<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
<meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
<meta name="categories" content="prime">
<meta name="product" content="Prime">
<title>Prime Structure Prediction Menu Bar</title>
<link rel="stylesheet" type="text/css" href="../support/help.css">
</head>
<script type="text/javascript">
function setTitle()
{
top.document.title = document.title + " - " + parent.parent.WINDOW_TITLE;
}
</script>
<body onload="setTitle();">
<table border=0 cellspacing=0 bgcolor=#dcdcdc width=100%>
<tr><td>
<p><img src="../support/schrodinger_logo.gif" border=0 alt="" align="left" vspace=5 hspace=5 /></p>
</td></tr>
<tr><td>
<h1 class=title><span class="prime">Prime Structure Prediction Menu Bar</span></h1>
</td></tr>
</table>

<ul>
<li><a href="#PspMenu1">Summary</a></li>
<li><a href="#PspMenu2">The File Menu</a></li>
<li><a href="#PspMenu3">The Edit Menu</a></li>
<li><a href="#PspMenu4">The Display Menu</a></li>
<li><a href="#PspMenu5">The Step Menu</a></li>
<li><a href="#PspMenu9">Related Topics</a></li>
</ul>

<a name="PspMenu1"></a>
<h2>Summary</h2>

<p>The Prime Structure Prediction menu bar contains four menus,
<span class="GUI">File</span>,
<span class="GUI">Edit</span>,
<span class="GUI">Display</span>,
and <span class="GUI">Step</span>.
Some of the options in these menus
are also available as toolbar buttons.</p>

<a name="PspMenu2"></a>
<h2>The File Menu</h2>

<p>The <span class="GUI">File</span> menu contains commands for creating,
naming, opening, and deleting Prime runs.  (Also see <a
href="info_runs.html">Using Prime Runs</a>.)  The other menu options are <span
class="GUI">Export Sequences</span> and the <span class="GUI">Job Options</span>
command, which allows you to designate a host machine and a user name for each
type of Prime job.</p>

<p>The run commands are:</p>

<dl>
<dt><span class="GUI">New</span> </dt>
<dd><p>Create a new run. You are asked for a name (all names must be unique.)
Prime starts the new run at the first step.</p></dd>

<dt><span class="GUI">Save As</span></dt>
<dd><p>Save the current state and provide a new run name.
Prime leaves the existing run
and switches to an identical run with the new run name.</p></dd>

<dt><span class="GUI">Rename</span></dt>
<dd><p>Specify a new name for the current run.
If the run name is not unique, a warning message appears.</p></dd>

<dt><span class="GUI">Delete</span></dt>
<dd><p>Delete the current run. You will be asked for confirmation.</p></dd>

<dt><span class="GUI">Open</span></dt>
<dd><p>View a list of options for switching to another run.
The list includes the four most recently used runs. If there
are more than four runs in the current project,
click the <span class="GUI">More</span> option
and select the run you want to open from the dialog
box.</p></dd>
</dl>

<p>The other options are:</p>
<dl>
<dt><span class="GUI">Export Sequences</span></dt>
<dd><p>Click to open the <span class="GUI">Export Sequences</span>
panel.  All sequences (including secondary structure assignments
and predictions) currently visible in the Sequence Viewer
will be exported to a single file in native Maestro format.
If HMMER/Pfam, ssa, or ssp data is currently hidden under
the parent sequence it describes, only the parent sequence
is exported.</p>
</dd>

<dt><span class="GUI">Job Options</span></dt>
<dd><p>Click this option to open the <span class="GUI">Job Options</span>
panel, which is described in the <a href="info_jobs.html">Prime Job Options</a> 
topic.
</p></dd>
</dl>


<a name="PspMenu3"></a>
<h2>The Edit Menu</h2>
<p>The Edit menu contains commands for editing sequences and secondary
structure predictions. These editing tools are also available
as buttons on the Prime toolbar, and are described in the
<a href="toolbar.html">Prime Toolbar</a> topic:</p>

<p>
<span class="GUI">Select</span><br />
<span class="GUI">Crop</span><br />
<span class="GUI">Edit SSP</span><br />
<span class="GUI">Set Template Regions</span><br />
<span class="GUI">Slide Freely</span><br />
<span class="GUI">Slide As Block</span><br />
<span class="GUI">Add Anchors</span><br />
<span class="GUI">Lock Gaps</span><br />
<span class="GUI">Unlock Gaps</span>
</p>

<a name="PspMenu4"></a>
<h2>The Display Menu</h2>

<p>The <span class="GUI">Display</span> menu contains the following
options governing the visualization of structures and sequences.
Color schemes are described briefly here;
use the <span class="GUI">Legend</span> command in this menu
to view the assignment of each color and
symbol used in each color scheme and sequence
representation.</p>

<ul>
<li><a href="#PspMenu4.1">Color Scheme</a></li>
<li><a href="#PspMenu4.2">Font Size</a></li>
<li><a href="#PspMenu4.3">View Structure</a></li>
<li><a href="#PspMenu4.4">View SSA</a></li>
<li><a href="#PspMenu4.4a">View Full PFAM Annotation</a></li>
<li><a href="#PspMenu4.5">Legend</a></li>
<li><a href="#PspMenu4.6">Preferences</a></li>
</ul>
<p></p>

<a name="PspMenu4.1"></a>
<h3>Color Scheme</h3>

<p>This command opens a menu of coloring options.
In Prime-SP, only one color scheme can be used at
any given time.
Each step in Prime has an appropriate default color scheme.
You may choose a different scheme using this menu.
When an amino acid sequence is added to
the Sequence Viewer, it is colored
according to the scheme that is currently selected.</p>

<p>When the structure associated with a sequence is displayed in the
Workspace, it is generally colored using the same scheme as the sequence.
However, protein structures in the Workspace are sometimes colored
using other color schemes:</p>

<ul>
<li><p>When a PDB structure is imported, it is displayed in the Workspace
using a Maestro error reporting color scheme
in which complete recognized residues are gray,
incomplete residues are red, unknown/nonstandard residues are orange,
residues with multiple location indicators are green,
and recognized residues with some unrecognized atoms are blue.
See the <a href="../misc/pdb.html#pdbc4">PDB Conversion</a>
topic section on Color-Based Error Reporting for more information.</p></li>

<li><p>At the end of the Prime workflow, when an output structure is added to
the Project table, it is colored to represent the quality of the model. The
final structural analysis provides information about potential problems:
clashes, bad bond lengths, and bad bond angles.  Each atom in the structure is
assigned a color, from blue (no problems) through gray to red (at least one
severe problem).  (This analysis can be viewed by using the
<a href="../tools_menu/protein_reports_panel.html">Protein Reports panel</a>.)</p></li>
</li> </ul>

The Prime color schemes are:<p>

<ul>
<li><a href="#PspMenu4.1.1">None</a></li>
<li><a href="#PspMenu4.1.2">Color by Sequence</a></li>
<li><a href="#PspMenu4.1.3">Residue Type</a></li>
<li><a href="#PspMenu4.1.4">Residue Property</a></li>
<li><a href="#PspMenu4.1.5">Residue Matching</a></li>
<li><a href="#PspMenu4.1.6">Residue Homology</a></li>
<li><a href="#PspMenu4.1.7">Secondary Structure</a></li>
<li><a href="#PspMenu4.1.8">Template ID</a></li>
<li><a href="#PspMenu4.1.9">Proximity</a></li>
</ul>

<dl>
<a name="PspMenu4.1.1"></a>
<dt>None </dt>
<dd><p>All sequences are colored standard gray.
</p></dd>

<a name="PspMenu4.1.2"></a>
<dt>Color by Sequence </dt>
<dd><p>The query sequence is colored standard gray.
Each of the other sequences uses a different color.
</p></dd>

<a name="PspMenu4.1.3"></a>
<dt>Residue Type </dt>
<dd><p>Each amino acid sequence is colored
according to the array of residue type colors.
</p></dd>

<a name="PspMenu4.1.4"></a>
<dt>Residue Property </dt>
<dd><p>Each amino acid sequence is colored
according to the array of residue property colors
(fewer colors than residue type).
</p></dd>

<a name="PspMenu4.1.5"></a>
<dt>Residue Matching </dt>
<dd><p>The query and templates are colored in a pairwise manner.
Matching residues are colored according to their residue type,
and non-matching residues are colored standard gray.
If more than one template is chosen,
the query will have more colored residues because the effect is cumulative.
When only the query sequence is shown, it is colored standard gray for all residues.
</p></dd>

<a name="PspMenu4.1.6"></a>
<dt>Residue Homology</dt>
<dd><p>Aligned query and template residues that are identical
(highly conserved) are colored red. Aligned query and template residues that
are similar according to the BLOSUM62 scoring matrix (conserved)
are colored yellow. Gray is used for residues with no homology.
</p></dd>

<a name="PspMenu4.1.7"></a>
<dt>Secondary Structure </dt>
<dd><p>The query sequence, which has no secondary structure
assignment (SSA), is colored standard gray.
All other amino acid sequences should have SSAs.
Secondary structure assignments are displayed as
a linear series of
red cylinders (alpha helices), cyan arrows (beta strands),
and gray line segments (loops/random coils.)
This option applies the same color scheme to the amino acid
sequences that have SSAs, and to the model in the Workspace.
</p></dd>

<a name="PspMenu4.1.8"></a>
<dt>Template ID </dt>
<dd><p>Only available for the Comparative Modeling path,
when a model is being built (in Build Structure)
or refined (in Refine Structure).
Before a structure is built, the query sequence is colored standard gray.
Afterward, both the query sequence and the predicted structure are colored
according to template ID for each region (using a cycle of assigned colors).
Each template sequence is colored with its assigned color,
but only in the region used to build the model.
The unused portion of the template is colored dark gray.
</p></dd>

<a name="PspMenu4.1.9"></a>
<dt>Proximity </dt>
<dd><p>At any step in which the Sequence Viewer is displaying sequences
that have associated structure (all steps except Input Sequence),
color by proximity mode is available. When you select this color scheme,
the current sequence colors do not change immediately.
Instead, when the mouse is over a sequence that has structure,
the cursor changes to indicate that proximity can be calculated.
You can then select one or more contiguous residues in the sequence,
and the system will color that sequence
based on the proximity to the selected residues.
No other sequences will have modified colors
(even if they have structure too and
are in proximity of the selected sequence).
Proximity is calculated based on the shortest distance
between any two atoms in two residues, not including the bonded C and N atoms
in the residues next to the selected residues.</p>

<p>To change the proximity cutoff, select <span class="GUI">Preferences</span>
from the Display menu and change the number in the <span class="GUI">Proximity
cutoff</span> text box.
</p></dd>
</dl>

<a name="PspMenu4.2"></a>
<h3>Font Size</h3>

<p>The font size of letters and numbers in the Sequence Viewer
can be changed by choosing a point size from this submenu. The default size is
10 points.</p>

<a name="PspMenu4.3"></a>
<h3>View Structure</h3>

<p>The options for showing structures in the Maestro Workspace include:</p>
<ul>
<li><span class="GUI">All</span></li>
<li><span class="GUI">None</span></li>
<li><span class="GUI">Predicted Only</span></li>
</ul>

<a name="PspMenu4.4"></a>
<h3>View SSA</h3>

<p>This option controls whether secondary structure assignments are shown in the
Sequence Viewer.</p>

<a name="PspMenu4.4"></a>
<h3>View Full PFAM Annotation</h3>

<p>This option controls whether the PFAM annotation shows only the highly
conserved residues, or shows other family annotations as well. In the full
annotation, capital letters indicate highly conserved residues, lowercase
letters indicate a match to the HMM, + means the match is conservative, and a
blank indicates that the residue does not match the HMM.</p>

<a name="PspMenu4.5"></a>
<h3>Legend</h3>

<p>This option opens the <span class="GUI">Sequence Viewer Legend</span>
panel, which allows you to view the assignment of each color and symbol used in
each color scheme and sequence representation.  It is also available when you
right-click on a sequence in the Sequence Viewer.</p>

<a name="PspMenu4.6"></a>
<h3>Preferences</h3>

<p>Select this option to open the <span class="GUI">Proximity</span> dialog box.
The value entered in this text box defines the proximity cutoff for the
<span class="GUI">Proximity</span> color scheme.  The default is
4.00&nbsp;&Aring;.</p>

<a name="PspMenu5"></a>
<h2>The Step Menu</h2>

<p>The <span class="GUI">Step</span> menu provides another way to navigate
between steps in Prime.  It includes the following options:</p>

<dl>
<dt><span class="GUI">Guide</span></dt>
<dd><p>When this check box is selected, the Prime Guide is displayed (the default).
Deselect the check box to hide the Guide.</p></dd>

<dt><span class="GUI">Input Sequence</span></dt>
<dd><p>Click this option to go to the Input Sequence step.</p></dd>

<dt><span class="GUI">Find Homologs</span></dt>
<dd><p>Click this option to go to the Find Homologs step.</p></dd>
</dl>

<p>There is an option to click to go to each step in Prime-SP.
A red diamond is displayed next to the option to go to the current step.
Options to go to steps that are not available appear dimmed.</p>

<a name="PspMenu9"></a>
<h2>Related Topics</h2>
<ul>
<li><a href="info_help.html">Prime General Help</a></li>
<li><a href="info_layout.html">Prime Panel Layout and Use</a></li>
<li><a href="info_runs.html">Using Prime Runs</a></li>
<li><a href="info_jobs.html">Prime Job Options and Results</a> </li>
<li><a href="toolbar.html">The Prime Toolbar</a></li>
</ul>

<hr />
<table width="100%">
<td><p class="small"><a href="../support/legal_notice.html" target="LegalNoticeWindow">Legal Notice</a></p></td>
<td>
<table align="right">
<td><p class="small">
File: prime/info_menu.html<br />
Last updated: 03 Aug 2012
</p></td>
</table>
</td>
</table>
</body>
</html>
