<!DOCTYPE html>
<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd" lang="en-US" xml:lang="en-US" data-mc-search-type="Stem" data-mc-help-system-file-name="Documentation.xml" data-mc-path-to-help-system="../" data-mc-target-type="WebHelp2" data-mc-runtime-file-type="Topic" data-mc-preload-images="false" data-mc-in-preview-mode="false" data-mc-toc-path="User Manuals|BioLuminate User Manual">
    <head>
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <meta charset="utf-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <link href="../Skins/Default/Stylesheets/Slideshow.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/TextEffects.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Topic.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Components/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" /><title>Homology Modeling of Proteins</title>
        <link href="../Resources/Stylesheets/docs.css" rel="stylesheet" />
        <script src="../Resources/Scripts/custom.modernizr.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/jquery.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.config.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/foundation.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/plugins.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/MadCapAll.js" type="text/javascript">
        </script>
    </head>
    <body><a name="aanchor730"></a>
        <MadCap:concept term="User Manuals" /><a name="aanchor731"></a>
        <MadCap:concept term="BioLuminate" /><a name="homology_modeling_1606914295_744641"></a><a name="homology_modeling_1606914295_98274"></a>
        <table class="NavLink" data-mc-conditions="Default.ScreenOnly">
            <tr class="NavLink">
                <td class="NavLeft">
                &#160;
        </td>
                <td class="NavCenter"><span class="NavText"><a class="NavLink MCXref xref xrefNavLink" href="bioluminate_user_manualTOC.htm">BioLuminate User Manual</a></span>
                </td>
                <td class="NavRight">
                &#160;
        </td>
            </tr>
        </table>
        <h1 class="Title">Homology Modeling of Proteins</h1>
        <p class="Tutorial" data-mc-autonum="For a tutorial example, see "><span class="autonumber"><span>For a tutorial example, see </span></span><a class="Tutorial MCXref xref xrefTutorial" href="../bioluminate_quick_start/homology_modeling_tut.htm#homology_modeling_tut_4015710100_98274">Simple Homology Modeling of Proteins</a>.</p>
        <p>If you have a protein sequence and want to build a homology model of the protein, there are three ways that you can proceed in the BioLuminate interface.</p>
        <p>Proteins for which you expect a high homology with the template and that require only a straightforward alignment to the template can be modeled in the <span class="GUI">Homology Model</span> panel. In the default mode, any missing loops are predicted using a curated database of known loops in the PDB. This approach is very fast and a full homology model can typically be generated using this panel in 2-5 minutes. To open this panel, choose <span class="GUI">Tasks → Biologics → Simple Homology Modeling</span> in the Task Tool. The use of this panel is described below.</p>
        <p class="image">
            <img src="../Resources/Images/bioluminate/homology_model_initial.png" style="margin: 2px;" />
        </p>
        <p>Proteins where the homology is not as high or where alignment of the template and the query is required can be modeled either in the <span class="GUI">Multiple Sequence Viewer</span> panel (opened with Tools → <span class="GUI">Multiple Sequence Viewer</span>), or with the <span class="GUI">Structure Prediction</span> panel (opened with <span class="GUI">Tasks</span> → <span class="GUI">Homology Modeling</span> → <span class="GUI">Advanced Homology Modeling</span>). For information on using these panels, see the <a href="../maestro_tools_help/multiple_sequence_viewer.html">Multiple Sequence Viewer Panel</a> document for the <span class="GUI">Multiple Sequence Viewer</span> and the <a class="Manual MCXref xref xrefManual" href="../prime_user_manual/prime_user_manualTOC.htm#prime_user_manualtoc_3112570731_64161">Prime User Manual</a> for the <span class="GUI">Structure Prediction</span> panel. </p>
        <p>If you are interested in homology modeling of antibodies, see <a class="TopicLink MCXref xref xrefTopicLink" href="antibody_modeling.htm#antibody_modeling_3917355573_741933">Antibody Modeling</a>.</p>
        <p>For a tutorial introduction to advanced homology modeling, see <a class="Manual MCXref xref xrefManual selected" href="#homology_modeling_1606914295_744641">Homology Modeling of Proteins</a>.</p>
        <p class="Task_LeadIn">To build a homology model:</p>
        <table class="AutoNumber_p_Numbered1" style="width: 100%; margin-left: 0;" cellspacing="0" cellpadding="0">
            <col style="width: 0px;" />
            <col style="width: 32px;" />
            <col style="width: auto;" />
            <tr>
                <td valign="top" />
                <td class="AutoNumber_p_Numbered1" valign="top"><span>1.</span>
                </td>
                <td class="AutoNumber_p_Numbered1" valign="top" data-mc-autonum="1.">Choose a source for the query (or reference) sequence using the <span class="GUI">Query</span> button. There are two choices: </td>
            </tr>
        </table>
        <ul>
            <li value="1">
                <p class="ListPara"><span class="GUI">From Workspace</span>—Use the sequence of the structure in the Workspace as the query.</p>
            </li>
            <li value="2">
                <p class="ListPara"><span class="GUI">Browse for File</span>—Opens a file selector so you can locate and import the sequence file for the query.</p>
            </li>
        </ul>
        <p class="Indented1">When you have chosen a query, the box to the right of the button displays the text <span class="GUI">( Query structure )</span> and the title of the query, if it has a title, or <span class="GUI">Query</span>, if no title is available.</p>
        <table class="AutoNumber_p_Numbered1_" style="width: 100%; margin-left: 0;" cellspacing="0" cellpadding="0">
            <col style="width: 0px;" />
            <col style="width: 32px;" />
            <col style="width: auto;" />
            <tr>
                <td valign="top" />
                <td class="AutoNumber_p_Numbered1" valign="top"><span>2.</span>
                </td>
                <td class="AutoNumber_p_Numbered1" valign="top" data-mc-autonum="2.">Choose a source for the template structure on which to build the model, using the <span class="GUI">Template</span> button.</td>
            </tr>
        </table>
        <p class="Indented1">You can either read in a template structure from a Maestro file or a PDB file (<span class="GUI">Browse for File</span>), or you can run a BLAST search, as follows:</p>
        <table class="AutoNumber_p_Lettered2" style="width: 100%; margin-left: 0;" cellspacing="0" cellpadding="0">
            <col style="width: 32px;" />
            <col style="width: 32px;" />
            <col style="width: auto;" />
            <tr>
                <td valign="top" />
                <td class="AutoNumber_p_Numbered1" valign="top"><span>a. </span>
                </td>
                <td class="AutoNumber_p_Numbered1" valign="top" data-mc-autonum="a. ">Choose <span class="GUI">Template</span> → <span class="GUI">BLAST Homology Search</span>.</td>
            </tr>
        </table>
        <table class="AutoNumber_p_Lettered2" style="width: 100%; margin-left: 0;" cellspacing="0" cellpadding="0">
            <col style="width: 32px;" />
            <col style="width: 32px;" />
            <col style="width: auto;" />
            <tr>
                <td valign="top" />
                <td class="AutoNumber_p_Numbered1" valign="top"><span>b. </span>
                </td>
                <td class="AutoNumber_p_Numbered1" valign="top" data-mc-autonum="b. ">Change settings if desired in the <span class="GUI">BLAST Search Settings</span> dialog box, and click <span class="GUI">Start Job</span>. </td>
            </tr>
        </table>
        <p class="Indented2">When the job finishes, the <span class="GUI">Job Progress</span> dialog box is replaced by the <span class="GUI">BLAST Search Results</span> dialog box. This dialog box lists the homologs found with various measures of the match: E-value, Score, Identity, Similarity, and Homology. </p>
        <table class="AutoNumber_p_Lettered2" style="width: 100%; margin-left: 0;" cellspacing="0" cellpadding="0">
            <col style="width: 32px;" />
            <col style="width: 32px;" />
            <col style="width: auto;" />
            <tr>
                <td valign="top" />
                <td class="AutoNumber_p_Numbered1" valign="top"><span>c. </span>
                </td>
                <td class="AutoNumber_p_Numbered1" valign="top" data-mc-autonum="c. ">Choose a template from the list of homologs, and click <span class="GUI">Select Template</span>.</td>
            </tr>
        </table>
        <p class="Indented2">Blast search results are listed in order of decreasing homology score. By default, the first structure in the list is chosen, and in most cases this will be the most appropriate selection.</p>
        <p class="Indented2">If you do not have a local installation of the BLAST or PDB databases, a warning is displayed by default: “Multiple Sequence Viewer is attempting to access a remote server. Would you like to continue?” If you have not already turned this warning off, Select <span class="GUI">Do not ask this question again</span>, to prevent it from being displayed each time a structure is downloaded, and click <span class="GUI">OK</span>. </p>
        <p class="Indented2">The template is selected and the <span class="GUI">BLAST Search Results</span> dialog box closes. </p>
        <p class="Graphics1">
            <img src="../Resources/Images/bioluminate/blast_search_results.png" style="margin: 2px;" />
        </p>
        <p class="Indented1">When you have chosen a template, the box to the right of the button displays the text <span class="GUI">(Template structure)</span> and the title of the template. The table rows in the <span class="GUI">Homology Model</span> panel are filled in with information on the template.</p>
        <p class="Graphics1">
            <img src="../Resources/Images/bioluminate/homology_model_template.png" style="margin: 2px;" />
        </p>
        <table class="AutoNumber_p_Numbered1_" style="width: 100%; margin-left: 0;" cellspacing="0" cellpadding="0">
            <col style="width: 0px;" />
            <col style="width: 32px;" />
            <col style="width: auto;" />
            <tr>
                <td valign="top" />
                <td class="AutoNumber_p_Numbered1" valign="top"><span>4.</span>
                </td>
                <td class="AutoNumber_p_Numbered1" valign="top" data-mc-autonum="4.">Enter a name in the <span class="GUI">Job name</span> text box and click <span class="GUI">Run</span>.</td>
            </tr>
        </table>
        <p class="Indented1">The job includes alignment of the template and the query using ClustalW, and the structure is built on the basis of the template and an analysis of structural elements in the PDB for non-templated regions (a “knowledge-based” selection of the coordinates). </p>
        <p>When the job finishes, the model is added to the Workspace, in cartoon representation. The cartoon is colored by how the template was used: dark blue for residues for which all coordinates were taken from the template, cyan for residues for which the backbone was taken from the template, and red for residues that were entirely modeled, not using the template. The title given to the model includes information on the query and the template.</p>
        <p>If the job fails, it is likely that there is insufficient homology or poor alignment between the reference and the template to build a model. In this case you should use the <span class="GUI">Advanced Homology Modeling</span> panel (<span class="GUI">Tasks</span> → <span class="GUI">Homology Modeling</span> → <span class="GUI">Advanced Homology Modeling</span>) or the <span class="GUI">Multiple Sequence Viewer</span> (<span class="GUI">Tools</span> → <span class="GUI">Multiple Sequence Viewer</span>).</p>
        <p>If you want to examine the quality of the model structure, click <span class="GUI">Examine Model Quality</span> to open the <span class="GUI">Protein Structure Quality Viewer</span> panel, where you can view reports on the protein structure, a Ramachandran plot, and plots of protein properties</p>
        <p>If you want to refine loops in the model, click <span class="GUI">Refine Loops</span>. The <span class="GUI">Refinement</span> panel opens with the <span class="GUI">Refine loops</span> task selected. You should only need to refine the loops if they were not predicted from the template. Click <span class="GUI">Non-Template</span> in the <span class="GUI">Refinement</span> panel to list only the loops that did not come from the template. See <a class="TopicLink MCXref xref xrefTopicLink" href="../prime_user_manual/refine_proteins.htm#refine_proteins_4294271795_13606">Refining Protein Structures</a> for information on this panel.</p>
        <table class="NavLink" data-mc-conditions="Default.ScreenOnly">
            <tr class="NavLink">
                <td class="NavLeft">
                &#160;
        </td>
                <td class="NavCenter"><span class="NavText"><a class="NavLink MCXref xref xrefNavLink" href="bioluminate_user_manualTOC.htm">BioLuminate User Manual</a></span>
                </td>
                <td class="NavRight">
                &#160;
        </td>
            </tr>
        </table>
    </body>
</html>