<!DOCTYPE html>
<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd" lang="en-US" xml:lang="en-US" data-mc-search-type="Stem" data-mc-help-system-file-name="Documentation.xml" data-mc-path-to-help-system="../" data-mc-target-type="WebHelp2" data-mc-runtime-file-type="Topic" data-mc-preload-images="false" data-mc-in-preview-mode="false" data-mc-toc-path="User Manuals|BioLuminate User Manual|Scanning for Residue Mutations|Using Homologs for Setting Up Mutations in Residue Scanning">
    <head>
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <meta charset="utf-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <link href="../Skins/Default/Stylesheets/Slideshow.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/TextEffects.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Topic.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Components/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" /><title>Selecting Residues by Homology for Residue Scanning</title>
        <link href="../Resources/Stylesheets/docs.css" rel="stylesheet" />
        <script src="../Resources/Scripts/custom.modernizr.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/jquery.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.config.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/foundation.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/plugins.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/MadCapAll.js" type="text/javascript">
        </script>
    </head>
    <body><a name="aanchor4577"></a>
        <MadCap:concept term="User Manuals" /><a name="aanchor4578"></a>
        <MadCap:concept term="BioLuminate" /><a name="residue_scanning_1293582990_777345"></a>
        <table class="NavLink" data-mc-conditions="Default.ScreenOnly">
            <tr class="NavLink">
                <td class="NavLeft"><span class="NavText">◀ <a class="NavLink MCXref xref xrefNavLink" href="residue_scanning_homologs_aligning.htm" xrefformat="{paratext}">Aligning Homologs for Residue Scanning</a></span>
                </td>
                <td class="NavCenter"><span class="NavText"><a class="NavLink MCXref xref xrefNavLink" href="bioluminate_user_manualTOC.htm" xrefformat="{paratext}">BioLuminate User Manual</a></span>
                </td>
                <td class="NavRight"><span class="NavText"><a class="NavLink MCXref xref xrefNavLink" href="residue_scanning_homologs_attributes.htm" xrefformat="{paratext}">Selecting Residues by Structural Attributes for Residue Scanning</a> ▶</span>
                </td>
            </tr>
        </table>
        <h1 class="Title">Selecting Residues by Homology for Residue Scanning</h1>
        <p>When the sequences are aligned, residues can be selected on the basis of the of the alignment. To choose residues to mutate, you may want to select residues at positions where the variability among the aligned homologs is high, or residues that vary in residue type, or residues that differ from most of the homologs. The <span class="GUI">Selection by homology criteria</span> section has a number of options that you can set to apply selection filters on the basis of the amount of variation. Each of the options that you select is applied, so the residues must meet all specified criteria.</p>
        <ul>
            <li value="1">
                <p class="ListPara"><span class="GUI">Variability at position &gt;/&lt; </span><i class="UserReplace">N</i><span class="GUI"> %</span>—Select residues based on the percentage variability at the residue position. The variability is the normalized Shannon entropy for residue variation at the position, expressed as a percentage. You can apply a minimum or a maximum variability by choosing from the option menu, and specify the cutoff in the text box.</p>
            </li>
            <li value="2">
                <p class="ListPara"><span class="GUI">Variability at position &gt;/&lt; </span><i class="UserReplace">N</i><span class="GUI"> residue types</span>—Select residues based on the residue type variability at the residue position. The variability is the number of residue types observed at the position. Choose whether to apply a minimum or a maximum variability from the option menu, and specify the threshold for the number of residue types that can vary in the text box.</p>
            </li>
            <li value="3">
                <p class="ListPara"><span class="GUI">Ignore positions with group conservations</span>—Ignore (do not select) residues that are strongly conserved or that are either strongly or weakly conserved. Select the appropriate option for strong conservation or both strong and weak conservation. </p>
                <p class="ListPara">Strong conservation means that all residues at a particular position are in one of the following groups: STA, NEQK, NHQK, NDEQ, QHRK, MILV, MILF, HY, FYW. Weak conservation means that all residues at a particular position are in one of the following groups: CSA, ATV, SAG, STNK, STPA, SGND, SNDEQK, NDEQHK, NEQHRK, FVLIM, HFY. These definitions are those used by ClustalW.</p>
            </li>
            <li value="4">
                <p class="ListPara"><span class="GUI">Ignore parent sequence in applying above criteria</span>—Ignore the parent (query) sequence when applying the variability or conservation criteria. The selection is then based on the variability in the homologs only. </p>
            </li>
            <li value="5">
                <p class="ListPara"><span class="GUI">Parent residue different than </span><i class="UserReplace">N</i><span class="GUI"> % homologs at position</span>—Select residues for which the parent residue is different from more than the specified percentage of homologs at the residue position. </p>
            </li>
        </ul>
        <p>When residues are selected on the basis of homology, a default set of mutations is also defined. The mutations for a given residue include all the variants found at that residue position.</p>
        <table class="NavLink" data-mc-conditions="Default.ScreenOnly">
            <tr class="NavLink">
                <td class="NavLeft"><span class="NavText">◀ <a class="NavLink MCXref xref xrefNavLink" href="residue_scanning_homologs_aligning.htm" xrefformat="{paratext}">Aligning Homologs for Residue Scanning</a></span>
                </td>
                <td class="NavCenter"><span class="NavText"><a class="NavLink MCXref xref xrefNavLink" href="bioluminate_user_manualTOC.htm" xrefformat="{paratext}">BioLuminate User Manual</a></span>
                </td>
                <td class="NavRight"><span class="NavText"><a class="NavLink MCXref xref xrefNavLink" href="residue_scanning_homologs_attributes.htm" xrefformat="{paratext}">Selecting Residues by Structural Attributes for Residue Scanning</a> ▶</span>
                </td>
            </tr>
        </table>
    </body>
</html>