<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--	hom_tut.htm
!!-->

<!--
!!    MOE Online Manuals
!!    COPYRIGHT (C) 1997-2015
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<head>
    
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

<script type="text/javascript" 
src="../include/jsincludes_moe.js"></script>

<link rel="stylesheet" type="text/css"
href="../include/manstyle.css" />


<title>Homology Model Tutorial</title>
</head>

<body>
<div id="MoeHeader"></div>
<noscript>
	<span class="warning">Warning: JavaScript is disabled. This page will not display correctly.</span>
	<h1 class="title">MOE Documentation</h1>
	<hr noshade="noshade" />
	<style>.LaTeX {color: #C08080;}</style>
</noscript>

<p>In this tutorial, we will introduce MOE's Protein Modeling applications.
The tutorial has been broken down into the following steps:</p>

<ul>
<li><a href="#SearchingCluster">Performing a Homology Search</a></li>
<li><a href="#BuildingModel">Building a Homology Model</a></li>
<li><a href="#ComparingModel">Comparing the Homology Model with the X-ray 
Structure</a></li>
<li><a href="#EvaluatingModel">Evaluating the Homology Model</a></li>
</ul>

<p>These MOE applications will be used:</p>

<table cellpadding="3" width="80%" style="margin-left:3em;">
    <tr>
    <td valign="top"><u>Application</u></td>
    <td valign="top"><u>Functions</u></td>
    </tr>
    <tr>
    <td valign="top"><a href="../proteins/prosearch.htm">MOE-SearchPDB</a></td>
    <td valign="top">Searches the protein family database for
	protein structures homologous to a query sequence.</td>
    </tr>
    <tr>
    <td valign="top"><a href="../proteins/align.htm">Protein Align</a></td>
    <td valign="top">Aligns protein structures and/or sequences based 
	on sequence and 3D features.</td>
    </tr>
    <tr>
    <td valign="top"><a href="../proteins/promodel.htm">Homology Model</a></td>
    <td valign="top">Builds homology models for a sequence from the
	structures of one or more aligned proteins.</td>
    </tr>
    <tr>
    <td valign="top"><a href="../apps/progeom.htm">Protein Geometry</a></td>
    <td valign="top">Calculates features of protein structure and
	geometry useful in quality and error analysis.</td>
    </tr>
</table>

<h3>Conventions Used in this Tutorial</h3>

<table cellpadding="3">
<tr> <td valign="top" bgcolor="#cccccc">MOE</td>
<td valign="top">MOE Window</td>
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">MOE | RHS</td>
<td valign="top">Right button bar of the MOE Window</td>
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">MOE | Footer</td>
<td valign="top">MOE footer button bar</td>
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">SE</td>
<td valign="top">Sequence Editor</td>
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">SE | Footer</td>
<td valign="top">Sequence Editor footer bar</td>
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">DBV</td>
<td valign="top">Database Viewer</td>
</tr>
</table>

<p>In this tutorial, we will assume that you are using a three-button mouse.
For information on using a two-button mouse see
<a href="../moe/gui/mouse.htm">Using the Mouse</a>.</p>

<p>We will build a model for PDB entry <tt>4MUZ</tt>, which is a
1.39&Aring; X-ray structure. It was released after the cutoff date for
structures in the <span class="nobr">MOE 2013.08</span> non-redundant
chain database in <tt>$MOE/lib/pdb.mdb</tt>.</p>

<h3><a name="SearchingCluster">Performing a Homology Search</a></h3>

<ol>
<p><li> Close the current system using </p>
<blockquote>
<span class="menu">MOE | File | Close</span>
</blockquote>
</li>

<li><p>Open the file containing the protein sequence that we will model.
The sequence for this protein is in the MOE sample directory:</p>

<blockquote>
    <span class="menu">MOE | File | Open  | 
	<tt>$MOE/sample/mol/4muz.fst</tt></span>
</blockquote>
</li>

<li><p>We will now use the <b>MOE-SearchPDB</b> application to search for a suitable template for homology modeling. The search is performed on a database of
protein structures and sequences that have been clustered into families.
Open the panel with</p>
<blockquote>
<span class="menu">MOE | Protein | Search | PDB</span>
</blockquote>

<p>Select chain #1 from the <b>Load Chain</b> option list
at the top right of the panel to load the <tt>4MUZ.A</tt> sequence into 
the search area and press <b>Search</b>. After starting the search, delete
chain 1 in the Sequence Editor by right-clicking on Chain 1 and selecting Delete
 &mdash; we will be loading an aligned version of the query after
the search.</p>

<p>As the search progresses, pre-aligned families appear in the Results section
of the panel with their calculated E-values. Once the screening of E-values is
complete, the Z-Scores are calculated and the Z-Score threshold applied. Note
that if the E-value is sufficiently low, the Z-Score is not calculated
as it presumably will be large.
(See <a href="../proteins/prosearch.htm">Homology Searching</a>
for more details.)</p>

<p align="center">
<img src="hom_tut/spanel.png" alt="MOE-SearchPDB" />
</p>

<p>In this example, the search identifies OMP decarboxylase
as a homolog of the target sequence.
Each item in the Results section of the panel represents one
or more PDB structures, possibly augmented by sequence-only entries from
the Uniref50 database, clustered into a family and aligned by a sequence
and structure based alignment protocol. The displayed code represents the 
PDB structure from within the family with the highest similarity score
to the target sequence.</li></p>

<p><li>Double-click on <tt>PDB_4LUI.B</tt> in the <b>Results</b> area,
or select the result and press the 
<span class="menu">Load Alignment...</span> button. The
<span class="menu">MOE-SearchPDB: Load Alignment</span> panel appears, 
displaying the pre-aligned family, with the chains sorted by sequence
similarity to the query sequence. Note that the PDB structure 
<tt>4LUI.B</tt>, which was displayed in the Search results area, is 
one of 13 PDB structures in this family.  </p>

<p align="center">
<img src="hom_tut/ldalign.png" alt="MOE-SearchPDB: Load Alignment" />
</li></p>

<p><li>Select the <span class="menu">Load Query Sequence</span> checkbox and
press <span class="menu">Load All</span> to load these chains into MOE. 
You may now close the <span class="menu">MOE-SearchPDB</span> panel.</p>

<p>In the MOE window, press <span class="menu">SEQ</span> to open the
Sequence Editor, you now see the query sequence (chain 1) as well
as the five chains from the homologous proteins. To annotate these with
the secondary structure, on <span class="menu">SE | Footer</span>, click 
<img src="hom_tut/secstruct.png" alt="Show Secondary Structure for Sequences"/> 
</p>


<p>To display the sequences with single letter codes, click
<img src="hom_tut/single.png" alt="Show Single Letter Sequences in the Editor"
/></p>

<p>To wrap the sequences to fit into the window, click
<img src="hom_tut/seqwrap.png" alt="Wrap Sequences in the Sequence Editor" />
</p>

<p align="center">
<img src="hom_tut/alignsec.png" alt="Secondary Structures" />
</p>

<blockquote>
<span class="tip"><b>Tip!</b></span> 
The colored bars above the sequences reflect the secondary structure of
the atoms in those residues having associated atomic coordinates.  
Chain 1 is a <i>sequence-only</i> chain, and therefore has no such bars.
</blockquote>

</li>
</ol>


<h3><a name="BuildingModel">Building a Homology Model</a></h3>

<p>In order to build a homology model, we will align the target sequence to the
homologous structures, decide which chain is to be the template for the model,
and then build the model.</p>
<ol>
<li><p>We will use the <span class="menu">Protein Similarity Monitor</span>
to compare the sequences. Open the panel with:</p>
    
<blockquote>
<span class="menu">SE | Alignment | Similarity</span>
</blockquote>

<p align="center">
<img src="hom_tut/simmon.png" alt="Protein Similarity Monitor" /></p>


<p>The sequence identity matrix is non-symmetric.  The percentage
sequence identity is calculated from the number of identical residues
between both chains, divided by the number of amino acids in the chain
corresponding to the cell column.
</p>

<p>The matrix shows that the <tt>4LUI.B</tt>, <tt>1DVJ.A</tt> and
<tt>2CZD.A</tt> chains
are much closer to the query sequence than the other ten chains.</li></p>

<p><li>Select chain 5 with the mouse, and then hold the <tt>&lt;Shift&gt;</tt>
key and select chain 14. Then press the keyboard 
<b>&lt;Delete&gt;</b> key and click <b>OK</b> to delete these chains.</p>

<p align="center">
<img src="hom_tut/deletesequence.png" 
alt="Delete Chains in the Sequence Editor"/> 
</li></p>

<p><li>From the Sequence Editor, we can raise the window for the
<span class="menu">Protein Similarity Monitor</span> using:</p>

    
<blockquote>
<span class="menu">SE | Alignment | Similarity</span>
</blockquote>

<p>Click the RMSD button in the
<span class="menu">Protein Similarity Monitor</span>.</p>

<p align="center">
<img src="hom_tut/simmonrmsd.png" 
alt="Protein Similarity Monitor RMSD" /></p>

<p>This shows the very large RMSDs  
in the top right half of the matrix.
The RMSDs are high because <b>MOE-SearchPDB</b> returns the structures with the
coordinates from the original PDB files. The sequences have been aligned
but the structures have not been superposed.  The bottom left half of the
matrix shows the RMSD for residues where there are atoms for all of the
chains.  The sequence-only query chain is included in these scores, so
no RMSD values are shown here.</li></p>

<p><li>We will use the <b>MOE-Align</b> application to align the target 
sequence to the other chains, and then to superpose the chains.  
Start <b>MOE-Align</b> with</p>
<blockquote>
<span class="menu">SE | Alignment | Align</span>
</blockquote>

<p align="center">
<img src="hom_tut/align.png" alt="MOE-Align"/></p>

<p>Press <b>OK</b> to perform the alignment. Note that the 
<span class="menu">Protein Similarity Monitor</span>
updates automatically to show the new RMSDs between the 
chains.</li></p>

<p><li>In this case, we are most interested in the residues which differ
between the sequences, so we can highlight residues in the sequence editor
which are not conserved between all the chains. Click 
<span class="menu">SE | Footer | Consensus</span>:
<img src="hom_tut/consensus.png" alt="Show Consensus Settings"/> 
</p>

<p>Select the <span class="menu">Similarity: * for Conserved Residues</span> 
check box:</p>

<p align="center">
<img src="hom_tut/hideconserved.png" alt="Show * for Conserved Residues"/> 
</li></p>

<p><li>Color the residues by the RMS deviation between the atoms using
the Residue <b>RMSD</b> button from the Sequence Editor footer:</p>
<blockquote>
<span class="menu">SE | Footer | Residues </span>
</blockquote>
<p align="center">
<img src="hom_tut/rescolor.png" alt="Color Residues" /></p>

<blockquote>
<span class="tip"><b>Tip!</b></span> Residues with no atoms, like in the 
sequence-only query chain, are not colored with the RMSD mode.
</blockquote>

<p align="center">
<img src="hom_tut/identseq.png" alt="Sequence identity"
/></p>

<p>Review the aligned sequences by scrolling down in the sequence editor.
Note that most of the residues in the three protein chains
are colored green, showing that they have similar conformations
to each other.
</p>

<blockquote>
<span class="tip"><b>Tip!</b></span>
It is good practice to check the positions of the mismatched residues
in the structure.  It may be best to choose a template structure
with a lower overall sequence identity, if more of the conserved 
residues are near a ligand binding site, or where the residues
pack into the middle of the protein.  Mismatches can be accommodated
more easily in surface loops, which are more flexible, and have less
steric hindrance.
</blockquote>

<!--
<p><tt>1F2V.A</tt> and <tt>3E7D.C</tt> match the query sequence in
134 and 128 positions respectively.  The query sequence has one 
extra residue at position 173 in the sequence alignment compared
to the two template structures, and one less residue at position
199 in the alignment.  There is 75% sequence identity between
the query sequence and the template structures between these two
gaps, which suggests that the sequences should not be aligned by
removing both of the gaps.  
</p>
!!-->
</li>

<li><p>Build the homology model using <tt>4LUI.B</tt> 
as the template structure.
Open <span class="menu">Homology Model</span> with</p>
<blockquote>
<span class="menu">SE | Protein | Homology Model</span>
</blockquote>

<p align="center">
<img src="hom_tut/hompanel.png" alt="MOE Homology Model" /></li></p>

<p><li>There is only one sequence loaded in MOE with no associated atoms
(Chain 1); it is therefore specified by default as the sequence to
be modeled.  The <b>Template</b> will default to the first chain in the
system where there are atomic coordinates: Chain 2 (<tt>4LUI.B</tt>).</p>

<p>Select <span class="menu">Open Database Viewer</span> near the top
of the <span class="menu">Homology Model</span> panel.</p> </li>

<li><p>The currently loaded forcefield is displayed at the bottom of the
panel. By default, MOE uses a version of the Merck Molecular Forcefield
[Halgren], however, this forcefield is not specifically designed for
protein modeling. Press <span class="menu">Potential
Setup</span> to select a more appropriate forcefield. We suggest <b>Amber12:EHT</b>
[Case 2012, Gerber 1995] with a Reaction Field (<b>R&ndash;Field</b>)
treatment of solvation electrostatics [Tironi 1995] for this exercise
(<b>Solvation: R-Field</b>).</p>

<p>The <span class="menu">Fix Hydrogens</span> and
<span class="menu">Fix Charges</span> buttons
can be ignored.  <b>Homology Model</b> will add or remove hydrogens and
lone pairs as necessary, and will assign partial charges to all atoms.</p>

<p>Finally, press <b>OK</b> to apply the changes and close the <span
class="menu">Potential Setup</span> window.</p>

<p align="center">
<img src="hom_tut/amber12eht.png" alt="Potential Setup" /></li></p>

<p><li>In the Homology Modeling panel press <b>OK</b>. In the output
database, <tt>promodel.mdb</tt>, you can view the results as they are
being calculated.</p> </li> </ol>

<p><b>Notes:</b></p>
<ul>

<p><li>The entire system is saved prior to modeling as a MOE file
specified in the <span class="menu">Current System</span> field.
The default filename is <tt>promodel.moe</tt>.</li></p>

<p><li>The various models will be written to the database specified in the
<span class="menu">Output Database</span> field.  The default filename is 
<tt>promodel.mdb</tt>.</li></p>

<p><li>The <span class="menu">Disable C-terminal &amp; N-terminal
Outgap Modeling</span> option is, by default, turned on.  When outgaps
are ignored, the amino acids of the target sequence that extend beyond
(either before or after) the template sequence in the alignment are
clipped off before the model is built.</li></p>

<p><li>By default, ten independent intermediate models will be built.
These different homology models will be the result of the permutational
selection of different loop candidates and sidechain rotamers.</li></p>

<p><li>The intermediate model which scores best according to
the chosen scoring function is chosen as the final model,  
subject to optional further energy minimization.</li></p>

<p><li>Once the calculations have finished, the output database will
include the ten intermediate models and a refined final model (entry
eleven).</li></p>
</ul>

<p>To get a better view of the molecules in the database, you can enlarge
the molecular cells in the Database Viewer.  Position the cursor over
any one of the cells in the first column (the <b>mol</b> field), press
the left mouse button and drag down and to the right, to enlarge the
molecular drawings. Click the middle mouse button over one of these and
drag the mouse to rotate the molecule.</p>

<p align="center">
<img src="hom_tut/db_hom.png" alt="Homology Models in Database Viewer" />
</p>

<h3><a name="ComparingModel">Comparing the Homology Model with the X-ray
Structure</a></h3>

<ol>
<p><li>When the calculation finishes, the final,
refined model will be loaded in the MOE window.  Open
the PDB file with the X-ray structure for <tt>4MUZ</tt>:</p>

<blockquote>
<span class="menu">SE | File | Open  | <tt>$MOE/sample/mol/4muz.pdb</tt></span>
</blockquote>

<p>This will bring up the <span class="menu">Load PDB File</span> panel. 
Keep the default options and press <b>OK</b>.</p>

<p>The <tt>4MUZ</tt> structure is a monomer, with a chain with a ligand
and sulfate ion, and another chain with water molecules.  </li></p>

<p><li>To compare the homology model with the X-ray
structure, we will use <span class="menu">Protein Superpose</span>:</p>

<blockquote>
<span class="menu">SE | Alignment | Superpose</span> 
</blockquote>
<p>Press <b>Superpose</b>.</p> 

<p align="center">
<img src="hom_tut/superpose.png" alt="Protein Superpose" />
</p>

<p>Press <b>Yes</b> when prompted to realign the sequences and 
<b>Close</b> the <span class="menu">Protein Superpose</span> panel.</li></p>

<p><li>In the Sequence Editor, click the <b>-</b> button to the left
of the Sequence Editor ruler to hide the chains with solvent molecules.
Open the <span class="menu">SE | Footer | Consensus</span> panel and
select the <span class="menu">Plot: RMSD</span> check box and
deselect the <span class="menu">Similarity: * for Conserved Residues</span> 
check box:</p>

<p align="center">
<img src="hom_tut/seqcompare.png" alt="Compare the Sequences" />
</li></p>

<p><li>Turn on the Selection Synchronize option, so that selecting
residues in the Sequence Editor will select the atoms in those
residues in the MOE window, and selecting atoms will select the
residues.
</p>
<blockquote>
<span class="menu">SE | Select | Synchronize</span> 
</blockquote>
</li>

<li><p>Hide the atoms, center the view point and show the 
protein backbone ribbon using the chain color:</p>
<blockquote>
<span class="menu">SE | Footer | Atoms </span> 
<img src="hom_tut/hideatoms.png" alt="Hide Atoms"/> 
</blockquote>
<blockquote>
<span class="menu">SE | Footer | Ribbon </span> 
<img src="hom_tut/ribbon.png" alt="Ribbons"/> 
</blockquote>
<p>to render the final model as shown here. The homology model
is shown in orange. The X-ray structure is magenta.</p>

<p align="center">
<img src="hom_tut/final.png" alt="Final Model" />
</li></p>

<p><li>Scroll down the Sequence Editor.  The RMSD plot shows
that the model is reasonably close to the X-ray structure for most
of the residues.  The major differences are near the gaps between
the query sequence and the template structure.</li></p>
</ol>

<h3><a name="EvaluatingModel">Evaluating the Homology Model</a></h3>

<p>It is good practice to examine the geometry of homology
models for unusual or unreasonable features. Serious
problems - particularly ones that cluster in one or a few
areas of the model - can suggest a either a problem with the choice
or template, or a problem with the alignment of
the target sequence to the template. </p>

<ol><p><li>In the Sequence Editor, select chain 1</p>
<p align="center">
<img src="hom_tut/chainselect.png" alt="Select Chain 1" /></li></p>
<p><li>Investigate the geometric quality of the model using
<span class="menu">Protein Geometry</span> from </p>
<blockquote>
<span class="menu">SE | Protein | Geometry | Phi-Psi Plot</span> 
</blockquote>
</li>

<li><p>From <b>Residues</b>, choose <span class="menu">Selected Chains</span>
</p>

<p align="center">
<img src="hom_tut/progeom.png" alt="Protein Geometry" /></li></p>
</ol>

<p>The user interface of the <span class="menu">Protein Geometry</span>
application presents
data in the form of plots, such as the Ramachandran plot shown above,
and as tabular lists. MOE uses stochastic algorithms in its homology modeling,
and results may differ slightly each time the model is constructed.
Therefore, your results and the results shown here may not be identical.
</em>
</p>


Click on <b>Display: Data</b> and select the outlier(s) in the list, 
and press <span class="menu">Select
Atoms</span> at the bottom of the panel. Now, with the atoms of the
outlier residue selected, you may perform operations such as a fine
grain energy minimization in an attempt to obtain more typical phi and
psi angles.</p>

<h2>Further Work</h2>
<p>To improve a homology model:</p>
<ul>
<p><li>Check to see if new protein structures have been released which 
are similar to the target sequence.</li></p>
<p><li>Adjust the position of gaps in the sequence alignment 
between the query sequence and the template structure, and
build models.  </li></p>
<p><li>Consider the flexibility of the protein.  Residues in surface
loops and at the chain termini are likely to be much more flexible
than residues in a helix or strand secondary structure, in the 
core of a domain.</li></p>
<p><li>Explore loop conformations with the
<a href="../apps/proloop.htm">Loop Modeler</a>.</li></p>
</ul>

<h2>Summary</h2>

<p>
In this tutorial, we have shown how to use the protein modeling tools in MOE,
searching with a protein amino acid sequence for a structure 
from a similar protein.  <b>MOE-Align</b> was used to refine the alignment 
between the sequence and the template.  A homology model was built using
<b>MOE-Homology</b>, and the geometry of the model was checked using the
<b>Protein Geometry</b> application.</p>

<h2>References</h2>

<table cellpadding="5">

<tr>
<td valign="top"><span class="nobr">[Case 2012]</span></td>
<td>Case, D.A., <i>et al.</i>; AMBER 12; University of California,
San Francisco (<b>2012</b>).
</td>
</tr>
<tr>
<td valign="top">[Gerber&nbsp;1995]</td>
<td>Gerber, P.R., M&uuml;ller, K.;
MAB, a generally applicable molecular force field for
structure modelling in medicinal chemistry;
<i>J. Comput. Aided Mol. Des. 9</i> (<b>1995</b>) 251&ndash;68.
</td>
</tr>
<tr>
<td valign="top"><span class="nobr">[Halgren 1996]</span>
</td>
<td>
Halgren, T.A.; The Merck Force Field;
<i>J. Comput. Chem. 17</i> (<b>1996</b>) 490&ndash;641.
</td>
</tr>
<tr>
<td valign="top"><span class="nobr">[Tironi 1995]</span>
</td>
<td>
Tironi, I.G., Sperb, R., Smith, P.E., Van Gunsteren, W.F.;
A generalized reaction field method for molecular dynamics simulations;
<i>J. Chem. Phys. 102</i> (<b>1995</b>) 5451&ndash;5459.
</td>
</tr>
</table>

<h2>See Also</h2>

<a href="../proteins/prosearch.htm">Homology Searching</a><br />
<a href="../proteins/align.htm">Sequence and Structure Alignment</a><br />
<a href="../proteins/promodel.htm">Building Homology Models</a><br />
<a href="../apps/progeom.htm">Protein Geometry</a><br />
<p></p>
<a href="../moe/pot.htm">Potential Energy Selection and Configuration</a><br />
<p></p>
<a href="../moe/gui/mainwin.htm">MOE Window</a><br />
<a href="../mdb/db.htm">Database Viewer Window</a><br />
<a href="../moe/gui/se.htm">Sequence Editor Window</a><br />


<noscript>
	<hr noshade="noshade" />
	<font size="2"> Copyright &copy; 1997&ndash;2015
	<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a> </font>
</noscript>
<div id="MoeFooter"></div>
</body>
</html>
