<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<!--
!!	moeconfig.htm		MOE Configuration Options chapter
!!
!!	05-nov-2010 (al) added LigX
!!	27-oct-2010 (al) changed to PSILO/RCSB Connectivity
!!	20-oct-2010 (jd) added Builder page, Chain Tag in SeqEdit, Kinome DB
!!	27-sep-2010 (hf) added PSILO
!!	28-oct-2008 (mk) fix examples
!!	28-oct-2008 (mk) removing visualization setup panel documentation
!!	24-sep-2008 (kk) copy edit
!!	20-sep-2008 (al) custom directory
!!-->



<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../manstyle.css" />


<title>Configuring and Customizing MOE</title>

<meta panel Configuration Options>

<meta keywords custom SVL files, resource configuration, user menu />
<meta keywords MOE_SVL_LOAD, MOE_SVL_RUNPATH, command line switch />
<meta keywords menu configuration, startup, master definition, redefine menu />
<meta keywords override standard, PREPEND, APPEND, ENDMENU />
<meta keywords keyboard accelerator, accel, ReadMenuFile, submenu, menu bar />
<meta keywords popup, button bar, function keys, JDBC, PSILO, RCSB />
<meta keywords mouse sensitivity, assign action, database viewer />
<meta keywords sequence editor, protein defaults />
<meta keywords LigX />

</head>

<body bgcolor="#ffffff">
<a href="../index.htm"><img src="../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>Configuring and Customizing MOE</h2>
<hr noshade="noshade" />

<ul>
<li><a href="#CustomSVL">Custom SVL Files</a>
<li><a href="#Menus">Menus</a>
<li><a href="#SVLPrograms">SVL Programs</a>
<li><a href="#FKeys">Function Keys</a>
<li><a href="#Builder">Builder</a>
<li><a href="#LigX">LigX</a>
<li><a href="#JDBC">JDBC Database Connections</a>
<li><a href="#PSILO">PSILO/RCSB Connectivity</a>
<li><a href="#MainMouse">Mouse (MOE Window)</a>
<li><a href="#DatabaseViewer">Database Viewer</a>
<li><a href="#SequenceEditor">Sequence Editor</a></li>
<li><a href="#ProteinDefaults">Protein Defaults</a></li>
<li><a href="#Miscellaneous">Miscellaneous</a></li>
</ul>

<p>Many aspects of the MOE interface can be configured and customized.
Many customizations are performed through the <b>Configuration&nbsp;Options</b>
panel, which uses an RC (resource configuration) file, <tt>$HOME/.moe-rc</tt>,
to save its settings.  The MOE menus and button bars are read from MOE menu
files, and can be modified using user menu files.  The standard SVL files
in the distribution can be augmented by custom user files.
There are also a variety of
<a href="../install/install.htm#RunSwitch">command line switches</a>
which can be used to customize a particular MOE session.
Furthermore, if the
<a class="svl" href="../install/install.htm#MOE_SVL_RUN">MOE_SVL_RUN</a>
or
<a class="svl" href="../install/install.htm#MOE_SVL_STARTSCRIPT">MOE_SVL_STARTSCRIPT</a>
environment variable has been set, the SVL source file or SVL script
file being pointed to will be run/executed when MOE is started.
</p>

<p><b>Note:</b>
The user resource configuration file is rewritten each time a MOE
session ends normally. Any changes made during the session are saved at
this time. This means that if you wish to edit your configuration file manually,
you must do so <b>after</b> closing MOE to
avoid losing any changes. Note that comments in a user configuration file
are not saved.</p>

<a name="CustomSVL"><h2>Custom SVL Files</h2></a>

<p>Custom SVL files can be automatically loaded on MOE start-up.
MOE automatically loads SVL files in the following order.
If the same filename is found in different directories,
the file that is loaded last is the one that takes precedence.
</p>

<ol>

<li>Standard MOE distribution source files, from <tt>$MOE/lib/svl</tt>.
</li>

<li>Patches (if any) to the standard MOE distribution, found in
<tt>$MOE/lib/svl/patch</tt>.
</li>

<li>Custom files from <tt>$MOE/lib/svl/custom</tt>.
If there is a file called <tt>moe-menus</tt> or
<tt>.moe-menus</tt> found here, it will also be read in.
</li>

<li>If the environment variable
<a class="svl" href="../install/install.htm#MOE_SVL_LOAD">MOE_SVL_LOAD</a> has
been set to a directory, then files from that directory.
</li>

<li>If the directory <tt>$HOME/svl</tt> exists, then
files from that directory.
</li>

</ol>

<p>
MOE can also be invoked with the
<a class="svl" href="../install/install.htm#-load">-load</a>
command line switch which specifies an SVL source file or directory
of SVL source files to be loaded automatically when MOE starts.
</p>

<p>The directory <tt>$MOE/lib/svl/custom</tt> has a subdirectory
<tt>$MOE/lib/svl/custom/run</tt> in which runnable custom files
can be stored.  These files will be accessible to all users of MOE.
</p>

<p>Runnable files are searched for in the following
directories in the order given.
If the same filename is found in different directories,
the first file of that filename that is encountered in the
search is the one that is used.
</p>

<ol>

<li>
The current working directory (as returned by the
<a class="svl" href="../svl/fcnref/cd.html#cd">cd</a> function).
</li>

<li>
<a class="svl"
href="../install/install.htm#MOE_SVL_RUNPATH">MOE_SVL_RUNPATH</a>,
if set to a valid directory.
</li>

<li>Custom directory, <tt>$MOE/lib/svl/custom/run</tt>.
</li>

<li>Standard MOE distribution patch directory,
<tt>$MOE/lib/svl/patch/run</tt>.
</li>

<li>Standard MOE distribution, <tt>$MOE/lib/svl/run</tt>.
</li>

</ol>

<p>Examples of using the custom directory can be found in:
</p>

<ul>
<li><tt>$MOE/sample/echo.svl</tt> A loadable custom file.
</li>
<li><tt>$MOE/sample/hello_world.svl</tt> A runnable custom file.
</li>
<li><tt>$MOE/sample/custom-menus</tt> A custom menu file.
</li>
</ul>

<p>Note that if MOE is invoked using the
<a class="svl" href="../install/install.htm#-std">-std</a>
command line switch, customizations will be ignored, and
only files in the patch and standard distribution directories
will be used.

<a name="Menus"><h2>Menus</h2></a>
<p>
Menus, popups and button bars in MOE are configured using menu
configuration files.  On startup, MOE first reads the master menu
configuration file located in <tt>$MOE/lib/moe-menus</tt> and then the
user menu file. User-specific menu configuration is achieved with the menu
configuration file named <tt>$HOME/moe-menus</tt> (if it exists).  On
Unix systems, the menu file can be named <tt>$HOME/.moe-menus</tt>.
If Unix is the only operating system being used then <tt>.moe-menus</tt> is
normally used as the filename.  However, if both Unix and Windows operating
systems are used then <tt>moe-menus</tt> should be used.
In this way, only one menu file need be maintained.
The user menu definitions <i>override</i> the master definitions on a
menu-by-menu basis; i.e. to redefine a particular menu, only that
menu need be specified in the user menu file.
</p>

<p>Menu files are read by MOE in the following order.  Subsequent
menus override previous menus.
</p>

<ol>

<li>MOE standard distribution menus, found in <tt>$MOE/lib</tt>.
</li>

<li><tt>$MOE/lib/svl/custom/moe-menus</tt>, if it exists.
</li>

<li><tt>$HOME/moe-menus</tt> if it exists, otherwise
<tt>$HOME/.moe-menus</tt> if it exists.

</li>

</ol>

<p>Note that if MOE is invoked using the
<a class="svl" href="../install/install.htm#-std">-std</a>
command line switch, customizations will be ignored, and
only the standard MOE menus will be used.
</p>

<p>
Each menu file consists of a sequence of menu specifications. Each menu
is given a name so that other menus can reference it. If a new menu
specification has the same name as a previous specification, the new menu
will override the old one.  Menus exist as items in one of the standard MOE
menu bars, button bars or popup menus.
For more complete information about menu files, please see
<a href="../moe/fcnref/menufile.htm">Menu File Format</a> or consult
<tt>$MOE/lib/moe-menus</tt> and the files it includes for detailed information
about the contents of the MOE menus.
<p>
To customize menus and buttons in the version of MOE you are running,
you must either a) create a user menu configuration file and save it in your
home directory; or b) create a menu file and load it using
<b><nobr>MOE | File | Open</nobr></b> or c) use the SVL function
<tt>ReadMenuFile</tt>.

<h3>Adding a Command to a Standard Menu</h3>
<p>
The <tt>MENU PREPEND</tt> and <tt>MENU APPEND</tt> constructs allow you
to add menu options to existing menus. <tt>PREPEND</tt> causes the new
option to be inserted at the top of the menu. <tt>APPEND</tt> puts the
new option at the bottom. <tt>ENDMENU</tt> terminates the current set of menu
modifications.
<p>
For example, to add a new entry to the Compute menu that will execute one
of MOE's sample programs (<tt>asurf.svl</tt> to plot the water accessible
surface area of a molecule):

<ol>
<li>Using a text editor, type the following text in a menu configuration
file named <tt>asurf.menu</tt>:
<pre>
  MENU PREPEND &quot;Main:Compute&quot;
    "Water Accessible Surface" exec "run '$MOE/sample/asurf.svl'"
        accel 'ctrl+alt+a'
        active atoms 
  ENDMENU
</pre>
<p>
The <tt>accel</tt> attribute allows you to invoke the Water Accessible
Surface command using the keyboard accelerator combination <tt>Ctrl-Alt-a</tt>.
The <tt>active</tt> attribute causes the menu button to be dimmed if no
atoms are present in the system.
<p>
<li>Save the file and type <tt>ReadMenuFile 'asurf.menu'</tt> at the
SVL command line.
</ol>
<p>
If, instead of <tt>asurf.menu</tt> the menu were added to
<tt>$HOME/moe-menus</tt> then this menu item would be added every time
MOE was started (unless the <tt>-std</tt> switch were used).
</ol>

<h3>Adding a Submenu to a Standard Menu</h3>
<p>
Adding new submenus to existing menu bars is very similar to adding menu
commands.  The standard menu bar names in MOE are: 
<tt>Main, SEMenuBar, dbvMenuBar, SVLTextEditor</tt>.
For example, to add a submenu <b>MyMenu</b> to the main MOE menu bar:

<ol>
<li>In <tt>$HOME/moe-menus</tt> type:
<pre>
    MENU APPEND &quot;Main&quot;
	&quot;MyMenu&quot; submenu &quot;Work on Selection&quot;
    ENDMENU

    MENU &quot;Work on Selection&quot;
	&quot;Keys&quot; exec 'Atoms[] | aSelected Atoms[]'
	       active sel_atom
	&quot;Total&quot; exec 'length (Atoms[] | aSelected Atoms[])'
	       active sel_atom
    ENDMENU
</pre>

<p>
<li>Use <tt>ReadMenuFile'$HOME/moe-menus'</tt> to load the changes.
The new menu will appear and will print its results to the SVL Commands
window.  <b>Note!</b> a new submenu <b>MyMenu</b> will appear every
time the menu file is loaded.
</ol>

<p>
Popup menus are treated as any other menus in MOE inasmuch as submenus
and options can be added to them. You cannot, however, create entirely new
(non-standard) popups. Only some windows support popups.  For example,
<pre>
    MENU APPEND &quot;SEPopupMenu&quot;
	&quot;num&quot; exec 'add rSelected Residues[]'
    ENDMENU
</pre>

<p>Popup menus that can be added to are:
<ul>
<li>MOE Window: <tt>MainPopup</tt>
<p><li>Sequence Editor: <tt>SEPopupMenu, SEChainPopupMenu, SEColumnPopupMenu,
SEResiduePopupMenu</tt>
<p><li>Database Viewer: <tt>dbvMolCellPopupMenu, dbvFieldPopupMenu,
dbvEntryPopupMenu, dbvCellPopupMenu, dbvPlotPopupMenu, dbvYAxisPopupMenu</tt>
<p><li>SVL Text Editor: <tt>SVLTextEditorPopup</tt>
</ul>

<h3>Button Bars</h3>
<p>
Adding a button to a button bar is the same as adding a menu option to
a menu. The standard button bars in MOE are: <tt>
RightButtonBar, LeftButtonBar, SERightButtonBar, SELeftButtonBar,
dbvRightButtonBar, dbvLeftButtonBar</tt>.
Note that, by default, only the right button bar appears in the MOE Window.

<p>
The <tt>PAGER</tt> construct is applicable to button bars.
Suppose you wish to create shortcuts to the atom selection and
rendering commands you use most often. In the next example, you will add
a left button bar to the MOE Window. It will contain two pages:
the first one (called SELECT) will include atom selection commands, and
the second (called RENDER) will include atom rendering commands.
<pre>
  MENU PAGER "LeftButtonBar"
    "Select"	submenu "SELECT"
    "Render"    submenu "RENDER"
  ENDMENU

  MENU "SELECT"
    "Hydrogen"          exec "aSetSelected [Atoms[], 'H'  == aElement Atoms[]]"
    "Carbon"            exec "aSetSelected [Atoms[], 'C'  == aElement Atoms[]]"
    "Nitrogen"          exec "aSetSelected [Atoms[], 'N'  == aElement Atoms[]]"
    "Oxygen"            exec "aSetSelected [Atoms[], 'O'  == aElement Atoms[]]"
    "Fluorine"          exec "aSetSelected [Atoms[], 'F'  == aElement Atoms[]]"
    "Phosphorus"        exec "aSetSelected [Atoms[], 'P'  == aElement Atoms[]]"
    "Sulfur"            exec "aSetSelected [Atoms[], 'S'  == aElement Atoms[]]"
    "Chlorine"          exec "aSetSelected [Atoms[], 'Cl' == aElement Atoms[]]"
    "LonePair"          exec "aSetSelected [Atoms[], 'LP' == aElement Atoms[]]"
  ENDMENU

  MENU "RENDER"
    "Line"              exec 'RenderLine[]'
    "Stick"             exec 'RenderStick[]'
    "Ball and Line"     exec 'RenderBallAndLine[]'
    "Ball and Stick"    exec 'RenderBallAndStick[]'
    "Space Filling"     exec 'RenderSpaceFilling[]'
  ENDMENU
</pre>

<a name="SVLPrograms"><h2>SVL Programs</h2></a>

<p>When MOE starts, a collection of SVL programs are automatically loaded.
These SVL programs form the basic tool set and interactive support for
MOE. These files reside in the directory <tt>$MOE/lib/svl</tt> and its
subdirectories.
If the environment variable <tt>MOE_SVL_LOAD</tt> exists, then its
value is interpreted as a file or a directory. The file is loaded as an SVL
module; the directory is searched for all files that match the wildcard
expression <tt>*.svl</tt>, which are loaded as well.
If the environment variable <tt>MOE_SVL_LOAD</tt> does not exist then
the files in <tt>$HOME/svl</tt> are loaded upon startup.
Files in the installation set can be overridden by copying them to
another directory and loading them automatically via the
<tt>MOE_SVL_LOAD</tt> environment variable.
<p>
For example, to automatically load the file <tt>myprogram.svl</tt> upon
MOE startup (assuming that <tt>MOE_SVL_LOAD</tt> does not exist):
<ol>
<li>Create the directory <tt>$HOME/svl</tt>.
<li>Copy <tt>myprogram.svl</tt> to <tt>$HOME/svl</tt>.
<li>Restart MOE.
</ol>

<a name="FKeys"><h2>Function Keys</h2> </a>
<div class="floatright">
<img src="moeconfig/fkeys.png" alt="Function Keys">
</div>

<p>
MOE normally does not make use of the F1-F10 keyboard function keys.  However,
these may be programmed to invoke SVL expressions.  To configure these keys
choose <b>MOE&nbsp;|&nbsp;Window&nbsp;|&nbsp;Options</b> and select the
<b>Function Keys</b> page in the menu.
<p>
Then, enter the expression to be associated with the function key, or
the Ctrl-, Alt- or Shift- modified function key.  Press <b>OK</b> or
<b>Apply</b> (the latter leaves the panel open).
The function key configuration will be written to the
MOE configuration file.
</p>

<p>
<b>Note:</b>
On some operating systems, certain function key combinations
have been appropriated by the system for keyboard shortcuts and cannot
be overridden by the Configuration Options panel.
For example, Alt+F4 is commonly used to exit programs and on GNOME,
Alt+F7 is used to move the focused window.
<br clear="both" />
</p>


<a name="Builder"> <h2>Builder</h2> </a>

<div class="floatright">
<img src="moeconfig/config_builder.png" alt="Builder" />
</div>


<p>To configure the <a href="../moe/molsystems/build.html">Molecule Builder</a>,
choose <span class="menu">MOE | Window | Options</span> and
select <b>Builder</b> from the menu.</p>
<p>
The <b>Controls</b> section is used to enable or disable the display
of the hybridization, SMILES input and atom name controls
in the molecule builder.
<p>
The <b>Options</b> section contains checkboxes used to turn on or off
warnings issued when a new molecule is created and the automatic
labeling of atoms with incorrect topology.
<p>
The <b>Chirality Limit</b> specifies the maximum number of atoms under which
the molecule builder stereochemistry controls are automatically updated to
reflect the current state. 
<p>
<b>R-Vector</b> specifies the van der Waals interaction
energy threshold and color used to draw
<a href="../apps/ligx.htm#RVectors">exit vectors</a>.
<br clear="both" />
</p>


<a name="LigX"> <h2>LigX</h2> </a>

<div class="floatright">
<img src="moeconfig/config_ligx.png" alt="LigX" />
</div>


<p>To configure the <a href="../apps/ligx.htm">LigX Right Button Bar</a>,
choose <span class="menu">MOE | Window | Options</span> and
select <b>LigX</b> from the menu.

<p>
Rendering style (<b>Solid</b>, <b>Line</b>, <b>Dot</b>),
material property, and coloring mode of &quot;Quick&quot; surfaces
(<b>Receptor Surface</b>, <b>Ligand Surface</b>, and
<b>Interaction Surface</b>), are set in the top part of the page.
The color widget is used for color selection when <b>Constant</b>
color is chosen.
</p>

<p>
<b>R-Vector</b> specifies the van der Waals interaction
energy threshold and color used to draw
<a href="../apps/ligx.htm#RVectors">exit vectors</a>.
</p>

<p>
The <b>Duplicate</b> options specify properties of the ligand
that are created using
<span class="menu">MOE | RHS | Ligand | Duplicate</span>.
</p>

<p>
<ul>
<li><b>Activate+Show</b>. If on, the duplicate will be shown and
all atoms will be active (the original ligand in all cases will
be hidden and made inert).  If off, the duplicate will be shown,
but the active state will be that of the original ligand atoms.
</li><li>
<b>Stick</b>.  If on, the duplicate will be rendered in stick mode.
If off, it will be rendered in the same way as the original ligand atoms.
</li><li>
<b>Element Color</b>.  If on, the duplicate atoms will be colored by
element.  If off, they will be colored in the same way as the original ligand
atoms.
</li><li>
<b>Hide and de-activate all other ligands</b>.
If on, any other ligands in the system will be hidden and made
inert.  If off, the active/inert state and visibility of other ligands is
untouched.
</li>
</ul>
</p>

<p>
The <b>Minimize (Vac)</b> options are used to control the
<span class="menu">MOE | RHS | Ligand | Minimize (Vac)</span>
operation.  If the <b>Display message</b> option is on, a
message reporting the minimized strain energy and RMSD will
be displayed for the specified number of seconds.  If off,
no message will be displayed.  If <b>Color newly created
carbons</b> is on, the duplicate of the ligand will have
its carbon atoms colored by the specified color.  If off,
the carbon atom coloring will be the same as that
of the original ligand.
<br clear="both" />
</p>

<a name="JDBC"><h2>JDBC Database Connections</h2>

<div class="floatright">
<img src="moeconfig/jdbc.png" alt="JDBC Options">
</div>

<p>
The <a href="samples.htm#JDBCViewer">JDBC Database Viewer</a> and other
applications that use the JDBC relational database access library generally
require server and user specification.  To configure these settings
choose <b>MOE&nbsp;|&nbsp;Window&nbsp;|&nbsp;Options</b> and select the
<b>JDBC Database Connections</b> page in the menu.</p>
<p>Up to four connection configurations may be specified.  For each, enter
the required information, then press <b>OK</b> or <b>Apply</b>.
For more information on the JDBC Connection information please see
<a href="samples.htm#JDBCViewer">Setting the JDBC Database Connections</a>.
<br clear="both" />
</p>

<p>
<a name="PSILO"><h2>PSILO/RCSB Connectivity</h2>
</p>

<div class="floatright">
<img src="moeconfig/psilo.png" alt="PSILO Options">
</div>

<p>MOE can connect to the RCSB with no special configuring, although
configuration options do exist.
Connecting to PSILO requires setting the PSILO URL and the user
name and password.
Choose <b>MOE&nbsp;|&nbsp;Window&nbsp;|&nbsp;Options</b> and select the
<b>PSILO/RCSB Connectivity</b> page in the menu.
</p>

<p>
If you have access to a
<a href="http://www.chemcomp.com/psilo-info.htm" target="_blank">PSILO</a>
server, you can configure MOE to connect to it, for example using the
<a href="../proteins/pdb_mdb.htm#PSILODownload">PSILO Browse</a> panel.

<p>
Enter the required information, then press <b>OK</b> or <b>Apply</b>.
<b>Name</b> is just an arbitrary name for the connection.
</p>


<p>Connecting to the RCSB requires no username or password.
By default, MOE will connect to the main RCSB site
<tt>ftp://ftp.wwpdb.org</tt>, as maintained by the
<a href="http://www.wwpdb.org">World Wide Protein Data Bank</a>
consortium.  However, a mirror site can be specified by
entering the appropriate URL in the <b>RCSB Mirror</b> field and pressing
<b>OK</b> or <b>Apply</b>. Note that this should be the
name of the root directory in which the <tt>/pub/pdb</tt> directory
is found.  For example, the following would be the URL
used to specify the Japanese RCSB mirror site:
</p>

<pre>
	ftp://protein.osaka-u.ac.jp
</pre>

<p>In the <a href="../proteins/pdb_mdb.htm#PSILODownload">PSILO Browse</a>
panel, the mirror will appear in the <b>Site</b> menu as the menu entry
&quot;Local&quot;.
</p>

<p>The <b>Directory</b> field is used to specify the directory in which
the downloaded PDB files will be saved.
Press <b>OK</b> or <b>Apply</b> to put any changes into effect.
</p>

<p><b>Note:</b> If your network uses an internet proxy server, you may
need to configure the JVM (Java Virtual Machine) for HTTP
transfers through the proxy server.  For more information,
please see the section on
<a href="../install/install.htm#JVM_OPTS">Options for
Starting the JVM</a>.
<br clear="both" />
</p>


<a name="MainMouse"></a><h2>Mouse (MOE Window)</h2>
<div class="floatright">
<img src="moeconfig/mouse.png" alt="MOE Window Mouse" />
</div>

To configure the behavior of the right (third) mouse button in the MOE
main window, choose <b>MOE&nbsp;|&nbsp;Window&nbsp;|&nbsp;Options</b> and
select <b>Mouse (MOE Window)</b> from the menu.
(A two-button mouse cannot be programmed in this way.)
<p>
The <b>Mouse Sensitivity</b> controls the sensitivity of
rotation and translation as a function of mouse movements.
Higher values provide more precise
control in the rendering area than lower values. The sensitivity is a
floating-point number with default value 250; this value provides finer
control without significant loss of speed.
Values ranging from 100 to 1000 produce good results. 
<p>
The <b>Mouse Action</b> settings are used to assign an SVL expression
to a right mouse click (possibly modified by a combination of Ctrl, Alt
or Shift).  Enter an SVL expression in the appropriate text field. In this 
expression, the <tt>$</tt> character is replaced by the atom key of the clicked 
atom (or zero if no atom is clicked).
For example, to toggle the label setting of selected atoms using the
shift right mouse button in the MOE Window, you would enter the
following SVL expression in the <b>Shift+Button3</b> text field:
<p>
<pre>
if $ then aSetLabelName[$, not aLabelName $]; endif
</pre>
<p>
Press <b>OK</b> or <b>Apply</b> to update the mouse configuration.  (Note
that the changes will be written to the user <tt>moe-rc</tt> file when MOE
is closed.) In the MOE Window, right-click on an atom while holding down
the Shift key. A label will appear. Repeat to remove the label.
<br clear="both" />
</p>

<a name="DatabaseViewer"></a><h2>Database Viewer</h2>

<div class="floatright">
<img src="moeconfig/dbv.png" alt="Database Viewer">
</div>

<p>
To configure the Database Viewer choose
<b>MOE&nbsp;|&nbsp;Window&nbsp;|&nbsp;Options</b> and select
<b>Database Viewer</b> from the menu.
<p>
The <b>Default Plot Range</b> values specify the default range of vertical
axis values of the Database Viewer plot area.
<p>
The <b>Modifier</b> fields contain SVL commands to execute when the
left mouse is double-clicked in any cell of the database viewer.
<b>None</b> means a plain double click, <b>Control</b> means a double-click
with the Ctrl key held down and similarly for <b>Shift</b> and <b>Alt</b>.

<p><b>Row Color</b> sets the color of alternate rows in the Database Viewer.
<br clear="both" />
</p>

<a name="SequenceEditor"></a> <h2>Sequence Editor</h2>

<div class="floatright">
<img src="moeconfig/seqed.png" alt="Sequence Editor">
</div>

<p>
To configure the Sequence Editor choose
<b>MOE&nbsp;|&nbsp;Window&nbsp;|&nbsp;Options</b> and select
<b>Sequence Editor</b> from the menu.
<p>
The <b>H-Bonds</b> parameter is used to color the hydrogen bond arcs, which are
drawn using <nobr><b>SE | Display | Hydrogen Bonds</b></nobr>.
The <b>Plot Scalar</b> parameters determine the color of the Sequence Editor
residue scalar plots, which are drawn beneath the residues in the Sequence
Editor drawing area.
<p>
The remaining display options set the default display parameters of the 
Sequence Editor. At any time during a MOE session, these parameters can
be modified using the Display menu in the
<a href="../moe/gui/se.html">Sequence Editor</a>;
such modifications will have no effect on the default values.
<br clear="both" />
</p>

<a name="ProteinDefaults"></a> <h2>Protein Defaults</h2>

<div class="floatright">
<img src="moeconfig/optprosetup.png" alt="Protein Defaults">
</div>

<p>
To configure the Protein database, Kinome database, modeling and
alignment defaults, choose
<b>MOE&nbsp;|&nbsp;Window&nbsp;|&nbsp;Options</b> and select
<b>Protein Defaults</b> from the menu.
<p>
The <b>Protein Database</b> field specifies the default molecular database
file opened in the Protein Database panel. Two protein databases with
data from the Protein Data Bank [Berman 2000] are delivered as part of the
MOE distribution. The file <tt>$MOE/lib/pdb.mdb</tt> is automatically installed
with MOE; it contains a non-redundant set of protein chains only. An mdb
containing the full release of the PDB is located on the MOE distribution DVD.
The <b>Protein Family Files</b> field specifies a file with a list of 
sequence alignment files for homology searching.
See <a href="../proteins/pdb_mdb.htm">
Protein Sequence and Structure Data</a> for more information on MOE's
protein databases. 

<p>
The <b>Residue Database</b> is a molecular database file containing
conformational data on the 20 standard amino acids, plus acetyl and
amide endcaps. This data is used by the Protein Builder and Create Sequence
applications  for constructing 3D polypeptide models in MOE. See 
<a href="../proteins/protool.htm">Building Proteins and DNA</a> for more
information.
<p>
The <b>Rotamer Library</b> and <b>Environment Data</b> are use by MOE's
<a href="../proteins/promodel.htm">Homology Model</a> application.
<b>Prediction Data</b> is used in secondary structure prediction.
<p>
<b>Similarity Matrix</b>, <b>Matrix Path</b>, <b>Gap Start</b> and
<b>Gap Extend</b> are the default values for sequence alignment applications.
<p>
The <b>Kinome Database</b> field specifies the default molecular database
file opened by the <a href="../proteins/kinase.htm">Kinase Search</a> panel.
<br clear="both" />
</p>


<a name="Miscellaneous"></a><h2>Miscellaneous</h2>

<div class="floatright">
<img src="moeconfig/misc.png" alt="Miscellaneous">
</div>

To configure the Miscellaneous settings,
<b><nobr>MOE | Window | Options</nobr></b> and select <b>Miscellaneous</b>
from the menu.
<p>
<b>Bubble Help Mode</b> controls the display of bubble help in all
MOE panels. To enable the display of bubble help, select <b>on</b>.
To disable the display of bubble help, select <b>off</b>.
Select <b>key</b> to display bubble help only when
the <tt>Ctrl</tt> key is held down for over a second.
<p>
<b>SVL Output History</b> controls the number of text lines to maintain in the
scrolling area of the SVL Commands Window.
<b>SVL Command History</b> controls  the number of previous commands to keep
in the command line (CLI) history buffer. Previous commands are retrieved
at the CLI with the up/down arrow keys.
<p>
<b>Window Positions Save</b> saves the position of all named windows in MOE.
<b>Window Positions Restore</b> moves all windows to their saved positions.


<h2>See Also</h2>

<a href="../moe/gui/mainwin.htm">The MOE Window</a><br>
<a href="samples.htm#JDBCViewer">JDBC Database Viewer</a><br>
<a href="../appendix/viewsetup.htm">Visualization Setup</a><br>
<a href="../moe/fcnref/menufile.htm">Menu File Format</a><br>
<a href="../moe/fcnref/rc.htm">Configuration File Functions</a>


<p><a href="../index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
