<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">


<!--
!!	samples.htm Guide to MOE sample applications
!!-->

<!--
!!    MOE Online Manuals
!!    COPYRIGHT (C) 1997-2015
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

<script type="text/javascript" 
src="../include/jsincludes_moe.js"></script>

<link rel="stylesheet" type="text/css"
href="../include/manstyle.css" />


<style type="text/css">
.index {
	font-weight:bold;
	font-family:Arial,sans-serif;
	color:#008080;
}
.index ul {
	font-weight:normal;
	font-family:serif;
	color:black;
	padding-bottom:1em;
}
pre {
	margin-left:2em;
}
</style>

<title>MOE Sample Programs</title>

<meta panel JDBC Database Viewer>

<meta main_keywords Java Database Connectivity />
<meta main_keywords SVL_JVM, CLASSPATH, SQL, PostgreSQL, MySQL />

<meta keywords MDB, password, URL, environment variable />
<meta keywords third party, import, alias, username, password />
<meta keywords mpu_password, driver />

</head>

<body>
<div id="useMathJax"></div>
<div id="MoeHeader"></div>
<noscript>
	<span class="warning">Warning: JavaScript is disabled. This page will not display correctly.</span>
	<h1 class="title">MOE Documentation</h1>
	<hr noshade="noshade" />
	<style>.LaTeX {color: #C08080;}</style>
</noscript>

<p>The MOE package includes a variety of sample programs that not
only illustrate the versatility and practicality of MOE's various
tools but also the ease with which one can program new applications
using the SVL programming language.  The SVL files can be found 
in the directory $MOE/sample.  For an alphabetical listing of these
files, see <a href="#QuickRef">Sample Programs Quick Reference</a>.</p>

<ol class="index">
<li>Molecular Systems
    <ul>
    <li><a href="#Typer">Atom Typer (patty.svl)</a></li>
    <li><a href="#EEM">Partial Charge Calculation (eem.svl)</a></li>
    <li><a href="#Scenes">Molecular Scenes Manager (scenes.svl)</a></li>
    </ul>
</li>
<li>Molecular Surfaces
    <ul>
    <li><a href="#Water">Water Accessible Surfaces (asurf.svl)</a></li>
    <li><a href="#Charge">Surface Coloring by Charge (csurf.svl)</a></li>
    <li><a href="#Electro">Electrostatic Surfaces (esurf.svl)</a></li>
    <li><a href="#Gridsurf">Grid Surface Demo (gridex.svl)</a></li>
    <li><a href="#Nabstack">DNA Base Stacking (nabstack.svl)</a></li>
    <li><a href="#Solv">Solvation Free Energy from Surface Area (vajdaweng.svl)
	</a></li>
    </ul>
    </li>

<li>Molecular Databases
    <ul>
    <li><a href="#Molshape">Molecular Shape Similarity (molshape.svl)</a></li>
    <li><a href="#Pairsim">Typed Pair Distance Similarity (pairsim.svl)</a>
	</li>
    <li><a href="cliprg.htm">Clipping R-Groups (dbvclip.svl)</a>
	</li>
    <li><a href="cdgen.htm">QuaSAR-CombiGen:
	Enumerative Combinatorial Library Generation (qcdgen.svl)</a>
	</li>
    <li><a href="qcombi.htm">QuaSAR-CombiDesign: Create Diverse Subset
	(qcombi.svl)</a>
	</li>
    <li><a href="qcrsel.htm">QuaSAR-Reagent: Reagent Activity Analysis
	(qcrsel.svl)</a>
	</li>
    <li><a href="#WebMake">Output As HTML (webmake.svl)</a></li>
    <li><a href="#JDBCViewer">JDBC Database Viewer (jdbcview.svl)</a></li>
    </ul></li>

<li>Quantitative Structure Activity Relationships
    <ul>
    <li><a href="#Kier">Kier Descriptors (q_estate.svl)</a></li>
    <li><a href="#Randic">Randic Descriptor (q_randic.svl)</a></li>
    <li><a href="#MACCS">MACCS Key Count Descriptors (q_maccs.svl)</a></li>
    </ul></li> 

<li>Molecular Modeling
    <ul>
    <li><a href="#4Dconf">4D Conformational Sampling (cs4d.svl)</a></li>
    <li><a href="#HMC">Hybrid Monte Carlo Simulation (hmc.svl)</a></li>
    <li><a href="#Langevin">Langevin Implicit Euler Dynamics (iedyn.svl)
	</a></li>
    <li><a href="#Molsup">Molecular Superposition (seal.svl)</a></li>
    <li><a href="#Flexsup">Flexible Molecular Superposition (sealflex.svl)
	</a></li>
    <li><a href="#NVE">NVE Molecular Dynamics Simulation (verlet.svl)</a></li>
    <li><a href="#dihmplot">Dihedral Energy Plot with Minimization (dihmplot.svl)</a></li>
    <li><a href="#eht_ui">Extended Hueckel Theory Properties
	 (eht_ui.svl)</a></li>
    <li><a href="#rism_hfe">3D-RISM Hydration Free Energy (rism_hfe.svl)</a></li>
    </ul></li>
<li>Molecular Dynamics
    <ul>
    <li><a href="#mdtools_animate">Trajectory Animation Panel</a></li>
    <li><a href="#mdtools_qp">Trajectory Quick Play</a></li>
    <li><a href="#mdtools_moevie">Trajectory MOEvie Generation</a></li>
    <li><a href="#mdtools_analysis">Trajectory Analysis Tools</a></li>
    </ul></li>

<li>Protein Analysis and Modeling
    <ul>
    <li><a href="#Alpha">Protein Reconstruction from Alpha Carbon Trace
    (alphabuild.svl)</a></li>
    <li><a href="#OCCO">OCCO Plot (occo.svl)</a></li>
    <li><a href="#DistPlot">Alpha Carbon Distance Matrix (distplot.svl)</a>
	</li>
    <li><a href="#Jack">Mean Pair Potential Jackknife Test (pairpot.svl)</a>
	</li>
    </ul></li>

<li>MOE/cluster (MOE/smp)</font></b>
    <ul>
    <li><a href="#smics">Parallelized Stochastic Search (smics.svl)</a></li>
    <li><a href="#psearch">Parallelized Homology Search (psearch.svl)</a></li>
    </ul></li>

<li>SVL Programming
    <ul>
    <li><a href="#grdemo">2D Graphics Demo (grdemo.svl)</a></li>
    <li><a href="#Gobj">3D Graphics Objects Demo (gobjdemo.svl)</a></li>
    <li><a href="#Gridcode">Grid Code Example (gridcode.svl)</a></li>
    <li><a href="#Window">SVL Window Toolkit Demo (windemo.svl)</a></li>
    </ul></li>
</ol>

<!-- ATOM TYPER -->
<hr noshade>
<h3> <a name="Typer">Atom Typer</a> </h3>
<ul>
    <li><b>File</b>: <tt>$MOE/sample/patty.svl</tt></li>
    <p><li><b>Description</b>: This sample types atoms in the system
    according to PATTY and draws the resulting type tokens next to
    their respective atoms in the MOE rendering window.</p>

    <p><cite> Reference: B.L.Bush, R.P.Sheridan
    PATTY: A Programmable Atom Typer and Language for
    Automatic Classification of Atoms in Molecular Databases.
    J.Chem.Info.Comp.Sci, 33, pp756-762, 1993.
    </cite></p>

    <p>The tokens returned are:</p>

    <table>

    	<tr>
    	<td valign="top">'CAT'</td>
    	<td valign="top">CATION</td>
    	<td>used for basic functionality; i.e. protonated</td>
    	</tr>

    	<tr>
    	<td valign="top">'ANI'   </td>
    	<td valign="top">ANION</td>
    	<td>used for acidic functionality; i.e. de-protonated</td>
    	</tr>

    	<tr>
    	<td valign="top">'POL'   </td>
    	<td valign="top">POLAR</td>
    	<td>atom is neither an ANION nor a CATION and has a lone pair
    	and a polar H, or (due to tautomerization) can be a donor
    	or an acceptor.</td>
    	</tr>

	<tr>
	<td valign="top">'DON'    </td>
	<td valign="top">DONOR</td>
	<td>an atom that is none of the above and has a polar H.</td>
	</tr>

	<tr>
	<td valign="top">'ACC'    </td>
	<td valign="top">ACCEPTOR</td>
	<td>an atom that is none of the above and has a lone pair.</td>
	</tr>

	<tr>
	<td valign="top">'HYD'   </td>
	<td valign="top"> HYDROPHOBIC</td>
	<td>an atom that is none of the above and is an
	accessible atom that has hydrophobic functionality</td>
	</tr>

	<tr>
	<td valign="top">'OTH'   </td>
	<td valign="top"> OTHER</td>
	<td>an atom that falls into none of the above classes.</td>
	</tr>
    </table>

    <p>(A polar H is an H that is strongly positively charged.)  Hydrogens are
    typed as either HYD or POL.

    The environment is assumed to be at a particular pH value or in a
    narrow (although unspecified) pH range.</p>


    <li><b>Instructions</b>:<br />

    <ol>
    <li>Open a molecule file.</li>
    <li>You can run the file by typing:

    <pre>run '$MOE/sample/patty.svl'</pre>

	In this case, all of the atoms in the current system will be used.
    </li>
    <li>Or you can load the file by typing:

    <pre>load '$MOE/sample/patty.svl'</pre>

	and then run the function, passing only the keys of atoms of
	interest:
<pre>PATTY_Type atom_keys</pre>
</li>
<li>Further, if you wish to have the types returned, use:
<pre>type_tokens = PATTY_Type atom_keys</pre>
</li>
    </ol>
    </li>
</ul>

<!-- EEM -->

<hr noshade>
<h3> <a name="EEM">Partial Charge Calculation</a> </h3>
<ul>
    <p><li><b>File</b>: <tt>$MOE/sample/eem.svl</tt></li></p>
    <p><li><b>Description</b>:

	SVL function <b><tt>EEM</tt></b>.
	Computes the partial charges according to the Electrostatic Energy
	Minimization partial charge model for all currently loaded atoms.
	The EEM charge model consists of an energy function as a function
	of charge for each atom in the system, along with Coulomb
	electrostatic energy terms, so that charge depends on molecular
	conformation. The current potential setup determines the form of
	the Coulombic terms.

	<p>Note: This model should not be used if the atomic coordinates are
	unreasonable. A topological algorithm (not dependent on
	conformation) such as PEOE should be used instead.</p>
    </li></p>
    <li><b>Instructions</b>:

	<ol>
	<li>Open a molecule file.</li>
	<li>Load the module using the following command:

	<pre>load '$MOE/sample/eem.svl'</pre>
	</li>
	<li>Run the function using:
	<pre>EEM []</pre>

	<p>EEM returns the partial charge for every atom in the system.

	<p>Note that <tt>EEM</tt> returns the partial charges but
	does not actually set the atom charges. To do so, use:

	<pre>aSetCharge [ Atoms [], EEM [] ] </pre>
	</li>
	</ol>
	</li>
</ul>

<!-- Scenes -->

<hr noshade>
<h3> <a name="Scenes">Molecular Scenes Manager</a> </h3>
<ul>
    <p><li><b>File</b>: <tt>$MOE/sample/scenes.svl</tt></li></p>
    <p><li><b>Description</b>:

	This sample program implements a Scenes Manager, which enables
	a user to save complete molecular scenes with all properties,
	such as molecule representation style, meters, constraints,
	surfaces, etc., in a database. The scenes can be commented and
	a button bar gives quick access to the first 10 scenes in
	the database.
    
    </li></p>
    <p><li><b>Instructions</b>: 
    <ol>
    	<p><li>
	Run the file by typing:
	<pre>run '$MOE/sample/scenes.svl'</pre>
	<p>The Scenes Manager prompter is displayed at the top
	of the MOE window:</p>
	<p style="text-align:center">
	<img src="samples/scenes_prompt.png" alt="Scenes Prompter" />
	</p>
	</li></p>
	<li>Choose a new scene database with <b>Set</b> or use the
	default <tt>scenes.mdb</tt> in the current work directory.
	</li>
	<li>New molecular scenes can be added to the database
	with the <b>Add</b> button.
	</li>
	<li>Scenes from the database can be loaded with
	the numbered <b>Scenes</b> buttons.
	</li>
	<li>To remove scenes, add comments or access more than 10
	scenes, open the Scenes Manager panel with the <b>Manage</b> button.
	<p style="text-align:center">
	<img src="samples/scenes_mgr.png" alt="Scenes Manager" />
	</p>
	</li>
    </ol>
    </li></p>
</ul>

<!--             asurf.svl   -->

<hr noshade>
<h3><a name="Water">Water Accessible Surface</a> </h3>
<ul>
    <li><b>File</b>: <tt>$MOE/sample/asurf.svl</tt></li>
    <p><li><b>Description</b>: Plots the water accessible surface 
    of a compound.


    <p style="text-align:center">
    <img src="samples/s_asurf2.png" alt="Water Accessible Surface" />
    </p>
    </li></p>
    <p><li><b>Instructions</b>: 
    <ol>
	<li>Open a molecule file.</li>
	<p><li>Run the file by typing:
	<pre>run '$MOE/sample/asurf.svl'</pre>
	<p>A panel titled Water Accessible Surface is displayed:</p>
	<p style="text-align:center">
	<img src="samples/s_asurf1.png" alt="Water Accessible Surface Panel">
	</p>
	</li></p>
	<li>The Name is used to identify the surface in the
	<a href="../moe/gui/gobjman.htm">Graphics Objects Manager</a>.
	</li>
	<li>The Radius value controls the radius of the spherical
	probe used to calculate the water accessible surface.
	</li>
	<li>The Density buttons are used to choose how many dots
	will be drawn to render the water accessible surface. 
	</li>
	<li>The Atoms buttons are used to determine whether to draw
	the surface around all atoms or a selected group of atoms.
	</li>
	<li>Press <b>OK</b> to draw the surface. Once drawn, the surface can be
	managed (shown or hidden in the MOE rendering window) or deleted using
	the <a href="../moe/gui/gobjman.htm">Graphics Objects Manager</a>.
	</li>
    </ol>
    <p> For more information, see
    <a href="../apps/surfmap.htm">Molecular Surfaces</a>.</p>
    </li></p>
</ul>


    
    <!--   ----------     csurf.svl   ----------- -->

<hr noshade>

<h3> <a name="Charge">Surface Coloring by Charge </a></h3>
<ul>
    <li><b>File</b>: <tt>$MOE/sample/csurf.svl</tt></li>
    <p><li><b>Description</b>: Plots a molecular surface
    colored by charge.

	<p style="text-align:center">
	<img src="samples/s_csurf2.png" alt="Color-Coded Surface" />
	</p>
	
	</li></p>
    <li><b>Instructions</b>: <br>
        <ol> 
	    <li>Open a molecule file.</li>
	    <p><li>Before running this sample, make sure partial charges
	    have been computed for the compound.</li></p>

	    <p><li>Type the following command at the SVL command line:
	    <pre>run '$MOE/sample/csurf.svl'</pre>
	    <p>A panel titled Charge Surface is displayed.</p>

	    <p style="text-align:center">
	    <img src="samples/s_csurf1.png" alt="Charge Surface Panel" />
	    </p>
		</li></p>
		<li>The Name is used to identify the surface in the
		<a href="../moe/gui/gobjman.htm">Graphics Objects Manager</a>.
		</li>
		<li>The Radius value controls the radius of the spherical
		probe used to calculate the surface. 
		</li>
		<li>The Density buttons are used to choose how many dots
		will be drawn to render the surface. 
		</li>
		<li>The Atoms buttons are used to determine whether to draw the
		surface around all atoms or a selected group of atoms.
		</li>
		<li>Press <b>OK</b> to draw the surface. Once drawn, the
		surface can be managed (shown or hidden in the MOE rendering
		window) or deleted using the
		<a href="../moe/gui/gobjman.htm">Graphics Objects Manager</a>.
		</li>
	</ol>
    <p>For more information, see
    <a href="../apps/surfmap.htm">Surfaces and Maps</a>.
    </li></p>
</ul>

<!--  esurf.svl  -->

<hr noshade>

<h3> <a name="Electro"> Electrostatic Surfaces </a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/esurf.svl</tt></li>
    <p><li><b>Description</b>: Calculates electrostatic potential and 
    plots iso-surfaces of specified levels.

    <p style="text-align:center">
    <img src="samples/s_esurf2.png" alt="Electrostatic Iso-Surface">
    </p>

    </li></p>
    <li><b>Instructions</b>: <br>
        <ol><p>
	    <li>Open a molecule file.</li>
	    <li> Before running this sample, make sure partial charges
	    have been computed.</li>

	    <li>The file may be run with
	    <pre>    run '$MOE/sample/esurf.svl' </pre>
	    A panel titled Electrostatic Field Iso-Surface is displayed.

	<p style="text-align:center"><img src="samples/s_esurf1.png" 
	alt="Electrostatic Field Iso-Surface Panel" /></p>
	
	</li>

        <li>The Name is used to identify the surface in the
	<a href="../moe/gui/gobjman.htm">Graphics Objects Manager</a>.
	</li>

	<li>The Probe Charge value is used to scale the iso-surface.</li>

	<li>The Levels value is used to specify the levels of iso-surfaces to
	be drawn. More than one level can be specified at once; separate each
	value by a space.</li>

	<li>The Spacing buttons are used to choose line density of the
	iso-surface mesh.</li>

	<li>The Use Selected Atoms Only checkbox is used to specify around
	which atoms the surface will be drawn.  If on, only selected atoms will
	be considered in the surface calculations, otherwise, all atoms will be
	used.</li>

	<li>Press <b>OK</b> to draw the surface. Once drawn, the surface can be
	managed (shown or hidden in the MOE rendering window) or deleted using
	the <a href="../moe/gui/gobjman.htm">Graphics Objects Manager</a>.
	</li>
    </p>
    </ol>

    <p> For more information, see <a href="../apps/surfmap.htm">Electrostatic
    Feature Maps</a>.

    <!-- <p> For more information, see <a
    href="../moe/molsystems/pboltz.htm">Poisson-Boltzmann Electrostatics</a>.
    -->

</ul>

<!--      gridex.svl -->

<hr noshade>

<h3> <a name="Gridsurf">Grid Surface Demo </a></h3>
<ul>
    <li><b>File</b>: <tt>$MOE/sample/gridex.svl</tt></li>

    <p><li><b>Description</b>: Demonstrates the various drawing 
    modes used to render a grid surface.</li></p>

    <p><li><b>Instructions</b>:
        <ol> 
	    <li>Open a molecule file.</li>
	    <p><li> Type the following command at the SVL Command Line:
	    <pre>    run '$MOE/sample/gridex.svl' </pre>
	    A panel titled Grid Surfaces Demo is displayed.
	    <p style="text-align:center">
	    <img src="samples/s_gridex1.png" alt="Grid code panel" />
	    </p>
	    </li></p>
        <li>Turn on the Apply Changes Immediately toggle box so that
        values are instantaneously reflected on the surface displayed in
        the MOE rendering window.
        </li>
        <li>The Camnet fill ratio and Tile/loop separation parameters
        are only available when the appropriate Surface type is in
        effect.
        <p style="text-align:center">
        <img src="samples/s_gridex2.png" alt="Grid Surface" />
        </p>
        </li>
        </ol>
    </li>
</ul>



<hr noshade>
<h3> <a name="Nabstack"> DNA Base Stacking </a> </h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/nabstack.svl</tt>
    <p><li><b>Description</b>: Displays base-base stacking contacts.</a>

	<p><cite> Reference:
	Finding and Visualizing Nucleic Acid Base Stacking.
	Journal of Molecular Graphics, Vol.14, No.1, February 1995, pp.6-11.
	H.A.Gabb, S.R.Sanghani, C.H.Robert, and C.Prevost
	</cite></p>

	<p style="text-align:center">
	<img src="samples/s_nabstack.png" alt="DNA Base Stacking" />
	</p>
    </li></p>
    <li><b>Instructions</b>:
	<ol>
	    <li>Open a DNA molecule file from the RCSB database (for example, PDB code 101D).</li>
	    <li>Run the program with:
	    <pre>run '$MOE/sample/nabstack.svl'</pre>
	    </li>
	    <li>Graphics objects are managed (shown or hidden in the MOE
	    rendering window) or deleted using the
	    <a href="../moe/gui/gobjman.htm">Graphics Objects Manager</a>.
	    </li>
	</ol>
	</li>
    </ul>

<!-- vadjaweng.svl -->
    
<hr noshade>
<h3> <a name="Solv">Solvation Free Energy From Surface Area </a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/vajdaweng.svl</tt></li>
    <p><li><b>Description</b>: Prints, at the CLI, an estimate of the 
    transfer free energy based on the area of the water-accessible surface.
    
    <p><cite> Reference:
    Extracting Hydrophobicity Parameters from Solute Partition
    and Protein Mutation/Unfolding Experiments.
    Protein Engineering Vol.8, No.11, November 1995, pp.1081-1092
    S.Vajda, Z.Weng
    </cite></p>
    </li></p>
    <p><li><b>Instructions</b>:</p>
    <ol><li>Open a molecule file.</li>
    <li>In the SVL Commands window, type:</li>
	<pre>run '$MOE/sample/vajdaweng.svl'</pre></li>

<p>The output will appear in the SVL Commands window.</p>
</li>
</ul>

<!-- molshape.svl -->

<hr noshade>
<h3> <a name="Molshape">Molecular Shape Similarity</a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/molshape.svl</tt></li>
    <p><li><b>Description</b>:
    Implementation of the Nilakantan, Bauman and  Venkataraghavan
    algorithm to find similarities in molecular shapes of molecules.  The
    search is performed by comparing one molecule, which is loaded in MOE,
    to molecules in a specified database.  The first molecule field found
    in the database is used as the search field. Progress of the search is
    printed to the SVL Commands window.

    <p><cite>
    Reference: R.Nilakantan, N.Bauman, R.Venkataraghavan.
    New Method for Rapid Characterization of Molecular
    Shapes: Applications in Drug Design.
    J. Chem. Info. and Comp. Sci., 33, pp79-85, 1993.
    </cite></p>
    </li></p>
    <p><li><b>Instructions</b>:
    <ol>
    <li>Load a Search Molecule into MOE.</li>
    <li>Open a database of molecules to search.</li>
    <li>In the SVL Commands window, type:
    <pre>run ['$MOE/sample/molshape.svl', tol]</pre>

    <p>where <tt>tol</tt> is in the range [0,1] and represents a search
    tolerance.  A value of 1 searches for exactly similar structures.  If
    no tolerance is given, then a tolerance of 0.5 is used.  In this case,
    you can run the program with:</p>

    <pre>run '$MOE/sample/molshape.svl'</pre>

    <p>Molecules ascertained to be similar will be selected in the Database 
    Viewer. Note that the default tolerance is fairly low, and will result
	in many molecules being flagged as similar.</p>
	</li>
    </ol>
    </li></p>
</ul>

<!-- pairsim.svl -->

<hr noshade>
<h3><a name="Pairsim">Typed Pair Distance Similarity </a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/pairsim.svl</tt></li>
    <p><li><b>Description</b>: Calculates real-valued fingerprints based
    on binned, Cartesian atom pair distances for atoms of selected types in
    a molecule. Output fingerprints have variable length based on the number
    of distance bins and the number of requested atom types.

    <p><cite>
    Reference:
    R.P.Sheridan, M.D.Miller, D.J.Underwood, S.K.Kearsley
    Chemical Similarity Using Geometric Atom Pair Descriptors.
    J. Chem. Info. Comput. Sci., Vol 36, No. 1, 1996
    </cite></p>
    </li></p>
    <p><li><b>Instructions</b>: 
    <ol>
    <p><li>Load the fingerprinting module with the following command:
    <pre>load '$MOE/sample/pairsim.svl'</pre>
    <p>This defines two functions:</p>
    <pre>fp2_FingerPrint [atom_keys, type_vector, sel_types]</pre>
    <p>which calculates the fingerprint for a single molecule, and</p>
    <pre>fp2_Similarity [fprint1, fprint2]</pre>
    <p>which yields a similarity between 0 and 1.</p>
    </li></p>
    <p><li>Create methanol, and generate its fingerprint:
<pre>
atoms1 = sm_Build 'CO';
type_vector1 = aElement atoms1;
fprint1 = fp2_FingerPrint [atoms1, type_vector1, ['C','O','H']];
</pre>
</li></p>
    <p><li>Create ethanol, and generate its fingerprint:
    <pre>
atoms2 = sm_Build 'CCO';
type_vector2 = aElement atoms2;
fprint2 = fp2_FingerPrint [atoms2, type_vector2, ['C','O','H']];
    </pre>
    <p>Both fprint1 and fprint2 are vectors of length 189 (21*9), containing
    the occupancy of each distance bin for each pair: C-C, C-O, etc.  The
    occupancies of the distance bins sum to the number of atom pairs of
    the specified type.</p>
    </li></p>
    <p><li>Generate the similarity metric between ethanol and methanol:
    <pre>print fp2_Similarity [fprint1, fprint2];</pre>
     <p>yielding roughly 0.55.</p>
    </li></p>
    <p><li>To calculate the fingerprint using all atoms of the molecule:
    <pre>print fp2_FingerPrint [atoms2, one atoms2, 1];</pre>
    <p>which indicates that all atoms are of type 1, and that all atoms
    of type 1 should be considered.</p>
    </li></p>
    </ol>
    </li></p>
</ul>

<!-- webmake.svl -->

<hr noshade>
<h3><a name="WebMake">Output As HTML</h3></a>

<ul>

<p><li><b>File</b>: <tt>$MOE/sample/webmake.svl</tt></li></p>
<p><li><b>Description</b>: 

    A program that can output the contents of a MOE database as an HTML page,
    showing the rows and columns in a table. Molecule fields are shown as 2D
    diagrams and stored as PNG (Portable Network Graphic) image files.</li></p>  
<p><li><b>Interactive Instructions</b>:

    <ol>
    
    <p><li>Use the Database Viewer to open a small molecule MOE database of choice.</li></p>
    
    <p><li>Run the program with:
    
<pre>run '$MOE/sample/webmake.svl'</pre>
    
    <p align="center"><img src="samples/webmake1.png" /></p>
    </li></p>
    <p><li>If necessary, change the default output directory, HTML filename and
    title.
    </li></p>
    <p><li>Adjust the dimensions (in pixels) for the molecular graphics.
    </li></p>
    <p><li>If the molecule-type fields already have 2D coordinates that are
    appropriate for diagrammatic display, uncheck the Depict 2D coordinates box.
    Otherwise the coordinates will be regenerated for aesthetic purposes.
    </li></p>
    <p><li>To limit the selection of fields or entries, toggle the Selected fields
    only and Selected entries only checkboxes.
    </li></p>
    </ol>
    </li></p>

<p><li><b>Batch Instructions</b>:

    <ol>
    
    <p><li>Load the program:
    
<pre>load '$MOE/sample/webmake.svl'</pre>
	</li></p>
    <p><li>Call the function:
    
<pre>webmake_BuildFiles [dbfn, opt]</pre>
	
    <p><tt>dbfn</tt> is the MOE database from which the web files will be
    built.  <tt>opt</tt> is a tag vector which contains various parameters, all
    of which are optional. See the SVL source of
    <tt>$MOE/sample/webmake.svl</tt> for a detailed description of the options.
    </p></li></p>
    </ol>
    </li></p>

<p><li><b>Output</b> : 

    <p>The following fragment is an example of the output,
    rendered with a browser:</p>

    <p style="text-align:center"><img src="samples/webmake2.gif" /></p>
</li></p>

</ul>

<!-- jdbcview.svl -->

<hr noshade>
<h3><a name="JDBCViewer">JDBC Database Viewer</a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/jdbcview.svl</tt></li>
    <p><li><b>Description</b>: 
<p>
Displays database information
obtained from third party relational database applications
such as IBM DB2, Oracle,
Microsoft SQL Server, PostgreSQL, MySQL, and SQLite.
<b>JDBC</b> is an acronym for
<i>Java Database Connectivity</i>, the protocol used to
make the connection to the external databases.
Tables and fields defined in the databases are
displayed and can be imported into MOE database formats using SQL (Standard
Query Language) commands.
Note that in order to use this application, Java must be installed and
the proper <a href="settings.htm#Environment">environment variables</a> must be set.</p>
</li></p>
    <p><li><b>Instructions</b>: 
<p>To open the JDBC Database Viewer, run the program with</p>

<pre>run '$MOE/sample/jdbcview.svl'</pre>

<p>The following panel appears:</p>

<p><center><img src="samples/jdbc_panel.png"></center></p>

<p>The <b>Database Alias</b> menu shows all defined JDBC Database Connection
information.  If no JDBC connection information has been defined, then the menu
will show <nobr><tt>'(no server defined)'</tt></nobr>.  The JDBC database
connection information can be set either directly in the <tt>.moe-rc</tt> file
or by using the 
<a href="../appendix/settings.htm#Config">Configuration Options Panel</a>.
If connection data is defined, the drop-down
menu will display the <i>alias</i> for each defined connection.  Choose the
database to be viewed and press <b>Connect</b> to establish the connection.</p>
</li></p>
<p>The JDBC Database Viewer consists of two lists and a textfield for entering
SQL commands.  SQL commands can be entered directly into the textfield;
selecting items in the lists will automatically
set the appropriate SQL command.
</p>

<p>To view the fields and data types of a particular table, select the table
name in the <b>Database Tables</b> list.  The fields contained in the selected
table are displayed in the <b>Table Fields</b> list.  The SQL command is
automatically updated to reflect the selected table.  In the following image,
a connection has been made
to an Oracle database and is currently displaying
the fields from the <tt>LIGAND</tt> table.  The SQL command is set to
retrieve all fields from this table.</p>

<p><center><img src="samples/jdbc_tables.png" alt="Connection to Oracle" />
</center></p>

<p>To create a MOE database, type in the SQL command (or select the
desired entries in the lists) and press <b>Save as MDB</b>.
A file prompter will ask for the new database file name and the created
database will be opened in a MOE Database Viewer.</p>



</li>
</ul>

<!-- q_estate.svl -->

<hr noshade>
<h3><a name="Kier">Kier Descriptors</h3></a>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/q_estate.svl</tt></li>
    <p><li><b>Description</b>:
    <p>This file contains over 100 new descriptors that all pertain to
    Kier's electrotopological state indices for atom types.  </p>

    <p><cite>
    Reference:
    Electrotopological State Indices for Atom Types: A Novel
    Combination of Electronic, Topological, and Valence State Information.
    Hall,L.H., Kier, L.B.
    Journal of Chemical Information and Computer Sciences, 1995, 35
    </cite></p>
    </li></p>

    <p><li><b>Instructions</b>:
    <ol>
	<li>Open a MOE Database.</li>
        <li>Load the file with:
	<pre>load '$MOE/sample/q_estate.svl'</pre>
	</li>
	<li>Open the QuaSAR-Descriptor panel 
	(<span class="menu">DBV | Compute | Descriptors | Calculate</span>) 
	from the Database Viewer.
	</li>

        <li>The codes of the Kier descriptors all begin with 'kS_*' or 'kC_*'.
	</li>
    </ol>
    </li></p>

</ul>

<!-- q_randic.svl -->

<hr noshade>
<h3><a name="Randic">Randic Descriptor</a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/q_randic.svl</tt></li>
    <p><li><b>Description</b>: QuaSAR descriptor used to calculate the Randic
Molecular Identification Number. This descriptor is based on the weighted
paths of the connectivity graph.
    <p><cite>
    Molecular Identification Numbers
    J. Chem. Inf. Comput. Sci. 1984, 24, 164-175.
    </cite></p>
    </li></p>
    <p><li><b>Instructions</b>: 
        <ol>
	<li>Open a MOE Database.</li>
        <li>Load the file with:</p>
	<pre>load '$MOE/sample/q_randic.svl'</pre>
	</li>
        <li>Open the QuaSAR-Descriptor panel 
	(<span class="menu">DBV | Compute | Descriptors | Calculate</span>) 
	from the Database Viewer.
	</li>
        <li>The Randic descriptor code is <tt>randicMID</tt>.</li>
        </ol>
	</li></p>
</ul>

<!-- q_maccs.svl -->

<hr noshade>
<h3><a name="MACCS">MACCS Key Count Descriptors</a></h3>
<ul>
    <li><b>File</b>: <tt>$MOE/sample/q_maccs.svl</tt></li>
    <p><li><b>Description</b>: Individual MACCS keys
	can be used as descriptors in QuaSAR model building.</p>
	</li>
    <p><li><b>Instructions</b>: 
        <ol>
	<li>Open a MOE Database.</li>
        <li>Load the file with:
	<pre>load '$MOE/sample/q_maccs.svl'</pre>
	</li>
        <li>Open the QuaSAR-Descriptor panel 
	(<span class="menu">DBV | Compute | Descriptors | Calculate</span>) 
	from the Database Viewer.
	</li>
        <li>The MACCS descriptor codes all begin with '<tt>MACCS</tt>'.</li>
        </ol>
	</li></p>
</ul>


<!-- cs4d.svl -->

<hr noshade>

<h3> <a name="4Dconf">4D Conformational Sampling </a></h3>
<ul>
    <li><b>File</b>: <tt>$MOE/sample/cs4d.svl</tt></li>
    <p><li><b>Description</b>:

    This sample demonstrates how to perform conformational sampling and 
    energy minimization in 4D coordinates.</li></p>

    <p><li><b>Instructions</b>:
        <ol>
	    <p><li>Type the following command to load the 4D conformational
	    sampling module in MOE:</p>
            <pre>load '$MOE/sample/cs4d.svl'</pre>
	    </li></p>
	    <li>To perform 4D energy minimization of a molecule, load the molecule into MOE and type:
	    <pre>CS4D []</pre>
	    </li>
	    <li>To perform 4D energy minimization of all compounds in a
	    molecular database:
	    <pre>CS4D_MDB 'dbname.mdb' </pre>
	    <p>The function minimizes the first molecule field in the database.
	    </p>
	    </li>
	</ol>
    </li></p>
</ul>

<!-- hmc.svl -->

<hr noshade>
<h3><a name="HMC">Hybrid Monte Carlo Simulation</a> </h3>
<ul>
    <li><b>File</b>: <tt>$MOE/sample/hmc.svl</tt></li>
    <p><li><b>Description</b>:

	Implementation of the Hybrid Monte Carlo simulation method.

    </li></p>
    <p><li><b>Instructions</b>:
	<ol>
	<p><li><a href="../apps/emin.htm">Minimize the energy</a>
	of the molecular system under study to an RMS gradient of 0.1
	or smaller.</p>
	</li>
	<p><li>Run the file using the following command:
	<pre>run '$MOE/sample/hmc.svl'</pre>
	<p>The following panel appears:</p>
	<p style="text-align:center"><img src="samples/hmc.png" /></p>
    </li></p>
	<p><li>Once all parameters are chosen, press <b>OK</b> to begin
	the simulation.</li></p>
	</ol>
    </li></p>
</ul>

<!-- iedyn.svl -->

<hr noshade>

<h3> <a name="Langevin">Langevin Implicit Euler Dynamics </a></h3>
<ul>
    <li><b>File</b>: <tt>$MOE/sample/iedyn.svl</tt></li>
    <p><li><b>Description</b>:
    This sample implements Langevin Implicit Euler dynamics.
    
    <p><cite> Reference: P.Derreumaux, T.Schlick
    Long Timestep Dynamics of Peptides by the Dynamics Driver Approach
    PROTEINS: Structure, Function, and Genetics 21:282-302 (1995)
    </cite></p>
    </li></p>
    <p><li><b>Instructions</b>:
    <ol>
	<p><li>Load a molecule in MOE.</li></p>
	<li>Load the Langevin Dynamics module:
	<pre>load '$MOE/sample/iedyn.svl' </pre>
	</li>
	<li>Start molecule animation by typing the following command:
	<pre>IE [] </pre>
	</li>
	<li>Use the Cancel menu to terminate the function.</li>
    </ol>
    </li></p>
</ul>

<!-- seal.svl -->

<hr noshade>
<h3><a name="Molsup">Molecular Superposition</a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/seal.svl</tt></li>
    <p><li><b>Description</b>: Performs an alignment of small molecules
    using geometry and electrostatic contacts. Creates a database of
    superpositions and their scores.

    <p><cite> Reference:
    An Alternative Method for the Alignment of Molecular
    Structures: Maximizing Electrostatic and Steric Overlap.
    Tetrahedron Computer Methodology, Vol. 3, No. 6C, pp615-633, 1990.
    </cite></p>
    </li></p>

    <p><li><b>Instructions</b>:
        <ol>
	    <li>Load the SEAL module:
	    <pre>load <tt>'$MOE/sample/seal.svl'</tt></pre></li>
	    <li>Load two molecules into MOE (e.g. 
	    <tt>Open '$MOE/sample/seal.moe'</tt>).
	    </li>
	    <li>Type <tt>SEAL []</tt> at the CLI.
	    <p style="text-align:center"><img src="samples/seal.png" /></p>
	    </li>

	    <li>Enter the name for the output database.</li>
	    <li>Press <b>OK</b> to start the procedure with all default values.
	    </li>
        </ol>
	</li></p>

</ul>

<!-- sealflex.svl -->

<hr noshade>
<h3><a name="Flexsup">Flexible Molecular Superposition</a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/sealflex.svl</tt></li>
    <p><li><b>Description</b>: Performs an alignment of small molecules
using geometry and electrostatic contacts. Bond angles may change during
the calculations. The resulting superposition appears in the MOE rendering
window.

    <p><cite> Reference:
    An Alternative Method for the Alignment of Molecular
    Structures: Maximizing Electrostatic and Steric Overlap.
    Tetrahedron Computer Methodology, Vol. 3, No. 6C, pp615-633, 1990.
    </cite></p>
    </li></p>

    <p><li><b>Instructions</b>: 
    
    <ol>
	<li>Load the SEAL Flex module:
	<pre>load '$MOE/sample/sealflex.svl'</pre></li>
	<li>Load two molecules into MOE (e.g. <tt>Open
	'$MOE/sample/seal.moe'</tt>).
	</li>

	<li>Type <tt>SEALFLEX []</tt> at the CLI.</li>
    </ol>
    </li></p>
</ul>

<!-- verlet.svl -->

<hr noshade>
<h3><a name="NVE">NVE Molecular Dynamics Simulation</a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/verlet.svl</tt></li>
    <p><li><b>Description</b>: 
    This SVL program loops indefinitely, performing velocity verlet integration
    steps.  At each iteration, the internal atom positions are updated
    so that the simulation is animated automatically.

    <span class='LaTeX'>
    \begin{align}
    r\left( t + \delta t \right) & =
          r\left( t \right)
        + v\left( t \right) \delta t
        + \delta t^2 a\left( t \right)
    \\
    a \left( t + \delta t \right) & = - \nabla U\left( r \left( t + \delta t \right) \right) / m
    \\
    v\left( t + \delta t \right) & =
          v\left( t \right)
        + \frac{1}{2} \delta t \left[
            a\left( t \right) + a\left( t + \delta t \right)
        \right)]
    \end{align}
    </span>

    </li></p>

    <p><li><b>Instructions</b>:
	<ol>
	<li>Load a molecule in MOE.</li>
	<li>Run the sample using:
        <pre>    run '$MOE/sample/verlet.svl'</pre></li>
	<li>Use the Cancel menu to stop the sample execution.</li>
	</ol>
    </li></p>
</ul>

<!-- dihmplot.svl -->

<hr noshade>
<h3> <a name ="dihmplot">Dihedral Energy Plot with Minimization</a> </h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/dihmplot.svl</tt></li>

    <p><li><b>Description</b>:   Given 4 atoms defining a torsion, this
	program
	performs an energy minimization at each dihedral angle increment
	(using torsion restraints) and produces a plot
	of the energy at equally spaced angles.
	</li></p>

    <p><li><b>Instructions</b>:
    <ol>
	<li>Before running this program, first select four atoms 
	    defining a torsion.</li>
	<li>Type the following command at the SVL 
	command line: <pre>    run '$MOE/sample/dihmplot.svl'</pre></li>

	<li>Once the calculation is complete, a plot panel will appear:
	<p style="text-align:center"><img src="samples/dihmplot.png" /></p>
	</li>
    </ol>
    </li></p>
</ul>

<!-- eht_ui.svl -->

<hr noshade>
<h3> <a name ="eht_ui">Extended Hueckel Theory Properties</h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/eht_ui.svl</tt></li>

    <p><li><b>Description</b>: 
	Computes Extended Hueckel properties and displays them
	in a table and annotates the atoms in MOE with the
	calculated pi bond orders.  The following are the
	tabulated values:

	<ul>
	    <li><b>Z</b>. Atomic number.
	    </li><li><b>h</b>. Hybridization (3=<i>sp</i><sup>3</sup>,
		2=<i>sp</i><sup>2</sup>, 1=<i>sp</i>).
	    </li><li><b>q</b>. Total charge.
	    </li><li><b>sigma</b>. Sigma charge.
	    </li><li><b>pi</b>. Pi charge.
	    </li><li><b>iH_q</b>. Charge on implicit hydrogens.
	    </li><li><b>ema</b>. H-bond acceptor strength.
	    </li><li><b>emd</b>. H-bond donor strength.
	    </li>
	</ul>
    </li></p>

    <p><li><b>Instructions</b>:
    <ol>
	<li>Load a molecule into MOE.</li>
	<li>Type the following command at the SVL 
	command line:
	<pre>    run '$MOE/sample/eht_ui.svl'</pre>
	</li>
	<li>In the Extended Hueckel Theory Properties panel,
	press <b>Apply</b>.
	<p style="text-align:center"><img src="samples/eht_ui_panel.png" />
	<img src="samples/eht_ui_mol.png" /></p>
	</li>
    </ol>
    </li></p>
</ul>

<!-- rism_hfe.svl -->

<hr noshade>
<h3> <a name ="rism_hfe">3D-RISM Hydration Free Energy (HFE)</a> </h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/rism_hfe.svl</tt></li>

    <p><li><b>Description</b>: 
	Computes 3D-RISM solutions for the set of input atoms,
	employing a theoretical model fitted on the David Mobley HFE
	set [Mobley&nbsp;2009], to produce non-polar HFE values.
	</li></p>

    <p><li><b>Instructions</b>:
    <ol>
	<li>Load a molecule into MOE.</li>
	<li>Assign partial charges (AM1-BCC preferred):
	<blockquote>
	<span class="menu">MOE | Compute | Prepare | Partial Charges</span>
	</blockquote>
	For the <b>Method</b>, choose <b>AM1-BCC</b>.  Press <b>OK</b>.
	<li>Type the following command at the SVL 
	command line:

	<pre>    run ['$MOE/sample/rism_hfe.svl', [output_filename, <i>atoms</i>]]
	</pre>

	where <i><tt>atoms</tt></i> is either <tt>Atoms[]</tt>, if all atoms
	are to be used in the calculation, or <tt>SelectedAtoms[]</tt>
	if a subset of atoms are to be used.  In the latter case, first
	select the atoms then issue the command.  <tt>output_filename</tt>
	is the filename prefix used to name the output files generated
	by the calculation.
	</li>

	<li>Once the calculation is complete, the computed
	hydration free energy will be printed to the SVL Commands window.
	</li>
    </ol>
    </li></p>
</ul>

<blockquote>
    [Mobley&nbsp;2009] Mobley, D., Bayly, C., Cooper, M., Shirts, M., and
	Dill, K.; Small molecule hydration free energies in explicit solvent:
	An extensive test of fixed-charge atomistic simulations;
	<i>J. Chem. Theory. Comput. 5(2)</i>, (<b>2009</b>), 350&ndash;358.
</blockquote>


<!-- md_tools.svl -->

<hr noshade>
<h3> <a name ="mdtools_animate">Molecular Dynamics Trajectory Animation Panel</a> </h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/md_tools.svl</tt></li>

    <p><b>Description</b>: 
    An example of a trajectory animation panel.
    </p></li>
    <p><b>Pre-Requisites</b>:
    <pre>A molecular dynamics trajectory.</pre></p></li>

    <li><b>Instructions</b>:
    <ol>
	<li>Type the following commands at the SVL command line:</li>
    <pre>
    load '$MOE/sample/md_tools.svl'<br>
    MD_Animate []
    </pre>
    <li>In the panel, enter the trajectory root or browse to a trajectory file.
    </li>
	<p style="text-align:center"><img src="samples/md_animate_panel.png" />
	</p>
    <li>If a reference molecule is detected, it will be auto-loaded.  Otherwise,
    the reference molecule for the trajectory needs to be manually selected.
    </li>
    <li>Press play button to begin playback.</li>
    </ol>
    </li>
</ul>
<hr noshade>
<h3> <a name ="mdtools_qp">Molecular Dynamics Trajectory Quick Play</a> </h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/md_tools.svl</tt></li>

    <p><b>Description</b>: 
    Demonstrates the use of MD Framework functions to animate a MD trajectory.
    </p>
    <p><b>Pre-Requisites</b>:
    <pre>A molecular dynamics trajectory.</pre></p>

    <li><b>Instructions</b>:
    <ol>
	<li>Type the following commands at the SVL command line:</li>
    <pre>
    load '$MOE/sample/md_tools.svl'<br>
    MD_QuickAnimate [ root, opt ]
    </pre>
    Where <tt>root</tt> is the root of your trajectory name.
    <p>The option block is a tagged vector supporting the following tags:
    <blockquote><dl>
    <dt><tt>interval:</tt> <i>integer</i>
    <dd>Specifies that every n<sup>th</sup> frame should be displayed.
    [Default: 1]
    <dt><tt>delay:</tt> <i>float</i>
    <dd>The minimum time in seconds to pause between frames. [Default: 0.01]
    <dt><tt>startFrame:</tt> <i>integer</i>
    <dd>Begin visualization starting with this frame index. [Default: 1]
    <dt><tt>endFrame:</tt> <i>integer</i>
    <dd>Terminate animation on specified frame. [Default: INT_MAX]
    <dt><tt>reference:</tt> <i>filename</i>
    <dd>Specify the reference molecule file.  This over-rides the auto-detected
    molecule.</tt></a>.
    <dt><tt>format:</tt> <i>token</i>
    <dd>See <a href="../moe/fcnref/mdxfcn.htm#MD_Open"><tt>MD_Open</tt></a>.
    <dt><tt>forceroot:</tt> <i>integer</i>
    <dd>See <a href="../moe/fcnref/mdxfcn.htm#MD_Open"><tt>MD_Open</tt></a>.
</pre>
    </ol>
    </li>
</ul>

<hr noshade>
<h3> <a name ="mdtools_moevie">Trajectory MOEvie Generation</a> </h3>
<ul>
    <li><b>File</b>: <tt>$MOE/sample/md_tools.svl</tt></li>

    <p><b>Description</b>: 
    Demonstrates the use of MD Framework functions to directly generate a MOEvie
    from MD trajectories.</p>
    <b>Pre-Requisites</b>:
    <pre>
    A molecular dynamics trajectory
    MEncoder
    A Media Player
    </pre>

    <li><b>Instructions</b>:
    <ol>
    <li>Ensure that mencoder is properly configured.</li>
    <pre>
    GetRC ['mencoder.path']
    SetRC ['mencoder.path', '<path to mencoder>/mencoder']</pre>
    <li>Type the following commands at the SVL command line:</li>
    <pre>
    load '$MOE/sample/md_tools.svl'
    MD_GenerateMOEvie [ trajectory, moevie, opt ]</pre>
    <p>Where <tt>trajectory</tt> is one or more trajectory root(s) and/or 
    trajectory segments.</p>
    <p><tt>moevie</tt> is the output moevie name to generate.</p>
    <p>The option block is a tagged vector supporting the following tags:
    <blockquote><dl>
    <dt><tt>reference:</tt> <i>token(s)</i>
    <dd>Load the indicated file(s) for reference and rendering
    <dt><tt>renderFrame:</tt>integer(s)</i>
    <dd>Frame(s) where the paired reference should be loaded. 
    [Default: startFrame]
    <dt><tt>moevieFormat:</tt> <i>token</i>
    <dd>Format to use for encoding. <tt>'x264', 'MJPEG', 'WMV', 'Xvid'</tt>
    [Default: 'x264']
    <dt><tt>interval:</tt> <i>integer</i>
    <dd>Read every nth frame. [Default: 1]
    <dt><tt>startFrame:</tt> <i>integer</i>
    <dd>First frame to render. [Default: 1]
    <dt><tt>endFrame:</tt>  <i>integer</i>
    <dd>Last frame to render. [Default: INT_MAX]
    <dt><tt>tmp_dir:</tt> <i>token</i>
    <dd>Path wheere PNG directory should be generated. [Default: $TMP]
    <dt><tt>imageSize:</tt> <i>[integer, integer]</i>
    <dd>[width, height] image size for rendering. [Default: [640, 480]]
    <dt><tt>hq:</tt> <i>boolean</i>
    <dd>High quality ribbon rendering. [Default: 0]
    <dt><tt>aa:</tt> <i>boolean</i>
    <dd>Anti-aliasing. [Default: 1]
    <dt><tt>keepImages:</tt> <i>boolean</i>
    <dd>Keep intermediate png files. [Default: 0]
    </blockquote>
    <li>On completion, the specified MOEvie may be opened by an external
    media player.</li>
    </ol>
    </li>
</ul>



<hr noshade>
<h3> <a name ="mdtools_analysis">Molecular Dynamics Trajectory Analysis Tools</a> </h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/md_tools.svl</tt></li>

    <p><li><b>Description</b>: 
    Demonstrates the use of MD Framework functions to animate MD trajectories
    and simultaneously compute descriptors for each frame.  The solvent
    molecules are ignored for the descriptor calculation.
	</li></p>
	
    <p><li><b>Pre-Requisites</b>:
    <pre>A molecular dynamics trajectory.</pre></li></p>

    <p><li><b>Instructions</b>:
    <ol>
	<li>Type the following commands at the SVL command line:
    <pre>
    load '$MOE/sample/md_tools.svl'<br>
    MD_DescriptorCalc [ root, descriptorList, opt ]</pre>
    Where <tt>root</tt> is the root of your trajectory name.  
    <tt>descriptorList</tt> is any combination of appropriate descriptors as
    listed by <tt>DescriptorList[]</tt>.</li>
	<li>Once the calculation is complete, a database of the computed
    descriptors will be created and opened. Note that the descriptors are calculated only for non-solvent chains.
	</li>
    </ol>
    </li></p>
</ul>

<!-- makelabel.svl -->


<!-- alphabuild.svl -->

<hr noshade>

<h3><a name="Alpha">Protein Reconstruction From Alpha-Carbon Traces </h3> </a>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/alphabuild.svl</tt></li>

    <p><li><b>Description</b>: Routines to build proteins from 
alpha-carbon traces based on the method developed by P. Correa. 

    <p><cite> Reference: The building of protein 
    structures from alpha-carbon coordinates,
    PROTEINS: Structure, Function, and Genetics, 7, 266-377, 1990.</cite></p>
    </li></p>

    <p><li><b>Instructions</b>: 
    <ol>
	<li>Load the protein <tt>$MOE/sample/mol/1crn.pdb</tt>
	into MOE.</li>

	<li>Select all alpha-carbons.</li>

	<li>Invert the selection. The selection set now comprises all atoms
	that are not alpha-carbons.</li>

	<li>Delete all selected atoms.</li>

	<li>Type the following command at the SVL Command Line to
	 reconstruct the protein from its alpha traces:
	<pre>    run '$MOE/sample/alphabuild.svl'</pre></li>

    </ol>
    </li></p>
</ul>

<!-- occo.svl -->

<hr noshade>
<h3> <a name="OCCO"> OCCO Plot </a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/occo.svl</tt></li>
    <p><li><b>Description</b>:
    Function <tt>Protein_OCCO_Plot</tt> computes the backbone OCCO dihedral
    angle and puts it in the ResScalar field of each residue (normalized to +/-
    1).  This plot can then be viewed in the Sequence Editor.

    <p><cite> Reference:
    Pseudotorsional OCCO backbone angle as a single descriptor of protein
    secondary structure. S. Laiter, et. al, Protein Science, Vol. 4, 1995
    </cite></p>
    </li></p>

    <p><li><b>Instructions</b>: 
        
    <p>Run the program with:</p>
    <pre>run '$MOE/sample/occo.svl'</pre>

    <p><b>Note:</b> This function assumes that all chains consist of amino 
    acids containing backbone carbonyl C and O atoms named 'C' and 'O'.</p>
    </li></p>
</ul>

<!-- distplot.svl -->

<hr noshade>
<h3> <a name="DistPlot">Alpha Carbon Distance Plot</a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/distplot.svl</tt></li>
    <p><li><b>Description</b>: 
Draws a matrix of the distances between alpha carbons in a protein chain.
    <p style="text-align:center">
    <img src="samples/distmat.png" alt="Distance Matrix" />
    </p>

    </li></p>

    <p><li><b>Instructions</b>:
    <ol>
        <li>Load a protein into MOE.</li>
	<p><li>Type the following command at the SVL Command Line:
	    <pre>run '$MOE/sample/distplot.svl' </pre>
    </li></p>
	<li>A prompter in the Sequence Editor allows a protein chain to be
	chosen.
	</li>
    </ol> 
    </li></p>
</ul>

<!-- pairpot.svl -->

<hr noshade>
<h3> <a name="Jack"> Mean Pair Potential Jackknife Test </a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/pairpot.svl</tt></li>
    <p><li><b>Description</b>: Implements a method that
    improves a simple pair potential of mean force in such a way
    that it recognizes native protein folds with high reliability.

    <p><cite> Reference:
    An Improved Pair Potential to Recognize Native Protein Folds.
    Aron Bauer and Anton Beyer.
    PROTEINS: Structure, Function and Genetics 18:254-261 (1994).
    </cite></p>
    </li></p>

    <p><li><b>Instructions</b>: 
    <ol>
	<p><li>Type the following command at the SVL Command Line:
	    <pre>run '$MOE/sample/pairpot.svl' </pre>
	    <p>A panel titled Mean Pair Potential Jackknife Test is
	displayed.</p>
	<p style="text-align:center">
	<img src="samples/s_pairpot.png" alt="Jackknife Panel" />
	</p>
	</li></p>
	<li>The PDB Code List is used to specify the file of PDB entries.
	All list entries will be used in the jackknife test.</li>
	<li>The Database File is used to specify the name of the
	database from which protein chains will be loaded.  </li>
	<li>The smoothing options let you select the smoothing treatment
	used in the test. You can use none, either or both of these
	smoothing treatments.</li>

    </ol> 
    </li></p>
</ul>

<!-- smics.svl -->

<hr noshade>
<h3> <a name="smics">Parallelized Stochastic Search</a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/smics.svl</tt></li>
    <p><li><b>Description</b>:
	A MOE/cluster (MOE/smp) multi-processor sample program for stochastic
	search of small molecules.
	</li></p>
    <p><li><b>Instructions</b>:
    <ol>
	<li>Create a file in your current working directory 
	called <tt>smics.smi</tt> which contains
	one smiles string per line.  A sample <tt>smics.smi</tt> can be
	found in the <tt>$MOE/sample</tt> directory.
	</li>
	<li>In either MOE or MOE/cluster (MOE/smp), run the file as follows:
	<pre>run '$MOE/sample/smics.svl'</pre>
	</li>

	<li>A MOE Database called <tt>smics.mdb</tt> will be created
	and opened so that you can see the results of the search as
	they are found.
	</li>
	</ol>
    </li></p>
</ul>

<!-- psearch.svl -->

<hr noshade>
<h3> <a name="psearch">Parallelized Homology Search</a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/psearch.svl</tt></li>
    <p><li><b>Description</b>: A MOE/cluster (MOE/smp) multi-processor sample program
    for homology search of protein sequences.  This program takes as
    input a Fasta file containing a set of query protein sequences.
    The output is a text file, located in MOE's current directory,
    containing one line for each hit of a query sequence to a family
    in MOE's family database.
    </li></p>

    <p><li><b>Instructions</b>:
    <ol>
	<li>Load the SVL file:<pre>load '$MOE/sample/psearch.svl'</pre></li>
	<li>In either MOE or MOE/batch run the function by typing the
	following at the SVL command line:
	<pre>svl&gt; mpu_psearch 'myfile'</pre>
	<p> where <tt>myfile</tt> is a Fasta file containing a set of
	query sequences.</li></p>
	<p><li>Output is written to a file <tt>myfile.log</tt> and
	errors are printed in the SVL Commands window.  A sample
	of the output file created after running <tt>mpu_psearch</tt>
	on <tt>'$MOE/sample/mol/p2genes500.fsa'</tt> may look like this:
	
<pre>
    SSO0044       PDB_1EHK.A   1.47e-032    Skip
    SSO0044       PDB_1AR1.A   2.37e-021    Skip
    SSO0050       PDB_1HZD.A   7.21e-020    Skip
    SSO0050       PDB_1EF8.A   8.81e-020    Skip
    SSO0053       PDB_1F3O.A   2.55e-031    Skip
    SSO0053       PDB_1G29.1   3.36e-022    Skip
    SSO0082       PDB_1B87.A   1.38e-003    8.71
    SSO0082       PDB_1BO4.A   1.93e-002    7.71
    SSO0090       PDB_1EOV.A   1.30e-076    Skip
    SSO0090       PDB_1KRS     6.61e-012    23.16
    SSO0091       PDB_1JJ2.F   5.15e-031    Skip
    SSO0091       PDB_1CK5.B   1.20e-002    10.74
</pre>

	<p>The columns of data show, in this order, the name of the query
	sequence, the PDB code of the best scoring structure from the family
	hit, the expectation value, and the Z-score.
	Note that if the E-value is lower than the default threshold then the
	Z-score calculation is skipped).</p>
	</li></p>
	</ol>
	<p>For detailed information on MOE's Homology Search application please
see <a href="../proteins/promodel.htm">Building 3D Protein Models</a>.</p>
    </li></p>
</ul>



<!-- grdemo.svl -->

<hr noshade>

<h3> <a name="grdemo"> 2D Graphics Demo </a></h3>
<ul>
    <li><b>File</b>: <tt>$MOE/sample/grdemo.svl</tt></li>

    <p><li><b>Description</b>: 
<p>    
The MOE 2D graphics features allow schematic drawings to be produced from a
concise alphabet of graphics primitives. These drawings can be rendered within
panels, exported to files, copied to the clipboard, or printed. This sample
program contains drawing instructions for several pages which showcase the range
and capabilities of the graphics primitives. It also provides a facility
for testing the file formats that are available for exporting the 2D
graphics objects.</p>
</li></p>

    <p><li><b>Instructions</b>: 
    <ol>
	<li>Type the following command at the SVL Command Line:
	<pre>    run '$MOE/sample/grdemo.svl' </pre>
	<p>The following panel will appear:</p>

	<p style="text-align:center">
	<img src="samples/grdemo.png" alt="2D Graphics Primitives" />
	</p>
	</li>
	<li>The default graphic, page 1, is shown above. To flip between pages,
	press the numbers labeled by <b>Graphics</b>.
	</li>

    	<li>To test the exporting abilities, make sure that the location given
	in <b>Directory</b> is suitable for writing a
	number of files, then press the <b>Export</b> button. For each of the
	pages of graphics, a file will be created for every available output
	format. Output formats include raster (PNG, JPG, GIF, BMP) and
	vector (EMF+, SVG, Grob) formats.
	</li>
    </ol>
    
    <p>For more information, see
    <a href="../svl/fcnref/grfcn.htm">Flat Graphics Object Functions</a>.</p>
</li></p>
</ul>

<hr noshade>

<h3> <a name="Gobj"> 3D Graphics Objects Demo </a></h3>
<ul>
    <li><b>File</b>: <tt>$MOE/sample/gobjdemo.svl</tt></li>

    <p><li><b>Description</b>: Running this program displays the Graphics
Objects Demo panel, which allows you to select a function defining a
graphic object and to modify the object's parameters on an interactive
basis.  The resulting graphic object is displayed in the MOE rendering
window.

<p><b>Note:</b> The objects are rendered using green lines and
triangles.  The direction of the normals is shown by red line segments
(optional).</p>


<p style="text-align:center">
<img src="gobjdemo/wire_spheres.png" alt="Wireframe Spheres in MOE" />
</p>
</li></p>

    <p><li><b>Instructions</b>: 
    <ol>
	<li>Type the following command at the SVL Command Line:
	<pre>run '$MOE/sample/gobjdemo.svl' </pre>
	<p>A panel titled Graphics Objects Demo is displayed.</p>
	<p style="text-align:center"><img src="samples/gobjdemo_panel.png" 
	alt="GOBJ Demo Panel" /></p>
	</li>
	<li>The Object section contains each available
	graphic object in MOE. Press a button to display the corresponding
	object in the MOE rendering window.
	</li>
	<li>The Show Normals toggle box controls the display of red arrows 
	indicating the graphic object normals. 
	</li>
	<li>The three wheels in the Center section are used
	to change the coordinates of the displayed object.
	</li>
	<li>The three wheels in the Axis section are used to change
	the height, declination from Z axis and angle about the Z axis
	of the currently displayed graphic object.
	</li>
	<li>The Radius scale determines the size of the displayed object.
	To modify this value, enter a number in the text field and
	press <tt>Enter</tt> or move the scale.
	</li>
	<li>The Quality scale controls the visual quality of the surface
	of the displayed objects. 
	</li>
	<li>The Replicate scale determines the number of graphic objects
	to display. 
	</li>
	<li>The Separation scale controls the amount of space between
	the replicated graphic objects.
	</li>
	<li>The SVL display area shows the SVL code used to create the
	currently displayed object.
	</li>
    </ol>
    </li></p>
</ul>




<p>Each object is defined by its basic shape and desired appearance
(solid, wireframe or polyhedral).  Wireframe objects are composed of
lines, whereas solid and polyhedral objects are composed of triangles that
form polyhedral facets.  Solid and polyhedral objects differ in their
normals:  while normals of a solid object are perpendicular to the
analytic surface of the object, normals of a polyhedral object are
perpendicular to each individual facet.

<table border="1" style="margin:1em auto;" class="titlebar"> 

<tr>
  <th>Object</th>
  <th>Solid </th>
  <th>Wireframe </th>
  <th>Polyhedral </th>

</tr><tr>
  <th>cross</th>
  <td>&nbsp; </td>
  <td><img src="gobjdemo/cross.gif" /> </td>
  <td>&nbsp; </td>

</tr><tr><th>circle</th>
  <td><img src="gobjdemo/circle.gif" /> </td>
  <td><img src="gobjdemo/wcircle.gif" /> </td>
  <td>&nbsp; </td>

</tr><tr>
  <th>box</th>
  <td><img src="gobjdemo/box.gif" /></td>
  <td><img src="gobjdemo/wbox.gif" /></td>
  <td>&nbsp; </td>

</tr><tr><th>sphere</th>
  <td><img src="gobjdemo/sphere.gif" /> </td>
  <td><img src="gobjdemo/wsphere.gif" /> </td>
  <td><img src="gobjdemo/psphere.gif" /></td>

</tr><tr>
  <th>cylinder</th>
  <td><img src="gobjdemo/cylinder.gif" /> </td>
  <td><img src="gobjdemo/wcylinder.gif" /> </td>
  <td><img src="gobjdemo/pcylinder.gif" /></td>

</tr><tr>
  <th>cone</th>
  <td><img src="gobjdemo/cone.gif" /> </td>
  <td><img src="gobjdemo/wcone.gif" /></td>
  <td><img src="gobjdemo/pcone.gif" /> </td>

</tr><tr>
  <th>arrow</th>
  <td><img src="gobjdemo/arrow_solid.png" /> </td>
  <td><img src="gobjdemo/arrow_wireframe.png" /></td>
  <td><img src="gobjdemo/arrow_polyhedral.png" /> </td>

</tr><tr><th>cap</th>
  <td><img src="gobjdemo/cap.gif" /> </td>
  <td><img src="gobjdemo/wcap.gif" /></td>
  <td><img src="gobjdemo/pcap.gif" /> </td>
</tr>
</table>

<!-- gridcode.svl -->

<hr noshade>

<h3> <a name="Gridcode"> Grid Code Example </a></h3>
<ul>
    <li><b>File</b>: <tt>$MOE/sample/gridcode.svl</tt></li>

    <p><li><b>Description</b>:
    Demonstrates how to use grid-related SVL functions.

<p style="text-align:center">
<img src="samples/s_gridcode2.png" alt="Gridcode Surface" /> 
</p>
</li></p>
    <p><li><b>Instructions</b>: <br><br>
    <ol>
	<li>Load the aspirin molecule <tt>'$MOE/sample/mol/asa.mol'</tt>
	into MOE.</li>

	<li>Type the following command at the SVL Command Line:
	<pre>run '$MOE/sample/gridcode.svl' </pre>
	<p>A panel titled Grid Surfaces SVL Demo is displayed.</p>
	<p style="text-align:center"><img src="samples/s_gridcode1.png" 
	alt="Grid Code Panel" /></p>

	</li>
	<li>Modify the various parameters of the program and press 
	Apply to see the corresponding surface in the MOE rendering window.
	</li>
    </ol>
    </li></p>
</ul>

<!-- windemo.svl -->

<hr noshade>

<h3><a name="Window"> SVL Window Toolkit Demo </a></h3>

<ul>
    <li><b>File</b>: <tt>$MOE/sample/windemo.svl</tt></li>
    <p><li><b>Description</b>:
    <p>Displays a panel showing the different widgets available in the 
    <a href="../svl/svlgui/wt_intro.htm">SVL Window Toolkit</a>
    and their respective
    attributes.  Adjusting the attributes allows you to
    see how different attribute values affect the widget layout.</p>

    <p style="text-align:center"><img src="samples/s_windemo.png" /></p>
    </li></p>
    <p><li><b>Instructions</b>: 
    Run the sample with 
	<pre>run '$MOE/sample/windemo.svl'</pre>
    <p>and follow the instructions.</p>
    </li></p>
</ul>

<hr noshade>


<a name="QuickRef">
<h2>Sample Programs Quick Reference</h2>
<dir>
<table>
<tr>
    <td><a href="#Alpha">alphabuild.svl</a></td>
    <td>Protein Reconstruction from Alpha Carbon Trace</td>
</tr><tr>
    <td><a href="#Water">asurf.svl</a></td>
    <td>Water Accessible Surfaces</td>

</tr><tr>
    <td><a href="cliprg.htm">dbvclip.svl</a></td>
    <td>Clippgin R-Groups</td>
</tr><tr>
    <td><a href="#4Dconf">cs4d.svl</a></td>
    <td>4D Conformational Sampling</td>
</tr><tr>
    <td>csearch.svl</td>
    <td>Systematic Conformational Search</td>
</tr><tr>
    <td><a href="#Charge">csurf.svl</a></td>
    <td>Surface Coloring by Charge</td>
</tr><tr>
    <td>dbimport.svl</td>
    <td>Legacy tool for importing data into a MOE database</td>
</tr><tr>
    <td>dgeom.svl</td>
    <td>Distance Geometry</td>
</tr><tr>
    <td><a href="#dihmplot">dihmplot.svl</a></td>
    <td>Dihedral Energy Plot with Minimization</td>
</tr><tr>
    <td><a href="#DistPlot">distplot.svl</a></td>
    <td>Alpha carbon distance matrix plot</td>
</tr><tr>
    <td><a href="#">echo.svl</a></td>
    <td>Example custom loaded SVL function</td>
</tr><tr>
    <td><a href="#EEM">eem.svl</a></td>
    <td>Partial Charge Calculation</td>
</tr><tr>
    <td><a href="#eht_ui">eht_ui.svl</a></td>
    <td>Extended Hueckel Theory Properties</td>
</tr><tr>
    <td><a href="#Electro">esurf.svl</a></td>
    <td>Electrostatic Surfaces</td>
</tr><tr>
    <td><a href="#Gobj">gobjdemo.svl</a></td>
    <td>3D Graphics Objects Demo</td>
</tr><tr>
    <td><a href="#Gobj">grdemo.svl</a></td>
    <td>2D Graphics Objects Demo</td>
</tr><tr>
    <td><a href="#Gridcode">gridcode.svl</a></td>
    <td>Grid Code Example</td>

</tr><tr>
    <td><a href="#Gridsurf">gridex.svl</a></td>
    <td>Grid Surface Demo</td>
</tr><tr>
    <td><a href="#">hello_world.svl</a></td>
    <td>Example custom run SVL function</td>
</tr><tr>
    <td><a href="#HMC">hmc.svl</a></td>
    <td>Hybrid Monte Carlo Simulation</td>
</tr><tr>
    <td><a href="#">hrot.svl</a></td>
    <td>Random Helix Rotation</td>
</tr><tr>
    <td><a href="#Langevin">iedyn.svl</a></td>
    <td>Langevin Implicit Euler Dynamics</td>


</tr><tr>
    <td><a href="#Molshape">molshape.svl</a> </td>
    <td>Molecular Shape Similarity</td>
</tr><tr>
    <td><a href="#Nabstack">nabstack.svl</a></td>
    <td>DNA Base Stacking</td>
</tr><tr>
    <td><a href="#OCCO">occo.svl</a></td>
    <td>OCCO Plot</td>
</tr><tr>
    <td><a href="#">open_template.svl</a></td>
    <td>Template SVL for adding a new file type to <b>File | Open</b></td>
</tr><tr>
    <td><a href="#Jack">pairpot.svl</a></td>
    <td>Mean Pair Potential Jackknife Test</td>
</tr><tr>
    <td><a href="#Pairsim">pairsim.svl</a></td>
    <td>Typed Pair Distance Similarity</td>
</tr><tr>
    <td><a href="#Typer">patty.svl</a></td>
    <td>Atom Typer</td>

</tr><tr>
    <td><a href="#psearch">psearch.svl</a></td>
    <td>Parallelized Homology Search</td>

</tr><tr>
    <td><a href="#Kier">q_estate.svl</a></td>
    <td>Kier Descriptors</td>
</tr><tr>
    <td><a href="#Randic">q_randic.svl</a></td>
    <td>Randic Descriptor</td>
</tr><tr>
    <td><a href="#MACCS">q_maccs.svl</a></td>
    <td>MACCS Key Count Descriptors</td>

</tr><tr>
    <td><a href="cdgen.htm">qcdgen.svl</a></td>
    <td>QuaSAR-CombiGen: Enumerative Combinatorial Library Generation</td>

</tr><tr>
    <td><a href="qcombi.htm">qcombi.svl</a></td>
    <td>QuaSAR-CombiDesign: Create Diverse Subset</td>

</tr><tr>
    <td><a href="qcrsel.htm">qcrsel.svl</a></td>
    <td>QuaSAR-Reagent: Reagent Activity Analysis</td>
    <td></td>

</tr><tr>
    <td><a href="#rism_hfe">rism_hfe.svl</a></td>
    <td>3D-RISM Hydration Free Energy</td>

</tr><tr>
    <td><a href="#Molsup">seal.svl</a></td>
    <td>Molecular Superposition</td>
</tr><tr>
    <td><a href="#Flexsup">sealflex.svl</a></td>
    <td>Flexible Molecular Superposition</td>

</tr><tr>
    <td><a href="#smics">smics.svl</a></td>
    <td>Parallelized Stochastic Search</td>
</tr><tr>
    <td><a href="#">sniffer.svl</a></td>
    <td>Sniffer Pseudodynamics</td>
</tr><tr>
    <td><a href="#Solv">vajdaweng.svl</a></td>
    <td>Solvation Free Energy from Surface Area</td>
</tr><tr>
    <td><a href="#NVE">verlet.svl</a></td>
    <td>NVE Molecular Dynamics Simulation</td>
</tr><tr>
    <td><a href="#WebMake">webmake.svl</a></td>
    <td>Output Database as HTML</td>
</tr><tr>
    <td><a href="#Window">windemo.svl</a></td>
    <td>SVL Window Toolkit Demo</td>
</tr>
</table>
</dir>


<noscript>
	<hr noshade="noshade" />
	<font size="2"> Copyright &copy; 1997&ndash;2015
	<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a> </font>
</noscript>
<div id="MoeFooter"></div>
</body>
</html>
