<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!	se.htm		Sequence Editor GUI chapter
!!-->

<!--
!!    MOE Online Manuals
!!    COPYRIGHT (C) 1997-2015
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

<script type="text/javascript" 
src="../../include/jsincludes_moe.js"></script>

<link rel="stylesheet" type="text/css"
href="../../include/manstyle.css" />


<meta panel Sequence Editor>
<meta panel Color Residues>

<meta keywords Selection Prompter>
<meta keywords Residue Selector>

<title>Sequence Editor</title>
</head>

<body>
<div id="MoeHeader"></div>
<noscript>
	<span class="warning">Warning: JavaScript is disabled. This page will not display correctly.</span>
	<h1 class="title">MOE Documentation</h1>
	<hr noshade="noshade" />
	<style>.LaTeX {color: #C08080;}</style>
</noscript>

<p>This document introduces the Sequence Editor and provides an
overview of its functionality.  If you are looking for information on a
specific menu or command, please refer to the
<a href="#MenuBar">Sequence Editor Menu Bar</a>.</p>

<h6 class="h-two">Contents</h6>
<div id="toc-list"><script>var gman_OrderedListTOC=false; </script></div>


<a name="Introduction"></a><h2>Introduction</h2>

<p>The Sequence Editor serves
as the primary launch point for MOE's protein modeling and protein structure
analysis commands. It can also be used to organize and manage residues, chains,
and tag groups. 
To open the Sequence Editor from the
MOE Window, select
<b>Sequence Editor</b> from the <b>Window</b> menu.
Alternatively, press the <b>SEQ</b> button in the upper
right corner of the MOE Window.
</p>


<p>In MOE, a molecular system consists of one or more chains,
possibly organized by chain tag into <i>tag groups</i>.
Each chain
consists of zero or more residues, and each residue consists of zero or more
atoms.
The Sequence Editor can be used to organize and manage these residues, chains,
and tag groups.  Note that
chains and tag groups can also be managed using
the <a href="system.htm">System Manager</a>.
</p>

<p>
The Sequence Editor and the
<a href="mainwin.htm">MOE Window</a> work in concert to present
complementary views of the molecular system.  The MOE Window
enables visualization and manipulation of the system at the atomic level,
whereas the Sequence Editor allows for visualization and
modification of molecules at the chain and residue levels.
Special annotation
operations, accessible from the Sequence Editor, act at both the
atom and residue levels, visually linking the atom and residue views.
Atom, residue, and chain selection are by default
synchronized so that selecting a residue in the Sequence Editor
will automatically cause the atoms of the residue to also be selected.
</p>

<p>The Sequence Editor displays the current molecular system as a residue
schematic, where each
chain is displayed as a list of residues.
For instance, the figure below shows a residue schematic of
<tt>$MOE/sample/mol/serine_prot.moe</tt>, a collection of serine
proteases.</p>

<p style="text-align:center"><img src="se/se_serineprot.png"
alt="Sequence Editor" /></p>

<p>Chain numbering is reflected in the order in which
chains are displayed in the Sequence
Editor.  Chain numbers are shown in the column labeled <b>Chain</b> and
are always displayed in ascending order.
Consecutive chains having the same chain tag are shown
associated with a common tag block.
The chain label always displays the chain number, but can also include
the chain name and header.  Control of the contents
of chain label can be found in the Chain popup menu (right-click on
<b>Chain</b>).
</p>

<p>Residues composing the chains are drawn with either
<a href="#ThreeLetterCode">1- or 3-letter residue codes</a>,
depending on the selected display mode.
The horizontal alignment ruler
in the Sequence Editor indicates the alignment position of residues,
and can be made to wrap to fit the current Sequence Editor
window size.
A given alignment position in the residue schematic
is referred to as a <i>residue column</i>.
Residue UID numbers and insertion codes can be added to
the display, along with a variety of other
<a href="#ResidueAnnotations">annotations</a>.
</p>

<p>To close the Sequence Editor, choose
<b><span class="nobr">SE | File | Quit</span></b>.
The Cancel menu, in the top right corner of the Sequence Editor window,
allows you to cancel tasks running in MOE.  Note that the
MOE Window and Database
Viewer also provide Cancel menus.  Next to the Cancel menu, the
<b>SVL</b> and <b>MOE</b> buttons raise the
<a href="svlcommands.htm">SVL Commands window</a> and MOE Window,
respectively.</p>

<a name="Components_of_the_Sequence_Editor"></a><a name="Components"></a>
<h2>Components of the Sequence Editor</h2>

<p>The Sequence Editor comprises a menu bar, SVL Command Line,
residue display area, and footer.
</p>

<p>
<center>
<img src="se/se_components.png" alt="Components of the Sequence Editor" />
</center>
</p>

<p>
The Sequence Editor also has horizontal and vertical sliders for
scrolling the residue display.  These sliders each provide a popup menu
for moving to the beginning or end positions of the slider, and for
restoring the previous (most recent) scroll position.
</p>

<a name="Sequence_Editor_Menu_Bar"></a><a name="MenuBar"></a>
<h3>Sequence Editor Menu Bar</h3>

<p>The Sequence Editor menu bar
provides direct access to commands, applications and tools,
some of which can also be found in the
<a href="mainwin.htm#MOEWindow">MOE Window</a> (e.g. Protein Builder).</p>


<table width="100%" cellpadding="5">
<td valign="top" nowrap="nowrap" width="25%">

<fieldset>
    <legend> <b>File</b> </legend>

    New<br />
	&nbsp;&nbsp;<a href="../../mdb/db.htm">Database</a><br />
	&nbsp;&nbsp;<a href="../../apps/ph4_query.htm">Pharmacophore Query</a>
	<br />
	&nbsp;&nbsp;<a href="ted.htm">Text Editor</a><br />
	&nbsp;&nbsp;<a href="../../apps/moevie.htm">MOEvie</a><br />
    <a href="fileopen.htm">Open</a><br />
    <a href="../../apps/filesave.htm">Save</a><br />
    <a href="mainwin.htm#ClosingTheSystem">Close</a><br />
    <a href="../../apps/print.htm">Print</a><br />
    Quit

</fieldset>

<br />

<fieldset>
    <legend> <b>Edit</b> </legend>

    <a href="mainwin.htm#Undo">Undo</a><br />
    <a href="mainwin.htm#Undo">Redo</a><br />
    <a href="#CopyPaste">Copy As</a><br />
    <a href="#CopyPaste">Copy</a><br />
    <a href="#CopyPaste">Paste</a><br />
    <a href="#CopyPaste">Paste Fragment</a><br />
    <a href="#DeletingChainsAndResidues">Delete</a><br />
    <a href="../../proteins/probuild.htm">Protein Builder</a><br />
    <a href="../../proteins/probuild.htm#PTM">PTM</a><br />
    <a href="#DNAbuilder">DNA Builder</a><br />
    <a href="#Rename">Rename Tag/Chain</a><br />
    <a href="#Renumber">Renumber</a><br />
    Split/Join Chains<br />
	&nbsp;&nbsp;<a href="#JoiningAndSplitting">Split Chains</a><br />
	&nbsp;&nbsp;<a href="#JoiningAndSplitting">Join Chains</a><br />
    <a href="#MovingChains">Move Chains</a><br />
<!--
	&nbsp;&nbsp;Top<br />
	&nbsp;&nbsp;Bottom<br />
	&nbsp;&nbsp;Before<br />
-->
    <a href="#SortChains">Order Chains</a><br />
<!--
	&nbsp;&nbsp;By Name<br />
	&nbsp;&nbsp;By Tag<br />
-->
<!--
	&nbsp;&nbsp;Chains<br />
	&nbsp;&nbsp;Residues<br />
-->

</fieldset>

<td valign="top" nowrap="nowrap" width="25%">
<fieldset>
    <legend> <b>Select</b> </legend>

    <a href="#Clear">Clear</a><br />
    Chains<br />
    &nbsp;&nbsp;<a href="#InvertChains">Invert</a><br />
    &nbsp;&nbsp;<a href="#ClearChains">Clear</a><br />
    &nbsp;&nbsp;<a href="#AllChains">All</a><br />
    &nbsp;&nbsp;<a href="#EmptyChains">Empty</a><br />
    &nbsp;&nbsp;<a href="#WaterChains">Water</a><br />
    &nbsp;&nbsp;<a href="#TagExtend">Tag Extend</a><br />
    &nbsp;&nbsp;<a href="#LetterExtend">Letter Extend</a><br />
    &nbsp;&nbsp;<a href="#ChainsWithSelectedAtoms">With Selected Atoms</a><br />
    &nbsp;&nbsp;<a href="#ChainsWithSelectedResidues">With Selected Residues</a><br />
    Residues<br />
	&nbsp;&nbsp;<a href="#InvertResidues">Invert</a><br />
	&nbsp;&nbsp;<a href="#ClearResidues">Clear</a><br />
	&nbsp;&nbsp;<a href="#All">All</a><br />
	&nbsp;&nbsp;<a href="#Empty">Empty</a><br />
	&nbsp;&nbsp;<a href="#ChainBreaks">Chain Breaks</a><br />
<!--
	&nbsp;&nbsp;<a href="#Sets">Sets</a><br />
-->
	&nbsp;&nbsp;<a href="#Conserved">Conserved</a><br />
<!--
	    &nbsp;&nbsp;&nbsp;&nbsp;<a href="#Identity">Identity</a><br />
	    &nbsp;&nbsp;&nbsp;&nbsp;<a href="#Similarity">Similarity</a><br />
	    &nbsp;&nbsp;&nbsp;&nbsp;<a href="#Structure">Structure</a><br />
	    &nbsp;&nbsp;&nbsp;&nbsp;<a href="#Identity">Property</a><br />
	    &nbsp;&nbsp;&nbsp;&nbsp;<a href="#Hydrophobic">Hydrophobic</a><br />
	    &nbsp;&nbsp;&nbsp;&nbsp;<a href="#Hydrophilic">Hydrophilic</a><br />
	    &nbsp;&nbsp;&nbsp;&nbsp;<a href="#Acidic">Acidic</a><br />
	    &nbsp;&nbsp;&nbsp;&nbsp;<a href="#Basic">Basic</a><br />
	    &nbsp;&nbsp;&nbsp;&nbsp;<a href="#Aromatic">Aromatic</a><br />
	    &nbsp;&nbsp;&nbsp;&nbsp;<a href="#Aliphatic">Aliphatic</a><br />
	    &nbsp;&nbsp;&nbsp;&nbsp;<a href="#Polar">Polar</a><br />
-->
	&nbsp;&nbsp;<a href="#FixedResidues">Fixed</a><br />
	&nbsp;&nbsp;<a href="#ConstrainedResidues">Constrained</a><br />
	&nbsp;&nbsp;<a href="#ChainInterfaces">Chain Interfaces</a><br />
	&nbsp;&nbsp;<a href="#ExtendResidues">Extend</a><br />
<!--
	    &nbsp;&nbsp;&nbsp;&nbsp;Bonded To<br />
	    &nbsp;&nbsp;&nbsp;&nbsp;Structure<br />
	    &nbsp;&nbsp;&nbsp;&nbsp;Chain<br />
	    &nbsp;&nbsp;&nbsp;&nbsp;Column<br />
	    &nbsp;&nbsp;&nbsp;&nbsp;Left<br />
	    &nbsp;&nbsp;&nbsp;&nbsp;Right<br />
-->
	&nbsp;&nbsp;<a href="#WithSelectedAtoms">With Selected Atoms</a><br />
	&nbsp;&nbsp;<a href="#OfSelectedChains">Of Selected Chains</a><br />
    Atoms<br />
	&nbsp;&nbsp;<a href="#AtomsOfSelectedResidues">Of Selected Residues</a><br />
	&nbsp;&nbsp;<a href="#AtomsOfSelectedChains">Of Selected Chains</a><br />
    <a href="#PTM-Sites">PTM Sites</a><br />
    <a href="#Set">Set</a><br />
    <a href="#SetCreate">Set&nbsp;Create</a><br />
    <a href="#SetDelete">Set&nbsp;Delete</a><br />
    <a href="sel_ui.htm">Selector</a><br />
    <a href="#Save">Save</a><br />
    <a href="#Restore">Restore</a><br />
    <a href="#Synchro">Synchronize</a>

</fieldset>

<td valign="top" nowrap="nowrap" width="25%">
<fieldset>
    <legend> <b>Annotate</b> </legend>

    <a href="#PredictSecondaryStructure">2o Structure</a><br />
    <a href="#Plot">Plot</a><br />
	&nbsp;&nbsp;<a href="../../proteins/propred.htm">Structure Propensity</a><br />
	&nbsp;&nbsp;<a href="../../proteins/propred.htm">Surface Accessibility</a><br />
	&nbsp;&nbsp;<a href="../../proteins/propred.htm">Stereochemical Quality</a><br />
    <a href="../../proteins/antibody.htm#MethodNumbering">Antibody</a><br />
    <a href="#AnnotateSequence">Sequence</a><br />
    <a href="../../proteins/propred.htm">Propensity</a><br />
    <a href="#ChainInfo">Chain Info</a>
</fieldset>

<br />

<fieldset>
    <legend> <b>Alignment</b> </legend>

    <a href="#ResetAlignment">Reset</a><br />
    <a href="#PackAlignment">Pack</a><br />
    <a href="#FixUnfix">Fix Residues</a><br />
    <a href="#FixUnfix">Unfix Residues</a><br />
    <a href="#Constraints">Constraints</a><br />
	&nbsp;&nbsp;<a href="#Constraints">Constrain Residues</a><br />
	&nbsp;&nbsp;<a href="#Constraints">Unconstrain Residues</a><br />
	&nbsp;&nbsp;<a href="#Constraints">Antibody Constraints</a><br />
	&nbsp;&nbsp;<a href="#Constraints">Automatic</a><br />
	&nbsp;&nbsp;<a href="#Constraints">Manage Constraints</a><br />
    <a href="../../proteins/align.htm">Align</a><br />
    <a href="../../proteins/align.htm#Monitor">Similarity</a><br />
    <a href="../../proteins/prosup.htm">Superpose</a><br />
    <a href="../../proteins/procore.htm">Consensus</a><br />
</fieldset>

<br />
<td valign="top" nowrap="nowrap" width="25%">

<fieldset>
    <legend><b>Protein</b></legend>

    Search<br />
  &#160;&#160;&#160;<a href="../../proteins/prosearch.htm">PDB Search</a><br />
  &#160;&#160;&#160;<a href="../../proteins/motifsearch.htm">Domain
	Motif</a><br />
  &#160;&#160;&#160;<a href="../../proteins/projsrch.htm">Project Search</a><br />
  <a href="../../apps/structprep.htm">Structure&nbsp;Preparation</a><br />
  <a href="../../proteins/probuild.htm">Protein Builder</a><br />
  <a href="../../proteins/promodel.htm">Homology Model</a><br />
  <a href="../../proteins/antibody.htm">Antibody Modeler</a><br />
  <a href="../../apps/proloop.htm">Loop&nbsp;Modeler</a><br />
  <a href="../../apps/proloop.htm">Linker&nbsp;Modeler</a><br />
  <a href="../../apps/prodesign.htm">Design</a><br />
  <a href="../../proteins/propatch.htm">Patch Analyzer</a><br />
  <a href="../../proteins/proprop.htm">Properties</a><br />
  <a href="../../apps/progeom.htm">Geometry</a><br />
  <a href="../../proteins/procontacts.htm">Contacts</a><br />

</fieldset>
</tr>
</table>

<a name="Command_Line"></a><a name="CommandLine"></a>
<h3>Command Line</h3>

<p>The <a href="svlcommands.htm">Command Line Interface (CLI)</a>
is where SVL commands are typed and issued.  While the Sequence Editor
has the focus, all keystrokes are automatically directed to the CLI
regardless of where the mouse is positioned within the window.</p>

<p>The command line area is also used as a prompt area for several MOE
applications and functions. For example, the
<a href="sel_ui.htm">Selection Prompter</a> occupies this area.
The prompt area may also be for entering data or simply for
displaying messages. In all cases, the prompt may be canceled by
pressing either the red 'X' or <tt>Esc</tt>. While the
Sequence Editor's CLI is
occupied by a prompt, the CLIs in other windows will function as usual.
</p>

<p>For more information on the Command Line Interface see
the <a href="svlcommands.htm">SVL Commands window</a>.</p>

<a name="Chain_Numbers"></a><a name="ChainNumbers"></a>
<h3>Tags and Chain Numbers</h3>

<p>Every chain is associated with a chain tag.  Chain tags
are located in the column under the Tag Header (<b>Tag</b>).
If consecutive chains share the same tag, the tag box will
span the height of the chains.  Pressing the Tag Header will
successively select then deselect all tags and hence all
chains.  Available on the Tag Header and all tags
is the <a href="#PopupMenus">Tag popup menu</a>.
</p>

<p><i>Chain numbers</i> are the boxes to the left of the residue display area
in the column headed by the Chain Header (<b>Chain</b>).
A chain is selected by pressing its chain number; this also results
in the associated chain tag being selected.
When the Chain Header is pressed successively, all chains are
alternately selected then unselected.
Available on the Chain Header and on all chain numbers
is the <a href="#PopupMenus">Chain popup menu</a>.
</p>

<p>The chain tag and number/name columns may be resized by click-dragging
the edge of the column. Double-clicking the column edges will auto-resize
the columns to fit the longest chain/tag label.
</p>

<div class="floatright">
<img src="se/se_chaincolor_popup.png" alt="Chain Color" />
<p class="caption">
Chain Color
</p>
</div>

<p>Chains are always numbered sequentially, and each chain number is
colored by its <i>chain color</i>.
When atoms are colored by chain color
<span class="nobr">(<span class="menu">MOE | Render | Atoms</span>)</span>,
this is the color that will be used.
Chain color can be set using the <a href="#ChainColor"><b>Chain Color</b></a>
command in the Chain popup menu.  The <b>T</b> option
colors the chain by tag; chains
of the same tag will thus be visually associated by color.
<b>#</b> colors the chain by number; i.e. chains will have
different colors, whether or not they have the same tag.
</p>

<a name="HideSolventLigand"></a>
<h3>Hiding/Showing Chains</h3>

<p>Chains recognized to contain only solvent or ligand residues
can be hidden by pressing the <b>-</b> button next to the Tag Header.
Hidden chains are denoted by a black fletch in the Chain column.
Pressing <b>+</b> restores the hidden chains to view.
</p>

<p>Any chain may be hidden using the
<b>Hide (SE)</b> option in either the chain or tag popup menu.
The <b>Show</b> and <b>Show All</b> options cause hidden
chains to be shown.  Note that the <b>+</b> and <b>-</b> buttons
only apply to ligand and solvent chains. Right-clicking on the
black fletch opens the hidden chain popup menu,
indicating the number of hidden chains represented by the
fletch and the possibility to reshow just these.
</p>

<a name="Residue_Display_Area"></a><a name="ResidueDisplayArea"></a>
<h3>Residue Display Area</h3>

<p>The residue display area comprises the column ruler, the header,
and the residue drawing area, including the residues themselves
and the empty space between the drawn residues.
Residue annotations, such as predicted secondary structure indicators,
residue UID numbers and INS codes, chain breaks, disulfides, and
constraints, are all displayed in the drawing area.
Consensus, similarity, and RMSD plots are displayed in the header area.
The <b>Residue</b> popup menu is located in the drawing area,
and the residue <b>Column</b> popup menu in the alignment ruler and header.
</p>

<h4>Header</h4>

<p>The area of the window between the alignment ruler and the
residue drawing area is referred to as the header and is used
to display similarity/percent identity, consensus, consensus
structure, and RMSD annotations.  These annotations are enabled
from the footer.  When the annotations are all disabled, the header
area is hidden.
</p>

<center>
<img src="se/se_ann2.png" alt="Sequence Editor Header" />
</center>

<a name="Footer"></a>
<h4>Footer</h4>

<p>The bottom part of the Sequence Editor window
is referred to as the footer and is used
to control the display of residues and of residue annotations
both in the residue drawing area and in the Sequence Editor header.
The components of the footer are described below.
</p>

<p>
<center>
<img src="se/se_footer.png" alt="Sequence Editor Footer">
</center>
</p>


<a name="Info_Area"></a>
<h4>Info Area</h4>

<p>In the leftmost part of the Sequence Editor footer,
informational messages are displayed.  If the mouse cursor
is positioned over a chain number or tag or over a residue,
the name and number of the chain or the name of the residue
is given, respectively.  Otherwise, the
number of residues and chains selected is reported.
Additional information is also shown if the mouse is over
residue and header annotations (e.g. specific type of
secondary structure, or per column values of RMSD plot, ...).
</p>

<a name="Consensus_Preferences"></a>
<h4>Consensus Preferences</h4>

<p>The Consensus Preferences panel is opened from the Sequence Editor
footer, and is used to control the display of consensus annotations
in the Sequence Editor header area.  The options are as follows:
</p>

<center>
<img src="se/footer_consensus.png" alt="Consensus Preferences Panel" />
</center>

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Footer | Consensus Preferences</th></tr>
	<tr>
	<th>Command</th>
	<th>Description</th>
	</tr>
    <tr>
    <td valign="top"><span class="menu">
	Consensus
    </span></td>
    <td valign="top">
	Hide/display the residue consensus in the Sequence Editor header.

	<ul>
	<li><b>None</b>. Turn off Consensus display.
	</li><li><b>Majority</b>. Display majority residue consensus.
	    The residue appearing most frequently is shown in the header.
	</li><li><b>Clustal-X</b>. Display residue consensus class based
	    on Clustal-X consensus calculation, classifying columns into
	    8 different types (hydrophilic, acidic, basic,  etc.).
	</li>
	</ul>

	<b>Note.</b> The consensus display setting is remembered between MOE sessions.

	<ul>
	<li><b>Color Consensus</b>. If on, the consensus display will be
	    colored by the Clustal-X consensus class color.
	</li>
	</ul>
    </td>
    </tr>

    <tr>
    <td valign="top"><span class="menu">
	Consensus | Structure
    </span></td>
    <td valign="top">
	Hide/display the consensus secondary structure information in the
	Sequence Editor header.
	    If <b>Master Chain</b> is
	    enabled, the consensus structure is calculated with respect to
	    the <i>master chain</i>,
	    otherwise it is the most common secondary structure
	    element across all residues in the residue column.
    </td>
    </tr>

    <tr>
    <td valign="top"><span class="menu">
	Plot
    </span></td>
    <td valign="top">
	Hide/display plots in the Sequence Editor header.

	<ul>
	<p>
	<li><b>Similarity</b>.
	    Hide/display residue similarity based on
	    substitution matrix (BLOSUM62) scores.  If <b>Master Chain</b> is
	    enabled, the similarity is calculated with respect to
	    the <i>master chain</i>,
	    otherwise it is calculated across all residues in the
	    residue column (sum of pairs).  The similarity plot and the
	    %-Identity plot share the same plot area.
	</li>
	</p>

	<p>
	<li><b>%-Identity</b>.
	    Hide/display percent residue identity.  If <b>Master Chain</b> is
	    enabled, the percentage of residues corresponding to the residue in
	    the <i>master chain</i> is shown,
	    otherwise the percentage of identical residues to the consensus
	    (most common) residue is plotted.  The %-Identity
	    plot and the similarity plot share the same plot area.
	</li>
	</p>

	<p>
	<li><b>RMSD</b>.
	    Hide/display RMSD of the C&alpha; atom positions of all
	    residues in that column.  If <b>Master Chain</b> is enabled,
	    the root mean square deviation is calculated with respect to
	    the corresponding residue in the <i>master chain</i>.
	</li>
	</p>

	</ul>
    </td>
    </tr>

    <tr>
    <td valign="top"><span class="menu">
	Similarity
    </span></td>
    <td valign="top">
	<ul>
	<p>
	<li><b>Use Master Chain</b>. If on, a reference chain called
	the <i>master chain</i> is used for calculations.
	The master chain is the first visible protein chain of the system.
	Otherwise, the calculations are made with respect to a
	value obtained from averaging over the residues in a residue column.
	</li>
	</p>

	<p>
	<li><b>Use * for Conserved Residues</b>.
	If on, conserved residues are represented by an asterisk (*).
	This helps to emphasize the residues which are not conserved.
	</li>
	</p>
	</ul>
    </td>
    </tr>

</table>
</center>
</p>

<a name="Residue_Color_Style"></a>
<h4>Residue Color Style</h4>

<p>The Residue Color Style panel controls residue coloring.  It also
has additional options for selecting, fixing, and constraining residues.
The commands will be applied to all selected residues, or all
residues if there are none selected. If the <tt>&lt;Alt&gt;</tt>
key modifier is pressed, the chain selection (if any) is intersected
with the residue selection.
</p>

<center>
<img src="se/footer_residues.png" alt="Residue Color Style Panel" />
</center>

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Footer | Residue Color Style</th></tr>
	<tr>
	<th>Command</th>
	<th>Description</th>
	</tr>
    <tr>

    <td valign="top"><span class="menu">
	Clear<br />
	Invert
    </span></td>
    <td valign="top">
	Clear residue selection or invert current residue selection.
	<tt>Alt</tt>-Click will apply the operation to only selected residues
	within selected chains.
    </td>
    </tr>

    <td valign="top"><span class="menu">
	<a name="FixUnfix"></a>
	Fix<br />
	Unfix
    </span></td>
    <td valign="top">
	Fix or unfix the selected residues.  Fixed residues are
	colored pink and the corresponding column indicated by
	a pink background in the alignment ruler.  A fixed
	residue will block attempts to move residues of the
	same chain in the Sequence Editor.
	<tt>Alt</tt>-Click will apply the operation to only selected residues
	within selected chains.
    </td>
    </tr>

    <td valign="top"><span class="menu">
	Constrain<br />
	Unconstrain
    </span></td>
    <td valign="top">
	Applies or removes residue alignment constraints.
	Constrained residues are outlined in blue and
	connected by blue lines to other residues belonging
	to the same constraint.
	<tt>Alt</tt>-Click will apply the operation to only selected residues
	within selected chains.
    </td>
    </tr>

    <a name="ResidueColorStyle"><a/>
    <td valign="top"><span class="menu">
	Color
    </span></td>
    <td valign="top">
	Specify residue coloring style.
	Residues can be colored by chain or tag color,
	secondary structure, domain (automatically detected),
	average temperature factor, amino acid type,
	terminus (directional),C&alpha;-RMSD,
	sequence identity, sequence similarity (BLOSUM62),
	Clustal-X consensus, and several plain colors.
	Pressing <b>None</b> turns off coloring.
	<p>
	The color styles <b>Identity</b>, <b>%-Identity</b>,
	<b>Similarity</b>, <b>Clustal-X</b>, and <b>RMSD</b>
	depend on the sequence alignment. 
	The residue colors will update dynamically when
	the alignment changes.
	</p>
    </td>
    </tr>

</table>
</center>
</p>

<a name="Atom_Rendering_Style"></a>
<h4>Atom Rendering Style</h4>

<p>The Atom Rendering Style panel controls atom coloring.  It also
has additional options for selecting, hiding, and isolating atoms.
The render and color style will be applied to the atoms of all selected
residues, or all atoms if there are no residues selected.
If the <tt>&lt;Alt&gt;</tt>
key modifier is pressed, the chain selection (if any) is intersected
with the residue selection.
</p>

<center>
<img src="se/footer_atoms.png" alt="Atom Rendering Panel" />
</center>

<p>The Atom Rendering Style panel is identical to that
found in the
<a href="mainwin.htm#RenderingAtoms">MOE Window</a>,
with the addition of the following buttons:
</p>

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Footer | Atom Rendering Style</th></tr>
	<tr>
	<th>Command</th>
	<th>Description</th>
	</tr>
    <tr>

    <td valign="top"><span class="menu">
	Select<br />
	Unselect
    </span></td>
    <td valign="top">
	Select/deselect atoms of selected residues, or all atoms if
	there are no selected residues.
	<tt>Alt</tt>-Click will apply the operation only to atoms
	of selected residues within selected chains.
    </td>
    </tr>

    <td valign="top"><span class="menu">
	Hide<br />
	Show
    </span></td>
    <td valign="top">
	Hide/show atoms of selected residues, or all atoms if
	there are no selected residues.
	<tt>Alt</tt>-Click will apply the operation only to atoms
	of selected residues within selected chains.
    </td>
    </tr>

    <td valign="top"><span class="menu">
	Isolate<br />
	Center
    </span></td>
    <td valign="top">
	Isolate or center atoms of selected residues, or all atoms if
	there are no selected residues.
	The isolate operation shows the atoms in question and hides
	all others.  The center operation adjusts the view so that
	the atoms in question are centered and all visible within
	the view.
	<tt>Alt</tt>-Click will apply the operation only to atoms
	of selected residues within selected chains.
    </td>
    </tr>

</table>
</center>
</p>

<a name="Ribbon_Rendering_Style"></a>
<h4>Ribbon Rendering Style</h4>

<p>The Ribbon Rendering Style panel is identical to the
one found in the
<a href="mainwin.htm#RenderingBackbone">MOE footer</a>.
The commands will be applied to the selected residues, or all
residues if there are no residues selected.  If the <tt>&lt;Alt&gt;</tt>
key modifier is pressed, the chain selection (if any) is intersected
with the residue selection.
</p>

<center>
<img src="se/footer_ribbon.png" alt="Ribbon Rendering Panel" />
</center>

<a name="ResScalarPlots"></a>
<h4>Residue Scalar Plots</h4>

<div class="floatright">
<img src="se/se_plot_generic.png" alt="Plot Scalar Values" />
</div>

<p>Residue scalar plots display numerical residue
attributes on a bar graph aligned with and displayed
beneath the associated residue. Up to three scalar values
may be plotted at the same time.
Plotted values range from -1 to 1, with a horizontal black line marking
0. Plotted values exceeding this range are "clamped"; i.e. the values are
clipped.  Colors assigned to the plots can be configured in the
<b>Plot</b> popup in the Sequence Editor footer.
</p>

<p>The <span class="menu">SE | Annotate | Plot</span> menu includes
three built-in measurements:
</p>

<ul>
<p>
<li><b>Structure Propensity</b>.  Probability of the
sequence having a particular secondary structure type.
</li>
</p>

<p>
<li><b>Surface Accessibility</b>.
Ratio of the solvent accessible surface
areas of all amino acid residues in the system
to pre-calculated values for each amino acid type in a GLY-X-GLY
tri-peptide in extended conformation.

The midline represents exactly 0.25 of the standard exposed surface area for
each residue type. Bars above the midline (pointing up) indicate buried
residues, below the midline, exposed residues.
</li>
</p>

<p>
<li><b>Stereochemical Quality</b>.

Histograms denoting backbone geometry, Ramachandran outliers and rotamer
energies. The geometry score is based on a Z-Score combination
of all backbone bond length, angles and dihedral components (13 in total).
The data of all categories are scaled between -1 and 1 for display.
After scaling, Ramachandran outliers have values above 0.5, the allowed set,
values between -0.5 and 0.5, and the core, values
between -0.5 and -1, using a upper cutoff of 0.1.
The rotamer energies are scaled between -0.5 and -1 if lower than 0,
between -0.5 and 0.5 for energies from 0 and 3 kcal/mol,
and between 0.5 and 1 for energies
above 3 kcal/mol (upper cutoff is 5 kcal).

</li>
</p>
</ul>

<div class="floatright">
<img src="se/se_plot_surfacc.png"
alt="Scalar Plots of Surface Accessibility" />
</div>

<p>When any of these built-in measurements is chosen, the
Plot popup checkbox labels are adjusted to
reflect the quantity being plotted.
Toggling the checkboxes hides/displays the corresponding plot
in the Sequence Editor residue drawing area.
</p>

<a name="ResidueAnnotations"></a>
<h4>Secondary Structure Annotation</h4>

<p>
Display the DSSP secondary structure assignments [Kabsch 1983]
as color-coded annotations above the amino acid residues to which they apply.
<ul>
<li>Yellow arrows for &beta;-sheets.</li>
<li>Red cylinders for &alpha;-helices.</li>
<li>Blue thin cylinders for turns.</li>
</ul>
<p><span class="tip">Tip</span>&nbsp;&nbsp;
Moving the mouse cursor over the secondary structure element indicator will
show its exact type (e.g. 5-turn) in the <a href="#Footer">Info Area</a>;
</p>


<h4>Residue Number/UID Labeling</h4>

<p>
The residue numbers and insertion codes can be displayed
above the corresponding residues in the Sequence Editor.
Successive clicks cycle between turning off residue
numbering, displaying both residue UID numbers and
insertion codes, and displaying the residue number (sequential
index).
The button icon changes to reflect the current display mode.
</p>
<p><span class="tip">Tip</span>&nbsp;&nbsp;
For space reasons, not all residue numbers will be shown.
Only the numbers of every fifth residue or a residue
at a chain break will be shown. Insertion codes will be shown
by default in 3-letter but not 1-letter mode. To determine
the UID number of any residue move the mouse cursor over
the residue in question; the UID number and INS code will
appear with the residue name in the <a href="#Footer">Info Area</a>;
</p>

<a name="ThreeLetterCode"></a>
<h4>Single/Three Letter Codes</h4>

<p>Switches between 1- and 3-letter residue codes.  In
single-letter mode, all non-amino-acid/non-nucleic-acid
residues are marked with vertical brackets.
Waters are represented by <tt>W</tt>, other solvents by <tt>S</tt>,
ligands by <tt>L</tt>, metals by <tt>M</tt>, anions by <tt>-</tt>,
cations by <tt>+</tt>, and dummies by <tt>*</tt>.
</p>
<b>Note.</b> The single/three letter code setting is remembered between
MOE sessions.

<h4>Wrap Mode</h4>

<p>Switches the display between linear and wrapped modes.
In wrapped mode, the horizontal extent of the residue
drawing area is sized to match the width of the Sequence Editor window.
</p>
<b>Note.</b> The wrap mode setting is remembered between
MOE sessions.

<h4>Font Size</h4>

<p>Changes the display font size.  The alignment ruler will
also resize.  The <b>R</b> button resets
the font size to its default value.
</p>

<a name="Popup_Menus"></a>
<a name="PopupMenus"></a>
<a name="SequenceEditorPopupMenus"></a>
<h2>Sequence Editor Popup Menus</h2>

<p>The Sequence Editor features several popup menus with a variety of
commands.  Familiarity with these popups will give you a much greater
control over the display of atoms in the MOE Window and of residues in the
Sequence Editor.  Press the <a href="mouse.htm">mouse popup button</a>
over each of the following objects to display their respective popup menus.
</p>

<p>
The popup menus by default apply to all associated selected objects.
For example, the Chain popup menu applies by default to all selected
chains.  When a popup is raised, the objects to which it applies are
highlighted in green to provide a visual cue as to which objects will
be affected.  The number of affected objects appears in the popup header.
Pressing <tt>Ctrl</tt> while raising the popup menu causes the menu
commands
to apply only to the object directly beneath the cursor at that time,
independent of the current selection.
</p>

<p>
<center>
<table class="titlebar" width="95%">
	<tr>
	<th width="20%">Popup Menu</th>
	<th>Location </th>
	<th>Action</th>
	</tr>
	<tr> <td valign="top">
	<b>Selected Tags</b></td>
	<td valign="top">Over any chain tag or the Tag Header.</td>
	<td valign="top">Commands apply to all chains of the selected tags.
	With <tt>Ctrl</tt>-click, commands apply to only the chains of the
	tag beneath the cursor.
	</td>
	</tr>

	<tr> <td valign="top">
	<b>Selected Chains</b></td>
	<td valign="top">Over any chain number/name or the Chain Header.</td>
	<td valign="top">Commands apply to all selected chains.
	With <tt>Ctrl</tt>-click, commands apply to only the chain beneath
	the cursor.
	</td>
	</tr>

	<tr>
	<td valign="top">
	<b>Selected Residues</b></td>
	<td valign="top">Anywhere within residue drawing area.</td>
	<td valign="top">Commands apply to all selected residues.
	With <tt>Ctrl</tt>-click, commands apply to only the residue beneath
	the cursor.
	</td>
	</tr>

	<tr>
	<td valign="top">
	<b>Residue Column</b></td>
	<td valign="top">In the alignment ruler.</td>
	<td valign="top">Commands apply to the residues in the current
	column only. </td>
	</tr>
</table>
</center>

<p>The Sequence Editor chain, tag, residue, and residue column popup
menus all contain options to open the
<a href="#Atom_Rendering_Style">Atom Rendering Style</a>,
<a href="#Residue_Color_Style">Residue Color Style</a>, and
<a href="#Ribbon_Rendering_Style">Ribbon Rendering Style</a> panels.
<p>
<center>
<a name="All_Popups"></a>
<table class="titlebar" width="95%">
	<tr><th colspan="2">Tag + Chain + Residue + Column Popup</th></tr>
	<tr>
	  <th>Command</th>
	  <th>Action</th>
	</tr>

	<tr>
	<td valign="top"><span class="menu">Atoms</span></td>
	<td valign="top">
	Set the <a href="#Atom_Rendering_Style">Atom Rendering Style</a> of
	the corresponding chains/residues.  Additionally, atoms can be selected, shown,
	hidden, isolated, and centered.
	</td>
	</tr>

	<tr>
	<td valign="top"><span class="menu">Residues</span></td>
	<td valign="top">
	Set the <a href="#Residue_Color_Style">Residue Color Style</a> of
	the corresponding chains/residues.  Additionally, residues can be selected,
	fixed, or constrained.
	</td>
	</tr>

	<tr>
	<td valign="top"><span class="menu">Ribbons</span></td>
	<td valign="top">
	Set the <a href="#Ribbon_Rendering_Style">Ribbon Rendering Style</a> of
	the corresponding chains/residues.
	</td>
	</tr>
</table>
</center>
</p>

<p>
All popup menus except the residue column popup include other options,
detailed below.
</p>

<p>
<a name="Tag_Popup"></a>
<h3>Tag Popup Menu</h3>
<center>
<table class="titlebar" width="95%">
	<tr>
	  <th>Command</th>
	  <th>Action</th>
	</tr>

	<tr>
	<td valign="top"><span class="menu">Select</span></td>
	<td valign="top">
	<ul>
	<li><b>Invert</b>. Invert tag selection; chains of selected
	    tags will become unselected, and chains of unselected
	    tags will become selected.
	</li><li><b>Clear</b>.  Clear chain selection.
	</li><li><b>All</b>. Select all chains.
	</li><li><b>Save</b>. Save the current selection set.
	</li><li><b>Restore</b>. Restore the last selection set.
	</li>
	</ul>
	</td>
	</tr>

	<tr>
	<td valign="top"><span class="menu">Extend</span></td>
	<td valign="top">
	Extend chain selection:
	<ul>
	<li><b>Tag</b>. Select all chains of selected tags.
	</li><li><b>Letter</b>.  Select all chains having the same
	    chain letter as any chain in a selected tag.
	</li>
	</ul>
	</td>
	</tr>

	<tr>
	<td valign="top"><span class="menu">Color</span></td>
	<td valign="top">
	<div class="floatright">
	<img src="se/se_chaincolor_popup.png" alt="Chain Color" />
	</div>
	Set the color of the chains in the selected tags with the
	<b>Chain Color</b> panel. The <b>T</b> option
	colors the chains by tag; <b>#</b> colors the chains by number.
	</td>
	</tr>

	<tr>
	<a name="SetTag"></a>
	<td valign="top"><span class="menu">Rename</span></td>
	<td valign="top">
	Opens a text field popup in the CLI prompt for changing the chain tag
	of all selected tags.  Enter the new chain tag and press
	<b>OK</b> to set the specified tag.
	An option allows to switch to setting chain name, header, and group
	labels instead.
	</td>
	</tr>

	<tr>
	<td valign="top">
	    <span class="menu">Hide (SE)</span>
	</td>
	<td valign="top">
	Hides the selected tags and the corresponding chains
	from view in the Sequence Editor.
	Hidden chains are
	indicated in the Chain column as black fletches.
	These markers have an associated popup which reports
	the number of hidden chains at that location, and
	provide an option to once again display the hidden chains.
	</td>
	</tr>

	<tr>
	<td valign="top">
	    <span class="menu">Show</span><br />
	    <span class="menu">Show All</span>
	</td>
	<td valign="top">
	Show the hidden chains of the selected tags; or all hidden chains.
	</td>
	</tr>

	<tr>
	<td valign="top">
	    <span class="menu">Label</span>
	</td>
	<td valign="top">
	Choose what label to show in the <b>Chain </b> column.
	The chain number is always shown (in the current chain color).
	<ul>
	<li><b>Name</b>. Include the chain name.
	</li><li><b>Header</b>.
	    Include the chain header as
	    part of the chain label.
	</li>
	</ul>
	</td>
	</tr>

	<tr>
	<td valign="top">
	    <span class="menu">Delete</span>
	</td>
	<td valign="top">
	Delete the selected tags and all associated chains.  A confirmation prompt
	will be raised.
	</td>
	</tr>
</table>
</center>
</p>

<p>
<a name="Chain_Popup"></a>
<h3>Chain Popup Menu</h3>
<center>
<table class="titlebar" width="95%">
	<tr>
	  <th>Command</th>
	  <th>Action</th>
	</tr>
	<tr>
	<td valign="top"><span class="menu">Select</span></td>
	<td valign="top">
	<ul>
	<li><b>Invert</b>. Invert chain selection.
	</li><li><b>Clear</b>.  Clear chain selection.
	</li><li><b>All</b>. Select all chains.
	</li><li><b>Save</b>. Save the current selection set.
	</li><li><b>Restore</b>. Restore the last selection set.
	</li>
	</ul>
	</td>
	</tr>

	<tr>
	<td valign="top"><span class="menu">Extend</span></td>
	<td valign="top">
	Extend chain selection:
	<ul>
	<li><b>Tag</b>. Select all chains belonging to any tags
	    associated with selected chains.
	</li><li><b>Letter</b>.  Select all chains within selected
	    tags having the same chain letter as any of the
	    currently selected chains.
	</li>
	</ul>
	</td>
	</tr>

	<a name="ChainColor"></a>
	<tr>
	<td valign="top"><span class="menu">Color</span></td>
	<td valign="top">
	<div class="floatright">
	<img src="se/se_chaincolor_popup.png" alt="Chain Color" />
	</div>
	Set the color of the selected chains with the
	<b>Chain Color</b> panel. The <b>T</b> option
	colors the chains by tag; <b>#</b> colors the chains by number.
	</td>
	</tr>
	<tr>

	<tr>
	<td valign="top"><span class="menu">Rename</span></td>
	<td valign="top">
	Opens a CLI prompter with a text field for changing the name
	of all selected chains.  Enter the new chain name and press
	<b>OK</b> to set the specified name.
	An option allows to switch to setting chain tag, header, and group
	labels  instead.
	</td>
	</tr>

	<tr>
	<td valign="top"><span class="menu"><a href="#Renumber">Renumber</a>
	</span>
	</td>
	<td valign="top">
	Modify the UID/INS number of selected chains.
	</td>
	</tr>

	<tr>
	<td valign="top">
	    <span class="menu">Hide (SE)</span>
	</td>
	<td valign="top">
	Hides the selected chains from view in the Sequence Editor.
	If all chains of a given tag are hidden, the tag
	is hidden too.  Selected chains or chains with selected residues
	cannot be hidden; in these cases, the chain number is
	placed in parentheses to indicate that the chain would
	otherwise be hidden.
	Hidden chains are
	indicated in the Chain column as black fletches.
	These markers have an associated popup which reports
	the number of hidden chains at that location, and
	provides an option to show the hidden chains.
	</td>
	</tr>

	<tr>
	<td valign="top">
	    <span class="menu">Show</span><br />
	    <span class="menu">Show All</span>
	</td>
	<td valign="top">
	Show selected or all hidden chains.
	</td>
	</tr>

	<tr>
	<td valign="top">
	    <span class="menu">Label</span>
	</td>
	<td valign="top">
	Choose what label to show in the <b>Chain </b> column.
	The chain number is always shown (in the current chain color).
	<ul>
	<li><b>Name</b>. Include the chain name.
	</li><li><b>Header</b>.
	    Include the chain header as
	    part of the chain label.
	</li>
	</ul>
	</td>
	</tr>

	<tr>
	<td valign="top">
	    <span class="menu">Delete</span>
	</td>
	<td valign="top">
	Delete the selected chains.  A confirmation prompt
	will be raised.
	</td>
	</tr>
</table>
</center>
</p>

<p>
<a name="Residue_Popup"></a>
<h3>Residue Popup Menu</h3>
<center>
<table class="titlebar" width="95%">
	<tr>
	  <th>Command</th>
	  <th>Action</th>
	</tr>

	<tr>
	<td valign="top"><span class="menu">Select</span>
	</td>
	<td valign="top">
		<ul>
		<li><b>Invert</b>. Invert residue selection.
		</li><li><b>Clear</b>. Clear residue selection.
		</li><li><b>All</b>. Select all residues.
		</li><li><b>Sets</b>. Select residues of specified named sets.
		</li><li><b>Save</b>. Save the current selection set.
		</li><li><b>Restore</b>. Restore the last selection set.
		</li>
		</ul>
	</td>
	</tr>

	<tr>
	<a name="ResiduePopupExtend"></a>
	<td valign="top"><span class="menu">Extend</span></td>

	<td valign="top">
	Extend chain selection:
	<ul>
	<li><b>Bonded To</b>. Select all residues bonded to currently
	    selected residues.
	</li>
	<li><b>Structure</b>. Select all residues belonging
	    to the same secondary structure element.
	</li>
	<li><b>Chain</b>. Select all residues belonging to the chains
	    of the selected residues.
	</li>
	<li><b>Domain</b>. Select all residues belonging to the
	    structural domains as the selected residues.
	</li>
	<li><b>Column</b>. Select all residues within the same
	    residue column as a selected residue.
	</li>
	<li><b>Left/Right</b>. Select all residues to the left/right of the most
	left/right selected residue in each chain.
	</li>
	</ul>
	</td>
	</tr>

	<tr>
	<td valign="top"><span class="menu"><a href="#CopyPaste">Copy/Paste</a>
	</span>
	</td>
	<td valign="top">
	Copy/paste selected residues to/from the clipboard.
	</td>
	</tr>

	<tr>
	<td valign="top"><span class="menu"><a href="#Renumber">Renumber</a>
	</span>
	</td>
	<td valign="top">
	Modify the UID/INS number of selected residues.
	</td>
	</tr>
	<tr>
	<td valign="top">
	    <span class="menu">Delete</span>
	</td>
	<td valign="top">
	Delete the selected residues.  A confirmation prompt
	will be raised.
	</td>
	</tr>
</table>
</center>
</p>

<p>
<a name="Column_Popup"></a>
<h3>Column Popup Menu</h3>
<center>
<table class="titlebar" width="95%">
	<tr>
	  <th>Command</th>
	  <th>Action</th>
	</tr>

	<tr>
	<td valign="top"><span class="menu">Select</span>
	</td>
	<td valign="top">
		<ul>
		<li><b>Invert</b>. Invert column residue selection.
		</li><li><b>Clear</b>. Clear column residue selection.
		</li><li><b>All</b>. Select all column residues.
		</li><li><b>Save</b>. Save the current selection set.
		</li><li><b>Restore</b>. Restore the last selection set.
		</li>
		</ul>
	</td>
	</tr>

	<tr>
	<td valign="top"><span class="menu">Hide (SE)</span></td>
	<td valign="top">
	Hide <b>Selected</b> or <b>Unselected Columns</b>
	in the Sequence Editor.
	</td>
	</tr>
	<tr>
	<td valign="top"><span class="menu">Show All</span></td>
	<td valign="top">
	Show all previously hidden columns.
	</td>
	</tr>
</table>
</center>
</p>

<a name="Selecting_Molecular_Objects"></a><a name="SelectingMolecularObjects"></a>
<h2>Selecting Molecular Objects</h2>

<p>You can select chains and residues as well as atoms in the Sequence
Editor using the mouse, the Sequence Editor popup menus, or the
<span class="menu">SE | Select</span> menu.</p>

<a name="SynchTip"></a>
<p><span class="tip">Tip</span>&nbsp;&nbsp;
<span class="menu">SE | Select | Synchronize</span> synchronizes the
selection of atoms in the MOE Window with the selection of residues in
the Sequence Editor. <b>Note.</b> The synchronize state is remembered
between MOE sessions.</p>

<a name="Using_the_Left_Mouse_Button"></a><a name="UsingTheLeftMouseButton">
<h3>Using the Left Mouse Button in the Sequence Editor</h3></a>

<p>
<center>
<table class="titlebar" width="95%">
	<tr><th colspan="2">Chains</th></tr>
	<tr>
	<td valign="top">Select a single chain, deselecting all other
	chains.
	</td>
	<td valign="top">Click on a chain number/name.
	</td>
	</tr>
	<tr>
	<td valign="top">Select all chains
	</td>
	<td valign="top">Click on <b>Chain</b> label 
	(if all chains are already selected, clicking on the <b>Chain</b> label 
	will deselect all chains).
	</td>
	</tr>
	<tr>
	<td valign="top">Select contiguous chains
	</td>
	<td valign="top">Click on first chain then <tt>Shift</tt> - 
	click on last chain.
	</td>
	</tr>
	<tr>
	<td valign="top">Toggle selection state of a single chain
	</td>
	<td valign="top"><tt>Ctrl</tt> - click on chain number.
	</td>
	</tr>
	<tr>
	<td valign="top">Select all chains of a tag
	</td>
	<td valign="top">Click on tag.
	</td>
	</tr>
	<tr>
	<td valign="top">Clear chain selection
	</td>
	<td valign="top">Click below chain numbers.
	</td>
	</tr>

<tr><th colspan="2">Residues</th></tr>

  <tr>
    <td valign="top">Select a single residue deselecting all other residues.
    </td>
    <td valign="top">Click on residue.
    </td>
  </tr>
  <tr>
    <td valign="top">Toggle selection state of a single residue
    </td>
    <td valign="top"><tt>Ctrl</tt> - click on residue.
    </td>
  </tr>
  <tr>
    <td valign="top">Draw selection box around residues
    </td>
    <td valign="top">Drag over desired residues using the left mouse button.
    </td>
  </tr>
  <tr>
    <td valign="top">Add to selection using a selection box
    </td>
    <td valin="top"><tt>Ctrl</tt> - drag over residues using
	the left mouse button.
    </td>
  </tr>
  <tr>
    <td valign="top">Select a column of residues, 
    i.e. residues across chains that occupy the same alignment position 
    (as indicated by the ruler)</td>
    </td>
    <td valign="top">Click in alignment ruler.
    </td>
  </tr>
  <tr>
    <td valign="top">Select contiguous residues (within the same chain) or
    residue columns
    </td>
    <td valign="top">Click on first residue (or residue column)
	then <tt>Shift</tt> - click  on last residue (or residue column).
    </td>
  </tr>
  <tr>
    <td valign="top">Toggle selection state of a column of residues
    </td>
    <td valign="top"><tt>Ctrl</tt> - click on alignment ruler
    (if all residues in the column are already selected, <tt>Ctrl</tt> - click
    will deselect all residues).
    </td>
  </tr>
  <tr>
    <td valign="top">Draw selection box over residue columns
    </td>
    <td valign="top">Drag over desired residue columns in ruler using the
	left mouse button.
    </td>
  </tr>
  <tr>
    <td valign="top">Add to column selection using a selection box
    </td>
    <td valin="top"><tt>Ctrl</tt> - drag over residue columns in ruler using
	the left mouse button.
    </td>
  </tr>
  <tr>
    <td valign="top">Select a group of connected residues deselecting all other residues.
    </td>
    <td valign="top">Double-click on residue.
    </td>
  </tr>
  <tr>
    <td valign="top">Toggle selection state of a group of connected residues
    </td>
    <td valign="top"><tt>Ctrl</tt> -double-click on residue.
    </td>
  </tr>
  <tr>
    <td valign="top">Toggle selection state of all residues of a chain or tag
    </td>
    <td valin="top">Double-click on a chain number/name or tag.
    </td>
  </tr>
  <tr>
    <td valign="top">Clear residue selection</td>
    <td valign="top">Click away from residues, chains or ruler.
    </td>
  </tr>
</table>
</center>
</p>

<a name="Using_Popup_Menus"></a><a name="UsingPopupMenus"><h3>Using Popup Menus</h3></a>

<p>The popup menus in the Sequence Editor allow you to select,
invert and clear the current chain or residue selection, or to
select all chains or residues.  There are also options to
extend the current selection.  For chains, the chain selection
can be extended to include all chains of the currently-selected tags,
or all chains of the same chain letter.  For residues,
the residue selection can be extended to residues bonded to any of
the currently selected residues, to residues in the same secondary
structure element, to residues of the same chain or column, or
to all the residues to the left or right of the currently selected
residues, in their respective chains.
For a full description
of the popup menus see <a href="#SequenceEditorPopupMenus">Sequence Editor
Popup Menus</a>.
</p>

<a name="Using_the_Select_Menu"></a><a name="UsingSelectMenu">
<h3>Using the Select Menu</h3></a>

<p>The <span class="menu">SE | Select</span> menu enables you to highlight
chains, residues or atoms of the molecular system.  Selection operations
located in the Select menu add to the current selection set.  In other
words, previously selected objects remain selected.</p>

<center>
<table class="titlebar" border="1" width="95%" cellpadding="3">
    <tr><th colspan="2">SE | Select</th></tr>
    <tr>
    <th>Command</th>
    <th>Action</th>
    </tr>
    <tr>
        <td style="vertical-align:top"><a name="Clear"><b>Clear</b></a></td>
        <td style="vertical-align:top">Deselect all chains and residues.</td>
    </tr>

    <tr>
        <td style="vertical-align:top"><b>Chains</b></a></td>
        <td style="vertical-align:top">
	<a href="#SelectChains">Select chains.</td>
    </tr>
    <tr>
        <td style="vertical-align:top"><b>Residues</b></a></td>
        <td style="vertical-align:top">
	<a href="#SelectResidues">Select residues.</td>
    </tr>
    <tr>
        <td style="vertical-align:top"><b>Atoms</b></a></td>
        <td style="vertical-align:top">
	<a href="#SelectingAtoms">Select atoms.</td>
    </tr>

    <tr>
        <td style="vertical-align:top">
	<a name="PTM-Sites"><b>PTM Sites</b></a></td>

        <td style="vertical-align:top">
	Select potential Post-Translational Modification sites.
	These are residues sidechains where PTMs may be attached,
	and will only be selected if the chosen modification can be
	chemically attached there and if there is enough space
	for the additional atoms.  A simple van der Waals potential is
	used to determine the latter, and solvent atoms are ignored.
	Modifications permitted are:
	<ul>
	<li>Fucose</li>
	<li>Glucose</li>
	<li>Galactose</li>
	<li>Mannose</li>
	<li>N-acetylglucosamine</li>
	<li>N-acetylgalactosamine</li>
	<li>Xylose</li>
	<li>Acetylation</li>
	<li>Methylation</li>
	<li>Nitrosylation</li>
	<li>Phosphorylation</li>
	<li>Sulfation</li>
	</ul>
	</td>
    </tr>

    <tr>
        <td style="vertical-align:top"><a name="Set"><b>Set</b></a></td>
        <td style="vertical-align:top">Select residues of a specified
	    named set.
	</td>
    </tr>

    <tr>
        <td style="vertical-align:top">
	<a name="SetCreate"><b>Set Create</b></a></td>
        <td style="vertical-align:top">Create a new named set from the
	    selected residues, or add them to an existing set.
	</td>
    </tr>

    <tr>
        <td style="vertical-align:top">
	<a name="SetDelete"><b>Set Delete</b></a></td>
        <td style="vertical-align:top">
	Delete specified named set definition.
	</td>
    </tr>

    <tr>
        <td style="vertical-align:top">
	<a name="Selector"><b>Selector</a></b></td>
        <td style="vertical-align:top">Open the
	    <a href="#SelectionPrompter">Selection Prompter</a>.
        </td>
    </tr>

    <tr>
        <td style="vertical-align:top"><a name="Save"><b>Save</b></a></td>

        <td style="vertical-align:top">Saves the current residue selection set.
	You can save up to 16 selection sets at any one time. Simply select
	one or more residues and choose the Save command. To view the saved
        selection, choose Restore (described next).</td>
    </tr>

    <tr>
        <td style="vertical-align:top"><a name="Restore"><b>Restore</b></a></td>

        <td style="vertical-align:top">Restores the last saved selection set.
	Selections are saved on a last-in, first-out basis. The last saved
	selection is the first to be restored. The selection is then removed
	from the stack and cannot be restored again, unless first saved.</td>
    </tr>

    <tr>
        <td style="vertical-align:top"><a name="Synchro"><b>Synchronize</b></a></td>

        <td style="vertical-align:top">
            Synchronizes the selection of residues in the Sequence Editor.
            with the selection of atoms in the MOE rendering area

            <p>Selecting a residue causes all its children atoms to become
	    selected (seen in the MOE Window), and selecting an atom causes
	    its parent residue to be selected (seen in the Sequence Editor),
            The exception is when a residue becomes selected because the
            monitor has detected that one of the residue's child atoms has
            been selected: the rest of the residue's children do not
            immediately become selected. A residue remains selected as
            long as any of its child atoms are selected. Deselecting a
            residue deselects its atoms. Deselecting all atoms of a
            residue deselects the residue.</p>
        </td>
    </tr>
</table>
</center>

<p><font color="#ff0000"><b>Tip</b></font>&#160;&#160; If you accidentally
click in the 3D rendering area and lose your current selection, you can
use the
<span class="nobr"><b>MOE | Select | Restore</b></span>
command to restore the last selection set.


<a name="Selecting_Residues_and_Chains"></a><a name="SelectingResChains"></a>
<h4>Selecting Chains, Residues and Atoms</h4>

<a name="SelectChains"></a>

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Select | Chains</th></tr>
    <tr>
    <th>Command</th>
    <th>Action</th>
    </tr>

    <tr>
    <td valign="top">
    <a name="InvertChains"></a>
	<span class="menu">Invert</span>
    </td>
    <td valign="top">
    Invert chain selection.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="ClearChains"></a>
	<span class="menu">Clear</span>
    </td>
    <td valign="top">
    Clear chain selection.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="AllChains"></a>
	<span class="menu">All</span>
    </td>
    <td valign="top">
    Select all chains visible in the Sequence Editor.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="EmptyChains"></a>
	<span class="menu">Empty</span>
    </td>
    <td valign="top">
    Select chains with no residues.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="WaterChains"></a>
	<span class="menu">Water</span>
    </td>
    <td valign="top">
    Select chains containing only water residues.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="TagExtend"></a>
	<span class="menu">Tag Extend</span>
    </td>
    <td valign="top">
    Extend chain selection to include all chains of tags
    containing selected chains.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="LetterExtend"></a>
	<span class="menu">Letter Extend</span>
    </td>
    <td valign="top">
    Extend chain selection to include all chains with
    the same chain letter as any selected chain.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="ChainsWithSelectedAtoms"></a>
	<span class="menu">With Selected Atoms</span>
    </td>
    <td valign="top">
    Select the chains that have at least one atom selected.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="ChainsWithSelectedResidues"></a>
	<span class="menu">With Selected Residues</span>
    </td>
    <td valign="top">
    Select the chains that have at least one residue selected.
    </td>
    </tr>

</table>
</center>
</p>

<a name="SelectResidues"></a>

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Select | Residues</th></tr>
    <tr>
    <th>Command</th>
    <th>Action</th>
    </tr>

    <tr>
    <td valign="top">
    <a name="InvertResidues"></a>
	<span class="menu">Invert</span>
    </td>
    <td valign="top">
    Invert the residue selection.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="ClearResidues"></a>
	<span class="menu">Clear</span>
    </td>
    <td valign="top">
    Clear the residue selection.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="All"></a>
	<span class="menu">All</span>
    </td>
    <td valign="top">
    Select all residues of chains visible in the Sequence Editor.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="Empty"></a>
	<span class="menu">Empty</span>
    </td>
    <td valign="top">
    Select residues with no atoms.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="ChainBreaks"></a>
	<span class="menu">Chain Breaks</span>
    </td>
    <td valign="top">
    Select non-empty amino acid residues that are not bonded through a peptide
    bond to the preceding residue in its chain.
    </td>
    </tr>

<!--
    <tr>
    <td valign="top">
    <a name="Sets"></a>
	<span class="menu">Sets</span>
    </td>
    <td valign="top">
    Select residues belonging to a named collection.
    </td>
    </tr>
-->

    <tr>
    <td valign="top">
    <a name="ConservedResidues"></a>
	<span class="menu">Conserved</span>
    </td>
    <td valign="top">
	Include in the residue selection residues with conserved
	properties, described <a href="#Conserved">below</a>.
	The calculations are performed over selected chains only,
	or all visible chains if none are selected.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="FixedResidues"></a>
	<span class="menu">Fixed</span>
    </td>
    <td valign="top">
    Select fixed residues.  Residues can be fixed and unfixed
    using the <span class="menu">SE | Footer | Residues</span> panel
    or the <span class="menu">SE | Alignment</span> menu.
	The calculations are performed over selected chains only,
	or all visible chains if none are selected.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="ConstrainedResidues"></a>
	<span class="menu">Constrained</span>
    </td>
    <td valign="top">
    Select constrained residues.  Residues can be constrained and unconstrained
    using the <span class="menu">SE | Footer | Residues</span> panel or in the
    <span class="menu">SE | Alignment | Constraints</span> menu.
	The calculations are performed over selected chains only,
	or all visible chains if none are selected.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="ChainInterfaces"></a>
	<span class="menu">Chain Interfaces</span>
    </td>
    <td valign="top">
	Select residues at chain interfaces.  These are
	surface residues where chains are in close proximity.
	The calculations are performed over selected chains only,
	or all visible chains if none are selected.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="ExtendResidues"></a>
	<span class="menu">Extend</span>
    </td>
    <td valign="top">
	<a href="#ResiduePopupExtend">Extend</a> the residue selection
	to residues <b>Bonded To</b> any selected
	residue, or to residues belonging to the same secondary
	<b>Structure</b> element, or to residues of the same <b>Chain</b>, 
	<b>Domain</b> or <b>Column</b>, or to residues to the <b>Left</b> or
	<b>Right</b> of any selected residue, in its respective chain.
    </td>
    </tr>

    <tr>
    <td valign="top">
    <a name="WithSelectedAtoms"></a>
	<span class="menu">With Selected Atoms</span>
    </td>
    <td valign="top">
    Select the residues with at least one atom selected.
    <p><span class="tip">Tip</span>&nbsp;&nbsp;
    <span class="menu">SE | Select | Synchronize</span>
    will keep the atom and residue selections synchronized automatically.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="OfSelectedChains"></a>
	<span class="menu">Of Selected Chains</span>
    </td>
    <td valign="top">
    Select the residues belonging to selected chains.
    </td>
    </tr>

</table>
</center>
</p>


<a name="Conserved"></a>
<p>The <span class="menu">SE | Select | Residues | Conserved</span>
menu is used to
select those residues where the specified property (identity, structure,
etc.) is conserved for the entire column (alignment position).  Only
alignment positions with no gaps are
considered conserved, i.e. the residue columns must be complete.
This command applies to amino acid residues only.</p>

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Select | Residues | Conserved</th></tr>
	<tr>
	<th>Command</th>
	<th>Action</th>
	</tr>
    <tr>
    <td valign="top">
    <a name="Identity"></a><span class="menu">Identity</span>
    </td>
    <td valign="top">
    Select residues at positions that conserve the residue single letter code.
    Residues for which the single letter code is 'X' will be compared based
    on their 3-letter code.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Similarity"></a><span class="menu">Similarity</span>
    </td>
    <td valign="top">
    Select residues at positions where all pairwise BLOSUM62 similarity scores
    are greater than zero.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Structure"></a><span class="menu">Structure</span>
    </td>
    <td valign="top">
    Select residues at positions that conserve secondary structure.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Property"></a><span class="menu">Property</span>
    </td>
    <td valign="top">
    Select residues at positions that conserve any of the amino acid
    properties described the remainder of this menu.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Hydrophobic"></a><span class="menu">Hydrophobic</span>
    </td>
    <td valign="top">
	Ala, Val, Ile, Leu, Met, Trp, Phe, Tyr
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Hydrophilic"></a><span class="menu">Hydrophilic</span>
    </td>
    <td valign="top">
	Ser, Thr, Asn, Gln, Glu, Asp, Arg, Lys, His
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Acidic"></a><span class="menu">Acidic</span>
    </td>
    <td valign="top">
	Glu, Asp
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Basic"></a><span class="menu">Basic</span>
    </td>
    <td valign="top">
	Arg, Lys, His
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Aromatic"></a><span class="menu">Aromatic</span>
    </td>
    <td valign="top">
	Phe, Tyr, Trp, His
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Aliphatic"></a><span class="menu">Aliphatic</span>
    </td>
    <td valign="top">
	Val, Leu, Ile, Gly, Ala, Met, Pro
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Polar"></a><span class="menu">Polar</span>
    </td>
    <td valign="top">
	Asn, Ser, Thr, Gln
    </td>
    </tr>

</table>
</center>
</p>

<a name="Selecting_Atoms_in_the_MOE_Window"></a><a name="SelectingAtoms"></a>
<h4>Selecting Atoms in the MOE Window</h4>

<p>Commands in the <span class="menu">SE | Select</span>
menu enable you to select
atoms of the molecular system.  Every selection operation adds
to the current selection set.  In other words, previously selected atoms
remain selected.</p>

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Select | Atoms</th></tr>
	<tr>
	<th>Command</th>
	<th>Action</th>
	</tr>
    <tr>
    <td valign="top">
    <a name="AtomsOfSelectedResidues"></a>
	<span class="menu">Of Selected Residues</span>
    </td>
    <td valign="top">
    Select the child atoms of all selected residues.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="AtomsOfSelectedChains"></a>
	<span class="menu">Of Selected Chains</span>
    </td>
    <td valign="top">
    Select the child atoms of all selected chains.
    </td>
    </tr>
</table>
</center>
</p>


<a name="Using_the_Residue_Selector_Panel"></a>
<a name="ResidueSelectorPanel"></a>
<a name="SelectionPrompter"></a>
<h3>Using the Selection Prompter</h3>

<p>For more complex selection criteria, MOE provides a
    <a href="sel_ui.htm">Selection Prompter</a>, which is opened with
<span class="nobr"><b>SE | Select | Selector</b></span>.
When active, the Selection Prompter appears in the Sequence Editor
immediately below the menu bar.  The Selection Prompter allows
for specifying residue selections using MOE's
<a href="../fcnref/aselect.htm">selection language</a>.
This language has built-in keywords representing common sets,
e.g. <tt>rsel</tt> for the current residue selection, and
<tt>receptor</tt> for all receptor residues, as well as logical,
comparison, property, substructure, and extension operations.
</p>

<p><span class="tip">Note:</span> Beginning in 2013, the
<a name="ResidueSelector"><a href="select.htm">Residue Selector</a></a>
panel is invoked through
the <b>UI</b> button in the Selection Prompter.
</p>

<a name="Annotating_Molecular_Objects"></a>
<a name="AnnotatingMolecularObjects"></a>
<h2>Annotating Molecular Objects</h2>


<div class="floatright">
<img src="se/footer_residues.png" alt="Residue Color Style Panel" />
</div>

<a name="Coloring_Residues"></a><a name="ColoringResidues"></a>
<h3>Coloring Residues</h3>

<p>Residues can be colored in the Sequence Editor using one of the
following:

<ul>

<li>The <b>Residues | Color</b> menu item from the Sequence Editor
popup menus
</li>

<li>The Residue Color Style panel in
<span class="nobr"><b>SE | Footer</b></span>
</li>

</ul>

<p>The Sequence Editor popup menus allow you to color single or
multiple residues.  The colors are selected from a predefined palette which
is accessible as a submenu of colors.

<p>The Residue Color Style panel defines a group of color-coded amino acid
equivalence classes. Choosing a color scheme and pressing Apply will color
all the residues currently in MOE.  When residues are created or mutated
they are <i>not</i> automatically colored.  In this case, you must reapply
the desired residue coloring.
</p>


<p>The <a href="mainwin.htm#Color">MOE Window</a> includes the rendering option 
<span class="menu">MOE | Render | Atoms | Color: Residue</span> to
automatically color atoms according to the color assigned to their parent
residue. Ribbons can also be automatically colored by their parent residues with
the option <span class="menu">MOE | Render | Ribbon | Color: Residue</span>.
In both cases the dynamic update of sequence alignment dependent
<a href="#ResidueColorStyle">residue color schemes</a> will also 
update the colors of atoms and ribbons colored by residue.
</p> 

<a name="ColorByChainTag"></a>
<h3>Coloring Chains by Chain Number and Tag</h3>

<p>
Chain colors can be assigned according to chain number and chain tag using the
<b>#</b> and <b>Tag</b> options in the <b>Color</b> menu of the chain popup.
The chain color is the color used to display the chain number in
the Sequence Editor.  Atoms and ribbons can be colored by tag and 
chain color using the 
<span class="menu">MOE | Render | Atoms | Color: Tag or Chain</span> and 
<span class="menu">MOE | Render | Ribbon | Color: Tag or Chain</span>
commands.
</p>

<a name="Coloring_the_Backbone"></a><a name="ColoringTheBackbone"></a>
<h3>Coloring the Backbone</h3>

<p>The display of the backbone ribbon in the MOE Window can be controlled
from within the Sequence Editor.  Each of the Sequence Editor popup menus
contains a submenu called <b>Ribbon</b> that allows you to render the
backbone ribbon of specific residues in various styles or to change the color of
that portion of the backbone to one of the pre-defined colors found in the
<span class="menu">Ribbon</span> submenu.  Note that changing the color
will not automatically turn on the backbone display, so to see the color,
first draw the backbone.  To see the different backbone drawing styles,
see <a href="mainwin.htm#RenderingBackbone">Rendering the Backbone</a>.

<a name="SpecialAnnotations"></a>
<a name="Annotate"></a>
<h3>Special Annotations</h3>

<p>The <span class="menu">SE | Annotate</span>
menu provides access to a series of functions and settings that aid in
highlighting structural topologies of the protein chains loaded in the system.

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Annotate</th></tr>
	<tr>
	<th>Command</th>
	<th>Description</th>
	</tr>
    <tr>
     <td valign="top"><span class="menu">
	<a name="PredictSecondaryStructure" />2o Structure</span></td>
     <td valign="top">

Manually assign to each amino acid residue of the selected
chains its most probable secondary structure label as listed in the
drop-down menu. The possible structure classifications include <i>None,
Helix, 3-10 Helix, Pi Helix, Strand, Anti-Strand, 3 Turn, 4 Turn and
5-Turn</i> Loops. For helices, the rotation directions (left/right-handed)
may also be specified. The predicted secondary structure display in
the SE is controlled by the <b>Display</b> checkbox in the menu option.
Note that the menu is grayed out if no chains/residues are selected.

     </td>
    </tr>

    <tr>
     <td valign="top"><span class="menu">
	<a name="Plot" />Plot</span></td>
     <td valign="top">

	Control the display of the
	<a href="#ResScalarPlots">Residue Scalar Plots</a> as
	outlined above. Residue scalar plots display numerical residue
	attributes on a bar graph aligned with and displayed beneath the
	associated residue. Up to three scalar values may be plotted at the
	same time that are controlled by <i>Scalar 1, Scalar2</i> and
	<i>Scalar 3</i> Submenu options. The following measurements may be
	selected:

	<br />
	<ul>
	  <li><a href="../../proteins/propred.htm"><b>Structure
		Propensity</b></a></li>
	  <li><a href="../../proteins/propred.htm"><b>Surface
		Accessibility</b></a></li>
	  <li><b>Stereochemical Quality</b></li>
	</ul>
     </td>
    </tr>
    <tr>
     <td valign="top"><a name="AnnotateAntibody" />
     <span class="menu">Antibody</span></td>
     <td valign="top">

    Launch the antibody detection method
    that scans the proteins in the current system for antibody
    chains. If detected, then the three CDR regions of V<sub>L</sub>
    and V<sub>H</sub> will be highlighted according to the one of the
    antibody numbering schemes <i>CCG, Kabat</i> and <i>Chothia</i> that
    may be selected in the submenu.  In addition to the visual assignment
    of the CDR regions in the SE, a series of dedicated residue sets
    are generated as well composed of the individual CDRs as well as
    the antibody framework region.  These sets may be selected with
    the <span class="menu">SE | Select | Set</span> menu command.

     </td>
    </tr>
    <tr>
     <td valign="top"><a name="AnnotateSequence" />
      <span class="menu">Sequence</span>
     </td>
     <td valign="top">

    Each protein chain in the system is compared to a set of standard
    reference families (databases found in the <b>Project Path</b> of the 
    <a href="../../appendix/settings.htm#ProteinDefaults">Protein Defaults</a>),
    which determines its membership to one or more protein families.
    The various annotations and renderings that are relevant to each family will be
    copied to the sequence editor and highlighted. This makes it easy to
    compare multiple proteins in the system in a topologically meaningful
    fashion.  Residue sets of interest will also be created and can be
    selected with the <span class="menu">SE | Select | Set</span> menu command.

     </td>
    </tr>

    <tr>
     <td valign="top"><span class="menu">
	<a name="StructurePropensity" />Structure Propensity</span></td>
     <td valign="top">

	Color the residues according to the probability of having
	helix, strand, or no secondary structure.  A darker color
	indicates greater
	<a href="../../proteins/propred.htm">propensity</a>.
     </td>
    </tr>

    <tr>
     <td valign="top"><a name="ChainInfo" />
        <span class="menu">Chain Info</span>
    </td>
     <td valign="top">
    Provide a set of checkboxes that control
    the visualization of issues often associated with working with protein
    structure collections:
	<ul>
	 <li><b>Compound Name</b>.
        If on, then the full compound name composed of the chain tag and the
	chain name is shown on top of each entry.
    </li>
	 <li><b>Empty Residues</b>.
    If on, then residues lacking atom coordinates will be displayed in the
    sequence editor in a regular font style. This is in contrast to residues
    with atom coordinates that are by default shown in a bold font in the SE.
    </li>
	<li><b>Chain Breaks</b>.
    If on, then structural gaps in the backbone within a protein chain or
    peptide are highlighted by a vertical line.
    </li>
	<li><b>Disulfide Bonds (intra)</b>.
	<div class="floatright">
	<img src="se/ss_msg.png" alt="SE Disulfide Bond" />
	</div>
    Disulfide bonds are marked in the Sequence Editor by yellow rectangles
    surrounding the SS-bond residues.
    If one of the disulfide bond residues is selected then the name and UID of
    the corresponding Cys/Met is displayed in the footer of the SE panel.
    The checkbox controls, if SS-bond residues are highlighted.
    </li>
	 <li><b>Inter-Residue Bonds</b>.
    If on, then residues that are connected by covalently bonded atoms are
    highlighted by a cyan rectangle. This is particular important when working
    protein/ligand complexes. The checkbox controls, if these residues are
    highlighted in the SE.
	 </li>
	 <li><b>Inter-Chain Bonds</b>.
    If residues originating from different chains are covalently bound
    then these
    residues are marked by a magenta rectangle. In most cases this applies for
    receptor/ligand complexes or disulfide bonds of Cys from different chains.
	 </li>
	</ul>
	<br />
	All the above mentioned settings maybe permanently stored
	by pressing the
    <b>Save As Default</b> button.
     </td>
    </tr>

</table>
</center>
</p>

<a name="Protein"></a>
<h2>Protein Modeling</h2>

<p>The <span class="menu">SE | Alignment</span>
and <span class="menu">SE | Protein</span>
menus provides access to MOE's suite of protein modeling
applications. 
</p>

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Alignment</th></tr>
	<tr>
	<th>Command</th>
	<th>Description</th>
	</tr>
    <tr>
    <td valign="top"><span class="menu">
	<a name="ResetAlignment"></a>Reset
    </span></td>
    <td valign="top">
	Reset the alignment positions of residues of selected chains. Residues
	become positioned according to cardinal numbering. This command is
	grayed out if no chain is selected.
    </td>
    </tr>

    <tr>
    <td valign="top"><span class="menu">
	<a name="PackAlignment"></a>Pack
    </span></td>
    <td valign="top">
	Eliminate empty residue columns from the current alignment.  Such
	empty columns may arise, for example, when chains or residues
	are deleted.
    </td>
    </tr>

    <tr>
    <td valign="top"><span class="menu">
	<a name="Fix"></a>Fix Residues
    </span></td>
    <td valign="top">
	Fix selected residues.  This prevents them from
	being moved during manual alignment.
    </td>
    </tr>

    <tr>
    <td valign="top"><span class="menu">
	<a name="Unfix"></a>Unfix Residues
    </span></td>
    <td valign="top">
	Unfix selected residues.
    </td>
    </tr>

    <tr>
    <td valign="top"><span class="menu">
	<a name="Constraints"></a>Constraints
    </span></td>
    <td valign="top">

    <div class="floatright">
    <img src="se/constraintsmanager.png" alt="Constraints Manager" />
    </div>

	Apply/remove alignment constraints to residues.
	<ul>
	<li>
    <b>Constrain Residues:</b><br>
	Opens a <a href="../../proteins/align.htm#CONSTRAINTS">
	CLI prompter</a> in which one can create new
	alignment constraints by picking residues.
	</li><li>
    <b>Unconstrain Residues:</b><br>
	Remove selected residues from constraints.
	</li><li>
    <b>Antibody Constraints:</b><br>
	Create constraints to align antibody sequences.
	</li><li>
    <b>Automatic:</b><br>
	Detect what family, if any, each chain belongs to, and create
	constraints specific to aligning that family, based on those stored in
	the databases found in the <b>Project Path</b> of the 
	<a href="../../appendix/settings.htm#ProteinDefaults">Protein Defaults</a>.
	</li><li>
    <b>Manage Constraints:</b><br> 
	Open the <a href="../../proteins/align.htm#ManagingConstraints">
	Residue Alignment Constraints manager.</a>

	</li>
    </td>
    </tr>

    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/align.htm">Align</a>
    </span></td>
    <td valign="top">
	Open the Protein Align panel, for performing multiple sequence
	and structure alignment.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/align.htm#Monitor">Similarity</a>
    </span></td>
    <td valign="top">
	Open the Protein Similarity panel, which measures all-against-all
	sequence identity, similarity and RMSD scores.
    </td>
    </tr>

    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/prosup.htm">Superpose</a>
    </span></td>
    <td valign="top">
	Open the Protein Superpose panel, for performing protein
	structure superposition.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/procore.htm">Consensus</a>
    </span></td>
    <td valign="top">
	Open the Protein Consensus panel, for performing an
	analysis of the structurally conserved core of a group
	of aligned proteins.
    </td>
    </tr>
</table>
</center>
</p>

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Protein</th></tr>
	<tr>
	<th>Command</th>
	<th>Description</th>
	</tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/prosearch.htm">PDB Search</a></span>
    </td>
    <td valign="top">
	Open the MOE-SearchPDB panel, for performing a homology search.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/motifsearch.htm">Domain Motif Search</a></span>
    </td>
    <td valign="top">
	Open the Domain Motif Search panel, for searching protein structures
	with similar secondary structure motifs to a query.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/projsrch.htm">Project Search</a></span>
    </td>
    <td valign="top">
	These menu options will vary depending on which databases are found in
	the <b>Project Path</b> of the <a href="../../appendix/settings.htm#ProteinDefaults">Protein Defaults</a>.
	Selecting any of them will open the <b>Project Search</b> panel, for browsing
	and searching the selected project database.
    </td>
    </tr>

   <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/probuild.htm">Protein Builder</a></span>
    </td>
    <td valign="top">Open the
	Protein Builder and Rotamer Explorer to build, modify amino acids
	and explore their conformations.</td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/promodel.htm">Homology Model</a></span>
    </td>
    <td valign="top">
	Open the Homology Model panel, for building a 3D protein model.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/antibody.htm">Antibody Modeler</a></span>
    </td>
    <td valign="top">
	Open the Antibody Modeler panel, for building a 3D protein
	model based on antibody templates.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../apps/proloop.htm">Loop Modeler</a></span>
    </td>
    <td valign="top">
	Open the Loop Modeler application.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../apps/proloop.htm">Linker Modeler</a></span>
    </td>
    <td valign="top">
	Open the Linker Modeler application.
    </td>
    </tr>
     <tr>
    <td valign="top"><span class="menu">
	<a href="../../apps/prodesign.htm">Design</a></span>
    </td>
    <td valign="top">
	Open one of MOE's Protein Design applications :
	<a href="../../apps/prodesign.htm#AlanineScanMethod">Alanine Scan
	</a></span>,
	<a href="../../apps/prodesign.htm#DisulfideScanMethod">Disulfide Scan
	</a></span>,
	<a href="../../apps/prodesign.htm#ResistanceScanMethod">Resistance Scan
	</a></span>,
	<a href="../../apps/prodesign.htm#ResidueScanMethod">Residue Scan
	</a></span> or
	<a href="../../apps/prodesign.htm#SequenceDesignMethod">Sequence Design
	</a></span>.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/propatch.htm">Patch Analyzer</a></span>
    </td>
    <td valign="top">
	Opens the Protein Patch Analyzer, which determines and
	analyzes hydrophilic and hydrophobic protein surface regions. 
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/proprop.htm">Properties</a></span>
    </td>
    <td valign="top">
	Opens the Protein Properties panel, which calculates various
	physicochemical protein properties.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../apps/progeom.htm">Geometry</a></span>
    </td>
    <td valign="top">
	Opens the Protein Geometry panel, for evaluating protein
	stereochemical quality.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/procontacts.htm">Contacts</a></span>
    </td>
    <td valign="top">
	Opens the Protein Contacts panel, for analyzing stabilizing contacts.
    </td>
    </tr>
   </table>
</center>
</p>


<a name="DNAbuilder"></a>
<h2>DNA Builder</h2>

<p>To open the DNA Builder panel, choose 
<span class="menu">SE | Edit | DNA Builder</span>
in the Sequence Editor.  This panel is used to create
DNA fragments from a string of single-letter identifiers.
(It can also be opened from the main MOE window:
<span class="menu">MOE | Edit | Build | DNA</span>.)
</p>

<p align="center">
<img src="se/DNAbuilder.png" width="300px" alt="DNA Builder Panel" />
</p>

<p>
The <b>Sequence</b> input line recognizes single-letter codes of residues (see
below).  <b>Conformation</b> options allow you to 
create fully-populated strands in A, B, or C conformation [Arnott 1975].
</p>

<p>
The name, tag, and header attributes of the chain can be set in this
panel. Press <b>OK</b> to create the sequence. Press <b>Apply</b> to create
the sequence without closing the panel.
</p>

<p>
<table class="titlebar" align="center">
  <tr>
    <th>1-letter code</th>
    <th>Nucleic Acid</th>
  </tr>
  <tr>
    <td><tt>a</tt></td>
    <td>Adenosine</td>
  </tr>
  <tr>
    <td><tt>c</tt></td>
    <td>Cytidine</td>
  </tr>
  <tr>
    <td><tt>g</tt></td>
    <td>Guanosine</td>
  </tr>
  <tr>
    <td><tt>t</tt></td>
    <td>Thymidine</td>
  </tr>
</table>
</p>


<a name="Editing_Molecular_Objects"></a>
<a name="EditingMolecularObjects"></a>
<h2>Editing Molecular Objects</h2>

<a name="Moving_Residues"></a><a name="MovingResidues"></a>
<h3>Moving Residues</h3>

<p>Residues are moved with either mouse drag movements or 
<tt>Left</tt>/<tt>Right</tt> arrow keys
within the residue display area. They can be moved to the right or left, 
but not beyond the start of the alignment ruler.
Fixed residues can also impede residue movement.
</p>

<p>
<center>
<table class="titlebar" width="95%">
<tr>
<th>Action</th>
<th>Description</th>
</tr>
<tr>
<td valign="top">To move one residue</td>
<td valign="top">Drag the residue in the Sequence Editor using the middle mouse
button or select the residue and press <tt>Left</tt>/<tt>Right</tt> arrows.
A red border highlights the residue being moved.

<p>The residue is moved right or left (changing its alignment
position). Any residues encountered will be pushed
along, like beads on a string (&quot;abacus&quot; mode).
Gaps between residues are not preserved.</p></td>
</tr>

<tr>
<td valign="top">To move several residues </td>
<td valign="top">
Select the residues to move (residues can belong to more than one chain
and do not have to be consecutive), and press <tt>Alt</tt> while dragging
the middle mouse button or press <tt>Left</tt>/<tt>Right</tt> arrows.
Red borders highlight all residues being moved.

<p>Selected residues are moved right or left. Any residues
encountered will be pushed along, like beads on a string.
Selected residues maintain their relative alignment positions while moving.</p></td>
</tr>

<tr>
<td valign="top">To slide all residues in an entire chain</td>
<td valign="top">Press <tt>Ctrl</tt> and drag the middle mouse button
over any residue in the chain.  Gaps are maintained.<br>
Red borders highlight all residues being moved.
</td>
</tr>

<tr>
<td valign="top">To move residues on one &quot;side&quot; of a chain</td>
<td valign="top">Press <tt>Shift</tt> 
while dragging the middle mouse button over a residue in the chain.
<p>
All residues to the right or left of the dragged residue are moved to
the right or left along with it, while maintaining gaps.
The &quot;side&quot; of the chain is determined with respect
to any gaps or breaks.  If no gaps or breaks exist, the residues on
the side in which direction one starts to drag is moved.
Red borders highlight all residues being moved.
</td>
</tr>

</table>
</center>
</p>

<a name="Moving_Chains"></a><a name="MovingChains"></a>
<h3>Reordering Chains</h3>

<p>You can reorder the chains in a system by dragging individual
chain numbers in the Chains column or tags in the Tags column using
the middle mouse button.
<b>Note:</b> Chain reordering using drag-middle can also be
done in the <a href="system.htm">System Manager</a>.
</p>

<p>Chains can also be reordered using
the <span class="menu">SE | Edit</span> menu <b>Move</b> commands.
With these, you can move several chains at once, but you
must first select the chains you wish to move.
There is also a set of <b>Order</b> commands in the menu:
</p>

<p>
<center>
<a name="SortChains"></a>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Edit</th></tr>
	<tr>
	<th>Command</th>
	<th>Action</th>
	</tr>
    <tr>
    <td valign="top">
	<span class="menu">Move | Top</span>
    </td>
    <td valign="top">
	Move the selected chains to the top of the chain list.
    </td>
    </tr>
    <tr>
    <td valign="top">
	<span class="menu">Move | Bottom</span>
    </td>
    <td valign="top">
	Move the selected chains to the bottom of the chain list.
    </td>
    </tr>
    <tr>
    <td valign="top">
	<span class="menu">Move | Before</span>
    </td>
    <td valign="top">
	Move the selected chains before a given chain.  You will
	be prompted to select the chain before which the selected chains will
	be moved.
    </td>
    </tr>
    <tr>
    <td valign="top">
	<span class="menu">Order | by Name</span>
    </td>
    <td valign="top">
	Order the chains alphabetically by chain name.
    </td>
    </tr>
    <tr>
    <td valign="top">
	<span class="menu">Order | by Tag</span>
    </td>
    <td valign="top">
	Order the chains alphabetically by chain tag.
	Chain tags can be edited using
	<span class="menu">SE | Edit | Chain Tag</span>
	or with the Chain popup menu.
    </td>
    </tr>
</table>
</center>
</p>

<!--
<p><b>Note:</b> When you reorder chains in the Sequence Editor display,
you cause the actual chain numbering to be reordered as well.</p>
-->

<a name="Joining_and_Splitting_Chains"></a><a name="JoiningAndSplitting"></a>
<h3>Joining and Splitting Chains</h3>

<p>To join two or more chains:</p>
<ol>
<li>Select the chains to join.</li>
<li>Choose <b><span class="nobr">SE | Edit | Join Chains</span></b> or choose
<b>Join Chains</b> from the Chain popup.</li>
</ol>

<p>The residues of the selected chains are re-parented to the lowest numbered
chain. <!--If the last residue of one chain and the first residue of
the next belong to the MOE standard residue database, the two residues are
automatically polymerized.--></p>

<p>To split a chain at a single residue, open the
Residue popup over the residue and select <b>Split Chain</b>.</p>

<p>To split the system at several residues:
<ol>
<li>Select the residues that will mark the beginning of the new chains.</li>
<li>Choose <span class="menu">SE | Edit | Split Chains</span>.
The selected residues each become the first residue of a new chain.  Each
new chain appears directly below the chain from which it was split.</li>
</ol>

<p><span class="tip">Tip</span>&nbsp;&nbsp;
Chains can also be split and joined with the Sequence Editor
<a href="#CopyPaste"><b>Copy As</b></a>,
<a href="#CopyPaste"><b>Paste</b></a>, and
<a href="#CopyPaste"><b>Paste Fragment</b></a>
commands. When joining chain fragments with the 
<b><span class="nobr">SE | Edit | Paste Fragment</span></b> 
set the <b>Coordinates</b> option to <b>Original</b> in order
to preserve the current atom positions during joining.
</p>

<a name="Deleting_Chains_and_Residues"></a><a name="DeletingChainsAndResidues"></a>
<h3>Deleting Chains and Residues</h3>

To delete a single chain:
<ol>
<li>Position the cursor over the number/name of the chain to delete.</li>
<li>Display the Chain popup (use <tt>Ctrl</tt> if the chain
is not selected) and choose <b>Delete</b>.</li>

</ol>

<p>To delete two or more chains at a time:
<ol>
<li>Select the chains to delete.</li>
<li>Display the Chain popup and choose <b>Delete</b>.
The same functionality is found in
<span class="menu">SE | Edit | Delete | Chains</span>.</li>

</ol>

<p>Deleting residues is accomplished similarly.  To delete a single
residue, you can use the <b>Delete</b> option from the Residue
popup.  To delete one or more residues, select the residues to
delete, then use either the <b>Delete</b> option in the
Residue popup, or the 
<span class="menu">SE | Edit | Delete| Residues </span> menu command.
To delete all residues in a residue column, select a column of
residues by clicking in the alignment ruler.  Then proceed as
for deleting selected residues.
After the residue column is deleted, the alignment is &quot;packed&quot;
(all residues
to the right of the deleted column are moved left by one column to
close the gap).
</p>

<p>When deleting chains or residues, MOE displays the total number of
chains, residues and atoms that will be deleted, and asks you to confirm
or cancel the operation.  You also have the option to &quot;pack&quot; the
alignment; i.e. eliminate empty residue columns that may
result from the deletion operation.</p>

<a name="Cap"></a>
<h3>Capping Terminal Residues</h3>

<p>The C terminus of proteins can be capped with an ACE,
and the N terminus with an NME using the
<span class="menu">SE | Edit | Cap</span> commands,
<b>Amidate C Terminus</b> and
<b>Acetylate N Terminus</b>, respectively.
Selected residues that are at the ends of a chain break
can be similarly capped.
</p>

<a name="Rename"></a>
<h3>Renaming Tags and Chains</h3>
<p>
To rename chains open the <b>Rename</b> prompter with the 
<span class="menu">SE | Edit | Rename Tag/Chain </span> menu command.
The prompter provides a text field for changing the name
of all selected chains.  In addition to renaming the chains, also
the tag, header and group label can be set for the selected chains or tags,
therefore reorganizing the chains as requested.  The prompter can also
be opened from the Chain and Tag popups.
</p>

<a name="Renumber"></a>
<h3>Renumbering Residues and Chains</h3>
<p>
The Sequence Editor provides an easy way of modifying the UID/INS number
of selected residues and chains. This operation uses a CLI prompt which is
accessible both through <span class="menu">SE | Edit | Renumber</span> and
Residue/Chain popup menu <b><b>Renumber</b></b> commands.
</p>
<center>
<img src="se/renumber.png" alt="Renumbering" />
</center>

<p>
Enter/Choose the new numbering scheme to be applied to either selected residues
or chains and then press <b>Apply</b>. Three numbering schemes are provided:
<ul>
    <li>Start from a given number;</li>
    <li>Set to be same as alignment position;</li>
    <li>Inherit numbering from a specified chain.</li>
</ul>
<b>Note.</b> If two non-consecutive residues on the same chain are selected, and 
<b>start from number</b> is used, all the intervening residues will be
renumbered as well.
</p>

<a name="CopyPaste"></a>
<h3>Copying and Pasting Residues and Chains</h3>

<p>The Sequence Editor supports copying and pasting selected
residues and chains.  The 
<span class="menu">SE | Edit | Copy As</span> command copies
all selected objects to the clipboard.  The
<span class="menu">SE | Edit | Paste</span> command pastes
the entire contents of the clipboard into its own chain.
</p>

<p>
A special paste operation, accessible both through
<span class="menu">SE | Edit | Paste Fragment</span>
and the Residue popup menu <b>Copy/Paste</b> commands,
allows for inserting the clipboard contents at a specific
location in a chain.  This operation uses a CLI prompt.
The insertion can be made
to either replace or displace a sequence of residues.
First the insertion point or region to replace is
chosen in the Sequence Editor using the mouse, then
specified whether the fragment should be grafted onto
the target chain, and finally <b>Paste</b> is pressed to
perform the paste action.
Insertion is only possible if the clipboard contents
is a contiguous sequence fragment. If multiple chains or
separate residue block are on the clipboard a normal 
<span class="menu">SE | Edit | Paste</span> operation
creating new chains is performed.
</p>

<center>
<img src="se/copypaste.png" alt="Copy/Paste" />
</center>

<p>
While in the <b>Paste Fragment</b> CLI prompt the
mouse cursor is changed to a special shape and can be either
placed between two residues to mark the spot where the
insertion will take place, or mark a sequence region
by left-dragging the mouse, drawing a box around the
residues to be replaced.
</p>

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Edit | Paste Fragment</th></tr>
	<tr>
	<th>Command</th>
	<th>Action</th>
	</tr>

    <tr>
    <td valign="top">
	<span class="menu">Coordinates</span>
    </td>
    <td valign="top">
	Specify how the residues on the clipboard should be
	inserted:
	<ul>
	<li><b>None.</b>
	Insert as sequence-only residues, discarding atom information.
	</li>
	<li><b>Original.</b>
	Clone atom information including coordinates. The sequence
	fragment should be superposed upfront to fit into the
	designated insertion area.
	</li>
	<li><b>Graft.</b>
	Fit sequence fragment into chain by superposing the flanking
	residues of the fragment and the insertion area. After the
	insertion strained geometry will be relaxed by a short
	minimization.
	</li>
	</ul>
    </td>
    </tr>

    <tr>
    <td valign="top">
	<span class="menu">Options</span>
    </td>
    <td valign="top">
	Specify:
	<ul>
	<li>
	If the residues to copy should be renumbered
	to fit into the target sequence.
	</li>
	<li>
	The number of flanking residues used for the grafting.
	</li>
	<li>
	The preview mode. The incoming fragment is shown superposed
	onto the flanking residues before the paste operation is performed.
	</li>
	</ul>
    </td>
    </tr>

    <tr>
    <td valign="top">
	<span class="menu">Paste</span>
    </td>
    <td valign="top">
	Perform the paste operation. If the fragment does not fit into
	the designated chain area, the user is notified and can undo
	the operation.
    </td>
    </tr>

</table>
</center>
</p>

<p>
<a name="Examples"></a>
<h2>Examples</h2>

<div class="example">

<a name="ExampleLoopGraft"></a>
<h3>Example: Loop Grafting</h3>
<p>
In this example, the inactive loop conformation of 1M52 will be replaced
with the active loop conformation of 1IEP

<ol>
<p><li><i>Load the 1IEP and 1M52 complexes into MOE.</i>
<p>
<span class="menu">MOE | File | Open |</span>
select the files <tt>$MOE/sample/mol/1iep.pdb</tt> and <tt>$MOE/sample/mol/1m52.pdb</tt>
then press <b>OK</b>.
</p>
</li></p><p><li>
<i>Open the Sequence Editor.</i>
<p>
<span class="menu">MOE | SEQ</span> (upper right corner)
</p>

</li></p><p><li>
<i>Delete unwanted chains in the system.</i>
<p>For clarity, we will delete the following chains: 2,3,4,5,6 from 1IEP
and 8,9,10,11,12 from 1M52. 
To do this select those chains in the Sequence Editor then 
<b>right click</b> on the Chain Header to bring up the <b>Chain Popup</b>
then choose <b>Delete</b>
<p>
<p align="center">

</li></p><p><li>
<i>Render the system.</i>

<p>
    <ul>
    <li><span class="menu">MOE | RHS | Hide | All Atoms</span></li>
    <li><span class="menu">MOE | Footer | Ribbon | Line</span></li>
    <li><span class="menu">MOE | Footer | Ribbon | Color | Chain</span></li>
    </ul>
<p>
This should render the system as shown below where 1IEP is colored in orange
and 1M52 in magenta.  
<p align="center">
<img src="se/example_render.png" alt="Example Render"/></a>

</li><li>
<i>Superpose the structures.</i>
<p>
Chose <span class="menu">SE | Alignment | Superpose</span>
</p>
<p>
Then in the Superpose Panel press the <b>Superpose</b> button.  This will 
perform an all alpha-carbon superposition, which could be refined further, but
will suffice for this exercise.  The two activation loop conformations however
should be evident. 

<p align="center">
<img src="se/example_superpose.png" alt="Example Superpose"/></a>

</li></p><p><li>
<i>Copy the 1IEP loop.</i>
<p>
Select residues HIS 375 to LYS 400 (columns: 153-178) of 1IEP (Chain 1)
then right-click on the selected residues to bring up the
<b>Selected Residues Popup</b>. 


<p>
Next, in the <b>Selected&nbsp;Residues&nbsp;Popup</b> choose <b>Copy/Paste</b>. 
This will change bring up a prompter in the Sequence Editor as shown below:
<p align="center">
<img src="se/example_seq_loop2.png" alt="Example SEQ"/>

</li></p><p><li>
<i>Paste the 1IEP loop &ndash; Preview</i>
<p>


<p>Now select the corresponding residues HIS 375 to LYS 400 (columns: 153-178)
in 1M52. Notice that the loop is now rendered in the MOE window. 


For clarity, in the
<b>Paste prompter </b> do one of the following:
<p>
<span class="menu">Paste prompter | Options | Show Preview | Ribbons</span>
</p>
or 
<p>
<span class="menu">Paste prompter | Options | Show Preview | Atoms</span>
</p>
<p align="center">
<img src="se/example_preview_options.png" alt="Example SEQ"/>
<p>
Below are two different views of the paste preview.  The image on the left 
shows an atomistic view while the one on the left is a loop view.  The 
atomistic view is useful for looking at potential side chain clashes while 
the loop view is good for looking at the flanking residue overlap.
<p>
Here is how to interpret the paste preview modes:

<p><center>
<blockquote>

<table class="titlebar" border="0" cellpadding="3">

<tr>
<th width="50%">Show Preview: Atoms</th>
<th width="50%">Show Preview:  Ribbon</th>
</tr>

<tr>
<td>
<b>Orange atoms</b> - original backbone atoms on destination chain 
<td>
<b>Orange ribbon</b> - original loop on destination chain 
</tr>

<tr>
<td align="center">
<b>Blue atoms</b> - flanking residues on destination chain 
</td>
<td>
<b>Blue ribbon</b> - flanking region on destination chain 
</td>
</tr>

<tr>
<td align="center">
<b>Green atoms</b> - replacement atoms 
</td>
<td>
<b>Green ribbon</b> - replacement loop
</td>
</tr>

<tr>
<td align="center">
<b>Cyan atoms</b> - flanking residues on the replacement 
</td>
<td>
<b>Cyan ribbon</b> - flanking region of the replacement loop 
</td>
</tr>

<tr>
<td align="center">
<b>Element-colored atoms</b> - nearby sidechain atoms on destination 
</td>
<td>
<b>Grey ribbon</b> - rest of the destination chain 
</td>
</tr>

</table>
</center>


<div align="center">
<img src="se/atoms.png" />&nbsp;<img src="se/ribbons.png" />
</div>

<p>
</li></p><p><li>
<i>Paste the 1IEP loop &ndash; Replace and Minimize</i>
<p>
Finally press
<span class="menu">Paste prompter | Paste</span>

<p>A warning message may appear stating that there is a poor superposition of 
flanking regions. Ideally one should verify that the flanking regions are
well superposed. If not, one may need to either extend the loop regions on 
either side by a few residues, increase the number of flanking residues or 
choose <b>Paste prompter | Coordinates | None</b>.  In this final case, one 
would need to use the homology modeler to model the loop coordinates.
<p>
If you get a warning message simply press <b>Yes</b>.
Once the loop is grafted, there is a quick relaxation procedure that is done
to remove strain in the flanking regions.  However this simple method was not 
designed to deal with sidechain clashes.  
<p>
If you get another warning message indicating that there may be clashes,
simply press <b>Yes</b>. 

Afterward one may need to perform some local minimizations or loop 
conformation sampling to correct the geometric issues.   The choice is up to
user of how this should be done.  
<p>
Here is how to interpret the view of the final result:

<p><center>
<blockquote>

<table class="titlebar" border="0" cellpadding="3">

<tr>
<th width="50%">Final result via Show Preview: Atoms</th>
<th width="50%">Final result via Show Preview: Ribbon</th>
</tr>

<tr>
<td align="center">
<b>Blue atoms</b> - flanking residues on destination chain 
</td>
<td>
<b>Blue ribbon</b> - flanking region on destination chain 
</td>
</tr>

<tr>
<td align="center">
<b>Green atoms</b> - replacement atoms 
</td>
<td>
<b>Green ribbon</b> - replacement loop
</td>
</tr>


</table>
</center>


<p align="center">
<img src="se/example_final3.png" 
alt="Example Final"/></a>

</div>



<a name="SVL_Commands"></a><a name="SVL"></a><h2>SVL Commands</h2>

<a href="../fcnref/chain.htm">Chain Functions</a><br />
<a href="../../proteins/fcnref/intropro.htm">Protein Functions</a><br />
<a href="../fcnref/residue.htm">Residue Functions</a>

<a name="References"></a><h2>References</h2>

<table>
<tr>
<td valign="top">[Arnott&nbsp;1975]</td>
<td valign="top">
Arnott, S., Selsing, E.; The conformation of C-DNA;
<i>Journal of Molecular Biology 98</i> (<b>1975</b>) 265&ndash;269.
</td>
</tr>

<tr>
<td valign="top">
[Eisenberg&nbsp;1984] 
<td valign="top">
Eisenberg, D., Schwarz, E., Komaromy, M., and Wall, R.;
<i>J.Mol.Biol. 179(1)</i> (<b>1984</b>) 125&ndash;142.

<tr>
<td valign="top">
[Kabsch&nbsp;1983] 
<td valign="top">
Kabsch, W. and Sander, C.;
Dictionary of Protein Secondary Structures: Pattern Recognition of
Hydrogen-Bonded and Geometrical Features; <i>Biopolymers 22</i>
(<b>1983</b>) 88&ndash;47.

</table>


<a name="See_Also"></a><h2>See Also</h2>

<a href="select.htm#ResidueSelector">Residue Selector</a><br />
<a href="../../proteins/probuild.htm">Protein Builder and Rotamer Explorer</a><br />
<a href="../../proteins/align.htm#CONSTRAINTS">Constraining
and Unconstraining Residues</a><br />
<a href="mainwin.htm#MOE_Window">MOE Window</a>


<noscript>
	<hr noshade="noshade" />
	<font size="2"> Copyright &copy; 1997&ndash;2015
	<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a> </font>
</noscript>
<div id="MoeFooter"></div>
</body>
</html>
