<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<!--
!!	moemanuals/moe/moeweb_intro.htm		MOE/web applications
!!
!!	05-nov-2009 (al) created
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../manstyle.css" />


<title>MOE/web Applications</title>
</head>

<body bgcolor="#ffffff">
<a href="../index.htm"><img src="../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>MOE/web Applications</h2>
<hr noshade="noshade" />

<ul>
<li><a href="#Homology_Search">Homology Search</a></li>
<li><a href="#SD_Filter">SD Filter</a></li>
<li><a href="#Small_Molecule_Properties">Small Molecule Properties</a></li>
<li><a href="#SFCHECK">Structure Factor Check</a></li>
<li><a href="#SAReport">Structure-Activity Report</a></li>
<li><a href="#Antibody_Homology_Modeling">Antibody Homology Modeling</a></li>
<li><a href="#Database_Browser">Database Browser</a></li>
<li><a href="#References">References</a></li>
</ul>

<p>The MOE distribution includes a number of MOE/web applications
that provide web access to built-in MOE applications.
A tutorial is also provided which describes how to
develop a new MOE/web application.  To run the MOE/web applications,
you must first have <a href="../install/web_inst.htm">installed MOE/web</a>.
In your browser, visit the web page
</p>

<pre>
    http://WEBSITE/moe/web/index.htm
</pre>

<p>The following page should appear:
</p>

<p>
<center>
<img src="moeweb_intro/moeweb_toc.png"
alt="MOE/web Table of Contents" border="1" />
</center>
</p>

<a name="Homology_Search"></a>
<h2>Homology Search</h2>

<p>The Protein Homology Search web application
provides web access to MOE's built-in
<a href="../proteins/prosearch.htm">Homology Searching</a> application.
The web application accepts an amino acid
sequence and returns the results of a homology search.  The output
can be filtered by applying a cutoff to the statistical significance
Z-scores of the search results.  More details of the methodology
can be found in the chapter on
<a href="../proteins/prosearch.htm">Homology Searching</a>.
Shown here is an example of the results page (truncated) generated by the
application.
</p>

<p>
<center>
<img src="moeweb_intro/web_homsrch_results.png"
alt="MOE/web Homology Search Results" border="1" />
</center>
</p>

<a name="SD_Filter"></a>
<h2>SD Filter</h2>

<p>The SD Filter web application provides web access to the
<a class="svl" href="../moe/fcnref/sdwash.htm">sdwash</a> and
<a class="svl" href="../moe/fcnref/sdfilter.htm">sdfilter</a>
tools of MOE's suite of SD tools.
The purpose of the SD Filter application is to
standardize and correct molecular data stored in SD files,
rejecting records that violate such rules as Lipinski's
rule-of-five [Lipinski&nbsp;1999], Oprea lead-like [Oprea&nbsp;2000],
or whether a molecule contains reactive groups.</p>

<p>The washing step normalizes the molecular data for
each record in the input SD file.  Such operations as
hydrogen adjustment, salt (small component) removal,
acid-base normalization,
ylide-bond standardization, wedge-bond regeneration for
correctness and aesthetics, and flat depiction
using MOE's <a href="../apps/view2d.htm">2D depictor</a>
are performed.  These operations are optional and
can be enabled or disabled as desired.
</p>

<p>The filtering step removes records from the input
SD files based on rules defined by programmable filters.
In addition to the Lipinski druglike and Oprea leadlike
rules, there are filters for reactive group content,
restriction on ring size, molecular weight, number of
chiral centers, number of rotatable bonds, and element composition,
among others.
There are also filters for substructure requirements that
are specified using
<a href="molsystems/smiles.html">SMARTS</a> strings.
</p>

<p>
All records that pass the filters are written to the
output file.  The rejected records are written to a separate file,
and both files are available for download.
</p>

<p>Here is an example of the output (truncated) generated by the
application:</p>

<p>
<center>
<img src="moeweb_intro/web_sdfilter_results.png"
alt="MOE/web SD Filter Results" border="1" />
</center>
</p>

<a name="Small_Molecule_Properties"></a>
<h2>Small Molecule Properties</h2>

<p>The Small Molecule Properties application provides a
drawing area in which to sketch the small molecule for
which properties will be calculated.  The sketching tool
is Peter Ertl's JME Molecular Editor,
available at
<a class="svl" href="http://www.molinspiration.com/jme">www.molinspiration.com/jme</a>.
This is a Java applet and therefore Java must be installed
in order for the Small Molecule Properties application to run.
</p>

<p>Various molecular properties are calculated, including
number of H-bond donors and acceptors, number of chiral centers,
number of rotatable bonds, logP (octanol/water), molecular weight,
and van der Waals surface area and volume.  These calculated
values correspond to molecular descriptors available
in MOE's <a href="../quasar/descr.htm">Calculate Descriptors</a>
application.
</p>

<p>Here is an example of the output generated by
the Small Molecule Properties application.  The results page
provides an Edit button for returning to the molecule editor
and making modifications.
</p>

<p>
<center>
<img src="moeweb_intro/web_molprop_results.png"
alt="MOE/web Small Molecular Properties Results" border="1" />
</center>
</p>

<a name="SFCHECK"></a>
<h2>Structure Factor Check</h2>

<p>The Structure Factor Check application is based on the
SFCHECK program [Vaguine&nbsp;1999] from the
<a href="http://www.ccp4.ac.uk">CCP4 suite</a>, and also
computes the Real Space R Factor [Jones&nbsp;1991];
Using experimental structure factor and model molecular coordinates,
it computes various electron density maps and assesses the
quality of the fit to the model.  The results may highlight
possible errors in the model.
</p>

<p>The application takes as input a structure file
containing molecular coordinates, and a structure factor
file.  The structure files can be provided in PDB,
mmCIF, <a href="fcnref/moefmt.htm">MOE</a>, or MOL2 formats.
The structure factor files can be provided in CIF, MTZ, or CNS formats.
</p>

<p>The space group, cell dimensions, and resolution
limit can be provided optionally as additional parameters,
when they are missing or when you wish to override
them.
</p>

<p>After the input files and parameters have been
specified, press <b>Launch Check</b>.  At this point, the
application will read the input files and, if there
is more than one intensity/magnitude or sigma column,
the input page will be regenerated with <b>Specify Columns</b>
options.  Press <b>Launch Check</b> again to do the calculations.
</p>

<p>The output page generated by the application is divided into
several areas, including an <b>Input</b> area that summarizes
the inputs, and a <b>Plots</b> area.
</p>

<p>
<center>
<img src="moeweb_intro/web_sfcheck_results.png"
alt="MOE/web Structure Facture Check Results" border="1" />
</center>
</p>

<p>The plots give a visual
indication of how well the model matches the electron
density.  Included are
such plots as density correlation [Vaguine&nbsp;1999],
where low values indicate poor agreement,
Real Space R Factor [Jones&nbsp;1991],
which measures the agreement between the observed and the
calculated electron densities, B-Factor, which is a measure
of atom mobility, and the Wilson Plot, which assesses data quality.
</p>

<p>
<center>
<img src="moeweb_intro/web_sfcheck_densityindex.png"
alt="MOE/web Structure Facture Check Density Index Plot" border="0" />
</center>
</p>

<a name="SAReport"></a>
<h2>Structure-Activity Report</h2>

<p>The Structure-Activity Report web application provides
web access to MOE's built-in
<a href="../apps/sar.htm">Structure-Activity Report</a> application.
The input of the web application is a molecular database containing
small molecules with their activity data.
The input molecules are assumed to be mostly based on a small number
of scaffolds which are sufficiently analogous that the implied
R-groups have common meaning throughout the series.
</p>

<p>The input database,
provided in either MOE
<a href="molsystems/molrep.html#MDB_CTAB_Formats">MDB</a>
(<tt>.mdb</tt>) or MDL SD (<tt>.sdf</tt>) file formats,
is read and parsed when <b>Submit Input Data</b> is pressed.
A new page containing various options pertaining to the
input data is then generated.
</p>

<p>This second page provides a means of specifying the meaning of
the numeric data fields contained in the database,
as well as of providing molecular scaffolds.
Note that activity data must be present in order for the
application to run successfully.
</p>

<p>
If scaffolds are not provided, they will be automatically detected.
Scaffolds can be provided in a molecular structure file or
using the molecular sketcher provided, Alex Clark's
<a href="http://sketchel.sf.net">SketchEl</a>
molecular drawing tool.  This is a Java applet, therefore
Java must be installed to be able to input a scaffold
using the sketcher.
</p>

<p>Pressing <b>Generate Report</b> produces a page
similar to the following (truncated):
</p>

<p>
<center>
<img src="moeweb_intro/web_sareport_results.png"
alt="MOE/web Structure-Activity Results" border="1" />
</center>
</p>


<a name="Antibody_Homology_Modeling"></a>
<h2>Antibody Homology Modeling</h2>

<p>The Antibody Homology Modeling web application provides web access
to MOE's built-in
<a href="../proteins/antibody.htm">Antibody Modeler</a>
application.
It takes as input query sequences for light and heavy antibody
chains and a set of template searching and forcefield parameters.
The VL and VH chains can be supplied as amino acid sequences,
or as a FastA (<tt>.fst</tt>) file.
</p>

<p>The outputs are presented over several pages:
<b>CDR Clusters</b> (Complementarity Determining Regions),
<b>Hydrogen Bonds</b>,
<b>ASA</b> (Accessible Surface Area) and <b>RMSD</b>
(Root Mean Square Distance).
An example of the generated CDR Clusters and ASA output pages
are as follows:
</p>

<p>
<center>
<img src="moeweb_intro/web_antibody_results.png"
alt="MOE/web Antibody Homology Results" border="1" />
</center>
</p>

<p>
<center>
<img src="moeweb_intro/web_antibody_asa.png"
alt="MOE/web Antibody Homology Results" border="1" />
</center>
</p>

<a name="Database_Browser"></a>
<h2>Database Browser</h2>

<p>The Database Browser web application provides a web
equivalent of MOE's
<a href="../mdb/dbbrowse.htm">Database Browser</a>
application.
It generates a 2D snapshot of each compound in a
molecular database, which can be viewed in either
table or tile format.
</p>

<p>The application generates 3 pages of output:
<b>Database Info</b>, <b>Tile Layout</b>, and <b>Table Layout</b>.
The Database Info gives information about the
input database.  The remaining two pages are launch
points for generating the tile and table layouts,
respectively.

<p>
In both Tile and Table mode, MOE's
<a href="../apps/view2d.htm">2D depictor</a> can
be chosen to produce aesthetically pleasing
2D renderings of the molecules.
The Table view can displays each row in the
database as a row in the page.  Numeric fields
can be shown.
In Tile view, multiple molecules can be placed
on each row.  Tiles can be annotated.
</p>

<p>Here is an example of the tiled output generated
by the application:
</p>

<p>
<center>
<img src="moeweb_intro/web_dbbrowse_results.png"
alt="MOE/web Database Browser Results" border="1" />
</center>
</p>

<a name="References"></a>
<h2>References</h2>

<p>
<table border="0" cellpadding="3">
<tr>
<td valign="top">[Jones&nbsp;1991]</td>
<td valign="top">
Jones, T.A., Zou, J.-Y., Cowan, S.W., and Kjeldgaard, M.;
Improved Methods for Building Protein Models in Electron Density Maps and the
Location of Errors in these Models;
<i>Acta Cryst. A47</i> (<b>1991</b>) 110&ndash;119.
</td>
</tr>
<tr>
<td valign="top">[Lipinski&nbsp;1999]</td>
<td valign="top">
Lipinski, C.A., et al.;
Experimental and Computational Approaches to Estimate Solubility and
Permeability in Drug Discovery and Development Settings;
<i>Adv. Drug Deliv. Rev. 23</i> (<b>1997</b>) 3–25. 
</td>
</tr>
<tr>
<td valign="top">[Oprea&nbsp;2000]</td>
<td valign="top">
Oprea, T.I.;
Property Distribution of Drug-Related Chemical Databases;
<i>J. Comp. Aid. Mol. Des. 14</i> (<b>2000</b>) 251–264.
</td>
</tr>
<tr>
<td valign="top">[Vaguine&nbsp;1999]</td>
<td valign="top">
Vaguine, A.A., Richelle, J., and Wodak, S.J.;
SFCHECK: a Unified Set of Procedures for Evaluating the Quality of
Macromolecular Structure-Factor Data and their Agreement with the Atomic Model;
<i>Acta Cryst. D55</i> (<b>1999</b>) 191&ndash;205.
</td>
</tr>
</table>
</p>

<p><a href="../index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
