<!DOCTYPE html>
<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd" lang="en-US" xml:lang="en-US" data-mc-search-type="Stem" data-mc-help-system-file-name="Documentation.xml" data-mc-path-to-help-system="../" data-mc-target-type="WebHelp2" data-mc-runtime-file-type="Topic" data-mc-preload-images="false" data-mc-in-preview-mode="false" data-mc-toc-path="Help|Prime Help">
    <head>
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <meta charset="utf-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <link href="../Skins/Default/Stylesheets/Slideshow.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/TextEffects.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Topic.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Components/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Resources/Stylesheets/docs.css" rel="stylesheet" type="text/css" /><title>Prime Structure Prediction — Edit Alignment Step</title>
        <script src="../Resources/Scripts/custom.modernizr.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/jquery.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.config.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/foundation.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/plugins.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/MadCapAll.js" type="text/javascript">
        </script>
    </head>
    <body><a name="aanchor1597"></a>
        <MadCap:concept term="Help Topics" /><a name="aanchor1598"></a>
        <MadCap:concept term="Prime" />
        <table class="navlink">
            <tr>
                <td class="navlink"><a class="navbutton" href="#using">Using</a>
                </td>
                <td class="navlink"><a class="navbutton" href="#features">Features</a>
                </td>
                <td class="navlink"><a class="navbutton" href="#links">Links</a>
                </td>
            </tr>
        </table>
        <h1 class="title">Prime Structure Prediction — Edit Alignment Step</h1><a name="summary"></a>
        <p>The purpose of this step is to find a sastisfactory sequence alignment between the templates and the query, which can be used to build a structure.</p>
        <p>For general information on the panel, see the topic <a href="structure_prediction.html">Structure Prediction Panel</a>.</p>
        <h2><a name="using"></a>Using the Edit Alignment Step</h2>
        <p>There are several ways to produce an alignment for use in the <span class="GUI">Build Structure</span> step:</p>
        <ul>
            <li value="1">Accept the BLAST/PSI-BLAST alignment generated in the <span class="GUI">Find Homologs</span> step and currently displayed in the Sequence Viewer.</li>
            <li value="2">Import an alignment, by clicking <span class="GUI">Import</span>, to the right of the <span class="GUI">Alignments</span> table.</li>
            <li value="3">Generate an alignment, either with ClustalW or with Prime SSP, by selecting the appropriate <span class="GUI">Alignment method</span> option, then clicking <span class="GUI">Align</span>. Prime SSP requires secondary structure predictions (SSPs) (see below).</li>
            <li value="4">Manually edit any of these alignments.</li>
            <li value="5">If you are modeling a GPCR, select <span class="GUI">GPCR-specific mode</span> and click <span class="GUI">Align</span> to create the alignment.</li>
        </ul>
        <p>For technical details about the alignment program, see the <a href="../prime_command_reference/prime_commands_ska.htm">ska—Protein Structure Alignment</a> topic in the <i>Prime Command Reference Manual</i>.</p>
        <h3>Secondary Structure Predictions</h3>
        <p>To generate an SSP for the query, click <span class="GUI">Run</span>. If you want to save an SSP for later use, right-click on an SSP for a menu of options, including deleting the SSP, hiding all SSPs, or opening the <span class="GUI">Export SSP</span> panel. If there is a difficulty with exporting a file, an error message is displayed. To import an SSP, click the <span class="GUI">Import</span> button for <span class="GUI">Secondary structure prediction</span>, and use the file selector to locate and import the SSP.</p>
        <p>If you believe an SSP is incorrect, you may delete it entirely or edit the incorrect portions. To delete the SSP, right-click on the SSP and choose <span class="GUI">Delete</span>. To edit the SSP, click the <span class="GUI">Edit SSP</span> toolbar button,</p>
        <img src="../Resources/Images/Toolbuttons/prime_edit_ssp.png" />
        <p>then highlight the part of the SSP you wish to change, and type the character <tt>e</tt>, <tt>h</tt>, or <tt>-</tt>. The highlighted region of the prediction is changed to your choice: E (strand), H (helix), or - (loop).</p>
        <p>To retrieve an unmodified SSP after editing, right-click on it and choose <span class="GUI">Revert</span>.</p>
        <h3>Editing the Alignment</h3>
        <p>You can edit any alignment using combinations of the Prime toolbar buttons <span class="GUI">Slide Freely</span>, <span class="GUI">Slide as Block</span>, <span class="GUI">Set Anchors</span>, <span class="GUI">Lock Gaps</span>, and <span class="GUI">Unlock Gaps</span>. A preliminary model of the query is displayed in the Workspace, colored by <span class="GUI">Residue Property</span>, revealing the location of charged, polar, and hydrophobic residues.</p>
        <h3> Error Messages</h3>
        <dl>
            <dt><span class="GUI">Alignment check</span>
            </dt>
            <dd>
                <p>This error dialog box lists possible alignment problems that you may want to fix before continuing to the next step. For example, gaps in secondary structure elements of the template, or gaps in the query that are aligned to secondary structure elements of the template. Click <span class="GUI">Continue</span> to go on to the next step anyway; click <span class="GUI">Cancel</span> to remain in <span class="GUI">Edit Alignment</span>.</p>
            </dd>
        </dl>
        <h2><a name="features"></a>Edit Alignment Step Features</h2>
        <ul>
            <li value="1"><a href="#feature4"><span class="GUI">Alignment method</span> options</a>
            </li>
            <li value="2"><a href="#feature5"><span class="GUI">GPCR-specific alignment</span> option</a>
            </li>
            <li value="3"><a href="#feature1"><span class="GUI">Run</span> button</a>
            </li>
            <li value="4"><a href="#feature2"><span class="GUI">Import</span> button</a>
            </li>
            <li value="5"><a href="#feature3"><span class="GUI">Align</span> button</a>
            </li>
            <li value="6"><a href="#feature7"><span class="GUI">Identify Globally Conserved Residues</span> button</a>
            </li>
            <li value="7"><a href="#feature6"><span class="GUI">Family name</span> text box</a>
            </li>
            <li value="8"><a href="#feature8"><span class="GUI">Alignments</span> table</a>
            </li>
            <li value="9"><a href="#feature9"><span class="GUI">Import</span> and <span class="GUI">Export</span> buttons</a>
            </li>
        </ul>
        <p />
        <dl>
            <dt><a name="feature4"></a><span class="GUI">Alignment method</span> options</dt>
            <dd>
                <p>Choose the alignment program to be used for alignment.</p>
                <ul>
                    <li value="1">
                        <p><span class="GUI">ClustalW</span>—Align the query and template using ClustalW, based only on the sequence information. This is usually very fast, and is best used when there is high sequence identity between the query and the template.</p>
                    </li>
                    <li value="2">
                        <p><span class="GUI">Prime STA</span>—Align the query and template using the Prime STA program, which uses secondary structure assignments as well as sequence information. This choice requires secondary structure predictions (SSPs).</p>
                    </li>
                </ul>
            </dd>
            <dt><a name="feature5"></a><span class="GUI">GPCR-specific alignment</span> option</dt>
            <dd>
                <p>Run an alignment that includes special features for aligning GPCRs, which include fingerprint matching, a customized GPCR sequence database, and identification of transmembrane helices.</p>
            </dd>
            <dt><a name="feature1"></a><span class="GUI">Run</span> button</dt>
            <dd>
                <p>Run all available secondary structure prediction programs. SSpro is bundled with Prime, but PSIPRED, which is an optional third-party program, is highly recommended for optimal results, especially for GPCRs. PSIPRED is not available on Windows. See <a href="info_third.html">Prime Third-Party Programs</a> for more information on this program.</p>
            </dd>
            <dt><a name="feature2"></a><span class="GUI">Import</span> button</dt>
            <dd>
                <p>Import a secondary structure prediction for the query. Opens a file selector in which you can select the SSP file. The SSP must be in FASTA, Maestro, or PSPIRED format. If the file contains bad data or the sequence doesn't match the query sequence, an error message is displayed.</p>
            </dd>
            <dt><a name="feature3"></a><span class="GUI">Align</span> button</dt>
            <dd>
                <p>Run a full alignment of the query to the template, using the chosen alignment method.</p>
            </dd>
            <dt><a name="feature6"></a><span class="GUI">Query family name</span> text box</dt>
            <dd>
                <p>The family name for the query is displayed in this text box after you search for families.</p>
            </dd>
            <dt><a name="feature7"></a><span class="GUI">Identify Globally Conserved Residues</span> button</dt>
            <dd>
                <p>Run HMMER on the Pfam database to search for globally conserved residues from a multiple sequence alignment of homologous proteins, and identify the protein family. The conserved residues are shown in the sequence viewer, colored according to the template color if both the template and the query have the conserved residue. This allows you to easily identify conserved residues that are not in the template or in the query.</p>
            </dd>
            <dt><a name="feature8"></a><span class="GUI">Alignments</span> table</dt>
            <dd>
                <p>Lists the templates with their alignment score, number of identities, positives, and gaps.</p>
            </dd>
            <dt><a name="feature9"></a><span class="GUI">Import</span> and <span class="GUI">Export</span> buttons</dt>
            <dd>
                <p>Import an alignment from a file, or export an alignment to a file. These buttons open a file selector, in which you can navigate to the desired location and select or name the file.</p>
                <p><b>Note:</b> The template PDB ID must be uppercase in the alignment file. If it is not, you must edit the alignment file before importing it.</p>
            </dd>
        </dl>
        <h2><a name="links"></a>Related Topics</h2>
        <ul>
            <li value="1"><a href="find_homologs_step.html" class="MCXref xref" xrefformat="{paratext}">Prime Structure Prediction — Find Homologs Step</a>
            </li>
            <li value="2"><a href="build_structure_step.html" class="MCXref xref" xrefformat="{paratext}">Prime Structure Prediction — Build Structure Step</a>
            </li>
            <li value="3"><a href="structure_prediction.html" class="MCXref xref" xrefformat="{paratext}">Structure Prediction Panel</a>
            </li>
            <li value="4"><a href="job_options.html" class="MCXref xref" xrefformat="{paratext}">Structure Prediction - Job Options Dialog Box</a>
            </li>
            <li value="5"><a href="toolbar.html" class="MCXref xref" xrefformat="{paratext}">Prime Structure Prediction Toolbar</a>
            </li>
            <li value="6"><a href="info_third.html" class="MCXref xref" xrefformat="{paratext}">Prime Third-Party Programs</a>
            </li>
        </ul>
    </body>
</html>