<!DOCTYPE html>
<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd" lang="en-US" xml:lang="en-US" data-mc-search-type="Stem" data-mc-help-system-file-name="Documentation.xml" data-mc-path-to-help-system="../" data-mc-target-type="WebHelp2" data-mc-runtime-file-type="Topic" data-mc-preload-images="false" data-mc-in-preview-mode="false" data-mc-toc-path="Help|Prime Help">
    <head>
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <meta charset="utf-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <link href="../Skins/Default/Stylesheets/Slideshow.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/TextEffects.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Topic.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Components/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Resources/Stylesheets/docs.css" rel="stylesheet" type="text/css" /><title>Prime Structure Prediction — Input Sequence Step</title>
        <script src="../Resources/Scripts/custom.modernizr.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/jquery.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.config.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/foundation.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/plugins.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/MadCapAll.js" type="text/javascript">
        </script>
    </head>
    <body><a name="aanchor6010"></a>
        <MadCap:concept term="Help Topics" /><a name="aanchor6011"></a>
        <MadCap:concept term="Prime" />
        <table class="navlink">
            <tr>
                <td class="navlink"><a class="navbutton" href="#using">Using</a>
                </td>
                <td class="navlink"><a class="navbutton" href="#features">Features</a>
                </td>
                <td class="navlink"><a class="navbutton" href="#links">Links</a>
                </td>
            </tr>
        </table>
        <h1 class="title">Prime Structure Prediction — Input Sequence Step</h1><a name="summary"></a>
        <p>This step is used to read in protein (amino acid) sequences and designate the query sequence.</p>
        <p>For general information on the panel, see the topic <a href="structure_prediction.html">Structure Prediction Panel</a>.</p>
        <h2><a name="using"></a>Using the Input Sequence Step</h2>
        <p>The purpose of this step is to define a query sequence. You can read the sequence from a sequence file, add it as text, obtain it from the PDB, or obtain it from the Workspace. Several common sequence file formats are supported, including FASTA, EMBL, GENBANK, and PIR. Sequences can be read from PBB files, from the SEQRES records, if present, or from the ATOM records if SEQRES records are missing. You can add more than one sequence, but only one can be used as a query. If you want to build a multimer, you can do so by combining information from separate runs for each chain in the Build Structure step.</p>
        <p><b>To read a sequence from a file:</b>
        </p>
        <ul>
            <li value="1">
                <p>Click <span class="GUI">File</span> and specify a file containing the desired sequence.</p> <p>Several file formats are supported, including FASTA, EMBL, GENBANK, PDB, and PIR. For a complete list, see Appendix C of the <i>Prime User Manual</i>. File formats are auto-detected by the program. Sequences must be in standard uppercase one-letter code format.</p></li>
        </ul>
        <p><b>To read a sequence from the PDB:</b>
        </p>
        <ul>
            <li value="1">
                <p>Click <span class="GUI">PDB ID</span> and specify the PDB ID and optionally the chain name.</p> <p>The PDB file is downloaded and the sequences are read in.</p></li>
        </ul>
        <p><b>To enter a sequence as text:</b>
        </p>
        <ul>
            <li value="1">
                <p>Click <span class="GUI">Text</span> and type in the sequence or paste it from the clipboard.</p> <p>If there are multiple chains with the same sequence, the sequence will be read in only once.</p></li>
        </ul>
        <p><b>To read a sequence from the Workspace:</b>
        </p>
        <ul>
            <li value="1">
                <p>Click <span class="GUI">Workspace</span>.</p> <p>All sequences represented in the Workspace are added to the Sequences table. If there are multiple chains with the same sequence, the sequence is read in only once.</p></li>
        </ul>
        <p>Once you have obtained one or more sequences, you can select the sequence you want to use for the query from the <span class="GUI">Sequences</span> list. By default, the first sequence read in is selected and displayed in the sequence viewer in the upper section of the panel.</p>
        <p>If you want to modify the sequence you have selected as a query, click <span class="GUI">Edit Sequence</span> and make any changes you want in the dialog box that opens. You might, for example, want to remove extra terminal residues or insertions, or mutate the sequence.</p>
        <p>Sequences longer than 1,000 residues generally include more than one domain. You should consider dividing these sequences into probable domains and treat each domain as an individual query.</p>
        <p>When you are satisfied with the query sequence, click <span class="GUI">Next</span> to go to the Find Homologs step.</p>
        <h3>Error Messages</h3>
        <p>If one of these messages appears when you click <span class="GUI">Next</span>, you will not be able to proceed until the problem is resolved.</p>
        <dl>
            <dt><span class="GUI">Invalid File Message</span>
            </dt>
            <dd>
                <p>This message is shown when a valid sequence cannot be extracted from the file.</p>
                <ul>
                    <li value="1">Check that the sequence is in one of the supported sequence file formats.</li>
                    <li value="2">If it is in FASTA format, check that there are no extra spaces between the "&gt;" and the sequence name and that there are no extra spaces at the end of the line.</li>
                </ul>
            </dd>
            <dt><span class="GUI">No Sequences Found</span>
            </dt>
            <dd>
                <p>This message is shown when no valid sequences can be extracted from the Maestro Workspace.</p>
                <ul>
                    <li value="1">Make sure that the correct project is open.</li>
                    <li value="2">Check the Project Table to make sure the correct entries are included in the Workspace.</li>
                    <li value="3">Display the Workspace sequence viewer to check that the structure has a sequence.</li>
                </ul>
            </dd>
            <dt><span class="GUI">Invalid Sequence Characters</span>
            </dt>
            <dd>
                <p>The sequence contains nonstandard one-letter codes.</p>
                <ul>
                    <li value="1">Check that the sequence is in one-letter code and is all in upper case.</li>
                    <li value="2">Change or edit out any nonstandard characters.</li>
                </ul>
            </dd>
        </dl>
        <h2><a name="features"></a>Input Sequence Step Features</h2>
        <ul>
            <li value="1"><a href="#feature1"><span class="GUI">Get sequence from</span> buttons</a>
            </li>
            <li value="2"><a href="#feature2"><span class="GUI">Sequences</span> table</a>
            </li>
            <li value="3"><a href="#feature3"><span class="GUI">Edit Sequence</span> button</a>
            </li>
        </ul>
        <dl>
            <dt><a name="feature1"></a><span class="GUI">Get sequence from</span> buttons</dt>
            <dd>
                <p>Use these buttons to get sequences from various sources.</p>
            </dd>
            <ul>
                <li value="1">
                    <p><span class="GUI">File</span>—Read the sequence from a file. Several file formats are supported, including PDB, FASTA, EMBL, GENBANK, and PIR. File formats are auto-detected by the program. Sequences from PDB files are read from the SEQRES records, if they exist; otherwise they are read from the ATOM records. If SEQRES records do not exist and residues are missing from the ATOM records, the query sequence will contain gaps. For other file formats, sequences must be in standard uppercase one-letter code.</p>
                </li>
                <li value="2">
                    <p><span class="GUI">Text</span> button—Enter the sequence as text. Opens the <span class="GUI">Get Sequence from Text</span> dialog box, in which you can type in the sequence, or paste it from the clipboard with <span class="Keys">Ctrl+V</span> (<span class="Keys">⌘V</span>).</p>
                </li>
                <li value="3">
                    <p><span class="GUI">PDB ID</span>—Obtain sequences from the Protein Data Bank (PDB). Opens the <a href="get_sequence_from_pdb.html"><span class="GUI">Get Sequence from PDB ID</span> dialog box</a>, in which you can enter the PDB ID and optional chain name. The PDB file is downloaded and the sequence read from the file, as described above.</p>
                </li>
                <li value="4">
                    <p><span class="GUI">Workspace</span>—Read sequences from proteins in the Workspace. If there are duplicates, the sequence is read in only once.</p>
                </li>
            </ul>
            <dt><a name="feature2"></a><span class="GUI">Sequences</span> table</dt>
            <dd>
                <p>This table lists the sequences that have been added to the run, in the order they were read in, with their chain lengths. You can select a single sequence as the query. This sequence is displayed in the sequence viewer.</p>
            </dd>
            <dt><a name="feature3"></a><span class="GUI">Edit Sequence</span> button</dt>
            <dd>
                <p>Edit the sequence that is selected in the <span class="GUI">Sequences</span> table. Opens a panel in which you can edit the sequence as a text string.</p>
            </dd>
        </dl>
        <h2><a name="links"></a>Related Topics</h2>
        <ul>
            <li value="1"><a href="find_homologs_step.html" class="MCXref xref" xrefformat="{paratext}">Prime Structure Prediction — Find Homologs Step</a>
            </li>
            <li value="2"><a href="toolbar.html" class="MCXref xref" xrefformat="{paratext}">Prime Structure Prediction Toolbar</a>
            </li>
            <li value="3"><a href="structure_prediction.html" class="MCXref xref" xrefformat="{paratext}">Structure Prediction Panel</a>
            </li>
        </ul>
    </body>
</html>