define({"1372":{i:0.000144746338163968,u:"../primex_user_manual/primex_input_calcsettings_bfactor.htm",a:"There are three options for overall B-factor scaling: Anisotropic—A single anisotropic scaling factor is optimized. Both isotropic and anisotropic components of this scale factor are applied, to the isotropic B-factors and to Fo, respectively This option is useful in most circumstances, and is also ...",t:"Overall B-Factor Scaling"},"1373":{i:0.000143314614650675,u:"../primex_user_manual/primex_input_calcsettings_filter.htm",a:"You can filter reflection data that are either too weak or too strong by selecting Reject reflections with and specifying the cutoff criteria. This option enables the rejection of reflections based on cutoffs on their F values. When you change any cutoff, the number of reflections rejected is ...",t:"Reflection Filtering"},"1374":{i:0.000144612772877476,u:"../primex_user_manual/primex_input_calcsettings_hydrogens.htm",a:"Proper handling of hydrogens is critical in PrimeX. The presence of hydrogens is essential in any of the calculations that use force fields, because PrimeX uses the OPLS all-atom force fields. Once you have added hydrogens, and optimized their positions, you may retain them for the rest of the ...",t:"Hydrogen Placement for PrimeX"},"1375":{i:0.000150625963869755,u:"../primex_user_manual/primex_input_calcsettings_forcefield.htm",a:"Force Field Solvation Model for PrimeX When performing real-space refinements of any kind or reciprocal-space minimizations and rigid-body refinements, you can use a surface generalized Born (SGB) implicit solvation model for the solvent regions of the crystal, by selecting Use implicit solvent. ",t:"Force Field Solvation Model for PrimeX"},"1376":{i:0.000170549622528987,u:"../primex_user_manual/primex_input_calcsettings_planar.htm",a:"The force fields used in PrimeX real-space refinement permit nominally planar groups to deviate from planarity. Such deviations can be an indication of the need for more refinement. However, you can also choose to restrain these groups more closely to planarity by including additional restraints on ...",t:"Restraints on Planar Groups in PrimeX"},"1377":{i:0.000175751577930589,u:"../primex_user_manual/primex_input_calcsettings_memory.htm",a:"You can set the amount of memory used in the fast Fourier transform by entering a value in the Maximum memory use in FFT text box. Larger values will result in faster calculations, but this value should not exceed the available physical memory. The default value is a safe choice for all but the most ...",t:"Memory Use in PrimeX FFT"},"1378":{i:0.000233286467752922,u:"../primex_user_manual/primex_input_calcsettings_realspace.htm",a:"There are a number of options that apply only to real-space refinement tasks.  For density maps, you can choose to use the phases from the model, or make use of phased reflection data. The options are: Observed amplitudes and phases calculated from model—phases are calculated from the model, using ...",t:"Real-Space Options for PrimeX"},"1379":{i:0.000171206150539604,u:"../primex_user_manual/primex_input_calcsettings_colors.htm",a:"To change the default map colors, click Set Default Map Colors. You can then choose colors for each of the map types in the Set Default Map Colors dialog box. Default map colors can also be set in the Preferences panel, in the Surfaces – Creation section. Colors for individual maps can be changed ...",t:"PrimeX Map Colors"},"1380":{i:0.000154507987789096,u:"../primex_user_manual/primex_input_display.htm",a:"When you click OK in the Input Data dialog box, the initial structure is displayed in the Workspace. If you have the sequence viewer displayed in the main window, the sequence of the initial structure is displayed in the sequence viewer. The Project Table panel opens with the initial structure ...",t:"Initial Structure and Sequence Display in PrimeX"},"1381":{i:0.000105104887766899,u:"../primex_user_manual/primex_input_proteinprep.htm",a:"The basic preparation of the protein structure (assignment of bond orders and metal treatment) are performed on import when you select Assign bond orders. If you did not select this option when you imported the structure, you can assign bond orders by choosing Assign Bond Orders from the Tools menu. ...",t:"Preparing the Protein Structure for PrimeX"},"1382":{i:0.000100091136123334,u:"../primex_user_manual/reciprocal_space.htm",a:"Often, the first step in a refinement is the overall positioning of the protein, which is done by treating the protein as a rigid body. Simulated annealing is generally used in the middle stages of the refinement, after the larger changes in positions have been completed with rigid-body and ...",t:"Reciprocal-Space Refinement"},"1383":{i:0.000136814001876671,u:"../primex_user_manual/reciprocal_space_general.htm",a:"In the General tab, you specify the target for refinement and the relative weighting of X-ray and energy terms in the target function, and choose how to set the isotropic B factors. There are two choices of refinement target: Maximum likelihood and Least squares. If you choose Least squares, the ...",t:"General Setup for Reciprocal-Space Refinement"},"1384":{i:0.000131027872966082,u:"../primex_user_manual/reciprocal_space_maps.htm",a:"The Maps tab provides options for defining the parameters and extent of the maps that will be created with a refinement job. These maps are created automatically after the refinement is completed. The options are synchronized with those in the Create Map panel. Four choices are provided for the ...",t:"Setting Up Maps for Reciprocal-Space Refinement"},"1385":{i:0.000109183873844793,u:"../primex_user_manual/reciprocal_space_restraints.htm",a:"In the NCS tab, you can define restraints on the optimization to satisfy noncrystallographic symmetry.  Noncrystallographic symmetry (NCS) is imposed at the chain level. To impose this symmetry, you create groups of chains or chain segments that are related by NCS. The parts of the chains that are ...",t:"Using Noncrystallographic Symmetry Restraints in Reciprocal-Space Refinement"},"1386":{i:0.000117723101625595,u:"../primex_user_manual/reciprocal_space_minimize.htm",a:"PrimeX allows you to minimize the R factors for the coordinates, the B-factors, grouped B-factors, and grouped occupancies. Settings for a minimization are made in the Minimization tab. Three minimizers are offered in the Minimizer option menu, Quasi-Newton (LBFGS), Truncated Newton, and Conjugate ...",t:"Reciprocal-Space Minimization"},"1387":{i:0.000102602169932856,u:"../primex_user_manual/reciprocal_space_rigidbody.htm",a:"Often, the first step in a refinement is the overall positioning of the protein, which is done by treating the protein as a rigid body. PrimeX provides the means to define multiple rigid bodies for the refinement.  To set up a rigid-body refinement, choose Rigid body from the Refinement method ...",t:"Rigid-Body Refinement"},"1388":{i:0.000103559900247699,u:"../primex_user_manual/reciprocal_space_simanneal.htm",a:"Simulated annealing is generally used after the larger changes in positions have been completed with rigid-body refinement. Usually the initial temperature is decreased with successive runs. This method is useful for moving across low energy barriers. Simulated annealing can be performed on selected ...",t:"Simulated Annealing in Reciprocal-Space Refinement"},"1389":{i:7.8931340063019E-05,u:"../primex_user_manual/reciprocal_space_summary.htm",a:"During post-processing after every reciprocal space refinement job, four statistics are calculated that summarize the quality of the molecular geometry. These quantities are: Steric clashes: The number of interactions of non-hydrogen atoms where the distance between atom centers is less than 0.85 of ...",t:"Summary of Molecular Geometry from Reciprocal-Space Refinement"},"1390":{i:9.98068593186909E-05,u:"../primex_user_manual/real_space.htm",a:"Real-space refinement in PrimeX makes use of Prime technology with OPLS force fields for minimization, loop building and side-chain placement, but with the addition of X-ray terms to the force field terms in the function to be optimized. The balance between the two can be set in the Calculation ...",t:"Real-Space Refinement"},"1391":{i:0.000111575654203039,u:"../primex_user_manual/real_space_loop.htm",a:"The controls for loop refinement consist of a set of options for setting the grid spacing and B-factors, a table that lists the loops, text boxes for the loop limits, a button for loading the loops table from the Workspace structure and an option to bias the loop prediction towards the existing ...",t:"Loop Refinement in PrimeX"},"1392":{i:9.75354698243584E-05,u:"../primex_user_manual/real_space_sidechain.htm",a:"Side-chain prediction is a necessary follow-up to mutation of the model to match the input sequence (which you must do before you can run a side-chain prediction). The mutation is performed automatically by the Mutate Model to Sequence task. To perform this task, first click Mutate Model to Sequence ...",t:"Side-Chain Prediction in PrimeX"},"1393":{i:7.8800589353496E-05,u:"../primex_user_manual/real_space_minimize.htm",a:"Real-space minimization performs a standard geometric variation of the coordinates to minimize the energy, or in PrimeX, the functional that includes both the force-field energy and the density fit. If you have alternate conformations in your structure, you can only perform a minimization on one of ...",t:"Real-Space Minimization"},"1394":{i:7.3902587996154E-05,u:"../primex_user_manual/ligand_solvent_placement.htm",a:"Placing Ligand, Solvent, and Water Molecules in PrimeX Once the protein is reasonably well refined, you can proceed to place the ligand, solvent molecules, and water molecules, as described in the topics listed below.",t:"Placing Ligand, Solvent, and Water Molecules in PrimeX"},"1395":{i:0.00011836831909907,u:"../primex_user_manual/ligand_solvent_placement_ligand.htm",a:"Ligand and solvent molecule placement works by docking the molecule with Glide to a protein site that is defined by a region of electron density. This procedure is especially useful if the ligand or the solvent molecule can adopt more than one conformation. Glide performs a rapid conformational ...",t:"Placing the Ligand and Solvent Molecules in a Region of Electron Density"},"1396":{i:7.75596522769317E-05,u:"../primex_user_manual/ligand_solvent_placement_waters.htm",a:"In the Water Placement panel you can set options and run a job to place water molecules. The options in this panel ensure that water molecules are placed in reasonable locations. To open the Water Placement panel, click the Water Placement button in the PrimeX panel. Water molecules are placed in ...",t:"Placing Water Molecules in Regions of Electron Density"},"1397":{i:6.13254596701412E-05,u:"../primex_user_manual/partial_occupancy.htm",a:"PrimeX can perform refinements on systems with partial occupancies—for example, where only a fraction of the proteins in the crystal have bound ligands, or where ligands (and the nearby active site residues) occupy different conformations. The refinement of the occupancy values can be set up as a ...",t:"Partial Occupancy Refinement"},"1398":{i:0.000105199487229519,u:"../primex_user_manual/primex_create_map.htm",a:"Electron Density Maps An essential part of the refinement process is the creation and display of electron density maps. These tasks are described in the topics listed below.",t:"Electron Density Maps"},"1399":{i:0.000189247371612569,u:"../primex_user_manual/create_map_create.htm",a:"Maps can be created automatically at the end of reciprocal-space refinement jobs, but there may be other times when you want to create a map. In addition to regular maps, you may want to create omit maps. You can set up a job to create a map in the Create Maps panel.  To open the Create Map panel, ...",t:"Creating Electron Density Maps"},"1400":{i:0.000114119551274754,u:"../primex_user_manual/create_map_display.htm",a:"From the perspective of Maestro, maps are simply another kind of volume that can be used to generate a surface. Maps are associated with project entries and are stored with the project.  The controls for display of surfaces are available in the Manage Surfaces panel. To open this panel, choose ...",t:"Displaying Electron Density Maps"},"1401":{i:0.000103379772959502,u:"../primex_user_manual/other_primex_tools.htm",a:"Other PrimeX Tasks and Tools There are several other PrimeX tasks that have not been described or have only been mentioned in the preceding chapters. These tasks are described in the topics listed below. As for the other tasks, you can initiate a task in the PrimeX panel.",t:"Other PrimeX Tasks and Tools"},"1402":{i:0.000148420337265236,u:"../primex_user_manual/other_primex_tools_mutate.htm",a:"The mutate model to sequence task enables you to automatically mutate all residues in the model to match those in the sequence. This task is performed by the Maestro builder. The Alignment to reference sequence text area in the Mutate Model to Sequence dialog box shows the alignment of the reference ...",t:"Mutating a Model to Match the Sequence"},"1403":{i:0.00013936036526409,u:"../primex_user_manual/other_primex_tools_hbond.htm",a:"Most X-ray experiments cannot distinguish between the N and the O of terminal amides in Gln and Asn, and cannot determine the orientation or ionization state of the ring in His, or the orientation of water molecules. The Optimize H-Bond Networks task analyzes the network of hydrogen bonds in the ...",t:"Optimizing H-Bond Networks in PrimeX"},"1404":{i:0.000152103755056398,u:"../primex_user_manual/other_primex_tools_rfactor.htm",a:"Most PrimeX tasks include an R-factor calculation, so there is usually no need for a separate calculation of R factors. At the beginning of a refinement, however, the R factors are not likely to have been calculated. You can calculate them with the R-Factor Calculation task. The R-Factor Calculation ...",t:"R-Factor Calculations"},"1405":{i:0.000140261799376828,u:"../primex_user_manual/other_primex_tools_blobs.htm",a:"The Ligand/Solvent Placement panel makes use of density “blobs” to define the location of the ligand or solvent molecule. Density blobs are defined as collections of density peaks from a difference map that are part of a continuous region of density difference, with some minimum value set for ...",t:"Locating Density Blobs"},"1406":{i:0.000101068082497769,u:"../primex_user_manual/other_primex_tools_export.htm",a:"When you are ready to generate a file with the completed structure (or to transfer a structure to some other location), you can run the Export task. This task uses the calculation settings and runs a job to generate a Maestro file. The job includes an R-factor calculation. When you click Export in ...",t:"Exporting a Structure from PrimeX"},"1407":{i:0.000131224837562225,u:"../primex_user_manual/primex_validation_tools.htm",a:"PrimeX includes several validation tools that you can use to check the predicted structure. Two of these tools, Ramachandran plots and protein reports, are a part of the Maestro tool set and have wider applicability. The third, density fit, is specific to PrimeX. The tools are described in the ...",t:"Validation Tools for Predicted X-ray Structures"},"1408":{i:0.000246691224998514,u:"../primex_user_manual/primex_validation_tools_density.htm",a:"The Density Fit panel displays a measure for each residue of how well the electron density predicted by the current structure fits the current electron density map. This panel can be used to identify residues that should be targeted for further refinement, based on the fit to the electron density ...",t:"Checking the Density Fit"},"1409":{i:0.000203820085261473,u:"../primex_user_manual/primex_validation_tools_report.htm",a:"Reports on various properties of the protein that is in the Workspace are presented in the Protein Reports panel, which you open by choosing Tools → Protein Reports. The Display option menu lists the various protein properties for which reports can be generated. When you choose an item from the ...",t:"Protein Reports"},"1410":{i:0.000122734488240986,u:"../primex_user_manual/primex_validation_tools_rama.htm",a:"The Ramachandran Plot panel displays a plot of the dihedral angles f and y for each residue in the protein that is displayed in the Workspace. To open the panel, choose Tasks → Protein Analysis → Ramachandran Plot. This panel is useful for identifying residues that fall in disallowed regions of ...",t:"Ramachandran Plots"},"1411":{i:8.727336318956E-05,u:"../primex_user_manual/primex_reserved_residues.htm",a:"PrimeX uses residue names to classify residues as protein residues, and hence determine whether chains are protein chains or not. In addition to the standard names for the 20 standard amino acids, the following names are also recognized as protein residues: You should ensure that these residue names ...",t:"PrimeX Reserved Residue Names"},"1412":{i:7.43575653581844E-05,u:"../primex_user_manual/primex_theory.htm",a:"Map generation in PrimeX is based on the sigmaA weighting scheme of Reed [ [] ], a data treatment that is shown to decrease the bias in electron density maps. Optionally, an unweighted map may be calculated. Treatment of omit maps is by the kicked method [ [] ]. Reciprocal-space refinement with ...",t:"PrimeX Implementation Details"},"1413":{i:0.000177763693113814,u:"../primex_user_manual/primex_references.htm",a:"Read, R. J. Improved Fourier Coefficients for Maps Using Phases from Partial Structures with Errors. Acta Cryst. 1986, A42, 140–149.  Bhat, T.N.; Cohen, G.H. OMIT MAP: An Electron Density Map Suitable for the Examination of Errors in a Macromolecular Model. J. Appl. Crystallog. 1984, 17, 244–248. ...",t:"PrimeX References"},"1414":{i:0.000103939671044817,u:"../protein_prep_user_manual/pprep_intro.htm",a:"The typical structure file from the PDB is not suitable for immediate use in molecular modeling calculations. A typical PDB structure file consists only of heavy atoms and may include a cocrystallized ligand, water molecules, metal ions, and cofactors. Some structures are multimeric, and may need to ...",t:"Protein Preparation"},"1415":{i:0.000100264534149173,u:"../protein_prep_user_manual/pprep_intro_preparation_process.htm",a:"The preparation of a protein involves a number of steps, which are outlined below. The procedure assumes that the initial protein structure is in a PDB-format file, includes a cocrystallized ligand, and does not include explicit hydrogens. The result is refined, hydrogenated structures of the ligand ...",t:"The Protein Preparation Process"},"1416":{i:0.000223014229667629,u:"../protein_prep_user_manual/prepwizard.htm",a:"Many of the protein preparation tasks can be performed automatically with the Protein Preparation Wizard. The Protein Preparation Wizard panel allows you to take a protein from its raw state, (which may be missing hydrogen atoms and have incorrect bond order assignments, charge states, or ...",t:"The Protein Preparation Wizard"},"1417":{i:0.000152899692604286,u:"../protein_prep_user_manual/prepwizard_input.htm",a:"The Protein Preparation Wizard uses the structure in the Workspace as its input. The Workspace structure may be an included entry that exists in the Project Table, or one that is imported using the Protein Preparation Wizard. The import controls in the Import structure into Workspace section of the ...",t:"Protein Preparation Structure Input"},"1418":{i:0.000100173546148466,u:"../protein_prep_user_manual/prepwizard_displayH.htm",a:"The many hydrogen atoms in an all-atom PDB structure can clutter the display, making it difficult to see what is important. The Protein Preparation Wizard provides three options for display of hydrogen atoms: None—Do not display hydrogens\n     Polar only—Display only the polar hydrogens\n     All ...",t:"Displaying Hydrogens for Protein Preparation"},"1419":{i:0.000105851075816803,u:"../protein_prep_user_manual/prepwizard_preprocessing.htm",a:"After importing a structure, the first stage of the process is to address the main structural issues: assignment of bonds and bond orders, addition of hydrogens, filling in missing loops or side chains, capping uncapped termini, adjusting bonds and formal charges for metals, and correcting ...",t:"Preprocessing the Protein Structure"},"1420":{i:0.000113938078018934,u:"../protein_prep_user_manual/prepwizard_fixing_structure.htm",a:"The Protein Preparation - Problems dialog box lists the atoms and residues for which atoms are missing (often the side chains), atom types are misassigned, atoms are overlapping, or atoms have alternate positions. It opens automatically at the end of preprocessing if any of these problems are ...",t:"Fixing Protein Structural Problems"},"1421":{i:0.00010841820337663,u:"../protein_prep_user_manual/prepwizard_deleting_groups.htm",a:"After fixing any structural defects, the next stage is to inspect the structure and delete unwanted parts of the structure, which is done in the Review and Modify tab.  If you are starting the process at this stage, without preprocessing, display the structure in the Workspace, and click Analyze ...",t:"Inspecting the Protein Structure and Deleting Unwanted Groups"},"1422":{i:0.000109233740359099,u:"../protein_prep_user_manual/prepwizard_het_groups.htm",a:"For het groups, you can use Epik to predict ionization and tautomeric states of the het groups in the pH range specified in the pH text boxes. Additional states that are suitable for binding to metals in metalloproteins are also generated, if the het group is coordinated to a metal. To generate the ...",t:"Ionizing and Tautomerizing Protein Het Groups"},"1423":{i:0.000117023085185889,u:"../protein_prep_user_manual/prepwizard_optimizing_Hbond.htm",a:"The next stage of protein preparation is to optimize the hydrogen-bonding network by reorienting hydroxyl and thiol groups, water molecules, amide groups of asparagine (Asn) and glutamine (Gln), and the imidazole ring in histidine (His); and predicting protonation states of histidine, His, aspartic ...",t:"Optimizing the Protein Hydrogen Bonding Network"},"1424":{i:0.000116450387418214,u:"../protein_prep_user_manual/prepwizard_optimizing_Hbond_automated.htm",a:"The automated optimization provides a limited amount of control over the process. You can also select Minimize hydrogens of altered species to minimize the energies of these hydrogen atoms once the hydrogen bonding network is optimized. On some occasions, you may want to manually orient the water ...",t:"Automated Optimization of the Hydrogen Bonding Network"},"1425":{i:0.000142376939208133,u:"../protein_prep_user_manual/prepwizard_optimizing_Hbond_interactive.htm",a:"The interactive optimization is performed in the Interactive H-bond Network Optimizer panel, which you open by clicking Interactive Optimizer. This panel is used to interactively perform the assignments of the H-bond network.  The Interactive H-bond Optimizer panel. The first task is to analyze the ...",t:"Interactive Optimization of the Hydrogen Bonding Network"},"1426":{i:0.000115097467687158,u:"../protein_prep_user_manual/prepwizard_retaining_waters.htm",a:"Once the hydrogen-bonding network has been refined, you can choose to retain or remove waters based on the number of hydrogen bonds they form with parts of the structure that are not other waters. This allows you to keep waters that have significant binding to the receptor, for example, forming ...",t:"Retaining Structural Waters in the Protein"},"1427":{i:0.000110656735250764,u:"../protein_prep_user_manual/prepwizard_minimizing.htm",a:"The final step in the preparation process is to refine the structure. A minimization is run in which the heavy atoms can be restrained, so that strain can be relieved but the final result does not deviate too much from the input geometry. Hydrogen atoms are not restrained at all, which allows the ...",t:"Minimizing the Protein Structure"},"1428":{i:0.000118011105787126,u:"../protein_prep_user_manual/prepwizard_checking.htm",a:"You should always check your structures before using them for an application. While a great deal of effort has been invested to make the Protein Preparation Wizard robust and cover a wide variety of cases, it cannot catch every problem in every structure.  To assist you in checking your structures, ...",t:"Checking the Final Protein Structure"},"1429":{i:7.80443805069963E-05,u:"../protein_prep_user_manual/prepwizard_new_prep.htm",a:"If you want to start a new protein preparation after you have completed a preparation, you can click Reset to reset all the values and options in the panel to their defaults, and clear all data. If you want to keep settings that you have already made, simply import the new protein and start the ...",t:"Starting a New Protein Preparation"},"1430":{i:0.000257012314836076,u:"../protein_prep_user_manual/manual_pprep.htm",a:"For the most part, protein preparation can be performed with the Protein Preparation Wizard. However, there are occasions when you may need to make corrections manually. This chapter lays out some procedures that you can use for manual preparation and correction of a complex. This chapter contains ...",t:"Manual Protein Preparation"},"1431":{i:0.000217414965383894,u:"../protein_prep_user_manual/manual_pprep_deleting_waters.htm",a:"Water molecules in the crystallographic complex are generally not used unless they are judged critical to the functioning of the protein–ligand interaction. When waters are used, they are later included in the protein as “structural” waters. This section contains the following topics:",t:"Manually Deleting Unwanted Waters"},"1432":{i:0.000165355293915769,u:"../protein_prep_user_manual/manual_pprep_deleting_waters_locating.htm",a:"Structural waters can be located by displaying atoms within a certain distance of the ligand. The procedure below displays water molecules as balls, which makes them easier to identify in the Workspace.  Click the  Select ligands (only) button under Quick Select  on the  toolbar.  The ligand should ...",t:"Locating Structural Waters in a Protein"},"1433":{i:0.000149728670393216,u:"../protein_prep_user_manual/manual_pprep_deleting_waters_all.htm",a:"Deleting All Water Molecules If you decide to delete all waters, choose Waters from the Choose items to select button menu on the toolbar, and press the Del key. All water molecules are deleted. ",t:"Deleting All Water Molecules"},"1434":{i:0.000146639358482716,u:"../protein_prep_user_manual/manual_pprep_deleting_waters_distant.htm",a:"If you want to keep one or more waters, it is a good idea to begin by removing those that are farther than a specified distance from the ligand. Click the  Select ligands (only) button under Quick Select  on the  toolbar.  The ligand is selected.  Click the Define button on the Select toolbar.  The ...",t:"Deleting Water Molecules Distant from the Ligand"},"1435":{i:0.000151363635873516,u:"../protein_prep_user_manual/manual_pprep_deleting_waters_remaining.htm",a:"After deleting water molecules beyond a certain distance from the ligand, examine the Workspace and delete any remaining water molecules you do not want to keep: Choose Molecules from the Set selection scope button on the toolbar: Click on a water oxygen and press the Del key to delete that water ...",t:"Deleting Remaining Unwanted Waters"},"1436":{i:0.000171126246183401,u:"../protein_prep_user_manual/manual_prep_simplifying_complex.htm",a:"For computational efficiency it is often desirable to keep the number of atoms in the complex structure to a minimum. If the complex is a multimer and the important interactions do not span more than one unit of the multimer, you can delete the others. This section contains the following topics:",t:"Simplifying a Protein Complex"},"1437":{i:0.000154835784020614,u:"../protein_prep_user_manual/manual_pprep_simplifying_determining.htm",a:"To determine whether the ligand-receptor complex is a multimer, compare the chains that appear in the sequence viewer. If there are two or more chains with identical sequences, the complex may be a multimer. If this is the case, there may be duplicate copies of the binding site of interest.  To see ...",t:"Determining Whether the Complex Is a Multimer"},"1438":{i:0.00015889296938702,u:"../protein_prep_user_manual/manual_pprep_simplifying_subunits.htm",a:"If the protein complex structure is a multimer with duplicate binding sites, it can be truncated by deleting all but a single ligand binding site and the associated receptor subunits. Choose Molecules from the Set selection scope button on the toolbar: Pick an atom from the desired ligand in the ...",t:"Removing Unwanted Subunits from a Protein Complex"},"1439":{i:0.000189501594492461,u:"../protein_prep_user_manual/manual_prep_selecting_residues.htm",a:"When you are fixing problems detected in the conversion of the structure from a PDB file, it is useful to select residues based on the conversion status. This status is stored as a Maestro property, and has an associated color scheme that can be applied at any time, by choosing PDB Conversion Status ...",t:"Selecting Residues by PDB Conversion Status"},"1440":{i:0.000186532547046819,u:"../protein_prep_user_manual/manual_prep_correcting.htm",a:"In some situations, you may need to correct bond orders, atom types, or formal charges. These operations are described in this section. The situations in which you may have to perform these actions are described in later sections. If the complex structure contains any bonds from the ligand or a ...",t:"Correcting Bonds, Atom Types, and Charges"},"1441":{i:0.000207017148193121,u:"../protein_prep_user_manual/manual_pprep_bonds_atoms_charges_display.htm",a:"for Correcting Protein Structures When making corrections, it is useful to display information that is relevant to the task: for example, to display formal charges when correcting the charge or the atom type. This section presents some procedures for displaying information. Display tasks are ...",t:"Display Tasks"},"1442":{i:0.000166425701720362,u:"../protein_prep_user_manual/manual_pprep_bonds_atoms_charges_correction.htm",a:"Atom and Bond Correction Tasks The basic tasks for correcting bond orders, atom types, and formal charges are given below. It is a good idea to display formal charges for both atom type changes and formal charge changes, since atom typing can change the formal charge, and Maestro corrects the formal ...",t:"Correction Tasks"},"1443":{i:0.000142582171492691,u:"../protein_prep_user_manual/manual_prep_atom_names.htm",a:"Prime requires that the PDB atom names of all atoms are correct. In the imported structure, residues that have incorrect PDB atom names are colored red or blue. You can select these residues using the procedure given in  Selecting Residues by PDB Conversion Status .  To correct PDB atom names: ...",t:"Correcting PDB Atom Names"},"1444":{i:0.000160464259527657,u:"../protein_prep_user_manual/manual_prep_fixing_residues.htm",a:"Incomplete residues are common errors in PDB structures, but may be relatively harmless if they are distant from the active site. Structures that have incomplete residues near the active site should be repaired. Often, incomplete residues are missing side chains.  When PDB structures are imported ...",t:"Fixing Incomplete Residues"},"1445":{i:0.000164937099637791,u:"../protein_prep_user_manual/manual_prep_missing_residues.htm",a:"Some structures are missing entire residues, even though the sequence may be complete. The residues next to where the break occurs are  colored cyan on import into Maestro, but the residues are not displayed in the sequence viewer. If your structure has missing residues that could be important, you ...",t:"Adding Missing Residues to a Protein"},"1446":{i:0.000214960656599737,u:"../protein_prep_user_manual/manual_prep_protonation.htm",a:"You should look for inconsistencies in hydrogen bonding to see whether a misprotonation of the ligand or the protein might have left two acceptor atoms close to one another without an intervening hydrogen bond. One or more residues may need to be modified to resolve such an acceptor-acceptor or ...",t:"Manually Adjusting Protonation and Tautomerization"},"1447":{i:0.000176100317809082,u:"../protein_prep_user_manual/manual_prep_orientations.htm",a:"If the orientation of water molecules is incorrect, reorient the molecules with the following procedure: Pick the water molecule in the Workspace. In the 3D Builder panel, click Move . The Move Atoms banner is displayed, giving instructions. Middle-click the oxygen atom. This ensures that the ...",t:"Correcting Orientations"},"1448":{i:0.00013736642148219,u:"../protein_prep_user_manual/protassign_notes.htm",a:"While X-ray structures are invaluable starting points in modeling studies, they have a critical weakness: due to the relative lack of electron density around hydrogens, it is nearly impossible to accurately resolve their locations in the structure. While many hydrogen positions are easily estimated ...",t:"H-Bond Optimization Technical Notes"},"1449":{i:9.15819262889216E-05,u:"../protein_prep_user_manual/pprep_references.htm",a:"Li, H.; Robertson, A. D.; Jensen J. H. Very Fast Empirical Prediction and Interpretation of Protein pKa Values. Proteins, 2005, 61, 704–721. Bas, D. C.; Rogers, D. M.; Jensen J. H. Very Fast Prediction and Rationalization of pKa Values for Protein-Ligand Complexes. Proteins, 2008, 73, 765–783. ...",t:"Protein Preparation References"},"1450":{i:9.73288892575178E-05,u:"../qikprop_user_manual/qikprop_intro.htm",a:"QikProp is a quick, accurate, easy-to-use absorption, distribution, metabolism, and excretion (ADME) prediction program designed by Professor William L. Jorgensen. QikProp predicts physically significant descriptors and pharmaceutically relevant properties of organic molecules. In addition to ...",t:"Introduction to QikProp"},"1451":{i:0.000133997592300323,u:"../qikprop_user_manual/qikprop_intro_properties.htm",a:"For each successfully processed molecule, QikProp produces the following descriptors and properties. Those that are not predicted in fast mode are marked with a dagger (†) in the table below. Those whose values differ between fast and normal mode are marked with a double dagger (‡).   ◀  ...",t:"QikProp Properties"},"1452":{i:8.41060177309644E-05,u:"../qikprop_user_manual/qikprop_intro_methodology.htm",a:"QikProp was developed with the following process. The BOSS program and the OPLS-AA force field were used to perform Monte Carlo statistical mechanics simulations on organic solutes in a periodic boxes of explicit water molecules. This process resulted in configurational averages for a number of ...",t:"QikProp Methodology"},"1453":{i:0.000101285208530887,u:"../qikprop_user_manual/running_qikprop.htm",a:"QikProp can be run either from the Maestro GUI or from the command line. Using Maestro to run QikProp enables you to build structures or import structures into Maestro, use them as input for a QikProp job, and then use Maestro’s analysis tools and Schrödinger’s other computational programs to ...",t:"Running QikProp"},"1454":{i:0.000137718540280799,u:"../qikprop_user_manual/running_qikprop_input.htm",a:"QikProp input structures and files must meet the following criteria: \n            Structures must be three-dimensional, not two-dimensional. \n            Hydrogen atoms must be explicit. You cannot, therefore, use structures with united-atom types from molecular mechanics programs. If your input ...",t:"QikProp Input Structures"},"1455":{i:0.000117683548458169,u:"../qikprop_user_manual/running_qikprop_maestro.htm",a:"To run QikProp from Maestro, choose Tasks → ADME and Molecular Properties → Ligand-Based ADME/Tox Prediction or Applications → Other Applications → QikProp in the Task Tool. The QikProp panel opens. The QikProp panel. To launch a job from the QikProp panel, specify an input source and select ...",t:"Running QikProp From Maestro"},"1456":{i:0.000116271204260565,u:"../qikprop_user_manual/running_qikprop_maestro_options.htm",a:"Specifying an input source You can use Workspace structures, selected entries in the Project Table panel, or structures from a Maestro file as input for a QikProp job. The default setting is Selected Entries. To use structures from a file, choose File from the Use structures from option menu, and ...",t:"Setting QikProp Options"},"1457":{i:0.000127057071423477,u:"../qikprop_user_manual/running_qikprop_maestro_job.htm",a:"The Job Settings dialog box provides a standard set of controls for incorporation of job output, naming the job, and selection of the execution host. These controls are described in detail in the help topic.  If you selected project entries rather than a file for input, a file named jobname.maegz ...",t:"Setting QikProp Job Options"},"1458":{i:0.000115712792514917,u:"../qikprop_user_manual/running_qikprop_maestro_fixing.htm",a:"If your input structures are missing hydrogen atoms or other structural information needed by QikProp, the Hydrogen/Lone-Pair Fixup dialog box ( Figure 1 ) is displayed when you click Run in the Job Settings dialog box. This dialog box presents a list of valid treatments to fix up the structures, ...",t:"Fixing Noncompliant Structures for QikProp"},"1459":{i:9.86400343521489E-05,u:"../qikprop_user_manual/qikprop_quality.htm",a:"Experimental results for more than 710 compounds including about 500 drugs and related heterocycles were used in developing QikProp. The following table summarizes the fits (first released with QikProp 2.2). In the QikProp output file for the log P (octanol/water), log S and log BB output, if the ...",t:"Quality of QikProp Results"},"1460":{i:0.00013169259572392,u:"../qikprop_user_manual/qikprop_quality_test_set.htm",a:"The following tables illustrate results for test sets of molecules that were not in the QikProp 2.2 training set. The 3D structures were obtained directly from SciFinder, then to gauge the effect of optimization, they were optimized with the BOSS program using the OPLS-AA force field and CM1P ...",t:"QikProp Test Set Results"},"1461":{i:0.00016730586214754,u:"../qikprop_user_manual/qikprop_quality_accuracy.htm",a:"QikProp predictions are performed for specific conditions (pH 7, for one), and comparison of the results with experiment must take into account any differences in those conditions. The following issues can affect the accuracy of the comparison:  active vs passive transport mechanisms\n     efflux ...",t:"QikProp Accuracy Issues"},"1462":{i:0.000141955134782089,u:"../qikprop_user_manual/qikprop_quality_conformation.htm",a:"\n            The dependence of the QikProp results on molecular conformation has been studied by performing conformational searches on many molecules using BOSS with the OPLS-AA force field, followed by QikProp calculations for each conformer. The results demonstrate the following points: There is ...",t:"Conformation Dependence of Results"},"1463":{i:0.000101547923021632,u:"../qikprop_user_manual/qikprop_quality_data_sources.htm",a:"The sources of the experimental log P and log S values are described in Refs.  ,  , and  . Additional data has been kindly provided by Pharmacia Inc. The databases of free energies of hydration and free energies of solvation in hexadecane are from Abraham et al [ [] ]. The log BB values are mostly ...",t:"QikProp Data Sources"},"1464":{i:8.41060177309644E-05,u:"../qikprop_user_manual/qikprop_descriptors.htm",a:"The following metabolic reactions contribute to the #metab descriptor. aromatic OH oxidation\n     enol oxidation\n     benzylic-like H → alcohol\n     allylic H → alcohol\n     secondary alcohol → ketone\n     primary alcohol → acid\n     tertiary alcohol E1 or SN1\n     amine dealkylation\n     ether ...",t:"QikProp Descriptor Information"},"1465":{i:0.000141322523600025,u:"../qikprop_user_manual/qikprop_references.htm",a:"Duffy, E. M.; Jorgensen, W. L. Prediction of Properties from Simulations: Free Energies of Solvation in Hexadecane, Octanol, and Water. J. Am. Chem. Soc. 2000, 122, 2878-2888. Jorgensen, W. L.; Duffy, E. M. Prediction of Drug Solubility from Monte Carlo Simulations. Bioorg. Med. Chem. Lett. 2000, ...",t:"QikProp References"},"1466":{i:0.000138682697855721,u:"../qpld_user_manual/qpld_panel.htm",a:"In the QM-polarized ligand docking protocol, ligands are docked with Glide, then charges on the ligand induced by the protein are calculated with QSite, and a set of the best ligand poses are redocked. The QM-Polarized Ligand Docking panel collects the relevant controls for each part of the ...",t:"The QM-Polarized Ligand Docking Panel"},"1467":{i:0.000113171264047047,u:"../qpld_user_manual/qpld_ligands.htm",a:"The ligands to be docked must be properly prepared (for example, by using LigPrep) and must be in a Maestro, SD, or PDB file. Because the QSite step takes much more time than the Glide docking steps, you should restrict the number of ligands to a relatively small set, which might be the results of a ...",t:"Selecting Ligands To Be Docked with QPLD"},"1468":{i:0.000103353727950371,u:"../qpld_user_manual/qpld_grid.htm",a:"The grid for the QM-polarized ligand docking job can be set up as part of the job, or it can be read from a previous Glide grid generation job. The QM-Polarized Ligand Docking panel offers a limited range of options for setting up the grid, in the Receptor tab. If you want greater flexibility, you ...",t:"Setting Up the Grid for QPLD"},"1469":{i:0.000105750597931594,u:"../qpld_user_manual/qpld_initial_docking.htm",a:"In the Initial docking tab, you can select the docking mode, set parameters for the elimination of duplicate poses, scale the van der Waals radii if necessary, and decide how many poses to keep per ligand. The defaults represent reasonable choices. The remaining Glide docking options are set to ...",t:"Setting Initial Glide Docking Parameters for QPLD"},"1470":{i:0.000112493638908036,u:"../qpld_user_manual/qpld_qm_treatment.htm",a:"In the QM charges tab, you can select a level of quantum-mechanical treatment of the ligand, and choose whether to treat the ligand in the field of the receptor or as a free ligand, in the gas phase or in water.  To calculate charges for the free ligand, select Calculate charges for free ligand. You ...",t:"Selecting a QM Treatment for QPLD"},"1471":{i:0.000117135696926214,u:"../qpld_user_manual/qpld_redocking.htm",a:"In the Redocking tab, you can choose the docking precision (SP or XP) for redocking of the ligands with updated QM charges, set the van der Waals scaling, set criteria for the elimination of duplicate poses, and enter the maximum number of poses to keep for each redocked ligand. These controls are ...",t:"Setting Glide Redocking Parameters for QPLD"},"1472":{i:0.000112633998537152,u:"../qpld_user_manual/qpld_poses.htm",a:"In the Final selection tab, you can specify how the poses are ranked and calculate an RMSD from a reference ligand.  The Final selection by option menu offers the choice of three quantities by which the poses are ranked: the Coulomb-van der Waals energy, the GlideScore, and Emodel. For a description ...",t:"Specifying the Final Selection of Poses for QPLD"},"1473":{i:8.52611708276799E-05,u:"../qpld_user_manual/qpld_job.htm",a:"When you are ready to run the job, click the Settings button and choose Job Settings to open the Job Settings dialog box. In this dialog box, you can make job settings and start the job. The dialog box has an Output section in which you can choose an option from the menu for incorporating the ...",t:"Running the QPLD Job"},"1474":{i:6.13254596701412E-05,u:"../qsite_user_manual/qsite_intro.htm",a:"QSite is a mixed mode Quantum Mechanics/Molecular Mechanics (QM/MM) program used to study geometries and energies of structures not parameterized for use with molecular mechanics, such as those that contain metals or represent transition states. QSite is uniquely equipped to perform QM/MM ...",t:"Introduction to QSite"},"1475":{i:0.000118443326653984,u:"../qsite_user_manual/qsite_structureprep.htm",a:"The quality of QSite results depends on reasonable starting structures. Schrödinger offers a comprehensive protein preparation facility in the Protein Preparation Wizard, which is designed to ensure chemical correctness and to optimize protein and protein-ligand complex structures for use as input. ...",t:"Protein Preparation for QSite"},"1476":{i:0.000121623769545073,u:"../qsite_user_manual/qsite_structureprep_protein.htm",a:"A typical PDB structure file consists only of heavy atoms, can contain waters, cofactors, and metal ions, and can also be multimeric. The structure generally has no information on bonding or charges. Terminal amide groups can also be misaligned, because the X-ray structure analysis cannot usually ...",t:"QSite Protein Preparation Procedure"},"1477":{i:8.20023668440522E-05,u:"../qsite_user_manual/qsite_structureprep_checking.htm",a:"After you have completed the protein preparation, you should check the completed ligand and protein structures. The Protein Preparation Wizard provides reports which allow you to examine and often to correct problems in the structure. In particular, you can use the interactive H-bond optimization ...",t:"Checking the Protein Structures for QSite"},"1478":{i:9.31101799633707E-05,u:"../qsite_user_manual/running_qsite.htm",a:"QSite performs mixed quantum mechanical/molecular mechanical (QM/MM) calculations, using Jaguar for the QM calculations and Impact for the MM calculations. Ligands and other specified regions of a protein complex can be studied using QM, while MM is used for the rest of the molecule.  At each step ...",t:"Running QSite From Maestro"},"1479":{i:0.000128387981887805,u:"../qsite_user_manual/running_qsite_panel.htm",a:"QSite calculations can be set up and run using the QSite panel. To open the QSite panel, choose Tasks → General Modeling → Minimization → QM-MM or Tasks → Applications → Other Applications → QSite in the Task Tool. The panel has eight tabs: QM Settings—specify the QM region and other QM options.\n    ...",t:"The QSite Panel"},"1480":{i:0.000110339831207518,u:"../qsite_user_manual/running_qsite_qmsettings.htm",a:"The QM Settings tab is used to enter information for the QM job and to define the QM region.  QM job information includes the quantum-mechanical method to be used, the charge and spin multiplicity of the QM system, and other keywords and options that may be required by Jaguar.  The QM region can be ...",t:"The QM Settings Tab"},"1481":{i:0.00012793529935483,u:"../qsite_user_manual/running_qsite_potential.htm",a:"The Potential tab provides options for the definition of the potential energy functions used in the molecular mechanics part of the calculation. One of these (continuum solvation) affects the QM potential energy as well. For detailed information on the options in this tab, see  QSite Panel — ...",t:"The QSite Potential Tab"},"1482":{i:0.000135459026913945,u:"../qsite_user_manual/running_qsite_mm_constraints.htm",a:"The MM Constraints tab is used to apply constraints to the Cartesian coordinates of selected atoms in the MM region. Specified atoms can be frozen at their input coordinates (frozen-atom constraints), or they can be constrained to remain near their initial coordinates by applying a harmonic force. ...",t:"The MM Constraints Tab"},"1483":{i:0.000127803065891888,u:"../qsite_user_manual/running_qsite_qmsettings_qmconstraints.htm",a:"The QM constraints tab is used to set constraints on geometric parameters in the QM region. It provides the same capabilities as in the Optimization tab of the Jaguar panel. For full details, see  Constraining Coordinates in Jaguar Calculations . Briefly, you can set constraints on distances, ...",t:"The QM Constraints Tab"},"1484":{i:0.000153340493089785,u:"../qsite_user_manual/running_qsite_mmminimization.htm",a:"The MM Minimization tab specifies settings for energy minimization of the MM region of the molecule. If the QM method chosen in the Optimization tab is Single point, these settings are not used, and no MM minimization is performed. The options available in this tab are described below. The MM ...",t:"The MM Minimization Tab"},"1485":{i:0.000157838769589303,u:"../qsite_user_manual/running_qsite_qmoptimization.htm",a:"The QM Optimization tab specifies the type of QM (Jaguar) calculation to be performed and provides information needed to set up the calculation. For transition state optimizations additional structures (reactant, product, and transition state guess structures) can be given to guide the search. QSite ...",t:"The QM Optimization Tab"},"1486":{i:0.000117446755894474,u:"../qsite_user_manual/running_qsite_qmoptimization_calctype.htm",a:"The Method menu controls the QM calculation type. The options are: Single point—Calculate the QM energy for the structure as it stands. No QM geometry optimization or MM minimization is performed. When Single point is selected, other options in this tab are unavailable. Settings in the Minimization ...",t:"Calculation Type for QSite Jobs"},"1487":{i:0.000107272803568602,u:"../qsite_user_manual/running_qsite_qmoptimization_tssearch.htm",a:"If you select Transition state from the Method menu, the default option for TS method is Standard. The following three methods for transition-state optimization are supported in QSite, corresponding to well-known ab initio techniques. See  Transition-State Optimizations  and  Transition-State Search ...",t:"QSite Transition State Searches"},"1488":{i:0.000104330903091775,u:"../qsite_user_manual/running_qsite_properties.htm",a:"In this tab, you can set options for the calculation of QM properties at the end of the QSite job. The Properties tab is essentially the same as in the Jaguar panel. For detailed information on the properties and the use of the tab, see  Molecular Properties from Jaguar Calculations ,  Vibrational ...",t:"The QSite Properties Tab"},"1489":{i:0.000111550112969618,u:"../qsite_user_manual/running_qsite_scan.htm",a:"In this tab, you set up the coordinates for a relaxed or a rigid coordinate scan. Whether the scan is relaxed or rigid depends on the method selection in the QM Optimization tab: selecting Single point from the Method option menu performs a rigid scan, selecting Minimization performs a relaxed scan. ...",t:"The QSite Scan Tab"},"1490":{i:0.000132542405988151,u:"../qsite_user_manual/running_qsite_jobs.htm",a:"When you have set the options in the QSite panel tabs to the desired value, enter a name in the Job name text box and click Run to run the job with the current job settings. If you want to make job settings, click the Settings button to open the Job Settings dialog box. In this dialog box you can ...",t:"Running QSite Jobs"},"1491":{i:0.000139792588211427,u:"../qsite_user_manual/running_qsite_troubleshooting.htm",a:"Troubleshooting QSite Jobs The topics listed below contain information on various conditions that you might encounter, and offer some suggestions to work around or fix the problems. ",t:"Troubleshooting QSite Jobs"},"1492":{i:0.00014438785279066,u:"../qsite_user_manual/running_qsite_troubleshooting_atomtype.htm",a:"By default, Impact performs a number of tests during atom typing to guide selection of optimal force-field parameters for the atoms in the input structure. The tests are applied to both the MM and the QM regions. For transition metals, force-field parameters are more limited than for the common ...",t:"mmlewis Warnings or Atom Typing Failures in QSite Jobs"},"1493":{i:0.000102237214460227,u:"../qsite_user_manual/running_qsite_troubleshooting_convergence.htm",a:"For difficult to converge QSite cases, the following recommendations may help: Do not use non-bonded cutoffs (clear the Use non-bonded cutoffs box in the Potential tab). Using non-bonded cutoffs leads to energy jumps when the number of non-bonded interactions changes, as residues or atoms cross the ...",t:"Convergence Problems in QSite Jobs"},"1494":{i:8.73407349296382E-05,u:"../qsite_user_manual/qsite_technotes.htm",a:"The study of reactive chemistry in a protein environment is an extremely challenging problem for computational chemistry. The only methods that can produce reliable results (particularly for structures containing transition metals) are those of ab initio quantum chemistry. However, such methods are ...",t:"QSite Technical Notes"},"1495":{i:0.000122419726435886,u:"../qsite_user_manual/qsite_technotes_active_sites.htm",a:"QSite is specifically designed to treat protein active sites. It combines Schrödinger’s powerful Jaguar program for ab initio electronic structure calculation with molecular-mechanics calculations that use the OPLS-AA force fields of Jorgensen and coworkers. The speed of Jaguar—augmented by an ...",t:"QM/MM for Protein Active Sites"},"1496":{i:0.000112809349884736,u:"../qsite_user_manual/qsite_technotes_transition_state.htm",a:"Jaguar and QSite can perform transition state (TS) searches by using a quasi-Newton method to find the TS nearest the initial geometry  [] . Alternatively, Jaguar can employ a Linear/Quadratic Synchronous Transit (LST/QST) approach, which is also known as Synchronous Transit Quasi-Newton (STQN), to ...",t:"QM/MM Transition State Modeling"},"1497":{i:0.000119847022600118,u:"../qsite_user_manual/qsite_technotes_methodology.htm",a:"Most approaches for developing robust and accurate QM/MM methods have been based on “link atom” approaches, in which QM and MM fragments are capped by hydrogens. These methods face nontrivial problems in constructing an accurate description of the QM/MM interface, particularly for polar systems, ...",t:"How QSite Works"},"1498":{i:0.000121930450521871,u:"../qsite_user_manual/qsite_technotes_validation.htm",a:"QSite Parametrization Validation The topics listed below provide information on validation of QSite.",t:"QSite Parametrization Validation"},"1499":{i:0.000165339253377579,u:"../qsite_user_manual/qsite_technotes_validation_deprotonation.htm",a:"Chemical reaction energetics provide one important measure of how well a QM/MM model reproduces accurate quantum mechanics. Below we examine a simple reaction—removal of a proton from the QM region.  Table 1  compares differences between the QM and QM/MM deprotonation energies for the capped ...",t:"Deprotonation Energies from QSite"},"1500":{i:0.000161539194053813,u:"../qsite_user_manual/qsite_technotes_validation_conformers.htm",a:"Conformational energies afford a second critical test of the quality of the QM/MM methodology.  Table 1  summarizes the RMS deviations between the QM/MM or MM (OPLS-AA) and the purely QM results for the relative energetics of the side-chain rotamer data set used to parameterize QSite. The QM/MM ...",t:"Conformational Energies from QSite"},"1501":{i:0.000144691409791087,u:"../qsite_user_manual/qsite_technotes_validation_other.htm",a:"QSite has also been shown to accurately reproduce patterns in quantum-mechanical hydrogen-bond dimerization energies and to reproduce LMP2/cc-pVTZ(-f) relative conformational energies for the alanine dipeptide more accurately than does a molecular force field such as OPLS-AA (see Ref.  ).",t:"Other Comparisons for QSite"},"1502":{i:0.000173062998220645,u:"../qsite_user_manual/qsite_technotes_illustration.htm",a:"Cytochrome P-450 is an enzyme whose variants are ubiquitously distributed across a wide variety of organisms. A human version of the enzyme in the liver is of great importance pharmaceutically because it is involved in a significant fraction of toxicity and drug metabolic pathways. While a ...",t:"An Illustrative Application of QSite"},"1503":{i:0.000190552670350753,u:"../qsite_user_manual/qsite_references.htm",a:"Bofill, J. M. J. Comp. Chem. 1994, 15, 1; Murtagh, B. A.; Sargent, R. W. H. Comput. J. 1970, 13, 185; Powell, M. J. D. Math. Prog. 1971, 1, 26. Peng, C.; Schlegel, H. B. Isr. J. Chem. 1993, 33, 449. Gogonea, V.; Suárez, D.; van der Vaart, A.; Merz, K. M. Jr. Curr. Opin. Struct. Biol. 2001, 11, ...",t:"QSite References"},"1504":{i:0.000155253641625655,u:"../rgroup_user_manual/rgroup.htm",a:"If you have a set of structures that are built on a common or similar scaffold, by attaching different groups at one or more points on the scaffold, you might want to see the properties of the structures as a function of the groups at the various attachment points. The R-Group Analysis facility ...",t:"R-Group Analysis"},"1505":{i:0.00013027162015579,u:"../rgroup_user_manual/rgroup_analyze.htm",a:"The analysis and display of R groups is done in the R-Group Analysis panel. The panel is available from Maestro, from Canvas, and from the command line. To open the R-Group Analysis panel: from Maestro, choose Tasks → Structure Analysis → Small Molecule Analysis → R-group Analysis in the Task Tool.  ...",t:"Analyzing a Set of Structures for R-Groups"},"1506":{i:0.000126888198027344,u:"../rgroup_user_manual/rgroup_analyze_input.htm",a:"First, you must specify the source of the structures to analyze, which you can do by selecting Analyze structures from and choosing a source from the option menu. The sources are: Project Table (selected entries)—Use the structures that are selected in the Project Table. Only available from Maestro. ...",t:"Specifying the Structures to Analyze for R-Groups"},"1507":{i:0.000134820943226152,u:"../rgroup_user_manual/rgroup_analyze_core.htm",a:"The second step in the analysis is to choose a method for the definition of the core of the structures. The analysis determines the R groups that are attached to this core. The core can be taken from CombiGlide attachment-bond labeling, specified by a SMARTS pattern, or determined by the maximum ...",t:"Defining the Core for R-Group Analysis"},"1508":{i:0.000132495583151014,u:"../rgroup_user_manual/rgroup_analyze_running.htm",a:"When you have selected the input structures and chosen a method for defining the core, click Start to run the analysis. Once the analysis finishes, the R-Group Viewer panel opens. In this panel you can view the groups, export them, and open other panels that display various analyses of the ...",t:"Running the R-Group Analysis"},"1509":{i:0.000150060813088564,u:"../rgroup_user_manual/rgroup_analyze_saveload.htm",a:"If you want to save the results of the analysis, click Save Results, and navigate to a location and save the results (as a zip file) in the file selector that opens. You can then read the results of this analysis back in at a later time, by selecting Read results from previous run, then clicking ...",t:"Saving and Loading Results of an R-Group Analysis"},"1510":{i:0.000180672509580321,u:"../rgroup_user_manual/rgroup_viewing.htm",a:"If you want to view the results of the analysis by displaying the structures with the R groups marked, you can do so in the R-Group Viewer panel. This panel displays the structures and the R groups for each structure. It also allows you to choose R groups at each position and display only the input ...",t:"Viewing and Exporting R-Groups"},"1511":{i:0.000154703255018494,u:"../rgroup_user_manual/rgroup_viewing_restrict.htm",a:"To restrict the structures that are shown in the Input Structures section, select Restrict visible input structures by selecting R-groups. By default, all R groups are selected for display at each position. For each position you can choose the R groups that you want to display by selecting them in ...",t:"Restricting the Structures Displayed for R-Group Analysis"},"1512":{i:0.000148879371779762,u:"../rgroup_user_manual/rgroup_viewing_export_structures.htm",a:"If you want to export the input structures, click Export Input Structures. A dialog box opens, in which you can make the following choices: Export the structures to a file or to the Project Table (only available from Maestro).  Export all structures or only the visible structures. Superimpose the ...",t:"Exporting Structures from R-Group Analysis"},"1513":{i:0.000154294656406387,u:"../rgroup_user_manual/rgroup_viewing_export_groups.htm",a:"To export the R groups as separate molecules, capped with methyl groups, to a set of Maestro files or to the Project Table, click Export R-Group Structures. The set of R groups at each position is exported to a separate file or entry group. The base name of the file or the entry group is given in ...",t:"Exporting R-Groups"},"1514":{i:0.000179230187624991,u:"../rgroup_user_manual/rgroup_properties.htm",a:"The R-group analysis facility offers several ways of displaying information on molecular properties and their relationship to the R-groups and the cores: an activity cliff plot, importance analysis, a heat map, R-group QSAR based on either pharmacophore features or electrotopological states ...",t:"Analyzing Molecular Properties from R-Group Analysis"},"1515":{i:0.000193837698326879,u:"../rgroup_user_manual/rgroup_properties_cliffs.htm",a:"When examining structure-activity relations, you may want to identify “activity cliffs”, where a small change in structure produces a large change in a property. This can be done in the Activity Cliffs panel, which you open by clicking Activity Cliffs in the R-Group Viewer panel. The first task is ...",t:"Activity Cliffs for R-Group Analysis"},"1516":{i:0.000143057133585819,u:"../rgroup_user_manual/rgroup_properties_importance.htm",a:"Importance analysis addresses the question of how sensitive the property of interest is to R-group variation at that position. A position is more important than another if varying the R group at that position leads to greater property differences than are observed when the R group at the other ...",t:"Importance Analysis of R-Groups"},"1517":{i:0.000143071522842256,u:"../rgroup_user_manual/rgroup_properties_heatmaps.htm",a:"You can display a heat map of a selected property for R groups at two positions, as a function of the values at all other positions, in the Heat Map panel. To open the Heat Map panel, click Heat Map in the R-Group Viewer panel. You can choose the property used in the heat map in the Displayed ...",t:"Heat Maps of R-Group Properties"},"1518":{i:0.000145444312990689,u:"../rgroup_user_manual/rgroup_properties_qsar.htm",a:"If you want some idea of what kinds of groups are most important for the properties of your compounds at each attachment position, you can generate a QSAR model, based either on pharmacophore features or E-state (electrotopological state) atom types. These two model types are supported in panels ...",t:"R-Group QSAR Models"},"1519":{i:0.000102536610000224,u:"../rgroup_user_manual/rgroup_properties_sar_table.htm",a:"If you want to view the property values in tabular form along with the R groups, click SAR Table in the R-Group Viewer panel. The SAR Table panel shows the 2D structures of the cores, at the top, with the R groups and properties below. The attachment positions on the core structures have labels Rn, ...",t:"R-Group SAR Table"},"1520":{i:0.000103811515648202,u:"../semiempirical_nddo_guide/nddo.htm",a:"Schrödinger provides access to programs for basic semiempirical NDDO capabilities. Two versions are available: the Semiempirical NDDO module is based on the public domain version of MOPAC 7.1 available at  www.openmopac.net ; and MOPAC2012 is the commercially available version of MOPAC, which ...",t:"Semiempirical NDDO Guide"},"1521":{i:0.000141817554327978,u:"../semiempirical_nddo_guide/nddo_panel.htm",a:"Semiempirical NDDO jobs can be set up and run from Maestro by using the Semiempirical NDDO Module panel or the MOPAC2012 panel. To open these panels, choose Tasks → Quantum Mechanics → Semiempirical NDDO or Tasks → Quantum Mechanics → MOPAC2012 in the Task Tool. The Semiempirical NDDO Module panel.  ...",t:"The MOPAC 2012 and Semiempirical NDDO Module Panels"},"1522":{i:0.000128360038867331,u:"../semiempirical_nddo_guide/nddo_command.htm",a:"To run a semiempirical NDDO job from the command line, use the following syntax: \n$SCHRODINGER/run semi_emp.py [-MOPAC2012] [options] [job-options] structure-file The input structure file can contain multiple structures, which will all be processed in the job. The file can be in Maestro, SD, or MOL2 ...",t:"Running a Semiempirical NDDO Job from the Command Line"},"1523":{i:0.000175802220988205,u:"../semiempirical_nddo_guide/nddo_keywords.htm",a:"The keywords that are supported in the Semiempirical NDDO module for use with the -keywords command option are given in this section. Keywords are not case-sensitive. The keywords listed below are the only keywords that are supported with this module. Any other keywords that exist in the original ...",t:"Keywords Supported in the Semiempirical NDDO Module"},"1524":{i:0.000181336638781338,u:"../semiempirical_nddo_guide/nddo_keywords_general.htm",a:"RM1, AM1, PM3, MNDO, MNDOD  These keywords select the method used to calculate the semi-empirical heat of formation. RM1 is the default, and is recommended. It is a reparameterization for some of the elements covered by AM1. The Semiempirical NDDO module automatically uses AM1 parameters for ...",t:"General NDDO Keywords"},"1525":{i:0.000148990333415419,u:"../semiempirical_nddo_guide/nddo_keywords_properties.htm",a:"MULLIK By default, atomic charges are calculated by summing up the density matrix elements for each atom’s basis functions. To take into account the overlap of basis functions between atoms, use this keyword to perform a Mulliken analysis. ESP Print the atomic charges that best reproduce the ...",t:"Molecular Properties Keywords for NDDO Calculations"},"1526":{i:0.000150144798763956,u:"../semiempirical_nddo_guide/nddo_keywords_plot.htm",a:"These keywords are used by the Semiempirical NDDO module to generate a 3-dimensional grid of electronic properties that can be used in Maestro to display surfaces. For ESP, ALIE, and ALEA surfaces, an analysis of the values of the property on the isodensity surface is included in the output and ...",t:"Plot Keywords for NDDO Calculations"},"1527":{i:0.000140943008966423,u:"../semiempirical_nddo_guide/nddo_limitations.htm",a:"Limitations on Semiempirical NDDO Calculations The following limitations should be noted: Spin states higher than sextet are not available. ESP and SUPER calculations cannot be run with the MNDO/d method.  UHF (unsupported keyword) cannot be used for ESP, MULLIKEN, or SUPER calculations.",t:"Limitations on Semiempirical NDDO Calculations"},"1528":{i:0.000168380326797681,u:"../semiempirical_nddo_guide/nddo_troubleshooting.htm",a:"If you use a Maestro structure file as input, and you experience a problem running the module and want to see the input file that is created from the .mae file, include the -SAVE option on the command line. This option creates a .zip file of the job scratch directory, including the input file, which ...",t:"Troubleshooting Semiempirical NDDO Calculations"},"1529":{i:0.00015791917108166,u:"../semiempirical_nddo_guide/nddo_references.htm",a:"Gancia, E.; Montana, J. G.; Manallack, D. T. J. Mol. Graph. Model. 2001, 19, 349. Karelson, M.; Lobanov, V. S.; Katritzky, A. R. Chem. Rev. 1996, 96, 1027. Stewart, J. J. P. Optimization of Parameters for Semiempirical Methods V: Modification of NDDO Approximations and Application to 70 Elements. J. ...",t:"References for Semiempirical NDDO"},"1530":{i:0.000169010201647821,u:"../shape_screening_user_manual/shape_screening.htm",a:"There are occasions on which the shape of the molecule is its most important feature, and a search for molecules that are most similar in shape is needed. This capability is provided with the Shape Screening program.  You can run shape screening from Maestro (see  Running Shape Searches from Maestro ...",t:"Searching for Molecules by Shape"},"1531":{i:0.000157544485685,u:"../shape_screening_user_manual/shape_screening_overview.htm",a:"Overview of Shape Screening The Shape Screening program can be used to screen one or more files or a Phase database against a shape query. Each conformer from a given molecule is aligned to the query in various ways, and a similarity is computed based on overlapping hard-sphere volumes. The ...",t:"Oveview of Shape Screening"},"1532":{i:0.000141881659475409,u:"../shape_screening_user_manual/shape_screening_maestro_options.htm",a:"To run a shape search from Maestro, you use the Shape Screening panel, which you open by choosing Tasks → Ligand-Based Virtual Screening → Shape Screening in the Task Tool.  The main panel contains the most commonly used options, but further options are available in a dialog box.  Setting Up a Basic ...",t:"Running Shape Searches from Maestro"},"1533":{i:8.11445873578983E-05,u:"../sitemap_user_manual/sitemap_intro.htm",a:"SiteMap is a tool for finding potential binding sites for small-molecule ligands on a protein. It maps and scores regions on the protein surface that are likely to accommodate a ligand. SiteMap can be run either from Maestro or from the command line. A SiteMap calculation typically takes a few ...",t:"Finding Binding Sites with SiteMap"},"1534":{i:6.99473853792205E-05,u:"../sitemap_user_manual/sitemap_intro_overview.htm",a:"The location of the primary binding site on a receptor such as a protein is often known from the structure of a co-crystallized complex. Efforts to design better ligands for these receptors can profit from an understanding of how well the known ligands complement the receptor, and how extension of ...",t:"Overview of SiteMap"},"1535":{i:0.000103392051880422,u:"../sitemap_user_manual/sitemap_methodology.htm",a:"The current SiteMap represents an expansion of the original SiteMap facility in Maestro (now referred to as hppmap). As in the original procedure  [4] , site mapping operates in a manner analogous to Goodford’s GRID algorithm  [5] . A SiteMap calculation has three stages. First, a grid is set up, ...",t:"How SiteMap Works"},"1536":{i:0.000118034984700979,u:"../sitemap_user_manual/sitemap_methodology_find_sites.htm",a:"The first stage of a SiteMap calculation is to locate the sites. A site is defined by a set of site points on a grid that are either contiguous or bridged by short gaps in solvent-exposed regions. The site-finding algorithm begins by placing a 1-Å grid of possible site points around the entire ...",t:"How SiteMap Finds Binding Sites"},"1537":{i:0.000129001001351457,u:"../sitemap_user_manual/sitemap_methodology_map_sites.htm",a:"The second stage of a SiteMap calculation generates the various “maps” that define the sites. A map is defined by a set of values of a property on a given 3D grid. First, SiteMap uses the site points from the preceding stage to position a mapping box for each site. This box defines a grid with a ...",t:"How SiteMap Maps Binding Sites"},"1538":{i:0.000120809499116869,u:"../sitemap_user_manual/sitemap_methodology_map_sites_hydrophilic.htm",a:"SiteMap constructs a measure of hydrophilicity by adding an “electric-field reward” term to the van der Waals energy:  where the oriented-dipole energy is necessarily negative. Hydrophilic regions are those within which the sum of the two terms is more negative than a given threshold, which by ...",t:"SiteMap Hydrophilic Map"},"1539":{i:0.000150911304376519,u:"../sitemap_user_manual/sitemap_methodology_map_sites_hydrophobic.htm",a:"The quantity representing hydrophobicity is constructed by adding an “electric-field penalty” (positive) term to the van der Waals term:  Grid_phobic =  vdW_energy − 0.30 * oriented-dipole_energy Hydrophobic regions thus are regions where something would like to be, but water would not. The starting ...",t:"SiteMap Hydrophobic Map"},"1540":{i:0.000128825840355446,u:"../sitemap_user_manual/sitemap_methodology_map_sites_hbondmetal.htm",a:"The hydrophilic map is further partitioned into separate hydrogen-bond donor and acceptor maps. When there is interaction with a metal center other than Ca2+, which (as in Glide XP) is not considered to be a metal-binding center, a separate metal-binding map is also formed. Metal-binding grid points ...",t:"SiteMap Donor, Acceptor, and Metal-Binding Regions"},"1541":{i:0.000125042383576137,u:"../sitemap_user_manual/sitemap_methodology_map_sites_surface.htm",a:"SiteMap Surface Map The surface map is obtained by removing attractive regions of the van der Waals grid and then contouring the repulsive part of this grid at a positive threshold value, which is set to +1 kcal/mol by default. ",t:"SiteMap Surface Map"},"1542":{i:0.000128063059104985,u:"../sitemap_user_manual/sitemap_methodology_evaluate_sites.htm",a:"This stage uses the site-point groups produced in the site-finding stage and the grids produced in the mapping stage to evaluate the sites in terms of a number of properties. The same modifications to van der Waals radii and formal-charge contributions and the same definition of hydrophobicity are ...",t:"How SiteMap Evaluates Sites"},"1543":{i:9.71554732182425E-05,u:"../sitemap_user_manual/running_sitemap.htm",a:"SiteMap calculations can be set up and run from the SiteMap panel 1 A more restricted facility for mapping just the hydrophobic and hydrophilic surfaces of sites is available at Workspace → Surface → Hydrophobic/philic. . To open this panel, choose Tasks → Structure Analysis → Binding Site ...",t:"Running SiteMap from Maestro"},"1544":{i:0.000128032579992944,u:"../sitemap_user_manual/running_sitemap_task.htm",a:"In the Specify task section, you can choose between two tasks: Identify top-ranked potential receptor binding sites Evaluate a single binding site region For each task, the structure to be used must be displayed in the Workspace. SiteMap panel showing options for mapping an entire protein. If you ...",t:"Specifying the SiteMap Task"},"1545":{i:0.000110893400869623,u:"../sitemap_user_manual/running_sitemap_options.htm",a:"In the Settings section, you can make the following choices to control the calculations: Require at least n site points per reported site Enter a value to set the minimum number of site points required in the initial site-finding stage to define a site. The default value of 15 site points should ...",t:"Setting Other SiteMap Options"},"1546":{i:0.000105218487822819,u:"../sitemap_user_manual/running_sitemap_job.htm",a:"To start the SiteMap job, click Run, or click the Settings button, make settings in the Job Settings dialog box, and click Run in the dialog box. You can set the job name, the host, the user name, and the number of processors to use, and you can specify how to incorporate the results.  When a ...",t:"Running the SiteMap Job"},"1547":{i:0.000117831986920262,u:"../sitemap_user_manual/running_sitemap_results.htm",a:"When the results are incorporated, the Manage Surfaces panel opens. You can use the controls in this panel to change the display attributes of the various maps. The accptr, donor, and phob maps are displayed by default; the phil and surf maps are not displayed by default.  The default appearance of ...",t:"Viewing the SiteMap Results"},"1548":{i:0.00012051249489271,u:"../sitemap_user_manual/running_sitemap_samples.htm",a:"To illustrate a typical application,  Figure 1  and  Figure 2  show the co-crystallized ligand for the thrombin 1ett receptor and the generated site points (white) in the context of the receptor structure and of the gray, translucent SiteMap surface.  Figure 1  focuses on relatively exposed regions ...",t:"Sample Site Maps"},"1549":{i:0.000147322013112678,u:"../sitemap_user_manual/sitemap_results.htm",a:"To calibrate and characterize the SiteMap properties, SiteMap has been applied to an extensive set of 230 proteins, which were taken either from the Glide database-enrichment suite or from the PDBbind database  [6] . These proteins bind ligands of molecular weight at least 150 with affinities of at ...",t:"SiteMap Calibration"},"1550":{i:0.000228446194267068,u:"../sitemap_user_manual/sitemap_references.htm",a:"Nayal, M.; Honig, B. On the nature of cavities on protein surfaces: Application to the identification of drug-binding sites. Proteins 2006, 63, 892. Halgren, T. A. New Method for Fast and Accurate Binding-site Identification and Analysis, Chem. Biol. Drug Des., 2007, 69, 146. Friesner, R. A., ...",t:"SiteMap References"},"1551":{i:0.000108044498484605,u:"../vsw_user_manual/vsw.htm",a:"The Virtual Screening Workflow is designed to run an entire sequence of jobs for screening large collections of compounds against one or more targets. The workflow includes ligand preparation using LigPrep, filtering using ligfilter on QikProp properties or other structural properties, and Glide ...",t:"Virtual Screening Workflow"},"1552":{i:0.000111942558614671,u:"../vsw_user_manual/vsw_panel.htm",a:"The Virtual Screening Workflow panel sets up the input files for LigPrep, QikProp, and Glide ligand docking and submits them to the selected host in order.  To open the Virtual Screening Workflow panel, choose Tasks → Receptor-Based Virtual Screening → Virtual Screening Workflow in the Task Tool. ...",t:"The Virtual Screening Workflow Panel"},"1553":{i:0.000132335849571437,u:"../vsw_user_manual/vsw_ligands.htm",a:"In the Input tab you specify the source of the ligands, and ensure that each ligand is uniquely identified. Ligands can be taken from two sources: files (in Maestro, SD, or SMILES format), or a Phase database. The Input tab of the Virtual Screening Workflow panel. The ligand sources and related ...",t:"Specifying the Ligands for VSW"},"1554":{i:0.00013867103882351,u:"../vsw_user_manual/vsw_ligands_files.htm",a:"To use ligands from one or more files, select Files for the Source of ligands option. To specify ligand files, you can enter a comma-separated list of file names in the Files text box, or click Browse to navigate to and select the files. To specify multiple ligand files with related names, you can ...",t:"Using Ligands from Files for VSW"},"1555":{i:0.000140635890936898,u:"../vsw_user_manual/vsw_ligands_phase.htm",a:"To use ligands from a Phase database, select Phase database for the Source of ligands option. You can type the name of the database into the text box, without the .phdb extension, or click Browse and navigate to the database. If you have a Phase database in an earlier format (with a _phasedb ...",t:"Using Ligands from a Phase Database for VSW"},"1556":{i:0.000141234934010911,u:"../vsw_user_manual/vsw_ligands_identifying.htm",a:"In the docking sequence, you can choose to keep all the ionization and tautomeric states of a given compound for which one of these states docks well. The ionization and tautomeric states that originate from the same compound are identified by a property that is unique for the compound. The controls ...",t:"Identifying Ligands and their States in VSW"},"1557":{i:0.000141384140901452,u:"../vsw_user_manual/vsw_filtering.htm",a:"In the Filtering tab you can set options for ligand filtering. This tab allows you to run QikProp and filter the ligands by property.  The Filtering tab of the Virtual Screening Workflow panel. The Virtual Screening Workflow offers three choices for prefiltering ligands: prefiltering using ...",t:"Filtering the Ligands for VSW"},"1558":{i:0.00014131164750839,u:"../vsw_user_manual/vsw_filtering_lipinski.htm",a:"To prefilter the ligands using Lipinski’s Rule of 5, select Prefilter by Lipinski’s Rule. Ligands that do not satisfy this rule are not docked. This option requires QikProp properties. If the input structure files do not have QikProp properties, select Run QikProp.",t:"Filtering by Lipinski’s Rule for VSW"},"1559":{i:0.000140906608704517,u:"../vsw_user_manual/vsw_filtering_reactive.htm",a:"To filter out ligands that have reactive functional groups, select Remove ligands with reactive functional groups. The functional groups that are considered reactive are: ◀  Filtering by Lipinski’s Rule for VSW Virtual Screening Workflow User Manual Filtering with Custom Filters for VSW  ▶",t:"Filtering Out Reactive Functional Groups for VSW"},"1560":{i:0.000139549697492167,u:"../vsw_user_manual/vsw_filtering_custom.htm",a:"To set up your own filter, select Retain only ligands matching criteria, then click Criteria to set up or read in the filter. The Ligand Filtering dialog box opens, which allows you to define the filtering criteria or read a file containing filtering criteria. The Ligand Filtering dialog box, ...",t:"Filtering with Custom Filters for VSW"},"1561":{i:0.000135166021763489,u:"../vsw_user_manual/vsw_ligprep.htm",a:"In the Preparation tab you set options for ligand preparation with LigPrep. If you only want to prepare and filter a set of ligands, you can run the job when you have made settings in this tab, and ignore the remaining two tabs, which specify the receptors and the docking options. The Preparation ...",t:"Preparing the Ligands for VSW"},"1562":{i:0.000121052101404384,u:"../vsw_user_manual/vsw_receptor.htm",a:"If you want to dock the ligands that you have prepared and filtered, you must specify one or more grids for the receptor in the Receptors tab. The ligands are docked to each receptor that you specify. Grids can be added to the Receptors for docking table by clicking Add or Add Multiple. The maximum ...",t:"Selecting or Setting Up the Receptor Grids for VSW"},"1563":{i:0.000126040573482326,u:"../vsw_user_manual/vsw_receptor_single.htm",a:"When you click Add, the VSW - Receptor dialog box opens. In this dialog box, you can select a pregenerated grid or generate a grid from the Workspace structure. You can also add constraints (described later), identify the receptor, and add an offset to the GlideScore. Any receptor that you specify ...",t:"Adding a Single Grid for VSW"},"1564":{i:0.000107344209275146,u:"../vsw_user_manual/vsw_receptor_multiple.htm",a:"If you want to add multiple receptors to the table in a single action, click Add Multiple. A file selector opens, in which you can browse to the location for the grid files and select multiple files. When you click OK, the grids are added to the Receptors for docking table, with the default Receptor ...",t:"Adding Multiple Pregenerated Grids for VSW"},"1565":{i:0.000144645953892155,u:"../vsw_user_manual/vsw_receptor_constraints.htm",a:"You can set up constraints for grids that are to be generated, you can request the use of constraints for both pregenerated and new grids, and you can set constraints on the ligand core position. All of these tasks are performed in the VSW - Receptor Constraints dialog box, which you open by ...",t:"Setting Constraints for VSW"},"1566":{i:0.000111091125951398,u:"../vsw_user_manual/vsw_receptor_aligning.htm",a:"If you are generating grids, you can align the receptors before the grids are generated, using the Protein Structure Alignment tool. To do so, select Align receptors. This option does not align receptors for pregenerated grids, only those that are generated by the current VSW run. The receptors are ...",t:"Aligning Grids for VSW"},"1567":{i:0.000118464356489744,u:"../vsw_user_manual/vsw_docking.htm",a:"If you specified grids in the Receptors tab, you can choose which of the three docking accuracy levels to include and set various options for docking in the Docking tab. The docking options are described in detail in  Ligand Docking . The Docking tab of the Virtual Screening Workflow panel. The ...",t:"Setting Docking Options for VSW"},"1568":{i:0.000120755885435047,u:"../vsw_user_manual/vsw_docking_common.htm",a:"At the top of the Docking tab are several options that apply to all docking calculations.  You can select Enhance planarity of conjugated pi groups, to increase the potential for out-of-plane deformations of planar groups such as phenyl rings.  Glide can add ionization and tautomerization penalties ...",t:"Setting Common Options for VSW"},"1569":{i:0.000120897158269836,u:"../vsw_user_manual/vsw_docking_stages.htm",a:"The Docking tab allows you to choose which Glide docking runs to include in the workflow, and contains options for HTVS, SP, and XP docking runs. To include a docking run in the workflow, select the corresponding option. Next to each option is an Option button, which opens the VSW - Docking Options ...",t:"Setting Up the VSW Docking Stages and Options"},"1570":{i:8.54214521413725E-05,u:"../vsw_user_manual/vsw_jobs.htm",a:"When the general setup and the docking setup has been completed, click the Settings button (see  Starting Jobs from Maestro ) to open the Job Settings dialog box, in which you can set job options and start the job. The run consists of a master (or “driver”) job and a set of subjobs. The master job ...",t:"Running VSW Jobs"},"1571":{i:0.000115180203477737,u:"../canvas_command_reference/canvas_command.htm",a:"All the Canvas command-line programs are available in $SCHRODINGER/utilities. The syntax statements below are therefore given with respect to this directory. For a description of the syntax and the options, run the program with the -h option. Canvas programs generally run as standalone processes in ...",t:"Running Canvas Applications from the Command Line"},"1572":{i:0.000132215807424675,u:"../canvas_command_reference/canvas_command_common.htm",a:"Many of the programs share common syntax, especially those that perform similar tasks. This section describes some elements of common usage. A range specification is a comma-separated list of indices or ranges, with no spaces. A range is defined by the endpoints of the index range, separated by a ...",t:"Common Syntax Descriptions for Canvas Programs"},"1573":{i:0.000218406969315364,u:"../canvas_command_reference/canvas_command_fingerprint.htm",a:"2D Fingerprint Programs The programs available for 2D fingerprints are listed in  Table 1 . For a description of the arguments and options, run the command for the program with the -h option.  ◀  Common Syntax Descriptions for Canvas Programs Canvas Command Reference Manual canvasFPGen  ▶",t:"2D Fingerprint Programs"},"1574":{i:0.000388554756399142,u:"../canvas_command_reference/canvas_command_canvasFPGen.htm",a:"This program generates fingerprints for a set of molecules. A variety of schemes for atom and bond typing are available, as listed in  Table 1 .There are also a number of scaling schemes, for the fingerprint values, listed in  Table 2 .    ◀  2D Fingerprint Programs Canvas Command Reference Manual \n ...",t:"canvasFPGen"},"1575":{i:0.000186520753282818,u:"../canvas_command_reference/canvas_command_3dpharm.htm",a:"\n            3D Pharmacophore Fingerprint Programs\n The program  canvasPharmFP generates fingerprints from 3D pharmacophores for the input structures. The fingerprint bits are set on the basis of the feature types and inter-feature distances.",t:"\n            3D Pharmacophore Fingerprint Programs\n"},"1576":{i:0.000197319465337833,u:"../canvas_command_reference/canvas_command_similarity.htm",a:"Similarity, Dissimilarity, and Clustering Programs The programs for generating similarity information, dissimilarity-based selections of molecules, and for clustering are listed in  Table 1 . Notes on the programs are given in the topics listed below.",t:"Similarity, Dissimilarity, and Clustering Programs"},"1577":{i:0.000193926114304517,u:"../canvas_command_reference/canvas_command_canvasDBCS.htm",a:"\n            canvasDBCS\n This program performs dissimilarity-based compound selection based on a pairwise matrix or a Canvas fingerprint file. Available metrics are listed in  Table 1 .  ◀  Similarity, Dissimilarity, and Clustering Programs Canvas Command Reference Manual \n            canvasLibOpt  ...",t:"\n            canvasDBCS\n"},"1578":{i:0.000172101888969895,u:"../canvas_command_reference/canvas_command_canvasLibOpt.htm",a:"\n            canvasLibOpt This program selects a subset of compounds from a pool by minimizing their similarity, and optionally by optimizing the ranges of specified properties. It can be used to fill holes in an existing library of compounds.  Given a pool of structures in a Canvas fingerprint ...",t:"\n            canvasLibOpt"},"1579":{i:0.000156248997063761,u:"../canvas_command_reference/canvas_command_canvasFPHist.htm",a:"\n            canvasFPHist \n This utility generates a histogram of nearest-neighbor similarities, by locating the largest off-diagonal value in each row of a similarity matrix for a single fingerprint file, or the largest value in each row of the similarity between two fingerprint files. ",t:"\n            canvasFPHist \n"},"1580":{i:0.000213950708918349,u:"../canvas_command_reference/canvas_command_canvasFPMatrix.htm",a:"\n            canvasFPMatrix This utility generates a pairwise similarity or distance matrix using binary or scaled fingerprints from one or two sets of molecules.  The metrics are listed in  Table 1 , along with their type and formula. The quantities in the formula are defined in  Table 2 , except ...",t:"\n            canvasFPMatrix"},"1581":{i:0.000167021387850998,u:"../canvas_command_reference/canvas_command_canvasKMeans.htm",a:"\n            canvasKMeans This program performs k-means clustering using Lloyd’s algorithm  [3] . The algorithm consists of a set of runs, each of which starts with the random selection of k data points (structures) as the initial centroids of the clusters. For each run, a series of iterations is ...",t:"\n            canvasKMeans"},"1582":{i:0.000153354292373588,u:"../canvas_command_reference/canvas_command_canvasLC.htm",a:"\n            canvasLC This program performs leader-follower clustering. In the leader-follower clustering method, data points are added to a cluster based on a cutoff on their distance from the cluster “leader”. The first leader is chosen by some method, then for each data point, the distance from ...",t:"\n            canvasLC"},"1583":{i:0.000187089241610142,u:"../canvas_command_reference/canvas_command_model.htm",a:"\n            Model Building and Related Application Programs\n The programs available for statistics, 2D QSAR, neural networks and machine learning are listed in  Table 1 . Notes on the programs are given in the topics listed below. ",t:"\n            Model Building and Related Application Programs\n"},"1584":{i:0.000147736279774815,u:"../canvas_command_reference/canvas_command_canvasMDS.htm",a:"\n            canvasMDS\n This program preforms multi-dimensional scaling (metric scaling based on principal coordinate analysis) on the output from canvasCSVMatrix or canvasFPMatrix, or an external distance matrix in CSV format.",t:"\n            canvasMDS\n"},"1585":{i:0.000162204645693564,u:"../canvas_command_reference/canvas_command_canvasMLR.htm",a:"\n            canvasMLR\n This program builds and tests multiple linear regression models using data supplied in a CSV (delimited) file. In the build mode, a model is developed on a training set and optionally applied to a test set contained in the same file as the training set. In the test mode, an ...",t:"\n            canvasMLR\n"},"1586":{i:0.000205847548895897,u:"../canvas_command_reference/canvas_command_canvasMolDescriptors.htm",a:"\n            canvasMolDescriptors \n            This program calculates molecular descriptors from a set of structures. You can create your own SMARTS patterns to define hydrogen-bond donors, acceptors, and rotatable bonds, and supply them in template files to canvasMolDescriptors. The format of the ...",t:"\n            canvasMolDescriptors"},"1587":{i:0.000161490086792618,u:"../canvas_command_reference/canvas_command_canvasPCAReg.htm",a:"\n            canvasPCAReg\n This program builds and tests principal component analysis regression models using data supplied in a CSV file. In build mode, a model is developed on a training set and optionally applied to a test set contained in the same file as the training set. In test mode, an ...",t:"\n            canvasPCAReg\n"},"1588":{i:0.00014521482239404,u:"../canvas_command_reference/canvas_command_canvasPLS.htm",a:"\n            canvasPLS\n This program builds and tests partial least squares regression models using data supplied in a CSV (delimited) file. In build mode, a model is developed on a training set and optionally applied to a test set contained in the same file as the training set. In test mode, an ...",t:"\n            canvasPLS\n"},"1589":{i:0.00013213370327743,u:"../canvas_command_reference/canvas_command_canvasKPLS.htm",a:"\n            canvasKPLS\n This program builds and tests kernel-based partial least squares regression models using data supplied in a CSV (delimited) file. In build mode, a model is developed on a training set and optionally applied to a test set contained in the same file as the training set. In ...",t:"\n            canvasKPLS\n"},"1590":{i:0.00013632397652878,u:"../canvas_command_reference/canvas_command_canvasBayes.htm",a:"\n            canvasBayes\n This program builds and tests a Bayes model from binary or continuous training data to predict the probability of a molecule at each activity level  [14] . ",t:"\n            canvasBayes\n"},"1591":{i:0.000130112881609692,u:"../canvas_command_reference/canvas_command_canvasNnet.htm",a:"\n            canvasNnet\n This program builds and tests an ensemble model of neural networks with data supplied in a CSV (delimited) file. The network has three layers, one input layer, one output layer, and one hidden layer. Networks are trained using a BFGS algorithm. In build mode, a model is ...",t:"\n            canvasNnet\n"},"1592":{i:0.000138082867158772,u:"../canvas_command_reference/canvas_command_canvas.htm",a:"\n            canvasSOM and canvasSOMBits\n These two program creates a Kohonen self-organizing map (SOM) from data in a CSV file or a binary fingerprint file. canvasSOM creates the map from scaled (real) values, whereas canvasSOMBits creates the map from binary fingerprint data.  For canvasSOM, the ...",t:"\n            canvasSOM and canvasSOMBits\n"},"1593":{i:0.000184453809124868,u:"../canvas_command_reference/canvas_command_utilities.htm",a:"Canvas Utilities In addition to the programs described above, various utilities are available for job management, format conversion, and various other tasks. These utilities are listed in  Table 1 . Notes are given in the topics listed below.",t:"Canvas Utilities"},"1594":{i:0.000158124656883137,u:"../canvas_command_reference/canvas_command_canvas_app.htm",a:"\n            canvas_app\n This utility runs a job according to the type and parameters specified in the input file. ",t:"\n            canvas_app\n"},"1595":{i:0.000200843960582151,u:"../canvas_command_reference/canvas_command_canvasConvert.htm",a:"\n            canvasConvert\n This utility converts between SD, Maestro, Canvas compact, CSV and SMILES molecular structure file formats.",t:"\n            canvasConvert\n"},"1596":{i:0.000217317664546487,u:"../canvas_command_reference/canvas_command_canvasJob.htm",a:"\n            canvasJob This program does setup and cleanup for jobs associated with a Canvas project.",t:"\n            canvasJob"},"1597":{i:0.000166453780292805,u:"../canvas_command_reference/canvas_command_canvasProjectDB.htm",a:"\n            canvasProjectDB\n This program can be used to create and update a Canvas project database. It can be run as a regular foreground process, or as a single-CPU job on any host that has access to the project directory.",t:"\n            canvasProjectDB\n"},"1598":{i:0.000151262393973998,u:"../canvas_command_reference/canvas_command_canvasSDMerge.htm",a:"canvasSDMerge This utility merges CSV data with an existing SD file into a new SD file, preserving the original SD file order.",t:"canvasSDMerge"},"1599":{i:0.000148512743932645,u:"../canvas_command_reference/canvas_command_canvasSearch.htm",a:"\n            canvasSearch\n This program searches a list of target molecules against a set of queries, composed of either molecules or partial structures. This program can also be used to filter a target list based on either standard REOS rules or a user-defined file containing SMILES queries and ...",t:"\n            canvasSearch\n"},"1600":{i:0.000154000352915443,u:"../canvas_command_reference/canvas_command_canvasMCS.htm",a:"\n            canvasMCS\n This utility finds the maximum common substructure (MCS) among a given set of molecules. ",t:"\n            canvasMCS\n"},"1601":{i:0.000176117428502887,u:"../canvas_command_reference/canvas_command_canvasScaffold.htm",a:"\n            canvasScaffold This utility decomposes structures into a set of scaffolds (detection), and searches a set of structures to determine which structures contain each scaffold in a set of scaffolds (matching).  Scaffold detection consists of identifying all unique scaffolds within a set of ...",t:"\n            canvasScaffold"},"1602":{i:0.000111227831107215,u:"../canvas_command_reference/canvas_command_scripting.htm",a:"In addition to the command-line utilities, Canvas has a Python API that can be used to write Python scripts. The API is described in the document  Canvas Python API . For the general Schrödinger Python API, visit the  Python API  web page. Canvas utilities are also available for use with KNIME, ...",t:"Scripting with Canvas"},"1603":{i:7.77757698572795E-05,u:"../combiglide_command_reference/combiglide_command_line.htm",a:"This chapter describes how to run various CombiGlide tasks from the command line. For information on command options, run the command with the -h option. See  Running Schrödinger Software  for information on using the command line. The tasks are described in the topics listed below.",t:"Running CombiGlide Tasks from the Command Line"},"1604":{i:0.000116114995815942,u:"../combiglide_command_reference/combiglide_command_reagentprep.htm",a:"\n            Reagent Preparation from the Command Line\n Reagent preparation is implemented as a set of additions to LigPrep, which includes two scripts, vcsprep and reagentprep, and a command-line option to the ligprep command. The script vcsprep is run at the beginning of the LigPrep workflow, to ...",t:"\n            Reagent Preparation from the Command Line\n"},"1605":{i:0.000115591104611342,u:"../combiglide_command_reference/combiglide_command_combgen.htm",a:"Enumeration of a combinatorial library can be done with the program combgen. The library can be constructed from a prepared core-containing structure, in which substitutions are made at the defined attachment points, or it can be constructed by linking several fragments in a linear sequence. The ...",t:"Combinatorial Library Enumeration from the Command Line"},"1606":{i:0.000133459814279038,u:"../combiglide_command_reference/combiglide_command_combgen_command.htm",a:"The combgen Command The syntax of the combgen command is as follows: \n$SCHRODINGER/combgen jobname [options]\n",t:"The combgen Command"},"1607":{i:0.000138989217552066,u:"../combiglide_command_reference/combiglide_command_combgen_input.htm",a:"The specification of the combinatorial library is made in the input file, which consists of a set of command lines. Each valid command line consists of a command name, followed by a colon (:), followed by a list of parameters specific to the command. The command names are case-insensitive. Comments ...",t:"The combgen Input File"},"1608":{i:0.000140634846144074,u:"../combiglide_command_reference/combiglide_command_combgen_input_core.htm",a:"The Core command specifies the core-containing molecule, if one is to be used. The syntax is as follows: \nCore: filename The file must be in Maestro format, and the first structure from this file is taken as the core-containing molecule. If no file name is given, a coreless chain is assumed. The ...",t:"The combgen Core Command"},});