define({"457":{i:0.000150710388164493,u:"../bioluminate_user_manual/crosslink_proteins_preparing.htm",a:"The link between the proteins is formed with standard peptide bonds, so the links must be formed from the N-terminus of a chain to the C-terminus of a chain. No adjustment of the relative position or orientation of the proteins is done in the process, so you must ensure that they are properly ...",t:"Preparing the Proteins for Cross-Linking"},"458":{i:0.00014471390172548,u:"../bioluminate_user_manual/crosslink_proteins_connection.htm",a:"The first task is to pick the residues on the proteins that you want to connect with a linker. These residues must be protein termini.  When you pick the termini, it can be useful to select them in the Workspace first. To do this, you can display the sequence viewer (Edit → Settings → Show Sequence ...",t:"Picking the Connection Residues for Cross-Linking Proteins"},"459":{i:0.000143587919713309,u:"../bioluminate_user_manual/crosslink_proteins_linkers.htm",a:"A great deal of flexibility is provided for the definition of the linkers used to connect the chain termini. The linkers are multimers, composed of monomers that are built from standard amino acids. By default, the standard amino acids are available as monomers, but you can define your own ...",t:"Defining the Linkers for Cross-Linking Proteins"},"460":{i:0.00014561058099443,u:"../bioluminate_user_manual/crosslink_proteins_job.htm",a:"The linkers are added by building a loop between the two selected termini. There are two methods available from the Linker conformation prediction option menu for determining the loop conformation: Loop lookup from curated PDB—Use a table of known loops taken from the PDB to determine the loop ...",t:"Choosing Methods and Running the Protein Cross-Linking Job"},"461":{i:0.000102583698892141,u:"../bioluminate_user_manual/crosslink_proteins_results.htm",a:"When the job finishes, the results are automatically listed in the Results tab. The results table lists the structures for each linker, showing details of the linker and its composition, the total strain energy, and the strain energy per amino acid. You can sort the table by the values in any column ...",t:"Examining the Results of Protein Cross-Linking"},"462":{i:0.000143858538971868,u:"../bioluminate_user_manual/residue_scanning.htm",a:"Scanning 1brs for Residue Mutations . This chapter describes how to scan a protein for potential residue mutations, generate mutated structures, and compare the properties of the mutated structures. The mutation sites and the mutations can be selected manually or selected automatically based on ...",t:"Scanning for Residue Mutations"},"463":{i:0.000126293612481244,u:"../bioluminate_user_manual/residue_scanning_preparation.htm",a:"The first step is to select the protein. The protein must be one that has been prepared for use in modeling. If it has not been prepared, we recommend that you prepare it with the Protein Preparation Wizard (on the Tools menu and the Tasks menu). Details of preparing a protein can be found in the  ...",t:"Selecting and Preparing the Protein for Residue Scanning"},"464":{i:0.0001139078693976,u:"../bioluminate_user_manual/residue_scanning_type.htm",a:"Once the structure is imported into the workflow, the next task is to choose the type of calculation to perform, on which the assessment of the mutations is made. This is done by selecting one of the Calculation type options. By default, only the protein stability is calculated (Stability). This is ...",t:"Choosing the Residue Scanning Calculation Type"},"465":{i:0.000121142192711039,u:"../bioluminate_user_manual/residue_scanning_manual.htm",a:"The residues that are available for mutation are listed in the table in the Residues tab. They are identified by the chain name, the residue number (and insertion code), and the 3-letter residue name. The table also shows information on the presence of disulfide bonds, the number of pi-pi ...",t:"Setting Up the Mutations Manually for Residue Scanning"},"466":{i:0.000125677668487412,u:"../bioluminate_user_manual/residue_scanning_homologs.htm",a:"In addition to manual selection of residues, you can use homology to other proteins to select residues to mutate and set the mutations for these residues, on the basis of variability or conservation of residues across the set of homologs, or on structural proximity or properties. You can do this in ...",t:"Using Homologs for Setting Up Mutations in Residue Scanning"},"467":{i:0.000136258542975976,u:"../bioluminate_user_manual/residue_scanning_homologs_blast.htm",a:"If you don’t have homologs for your protein, you can run a BLAST search to find homologs. Click Run a Blast Search.  The BLAST Search Settings dialog box opens, so you can change settings if you want.  Click Start Job in this dialog box, after changing any settings.  The job starts, and its log file ...",t:"Obtaining Homologs from a BLAST Search for Residue Scanning"},"468":{i:0.000138779942289337,u:"../bioluminate_user_manual/residue_scanning_homologs_importing.htm",a:"If you already have a set of homologs for a structure that you want to use to identify potential mutation sites, or if you have other structures that you want to use as homologs, you can import them.  If you have a set of homologs in a file, click the Import homologs button and choose From file. A ...",t:"Importing Homologs for Residue Scanning"},"469":{i:0.000137097588516478,u:"../bioluminate_user_manual/residue_scanning_homologs_aligning.htm",a:"To make use of the homologs, they must be aligned to the parent (query), if they are not already aligned. You can run a job to align the homologs, or you can do a manual alignment, or both. To run the alignment job, click Align Homologs, or click the Multiple Alignment toolbar button.  The ...",t:"Aligning Homologs for Residue Scanning"},"470":{i:0.000128638762509557,u:"../bioluminate_user_manual/residue_scanning_homologs_selecting.htm",a:"When the sequences are aligned, residues can be selected on the basis of the of the alignment. To choose residues to mutate, you may want to select residues at positions where the variability among the aligned homologs is high, or residues that vary in residue type, or residues that differ from most ...",t:"Selecting Residues by Homology for Residue Scanning"},"471":{i:0.000133957090980327,u:"../bioluminate_user_manual/residue_scanning_homologs_attributes.htm",a:"Residue selection filters can also be applied on the basis of solvent-accessible surface area and contact with other parts of the protein or with a ligand or cofactor. To apply these criteria to residue selection, select one or more of the following options in the Parent structure 3D criteria ...",t:"Selecting Residues by Structural Attributes for Residue Scanning"},"472":{i:0.000127696486819868,u:"../bioluminate_user_manual/residue_scanning_options.htm",a:"Setting Options for Residue Scanning When you have set up the mutations, you can set options for the job, which cover refinement of the mutated protein, and optimization of the stability or the affinity property by performing successive mutations. These options are set in the Options tab.",t:"Setting Options for Residue Scanning"},"473":{i:0.000133772127038946,u:"../bioluminate_user_manual/residue_scanning_options_refinement.htm",a:"When the residues are mutated, the protein structure around the mutation site should be allowed to relax in response to the mutation. Relaxation is carried out based on the refinement method selected, which is described below. The region around the mutation site that is relaxed during the ...",t:"Choosing Refinement Options for Residue Scanning"},"474":{i:0.000127985924609666,u:"../bioluminate_user_manual/residue_scanning_options_affinity.htm",a:"In affinity maturation, a search is performed to find the optimum binding affinity between two sets of protein chains, or the maximum stability of the entire protein. To perform an affinity calculation, select Perform affinity maturation in the Options tab. You can choose the property by selecting ...",t:"Setting Optimization Options for Affinity Maturation"},"475":{i:0.000135318775249235,u:"../bioluminate_user_manual/residue_scanning_options_properties.htm",a:"As part of the calculation, the change in a number of properties can also be calculated. These properties are displayed when you view the results. The change in property value that is displayed is the value for the mutated structure minus the value for the parent structure, so a positive value for ...",t:"Selecting Properties to Calculate for Residue Scanning"},"476":{i:0.00013315419725262,u:"../bioluminate_user_manual/residue_scanning_job.htm",a:"To run the job, you first set job parameters, and then submit the job to a host for execution. Click the Settings button to open the Job Settings dialog box. In this dialog box, you can choose whether to append the results to your project, or leave the results on disk in the current directory, where ...",t:"Running the Residue Scanning Job"},"477":{i:0.000354690870025828,u:"../bioluminate_user_manual/residue_scanning_results.htm",a:"When the job finishes, the Residue Scanning Viewer or Affinity Maturation Viewer panel opens, displaying changes in properties for each mutation, and a graph of one of the properties against the row number. If you want to examine results from a job that was completed earlier, you can open this panel ...",t:"Examining the Mutation Results for Residue Scanning"},"478":{i:0.000164983684642202,u:"../bioluminate_user_manual/residue_scanning_results_affinity.htm",a:"The change in the binding affinity of the protein due to the mutation is calculated from a thermodynamic cycle, which can be represented as follows: where R is the receptor, L is the ligand in the parent, and L\u0027 is the mutated ligand. R+L and R+L\u0027 represent the separated receptor and ligand. R·L and ...",t:"Binding Affinity Prediction from Residue Scanning"},"479":{i:0.000213820428944272,u:"../bioluminate_user_manual/residue_scanning_results_stability.htm",a:"The stability of the protein due to the mutation is calculated from a thermodynamic cycle, which can be represented as follows: where L(u) is the unfolded parent protein, L(f) is the folded parent protein, L\u0027(u) is the unfolded mutated protein, and L\u0027(f) is the folded mutated protein. The change in ...",t:"Stability Prediction from Residue Scanning"},"480":{i:0.000157150838274691,u:"../bioluminate_user_manual/residue_scanning_results_sasa.htm",a:"Solvent-Accessible Surface Areas from Residue Scanning Solvent-accessible surface areas are calculated with the Lee-Richards method  [12] , which is the same method used in the Canvas and Phase applications.",t:"Solvent-Accessible Surface Areas from Residue Scanning"},"481":{i:0.000165822679714361,u:"../bioluminate_user_manual/residue_scanning_results_complementarity.htm",a:"The surface complementarity is calculated using the method of Lawrence and Colman  [10] , which is outlined here. The complementarity at any point on the molecular surface (Connolly surface  [11] ) of one protein (A) is determined by finding the nearest point on the molecular surface of the other ...",t:"Surface Complementarity from Residue Scanning"},"482":{i:0.000131893051918488,u:"../bioluminate_user_manual/residue_scanning_command_line.htm",a:"You can run residue scanning and affinity maturation jobs from the command line with the following command: \n$SCHRODINGER/run residue_scanning_backend.py [options] structure-file If you are running on Windows from a Schrodinger Command Prompt window, you can omit $SCHRODINGER/. To view the command ...",t:"Running Residue Scanning from the Command Line"},"483":{i:0.000124682432291568,u:"../bioluminate_user_manual/residue_scanning_nonstandard_aa.htm",a:"If you want to use nonstandard amino acids in your residue scanning job, you can create a database of nonstandard residues, and select the residues from the database that will be shown in the residue selection tools. You can create or open a database and make the residue selection in the Nonstandard ...",t:"Residue Scanning with Nonstandard Amino Acids"},"484":{i:0.000112123941927512,u:"../bioluminate_user_manual/cysteine_mutation.htm",a:"Locating Possible Mutations for Disulfide Bridges in 1vhu . Disulfide bridges between cysteine residues add to the stability of a protein structure. Mutating residues to form or break disulfide bridges offers a way of controlling the stability of a protein. This chapter describes how to run a ...",t:"Locating Possible Mutations for Disulfide Bridges"},"485":{i:0.000117557311116664,u:"../bioluminate_user_manual/cysteine_mutation_summary.htm",a:"Display the protein you want to analyze in the Workspace. You should ensure that it is properly prepared (with the Protein Preparation Wizard).  See  Selecting and Analyzing the Protein for Cysteine Mutation . If you want to analyze only part of the protein, select the residues that you want to ...",t:"Cysteine Mutation Workflow Summary"},"486":{i:0.000115110310167892,u:"../bioluminate_user_manual/cysteine_mutation_protein.htm",a:"The Cysteine Mutation panel can analyze a single protein for potential mutations, such as a structure from the PDB, or it can analyze a molecular dynamics trajectory, to find frames in which suitable residues come close enough to form disulfide bridges.  If you want to analyze a single protein, ...",t:"Selecting and Analyzing the Protein for Cysteine Mutation"},"487":{i:0.000185916993748459,u:"../bioluminate_user_manual/cysteine_mutation_residue_pairs.htm",a:"The Residue pairs for mutation table lists the pairs of residues that have the potential to form or break disulfide bonds. The table rows that are shown are controlled by the options below the table. The table columns include a range of relevant properties, which are described in  Table 1 . You can ...",t:"Choosing Residue Pairs to Mutate to or from Cysteine"},"488":{i:0.000163560748302419,u:"../bioluminate_user_manual/cysteine_mutation_relaxation.htm",a:"When a pair of residues is mutated, the side-chain conformations are sampled to find the best geometry. The residues are then minimized, to relieve strain. You can include additional residues around the mutation site in the minimization, to extend the relaxation of the protein. There are three ...",t:"Relaxing the Structure Around the Cysteine Mutation Site"},"489":{i:0.000170992635671316,u:"../bioluminate_user_manual/cysteine_mutation_job.htm",a:"When you have selected the residue pairs that you want to mutate, and set any options for relaxing the structure around the residue pairs, enter a name in the Job name text box, and click Run.  If you want to choose whether to incorporate the results into the project, click the Settings button. The ...",t:"Running the Cysteine Mutation Job"},"490":{i:0.000173098469847196,u:"../bioluminate_user_manual/cysteine_mutation_results.htm",a:"The results of a cysteine mutation job are automatically displayed in the Results tab if you chose to append new entries in the Start dialog box. To view results of any other cysteine mutation job, click Load Results from Previous Run. This button opens a file selector, in which you can browse to ...",t:"Examining the Cysteine Mutation Results"},"491":{i:0.000109028779686236,u:"../bioluminate_user_manual/antibody_modeling.htm",a:"Homology Modeling of an Antibody . The main task of antibody modeling in BioLuminate is to construct a homology model based on a database of antibody templates. Once the model is constructed, it can be humanized if required. A database of antibody templates is provided with BioLuminate, based on a ...",t:"Antibody Modeling"},"492":{i:0.000126927507837093,u:"../bioluminate_user_manual/antibody_modeling_structure.htm",a:"Modeling of the Fv region of an antibody involves prediction of both the framework (FR) region and the variable loop (CDR) region. These regions are identified automatically and their structure predicted by homology, based on known antibody structures from the PDB, or from your own database of ...",t:"Modeling an Antibody Structure"},"493":{i:0.000135625568021109,u:"../bioluminate_user_manual/antibody_modeling_structure_summary.htm",a:"The basic procedure for running a prediction using a homology model is as follows: Import the sequences for the light and heavy chains: Click the part of the diagram for the region you want to import, and choose a source from the list that is displayed. Set up the antibody databases that you want to ...",t:"Antibody Homology Modeling Summary"},"494":{i:0.000135296167726951,u:"../bioluminate_user_manual/antibody_modeling_structure_import_sequence.htm",a:"The first step in the modeling process is to import the antibody sequence, and if you want to re-predict only a part of the antibody, the existing structure.  The left part of the panel displays a diagram of the Fv region of an antibody. Clicking on the light variable or heavy variable region in the ...",t:"Importing the Antibody Sequence"},"495":{i:0.000167246636424913,u:"../bioluminate_user_manual/antibody_modeling_structure_choosing_database.htm",a:"The model of the antibody is built using a database of antibodies, each of which has been analyzed and curated in terms of framework and hypervariable loop regions. This database is used to model antibody structures from a sequence. A curated database obtained from the PDB is provided with the ...",t:"Choosing an Antibody Database"},"496":{i:0.000168164142929861,u:"../bioluminate_user_manual/antibody_modeling_structure_fr_coords.htm",a:"You can choose between two sources for the coordinates of the framework region: the input structure, or a homology model.  If you want to use the coordinates from the input structure, select Input structure. The sequences that you imported for the light and heavy regions must of course be associated ...",t:"Selecting the Coordinates for the Framework Region of the Antibody"},"497":{i:0.000125500947791731,u:"../bioluminate_user_manual/antibody_modeling_structure_filtering_database.htm",a:"By default, the homology search scans all of the structures in the databases you have chosen. If you want to restrict the search to only the structures that have certain properties, you can set up a filter on these properties before conducting the search. To set up the filter, click Filter Search ...",t:"Filtering the Antibody Database by Property"},"498":{i:0.000133823266032581,u:"../bioluminate_user_manual/antibody_modeling_structure_generating_model.htm",a:"With the framework region defined, you can now generate a model for the six loops. As for the framework region, you have the choice of using input coordinates for a loop or predicting the loop from the database, either by automatic selection or manual selection of a template.  The six CDR loops are ...",t:"Generating the Antibody Loop Model"},"499":{i:0.000130361400919186,u:"../bioluminate_user_manual/antibody_modeling_structure_model_options.htm",a:"By default, only the variable region is modeled, with automatic detection of the loops. If you want to override either of these defaults, click Advanced Options, and make settings in the Antibody Prediction - Advanced Options dialog box. To model the full Fab region, select Generate Fab model, then ...",t:"Advanced Antibody Model Building Options"},"500":{i:0.000191035526660999,u:"../bioluminate_user_manual/antibody_modeling_structure_clusters.htm",a:"If you want to select the loop templates that are used to build the model, you can do so in the Loop Clusters dialog box, which you open by double-clicking in the Loop Template column of the input table in the Basic Loop Model tab, or right-clicking and choosing Select Cluster. When this dialog box ...",t:"Selecting Clusters for an Antibody Loop"},"501":{i:0.000139620334367209,u:"../bioluminate_user_manual/antibody_modeling_structure_refining_loops.htm",a:"Frequently, the model based on homology is adequate, and no further prediction is needed. However, in some cases, prediction of certain loops (especially H3) can be problematic, and advanced methods can result in better predictions. You can carry out advanced loop refinement using the Advanced Loop ...",t:"Refining Antibody Loops"},"502":{i:0.000133044664006601,u:"../bioluminate_user_manual/antibody_modeling_humanize_mutation.htm",a:"If you have an antibody model that is not based on a human species, you can humanize it by mutating residues to create a more human-compatible protein. Appropriate sites for mutation can be found by comparing the antibody to homologs and identifying residues that satisfy various criteria, such as ...",t:"Humanizing Antibodies by Residue Mutation"},"503":{i:0.000138859001158099,u:"../bioluminate_user_manual/antibody_modeling_humanize_mutation_summary.htm",a:"Import a structure into the workflow, either from the Workspace or from a Maestro file (Import structure from). Click Homology Criteria. (Optional) Select databases for running the search for homologs. Run the search for homologs (Search Antibody Database for Homologs) and align the homologs (Align ...",t:"Summary of Antibody Humanization"},"504":{i:0.000140590349031002,u:"../bioluminate_user_manual/antibody_modeling_humanize_mutation_importing.htm",a:"The first step in the process is to import the antibody into the workflow using the Import structure from tools, and then analyze the antibody to locate the antibody regions and calculate solvent-accessible surface areas (SASA). By default, the entire structure is analyzed, but if you want to limit ...",t:"Importing the Antibody for Humanization"},"505":{i:0.00014088623716562,u:"../bioluminate_user_manual/antibody_modeling_humanize_mutation_homologs.htm",a:"The next step is to search for homologs in an antibody database. These homologs are used to identify residues that could be mutated, based on a comparison of the homologs with the parent sequence. These tasks are done in the Homology Suggestions dialog box, which you open by clicking Homology ...",t:"Finding Homologs for Antibody Humanzation"},"506":{i:0.00014019911863452,u:"../bioluminate_user_manual/antibody_modeling_humanize_mutation_regions.htm",a:"If you want to limit the mutations to specified regions of the antibody, you can do so by selecting one or more of the Substitute residues in options. The regions can be selected by group: CDR, Non-CDR, VH, VL, Fab, Fc, or Hinge. These options are available by default. For finer control, you can ...",t:"Selecting Regions to Mutate for Antibody Humanization"},"507":{i:0.000137478285144972,u:"../bioluminate_user_manual/antibody_modeling_humanize_mutation_criteria.htm",a:"The next task is to set up criteria for selecting the residues to mutate. You can use two kinds of criteria: ones based on homology, and ones based on the 3D structure of the parent.  Criteria that are based on homology use the variations in the residues at each residue position among the homologs, ...",t:"Setting Up Residue Selection Criteria for Antibody Humanization"},"508":{i:0.000128563058901231,u:"../bioluminate_user_manual/antibody_modeling_humanize_mutation_selecting.htm",a:"When you have finished setting up the selection criteria, click Save. The dialog box closes, and the residues that match the criteria you chose are selected in the residues table of the Residues tab. A set of mutations based on the variations in the homologs is chosen for these residues. The ...",t:"Selecting the Residues and Their Mutations for Antibody Humanization"},"509":{i:0.000133093247275031,u:"../bioluminate_user_manual/antibody_modeling_humanize_mutation_properties.htm",a:"As part of the calculation, the change in a number of properties can also be calculated. These properties are displayed when you view the results. The change in property value that is displayed is the value for the mutated structure minus the value for the parent structure, so a positive value for ...",t:"Selecting Properties to Calculate for Antibody Humanization"},"510":{i:0.000124723558521755,u:"../bioluminate_user_manual/antibody_modeling_humanize_mutation_refinement.htm",a:"In the job that performs the mutations, the structure of the mutated residue is optimized, but you can also refine the structures of adjacent residues. Settings for refinement can be made in the Options tab. The residues that are included in the refinement are selected by distance from the mutated ...",t:"Setting Refinement Options for Mutated Residues in Antibody Humanization"},"511":{i:0.000123927978462567,u:"../bioluminate_user_manual/antibody_modeling_humanize_mutation_running.htm",a:"When you have finished making settings, you can run the job: To run the job with the current job settings, enter a name in the Job Name dialog box and click Run. To make job settings in the Job Settings dialog box, including the host, job name, number of processors, number of subjobs, and treatment ...",t:"Running the Mutation Job for Antibody Humanization"},"512":{i:0.000128288700069564,u:"../bioluminate_user_manual/antibody_modeling_humanize_mutation_results.htm",a:"When the job finishes, the Humanize Antibody Viewer opens automatically, with the results loaded. You can also open this panel by choosing Tasks → Antibody Modeling → Humanization → Mutation Results, and then clicking Import in the panel to locate and load a set of results. The viewer is the same as ...",t:"Analyzing the Antibody Humanization Results"},"513":{i:0.00014986924345205,u:"../bioluminate_user_manual/antibody_modeling_humanize_grafting.htm",a:"Another way of creating humanized antibodies is to graft the CDR loops of a murine antibody onto a human framework. This task can be performed in the Antibody Humanization: CDR Grafting panel, which you open by choosing Tasks → Biologics → Humanization → CDR Grafting in the Task Tool. The human ...",t:"Humanizing Antibodies by CDR Grafting"},"514":{i:0.000185792459838001,u:"../bioluminate_user_manual/antibody_modeling_databases.htm",a:"Modeling antibodies is done with the help of a database of prepared antibody structures. A database is supplied with the distribution, and is the default database. You can create and manage your own databases in the Antibody Database Management, which you open by choosing Tasks → Biologics → ...",t:"Antibody Databases"},"515":{i:0.000113969279812536,u:"../bioluminate_user_manual/antibody_modeling_numbering.htm",a:"If you want to use one of the common residue numbering schemes for antibodies, (Chothia, Enhanced Chothia, Kabat, IMGT, AHo), you can apply the numbering scheme from the Multiple Sequence Viewer panel, which you open from the Tools menu. First display the structure in the Workspace, then open the ...",t:"Antibody Numbering Schemes"},"516":{i:0.000421044868947245,u:"../bioluminate_user_manual/bioluminate_references.htm",a:"Olsson, M. H. M.; Søndergard, C. R.; Rostkowski, M.; Jensen J. H. PROPKA3: Consistent Treatment of Internal and Surface Residues in Empirical pKa predictions. J. Chem. Theor. Comput., 2011, 7, 525–537. North, B.; Lehmann, A.; Dunbrack, R. L., Jr. A new clustering of antibody CDR loop conformations. ...",t:"BioLuminate References"},"517":{i:6.13254596701412E-05,u:"../canvas_user_manual/canvas_intro.htm",a:"Canvas is a cheminformatics package that provides a range of applications for structural and data analysis, including fingerprints, similarity searching, substructure searching, selection by diversity, clustering, building regression and classification models. The Canvas graphical interface is ...",t:"About Canvas"},"518":{i:9.64903045792281E-05,u:"../canvas_user_manual/canvas_gui.htm",a:"The Canvas graphical interface is designed for project-oriented work and provides chemical structure storage, data analysis and visualization, and access to a wide range of applications that are also available through the command line. The interface also provides links to Maestro that allow you to ...",t:"The Canvas Graphical Interface"},"519":{i:0.000124106035280881,u:"../canvas_user_manual/canvas_gui_starting.htm",a:"Linux: To start the Canvas interface, enter the following command: \n$SCHRODINGER/canvas [ -proj projName.cnv [ -mae maeFile|-sd sdFile\n[-append|-replace] [-autodelete]] | -zippedproj projName.cnv[.]zip ]\n The -proj option opens the specified project. If the project exists, it is opened. If the ...",t:"Starting and Quitting Canvas"},"520":{i:0.00010244545650411,u:"../canvas_user_manual/canvas_gui_main_window.htm",a:"The Canvas main window has a menu bar, toolbars, a spreadsheet area, a Project View panel, a Messages View panel, and a status area. The two panels are docked into the main window by default but can be undocked. You can show or hide these views from the Project menu or from the context menu ...",t:"The Canvas Main Window"},"521":{i:9.10751723159946E-05,u:"../canvas_user_manual/canvas_gui_main_window_menu_bar.htm",a:"The menu bar has the following menus: File—Open and close projects, import and export structures and data, set preferences, and quit Canvas. This menu is described in detail in  Canvas Projects . Edit—Copy cells, clear cell content, add or delete rows or columns, select cells, rows, or columns, find ...",t:"The Canvas Menu Bar"},"522":{i:0.000131415529877412,u:"../canvas_user_manual/canvas_gui_main_window_toolbars.htm",a:"The Canvas interface has four toolbars: File, Edit, Select Rows, and Structure Size. You can show or hide the toolbars from the shortcut menu, and you can show or hide toolbar buttons in the Preferences panel (File → Preferences → Toolbar Buttons). The File toolbar has buttons for many of the common ...",t:"The Canvas Toolbars"},"523":{i:0.000138449975148041,u:"../canvas_user_manual/canvas_gui_main_window_keyboard.htm",a:"Keyboard Shortcuts in Canvas Many of the menu items have keyboard shortcuts. These shortcuts are listed in  Table 1 . ◀  The Canvas Toolbars Canvas User Manual Canvas Projects  ▶",t:"Keyboard Shortcuts in Canvas"},"524":{i:0.000140193297499007,u:"../canvas_user_manual/canvas_gui_projects.htm",a:"Canvas stores all of its data in a project, which is simply a directory that contains a number of subdirectories and files. A Canvas project directory has the extension .cnv. Canvas does not have scratch projects, so you must create a project before you can import structures. Information on Canvas ...",t:"Canvas Projects"},"525":{i:0.000153488906297744,u:"../canvas_user_manual/canvas_gui_projects_openclose.htm",a:"The basic operations of opening and closing projects can be performed from the File menu:  New Project—Create a new Canvas project. You can also use CTRL+N (⌘N), or click the New Project toolbar button.  Opens a file selector in which you can navigate to a location and name the project. The default ...",t:"Opening and Closing Canvas Projects"},"526":{i:0.000156913114879466,u:"../canvas_user_manual/canvas_gui_projects_import.htm",a:"There are several ways of importing structures and data into a Canvas project. From inside Canvas, you can import from a file or from a project. From Maestro, you can export selected entries to a Canvas project—see  Exporting Structures and Data to a Canvas Project  for details. To import structures ...",t:"Importing Structures and Data into Canvas Projects"},"527":{i:0.000155702125934678,u:"../canvas_user_manual/canvas_gui_projects_export.htm",a:"Canvas provides several ways of exporting structures and data. You can export the selected rows (rows in which all columns are selected) to a file or to Maestro. You can also copy and paste from Canvas into a spreadsheet. Properties are exported in the order in which they appear in the view, so you ...",t:"Exporting Structures and Data from Canvas Projects"},"528":{i:0.000148003991524527,u:"../canvas_user_manual/canvas_gui_projects_view.htm",a:"This panel gives a summary of all the information that is available in the project. It lists the applications and the various classifications of the data (filters, views, partitions) as a tree. When information exists in any of these categories, you can expand the display by clicking the turner (the ...",t:"The Canvas Project View Panel"},"529":{i:0.000150206837701459,u:"../canvas_user_manual/canvas_gui_projects_messages.htm",a:"The Messages View panel displays messages generated by actions on the project, including the running of applications. Log messages and job error messages are displayed in this panel. You can show and hide this panel by choosing Project → Messages View. You can hide the panel by clicking the close ...",t:"The Canvas Messages View Panel"},"530":{i:0.000165679500395132,u:"../canvas_user_manual/canvas_gui_spreadsheet.htm",a:"The spreadsheet is the area where structures and properties are displayed. Although it does not have the full capabilities of a spreadsheet program such as Microsoft Excel or OpenOffice Calc, it does function in many respects in the same way. Information on the spreadsheet is given in the topics ...",t:"The Canvas Spreadsheet"},"531":{i:0.000160310252166715,u:"../canvas_user_manual/canvas_gui_spreadsheet_configure.htm",a:"The spreadsheet can be configured and navigated in much the same way as other spreadsheets, as described below.  You can resize individual rows and columns by dragging their borders. To resize multiple rows or columns simultaneously, select the rows or columns and drag one of the borders inside the ...",t:"Configuring and Navigating the Canvas Spreadsheet"},"532":{i:0.00015550105171812,u:"../canvas_user_manual/canvas_gui_spreadsheet_shortcut.htm",a:"The spreadsheet has shortcut menus for the selected rows, for the column headings, and for the selected structure cell. These are displayed when you right-click and hold.  The selected rows shortcut menu has the following items, given with their menu equivalents: Delete—delete the selected rows ...",t:"Shortcut Menus in the Canvas Spreadsheet"},"533":{i:0.000157314373040016,u:"../canvas_user_manual/canvas_gui_spreadsheet_select_copy.htm",a:"There are several ways of selecting cells in the spreadsheet. The selected cells are highlighted with a blue background.  \n            Selection of cells with the mouse follows the usual rules for tables:  To select a row, click the row index. \n     To select multiple rows, use shift-click and ...",t:"Selecting and Copying Rows, Columns, and Cells in the Canvas Spreadsheet"},"534":{i:0.000155106174474322,u:"../canvas_user_manual/canvas_gui_spreadsheet_finding.htm",a:"If you want to search for text in the spreadsheet, you can use the Find panel. To open this panel, choose Edit → Find in the main window, or click the Find toolbar button. To specify the search string, enter it in the Find what text box. You can qualify the search to match the entire contents of the ...",t:"Finding Text in the Canvas Spreadsheet"},"535":{i:0.00014549957061459,u:"../canvas_user_manual/canvas_gui_spreadsheet_color.htm",a:"You can control some aspects of the structure coloring. To color the non-carbon atoms, choose Structure → Color Heteroatoms. If you do a substructure query, you can color the matches by choosing Structure → Highlight Matches. The color used can be set by choosing Structure → Choose Highlight Color. ",t:"Coloring Structures in the Canvas Spreadsheet"},"536":{i:0.000141964225864816,u:"../canvas_user_manual/canvas_gui_spreadsheet_copypaste.htm",a:"\n            Structures can be copied and pasted in MDL (SD) format within Canvas. To copy the current structure in MDL format, choose Structure → Copy Structure; to paste a structure into a cell (replacing the existing structure), select the cell and choose Structure → Paste. Pasting cannot be ...",t:"Copying and Pasting Structures in the Canvas Spreadsheet"},"537":{i:0.00013909380682323,u:"../canvas_user_manual/canvas_gui_spreadsheet_rename.htm",a:"You can change the structure name by selecting the structure and choosing Structure → Edit Name or right-clicking in the structure cell and choosing Edit Name. In the dialog box that opens, you can type in a new name. You can also change the names used for all the structures by choosing a property ...",t:"Renaming Structures in the Canvas Spreadsheet"},"538":{i:0.000176664715062173,u:"../canvas_user_manual/canvas_gui_spreadsheet_edit.htm",a:"If you want to edit a structure, you can open the 2D structure editor by any of these methods: Double-click the cell containing the structure.\n     Right-click on the structure and choose Edit Structure.\n     Select the structure in the spreadsheet and choose Structure → Edit Structure.\n     The 2D ...",t:"Editing Structures in the Canvas Spreadsheet"},"539":{i:0.000135467194481735,u:"../canvas_user_manual/canvas_gui_spreadsheet_edit_data.htm",a:"You can edit the data in the spreadsheet cells directly by choosing Data → Edit Spreadsheet, or clicking the Edit Spreadsheet toolbar button. The spreadsheet is placed in editing mode, and you can click in any cell and change the data in the cell, with the exception of the structure cells (first ...",t:"Editing Data Cells in the Canvas Spreadsheet"},"540":{i:0.000129165365506548,u:"../canvas_user_manual/canvas_gui_spreadsheet_add_delete.htm",a:"To append a row to the spreadsheet, choose Edit → Add Row or click the toolbar button. A new empty row is added, and the Structure cell is selected, so that you can paste a structure into the cell, or build one in the Edit Structure dialog box. , To append a column to the spreadsheet, choose Edit → ...",t:"Adding and Deleting Rows and Columns in the Canvas Spreadsheet"},"541":{i:0.000138608621841601,u:"../canvas_user_manual/canvas_gui_spreadsheet_views.htm",a:"There are occasions on which you might want to see only a particular range of rows and columns, arranged in a particular way. Such an selection and arrangement is called a “view”. Changes to the view can be made manually, or as a result of some other process. When the view is changed, the title bar ...",t:"Displaying Different Views of the Canvas Spreadsheet"},"542":{i:0.000125878063794803,u:"../canvas_user_manual/canvas_gui_spreadsheet_custom_views.htm",a:"Changes made to the view are transient: as soon as another action is taken, the previous arrangement and visibility of the rows and columns is modified. If you want to preserve a particular view of the data, you can create a custom view that is saved in the project and can be displayed later. As ...",t:"Creating and Using Custom Views of the Canvas Spreadsheet"},"543":{i:0.000123865956305809,u:"../canvas_user_manual/canvas_gui_spreadsheet_custom_views_apply.htm",a:"Custom views are independent views of the spreadsheet. However, you can apply a custom view to the master view by choosing View → Apply to Master. This action is also available from the shortcut menu for the view in the Project View panel. Choosing View → Apply to Master allows you to apply the ...",t:"Applying a Custom View to the Master View"},"544":{i:0.000126312695150177,u:"../canvas_user_manual/canvas_gui_spreadsheet_custom_views_combine.htm",a:"Custom views can be combined with logical operations to update the current view or create new custom views. This capability is useful, for example if you have created custom views with several different filters, and want to combine the results. To combine other custom views with the current view, ...",t:"Combining Views with Logical Operations"},"545":{i:0.000140653826132135,u:"../canvas_user_manual/canvas_gui_organize.htm",a:"Canvas provides various ways of organizing the rows in the spreadsheet. Some of them, such as clustering, involve running an application. These means of organizing data are treated in  Running Applications from Canvas . The simplest ways of organizing the data are sorting and partitioning, which are ...",t:"Organizing the Canvas Spreadsheet Data"},"546":{i:0.000132639710193563,u:"../canvas_user_manual/canvas_gui_organize_sort.htm",a:"One of the simplest ways of organizing the data is sorting. To sort the rows in any view of the spreadsheet, choose Data → Sort, which opens the Sort dialog box. This dialog box allows you to sort the data (New sort), to reverse the order of the previous sort (Reverse current order), or restore the ...",t:"Sorting the Canvas Spreadsheet"},"547":{i:0.000146195111637801,u:"../canvas_user_manual/canvas_gui_organize_partition.htm",a:"Another way of organizing the structures (or rows) is to partition them into classes. The classes can be based on the value of some property, or they can be assigned manually or randomly.  Partitioning the rows creates a new property, which is shown as a column in the spreadsheet whose cells are ...",t:"Partitioning Rows Into Classes in the Canvas Spreadsheet"},"548":{i:0.000145868678469876,u:"../canvas_user_manual/canvas_gui_organize_partition_create.htm",a:"There are three ways of creating a partition from scratch: from the selected rows, from values of a property, and by random assignment. You can also save an existing partition with a new name (Partition → Save As) and then edit the partition (Partition → Edit). To create a new partition from the ...",t:"Creating Partitions"},"549":{i:0.000152178022092904,u:"../canvas_user_manual/canvas_gui_organize_partition_change.htm",a:"When a partition has been created, you can add classes, move rows to new classes or declassify them, edit the partition, or delete the partition. To create a new class, first select the rows in the spreadsheet. The rows can be unclassified rows, or rows that are already in one or more classes. Next, ...",t:"Changing the Classes in a Partition"},"550":{i:0.000174771252750172,u:"../canvas_user_manual/canvas_gui_organize_heatmap.htm",a:"To highlight the values of a property, you can apply a heat map for the property, which colors the cells in the spreadsheet according to the property value. The colors provide a visual organization of the property values. To set up the heat maps, choose Data → Heat Map, which opens the Heat Map ...",t:"Creating a Property Heat Map in the Canvas Spreadsheet"},"551":{i:0.000189634547131562,u:"../canvas_user_manual/canvas_gui_filtering.htm",a:"Canvas allows you to filter the data on the values of properties or on counts of functional groups, which are represented as SMARTS patterns. You can also run a query to find all molecules that have a particular substructure. Filters and queries can be stored and be reused. All types of filters and ...",t:"Filtering and Querying the Data in the Canvas Spreadsheet"},"552":{i:0.000153088189783211,u:"../canvas_user_manual/canvas_gui_filtering_property.htm",a:"You can filter the structures in the spreadsheet by a combination of property values. This task is performed in the Property Filter dialog box, which you can open by choosing Data → Property Filter or Query → Properties. To construct a new filter, you first set up a condition on a property by ...",t:"Filtering the Canvas Spreadsheet by Property Values"},"553":{i:0.000134217591426894,u:"../canvas_user_manual/canvas_gui_filtering_classes.htm",a:"In a similar way to filtering by property values, you can filter the data by the property classes stored in a partition. This feature allows you to perform more complex filtering on a single property. To do so, first open the partition, then choose Partition → Filter. In the Partition Filter panel, ...",t:"Filtering the Canvas Spreadsheet by Property Classes"},"554":{i:0.000138887238158948,u:"../canvas_user_manual/canvas_gui_filtering_structure.htm",a:"You can filter the structures based on their chemical content, which includes the usual functional groups. This task is performed in the Structure Filter dialog box, which you open by choosing Structure → Structure Filter.  Each condition on the occurrence of a chemical feature is specified by a row ...",t:"Filtering the Canvas Spreadsheet by Structure"},"555":{i:0.000139516685482094,u:"../canvas_user_manual/canvas_gui_filtering_substructure.htm",a:"You can filter the structures in the database by running a substructure query. This task is performed in the Substructure Query dialog box, which you open by choosing Query → Substructure Query or clicking the Substructure Query toolbar button. The query can be run on the selected rows in the ...",t:"Filtering the Canvas Spreadsheet by Substructure"},"556":{i:0.000143629874765129,u:"../canvas_user_manual/canvas_gui_filtering_duplicates.htm",a:"There is no restriction in Canvas on the structures that you can import, other than any filters you apply in the process of importing. This means that the project can contain duplicates. If you want to locate the duplicates, you can do so by choosing Structure → Detect Duplicates. Two columns are ...",t:"Detecting Duplicate Structures in the Canvas Spreadsheet"},"557":{i:0.000150955368815813,u:"../canvas_user_manual/canvas_gui_charts.htm",a:"Canvas provides several kinds of charts for visualizing the data: scatter plots, histograms, and frequency pie charts. To create a chart, choose the chart type from the Chart menu, or click the toolbar button for the chart type.  Each chart panel has a File menu, a Settings menu, and one or more ...",t:"Making Charts of the Canvas Spreadsheet Data"},"558":{i:0.000202237570587647,u:"../canvas_user_manual/canvas_gui_charts_scatter.htm",a:"\n            Scatter Plots of Canvas Data Scatter plots can be used to represent up to four different variables. The default plot shows two variables, on the x and y axes. The other two variables can be represented by the color and the size of the plot symbols. The property values for each point are ...",t:"\n            Scatter Plots of Canvas Data"},"559":{i:0.000159853527877344,u:"../canvas_user_manual/canvas_gui_charts_histogram.htm",a:"\n            Histograms of Canvas Data Histograms show the frequency of occurrence of values of a single variable, which you can choose from the Property option menu. The variable can be numeric, and the bars defined in terms of ranges of the variable; or it can be categorical, and the bars record ...",t:"\n            Histograms of Canvas Data"},"560":{i:0.000167889390290034,u:"../canvas_user_manual/canvas_gui_charts_pie.htm",a:"\n            Pie Charts of Canvas Data Pie charts also show the values of a single variable. You can choose the property that is represented from the Property menu. The variable can be numeric, and the sectors defined in terms of ranges of the variable; or it can be categorical, and the sectors ...",t:"\n            Pie Charts of Canvas Data"},"561":{i:0.000228037011706277,u:"../canvas_user_manual/canvas_gui_charts_settings.htm",a:"The common settings for each chart are made in the General dialog box, which you open by choosing Settings → General in the chart panel. You can set the title of the chart and change its font size and color, and set the background and foreground colors for the chart.  For scatter plots and ...",t:"Canvas Chart Settings"},"562":{i:0.000148905058650591,u:"../canvas_user_manual/canvas_gui_charts_images.htm",a:"Saving Images of Canvas Charts You can save an image of any chart by choosing File → Save Image in the chart panel. A file selector opens, in which you can navigate to a location, name the image, and choose the image format. The available formats are PNG, TIFF, and JPEG. ",t:"Saving Images of Canvas Charts"},"563":{i:0.00013508455405662,u:"../canvas_user_manual/canvas_gui_statistics.htm",a:"You can compute various univariate and bivariate statistics for properties in the Statistics dialog box, which you open by choosing Data → Statistics. See  Univariate Statistics Definitions  and  Bivariate Statistics: Covariance and Correlation  for definitions. The statistics are calculated for the ...",t:"Calculating Statistics from Canvas Data"},"564":{i:0.00011140832781328,u:"../canvas_user_manual/canvas_gui_properties.htm",a:"Canvas provides a calculator that can be used to combine properties by constructing a formula. The calculator provides mathematical functions as well as basic arithmetical operations. It is designed to generate entire new properties from existing properties by operating on property columns, rather ...",t:"Calculating New Properties from Canvas Data"},"565":{i:0.000127605457236185,u:"../canvas_user_manual/canvas_gui_pymol.htm",a:"The Canvas spreadsheet only displays structures in 2D. To view them in 3D, you would normally have to export them and then import them into a 3D viewer (like Maestro). Canvas provides an interface to PyMOL that allows you to select structures in the Canvas spreadsheet and display them directly in ...",t:"Viewing Canvas Spreadsheet Structures in PyMOL"},"566":{i:0.000122510470768188,u:"../canvas_user_manual/canvas_gui_preferences.htm",a:"Some customization of the interface features and other settings can be made in the Preferences dialog box, which you open with File → Preferences. The panel has a list of preference categories on the left. When you select an item from this list, the preferences in that category are displayed on the ...",t:"Setting Canvas Preferences"},"567":{i:0.000122928588831206,u:"../canvas_user_manual/running_canvas.htm",a:"The Canvas interface provides access to a range of applications for molecular descriptors, fingerprints, similarity and clustering, calculating statistics, and building regression and other predictive models based on properties of the structures. The applications can only be run from the master ...",t:"Running Applications from Canvas"},"568":{i:0.000181810415957228,u:"../canvas_user_manual/running_canvas_general.htm",a:"Nearly all of the applications run on a data set that includes both structures and properties. The Data set option menu is used to choose the data set. You can choose the selected rows or the visible rows in the master view, which makes all properties in the spreadsheet available for selection. You ...",t:"General Features of Canvas Applications"},"569":{i:0.000100305991725855,u:"../canvas_user_manual/running_canvas_properties.htm",a:"Molecular properties are used as the basis of modeling applications such as QSAR or QSPR. Canvas provides a means of calculating a wide range of molecular properties and descriptors based on the 2D structure. 3D properties can be generated by other applications and imported into Canvas. With a large ...",t:"Molecular Properties in Canvas"},"570":{i:0.000108889781526278,u:"../canvas_user_manual/running_canvas_properties_calculating.htm",a:"Canvas can calculate a range of molecular properties from the 2D structures that can be used as descriptors. Some of these are simple counts of features, others are numerical properties evaluated from a set of rules. The calculation of these properties can be done in the Molecular Properties dialog ...",t:"Calculating Molecular Properties in Canvas"},"571":{i:9.00816596847439E-05,u:"../canvas_user_manual/running_canvas_properties_representative.htm",a:"Often, the set of molecular properties or descriptors that you obtain from various sources can have a high degree of correlation. For model-building, it is useful to remove the linear dependence in the properties, to speed up the calculation and to reduce numerical error. One way of doing this is to ...",t:"Selecting Representative Properties in Canvas"},"572":{i:0.000109642564647282,u:"../canvas_user_manual/running_canvas_fingerprints.htm",a:"Canvas can calculate hashed fingerprints and structural keys from the 2D structure,, which are used by other Canvas applications. Canvas can also calculate fingerprints from 3D all-atom structures, based on 3-point or 4-point pharmacophore models. Once you have a fingerprint, you can create modal ...",t:"Fingerprints"},"573":{i:0.000102962855973739,u:"../canvas_user_manual/running_canvas_fingerprints_2D.htm",a:"Binary fingerprints from the 2D structure are calculated using the Binary Fingerprints from Structures dialog box (Applications → Binary Fingerprints from Structures). You can calculate the properties for the selected rows or the visible rows, a saved view, or the data set from a previous run, and ...",t:"2D Fingerprints"},"574":{i:0.000122837450473557,u:"../canvas_user_manual/running_canvas_fingerprints_3D.htm",a:"Fingerprints based on pharmacophore models of the 3D structure are calculated using the 3D Pharmacophore Fingerprints dialog box, which you open by choosing Applications → 3D Pharmacophore Fingerprints. The fingerprint bits are assigned based on the feature type and the distances between the ...",t:"3D Fingerprints from Pharmacophores"},"575":{i:0.000114129224463846,u:"../canvas_user_manual/running_canvas_similarity.htm",a:"Molecular similarity or dissimilarity is measured by a distance between molecules in the space of a property set. The similarity or distance is defined by a metric. Similarity or distance can be used for clustering and screening of structures, or selection of diverse structures. The use of ...",t:"Similarity, Dissimilarity, and Clustering in Canvas"},"576":{i:0.000125640217663467,u:"../canvas_user_manual/running_canvas_similarity_matrix.htm",a:"Similarity and distance matrices can be computed in Canvas based on fingerprints or on molecular properties, and over a range of metrics. The computed matrices can be examined in table form, and they can be exported to a file, where they can be used by other applications. The structures for any ...",t:"Computing a Similarity or Distance Matrix in Canvas"},"577":{i:0.000112854416037572,u:"../canvas_user_manual/running_canvas_similarity_screening.htm",a:"Similarity or distance matrices computed from fingerprints can be used to screen structures by similarity to one or more reference structures. The results are returned as properties that contain the similarity to the reference structure.  To set up a screen, choose Applications → Similarity/Distance ...",t:"Screening Structures by Similarity or Distance in Canvas"},"578":{i:0.000116838633568127,u:"../canvas_user_manual/running_canvas_similarity_shape.htm",a:"If you have 3D structures, you can select structures that have a similar shape to a query molecule in the Shape Screen dialog box, which you open from the Applications menu. The screening is done with the phase_shape program, and requires a Phase license. For details on shape screening, see  ...",t:"Screening Structures by Shape in Canvas"},"579":{i:0.000110754915925735,u:"../canvas_user_manual/running_canvas_similarity_diverse.htm",a:"It is often useful to be able to select structures that are as different as possible from each other. You can do this in Canvas in the Diversity-Based Selection dialog box, which you open from the Applications menu. The analysis can be run on the selected rows of the spreadsheet or on all rows, and ...",t:"Selecting Diverse Structures in Canvas"},"580":{i:0.000115760783671239,u:"../canvas_user_manual/running_canvas_similarity_library.htm",a:"If you want to choose a set of compounds based on their dissimilarity and optionally by optimizing the values of a set of properties, you can do so with in the Hole-Filling and Library Optimization dialog box, which you open from the Applications menu. With this application you can choose a set of ...",t:"Hole-Filling and Library Optimization in Canvas"},"581":{i:0.000109050503391315,u:"../canvas_user_manual/running_canvas_similarity_comparing.htm",a:"A histogram of nearest neighbor similarities is a convenient way to assess the diversity of a single collection of compounds, or the similarity between two collections of compounds. You can create the data for this histogram in the Library Comparison dialog box, which you open from the Applications ...",t:"Comparing Libraries in Canvas"},"582":{i:0.00010881831723497,u:"../canvas_user_manual/running_canvas_similarity_clustering.htm",a:"Canvas provides three methods for clustering structures: hierarchical clustering, leader-follower clustering, and k-means clustering. Leader-follower and k-means clustering can be used on much larger data sets than hierarchical clustering. Clustering by these methods can be done in the Hierarchical ...",t:"Clustering Structures by Similarity or Distance in Canvas"},"583":{i:0.00011712879938062,u:"../canvas_user_manual/running_canvas_similarity_clustering_hierarchichal.htm",a:"The specific options for hierarchical clustering include the following: If the matrix is asymmetric, you can select Matrix is asymmetric and then choose how to treat the off-diagonal elements.  You must choose a clustering linkage method, from the nine available methods. The linkage method describes ...",t:"Hierarchical Clustering Options"},"584":{i:9.46112938569659E-05,u:"../canvas_user_manual/running_canvas_similarity_clustering_lf_kmeans.htm",a:"Information on the algorithms for k-means and leader-follower clustering is given in  \n            canvasKMeans  and  \n            canvasLC . For leader-follow clustering, you can sort the fingerprints by the count of “on” bits, in descending order. This option is only available with some of the ...",t:"Leader-Follower and K-Means Clustering Options"},"585":{i:9.81837310833601E-05,u:"../canvas_user_manual/running_canvas_model.htm",a:"Canvas provides a range of methods for building a model that can be used to predict properties. These models include the common regression models—multiple linear regression, partial least-squares regression, and principal components regression—and also Bayes classification, recursive partitioning, ...",t:"Building a Predictive Model in Canvas"},"586":{i:9.77546694517674E-05,u:"../canvas_user_manual/running_canvas_model_mlr.htm",a:"For multiple linear regression, you can build a model from the selected X variables, or you can choose the best subsets. If you select Choose best subsets, click Options to set parameters for finding the best subsets in the Multiple Linear Regression - Best Subsets Options dialog box.  The best ...",t:"Multiple Linear Regression in Canvas"},"587":{i:0.000116096581321859,u:"../canvas_user_manual/running_canvas_model_pls.htm",a:"For partial least-squares (PLS) regression, you can set the maximum number of PLS factors, and you can stop adding PLS factors when the standard deviation drops below a limit that you set. The dependent variables can be scaled automatically so that they lie on the same range. You can also set a ...",t:"Partial Least-Squares Regression in Canvas"},"588":{i:0.000159979572294803,u:"../canvas_user_manual/running_canvas_model_kpls.htm",a:"Kernel-based partial least-squares regression is an extension of partial least-squares regression that introduces some nonlinearity into the scalar products of X variables used in the regression via a “kernel”, which is some nonlinear function of these scalar products  [7] . In Canvas, the kernel is ...",t:"Kernel-Based Partial Least-Squares Regression in Canvas"},"589":{i:0.000127255836101871,u:"../canvas_user_manual/running_canvas_model_pcr.htm",a:"For principal components (PCA) regression, you can set the maximum number of principal components, and the dependent variables can be scaled automatically so that they lie on the same range. See  Principal Component Analysis Overview  for a brief description of principal components regression.",t:"Principal Components Regression in Canvas"},"590":{i:0.000150265769663661,u:"../canvas_user_manual/running_canvas_model_bayes.htm",a:"For Bayes classification, you can choose a Canvas fingerprint for the independent variables instead of selecting properties, and set options for partitioning the dependent variable. When a model is built, the tabs to the right of the structures table show the classes and the number of members of ...",t:"Bayes Classification in Canvas"},"591":{i:0.000120899263856084,u:"../canvas_user_manual/running_canvas_model_nnet.htm",a:"Neural Networks in Canvas For neural networks, you can specify the number of networks to train, the number to keep, the number of training cycles, and the fraction of the training set to use for cross-validation, selected randomly. ",t:"Neural Networks in Canvas"},"592":{i:0.000130068719111128,u:"../canvas_user_manual/running_canvas_model_rp.htm",a:"Recursive partitioning is a method like Bayes classification that can be used to build a model for the prediction of property ranges or categories, using a decision tree. You can set options for partitioning the dependent variable and for building a single tree or an ensemble model, which generates ...",t:"Recursive Partitioning in Canvas"},"593":{i:0.000121713287269888,u:"../canvas_user_manual/running_canvas_model_som.htm",a:"Kohonen self-organizing maps can be built and applied in the Self-Organizing Map panel, which you open from the Applications menu. The panel has some features in common with those of the regression and other models. The models can be built or applied for the selected rows or the visible rows, a ...",t:"Self-Organizing Maps in Canvas"},"594":{i:0.000115572682389558,u:"../canvas_user_manual/running_canvas_other.htm",a:"Other Canvas Applications The remaining applications on the Applications menu are described in the topics listed below. ",t:"Other Canvas Applications"},"595":{i:0.000116232086142187,u:"../canvas_user_manual/running_canvas_other_pca.htm",a:"Analyzing the principal components of a set of independent variables can help in the selection of a subset of variables for use in other applications. See  Principal Component Analysis Overview  for an brief description. The analysis can be done in the Principal Components Analysis dialog box, which ...",t:"Principal Components Analysis in Canvas"},"596":{i:0.000104276334243574,u:"../canvas_user_manual/running_canvas_other_mcs.htm",a:"The task of finding the maximum common substructure (MCS) of a number of structures can be run from the Maximum Common Substructure dialog box, which you open from the Applications menu. As for other applications, the structures that are processed can be the selected or the visible rows in the ...",t:"Finding the Maximum Common Substructure in Canvas"},"597":{i:0.000109127604950937,u:"../canvas_user_manual/running_canvas_other_scaffold.htm",a:"Another way of finding common structural features in a set of structures is to decompose each molecule into a set of scaffolds, then make a list for each scaffold of the structures that contain that scaffold. You can perform this scaffold decomposition in the Scaffold Decomposition panel, which you ...",t:"Scaffold Decomposition in Canvas"},"598":{i:0.000120665412147045,u:"../canvas_user_manual/running_canvas_other_rgroup.htm",a:"R-group analysis identifies the R groups that are attached to a scaffold for a set of structures, and presents analyses of properties as a function of the attachment points and the R groups at each point. This application is described in a separate document,  R-Group Analysis .",t:"R-Group Analysis in Canvas"},"599":{i:0.000127580329971854,u:"../canvas_user_manual/running_canvas_other_minimize.htm",a:"The structures that are imported into Canvas or created in Canvas do not have to be 3D structures. You can convert them into 3D structures and minimize them with the 3D Minimization dialog box, which you open from the Applications menu. You can minimize the structures in the selected rows or the ...",t:"Running a 3D Minimization from Canvas"},"600":{i:0.000143331296839195,u:"../canvas_user_manual/running_canvas_external.htm",a:"If you want to run an external application and incorporate the results into Canvas, you can do so with the External Application dialog box, which you open from the Applications menu. In common with other applications panels, you can use the selected rows or the visible rows, a saved view, or the ...",t:"Running External Applications from Canvas"},"601":{i:0.000107892969686681,u:"../canvas_user_manual/running_canvas_scripts.htm",a:"Canvas provides two ways of running Python scripts from within the GUI.  You can install your own Python scripts on the Scripts menu with Scripts → Install and manage them (delete, reload) with Scripts → Manage. Once installed, you can run them by choosing the item on the Scripts menu. In general, ...",t:"Running Python Scripts from Canvas"},"602":{i:0.000101199642931541,u:"../canvas_user_manual/using_canvas.htm",a:"Using Canvas Canvas has many possible uses. In the topics listed below, instructions are given for some of the scenarios in which Canvas can be used to solve a particular problem. ",t:"Using Canvas"},"603":{i:0.000140726127255197,u:"../canvas_user_manual/using_canvas_holefilling.htm",a:"For purchasing new compounds, it can be useful to determine which compounds in a given set (available for purchase, for example) are not represented in your existing library. You can use diversity-based selection to find these compounds and export the structures to a file as follows. It is assumed ...",t:"Selecting Compounds Not Represented in a Library with Canvas"},"604":{i:0.000128426048012872,u:"../canvas_user_manual/using_canvas_duplicate_removal.htm",a:"Choose Structure → Detect Duplicates. A new column, labeled Duplicate of is added to the spreadsheet. Choose Data → Property Filter. The Property Filter dialog box opens. Ensure that Current view is selected for Store results in. Choose Duplicate of for the property under Show rows where. Choose has ...",t:"Removing Duplicate Structures from a Canvas Project"},"605":{i:0.000128118561464152,u:"../canvas_user_manual/using_canvas_choosing_fingerprints.htm",a:"Choosing which fingerprint to use and the options to set can be difficult, as no single setting is best for all targets. Some large scale screening studies [ 5 , 6 ] reported the following findings: MOLPRINT2D, Dendritic, Radial, Triplet, and Pairwise fingerprints performed better than others. More ...",t:"Choosing a Fingerprint"},"606":{i:0.000143071833632257,u:"../canvas_user_manual/using_canvas_modal_fingerprints.htm",a:"Modal fingerprints  [6]  are fingerprints that are averaged over several structures, thus combining the information from several query molecules into a single fingerprint.  To create a modal fingerprint: Create a binary fingerprint with the type of your choice.  Select the rows in the spreadsheet ...",t:"Creating Modal Fingerprints"},"607":{i:9.1729554869168E-05,u:"../canvas_user_manual/using_canvas_tips.htm",a:"This section contains some tips for common operations. To define training and test sets before building a model: Create a partition from a property with classes for the training set and the test set, and select Assign from partition to assign the sets in the model-building panel. To copy model ...",t:"Canvas Tips"},"608":{i:0.000110403694358259,u:"../canvas_user_manual/canvas_statdefs.htm",a:"Statistical Definitions and Methods The topics listed below define statistical quantities, algorithms, and regression methods used in Canvas.",t:"Statistical Definitions and Methods"},"609":{i:0.00017320854734822,u:"../canvas_user_manual/canvas_statdefs_univariate.htm",a:"Univariate Statistics Definitions The topics listed below define some symbols, quantities, and equations relating to the statistics of a single variable.",t:"Univariate Statistics Definitions"},"610":{i:0.000148205966848611,u:"../canvas_user_manual/canvas_statdefs_univariate_symbols.htm",a:"N—Number of data points (observations) in a sample. There is no hard limit on sample size (number of molecules) in Canvas, but for large samples (millions of molecules) practical issues such as system memory limitations may apply. xi —value of data point i in a sample of variable x x —mean of ...",t:"Symbols Used for Univariate Statistics"},"611":{i:0.00017788336220126,u:"../canvas_user_manual/canvas_statdefs_univariate_mean.htm",a:"These statistics describe the “central tendency” of a variable. Mean The mean of variable x is defined by  . Median For an even number of data points, the median is the mean of the middle-most two values in the ordered sample.  For an odd number of data points, the median is the middle-most value in ...",t:"Mean, Median, and Mode"},"612":{i:0.000172987238661529,u:"../canvas_user_manual/canvas_statdefs_univariate_variance.htm",a:"The statistical quantities defined in this section are measures of the spread of values in a sample about the mean. Variance The variance is defined as: If x is known or if one is examining a complete population, the N−1 term reverts to N. Canvas calculates all variances assuming a sample, i.e. ...",t:"Variance and Deviation"},"613":{i:0.00013092650115553,u:"../canvas_user_manual/canvas_statdefs_univariate_skewness.htm",a:"These statistics are measures of the extent to which a sample differs from a normal distribution. Both have a value of zero for a normal distribution. Skewness  Canvas calculates the Fisher Skewness for a sample. Normal distributions have a skewness of zero as they are perfectly symmetrical about ...",t:"Skewness and Kurtosis"},"614":{i:0.000193161833407891,u:"../canvas_user_manual/canvas_statdefs_bivariate.htm",a:"The statistics in this section describe the relationship between two variables in terms of covariance and correlation. These statistics are also applied to pairs of variables in models with multiple independent variables. Correlation coefficient See Pearson r or r-squared Correlation matrix The ...",t:"Bivariate Statistics: Covariance and Correlation"},"615":{i:0.000137406740210188,u:"../canvas_user_manual/canvas_statdefs_model.htm",a:"Definitions for Model-Building Methods The topics listed below define terms and methods used in building QSAR/QSPR models. They briefly introduces three regression methods available for model-building: partial least squares, principal component analysis, and multiple linear regression. ",t:"Definitions for Model-Building Methods"},"616":{i:0.000152610922043737,u:"../canvas_user_manual/canvas_statdefs_model_variables.htm",a:"Dependent variable The dependent variable (or response variable) is the variable that is being fitted to in a regression model. It is referred to as dependent as it is assumed that its values are dependent on the values of independent variables that will be used to generate the predictive model. ...",t:"Independent and Dependent Variables—Definitions"},"617":{i:0.000184760326031332,u:"../canvas_user_manual/canvas_statdefs_model_pls.htm",a:"The partial least squares (PLS) method generates linear equations that describe the relationship between a number of factors derived from a set of independent descriptors and a dependent descriptor. The PLS procedure works by extracting successive linear combinations of the factors (also called ...",t:"Partial Least Squares Overview"},"618":{i:0.000213360955805832,u:"../canvas_user_manual/canvas_statdefs_model_pca.htm",a:"Principal component analysis (PCA) transforms a number of independent variables into a number of uncorrelated factors that explain the variance of the dependent variable. The first factor accounts for as much of the variability in the data as possible, and each succeeding factor accounts for as much ...",t:"Principal Component Analysis Overview"},"619":{i:0.000186627102947009,u:"../canvas_user_manual/canvas_statdefs_model_mlr.htm",a:"Multiple linear regression (MLR) generates linear equations that describe the relationship between a set of independent descriptors and a dependent descriptor. As used in Canvas, MLR fits a straight line to the dependent descriptor using the following linear relationship: In the above equation, Pj ...",t:"Multiple Linear Regression Overview"},"620":{i:0.000155541555958211,u:"../canvas_user_manual/canvas_statdefs_validation.htm",a:"The statistics in this section are used to analyze and validate QSAR/QSPR models built using regression techniques. Cross-validation or  leave-n-out validation Cross-validation tests how dependent a generated regression is on the samples used to generate the regression. In leave-group-out (LGO) or ...",t:"Model Analysis and Validation Methods and Statistics"},"621":{i:0.000145883091388105,u:"../canvas_user_manual/canvas_statdefs_outliers.htm",a:"Outlier Detection Methods The topics listed below describe two methods of detecting outliers. ",t:"Outlier Detection Methods"},"622":{i:0.000142882098087992,u:"../canvas_user_manual/canvas_statdefs_outliers_loci.htm",a:"\n            Local Correlation Integral Outlier (LOCI) Detection\n The LOCI outlier detection methodology uses a density-based approach to identifying outliers within a sample. It works by comparing the density of points surrounding a given point with the densities of the surrounding neighbor points. ...",t:"\n            Local Correlation Integral Outlier (LOCI) Detection\n"},"623":{i:0.000141949693048041,u:"../canvas_user_manual/canvas_statdefs_outliers_mlr.htm",a:"A second outlier detection method is used in conjunction with multiple linear regression (MLR). In this case, the model is run first and a set of five statistical tests on the final MLR model is run to identify the outliers. The five tests are: Standardized residual\n     Studentized residual\n     ...",t:"Outliers Identified in Multiple Linear Regression"},"624":{i:0.000141660196947961,u:"../canvas_user_manual/canvas_statdefs_similarity.htm",a:"Similarity Statistics The statistics defined in the topics listed below are measures of similarity.",t:"Similarity Statistics"},"625":{i:0.000141570994385947,u:"../canvas_user_manual/canvas_statdefs_similarity_atom_pair.htm",a:"mAB is the total number of unique atom pair types found on molecules A and B  is the number of times atom pair type k was found on molecule A wi is the weight for atom pair type k To calculate the atom-pair similarity of two molecules, a set of atom-pair types is developed for each molecule. The ...",t:"Atom-Pair Similarity"},"626":{i:0.000141545714615284,u:"../canvas_user_manual/canvas_statdefs_similarity_descriptor.htm",a:"The four quantities defined here are measures of distance in descriptor space. Manhattan Distance The Manhattan distance metric, also known as the city-block distance, is a measure of the sum of geometric distances between points measured along axes at right angles. The distance being measured is ...",t:"Similarity Measures in Descriptor Space"},"627":{i:0.000425798490953923,u:"../canvas_user_manual/canvas_references.htm",a:"Leach, A. R.; Gillet V. J. An Introduction to Cheminformatics. Springer: Dordrecht, 2007. Gobbi, A.; Lee, M.-L. J. Chem. Inf. Comput. Sci. 2003, 43, 317. Lloyd, S. P. IEEE Transactions on Information Theory 1982, 28, 129. Kelley, L. A.; Gardner, S. P.; Sutcliffe, M. J. An automated approach for ...",t:"Canvas References"},"628":{i:8.59970897840058E-05,u:"../combiglide_user_manual/combiglide_intro.htm",a:"CombiGlide employs combinatorial technology for lead identification and optimization. It presents three distinct workflows: The Interactive Enumeration workflow—Interactively construct a combinatorial library by selection of fragments for each attachment point.  The Library Enumeration ...",t:"Introduction to CombiGlide"},"629":{i:8.70730619846764E-05,u:"../combiglide_user_manual/combiglide_intro_features.htm",a:"CombiGlide includes the following capabilities and features. Library enumeration: Enumerate complete combinatorial libraries.\n     Untangles and minimizes structures\n     Interactive enumeration: Define and manage collections of fragments (R groups).\n     Enumerate complete combinatorial libraries.\n ...",t:"CombiGlide Features"},"630":{i:0.000105490548250674,u:"../combiglide_user_manual/combiglide_intro_screening.htm",a:"CombiGlide can be used for virtual combinatorial screening, in which a focused combinatorial library is obtained at the end of the process. The steps in this process are summarized here. The last five steps constitute the combinatorial screening process (which is summarized with some more detail in  ...",t:"Overview of Virtual Combinatorial Screening"},"631":{i:6.87979949525788E-05,u:"../combiglide_user_manual/combiglide_intro_enumeration.htm",a:"If you already have a focused set of reagents, or if you simply want to enumerate a library, you can run a combinatorial library enumeration job. You must obtain and prepare the reagents in the same way as for library design ( Step    and  Step   ); and you must prepare the core, define the ...",t:"Overview of Library Enumeration"},"632":{i:0.000120163835687303,u:"../combiglide_user_manual/combiglide_enumerate.htm",a:"CombiGlide provides tools for simple library enumeration in addition to the tools for virtual combinatorial screening; that is, for creating a library of ligands from a core structure to which side chains are added in user-specified positions. There are two tools for this task, the Interactive ...",t:"Enumerating a Combinatorial Library"},"633":{i:0.00013885830272862,u:"../combiglide_user_manual/combiglide_enumerate_interactive.htm",a:"The Interactive Enumeration panel provides tools for all the tasks necessary to set up and enumerate a combinatorial library. The panel provides the means to define the attachment points on the core-containing structure and to set up the libraries for each attachment point. For each attachment ...",t:"Interactive Enumeration"},"634":{i:0.000124517241164816,u:"../combiglide_user_manual/combiglide_enumerate_interactive_setup.htm",a:"To set up a combinatorial library, you need to choose a core-containing molecule, define its attachment points, and add a fragment collection at each attachment point. These tasks are done in the Combinatorial Library tab. You can use an existing library and modify it, or you can create a new ...",t:"Setting Up the Combinatorial Library"},"635":{i:0.000112215796095556,u:"../combiglide_user_manual/combiglide_enumerate_interactive_collection.htm",a:"The Fragment Collection tab provides the tools to design your own fragment collection. You can import fragments, create fragments, delete fragments, and export fragments. You can either set up a fragment collection independent of a core position, or you can set up a fragment collection for a ...",t:"Setting Up a Fragment Collection"},"636":{i:0.000114956474661062,u:"../combiglide_user_manual/combiglide_enumerate_interactive_create.htm",a:"If you want to create a new fragment collection by selecting from a set of structures, you can do so in the Create Fragment Collection panel. In this panel, you can import or draw structures, select the ones that you want, define their attachment bonds, and add them to the current fragment ...",t:"Creating a Fragment Collection"},"637":{i:0.000105116277652624,u:"../combiglide_user_manual/combiglide_enumerate_interactive_library.htm",a:"When you have defined a combinatorial library, you can then proceed to enumerate it. When you click Enumerate, the Start dialog box opens, and you can set job options and start the enumeration job.  The results can be appended to the Project Table or written to the current directory. The default is ...",t:"Enumerating the Library"},"638":{i:9.11096107168816E-05,u:"../combiglide_user_manual/combiglide_enumerate_library.htm",a:"The combinatorial library enumeration is carried out by substitution of side chains on the core structure with fragments from the reagent structures. For both the core and the reagents, the bonds to be broken must be designated. The fragments from the reagents are then attached to the core at the ...",t:"Combinatorial Library Enumeration"},"639":{i:0.000214877122339494,u:"../combiglide_user_manual/combi_chemistry.htm",a:"The first step in the design of a focused library using CombiGlide is to identify the chemistry associated with desired library. Most likely, this will be done in collaboration with one of your medicinal or combinatorial chemists.  For example, if you were interested in generating a focused pyrazole ...",t:"Combinatorial Screening: Defining the Chemistry"},"640":{i:0.000148530373620272,u:"../combiglide_user_manual/combi_chemistry_structures.htm",a:"Once the synthetic approach has been determined, it will be apparent which types of reagents will be required for the library synthesis. For example, in the pyrazole library described in  Figure 1 in Combinatorial Screening: Defining the Chemistry , the library will be generated by the combination ...",t:"Obtaining Reagent and Protein Structure Files"},"641":{i:9.28897263121528E-05,u:"../combiglide_user_manual/combi_chemistry_core.htm",a:"CombiGlide works by building side chains onto defined positions on a core-containing structure. You must therefore supply a structure that contains the core in addition to supplying the reagents. The reagent structures are used as a source of side chains. Even though in the actual chemical ...",t:"Choosing the Core-Containing Molecule"},"642":{i:0.000144673161780043,u:"../combiglide_user_manual/combi_reagent_prep.htm",a:"When you have decided on the biological target and the synthetic route to the library you want to generate, and obtained structure files for the reagents and the receptor, you can proceed to the preparation of the structures for CombiGlide. Both the protein structure and the reagent structures must ...",t:"Combinatorial Screening: Preparing Structures"},"643":{i:0.000150124268908317,u:"../combiglide_user_manual/combi_reagent_prep_grid.htm",a:"A prepared protein structure is required for CombiGlide docking. Protein preparation can include simplifying multimeric complexes, deleting unwanted waters or cofactors, correcting serious errors in the protein structure, adjusting bond orders, ionization states, and formal charges of the protein, ...",t:"Protein Preparation and Grid Generation for Combinatorial Screening"},"644":{i:0.000160639079025341,u:"../combiglide_user_manual/combi_reagent_prep_running.htm",a:"\n            Reagent preparation ensures that the input structures are all-atom, 3D structures, and that they have the appropriate information stored with them to construct the molecules that are docked or used for library enumeration. The main tasks in the reagent preparation process are to select ...",t:"Reagent Preparation"},"645":{i:0.000133221047221477,u:"../combiglide_user_manual/combi_reagent_prep_running_source.htm",a:"The first task is to choose the source of the reagent structures for a given reagent type. Because the reagent preparation job matches the selected functional group in each structure to a SMARTS pattern, structures that do not match are filtered out. The structures that match are written to a file ...",t:"Choosing the Source of Reagent Structures"},"646":{i:0.000262254319886675,u:"../combiglide_user_manual/combi_reagent_prep_running_type.htm",a:"Selecting the reagent type involves choosing a functional group along with the bond that is to be replaced when the reagent is used to build structures for the library. A variety of predefined functional groups with corresponding bonds to be broken are displayed as a set of buttons in the center of ...",t:"Selecting the Reagent Type"},"647":{i:0.000134004144952452,u:"../combiglide_user_manual/combi_reagent_prep_running_options.htm",a:"In addition to defining the atoms of the reagents that will become the side chains, the reagent preparation process performs a 2D-to-3D conversion, generates all reasonable ionization and tautomeric states, and expands the stereochemistry. If the structures you have are already 3D structures in the ...",t:"Setting Options for Structure Variation and Cleanup"},"648":{i:0.000132196454032831,u:"../combiglide_user_manual/combi_reagent_prep_running_job.htm",a:"When you have completed the tasks above, enter a name in the Job name text box and click Run. You can also click the Settings button and choose Job Settings, to select job options and start the job. See  Starting Jobs from Maestro  for details. The output structure file contains the prepared ...",t:"Running the Reagent Preparation Job"},"649":{i:0.000116117162302364,u:"../combiglide_user_manual/combi_reagent_prep_running_summary.htm",a:"Task Summary To set up and run a reagent preparation job: Choose the source of reagent structures Select the functional group Select 2D-to-3D conversion and structure variation (LigPrep) options Click Run.",t:"Task Summary"},"650":{i:9.42267378226929E-05,u:"../combiglide_user_manual/combi_reagent_prep_core.htm",a:"The structures that you select for the core-containing molecules must be all-atom, minimized, 3D structures. If the structure you have does not meet these criteria, you can prepare it using LigPrep. For more information, see the  LigPrep User Manual . Note that only the ionization state and tautomer ...",t:"Preparing the Core-Containing Molecules"},"651":{i:9.05741538978076E-05,u:"../combiglide_user_manual/comb_screening.htm",a:"The goal of combinatorial screening is to create a small combinatorial library of structures that are likely to have high activity towards to a selected target. The library is screened, first by docking to the target receptor and eliminating structures that do not dock well, then by analyzing the ...",t:"Combinatorial Screening: The Process"},"652":{i:0.000153707153031121,u:"../combiglide_user_manual/comb_screening_panel.htm",a:"The Combinatorial Screening panel is designed like a wizard, with five steps. To perform a combinatorial screening, you proceed through each step in turn. You do not have to complete all steps in a single session. The design allows you to exit at an intermediate stage and pick up the process again ...",t:"The Combinatorial Screening Panel"},"653":{i:0.000113324274634826,u:"../combiglide_user_manual/comb_screening_process.htm",a:"The tasks involved in each step of the combinatorial screening process are summarized below. The Combinatorial Screening panel. The Define Combinations step In this step, you first select the core-containing molecule that you want to use for generation of the library. The library is generated by ...",t:"The Combinatorial Screening Process"},"654":{i:0.000124388576928533,u:"../combiglide_user_manual/define_combinations.htm",a:"The combinatorial screening process begins with the definition of the core and its attachments. The core-containing structure must be a 3D, all-atom, minimized structure. Likewise, the reagents used for the attachments must be 3D, all-atom, minimized structures, and also include information that ...",t:"Combinatorial Screening: Defining the Core and Its Attachments"},"655":{i:0.000170481012113944,u:"../combiglide_user_manual/define_combinations_core.htm",a:"The first task in this step is to select the core-containing molecule. When you first enter the Define Combinations step, Pick molecule is selected in the Core structure section, and most other controls are unavailable (see  Figure 1 ).  To select the core-containing molecule: Ensure that the ...",t:"Selecting the Core-Containing Molecule for Combinatorial Screening"},"656":{i:0.000177315833206164,u:"../combiglide_user_manual/define_combinations_attachments.htm",a:"Once you have a core-containing structure, you can define the attachments. An attachment is defined as the combination of an attachment position and a file that contains the reagents that will be attached at this position. The attachment position is defined by the atom in the core to which a side ...",t:"Defining Attachments for Combinatorial Screening"},"657":{i:9.9006966944072E-05,u:"../combiglide_user_manual/define_combinations_library.htm",a:"The combination of a core structure and its attachments defines a combinatorial library. You can save this definition for future use, by clicking Export Definition. A file browser opens, in which you can navigate to the desired location and specify a name. The library definition is saved as a ...",t:"Saving and Loading Library Definitions for Combinatorial Screening"},"658":{i:9.6153162183904E-05,u:"../combiglide_user_manual/combi_setup_docking.htm",a:"The center of the combinatorial screening process is the docking of the library members using CombiGlide XP docking. After you have defined the library, the next task is to set up the parameters and options for docking.  The docking of the library is performed in three stages: Core docking. In this ...",t:"Combinatorial Screening: Setting Up for Docking"},"659":{i:0.000128724680895098,u:"../combiglide_user_manual/combi_setup_docking_settings.htm",a:"The Glide settings are made in the Settings tab of the Configure Docking step. This tab is a combination of parts of the Settings tab and the Ligand tab from the Glide Ligand Docking panel. Because CombiGlide uses Glide XP docking and the ligands are predefined, only the relevant options from these ...",t:"Making Glide Settings for Combinatorial Screening"},"660":{i:0.000124853251250125,u:"../combiglide_user_manual/combi_setup_docking_constraints.htm",a:"Constraints can also be used with CombiGlide and are specified in the same manner as with Glide. With CombiGlide, constraints are only turned on during the combinatorial docking stage, i.e. docking of the fully substituted structures. This prevents the docking algorithm from forcing the side chains ...",t:"Setting Glide Constraints"},"661":{i:0.000127662411843748,u:"../combiglide_user_manual/combi_setup_docking_core.htm",a:"The final setup task is to determine how the core poses will be generated in the first stage of the docking step. The core poses can be generated with any molecule that includes the core, not just the structure you selected to define the core in the Define Combinations step. This task is carried out ...",t:"Defining the Core Poses"},"662":{i:0.000140480113281908,u:"../combiglide_user_manual/combi_setup_docking_core_mode.htm",a:"The first stage in the docking step is to dock the core-containing molecules. This stage does not apply any Glide constraints that you have set. If the molecules used to define the core poses are smaller than the original core-containing molecule, they might bind to locations in the active site that ...",t:"Selecting a Docking Mode"},"663":{i:0.000101129931018897,u:"../combiglide_user_manual/combi_setup_docking_core_structures.htm",a:"The structures that will be docked to define the core poses are listed in the Core Structures table. You can add structures to the table with the Add buttons below the table, and you can delete structures from the table by selecting them and clicking Delete.  By default, only the user-capped core is ...",t:"Specifying Structures for the Core Poses"},"664":{i:9.83485342209803E-05,u:"../combiglide_user_manual/combi_dock_library.htm",a:"The docking step is the heart of the combinatorial screening process, where the docking jobs are run to screen out molecules that do not dock well and thereby eliminate reagents. This step is carried out in the Dock Library step of the Combinatorial Screening panel. First, the core-containing ...",t:"Combinatorial Screening: Docking the Library"},"665":{i:0.000139055241629231,u:"../combiglide_user_manual/combi_dock_library_jobs.htm",a:"The docking jobs are set up and run from the Dock dialog box, which opens when you click Dock. This dialog box offers choices for the docking task and options for each of these tasks. The Dock dialog box. If you want to continue on in the workflow to analyze the docking results and select the ...",t:"Running the Combinatorial Docking Jobs"},"666":{i:0.00012680594098325,u:"../combiglide_user_manual/combi_dock_library_results.htm",a:"Statistics on the reagents that survive the various docking stages are added to the Reagent retention table when the docking jobs are finished. The table is noneditable, and its columns are described in  Table 1 . The number of structures actually generated is reported in the Combinatorial results ...",t:"Combinatorial Docking Results"},"667":{i:0.000141331756284182,u:"../combiglide_user_manual/combi_focus_library.htm",a:"The final step in the combinatorial screening process is to focus the library to a small number of compounds generated from a few reagents at each position. This focused library should contain a large proportion of compounds that are likely to bind well. The focusing is carried out by applying ...",t:"Combinatorial Screening: Focusing the Library"},"668":{i:0.000206161379395976,u:"../combiglide_user_manual/combi_focus_library_strategies.htm",a:"CombiGlide has three automated selection strategies, which can be supplemented with manual selection. The automated strategies use the GlideScore to select the reagents that produce the greatest number of structures that have a good GlideScore. The selection strategies work by filling lists of ...",t:"Selection Strategies for Combinatorial Library Focusing"},"669":{i:0.000144540733533823,u:"../combiglide_user_manual/combi_focus_library_manual.htm",a:"You can manually select reagents that must be included in the library or that must be excluded from the library in the Include/Exclude tab of the Filter and Select dialog box. This allows you to enforce the selection of reagents that did not score well, or exclude reagents that did score well, based ...",t:"Manual Selection of Reagents to Focus the Combinatorial Library"},"670":{i:0.000139064635087481,u:"../combiglide_user_manual/combi_focus_library_filtering.htm",a:"In addition to selecting reagents with one of the selection strategies, you can filter the combinatorial results based on various predicted ADME properties of the molecules. These properties are generated by QikProp. Several filters are provided, and you can customize these filters by including or ...",t:"Filtering the Focused Combinatorial Library"},"671":{i:0.000173091818721715,u:"../combiglide_user_manual/combi_focus_library_results.htm",a:"The selection and filtering job is run when you click OK or Apply in the Filter and Select dialog box. After the job finishes, the results of the selection and filtering process are displayed in the table in the upper part of the panel. The results—the current filter and selection settings and the ...",t:"Results of Selection and Filtering of the Combinatorial Library"},"672":{i:0.000125566721753333,u:"../combiglide_user_manual/combi_focus_library_compare.htm",a:"In the process of selecting the optimal library, you may want to compare several library selections. To do so, these definitions must be saved in the run, using the Save button. To compare the saved library selections, click Compare, then in the Compare Results dialog box, select the library ...",t:"Comparing Results of Combinatorial Library Focusing"},"673":{i:9.69043358211441E-05,u:"../combiglide_user_manual/combi_focus_library_create.htm",a:"When you have decided on the library selection, you can proceed to create the library. The library is a Maestro file that contains structures generated from the reagent variants, subject to a GlideScore threshold. A reagent variant is discarded if the GlideScore of the best-scoring molecule to which ...",t:"Creating and Docking the Focused Combinatorial Library"},"674":{i:0.000111089605975782,u:"../combiglide_user_manual/combi_analyze_library.htm",a:"When you have selected a small library of compounds that is likely to bind well, you may want more information about the library than is given by the GlideScore or the filtering. For example, the full library might include known actives, and you want to know how well the focused library recovers ...",t:"Combinatorial Screening: Analyzing the Library"},"675":{i:0.000161341822530785,u:"../combiglide_user_manual/combi_analyze_library_chemfeatures.htm",a:"Analysis of Chemical Features from Combinatorial Screening . When selecting a small, high-ranking combinatorial library, you may be more interested in trends than in specific reagents. For example, you might like to know if hydrogen-bond donors are more likely than average to occur at a particular ...",t:"Chemical Feature Analysis of a Combinatorial Library"},"676":{i:0.000186347703566661,u:"../combiglide_user_manual/combi_analyze_library_chemfeatures_types.htm",a:"\n            Chemical features can be classified into one of the following categories: A—hydrogen bond acceptor\n     D—hydrogen bond donor\n     H—hydrophobe\n     N—negatively charged center\n     P—positively charged center\n     R—aromatic\n     These categories reflect standard pharmacophore types, ...",t:"Features and Types in Library Analysis"},"677":{i:0.000157334428703584,u:"../combiglide_user_manual/combi_analyze_library_chemfeatures_display.htm",a:"Three analyses are presented in the Chemical Features panel, which can be selected with the options at the top of the panel: Side Chain Features Overall Type Selectivity Type Preferences These analyses are described in the following subsections. You can save the figures that are displayed in the ...",t:"Displaying Feature and Type Analyses of a Combinatorial Library"},"678":{i:0.000152489923312412,u:"../combiglide_user_manual/combi_analyze_library_chemfeatures_display_sidechain.htm",a:"The initial and final feature frequencies are displayed side by side. Error bars show the standard error, based on counting statistics. When the two frequencies differ by more than one standard error, the difference is significant. The results shown below indicate strong enrichment of acceptors, ...",t:"Side Chain Features in Library Analysis"},"679":{i:0.000219206927484062,u:"../combiglide_user_manual/combi_analyze_library_chemfeatures_display_selectivity.htm",a:"Entropy-based statistics can be used for fractional compositions that sum to unity, such as for the type data. We use a measure based on the Kullback-Liebler divergence to compute the selectivity of each position on the core. If a position has a selectivity of N, it is similar to saying that only ...",t:"Type Selectivity in Library Analysis"},"680":{i:0.000144350738713508,u:"../combiglide_user_manual/combi_analyze_library_chemfeatures_display_typeprefs.htm",a:"This chart visualizes how the distribution of chemical types changes between the initial and final collections. The upper bar in the display represents the initial side-chain collection and the lower bar represents the final side-chain collection. The width is normalized to 100% in both cases. Each ...",t:"Type Preferences in Library Analysis"},"681":{i:0.000128609944540224,u:"../combiglide_user_manual/combi_analyze_library_chemfeatures_using.htm",a:"The results of a chemical feature analysis can be used in library refinement. Based on results with a diverse side-chain collection, a second-generation library can be designed that is enriched in the types and features shown to dominate at the various positions. Positions that exhibit low ...",t:"Using Chemical Feature Analysis Results"},"682":{i:0.000124149697402779,u:"../combiglide_user_manual/combi_analyze_library_actives.htm",a:"Enrichment of Actives from Combinatorial Screening . If you are interested in how well the focused library picks up known actives, you can use the active compound analysis capabilities to generate enrichment data and an enrichment plot. The analysis is done when you run a selection job from the ...",t:"Active Compound Analysis of a Combinatorial Library"},"683":{i:0.000212811487645814,u:"../combiglide_user_manual/combi_functional_groups.htm",a:"This appendix describes the procedures for adding custom functional groups or modifying existing functional groups.  The definitions of the functional groups are stored in two files in the CombiGlide distribution, reagentprep.ini, which is a plain text file, and reagentprep.mae, which is a structure ...",t:"Creating Custom Functional Groups and Custom Minimal Capping Groups"},"684":{i:6.13254596701412E-05,u:"../combiglide_user_manual/combiglide_glossary.htm",a:"Attachment—Combination of an attachment position and a reagent file. Attachment position—Position on a core to which side chains are attached (synonymous with ‘site of diversity’). Chemical feature—One of six standard types of functional group: hydrogen bond acceptor (A), hydrogen bond donor (D), ...",t:"CombiGlide Glossary"},});