define({"1146":{i:7.37021065486678E-05,u:"../materials_user_manual/matsci_intro_overview_building.htm",a:"The Materials Science suite offers several specialized tools for building different types of structures: Organometallic complexes ( Building Organometallic Complexes ,  Build Single Complex Panel ,  Build Multiple Complexes Panel )\n     Structure libraries ( Building Structure Libraries by Adding to ...",t:"Building Structures with the Materials Science Suite"},"1147":{i:7.07304025429627E-05,u:"../materials_user_manual/matsci_intro_overview_analysis.htm",a:"Structure Analysis and Annotation with the Materials Science Suite Several tools that can be used for analysis and annotation of common elements in a structure or set of structures are provided in the Materials Science interface: R-group analysis—analyze the properties of a set of structures as a ...",t:"Structure Analysis with the Materials Science Suite"},"1148":{i:7.30454300860707E-05,u:"../materials_user_manual/matsci_intro_overview_optimization.htm",a:"Tools for minimization of the energy of structures with respect to their geometry (geometry optimization) are available under Tasks  → General Modeling → Minimization. These include  force-field minimization of single structures with MacroModel ( MacroModel Minimizations ) force-field minimization ...",t:"Structure Optimization in the Materials Science Suite"},"1149":{i:7.70740723957278E-05,u:"../materials_user_manual/matsci_intro_overview_reactions.htm",a:"Several tools are provided for calculating reaction energetics, reaction barriers, and reaction rates.  Transition state search—locate the transition state for a reaction ( Transition-State Optimizations ) and generate an IRC or MEP reaction path ( Intrinsic Reaction Coordinate Calculations ) ...",t:"Chemical Reaction Tools in the Materials Science Suite"},"1150":{i:7.2876717342689E-05,u:"../materials_user_manual/matsci_intro_overview_surface.htm",a:"Surface Physics and Chemistry in the Materials Science Suite For surface physics and chemistry, the following tools are available: Preparing slabs and interfaces—prepare a slab by slicing a crystal along a specified plane, or slabs from two different structures with or without a vacuum layer ( Build ...",t:"Molecular Dynamics in the Materials Science Suite"},"1151":{i:7.66044988957198E-05,u:"../materials_user_manual/matsci_intro_overview_optoelectronics.htm",a:"The tools below calculate various optoelectronics-related properties. Optoelectronic properties—calculate oxidation and reduction potentials, reorganization energies, triplet energies, and absorption spectra ( Calculating Optoelectronic Properties ) Charge transfer rate—calculate the Marcus charge ...",t:"Optoelectronics Tools in the Materials Science Suite"},"1152":{i:8.32010545008329E-05,u:"../materials_user_manual/matsci_intro_overview_md.htm",a:"Most of the molecular dynamics capabilities are described in the  Desmond User Manual . The following features for general MD simulations are specifically designed for the Materials Science Suite: Disordered System Builder — build a multicomponent system that is randomly distributed in a simulation ...",t:"Molecular Dynamics in the Materials Science Suite"},"1153":{i:6.77548672083201E-05,u:"../materials_user_manual/matsci_intro_overview_bulk.htm",a:"Bulk Properties in the Materials Science Suite Properties of bulk materials can be calculated with the following tools: Thermophysical properties—coefficient of thermal expansion and glass transition temperature ( Thermophysical Properties Panel ) Elastic constants—Tension and shear anisotropy, Lamé ...",t:"Molecular Dynamics in the Materials Science Suite"},"1154":{i:0.000101732041972348,u:"../materials_user_manual/matsci_intro_overview_vis.htm",a:"The graphical interface has a number of visualization tools for display of the results of calculations.  Plotting spectra—plot UV/Vis, infrared, or VCD spectra from Jaguar, or imported as raw data ( Plotting Spectra ,  Spectrum Plot Panel ) Plotting potential energy surfaces—plot the results of a ...",t:"Visualization Tools in the Materials Science Suite"},"1155":{i:0.000170303207447077,u:"../materials_user_manual/matsci_intro_overview_interface.htm",a:"The Maestro - Materials Science interface is a customized form of the Maestro interface that is specially designed for materials science use. It inherits most of the capabilities of the Maestro interface, and it has features of its own.  If you prefer to use the standard Maestro interface, you can ...",t:"The Maestro - Materials Science Interface"},"1156":{i:0.000242842106849125,u:"../materials_user_manual/matsci_intro_overview_periodicity.htm",a:"Workspace Tools for Periodic Structures In the Materials Science profile, Maestro has additional tools for working with periodic structures.  The Status bar is customized  to provide information on periodic structures and access to tools for display of periodic structures. Cell information. When a ...",t:"The Maestro - Materials Science Interface"},"1157":{i:0.000106131137358875,u:"../materials_user_manual/matsci_building.htm",a:"Maestro provides facilities for building small molecules either in 2D or 3D, by sketching the structure or by combining predefined and preminimized fragments. These tools are described in  Building and Adjusting Structures .  In addition to these tools, the Materials Science Suite provides tools for ...",t:"Building Structures for Materials Science"},"1158":{i:0.000158130416812351,u:"../materials_user_manual/matsci_building_complexes.htm",a:"Two tools are available for building octahedral, tetrahedral, or square planar organometallic complexes. The first is a tool to build a single organometallic complex with a single metal center, using one or more ligand types. The second is a tool to build a series of complexes from a set of ligands, ...",t:"Building Organometallic Complexes"},"1159":{i:0.000141100001392021,u:"../materials_user_manual/matsci_building_complexes_single.htm",a:"You can build a single organometallic complex with a single metal center, in octahedral, tetrahedral, or square planar geometry, in the Build Single Complex panel, which you open by choosing Tasks → Materials → Build → Single Complex in the Task Tool. There are three main steps to building an ...",t:"Building a Single Organometallic Complex"},"1160":{i:0.0001234139639783,u:"../materials_user_manual/matsci_building_complexes_single_geometry.htm",a:"The metal, coordination geometry, and other information can be set in the Complex information section.  To choose the metal, type the atomic symbol for the metal in the Central atom text box. To set the geometry, choose Octahedral, Tetrahedral, or Square planar from the Geometry option menu. To set ...",t:"Choosing the Metal and Coordination Geometry for the Organometallic Complex"},"1161":{i:0.000130043126306112,u:"../materials_user_manual/matsci_building_complexes_single_ligands.htm",a:"In this step, you decide how many different ligands you want to use, how many of each ligand to use, and select or sketch the ligands.  Each ligand is represented by a row in the Ligands section. Each row specifies the number of copies of the ligand, and has a button to assign a ligand structure to ...",t:"Choosing the Ligands for the Organometallic Complex"},"1162":{i:0.000119108168748054,u:"../materials_user_manual/matsci_building_complexes_single_arrangement.htm",a:"For octahedral complexes with three copies of a single bidentate ligand or three copies of each of two monodentate ligands, you can automatically arrange them in a facial or meridional conformation, by choosing the isomer in the Complex information section. Both isomers are chiral, but only one of ...",t:"Choosing the Ligand Arrangement for the Organometallic Complex"},"1163":{i:0.000125903955368374,u:"../materials_user_manual/matsci_building_complexes_single_saving.htm",a:"When you have finished defining the complex, click one of the action buttons to save it, either to the Maestro project with Create Project Entry or to a file, with Write File. By default, the complexes are cleaned up after building (using the same method as the Ligand Cleanup panel). You can ...",t:"Saving the Organometallic Complex"},"1164":{i:0.000145074202792378,u:"../materials_user_manual/matsci_building_complexes_single_summary.htm",a:"The procedures for building complexes and ligands are summarized here. To build a complex: Enter the atomic symbol for the metal in the Central atom text box. Choose the coordination geometry from the Geometry option menu. Select a ligand template from the Template option menu. (Optional) Modify the ...",t:"Summary of Single Organometallic Complex Building"},"1165":{i:0.000121721678939364,u:"../materials_user_manual/matsci_building_complexes_series.htm",a:"The Build Multiple Complexes panel is intended for building a series of organometallic complexes from a set of ligands that are variations on a common core. Each complex contains only one of the ligands: thus, the number of complexes generated is equal to the number of ligands that you supply. In ...",t:"Building a Series of Organometallic Complexes"},"1166":{i:0.000104348035355401,u:"../materials_user_manual/matsci_building_complexes_maestro.htm",a:"If you want to build complexes with coordination geometries other than octahedral, tetrahedral, or square planar, you can use the metal fragments and metal ligands in the 3D Builder panel, which you open with Edit → 3D Builder, or click the Build button on the toolbar. To display the fragments for ...",t:"Building Coordination Complexes with the Maestro Builder"},"1167":{i:0.000121302454973256,u:"../materials_user_manual/matsci_building_complexes_cleanup.htm",a:"After building complexes, you should clean up the structures of the complexes. You can do this in the Clean Up Complexes panel, which cleans up the structure of one or more complexes by performing a force-field minimization. The metal atom is treated as though the metal-ligand bonds are at their ...",t:"Cleaning Up the Structures of the Organometallic Complexes"},"1168":{i:0.00017887665696673,u:"../materials_user_manual/matsci_building_libraries.htm",a:"The search for new materials can involve tuning the structure of a molecule, such as a ligand in an organometallic complex, to optimize a particular property or set of properties. Computationally, this can easily be done by replacement of one or more target groups on a scaffold with sets of ...",t:"Building Structure Libraries by Adding to a Scaffold"},"1169":{i:0.000114689547743712,u:"../materials_user_manual/matsci_building_libraries_importing.htm",a:"The first step is to import a scaffold, or “core-containing molecule”, which is used to define the combinatorial library. The choices are: Maestro file—Import the core-containing molecule from a Maestro file. Click Browse to navigate to the file and select it. The first structure in the file is ...",t:"Importing a Scaffold for a Structure Library"},"1170":{i:0.000135015130620905,u:"../materials_user_manual/matsci_building_libraries_attachment.htm",a:"If you imported a combinatorial definition, or a molecule that already has attachment bonds defined, the bonds are marked by colored arrows on the 2D structure, and rows in the Combinatorial definition table are added for each attachment bond. The arrow color matches the rows in the table. The head ...",t:"Attachment Points on the Scaffold for a Structure Library"},"1171":{i:0.000124057586462274,u:"../materials_user_manual/matsci_building_libraries_existingfrag.htm",a:"To create a combinatorial library, a collection of fragments must be applied to each attachment point (core position). Collections are stored in files with a .bld extension. If you already have fragment collections saved, you can apply them with one of the following methods: Define a default ...",t:"Applying an Existing Fragment Collection to a Scaffold"},"1172":{i:0.000140388185536678,u:"../materials_user_manual/matsci_building_libraries_fragcollection.htm",a:"The Fragment Collection tab provides the tools to select and manage your own fragment collection. A fragment collection is a set of molecules, each of which has a bond marked as the attachment bond. As for the core, there is a direction associated with the attachment bond: one part of the molecule ...",t:"Setting Up Fragment Collections for Building a Structure Library"},"1173":{i:0.000139973008916196,u:"../materials_user_manual/matsci_building_libraries_newcollection.htm",a:"To create a new fragment collection by selecting from a set of structures, you can use the Create Fragment Collection panel. In this panel, you can import or draw structures, select the desired structures, define their attachment bonds, and add them to the fragment collection. To open the Create ...",t:"Creating a Fragment Collection for Building a Structure Library"},"1174":{i:0.000137178287286101,u:"../materials_user_manual/matsci_building_libraries_saving.htm",a:"When you have finished adding fragments to each attachment point on the core, you can save the library or enumerate it.  To save the library definition, click Export Library, and navigate to the desired location in the file selector that opens. The core molecule with the defined attachments and the ...",t:"Saving or Enumerating the Structure Library"},"1175":{i:0.000162723175502718,u:"../materials_user_manual/matsci_building_elements.htm",a:"If you want to create variations on a structure that involve a change of element, such as replacement of CH groups with N, you can use the Elemental Enumeration panel to perform the transmutations on the Workspace structure. All possible transmutations are performed on the selected atoms, within ...",t:"Varying the Elemental Composition in a Structure"},"1176":{i:0.000150973943716938,u:"../materials_user_manual/matsci_building_nanostructures.htm",a:"The Nanostructure Builder panel allows you to build nanotubes or nanosheets from a specified unit cell and geometry to a given size. Multi-walled nanotubes and multi-layer nanosheets can be constructed as well as single-walled nanotubes and single nanosheets, and nanotubes with a range of chiral ...",t:"Building Nanostructures"},"1177":{i:0.000144823618576735,u:"../materials_user_manual/matsci_building_nanostructures_composition.htm",a:"The first step is to define the basic building block and choose how to terminate the sheets or tubes. This task is common to both types of nanostructure. Specify the composition of the unit cell, by entering the element symbols in the Elemental composition text boxes. There are no limitations ...",t:"Defining the Composition and Bonding of the Nanostructures"},"1178":{i:0.000143711000486684,u:"../materials_user_manual/matsci_building_nanostructures_sheets.htm",a:"You can build structures that consist of a single nanosheet or of multiple bilayers. First, the size and shape of each nanosheet in the structure must be set. Choose the edge types for the two edges of the sheet, from armchair and zigzag. Set the dimensions of the sheet in terms of the unit cell ...",t:"Building Nanosheets"},"1179":{i:0.000145934599444518,u:"../materials_user_manual/matsci_building_nanostructures_tubes.htm",a:"You can build single-walled or multi-walled nanotubes, for a fixed set of chiral indices, and you can also build nanotubes with a range of chiral indices. The first stage is to set up the parameters that define the geometry and number of tubes.  Set the chiral indices (n,m) for the tube (or the ...",t:"Building Nanotubes"},"1180":{i:0.000154894762726902,u:"../materials_user_manual/matsci_building_nanoparticles.htm",a:"The Nanoparticle Builder panel allows you to build a nanoparticle of a particular shape and size from the structure in the Workspace. To open the Nanoparticle Builder panel, choose Tasks → Materials → Build → Nanoparticles in the Task Tool. Nanoparticles are built from a larger structure by imposing ...",t:"Building Nanoparticles"},"1181":{i:0.000175449540098742,u:"../materials_user_manual/matsci_building_mixture.htm",a:"If you want to create a disordered mixture of compounds that can be used for molecular dynamics simulations or other purposes, you can build the mixture in the Disordered System Builder panel, which you open by choosing Tasks → Materials → Build  → Disordered System in the Task Tool. This panel ...",t:"Building a Disordered Molecular Mixture"},"1182":{i:0.000150977709673695,u:"../materials_user_manual/matsci_building_mixture_composition.htm",a:"The first stage is to set up the mixture composition and provide the structures of each component.  Choose the organic compounds you want to use.  You can take structures from the selected entries in the Project Table, the Workspace, or a single file that contains all the compounds. The compounds ...",t:"Specifying the Disordered Mixture Composition"},"1183":{i:0.000140954707846189,u:"../materials_user_manual/matsci_building_mixture_substrate.htm",a:"To build a disordered system on a substrate, you must load the substrate and identify the type of substrate it is. You can load the substrate from a file or from the Workspace, using the Structure option menu and associated button. The file must be a Maestro file.  The type of substrate that you ...",t:"Building a Disordered Mixture on a Substrate"},"1184":{i:0.000130053203434601,u:"../materials_user_manual/matsci_building_mixture_placement.htm",a:"Specifying Placement and Distribution in the Disordered Mixture The next stage is to specify how the molecules are to be placed and distributed in the box or in relation to the substrate. The molecules are placed initially in the box, and then undergo up to three processes to produce a final ...",t:"Specifying the Placement and Distribution of the Molecules in the Disordered Mixture"},"1185":{i:0.000135469791808639,u:"../materials_user_manual/matsci_building_mixture_job.htm",a:"After the parameters are set, use the Job toolbar to run the job to create the disordered system. See  Starting Jobs from Maestro  for more information on this toolbar.  If you chose to prepare the systems for Desmond simulations, the result is one or more CMS files, which can be used directly as ...",t:"Running the Job to Create the Disordered Mixture"},"1186":{i:0.00013162061503539,u:"../materials_user_manual/matsci_building_crystal.htm",a:"The crystal structure builder allows you to build a crystal structure, either from text data, which may be taken from a publication, for example, or from an existing 3D structure. The crystal structure is stored as a project entry with crystal properties, and can be used in Schrödinger applications ...",t:"Building a Crystal Structure"},"1187":{i:0.000137642629647101,u:"../materials_user_manual/matsci_building_polymers.htm",a:"If you want to build a polymer from one or more monomer units, you can use the Polymer Builder panel. The polymer can be linear, branched, or dendritic; with different monomers it can also be a periodic or block copolymer or a random copolymer. You can specify the composition, arrangement and ...",t:"Building Polymers"},"1188":{i:0.000128877654252117,u:"../materials_user_manual/matsci_building_polymers_choosing.htm",a:"The first task in building a polymer is deciding on the groups from which the polymer is built: the initiator, the terminator, the cascader for dendritic structures, and the monomer, or monomers for a copolymer. The end groups (initiator, cascader, and terminator) can be chosen in the End groups ...",t:"Choosing Monomer and End Groups for the Polymer"},"1189":{i:0.000134354597657814,u:"../materials_user_manual/matsci_building_polymers_sketching.htm",a:"If you want to supply your own end groups or monomers, you can sketch them in 2D. Click the Sketch button for the group to open a panel in which you can sketch the structure. In addition to drawing the structure, you will also have to label groups as R groups, which you can do from the shortcut menu ...",t:"Sketching Polymer End Groups and Monomers"},"1190":{i:0.000128860458031063,u:"../materials_user_manual/matsci_building_polymers_dendrimers.htm",a:"To build a dendrimer, you must choose a cascader in the End groups section. In addition to the controls to define the cascader group, there is a Generations box, in which you can define the number of replications of the polymer chain in the dendrimer. The minimum number is 2, which means that there ...",t:"Building Dendrimers"},"1191":{i:0.000137581812975931,u:"../materials_user_manual/matsci_building_polymers_branched.htm",a:"If the monomer group you choose has an R3 group, you can use it to build a branched polymer. The Branching controls are enabled, and you can specify the probability of branching and the maximum number of generations when branching.  The branching probability is applied to each monomer unit ...",t:"Building Branched Polymers"},"1192":{i:0.000140267063329478,u:"../materials_user_manual/matsci_building_polymers_composition_random.htm",a:"In the Composition section of the Configuration tab, you specify how the monomers are arranged to form the polymer, and the number of repeat units in the polymer. The type of polymer that can be formed depends on the number of monomer units, and is reflected in the options of this section. These ...",t:"Defining the Polymer Composition"},"1193":{i:0.000141022477214941,u:"../materials_user_manual/matsci_building_polymers_configuration.htm",a:"The chain configuration can be set by specifying backbone dihedrals, and if the monomer is chiral, the tacticity. You can set the backbone dihedral angles both between and within the monomer units or specify a range of angles with the Backbone dihedral options.  Set—Select this option to set ...",t:"Setting the Polymer Chain Configuration"},"1194":{i:0.000141003195835504,u:"../materials_user_manual/matsci_building_polymers_coupling.htm",a:"If you want to allow tail-to-tail or head-to-head couplings, instead of the normal head-to-tail couplings, you can set the percentage probability that a nonstandard coupling occurs on the addition of another monomer, in the Tail-tail and Head-head boxes in the Chain growth section of the ...",t:"Allowing Nonstandard Couplings of Polymer Units"},"1195":{i:0.000140179742966598,u:"../materials_user_manual/matsci_building_polymers_seed.htm",a:"Several of the options for building polymers involve random selection. The random number generator that is used for the random selection processes is initialized by a seed, which you can specify, by selecting one of the Random seed options: Variable—Select this option to use a different seed each ...",t:"Setting the Random Number Seed for Polymer Building"},"1196":{i:0.000137292956426351,u:"../materials_user_manual/matsci_building_polymers_representation.htm",a:"You can set the color and molecular representation that are used in Maestro to display the polymer when it is built.  To set the color, choose one of the following schemes from the Color by option menu: Element—Color each atom by element. This is the standard Element color scheme, with gray carbons, ...",t:"Setting Color and Molecular Representation for Polymers"},"1197":{i:0.000127928417367601,u:"../materials_user_manual/matsci_building_polymers_title.htm",a:"Setting the Structure Title for the Polymer To specify the title for the polymer, you can edit the text in the Title text box. A default title is built up from the IUPAC name of the polymer, as far as possible. This default scheme does not work for more complicated polymers. ",t:"Setting the Structure Title for the Polymer"},"1198":{i:0.000130354106025451,u:"../materials_user_manual/matsci_building_polymers_amorphous.htm",a:"As well as building a single polymer chain or dendrimer, you can create an amorphous cubic cell consisting of multiple polymer chains. The cell may have a nonuniform composition of the polymers if you chose any random element in the generation of the polymer chain.  The cell is built by randomly ...",t:"Creating an Amorphous Cell of Polymers"},"1199":{i:0.000115691398885177,u:"../materials_user_manual/matsci_building_polymers_creating.htm",a:"When you have made all the settings you want to make, click Run. The job should take less than a minute for a polymer, and somewhat longer for an amorphous cell. The polymer or the amorphous cell (or Desmond model system) is added as an entry to the Project Table, and is displayed in the Workspace.  ...",t:"Creating the Polymer or Amorphous Cell"},"1200":{i:9.4106077699039E-05,u:"../materials_user_manual/matsci_building_polymers_summary.htm",a:"To sketch an initiator, cascader, terminator, or monomer group: Click Sketch. The Sketcher panel for the group type opens. Draw the group as it appears in the polymer using the drawing tools. This means that monomers should not have double bonds as they exist in the isolated monomer, but single ...",t:"Summary of Basic Polymer-Building Procedures"},"1201":{i:9.3259481496899E-05,u:"../materials_user_manual/matsci_reactions.htm",a:"The Materials Science suite contains a number of tools for investigating chemical reactions: reaction channel enumeration, for generating possible products from reactants; reaction path interpolation, for generating structures along an approximate reaction path by linear interpolation; reaction ...",t:"Investigating Chemical Reactions"},"1202":{i:0.000112703314523338,u:"../materials_user_manual/matsci_reactions_enumeration.htm",a:"Before proceeding with the calculation of a reaction path or reaction energetics, you might want to explore the possible reaction channels. This can be done in the Reaction Channel Enumeration panel, which you open by choosing  Tasks → Materials → Enumerate →  Reaction Channel  in the Task Tool. ...",t:"Reaction Channel Enumeration"},"1203":{i:0.000167159355260142,u:"../materials_user_manual/matsci_reactions_enumeration_specific.htm",a:"If you want to generate the reaction channels for specific reactions, i.e. breaking or forming specific bonds, you can use the tools in the Specific channels section. Each reaction channel is defined in a row of the table in this section. The table initially contains one reaction channel but with no ...",t:"Setting Up Enumeration of Specific Reaction Channels"},"1204":{i:0.000135441062971057,u:"../materials_user_manual/matsci_reactions_enumeration_random.htm",a:"Setting Up Random or Exhaustive Enumeration of Reaction Channels If you want to generate a random selection of reaction channels, or exhaustively enumerate all possible reaction channels, you can do so in the Random channels section.  All molecules in the Workspace are considered in the generation ...",t:"Setting Up Random Exhaustive Enumeration of Reaction Channels"},"1205":{i:0.000136202517321863,u:"../materials_user_manual/matsci_reactions_enumeration_output.htm",a:"The reaction products for a given channel are created by breaking bonds and forming new bonds. The product structures can be minimized with the OPLS_2005 force field by selecting Clean up geometries. The reactant structures are also minimized if you choose this option. You should ensure that the ...",t:"Post-Processing and Output of Reaction Channel Enumeration"},"1206":{i:0.000128818615218331,u:"../materials_user_manual/matsci_reactions_enumeration_summary.htm",a:"To set up an enumeration for a random selection of channels: Display the reactant molecules in the Workspace. Choose whether to allow bonds to hydrogen to be considered in the reaction channels, by selecting or deselecting Allow reactive R-H bonds. Choose whether to keep only the nonredundant ...",t:"Summary of Reaction Channel Enumeration Processes"},"1207":{i:0.000135996837153306,u:"../materials_user_manual/matsci_reactions_dissociation.htm",a:"When evaluating compounds for their reactivity or stability, it can be useful to know what the bond energies are. This information may be useful in the assessment of chemical lifetimes of optoelectronic materials, for example. Similarly, when evaluating ligands for use in transition metal complexes, ...",t:"Bond and Ligand Dissociation Energies"},"1208":{i:0.000155905612772327,u:"../materials_user_manual/matsci_reactions_interpolation.htm",a:"Locating the transition state for a reaction is a critical task in determining the kinetics of a reaction. It is also a difficult task, as it is usually necessary to have a good guess at the desired transition state. For the search, it is also necessary to maintain a proper correspondence between ...",t:"Reaction Path Interpolation"},"1209":{i:0.000130335911066789,u:"../materials_user_manual/matsci_reactions_interpolation_structures.htm",a:"In general, the reaction and product structures should be constructed so that the structures are pre-positioned for reaction (in either direction). It is highly recommended that you build the product structures by modifying the reactant structures, breaking bonds and forming new bonds as necessary ...",t:"Choosing Reactant and Product Structures for Reaction Path Interpolation"},"1210":{i:0.000126626022351011,u:"../materials_user_manual/matsci_reactions_interpolation_renumbering.htm",a:"You can renumber the atoms if the reactant and product numbering schemes do not match, in the Reorder Atoms dialog box, which you open by clicking Reorder Product Atoms. This dialog box shows 2D images of the reactant and product complexes, and allows you to visually select the corresponding atoms ...",t:"Renumbering Atoms for Reaction Path Interpolation"},"1211":{i:0.000133501587859351,u:"../materials_user_manual/matsci_reactions_interpolation_positioning.htm",a:"If you have not already positioned the reactants and products in a pre-reactive or post-reactive complex, you should do so before proceeding. Structures that are pre-positioned close to the transition state provide better reaction paths than structures that are at their equilibrium geometries and ...",t:"Positioning the Reactants and Products for Reaction Path Interpolation"},"1212":{i:0.000128101734084397,u:"../materials_user_manual/matsci_reactions_interpolation_path.htm",a:"The points along the reaction path can be chosen as an evenly-spaced set or as a list of points. The path is defined with a value of 0 for the reactants and 1 for the products. The points that you specify do not include the reactant or product structures, so for example if you choose 10 evenly ...",t:"Specifying the Reaction Path for Interpolation"},"1213":{i:0.000135544029014298,u:"../materials_user_manual/matsci_reactions_interpolation_coords.htm",a:"The choice of a coordinate system that is interpolated is critical to finding a suitable reaction path. Cartesian coordinates may be suitable for some reactions, such as the classic SN2 reaction, X− + CH3Y → CH3X + Y− (if the halides X and Y and the carbon atom remain in the same order). The ...",t:"Choosing a Coordinate System for Reaction Path Interpolation"},"1214":{i:0.000133834290144725,u:"../materials_user_manual/matsci_reactions_interpolation_optimizing.htm",a:"As a nonlinear least squares procedure is required for optimizing the path for distance and internal coordinates, some parameters can be set to control the optimization. Further, since short distances are to be avoided, a penalty function can be added for short distances.  To help improve ...",t:"Setting Parameters for Optimizing the Reaction Path"},"1215":{i:0.00012035800790827,u:"../materials_user_manual/matsci_reactions_interpolation_output.htm",a:"When the path has been generated, the output Maestro file contains all of the structures: first the reactants, then the reactant complex (if requested), the steps in the path, the product complex (if requested), then the products. The step index and distance along the path (in the range 0 to 1) are ...",t:"Reaction Path Interpolation Output"},"1216":{i:0.000124177716644414,u:"../materials_user_manual/matsci_reactions_energetics.htm",a:"When studying the effect of substituents on a reaction barrier or a reaction enthalpy, it is useful to have a semiquantitative guide to the relative effects of the substituents at an approximate level before performing full calculations. The more expensive calculations can then be focused on the ...",t:"Reaction Energetics for a Compound Series"},"1217":{i:0.000111570432801962,u:"../materials_user_manual/matsci_reactions_rate.htm",a:"The TST Rate Calculation panel allows you to calculate the rate of a reaction using transition state theory, from the output of quantum mechanical calculations on the reactants and the transition state. The calculations allow you to select the prefactor partition functions to use and to include the ...",t:"TST Rate Calculation"},"1218":{i:8.50351422845828E-05,u:"../materials_user_manual/matsci_reactions_heat.htm",a:"Heats of Formation Heats of formation and atomization energies can be calculated with the Heat of Formation panel, which you open by choosing Tasks → Quantum Mechanics → Heat of Formation in the Task Tool. This panel is described in detail in  Heat of Formation .  ",t:"Heats of Formation"},"1219":{i:8.78643860372299E-05,u:"../materials_user_manual/optoelectronics.htm",a:"Optoelectronics The Materials Science suite provides some capabilities for calculating properties that are relevant to optoelectronics, in addition to existing Jaguar features for calculating molecular properties. ",t:"Optoelectronics"},"1220":{i:0.000156104041106768,u:"../materials_user_manual/optoelectronics_properties.htm",a:"Electronic properties of molecules relevant to optoelectronics can be calculated via the Optoelectronics Calculations panel, and viewed in the Optoelectronics Results panel. The properties and the methods used for the calculations are described in detail in  Optoelectronics Properties .  To open the ...",t:"Calculating Optoelectronic Properties"},"1221":{i:0.00015247441547558,u:"../materials_user_manual/optoelectronics_properties_setup.htm",a:"To set up a calculation:  Choose the source of the structures for the current task from the Use structures from option menu.  Project Table (selected entries)—Use the entries that are currently selected in the Project Table. Workspace (included entries)—Use the entries that are currently included in ...",t:"Setting Up and Running an Optoelectronics Properties Calculation"},"1222":{i:0.00013308639903727,u:"../materials_user_manual/optoelectronics_properties_results.htm",a:"The Optoelectronics Results panel can be used to detect trends and outliers in properties calculated with the Optoelectronics Calculation panel. The panel contains a table that displays all the data, and a plot that displays user-specified properties.  The plot makes it easy to find interesting ...",t:"Viewing the Optoelectronics Results"},"1223":{i:0.000138905442303115,u:"../materials_user_manual/optoelectronics_properties_results_table.htm",a:"The properties table displays properties for the structures. By default, the entry title, the 2D structure, and the primary properties for optoelectronics calculations are shown. You can select multiple rows, to apply an action. You can also copy and paste data from the table in to another ...",t:"Optoelectronics Results: Properties Table"},"1224":{i:0.000140712109236568,u:"../materials_user_manual/optoelectronics_properties_results_plot.htm",a:"The plotting tool allows you to show a scatter plot of two properties, which you can choose from the X-Axis and Y-Axis option menus. Only numerical properties shown in the table can be selected.  You can color the plot points by a chosen property, using a blue-green-red color ramp, with blue for low ...",t:"Optoelectronics Results: Property Scatter Plot"},"1225":{i:0.000141269316930756,u:"../materials_user_manual/optoelectronics_optimization.htm",a:"The Materials Science suite provides a tool for modifying structures to optimize a selection of optoelectronic properties, by mutating structures with a genetic algorithm. The goal is to generate a diverse set of structures that have the desired properties. In this sense, it is a discovery tool ...",t:"Optimization of Optoelectronic Properties"},"1226":{i:0.000141429178959444,u:"../materials_user_manual/optoelectronics_optimization_initial.htm",a:"The first step is to set up the initial state for the evolution of the structures.  As the goal of the optimization is to find new, diverse structures, the input structures should not be closely related (as in a congeneric series, for example). You can choose the source of the structures for the ...",t:"Selecting the Initial Structures and State for Property Optimization"},"1227":{i:0.000141436164035905,u:"../materials_user_manual/optoelectronics_optimization_properties.htm",a:"You can optimize several properties simultaneously, if you wish, set targets for the properties, and set weights in the optimization for each property. Each property to be optimized is specified by a row in the Properties table. You can choose the property to be optimized from the option menu in the ...",t:"Selecting the Properties to Optimize"},"1228":{i:0.000141300969568756,u:"../materials_user_manual/optoelectronics_optimization_selection.htm",a:"Several controls are provided on how to individuals are selected as parents for the next generation. The selection may be based on the scoring function for the fitness of the individuals. To specify the number of individuals to carry forward to the next generation without modification, set the ...",t:"Setting Up the Selection Process for Property Optimization"},"1229":{i:0.000140816894116388,u:"../materials_user_manual/optoelectronics_optimization_mutations.htm",a:"The types of mutations that are made to produce new structures can be specified in the Genetic diversity section. You can choose one or more of the following mutation types: Bond crossover—Swap fragments from two structures. An acyclic single bond that does not involve hydrogen is chosen at random ...",t:"Setting Up the Mutations for Property Optimization"},"1230":{i:0.000139243765572732,u:"../materials_user_manual/optoelectronics_optimization_termination.htm",a:"You can terminate the optimization in several ways. To limit the length of the run, you can specify the maximum number of generations in the Maximum generations box, after which the optimization stops. Otherwise, you can terminate the optimization early by selecting options for conditions on the ...",t:"Setting Termination Criteria for Property Optimization"},"1231":{i:0.000134176067480241,u:"../materials_user_manual/optoelectronics_optimization_job.htm",a:"The optimization involves running a set of Jaguar jobs on each structure in each generation. You can set keywords for the Jaguar calculations in the Jaguar keywords box, if you want to use something other than the defaults. A geometry optimization is performed on each structure, so you may want to ...",t:"Setting Job Options and Starting the Property Optimization Job"},"1232":{i:0.000117864308006629,u:"../materials_user_manual/optoelectronics_optimization_results.htm",a:"As the jobs can take a long time, it may be useful to examine the results as it is running, for example to assess the progress or success of the optimization, or to make use of structures that meet some criteria before the job finishes. You can do this in the Optoelectronics Genetic Optimization ...",t:"Monitoring and Examining the Property Optimization Results"},"1233":{i:0.000130727678214144,u:"../materials_user_manual/optoelectronics_charge_transfer.htm",a:"The Electron Coupling panel can be used to calculate carrier hopping rates in an amorphous structure. It is mainly designed for the results of an MD simulation with a homogeneous system, but it can be used for any structure consisting of multiple molecules. Marcus theory is used for the hopping ...",t:"Charge Transfer Rates"},"1234":{i:0.000104093034131106,u:"../materials_user_manual/optoelectronics_spin_states.htm",a:"The location of different spin states is important when searching for spin-forbidden transitions, for example in transition metal complexes. The Spin States panel enables you to generate spin states for transition metal complexes. It is primarily intended for complexes with a single metal, where the ...",t:"Locating Spin States of Metal Complexes"},"1235":{i:0.000114098896741399,u:"../materials_user_manual/optoelectronics_spin_states_prep.htm",a:"Tools for building complexes are described in  Building Organometallic Complexes . For mononuclear transition metal complexes, you must assign formal charges to the metal center and the ligands so that the ligands are represented as closed-shell molecules. Assigning atomic formal charges helps ...",t:"Preparing Complexes for Spin State Calculations"},"1236":{i:0.000109544716191109,u:"../materials_user_manual/optoelectronics_spin_states_setup.htm",a:"The Spin States panel offers a selection of the most important settings available in the Jaguar panel, but you can add settings if you need to.  Use structures from—Choose the source of the structures for the current task. Project Table (selected entries)—Use the entries that are currently selected ...",t:"Setting Up the Spin States Calculation"},"1237":{i:8.46046541510951E-05,u:"../materials_user_manual/optoelectronics_spin_states_job.htm",a:"After making settings, you can run the job. To make settings for the host and number of CPUs, and how subjobs are distributed, click the Settings button and make your choices in the Job Settings dialog box, then click Run to run the job.  If you are running calculations on more than one complex, the ...",t:"Running the Spin States Job"},"1238":{i:0.000103081776837673,u:"../materials_user_manual/matsci_md.htm",a:"Molecular Dynamics Tools in the Materials Science Suite The Materials Science suite provides access to the Desmond molecular dynamics program. The Desmond panels in the standard Maestro are mostly designed for general applications and for protein modeling. The Materials Science suite provides some ...",t:"Molecular Dynamics in the Materials Science Suite"},"1239":{i:0.000166236355971969,u:"../materials_user_manual/matsci_md_multistage.htm",a:"The molecular dynamics panels that are provided in the standard Maestro interface as well as the Materials Science interface are suitable for the most common workflows, and include a stage for relaxing the model system before performing the main simulation.  Sometimes more complex workflows are ...",t:"Multistage Simulation Workflow"},"1240":{i:9.66523692994944E-05,u:"../materials_user_manual/matsci_md_analysis.htm",a:"Information on bulk properties can be extracted from an analysis of an MD trajectory. The MS MD Trajectory Analysis panel analyzes a trajectory from a Desmond MD simulation and presents information about bulk properties derived from the simulation, in graphical form. The properties displayed are ...",t:"Simulation Analysis"},"1241":{i:9.05925835502992E-05,u:"../materials_user_manual/coarse_grain.htm",a:"Coarse-Grained Modeling in the Materials Science Suite The Material Science Coarse-Grained Modeling facility is intended for molecular dynamics simulations of larger bulk systems over more extended period of time than all-atom (AA) models. It achieves this goal by mapping multiple atoms onto single ...",t:"Molecular Dynamics in the Materials Science Suite"},"1242":{i:0.000107467230096775,u:"../materials_user_manual/coarse_grain_potentials.htm",a:"  Two general types of non-bonded interactions are supported for coarse-grained modeling. You can only use one of these types of potentials in a simulation: Lennard-Jones potentials in combination with Coulombic potentials (LJC). A variety of Lennard-Jones potentials beyond the usual 12-6 potentials ...",t:"Potentials and Simulation Types for Coarse-Grained Modeling"},"1243":{i:0.0001078312193187,u:"../materials_user_manual/coarse_grain_workflow.htm",a:"The general workflow for a coarse-grained modeling project involves several steps.  For some steps, the detailed workflow may depend on the model type or the potentials used. Links to the topics that describe the panels used or details of specific tasks are included in the text.  The following ...",t:"Coarse-Grained Modeling Workflow"},"1244":{i:0.000194898171633305,u:"../materials_user_manual/coarse_grain_disordered_system.htm",a:"Disordered molecular systems can be assembled for coarse-grained modeling in the  Disordered System Builder Panel  from a set of source all-atom molecules or a set of source molecules constructed from coarse-grained particles.  If you want to build the system from all-atom molecules, you can use the ...",t:"Constructing Disordered Systems for Coarse-Grained Modeling"},"1245":{i:0.000112652174307639,u:"../materials_user_manual/coarse_grain_rh_volume_adjust.htm",a:"Repulsive harmonic (RH) systems will nearly always have a volume that is different from the desired one when they are built. A tool has been provided , scale_simulation_box.py, to adjust the system volume by scaling the box size and site coordinates uniformly. This scaling should be done after the ...",t:"Adjusting the Volume for a Repulsive Harmonic System"},"1246":{i:0.000141311199021758,u:"../materials_user_manual/coarse_grain_simulation_setup.htm",a:"Coarse-grained simulations should be set up in the  MD Multistage Workflow Panel . When a coarse-grained system is loaded into this panel the relaxation protocol is disabled to ensure that stages are defined appropriately for coarse-grained simulations.  You can define multiple stages, optionally ...",t:"Setting Up the Coarse-Grained Simulation"},"1247":{i:0.000198301072532693,u:"../materials_user_manual/matsci_probe_scan.htm",a:"Finding out how one molecule interacts with another is an important task in many areas of materials science, from determining low-energy adsorption sites of a molecule on some substrate, which might be another molecule, a nanoparticle, a catalytic oxide, to finding the pre-positioned reactant and ...",t:"Scanning Probe-Target Interactions"},"1248":{i:0.000189974837382398,u:"../materials_user_manual/matsci_probe_scan_target.htm",a:"The target is specified at the top of the panel, by choosing a source from the Target structure option menu Workspace (included entry)—Use the entry that is currently included in the Workspace. Only one entry must be included in the Workspace. File—Use the (first) structure from the specified file. ...",t:"Specifying the Target for a Probe Scan"},"1249":{i:0.000156582055324094,u:"../materials_user_manual/matsci_probe_scan_probe.htm",a:"There are three choices for specifying the probe to use: Point charge—Use a point charge as the probe, and specify the charge value in the Charge text box. Single atom—Use a single atom as the probe, and specify the atomic symbol in the Atomic symbol text box. Molecule—Use the molecule that is in ...",t:"Specifying the Probe for a Probe Scan"},"1250":{i:0.000146207552012994,u:"../materials_user_manual/matsci_probe_scan_system.htm",a:"With the probe-target system defined, you will need to specify the charge and spin of the system, and set any other parameters you want to use for the QM calculation with Jaguar. Charge—Specify the total charge on the probe-target system in this box. If you chose a point charge or a charged atom or ...",t:"Setting System Parameters for a Probe Scan"},"1251":{i:0.000142986685929583,u:"../materials_user_manual/matsci_probe_scan_grid.htm",a:"The grid on which the probe or probe atom is placed is constructed at a specified distance from the van der Waals surface of the target, and thus represents a surface around the target. You can specify multiple surfaces, if you want to map a volume rather than a surface.  To set the distance from ...",t:"Specifying the Grid for a Probe Scan"},"1252":{i:0.000141994332298609,u:"../materials_user_manual/matsci_probe_scan_optimize.htm",a:"If you want to optimize the geometry of the target or the probe (or both) at each grid point, select the Geometry optimization option. For each grid point, the target and probe geometries are set to their input geometries, and then optimized. There are a number of choices of the parts of the system ...",t:"Optimizing the Geometry in a Probe Scan"},"1253":{i:0.000141713129864991,u:"../materials_user_manual/matsci_probe_scan_output.htm",a:"For some kinds of scans, it may not be useful to return all the results, so you can choose an option that determines which of the resulting structures you want to return as the results of the scan. All results—return all structures. This is probably the most useful for examining the energy surface.  ...",t:"Specifying the Probe Scan Output"},"1254":{i:0.000210082594487651,u:"../materials_user_manual/matsci_references.htm",a:"Halgren, T. A.; Lipscomb, W. N. The synchronous-transit method for determining reaction pathways and locating molecular transition states. Chem. Phys. Lett. 1977, 49, 225. Coropceanu,V.; Cornil, J.; da Silva Filho, D. A.; Olivier, Y.; Silbey, R.; Brédas, J.-L. Charge Transport in Organic ...",t:"Materials Science References"},"1255":{i:6.13254596701412E-05,u:"../adme_tox_user_manual/adme_tox_intro.htm",a:"The Physics-Based ADME/Tox suite is a set of tools for evaluating ADME/Tox properties. The current suite contains three tools: P450 Site of Metabolism —a tool for identifying likely sites of metabolism based on Hammet and Taft- type rules and 3D spatial information in several P450 isoforms.  ...",t:"Physics-Based ADME/Tox Introduction"},"1256":{i:9.7960576092735E-05,u:"../adme_tox_user_manual/cyp450som.htm",a:"Cytochrome P450 enzymes play an integral role in the metabolism pathway of drugs. These heme-containing enzymes catalyze a variety of reactions, such as hydroxylation, dealkylation, and double-bond oxidation, that result in the degradation of small molecules. Predicting the sites of metabolism of ...",t:"P450 Site of Metabolism"},"1257":{i:0.000123333360656307,u:"../adme_tox_user_manual/cyp450som_methodology.htm",a:"The reactivity rules have been parameterized to predict atomic reactivity profiles for promiscuous P450 enzymes that are thought to be mostly independent of structural restrictions on the binding poses. The reactivity is predicted with a linear free energy approach based on the Hammett and Taft ...",t:"Methodology for P450 Site of Metabolism"},"1258":{i:0.000119836188709773,u:"../adme_tox_user_manual/cyp450som_running.htm",a:"The calculation can be set up from the P450 Site of Metabolism - Perform Calculation panel, which you open by choosing Tasks → ADME and Molecular Properties → Structure Based P450 Site of Metabolism → Perform Calculation. The ligands can be taken from the Workspace (included entries), the selected ...",t:"Running the P450 Site of Metabolism Calculation"},"1259":{i:0.000113999773039427,u:"../adme_tox_user_manual/cyp450som_results.htm",a:"When the job finishes, you can examine the results in the P450 Site of Metabolism - Examine Results panel, which you open by choosing Tasks → ADME and Molecular Properties →  Structure Based P450 Site of Metabolism → Examine Results.  First, the results must be imported. To do so, click Import ...",t:"Examining the P450 Site of Metabolism Results"},"1260":{i:0.000101581904127725,u:"../adme_tox_user_manual/membrane_permeability.htm",a:"The membrane permeability tool allows you to calculate the passive membrane permeability of a set of molecules. It is primarily intended for use on congeneric series of ligands to evaluate the relative permeability of similar ligands. The calculations are based on a physical model with the ...",t:"Membrane Permeability"},"1261":{i:7.17511442205757E-05,u:"../adme_tox_user_manual/amphiphilic_moment.htm",a:"The structure-based adme tool allows you to calculate the amphipilic moment of one or more positively charged molecules. This property often correlates with certain ADME/Tox end points, such as phospholipidosis. The algorithm involves calculation of ClogP contributions (octanol/water partition ...",t:"Amphiphilic Moment"},"1262":{i:7.44993728166168E-05,u:"../prime_user_manual/prime_intro.htm",a:"Prime is a highly accurate suite of programs for protein structure prediction. The Homology Modeling workflow incorporates the complete protein structure prediction process from template identification, to alignment, to model building. Refinement can then be done from a separate set of panels, and ...",t:"Introduction to Prime"},"1263":{i:9.29888393567672E-05,u:"../prime_user_manual/prime_intro_using.htm",a:"Maestro is the graphical user interface (GUI) for all of Schrödinger’s products.  Prime provides the tools for three main tasks: homology modeling, structure refinement, and binding energy estimation, all of which are accessible from the Maestro interface. Homology modeling can be done through one ...",t:"Using Prime from Maestro"},"1264":{i:0.00010309972608866,u:"../prime_user_manual/protein_structure_prediction.htm",a:"The Prime Structure Prediction (PSP) workflow offers the most flexibility of the interfaces available for homology modeling in Maestro. You can make use of multiple templates, generate and edit the alignment, and build single-chain or multimer models. The workflow is presented as a series of steps ...",t:"Protein Structure Prediction"},"1265":{i:0.000110616361876143,u:"../prime_user_manual/structure_prediction_sequence.htm",a:"This step is used to read in an amino acid sequence, edit the sequence if desired, and select the query sequence. When you have selected a query sequence, click Next to go to the Find Homologs step.  The tasks that can be performed in this step are described in the topics listed below.",t:"PSP—Input Sequence Step"},"1266":{i:0.000128847613007037,u:"../prime_user_manual/structure_prediction_sequence_reading.htm",a:"To read a sequence from a file: Click File and specify a file containing the desired sequence. \n     Several file formats are supported, including FASTA, EMBL, GENBANK, PDB, and PIR. For a complete list, see  Sequence File Formats Used by Prime . File formats are auto-detected by the program. ...",t:"Reading or Entering a Sequence for PSP"},"1267":{i:0.000127686716338662,u:"../prime_user_manual/structure_prediction_sequence_selecting.htm",a:"To select a query from the Sequences table: Click on a sequence to select it for this run. \n     This brings the selected sequence into the sequence viewer. The sequences are named by extracting the name from either the PDB header or the FASTA file header. Non-unique sequences are ignored. ...",t:"Selecting a Query Sequence for PSP"},"1268":{i:0.000137569814263105,u:"../prime_user_manual/structure_prediction_sequence_editing.htm",a:"To edit a sequence: Select the sequence in the Sequences table. Click Edit Sequence. Edit the sequence in the dialog box that opens, and click OK. Sequences longer than 1,000 residues generally include more than one domain. You should consider dividing these sequences into probable domains and treat ...",t:"Editing a Sequence for PSP"},"1269":{i:0.000141398355320803,u:"../prime_user_manual/structure_prediction_sequence_messages.htm",a:"If an error message appears when you click Next, you cannot proceed until the problem is resolved. The error messages that you may encounter are: Invalid File This message appears if the system fails to extract a valid sequence from the file. Check that the sequence is in one of the supported ...",t:"PSP Input Sequence Error Messages"},"1270":{i:0.000193368274365285,u:"../prime_user_manual/structure_prediction_homologs.htm",a:"This step is used to search for homologs of the query sequence. These homologs can then be used as templates to build a model structure. When you are satisfied with the template (or structurally aligned multiple templates) you have selected, you can proceed to the Edit Alignment step by clicking ...",t:"PSP—Find Homologs Step"},"1271":{i:0.000204341056437564,u:"../prime_user_manual/structure_prediction_homologs_importing.htm",a:"To import homologs, use the Import buttons. You might want to do this for homologs that are not in the NCBI sequence database, or homologs that have been rotated into a common reference frame when multiple templates have been selected. You can import from two sources: From File—You can import ...",t:"Importing Homologs for PSP"},"1272":{i:0.000196018359607578,u:"../prime_user_manual/structure_prediction_homologs_finding.htm",a:"To launch a search for homologs: Click BLAST Homology Search (near the middle of the panel).  Depending on what is selected in the Find Homologs–Options dialog box, BLAST or PSI-BLAST is used to search for templates. The default is to use BLAST to search the non-redundant PDB database that has been ...",t:"Searching for Homologs in PSP"},"1273":{i:0.000173078055941945,u:"../prime_user_manual/structure_prediction_homologs_options.htm",a:"To change search options: Click Options to open the Find Homologs–Options dialog box. \n     You can use this dialog box to specify a PSI-BLAST or BLAST search, change the settings for either type of search, and choose whether to run the search on the BLAST web site or use a local installation of the ...",t:"Find Homologs Search Options"},"1274":{i:0.000187838993936854,u:"../prime_user_manual/structure_prediction_homologs_results.htm",a:"The output from the homolog search appears in the table labeled Homologs: (N found, n selected). For each potential template, the row contains the ID, the name, score, expectation value, % identities, % positives, % gaps, and header information (if taken from a PDB file). All percentages are rounded ...",t:"PSP Homolog Search Results"},"1275":{i:0.000118455613469138,u:"../prime_user_manual/structure_prediction_homologs_alignment.htm",a:"Multiple templates can be used in several ways in Prime to build a model. For a single query chain, you can build a composite model, where each template contributes to a different part of the structure, or you can build a consensus model, where each template contributes to all parts of the ...",t:"Multiple Templates and Structure Alignment in PSP"},"1276":{i:0.000107706553469419,u:"../prime_user_manual/structure_prediction_homologs_runs.htm",a:"If you decide to try different templates, you can avoid overwriting your existing workflow by starting a new one before you navigate back to the Find Homologs step to choose a different homolog. Choose Save As from the File menu. This creates a copy of the current run, allowing you to select a new ...",t:"Saving Prime Runs With Different Templates"},"1277":{i:0.000174103691139346,u:"../prime_user_manual/structure_prediction_homologs_messages.htm",a:"If one of the following messages appears when you click Next to go to the Edit Alignment step, you may not be able to proceed until the problem is resolved: Multiple Templates Not Aligned If you select multiple templates, you are reminded that the templates must be structurally aligned to one ...",t:"Find Homologs Error Messages"},"1278":{i:0.00014528828742385,u:"../prime_user_manual/structure_prediction_homologs_tablewarn.htm",a:"These messages appear in the Warning column of the Homologs table for sequences that are likely to be unusable for model building: Many missing residues Too many consecutive missing residues to guarantee model building will succeed.\n     !UNUSABLE: Alternate residue type The residue type was ...",t:"Homologs Table Warnings"},"1279":{i:0.000163237681336249,u:"../prime_user_manual/structure_prediction_alignment.htm",a:"In order to build a model structure of the query, the Build Structure step requires a satisfactory alignment between templates and query. The alignment of templates to query generated in the Find Homologs step is based only on sequence. This alignment usually needs some improvement. The Edit ...",t:"PSP—Edit Alignment Step"},"1280":{i:0.000120909039578534,u:"../prime_user_manual/structure_prediction_alignment_existing.htm",a:"When you start the Edit Alignment step, the BLAST/PSI-BLAST alignment for the template you selected in the Find Homologs step is displayed in the sequence viewer. If you chose a single template and this is the template and the alignment you want to use, click Next to go to the Build Structure step.  ...",t:"Using Existing Sequence Alignments in PSP"},"1281":{i:0.000112342730927684,u:"../prime_user_manual/structure_prediction_alignment_generating.htm",a:"If the existing alignments are not satisfactory, you can generate an alignment using one of two programs, listed under Alignment method: ClustalW—Running this program generates an alignment quickly, from sequence information only. It is best used when there is high sequence identity between the ...",t:"Generating a Sequence Alignment in PSP"},"1282":{i:0.000119161663912136,u:"../prime_user_manual/structure_prediction_alignment_generating_ssp.htm",a:"To run all available SSP programs, click Run. One secondary structure prediction program (SSpro) is bundled with Prime. However, the optional third-party program PSIPRED is highly recommended for optimal results, especially for GPCRs. PSIPRED is not available on Windows, but you can use the PSIPRED ...",t:"Generating Secondary Structure Predictions for PSP"},"1283":{i:0.000122368238757762,u:"../prime_user_manual/structure_prediction_alignment_importing_ssp.htm",a:"To import a secondary structure prediction for the query, click the Import button for Secondary structure prediction and select the SSP file, which may be in FASTA, PSIPRED, or Maestro format. If necessary, use the seqconvert utility to change the file format—see  \n            Sequence Format ...",t:"Importing and Exporting Secondary Structure Predictions in PSP"},"1284":{i:0.000126064319622803,u:"../prime_user_manual/structure_prediction_alignment_adjusting_ssp.htm",a:"If you believe an SSP is incorrect, you can delete it entirely or edit the incorrect portions. You can revert to the original SSP if you are not satisfied with your adjustments. To delete an SSP:  Right-click on the SSP and choose Delete from the shortcut menu. To edit an SSP: Click the Edit SSP ...",t:"Adjusting Secondary Structure Predictions in PSP"},"1285":{i:0.000136703564670591,u:"../prime_user_manual/structure_prediction_alignment_generating_gpcr.htm",a:"Before running an alignment, if you want to take advantage of the special capabilities for aligning GPCRs, select GPCR-specific alignment. These capabilities include fingerprint matching, a customized GPCR sequence database, and identification of transmembrane helices. For GPCR alignment, a set of ...",t:"GPCR-Specific Sequence Alignment in PSP"},"1286":{i:0.000139963837138105,u:"../prime_user_manual/structure_prediction_alignment_generating_constraints.htm",a:"To take advantage of your knowledge of the system, one or more pairs of query/template residues that you know should be aligned can be constrained to remain so, by using alignment constraints.  To constrain a pair of residues to remain aligned:  Click the Add Anchors toolbar button. Select one of ...",t:"Constraining the Sequence Alignment in PSP"},"1287":{i:0.000140830744243285,u:"../prime_user_manual/structure_prediction_alignment_generating_job.htm",a:"When you are satisfied with the secondary structure predictions for the query, click Align to start the alignment job.  After the job finishes, the Alignments table is updated with the new alignment score, the percent identity, the percent similarity, and the percent gaps, all rounded to the nearest ...",t:"Running the PSP Sequence Alignment"},"1288":{i:0.000140629964580185,u:"../prime_user_manual/structure_prediction_alignment_editing.htm",a:"You can edit any alignment, however obtained, using combinations of these Prime toolbar buttons: Slide Freely, Slide as Block, Add(/Remove) Anchors, Lock Gaps, and Unlock Gaps.  The Workspace shows the template structure colored by query residue property, displayed as a ribbon and side chains. Each ...",t:"Manually Editing a Sequence Alignment in PSP"},"1289":{i:0.000139054502462205,u:"../prime_user_manual/structure_prediction_alignment_messages.htm",a:"When you click Next at the end of the Edit Alignment step, the Check Alignment warning dialog box may appear. The warning lists possible alignment problems that you may want to fix before continuing to the next step—for example, gaps in secondary structure elements of the template, or gaps in the ...",t:"Edit Alignment Error Messages"},"1290":{i:0.000178260007612116,u:"../prime_user_manual/structure_prediction_build.htm",a:"The process of building a structure from one or more templates involves the following steps. Copying of backbone atom coordinates for aligned regions and side chains of conserved residues\n     Optimization of side chains\n     Minimization of non-template residues\n     Building of insertions and ...",t:"PSP—Build Structure Step"},"1291":{i:0.000139091011213446,u:"../prime_user_manual/structure_prediction_build_choose.htm",a:"If you are using multiple templates, you must choose the type of model you are building: composite, consensus, homo-multimer, or hetero-multimer. For composite models, you must select the regions from each template that are used to build the structure. The following sections provide more detail on ...",t:"Choosing the Model Type for the Protein Structure in PSP"},"1292":{i:0.000140758794562732,u:"../prime_user_manual/structure_prediction_build_choose_composite.htm",a:"If you want to build a composite model from the templates you selected in the Find Homologs step, select Composite/Chimera under Multi-template model type section. You must specify which template should be used for each region of the query, otherwise, the first template in the table is used for the ...",t:"Building a Composite Protein Structure Model with PSP"},"1293":{i:0.000141248859775752,u:"../prime_user_manual/structure_prediction_build_choose_consensus.htm",a:"If you want to build a consensus model from the templates, select Consensus under Multi-template model type. The model is then built from the consensus between the templates at each residue position: for example, if two templates have a loop in the same spatial region but in the third template, the ...",t:"Building a Consensus Protein Structure Model with PSP"},"1294":{i:0.000141310534473958,u:"../prime_user_manual/structure_prediction_build_choose_multimer.htm",a:"In addition to building models for a single chain, you can build models for multimers that are composed of copies of the same chain (homomultimers) or of different chains (heteromultimers). To build the multimer, the structures for all chains in the templates must be in the correct location ...",t:"Building Protein Multimers with PSP"},"1295":{i:0.000141038067503772,u:"../prime_user_manual/structure_prediction_build_preparing.htm",a:"Preparing to Build a Protein Structure Model in PSP Before you build the structure, you can make settings that affect the build process. These settings are described in the following sections.",t:"Preparing to Build a Protein Structure Model in PSP"},"1296":{i:0.000140014802618296,u:"../prime_user_manual/structure_prediction_build_preparing_ligands.htm",a:"Any cofactors and ligands present in the template you have chosen (except water) are listed in the Include ligands and cofactors list. To include one or more ligands or cofactors, select them in this list. You can use shift-click and control-click to select multiple rows. The selections are marked ...",t:"Including Ligands and Cofactors in the Protein Structure in PSP"},"1297":{i:0.000136676031772938,u:"../prime_user_manual/structure_prediction_build_preparing_constraints.htm",a:"If you know that certain residues must be in close contact in the final structure, such as cysteines forming a disulfide bond, you can constrain their positions in the final model by setting proximity constraints on residue pairs. A disulfide bond is formally constructed between pairs of CYS ...",t:"Setting Constraints on the Protein Structure in PSP"},"1298":{i:0.000125916180628872,u:"../prime_user_manual/structure_prediction_build_preparing_options.htm",a:"Click Options to open the Build Structure–Options dialog box and specify which optional model building steps to perform: Preserve the residue numbers of the template—Number the residues in the built structure the same as in the template, as far as possible. Does not apply to consensus modeling. When ...",t:"PSP Protein Structure Building Options"},"1299":{i:0.000121707979289317,u:"../prime_user_manual/structure_prediction_build_building.htm",a:"There are two choices of method for building the structure: Knowledge-based—Construct insertions and close gaps using segments from known structures. With this option, you can choose to return multiple models of the structure. By default, only one is returned. The models are approximately ordered by ...",t:"Building the Protein Structure in PSP"},"1300":{i:8.71895162362336E-05,u:"../prime_user_manual/structure_prediction_build_technotes.htm",a:"The energy-based model building method in the Build Structure step uses the following resources and methods: The OPLS_2005 all-atom force field for energy scoring of proteins as well as for ligands and other non-amino-acid residues. A Surface Generalized Born (SGB) continuum solvation model for ...",t:"Technical Notes for the Build Structure Step"},"1301":{i:0.000165590306586742,u:"../prime_user_manual/refine_proteins.htm",a:"The Refinement module in Prime is used to refine protein structures. It can be used for loop refinement, side-chain prediction, minimization, rigid-body minimization, hybrid Monte Carlo conformational searching, binding site refinement, and energy analysis at the current geometry of the protein ...",t:"Refining Protein Structures"},"1302":{i:0.000120439419463259,u:"../prime_user_manual/refine_proteins_prep.htm",a:"The refinement module can handle many types of structures, and can fix many structural problems. For standard residues, it fixes formal charges and bond orders, and corrects disparities between the sequence and the structure. For example, if a residue has the coordinates of ALA but is called SER, ...",t:"Preparing Protein Structures for Refinement"},"1303":{i:0.000100421325170361,u:"../prime_user_manual/refine_proteins_panel.htm",a:"The refinement task panels have a number of features in common, which are described below. The task-specific features are described in the following sections. When selecting regions for refinement, you may want to view information about possible structural problems. This information is available in ...",t:"The Protein Refinement Panels"},"1304":{i:0.000131431339840399,u:"../prime_user_manual/refine_proteins_loops.htm",a:"The loop refinement task is capable of refining loop structures of various lengths, and provides algorithms for different loop lengths. In addition, loops whose structure affects other loops can be cooperatively refined in pairs.  To refine one or more loops serially: Click one of the Find loops in ...",t:"Refining Loops"},"1305":{i:0.000124404970190923,u:"../prime_user_manual/refine_proteins_sidechains.htm",a:"The Predict Side Chains task predicts the conformation of side chains in one or more proteins by sampling orientations to minimize the energy. The sampling can include the orientation with respect to the backbone, and sampling of the backbone as well.  You can select residues for side-chain ...",t:"Predicting Side Chains"},"1306":{i:0.000148724506439131,u:"../prime_user_manual/refine_proteins_minimize.htm",a:"The Minimize refinement task minimizes the energy of selected parts of proteins or entire proteins with respect to coordinate changes. You can specify a set of atoms or residues for minimization with the Atoms for minimization picking tools, or click Select Non-Conserved Residues to minimize atoms ...",t:"Minimizing Protein Structures"},"1307":{i:0.000127960723504804,u:"../prime_user_manual/refine_proteins_rigidbody.htm",a:"The Minimize as Rigid Body task allows you to treat part of a system as a rigid body or a set of rigid bodies, with optional freely moving atoms, and freeze the rest of the system. The panel offers two standard selections of the rigid body, Protein and Ligand, which are detected automatically, and a ...",t:"Minimizing Parts of Protein Structures as Rigid Bodies"},"1308":{i:0.000123629442211624,u:"../prime_user_manual/refine_proteins_hybridmc.htm",a:"The Hybrid Monte Carlo task uses high-temperature molecular dynamics simulations followed by minimization to explore conformational space. Each MD/minimization stage produces a minimized structure. The high temperature MD simulation allows the structure to move between minima on the potential energy ...",t:"Hybrid Monte Carlo Conformational Sampling for Protein Minimization"},"1309":{i:0.000122567918754624,u:"../prime_user_manual/refine_proteins_complexes.htm",a:"The Refine Protein-Ligand Complex task refines the structure of protein-ligand complexes. The ligand can be covalently bound or bound by noncovalent interactions. The refinement can be restricted to the region around the ligand, and can sample positions and conformations of the ligand. To set up a ...",t:"Refining Protein-Ligand Complexes"},"1310":{i:0.000123347840981153,u:"../prime_user_manual/refine_proteins_energy.htm",a:"The Calculate Energy task performs a single-point molecular mechanics energy calculation which includes a breakdown of the energy into various terms. The calculation uses the OPLS_2005 all-atom force field for protein residues as well as for ligands and cofactors. The output includes a breakdown of ...",t:"Calculating the MM Energy of a Protein"},"1311":{i:0.00012696692112497,u:"../prime_user_manual/refine_proteins_options.htm",a:"The Options dialog boxes for the refinement tasks have several common options at the top, followed by options for the specific task. The Calculate Energy and Predict Side Chains tasks only have the common options. Click Options in any of the refinement panels to view or change settings. The options ...",t:"Setting Protein Refinement Options"},"1312":{i:0.000128155909940505,u:"../prime_user_manual/refine_proteins_options_common.htm",a:"These options apply to all types of refinement or to more than one type of refinement. The Seed option is not available for minimization tasks, which do not use a random seed.  The Environment options allow you to set up a special environment for the protein. You can choose one of the three options. ...",t:"Common Protein Refinement Options"},"1313":{i:0.000145192679129963,u:"../prime_user_manual/refine_proteins_options_loop.htm",a:"The loop refinement options include selecting the sampling method, subjob options, output options, and some general constraints on the optimization. Prime can perform cooperative loop sampling as well as serial loop sampling. Both the size of the loop and the sampling options chosen affect the time ...",t:"Loop Refinement Options"},"1314":{i:0.000134155635722205,u:"../prime_user_manual/refine_proteins_options_minimization.htm",a:"The minimization options allow you to specify the minimization method and set options for convergence. These options are the same for the Minimize and Minimize as Rigid Body tasks. Method  Choose the minimization method from this option menu. The choices are Quasi-Newton (LFBGS), Truncated Newton, ...",t:"Protein Minimization Options"},"1315":{i:0.000148140516368367,u:"../prime_user_manual/refine_proteins_options_hybridmc.htm",a:"The options presented in the Hybrid Monte Carlo - Options dialog box allow you to set the temperature and length of the sampling, and decide which structures to return. Number of steps  Specify the number of steps to take in the sampling procedure. Each step consists of a short, high-temperature MD ...",t:"Protein Hybrid Monte Carlo Options"},"1316":{i:0.000172234738942033,u:"../prime_user_manual/refine_proteins_options_complex.htm",a:"The options presented in the Refine Protein-Ligand Complex - Options dialog box depend on the sampling algorithm chosen in the Refine Protein-Ligand Complex panel. The options are described below for each algorithm. Local Optimization Options  Set the number of passes through the prediction of side ...",t:"Protein-Ligand Complex Refinement Options"},"1317":{i:0.000243283362711015,u:"../prime_user_manual/refine_proteins_membrane.htm",a:"The Prime implicit membrane model is intended for proteins embedded in a membrane. The implicit membrane is a low-dielectric slab-shaped region, which is treated in the same way as the high-dielectric implicit solvent region. Hydrophobic groups, which normally pay a solvation penalty for creating ...",t:"Setting Up an Implicit Membrane"},"1318":{i:0.000216312084187571,u:"../prime_user_manual/refine_proteins_constraints.htm",a:"Prime provides five kinds of constraints that you can apply in a refinement: distance constraints (also known as pairwise constraints), cartesian constraints, helical constraints, spatial constraints, and membrane constraints. You can apply the constraints in the Constraints dialog box for the ...",t:"Applying Constraints for Protein Refinement"},"1319":{i:0.000151767635116284,u:"../prime_user_manual/refine_proteins_technical.htm",a:"Technical Details for Protein Refinement The topics listed below give details of the methods and algorithms used in protein structure refinement.",t:"Technical Details for Protein Refinement"},"1320":{i:0.000209273956748606,u:"../prime_user_manual/refine_proteins_technical_loop.htm",a:"Loop refinement using the Default option proceeds as follows: The loop is reconstructed using the backbone dihedral library, by building up half from each direction. The resolution starts off very coarsely, becoming finer until it manages to produce a required number of physically realistic loop ...",t:"Loop Refinement Technical Details"},"1321":{i:0.000196937202849188,u:"../prime_user_manual/refine_proteins_technical_cooploop.htm",a:"Cooperative loop refinement proceeds as follows: A list of side chains that will be optimized is generated. By default this list includes any residue with an atom within 7.5 Å of either loop 1 or loop 2. Loop 1 is sampled, allowing side chains in the list from  Step 1   to move, holding the backbone ...",t:"Cooperative Loop Refinement Technical Details"},"1322":{i:0.000231923263506372,u:"../prime_user_manual/refine_proteins_technical_sidechain.htm",a:"The structure refinement program uses the following procedure to re-predict conformations for the side chain set that you select. The default algorithm is as follows: Side-chain rotamers are randomized for nonconserved residues (the default) or for all residues. Beginning with the first residue to ...",t:"Side-Chain Prediction Technical Details"},"1323":{i:0.00024950062210195,u:"../prime_user_manual/refine_proteins_technical_hierarchical_sampling.htm",a:"Hierarchical sampling in the refinement of protein-ligand complexes and in MM-GSBA calculations is an implementation of the PGL Sampling approach described in \"Exploring hierarchical refinement techniques for induced fit docking with protein and ligand flexibility\"  [1] , and applied with other ...",t:"Hierarchical Sampling Technical Details"},"1324":{i:0.000159637083219924,u:"../prime_user_manual/refine_proteins_nonstandard.htm",a:"If you want to refine a structure with nonstandard residues, you can make changes to the residues in the structures before refinement. A number of nonstandard residues are supported: the neutralized forms of ASP, LYS, ARG, and GLU, which are named ASH, LYN, ARN, and GLH; and the tautomers of HIS ...",t:"Protein Refinement with Nonstandard Residues"},"1325":{i:0.000106558401049632,u:"../prime_user_manual/refine_proteins_output.htm",a:"By default, the refined structures are appended to the Project Table for the project currently open. If you would prefer the output structure to replace the input structure, or would rather not incorporate the output structure into the project at all, select the appropriate option from the ...",t:"Output of Protein Refinement Jobs"},"1326":{i:0.000103363152142715,u:"../prime_user_manual/protein_structalign.htm",a:"Protein Structure Alignment It is sometimes useful to be able to align proteins outside the usual Prime workflow. Maestro provides access to the structure alignment facilities of Prime so that you can perform such alignments. Two panels are available from the tools menu:",t:"Protein Structure Alignment"},"1327":{i:0.000148361831680645,u:"../prime_user_manual/structalign_panel.htm",a:"To open the Protein Structure Alignment panel, choose Tasks → Structure Alignment → Protein Structure Alignment in the Task Tool. The Protein Structure Alignment panel.  In the Protein Structure Alignment panel, structure alignment is performed on Project Table entries that have been included in the ...",t:"The Protein Structure Alignment Panel"},"1328":{i:0.000114130109270212,u:"../prime_user_manual/structalign_binding_sites.htm",a:"The Align Binding Sites panel allows you to align the binding sites (or other structural features) of members of a family of proteins. The Align Binding Sites job first runs a Protein Structure Alignment to obtain the global alignment and then automatically generates the list of atoms to use in a ...",t:"The Align Binding Sites Panel"},"1329":{i:0.0001051168029313,u:"../prime_user_manual/mm_gbsa.htm",a:"The Prime MM-GBSA panel can be used to calculate ligand binding energies and ligand strain energies for a set of ligands and a single receptor, using the MM-GBSA technology available with Prime. You can also run the job from the command line, which offers more flexibility than the panel. The ligands ...",t:"Prime MM-GBSA"},"1330":{i:0.000139634069819622,u:"../prime_user_manual/mm_gbsa_maestro.htm",a:"To open the Prime MM-GBSA panel, choose Tasks → Lead Optimization → MM-GBSA. The Prime MM-GBSA panel.  After preparing your structures, you can set up the calculation as follows: Specify the source of the structures. You can take structures from a Pose Viewer file (Glide output), or from separated ...",t:"Running Prime MM-GBSA from Maestro"},"1331":{i:0.000118696398518968,u:"../prime_user_manual/mm_gbsa_output.htm",a:"The output Maestro structure file, which is compressed by default, includes a number of properties. Properties are calculated for the complex, the ligand, and the receptor, and strain energies can be calculated for the ligand and the receptor. For each of these properties, individual contributions ...",t:"Prime MM-GBSA Output"},"1332":{i:0.000120055267856462,u:"../prime_user_manual/prime_energy_visualizer.htm",a:"The Prime energy and its components from an Analyze Energy job, a WM/MM job, or an MM-GBSA job can be visualized in the Workspace, by using the Prime Energy Visualizer panel. This panel also supports visualization of other properties, such as the partial charge, ligand strain, solvent accessibility, ...",t:"Visualizing Prime Energy Terms"},"1333":{i:8.18010674841615E-05,u:"../prime_user_manual/prime_glossary.htm",a:"alignment—The optimal matching of residue positions between sequences, typically a query sequence and one or more template sequences. all-atom structure—A structure in which all atoms, including hydrogen atoms, are explicitly represented. This means that the structure must have coordinates for all ...",t:"Prime Glossary"},"1334":{i:0.000114347627840414,u:"../prime_user_manual/prime_references.htm",a:" Borrelli, K. W.; Cossins, B.; Guallar, V.  Exploring hierarchical refinement techniques for induced fit docking with protein and ligand flexibility.  J Comput Chem. 2010, 31, 1224-35. doi: 10.1002/jcc.21409.  Knight, J. L.; Krilov, G.; Borrelli, K. W. ; Williams, J.; Gunn, J. R.; Clowes, A.; Cheng, ...",t:"Prime References"},"1335":{i:6.13254596701412E-05,u:"../primex_user_manual/primex_intro.htm",a:"PrimeX performs refinement of molecular replacement solutions in X-ray crystal structure determination. Rigid-body refinement, individual coordinate and B-factor refinement, and grouped occupancy refinement can be performed in reciprocal space, with maximum likelihood and least-squares targets. With ...",t:"About PrimeX"},"1336":{i:7.43575653581844E-05,u:"../primex_user_manual/primex_overview.htm",a:"PrimeX has two unique features that change the way that this program is employed in refining a molecular replacement crystal structure solution, as compared to other refinement programs. Although the workflow discussed here for PrimeX will seem familiar to protein crystallographers, the use of the ...",t:"The PrimeX Process"},"1337":{i:0.00010381040824137,u:"../primex_user_manual/primex_basics.htm",a:"The graphical interface for all Schrödinger products, including PrimeX, is Maestro. Maestro provides many features for storing, managing, manipulating, creating, and modifying structures, surfaces and associated data. Information on the general capabilities of Maestro can be found in the  Maestro ...",t:"PrimeX and Maestro Basics"},"1338":{i:0.000125613092599104,u:"../primex_user_manual/primex_basics_mainwindow.htm",a:"At the top of the Maestro main window are the title bar and the menu bar, followed by the Selection toolbar. In the center is the Workspace, where structures and surfaces are displayed. At the bottom is the Footer bar, which contains. Several optional components can be displayed at the bottom of the ...",t:"PrimeX Basics: The Maestro Main Window"},"1339":{i:0.000123077481701738,u:"../primex_user_manual/primex_basics_projects.htm",a:"Molecular structures, surfaces and data are managed in a Maestro project. When you start Maestro, a project is opened; if none is specified, the project is a scratch project, which can be saved as a named project. All project data is automatically saved to disk.  Structures are stored in project ...",t:"PrimeX Basics: Maestro Projects"},"1340":{i:0.000123728998149325,u:"../primex_user_manual/primex_basics_manipulate.htm",a:"The topics listed below describe the tools available in Maestro for manipulating structures in the Workspace. You can change the orientation or location of the entire Workspace contents, or of any part of a structure. The coordinate system is set up so that the x axis is horizontal, the y axis is ...",t:"PrimeX Basics: Manipulating Structures"},"1341":{i:0.000129547519592874,u:"../primex_user_manual/primex_basics_manipulate_view.htm",a:"Changing the view of the Workspace in Maestro is done with the middle and right mouse buttons. To translate the entire contents of the Workspace, drag with the right mouse button. To rotate the Workspace contents, drag with the middle mouse button for rotation about the x and y axes, and hold down ...",t:"PrimeX Basics: Changing the View (Global Transformations)"},"1342":{i:0.000117044244317979,u:"../primex_user_manual/primex_basics_manipulate_transform.htm",a:"If you have multiple structures in the Workspace and want to reorient only one of them, or if you want to move only part of a structure, you can perform a local transformation. When you do, the coordinates of the structure you move are changed. You can also use local transformations to move a set of ...",t:"PrimeX Basics: Local Transformations"},"1343":{i:0.000132647040735238,u:"../primex_user_manual/primex_basics_manipulate_adjust.htm",a:"You can change individual bond lengths, bond angles, and dihedral angles (torsions), by using the toolbar buttons and the mouse. On the Proteins toolbar, the Quick Torsion tool allows you to pick a bond, then drag with the left mouse button or use the mouse wheel to rotate around that bond. The ...",t:"PrimeX Basics: Structure Adjustments"},"1344":{i:0.000114534141894663,u:"../primex_user_manual/primex_basics_identify.htm",a:"Maestro provides a variety of ways to identify structures or features in a structure. The topics listed below summarize the use of color schemes, labeling, displaying atoms and bonds, and displaying entry properties. For complete information, see the  Maestro Quick Reference Guide  and the online ...",t:"PrimeX Basics: Identifying Structural Features"},"1345":{i:0.000117735969898159,u:"../primex_user_manual/primex_basics_identify_color.htm",a:"Color schemes are useful for identifying parts of a structure by the values of a property. The schemes can be applied to some or all of the atoms in the structure displayed in the Workspace. The color scheme is also applied to the residues in the sequence viewer.  To color all atoms with a ...",t:"PrimeX Basics: Using Color Schemes"},"1346":{i:0.000113186831009157,u:"../primex_user_manual/primex_basics_identify_label.htm",a:"Adding labels to atoms or residues can be used to identify them, but can also be used to provide information about their properties. For a protein, it may not be useful to apply labels to all atoms, because doing so can make Workspace operations slow. You can apply labels selectively, either by ...",t:"PrimeX Basics: Labeling Atoms"},"1347":{i:0.000126307032508291,u:"../primex_user_manual/primex_basics_identify_display.htm",a:"Maestro includes a range of tools that you can use to selectively display atoms in a structure and to change the appearance of atoms and bonds. When you want to focus on a particular region of a molecule—for example, a loop that has just been refined—it is useful to display only the atoms of ...",t:"PrimeX Basics: Displaying Atoms and Bonds"},"1348":{i:0.000116150063921953,u:"../primex_user_manual/primex_basics_identify_feedback.htm",a:"When you pause the cursor over an atom in the Workspace, information on the atom is displayed in the status area immediately below the Workspace. This information includes the chain name, residue name and number, and atom name. The display of information is a preference that can be set in the ...",t:"PrimeX Basics: Workspace and Sequence Viewer Feedback Information"},"1349":{i:0.000131679817440643,u:"../primex_user_manual/primex_basics_identify_find.htm",a:"If you want to find an atom, molecule, residue, chain, or other group of atoms in the Workspace, you can do so with the Find toolbar, which you open from the Edit menu or by pressing CTRL+F (⌘F). All instances of the object that match are located. This toolbar is described in  Find Toolbar .  If you ...",t:"PrimeX Basics: Finding Structural Components in the Workspace"},"1350":{i:0.000132148699168939,u:"../primex_user_manual/primex_basics_modify.htm",a:"When you modify a structure in the Workspace, the changes are also made in the project entry to which the structure belongs. This is the default behavior. You can change this behavior in the Project tab of the Preferences panel, under Synchronize Workspace changes, so that changes in the Workspace ...",t:"PrimeX Basics: Modifying Structures"},"1351":{i:0.000102426275384536,u:"../primex_user_manual/primex_basics_track.htm",a:"PrimeX tasks return a structure as a new entry in the Project Table (except for R-factor calculation, which adds to an existing entry). To track the changes made to a structure as tasks are performed, you can use any of the following properties to track information about the progress of the ...",t:"PrimeX Basics: Tracking Changes"},"1352":{i:9.0347440154918E-05,u:"../primex_user_manual/primex_basics_panel.htm",a:"The PrimeX panel provides access to all the tools that are available for protein X-ray structure refinement. In the process of refining a structure, you will use the tools in this panel multiple times, so it is useful to leave it open. By default it is docked into the Workspace, but you can undock ...",t:"The PrimeX Panel"},"1353":{i:9.36969663164933E-05,u:"../primex_user_manual/non_crystallographers.htm",a:"The purpose of this section is to provide additional background and guidance to users of PrimeX who have limited experience with protein crystallography. One popular application by non-crystallographers is the preparation of a structure refined with other programs for use in computational chemistry. ...",t:"PrimeX for Non-Crystallographers"},"1354":{i:0.000114247630438918,u:"../primex_user_manual/non_crystallographers_rfactors.htm",a:" Protein crystal structures are molecular models created and refined to maximize agreement with X-ray diffraction data. The vast majority of protein crystal structures are solved in a resolution range where the coordinates of individual atoms cannot be discerned directly from the diffraction ...",t:"R-factors and Refinement in PrimeX"},"1355":{i:0.00012410597336294,u:"../primex_user_manual/non_crystallographers_restraints.htm",a:"Certain aspects of molecular geometry are more accurately known than can be obtained from protein crystal diffraction. Bond lengths, bond angles, group planarity and non-bonded contacts can be derived from small molecule crystal structures, which are typically known to a much higher resolution than ...",t:"Molecular Geometry, Restraints, and Hydrogen Atoms in PrimeX"},"1356":{i:0.000107321446047893,u:"../primex_user_manual/non_crystallographers_bfactors.htm",a:"X-ray diffraction data contains information about atomic motions as well as their positions. The isotropic B-factor (also known as the isotropic temperature factor) is an atomic quantity proportional to the root-mean-square displacement of that atom in space due to thermal motion. However, errors in ...",t:"B-factors and Refinement in PrimeX"},"1357":{i:0.000115420729943798,u:"../primex_user_manual/non_crystallographers_input.htm",a:"PrimeX requires three types of data. First, you need to have a list of coordinates for an existing model, usually available as a PDB file. After the selection of the model that you wish to work on, you will be asked about setting the isotropic B-factors. At this stage, you should set them all to a ...",t:"PrimeX Input Data"},"1358":{i:0.000111460104690261,u:"../primex_user_manual/non_crystallographers_start.htm",a:"To calculate the R-factors, click R-Factor Calculation in the PrimeX panel (see  R-Factor Calculations ). Default values are almost always appropriate for this calculation, although a number of options are available in the Calculation Settings panel. After the calculation is complete, find the place ...",t:"Where to Start with PrimeX: R-Factors"},"1359":{i:0.000107685924298606,u:"../primex_user_manual/non_crystallographers_maps.htm",a:"Calculation of an electron density map is a good next step. Click the Create Map button in the PrimeX panel to open the panel. For the map coefficients, select 2FoFc. All other parameters can be left at the defaults. After running the job, the Manage Surfaces panel opens, which allows you to ...",t:"Electron Density Maps"},"1360":{i:0.000133371682830642,u:"../primex_user_manual/non_crystallographers_residues.htm",a:"Crystallographers have three main ways of designating which residues are not well defined in the electron density. The least common practice is to set the B-factors of the atoms that are not observed to a very high value (80 Å2 or more) to indicate that that portion of the structure is disordered in ...",t:"Missing Residues in PrimeX"},"1361":{i:0.000146583631147709,u:"../primex_user_manual/non_crystallographers_recipmin.htm",a:"When you are satisfied that all previously missing atoms in the model have been adequately placed into the electron density, reciprocal-space minimization is a logical next step. Choose Reciprocal-Space Refinement from the Select a task button menu. In the panel that opens choose Minimization as the ...",t:"Reciprocal-Space Minimization of Coordinates"},"1362":{i:0.000128915625179375,u:"../primex_user_manual/non_crystallographers_hbond.htm",a:"You will note that PrimeX has added hydrogens to your model. Hydrogens are essential to refinement in PrimeX. Further, it is important that the position of hydrogens be optimized to form the most likely hydrogen bonds. At this point, choose the Optimize H-Bond Network task from the PrimeX task ...",t:"Optimizing H-Bond Networks in PrimeX"},"1363":{i:0.000122612318983228,u:"../primex_user_manual/non_crystallographers_minrefine.htm",a:"You should minimize the atomic positions at least one more time, as described in  Reciprocal-Space Minimization of Coordinates , before using the model for additional calculations. Pay attention to the value used for the X-ray weight to derive the best balance between the force field and the X-ray ...",t:"Additional Minimization and B-Factor Refinement"},"1364":{i:8.7381700043253E-05,u:"../primex_user_manual/non_crystallographers_remediation.htm",a:"If you have reason to believe that the crystal structure that you are working with requires more substantial remediation, a possible workflow is outlined here. The  PrimeX Quick Start Guide  is a useful resource for additional information on the new operations not previously mentioned in this ...",t:"A More Complex Remediation Workflow"},"1365":{i:9.09304274703377E-05,u:"../primex_user_manual/primex_input.htm",a:"The first step in the PrimeX refinement process is to specify the source of the model structure, the sequence, the reflection data, optional maps, and set the space group and unit cell parameters. These tasks are performed in the Input Data dialog box, and are described in the topic listed below. To ...",t:"Setting Up a PrimeX Refinement"},"1366":{i:0.000122102547341641,u:"../primex_user_manual/primex_input_structseq.htm",a:"The initial structure is specified with the Use structure from options. You can use one of two sources: File—Read the initial structure from a file. You can specify the file name in the File name text box, or click Browse and navigate to the desired file. The file must be in PDB or Masetro format. ...",t:"Specifying the Initial Structure and Reference Sequence for PrimeX"},"1367":{i:0.000130171053636434,u:"../primex_user_manual/primex_input_reflections.htm",a:"To specify the file containing the reflection data, enter the file name in the Reflection data text box, or click Browse and navigate to the file. In addition to the reflections, the space group and cell parameters are read from the file if possible, and replace any existing definitions. If the ...",t:"Specifying the Reflection Data for PrimeX"},"1368":{i:0.000151395971198079,u:"../primex_user_manual/primex_input_crystalparams.htm",a:"The two sections in the lower part of the Input Data dialog box provide a place to specify the space group and the unit cell parameters.  In the Space group section, you can enter the group number in the text box, or click Choose and choose the space group from the list provided in the Choose Space ...",t:"Specifying Crystallographic Parameters for PrimeX"},"1369":{i:0.000239777251492492,u:"../primex_user_manual/primex_input_calcsettings.htm",a:"In addition to specifying the basic input data, you can set a number of calculation settings prior to running calculations. These settings are made in the Calculation Settings dialog box, which you open by clicking the Calculation Settings button. They include setting resolution limits on and ...",t:"Specifying Calculation Settings for PrimeX"},"1370":{i:0.000172078924898631,u:"../primex_user_manual/primex_input_calcsettings_resolution.htm",a:"The Resolution limits options allow you to either calculate the limits from the data, by selecting Calculate from data and cell constants, or to specify the limits, by selecting Use values and entering the limits in the Low and High text boxes. Only the reflection data that lie between these two ...",t:"Resolution Limits in PrimeX"},"1371":{i:0.000151097225842506,u:"../primex_user_manual/primex_input_calcsettings_solvent.htm",a:"There are three options for bulk solvent correction: None, Babinet, and Mask method. The Babinet method is less accurate, but does not depend on having a good estimate of the structure. The Mask method relies on a definition of the boundary between the protein and the solvent, and is controlled by ...",t:"Bulk Solvent Corrections for PrimeX"},});