define({"3729":{i:0.000130685312298282,u:"../install_guide/mac_install_suite.htm",a:"Click the disk image (.dmg) in your Downloads folder or insert the DVD.  Double-click the installer package (.pkg). The installer starts. Follow the prompts in the installer. The installation path is /opt/schrodinger/suites2017-1. You can use a different disk by clicking Change Location. Enter the ...",t:"Installing a Schrödinger Suite on a Mac"},"3730":{i:0.000144094026992842,u:"../install_guide/mac_install_thirdparty.htm",a:"Prime requires access to the PDB and BLAST databases. If these databases are not installed locally, you can use web access to send queries and retrieve the results. If you want to use a local installation instead (for security reasons, for example), use the following instructions. Check that you ...",t:"Installing Prime Third-Party Databases on a Mac"},"3731":{i:8.17397084443522E-05,u:"../install_guide/mac_install_uninstall.htm",a:"Schrödinger software comes with an application for uninstalling the software. This application performs all the necessary actions to completely remove the software from your Mac. You should not use any other method for uninstallation. To uninstall Schrödinger software: Double-click the ...",t:"Uninstalling Schrödinger Software on a Mac"},"3732":{i:0.000155683860443632,u:"../install_guide/knime_install.htm",a:"The Schrödinger KNIME Extensions are available with the software distribution, either on DVD or the software download page, or from the Schrödinger KNIME update web site. The software distribution includes KNIME itself as well as the extensions. To install KNIME and the Schrödinger extensions from ...",t:"Installing and Updating KNIME Extensions"},"3733":{i:0.00014737299818622,u:"../install_guide/knime_install_existing.htm",a:"If you want to install the Schrödinger extensions into an existing version of KNIME, you can do so from the KNIME interface. If the version of KNIME does not meet the minimum requirements, you must update your version of KNIME first. The installation requires access to the Schrödinger KNIME update ...",t:"Installing into an Existing Version of KNIME"},"3734":{i:0.000147999012224633,u:"../install_guide/knime_install_existing_base.htm",a:"Start KNIME. Choose Help → Install New Software. The Install Available Software panel opens. If you already have the Base Chemistry Types and Nodes installed, you can skip to  Step   . If you are not sure, proceed with the next step to check.  From the Work with option menu, choose the KNIME Update ...",t:"Installing the KNIME Base Chemistry Types and Nodes"},"3735":{i:0.000158518473461607,u:"../install_guide/knime_install_existing_extensions.htm",a:"Start KNIME, if it is not already running. Choose Help → Install New Software. The Install Available Software panel opens. In the Available Software tab, click Add.  The Add Repository dialog box opens. Specify the location of the update site: If you are using the web site directly: Enter the ...",t:"Installing the Schrödinger KNIME Extensions"},"3736":{i:0.000195017682679064,u:"../install_guide/knime_install_existing_identifying.htm",a:"Start KNIME, if it is not already running. Choose File → Preferences. The Preferences panel opens. Under KNIME, choose Schrodinger. The Schrodinger preferences are displayed. There are two environment variables that you can set: SCHRODINGER, and PYMOL4MAESTRO. If either of the fields for these ...",t:"Identifying the Schrödinger Software Installation to KNIME"},"3737":{i:0.000157629346221425,u:"../install_guide/knime_install_updating.htm",a:"If you have direct access to the web, you can update the existing Schrödinger KNIME extensions to new versions using the update mechanism in KNIME, as follows. Start KNIME. Choose Help → Check for Updates. The Contacting Software Sites panel opens, and checks for updates to the software, by ...",t:"Updating Existing Schrödinger KNIME Extensions"},"3738":{i:0.000144861006083353,u:"../install_guide/knime_install_adding.htm",a:"Adding new Schrödinger extensions uses a very similar mechanism to installing the Schrödinger extensions into an existing KNIME installation. You can do this from the Schrödinger KNIME update web site or from a downloaded archive of this web site.  Start KNIME. Choose Help → Install New Software. ...",t:"Adding New Schrödinger KNIME Extensions"},"3739":{i:0.000182916097974513,u:"../install_guide/knime_install_workflows.htm",a:"KNIME workflows that have been wrapped to run from Maestro can be installed for the use of all users into the installation. The standard location for these workflows is in the mmshare-vversion/python/workflows directory of the installation. The installation can be performed with one of the following ...",t:"Installing KNIME Workflows"},"3740":{i:0.000137187048635587,u:"../install_guide/knime_install_web_service.htm",a:"Some KNIME nodes can be run through a web service. The instructions for setting up the web service and the necessary configuration are given below. To set up the web server: Install the Apache Tomcat web server on the machine where the Schrödinger software is installed. If you already have the web ...",t:"Installing A Web Service for KNIME Node Execution"},"3741":{i:0.000380168286074152,u:"../install_guide/job_setup.htm",a:"Schrödinger products use a common Job Control facility, which allows the user to submit, monitor, suspend and terminate jobs. The Job Control facility is described in detail in the  Job Control Guide . To run jobs on the local host only, no additional configuration is needed, but you should test ...",t:"Preparing for Remote Job Submission"},"3742":{i:0.000569204963573363,u:"../install_guide/job_setup_hosts.htm",a:"The Job Control facility obtains information about the hosts on which it will run jobs from the hosts file. The default name for this file is schrodinger.hosts. Maestro also uses the hosts file to set up the menus in the Job Settings dialog box.  This file must contain information on all hosts to ...",t:"The Hosts File"},"3743":{i:0.00016943755191961,u:"../install_guide/job_setup_hosts_name.htm",a:"The name setting must be the first line for each entry. This is the name that is used to select the host (or batch queue) with the configuration specified in the following settings. It is displayed in the list of hosts in the Start dialog box Host menu or table. Usually, entry-label is the name of a ...",t:"The name and host Settings in the Hosts File"},"3744":{i:0.000137604957596187,u:"../install_guide/job_setup_hosts_user.htm",a:"If you have different user names on the submission and execution hosts, you can include a user setting for the execution host in the hosts file on the submission host. The user setting should never be added to entries in the hosts file in the installation directory, because this would prevent other ...",t:"The user Setting in the Hosts File"},"3745":{i:0.000189504542172451,u:"../install_guide/job_setup_hosts_tmpdir.htm",a:"The tmpdir setting specifies the scratch directory, where temporary files can be written. Examples are /scr or /temp. The file system on which this directory is mounted should be large enough for the largest temporary files, should be mounted locally, and should be writable by the user. The actual ...",t:"The tmpdir Setting in the Hosts File"},"3746":{i:0.000141480552689551,u:"../install_guide/job_setup_hosts_processors.htm",a:"For hosts with multiple processors that are not running a queueing system, set processors to the number of processors on the host. For batch queues, set processors to the number of processors available to the queue. Here, “processors” refers to hardware units capable of running a job, which for most ...",t:"The processors and processors_per_node Settings in the Hosts File"},"3747":{i:0.000140757291294279,u:"../install_guide/job_setup_hosts_parallel.htm",a:"The parallel setting allows you to designate hosts that do or do not support MPI parallel job execution. By default, it is assumed that a host supports MPI parallel jobs: thus, the main need for this setting is to designate hosts that do not support MPI parallel jobs. The setting allows applications ...",t:"The parallel Setting in the Hosts File"},"3748":{i:0.000138853617141738,u:"../install_guide/job_setup_hosts_gpgpu.htm",a:"To make use of GPU cards for Desmond calculations, the hosts file must be configured for each host. For example, on a local workstation, you could use an entry like this: \n# Local workstation\nname: myhostname\nhost: localhost\nprocessors: 4\ntmpdir /usr/tmp\ngpgpu: 0, GTX 680\n This entry indicates that ...",t:"The gpgpu Setting in the Hosts File"},"3749":{i:0.000132858728774832,u:"../install_guide/job_setup_hosts_schrodinger.htm",a:"The schrodinger setting specifies the directory in which your Schrödinger software is installed on this host (the installation directory). If this setting is present, it is used by Job Control as the location of the software installation on the remote machine. If it is not present, the path defined ...",t:"The schrodinger Setting in the Hosts File"},"3750":{i:0.000151473199419947,u:"../install_guide/job_setup_hosts_env.htm",a:"The env setting specifies an environment variable that is to be set on this host when any job is started. The syntax of the setting is variable=value (regardless of the UNIX shell used), where variable is the environment variable and value is its value. For example, \nenv: ...",t:"The env Setting in the Hosts File"},"3751":{i:0.000185294197556274,u:"../install_guide/job_setup_hosts_include_base.htm",a:"The include and base settings offer tools for maintaining a hosts file that can be used across multiple installations of the software. The include setting permits you to include a hosts file in another hosts file. This allows you, for example, to make release-independent settings in one hosts file, ...",t:"The include and base Settings in the Hosts File"},"3752":{i:0.000158179896546893,u:"../install_guide/job_setup_hosts_custom.htm",a:"You can copy and edit the schrodinger.hosts file from the installation directory to customize its settings. The edited hosts file should be placed in your  Schrödinger user resources directory . You usually do not need to do this unless you have different user names on different hosts. If you have ...",t:"Customizing the Hosts File"},"3753":{i:0.00020870219994686,u:"../install_guide/job_setup_remote.htm",a:"Setting Up Access to Remote Hosts To run jobs on remote hosts, you must set up access to these hosts by allowing use of the ssh command without specifying a password. If you have a firewall, you must allow access across the firewall. Setting up access is described in the subsections below. Remote ...",t:"Setting Up Access to Remote Hosts Manually"},"3754":{i:0.000201065706460651,u:"../install_guide/job_setup_remote_linux_mac.htm",a:"The following steps allow you to use ssh between computers that share your login directory without specifying a password.  Open a terminal window. Generate a public/private RSA key pair on a host of your choice, whose home directory is shared with the remote hosts that you want to run jobs on:  cd ...",t:"Setting Up Access Manually From Linux and Mac Hosts"},"3755":{i:0.000247792813117577,u:"../install_guide/job_setup_remote_windows.htm",a:"\n            Setting Up Access from a Windows Host to Linux Hosts On Windows, passwordless SSH is handled using plink.exe, which is provided in the Schrödinger software distribution. To set up passwordless SSH, you can use the Remote Login Configuration tool to guide you through the process, which ...",t:"\n            Setting Up Access from a Windows Host to Linux Hosts"},"3756":{i:0.000194357141466531,u:"../install_guide/job_setup_remote_firewall.htm",a:"\n            Configuring the Firewall If you have a firewall, some configuration needs to be made in order to run remote jobs. If you do not have a firewall, this section is not applicable. Linux and Mac Firewalls For remote job submission, you must open ports above 1024 on the remote (Linux) host. ...",t:"\n            Configuring the Firewall"},"3757":{i:0.000283649439729011,u:"../install_guide/job_setup_clusters.htm",a:"Configuring Clusters The configuration of a cluster to run Schrödinger software must take into account the special issues of communication between the compute nodes, the manager nodes, and the job submission host, and the impact that this communication might have on performance. ",t:"Configuring Clusters"},"3758":{i:0.000251649033278662,u:"../install_guide/job_setup_clusters_requirements.htm",a:"Cluster Configuration Requirements Like any other host, each compute node must have access to a license, the software and the job-related files. The requirements can be stated as follows: Schrödinger software installations must be available to all hosts: the job submission host, the manager node, ...",t:"Configuration Requirements"},"3759":{i:0.000371314337459512,u:"../install_guide/job_setup_clusters_optimization.htm",a:"of Clusters To optimize the performance of a cluster for Schrödinger software, we suggest that you consider the following options when purchasing, upgrading, or configuring a cluster: Invest in a highly capable file server for the external network. Invest in shared storage for the private ...",t:"Performance Optimization"},"3760":{i:0.000659888382171478,u:"../install_guide/job_setup_offline.htm",a:"Configuring Offline Job Management When a user submits a job from a host such as a laptop and the host is taken offline, a staging area for the job output is needed until the host is online again. This is done by defining a job server host, which is simply a host that has a designated nonvolatile ...",t:"Configuring Clusters"},"3761":{i:0.000273698129911309,u:"../install_guide/job_setup_offline_server.htm",a:"Configuring the Job Server for Offline Job Management The job server host must be a machine that can provide a staging area for job input and output while the client is offline. Job control runs a server on this host that manages jobs for all users. Here it is assumed that the execution host is a ...",t:"Configuring Clusters"},"3762":{i:0.000119489649549956,u:"../install_guide/job_setup_offline_client.htm",a:"Configuring the Client for Offline Job Management Once the job server host is set up, you can set up the client to make use of the job server.  Copy the hosts file that you modified on the job server host to the client Schrödinger software installation. Set the following Maestro preferences, which ...",t:"Configuring Clusters"},"3763":{i:0.000245385690537086,u:"../install_guide/job_setup_testing.htm",a:"Testing the Installations and Connections Once you have installed the software and set up the hosts file on the desired hosts, you should test the installation. To do this, you can use the  Diagnostics panel, or you can run checks from the command line.",t:"Testing the Installations and Connections"},"3764":{i:0.000231280749277977,u:"../install_guide/job_setup_testing_diagnostics.htm",a:"The Diagnostics panel runs checks on the hosts file, and can be used to run test applications on hosts to ensure that the connections are working properly. To open the Diagnostics panel, use one of the following methods: Linux: Enter $SCHRODINGER/diagnostics in a terminal window. Windows: Choose ...",t:"The Diagnostics Panel"},"3765":{i:0.000110475666055233,u:"../install_guide/job_setup_testing_command.htm",a:"You can test the connections to remote hosts with the installation_check application. This application reads a hosts file and runs a test job using the host entries defined in the file. Errors in the hosts file and failures in the test jobs are reported and (to the extent possible) recommendations ...",t:"Command-Line Tools for Testing the Installation"},"3766":{i:0.000389041002508807,u:"../install_guide/batch_queue.htm",a:"Schrödinger supplies support for the PBS Pro, Grid Engine family (Son of Grid Engine, Open Grid Scheduler, and Univa Grid Engine), LSF, SLURM, and Torque queuing systems in the standard software installation. Below are links to information about these queuing systems: PBS Pro:  ...",t:"Preparing for Batch Queue Submission"},"3767":{i:0.000274063570475521,u:"../install_guide/job_setup_batch_hosts.htm",a:"To enable job submissions to a batch queue on a supported queueing system, you must add host entries that define the available queues to the schrodinger.hosts file. The command syntax is described in  The Hosts File . There are two settings that define the queue: the Queue setting and the Qargs ...",t:"Setting Up the Hosts File for Batch Queues"},"3768":{i:0.000150504180316539,u:"../install_guide/batch_queue_access.htm",a:"Support for queueing systems is provided through a set of scripts and a template that contain the specific syntax required to submit jobs to a queue. These files are installed in a subdirectory of the $SCHRODINGER/queues directory that is named after the queue for each of the supported queueing ...",t:"Setting Up Access to a Queueing System"},"3769":{i:0.000267130315878106,u:"../install_guide/batch_queue_license_check.htm",a:"On large clusters, it is possible for jobs to fail because there are too few licenses for all the jobs to run. A mechanism has been provided to ensure that a queued job will not run unless all the licenses it requires are available. This facility is available for the SGE 6.0, PBS Pro 10.4, and ...",t:"Setting Up License Checking for Queueing Systems"},"3770":{i:0.000152882019383796,u:"../install_guide/batch_queue_GPU.htm",a:"Queuing systems must assign the appropriate resources to the GPU. This can be a bit challenging, as not all queueing systems fully manage GPU resources. Desmond MD jobs run on a GPU require 1 CPU per GPU. The %NPROC% placeholder will be substituted in the hosts file for the number of GPUs requested ...",t:"Setting Up Queuing Systems for GPU Use"},"3771":{i:0.000189557422777028,u:"../install_guide/batch_queue_openmpi.htm",a:"Several programs distributed by Schrödinger (Desmond, Piper, and Quantum ESPRESSO) can use Open MPI for parallel execution, and can operate with a number of queuing systems. Open MPI provides tight integration that is compatible with many queuing systems via the PLS (Process Launch Subsystem) and ...",t:"Setting Up Queuing Systems for Open MPI Parallel Execution"},"3772":{i:0.000169347631841018,u:"../install_guide/batch_queue_sge.htm",a:"Setup for Grid Engine Queueing Systems First, set up the config file in your Schrödinger software installation: Edit the file $SCHRODINGER/queues/SGE/config. Set the QPATH keyword to the full path to the bin directory of the queueing software installation. Set the QPROFILE keyword in the config file ...",t:"Setup for SGE Queueing Systems"},"3773":{i:0.000191676067938504,u:"../install_guide/batch_queue_sge_license_check.htm",a:"License Checking for Grid Engine On large clusters, it is possible for jobs to fail because there are too few licenses for all the jobs to run. A mechanism has been provided to ensure that a queued job will not run unless all the licenses it requires are available. This facility is supported for the ...",t:"License Checking for SGE"},"3774":{i:0.000157122256015582,u:"../install_guide/batch_queue_sge_license_check_attributes.htm",a:"Adding License Attributes for Grid Engine A resource attribute representing each license must be defined in the queuing system configuration. The utility licutil can be used to parse your license file or query the license server and generate the configuration text that needs to be added to the ...",t:"Adding License Attributes for SGE"},"3775":{i:0.000146411737746257,u:"../install_guide/batch_queue_sge_license_check_configure.htm",a:"Configuring the Load Scripts for Grid Engine A script is provided that queries the FLEXlm license server and reports the number of tokens available for each license in a format that the queueing software can understand. To make sure the lmutil utility and license file can be found when the script is ...",t:"Configuring the Load Scripts for SGE"},"3776":{i:0.00014316664571841,u:"../install_guide/batch_queue_sge_license_check_install.htm",a:"Installing the Load Script for Grid Engine To install flexlm_sensor.pl, you need to add a load_sensor attribute to the host configuration of one of the hosts managed by Grid Engine. The load sensor is executed periodically on that host and feeds information about license availability back into the ...",t:"Installing the Load Script for SGE"},"3777":{i:0.000142425009051558,u:"../install_guide/batch_queue_sge_license_check_turnon.htm",a:"Turning On License Checking in Job Control for Grid Engine The final step is to instruct Job Control to specify license requirements when submitting jobs to the queue. This step must be done after the steps above, otherwise jobs submitted to the queue will fail because the license software will ...",t:"Turning On License Checking in Job Control for SGE"},"3778":{i:0.00014305285855183,u:"../install_guide/batch_queue_sge_gpu.htm",a:"GPU Configuration for Grid Engine Here is an example that can be used for GPU jobs in a very simple scheduling environment: \n# Grid Engine\nname: sge-gpu\nhost: headnode.mycluster.com\nqueue: SGE\nqargs: -q myqueue.q -pe smp %NPROC%\ntmpdir: /usr/local/tmp\nschrodinger: /opt/bin/schrodinger2017-1\ngpgpu: ...",t:"GPU Configuration for SGE"},"3779":{i:0.000146010189786878,u:"../install_guide/batch_queue_sge_openmpi.htm",a:"OpenMPI Configuration for Grid Engine Any Grid Engine queue to which parallel jobs are submitted must be configured to support multi-processor jobs. The schrodinger.hosts file must include an entry for each queue, which should look like the following:  \nName: my-queue-name\nQueue: SGE \nQargs: -q ...",t:"OpenMPI Configuration for SGE"},"3780":{i:0.000155818980019718,u:"../install_guide/batch_queue_pbspro.htm",a:"First, you must set up the config file in your Schrödinger software installation: Edit the file $SCHRODINGER/queues/PBS10.4/config (or the file in the relevant versioned subdirectory). Set the QPATH keyword to the bin directory of your PBS Pro queueing software installation. No instructions are ...",t:"Setup for PBS Pro Queueing Systems"},"3781":{i:0.000187478264589323,u:"../install_guide/batch_queue_pbspro_license_check.htm",a:"On large clusters, it is possible for jobs to fail because there are too few licenses for all the jobs to run. A mechanism has been provided to ensure that a queued job will not run unless all the licenses it requires are available. This facility is supported for the PBS Pro queueing system from ...",t:"License Checking for PBS Pro"},"3782":{i:0.000155836319712032,u:"../install_guide/batch_queue_pbspro_license_check_attributes.htm",a:"A resource attribute representing each license must be defined in the queueing system configuration. The utility licutil can be used to parse your license file or query the license server and generate the configuration text that needs to be added to the queueing system configuration files. The ...",t:"Adding License Attributes for PBS Pro"},"3783":{i:0.000146071377852442,u:"../install_guide/batch_queue_pbspro_license_check_configure.htm",a:"A script is provided that queries the FLEXlm license server and reports the number of tokens available for each license in a format that the queueing software can understand. To make sure the lmutil utility and license file can be found when the script is run by the queueing software, you need to ...",t:"Configuring the Load Script for PBS Pro"},"3784":{i:0.000143251445850906,u:"../install_guide/batch_queue_pbspro_license_check_install.htm",a:"The location of the load sensor script, pbs_lic_sensor.pl, is specified in the sched_priv/sched_config file (see  Configuring the Load Script for PBS Pro ). The configuration produced by licutil points to the location of this script within the Schrödinger installation, ...",t:"Installing the Load Script for PBS Pro"},"3785":{i:0.000143064640013234,u:"../install_guide/batch_queue_pbspro_license_check_turnon.htm",a:"The final step is to instruct Job Control to specify license requirements when submitting jobs to the queue. This step must be completed after the steps above, otherwise jobs submitted to the queue will fail because the license software will assume that the required licenses are not available. Open ...",t:"Turning On License Checking in Job Control for PBS Pro"},"3786":{i:0.000145225388437843,u:"../install_guide/batch_queue_pbspro_openmpi.htm",a:"The TM API is used to allocate slots and launch processes in Open MPI for all of the family of PBS queuing systems. For more information on running jobs under PBS Pro, check the OpenMPI FAQs at  http://www.open-mpi.org/faq/?category=tm . Tight integration for PBS Pro 9.1 and later should work ...",t:"OpenMPI Configuration for PBS Pro"},"3787":{i:0.000153037784889235,u:"../install_guide/batch_queue_lsf.htm",a:"Setup for LSF Queuing Systems First, you must set up the config file in your Schrödinger software installation: Edit the file $SCHRODINGER/queues/LSF/config. Set the QPATH keyword to the full path to the bin directory of your LSF queuing software installation. Set the QPROFILE keyword in the config ...",t:"Setup for LSF Queueing System"},"3788":{i:0.000178448386576153,u:"../install_guide/batch_queue_lsf_license_check.htm",a:"On large clusters, it is possible for jobs to fail because there are too few licenses for all the jobs to run. A mechanism has been provided to ensure that a queued job will not run unless all the licenses it requires are available. This facility is supported for the Platform LSF queuing system from ...",t:"License Checking for LSF"},"3789":{i:0.000151807773922437,u:"../install_guide/batch_queue_lsf_license_check_attributes.htm",a:"A resource attribute representing each license must be defined in the queueing system configuration. The utility licutil can be used to parse your license file or query the license server and generate the configuration text that needs to be added to the queueing system configuration files. The ...",t:"Adding License Attributes for LSF"},"3790":{i:0.000140884651612721,u:"../install_guide/batch_queue_lsf_license_check_configure.htm",a:"A script is provided that queries the FLEXlm license server and reports the number of tokens available for each license in a format that the queueing software can understand. To make sure the lmutil utility and license file can be found when the script is run by the queueing software, you need to ...",t:"Configuring the Load Script for LSF"},"3791":{i:0.000128975777001281,u:"../install_guide/batch_queue_lsf_license_check_install.htm",a:"Copy the edited script into the $LSF_SERVERDIR directory on the LSF master host: cp $SCHRODINGER/utilities/elim.schrodinger $LSF_SERVERDIR The $LSF_SERVERDIR directory is typically named something like /usr/lsf/6.2/linux2.6-glibc2.3-x86/etc/ and also contains the program melim, which manages ELIMs.  ...",t:"Installing the Load Script for LSF"},"3792":{i:9.78702584208908E-05,u:"../install_guide/batch_queue_lsf_license_check_turnon.htm",a:"The final step is to instruct Job Control to specify license requirements when submitting jobs to the queue. This step must be done after the steps above, otherwise jobs submitted to the queue will fail because the license software will assume that the required licenses are not available. Open the ...",t:"Turning On License Checking in Job Control for LSF"},"3793":{i:0.000127980634742815,u:"../install_guide/batch_queue_lsf_gpu.htm",a:"First, you must configure LSF for GPU resource usage, according to the instructions in the LSF documentation. Once configured, you can set up hosts file entries for use by Schrödinger software.To configure LSF you will need to define the GPU resources using the native support as defined on the LSF ...",t:"GPU Setup for LSF"},"3794":{i:0.000137372825603575,u:"../install_guide/batch_queue_lsf_openmpi.htm",a:"Loose integration for LSF should work without change. Tight integration is supported by LSF from version 7.0.2 on. It is generally recommended to use a single node. Here is an example of a hosts file entry: \n# Request %NPROC% processors on a single node\nName: label\nQueue: LSF\nQargs: -q LSF-queue -n ...",t:"OpenMPI Setup for LSF"},"3795":{i:0.000140409506506492,u:"../install_guide/batch_queue_slurm.htm",a:"Setup for SLURM Queueing Systems First, you must set up the config file in your Schrödinger software installation: Edit the file $SCHRODINGER/queues/SLURM2.1/config. Set QPATH to the bin directory of the SLURM queueing software installation. No instructions are given here for installing and general ...",t:"Configuring a SLURM Queueing System"},"3796":{i:0.000141734259979058,u:"../install_guide/batch_queue_slurm_openmpi.htm",a:"OpenMPI Setup for SLURM Below are some examples of SLURM entries in the hosts file. \n# Request %NPROC% processors on a single node\nName: label\nQueue: SLURM2.1\nQargs: --partition=SLURM-partition --nodes=1 --ntasks-per-node=%NPROC%\nHost: submit-node\nProcessors: processors-in-queue\n \n# Request %NPROC% ...",t:"OpenMPI configuration"},"3797":{i:0.000143372857883683,u:"../install_guide/batch_queue_torque.htm",a:"First, you must set up the config file in your Schrödinger software installation: Edit the file $SCHRODINGER/queues/Torque/config. Set the QPATH keyword to the bin directory of your Torque queueing software installation. No instructions are given here for installing and general configuration of the ...",t:"Setup for Torque Queueing Systems"},"3798":{i:0.00014783121299618,u:"../install_guide/batch_queue_torque_gpu.htm",a:"Here is an example that can be used for GPU jobs in a very simple scheduling environment: \n# Torque\nname: torque-gpu\nhost: headnode.mycluster.com\nqueue: Torque\nqargs: -q myqueue -l nodes=1:ppn=%NPROC%\ntmpdir: /usr/local/tmp\nschrodinger: /opt/bin/schrodinger2017-1\ngpgpu: 0, Tesla K40\ngpgpu: 1, Tesla ...",t:"GPU Configuration for Torque"},"3799":{i:0.000107951859275803,u:"../install_guide/batch_queue_torque_openmpi.htm",a:"The TM API is used to allocate slots and launch processes in Open MPI for all of the family of PBS queuing systems. For more information of running jobs under Torque, check the OpenMPI FAQs at  http://www.open-mpi.org/faq/?category=tm . Due to a naming conflict, components for Torque have been ...",t:"OpenMPI Configuration for Torque"},"3800":{i:0.000529575300067116,u:"../install_guide/setting_envvar.htm",a:"Setting Environment Variables This appendix describes how to set an environment variable on the supported platforms. ",t:"Setting Environment Variables"},"3801":{i:0.000404444028868802,u:"../install_guide/setting_envvar_windows.htm",a:"You can create or change environment variables in the Environment Variables dialog box. If you are adding to the PATH environment variable, you should separate each field with a semicolon (;).  To open the Environment Variables dialog box: Click Start, then click Control Panel. The Control Panel ...",t:"Setting Environment Variables on Windows"},"3802":{i:0.00022964722560625,u:"../install_guide/setting_envvar_mac.htm",a:"To set an environment variable on Mac OSX, first open a terminal window. If you are setting the environment variable to run jobs from the command line in a terminal window, use the following command: export variable=value where variable is the name of the environment variable (such as SCHRODINGER) ...",t:"Mac OSX"},"3803":{i:0.000126396469093253,u:"../install_guide/setting_envvar_linux.htm",a:"Linux To set an environment variable on Linux, enter the following command at a shell prompt: where variable is the name of the environment variable (such as SCHRODINGER) and value is the value you want to assign to the variable, (such as /opt/schrodinger2017-1).  ",t:"Linux"},"3804":{i:6.13254596701412E-05,u:"../install_guide/windows_locations.htm",a:"This topic contains information on where various resources or files are kept on Windows. These are relative to the standard application data location (%APPDATA%), local application data location (%LOCALAPPDATA%), and user location (%USERPROFILE)%), whose definitions are listed below. %USERPROFILE% ...",t:"Schrödinger File and Resource Locations on Windows"},"3805":{i:0.000204081777310533,u:"../install_guide/web_proxy.htm",a:"Some Schrödinger applications can retrieve information from the web. If you are using a proxy server for web access, you may have to perform some configuration to allow these applications access to the web. For such applications, a script has been provided that creates and stores the necessary ...",t:"Access to the Web via a Proxy for Schrödinger Applications"},"3806":{i:6.13254596701412E-05,u:"../job_control/jobcontrol_intro.htm",a:"Nearly all computational jobs launched from Maestro are run under Schrödinger’s Job Control facility. The Job Control facility provides a uniform mechanism for launching, monitoring and controlling calculations, both for jobs launched from Maestro and for jobs launched from the command line. The Job ...",t:"The Job Control Facility"},"3807":{i:0.000200453455387844,u:"../job_control/running_jobs.htm",a:"Jobs can be submitted to a designated host from a Maestro session or from the command line. The details of setting up the data in Maestro or submitting jobs from the command line for a given product are described in the documentation for the product. However, there are some common elements that ...",t:"Running Jobs"},"3808":{i:0.000349647555183739,u:"../job_control/running_jobs_life_cycle.htm",a:"To understand how information is obtained and passed on by Job Control, it is useful to have an understanding of how Schrödinger jobs are run. The “life cycle” of a job can be summarized as follows. A top-level script is run locally (on the submission host). This script parses command-line arguments ...",t:"The Job Life Cycle"},"3809":{i:0.00117957000371586,u:"../job_control/running_jobs_command_line.htm",a:"For most purposes, you can start jobs from Maestro. You can also run or submit jobs from the command line on Linux hosts or from a Schrödinger shell on Windows hosts (see below). The Job Control facility recognizes a number of command-line options that can be used to control the behavior of the job. ...",t:"Running Jobs From the Command Line"},"3810":{i:0.00030252450653555,u:"../job_control/running_jobs_command_line_scripts.htm",a:"You can run scripts from the command line with the following command: \n$SCHRODINGER/run scriptname [-FROM product] [-FIND|-FINDALL] [options] \n This command runs a top-level script like those for Schrödinger products, which recognizes the Job Control options listed in the tables above (except for  ...",t:"Running Scripts"},"3811":{i:0.000191321661520945,u:"../job_control/running_jobs_command_line_host_options.htm",a:"Jobs can be submitted to a remote host using the -HOST option to specify the remote host name. This name must be the value of a name setting in the hosts file (a “host entry name”), or an actual host name. For example:  $SCHRODINGER/bmin -HOST hostjobname For programs that can run a single job in ...",t:"The HOST, DRIVERHOST, and SUBHOST Options"},"3812":{i:0.000156255356223657,u:"../job_control/running_jobs_command_line_wait_option.htm",a:"All jobs are run in the background automatically, dissociated from the terminal session or application from which they were launched. As a result, the job continues to run even if you quit Maestro or the terminal session from which you launched the job. For command-line jobs, this means that the ...",t:"The WAIT option"},"3813":{i:0.000191535522348191,u:"../job_control/running_jobs_command_line_local_option.htm",a:"For some applications, you can request temporary files to be written to the submission directory (the directory from which you submitted the job), instead of to the job directory, which is a subdirectory of the scratch directory. This is done with the -LOCAL option. For example, $SCHRODINGER/bmin ...",t:"The LOCAL Option"},"3814":{i:0.000195295895091372,u:"../job_control/running_jobs_command_line_hosts_file.htm",a:"When you start a job from the command line, one of the first tasks of Job Control is to locate a hosts file. The hosts file used for a given job is the first one found in the following list: The file specified by the environment variable SCHRODINGER_HOSTS The schrodinger.hosts file in the current ...",t:"Location of the Hosts File for a Job"},"3815":{i:0.000143984086062021,u:"../job_control/running_jobs_scratch_dir.htm",a:"Most jobs now run in a scratch directory by default, rather than in the directory from which the job was started (the submission directory, also called the launch directory). When a job runs in a scratch directory, a subdirectory is created in it for the job, named tmpdir/username/unique_name. Here, ...",t:"Location of the Scratch Directory"},"3816":{i:0.000128566162126012,u:"../job_control/running_jobs_software_version.htm",a:"When a job is started, the Job Control facility checks that the software installation specified for the job exists and that the software (mmshare) version matches that on the submission host.  The software installation used on the execution host is determined by the following rules: The installation ...",t:"Software Version Selection for a Job"},"3817":{i:0.000129320305735511,u:"../job_control/running_jobs_license_location.htm",a:"When an executable starts or the queueing software checks the availability of licenses, the following locations are searched, in order, for the license file; the first one found is used. $SCHROD_LICENSE_FILE $LM_LICENSE_FILE $SCHRODINGER/licenses/ $SCHRODINGER/license or $SCHRODINGER/license.txt. ...",t:"License File Location"},"3818":{i:0.000143545113689789,u:"../job_control/running_jobs_environment_variables.htm",a:"Environment variables that can be used on the execution host can come from the following sources: Environment variables set in the shell startup script (.bashrc, .cshrc, and so on) \n     Environment variables set on the submission host when the job is launched.\n     Environment variables set in the ...",t:"Environment Variables Used on the Execution Host"},"3819":{i:0.000160850775113944,u:"../job_control/running_jobs_input_output.htm",a:"When you run a job from the command line you can specify input files in either of two ways: using absolute paths or relative paths. The way Job Control treats these differs. The configuration considerations for output files and for updates of the Job Control database are similar to those for input ...",t:"Input and Output Files"},"3820":{i:0.000415396070197518,u:"../job_control/running_jobs_incorporation.htm",a:"When Maestro project entries are used as input for jobs, the structure and property output can be incorporated into the project. The output files are always copied back to the submission directory: the incorporation options affect what is done in the Maestro project. You can specify how you want the ...",t:"Incorporation of Job Output"},"3821":{i:0.000177945580812278,u:"../job_control/jobcontrol.htm",a:"The Job Control facility provides tools for monitoring and controlling the jobs that it runs. These tools have both a graphical user interface in the Maestro Monitor panel and a command-line interface in the jobcontrol command. The common tasks can be performed from the Monitor panel, but the full ...",t:"Managing Jobs"},"3822":{i:0.000215239415471779,u:"../job_control/jobcontrol_database.htm",a:"Information about each job is kept in the user’s job database. The database contains a record for each job. You can determine which jobs are in the database by using the Maestro Monitor panel or the command-line jobcontrol utility.  By default, this database is kept in the directory ...",t:"The Job Database"},"3823":{i:0.000255950206654755,u:"../job_control/jobcontrol_database_job_record.htm",a:"Each job has a job record in the job database. The job record is a list of fields, one on each line, each consisting of a field name and its value. Many of these fields contain information that is only useful to the job control system, but a number may also be useful to users. Some of the latter are ...",t:"The Job Record"},"3824":{i:0.000182243014309952,u:"../job_control/jobcontrol_database_job_status.htm",a:"Every job proceeds through a series of well-defined stages (see  The Job Life Cycle ). The current stage of a job is displayed in the Status column of the Monitor panel. The Status of a completed job indicates the conditions under which it stopped. The job status descriptors that can appear in the ...",t:"Job Status"},"3825":{i:0.000167247433359595,u:"../job_control/jobcontrol_command_line.htm",a:"The job control utility allows you to perform a number of job control tasks from the command line. The syntax for job control utility commands is: $SCHRODINGER/jobcontrol action [options] query  where action is the command for the action you want to perform, options qualifies the scope of the ...",t:"Managing Jobs From the Command Line"},"3826":{i:0.00013159895061821,u:"../job_control/jobcontrol_command_line_queries.htm",a:"More general queries than those given above are also possible. Formally, a query consists of one or more conditions, optionally separated by the Boolean operators AND, OR or NOT. If the operators are omitted, OR is assumed. A condition takes the form \u003cfield\u003e\u003cop\u003e\u003cvalue\u003e, where \u003cfield\u003e is one of the ...",t:"General Job Control Queries"},"3827":{i:0.000122576419892419,u:"../job_control/jobcontrol_command_line_stranded.htm",a:"Job Control periodically tries to re-establish contact with stranded jobs. It is also possible to re-establish contact with the job manually, with the command: \n$SCHRODINGER/jobcontrol -recover jobid If this command fails, however, the job is cleaned up and the results are no longer available. If ...",t:"Recovering Stranded Jobs"},"3828":{i:0.00015662641149251,u:"../job_control/jobcontrol_command_line_jserver.htm",a:"\n            Using jserver\n The jserver process on the submission host sets up and manages the connection to remote hosts. Normally it is started automatically. If it has stopped, you can restart it on a UNIX system with the following command: \n$SCHRODINGER/utilities/jserver [-proxy] [action] ...",t:"\n            Using jserver\n"},"3829":{i:0.000211319876508685,u:"../job_control/jobcontrol_command_line_purging.htm",a:"If the job database is not purged, job records can accumulate and the job database directory can become quite large over time. The jobcontrol utility can be used to purge the job database of records for completed jobs. For example, the following command purges the entire database: ...",t:"Purging the Job Database"},"3830":{i:0.000238604353436909,u:"../job_control/jobcontrol_command_line_jnanny.htm",a:"\n            Performing Actions on Jobs with jnanny\n The jnanny script scans the job database and performs tasks defined by plugin modules. These tasks include checking for jobs that are stranded or appear to be stuck in a particular state, and then initiating a recovery process and updating the job ...",t:"\n            Performing Actions on Jobs with jnanny\n"},"3831":{i:0.000299558936512142,u:"../job_control/postmortem.htm",a:"\n            The postmortem Utility\n The postmortem utility archives information that is useful for understanding why jobs do not run as expected. It creates a zip archive containing the current environment, file system information, a list of installed Schrodinger software packages, the ...",t:"\n            The postmortem Utility\n"},"3832":{i:0.000519856615383878,u:"../job_control/jobcontrol_envvar.htm",a:"There are a number of environment variables that can be used to specify the location of resources, manage the job database, and control various aspects of job execution. These environment variables are given in  Table 1 . Some of these environment variables provide alternate means of specifying the ...",t:"Job Control Environment Variables"},"3833":{i:0.00157858206533338,u:"../job_control/job_control_glossary.htm",a:"entry—(1) A collection of settings in the hosts file that defines a configuration for running jobs on a given host. There can be more than one entry for a given host, with different settings.  (2) A structure and its properties in a Maestro project. execution host —The computer that a job runs on. ...",t:"Job Control Glossary"},"3834":{i:0.000352159458180274,u:"../autoqsar_help/autoqsar.html",a:"Generate QSAR models for a chosen property of a set of compounds, and apply the QSAR model to other compounds. Descriptor data and fingerprints are automatically generated. Multiple QSAR models are generated with different methods, the models are ranked, and the best models are returned. To open ...",t:"AutoQSAR Panel"},"3835":{i:0.000181035945454,u:"../autoqsar_help/autoqsar_advanced_options.html",a:"Make additional AutoQSAR settings, for definition of categorical properties from numerical data, for model building, and for manual selection of independent variables. AutoQSAR - Advanced Options Dialog Box Features Define Categories section Number of categories box Category distribution options ...",t:"AutoQSAR - Advanced Options Dialog Box"},"3836":{i:0.000181035945454,u:"../autoqsar_help/autoqsar_validation_set.html",a:"Select a set of structures to use for validation of the QSAR models. The structures are not used in any way in model development, so they are a true test set. Select Validation Set Dialog Box Features Select structures for separate validation set options Use random validation set option and box ...",t:"Select Validation Set Dialog Box"},"3837":{i:0.000104090622390322,u:"../autoqsar_help/autoqsar_properties.html",a:"AutoQSAR Output Properties The table below lists the Maestro properties that are added when a prediction is made for a structure. The value of label in the property name is the label you set in the AutoQSAR Prediction text box. Related Topics AutoQSAR Panel",t:"AutoQSAR Output Properties"},"3838":{i:0.000137943484458883,u:"../bioluminate_help/aggregation_profiles.html",a:"Display bar charts of aggregation-related properties on a per-residue basis. The properties are the Aggrescan and Zyggregator scores, the Side chain accessible surface area ratio, and the three contributions to the energy score of the patch (positive, negative, and hydrophobic).  To open this panel, ...",t:"Aggregation Profiles Panel"},"3839":{i:0.00010665618148832,u:"../bioluminate_help/aggregation_surface_options.html",a:"In this dialog box you can set parameters that control the quality and appearance of the surface, and the parameters for locating aggregation sites. Aggregation Surface - Options Dialog Box Features Surface grid spacing box Probe radius box Transparency box Radius to find neighbors box Hydrophobic ...",t:"Aggregation Surface - Options Dialog Box"},"3840":{i:0.00010665618148832,u:"../bioluminate_help/aggregation_surface.html",a:"In this panel you can generate a surface that indicates regions with a propensity for aggregation. To open this panel, choose Tasks → Biologics → Aggregation Surface in the Task Tool. Using the Aggregation Surface Panel Aggregation regions are defined by identifying clusters of exposed hydrophobic ...",t:"Aggregation Surface Panel"},"3841":{i:0.00063404536740914,u:"../bioluminate_help/antibody_database_options.html",a:"Set options to use when importing structures into the antibody database. The options include specification of file types when importing from a directory, and setting of criteria for determining whether a structure is an antibody or not. The determination is based on sequence similarity between the ...",t:"Antibody Database - Advanced Options Dialog Box"},"3842":{i:0.000714501020483869,u:"../bioluminate_help/antibody_database.html",a:"In this panel you can create and edit antibody databases, export data, and set the default database to use for prediction tasks. To open this panel, choose Tasks → Biologics → Antibody Modeling → Database Management in the Task Tool. Using the Antibody Database Management Panel In this panel you can ...",t:"Antibody Database Management Panel"},"3843":{i:0.000241101433159994,u:"../bioluminate_help/antibody_cdr_grafting.html",a:"Graft the CDR loops of a murine antibody onto a human framework, with mutations around the join points of the loops to improve the geometry. To open this panel, choose Tasks → Biologics → Humanization → CDR Grafting in the Task Tool. Using the Antibody Humanization: CDR Grafting Panel Before you can ...",t:"Antibody Humanization: CDR Grafting Panel"},"3844":{i:0.000142659178177427,u:"../bioluminate_help/antibody_mutation_homology_suggestions.html",a:"Identify candidate residues for mutation, using homology modeling or structural criteria. Homology modeling can be used to select residues by variability or conservation. Structural criteria can be applied for solvent accessible surface area and interactions (contacts) with the protein or a selected ...",t:"Antibody Humanization: Homology Suggestions Dialog Box"},"3845":{i:0.000288761576038461,u:"../bioluminate_help/antibody_residue_mutation.html",a:"This panel allows you to humanize an antibody by mutating residues to create a more human-compatible protein. Appropriate sites for mutation can be found by comparing the antibody to homologs and identifying residues that satisfy various criteria, such as solvent accessibility, maximum number of ...",t:"Antibody Humanization: Residue Mutation Panel"},"3846":{i:0.000375893857014363,u:"../bioluminate_help/antibody_prediction.html",a:"This panel provides a set of tools for prediction of the variable loop (CDR) region of an antibody. It includes: Automated identification of the six CDR region loops based on sequence Prediction of the loops using lookup tables and rules derived from an analysis of antibodies in the PDB (based on a ...",t:"Antibody Prediction Panel"},"3847":{i:0.000157578139969178,u:"../bioluminate_help/antibody_cdr_grafting_options.html",a:"Set options for additional regions to graft and for customizing the CDR loop region. To open this dialog box, click the Advanced Options button in the Antibody Humanization: CDR Grafting panel.\n CDR Grafting - Advanced Options Dialog Box Features Numbering scheme text Additional regions to graft ...",t:"CDR Grafting - Advanced Options Dialog Box"},"3848":{i:7.78743621210405E-05,u:"../bioluminate_help/choose_database.html",a:"In this dialog box you can choose the databases to use for antibody prediction or CDR grafting. To open this dialog box, click Choose Databases in the Framework Search tab of the Antibody Prediction panel or in the Antibody Humanization: CDR Grafting panel.\n Using the Choose Database Dialog Box To ...",t:"Choose Database Dialog Box"},"3849":{i:7.78743621210405E-05,u:"../bioluminate_help/consensus_visualization.html",a:"In this panel, you can identify structural elements, such as ligands, waters, or counter ions, that are in the same position in a set of related proteins. To open this panel, choose Tasks → Biologics → Consensus Viewer in the Task Tool. Using the Consensus Visualization Panel This panel is intended ...",t:"Consensus Visualization Panel"},"3850":{i:8.55716728406209E-05,u:"../bioluminate_help/crosslink_proteins.html",a:"Cross-link two protein chains with a peptide linker connecting the termini. The linker can be composed of several copies of a specified monomer unit. A loop prediction is performed on the linker to obtain a reasonable conformation. To open this panel, choose Tasks → Biologics → Crosslink Proteins in ...",t:"Crosslink Proteins Panel"},"3851":{i:0.000149225547727748,u:"../bioluminate_help/cysteine_mutation.html",a:"This panel allows you to find residues that can be mutated to Cys and form disulfide bonds, and to mutate Cys to other residues to break disulfide bonds. To open this panel, choose Tasks → Biologics → Cysteine Mutation in the Task Tool. Using the Cysteine Mutation Panel The basic workflow for using ...",t:"Cysteine Mutation Panel"},"3852":{i:0.000179788270003978,u:"../bioluminate_help/homology_model.html",a:"In this panel you can create a simple homology model from a query and a template. No alignment is done, so the sequence to be modeled must have a very high homology with the template. To open this panel, choose Tasks → Biologics → Simple Homology Modeling in the Task Tool. Using the Homology Model ...",t:"Homology Model Panel"},"3853":{i:0.000157578139969178,u:"../bioluminate_help/antibody_cdr_grafting_import_seq.html",a:"Import sequences for the light and heavy human framework regions and the human J region. To open this dialog box, choose Framework Sequences from the Import framework from option menu and click the Browse button next to it, in the Antibody Humanization: CDR Grafting panel.\n Import Framework ...",t:"Import Framework Sequences Dialog Box"},"3854":{i:0.000119725343903197,u:"../bioluminate_help/insertion_deletion_loop_swap.html",a:"In this panel you can specify a residue to be inserted or deleted, or you can define a string of residues to be modified in a loop swap. Because insertions or deletions require additional refinement of the residues around the modification site, all modifications made by this panel require a loop ...",t:"Insertions, Deletions, and Loop Swaps Panel"},"3855":{i:0.000217807383493845,u:"../bioluminate_help/loop_clusters.html",a:"In this dialog box you can examine the loop clusters for each of the CDR regions, filtered by similarity, and select a cluster to compute the loop. To open this panel, choose Tasks → Biologics → Antibody Modeling → Prediction in the Task Tool. Using the Loop Clusters Dialog Box When this dialog box ...",t:"Loop Clusters Dialog Box"},"3856":{i:0.000407685701006683,u:"../bioluminate_help/low_mode_sampling.html",a:"Calculate and visualize the low vibrational modes of a structure. To open this panel, choose Tasks → Biologics → Low Normal Mode Analysis → Calculate in the Task Tool. Using the Low Mode Vibrational Sampling Panel Visualizing the low vibrational modes of a protein can help in identifying domain ...",t:"Low Mode Vibrational Sampling Panel"},"3857":{i:0.000341039132698163,u:"../bioluminate_help/nonstandard_residues.html",a:"Set up a database of nonstandard residues, by building new residues or importing existing residues, and choose the nonstandard residues that are shown in the residue selection tools for residue scanning. To open this panel, choose Tasks → Biologics → Residue Scanning/Affinity Maturation → Manage ...",t:"Nonstandard Residues Panel"},"3858":{i:8.55716728406209E-05,u:"../bioluminate_help/peptide_docking.html",a:"This panel allows you to set up and run a job to dock peptides to a protein using Glide. The protein is treated as a rigid structure, but with some softening of the potentials for flexibility; the peptides are treated as flexible structures. To open this panel, choose Tasks → Biologics → Peptide ...",t:"Peptide Docking Panel"},"3859":{i:0.000407685701006683,u:"../bioluminate_help/peptide_helicity.html",a:"Predict the alpha helical tendency of one or more peptide sequence using molecular dynamics. To open this panel, choose Tasks → Biologics → Peptide Alpha Helicity in the Task Tool. Using the Peptide Helicity Panel The purpose of this panel is to predict the stability of alpha-helices. The sequences ...",t:"Peptide Helicity Panel"},"3860":{i:0.000364233386507755,u:"../bioluminate_help/peptide_qsar_load.html",a:"Load peptide sequences and the observable for each sequence that is being fit or predicted. To open this panel, click Load Sequences and Observables in the Setup tab of the Peptide QSAR panel. Peptide QSAR - Load Sequences and Observables Dialog Box Features The features shown in this dialog box ...",t:"Peptide QSAR - Load Sequences and Observables Dialog Box"},"3861":{i:0.000265965087301902,u:"../bioluminate_help/peptide_qsar.html",a:"Build a QSAR model for the properties of peptides, based on descriptors for individual residues in the peptide sequence. The sequences must be of the same length. To open this panel, choose Tasks → Biologics → Peptide QSAR in the Task Tool. Using the Peptide QSAR Panel This panel can be used to ...",t:"Peptide QSAR Panel"},"3862":{i:0.000265965087301902,u:"../bioluminate_help/peptide_qsar_kpls_options.html",a:"Set options for the kernel-based partial least-squares procedure for fitting the descriptors. To open this panel, click Advanced Options for KPLS in the Setup tab of the Peptide QSAR panel. Advanced Options - KPLS Dialog Box Features Maximum number of KPLS factors box Kernel nonlinearity slider, ...",t:"Peptide QSAR — Advanced Options - KPLS Dialog Box"},"3863":{i:0.000265965087301902,u:"../bioluminate_help/peptide_qsar_pls_options.html",a:"Set options for the partial least-squares procedure for fitting the descriptors. To open this panel, click Advanced Options for PLS in the Setup tab of the Peptide QSAR panel. Advanced Options - PLS Dialog Box Features Maximum number of PLS factors box Autoscale X variables option Stop adding PLS ...",t:"Peptide QSAR — Advanced Options - PLS Dialog Box"},"3864":{i:0.000265965087301902,u:"../bioluminate_help/peptide_qsar_results.html",a:"This tab displays the results of building the model: statistics for the training and test set, and predicted values for the peptides. Results Tab Features PLS/KPLS factors box Statistics tables Results table Export Predictions button Export Model button Plot button Reset button PLS/KPLS factors box ...",t:"Peptide QSAR — Results Tab"},"3865":{i:0.000646790729158221,u:"../bioluminate_help/peptide_qsar_setup.html",a:"In this tab you import the sequences and observable to be fit, choose a test and training set and set options for the model, then build the model. You can also apply a previously built model. Setup Tab Features Load Sequences and Observables button Sequence table Set selected rows as option menu and ...",t:"Peptide QSAR — Setup Tab"},"3866":{i:0.000139517366514962,u:"../bioluminate_help/protein_interaction_analysis.html",a:"Analyze the interactions at the interface of two proteins, defined as two sets of chains. The analysis locates residues in the second set of chains that are within a given distance of residues in the first set of chains, and presents counts of hydrogen bonds, salt bridges, disulfide bonds, pi-pi ...",t:"Protein Interaction Analysis Panel"},"3867":{i:0.000126943081515151,u:"../bioluminate_help/protein_interaction_analysis_options.html",a:"Make settings to define the interactions between two proteins for the analysis in the Protein Interaction Analysis panel. Advanced Options Dialog Box Features List closest interaction neighbors within box Ignore interactions between backbone atoms option Hydrogen bonds settings Salt bridges setting ...",t:"Protein Interaction Analysis — Advanced Options Dialog Box"},"3868":{i:0.000251802706853256,u:"../bioluminate_help/protein_structure_quality.html",a:"Analyze the protein in the Workspace and display tabular and graphical reports on its quality, as measured by deviations from reported values. To open this panel, choose Tasks → Biologics → Structure Quality in the Task Tool. Protein Structure Quality Viewer Panel Features Protein table Ramachandran ...",t:"Protein Structure Quality Viewer Panel"},"3869":{i:0.000110673601120885,u:"../bioluminate_help/protein_surface_analyzer_settings.html",a:"Make settings for the display of the protein surface analysis and for the inclusion of heterogroups in the analysis. To open this dialog box, click the Settings button in the Protein Surface Analyzer panel.\n Using the Protein Surface Analyzer - Settings Dialog Box Add using section here Protein ...",t:"Protein Surface Analyzer - Settings Dialog Box"},"3870":{i:0.000169302373643634,u:"../bioluminate_help/protein_surface_analyzer.html",a:"Analyze a protein surface into patches that represent distinct types of interactions: positively charged, negatively charged, and hydrophobic. For each patch a range of properties is calculated, to characterize the interaction and its intensity. These patches can be used to identify possible ...",t:"Protein Surface Analyzer Panel"},"3871":{i:0.000407685701006683,u:"../bioluminate_help/protein_titration_curve.html",a:"Generate and display a titration curve for one or more proteins. To open this panel, choose Tasks → Biologics → Protein Titration Curve in the Task Tool. Using the Protein Titration Curve Panel The titration curve for a protein is calculated from the pKa values of titratable groups—individual ...",t:"Protein Titration Curve Panel"},"3872":{i:0.000407685701006683,u:"../bioluminate_help/protein_titration_curve_settings.html",a:"Change settings for the titration curve and charge profile plots. Protein Titration Curve Settings Dialog Box Features Titration Curve tab Data set table Backgroundoption menu Legend option menu Charge Profile tab Data set option menu Ionizable residue table Background option menu Legend option menu ...",t:"Protein Titration Curve Settings Dialog Box"},"3873":{i:0.000103548538988497,u:"../bioluminate_help/protein_protein_docking.html",a:"In this panel you can set up and run a job to dock a ligand protein to a receptor protein, an antibody to an antigen, or form a dimer or trimer of a single protein. Using the Protein-Protein Docking Panel The Protein-Protein Docking panel provides an interface to the Piper program, licensed from ...",t:"Protein-Protein Docking Panel"},"3874":{i:0.000137943484458883,u:"../bioluminate_help/reactive_residues.html",a:"In this panel you can identify reactive sequences in the Workspace structure, by matching them to extended PROSITE patterns that define the reactive sites. To open this panel, choose Tasks → Biologics → Reactive Residues in the Task Tool. Using the Reactive Protein Residues Panel The main workflow ...",t:"Reactive Protein Residues Panel"},"3875":{i:0.000167381322854615,u:"../bioluminate_help/residue_analysis.html",a:"Run an analysis of the protein in the Workspace to produce properties of the residues, including hydropathy, energies, solvent-accessible surface areas, and rotatable bonds. To open this panel, choose Tasks → Biologics → Residue Analysis in the Task Tool. Using the Residue Analysis Panel To analyze ...",t:"Residue Analysis Panel"},"3876":{i:0.000312632134515127,u:"../bioluminate_help/residue_scanning_options.html",a:"Set options for affinity maturation: the effective temperature of the simulation, the seed for random number generation, options for accepting or rejecting moves or terminating the calculation. Also add an implicit solvent membrane if desired. Residue Scanning - Advanced Options Dialog Box Features ...",t:"Residue Scanning - Advanced Options Dialog Box"},"3877":{i:0.000269883657273529,u:"../bioluminate_help/residue_scanning_homology_suggestions.html",a:"Identify candidate residues for mutation, using homology modeling or structural criteria. Homology modeling can be applied to select residues by variability or conservation. Structural criteria can be applied for solvent accessible surface area and interactions (contacts) with the protein or a ...",t:"Residue Scanning - Homology Suggestions Dialog Box"},"3878":{i:0.000662385873753347,u:"../bioluminate_help/residue_scanning.html",a:"In this panel you can scan a protein for residues that are candidates for mutation, using homology modeling to identify variable or conserved residues, solvent-accessible surface area to identify surface residues, distances to identify contacts with other residues, surface shape complementarity, and ...",t:"Residue Scanning Panel"},"3879":{i:0.000143971557073677,u:"../bioluminate_help/residue_loop_mutation.html",a:"In this panel you can create a new structure from an existing protein by residue insertion, deletion, mutation or chirality inversion, or by a loop swap for a string of residues. The new structure can be refined by minimization or simulation. To open this panel, choose Tasks → Biologics → Residue ...",t:"Residue and Loop Mutation Panel"},"3880":{i:0.000249000905256192,u:"../bioluminate_help/mutation_results_viewer.html",a:"This panel displays the results of residue mutations that are made either in the Residue Scanning panel or the Humanize Antibody panel. You can examine changes in various properties, both in a table and in a graph, and you can view mutations in the Workspace. The panel name depends on the choice you ...",t:"Results Viewer Panel"},"3881":{i:0.00010665618148832,u:"../bioluminate_help/toggle_table.html",a:"This panel provides a means of interacting with the Workspace and the project that is similar to PyMOL. To open this panel, choose Window → Toggle Table in the Task Tool. Using the Toggle Table Panel The Toggle Table provides easy ways to manipulate the Workspace. Either the entire Workspace or ...",t:"Toggle Table Panel"},"3882":{i:0.000652296471698897,u:"../canvas_help/3d_minimization.html",a:"In this dialog box you can run an energy minimization on a set of structures to determine their 3D geometry. The minimization is done with a force field, using LigPrep. If the stereochemistry of chiral centers is not specified, LigPrep chooses a stereochemistry that is consistent with a database of ...",t:"3D Minimization Dialog Box"},"3883":{i:0.000548501050393404,u:"../canvas_help/3d_pharmacophore_fingerprints.html",a:"In this dialog box you can set up and run a job to create fingerprints from 3D structures using 3-point or 4-point pharmacophore features, with optional generation of conformers of the 3D structures. The structures must be all-atom structures: structures that do not have hydrogen atoms will not be ...",t:"3D Pharmacophore Fingerprints Dialog Box"},"3884":{i:0.000511481122312175,u:"../canvas_help/3d_pharmacophore_fingerprints_advanced.html",a:"In this dialog box you can set options and thresholds for pharmacophore feature definitions, inter-feature distance binning, conformer generation, and bit filtering. The options used for the canvasPharmFP command are indicated in the descriptions. 3D Pharmacophore Fingerprints - Advanced Options ...",t:"3D Pharmacophore Fingerprints - Advanced Options Dialog Box"},"3885":{i:0.00111671570626522,u:"../canvas_help/applications_menu.html",a:"The Applications menu provides access to the Canvas computational programs. When you start a job, the Start Application dialog box opens, in which you can choose a host and the number of CPUs (when relevant). When you run a job, messages about the job are displayed in the Messages View window, and a ...",t:"Applications Menu"},"3886":{i:0.000114936942379589,u:"../canvas_help/apply_logic_to_view.html",a:"In this dialog box, you can perform a logical AND, OR, NOT, or XOR operation on the current view and another view, and update the current view with the results, or create a new view. To open the Apply Logic to View dialog box, choose View \u003e  Logic in the view window. The logical operations can be ...",t:"Apply Logic to View Dialog Box"},"3887":{i:0.00010665618148832,u:"../canvas_help/apply_to_master.html",a:"Apply row and column selection and visibility to the master view from a custom view. Apply to Master Dialog Box Features Rows options Columns options Allow expansion of Master View row set option Rows options Specify how the rows in the current (custom) view are applied to the master view. Retain ...",t:"Apply to Master Dialog Box"},"3888":{i:0.000291293382134291,u:"../canvas_help/bayes_classification.html",a:"Build a Bayes model from binary or continuous training data to predict the probability of a molecule in each category, or each property range, of the Y variable. Use of more than 10 categories is unlikely to produce meaningful predictions. Bayes Classification Dialog Box Features Data set option ...",t:"Bayes Classification Dialog Box"},"3889":{i:0.000178434146772278,u:"../canvas_help/bayes_classification_bins.html",a:"In this dialog box, you can set the number of bins used for Bayes classification and their widths. Bayes Classification - Bins Dialog Box Features Y variable text box Number of bins box Equal widths option Equal populations option Apply button Display area Slider grid spacing option menu Y variable ...",t:"Bayes Classification - Bins Dialog Box"},"3890":{i:0.000178434146772278,u:"../canvas_help/bayes_classification_options.html",a:"Set advanced options for Bayes classification, including Kullback-Leibler acceptance criteria for fingerprint data, and the smoothing coefficient. Bayes Classification - Advanced Options Dialog Box Features Smoothing coefficient text box Keep bits in top box Apply distance cutoffs option Positive ...",t:"Bayes Classification - Advanced Options Dialog Box"},"3891":{i:0.000283288950824298,u:"../canvas_help/binary_fingerprints_structures.html",a:"In this dialog box you can set up and run a job to create hashed binary fingerprints or structural keys for the input structures. Binary Fingerprints from Structures Dialog Box Features Data set option menu New fingerprint column name text box Fingerprint type options Hashed fingerprint list ...",t:"Binary Fingerprints from Structures Dialog Box"},"3892":{i:0.000839667561042535,u:"../canvas_help/calculator.html",a:"In this dialog box you can create new properties by performing mathematical operations on existing properties. Calculator Dialog Box Features Operate on options Create column text box Formula display area Functions button Properties button Keypad Execute button Save button Load button Operate on ...",t:"Calculator Dialog Box"},"3893":{i:0.000446084378311636,u:"../canvas_help/chart_menu.html",a:"Chart Menu The Chart menu allows you to create and display various kinds of charts. New Create a new chart. The submenu offers choices of the types of chart:  Scatter Plot ,  Histogram , and  Pie Chart .",t:"Chart Menu"},"3894":{i:0.00016038069845643,u:"../canvas_help/choose_random_subset.html",a:"Choose a random subset of the rows from the current view. Choose Random Subset Dialog Box Features Choose rows option menu and box using seed text box Generate Seed button Store results in options Permute rows option Choose rows option menu and box Select the means of choosing the number of rows, ...",t:"Choose Random Subset Dialog Box"},"3895":{i:0.000257332455980548,u:"../canvas_help/choosing_model.html",a:"Canvas provides a range of models for predicting values of properties. Some advice on how to choose the best model for the application that you are interested in is given below. By method: Partial Least Squares (PLS) and Kernel-Based Partial Least Squares (KPLS)—These are good methods for large ...",t:"Choosing a Model"},"3896":{i:8.55716728406209E-05,u:"../canvas_help/clustering_results.html",a:"This panel displays clustering results for K-means and leader-follower clustering in a limited custom view, in which you can display structures for each cluster. Clustering Results Panel Features Menu bar View cluster box Spreadsheet Menu bar The menu bar has a reduced set of menus and items from ...",t:"Clustering Results Panel"},"3897":{i:0.000307085600287506,u:"../canvas_help/data_menu.html",a:"The Data menu allows you to perform various operations on the table data. The items on the Data menu are as follows. Sort Sort the table rows by the values in one of the columns. Opens the  Sort dialog box . Property Filter Filter the data by a set of conditions on the properties. Opens the  ...",t:"Data Menu"},"3898":{i:0.00316926954853869,u:"../canvas_help/details.html",a:"This dialog box provides information on the number of rows and columns in the view, and the times it was created, modified, and saved. It also allows you to change the description of the view. Details Dialog Box Features Name text box Description text area Number of rows text box Number of columns ...",t:"Details Dialog Box"},"3899":{i:0.000469907861900766,u:"../canvas_help/diversity_based_selection.html",a:"Select structures for maximum dissimilarity based on a similarity or distance matrix, or on a set of fingerprints. This method is useful for selecting a set of compounds that is different from an existing set, for example. Using the Diversity-Based Selection Dialog Box One typical scenario for using ...",t:"Diversity-Based Selection Dialog Box"},"3900":{i:0.000336380692179228,u:"../canvas_help/edit_menu.html",a:"The items on the Edit menu are listed below. Only Copy and Find are on the Edit menu in the custom view windows. Copy Copy the table selection to the clipboard. The selection can consist of entire rows or columns, or a rectangular subset of rows and columns. You can then paste them into a ...",t:"Edit Menu"},"3901":{i:0.000236294488714227,u:"../canvas_help/edit_partition_db.html",a:"In this dialog box you can change the name and description of a partition, and edit the class values and the colors used for each class. Edit Partition Dialog Box Features Name text box Description text box Classes table Name text box Name of the partition. You can change the name of the partition ...",t:"Edit Partition Dialog Box"},"3902":{i:0.000211456230719085,u:"../canvas_help/edit_structure.html",a:"In this panel you can edit molecular structures in 2D, and save an image of the molecule. On Windows, you can use ChemDraw to edit structures if you have the ChemDraw ActiveX plug-in or the full ChemDraw software package installed. To use ChemDraw in preference to the built-in editor, you can set a ...",t:"Edit Structure Dialog Box"},"3903":{i:6.13254596701412E-05,u:"../canvas_help/edit_structure_name.html",a:"Renaming Structures You can rename structures by choosing Structure → Edit Name, or right-clicking on the structure choosing Edit Structure Name, then entering the new name in the dialog box that opens.",t:"Renaming Structures"},"3904":{i:0.000255758843295397,u:"../canvas_help/export_options_db.html",a:"In this dialog box, you can select the properties to export, including fingerprints, and set options for how structures and fingerprints are exported. Note that properties are exported in the order that they appear in the view you export them from. Export Options Dialog Box Features Export random ...",t:"Export Options Dialog Box"},"3905":{i:0.000171042779022239,u:"../canvas_help/external_application.html",a:"In this panel you can set up a job to run an external application and incorporate the data into the Canvas project. External Application Dialog Box Features Data set option menu Save results as text box Previous jobs option menu Load Settings button Application text box and Browse button Application ...",t:"External Application Dialog Box"},"3906":{i:0.000236161533398185,u:"../canvas_help/feature_selection.html",a:"Identify a subset of properties that are representative of a larger set, using hierarchical clustering on the correlation matrix, followed by multiple linear regression on the subset. This procedure is useful for reducing a large number of related properties to a smaller set that has little linear ...",t:"Feature Selection Dialog Box"},"3907":{i:0.000259064365065377,u:"../canvas_help/feature_selection_correlation_matrix.html",a:"This dialog box shows the correlation matrix for either the original property set or the reduced property set from a feature selection job. Correlation Matrix Dialog Box Features Color by value option and controls Order by cluster option Correlation matrix Color by value option and controls Color ...",t:"Correlation Matrix Dialog Box"},"3908":{i:0.000309252288378873,u:"../canvas_help/feature_selection_viewer.html",a:"Using the Feature Selection Viewer Panel The initial set of properties displayed in the panel is selected from the hierarchical clustering results, with Kelley criteria to choose the default number of clusters. To determine how well the selected properties (reduced list) represents the orginal ...",t:"Feature Selection Viewer Panel"},"3909":{i:0.000238598962941757,u:"../canvas_help/file_menu.html",a:"The File menu allows you to create, open and close projects, import and export data, and quit Canvas. Canvas projects are saved whenever a change is made, so there is no Save menu item. You must have a Canvas project open to import structures and data. New Project Create a new Canvas project. Opens ...",t:"File Menu"},"3910":{i:0.000611735082945236,u:"../canvas_help/find_panel.html",a:"In this panel you can search for a text string or a regular expression, and select the matches to the search string. Find Panel Features Find what text box Find button Literal string option Allow wildcards option Interpret as regular expression option Interpret as floating point value option Match ...",t:"Find Panel"},"3911":{i:0.000117522769877674,u:"../canvas_help/getting_help.html",a:"Canvas Help The Canvas help facility contains information about the panels in Canvas and how to use them. For information about a panel or the tab that is displayed in a panel, click the Help button in the panel, or press F1. The relevant help topic is displayed in the Help panel. A full list of ...",t:"Getting Help in Canvas"},"3912":{i:0.000246269022743069,u:"../canvas_help/hashed_fingerprints_options.html",a:"In this dialog box you can set the parameters used to generate hashed fingerprints. Hashed Fingerprints - Advanced Options Dialog Box Features Atom/bond typing section Use 3D distancesoption Precision options Minimum linear path box Maximum linear path box Maximum linear + ring closure path box ...",t:"Hashed Fingerprints - Advanced Options Dialog Box"},"3913":{i:0.000204511836821784,u:"../canvas_help/heat_map.html",a:"Color the cells in the current view according to values of their properties. Heat Map Dialog Box Features Show Heat Map option Color Space options Properties list and search box Add and Remove buttons Heat map table Show Heat Map option Select this option to display the background coloring of the ...",t:"Heat Map Dialog Box"},"3914":{i:6.61120384018778E-05,u:"../canvas_help/help_menu.html",a:"The Help menu provides access to information about Canvas. About Canvas—Displays a panel that shows the Canvas version and legal notices relating to Canvas. Online Help—Opens the Canvas Online Help in your browser. See  Getting Help  for more information about the help system, other sources of ...",t:"Help Menu"},"3915":{i:0.000285770644109981,u:"../canvas_help/hierarchical_clustering.html",a:"Perform hierarchical agglomerative clustering on a set of structures using a similarity or distance matrix. Hierarchical Clustering Dialog Box Features Data set option menu Save results as text box From fingerprints option Fingerprint column option menu Metric option menu Formula Key button ...",t:"Hierarchical Clustering Dialog Box"},"3916":{i:0.000198562792715256,u:"../canvas_help/hierarchical_clustering_dendrogram.html",a:"Display a dendrogram from the results of a hierarchical clustering job. Hierarchical Clustering Dendrogram Panel Features Number of clusters option and box Use Kelley criterion option Merging distance option and text box Update button Dendrogram Color by cluster option Color by property option and ...",t:"Hierarchical Clustering Dendrogram Panel"},"3917":{i:0.000638917650220888,u:"../canvas_help/histogram.html",a:"This panel displays a histogram plot for a selected property. Histogram Panel Features Menu bar Chart area Property option menu Property type options Menu bar The menu bar has two menus: File and Settings. The File menu has two items, Save Image, for saving the image in png, jpeg, or tiff format, ...",t:"Histogram Panel"},"3918":{i:0.000391643694079102,u:"../canvas_help/histogram_bars.html",a:"In this dialog box you can change the number and appearance of the bars, and the display of the horizontal axis. Bars Dialog Box Features Bar specification section Axis line section Tick marks section Tick mark labels or Category labels section Axis title section Color buttons Bar specification ...",t:"Bars Dialog Box"},"3919":{i:0.000391643694079102,u:"../canvas_help/histogram_y_axis.html",a:"In this dialog box you can set up the y axis and its labels. Y Axis Dialog Box Features Axis limits section Axis line section Tick marks section Tick mark labels section Axis title section Color buttons Axis limits section Set the minimum and maximum values on the axis, and choose an option for ...",t:"Y Axis Dialog Box"},"3920":{i:0.000568018599058819,u:"../canvas_help/hole_filling_library_optimization.html",a:"This application allows you to select a subset of structures from a pool to minimize the similarity to other structures and optionally to minimize the number of properties whose values lie outside given ranges (property optimization). The structures can be chosen to fill holes in a library, and the ...",t:"Hole-Filling and Library Optimization Dialog Box"},"3921":{i:0.000316570540808844,u:"../canvas_help/hole_filling_library_optimization_options.html",a:"In this dialog box, you can set options to control the optimization of the library when doing hole filling and property optimization. Hole-Filling and Library Optimization - Advanced Options Dialog Box Features The command-line options for the canvasLibOpt program are given in the description of the ...",t:"Hole-Filling and Library Optimization - Advanced Options Dialog Box"},"3922":{i:0.000316570540808844,u:"../canvas_help/hole_filling_library_optimization_properties.html",a:"In this dialog box you can select the properties to optimize and set the desired range of property values. Using the Hole-Filling and Library Optimization - Property Filter Dialog Box To specify the properties to optimize, select them in the Available properties list and click Add to add them to the ...",t:"Hole-Filling and Library Optimization - Property Filter Dialog Box"},"3923":{i:0.000255758843295397,u:"../canvas_help/import_options_db.html",a:"The import options dialog box allows you to specify which structures to import and how they are to be imported into the Canvas project. It opens automatically after you select the structure files or project to import, and is labeled Import Structure File, Import SMILES and Properties, Import SMILES, ...",t:"Import Options Dialog Box"},"3924":{i:0.000594340613520033,u:"../canvas_help/install_scripts.html",a:"In this dialog box you can install Python scripts so that they appear on the Scripts menu. Using the Install Scripts Dialog Box To install a script: If necessary, click Browse to find a directory containing Python scripts. The default directory is $HOME on Linux and %USERPROFILE% on Windows. Note: ...",t:"Install Scripts Dialog Box"},"3925":{i:0.000359928580162549,u:"../canvas_help/kernel_based_pls_regression.html",a:"Build a model based on a kernel-based partial least squares regression for a selection of properties over a data set of structures, and apply the model to other data sets. This method is useful for large numbers of X variables, with both underdetermined and overdetermined systems. KPLS is available ...",t:"Kernel-Based Partial Least-Squares Regression Dialog Box"},"3926":{i:0.000213211796317637,u:"../canvas_help/k-means_clustering_db.html",a:"In this dialog box you can set up and start a job to perform k-means clustering. This clustering method can handle much larger data sets than hierarchical clustering. It is useful when you know the number of clusters you want. Two properties are added by the clustering: the cluster number for each ...",t:"K-Means Clustering Dialog Box"},"3927":{i:0.000165783487839273,u:"../canvas_help/kpls_advanced_fingerprints_options.html",a:"In this dialog box you can filter the fingerprints used in the KPLS regression model either by ON/OFF frequency or by information content. KPLS - Advanced Fingerprints Options Dialog Box Features There is one section in this dialog box, labeled Filter bits by. ON/OFF frequency option Information ...",t:"KPLS - Advanced Fingerprints Options Dialog Box"},"3928":{i:0.000264199944389133,u:"../canvas_help/kpls_analyze_atomic_contributions.html",a:"This dialog box presents an analysis of the atomic contributions of a structure to its predicted property value from the kernel-based PLS regression. Using the Analyze Atomic Contributions Dialog Box You can use this dialog box to examine the contributions from each atom to the predicted property. ...",t:"Analyze Atomic Contributions Dialog Box"},"3929":{i:0.000204311029802017,u:"../canvas_help/kpls_fragment_contributions.html",a:"This dialog box displays contributions to the predicted property from a particular atom in the structure, broken down into contributions from each fragment that contain the atom. Each row of the table corresponds to a fragment. The fragment is marked on the structure. The Fragment column contains ...",t:"Fragment Contributions Dialog Box"},"3930":{i:0.000211356932586557,u:"../canvas_help/kpls_model_visualization.html",a:"This dialog box presents the structures with the atoms color-coded by the size of their contribution to the predicted property values from the KPLS regression. Model Visualization Dialog Box Features KPLS factors box Sort by option menu Min and Max buttons Color scale slider and boxes Limits button ...",t:"Model Visualization Dialog Box"},"3931":{i:0.000213211796317637,u:"../canvas_help/leader-follower_clustering_db.html",a:"In this dialog box you can set up and run a leader-follower clustering job. Leader-follower clustering can be used on much larger data sets than hierarchical clustering. Background In the leader-follower clustering method, data points are added to a cluster based on a cutoff on their distance from ...",t:"Leader-Follower Clustering Dialog Box"},"3932":{i:0.000124884803140611,u:"../canvas_help/library_comparison.html",a:"Set up and run a job to find nearest-neighbor similarities between two collections of compounds or within one collection. Using the Library Comparison Dialog Box The library comparison works by finding the nearest neighbor in the reference library for each compound in the query library, using ...",t:"Library Comparison Dialog Box"},"3933":{i:0.000106987687585378,u:"../canvas_help/main_topic.html",a:"The Canvas graphical interface is designed for project-oriented work and provides chemical structure storage, data analysis and visualization, and access to a wide range of applications that are also available through the command line. Key features of the Canvas graphical interface include: ...",t:"Canvas Graphical Interface"},"3934":{i:0.000199899425664653,u:"../canvas_help/manage_properties.html",a:"In this dialog box you can select the properties that are displayed in the current view. Using the Manage Properties Dialog Box To add properties to the list that is shown in the view, select the properties in the Available properties list and click Add. To remove properties from the list that is ...",t:"Manage Properties Dialog Box"},"3935":{i:0.000594340613520033,u:"../canvas_help/manage_scripts.html",a:"In this dialog box you can perform tasks to manage the scripts on the Scripts menu. Manage Scripts Dialog Box Features Contents of the Scripts menu table Delete button Description button Reload button Contents of the Scripts menu table This table lists all the scripts currently installed and visible ...",t:"Manage Scripts Dialog Box"},"3936":{i:0.000171042779022239,u:"../canvas_help/maximum_common_substructure.html",a:"Determine the maximum common substructure for a number of structures within a set between a minimum and maximum number. The results include the substructure and the list of molecules that match the substructure. Maximum Common Substructure Dialog Box Features Data set option menu Save results as ...",t:"Maximum Common Substructure Dialog Box"},"3937":{i:0.00017426864413988,u:"../canvas_help/molecular_properties.html",a:"In this dialog box you can set up and run a job to calculate various molecular properties (descriptors). Molecular Properties Dialog Box Features Data set option menu Save results as text box Descriptor class tree Calculate class Descriptors option classDescriptors Job Options section ...",t:"Molecular Properties Dialog Box"},"3938":{i:0.000137248716058651,u:"../canvas_help/molecular_properties_advanced_db.html",a:"In this dialog box you can customize the definitions of hydrogen bond acceptors and donors and rotatable bonds, and make settings for the calculation of electrotopological states. Molecular Properties - Advanced Options Dialog Box Features Custom definitions section Electrotopological states section ...",t:"Molecular Properties - Advanced Options Dialog Box"},"3939":{i:0.000167391063585577,u:"../canvas_help/multiple_linear_regression.html",a:"Build and apply a multiple linear regression model. This method is useful for a small number of X variables, and (many) more Y values than X variables. For larger numbers of X variables, consider the partial least squares methods instead. Multiple Linear Regression Dialog Box Features Data set ...",t:"Multiple Linear Regression Dialog Box"},"3940":{i:0.00010665618148832,u:"../canvas_help/multiple_linear_regression_subsets_options.html",a:"Set parameters for determining the best subsets of the X variables to use in a multiple linear regression. Multiple Linear Regression - Best Subsets Options Dialog Box Features Number of X variables box Use average of N best models option and box Weight models by R^2 option Number of Monte Carlo ...",t:"Multiple Linear Regression - Best Subsets Options Dialog Box"},"3941":{i:0.000137229046352846,u:"../canvas_help/neural_networks.html",a:"Build and test an ensemble model of neural networks for selected properties. The neural networks have a standard achitechture: one input layer, one hidden layer, and one output layer. The number of neurons in the hidden layer is (input+output)/2. The networks are trained using a BFGS algorithm. ...",t:"Neural Networks Dialog Box"},"3942":{i:0.000135874437211345,u:"../canvas_help/new_class_db.html",a:"In this dialog box you can create a new class in the current partition. The existing classes for the partition are listed in the Class table. This table has three columns: Value—Value for the class. For values assigned from a property, these values are a sequence of integers. Count—Number of members ...",t:"New Class Dialog Box"},"3943":{i:0.000250482149847763,u:"../canvas_help/new_partition_from_property_bins.html",a:"In this dialog box, you can set the number of bins used to define a partition and their widths. New Partition from Property - Bins Dialog Box Features Property text box Number of bins box Equal widths option Equal populations option Apply button Display area Slider grid spacing option menu Property ...",t:"New Partition from Property - Bins Dialog Box"},"3944":{i:0.000486567984356293,u:"../canvas_help/new_partition_from_property_db.html",a:"Create a partition by assigning classes from the values of a property. You can split a property into classes of widths that you determine, with multiple property values included in each class, or you can create classes for each property value. New Partition from Property Dialog Box Features Property ...",t:"New Partition from Property Dialog Box"},"3945":{i:0.00038012045707367,u:"../canvas_help/new_partition_from_random_split_db.html",a:"In this dialog box you can create a partition in which the rows are distributed randomly between the classes. This feature can be used to creating training and test sets of structures, for example, for applications that do not offer an assignment choice. Create Random Partition Dialog Box Features ...",t:"New Partition from Random Split Dialog Box"},"3946":{i:0.00038012045707367,u:"../canvas_help/new_partition_from_selection_db.html",a:"In this dialog box, you can create a new partition of the structures (rows) into classes. The partition can be random, based on property values, or empty when it is created. You can create classes later, if you wish. The partition is displayed as a column in the spreadsheet, colored by class. New ...",t:"New Partition from Selection Dialog Box"},"3947":{i:0.000206373012436177,u:"../canvas_help/partial_least_squares_regression.html",a:"Build a multiple linear regression model based on variables derived from a partial least squares (PLS) analysis of a selection of properties for a data set of structures, and apply the model to other data sets. This method is useful for large numbers of X variables, with both underdetermined and ...",t:"Partial Least-Squares Regression Dialog Box"},"3948":{i:0.00063636307991459,u:"../canvas_help/partition_filter_db.html",a:"In this panel you can filter the rows in the current view by selection of classes from the current partition. The panel allows you to interact with the spreadsheet, so you can choose classes and apply the filter multiple times without closing the panel. Using the Partition Filter Panel To filter the ...",t:"Partition Filter Panel"},"3949":{i:0.000526266975725064,u:"../canvas_help/partition_menu.html",a:"The Partition menu provides tools for creating, storing, modifying, applying, and deleting partitions of the spreadsheet rows into classes. The items on the menu are listed below. Most of the items are only available in the Master View. From custom views, you can open and close partitions, but you ...",t:"Partition Menu"},"3950":{i:0.000560986379808912,u:"../canvas_help/pie_chart.html",a:"This panel displays a pie chart representing the distribution of values of a property. Frequency Pie Chart Panel Features Menu bar Chart area Property option menu Property type options Menu bar The menu bar has two menus: File and Settings. The File menu has two items, Save Image, for saving the ...",t:"Frequency Pie Chart Panel"},"3951":{i:0.000313712423667126,u:"../canvas_help/pie_chart_sectors.html",a:"In this dialog box you can change the number and appearance of the sectors and the legend for the pie chart. Sectors Dialog Box Features Sector specification section Sector perimeter section Legend section Color buttons Sector specification section Set the color for each of the sectors. For ...",t:"Sectors Dialog Box"},"3952":{i:0.000268580263850441,u:"../canvas_help/preferences.html",a:"In this dialog box you can set preferences for the behavior of Canvas. Using the Preferences Dialog Box To set preferences in a particular category, click the category in the list on the left. The preferences for that category are then displayed to the right of the list, and you can make your ...",t:"Preferences Dialog Box"},"3953":{i:0.000148918876494102,u:"../canvas_help/principal_components_analysis.html",a:"Perform a principal components analysis for a set of source properties. Principal Components Analysis Dialog Box Features Data set option menu Save results as text box Options tab Scores tab Principal Components tab Action buttons Data set option menuSelect the source of the data. The data source ...",t:"Principal Components Analysis Dialog Box"},"3954":{i:0.000207975725933137,u:"../canvas_help/principal_components_regression.html",a:"Build a multiple linear regression model based on variables derived from a principal components analysis of a selection of properties for a data set of structures, and apply the model to other data sets. This model is useful when there are many X variables. It seeks to encapsulate the bulk of the ...",t:"Principal Components Regression Dialog Box"},"3955":{i:0.000165183062980104,u:"../canvas_help/project_menu.html",a:"The Project menu gives you access to information about the Canvas project. Project View—This option displays or hides the Project View panel. This panel displays a tree view of the project, which provides access to all the objects created in the project, such as views, jobs run, and so on. By ...",t:"Project Menu"},"3956":{i:0.000233093875519236,u:"../canvas_help/project_view.html",a:"This panel displays a tree view of the project, which provides access to all the objects created in the project, such as views, jobs run, and so on. Using the Project View Panel The project tree lists the categories for the information that is stored in the project. By default, the panel is docked ...",t:"Project View Panel"},"3957":{i:0.000692506100332539,u:"../canvas_help/property_filter.html",a:"Filter all rows in the current view by conditions on the values of properties. Property Filter Dialog Box Features Use existing filter option menu Import button Return non-matches option Store results in options Show rows where controls And and Or options Add button Undo button Clear button Filter ...",t:"Property Filter Dialog Box"},"3958":{i:0.000559641939099455,u:"../canvas_help/property_selector.html",a:"Property selection tools are used in a number of panels to select one or more properties. This tool set consists of the following features: Available properties list This list contains properties that are available for use but are not in the set selected for use. You can select multiple rows with ...",t:"Property Selection Tools"},"3959":{i:6.61120384018778E-05,u:"../canvas_help/python_menu.html",a:"Python Menu This menu contains a single item, Open Shell, which opens a  window  in which you can enter Python commands and execute them, within an environment that includes Schrödinger and Canvas modules. The shortcut for this item is Ctrl+P (⌘P).",t:"Python Menu"},"3960":{i:0.000153946146386493,u:"../canvas_help/query_menu.html",a:"Query Menu The Query menu provides tools for querying the database and returning rows that match the query. Substructure Query Run a query on a substructure. Opens the  Substructure Query dialog box . Properties Run a query on one or more properties. Opens the  Property Filter dialog box .",t:"Query Menu"},"3961":{i:0.000384641237516477,u:"../canvas_help/recursive_partitioning.html",a:"Use recursive partitioning to build a model that can be used to predict property ranges or categories. Recursive Partitioning Dialog Box Features Data set option menu Save results as text box Options tab Training Set tab Results tab Action buttons Data set option menuSelect the source of the data. ...",t:"Recursive Partitioning Dialog Box"},"3962":{i:0.000187897404128007,u:"../canvas_help/recursive_partitioning_bins.html",a:"In this dialog box, you can set the number of bins used for recursive partitioning and their widths. Recursive Partitioning - Bins Dialog Box Features Y variable text box Number of bins box Equal widths option Equal populations option Apply button Display area Y variable text box Noneditable text ...",t:"Recursive Partitioning - Bins Dialog Box"},"3963":{i:0.000187897404128007,u:"../canvas_help/recursive_partitioning_options.html",a:"In this dialog box you can set options for the building of a model by recursive partitioning. Recursive Partitioning - Advanced Options Dialog Box Features Splitting section Ensemble model section Splitting section In this section you can set various parameters that control how the splitting ...",t:"Recursive Partitioning - Advanced Options Dialog Box"},"3964":{i:0.000187897404128007,u:"../canvas_help/recursive_partitioning_trees.html",a:"In this panel you can display the decision trees from a recursive partitioning model, one at a time. Recursive Partitioning Trees Panel Features Tree number box View classifications for options Categories Key button Tree display area Tree number box Specify the index number of the tree that you want ...",t:"Recursive Partitioning Trees Panel"},"3965":{i:0.000859640012848829,u:"../canvas_help/save_custom_view.html",a:"Save a custom view as a named view. Save Custom View Dialog Box Features Use selected rows and Use all rows options Name text box Description text box Number of rows text box Number of columns text box Open view option Use selected rows and Use all rows options Select Use selected rows to include ...",t:"Save Custom View Dialog Box"},});