define({"4446":{i:9.91565911213727E-05,u:"../materials_help/ms_md_simulation_analysis.html",a:"Analyze a trajectory from a Desmond MD simulation and present information on bulk properties derived from the trajectory, such as plots of the density and the cohesive energy. To open this panel, choose Tasks → Materials → Classical Mechanics → MS MD Trajectory Analysis in the Task Tool. Using the ...",t:"MS MD Trajectory Analysis Panel"},"4447":{i:0.000352182472561651,u:"../materials_help/coarse_grain_particulate.html",a:"Iteratively select groups of atoms in a structure and specify them as coarse-grain particles for molecular dynamics simulations, then generate a project entry with the coarse-grain model. To open this panel, choose Tasks → Materials → Coarse-Grained Modeling → Map Atoms to Particles in the Task ...",t:"Map Atoms to Particles Panel"},"4448":{i:0.000452067567304516,u:"../materials_help/mark_head_tail.html",a:"Mark the head and tail atoms of a monomer unit so that they can be identified in subsequent applications, such as for use in Canvas QSAR modeling. To open this panel, choose Tasks → Materials → Mark Monomer Head and Tail in the Task Tool. Using the Mark Monomer Head and Tail Panel The monomer ...",t:"Mark Monomer Head and Tail Panel"},"4449":{i:6.70904936395816E-05,u:"../materials_help/explorer.html",a:"This panel uses a self-organizing map to cluster structures by sets of properties, and allows you to explore the properties of the structures based on this clustering. A Canvas license is required (to create the self-organizing map). Using the Materials Explorer Panel The project entries are ...",t:"Materials Explorer Panel"},"4450":{i:6.13254596701412E-05,u:"../materials_help/explorer_add_property_sets.html",a:"In this panel you can add property sets to the properties that are available for use in clustering or mapping to clusters. The properties can be calculated or read in. Add Property Sets Panel Settings Remove constant descriptors during import option Calculate Descriptors section Read from text file ...",t:"Materials Explorer — Add Property Sets Panel"},"4451":{i:6.13254596701412E-05,u:"../materials_help/explorer_project_options.html",a:"This panel contains information about the project used and allows you to change the structures, and add properties. Advanced Project Options Panel Settings Change Structures button Advanced Property Preparation button Change Structures button Select the structures that you want to cluster. Opens the ...",t:"Materials Explorer — Advanced Project Options Panel"},"4452":{i:6.13254596701412E-05,u:"../materials_help/explorer_compute_maps.html",a:"In this panel you can select the maps to calculate, and cluster the structures by use of self-organizing maps. Compute Maps Panel Settings Cluster Maps to Calculate section Map Grid Size section Compute Maps button Reset Panel button Cluster Maps to Calculate section Choose the property sets that ...",t:"Materials Explorer — Compute Maps Panel"},"4453":{i:6.13254596701412E-05,u:"../materials_help/explorer_new_property_projection.html",a:"In this dialog box you can project a property onto a particular clustering of the structures. New Property Projection Dialog Box Settings Map option menu Property list Property filter text box Statistic option menu Calculate Projection button Map option menu Choose the self-organizing map (structure ...",t:"Materials Explorer — New Property Projection Dialog Box"},"4454":{i:6.13254596701412E-05,u:"../materials_help/explorer_property_preparation.html",a:"This panel allows you to prepare properties before doing the clustering of structures, to add properties from an external source, and to remove properties that are not of interest. Using the Property Preparation Panel The property preparation panel is intended to enable properties to be calculated ...",t:"Materials Explorer — Property Preparation Panel"},"4455":{i:0.000173165691477995,u:"../materials_help/nanoparticle_builder.html",a:"In this panel you can build a nanoparticle of a particular shape and size from the structure in the Workspace. To open this panel, choose Tasks → Materials → Build → Nanoparticles in the Task Tool. Using the Nanoparticle Builder Panel Nanoparticles are built from a larger structure by imposing a ...",t:"Nanoparticle Builder Panel"},"4456":{i:0.00017065972964336,u:"../materials_help/nanoparticle_alignment_options.html",a:"Align the nanoparticle boundaries automatically to crystal faces and edges, or manually by translating and rotating the shape template in the Workspace. Using the Alignment Options Dialog Box By default, the template shape for the nanoparticle is centered at the origin and aligned to the coordinate ...",t:"Nanoparticle Builder — Alignment Options Dialog Box"},"4457":{i:0.000668430614337456,u:"../materials_help/nanostructure_builder.html",a:"Build nanotubes or nanosheets from a specified asymmetric unit, composition, and geometry to a given size. Multi-walled nanotubes and multi-layer nanosheets can be constructed as well as single-walled nanotubes and single nanosheets, and nanotubes with a range of chiral indices can be generated. To ...",t:"Nanostructure Builder Panel"},"4458":{i:0.00060762289348692,u:"../materials_help/new_crystal_structure.html",a:"In this panel you can create a crystal structure entirely from text data, such as data found in a publication, for example, or from a 3D structure and information on the unit cell. To open this panel, choose Tasks → Materials → Build → Crystal Structure in the Task Tool. Using the New Crystal ...",t:"New Crystal Structure Panel"},"4459":{i:0.000164657884298081,u:"../materials_help/optoelectronics_advanced_options.html",a:"Set options for the calculation of oxidation and reduction potentials and the triplet energy, and specify Jaguar keywords to control the calculations. Options can be set separately for each of the calculation modes. Optoelectronics - Advanced Options Dialog Box Features Set options for options ...",t:"Optoelectronics - Advanced Options Dialog Box"},"4460":{i:0.000253163409342198,u:"../materials_help/optoelectronics_calculations.html",a:"Calculate electronic properties of molecules relevant to optoelectronics: oxidation and reduction potentials, hole and electron reorganization energies, singlet-triplet excitation energies, and electronic absorption spectra. To open this panel, choose Tasks → Quantum Mechanics → Optoelectronics → ...",t:"Optoelectronics Calculations Panel"},"4461":{i:0.000366726273474622,u:"../materials_help/optoelectronics_ga_optimization.html",a:"Optimize one or more optoelectronic properties using a genetic algorithm for modifying the structure. To open this panel, choose Tasks → Quantum Mechanics → Optoelectronics → Genetic Optimization in the Task Tool. Using the Optoelectronics Genetic Optimization Panel The main purpose of this panel is ...",t:"Optoelectronics Genetic Optimization Panel"},"4462":{i:0.000198496589691123,u:"../materials_help/optoelectronics_go_viewer.html",a:"Monitor the progress of a genetic optimization of optoelectronic properties while a job is running, or examine a completed job. To open this panel, choose Tasks → Quantum Mechanics → Optoelectronics → Genetic Optimization Monitoring in the Task Tool. Using the Optoelectronics Genetic Optimization ...",t:"Optoelectronics Genetic Optimization Viewer Panel"},"4463":{i:0.000140116334302778,u:"../materials_help/optoelectronics_properties.html",a:"This topic describes the optoelectronic properties generated in an optoelectronics calculation, and the methods used for the calculation. The calculations can be set up in the  Optoelectronics Calculations panel . All calculations begin with a gas phase optimization of the ground state, neutral ...",t:"Optoelectronics Properties"},"4464":{i:0.000149118742926282,u:"../materials_help/optoelectronics_results.html",a:"View the results of optoelectronics calculations, as tabular data and as scatter plots. To open this panel, choose Tasks → Quantum Mechanics → Optoelectronics → View Results in the Task Tool. Using the Optoelectronics Results Panel The Optoelectronics Results panel can be used to detect trends and ...",t:"Optoelectronics Results Panel"},"4465":{i:0.00018849830897053,u:"../materials_help/polymer_builder.html",a:"Build a polymer from one or more monomer units. The polymer can be linear, branched, or dendritic; with different monomers it can also be a periodic or block copolymer or a random copolymer. You can specify the composition, arrangement and configuration of the monomers; the initiator, cascader (if ...",t:"Polymer Builder Panel"},"4466":{i:0.000117838193756111,u:"../materials_help/prepare_for_md.html",a:"Prepare the structure in the Workspace for MD simulations, by cleaning up bond orders, selecting a force field, and creating a simulation box by running the Desmond System Builder. To open this panel, choose Tasks → Materials → Prepare for Molecular Dynamics in the Task Tool. Using the Prepare for ...",t:"Prepare for MD Panel"},"4467":{i:0.000675151056311691,u:"../materials_help/probe_scan.html",a:"Explore the potential energy surface of a molecule with respect to a probe molecule, by calculating QM energies of the probe-target system at a set of probe grid points. The grid is a set of points on one or more surfaces surrounding the target. To open this panel, choose Tasks → Quantum Mechanics → ...",t:"Probe Scan Panel"},"4468":{i:0.000279010129303959,u:"../materials_help/pdos_viewer.html",a:"Plot the projected density of states from a periodic DFT calculation as a function of energy. The density of states is projected to atoms and atomic orbitals so you can view contributions to the DOS from each element, each orbital for each element, and each orbital for each atom. To open this panel, ...",t:"Projected Density of States Viewer Panel"},"4469":{i:6.52896090655546E-05,u:"../materials_help/qm_multistage_workflow.html",a:"Set up a multistage workflow of quantum mechanical single point, optimization, and transition state search jobs, in which the output from each stage is used as input to the next. The workflow is run as a Jaguar batch job. To open this panel, choose Tasks → Materials → Quantum Mechanics → QM ...",t:"QM Multistage Workflow Panel"},"4470":{i:0.000396020848546332,u:"../materials_help/periodic_dft_options.html",a:"Set options for running periodic DFT calculations. The options are saved as preferences and used in subsequent calculations. Quantum ESPRESSO Calculations - Advanced Options Dialog Box Features The options are contained in several tabs. Some tabs have the same options, but these options are applied ...",t:"Quantum ESPRESSO Calculations - Advanced Options Dialog Box"},"4471":{i:0.000201769418528331,u:"../materials_help/periodic_dft_pp.html",a:"Specify pseudopotentials for the periodic DFT calculations. Quantum ESPRESSO Calculations - Pseudopotentials Dialog Box Features Path text box and Browse button Pseudopotential list Element label Path text box and Browse button Valence electrons text box Pseudopotential type text box Path text box ...",t:"Quantum ESPRESSO Calculations - Pseudopotentials Dialog Box"},"4472":{i:0.000613359492632859,u:"../materials_help/periodic_dft.html",a:"Run a periodic DFT calculation with the Quantum Espresso package.  To open this panel, choose Tasks → Materials → Quantum ESPRESSO → Perform Calculations in the Task Tool. Using the Quantum ESPRESSO Calculations Panel You must install and configure the Quantum Espresso package to run jobs with this ...",t:"Quantum ESPRESSO Calculations Panel"},"4473":{i:0.00010665618148832,u:"../materials_help/reaction_channel_enumeration.html",a:"For a given set of reactants, enumerate possible reaction channels for several kinds of reactions, either by choosing specific channels, by random selection, or by exhaustive enumeration. To open this panel, choose Tasks → Materials → Enumerate →  Reaction Channel in the Task Tool. Using the ...",t:"Reaction Channel Enumeration Panel"},"4474":{i:7.14475553967327E-05,u:"../materials_help/reaction_energetics_enumeration.html",a:"Calculate reaction barriers, enthalpies, and energies of intermediates for a series of related compounds, based on a prototype reaction, which can include multiple transition states and intermediates. The structures for the reactions are enumerated by addition of functional groups to the structures ...",t:"Reaction Energetics Enumeration Panel"},"4475":{i:0.00067568358731287,u:"../materials_help/reaction_path_interpolation.html",a:"Create a linear synchronous transit reaction path by interpolation between a reactant structure and a product structure. The path is returned as a set of structures with a property that defines the distance along the path. To open this panel, choose Tasks → Materials → Reaction Path Interpolation in ...",t:"Reaction Path Interpolation Panel"},"4476":{i:0.00043917946477562,u:"../materials_help/reorder_atoms.html",a:"Reorder the atom numbers in a comparison structure to match the ordering in a reference structure either automatically or by manual selection on two diagrams or from two lists. Using the Reorder Atoms Dialog Box There are several ways in which you can use this dialog box to map the atoms of the ...",t:"Reorder Atoms Dialog Box"},"4477":{i:0.000743802295666027,u:"../materials_help/restart_workflow.html",a:"Restart one of the restartable Materials Science workflows, chosen by job type. Restarting can be done on jobs that fail due to problems such as resource, hardware, or network issues. To open this panel, choose Tasks → Materials → Restart Workflows in the Task Tool. Using the Restart Workflow Panel ...",t:"Restart Workflow Panel"},"4478":{i:0.000405684160861726,u:"../materials_help/select_space_group.html",a:"In this dialog box you can choose the space group to use for your crystal structure. Using the Select Space Group Dialog Box To choose the desired space group, select it in the list of groups. You can filter the list by choosing a crystal system from the System option menu and a lattice type from ...",t:"Select Space Group Dialog Box"},"4479":{i:0.000270175304580945,u:"../materials_help/simulated_annealing_options.html",a:"Set options for the simulated annealing optimization of the disordered system. Simulated Annealing Options Dialog Box Features Rotation settings Weight text box Maximum rotation text box Translation settings Weight text box μ text box σ text box Temperatures text box Iterations text box Energy terms ...",t:"Simulated Annealing Options Dialog Box"},"4480":{i:0.00011798117097959,u:"../materials_help/spin_states.html",a:"Calculate high-spin and low-spin states of transition metal complexes, exploring some or all of the possible states produced by Jaguar\u0027s transition metal initial guess code. Calculations can also be done on organic molecules and on complexes with more than one transition metal. To open this panel, ...",t:"Spin States Panel"},"4481":{i:0.000171111451477018,u:"../materials_help/stress_strain.html",a:"Calculate the stress on a system as a function of the von Mises strain, from a set of molecular dynamic simulations. To open this panel, choose Tasks → Stress Strain → Perform Calculations in the Task Tool. Using the Stress Strain Panel The stress-strain calculations are run as a series of molecular ...",t:"Stress Strain Panel"},"4482":{i:0.000167277541977591,u:"../materials_help/stress_strain_viewer.html",a:"Display a plot of the stress against the strain from a stress-strain calculation. To open this panel, choose Tasks → Stress Strain → Analyze Results in the Task Tool. Stress Strain Viewer Panel Features Load Data from Workspace button Plot toolbar Plot area Load Data from Workspace button Load the ...",t:"Stress Strain Viewer Panel"},"4483":{i:0.000197024085946596,u:"../materials_help/optoelectronics_structure_property.html",a:"Define a structure property in terms of a SMARTS pattern. The number of occurrences of this pattern in a structure is the property value that is used in the optimization. To open this dialog box, choose Define new SMARTS property from the Property menu in the Optoelectronics Genetic Optimization ...",t:"Structure Property Definition Dialog Box"},"4484":{i:8.55716728406209E-05,u:"../materials_help/tst_rate_calculation.html",a:"Calculate the rate of a reaction using transition state theory, from the output of quantum mechanical calculations on the reactants and the transition state. To open this panel, choose Tasks → Quantum Mechanics → TST Rate Calculation in the Task Tool. Using the TST Rate Calculation Panel Before ...",t:"TST Rate Calculation Panel"},"4485":{i:0.000205503008781807,u:"../materials_help/thermophysical_properties_analysis.html",a:"Analyze the density of a solid as a function of temperature to determine the coefficient of thermal expansion and the glass transition temperature. The densities are obtained from MD simulations at a set of temperatures in the NPT ensemble. To open this panel, choose Tasks → Materials → ...",t:"Thermophysical Properties Analysis Panel"},"4486":{i:0.000164006196463055,u:"../materials_help/thermophysical_properties.html",a:"Calculate the density of a model system using MD simulations at a single temperature, or as a function of temperature to generate data for evaluation of the coefficient of thermal expansion and the glass transition temperature (Tg). To open this panel, choose Tasks → Materials → Thermophysical ...",t:"Thermophysical Properties Panel"},"4487":{i:7.78743621210405E-05,u:"../mcpro_help/simulation_analysis.html",a:"This panel provides the means to plot various thermodynamic and energetic quantities that are generated from free-energy perturbation simulations (including relative binding affinities). MCPRO+ Analyze FEP Panel Features Import section Plot type option menu Plot button Reset button Import section In ...",t:"MCPRO+ Analyze FEP Panel"},"4488":{i:0.000239515111395576,u:"../mcpro_help/mcpro_fep.html",a:"MCPRO+ can perform free-energy perturbation (FEP) calculations on ligands, proteins and complexes. Setting Up Structures for Free-Energy Perturbation The structures that you use for an FEP calculation must have the same atom ordering (by atom number) in the structures at either end of the ...",t:"MCPRO+ Free-Energy Perturbation Calculations"},"4489":{i:0.000129463790443061,u:"../mcpro_help/mcpro_free_energy_diff.html",a:"In MCPRO+ free-energy difference calculations, the free-energy difference between two structures is obtained by free-energy perturbation. The setup and requirements are laid out in the topic  MCPRO+ Free-Energy Perturbation Calculations . You can set up and run a free-energy difference calculation ...",t:"MCPRO+ Free-Energy Difference Calculations"},"4490":{i:0.000157560223057718,u:"../mcpro_help/mcpro_linear_response.html",a:"The purpose of MCPRO+ linear response calculations is to provide data for the generation of linear response models, for example using  Strike  to fit the data. You can set up and run a relative binding affinity calculation in the MCPRO+ Linear Response panel. The general layout and common features ...",t:"MCPRO+ Linear Response Calculations"},"4491":{i:6.13254596701412E-05,u:"../mcpro_help/mcpro_minimization.html",a:"MCPRO+ minimization calculations can be performed on a protein, a protein-ligand complex, or a single ligand. You can set up and run a minimization calculation in the MCPRO+ Minimization panel. The general layout and common features of this panel are described in the topic  MCPRO+ Panels . ...",t:"MCPRO+ Minimizations"},"4492":{i:6.13254596701412E-05,u:"../mcpro_help/mcpro_monte_carlo_sampling.html",a:"Monte Carlo sampling calculations are the foundation of MCPRO+. A brief description of the sampling procedure is given below. The MCPRO+ Monte Carlo Sampling panel allows you to run a general-purpose Monte Carlo sampling calculation, and analyze the energy averages. A Monte Carlo simulation ...",t:"MCPRO+ Monte Carlo Sampling"},"4493":{i:0.000129463790443061,u:"../mcpro_help/mcpro_relative_binding_affinity.html",a:"In MCPRO+ relative binding affinity calculations, the free-energy difference for the binding of two ligands to a receptor is obtained by free-energy perturbation, and the results for each ligand are subtracted to obtain a relative binding affinity. The setup and requirements are similar to those for ...",t:"MCPRO+ Relative Binding Affinity Calculations"},"4494":{i:0.000849632217592342,u:"../mcpro_help/mcpro_panel.html",a:"The MCPRO+ panels provide an interface for setting up and running MCPRO+ jobs. The controls in the panels are configured differently for each kind of MCPRO+ task. For information on the controls in a tab, see the help topic for the tab, which you can open by clicking Help when the tab is displayed.  ...",t:"MCPRO+ Panel"},"4495":{i:0.000157560223057718,u:"../mcpro_help/create_model_system_tab.html",a:"In this tab, you specify the model system for the MCPRO+ calculation. This task is required for all MCPRO+ calculations, and involves selecting the receptor and ligands as appropriate, truncating (chopping) and capping the protein, neutralizing the system, and defining constraints. The resulting ...",t:"MCPRO+ — Model System Creation Tab"},"4496":{i:0.000196385203275742,u:"../mcpro_help/free_energy_difference_tab.html",a:"In this tab you set up the calculation and job parameters for a free-energy perturbation calculation. Free Energy Difference by FEP Tab Features Monte Carlo settings section Pre-FEP Minimization section FEP workflow section Monte Carlo settings section In this section you set the pressure and ...",t:"MCPRO+ — Free Energy Difference by FEP Tab"},"4497":{i:0.000157560223057718,u:"../mcpro_help/linear_response_tab.html",a:"In this tab you set up the parameters of the simulations that will be done for the linear response calculation. For more information on these calculations, see  MCPRO+ Linear Response Calculations . Linear Response Tab Features Monte Carlo settings section Bound Monte Carlo sampling section Free ...",t:"MCPRO+ — Linear Response Tab"},"4498":{i:0.000377556613125196,u:"../mcpro_help/minimization_tab.html",a:"In this tab, you choose the optimizer for the minimization process and set the number of cycles for the minimization. The choice of optimizer also influences the availability of some solvation options—see the  Solvation tab  topic. Minimization Tab Features Number of optimization cycles text box ...",t:"MCPRO+ — Minimization Tab"},"4499":{i:0.000196385203275742,u:"../mcpro_help/monte_carlo_tab.html",a:"This tab allows you to set up the basic parameters for a Monte Carlo simulation of the model system. Monte Carlo Tab Features Monte Carlo settings section MC run section Monte Carlo settings section In this section you set the pressure and temperature for the ensemble, and specify the block size. ...",t:"MCPRO+ — Monte Carlo Tab"},"4500":{i:0.000196385203275742,u:"../mcpro_help/relative_binding_affinity_tab.html",a:"In this tab you set up the parameters for the free-energy perturbation calculations that allow you to evaluate the relative free-energy differences, which give a prediction of the relative binding affinities. Relative Binding Affinity by FEP Tab Features Monte Carlo settings section Pre-FEP ...",t:"MCPRO+ — Relative Binding Affinity by FEP Tab"},"4501":{i:0.000306847066683553,u:"../mcpro_help/solvation_tab.html",a:"In this tab you can select the solvent model and set options for the solvent. Solvation Tab Features The Solvation tab has a single section, Solvent Options, with up to five choices for a solvation model. The choices available depend on the type of calculation and the model system. Some choices are ...",t:"MCPRO+ — Solvation Tab"},"4502":{i:0.000234824169735968,u:"../phase_help/add_from_project_db.html",a:"In this dialog box, you select the entries from the Project Table that you want to add to the QSAR panels. Using the Add From Project Dialog Box To choose the ligands, select them in the table. You can use shift-click and control-click to select multiple ligands, and you can sort the table columns ...",t:"Add From Project Dialog Box"},"4503":{i:0.000432630235233073,u:"../phase_help/atom_based_qsar.html",a:"In this panel you can set up a Phase 3D QSAR model from a set of aligned ligands, and use the model to predict activities for other molecules. You can also visualize the QSAR model in the Workspace, add the QSAR model to a hypothesis, and create a scatter plot of the activities. Atom-Based QSAR ...",t:"Atom-Based QSAR Panel"},"4504":{i:0.000213266342785143,u:"../phase_help/build_atom_based_model.html",a:"In this dialog box you set options for building an atom-based QSAR model. Build Atom-Based Model Dialog Box Features Grid spacing text box Maximum PLS factors text box Eliminate variables with |t-value| \u003c option and text box Number of ligands to leave out text box Grid spacing text box Length of the ...",t:"Build Atom-Based Model Dialog Box"},"4505":{i:0.00021501257593,u:"../phase_help/build_qsar_model_options.html",a:"The Build QSAR Model - Options dialog box provides controls for the randomization of the training set and for the definition of the QSAR model. Build QSAR Model - Options Dialog Box Features The dialog box is divided into two sections: Training set and QSAR model. Random seed text box Keep actives ...",t:"Build QSAR Model - Options Dialog Box"},"4506":{i:0.000258036677275445,u:"../phase_help/choose_activity_property_db.html",a:"In this dialog box, you can choose an activity property for development of a pharmacophore model or a QSAR model, and for the latter, choose a property to define the training and test sets. If you click Cancel, the properties are not set, but the import proceeds. Using the Choose Activity Property ...",t:"Choose Activity Property Dialog Box"},"4507":{i:8.55716728406209E-05,u:"../phase_help/convert_phase_database.html",a:"This panel can be used to convert a Phase database from the earlier format in which the database was marked by a file whose name ends in _phasedb to the current format, in which the database is a directory with the extension .phdb. Using the Convert Phase Database Panel The Convert Phase Database ...",t:"Convert Phase Database Panel"},"4508":{i:0.000562765085401269,u:"../phase_help/create_phase_database.html",a:"Generate a database of ligands for screening, with optional structure preparation, filtering by properties, duplicate elimination, conformer and site generation. The database can be used for shape and pharmacophore screening, and in the Virtual Screening Workflow for ligand docking. Create Phase ...",t:"Create Phase Database Panel"},"4509":{i:0.000166664370768403,u:"../phase_help/define_actives_inactives.html",a:"Assign ligands to the active set or the inactive set for pharmacophore model development from multiple ligands, and optionally set the activities of the ligands. To open this dialog box, click the Define button in the Actives / Inactives Split section of the Develop Pharmacophore Model panel. Using ...",t:"Define Actives and Inactives Dialog Box"},"4510":{i:0.000166664370768403,u:"../phase_help/detect_binding_modes.html",a:"Cluster hypotheses that do not match all actives to detect possible different binding modes.  To open this panel, choose Tasks → Ligand-Based Virtual Screening → Detect Binding Modes or click on the H button for a hypothesis group and choose Detect Binding Modes. Using the Detect Binding Modes Panel ...",t:"Detect Binding Modes Panel"},"4511":{i:0.000365282738471555,u:"../phase_help/develop_pharmacophore_model.html",a:"Develop pharmacophore models from one or more ligands, a ligand-receptor complex, a receptor, a set of protein residues, or multiple existing hypotheses. To open this panel, choose Tasks → Ligand-Based Virtual Screening → Pharmacophore Modeling → Develop Pharmacophore Model. Using the Develop ...",t:"Develop Pharmacophore Model Panel"},"4512":{i:0.000133715662268526,u:"../phase_help/edit_feature.html",a:"Edit Feature  Dialog Box Edit the properties of a hypothesis feature that is in the Workspace, including position and settings related to screening. To open this dialog box, right-click on the feature in the Workspace. Using the Edit Feature  Dialog Box To reposition a feature, select Manually ...",t:"Edit Feature X Dialog Box"},"4513":{i:0.000597438557809578,u:"../phase_help/edit_features.html",a:"The Edit Features dialog box allows you to change the content of pharmacophore features. Features are defined in terms of SMARTS patterns. You can add patterns to both built-in features and up to three custom features. You can edit and delete custom patterns, and you can exclude or ignore both ...",t:"Edit Features Dialog Box"},"4514":{i:0.000321553741785431,u:"../phase_help/edit_pattern.html",a:"The Edit Pattern dialog box provides tools for editing custom SMARTS patterns and redefining the geometry of the pharmacophore site. Edit Pattern Dialog Box Features SMARTS pattern text box Get from selection button Geometry option menu Projected point type option menu Atom definition controls ...",t:"Edit Pattern Dialog Box"},"4515":{i:0.00012164247890383,u:"../phase_help/enrichment_viewer.html",a:"Displays the results of a hypothesis validation, showing enrichment metrics for each hypothesis and giving access to full reports and plots. To open this panel, choose Tasks → Ligand-Based Virtual Screening →  Pharmacophore Modeling → Enrichment Viewer.\n Enrichment Viewer Panel Features Import ...",t:"Enrichment Viewer Panel"},"4516":{i:0.000170719139451935,u:"../phase_help/excluded_volumes_inactives_actives.html",a:"Create excluded volumes for a Phase hypothesis in regions occupied by atoms in a set of inactives that do not overlap with atoms in a set of actives. The actives and inactives must be prealigned to the hypothesis for which the excluded volumes are created. To open this dialog box, click Inactives ...",t:"Excluded Volumes from Inactives and Actives Dialog Box"},"4517":{i:0.000281715220393316,u:"../phase_help/excluded_volumes_receptor.html",a:"Create excluded volumes for a Phase hypothesis from a given receptor or part of a receptor to which the reference ligand of the hypothesis is properly aligned. To open this dialog box, click Receptor in the Manage Excluded Volumes panel. Using the Excluded Volumes from Receptor Dialog Box The ...",t:"Excluded Volumes from Receptor Dialog Box"},"4518":{i:0.000161976752134188,u:"../phase_help/excluded_volumes_reference_structures.html",a:"Create excluded volumes in a full or partial shell around one or more reference structures. To open this dialog box, click Reference Structures in the Manage Excluded Volumes panel. Using the Excluded Volumes from Reference Structures Dialog Box Placing excluded volumes around a reference ligand or ...",t:"Excluded Volumes from Reference Structures Dialog Box"},"4519":{i:0.00012164247890383,u:"../phase_help/hypotheses_validation.html",a:"Validate one or more hypotheses by running a screen on a set of active ligands and decoy ligands. The results of the validation can be viewed in the  Enrichment Viewer Panel . To open this panel, you can: choose Tasks → Ligand-Based Virtual Screening → Pharmacophore Modeling → Hypotheses Validation  ...",t:"Hypotheses Validation Panel"},"4520":{i:0.000181083948134717,u:"../phase_help/hypothesis_settings.html",a:"Make settings that determine how the hypothesis is built, what its structure is, how it is scored.  To open this dialog box, click Hypothesis Settings in the Develop Pharmacophore Model panel.\n Hypothesis Settings Dialog Box Features Features tab Ligand-based and residue-based workflows Hypothesis ...",t:"Hypothesis Settings Dialog Box"},"4521":{i:0.000330138840206637,u:"../phase_help/manage_excluded_volumes.html",a:"Create, modify, move, and delete excluded volumes for a hypothesis.  To open this panel, right-click the H button menu in the Entry List panel and  choose Manage excluded volumes.\nThe hypothesis must be included in the Workspace.  Using the Manage Excluded Volumes Panel You can create new excluded ...",t:"Manage Excluded Volumes Panel"},"4522":{i:0.000426164589542403,u:"../phase_help/manage_phase_database.html",a:"In this panel you can manage a Phase 3D database, by adding, deleting, or exporting structures and creating and deleting subsets. Manage Phase Database Panel Features Open database text box and Browse button New button Structures table Add button Delete button Update button Select All button Select ...",t:"Manage Phase Database Panel"},"4523":{i:0.000558955303009441,u:"../phase_help/new_pattern.html",a:"The New Pattern dialog box provides tools for adding a new SMARTS pattern and associated data to a pharmacophore feature. Using the New Pattern Dialog Box To add a pattern to a feature, you must provide the SMARTS string for the desired arrangement of atoms, and define the corresponding ...",t:"New Pattern Dialog Box"},"4524":{i:0.00116144929641052,u:"../phase_help/ligand_screening.html",a:"Screen ligands against one or more pharmacophore hypotheses. To open this panel, choose Tasks → Ligand-Based Virtual Screening → Pharmacophore Modeling → Ligand Screening.\n Using the Phase Ligand Screening Panel This panel allows you to screen a set of ligands for matches to one or more ...",t:"Phase Ligand Screening Panel"},"4525":{i:6.13254596701412E-05,u:"../phase_help/overview.html",a:"Phase is a versatile product for developing a pharmacophore model from a set of ligands, creating a 3D QSAR (quantitative structure-activity relationship) model, and searching a set of structures for matches to the pharmacophore model. Phase can be used for lead discovery, optimization, and ...",t:"Phase Overview"},"4526":{i:0.000213266342785143,u:"../phase_help/qsar_vis_settings.html",a:"This panel provides controls for the visualization of the Phase QSAR model in the Workspace. To open this panel, click QSAR Visualization in the  Atom-Based QSAR Panel Using the QSAR Visualization Settings Panel This panel allows you to view the QSAR model in 3 dimensions, and to examine the spatial ...",t:"QSAR Visualization Settings Panel"},"4527":{i:0.00108957360076052,u:"../phase_help/screening_settings.html",a:"Make settings for various aspects of screening a set of ligands against Phase pharmacophore hypotheses. To open this dialog box, click the Screening Settings button in the Phase Ligand Screening panel. Screening Settings Dialog Box Features Conformers tab Use existing conformers options Score in ...",t:"Screening Settings Dialog Box"},"4528":{i:0.000250123267259115,u:"../phase_help/simplified_pharmacophore_screening.html",a:"In this panel you can create a hypothesis from a ligand and screen compounds against this hypothesis. Using the Simplified Pharmacophore Modeling and Screening Panel This panel provides a simple interface for the creation of a pharmacophore hypothesis and its use to screen a set of compounds. The ...",t:"Simplified Pharmacophore Modeling and Screening Panel"},"4529":{i:0.000239679471489284,u:"../phase_help/simplified_pharmacophore_screening_edit.html",a:"In this tab you can define the hypothesis to use for screening, either by creating it from a ligand, or importing it. Edit Tab Features Title text box Ligand from Workspace button Ligand from File button Features table Import button Export button Title text box Specify the title of the hypothesis. ...",t:"Simplified Pharmacophore Modeling and Screening — Edit Tab"},"4530":{i:0.000239679471489284,u:"../phase_help/simplified_pharmacophore_screening_find_matches.html",a:"In this tab, you can select the source of structures to screen, and set up and run the screening job. Find Matches Tab Features Screen structures in option menu Remote option File path text box and Browse button Source of conformations options Must match on at least N site points text box Return at ...",t:"Simplified Pharmacophore Modeling and Screening — Find Matches Tab"},"4531":{i:7.78743621210405E-05,u:"../adme_tox_help/amphiphilic_moment.html",a:"Calculate the amphiphilic moment of a set of molecules, which must be positively charged. To open this panel, choose Tasks → ADME and Molecular Properties → ADME/Tox → Amphiphilic Moment in the Task Tool. Using the Amphiphilic Moment Panel This panel allows you to calculate the amphiphilic moment of ...",t:"Amphiphilic Moment Panel"},"4532":{i:7.38866371065594E-05,u:"../adme_tox_help/membrane_permeability.html",a:"Calculate the passive membrane permeability of a set of congeneric ligands. To open this panel, choose Tasks → ADME and Molecular Properties → Physics-Based Membrane Permeability in the Task Tool. Using the Membrane Permeability Panel This panel allows you to calculate the passive membrane ...",t:"Membrane Permeability Panel"},"4533":{i:0.000407685701006683,u:"../adme_tox_help/p450_examine_results.html",a:"In this panel you can examine the predictions for the site of metabolism of a set of molecules by various P450 isoforms. To open this panel, choose Tasks → ADME and Molecular Properties →  Structure Based P450 Site of Metabolism → Examine Results in the Task Tool. P450 Site of Metabolism - Examine ...",t:"P450 Site of Metabolism - Examine Results Panel"},"4534":{i:0.000407685701006683,u:"../adme_tox_help/p450_perform_calculation.html",a:"In this panel you can set up and run a calculation to evaluate a set of ligands for the likely sites of metabolism by various P450 isoforms. To open this panel, choose Tasks → ADME and Molecular Properties → Structure Based P450 Site of Metabolism → Perform Calculation in the Task Tool. Using the ...",t:"P450 Site of Metabolism - Perform Calculation Panel"},"4535":{i:0.00027140466316373,u:"../prime_help/analyze_energy_options.html",a:"In the Analyze Energy - Options dialog box you can select options for the loop refinement, side-chain prediction and minimization tasks of Prime structure refinement. To open this dialog box, click Options in the Analyze Energy panel. Analyze Energy - Options Dialog Box Features Environment options ...",t:"Analyze Energy - Options Dialog Box"},"4536":{i:0.000172959551487836,u:"../prime_help/build_structure_options.html",a:"In this dialog box, you can select a number of options for building the structure from the templates in the Build Structure step. To open this dialog box, click Options in the Input Sequence step of the Structure Prediction panel. Build Structure - Options Dialog Box Features Preserve the residue ...",t:"Build Structure - Options Dialog Box"},"4537":{i:0.000226936839081282,u:"../prime_help/analyze_energy.html",a:"Evaluate the energy and its components for one or more proteins. To open this panel, choose Tasks → Protein Preparation and Refinement → Calculate Energy in the Task Tool. Calculate Energy Panel Features Use structures from option menu File name text box and Browse button Solvation model option menu ...",t:"Calculate Energy Panel"},"4538":{i:0.000193081732087267,u:"../prime_help/get_sequence_from_pdb.html",a:"This dialog box allows you to obtain a protein sequence from either the PDB web site or a local copy of the PDB database and import it into Maestro. The PDB file is downloaded first, and the sequence obtained from the PDB file. To open this dialog box, click PDB ID in the Input Sequence step of the ...",t:"Get Sequence from PDB ID Dialog Box"},"4539":{i:0.000239593741789451,u:"../prime_help/hybrid_monte_carlo.html",a:"Run hybrid Monte Carlo simulations to search nearby conformational space, through a sequence of short, high-temperature molecular dynamics simulations followed by minimization. New conformations are accepted or rejected based on standard Monte Carlo criteria. The result is either the lowest-energy ...",t:"Hybrid Monte Carlo Panel"},"4540":{i:0.000195214771281456,u:"../prime_help/hybrid_monte_carlo_constraints.html",a:"In this dialog box, you can set up constraints on distances and restraints of atoms to initial positions when performing minimizations. To open this dialog box, click Constraints in the Hybrid Monte Carlo panel. Hybrid Monte Carlo - Constraints Dialog Box Features Distance tab Cartesian tab (missing ...",t:"Hybrid Monte Carlo - Constraints Dialog Box"},"4541":{i:0.000156621709375915,u:"../prime_help/hybrid_monte_carlo_options.html",a:"In the Hybrid Monte Carlo - Options dialog box you can select options for Monte Carlo refinement of a protein structure, including use of an implicit membrane and crystal symmetry, the solvent dielectric constant, algorithm selection, and convergence criteria. To open this dialog box, click Options ...",t:"Hybrid Monte Carlo - Options Dialog Box"},"4542":{i:0.000428740472978066,u:"../prime_help/minimize.html",a:"Minimize the energy of selected parts of proteins or entire proteins with respect to coordinate changes. The minimization can be run with periodic boundary conditions, in implicit solvent, and with an implicit membrane. To open this panel, choose Tasks → Protein Preparation and Refinement → Minimize ...",t:"Minimize Panel"},"4543":{i:0.000354815089743377,u:"../prime_help/minimize_constraints.html",a:"In this dialog box, you can set up constraints on distances and restraints of atoms to initial positions when performing minimizations. To open this dialog box, click Constraints in the Minimize panel. Minimize - Constraints Dialog Box Features Distance tab Cartesian tab (missing or bad snippet) ...",t:"Minimize - Constraints Dialog Box"},"4544":{i:0.000398546473878834,u:"../prime_help/minimize_options.html",a:"In the Minimize - Options dialog box you can select options for minimization of a protein structure, including use of an implicit membrane and crystal symmetry, the solvent dielectric constant, algorithm selection, and convergence criteria. To open this dialog box, click Options in the Minimize ...",t:"Minimize - Options Dialog Box"},"4545":{i:0.00024265191760569,u:"../prime_help/minimize_rigid_body.html",a:"Minimize the energy of part of a protein by treating it as a rigid body and adjusting its location and orientation. The minimization can include freely moving atoms and multiple rigid bodies. The rest of the protein is frozen. The minimization can be run with periodic boundary conditions, in ...",t:"Minimize as Rigid Body Panel"},"4546":{i:0.00010665618148832,u:"../prime_help/minimize_rigid_body_constraints.html",a:"In this dialog box, you can set up constraints on distances and restraints of atoms to initial positions when performing rigid-body minimizations. To open this dialog box, click Constraints in the Minimize as Rigid Body panel. Using the Minimize as Rigid Body - Constraints Dialog Box Distance ...",t:"Minimize as Rigid Body - Constraints Dialog Box"},"4547":{i:0.000253632309890934,u:"../prime_help/minimize_rigid_body_options.html",a:"In the Minimize as Rigid Body - Options dialog box you can select options for minimization of a protein structure, including use of an implicit membrane and crystal symmetry, the solvent dielectric constant, algorithm selection, and convergence criteria. To open this dialog box, click Options in the ...",t:"Minimize as Rigid Body - Options Dialog Box"},"4548":{i:0.000363489995525077,u:"../prime_help/predict_side_chains.html",a:"Predict the conformation of side chains in one or more proteins by sampling orientations to minimize the energy. The sampling can include the orientation with respect to the backbone, and sampling of the backbone as well. The sampling can be run with periodic boundary conditions, in implicit ...",t:"Predict Side Chains Panel"},"4549":{i:0.00021413615967615,u:"../prime_help/predict_side_chains_constraints.html",a:"In this dialog box, you can set up constraints on distances and restraints of atoms to initial positions when performing side-chain predictions. To open this dialog box, click Constraints in the Predict Side Chains panel. Predict Side Chains - Constraints Dialog Box Features Distance tab Cartesian ...",t:"Predict Side Chains - Constraints Dialog Box"},"4550":{i:0.000267908047379015,u:"../prime_help/predict_side_chains_options.html",a:"In the Predict Side Chains - Options dialog box you can select options for the loop refinement, side-chain prediction and minimization tasks of Prime structure refinement. To open this dialog box, click Options in the Predict Side Chains panel. Predict Side Chains - Options Dialog Box Features ...",t:"Predict Side Chains - Options Dialog Box"},"4551":{i:0.000114840007014853,u:"../prime_help/energy_visualizer.html",a:"Visualize the Prime energy and its components from an Analyze Energy  job, a WM/MM  job or an MM-GBSA job. The visualization colors the structure in the Workspace and the surface (if generated) by the values of a chosen property or sum of properties. To open this panel, choose Tasks → Structure ...",t:"Prime Energy Visualizer Panel"},"4552":{i:0.000407685701006683,u:"../prime_help/prime_macrocycle_sampling.html",a:"Sample the conformations of a ligand-sized structure that contains macrocycles. To open this panel, choose Tasks → Lead Optimization → MM-GBSA in the Task Tool. Using the Prime Macrocycle Sampling Panel The sampling algorithm takes the macrocycle backbone and splits it into two pieces, which are ...",t:"Prime Macrocycle Sampling Panel"},"4553":{i:0.000810190597834083,u:"../prime_help/membrane_setup_panel.html",a:"This panel is used to set up an implicit membrane model for Prime refinement calculations. To open this panel, choose Tasks → Protein Preparation and Refinement → Calculate Energy in the Task Tool. Using the Prime Membrane Setup Panel The implicit membrane is a slab-shaped region with a low ...",t:"Prime Membrane Setup Panel"},"4554":{i:0.000179345640001599,u:"../prime_help/prime_mmgbsa_panel.html",a:"This panel can be used to calculate ligand binding energies and ligand strain energies for a set of ligands and a single receptor, using the MM-GBSA technology available with Prime. To open this panel, choose Tasks → Lead Optimization → MM-GBSA in the Task Tool. Using the Prime MM-GBSA Panel The ...",t:"Prime MM-GBSA Panel"},"4555":{i:0.000126462598933927,u:"../prime_help/sequence_viewer.html",a:"The Prime sequence viewer displays the sequences for the query and templates, and in some steps it shows secondary structure assignments (SSAs) and secondary structure predictions (SSPs). It is divided into two sections. The name of the sequence, SSA, or SSP is given in the left section, and the ...",t:"Prime Sequence Viewer"},"4556":{i:0.000269090773174007,u:"../prime_help/build_structure_step.html",a:"In this step, you build a model structure for the query based on the template chosen in the Find Homologs step and aligned in the Edit Alignment step. For general information on the panel, see the topic  Structure Prediction Panel . Using the Build Structure Step The process of building a structure ...",t:"Prime Structure Prediction — Build Structure Step"},"4557":{i:0.000291691864248671,u:"../prime_help/edit_alignment_step.html",a:"The purpose of this step is to find a sastisfactory sequence alignment between the templates and the query, which can be used to build a structure. For general information on the panel, see the topic  Structure Prediction Panel . Using the Edit Alignment Step There are several ways to produce an ...",t:"Prime Structure Prediction — Edit Alignment Step"},"4558":{i:0.000481076159993973,u:"../prime_help/find_homologs_step.html",a:"This step is used to search for homologs of the query sequence. These homologs can then be used as templates to build a model structure. For details about the Find Homologs step, see the topic  Find Homologs Step . For general information on the panel, see the topic  Structure Prediction Panel . ...",t:"Prime Structure Prediction — Find Homologs Step"},"4559":{i:0.000292297115220118,u:"../prime_help/input_sequence_step.html",a:"This step is used to read in protein (amino acid) sequences and designate the query sequence. For general information on the panel, see the topic  Structure Prediction Panel . Using the Input Sequence Step The purpose of this step is to define a query sequence. You can read the sequence from a ...",t:"Prime Structure Prediction — Input Sequence Step"},"4560":{i:0.000448317320400846,u:"../prime_help/info_menu.html",a:"The Prime Structure Prediction menu bar contains four menus, File, Edit, Display, and Step. Some of the options in these menus are also available as toolbar buttons. The File Menu The Edit Menu The Display Menu The Step Menu Related Topics The File Menu The File menu contains commands for creating, ...",t:"Prime Structure Prediction Menu Bar"},"4561":{i:0.000304742344326572,u:"../prime_help/toolbar.html",a:"The Prime Structure Prediction toolbar provides convenient access to commonly used structure prediction and display tasks. The names of the tasks are given in the button tooltips. The tasks can also be found on the Edit and Display menus. The buttons are described below. Related Topics Prime ...",t:"Prime Structure Prediction Toolbar"},"4562":{i:0.000836429110547,u:"../prime_help/info_third.html",a:"Summary BLAST/PSI-BLAST, Pfam, and the PDB Secondary Structure Prediction: Prime and SSpro Secondary Structure Prediction: PSIPRED Third-Party Program Use Agreement Summary As well as internal and bundled software and data, Prime can use certain external third-party programs and databases for ...",t:"Prime Third-Party Programs"},"4563":{i:0.000641162202334165,u:"../prime_help/refine_loops.html",a:"Refine the structure of loops in the protein that is displayed in the Workspace. The refinement can be run with periodic boundary conditions, in implicit solvent, and with an implicit membrane. To open this panel, choose Tasks → Protein Preparation and Refinement → Refine Loops in the Task Tool. ...",t:"Refine Loops Panel"},"4564":{i:0.000343646448409932,u:"../prime_help/refine_loops_constraints.html",a:"In this panel, you can set up various kinds of constraints when performing loop refinements: constraints on distances, restraints of atoms to initial positions, restraints of C-alpha atoms to specified positions, constraints of parts of the loop to helical conformation, and constraints on the loop ...",t:"Refine Loops - Constraints Dialog Box"},"4565":{i:0.000371812444683618,u:"../prime_help/refine_loops_options.html",a:"In the Refine Loops - Options dialog box you can select options for the loop refinement, including use of an implicit membrane and crystal symmetry, choosing the method, options, and job management. To open this dialog box, click Options in the Refine Loops panel. Refine Loops - Options Dialog Box ...",t:"Refine Loops - Options Dialog Box"},"4566":{i:0.000248216154089134,u:"../prime_help/refine_protein_ligand_complex.html",a:"Refine the structure of protein-ligand complexes. The ligand can be covalently bound or bound by noncovalent interactions. The refinement can be restricted to the region around the ligand, and can sample positions and conformations of the ligand. The refinement can be run with periodic boundary ...",t:"Refine Protein-Ligand Complex Panel"},"4567":{i:0.000180051744020264,u:"../prime_help/refine_protein_ligand_complex_constraints.html",a:"In this dialog box, you can set up constraints on distances and restraints of atoms to initial positions when performing minimizations. To open this dialog box, click Constraints in the Refine Protein-Ligand Complex panel. Refine Protein-Ligand Complex - Constraints Dialog Box Features Distance tab ...",t:"Refine Protein-Ligand Complex - Constraints Dialog Box"},"4568":{i:0.000240561796043786,u:"../prime_help/refine_protein_ligand_complex_options.html",a:"In the Refine Protein-Ligand Complex - Options dialog box you can select options for refinement of a complex, including use of an implicit membrane and crystal symmetry, the solvent dielectric constant, algorithm selection, and options for each algorithm. To open this dialog box, click Options in ...",t:"Refine Protein-Ligand Complex - Options Dialog Box"},"4569":{i:0.000323876538589401,u:"../prime_help/job_options.html",a:"Select host and login information for each type of job in the Prime Structure Prediction workflow. To open this dialog box, click Job Options in any step of the Structure Prediction panel. Using the Structure Prediction - Job Options Dialog Box The Structure Prediction workflow involves running a ...",t:"Structure Prediction - Job Options Dialog Box"},"4570":{i:0.000842909443708471,u:"../prime_help/structure_prediction.html",a:"Predict the structure of a protein from its sequence by homology modeling. To open this panel, choose Tasks → Protein Preparation and Refinement → Homology Modeling in the Task Tool. Using the Structure Prediction Panel The Structure Prediction workflow is presented as a sequence of steps, each of ...",t:"Structure Prediction Panel"},"4571":{i:0.00035287377721106,u:"../prime_help/info_runs.html",a:"A run in Prime is an execution of the Prime Structure Prediction workflow using a particular set of choices of templates, paths, and settings. A Maestro project can contain more than one Prime run. This is useful for comparing the results of runs made with different selections and options chosen. ...",t:"Using Prime Runs"},"4572":{i:0.000620669982347036,u:"../primex_help/calculation_settings_db.html",a:"In this dialog box, you can make settings that affect multiple job types. These include setting resolution limits on and filtering the reflection data, choice of bulk solvent corrections, control of B-factor scaling, and inclusion of H atoms in structure output. Calculation Settings Dialog Box ...",t:"Calculation Settings Dialog Box"},"4573":{i:0.000127246470151116,u:"../primex_help/create_map_panel.html",a:"In the Create Map panel, you can set up and run a job to create various types of electron density maps. To open this panel, click Create Map in the PrimeX panel. Create Map Panel Features Map source options Map type options Coefficients options Weighting options Grid size options Extent options ...",t:"Create Map Panel"},"4574":{i:0.000120950173520384,u:"../primex_help/density_blobs.html",a:"This panel lists density \"blobs\"—collections of density peaks from a difference map that are part of a continuous region of density difference. To open this panel, click Density Blobs in the PrimeX panel. Using the Density Blobs Panel Before opening the panel, ensure that an Fo-Fc map is displayed ...",t:"Density Blobs Panel"},"4575":{i:0.00205191995824899,u:"../primex_help/density_fit_panel.html",a:"This panel displays a measure for each residue of how well the electron density predicted by the current structure fits the current electron density map. To open this panel, click Density Fit in the PrimeX panel. Using the Density Fit Panel This panel can be used to identify residues that should be ...",t:"Density Fit Panel"},"4576":{i:0.00042684604497039,u:"../primex_help/input_data_db.html",a:"In this dialog box you specify the source of the model structure, sequence, reflections, and optional map, and set the space group and unit cell parameters. When you click OK, the data is copied into the project and any requested initial processing is performed. To open this dialog box, click Input ...",t:"Input Data Dialog Box"},"4577":{i:0.000157509670287073,u:"../primex_help/ligand_solvent_placement_panel.html",a:"This panel allows you to place a ligand or solvent molecules in a region that contains an appropriate amount of density, and run a job to optimize the placement. To open this panel, click Ligand/Solvent Placement in the PrimeX panel. Using the Ligand/Solvent Placement Panel Ligand and solvent ...",t:"Ligand/Solvent Placement Panel"},"4578":{i:0.000120950173520384,u:"../primex_help/mutate_model_to_sequence.html",a:"In this dialog box, you can mutate the model to match the sequence and start a side-chain prediction job for the mutated residues. To open this panel, click Mutate Model to Sequence in the PrimeX panel. Mutate Model to Sequence Dialog Box Features Alignment to reference sequence text area Mutate to ...",t:"Mutate Model to Sequence Dialog Box"},"4579":{i:0.000116023470937823,u:"../primex_help/primex_panel.html",a:"This panel provides access to all the PrimeX functions in a dockable panel. To open this panel, choose Tasks → Protein Preparation and Refinement → Protein X-Ray Refinement in the Task Tool. Using the PrimeX Panel Refinement of an X-ray structure usually involves many passes through different tasks. ...",t:"PrimeX Panel"},"4580":{i:0.000168770793545298,u:"../primex_help/real_space_refinement_panel.html",a:"Perform real-space refinements to optimize the geometries of side chains, loops, or selected residues. The refinement minimizes a weighted function of the force-field energy and the density fit. To open this panel, click Real-Space Refinement in the PrimeX panel. Real-Space Refinement Panel Features ...",t:"Real-Space Refinement Panel"},"4581":{i:0.000477127470391112,u:"../primex_help/reciprocal_space_refinement_panel.html",a:"Perform reciprocal space refinements. These can include minimization of R-factors for the coordinates, B-factors and grouped occupancies; adjustment of parts or all of the protein as rigid bodies, and simulated annealing. To open this panel, click Reciprocal-Space Refinement in the PrimeX panel. ...",t:"Reciprocal-Space Refinement Panel"},"4582":{i:0.00019483523501568,u:"../primex_help/reciprocal_space_general_tab.html",a:"In this tab, you specify the target for refinement, the weighting of X-ray vs energy terms in the target function, and how to set the isotropic B factors. General Tab Features Target options X-ray term weight options Isotropic B-factor values options Target options There are two choices of ...",t:"Reciprocal Space Refinement — General Tab"},"4583":{i:0.000169726510449236,u:"../primex_help/reciprocal_space_maps_tab.html",a:"The Maps tab provides options for defining the parameters and extent of the maps that will be created with a refinement job. Maps Tab Features Coefficients options Weighting options Grid size options Scale map to sigma units option Extent options Coefficients options Four options are provided for ...",t:"Reciprocal Space Refinement — Maps Tab"},"4584":{i:0.000156316983188536,u:"../primex_help/reciprocal_space_minimization_tab.html",a:"In this tab, you can make settings for a minimization. Minimizations are usually performed towards the end of the refinement. Minimization Tab Features Minimizer options Number of cycles text box Maximum steps per cycle text box Select atoms to minimize section Minimize options Isotropic B-factor ...",t:"Reciprocal Space Refinement — Minimization Tab"},"4585":{i:0.00019483523501568,u:"../primex_help/reciprocal_space_NCS_tab.html",a:"In this tab, you can define restraints on the optimization to satisfy noncrystallographic symmetry. The restraints are applied to specified groups of chains. Noncrystallographic symmetry restraints are generally used only in the early stages of refinement. NCS Tab Features The features in this tab ...",t:"Reciprocal Space Refinement — NCS Tab"},"4586":{i:0.000156316983188536,u:"../primex_help/reciprocal_space_rigid_body_tab.html",a:"In this tab you can define one or more rigid bodies for rigid-body refinement. Each rigid body is defined by a group of atoms, which you can select at will. You can also decide what to do with atoms that are not part of any explicitly chosen rigid body. Rigid Bodies Tab Features Minimizer options ...",t:"Reciprocal Space Refinement — Rigid Bodies Tab"},"4587":{i:0.000156316983188536,u:"../primex_help/reciprocal_space_simulated_annealing_tab.html",a:"This tab provides controls for simulated annealing calculations. Simulated Annealing Tab Features Select atoms to minimize section Energy model options Initial minimization steps text box MD energy scale estimation steps text box Temperatures text boxes Heating steps text box Cooling steps text box ...",t:"Reciprocal Space Refinement — Simulated Annealing Tab"},"4588":{i:6.13254596701412E-05,u:"../primex_help/reference_sequence_db.html",a:"The Reference Sequence panel displays the reference sequence. You can use the Find text box and the Previous and Next buttons to find a particular sequence of residues within the reference sequence. To open this panel, click Reference Sequence in the PrimeX panel. The reference sequence is not ...",t:"Reference Sequence Panel"},"4589":{i:0.00010665618148832,u:"../primex_help/r-factor_calculation.html",a:"In this panel you can set up and run a job to calculate R-factors for the structure in the Workspace. The R values are returned as Maestro properties. To open this panel, click R-Factor Calculation in the PrimeX panel. R-Factor Calculation Panel Features Calculation Settings button Job toolbar ...",t:"R-Factor Calculation Panel"},"4590":{i:0.000258831480589518,u:"../primex_help/set_default_map_colors.html",a:"This dialog box allows you to set the default colors for the various types of maps. Each map has an option menu that offers a choice of colors. For the Fo-Fc map, there is an option menu for negative values of the difference density as well as for the positive values. The color settings that you ...",t:"Set Default Map Colors Dialog Box"},"4591":{i:8.37887830029548E-05,u:"../primex_help/water_placement_panel.html",a:"In this panel you set options and run a job to place water molecules. The options in this panel ensure that water molecules are placed in reasonable locations. To open this panel, click Water Placement in the PrimeX panel. Using the Water Placement Panel To place water molecules, you must first have ...",t:"Water Placement Panel"},"4592":{i:7.78743621210405E-05,u:"../protein_prep_help/one_step_protein_preparation.html",a:"Prepare one or more proteins for modeling, with a limited range of settings. One-Step Protein Preparation Panel Features Use structures from option menu Open Project Table button File name text box and Browse button Create disulfide bonds option Fill in missing side chains using Prime option Fill in ...",t:"One-Step Protein Preparation Panel"},"4593":{i:0.000353950838091359,u:"../protein_prep_help/proteinprep_problems.html",a:"This dialog box contains tables that list problems found with the structure after preprocessing in the Protein Preparation Wizard panel. Using the Protein Preparation - Problems Dialog Box The purpose of this panel is to list the problems found during processing of a protein structure, so that you ...",t:"Protein Preparation - Problems Dialog Box"},"4594":{i:0.00142524908731778,u:"../protein_prep_help/protein_preparation.html",a:"The purpose of this panel is to locate and fix structural defects in imported protein structures and prepare them for use by various Schrödinger products. Using the Protein Preparation Wizard Panel The Protein Preparation Wizard allows you to take a protein from its raw state, (which may be missing ...",t:"Protein Preparation Wizard Panel"},"4595":{i:0.000417206470879255,u:"../protein_prep_help/proteinprep_import_process_tab.html",a:"In this tab you can import a structure if there is not already a structure in the Workspace, select options to amend the structure, and perform the basic preparation tasks. Note: You should always check that the structure is correct after any automatic procedure is run. Import and Process Tab ...",t:"Protein Preparation Wizard — Import and Process Tab"},"4596":{i:0.000282414298405072,u:"../protein_prep_help/proteinprep_refine_tab.html",a:"In this tab you can refine the assignment of hydrogen bonds, including terminal chi flips and histidines, remove waters based on their hydrogen bonding pattern, and perform a restrained minimization. Refine Tab Features H-bond assignment section Remove waters section Restrained minimization section ...",t:"Protein Preparation Wizard — Refine Tab"},"4597":{i:0.000297656815743878,u:"../protein_prep_help/proteinprep_review_modify_tab.html",a:"In this tab you can inspect parts of the structure in the Workspace by selecting them in tables, deleted unwanted parts, and generate ionization, tautomeric, and metal-binding states for het groups. When you select a chain, water, or het group, the color scheme with green carbons is used for the ...",t:"Protein Preparation Wizard — Review and Modify Tab"},"4598":{i:0.000177746726179124,u:"../qikprop_help/qikprop.html",a:"Use this panel to set up a QikProp calculation of ADME properties and 2D and 3D QSAR descriptors. This calculation can be performed using input taken from a project, allowing properties and descriptors to be rapidly calculated for a set of entries. To open this panel, choose Tasks → ADME and ...",t:"QikProp Panel"},"4599":{i:0.000157505801312969,u:"../qsite_help/constrained_atoms_panel.html",a:"The Constrained Atoms panel is used to specify a set of atoms to be harmonically restrained during a molecular mechanics calculation. Such atoms are referred to as \"constrained\" atoms to distinguish them from \"frozen\" (completely fixed) atoms. The strength of the harmonic constraining force can be ...",t:"QSite — Constrained Atoms Panel"},"4600":{i:0.000272595086848755,u:"../qsite_help/continuum_solvation_panel.html",a:"The effects of solvent can be implicitly simulated using the finite-element Poisson Boltzmann Solver (PBF). To open this panel, click the Settings button for Use continuum solvation in the Potential tab of the QSite panel. Continuum Solvation Panel Features Solvation method option menu SGB ...",t:"Continuum Solvation Panel"},"4601":{i:0.000157505801312969,u:"../qsite_help/frozen_atoms_panel.html",a:"QSite — Frozen Atoms Panel The Frozen Atoms panel is used to specify a set of atoms to be frozen during a QSite calculation. To open this panel, click Frozen Atoms in the MM Constraints tab of the QSite panel. Frozen Atoms Panel Features Frozen atoms list Define frozen atoms picking controls Markers ...",t:"Frozen Atoms Panel"},"4602":{i:0.000509130966674867,u:"../qsite_help/qsite_panel.html",a:"QSite performs mixed quantum mechanical/molecular mechanical (QM/MM) calculations, using Jaguar for the QM calculations and Impact for the MM calculations. Ligands and other specified regions of a protein complex can be studied using QM while MM is used for the rest of the complex. At each step of a ...",t:"QSite Panel"},"4603":{i:0.00016776851937756,u:"../qsite_help/mm_constraints_tab.html",a:"Specified atoms can be frozen at their input coordinates, or they can be constrained to remain near their initial coordinates by applying a harmonic force. In QSite, atoms in both the QM and the MM regions can be frozen or constrained. In this tab you specify the MM atoms to be frozen or ...",t:"QSite Panel — MM Constraints Tab"},"4604":{i:0.000164559261066128,u:"../qsite_help/mm_minimization_tab.html",a:"The QSite Minimization tab is used to set options for the minimization of the molecular mechanics (MM) region of the system. MM Minimization Tab Features  Maximum minimization cycles text box  Algorithm option menu  Initial step size text box  Maximum step size text box  Convergence criteria option ...",t:"QSite Panel — MM Minimization Tab"},"4605":{i:0.0004489368165332,u:"../qsite_help/potential_tab.html",a:"The Potential tab contains options for controlling the calculation of the molecular-mechanics energy in a QSite calculation. Potential Tab Features  Force field option menu  Electrostatic treatment option menu  Dielectric constant text box  Use non-bonded cutoffs option and Settings button  Use ...",t:"QSite Panel — Potential Tab"},"4606":{i:0.000120150690733276,u:"../qsite_help/properties_tab.html",a:"In this tab, you select the properties you want to calculate and set any relevant options. Before you can do so, you must specify the QM region in the  QM Settings tab . The controls are the same as in the Jaguar Properties tab, with a few exceptions: Frequencies, polarizabilities, NMR shielding ...",t:"QSite Panel — Properties Tab"},"4607":{i:0.000214166343085425,u:"../qsite_help/qm_constraints_tab.html",a:"Specified atoms can be frozen at their input coordinates, or they can be constrained to remain near their initial coordinates by applying a harmonic force. In QSite, atoms in both the QM and the MM regions can be frozen or constrained. In this tab you specify the QM atoms to be frozen or ...",t:"QSite Panel — QM Constraints Tab"},"4608":{i:0.000180982843769934,u:"../qsite_help/qm_optimization_tab.html",a:"The QSite Optimization tab specifies the QM (Jaguar) calculation to be performed, the number of iterations to be performed, and any additional structures that may be needed for transition state optimizations (reactant, product, and transition state guess structures). Optimizations (minimizations and ...",t:"QSite Panel — QM Optimization Tab"},"4609":{i:0.000192554699745123,u:"../qsite_help/qm_settings_tab.html",a:"Use this tab to enter information for the quantum mechanics (QM) calculation, and to define the region to be treated quantum-mechanically. The QM information includes the QM method to be used, the charge and spin multiplicity of the QM system, and other keywords and options that may be required by ...",t:"QSite Panel — QM Settings Tab"},"4610":{i:0.000132027054355746,u:"../qsite_help/scan_tab.html",a:"In this tab, you set up the coordinates for a relaxed or a rigid coordinate scan. Whether the scan is relaxed or rigid depends on the method selection in the  QM Optimization tab : selecting Single point from the Method option menu performs a rigid scan, selecting Minimization performs a relaxed ...",t:"QSite Panel — Scan Tab"},"4611":{i:0.000272595086848755,u:"../qsite_help/truncation_panel.html",a:"The Truncation panel defines the non-bonded cutoffs settings for a QSite calculation. To open this panel, click the Settings button for Use non-bonded cutoffs in the Potential tab of the QSite panel. The use of non-bonded cutoffs to remove interactions between widely separated pairs of atoms is an ...",t:"Truncation Panel"},"4612":{i:0.000226144909123469,u:"../rgroup_help/custom_rgroup_enumeration.html",a:"Add custom or standard fragment collections to selected positions on a core-containing molecule, and enumerate the structures resulting from all possible combinations of fragments at each position. To open this panel, click Create Enumerated Entries in the 3D Builder panel, after selecting ...",t:"Custom R-Group Enumeration Panel"},"4613":{i:6.13254596701412E-05,u:"../rgroup_help/r_group_estate_atomic_contributions.html",a:"Examine contributions from each atom to the predicted property. Using the E-State RQSAR - Atomic Contributions Dialog Box You can use this dialog box to examine the contributions from each atom to the predicted property. Pausing the pointer over the atom shows its contribution. The observed and ...",t:"E-State RQSAR - Atomic Contributions Dialog Box"},"4614":{i:9.66860920806163E-05,u:"../rgroup_help/r_group_estate_rqsar.html",a:"In this panel you can create a QSAR model for the activity as a function of the kind of R-group at each attachment position, and display the results qualitatively in terms of increasing the activity, decreasing the activity, or little effect on the activity. Using the E-State RQSAR Panel The QSAR ...",t:"E-State RQSAR Panel"},"4615":{i:6.13254596701412E-05,u:"../rgroup_help/r_group_pharma_atomic_contributions.html",a:"Examine contributions from each atom to the predicted property. Using the Pharma RQSAR - Atomic Contributions Dialog Box You can use this dialog box to examine the contributions from each atom to the predicted property. Pausing the pointer over the atom shows its contribution. The observed and ...",t:"Pharma RQSAR - Atomic Contributions Dialog Box"},"4616":{i:9.66860920806163E-05,u:"../rgroup_help/r_group_pharma_rqsar.html",a:"In this panel you can create a QSAR model for the activity as a function of the kind of R-group at each attachment position, and display the results qualitatively in terms of increasing the activity, decreasing the activity, or little effect on the activity. Using the Pharma RQSAR Panel The QSAR ...",t:"Pharma RQSAR Panel"},"4617":{i:0.000224345987266125,u:"../rgroup_help/r_group_activity_cliffs.html",a:"In this panel, the differences in a selected property are plotted against differences in structure evaluated using fingerprints. The results are also listed in a table. Activity Cliffs Panel Features Property option menu Show pairs varying in options Fingerprint settings options Run button Last run ...",t:"R-Group Analysis — Activity Cliffs Panel"},"4618":{i:0.00033382403072393,u:"../rgroup_help/r_group_analysis.html",a:"The R-Group Analysis panel identifies the groups that are attached to a core and provides tools for displaying properties of the R groups and for exporting them to files. The core can be taken from CombiGlide attachment bond labeling, specified by a SMARTS pattern, or determined by the maximum ...",t:"R-Group Analysis Panel"},"4619":{i:0.00120607445605144,u:"../rgroup_help/r_group_display_settings.html",a:"In this dialog box you can make settings for the display of the R-Group heat map. Display Settings Dialog Box Features Notation options Digits after decimal box Maximum value and Minimum value text boxes Min and Max sliders Reverse Map button Notation options Select a notation option for the display ...",t:"R-Group Analysis — Display Settings Dialog Box"},"4620":{i:0.000262592052157653,u:"../rgroup_help/r_group_heat_map.html",a:"This panel displays a heat map of a selected property for R groups at two positions, averaged over values at any other positions. Heat Map Panel Features Displayed property section R groups section R group position importance section Heat map Display Settings button Cell display option menu Always ...",t:"R-Group Analysis — Heat Map Panel"},"4621":{i:0.000217261330339475,u:"../rgroup_help/r_group_importance_analysis.html",a:"The importance analysis panel displays information on the range of property values for each attachment position and the values of the properties for each of the R groups at a selected position. Importance Analysis Panel Features Displayed property section Cores display R-Group position importance ...",t:"R-Group Analysis — Importance Analysis Panel"},"4622":{i:0.000168155287383353,u:"../rgroup_help/r_group_sar_table.html",a:"In this panel you can view the properties of the input structures in a table, with the R groups shown for each structure. SAR Table Panel Features Cores display View SMILES button View 2D Images button Select Properties button Sort button Table R-group size slider and text box Export Image button ...",t:"R-Group Analysis — SAR Table Panel"},"4623":{i:0.00010665618148832,u:"../rgroup_help/r_group_select_rgroups.html",a:"In this panel you can select the R groups for each position. Select R Groups Panel Features Position option menu R-group table Select All R-Groups, All Positions button Position option menu Choose the position for whicy you want to select R groups. The R groups for the chosen position are displayed ...",t:"R-Group Analysis — Select R Groups Panel"},"4624":{i:0.000332792839089739,u:"../rgroup_help/r_group_viewer.html",a:"In this panel you can display the structures and the R Groups for each structure, set limits on what is displayed, and export the R Groups and the structures. The panel opens automatically after the structure analysis has been run. R-Group Viewer Panel Features Input Structures section R Groups ...",t:"R-Group Viewer Panel"},"4625":{i:0.00010665618148832,u:"../semiempirical_nddo_help/mopac2012.html",a:"In this panel, you can set up and run a semiempirical calculation on one or more structures using the MOPAC2012 code. To open this panel, choose Tasks → Quantum Mechanics → MOPAC2012 in the Task Tool. MOPAC2012 Panel Features Use structures from option menu Open Project Table button File name text ...",t:"MOPAC2012 Panel"},"4626":{i:0.00010665618148832,u:"../semiempirical_nddo_help/semiempirical_nddo.html",a:"In this panel, you can set up and run a semiempirical calculation on one or more structures using the semiempirical NDDO module. To open this panel, choose Tasks → Quantum Mechanics → Semiempirical NDDO in the Task Tool. Semiempirical NDDO Module Panel Features Use structures from option menu Open ...",t:"Semiempirical NDDO Module Panel"},"4627":{i:0.00026296630676164,u:"../shape_screening_help/shape_screening.html",a:"In this panel you can set up and run a search for molecules that have similar shape to a shape query, which is either a template molecule or a set of spheres of defined radius and coordinates. To open this panel, choose Tasks → Ligand-Based Virtual Screening → Shape Screening in the Task Tool. Using ...",t:"Shape Screening Panel"},"4628":{i:0.000184412655957943,u:"../shape_screening_help/shape_screening_options_db.html",a:"In this dialog box you can select a variety of options to control the process and results of a shape screen. To open this dialog box, click Options in the Shape Screening panel. Shape Screening - Options Dialog Box Features Write out at most N alignments per ligand text box Use excluded volumes ...",t:"Shape Screening - Options Dialog Box"},"4629":{i:7.78743621210405E-05,u:"../sitemap_help/sitemap_panel.html",a:"SiteMap is a tool for finding, visualizing and characterizing protein binding sites. Sites are represented as collections of site points at or near the surface of a protein that are contiguous or are separated in solvent-exposed region by short gaps that could plausibly be spanned by ligand ...",t:"SiteMap Panel"},"4630":{i:0.000130769215300625,u:"../strike_help/strike_calcsim.html",a:"Use this panel to determine similarities in descriptor or 2D-structure space. Several distance-based descriptor similarity measures are available. Similarity in 2D-structure space is determined using an atom-pair-based approach. To open this panel, choose Tasks → Discovery Informatics and QSAR → ...",t:"Calculate Similarity Panel"},"4631":{i:0.000130769215300625,u:"../strike_help/strike_build.html",a:"The Build QSAR Model panel is used to generate QSAR/QSPR models using multiple linear regression (MLR), partial least squares (PLS), or principal component analysis (PCA). To open this panel, choose Tasks → Discovery Informatics and QSAR → Property-Based QSAR → Build Model in the Task Tool. Using ...",t:"Build QSAR Model Panel"},"4632":{i:0.00010665618148832,u:"../strike_help/factor_analysis.html",a:"Performs a principal component analysis of a set of descriptors for selected entries and displays information about the analysis. To open this panel, choose Tasks → Discovery Informatics and QSAR → Factor Analysis in the Task Tool. Using the Factor Analysis Panel To do an analysis, you must select a ...",t:"Factor Analysis Panel"},"4633":{i:0.000130769215300625,u:"../strike_help/strike_predict.html",a:"Use this panel to make predictions about molecular properties based on a QSAR/QSPR model. To open this panel, choose Tasks → Discovery Informatics and QSAR →\nProperty-Based QSAR → Predict in the Task Tool. Using the Predict Based on QSAR Model Panel Models to be used for property predictions can be ...",t:"Predict Based on QSAR Model Panel"},"4634":{i:0.000221892857605206,u:"../strike_help/strike_intro.html",a:"Strike is a chemically-aware statistical package which is integrated with Maestro to provide a flexible, intuitive interface. Using molecular data generated by Schrödinger software or other sources such as third-party software or experimental data, Strike can be used to generate basic univariate and ...",t:"Strike Overview"},"4635":{i:7.78743621210405E-05,u:"../strike_help/strike_getstats.html",a:"This panel displays univariate and bivariate statistics for the selected descriptors. To open this panel, choose Tasks → Discovery Informatics and QSAR → Property-Based QSAR → Statistics in the Task Tool. Using the Univariate and Bivariate Statistics Panel The Univariate and Bivariate Statistics ...",t:"Univariate and Bivariate Statistics Panel"},"4636":{i:0.000510570857874618,u:"../vsw_help/vsw_panel.html",a:"The Virtual Screening Workflow panel enables you to set up and run jobs to screen ligands against one or more targets. The workflow includes ligand preparation, filtering, and up to three docking stages with Glide, progressing from HTVS to SP to XP docking. For docking to multiple targets (ensemble ...",t:"Virtual Screening Workflow Panel"},"4637":{i:0.000218551036277074,u:"../vsw_help/vsw_docking_tab.html",a:"In this tab and its dependent dialog box, you can set up parameters for the ligand docking, and choose the docking stages that are run. Docking Tab Features Enhance planarity of conjugated pi groups option Use Epik state penalties for docking option Write interaction scores for residues within N Å ...",t:"Virtual Screening Workflow — Docking Tab"},"4638":{i:0.000136039902275355,u:"../vsw_help/vsw_filtering_tab.html",a:"In this tab you can specify how the ligands will be filtered before they are docked. Filtering Tab Features Run QikProp option Prefilter by Lipinski\u0027s Rule option Remove ligands with reactive functional groups option Retain only ligands matching criteria option and buttons Run QikProp option Select ...",t:"Virtual Screening Workflow — Filtering Tab"},"4639":{i:0.000136039902275355,u:"../vsw_help/vsw_input_tab.html",a:"In this tab you specify the source of the ligands to dock, and set options for how they are used in subjobs. Input Tab Features Source of ligands options Identify unique compounds by options Source of ligands options These options and the associated controls allow you to specify the source of the ...",t:"Virtual Screening Workflow — Input Tab"},"4640":{i:0.000169560553970675,u:"../vsw_help/vsw_preparation_tab.html",a:"This tab allows you to make settings for ligand preparation, which is done with LigPrep. If your ligands are already 3D structures in the appropriate ionization and tautomeric state, you can deselect the Prepare ligands option. Preparation Tab Features Prepare ligands option Regularize input ...",t:"Virtual Screening Workflow — Preparation Tab"},"4641":{i:0.000146589105561428,u:"../vsw_help/vsw_receptors_tab.html",a:"In this tab you select the receptors you want to use, and set options for grid generation. Receptors Tab Features Receptors for docking table Add button Edit button Delete button Add Multiple button Align receptors option Receptors for docking table Lists the receptors to use for docking. You can ...",t:"Virtual Screening Workflow — Receptors Tab"},"4642":{i:9.55062380776321E-05,u:"../vsw_help/vsw_receptor_constraints_db.html",a:"In this dialog box you can select receptor-based constraints, and specify a ligand core constraint, for Virtual Screening Workflow jobs. To open this dialog box, click Constraints in the VSW - Receptor dialog box. Receptor Constraints Dialog Box Features Constraints table New Positional button ...",t:"VSW - Receptor Constraints Dialog Box"},"4643":{i:0.00016971428549674,u:"../vsw_help/vsw_docking_options_db.html",a:"This dialog box provides access to options that can be applied to each docking stage. It is opened for each docking stage with appropriate default options set and with the features that apply to the docking stage. To open this dialog box, click Options in the Docking tab of the Virtual Screening ...",t:"VSW - Docking Options Dialog Box"},"4644":{i:0.000126658070427159,u:"../vsw_help/vsw_receptor_db.html",a:"In this dialog box you specify the options for reading or generating the receptor grids. In addition, you can select constraints to be used in docking to the receptor, and provide an identifier and a GlideScore offset. To open this dialog box, click Add or Edit in the Receptors tab of the Virtual ...",t:"VSW - Receptor Dialog Box"},"4645":{i:7.32720296430132E-05,u:"../watermap_help/rendering_sites.html",a:"If you want to render water sites generated by WaterMap using CPK or ball-and-stick rather than using the WaterMap analysis interface, but color the sites by free energy using the same color scheme as in the WaterMap analysis interface, you can do so as follows. Copy the color scheme resource files ...",t:"Custom Rendering of WaterMap Sites"},"4646":{i:0.00012648788047902,u:"../watermap_help/examine_results.html",a:"In this panel you can examine WaterMap simulation results, the enthalpy and entropy of individual water sites in the ligand binding sites of proteins relative to bulk water derived from the simulation, and visualize these sites (and their thermodynamic properties) in the Workspace, in relation to ...",t:"WaterMap - Examine Results Panel"},"4647":{i:0.00010224166730854,u:"../watermap_help/perform_calculation.html",a:"In this panel you can set up and run a WaterMap simulation, which uses Desmond to run a molecular dynamics simulation of water molecules in the ligand binding site of a receptor, which is restrained during the simulation. The excess enthalpy and entropy, and thus the free energy, of the water sites ...",t:"WaterMap - Perform Calculation Panel"},"4648":{i:8.55716728406209E-05,u:"../watermap_help/wmmm_scoring_panel.html",a:"The WM/MM Scoring panel allows you to set up and run a job for estimation of the free energy of binding, using Prime MM-GBSA for the terms involving the internal energies, Coulomb and van der Waals interactions, ligand desolvation, and internal strain, and the WaterMap score for the water ...",t:"WM/MM Scoring Panel"},"4649":{i:0.000271605558175138,u:"../wscore_help/wscore_docking.html",a:"Dock ligands to a WScore model, which includes an ensemble of protein structures for assessment of small induced fit effects and protein reorganization. The scoring includes terms for water displacement and desolvation effects, molecular recognition (H-bonds, hydrophobic enclosure, etc), ligand ...",t:"WScore - Docking Panel"},"4650":{i:0.000149461312155822,u:"../wscore_help/wscore_model_generation.html",a:"Build a trained scoring model for docking to a particular protein target. The model uses an ensemble of protein structures (conformations) for the target, data from assessment of water thermodynamics from MD simulations (WaterMap), experimental binding affinities for known ligands (which form a ...",t:"WScore - Model Generation Panel"},"4651":{i:0.000173582347483542,u:"../wscore_help/wscore_advanced_options.html",a:"Edit WScore parameters for the ensemble selected in the WScore - Docking panel. This panel is not intended for general use, as the parameters shown here are already optimized in the model generation.  To open this panel, click Advanced Options in the WScore - Docking panel. Advanced Options Dialog ...",t:"WScore Docking—Advanced Options Dialog Box"},"4652":{i:0.000173582347483542,u:"../wscore_help/define_core_constraints.html",a:"Set constraints on the core of the docked ligands, to reproduce the position of a reference ligand. To open this dialog box, click Define Core Constraints in the WScore - Docking panel. Define Core Constraints Dialog Box Features Restrict docking to reference position option Workspace ligand option, ...",t:"WScore Docking—Define Core Constraints Dialog Box"},"4653":{i:8.55716728406209E-05,u:"../wscore_help/wscore_visualizer.html",a:"WScore Visualizer Panel Visualize WScore scoring terms in the Workspace, showing the parts of the structure that contribute to the score. To open this panel, click the WS button for the desired ligand in the Entry List.\n WScore Visualizer Panel Features Receptor and ligand titles Table Receptor and ...",t:"Wscore Visualizer Panel"},"4654":{i:0.00286063998328228,u:"../desmond_command_reference/desmond_command_referenceTOC.htm",a:"Desmond Command Reference Manual",t:"Desmond Command Reference Manual"},"4655":{i:9.02151491469484E-05,u:"../bioluminate_user_manual/protein_aggregation_using_results.htm",a:"To reduce the likelihood that the protein will aggregate, you might want to mutate residues in the aggregation regions.  One way of doing this is to perform a set of mutations to reduce the size of the aggregation regions. You can create a set of structures that have single mutations at selected ...",t:"Using the Aggregation Results for Mutation Studies"},"4656":{i:0.000104397313827884,u:"../bioluminate_user_manual/protein_aggregation.htm",a:"Protein aggregation often occurs via hydrophobic regions. You can locate and analyze these regions using the Aggregation Surface panel, which you open by choosing Tasks → Biologics → Aggregation Surface in the Task Tool. Aggregation regions are defined by identifying clusters of exposed hydrophobic ...",t:"Predicting Aggregation Regions"},"4657":{i:0.00076414818894072,u:"../vsw_user_manual/vsw_user_manualTOC.htm",a:"Virtual Screening Workflow User Manual",t:"Virtual Screening Workflow User Manual"},"4658":{i:0.00198683226958624,u:"../jaguar_quick_start/jaguar_quick_startTOC.htm",a:"Jaguar Quick Start Guide",t:"Jaguar Quick Start Guide"},"4659":{i:0.000119677122257289,u:"../vsw_user_manual/vsw_docking_stages_common.htm",a:"There are two common options for the docking method for all stages: Dock flexibly and Dock rigidly. For SP and XP docking, two more options are available: Refine and Score in place.  The VSW - Docking Options dialog box. Under Dock flexibly, the Amide bonds option menu enables you to specify how to ...",t:"VSW Common Docking Options"},"4660":{i:0.00104447641229145,u:"../qsite_user_manual/qsite_user_manualTOC.htm",a:"QSite User Manual",t:"QSite User Manual"},"4661":{i:0.000384772516229199,u:"../field_based_qsar_user_manual/field_based_qsar_user_manualTOC.htm",a:"Field-Based QSAR User Manual",t:"Field-Based QSAR User Manual"},"4662":{i:0.000129819586052665,u:"../field_based_qsar_tutorial/fqsar_tutorial.htm",a:"Field-Based QSAR Tutorial This tutorial provides exercises for developing a field-based QSAR model from Maestro. ",t:"Field-Based QSAR Tutorial"},"4663":{i:0.00429060981071789,u:"../canvas_user_manual/canvas_user_manualTOC.htm",a:"Canvas User Manual",t:"Canvas User Manual"},"4664":{i:0.00114191594309735,u:"../prime_quick_start/prime_quick_startTOC.htm",a:"Prime Quick Start Guide",t:"Prime Quick Start Guide"},"4665":{i:0.000127158704216267,u:"../bioluminate_user_manual/protein_aggregation_creating_surface.htm",a:"Follow the steps below to create a surface that displays the likely aggregation regions. (Optional) Select atoms to define the structure for the surface. If you want to create a surface for the entire protein and you have only one entry in the Workspace, you can skip this step. Otherwise, you can ...",t:"Creating an Aggregation Surface"},"4666":{i:0.00215574966989396,u:"../combiglide_user_manual/combiglide_user_manualTOC.htm",a:"CombiGlide User Manual",t:"CombiGlide User Manual"},"4667":{i:0.000571678715449052,u:"../inducedfit_tutorial/inducedfit_tutorialTOC.htm",a:"Induced Fit Docking Tutorial",t:"Induced Fit Docking Tutorial"},"4668":{i:0.000794398799847749,u:"../combiglide_command_reference/combiglide_command_referenceTOC.htm",a:"CombiGlide Command Reference Manual",t:"CombiGlide Command Reference Manual"},"4669":{i:6.13254596701412E-05,u:"../Home.htm",a:"This page provides information on the documentation for the 2017-1 release of Schrödinger software. The pane to the left contains a Contents tree, in which you can find documents by type and product, organized in a tree structure. To search the documentation, use the search box at the top left of ...",t:"Schrödinger Documentation—2017-1 Release"},"4670":{i:0.00163197879867577,u:"../ligprep_command_reference/ligprep_command_referenceTOC.htm",a:"LigPrep Command Reference Manual",t:"LigPrep Command Reference Manual"},"4671":{i:0.000955604161289355,u:"../inducedfit_user_manual/inducedfit_user_manualTOC.htm",a:"Induced Fit Docking User Manual",t:"Induced Fit Docking User Manual"},"4672":{i:0.000113388189324406,u:"../vsw_user_manual/vsw_docking_stages_xp.htm",a:"For XP docking there are several additional options: Generate multiple input conformations Run a MacroModel conformational search job on the input structures corresponding to the best poses from the previous stage to locate the lowest-energy conformer, using two different force fields. This option ...",t:"Specific VSW Options for XP Docking"},"4673":{i:0.00169391921701498,u:"../general_utilities/general_utilitiesTOC.htm",a:"General Utilities",t:"General Utilities"},"4674":{i:0.000125423439623084,u:"../qpld_user_manual/qpld.htm",a:"The QM-Polarized Ligand Docking (QPLD) protocol aims to improve the partial charges on the ligand atoms in a Glide docking run by replacing them with charges derived from quantum mechanical calculations on the ligand in the field of the receptor. In this way the polarization of the charges on the ...",t:"QM-Polarized Ligand Docking"},"4675":{i:0.000970734645193537,u:"../primex_command_reference/primex_command_referenceTOC.htm",a:"PrimeX Command Reference Manual",t:"PrimeX Command Reference Manual"},"4676":{i:0.000202585164095468,u:"../vsw_command_reference/vsw_command_referenceTOC.htm",a:"Virtual Screening Workflow Command Reference Manual",t:"Virtual Screening Workflow Command Reference Manual"},"4677":{i:0.00316545800877602,u:"../glide_user_manual/glide_user_manualTOC.htm",a:"Glide User Manual",t:"Glide User Manual"},"4678":{i:0.000686221278296116,u:"../python_pipeline/python_pipelineTOC.htm",a:"Python Pipeline Command Reference Manual",t:"Python Pipeline Command Reference Manual"},"4679":{i:0.000237584339171566,u:"../general_utilities/general_utilities_applyhtreat.htm",a:"\n            Structure Preparation: applyhtreat \n Apply a hydrogen treatment (add or delete hydrogens) to one or more structures in a file. The syntax is: applyhtreat input-file output-file [options] [-t] treatment [[-a] ASL-expression] The input file and output file can be in any format that is ...",t:"\n            Structure Preparation: applyhtreat \n"},"4680":{i:0.000927682112912416,u:"../maestro_command_reference/maestro_crm_overview.html",a:"The Maestro Command Reference Manual provides information on Maestro\u0027s internal languages and commands. The Atom Specification Language  defines the language used to specify a set of atoms for some action. The mmsubs Substructure Notation  defines a linear substructure notation used for matching ...",t:"Maestro Command Reference Overview"},"4681":{i:0.000246891300975298,u:"../shape_screening_user_manual/shape_screening_user_manualTOC.htm",a:"Shape Screening User Manual",t:"Shape Screening User Manual"},"4682":{i:0.000311470618594294,u:"../qsite_command_reference/qsite_command_referenceTOC.htm",a:"QSite Command Reference Manual",t:"QSite Command Reference Manual"},"4683":{i:0.00387873782587985,u:"../materials_user_manual/materials_user_manualTOC.htm",a:"Materials Science User Manual",t:"Materials Science User Manual"},"4684":{i:0.00275059229864051,u:"../job_control/job_controlTOC.htm",a:"Job Control Guide",t:"Job Control Guide"},"4685":{i:0.00073715482669171,u:"../knime_user_manual/knime_user_manualTOC.htm",a:"KNIME Extensions User Manual",t:"KNIME Extensions User Manual"},"4686":{i:0.000268553455088765,u:"../sitemap_tutorial/sitemap_tutorialTOC.htm",a:"SiteMap Tutorial",t:"SiteMap Tutorial"},"4687":{i:9.87410140967586E-05,u:"../qsite_tutorial/qsite_tutorial.htm",a:"This tutorial is designed to help you quickly become familiar with QSite using the Maestro interface. In this tutorial, you will perform a QSite geometry minimization on a protein-ligand complex. Density functional theory (DFT) will be used to treat the QM region, which will consist of the ligand ...",t:"Modeling Ligand-Receptor Interactions with QSite"},"4688":{i:0.0013702921055864,u:"../primex_quick_start/primex_quick_startTOC.htm",a:"PrimeX Quick Start Guide",t:"PrimeX Quick Start Guide"},"4689":{i:0.000102537119998624,u:"../ligprep_tutorial/ligprep_tutorial.htm",a:"Preparing Ligands with LigPrep The LigPrep panel in Maestro enables you to prepare and run a variety of ligand preparation jobs. This tutorial provides examples of some ligands and illustrates the effect that LigPrep has on their structure.",t:"Preparing Ligands with LigPrep"},"4690":{i:0.00293221560414363,u:"../primex_user_manual/primex_user_manualTOC.htm",a:"PrimeX User Manual",t:"PrimeX User Manual"},"4691":{i:0.000119258520051297,u:"../materials_command_reference/qe_command.htm",a:"The Quantum ESPRESSO interface in the Materials Science Suite supports only a limited number of the options that are available in the Quantum ESPRESSO code (PW). Nevertheless, you can still combine the structure building and visualization functions of the Materials Science Suite with full ...",t:"Running Quantum ESPRESSO from the Command Line"},"4692":{i:0.000127506242709935,u:"../field_based_qsar_command_reference/field_based_qsar_command_referenceTOC.htm",a:"Field-Based QSAR Command Reference Manual",t:"Field-Based QSAR Command Reference Manual"},"4693":{i:0.00284704284041347,u:"../bioluminate_quick_start/bioluminate_quick_startTOC.htm",a:"BioLuminate Quick Start Guide",t:"BioLuminate Quick Start Guide"},"4694":{i:0.000231992813352223,u:"../shape_screening_command_reference/shape_screening_command_referenceTOC.htm",a:"Shape Screening Command Reference Manual",t:"Shape Screening Command Reference Manual"},"4695":{i:0.000436128137443286,u:"../glide_command_reference/glide_command_referenceTOC.htm",a:"Glide Command Reference Manual",t:"Glide Command Reference Manual"},"4696":{i:0.00131507423283259,u:"../prime_command_reference/prime_command_referenceTOC.htm",a:"Prime Command Reference Manual",t:"Prime Command Reference Manual"},"4697":{i:0.000126244986786909,u:"../bioluminate_user_manual/protein_aggregation_surface_options.htm",a:"You can control some of the parameters of the quality and appearance of the aggregation surface and the parameters that are used for locating aggregation regions in the Aggregation Surface - Options dialog box. To open this dialog box, click Options in the Aggregation Surface panel. The parameters ...",t:"Setting Options for Aggregation Surfaces"},"4698":{i:0.00180165664846057,u:"../protein_prep_user_manual/protein_prep_user_manualTOC.htm",a:"Protein Preparation Guide",t:"Protein Preparation Guide"},"4699":{i:0.00543282593659259,u:"../jaguar_command_reference/jaguar_command_referenceTOC.htm",a:"Jaguar Command Reference Manual",t:"Jaguar Command Reference Manual"},"4700":{i:0.00148694725017507,u:"../canvas_command_reference/canvas_command_referenceTOC.htm",a:"Canvas Command Reference Manual",t:"Canvas Command Reference Manual"},"4701":{i:0.000644627627544546,u:"../core_hopping_user_manual/core_hopping_user_manualTOC.htm",a:"Core Hopping User Manual",t:"Core Hopping User Manual"},"4702":{i:0.000222572023104773,u:"../sitemap_command_reference/sitemap_command_referenceTOC.htm",a:"SiteMap Command Reference Manual",t:"SiteMap Command Reference Manual"},"4703":{i:0.000123945629730972,u:"../phase_help/excluded_volumes_db.html",a:"In the Excluded Volumes dialog box, you can pick atoms to define a volume that should not contain atoms in any active ligand that matches the hypothesis, or generate them using some automated procedures. Using the Excluded Volumes Dialog Box The Excluded Volumes dialog box provides means to add ...",t:"Excluded Volumes Dialog Box"},"4704":{i:0.00212749013972803,u:"../glide_quick_start/glide_quick_startTOC.htm",a:"Glide Quick Start Guide",t:"Glide Quick Start Guide"},"4705":{i:0.000101959489909807,u:"../bioluminate_user_manual/protein_aggregation_analyzing_surface.htm",a:"The Analysis tab provides some tools for analyzing which residues contribute to the aggregation regions. To analyze a surface, choose it from the Surface option menu and click Analyze. When the analysis finishes, the table is filled in with a list of residues that contribute to the aggregation ...",t:"Analyzing the Aggregation Surface"},});