define({"229":{i:0.00014307771703383,u:"../ligprep_tutorial/ligprep_tutorial_variations_importing.htm",a:"Choose File → Import Structures. The Import panel opens. Select the file 2D_variations.sdf. If the import options are not displayed, click Options. Ensure that Import all structures is selected. Click Open. The first structure should be displayed in the Workspace. This structure has two ionizable ...",t:"Importing and Examining the Structures To Be Varied"},"230":{i:0.000145446208556413,u:"../ligprep_tutorial/ligprep_tutorial_variations_running.htm",a:"In the Entry List, click on the 2D_variations entry group row. In the Task Tool browser, choose Tasks → Ligand Preparation and Library Design → LigPrep.  The LigPrep panel opens. For Use structures from, select Project Table (2 selected entries). Check that the default options are selected: ...",t:"Setting Up and Running the LigPrep Variation Job"},"231":{i:0.000102537119998624,u:"../ligprep_tutorial/ligprep_tutorial_variations_examining.htm",a:"In the Entry List, click the In column for entry 3, then control-click the In column for entry 4 (command-click on Mac).  The two structures generated from the first 2D structure are displayed in the Workspace, superimposed. Note the variations in the structures and the absence of the quaternary ...",t:"Examining the Output Variations from LigPrep"},"232":{i:0.000110946176837651,u:"../macromodel_quick_start/macromodel_tutorial.htm",a:"These exercises are designed to help you learn the basic tasks for preparing and initiating MacroModel calculations from Maestro. Once you have worked through these exercises, you will have an understanding of the basic MacroModel features. The exercises are divided into groups: Energy Calculation ...",t:"MacroModel Tutorial"},"233":{i:0.000175122691552226,u:"../macromodel_quick_start/preparation_copying_files.htm",a:"The files you need to run the tutorial are included in the software distribution. You can create a working directory and copy the files to it from the Tutorials panel.  If the Tutorials panel is not already open: Choose Help → Tutorials. The Tutorials panel opens.  Make choices from the option menus ...",t:"Copying the MacroModel Tutorial Files"},"234":{i:0.000224763120596921,u:"../macromodel_quick_start/maestro_import.htm",a:"To display an existing structure in the Workspace, you must import the structure into the current project. Follow the instructions below to import the 1err protein-ligand complex into the project. Choose File → Import Structures. The Import panel opens. It should display the common structure files ...",t:"Importing the 1err Structure"},"235":{i:0.000180331556648603,u:"../macromodel_quick_start/mm_energy_calcs_tut.htm",a:"Energy Calculation Exercises Many types of energetic calculations are available using MacroModel. This section introduces the MacroModel energetic panels and basic energetic parameters. These exercises calculate the molecular mechanics energy of a structure in gas phase, then in solution phase. ...",t:"Energy Calculations"},"236":{i:0.000153632394432889,u:"../macromodel_quick_start/energy_calcs_gas.htm",a:"Choose Applications → MacroModel → Current Energy in the Task Tool.  Choose Workspace (included entries) from the Use structures from option menu. In the Potential tab, choose MMFFs from the Force Field option menu and choose None from the Solvent option menu. In the ECalc tab, choose Complete from ...",t:"Calculating the Gas-Phase Energy"},"237":{i:0.000127016452214403,u:"../macromodel_quick_start/energy_calcs_solution.htm",a:"Include the Ecalc entry in the Workspace. Choose Applications → MacroModel → Current Energy in the Task Tool. Choose Workspace (included entries) from the Use structures from option menu. In the Potential tab, choose MMFFs from the Force Field option menu, and choose Water from the Solvent option ...",t:"Calculating the Solution-Phase Energy"},"238":{i:0.000104276421205877,u:"../macromodel_quick_start/mm_minimization_tut.htm",a:"Exercises MacroModel energy minimizations are set up from the Minimization panel in Maestro. Minimization calculations can be performed on single structures and multi-structure collections. In addition, for single structure calculations, the MacroModel substructure facility can be used to select ...",t:"Energy Minimization"},"239":{i:0.000109980210189403,u:"../macromodel_quick_start/minimization_single.htm",a:"Choose Tasks → General Modeling → Minimization → Force-Field or Tasks → General Modeling → Minimization →  Force-Field in the Task Tool. Choose Workspace (included entries) from the Use structures from option menu.  Click the Settings button to open the Job Settings dialog box. Under Incorporate, ...",t:"Energy Minimization of a Single Structure"},"240":{i:0.000124677372549715,u:"../macromodel_quick_start/minimization_comparing.htm",a:"One useful way of comparing structural results is to superimpose them. Maestro provides tools for superimposing molecules based on a selection of atoms. For a pair of molecules, you can select the corresponding atoms manually, and Maestro superimposes them by minimizing the RMSD of the selected atom ...",t:"Comparing Structural Results by Superposition"},"241":{i:0.000113605305376412,u:"../macromodel_quick_start/minimization_multiple.htm",a:"A collection of structures, either conformers or non-conformers, can be minimized in one computation using the Minimization panel.  Choose File → Import Structures The Import panel opens. If the options are not displayed, click Options. Ensure that Import all structures is selected. Select ...",t:"Energy Minimization of Multiple Structures"},"242":{i:0.00012133476837151,u:"../macromodel_quick_start/minimization_substructure.htm",a:"The time required to minimize large structures can be drastically reduced by focusing on a particularly important section of the structure and restraining, freezing, or ignoring the rest. This exercise uses the protein-ligand complex from  Importing the 1err Structure  to perform a substructure ...",t:"Energy Minimization of a Substructure"},"243":{i:0.000112021290906687,u:"../macromodel_quick_start/mm_conf_search_tut.htm",a:"Conformational Search Exercises The goal of conformational searching is to locate the low-energy configurations of a molecular structure. MacroModel includes a number of conformational searching algorithms as well as mixed methods. This exercise first explores three standard conformational searches, ...",t:"Conformational Searches"},"244":{i:0.000137313451781853,u:"../macromodel_quick_start/conf_search_mcmm.htm",a:"The first conformational search is a Monte Carlo Multiple Minimum (MCMM), which generates trial conformations by randomly adjusting rotatable bonds.  Import MCMM.mae from your working directory.  Need help?  Use the directions in  Importing the 1err Structure . Choose Tasks → General Modeling → ...",t:"MCMM Conformational Search"},"245":{i:0.000133532086528829,u:"../macromodel_quick_start/conf_search_serial_mcmm.htm",a:"Serial MCMM conformational searches perform an MCMM conformation search on each input structure, with MCMM parameters that are set up automatically (by means of an AUTO opcode in the command file). Import Serial.mae from your working directory.  Need help?  Use the directions in  Importing the 1err ...",t:"Serial MCMM Conformational Search"},"246":{i:0.000117522072496264,u:"../macromodel_quick_start/conf_search_serial_lmod.htm",a:"Low-mode searching explores the low-frequency eigenvectors of the system to generate new conformations. A low-mode calculation does not require the designation of ring structures and variable torsion angles. Import Serial.mae from your working directory, if you did not do so in  Serial MCMM ...",t:"Serial Low-Mode Search"},"247":{i:0.000130910631160986,u:"../macromodel_quick_start/conf_search_ligand_frozen_receptor.htm",a:"In  Energy Minimization of a Substructure , a protein-ligand complex was minimized using the OPLS_2005 force field. There are multiple approaches to performing a subsequent conformational search on the complex. Two methods are demonstrated in the following two sections. This first exercise ...",t:"Ligand Conformational Search with a Frozen Receptor"},"248":{i:0.000128061628440289,u:"../macromodel_quick_start/conf_search_ligand_frozen_receptor_setup.htm",a:"Import the minimized structure LigandMCMM.mae from your working directory.  Need help?  Use the directions in  Importing the 1err Structure . Choose Tasks → General Modeling → Conformational Search → Advanced Search in the Task Tool. Choose Workspace (included entries) from the Use structures from ...",t:"Initial Setup for Ligand Conformational Search"},"249":{i:0.000137345617119286,u:"../macromodel_quick_start/conf_search_ligand_frozen_receptor_parameters.htm",a:"Note that this setup would not be adequate for a complete search of conformational space.  Click the  Select ligands (only) button under Quick Select on the toolbar. In the Style toolbox, click the Display selected atoms only button. Click the Fit to Ligand button on the toolbar. In the CSearch tab ...",t:"Setting Up Conformational Search Parameters Manually for the Ligand"},"250":{i:0.000140232341587948,u:"../macromodel_quick_start/conf_search_ligand_frozen_receptor_protein.htm",a:"This exercise uses the Substructure facility to freeze the receptor atoms that are within 6 Å of the ligand. MacroModel automatically ignores any remaining atoms in the computation. Display all atoms by clicking in the Workspace, then clicking the Display atomsbutton in the Style toolbox. In the ...",t:"Freezing the Protein in the Conformational Search"},"251":{i:0.000141136134214894,u:"../macromodel_quick_start/conf_search_ligand_frozen_receptor_run.htm",a:"Click the Job Settings button to open the Job Settings dialog box. Choose Append new entries as a new group from the Incorporate option menu. Enter LigandMCMM in the Name text box. Click Run to launch the job.  This computation will take one to three hours, depending on your computer. When the ...",t:"Running the Ligand Conformational Search Job and Examining the Results"},"252":{i:0.000141438999935942,u:"../macromodel_quick_start/conf_search_subs_auto.htm",a:"The last exercise demonstrated a computation in which the ligand was manually assigned Monte Carlo conformational search parameters while the entire receptor was held frozen. This exercise demonstrates a modified conformational search that enables increased receptor flexibility, using Perform ...",t:"Substructure Conformational Search with Automatic Setup"},"253":{i:0.000141604042301056,u:"../macromodel_quick_start/conf_search_subs_auto_initial.htm",a:"Import the structure in SubsAuto.mae from your working directory.  Need help?  Use the directions in  Importing the 1err Structure . Display the structure in the Workspace. Choose Tasks → General Modeling → Conformational Search → Advanced Search in the Task Tool. Choose Workspace (included entries) ...",t:"Making Initial Settings for Substructure Conformational Search"},"254":{i:0.00014188364253299,u:"../macromodel_quick_start/conf_search_subs_auto_shells.htm",a:"Setting Up Moving, Fixed, and Frozen Atoms In this part of the exercise, the freely-moving portion includes the ligand, as well as all residues within 3 Å of the ligand. Click the Substructure tab. In the Freely moving atoms (substructure) section, choose Molecules from the Pick menu and click on an ...",t:"Setting Up the Substructure and Shells that Define Moving, Fixed, and Frozen Atoms"},"255":{i:0.000142705429287629,u:"../macromodel_quick_start/conf_search_subs_auto_variables.htm",a:"In this part of the exercise, you will use the automatic setup features to define the MCMM conformational search variables. You can set up variables either for the entire moving region (the substructure) or only for the ligand molecule. Atoms in the fixed or frozen atom regions do not have ...",t:"Setting Up the Search Method and Search Variables"},"256":{i:0.00014532629207457,u:"../macromodel_quick_start/conf_search_subs_auto_run.htm",a:"Click the Settings button to open the Job Settings dialog box. Choose Append new entries as a new group from the Incorporate option menu. Enter SubsAuto in the Name text box. Click Run to launch the job. The sample files included in the distribution have conformational search variables defined only ...",t:"Starting the Substructure Conformational Search Job"},"257":{i:0.000102503145424313,u:"../macromodel_quick_start/conf_search_llmod.htm",a:"The large-scale low-mode (LLMOD) conformational searching routine is a unique method for generating candidate conformations of very large structures, including full proteins. Combinations of low-frequency vibrational modes are used to produce candidate structures. These modes represent simultaneous, ...",t:"Large-Scale Low-Mode Conformational Search"},"258":{i:0.000112244518966743,u:"../macromodel_quick_start/mm_other_tut.htm",a:"Other MacroModel Calculations MacroModel has a range of other capabilities than those already encountered. These exercises illustrate some of these capabilities. The exercises are listed below.",t:"Other MacroModel Calculations"},"259":{i:0.000138101239924247,u:"../macromodel_quick_start/other_embrace.htm",a:"Embrace is an automated routine that uses a collection of individual ligands, each pre-positioned with respect to a given receptor. Embrace automatically performs energetic calculations on each complex formed from the receptor and the individual ligands. Embrace can work in two modes—interaction ...",t:"Ligand Binding using Embrace"},"260":{i:0.000136792578772453,u:"../macromodel_quick_start/other_md.htm",a:"Molecular dynamics calculations simulate molecular movement over time using Newton’s equations of motion. In this exercise, you will run an MC/SD dynamics calculation.  In the MC/SD simulation, an initial minimization is performed to ensure that the structure is at a minimum on the potential energy ...",t:"Molecular Dynamics"},"261":{i:0.000128241218471709,u:"../macromodel_quick_start/other_scans.htm",a:"A contour diagram describing the molecular mechanics potential energy of a structure, relative to the value of either one or two coordinates (distances, angles, or dihedrals), can be generated with MacroModel. These exercises demonstrate how to produce a contour diagram describing the variation in ...",t:"Creating Energy Profiles from Coordinate Scans"},"262":{i:0.000133568117521779,u:"../macromodel_quick_start/other_scans_running.htm",a:"Import the structure in Ddrive.mae from your working directory. Need help?  Use the directions in  Importing the 1err Structure . Choose Tasks → Structure Analysis → Coordinate Scan → Force-Field in the Task Tool. The Coordinate Scan panel opens. Choose Workspace (included entry) from the Use ...",t:"Performing a Coordinate Scan Calculation"},"263":{i:0.000126721515692607,u:"../macromodel_quick_start/other_scans_analyzing.htm",a:"You can create a contour diagram of Ddrive-out.grd in the Plot of Two-Coordinate Scan panel: Choose Tasks → Structure Analysis → Coordinate Scan → Plot Two Coordinates in the Task Tool. Click Open and select Ddrive-out.grd from your working directory. Click Open. The output structure file is ...",t:"Analyzing the Results of the Coordinate Scan"},"264":{i:9.72314635193575E-05,u:"../macromodel_quick_start/other_partition_coefficient.htm",a:"MacroModel can estimate the logarithm of the partition coefficient of a solute between two solvents. The GB/SA parameterized solvents available are water, octanol, and chloroform. Multiple solutes can be used as input by selecting entries in the Project Table. The input structures are minimized in ...",t:"Partition Coefficient Between Two Solvents"},"265":{i:9.35474353023508E-05,u:"../phase_tutorial/phase_tutorial.htm",a:"These exercises are designed to help you quickly become familiar with performing pharmacophore modeling with Phase. Basic familiarity with Maestro is assumed. In these exercises you will learn to create several pharmacophore hypotheses (models) using different approaches suited to available data, ...",t:"Pharmacophore Modeling with Phase"},"266":{i:0.00011371966924633,u:"../phase_tutorial/phase_tut_copying.htm",a:"The files you need to run the tutorial are included in the software distribution. You can create a working directory and copy the files to it from the Tutorials panel.  If the Tutorials panel is not already open: Choose Help → Tutorials. The Tutorials panel opens.  Make choices from the option menus ...",t:"Copying the Phase Tutorial Files"},"267":{i:0.000144820712986058,u:"../phase_tutorial/phase_tut_creating.htm",a:"These exercises demonstrate the use of Phase to create pharmacophore hypotheses from aligned or unaligned collections of active and inactive ligands, from a single ligand, from an apo protein, from protein-ligand complexes and by merging hypotheses. If you have not copied the tutorial files, do so ...",t:"Creating Hypotheses"},"268":{i:0.000140864545848343,u:"../phase_tutorial/phase_tut_creating_single_ligand.htm",a:"In this exercise we will create a pharmacophore hypothesis from a ligand in the Maestro Workspace Choose File → Import Structures. The Import dialog box opens. From the Files of type option menu, ensure that Maestro is selected. If the options are not displayed, click Options. Ensure that Import all ...",t:"Creating a Hypothesis from a Single Ligand"},"269":{i:0.000135892505256615,u:"../phase_tutorial/phase_tut_creating_manual_complex.htm",a:"Creating a Hypothesis Manually from a Protein-Ligand Complex  In this exercise we will create a pharmacophore hypothesis from a ligand in the Maestro Workspace with perception of pharmacophore features coming from recognition of key protein-ligand interactions based on geometry. Choose File → Import ...",t:"Creating a Hypothesis Manually from a Protein Ligand Complex"},"270":{i:0.000122301006423517,u:"../phase_tutorial/phase_tut_creating_epharmacophore.htm",a:"Creating a Hypothesis with e-Pharmacophores from a Protein-Ligand Complex  In this exercise we will create a pharmacophore hypothesis from a ligand in the Maestro Workspace with perception of pharmacophore features coming from recognition of key protein-ligand interactions based on geometry. Choose ...",t:"Creating a Hypothesis with e Pharmacophores from a Protein Ligand Complex"},"271":{i:0.000158609872856568,u:"../phase_tutorial/phase_tut_creating_common_inplace.htm",a:"In this exercise we will create a pharmacophore hypothesis from a collection of pre-aligned factor Xa ligands. For this example a diverse set of factorXa protein-ligand complexes were aligned by binding site to create the ligand alignments. Choose File → Import Structures. The Import dialog box ...",t:"Creating a Common Hypothesis from Prealigned Ligands"},"272":{i:0.000103686809695841,u:"../phase_tutorial/phase_tut_creating_common_diverse.htm",a:"In this exercise we will create a pharmacophore hypothesis from a collection of known actives to estrogen receptor beta. For this example a diverse set of known estrogen receptor active ligands is used to recognize common pharmacophore hypotheses. The resulting hypotheses are recommended for ...",t:"Identifying Common Pharmacophore Hypotheses from Diverse Ligands for Virtual Screening"},"273":{i:0.000116997453475483,u:"../phase_tutorial/phase_tut_creating_common_congeneric.htm",a:"In this experiment common pharmacophore hypotheses will be identified from known active and inactive ligands in a congeneric series for use in a lead optimization project. Excluded volumes are derived from active and inactive alignments. The resulting hypotheses are recommended for use in lead ...",t:"Identifying Common Pharmacophore Hypotheses for Lead Optimization from Ligands in a Congeneric Series"},"274":{i:0.000144636626807702,u:"../phase_tutorial/phase_tut_modifying.htm",a:"In Phase the paradigm is to set options in a hypothesis and then screen compounds from the hypothesis rather than set all options when screening. In this section you will learn how to adjust attributes of a hypothesis including changing and viewing hypothesis attributes, merging hypotheses to create ...",t:"Modifying Hypothesis"},"275":{i:0.000183734832061553,u:"../phase_tutorial/phase_tut_modifying_attributes.htm",a:"In this exercise we will view different aspects of a hypothesis including excluded volumes, feature tolerances, and the reference ligand and modify features including adjusting their position and orientation, changing their type, changing what chemistry is able to match a given feature and what ...",t:"Changing and Viewing Hypothesis Attributes"},"276":{i:0.000156226637984324,u:"../phase_tutorial/phase_tut_modifying_merging.htm",a:"In this section we will merge features from multiple hypotheses to create a new hypothesis. One caveat is that the feature definitions used to create the hypotheses must be the same in order to merge their features into a new hypothesis. Choose File → Import Structures. The Import dialog box opens. ...",t:"Merging Hypotheses"},"277":{i:0.00015119322941424,u:"../phase_tutorial/phase_tut_modifying_xvols.htm",a:" In this exercise we will add excluded volumes to an existing hypothesis, then manipulate the excluded volume site locations and scaling, and selectively delete individual excluded volume sites. Choose File → Import Structures. The Import dialog box opens. From the Files of type option menu, ensure ...",t:"Working with Excluded Volumes"},"278":{i:0.000160937381413381,u:"../phase_tutorial/phase_tut_screening.htm",a:"Screening Ligands Against Hypotheses These exercises illustrate the use of a variety of options for screening ligands against hypotheses.",t:"Screening Ligands Against Hypotheses"},"279":{i:0.000160268661193455,u:"../phase_tutorial/phase_tut_screening_database.htm",a:"In this exercise we will screen a collection of compounds stored in multiple Phase databases against a pharmacophore hypotheses with constraints and analyze the results. Choose File → Import Structures. The Import dialog box opens. From the Files of type option menu, ensure that Maestro is selected. ...",t:"Screening Ligands From Phase Databases and Analyzing Hits"},"280":{i:0.000188255538943868,u:"../phase_tutorial/phase_tut_screening_files.htm",a:"In this exercise we will screen a collection of compounds without conformers against two pharmacophore hypotheses and analyze the results. We will employ feature definition presets to broaden chemistry that will match sites in the hypothesis without having to change feature definitions. Choose File ...",t:"Screening Ligand Files using Feature Definition Presets"},"281":{i:0.000188268605970752,u:"../phase_tutorial/phase_tut_analyzing.htm",a:"Analyzing Hypotheses The exercises listed below demonstrate two ways of analyzing hypothesis: checking enrichment on a ligand set with actives and decoys, and detecting multiple binding modes in a set of hypotheses developed by identifying common pharmacophores.",t:"Analyzing Hypotheses"},"282":{i:0.00021966337735048,u:"../phase_tutorial/phase_tut_analyzing_enrichment.htm",a:"In this exercise we will analyze enrichment performance for a set of hypotheses generated by common pharmacophore perception. This analysis can also be performed with a single hypothesis using the Validate hypothesis workflow. Choose File → Import Structures. The Import dialog box opens. From the ...",t:"Viewing Enrichment for a Set of Hypotheses"},"283":{i:0.000225593390111085,u:"../phase_tutorial/phase_tut_analyzing_binding_mode.htm",a:"In this exercise we will visualize the hypotheses and ligand alignments from an in-place common pharmacophore perception experiment to identify multiple binding modes. This uses the pharmacophore hypotheses created in  Creating a Common Hypothesis from Prealigned Ligands .   Choose File → Import ...",t:"Identifying Multiple Binding Modes from a Set of Hypotheses"},"284":{i:0.000158729126512771,u:"../phase_tutorial/phase_tut_databases.htm",a:"This exercise demonstrates how to create a Phase database for a collection of ligands through Maestro. While compounds can be screened against pharmacophore hypotheses from structure files or from entries in the Maestro Project Table, screening compounds from a prepared Phase database is typically ...",t:"Creating and Modifying Phase Databases"},"285":{i:0.000134473517755432,u:"../phase_tutorial/phase_tut_databases_creating.htm",a:"  Click the Tasks button, type “phase” and press Enter. Select  Create Database from the results list. The  Create Phase Database panel opens. We will be creating a new Phase database so ensure that Create new database is selected. You can also add compounds to an existing database with the Append ...",t:"Creating a Phase Database"},"286":{i:9.94280974618497E-05,u:"../phase_tutorial/phase_tut_databases_features.htm",a:"In this exercise the default feature definitions will be changed to no longer recognize trifluoromethyl groups as hydrophobic. Modifying default feature definitions is an advanced pharmacophore modeling capability and requires understanding of SMARTS patterns. In Phase, feature definitions are ...",t:"Modifying Default Phase Feature Definitions"},"287":{i:9.43794753666887E-05,u:"../prime_quick_start/prime_tutorial.htm",a:"This set of exercises provides a tutorial introduction to using Prime for protein structure prediction by homology modeling.  Homology Modeling of a Single Chain  provides an introduction to homology modeling for a single chain that covers most of the tasks that are performed in homology modeling, ...",t:"Homology Modeling With Prime"},"288":{i:0.000116656133852813,u:"../prime_quick_start/prime_tutorial_copying.htm",a:"The files you need to run the tutorial are included in the software distribution. You can create a working directory and copy the files to it from the Tutorials panel.  If the Tutorials panel is not already open: Choose Help → Tutorials. The Tutorials panel opens.  Make choices from the option menus ...",t:"Copying the Prime Tutorial Files"},"289":{i:0.00014808629799195,u:"../prime_quick_start/prime_tutorial_project.htm",a:"You can create a new named Maestro project to save your work, in case you want to complete the exercises at a later time. If you are using an existing Maestro session, it is advisable to create a new project to keep the tutorial separate from your other work. When you start Maestro, a scratch ...",t:"Creating a Maestro Project for the Prime Tutorial"},"290":{i:0.000126363367219419,u:"../prime_quick_start/prime_tutorial_preferences.htm",a:"Maestro has many options for displaying structures in the Workspace. Here you will set some preferences that are needed for the first exercise. You will also need to display the Saved Views toolbar. Choose Edit → Preferences. The Preferences panel opens. On the left, under Molecular representation, ...",t:"Setting Maestro Preferences for the Prime Tutorial"},"291":{i:0.00010859870452339,u:"../prime_quick_start/single_chain_tut.htm",a:"This chapter provides an exercise for building and refining a model of a query sequence for which a sequence homolog can be identified using BLAST. While the tutorial is self-contained, you may find it useful to refer to the  Prime User Manual  or the online help for more detailed information on the ...",t:"Homology Modeling of a Single Chain"},"292":{i:0.000119649713094779,u:"../prime_quick_start/single_chain_importing_query.htm",a:"The query sequence that will be used is closely related to that of phosphoglycerate kinase from Pyrococcus furiosus, but has been modified slightly to provide a case that best demonstrates various features of Prime’s Homology Modeling workflow: \n\u003eQuery ...",t:"Importing the Query Sequence"},"293":{i:0.000165855739338241,u:"../prime_quick_start/single_chain_finding_homologs.htm",a:"In this step, you will search for homologous proteins with known structure using BLAST, then select one homolog as a template. Click BLAST Homology Search. The search job is started. This search usually takes less than 1 minute on a 1-GHz processor. When the job finishes, a list of potential ...",t:"Finding Sequence Homologs"},"294":{i:0.000124871740814701,u:"../prime_quick_start/single_chain_alignment.htm",a:"Before making changes to the initial BLAST alignment, it is a good idea to save the results so that you can return to them later if you need to start over with the alignment. To do this, you will save the current run and give the run you are going to work in a new name. From the File menu, choose ...",t:"Saving the Alignment"},"295":{i:0.000133171443932708,u:"../prime_quick_start/single_chain_improving_alignment.htm",a:"Because the alignment provided by the Find Homologs step is based only on sequence information, there is room for improvement. For example, the default alignment has placed a gap at query residue His59, which corresponds to the middle of a helix in the template. Therefore, it is unlikely that the ...",t:"Improving the Alignment"},"296":{i:0.000128690663051906,u:"../prime_quick_start/single_chain_manual_alignment.htm",a:"While the alignment program took secondary structure into account in producing the alignment between query and template, it did not explicitly consider tertiary structure. There are still some issues in the alignment that need to be fixed. You will next perform some manual editing of the alignment ...",t:"Manual Editing of the Alignment"},"297":{i:0.00013786994914835,u:"../prime_quick_start/single_chain_building.htm",a:"The Build Structure program builds insertions, closes gaps, and predicts side-chain conformations of non-conserved residues to produce a model with no unphysical clashes. However, it does this efficiently, without extensive conformational sampling. The structure produced in the Build Structure step ...",t:"Building a Model Structure"},"298":{i:0.0001414538504386,u:"../prime_quick_start/single_chain_refining.htm",a:"To improve the structure most efficiently, you should focus refinement efforts on areas of the structure that are likely to be problematic. In general terms, this means refining loops (particularly where insertions have been made or gaps closed) and re-predicting side-chain conformations. A ...",t:"Refining Target Regions of the Structure"},"299":{i:0.000144923064826444,u:"../prime_quick_start/single_chain_refining_loops.htm",a:"In the first refinement exercise, you will refine one of the non-templated loops using the default sampling method.   If the Refine Loops panel is not open, open it by choosing Tasks → Protein Preparation and Refinement → Refine Loops in the Task Tool. The Refine Loops panel opens.  For Find loops ...",t:"Refining Loops"},"300":{i:0.000102388880035177,u:"../prime_quick_start/single_chain_refining_minimizing.htm",a:"Since only side chains (not the backbone) of residues within 7.5 Å were sampled during the previous loop refinement, it is not unreasonable to minimize the local environment of the loop before considering refinement complete. Open the Minimize panel by choosing Tasks → Protein Preparation and ...",t:"Minimizing Target Regions"},"301":{i:0.000117909883320606,u:"../prime_quick_start/homomultimer_tut.htm",a:"This exercise builds a homomultimer for hERG, which is a tetramer. The template that will be used is 1orq. This is not the highest scoring template that can be found from a BLAST search, but it is chosen to illustrate the use of a template in a realistic situation, where the similarity may not be ...",t:"Homology Modeling of a Homomultimer"},"302":{i:0.00015251083991029,u:"../prime_quick_start/homomultimer_preparing_template.htm",a:"Preparing the 1orq Template The template can be prepared by using the Protein Preparation Wizard. In addition to generating the tetramer, the parts of the structure that are not needed are deleted, and the chains are renamed so they can be distinguished when building the homology model. ",t:"Preparing the 1orq Template"},"303":{i:0.000144951082757413,u:"../prime_quick_start/homomultimer_preparing_template_importing.htm",a:"Click the Protein Preparation Wizard button on the Favorites toolbar to open the Protein Preparation Wizard panel, or choose Tasks → Protein Preparation and Refinement → Protein Preparation Wizard in the Task Tool. Enter 1orq in the PDB text box. Select Biological unit. Selecting this option applies ...",t:"Importing the 1orq Template Structure"},"304":{i:0.000142624335983805,u:"../prime_quick_start/homomultimer_preparing_template_deleting.htm",a:"In this stage of the preparation, unwanted parts of the structure are deleted, such as cadmium atoms added for the X-ray structure determination, waters, duplicate potassium ions, and unneeded chains. In the Review and Modify tab of the Protein Preparation Wizard panel, click Analyze Workspace. The ...",t:"Deleting Unwanted Parts of the 1orq Structure"},"305":{i:0.000141972356588175,u:"../prime_quick_start/homomultimer_preparing_template_relabeling.htm",a:"The next step is to assign the potassium ions to a separate chain.  In the Structure Hierarachy in the main window, open the tree for the 1orq structure to Other  → Metals / Ions. Select the six potassium ions, which are labeled K1 through K6. These ions are selected in the Workspace. In the 3D ...",t:"Assigning a Chain Name in the 1orq Template"},"306":{i:0.000141998126438912,u:"../prime_quick_start/homomultimer_preparing_template_trimming.htm",a:"Some further trimming down of the structure will be done before using the structure as a template.  In the main window, click the Define button on the Select toolbar. The Atom Selection dialog box opens. In the Residue tab, choose Residue number from the list. Enter 1-104 in the Residue number text ...",t:"Trimming the 1orq Template Structure"},"307":{i:0.000142741131143706,u:"../prime_quick_start/homomultimer_importing_sequence.htm",a:"Now that the template is prepared, you can start the homology modeling.  If the Structure Prediction panel is not already open, open it now by choosing Tasks → Biologics → Advanced Homology Modeling in the Task Tool. The Structure Prediction panel opens at the first step, Input Sequence. By default ...",t:"Importing the hERG Sequence"},"308":{i:0.000145337795278771,u:"../prime_quick_start/homomultimer_importing_template.htm",a:"Select the 1ORQ entry in the Entry List.  If this is a new project, it should be the only entry in the list. It should be the one that is marked in the In column, as it is displayed in the Workspace. Alternatively you can select the entry in the Project Table panel, which you can open by pressing ...",t:"Importing and Selecting the 1ORQ Template"},"309":{i:0.000102506405554292,u:"../prime_quick_start/homomultimer_building_model.htm",a:"The initial alignment of the query is satisfactory for model building, so you do not need to perform any alignment.  Click Next to proceed to the Build Structure step. Click Continue in the warning dialog box that is displayed.  The warnings are largely related to the missing loops in the template, ...",t:"Building the Homomultimer Model"},"310":{i:0.000118269743900171,u:"../prime_quick_start/heteromultimer_tut.htm",a:"This chapter provides an exercise for building a heteromultimer, consisting of CDK4 and Cyclin-E. To do this, you model the monomers up to the beginning of the Build Structure step in separate runs, then combine the runs to build the heteromultimer. If the Structure Prediction panel is not already ...",t:"Homology Modeling of a Heteromultimer"},"311":{i:0.000153780965875414,u:"../prime_quick_start/heteromultimer_preparing_cdk4.htm",a:"Choose File → New. Name the run CDK4. For Get sequence from, click File. Select cdk4.fasta and click Open.  The sequence is displayed in the Prime sequence viewer.  Click Next to proceed to the next step, Find Homologs. For Import, click From File.  Choose 3f5x.pdb and click Open.  Four sequences ...",t:"Preparing the CDK4 Chain"},"312":{i:0.000148894166100719,u:"../prime_quick_start/heteromultimer_preparing_cyclin.htm",a:"Choose File → New. Name the run CyclinE. For Get sequence from, click File. Select cyclinE.fasta and click Open.  The sequence is displayed in the Prime sequence viewer.  Click Next to proceed to the next step, Find Homologs. For Import, click From File.  Choose 3f5x.pdb and click Open.  Four ...",t:"Preparing the Cyclin-E Chain"},"313":{i:0.000103514095074655,u:"../prime_quick_start/heteromultimer_building_model.htm",a:"Now that both chains of the heteromultimer are prepared, you can build the model. Under Model-building method, choose Energy-based. Click Hetero-multimers. The Hetero-multimer Modeling dialog box opens. The instructions at the top describe the process you have just completed.  Select the CDK4 and ...",t:"Building the Model"},"314":{i:9.28846132231373E-05,u:"../primex_quick_start/primex_tutorial.htm",a:"This tutorial shows how to use the tools available with PrimeX and Maestro for performing an X-ray structure refinement, starting from a model, a sequence, and a set of reflection data. In the early parts of the tutorial, the steps for some actions are given in detail so that you can learn how to ...",t:"Solving an X-ray Structure with PrimeX"},"315":{i:0.000111380575726312,u:"../primex_quick_start/primex_tutorial_copying_files.htm",a:"The files you need to run the tutorial are included in the software distribution. You can create a working directory and copy the files to it from the Tutorials panel.  If the Tutorials panel is not already open: Choose Help → Tutorials. The Tutorials panel opens.  Make choices from the option menus ...",t:"Copying the PrimeX Tutorial Files"},"316":{i:0.000120926880444539,u:"../primex_quick_start/tutorial_skipping.htm",a:"Several of the steps are time-consuming, and you might want to skip over the actual calculations. Structures for various stages of the process have been provided so that you can so do. The structures can be imported as follows. Click the Import button on the Project toolbar. The Import panel opens. ...",t:"Skipping PrimeX Tutorial Steps"},"317":{i:0.000131957679942031,u:"../primex_quick_start/tutorial_creating_project.htm",a:"When you start Maestro, a scratch project is created. This project must be named in order to keep it for later use. You should therefore create a named Maestro project to save your work, in case you want to complete the exercises at a later time. For more information on Maestro projects, see  ...",t:"Creating a Maestro Project for the PrimeX Tutorial"},"318":{i:0.000128352203693493,u:"../primex_quick_start/tutorial_reading_input.htm",a:"The first step is to read the input data, which includes the starting molecular replacement structure, the sequence, and the reflection data.  Choose Applications → PrimeX. The PrimeX panel opens. This panel provides access to all the PrimeX features. Click Input Data.  The Input Data dialog box ...",t:"Reading the Input Data for p38 Kinase"},"319":{i:0.00013758581117319,u:"../primex_quick_start/tutorial_initial_R_factors.htm",a:"In the PrimeX panel, click R-Factor Calculation.  The R-Factor Calculation panel opens. Since the calculation settings have already been made, there is no need to change them. There is also no need to make job settings as the default settings replace the structure in the Project Table, and run the ...",t:"Generating Initial R-Factors for p38 Kinase"},"320":{i:0.000140789354066951,u:"../primex_quick_start/tutorial_rigid_body.htm",a:"Rigid-Body Refinement of p38 Kinase The rigid-body refinement in this tutorial is performed in two steps. First, the structure as a whole is moved, then the two domains of the protein are reoriented independently.",t:"Rigid-Body Refinement of p38 Kinase"},"321":{i:0.000142861489554746,u:"../primex_quick_start/tutorial_rigid_body_overall.htm",a:"In the PrimeX panel, click Rigid Bodies in the Reciprocal-Space Refinement section. The Reciprocal-Space Refinement panel opens with the Rigid bodies tab displayed.  Choose Truncated Newton from the Minimizer option menu. In the Select atoms to define group section, click the All button. This ...",t:"Refining the Overall Position of p38 Kinase"},"322":{i:0.00014697057583543,u:"../primex_quick_start/tutorial_rigid_body_domains.htm",a:"First, you need to clear the previous group selections. Click Delete All in the Rigid Bodies tab of the Reciprocal-Space Refinement panel. The group is deleted from the Rigid body groups table. Click the delete button in the Select atoms to define group text box. The ASL text box is cleared. Enter 8 ...",t:"Refining the Two Domains of p38 Kinase"},"323":{i:0.000159399677650129,u:"../primex_quick_start/tutorial_simulated_annealing.htm",a:"In the PrimeX panel, click Simulated Annealing in the Reciprocal-Space Refinement section. The Simulated Annealing tab is displayed in the Reciprocal-Space Refinement panel. Ensure that the text in the text box of the Select atoms to minimize section is all. If not, click the All button. For Energy ...",t:"Simulated Annealing Refinement for p38 Kinase"},"324":{i:0.000199154754096975,u:"../primex_quick_start/tutorial_mutating.htm",a:"In the next step, the model structure will be mutated to match the reference sequence, and side chains will be placed.  Click Mutate Model to Sequence in the PrimeX panel. The Mutate Model to Sequence panel opens. You will see the one substitution highlighted in the model (target) sequence. Click ...",t:"Mutating Amino Acids and Placing Side Chains in p38 Kinase"},"325":{i:0.000327029670743013,u:"../primex_quick_start/tutorial_reciprocal_space_minimization.htm",a:"In the PrimeX panel, click Minimization in the Reciprocal-Space Refinement section. The Reciprocal-Space Refinement panel opens with the Minimization tab displayed. Ensure that Optimal is chosen from the Minimizer option menu. The quasi-Newton minimizer is usually the most effective for this type of ...",t:"Reciprocal-Space Minimization for p38 Kinase"},"326":{i:0.000220069934672967,u:"../primex_quick_start/tutorial_examining_density_fit.htm",a:"Click Density Fit in the PrimeX panel. The Density Fit panel opens. After a short delay, the table is filled with real-space R-factors for each residue and for the side chain and backbone of each residue. Lower values indicate a better fit. Click the Backbone Fit column heading twice. The values are ...",t:"Examining the Density Fit for p38 Kinase"},"327":{i:0.000166378042696263,u:"../primex_quick_start/tutorial_loop_refinement.htm",a:"Loop Refinement for p38 Kinase In this step, two of the loops are refined, including the one identified in the previous section, and one of the loops is deleted and is rebuilt later. ",t:"Loop Refinement for p38 Kinase"},"328":{i:0.000150683528041817,u:"../primex_quick_start/tutorial_loop_refinement_refining_first.htm",a:"Click Refine loops in the Real-Space Refinement section of the PrimeX panel. The Real-Space Refinement panel opens. A table of loops is displayed in the center of the panel. These are regions lacking secondary structure, which are most likely to need to be rebuilt. The loops are identified by a ...",t:"Refining the Loop from Residues 118 to 123"},"329":{i:0.000148986998697222,u:"../primex_quick_start/tutorial_loop_refinement_comparing.htm",a:"To compare the new loop to the original structure, the new entry will be fixed in the Workspace, and the original structure also displayed. The atoms that are not in the loops will be hidden. The atoms in the entries will also be colored so that they can be distinguished. In the Project Table, ...",t:"Comparing the New and Original Loops from Residues 118 to 123"},"330":{i:0.000122478330817533,u:"../primex_quick_start/tutorial_loop_refinement_minimizing_first.htm",a:"After performing the loop refinement, a reciprocal-space minimization should be performed. Include the predicted loop in the Workspace (click its In column in the Project Table).  Follow the instructions in  Reciprocal-Space Minimization for p38 Kinase  to perform a minimization.",t:"Minimizing the Loop from Residues 118 to 123"},"331":{i:0.000111394263028972,u:"../primex_quick_start/tutorial_loop_refinement_refining_second.htm",a:"The next task is to refine the loop that was examined in  Examining the Density Fit for p38 Kinase , using the results of the minimization just performed.  Click Refine loops in the Real-Space Refinement section of the PrimeX panel. The Real-Space Refinement panel opens.  Ensure that the loop to be ...",t:"Refining the Loop from Residues 167 to 175"},"332":{i:0.000113129121256344,u:"../primex_quick_start/tutorial_loop_refinement_minimizing_second.htm",a:"After performing the loop refinement, a reciprocal-space minimization should be performed. Include one of the loop predictions in the Workspace. The first loop has the lowest Ref Score, and thus by this measure has the best combined fitting to the electron density with a low-energy conformation. ...",t:"Minimizing the Loop from Residues 167 to 175"},"333":{i:0.000115991891390671,u:"../primex_quick_start/tutorial_loop_refinement_deleting.htm",a:"Finally, the loop spanned by residues 30–38 should be deleted. The electron density is not yet accurate enough to place these residues with Loop Refinement. Before you delete the residues, the entry will be duplicated so that the results of the minimization are preserved. Ensure that the new loop ...",t:"Deleting the Loop from Residues 30 to 38"},"334":{i:0.000144112989413817,u:"../primex_quick_start/tutorial_optimizing_hbonds.htm",a:"At the moderate resolution of most protein crystal structures, it is difficult or impossible to determine from X-ray data alone which orientation the terminal groups in residues like His, Gln, and Asn should take. However, the most likely orientation can be deduced from optimizing the number and ...",t:"Optimizing Hydrogen-Bonding Networks for p38 Kinase"},"335":{i:0.000176185694287993,u:"../primex_quick_start/tutorial_bfactor_refinement.htm",a:"Click Minimization in the Reciprocal-space refinement section of the PrimeX panel.  The Minimization tab is displayed in the Reciprocal-Space Refinement panel.  Set the number of cycles to 8, and ensure that the maximum steps per cycle is 8. Under Minimize, select Individual B-factors. In the ...",t:"B-Factor Refinement for p38 Kinase"},"336":{i:0.000152240728576752,u:"../primex_quick_start/tutorial_ligand_placement.htm",a:"Ligand Placement for p38 Kinase At this point, the ligand can be placed. ",t:"Ligand Placement for p38 Kinase"},"337":{i:0.000144676113378065,u:"../primex_quick_start/tutorial_ligand_placement_importing.htm",a:"First, the ligand is imported and its structure corrected. Click the Import button on the Project toolbar. The Import panel opens. From the Files of type option menu, choose Common or PDB.  Select 358.pdb. If the import options are not visible, click Options. Ensure that Replace Workspace is ...",t:"Importing and Correcting the p38 Kinase Ligand"},"338":{i:0.000141923749086623,u:"../primex_quick_start/tutorial_ligand_placement_difference_map.htm",a:"Next, a difference map is created for the protein. Display the output structure from the B-factor refinement in the Workspace (click its In column in the Project Table). Click Create Map in the PrimeX panel. The Create Map panel opens.  For Coefficients, select Fo-Fc and deselect 2Fo-Fc.  You can ...",t:"Creating a Difference Map for p38 Kinase"},"339":{i:0.000139774804164066,u:"../primex_quick_start/tutorial_ligand_placement_centering_display.htm",a:"Next, the display will be centered on residue 167. Type Ctrl+F (⌘F)\n in the main window. The Find toolbar is displayed. From the Find option menu, select Residue number. Enter A:167 in the Residue number text box. Click N. The display is centered on residue 167 and zoomed in.  In the Manage Surfaces ...",t:"Centering the Display on Residue 167"},"340":{i:0.000134943269729952,u:"../primex_quick_start/tutorial_ligand_placement_placing_ligand.htm",a:"Click the Ligand/Solvent Placement button in the PrimeX panel. The Ligand/Solvent Placement panel opens. A progress box may be displayed for the detection of density “blobs”, which are used to place the ligand. Under Use ligands from, select Selected entries. In the Project Table, select the entry ...",t:"Placing the p38 Kinase Ligand"},"341":{i:0.000120040842085739,u:"../primex_quick_start/tutorial_ligand_placement_examining_poses.htm",a:"The next steps demonstrate the examination of the ligand poses that were returned, and how well they fit the density. Normally, you would choose the best fit to the electron density, and you can base your decision on the scores in the Project Table. Thus, for this tutorial, selection of the first ...",t:"Examining the p38 Kinase Ligand Poses"},"342":{i:0.0001204635705993,u:"../primex_quick_start/tutorial_ligand_placement_refining.htm",a:"Include the first ligand pose in the Workspace. If not all the atoms are displayed, choose All from the Display Only menu button on the Display Atoms toolbar. Run a reciprocal-space coordinate minimization with an appropriate entry title, following the instructions in  Reciprocal-Space Minimization ...",t:"Refining the p38 Kinase Ligand"},"343":{i:0.000118230435633536,u:"../primex_quick_start/tutorial_rebuilding_loop.htm",a:"At this point, the loop that was deleted in an earlier step can be built back in. The loop in question is from residues 28 to 40. The region around residues 28–40 will have clearer electron density now, but correct building of this loop is still difficult.  Click Refine Loops in the Real-Space ...",t:"Rebuilding the Loop from Residues 28 to 40"},"344":{i:0.000121277826634473,u:"../primex_quick_start/tutorial_adding_waters.htm",a:"Now that the protein and ligand have been fairly well refined, the next step is to add waters. Click Water Placement in the PrimeX panel. The Water Placement panel opens. The defaults can be used for this job. Start the job with an appropriate entry title. Run a reciprocal-space coordinate ...",t:"Adding Waters to p38 Kinase"},"345":{i:8.19435416355684E-05,u:"../primex_quick_start/tutorial_assessing_results.htm",a:"The published structure that you have solved is 1OVE. To compare the structures you can import 1OVE.pdb from the tutorial directory. Although you will find some differences, the bulk of the model refinement is finished at this point, and this is the end of the tutorial. In the course of the ...",t:"Assessing the Results for p38 Kinase"},"346":{i:8.93938035848629E-05,u:"../pldb_tutorial/pldb_tut_intro.htm",a:"Searching the Protein-Ligand Database (PLDB) The Protein-Ligand Database (PLDB) is designed to provide insight from protein-ligand complexes, validate unusual interactions from crystal structures, docking results, and design ideation. Access to the PLDB is done through a connection made to a server ...",t:"Protein Ligand Database (PLDB) Tutorial"},"347":{i:9.90606531025079E-05,u:"../pldb_tutorial/pldb_tut_login.htm",a:"These exercises use the PLDB Search panel from Maestro. To do a search you must log in to the PLDB. If you haven\u0027t already done so, start Maestro. Open the  PLDB Search panel from the Task Tool.  You can do this by searching for PLDB or using the Browse tool to go to the Structure Analysis section. ...",t:"Logging In to the PLDB"},"348":{i:0.000137839096796915,u:"../pldb_tutorial/pldb_tut_search_cdk2.htm",a:"In this exercise, we\u0027ll retrieve results for the CDK2 Gene. Click the New Search button in the history panel to open the PLDB search form.  On the left side of the form, you will find search fields for filtering query results. The sketcher and associated toolbars on the right side of the form are ...",t:"Searching for Entries for the CDK2 Gene"},"349":{i:0.0001411778063996,u:"../pldb_tutorial/pldb_tut_serine_proteases.htm",a:"In this exercise, we want to search for structures whose sequences have at least 80% sequence similarity with the serine protease, 1S0Q. Open the search form, and run a search for PDB Id 1S0Q. The results of this query should be available in under five seconds. Click the View Results button from the ...",t:"Finding Serine Proteases by Sequence Similarity"},"350":{i:0.000148641861638745,u:"../pldb_tutorial/pldb_tut_macrocycles.htm",a:" In this exercise we will search for macrocycles of 25 atoms or larger. In the history panel, click the New Search button to begin a new query. We will be using the sketcher on the right side of the search form for this exercise. Select Ligand from the Draw drop-down menu at the bottom of the ...",t:"Finding Macrocycles"},"351":{i:0.000137185838347107,u:"../pldb_tutorial/pldb_tut_ligand_sim.htm",a:"In this exercise, we will look at ligands similar to the 1F0R ligand that binds to Factor Xa. 1F0R\u0027s ligand is a cyclic amidine inhibitor for Factor Xa. We know that this class had some of the most potent binders known to Factor Xa, but that some of the properties of amidines make drug development ...",t:"Using a Ligand Similarity Search to Find Factor Xa Binders"},"352":{i:0.000148817519216819,u:"../pldb_tutorial/pldb_tut_analyze_interactions.htm",a:"In this exercise, we will go through the creation of a sample geometric search where we\u0027ll analyze how lysines interact with ligand pyrroles. Details of using the geometric search functionality can be found by clicking the Help button in the bottom-left corner of the search form. Creating and ...",t:"Analyzing Lysine Interactions with Ligand Pyrroles"},"353":{i:0.000141797902997611,u:"../pldb_tutorial/pldb_tut_hinge_indoles.htm",a:"For commonly-defined interactions and annotations such as hydrogen bonds, pi-pi interactions, the regions of kinase binding sites, and solvent accessibility, the PLDB offers annotation search. Let\u0027s use these search tools to find ligand indoles forming a hydrogen bond to the hinge region of a ...",t:"Finding Hinge-Binding Ligand Indoles"},"354":{i:0.000109946410460267,u:"../pldb_tutorial/pldb_tut_arginine_interactions.htm",a:"Create a search to retrieve all complexes where the ligand meets all of the following criteria: 1.) it has two hydrogen bonds with an arginine amino acid side chain, 2.) it has a π system roughly in the same plane defined by the arginine hydrogen bonds, and 3.) Its π system has a π-π stacking ...",t:"Searching for Arginine-Ligand Interactions"},"355":{i:0.000145446208556413,u:"../qikprop_tutorial/qikprop_tutorial_copying_files.htm",a:"The files you need to run the tutorial are included in the software distribution. You can create a working directory and copy the files to it from the Tutorials panel.  If the Tutorials panel is not already open: Choose Help → Tutorials. The Tutorials panel opens.  Make choices from the option menus ...",t:"Copying the QikProp Tutorial Files"},"356":{i:0.00014307771703383,u:"../qikprop_tutorial/qikprop_tutorial_importing.htm",a:"When running QikProp from Maestro, you can use as input for a job either the structures in the Workspace or entries that are selected in the Project Table panel. In this exercise, you will import structures into Maestro to use as QikProp input. Choose File → Import Structures.  The Import panel ...",t:"Importing Structures"},"357":{i:0.00014307771703383,u:"../qikprop_tutorial/qikprop_tutorial_normal_mode.htm",a:"In this exercise, you will run QikProp in normal processing mode on the imported structures.  Choose Browse → Tasks → ADME and Molecular Properties → Ligand-Based ADME/Tox Prediction in the Task Tool. The QikProp panel opens. Choose Project Table (selected entries) from the Use structures from ...",t:"Running QikProp in Normal Processing Mode"},"358":{i:0.000145446208556413,u:"../qikprop_tutorial/qikprop_tutorial_fast_mode.htm",a:"In this exercise, you will run QikProp in fast mode. You will use the same set of structures you processed in normal mode in the previous exercise so that you can compare the output files produced by each mode.  Ensure that all entries in the Project Table panel are selected. This should be the case ...",t:"Running QikProp in Fast Processing Mode"},"359":{i:0.000102537119998624,u:"../qikprop_tutorial/qikprop_tutorial_comparing_results.htm",a:"In this exercise, you will sort the entries in the Project Table to compare the results from fast mode to the results from normal mode.  In the Project Table panel, choose Fit To Header+Data from the Columns button menu on the toolbar.  Scroll through the Project Table and compare the fast mode ...",t:"Comparing Normal and Fast Mode Results"},"360":{i:0.00013204909079999,u:"../qsite_tutorial/qsite_tutorial_copying_files.htm",a:"The files you need to run the tutorial are included in the software distribution. You can create a working directory and copy the files to it from the Tutorials panel.  If the Tutorials panel is not already open: Choose Help → Tutorials. The Tutorials panel opens.  Make choices from the option menus ...",t:"Copying the QSite Tutorial Files"},"361":{i:0.00013532071818918,u:"../qsite_tutorial/qsite_tutorial_importing.htm",a:"Use the following steps to import the protein-ligand complex in the file qsite-1tpb.mae.  Choose File → Import Structures.  The Import panel opens. Choose Maestro or Common from the Files of type option menu. Navigate to your working directory (if necessary) and select the file 1tpb.mae. Click Open. ...",t:"Importing the 1tpb Complex"},"362":{i:0.000129098930980024,u:"../qsite_tutorial/qsite_tutorial_display.htm",a:"Given the number of atoms in the typical receptor-ligand complex, it can be hard to identify the ligand. In this exercise you will locate the ligand and set up the display to view only the ligand and the protein residues close to it.  Click the  Select ligands (only) button under Quick Select  on ...",t:"Setting Up the Display"},"363":{i:0.00013769891328828,u:"../qsite_tutorial/qsite_tutorial_job_type.htm",a:"Follow the instructions below to open the QSite panel and set the job type. In the main window, choose Tasks → General Modeling → Minimization → QM-MM. The QSite panel opens. In the Potential tab, ensure that OPLS_2005 is selected in the Force field option menu, and deselect Use non-bonded cutoffs ...",t:"Selecting the QSite Job Type"},"364":{i:0.000140441890533315,u:"../qsite_tutorial/qsite_tutorial_qm_region.htm",a:"You can select a ligand molecule, a lone ion, or a metal for the QM region by simple picking. To select entire residues from protein chains, you must make backbone cuts or use hydrogen caps. It is often useful to make side chain cuts, adding only the side chain rather than the entire residue to the ...",t:"Defining a QM Region"},"365":{i:0.000141522402894557,u:"../qsite_tutorial/qsite_tutorial_running.htm",a:"Change the job name to qsite_tutorial. Click Run. The job starts. You can view its progress in the Monitor panel.  Close the QSite panel. This job runs for several hours. If you wish to examine the results without waiting for the job to finish, you may simply import the output structure file, ...",t:"Running the QSite Job"},"366":{i:0.000142592774914321,u:"../qsite_tutorial/qsite_tutorial_importing_results.htm",a:"In the next two exercises, you will examine the results of the calculation. If you decided not to run the job, you can import the results from your working directory, as follows: Choose File → Import Structures.  The Import panel opens. Choose Maestro from the Files of type option menu. Select the ...",t:"Importing Results"},"367":{i:0.000145289993188358,u:"../qsite_tutorial/qsite_tutorial_comparing_structures.htm",a:"Click the Project Table toolbar button. The Project Table panel opens. The output structure has been appended to the project as an entry, with properties QM/MM Energy, QM Basis, QM Method, and Job Name (and others). By default, the output structure is included in the Workspace.  Control-click the ...",t:"Comparing Input and Output Structures"},"368":{i:0.000102492857915629,u:"../qsite_tutorial/qsite_tutorial_comparing_interactions.htm",a:"In this exercise you will compare the hydrogen bonding interactions between ligand and receptor in the input complex and the output complex. Some of the hydrogen bonds are around the limit at which Maestro displays them, so the settings will be changed.  Choose Edit → Preferences. The Preferences ...",t:"Comparing Ligand-Receptor Interactions"},"369":{i:0.000193272500898707,u:"../sitemap_tutorial/sitemap_tutorial.htm",a:"This tutorial exercise sets up a job to search a protein for possible active sites. The protein is 1ke8, which has a known site and ligand. This information will be used to assess how well the site is located. The results can be used to prepare a grid for Glide docking—see  Docking to a Site Found ...",t:"SiteMap Tutorial Introduction"},"370":{i:0.000158729242825532,u:"../sitemap_tutorial/tutorial_copying_files.htm",a:"The files you need to run the tutorial are included in the software distribution. You can create a working directory and copy the files to it from the Tutorials panel.  If the Tutorials panel is not already open: Choose Help → Tutorials. The Tutorials panel opens.  Make choices from the option menus ...",t:"Copying the SiteMap Tutorial Files"},"371":{i:0.000150488481442934,u:"../sitemap_tutorial/tutorial_finding_sites.htm",a:"In this exercise, you will set up and run the SiteMap calculation to locate sites on 1ke8.  Choose File → Import Structures.  The Import panel opens. From the Files of type option menu, ensure that Common is chosen.  Select the file 1ke8_protein.maegz and click Open. The protein is displayed in the ...",t:"Finding Sites with SiteMap"},"372":{i:0.000103965915788829,u:"../sitemap_tutorial/tutorial_examining_results.htm",a:"The main results of the calculation are displayed in the Workspace when the job finishes. In this exercise, you will see how well the results fit the native ligand, and assess the other sites on the protein.  Choose File → Import Structures.  The Import panel opens. From the Files of type option ...",t:"Examining the SiteMap Results"},"373":{i:6.83674761879237E-05,u:"../bioluminate_user_manual/bioluminate_overview.htm",a:"BioLuminate offers a wide range of tools for protein modeling, protein engineering, protein analysis, peptide analysis, and antibody modeling. In addition to the unique tools, BioLuminate provides access to many of the related tools in the Schrödinger software suite. The BioLuminate interface is a ...",t:"Overview of BioLuminate"},"374":{i:9.11285778661984E-05,u:"../bioluminate_user_manual/overview_protein_analysis.htm",a:"The protein analysis tools provide information on a protein and its properties. No change is made to the protein structure.  Protein Structure Quality Viewer (Tools → Protein Structure Quality): Show reports on deviations of protein parameters from standard values, in graphical and tabular form. See ...",t:"Protein Analysis Tools"},"375":{i:8.59044903323712E-05,u:"../bioluminate_user_manual/bioluminate_overview_protein_structure.htm",a:"The protein structure tools allow you to fix structures from the PDB or other sources that are missing information needed for modeling or are missing atoms, predict the structure of proteins by homology modeling, and predict the structure and stability of alpha helices in small peptides.  Protein ...",t:"Protein Structure Tools"},"376":{i:8.2524065604839E-05,u:"../bioluminate_user_manual/bioluminate_overview_peptide.htm",a:"The peptide tools allow you to predict various properties of small peptides from their sequences—see  Predicting Peptide Properties . Peptide Helicity (Tasks → Peptide Alpha Helicity): Predict the stability of alpha helices for small peptide sequences, using molecular dynamics.  Peptide Docking ...",t:"Peptide Tools"},"377":{i:8.86657355598894E-05,u:"../bioluminate_user_manual/bioluminate_overview_protein_alignment.htm",a:"The alignment tools include tools that structurally superimpose two proteins (or structures), and a tool for docking one protein to another. These tools perform rigid-body translation of the structures to obtain the best alignment.  Align Binding Sites (Tools → Binding Site Alignment): Align the ...",t:"Protein Alignment and Docking Tools"},"378":{i:8.07297834274414E-05,u:"../bioluminate_user_manual/bioluminate_overview_protein_mutation.htm",a:"Residue Scanning (Tasks → Residue Scanning): Systematically perform single mutations of protein residues to determine how energetic and other properties change, and to identify mutations that can effect desired changes. See  Scanning for Residue Mutations . Affinity Maturation (Tasks → Affinity ...",t:"Protein Mutation and Modification Tools"},"379":{i:7.11287214154418E-05,u:"../bioluminate_user_manual/bioluminate_overview_antibody.htm",a:"BioLuminate provides a specialized set of tools for modeling antibodies, including managing databases of antibody structures, homology modeling of antibodies, antibody humanization, and antigen-antibody docking.  Antibody Prediction (Tasks → Antibody Modeling → Prediction): Predict the structure of ...",t:"Antibody Tools"},"380":{i:7.96248417753833E-05,u:"../bioluminate_user_manual/bioluminate_interface.htm",a:"The BioLuminate interface is a customized form of the Maestro interface that is specially designed for biologics use. It inherits most of the capabilities of the Maestro interface , and it has features of its own.  This chapter focuses on the features that are unique to BioLuminate. Summaries of the ...",t:"The BioLuminate Interface"},"381":{i:0.000107639176874454,u:"../bioluminate_user_manual/bioluminate_interface_main_window.htm",a:"The Maestro BioLuminate Main Window The Maestro BioLuminate main window opens with the following features displayed by default: Menu bar. This is at the top of the window on Linux and Windows, and is the menu bar on the Mac.  Main toolbar. This toolbar has tools for selecting and styling atoms, ...",t:"The BioLuminate Main Window"},"382":{i:0.000102705564841708,u:"../bioluminate_user_manual/bioluminate_interface_structures_projects.htm",a:"When you start Maestro BioLuminate, a new, temporary project is created. Projects are the data structures that Maestro uses to store and manage molecular structures, such as proteins and ligands. Each such molecular structure in a project is stored in an entry. An entry can consist of multiple ...",t:"Structures and Projects in BioLuminate"},"383":{i:0.000108610578956696,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table.htm",a:"Using the Toggle Table . The Toggle Table panel can be used to interact with structures in the Workspace. The panel has a set of buttons at the top for quick access to some common actions. The main part of the panel consists of a row for each entry in the Workspace, with a set of buttons, or ...",t:"The BioLuminate Toggle Table"},"384":{i:0.000122864763227978,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_quick_access.htm",a:"This set of buttons at the top of the Toggle Table panel provides quick access to a number of actions, some of which are also on the menus.  Reset—Reset the view to the default view, in which the view axes are aligned with the coordinate axes of the structure.  Zoom—Change the view of the Workspace ...",t:"Quick Access Buttons in the Toggle Table"},"385":{i:0.000135729924325088,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_rows.htm",a:"Each row in the Toggle Table has a title and a set of five action buttons, labeled A, S, H, L, and C. These buttons open cascading menus, and are described in the following sections. There are three distinct types of rows in the Toggle Table: The All row: Actions taken in this row apply to all ...",t:"Toggle Table Rows"},"386":{i:0.000139727224281171,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu.htm",a:"The Toggle Table Action Menu The A button opens the Action menu, from which you can choose a variety of actions for the structure defined by the table row. Not all of the actions are available in every table row. ",t:"The Toggle Table Action Menu"},"387":{i:0.000140969245907307,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_view.htm",a:"The first four actions on the Action menu change the view (camera angle) of the structure defined by the table row. The Workspace coordinate system has the origin in the middle of the Workspace, the x axis is the horizontal axis, the y axis is the vertical axis, and the z axis is coming out of the ...",t:"Changing the View with the Toggle Table"},"388":{i:0.000141355173771301,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_minimize.htm",a:"You can minimize the energy of the Workspace or the selected atoms using the OPLS_2005 force field, by choosing Clean from the Action menu. The current selection is updated (or a new selection created) to refer to the minimized atoms. To avoid starting a lengthy calculation that is better performed ...",t:"Minimizing the Structure Energy from the Toggle Table"},"389":{i:0.000141475096771502,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_appearance.htm",a:"The set of actions on the Preset submenu change the appearance (representation) of the structures using various preset styles. Some of these representations take a little time to set up, and a progress bar is displayed at the bottom of the table. Simple—Show proteins as ribbons (C alpha trace) ...",t:"Changing the Appearance of Structures with the Toggle Table"},"390":{i:0.00014151236385815,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_contacts.htm",a:"You can turn on or off the display of polar contacts (hydrogen bonds) between various groups of atoms from the Polar Contacts submenu. The display is turned on with A → Polar Contacts → Find; it is turned off with A → Polar Contacts → Remove All. The atom groupings are: Within Object Involving Side ...",t:"Displaying Polar Contacts with the Toggle Table"},"391":{i:0.000141523945902263,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_selection.htm",a:"You can select atoms in predefined groups by choosing A → Generate → Selection and then choosing the group. The Generate item is not available for the All or Selection rows. The predefined groups are: All—All atoms.\n     Polymer—Backbone and side-chain atoms.\n     Organic—Ligand atoms.\n     ...",t:"Generating an Atom Selection with the Toggle Table"},"392":{i:0.000141527545992915,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_crystal_mates.htm",a:"If you want to see atoms from nearest-neighbor crystal symmetry mates of your structure, you can choose A → Generate → Symmetry Mates. To show symmetry-related atoms requires crystal symmetry information to be present for the structure. You should also ensure that you have only the structure that ...",t:"Displaying Atoms Related By Crystallographic Symmetry with the Toggle Table"},"393":{i:0.00014152866573924,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_modifying_selecti.htm",a:"The Modify item is available for selection rows only, and replaces Generate on the Action menu. It allows you to alter the group of selected atoms to include or exclude other atoms. After a Modify action, the selection row applies to the new group of atoms. The Modify actions all have a choice of ...",t:"Modifying an Atom Selection with the Toggle Table"},"394":{i:0.000141529015775725,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_removing_selectio.htm",a:"For the All row, Delete Selections removes any selection rows from the Toggle Table. All atoms are deselected but are otherwise unaltered. For selection rows Delete Selection just removes the row from the Toggle Table. The atoms in this selection group are deselected but otherwise remain unaltered.",t:"Removing Selections with the Toggle Table"},"395":{i:0.000141529130841644,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_renaming.htm",a:"You can change the name of an entry row, and thereby change the Title of the project entry with the Rename action. A text box is displayed instead of the name, and you can type a new name in the box. If you decide you do not want to change the name after all, press Esc.  You can change the name of a ...",t:"Renaming Rows with the Toggle Table"},"396":{i:0.00014152918633917,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_duplicating.htm",a:"You can duplicate table rows with the Duplicate action, with the exception of the All row. The action is different for entry rows and for selection rows. For entry rows, the action creates a new project entry below the entry that is the duplicate of the entry. Both structures remain in the Workspace ...",t:"Duplicating Rows with the Toggle Table"},"397":{i:0.000141529267454197,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_exclude.htm",a:"To remove (exclude) an entry from the Workspace, choose A → Remove from Workspace for that row. Removing an entry from the Workspace just means that it is no longer in your working area. The entry remains in the project, and is listed in the Project Table. You can add it back to the Workspace by ...",t:"Removing Entries from the Workspace with the Toggle Table"},"398":{i:0.00014152949766274,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_deleting.htm",a:"Deleting Entries from the Project with the Toggle Table To remove an entry from the project, choose Delete from Project in the entry row. The structure is removed from the Workspace and from the project. It no longer appear in the Project Table.",t:"Deleting Entries from the Project with the Toggle Table"},"399":{i:0.000141530229992272,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_creating.htm",a:"You can create new project entries either from a selection or from an entry row, by choosing Copy to New Project Entry. The new project entry is created by copying the selected atoms or entries, and it is added to the Workspace. This command is the same as Duplicate if you choose it for an entry ...",t:"Creating Project Entries with the Toggle Table"},"400":{i:0.00014153258377618,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_addh.htm",a:"You can add or remove hydrogen atoms from a row by choosing Hydrogens → Add or Hydrogens → Remove. If you perform this action in the All row or an entry row, the atoms are removed from the structure. If you perform this action in a selection row, the hydrogens in the selection are removed, or they ...",t:"Adding and Removing Hydrogens with the Toggle Table"},"401":{i:0.000141540161038823,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_waters.htm",a:"You can remove waters from structures with the Remove Waters action. If you perform this action in the All row or an entry row, the atoms are removed from the structure. If you perform this action in a selection row, the waters in the selection are removed from the structure.",t:"Removing Waters with the Toggle Table"},"402":{i:0.000141564550819168,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_properties.htm",a:"To compute simple properties of the object (selection or entry), you can use the Compute action. The properties you can compute are: Atom Count—The number of atoms. Formal Charge Sum—Sum of atom formal charges. Partial Charge Sum—Sum of atom partial charges. Surface Areas—Compute surface areas. ...",t:"Computing Properties with the Toggle Table"},"403":{i:0.0001416430610569,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_show_menu.htm",a:"The S button opens the Show menu, which contains commands that alter the display of structures in the Workspace. There are three different types of display for structures: Atomic representations such as lines, sticks, ball and sticks or spheres\n     Ribbon representations such as ribbons and ...",t:"The Toggle Table Show Menu"},"404":{i:0.000141895773691809,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_hide_menu.htm",a:"The H button opens the Hide menu, which contains commands that hide features in the Workspace. To redisplay these features, use the Show menu. Each item hides only the particular feature for the row, and leaves any other features as they are. Hiding atoms also hides any representation of the bonds ...",t:"The Toggle Table Hide Menu"},"405":{i:0.000142709213643204,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_label_menu.htm",a:"The L button opens the Label menu, which contains commands that label features in the Workspace. Unless otherwise specified, labels are created for every atom the row applies to. Atom labels created by these commands are not cumulative. Any existing atom labels are removed when new ones are created. ...",t:"The Toggle Table Label Menu"},"406":{i:0.00014532751195167,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_color_menu.htm",a:"The C button opens the Color menu, which contains commands to color the atom representations by various coloring schemes including by element, chain, substructure, B-factor and entry. The schemes are grouped into classes, each of which is represented by a menu item. Color by Element—Color H, C, N, O ...",t:"The Toggle Table Color Menu"},"407":{i:0.000102503491306896,u:"../bioluminate_user_manual/bioluminate_interface_toggle_table_shortcut_menus.htm",a:"The Workspace has two shortcut (context) menus when the Toggle Table panel is open. One is for the selected atoms, and one is for the entire Workspace. To show the shortcut menu for the Workspace, right-click in an empty part of the Workspace. To show the shortcut menu for the selection, right-click ...",t:"Shortcut Menus in BioLuminate"},"408":{i:0.000111830669041571,u:"../bioluminate_user_manual/protein_structure_quality.htm",a:"The quality of a protein structure is often measured by deviations from values reported in the PDB. You can analyze a protein and display tabular and graphical reports on its quality in the Protein Structure Quality Viewer panel, which you open by choosing Tasks → Biologics → Structure Quality.  If ...",t:"Analyzing Protein Quality"},"409":{i:0.000153392611059193,u:"../bioluminate_user_manual/protein_structure_quality_ramachandran.htm",a:"The Ramachandran Plot tab displays a Ramachandran plot of the dihedral angles. Glycines are plotted as triangles, prolines are plotted as squares, all other residues are plotted as circles. The plot has three regions: favored, allowed, and disallowed, which are colored by the color scheme selected ...",t:"Ramachandran Plot Tab"},"410":{i:0.000104788652523789,u:"../bioluminate_user_manual/protein_structure_quality_report.htm",a:"The property table displays a list of values of the property chosen from the Display menu. Selecting table rows zooms the Workspace view in to the structural features listed in those rows, highlights them with a change in representation, and selects them. The average of the selected property over ...",t:"Protein Report Tab"},"411":{i:9.53979351924541E-05,u:"../bioluminate_user_manual/residue_analysis.htm",a:"Identifying stable or unstable residues, or residues with desirable or undesirable properties may be a useful precursor to mutation studies. The Residue Analysis panel analyzes a protein to produce properties of the residues, including hydropathy, various energies, solvent-accessible surface areas, ...",t:"Analyzing Residue Properties"},"412":{i:0.000105596410935652,u:"../bioluminate_user_manual/consensus_viewer.htm",a:"Identifying Consensus Waters . The Consensus Visualization panel helps you to identify conserved waters, counter ions, and ligands for a protein. To open this panel, choose Tasks → Biologics → Consensus Viewer in the Task Tool.   Homologs of the target protein are identified by a BLAST search. You ...",t:"Identifying Consensus Molecules"},"413":{i:0.000131390444882589,u:"../bioluminate_user_manual/consensus_viewer_target.htm",a:"First, you must import your target protein. You can do this by clicking Import and choosing a source. The choices are: Browse for File—Open a file browser in which you can navigate to the desired location and select the file that contains the structure. The allowed file types are Maestro and PDB.  ...",t:"Choosing the Target Protein for Consensus Viewing"},"414":{i:0.000141681049931627,u:"../bioluminate_user_manual/consensus_viewer_homologs.htm",a:"The homologs that are used to identify consensus molecules can be imported, or you can run a BLAST search to locate homologs.  If you already have a set of homologs, you can simply import them with the Import button. Once they are imported you can align them structurally by clicking Align.  To run a ...",t:"Finding and Aligning Homologs for Consensus Viewing"},"415":{i:0.000101470210719922,u:"../bioluminate_user_manual/consensus_viewer_results.htm",a:"After the results are available, the next task is to decide how to define a consensus between the set of structures. There are two choices, available under Define consensus: a minimum percentage or a minimum number of structures. A match between the parent and a homolog for a particular molecule ...",t:"Viewing Consensus Molecules"},"416":{i:0.000117696016482392,u:"../bioluminate_user_manual/reactive_residues.htm",a:"Identifying Reactive Residues in 2pcy . You may need to identify reactive residues in a protein, so that you can mutate them to improve the protein properties. This can be done in the Reactive Protein Residues panel, which you open by choosing Tasks → Biologics → Reactive Residues in the Task Tool. ...",t:"Identifying Reactive Residues"},"417":{i:0.000116062230294524,u:"../bioluminate_user_manual/reactive_residues_custom.htm",a:"If you want to define your own reactive groups, click Edit, to open the Edit Patterns dialog box. This dialog box lists the patterns in a table, giving the pattern name, the definition, and a hotspot index. To edit an existing pattern, double-click the table cell that you want to edit, and enter the ...",t:"Defining Custom Reactive Groups"},"418":{i:0.000109368876897728,u:"../bioluminate_user_manual/protein_interfaces.htm",a:"BioLuminate provides tools for analyzing the specific nature of the interactions at the interface of two proteins, in the Protein Interaction Analysis panel. The analysis locates residues in one protein that are within a given distance of residues in another protein, and presents counts of hydrogen ...",t:"Analyzing Protein Interface Interactions"},"419":{i:0.000144704251458566,u:"../bioluminate_user_manual/protein_interfaces_selecting.htm",a:"The two proteins whose interface you want to analyze must be identified in the Workspace structure. This is done by defining two groups of chains, one for each protein. When you open the Protein Interaction Analysis panel, the chains are listed in the Unassigned chains list in the Define interacting ...",t:"Selecting the Interacting Proteins for Analysis"},"420":{i:0.00013021179733156,u:"../bioluminate_user_manual/protein_interfaces_running.htm",a:"Once you have defined the two groups, you can analyze the interactions at the interface, by clicking Determine Protein Interactions. This may take a minute or two. When the analysis is done, the results are displayed in the table.  The analysis is performed using settings that characterize the ...",t:"Running the Analysis of the Protein Interface"},"421":{i:0.000164021406306044,u:"../bioluminate_user_manual/protein_interfaces_results.htm",a:"The results of the analysis are listed in the table in the center part of the panel. The table columns are described in  Table 1 . You can click on a column heading to sort the table by the values in that column (except for the Specific Interactions column). When you select table rows, the residues ...",t:"Protein Interface Analysis Results"},"422":{i:8.92103966217551E-05,u:"../bioluminate_user_manual/protein_interface_viewing.htm",a:"You can color residues in the Workspace by the value of one of three interaction properties. The color scheme is a linear ramp between the color for a chosen minimum value and the color for a chosen maximum value of the property. The minimum and maximum values can be set to the range of values in ...",t:"Viewing Protein Interaction Properties in the Workspace"},"423":{i:6.83674761879237E-05,u:"../bioluminate_user_manual/protein_low_modes.htm",a:"Finding large-scale motions in proteins can provide information on the flexibility or rigidity of parts of the protein, on domain movements, or give some clues to biochemical processes. Trajectories from molecular dynamics simulations can show the large-scale motions of proteins, but these ...",t:"Locating Large-Scale Motions"},"424":{i:0.000106265144660584,u:"../bioluminate_user_manual/peptide_properties.htm",a:"Predicting Peptide Properties BioLuminate provides several tools for peptides: determining the stability of alpha-helical peptides, developing a QSAR model from peptide sequences, and docking peptides to a protein.",t:"Predicting Peptide Properties"},"425":{i:0.000161184723551008,u:"../bioluminate_user_manual/peptide_properties_stability.htm",a:"Determining the stability of alpha-helical peptides is important for their use as therapeutic agents. You can obtain an estimate of the stability by using the Peptide Helicity panel, which you can open by choosing Tasks → Peptide Alpha Helicity. The primary purpose of this panel is to provide ...",t:"Alpha Helix Stability"},"426":{i:0.000128427164426047,u:"../bioluminate_user_manual/peptide_properties_qsar.htm",a:"Predicting the property of a peptide based on its sequence can be useful when designing and selecting peptides for application as therapeutics. You can build and apply a QSAR-type model that relates properties of peptide sequences to an observed property in the Peptide QSAR panel.  The independent ...",t:"Peptide QSAR from Sequences"},"427":{i:0.000140107738631638,u:"../bioluminate_user_manual/peptide_properties_qsar_input.htm",a:"The first task is to load the sequences and observable property for each sequence, whether you are building a model or applying a model. To do this, click Load Sequences and Observables, and make choices in the Load Sequences and Observables dialog box. The sequences can come from peptide structures ...",t:"Loading the Sequences and Properties"},"428":{i:0.000132990355708408,u:"../bioluminate_user_manual/peptide_properties_qsar_building.htm",a:"When you have a set of sequences with an observable property, you can proceed to building a QSAR model. Select Build a new model to activate the model-building settings. First, you need to choose a training set and a test set. There are three options: You can choose the sets explicitly. Select Use ...",t:"Building a Peptide QSAR Model"},"429":{i:0.000112815715453864,u:"../bioluminate_user_manual/peptide_properties_qsar_examining.htm",a:"When a model is built or applied, the predicted values and statistics for the prediction are shown in the Results tab. Model building actually builds multiple models for increasing numbers of partial least squares factors. To examine the results for a particular number of factors, set the number of ...",t:"Examining the Model"},"430":{i:0.000135903689535295,u:"../bioluminate_user_manual/peptide_properties_qsar_applying.htm",a:"To apply an existing model, follow the steps below. Load the sequences you want to apply the model to, if you do not already have the sequences loaded. See  Loading the Sequences and Properties  for information on loading sequences. Choose the QSAR method for the model you want to apply. Select ...",t:"Applying the Peptide QSAR Model to Other Sequences"},"431":{i:0.000150389492300644,u:"../bioluminate_user_manual/peptide_properties_qsar_options.htm",a:"The default options for the PLS methods usually give good results. However, if you want more control over the application of these methods when building a model, you can click Advanced Options for method and make settings in the dialog box that opens. The options are described in the topics listed ...",t:"Setting Peptide QSAR Model Options for PLS Methods"},"432":{i:0.000132607176646154,u:"../bioluminate_user_manual/peptide_properties_qsar_options_PLS.htm",a:"Set values for the standard PLS method with the following options: Maximum number of PLS factors Specify the maximum number of PLS factors to use in the regression model. Regression models are built for increasing numbers of PLS factors up to this number. The maximum number that can be used is ...",t:"PLS Options for Peptide QSAR"},"433":{i:0.000134908389199932,u:"../bioluminate_user_manual/peptide_properties_qsar_options_KPLS.htm",a:"Set values for the kernel-based PLS method with the following options. For more information on this method, see  Kernel-Based Partial Least-Squares Regression in Canvas . Maximum number of KPLS factors  Specify the maximum number of KPLS factors to use in the regression model. Regression models are ...",t:"KPLS Options for Peptide QSAR"},"434":{i:0.000127085409910961,u:"../bioluminate_user_manual/peptide_properties_docking.htm",a:"As small peptides are of interest as therapeutic agents, it is useful to dock them to a receptor and examine the docked poses and a measure of their binding affinity. This can be done with the Peptide Docking panel, which you open by choosing Tasks → Biologics → Peptide Docking in the Task Tool. The ...",t:"Peptide Binding to a Receptor"},"435":{i:0.000130902161649953,u:"../bioluminate_user_manual/peptide_properties_docking_receptor.htm",a:"The first task is to define the binding site for the peptides on the protein. The grids that are generated for the docking are centered on the location that you choose for the binding site. Two options are available: Centroid of Workspace ligand—Center the binding site and thus the docking grids at ...",t:"Defining the Protein Receptor Region for Peptide Docking"},"436":{i:0.000118468568086291,u:"../bioluminate_user_manual/peptide_properties_docking_ligands.htm",a:"The next task is to specify the peptides to dock, which can be taken from the Project Table or a file, or can be entered by hand. The sequences are listed in the peptide list. You can add peptides from multiple sources, with the Add sequences from buttons: File—Add peptide sequences from a file. ...",t:"Specifying the Peptides to Dock"},"437":{i:0.000117950983381329,u:"../bioluminate_user_manual/peptide_properties_docking_options.htm",a:"Most of the Glide docking options are set by the peptide docking protocol, or have their default values. A few relevant options are available in the panel.  The structures that are built do not have capping groups, and by default are in zwitterionic form, i.e. with charged termini. (The option to ...",t:"Setting Peptide Docking Options"},"438":{i:8.13779670846866E-05,u:"../bioluminate_user_manual/peptide_properties_docking_job.htm",a:"When you have made all the settings, click Run to run the job with the current job settings, or click the Settings button to open the Job Settings dialog box and make settings for the host, distribution of the work, and return of the results.  Docking peptides is time consuming as there are many ...",t:"Running the Peptide Docking Job"},"439":{i:0.000101368373882317,u:"../bioluminate_user_manual/protein_protein_docking.htm",a:"The question of whether one protein binds to another, and where, can be addressed by protein-protein docking. Protein-protein docking in BioLuminate is performed using the Piper program  [5] , under license from Boston University. The job can be set up in the Protein-Protein Docking panel (Tasks → ...",t:"Protein-Protein Docking"},"440":{i:0.00011788915449076,u:"../bioluminate_user_manual/protein_protein_docking_preparation.htm",a:"Although it is not necessary to prepare proteins for docking, it may be advisable to do so. Proteins from the PDB often do not have coordinates for side chains on the surface of the protein, or even for whole chains that are solvent-exposed, as these are usually more mobile and it can be difficult ...",t:"Preparation of Proteins for Protein-Protein Docking"},"441":{i:0.000131013030090055,u:"../bioluminate_user_manual/protein_protein_docking_standard.htm",a:"Docking 1ba7 to 1qqu . To dock a protein to another protein, without any special conditions on the type of protein, choose Standard in the Mode section of the Protein-Protein Docking panel.  Next, choose the receptor protein and the ligand protein. It does not matter which of your two proteins you ...",t:"Docking a Protein to Another Protein"},"442":{i:0.000128705618006092,u:"../bioluminate_user_manual/protein_protein_docking_constraints.htm",a:"Protein-Protein Docking Constraints You can add constraints in the docking process, by providing an additional attractive term to the potential, remove the attractive potential, or declare residues to be buried, for residues that you select. Constraints are implemented as a bias during the docking ...",t:"Applying Protein-Protein Docking Constraints"},"443":{i:0.000126885643004999,u:"../bioluminate_user_manual/protein_protein_docking_restraints.htm",a:"Protein-Protein Docking Distance Restraints If you know which parts of the protein should be in contact, you can add distance restraints between residues in the two proteins. These restraints define a minimum and a maximum distance between a residue in one protein (the receptor) and a residue in the ...",t:"Applying Protein-Protein Docking Constraints"},"444":{i:0.000120628938022831,u:"../bioluminate_user_manual/protein_protein_docking_multimers.htm",a:"You can dock a protein to itself to create a dimer or a trimer. To do so, choose Dimer or Trimer in the Mode section of the Protein-Protein Docking panel. When you do, only one of the menus in the Protein structures section is available, and it is labeled Monomer. You can select the monomer protein ...",t:"Creating Dimers and Trimers in Protein-Protein Docking"},"445":{i:0.000114133243797885,u:"../bioluminate_user_manual/protein_protein_docking_antibody.htm",a:"Antibody-Antigen Docking with 1fsk . The docking of an antigen to an antibody can be performed, with constraints for the target region of the antibody. To dock an antigen to an antibody, choose Antibody in the Mode section. If you want to prevent docking to the non-CDR region, ensure that Mask ...",t:"Docking an Antigen to an Antibody"},"446":{i:9.66699694953971E-05,u:"../bioluminate_user_manual/homology_modeling.htm",a:"Simple Homology Modeling of Proteins . If you have a protein sequence and want to build a homology model of the protein, there are three ways that you can proceed in the BioLuminate interface. Proteins for which you expect a high homology with the template and that require only a straightforward ...",t:"Homology Modeling of Proteins"},"447":{i:0.000125734839404508,u:"../bioluminate_user_manual/residue_loop_mutation.htm",a:"At some point in a workflow, you might want to mutate a single residue, or replace a single loop with another loop. You can do this in the Residue and Loop Mutation panel, which you open from the Tasks menu.  BioLuminate provides other tools for mutations of more than one residue. The Residue ...",t:"Residue and Loop Mutation"},"448":{i:0.000125058007281016,u:"../bioluminate_user_manual/residue_loop_mutation_residues.htm",a:"There are three types of single-point mutation that you can make in this panel: mutation to a standard amino acid, mutation to a nonstandard amino acid, and chirality inversion. You can choose which type to do with the options in the Type section. Select Single mutation for both standard and ...",t:"Residue Mutations with the Residue and Loop Mutation Panel"},"449":{i:0.000130627498299798,u:"../bioluminate_user_manual/residue_loop_mutation_residues_selecting.htm",a:"You can select the residue to mutate by picking it in the Workspace structure or in the sequence viewer, then clicking Workspace Selection in the Original From section to register your choice. To find a particular residue in the Workspace, you can use the Find tool. Type Ctrl+F (⌘F) in the Workspace ...",t:"Selecting the Residue to Mutate"},"450":{i:0.000119526749330959,u:"../bioluminate_user_manual/residue_loop_mutation_residues_mutation.htm",a:"When you have selected the residue to mutate, you can define the mutated residue. The controls to do so are only available if you selected Single mutation in the Type section. If you selected Invert chirality, the mutation is already defined. You can mutate to a standard amino acid or a custom amino ...",t:"Defining the Residue Mutation"},"451":{i:0.000124633014316068,u:"../bioluminate_user_manual/residue_loop_mutation_residues_refining.htm",a:"If you want to refine the mutated structure, click Advanced Refinement Options and make selections in the Refinement Options dialog box. It is usually a good idea to do some sort of refinement, to allow the protein to adjust to the new residue. This is particularly so if you created a custom ...",t:"Refining the Mutated Structure from Residue Mutation"},"452":{i:0.000103902147295126,u:"../bioluminate_user_manual/residue_loop_mutation_loops.htm",a:"You may want to perform larger structural changes than a single residue mutation, such as deleting or inserting multiple residues, or replacing a loop with another loop. To do this, select Deletion, insertion, or loop swap in the Type section, then click Define to define the changes to be made in ...",t:"Insertions, Deletions, and Loop Swaps with the Residue and Loop Mutation Panel"},"453":{i:0.000125807314176684,u:"../bioluminate_user_manual/residue_loop_mutation_loops_choosing.htm",a:"When the Insertions, Deletions, and Loop Swaps panel opens, the sequence viewer at the top is loaded with the sequence of the structure in the Workspace. If you want to use a different s structure, you can replace the Workspace structure with another structure, or you can import a structure from a ...",t:"Choosing the Original and Replacement Loop"},"454":{i:0.000123671314076425,u:"../bioluminate_user_manual/residue_loop_mutation_loops_editing.htm",a:"If you want to modify the replacement loop residues to create the new loop that will replace the original loop, you can do so in the table in the lower part of the panel. Clicking on a table cell opens a menu that allows you to delete or mutate the current residue or insert a new residue before or ...",t:"Editing the Replacement Loop"},"455":{i:9.63671466033467E-05,u:"../bioluminate_user_manual/residue_loop_mutation_loops_output.htm",a:"The replacement loop is built with the Prime structure building software. By default, only the best structure is returned, but if you want to examine more than one structure, you can specify the number of new structures to be returned in the Number of models to generate box.  You can also choose ...",t:"Choosing the Output and Method for Loop Replacement"},"456":{i:0.000113831993491584,u:"../bioluminate_user_manual/crosslink_proteins.htm",a:"As part of protein design, it can be useful to cross-link two proteins. For example, suppose you have two oligopeptide fragments or protein domains that bind to a third protein. Both fragments or domains need to bind to the third protein for function. To increase efficacy, you might want to try to ...",t:"Cross-Linking Proteins"},});