define({'/phase_tutorial/phase_tut_analyzing_enrichment.htm':{i:[296],t:['Viewing Enrichment for a Set of Hypotheses'],b:['']},'/phase_tutorial/phase_tut_copying.htm':{i:[280],t:['Copying the Phase Tutorial Files'],b:['']},'/phase_tutorial/phase_tut_creating.htm':{i:[281],t:['Creating Hypotheses'],b:['']},'/phase_tutorial/phase_tut_creating_common_congeneric.htm':{i:[287],t:['Identifying Common Pharmacophore Hypotheses for Lead Optimization from Ligands in a Congeneric Series'],b:['']},'/phase_tutorial/phase_tut_creating_common_diverse.htm':{i:[286],t:['Identifying Common Pharmacophore Hypotheses from Diverse Ligands for Virtual Screening'],b:['']},'/phase_tutorial/phase_tut_creating_common_inplace.htm':{i:[285],t:['Creating a Common Hypothesis from Prealigned Ligands'],b:['']},'/phase_tutorial/phase_tut_creating_epharmacophore.htm':{i:[284],t:['Creating a Hypothesis with e Pharmacophores from a Protein Ligand Complex'],b:['']},'/phase_tutorial/phase_tut_creating_manual_complex.htm':{i:[283],t:['Creating a Hypothesis Manually from a Protein Ligand Complex'],b:['']},'/phase_tutorial/phase_tut_creating_single_ligand.htm':{i:[282],t:['Creating a Hypothesis from a Single Ligand'],b:['']},'/phase_tutorial/phase_tut_databases.htm':{i:[298],t:['Creating and Modifying Phase Databases'],b:['']},'/phase_tutorial/phase_tut_databases_creating.htm':{i:[299],t:['Creating a Phase Database'],b:['']},'/phase_tutorial/phase_tut_databases_features.htm':{i:[300],t:['Modifying Default Phase Feature Definitions'],b:['']},'/phase_tutorial/phase_tut_modifying.htm':{i:[288],t:['Modifying Hypothesis'],b:['']},'/phase_tutorial/phase_tut_modifying_attributes.htm':{i:[289],t:['Changing and Viewing Hypothesis Attributes'],b:['']},'/phase_tutorial/phase_tut_modifying_merging.htm':{i:[290],t:['Merging Hypotheses'],b:['']},'/phase_tutorial/phase_tut_modifying_xvols.htm':{i:[291],t:['Working with Excluded Volumes'],b:['']},'/phase_tutorial/phase_tut_screening.htm':{i:[292],t:['Screening Ligands Against Hypotheses'],b:['']},'/phase_tutorial/phase_tut_screening_database.htm':{i:[293],t:['Screening Ligands From Phase Databases and Analyzing Hits'],b:['']},'/phase_tutorial/phase_tut_screening_files.htm':{i:[294],t:['Screening Ligand Files using Feature Definition Presets'],b:['']},'/phase_tutorial/phase_tutorial.htm':{i:[279],t:['Pharmacophore Modeling with Phase'],b:['']},'/pldb_tutorial/pldb_tut_analyze_interactions.htm':{i:[369],t:['Analyzing Lysine Interactions with Ligand Pyrroles'],b:['']},'/pldb_tutorial/pldb_tut_arginine_interactions.htm':{i:[371],t:['Searching for Arginine-Ligand Interactions'],b:['']},'/pldb_tutorial/pldb_tut_hinge_indoles.htm':{i:[370],t:['Finding Hinge-Binding Ligand Indoles'],b:['']},'/pldb_tutorial/pldb_tut_intro.htm':{i:[363],t:['Protein Ligand Database (PLDB) Tutorial'],b:['']},'/pldb_tutorial/pldb_tut_ligand_sim.htm':{i:[368],t:['Using a Ligand Similarity Search to Find Factor Xa Binders'],b:['']},'/pldb_tutorial/pldb_tut_login.htm':{i:[364],t:['Logging In to the PLDB'],b:['']},'/pldb_tutorial/pldb_tut_macrocycles.htm':{i:[367],t:['Finding Macrocycles'],b:['']},'/pldb_tutorial/pldb_tut_search_cdk2.htm':{i:[365],t:['Searching for Entries for the CDK2 Gene'],b:['']},'/pldb_tutorial/pldb_tut_serine_proteases.htm':{i:[366],t:['Finding Serine Proteases by Sequence Similarity'],b:['']},'/prime_command_reference/mm_gbsa_output.htm':{i:[3799],t:['prime_mmgbsa—Calculate Ligand Binding Energies'],b:['#prime_commands_1525853049_786431']},'/prime_command_reference/prime_commands.htm':{i:[3787],t:['Prime Command Syntax'],b:['#prime_commands_1525853049_771395']},'/prime_command_reference/prime_commands_blast.htm':{i:[3788],t:['blast—Run Blast Searches'],b:['#prime_commands_1525853049_782924']},'/prime_command_reference/prime_commands_mmgbsa_asl.htm':{i:[3801],t:['ASL Expressions for Prime MM-GBSA'],b:['#prime_commands_1525853049_786457']},'/prime_command_reference/prime_commands_mmgbsa_input.htm':{i:[3800],t:['Prime MM-GBSA Command Input File'],b:['#prime_commands_1525853049_786444']},'/prime_command_reference/prime_commands_prime.htm':{i:[3789],t:['prime—Build and Refine Protein Structures'],b:['#prime_commands_1525853049_776785']},'/prime_command_reference/prime_commands_prime_constraints.htm':{i:[3791],t:['Prime Constraint Syntax'],b:['#prime_commands_1525853049_774992']},'/prime_command_reference/prime_commands_prime_examples.htm':{i:[3793],t:['Examples of Prime Input Files'],b:['#prime_commands_1525853049_748342']},'/prime_command_reference/prime_commands_prime_examples_building.htm':{i:[3794],t:['Protein Model Building Input'],b:['#prime_commands_1525853049_775594']},'/prime_command_reference/prime_commands_prime_examples_energy.htm':{i:[3795],t:['Prime Energy Calculation Input'],b:['#prime_commands_1525853049_777135']},'/prime_command_reference/prime_commands_prime_examples_loop.htm':{i:[3798],t:['Loop Prediction Input'],b:['#prime_commands_1525853049_777172']},'/prime_command_reference/prime_commands_prime_examples_minimization.htm':{i:[3797],t:['Prime Minimization Input'],b:['#prime_commands_1525853049_777160']},'/prime_command_reference/prime_commands_prime_examples_sidechain.htm':{i:[3796],t:['Side-Chain Prediction Input'],b:['#prime_commands_1525853049_777145']},'/prime_command_reference/prime_commands_prime_input.htm':{i:[3790],t:['Prime Command Input File'],b:['#prime_commands_1525853049_764964']},'/prime_command_reference/prime_commands_prime_restart.htm':{i:[3792],t:['Restarting an Incomplete Loop Refinement Job'],b:['#prime_commands_1525853049_776882']},'/prime_command_reference/prime_commands_ska.htm':{i:[3802],t:['ska—Protein Structure Alignment'],b:['#prime_commands_1525853049_748145']},'/prime_command_reference/prime_commands_ssp.htm':{i:[3803],t:['ssp—Secondary Structure Prediction'],b:['#prime_commands_1525853049_764724']},'/prime_command_reference/prime_commands_sta.htm':{i:[3804],t:['sta—Single Template Alignment'],b:['#prime_commands_1525853049_745390']},'/prime_command_reference/prime_sequence_file_formats.htm':{i:[3813],t:['Sequence File Formats Used by Prime'],b:['#prime_sequence_file_formats_233326424_739791']},'/prime_command_reference/prime_utilities.htm':{i:[3805],t:['Prime Command-Line Utilities'],b:['#prime_utilities_3940210242_742746']},'/prime_command_reference/prime_utilities_binding_sites.htm':{i:[3808],t:['Structure Alignment: align_binding_sites'],b:['#prime_utilities_3940210242_63741']},'/prime_command_reference/prime_utilities_buildloop.htm':{i:[3809],t:['Fast Loop Building: buildloop'],b:['#prime_utilities_3940210242_buildloop']},'/prime_command_reference/prime_utilities_database.htm':{i:[3810],t:['Prime Database Update Utilities'],b:['#prime_utilities_3940210242_743988']},'/prime_command_reference/prime_utilities_rsync_pdb.htm':{i:[3812],t:['Updating the PDB Datbase: rsync_pdb'],b:['#prime_utilities_3940210242_rsync_pdb']},'/prime_command_reference/prime_utilities_seqconvert.htm':{i:[3807],t:['Sequence Format Conversion: seqconvert'],b:['#prime_utilities_3940210242_80465']},'/prime_command_reference/prime_utilities_structalign.htm':{i:[3806],t:['Multiple Template Alignment: structalign'],b:['#prime_utilities_3940210242_26664']},'/prime_command_reference/prime_utilities_update_blastdb.htm':{i:[3811],t:['Updating the BLAST Database: update_BLASTDB'],b:['#prime_utilities_3940210242_743499']},'/prime_help/analyze_energy.html':{i:[4816],t:['Calculate Energy Panel'],b:['']},'/prime_help/analyze_energy_options.html':{i:[4814],t:['Analyze Energy - Options Dialog Box'],b:['']},'/prime_help/build_structure_options.html':{i:[4815],t:['Build Structure - Options Dialog Box'],b:['']},'/prime_help/build_structure_step.html':{i:[4835],t:['Prime Structure Prediction — Build Structure Step'],b:['']},'/prime_help/edit_alignment_step.html':{i:[4836],t:['Prime Structure Prediction — Edit Alignment Step'],b:['']},'/prime_help/energy_visualizer.html':{i:[4830],t:['Prime Energy Visualizer Panel'],b:['']},'/prime_help/find_homologs_step.html':{i:[4837],t:['Prime Structure Prediction — Find Homologs Step'],b:['']},'/prime_help/get_sequence_from_pdb.html':{i:[4817],t:['Get Sequence from PDB ID Dialog Box'],b:['']},'/prime_help/hybrid_monte_carlo.html':{i:[4818],t:['Hybrid Monte Carlo Panel'],b:['']},'/prime_help/hybrid_monte_carlo_constraints.html':{i:[4819],t:['Hybrid Monte Carlo - Constraints Dialog Box'],b:['']},'/prime_help/hybrid_monte_carlo_options.html':{i:[4820],t:['Hybrid Monte Carlo - Options Dialog Box'],b:['']},'/prime_help/info_menu.html':{i:[4839],t:['Prime Structure Prediction Menu Bar'],b:['']},'/prime_help/info_runs.html':{i:[4850],t:['Using Prime Runs'],b:['']},'/prime_help/info_third.html':{i:[4841],t:['Prime Third-Party Programs'],b:['']},'/prime_help/input_sequence_step.html':{i:[4838],t:['Prime Structure Prediction — Input Sequence Step'],b:['']},'/prime_help/job_options.html':{i:[4848],t:['Structure Prediction - Job Options Dialog Box'],b:['']},'/prime_help/membrane_setup_panel.html':{i:[4832],t:['Prime Membrane Setup Panel'],b:['']},'/prime_help/minimize.html':{i:[4821],t:['Minimize Panel'],b:['']},'/prime_help/minimize_constraints.html':{i:[4822],t:['Minimize - Constraints Dialog Box'],b:['']},'/prime_help/minimize_options.html':{i:[4823],t:['Minimize - Options Dialog Box'],b:['']},'/prime_help/minimize_rigid_body.html':{i:[4824],t:['Minimize as Rigid Body Panel'],b:['']},'/prime_help/minimize_rigid_body_constraints.html':{i:[4825],t:['Minimize as Rigid Body - Constraints Dialog Box'],b:['']},'/prime_help/minimize_rigid_body_options.html':{i:[4826],t:['Minimize as Rigid Body - Options Dialog Box'],b:['']},'/prime_help/predict_side_chains.html':{i:[4827],t:['Predict Side Chains Panel'],b:['']},'/prime_help/predict_side_chains_constraints.html':{i:[4828],t:['Predict Side Chains - Constraints Dialog Box'],b:['']},'/prime_help/predict_side_chains_options.html':{i:[4829],t:['Predict Side Chains - Options Dialog Box'],b:['']},'/prime_help/prime_macrocycle_sampling.html':{i:[4831],t:['Prime Macrocycle Sampling Panel'],b:['']},'/prime_help/prime_mmgbsa_panel.html':{i:[4833],t:['Prime MM-GBSA Panel'],b:['']},'/prime_help/refine_loops.html':{i:[4842],t:['Refine Loops Panel'],b:['']},'/prime_help/refine_loops_constraints.html':{i:[4843],t:['Refine Loops - Constraints Panel'],b:['']},'/prime_help/refine_loops_options.html':{i:[4844],t:['Refine Loops - Options Dialog Box'],b:['']},'/prime_help/refine_protein_ligand_complex.html':{i:[4845],t:['Refine Protein-Ligand Complex Panel'],b:['']},'/prime_help/refine_protein_ligand_complex_constraints.html':{i:[4846],t:['Refine Protein-Ligand Complex - Constraints Dialog Box'],b:['']},'/prime_help/refine_protein_ligand_complex_options.html':{i:[4847],t:['Refine Protein-Ligand Complex - Options Dialog Box'],b:['']},'/prime_help/sequence_viewer.html':{i:[4834],t:['Prime Sequence Viewer'],b:['']},'/prime_help/structure_prediction.html':{i:[4849],t:['Structure Prediction Panel'],b:['']},'/prime_help/toolbar.html':{i:[4840],t:['Prime Structure Prediction Toolbar'],b:['']},'/prime_quick_start/heteromultimer_building_model.htm':{i:[328],t:['Building the Model'],b:['#heteromultimer_tut_344636661_752962']},'/prime_quick_start/heteromultimer_preparing_cdk4.htm':{i:[326],t:['Preparing the CDK4 Chain'],b:['#heteromultimer_tut_344636661_754995']},'/prime_quick_start/heteromultimer_preparing_cyclin.htm':{i:[327],t:['Preparing the Cyclin-E Chain'],b:['#heteromultimer_tut_344636661_752016']},'/prime_quick_start/heteromultimer_tut.htm':{i:[325],t:['Homology Modeling of a Heteromultimer'],b:['#heteromultimer_tut_344636661_741933']},'/prime_quick_start/homomultimer_building_model.htm':{i:[324],t:['Building the Homomultimer Model'],b:['#homomultimer_tut_3069724356_751512']},'/prime_quick_start/homomultimer_importing_sequence.htm':{i:[322],t:['Importing the hERG Sequence'],b:['#homomultimer_tut_3069724356_754247']},'/prime_quick_start/homomultimer_importing_template.htm':{i:[323],t:['Importing and Selecting the 1ORQ Template'],b:['#homomultimer_tut_3069724356_751495']},'/prime_quick_start/homomultimer_preparing_template.htm':{i:[317],t:['Preparing the 1orq Template'],b:['#homomultimer_tut_3069724356_753263']},'/prime_quick_start/homomultimer_preparing_template_deleting.htm':{i:[319],t:['Deleting Unwanted Parts of the 1orq Structure'],b:['#homomultimer_tut_3069724356_753331']},'/prime_quick_start/homomultimer_preparing_template_importing.htm':{i:[318],t:['Importing the 1orq Template Structure'],b:['#homomultimer_tut_3069724356_753282']},'/prime_quick_start/homomultimer_preparing_template_relabeling.htm':{i:[320],t:['Assigning a Chain Name in the 1orq Template'],b:['#homomultimer_tut_3069724356_751432']},'/prime_quick_start/homomultimer_preparing_template_trimming.htm':{i:[321],t:['Trimming the 1orq Template Structure'],b:['#homomultimer_tut_3069724356_753994']},'/prime_quick_start/homomultimer_tut.htm':{i:[316],t:['Homology Modeling of a Homomultimer'],b:['#homomultimer_tut_3069724356_741933']},'/prime_quick_start/prime_tutorial.htm':{i:[302],t:['Homology Modeling With Prime'],b:['#prime_tutorial_523742316_1011524']},'/prime_quick_start/prime_tutorial_copying.htm':{i:[303],t:['Copying the Prime Tutorial Files'],b:['#prime_tutorial_523742316_1010882']},'/prime_quick_start/prime_tutorial_preferences.htm':{i:[305],t:['Setting Maestro Preferences for the Prime Tutorial'],b:['#prime_tutorial_523742316_1006072']},'/prime_quick_start/prime_tutorial_project.htm':{i:[304],t:['Creating a Maestro Project for the Prime Tutorial'],b:['#prime_tutorial_523742316_1011356']},'/prime_quick_start/single_chain_alignment.htm':{i:[309],t:['Saving the Alignment'],b:['#single_chain_tut_22544230_744704']},'/prime_quick_start/single_chain_building.htm':{i:[312],t:['Building a Model Structure'],b:['#single_chain_tut_22544230_746425']},'/prime_quick_start/single_chain_finding_homologs.htm':{i:[308],t:['Finding Sequence Homologs'],b:['#single_chain_tut_22544230_749537']},'/prime_quick_start/single_chain_importing_query.htm':{i:[307],t:['Importing the Query Sequence'],b:['#single_chain_tut_22544230_741068']},'/prime_quick_start/single_chain_improving_alignment.htm':{i:[310],t:['Improving the Alignment'],b:['#single_chain_tut_22544230_758751']},'/prime_quick_start/single_chain_manual_alignment.htm':{i:[311],t:['Manual Editing of the Alignment'],b:['#single_chain_tut_22544230_744738']},'/prime_quick_start/single_chain_refining.htm':{i:[313],t:['Refining Target Regions of the Structure'],b:['#single_chain_tut_22544230_741744']},'/prime_quick_start/single_chain_refining_loops.htm':{i:[314],t:['Refining Loops'],b:['#single_chain_tut_22544230_749456']},'/prime_quick_start/single_chain_refining_minimizing.htm':{i:[315],t:['Minimizing Target Regions'],b:['#single_chain_tut_22544230_744461']},'/prime_quick_start/single_chain_tut.htm':{i:[306],t:['Homology Modeling of a Single Chain'],b:['#single_chain_tut_22544230_708538']},'/prime_user_manual/mm_gbsa.htm':{i:[1368],t:['Prime MM-GBSA'],b:['#mm_gbsa_4137767329_741933']},'/prime_user_manual/mm_gbsa_maestro.htm':{i:[1369],t:['Running Prime MM-GBSA from Maestro'],b:['#mm_gbsa_4137767329_743614']},'/prime_user_manual/mm_gbsa_output.htm':{i:[1370],t:['Prime MM-GBSA Output'],b:['#mm_gbsa_4137767329_744463']},'/prime_user_manual/prime_energy_visualizer.htm':{i:[1371],t:['Visualizing Prime Energy Terms'],b:['#prime_energy_visualizer_535619057_741933']},'/prime_user_manual/prime_glossary.htm':{i:[1372],t:['Prime Glossary'],b:['#prime_glossary_1882826527_743093']},'/prime_user_manual/prime_intro.htm':{i:[1301],t:['Introduction to Prime'],b:['#prime_intro_1360576712_10346']},'/prime_user_manual/prime_intro_using.htm':{i:[1302],t:['Using Prime from Maestro'],b:['#prime_intro_1360576712_425962']},'/prime_user_manual/prime_references.htm':{i:[1373],t:['Prime References'],b:['']},'/prime_user_manual/protein_structalign.htm':{i:[1365],t:['Protein Structure Alignment'],b:['#protein_structalign_2414557610_743042']},'/prime_user_manual/protein_structure_prediction.htm':{i:[1303],t:['Protein Structure Prediction'],b:['#protein_structure_prediction_1167420164_742191']},'/prime_user_manual/refine_proteins.htm':{i:[1340],t:['Refining Protein Structures'],b:['#refine_proteins_4294271795_751322']},'/prime_user_manual/refine_proteins_complexes.htm':{i:[1348],t:['Refining Protein-Ligand Complexes'],b:['#refine_proteins_4294271795_760387']},'/prime_user_manual/refine_proteins_constraints.htm':{i:[1357],t:['Applying Constraints for Protein Refinement'],b:['#refine_proteins_4294271795_742924']},'/prime_user_manual/refine_proteins_energy.htm':{i:[1349],t:['Calculating the MM Energy of a Protein'],b:['#refine_proteins_4294271795_760374']},'/prime_user_manual/refine_proteins_hybridmc.htm':{i:[1347],t:['Hybrid Monte Carlo Conformational Sampling for Protein Minimization'],b:['#refine_proteins_4294271795_763794']},'/prime_user_manual/refine_proteins_loops.htm':{i:[1343],t:['Refining Loops'],b:['#refine_proteins_4294271795_748877']},'/prime_user_manual/refine_proteins_membrane.htm':{i:[1356],t:['Setting Up an Implicit Membrane'],b:['#refine_proteins_4294271795_761171']},'/prime_user_manual/refine_proteins_minimize.htm':{i:[1345],t:['Minimizing Protein Structures'],b:['#refine_proteins_4294271795_742867']},'/prime_user_manual/refine_proteins_nonstandard.htm':{i:[1363],t:['Protein Refinement with Nonstandard Residues'],b:['#refine_proteins_4294271795_773247']},'/prime_user_manual/refine_proteins_options.htm':{i:[1350],t:['Setting Protein Refinement Options'],b:['#refine_proteins_4294271795_760908']},'/prime_user_manual/refine_proteins_options_common.htm':{i:[1351],t:['Common Protein Refinement Options'],b:['#refine_proteins_4294271795_751240']},'/prime_user_manual/refine_proteins_options_complex.htm':{i:[1355],t:['Protein-Ligand Complex Refinement Options'],b:['#refine_proteins_4294271795_761084']},'/prime_user_manual/refine_proteins_options_hybridmc.htm':{i:[1354],t:['Protein Hybrid Monte Carlo Options'],b:['#refine_proteins_4294271795_765472']},'/prime_user_manual/refine_proteins_options_loop.htm':{i:[1352],t:['Loop Refinement Options'],b:['#refine_proteins_4294271795_747337']},'/prime_user_manual/refine_proteins_options_minimization.htm':{i:[1353],t:['Protein Minimization Options'],b:['#refine_proteins_4294271795_756812']},'/prime_user_manual/refine_proteins_output.htm':{i:[1364],t:['Output of Protein Refinement Jobs'],b:['#refine_proteins_4294271795_746455']},'/prime_user_manual/refine_proteins_panel.htm':{i:[1342],t:['The Protein Refinement Panels'],b:['#refine_proteins_4294271795_743014']},'/prime_user_manual/refine_proteins_prep.htm':{i:[1341],t:['Preparing Protein Structures for Refinement'],b:['#refine_proteins_4294271795_742770']},'/prime_user_manual/refine_proteins_rigidbody.htm':{i:[1346],t:['Minimizing Parts of Protein Structures as Rigid Bodies'],b:['#refine_proteins_4294271795_759389']},'/prime_user_manual/refine_proteins_sidechains.htm':{i:[1344],t:['Predicting Side Chains'],b:['#refine_proteins_4294271795_770892']},'/prime_user_manual/refine_proteins_technical.htm':{i:[1358],t:['Technical Details for Protein Refinement'],b:['#refine_proteins_4294271795_746987']},'/prime_user_manual/refine_proteins_technical_cooploop.htm':{i:[1360],t:['Cooperative Loop Refinement Technical Details'],b:['#refine_proteins_4294271795_746502']},'/prime_user_manual/refine_proteins_technical_hierarchical_sampling.htm':{i:[1362],t:['Hierarchichal Sampling Technical Details'],b:['']},'/prime_user_manual/refine_proteins_technical_loop.htm':{i:[1359],t:['Loop Refinement Technical Details'],b:['#refine_proteins_4294271795_746462']},'/prime_user_manual/refine_proteins_technical_sidechain.htm':{i:[1361],t:['Side-Chain Prediction Technical Details'],b:['#refine_proteins_4294271795_746536']},'/prime_user_manual/structalign_binding_sites.htm':{i:[1367],t:['The Align Binding Sites Panel'],b:['#protein_structalign_2414557610_746605']},'/prime_user_manual/structalign_panel.htm':{i:[1366],t:['The Protein Structure Alignment Panel'],b:['#protein_structalign_2414557610_743921']},'/prime_user_manual/structure_prediction_alignment.htm':{i:[1318],t:['PSP—Edit Alignment Step'],b:['#protein_structure_prediction_1167420164_749039']},'/prime_user_manual/structure_prediction_alignment_adjusting_ssp.htm':{i:[1323],t:['Adjusting Secondary Structure Predictions in PSP'],b:['#protein_structure_prediction_1167420164_749112']},'/prime_user_manual/structure_prediction_alignment_editing.htm':{i:[1327],t:['Manually Editing a Sequence Alignment in PSP'],b:['#protein_structure_prediction_1167420164_749155']},'/prime_user_manual/structure_prediction_alignment_existing.htm':{i:[1319],t:['Using Existing Sequence Alignments in PSP'],b:['#protein_structure_prediction_1167420164_749057']},'/prime_user_manual/structure_prediction_alignment_generating.htm':{i:[1320],t:['Generating a Sequence Alignment in PSP'],b:['#protein_structure_prediction_1167420164_749079']},'/prime_user_manual/structure_prediction_alignment_generating_constraints.htm':{i:[1325],t:['Constraining the Sequence Alignment in PSP'],b:['#protein_structure_prediction_1167420164_752329']},'/prime_user_manual/structure_prediction_alignment_generating_gpcr.htm':{i:[1324],t:['GPCR-Specific Sequence Alignment in PSP'],b:['#protein_structure_prediction_1167420164_749133']},'/prime_user_manual/structure_prediction_alignment_generating_job.htm':{i:[1326],t:['Running the PSP Sequence Alignment'],b:['#protein_structure_prediction_1167420164_752291']},'/prime_user_manual/structure_prediction_alignment_generating_ssp.htm':{i:[1321],t:['Generating Secondary Structure Predictions for PSP'],b:['#protein_structure_prediction_1167420164_749086']},'/prime_user_manual/structure_prediction_alignment_importing_ssp.htm':{i:[1322],t:['Importing and Exporting Secondary Structure Predictions in PSP'],b:['#protein_structure_prediction_1167420164_749096']},'/prime_user_manual/structure_prediction_alignment_messages.htm':{i:[1328],t:['Edit Alignment Error Messages'],b:['#protein_structure_prediction_1167420164_749215']},'/prime_user_manual/structure_prediction_build.htm':{i:[1329],t:['PSP—Build Structure Step'],b:['#protein_structure_prediction_1167420164_749381']},'/prime_user_manual/structure_prediction_build_building.htm':{i:[1338],t:['Building the Protein Structure in PSP'],b:['#protein_structure_prediction_1167420164_749456']},'/prime_user_manual/structure_prediction_build_choose.htm':{i:[1330],t:['Choosing the Model Type for the Protein Structure in PSP'],b:['#protein_structure_prediction_1167420164_749400']},'/prime_user_manual/structure_prediction_build_choose_composite.htm':{i:[1331],t:['Building a Composite Protein Structure Model with PSP'],b:['#protein_structure_prediction_1167420164_749404']},'/prime_user_manual/structure_prediction_build_choose_consensus.htm':{i:[1332],t:['Building a Consensus Protein Structure Model with PSP'],b:['#protein_structure_prediction_1167420164_749407']},'/prime_user_manual/structure_prediction_build_choose_multimer.htm':{i:[1333],t:['Building Protein Multimers with PSP'],b:['#protein_structure_prediction_1167420164_749415']},'/prime_user_manual/structure_prediction_build_preparing.htm':{i:[1334],t:['Preparing to Build a Protein Structure Model in PSP'],b:['#protein_structure_prediction_1167420164_749422']},'/prime_user_manual/structure_prediction_build_preparing_constraints.htm':{i:[1336],t:['Setting Constraints on the Protein Structure in PSP'],b:['#protein_structure_prediction_1167420164_749429']},'/prime_user_manual/structure_prediction_build_preparing_ligands.htm':{i:[1335],t:['Including Ligands and Cofactors in the Protein Structure in PSP'],b:['#protein_structure_prediction_1167420164_749424']},'/prime_user_manual/structure_prediction_build_preparing_options.htm':{i:[1337],t:['PSP Protein Structure Building Options'],b:['#protein_structure_prediction_1167420164_749437']},'/prime_user_manual/structure_prediction_build_technotes.htm':{i:[1339],t:['Technical Notes for the Build Structure Step'],b:['#protein_structure_prediction_1167420164_749475']},'/prime_user_manual/structure_prediction_homologs.htm':{i:[1309],t:['PSP—Find Homologs Step'],b:['#protein_structure_prediction_1167420164_742262']},'/prime_user_manual/structure_prediction_homologs_alignment.htm':{i:[1314],t:['Multiple Templates and Structure Alignment in PSP'],b:['#protein_structure_prediction_1167420164_742431']},'/prime_user_manual/structure_prediction_homologs_finding.htm':{i:[1311],t:['Searching for Homologs in PSP'],b:['#protein_structure_prediction_1167420164_742275']},'/prime_user_manual/structure_prediction_homologs_importing.htm':{i:[1310],t:['Importing Homologs for PSP'],b:['#protein_structure_prediction_1167420164_742271']},'/prime_user_manual/structure_prediction_homologs_messages.htm':{i:[1316],t:['Find Homologs Error Messages'],b:['#protein_structure_prediction_1167420164_751258']},'/prime_user_manual/structure_prediction_homologs_options.htm':{i:[1312],t:['Find Homologs Search Options'],b:['#protein_structure_prediction_1167420164_742296']},'/prime_user_manual/structure_prediction_homologs_results.htm':{i:[1313],t:['PSP Homolog Search Results'],b:['#protein_structure_prediction_1167420164_742319']},'/prime_user_manual/structure_prediction_homologs_runs.htm':{i:[1315],t:['Saving Prime Runs With Different Templates'],b:['#protein_structure_prediction_1167420164_744136']},'/prime_user_manual/structure_prediction_homologs_tablewarn.htm':{i:[1317],t:['Homologs Table Warnings'],b:['#protein_structure_prediction_1167420164_751773']},'/prime_user_manual/structure_prediction_sequence.htm':{i:[1304],t:['PSP—Input Sequence Step'],b:['#protein_structure_prediction_1167420164_742206']},'/prime_user_manual/structure_prediction_sequence_editing.htm':{i:[1307],t:['Editing a Sequence for PSP'],b:['#protein_structure_prediction_1167420164_751092']},'/prime_user_manual/structure_prediction_sequence_messages.htm':{i:[1308],t:['PSP Input Sequence Error Messages'],b:['#protein_structure_prediction_1167420164_751118']},'/prime_user_manual/structure_prediction_sequence_reading.htm':{i:[1305],t:['Reading or Entering a Sequence for PSP'],b:['#protein_structure_prediction_1167420164_750989']},'/prime_user_manual/structure_prediction_sequence_selecting.htm':{i:[1306],t:['Selecting a Query Sequence for PSP'],b:['#protein_structure_prediction_1167420164_751055']},'/primex_command_reference/primex_command.htm':{i:[3815],t:['Running PrimeX Tasks from the Command Line'],b:['#primex_command_3699188359_750168']},'/primex_command_reference/primex_command_export.htm':{i:[3834],t:['PrimeX Keywords for Exporting a Structure to a File'],b:['#primex_command_3699188359_745979']},'/primex_command_reference/primex_command_global.htm':{i:[3816],t:['PrimeX Input File Global Keywords'],b:['#primex_command_3699188359_746336']},'/primex_command_reference/primex_command_global_calcsettings.htm':{i:[3817],t:['Primex Keywords for Calculation Settings'],b:['#primex_command_3699188359_744589']},'/primex_command_reference/primex_command_global_map.htm':{i:[3818],t:['PrimeX Keywords for Map Settings'],b:['#primex_command_3699188359_745620']},'/primex_command_reference/primex_command_global_reflection.htm':{i:[3819],t:['PrimeX Keywords for Reflection Data Settings'],b:['#primex_command_3699188359_744624']},'/primex_command_reference/primex_command_global_selection.htm':{i:[3820],t:['PrimeX Atom Selection Keywords'],b:['#primex_command_3699188359_744641']},'/primex_command_reference/primex_command_global_symmetry.htm':{i:[3821],t:['PrimeX Keywords for Noncrystallographic Symmetry'],b:['#primex_command_3699188359_745720']},'/primex_command_reference/primex_command_ligandsolvent.htm':{i:[3831],t:['PrimeX Keywords for Ligand and Solvent Placement'],b:['#primex_command_3699188359_745948']},'/primex_command_reference/primex_command_map.htm':{i:[3822],t:['PrimeX Keywords for Map Generation'],b:['#primex_command_3699188359_745838']},'/primex_command_reference/primex_command_real.htm':{i:[3823],t:['PrimeX Keywords for Real-Space Refinement'],b:['#primex_command_3699188359_745849']},'/primex_command_reference/primex_command_real_loop.htm':{i:[3824],t:['PrimeX Keywords for Loop Prediction'],b:['#primex_command_3699188359_745861']},'/primex_command_reference/primex_command_real_minimize.htm':{i:[3826],t:['PrimeX Keywords for Minimization'],b:['#primex_command_3699188359_745883']},'/primex_command_reference/primex_command_real_sidechain.htm':{i:[3825],t:['PrimeX Keywords for Side Chain Prediction'],b:['#primex_command_3699188359_745877']},'/primex_command_reference/primex_command_reciprocal.htm':{i:[3827],t:['PrimeX Keywords for Reciprocal-Space Refinement'],b:['#primex_command_3699188359_745889']},'/primex_command_reference/primex_command_reciprocal_minimize.htm':{i:[3830],t:['PrimeX Keywords for Reciprocal-Space Minimization'],b:['#primex_command_3699188359_751984']},'/primex_command_reference/primex_command_reciprocal_rigidbody.htm':{i:[3828],t:['PrimeX Keywords for Rigid-Body Refinement'],b:['#primex_command_3699188359_745921']},'/primex_command_reference/primex_command_reciprocal_simanneal.htm':{i:[3829],t:['PrimeX Keywords for Simulated Annealing'],b:['#primex_command_3699188359_751977']},'/primex_command_reference/primex_command_rfactor.htm':{i:[3833],t:['PrimeX Keywords for R-Factor Calculation'],b:['#primex_command_3699188359_745972']},'/primex_command_reference/primex_command_utilities.htm':{i:[3835],t:['PrimeX Utilities'],b:['#primex_command_3699188359_748443']},'/primex_command_reference/primex_command_utilities_ccp42cns.htm':{i:[3838],t:['ccp42cns'],b:['#primex_command_3699188359_ccp42cns']},'/primex_command_reference/primex_command_utilities_mtzprint.htm':{i:[3836],t:['mtzprint'],b:['#primex_command_3699188359_mtzprint']},'/primex_command_reference/primex_command_utilities_parameter.htm':{i:[3840],t:['Writing PrimeX Parameter Files for Ligands'],b:['#primex_command_3699188359_748432']},'/primex_command_reference/primex_command_utilities_pprep.htm':{i:[3839],t:['Protein Preparation Command for PrimeX'],b:['#primex_command_3699188359_750482']},'/primex_command_reference/primex_command_utilities_refconvert.htm':{i:[3837],t:['refconvert'],b:['#primex_command_3699188359_refconvert']},'/primex_command_reference/primex_command_water.htm':{i:[3832],t:['PrimeX Keywords for Water Placement'],b:['#primex_command_3699188359_745962']},'/primex_help/calculation_settings_db.html':{i:[4852],t:['Calculation Settings Dialog Box'],b:['']},'/primex_help/create_map_panel.html':{i:[4853],t:['Create Map Panel'],b:['']},'/primex_help/density_blobs.html':{i:[4854],t:['Density Blobs Panel'],b:['']},'/primex_help/density_fit_panel.html':{i:[4855],t:['Density Fit Panel'],b:['']},'/primex_help/input_data_db.html':{i:[4856],t:['Input Data Dialog Box'],b:['']},'/primex_help/ligand_solvent_placement_panel.html':{i:[4857],t:['Ligand/Solvent Placement Panel'],b:['']},'/primex_help/mutate_model_to_sequence.html':{i:[4858],t:['Mutate Model to Sequence Dialog Box'],b:['']},'/primex_help/primex_panel.html':{i:[4859],t:['PrimeX Panel'],b:['']},'/primex_help/r-factor_calculation.html':{i:[4869],t:['R-Factor Calculation Panel'],b:['']},'/primex_help/real_space_refinement_panel.html':{i:[4860],t:['Real-Space Refinement Panel'],b:['']},'/primex_help/reciprocal_space_NCS_tab.html':{i:[4865],t:['Reciprocal Space Refinement — NCS Tab'],b:['']},'/primex_help/reciprocal_space_general_tab.html':{i:[4862],t:['Reciprocal Space Refinement — General Tab'],b:['']},'/primex_help/reciprocal_space_maps_tab.html':{i:[4863],t:['Reciprocal Space Refinement — Maps Tab'],b:['']},'/primex_help/reciprocal_space_minimization_tab.html':{i:[4864],t:['Reciprocal Space Refinement — Minimization Tab'],b:['']},'/primex_help/reciprocal_space_refinement_panel.html':{i:[4861],t:['Reciprocal-Space Refinement Panel'],b:['']},'/primex_help/reciprocal_space_rigid_body_tab.html':{i:[4866],t:['Reciprocal Space Refinement — Rigid Bodies Tab'],b:['']},'/primex_help/reciprocal_space_simulated_annealing_tab.html':{i:[4867],t:['Reciprocal Space Refinement — Simulated Annealing Tab'],b:['']},'/primex_help/reference_sequence_db.html':{i:[4868],t:['Reference Sequence Panel'],b:['']},'/primex_help/set_default_map_colors.html':{i:[4870],t:['Set Default Map Colors Dialog Box'],b:['']},'/primex_help/water_placement_panel.html':{i:[4871],t:['Water Placement Panel'],b:['']},'/primex_quick_start/primex_tutorial.htm':{i:[330],t:['Solving an X-ray Structure with PrimeX'],b:['#primex_tutorial_4101874975_741933']},'/primex_quick_start/primex_tutorial_copying_files.htm':{i:[331],t:['Copying the PrimeX Tutorial Files'],b:['#primex_tutorial_4101874975_764119']},'/primex_quick_start/tutorial_adding_waters.htm':{i:[360],t:['Adding Waters to p38 Kinase'],b:['#primex_tutorial_4101874975_745871']},'/primex_quick_start/tutorial_assessing_results.htm':{i:[361],t:['Assessing the Results for p38 Kinase'],b:['#primex_tutorial_4101874975_745873']},'/primex_quick_start/tutorial_bfactor_refinement.htm':{i:[351],t:['B-Factor Refinement for p38 Kinase'],b:['#primex_tutorial_4101874975_745862']},'/primex_quick_start/tutorial_creating_project.htm':{i:[333],t:['Creating a Maestro Project for the PrimeX Tutorial'],b:['#primex_tutorial_4101874975_767874']},'/primex_quick_start/tutorial_examining_density_fit.htm':{i:[342],t:['Examining the Density Fit for p38 Kinase'],b:['#primex_tutorial_4101874975_744729']},'/primex_quick_start/tutorial_initial_R_factors.htm':{i:[335],t:['Generating Initial R-Factors for p38 Kinase'],b:['#primex_tutorial_4101874975_767852']},'/primex_quick_start/tutorial_ligand_placement.htm':{i:[352],t:['Ligand Placement for p38 Kinase'],b:['#primex_tutorial_4101874975_745863']},'/primex_quick_start/tutorial_ligand_placement_centering_display.htm':{i:[355],t:['Centering the Display on Residue 167'],b:['#primex_tutorial_4101874975_746200']},'/primex_quick_start/tutorial_ligand_placement_difference_map.htm':{i:[354],t:['Creating a Difference Map for p38 Kinase'],b:['#primex_tutorial_4101874975_765170']},'/primex_quick_start/tutorial_ligand_placement_examining_poses.htm':{i:[357],t:['Examining the p38 Kinase Ligand Poses'],b:['#primex_tutorial_4101874975_749883']},'/primex_quick_start/tutorial_ligand_placement_importing.htm':{i:[353],t:['Importing and Correcting the p38 Kinase Ligand'],b:['#primex_tutorial_4101874975_765083']},'/primex_quick_start/tutorial_ligand_placement_placing_ligand.htm':{i:[356],t:['Placing the p38 Kinase Ligand'],b:['#primex_tutorial_4101874975_765253']},'/primex_quick_start/tutorial_ligand_placement_refining.htm':{i:[358],t:['Refining the p38 Kinase Ligand'],b:['#primex_tutorial_4101874975_765343']},'/primex_quick_start/tutorial_loop_refinement.htm':{i:[343],t:['Loop Refinement for p38 Kinase'],b:['#primex_tutorial_4101874975_744730']},'/primex_quick_start/tutorial_loop_refinement_comparing.htm':{i:[345],t:['Comparing the New and Original Loops from Residues 118 to 123'],b:['#primex_tutorial_4101874975_746159']},'/primex_quick_start/tutorial_loop_refinement_deleting.htm':{i:[349],t:['Deleting the Loop from Residues 30 to 38'],b:['#primex_tutorial_4101874975_744732']},'/primex_quick_start/tutorial_loop_refinement_minimizing_first.htm':{i:[346],t:['Minimizing the Loop from Residues 118 to 123'],b:['#primex_tutorial_4101874975_745789']},'/primex_quick_start/tutorial_loop_refinement_minimizing_second.htm':{i:[348],t:['Minimizing the Loop from Residues 167 to 175'],b:['#primex_tutorial_4101874975_766130']},'/primex_quick_start/tutorial_loop_refinement_refining_first.htm':{i:[344],t:['Refining the Loop from Residues 118 to 123'],b:['#primex_tutorial_4101874975_765730']},'/primex_quick_start/tutorial_loop_refinement_refining_second.htm':{i:[347],t:['Refining the Loop from Residues 167 to 175'],b:['#primex_tutorial_4101874975_766154']},'/primex_quick_start/tutorial_mutating.htm':{i:[340],t:['Mutating Amino Acids and Placing Side Chains in p38 Kinase'],b:['#primex_tutorial_4101874975_752939']},'/primex_quick_start/tutorial_optimizing_hbonds.htm':{i:[350],t:['Optimizing Hydrogen-Bonding Networks for p38 Kinase'],b:['#primex_tutorial_4101874975_745270']},'/primex_quick_start/tutorial_reading_input.htm':{i:[334],t:['Reading the Input Data for p38 Kinase'],b:['#primex_tutorial_4101874975_767859']},'/primex_quick_start/tutorial_rebuilding_loop.htm':{i:[359],t:['Rebuilding the Loop from Residues 28 to 40'],b:['#primex_tutorial_4101874975_745868']},'/primex_quick_start/tutorial_reciprocal_space_minimization.htm':{i:[341],t:['Reciprocal-Space Minimization for p38 Kinase'],b:['#primex_tutorial_4101874975_744727']},'/primex_quick_start/tutorial_rigid_body.htm':{i:[336],t:['Rigid-Body Refinement of p38 Kinase'],b:['#primex_tutorial_4101874975_744023']},'/primex_quick_start/tutorial_rigid_body_domains.htm':{i:[338],t:['Refining the Two Domains of p38 Kinase'],b:['#primex_tutorial_4101874975_766545']},'/primex_quick_start/tutorial_rigid_body_overall.htm':{i:[337],t:['Refining the Overall Position of p38 Kinase'],b:['#primex_tutorial_4101874975_766447']},'/primex_quick_start/tutorial_simulated_annealing.htm':{i:[339],t:['Simulated Annealing Refinement for p38 Kinase'],b:['#primex_tutorial_4101874975_745715']},'/primex_quick_start/tutorial_skipping.htm':{i:[332],t:['Skipping PrimeX Tutorial Steps'],b:['#primex_tutorial_4101874975_745946']},'/primex_user_manual/create_map_create.htm':{i:[1439],t:['Creating Electron Density Maps'],b:['#primex_create_map_3045762129_744503']},'/primex_user_manual/create_map_display.htm':{i:[1440],t:['Displaying Electron Density Maps'],b:['#primex_create_map_3045762129_744604']},'/primex_user_manual/ligand_solvent_placement.htm':{i:[1434],t:['Placing Ligand, Solvent, and Water Molecules in PrimeX'],b:['#ligand_solvent_placement_2219430518_741933']},'/primex_user_manual/ligand_solvent_placement_ligand.htm':{i:[1435],t:['Placing the Ligand and Solvent Molecules in a Region of Electron Density'],b:['#ligand_solvent_placement_2219430518_744853']},'/primex_user_manual/ligand_solvent_placement_waters.htm':{i:[1436],t:['Placing Water Molecules in Regions of Electron Density'],b:['#ligand_solvent_placement_2219430518_744551']},'/primex_user_manual/non_crystallographers.htm':{i:[1393],t:['PrimeX for Non-Crystallographers'],b:['#non_crystallographers_3322156840_743710']},'/primex_user_manual/non_crystallographers_bfactors.htm':{i:[1396],t:['B-factors and Refinement in PrimeX'],b:['#non_crystallographers_3322156840_743744']},'/primex_user_manual/non_crystallographers_hbond.htm':{i:[1402],t:['Optimizing H-Bond Networks in PrimeX'],b:['#non_crystallographers_3322156840_743987']},'/primex_user_manual/non_crystallographers_input.htm':{i:[1397],t:['PrimeX Input Data'],b:['#non_crystallographers_3322156840_743781']},'/primex_user_manual/non_crystallographers_maps.htm':{i:[1399],t:['Electron Density Maps'],b:['#non_crystallographers_3322156840_743861']},'/primex_user_manual/non_crystallographers_minrefine.htm':{i:[1403],t:['Additional Minimization and B-Factor Refinement'],b:['#non_crystallographers_3322156840_744036']},'/primex_user_manual/non_crystallographers_recipmin.htm':{i:[1401],t:['Reciprocal-Space Minimization of Coordinates'],b:['#non_crystallographers_3322156840_743906']},'/primex_user_manual/non_crystallographers_remediation.htm':{i:[1404],t:['A More Complex Remediation Workflow'],b:['#non_crystallographers_3322156840_744070']},'/primex_user_manual/non_crystallographers_residues.htm':{i:[1400],t:['Missing Residues in PrimeX'],b:['#non_crystallographers_3322156840_743862']},'/primex_user_manual/non_crystallographers_restraints.htm':{i:[1395],t:['Molecular Geometry, Restraints, and Hydrogen Atoms in PrimeX'],b:['#non_crystallographers_3322156840_743742']},'/primex_user_manual/non_crystallographers_rfactors.htm':{i:[1394],t:['R-factors and Refinement in PrimeX'],b:['#non_crystallographers_3322156840_743726']},'/primex_user_manual/non_crystallographers_start.htm':{i:[1398],t:['Where to Start with PrimeX: R-Factors'],b:['#non_crystallographers_3322156840_743745']},'/primex_user_manual/other_primex_tools.htm':{i:[1441],t:['Other PrimeX Tasks and Tools'],b:['#other_primex_tools_3811768154_741933']},'/primex_user_manual/other_primex_tools_blobs.htm':{i:[1445],t:['Locating Density Blobs'],b:['#other_primex_tools_3811768154_744726']},'/primex_user_manual/other_primex_tools_export.htm':{i:[1446],t:['Exporting a Structure from PrimeX'],b:['#other_primex_tools_3811768154_747520']},'/primex_user_manual/other_primex_tools_hbond.htm':{i:[1443],t:['Optimizing H-Bond Networks in PrimeX'],b:['#other_primex_tools_3811768154_744923']},'/primex_user_manual/other_primex_tools_mutate.htm':{i:[1442],t:['Mutating a Model to Match the Sequence'],b:['#other_primex_tools_3811768154_745087']},'/primex_user_manual/other_primex_tools_rfactor.htm':{i:[1444],t:['R-Factor Calculations'],b:['#other_primex_tools_3811768154_744727']},'/primex_user_manual/partial_occupancy.htm':{i:[1437],t:['Partial Occupancy Refinement'],b:['#partial_occupancy_1671719296_741933']},'/primex_user_manual/primex_basics.htm':{i:[1377],t:['PrimeX and Maestro Basics'],b:['#primex_basics_2045212017_741933']},'/primex_user_manual/primex_basics_identify.htm':{i:[1384],t:['PrimeX Basics: Identifying Structural Features'],b:['#primex_basics_2045212017_744988']},'/primex_user_manual/primex_basics_identify_color.htm':{i:[1385],t:['PrimeX Basics: Using Color Schemes'],b:['#primex_basics_2045212017_745011']},'/primex_user_manual/primex_basics_identify_display.htm':{i:[1387],t:['PrimeX Basics: Displaying Atoms and Bonds'],b:['#primex_basics_2045212017_755236']},'/primex_user_manual/primex_basics_identify_feedback.htm':{i:[1388],t:['PrimeX Basics: Workspace and Sequence Viewer Feedback Information'],b:['#primex_basics_2045212017_745095']},'/primex_user_manual/primex_basics_identify_find.htm':{i:[1389],t:['PrimeX Basics: Finding Structural Components in the Workspace'],b:['#primex_basics_2045212017_745393']},'/primex_user_manual/primex_basics_identify_label.htm':{i:[1386],t:['PrimeX Basics: Labeling Atoms'],b:['#primex_basics_2045212017_745027']},'/primex_user_manual/primex_basics_mainwindow.htm':{i:[1378],t:['PrimeX Basics: The Maestro Main Window'],b:['#primex_basics_2045212017_758227']},'/primex_user_manual/primex_basics_manipulate.htm':{i:[1380],t:['PrimeX Basics: Manipulating Structures'],b:['#primex_basics_2045212017_744785']},'/primex_user_manual/primex_basics_manipulate_adjust.htm':{i:[1383],t:['PrimeX Basics: Structure Adjustments'],b:['#primex_basics_2045212017_745816']},'/primex_user_manual/primex_basics_manipulate_transform.htm':{i:[1382],t:['PrimeX Basics: Local Transformations'],b:['#primex_basics_2045212017_758607']},'/primex_user_manual/primex_basics_manipulate_view.htm':{i:[1381],t:['PrimeX Basics: Changing the View (Global Transformations)'],b:['#primex_basics_2045212017_757701']},'/primex_user_manual/primex_basics_modify.htm':{i:[1390],t:['PrimeX Basics: Modifying Structures'],b:['#primex_basics_2045212017_745010']},'/primex_user_manual/primex_basics_panel.htm':{i:[1392],t:['The PrimeX Panel'],b:['#primex_basics_2045212017_745671']},'/primex_user_manual/primex_basics_projects.htm':{i:[1379],t:['PrimeX Basics: Maestro Projects'],b:['#primex_basics_2045212017_744912']},'/primex_user_manual/primex_basics_track.htm':{i:[1391],t:['PrimeX Basics: Tracking Changes'],b:['#primex_basics_2045212017_744984']},'/primex_user_manual/primex_create_map.htm':{i:[1438],t:['Electron Density Maps'],b:['#primex_create_map_3045762129_741933']},'/primex_user_manual/primex_input.htm':{i:[1405],t:['Setting Up a PrimeX Refinement'],b:['#primex_input_74285328_741933']},'/primex_user_manual/primex_input_calcsettings.htm':{i:[1409],t:['Specifying Calculation Settings for PrimeX'],b:['#primex_input_74285328_743709']},'/primex_user_manual/primex_input_calcsettings_bfactor.htm':{i:[1412],t:['Overall B-Factor Scaling'],b:['#primex_input_74285328_744008']},'/primex_user_manual/primex_input_calcsettings_colors.htm':{i:[1419],t:['PrimeX Map Colors'],b:['#primex_input_74285328_744835']},'/primex_user_manual/primex_input_calcsettings_filter.htm':{i:[1413],t:['Reflection Filtering'],b:['#primex_input_74285328_744024']},'/primex_user_manual/primex_input_calcsettings_forcefield.htm':{i:[1415],t:['Force Field Solvation Model for PrimeX'],b:['#primex_input_74285328_744731']},'/primex_user_manual/primex_input_calcsettings_hydrogens.htm':{i:[1414],t:['Hydrogen Placement for PrimeX'],b:['#primex_input_74285328_744038']},'/primex_user_manual/primex_input_calcsettings_memory.htm':{i:[1417],t:['Memory Use in PrimeX FFT'],b:['#primex_input_74285328_744735']},'/primex_user_manual/primex_input_calcsettings_planar.htm':{i:[1416],t:['Restraints on Planar Groups in PrimeX'],b:['#primex_input_74285328_744351']},'/primex_user_manual/primex_input_calcsettings_realspace.htm':{i:[1418],t:['Real-Space Options for PrimeX'],b:['#primex_input_74285328_747866']},'/primex_user_manual/primex_input_calcsettings_resolution.htm':{i:[1410],t:['Resolution Limits in PrimeX'],b:['#primex_input_74285328_744425']},'/primex_user_manual/primex_input_calcsettings_solvent.htm':{i:[1411],t:['Bulk Solvent Corrections for PrimeX'],b:['#primex_input_74285328_747795']},'/primex_user_manual/primex_input_crystalparams.htm':{i:[1408],t:['Specifying Crystallographic Parameters for PrimeX'],b:['#primex_input_74285328_743701']},'/primex_user_manual/primex_input_display.htm':{i:[1420],t:['Initial Structure and Sequence Display in PrimeX'],b:['#primex_input_74285328_744045']},'/primex_user_manual/primex_input_proteinprep.htm':{i:[1421],t:['Preparing the Protein Structure for PrimeX'],b:['#primex_input_74285328_744493']},'/primex_user_manual/primex_input_reflections.htm':{i:[1407],t:['Specifying the Reflection Data for PrimeX'],b:['#primex_input_74285328_743693']},'/primex_user_manual/primex_input_structseq.htm':{i:[1406],t:['Specifying the Initial Structure and Reference Sequence for PrimeX'],b:['#primex_input_74285328_743669']},'/primex_user_manual/primex_intro.htm':{i:[1375],t:['About PrimeX'],b:['#primex_intro_1613245333_741933']},'/primex_user_manual/primex_overview.htm':{i:[1376],t:['The PrimeX Process'],b:['#primex_overview_501504530_741933']},'/primex_user_manual/primex_references.htm':{i:[1453],t:['PrimeX References'],b:['#primex_references_1700234358_742611']},'/primex_user_manual/primex_reserved_residues.htm':{i:[1451],t:['PrimeX Reserved Residue Names'],b:['#primex_reserved_residues_3453932903_739664']},'/primex_user_manual/primex_theory.htm':{i:[1452],t:['PrimeX Implementation Details'],b:['#primex_theory_2776509258_740669']},'/primex_user_manual/primex_validation_tools.htm':{i:[1447],t:['Validation Tools for Predicted X-ray Structures'],b:['#primex_validation_tools_1997670646_741933']},'/primex_user_manual/primex_validation_tools_density.htm':{i:[1448],t:['Checking the Density Fit'],b:['#primex_validation_tools_1997670646_744004']},'/primex_user_manual/primex_validation_tools_rama.htm':{i:[1450],t:['Ramachandran Plots'],b:['#primex_validation_tools_1997670646_743741']},'/primex_user_manual/primex_validation_tools_report.htm':{i:[1449],t:['Protein Reports'],b:['#primex_validation_tools_1997670646_743740']},'/primex_user_manual/real_space.htm':{i:[1430],t:['Real-Space Refinement'],b:['#real_space_3340644892_741933']},'/primex_user_manual/real_space_loop.htm':{i:[1431],t:['Loop Refinement in PrimeX'],b:['#real_space_3340644892_745393']},'/primex_user_manual/real_space_minimize.htm':{i:[1433],t:['Real-Space Minimization'],b:['#real_space_3340644892_744482']},'/primex_user_manual/real_space_sidechain.htm':{i:[1432],t:['Side-Chain Prediction in PrimeX'],b:['#real_space_3340644892_744488']},'/primex_user_manual/reciprocal_space.htm':{i:[1422],t:['Reciprocal-Space Refinement'],b:['#reciprocal_space_2081142045_741933']},'/primex_user_manual/reciprocal_space_general.htm':{i:[1423],t:['General Setup for Reciprocal-Space Refinement'],b:['#reciprocal_space_2081142045_743694']},'/primex_user_manual/reciprocal_space_maps.htm':{i:[1424],t:['Setting Up Maps for Reciprocal-Space Refinement'],b:['#reciprocal_space_2081142045_743718']},'/primex_user_manual/reciprocal_space_minimize.htm':{i:[1426],t:['Reciprocal-Space Minimization'],b:['#reciprocal_space_2081142045_743856']},'/primex_user_manual/reciprocal_space_restraints.htm':{i:[1425],t:['Using Noncrystallographic Symmetry Restraints in Reciprocal-Space Refinement'],b:['#reciprocal_space_2081142045_743799']},'/primex_user_manual/reciprocal_space_rigidbody.htm':{i:[1427],t:['Rigid-Body Refinement'],b:['#reciprocal_space_2081142045_744256']},'/primex_user_manual/reciprocal_space_simanneal.htm':{i:[1428],t:['Simulated Annealing in Reciprocal-Space Refinement'],b:['#reciprocal_space_2081142045_744401']},'/primex_user_manual/reciprocal_space_summary.htm':{i:[1429],t:['Summary of Molecular Geometry from Reciprocal-Space Refinement'],b:['#reciprocal_space_2081142045_746703']},'/protein_prep_command_reference/pprep_command_tools.htm':{i:[3842],t:['Protein Preparation Command-Line Tools'],b:['#pprep_command_tools_1850852907_744119']},'/protein_prep_command_reference/pprep_command_tools_impref.htm':{i:[3843],t:['The impref Utility'],b:['#pprep_command_tools_1850852907_impref']},'/protein_prep_command_reference/pprep_command_tools_prepwizard.htm':{i:[3844],t:['The prepwizard Utility'],b:['#pprep_command_tools_1850852907_prepwizard']},'/protein_prep_command_reference/pprep_command_tools_protassign.htm':{i:[3845],t:['The protassign Utility'],b:['#pprep_command_tools_1850852907_protassign']},'/protein_prep_help/one_step_protein_preparation.html':{i:[4873],t:['One-Step Protein Preparation Panel'],b:['']},'/protein_prep_help/protein_preparation.html':{i:[4875],t:['Protein Preparation Wizard Panel'],b:['']},'/protein_prep_help/proteinprep_import_process_tab.html':{i:[4876],t:['Protein Preparation Wizard — Import and Process Tab'],b:['']},'/protein_prep_help/proteinprep_problems.html':{i:[4874],t:['Protein Preparation - Problems Dialog Box'],b:['']}});