<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="manstyle.css" />



<head>
<title>MOE 2010.10 Release Notes</title>
</head>

<body bgcolor="#ffffff">
<a href="index.htm"><img src="images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>MOE 2010.10 Release Notes</h2>
<hr noshade="noshade" />

<p>
The 2010.10 release of Chemical Computing Group's Molecular Operating
Environment (MOE) software includes a variety of new features, enhancements
and changes that are summarized here.
</p>

<!-- ----------------------------- CORE SYSTEM ------------------------ -->
<p>
<table width="100%"><tr bgcolor="#cccccc"><td align="center"><b>
Core System
</b></td></table>

<p>
<img src="images/blueball.gif">
<span class="new"><b>NEW!</b></span>
<b>MOE System Popup</b>.
The System button on the MOE Button Bar now raises a pop-up panel that
provides streamlined control over the objects in MOE.  All chains are
grouped by chain tag into a collapsible listing.  Within each tag group,
the atoms are partitioned into Receptor, Solvent and Ligand classes and
a control line presented.  Each control line can affect the display
of the atoms it represents:
<ol>
<li><i>Select/deselect</i>.</li>
<li><i>Active/Inert</i> without affecting visibility</li>
<li><i>Hide/Show</i> and simultaneously activate/deactivate</li>
<li><i>Color Atoms</i> (carbon atoms only or all atoms)</li>
<li><i>Render Mode</i> (point, line, stick, ball and stick, etc.)</li>
<li><i>Hide/Show Surfaces</i> associated with the atoms</li>
<li><i>Delete</i> atoms or surfaces or both</li>
</ol>
<p>
The Ctrl and Shift modifiers affect the behavior; for example,
Ctrl+Click will on the Hide/Show will show the clicked line and hide
all other lines within the same tag/receptor/solvent/ligand class;
<i>i.e.</i>, "isolate this line".  Consult the bubble help in the
popup for more information about the keyboard modifiers.
In previous versions, the System button raised the Atom Manager window.
This window must now be raised with
<span class="menu">MOE | Window | Atom Manager</span>.


<p>
<img src="images/blueball.gif">
<span class="new"><b>NEW!</b></span>
<b>MOE Footer</b>.
The MOE Footer now adapts to the current selection and provides information
and controls for editing objects.  Atom names, bond lengths, bond angles
and dihedral angles can modified directly from the footer.  In the case
of dihedral angles, a wheel can be used to modify a torsion, and the
<b>Profile</b> button displays a graph of energy vs. dihedral angle.
Clicking in the plot will modify the structure.
These footer controls supersede the Builder controls which have been
eliminated.
The footer bubble help contains additional information about the
selected atoms.

<p>
<img src="images/blueball.gif">
<span class="new"><b>NEW!</b></span>
<b>Atom Selection Masking</b>.
The <b>R</b>, <b>S</b>, and <b>L</b> buttons on the MOE Button Bar
(above the trackball) facilitate atom selection by mouse in the MOE Window.
Mouse selection of receptor, solvent and ligand atoms can be individually
enabled or disabled.  Double-click to enable selection of one class of
atoms and simultaneously disable selection of other classes.

<p>
<img src="images/blueball.gif">
<span class="new"><b>NEW!</b></span>
<b>Nonbonded Contacts Visualization</b>.
Hydrogen bonds and van der Waals contact control has been enhanced and
are now controlled in the <span class="menu">MOE | Footer | Contacts</span>
popup.
<ol>
<p><li><i>Rendering</i>.
Contacts can be displayed in a number of styles: dashed line with or
without an indicator of contact energy (in kcal/mol) - either a strength
bar or a text label.  Contacts can be enabled, disabled or drawn faintly
by interaction class combinations (Receptor, Ligand and Solvent).
Backbone-backbone interactions can be filtered and energy thresholds
are adjustable.
The settings can saved by right-clicking in the Contacts panel.
</li></p>
<p><li><i>Definition</i>.
Hydrogen bond donor and acceptor definitions are controlled by
the <tt>$MOE/lib/hbond.svl</tt> configuration file which contains
SMARTS pattern tables that determine candidates for hydrogen bonds.
Non-standard CH..X hydrogen bonds are permitted.
</li></p>
<p><li><i>Proton-&pi; Interactions</i>.
A geometric model for proton-&pi; interactions has been added.  Interactions
between planar rings and protons are calculated and drawn in a similar
fashion to hydrogen bonds.  These interactions are intended to represent
non-standard hydrophobic interactions.
</li></p>
</ol>
<p>

<p>
<img src="images/blueball.gif">
<span class="new"><b>NEW!</b></span>
<b>Integration of LigX and the Right Button Bar</b>.
The MOE Right Button Bar and the specialized LigX Button Bar have been
integrated into the new default button bar.  Common functions such as
Open, Close, Delete, Select, Minimize, etc. have been retained and other
functions have been streamlined and/or added.  The View button has
been renamed to the Center button.
<ol>
<p><li><i>SiteView</i>.
The SiteView button renders all objects in the system in a manner similar
to the Isolate function of several applications.  Pocket atoms are
rendered in line mode and ligands (or the selected atoms) in stick.
Atoms far from ligands are hidden.
</li></p>
<p><li><i>Hydrogens</i>.
The Hydrogens button cycles the display of hydrogens between three
states: no hydrogens, polar hydrogens only and all hydrogens.  The display
is restricted to the atom selection (all atoms if nothing is selected).
</li></p>
<p><li><i>Surface</i>.
The Surface menu provides streamlined access to Surfaces and Maps.
Molecular surfaces colored by the new Atom Coloring can be created around
active receptors and ligands.  Van der Waals surfaces (as in LigX) can
also be created.  In addition, the Surfaces and Maps application
can be raised (to edit the created surfaces).
</li></p>
<p><li><i>Constrain</i>.
The Constrain menu allows for Fixing, Unfixing and Tethering atoms as
in LigX.  LigX rendering has been replaced by an on-demand menu item
in the Constrain menu.
</li></p>
<p><li><i>LigX</i>.
The LigX button raises an application for preparing systems for LigX.
The Protonate3D parameters and tether settings are adjustable in the
LigX prepare application.  Multiple systems can be prepared simultaneously.
</ol>

<p>
<img src="images/blueball.gif">
<span class="new"><b>NEW!</b></span>
<b>Ligand Menu on the Button Bar</b>.
The Ligand menu on the MOE Button Bar contains functions related to
structure-based design focusing on the ligand.  Ligand Interactions and
the 2D Molecules applications can be raised.  In addition:
<ol>
<p><li><i>Choose Ligand</i>.
The Choose Ligand function provides a means to focus on a particular
ligand (and classify others as Receptor) similar to the LigX application.
</li></p>
<p><li><i>Ligand Properties</i>
displays the LigX ligand properties: logP, logS, toxicity, rsynth, donor
and acceptor counts.  If a ligand is minimized in a pocket with the
<b>Minimize</b> button then solvation energy of complexation and the
docking score are displayed.  Also, the local ligand strain is calculated
(by minimizing the ligand without the receptor) and the uncomplexed
RMSD is displayed.  Energy values are also displayed in "efficiency" form
(energy divided by heavy atom count).
</li></p>
<p><li><i>Ligand R-Vectors</i>
enables the display of "exit vectors" from visible hydrogens on ligand
atoms.  Each exit vector represents the potential for a methyl substitution
without steric clash (either to the receptor or the ligand).  These vectors
contain much of the information of the LigX Interaction (VDW) surface without
the need to display the surface.
</li></p>
<p><li><i>Duplicate</i>.
The Duplicate function duplicates a single selected ligand and hides the
original.  Fixed status, tethers and set membership are replicated.
</li></p>
<p><li><i>Minimize (Vac)</i>.
The Minimize (Vac) function duplicates a single selected ligand and
performs an isolated energy minimize (in the background) for more detailed
investigation of local strain.
The results are superposed onto the original and the newly created ligand
is set to visible but inert.
</li></p>
</ol>

<p>
<img src="images/blueball.gif">
<b>Enhanced Graphics</b>.
The GPU-accelerated graphics in MOE have been enhanced for further speed,
especially when displaying backbone ribbons and non-bonded contacts.
In addition
<ol>
<p><li>
<span class="new"><b>NEW!</b></span>
<i>Surface Z-Clipping</i>.
An additional pair of clipping planes applying exclusively
to surfaces is now available in MOE and can be accessed through
the new <b>SF</b> and <b>SW</b> sliders under <b>ZClip</b>
in the MOE Footer.
</li></p><p><li>
<span class="new"><b>NEW!</b></span>
<i>Anaglyph Stereo</i>.
MOE now provides Green/Magenta and Red/Cyan anaglyph stereo graphics
that can be perceived using the stereo glasses that have provided
as part of the MOE distribution.  Anaglyph stereo rendering is accessed from
the <span class="menu">MOE | Render | Stereo</span> menu.
</li></p><p><li>
<span class="new"><b>NEW!</b></span>
<i>Anaglyph Stereo Theme</i>.
To enhance color perception when in anaglyph
stereo mode, the <b>Anaglyph Stereo</b>
visualization theme can be used.
This can be selected from the Visualization Setup panel
(<span class="menu">MOE | Render | Setup</span>).
</li></p>
</ol>

<p>
<img src="images/blueball.gif">
<b>Bond Selection</b>.
Bonds are now selectable in the MOE Window.  Clicking towards the middle
of a bond will select the atoms on either end of the bond.

</p>
<p>
<img src="images/blueball.gif">
<span class="new"><b>NEW!</b></span>
<b>Mouse Reference Panel</b>.

A panel describing mouse commands in the MOE Window,
Sequence Editor, and Database Viewer can now be opened
from the <b>Help</b> menu in each of these windows.
</p>

<p>
<img src="images/blueball.gif">
<b>SKC Sketch Files</b>.
2D and 3D queries can now be pasted into MOE.  The new Sketch View
panel offers a 2D diagram view of the molecular structure of a
query, and provides export and print options.  The fully
annotated query can be visualized in the MOE Window.
</p>

<p>
<img src="images/blueball.gif">
<b>Sequence Editor</b>.

<ol>
    <p><li><b>Chain Tag Display</b>.
    Chain tags can now be displayed in the Sequence Editor
    residue area using the <span class="menu">SE | Display | Chain Tag</span>
    command.
    </li></p><p><li>
    <b>Menus</b>.
    New menu commands for displaying chain tags,
    for manipulating chains by tag, and for chain
    coloring have been added to the Sequence Editor menus.
    The Edit and Display menus and the Chain and Selected Chain
    popups are affected.  The new commands allow for setting chain
    tags, selecting chains with the same tag, sorting chains by name or
    tag, and coloring chains by number or by tag.
    </li></p>
</ol>
</p>

<!-- ----------------------------- APPLICATIONS ------------------------- -->
<p>
<table width="100%"><tr bgcolor="#cccccc"><td align="center"><b>
Applications
</b></td></table>

<p>
<img src="images/blueball.gif">
<b>Fragment-Based Drug Design</b>.
Scaffold Replacement, Fragment Linking and Growing, BREED and the new
MedChem Transformations applications have been consolidated into a
single panel opened with from the
<span class="menu">MOE | Compute | Fragments</span> menu.

<ul>
<p><li>
<span class="new"><b>NEW!</b></span>
<b>MedChem Transformations</b>.
A new application to perform common medicinal transformations in 3D
has been added.  A database transformations is systematically applied
to a given molecule in an active site.  The resulting molecules are
refined and scored.  Transformation rules are specified using MDL
format RXN (reaction) files and stored in a database
(<tt>$MOE/lib/medchem_rxn.mdb</tt> by default).  ChemDraw or SymyxDraw
can be used to create or edit the rules.
</li></p><p><li>
<b>Scaffold Replacement</b>.
A new Settings panel provides access to some specialized
parameters for Scaffold Replacement, such as
Link Feature Radius or Intra-Ligand and Protein-Ligand
clash energy thresholds.
</p>

</li></p>
</ul>
</p>

<p>
<img src="images/blueball.gif">
<b>Molecular Dynamics</b>.
Molecular dynamics simulations in MOE have been enhanced in the
following ways:

<ol>
<p><li>
<span class="new"><b>NEW!</b></span>
MOE now provides an interface to the NAMD program through the
Dynamics panel.  The molecular system is prepared in the same way
as for MOE dynamics.  Once the simulation is done, the resulting
trajectory file(s) are imported into MOE using the <tt>NAMD_Import</tt>
SVL function.
</li></p><p><li>
The stages of the dynamics simulation can now be specified
using a new protocol language.  The protocol language describes
a sequence of stages, each with a name (e.g. <tt>heat</tt> or
<tt>equil</tt>) and a set of parameters enclosed in braces
(e.g. <span class="nobr"><tt>{ ps=100 T=(0,300) }</tt></span>).
The sampling intervals and splitting of the output trajectory files
can also now be specified.
</li></p><p><li>
<span class="new"><b>NEW!</b></span>
<b>DCD Trajectory Files</b>.
MOE can now read dynamics coordinate data (DCD) trajectory files
produced by molecular dynamics and crystallography programs
such as NAMD, CHARMM, and XPLOR.
</li></p>
</ol>
</p>

<p>
<img src="images/blueball.gif">
<span class="new"><b>NEW!</b></span>
<b>Solvate</b>.
A new application for solvation replaces the previous Water Soak application.
<span class="menu">MOE | Edit | Build | Solvate</span> adds solvent
molecules to a molecular system, typically prior to a molecular dynamics
simulation, and offers the following features:
<p>
<ol>
<li>
Selection of solvent: water, DMSO, methanol, chloroform, carbon tetrachloride.
</li><li>Support for periodic (cell) or aperiodic (droplet) modes.
</li><li>Configuration of Crystal Parameters or Wall Restraint.
</li><li>Centering and axis aligning the solute.
</li><li>Optional addition of salt counterions to neutralize the system.
</li>
</ol>
</p>
<p>The solvent is read from precalculated periodic solvent files, stored
in SD file format, in the <tt>$MOE/lib/solvent</tt> directory.
</p>

<p>
<img src="images/blueball.gif">
<span class="new"><b>NEW!</b></span>
<b>Kinase Database</b>.
New applications have been added for managing and exploring kinases.
An application is
now available for creating an aligned kinase database
and a specialized search application can be used to locate and browse
entries.
<ol>
<p><li>
<i>Kinase Identification</i>.
Special SVL functions have been created to identify kinase domains
and important structural regions.  These regions are used to align
the domains into an absolute frame of reference.
</li></p><p><li>
<i>Kinase Database Update</i>.
Kinase Database Update is
a new application for preparing and adding
structures to kinase structure databases.
This application was run on the PDB to produce
<tt>$MOE/lib/kinome.mdb</tt>.

</li></p><p><li>
<i>Kinase Search</i>.
The Kinase Search application searches the kinase database
based on specific criteria:

<p>
<ol>
<li><i>Text</i>. Partial, exact, and phrase searches, with
include all, include any, and include none conditions.
</li><li><i>Date</i>. Search for records after a given date; useful for
updates.
</li><li><i>Species</i>, <i>Family</i>. Restrict search to specific
species and families.
</li><li><i>DFG</i>, <i>alphaC</i>, <i>HRD</i>. Search for specific
activation loop conformations or arginine within the HRD motif.
</li><li><i>Sequence/Structure</i>. Sequence only or sequence and
structure search, subject to identity and RMSD cutoffs.
</li><li><i>Ligand Descriptor Filter</i>. Filter for molecular
descriptor criteria, expressed as thresholds or binary conditions.
</li><li><i>Ligand Fingerprint Filter</i>. Filter for ligands
that are similar to a specified molecule in MOE.
</li><li><i>Ligand Class</i>, <i>Binding</i>. Restrict ligands
to specified classes and binding modes.
</li>
</ol>
</p>

<p>
Results from successive searches can be combined using logical
operators.
The panel is opened with <span class="menu">MOE | Compute |
Biopolymer | Kinase Search</span>.
</p>

</li></p><p><li>
<i>Kinase Statistics</i>.
Statistics on the kinome database or on just the
search results can be viewed as a series of pie charts.
The charts display such information as the distribution of
groups, families, species, DFG and alphaC loop activation
conformations, and date, among others. 
The images can be exported for inclusion in presentations.

</li></p><p><li>
<i>Kinase 3D Browser</i>.
The search results (or the entire database) can be browsed using
the Kinome Database Browser, which can be started from
either the Kinase Search application or a Database Viewer.
The Browser can be used to view different fields
of the kinome database.  When the kinome complex field
is chosen for display, the complex is shown in 3D in the MOE
Window, with the pocket region isolated and important regions color coded.
</li></p>
</ol>
</p>
<p>
The Kinase Database and related applications are intended for use
in homology modeling and in structure-based design of kinase inhibitors.
</p>

<p>
<img src="images/blueball.gif">
<b>LowModeMD</b>.
The Conformational Search application has been enhanced to provide
for explicit water conformational search using LowModeMD.  Two new
options are available for excluding water molecules and counter ions
from the RMSD calculation used for duplicate testing.  In this way,
the conformations of a solvated peptide, for example, can be generated
with duplicate testing applying only to the peptide.

<p>
<img src="images/blueball.gif">
<b>Surfaces and Maps</b>.
The Surfaces and Maps application has been rewritten to include
integrated Graphics Object Manager capabilities.
The panel now features a list of the surfaces in the system,
providing controls for
showing, hiding, deleting, and renaming them.  An existing surface can
be edited by selecting it in the list and then changing the surface
parameters in the panel.  The new layout is intended to
improve ease of use and streamline workflow.
In addition, there are new options for Atom Coloring: an option to force
coloring by element and an option to add hydrogen bond geometry information
onto the surface.  This coloring option is the new default.
</p>

<p>
<img src="images/blueball.gif">
<b>PSILO/RCSB Browse</b>.  The RCSB Browse panel has been enhanced
and is now renamed to PSILO/RCSB Browse.
The panel now functions as a direct link between MOE and
a PSILO server in addition to the RCSB Protein Data Bank internet
site as before.  Access to PSILO requires PSILO connectivity and
a PSILO license.  The panel is opened using
<span class="menu">MOE | File | PSILO/RCSB</span>.
The 
</p>

<p>
<img src="images/blueball.gif">
<span class="new"><b>NEW!</b></span>
<b>Molecular Descriptors</b>.  There is a new <tt>'mutagenic'</tt>
molecular descriptor that identifies potential toxic substructures
in ligands.  The <tt>mutagenic</tt> descriptor is a rule-based method
[Kazius <i>et al</i>;
<i>J.&nbsp;Med. Chem., 48</i> (<b>2005</b>) 312&ndash;320] which
identifies (bio)reactive groups including polycyclic aromatic motifs.

</p>

<p>
<img src="images/blueball.gif">
<b>Builder</b>.
The small molecule Builder has been updated in several respects.
A new narrow layout reduces screen space while still providing most
of the functionality of the old builder.
<ol>
<li>The behavior of many of the editing operations (including Undo)
has been made more consistent and more clearly defined.
<li>Addition of functional groups and rings includes a 3D torsion
sampler to generate better initial placements.
<li>A Minimize button (integrated with Ligand Properties) has been added
to streamline 3D building in the style of LigX.
<li>The bond length, angle and dihedral angle controls have been
moved to the MOE Window footer.
</ol>
<p>In addition, the <span class="menu">MOE | Window | Options | Builder</span>
contains configuration settings for the Builder.  The SMILES and Atom Name
and Hybridization (HintLP functionality of the &quot;old&quot; builder)
controls can be enabled or disabled.

<p>
<img src="images/blueball.gif">
<span class="new"><b>NEW!</b></span>
<b>Tutorials</b>.
The following new tutorials have been added to the manual.  These are
available from <span class="menu">MOE | Help</span>.

<ul>
<li>Visualization and Making Pictures
</li><li>LowModeMD
</li><li>MedChem Transformations In-Cleft
</li><li>Protein Alignment
</li><li>SD Pipelining Tools: How to Create a Fragment Database
</li><li>Prepare a Small Molecule
</li><li>Prepare a Protein or a Protein-Ligand Complex
</li><li>Prepare a Small Molecule Dataset
</li><li>Prepare a Conformation Database
</li>
</ul>

<p>
<img src="images/blueball.gif">
<b>Residue Selector</b>.
The residue selector now has an option to extend the current
residue selection to neighboring residues, with or without
taking into account gaps.
</p>

<p>
<img src="images/blueball.gif">
<b>SAReport</b>. A new Grow Scaffolds option provides control
over whether to grow scaffolds whose implied R-group
substituents are completely uniform; such scaffolds can be
grown by absorbing the implied substituents or used as is.
</p>

<p>
<img src="images/blueball.gif">
<b>Docking</b>.
Receptor sidechains can now be tethered using tether weights
derived from atom temperature (B) factors.  This option can be accessed
through the Configure Forcefield Refinement panel.  In addition,
the GOLD interface has been updated to provide for more control over
Genetic Algorithm options.
</p>

<p>
<img src="images/blueball.gif">
<b>Pre-defined Material Styles for POV-Ray Images</b>.
New pre-defined material styles for rendering atoms, bonds,
and ribbons are now available when generating ray-traced images
of the MOE Window.  The themes are available in the
Ray Tracing Settings panel, which is opened from the Save panel.
</p>

<p>
<img src="images/blueball.gif">
<b>Protein Structure and Family Databases</b>.
The PDB databases included with MOE have been updated. They are derived
from all entries in the PDB up to September 21, 2010.
The full database <tt>pdb_2010_09_21.mdb</tt> is located on the DVD.
The non-redundant set is located in <tt>$MOE/lib</tt>.
The structural family database is located in <tt>$MOE/lib/pdb.fam</tt>.
The Antibody Structure database is located in <tt>$MOE/lib/fab*.mdb</tt>.
The Kinase Database is in <tt>$MOE/lib/kinome.mdb</tt>.
</p>

<!-- ----------------------------- SVL PROGRAMMING ------------------------ -->

<p>
<table width="100%"><tr bgcolor="#cccccc"><td align="center"><b>
SVL Programming
</b></td></table>

<p>
<img src="images/blueball.gif">
<span class="new"><b>NEW!</b></span>
<b>Material Widget</b>.
The SVL widget toolkit now includes a pull-down material chooser widget
for selecting material properties (shading, gloss, brightness and transparency).
A preview area shows the current settings and preset values are easily
accessed.

<p>
<img src="images/blueball.gif">
<span class="new"><b>NEW!</b></span>
<b>Special Shell Widgets</b>.  The SVL widget toolkit now offers two new
specialized shells:
    <ul>
    <li>
    <b>Popup Shell</b>.
    The shell manager widget now supports a <tt>popup</tt> mode.
    In this mode, a panel is created as a popup shell that
    will open at the location specified by the <tt>location</tt>
    attribute.
    The location can be with respect to an SVL panel,
    or to a particular widget in an SVL panel.  Clicking away
    from the Popup or pressing <tt>Esc</tt> dismisses the panel.
    </li></p>

    </li></p><p><li>
    <b>Sidepanel Widget</b>.
    Left or right sidepanels can be attached to the MOE Window
    by specifying a shell widget with the <tt>location</tt>
    attribute set to <tt>MOE_Left</tt> or <tt>MOE_Right</tt>,
    respectively.

    </li></p>
    </ul>

<p>
<img src="images/blueball.gif">
<b>Text Widget</b>.
The text widget now has the ability to display text when the
field is empty.  This text disappears when characters are entered
in the text field, and reappears when the field is cleared.
The text to be displayed is assigned to the
<tt>emptyText</tt> attribute, with the display color controlled
using the <tt>emptyForeground</tt> attribute.
</p>

<p>
The following are SVL functions that have been added to MOE.
Consult the SVL Function Index for more information.
</p>

<p>
<table class="titlebar" width="100%">
<tr>
<th>Function</td>
<th>Description</td>
</tr>
<tr>
<td valign="baseline">
<tt>dcd_Open</tt><br />
<tt>dcd_Close</tt><br />
<tt>dcd_*</tt>
</td>
<td valign="baseline">
Open/close/use a DCD trajectory file.
</td>
</tr>
<tr>
<td valign="baseline">
<tt>G_Arrow</tt>
</td>
<td valign="baseline">
Generate a graphics object arrow composed of a cylinder capped
by one or two cones.
</td>
</tr>
<tr>
<td valign="baseline">
<tt>GOpen</tt><br />
<tt>GClose</tt>
</td>
<td valign="baseline">
Create a new or reopen an existing graphics object/close
an existing graphics object in the current task.
</td>
</tr>
<tr>
<td valign="baseline"><tt>jvm</tt></td>
<td valign="baseline">
Execute Java class code from within an SVL program.
</td>
</tr>
<tr>
<td valign="baseline"><tt>matdet</tt></td>
<td valign="baseline">
Given a matrix,
return a vector of numbers whose product is equal to the determinant of
that matrix. 
</td>
</tr>
<tr>
<td valign="baseline"><tt>matmgs</tt></td>
<td valign="baseline">
Apply modified Gram-Schmidt orthogonalization to a set of vectors.
</td>
</tr>
<tr>
<td valign="baseline"><tt>matsvdt</tt></td>
<td valign="baseline">
Calculate an approximate truncated singular value decomposition
of a matrix.
</td>
</tr>
</tr>
<tr>
<td valign="baseline">
<tt>md_MassCenter</tt><br />
<tt>md_RemoveAngularVelocity</tt><br />
<tt>md_KineticEnergy</tt><br />
<tt>md_BoltzmannVelocity</tt>
</td>
<td valign="baseline">
Functions for creating and manipulating velocity vectors
in molecular dynamics simulations.
</td>
<tr>
<td valign="baseline">
<tt>md_WrapWater*</tt>
</td>
<td valign="baseline">
Functions used to wrap water molecules into a molecular
dynamics simulation box.
</td>
</tr>
<tr>
<td valign="baseline">
<tt>md_Constraint*</tt>
</td>
<td valign="baseline">
Functions used to constrain bond lengths during molecular
dynamics simulations.
</td>
</tr>
<tr>
<td valign="baseline"><tt>mdl_ParseCTAB</tt></td>
<td valign="baseline">
Obtain a CTAB structure from an MDL MOL/SD file contents.
</td>
</tr>
</tr>
<tr>
<td valign="baseline">
<tt>mdlq_Open</tt><br />
<tt>mdlq_Close</tt><br />
<tt>mdlq_Match*</tt>
</td>
<td valign="baseline">
Open/close/use an MDL query context
for substructure searching using MDL-style queries.
</td>
</tr>
<tr>
<td valign="baseline">
<tt>mdlq3d_Open</tt><br />
<tt>mdlq3d_Close</tt><br />
<tt>mdlq3d_*</tt>
</td>
<td valign="baseline">
Open/close/use a 3D MDL query context
for 3D substructure searching using MDL-style query features.
</td>
</tr>
<tr>
<td valign="baseline">
<tt>rRibbonEnable</tt><br />
<tt>rSetRibbonEnable</tt>
</td>
<td valign="baseline">
Report/set ribbon display flag.  When the flag is non-zero,
the ribbon will be displayed, otherwise it will be hidden.
</td>
</tr>
<tr>
<td valign="baseline"><tt>tofupper</tt></td>
<td valign="baseline">
Change the capitalization of one or more tokens so that
the first letter is uppercase and the remainder lowercase.
</td>
</tr>
<tr>
<td valign="baseline"><tt>ViewSetFocus</tt></td>
<td valign="baseline">
Set the view focus to place a specified coordinate in the focal
plane.
</td>
</tr>
<tr>
<td valign="baseline"><tt>ViewSetRotationCenter</tt></td>
<td valign="baseline">
Set the center of rotation of the view in the MOE Window.
</td>
</tr>
<tr>
<td valign="baseline"><tt>ViewZSFront</tt></td>
<td valign="baseline">
Set the location of the front surface-only clipping plane.
</td>
</tr>
<tr>
<td valign="baseline"><tt>ViewZSWidth</tt></td>
<td valign="baseline">
Set the separation between the front and back surface-only clipping planes.
</td>
</tr>
<tr>
<td valign="baseline">
<tt>xml_fopen*</tt><br />
<tt>xml_fclose</tt><br />
<tt>xml_*</tt>
</td>
<td valign="baseline">
Open/close/use a previously an XML file.
</td>
</tr>
</table>
</p>

<p>
The following are SVL functions that have been modified in this release.
Consult the SVL Function Index for more information.
</p>

<p>
<table class="titlebar" width="100%">
<tr>
<th>Function</td>
<th>Description</td>
</tr>
<tr>
<td valign="baseline">
<tt>diff_GaussElectronDensityParam</tt>
</td>
<td>
This function now accepts anisotropic displacements as a parameter, which,
if provided, will be used instead of B factors for calculating the
electron density.
</td>
</tr>
<tr>
<td valign="baseline">
<tt>diff_CalcStructureFactorsFFT</tt><br />
<tt>diff_CalcStructureFactorsSum</tt>
</td>
<td valign="baseline">
These functions now accept anisotropic displacements as a parameter.
Also, the return value is now a grid of complex structure factors defined
by hkl indices.
</td>
</tr>
<tr>
<td valign="baseline">
<tt>Draw2DStructure</tt><br />
<tt>Draw2DStructureCTAB</tt>
</td>
<td valign="baseline">
In addition to atom annotations (<tt>annotations</tt>,
<tt>annotcol</tt>), bond annotations can now be
specified in the option vector using the <tt>annotbond</tt> tag.
</td>
</tr>
<tr>
<td valign="baseline">
<tt>Stereo</tt>
</td>
<td valign="baseline">
Now takes the option <tt>'anaglyph'</tt>.
</td>
</tr>
<tr>
<td valign="baseline">
<tt>ViewSetup</tt>
</td>
<td valign="baseline">
Now takes the option <tt>anaglyph_mode</tt>;
1 means green-magenta (default), 0, red-cyan.
</td>
</tr>
</table>
</p>

<p><a href="index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
