<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!   	kinase.htm	Kinase Search and Kinase Update
!!-->

<!--
!!    MOE Online Manuals
!!    COPYRIGHT (C) 1997-2015
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

<script type="text/javascript" 
src="../include/jsincludes_moe.js"></script>

<link rel="stylesheet" type="text/css"
href="../include/manstyle.css" />


<title>Kinase Search</title>

<meta panel Kinase Database Search>

</head>

<body>
<div id="MoeHeader"></div>
<noscript>
	<span class="warning">Warning: JavaScript is disabled. This page will not display correctly.</span>
	<h1 class="title">MOE Documentation</h1>
	<hr noshade="noshade" />
	<style>.LaTeX {color: #C08080;}</style>
</noscript>

<div id="toc-list"><script>var gman_OrderedListTOC=false; </script></div>
<!--
<ul>
<li><a href="#Background">Background and Methods</a>
  <ul>
    <li><a href="#PKClassification">Classification of Protein Kinases</a>
    <li><a href="#PKIdentification">Identification of Protein Kinases in MOE</a>
    <li><a href="#PKStructure">ATP Pocket Architecture and Catalytic
        Reaction Mechanism
        </a>
    <li><a href="#kinRegulate">Kinase Regulation</a>
    <li><a href="#kinInhibitors">Kinase Inhibitors</a>
        <ul>
            <li><a href="#kinInhClasses">Kinase Inhibitor Classes</a>
        </ul>
    <li><a href="#kinClustering">Kinase Structure Clustering in MOE</a>
  </ul>
<li><a href="#kinSearch">Searching the Kinase Database</a>
  <ul>
    <li><a href="#paramGeneral">General Search/Filter Parameters</a>
    <li><a href="#paramKinase">Kinase-specific Search Parameters</a>
    <li><a href="#paramLigand">Ligand-specific Search/Filter Parameters</a>
    <li><a href="#SearchMode">Kinase Database Search Mode</a>
    <li><a href="#SearchResults">Kinase Search Results</a>
	<ul>
	<li><a href="#Browser">Browser</a></li>
	</ul>
  </ul>

<li><a href="#Update">Compiling and Updating the Kinase Database</a></li>

<li><a href="#SeeAlso">See Also</a>
<li><a href="#References">References</a>

</ul>
-->

<!-- ====================== Background and Methods ======================== -->

<p><a name="Background"></a><h2>Background and Methods</h2>
The Protein Kinases (PKs) represent one of the largest superfamilies in eukaryotes 
(~ 2% of vertebrate genes encode kinases). They are functionally related by their 
catalytic domain, a region of approximately 250-300 amino acids.
PKs are ATP-dependent enzymes that facilitate the transfer of a phosphoryl
group from the &gamma;-position of ATP to
hydroxyls of serine, threonine or tyrosine of their respective substrates. Since
the phosphoryl transfer reaction represents one of the most important
posttranslational modifications in the cell it is not surprising to find PKs in
pivotal roles in regulation and control of a wide variety of critical cellular
processes including cell cycle, cell growth, cell death, metabolism and
differentiation.
<br>
Initial interest in protein kinases as pharmacological targets was triggered
by the finding that many viral oncogenes encode structurally modified cellular
PKs with constitutive enzyme activity. Consequently, deregulated kinase
activity has been implicated in the pathology of a number of important human
diseases, including cancer and disorders in the immune system
[Kolibaba&nbsp;1997].

<p>
<a name="PKClassification"></a><h3>Classification of Protein Kinases</h3>
<p>
Despite the fact that almost all kinases use the same donor (usually ATP) and
catalyze the same transfer reaction, PKs display a remarkable
diversity in structure and substrate recognition mechanism. One of the most
elaborate efforts to group PKs based on structures/sequence criteria was
presented by Grishin <i>et al.</i> [Grishin&nbsp;2005] classifying the majority
of PKs (> 98%) into 10 distinct fold groups. The most prominent kinase fold
architecture is represented by Ser/Thr-Tyr which is pharmacologically the most
important group and the sole focus of the MOE Kinase Search application.
<br>
Based on sequence homology, substrate recognition and specificity as well as
cellular locus of enzyme activity, the Ser/Thr Tyr kinases may be further
subdivided in distinct kinase groups, families and subfamilies. Commonly
accepted classification schemes distinguish "conventional" protein kinases
(ePKs) and "atypical" protein kinases [Manning&nbsp;2002].
The ePKs - the larger group - have been subclassified into 8 groups by
examining sequence similarity between catalytic domains, the presence of
accessory domains, and regulation mode  [Manning&nbsp;2002][Hanks&nbsp;1995].
Figure 1 shows a classification tree of the Ser/Thr-Tyr kinases
applied in MOE to classify kinases.
<p align="center">
<img src="kinase/grouptree.png" alt="Kinase classification tree">
<br>&nbsp;<br>
<b>Figure 1:</b> <i>Kinase Classification Tree</i>
</p>
<p>
The catalytic core of PKs can be further subdivided into 12 smaller subdomains
termed <i>Hank's Regions</i> defined as regions uninterrupted by large amino
acid insertions as well as containing a characteristic set of conserved residues
[Hanks&nbsp;1995]. Sequence similarity within these domains
is only significantly high within close relatives of the kinase families and
subfamilies. Only twelve residues within the catalytic core of approx.
300 amino acids are recognized as being invariant or nearly invariant
(conserved in over 95%) and hence strongly implicated in playing an essential
role in ATP-binding and enzyme catalysis.


<!-- ======================= Kinase Identification ======================= -->


<!-- ======================= Kinase Search Panel ====================== -->

<a name="kinSearch"></a><h2>Searching the Kinase Database</h2>

<p>The <a href="../proteins/projsrch.htm">Project Search Panel</a> provides
a convenient way to search kinases. It supersedes the previous 
<b>Kinase Search Panel</b>.</p>


<!-- ======================= Datbase Update ====================== -->

<a name="Update"></a><h2>Compiling and Updating the Kinase Database</h2>
<p>Background information on compiling and updating the MOE Kinase Database can
be viewed in the <a href="pdb_mdb.htm#kinDB">Protein Database</a> section of
the manual.


<!-- =============================== FOOTER =============================== -->

<!-- ----------------------  References ----------------------------------- -->
<a name="References"></a><h2>References</h2>

<table cellpadding="5" width="100%">
<tr>
  <td valign="top">[De&nbsp;Moliner&nbsp;2003]</td>
  <td valign="top">De Moliner E, Moro S, Sarno S, Zagotto G, Zanotti G, Pinna
    LA, Battistutta R;
    Inhibition of Protein Kinase CK2 by Anthraquinone-related Compounds
    <span class="nobr"><i>J Bio Chem 278:3</i></span>
    (<b>2003</b>) 1831&ndash;1836
  </td>
</tr>
 <tr>
  <td valign="top">[Ghose&nbsp;2008]</td>
  <td valign="top">Ghose AK, Herbertz T, Pippin DA, Salvino JM, Mallamo JP;
    Knowledge Bases Prediction of Ligand Binding Modes and Rational Inhibitor
    Design for Kinase Drug Discovery
    <span class="nobr"><i>J Med Chem 51:17</i></span>
    (<b>2008</b>) 5149&ndash;5171
  </td>
 </tr>
 <tr>
  <td valign="top">[Grishin&nbsp;2005]</td>
  <td valign="top">Cheek S, Ginalski K, Zhang H, Grishin NV;
    A comprehensive update of the sequence and structure classification of
    kinases
    <span class="nobr"><i>BMC Structural Biology 5:6</i></span>
    (<b>2005</b>)
  </td>
</tr>
<tr>
  <td valign="top">[Hanks&nbsp;1995]</td>
  <td valign="top">Hanks SK, Hunter T;
    The eukaryotic protein kinase superfamily: kinase (catalytic) domain
    structure and classification
    <span class="nobr"><i>FASEB J 9:8</i></span>
    (<b>1995</b>) 576&ndash;596
  </td>
</tr>
<!--<tr>
  <td valign="top">[Hanks&nbsp;2003]</td>
  <td valign="top">Steven K. Hanks;
     Genomic analysis of the eukaryotic protein kinase superfamily:
     a perspective
      <span class="nobr"><i>Genome Biology</i></span> Vol 4, Issue 5
      (<b>2003</b>) 111.1&ndash;111.7.
  </td>
</tr>-->
<tr>
  <td valign="top">[Knighton&nbsp;1991]</td>
  <td valign="top">Knighton DR, Zheng JH, Ten Eyck LF, Ashford VA, Xuong NH,
    Taylor SS, Sowadski JM;
    Crystal structure of the catalytic subunit of cyclic adenosine
    monophosphate-dependent protein kinase
    <span class="nobr"><i>Science 253</i></span>
    (<b>1991</b>) 407&ndash;414
  </td>
</tr>
<tr>
  <td valign="top">[Kolibaba&nbsp;1997]</td>
  <td valign="top">Kolibaba KS and Druker BJ;
    Protein tyrosine kinases and cancer
    <span class="nobr"><i>Biochim Biophys Acta 13333:3</i></span>
    (<b>1997</b>) F217&ndash;248
  </td>
</tr>
<tr>
  <td valign="top">[Manning&nbsp;2002]</td>
  <td valign="top">Manning G, Whyte DB, Martinez R, Hunter T, Sudarsanam S;
    The Protein Kinase Complement of the Human Genome
    <span class="nobr"><i>Science 298</i></span>
    (<b>2002</b>) 1912&ndash;1934
  </td>
</tr>
<tr>
  <td valign="top">[Nolen&nbsp;2004]</td>
  <td valign="top">Nolen B, Taylor S, Ghosh G;
    Regulation of Protein Kinases: Controlling Activation through Activation
    Segment Conformation
    <span class="nobr"><i>Molecular Cell 15</i></span>
    (<b>2004</b>) 661&ndash;675
  </td>
</tr>
<tr>
  <td valign="top">[Pargellis&nbsp;2002]</td>
  <td valign="top">Pargellis C, Tong L, Churchill L, Cirillo PF et al.;
    Inhibition of p38 MAP kinase by utilizing a novel allosteric binding site
    <span class="nobr"><i>Nature Structural Biology 9:4</i></span>
    (<b>2002</b>) 268&ndash;272
  </td>
</tr>
<!--<tr>
  <td valign="top">[Scheeff&nbsp;2009]</td>
  <td valign="top">Scheeff ED, Eswaran J, Bunkoczi G, Knapp S, Manning, G;
    Structure of the Pseudokinase VRK3 Reveals a Degraded Catalytic Site, a
    Highly Conserved Kinase Fold, and a Putative Regulatory Binding Site
    <span class="nobr"><i>Structure 17</i></span>
    (<b>2009</b>) 128&ndash;138
  </td>
</tr>-->

<tr>
  <td valign="top">[Shan&nbsp;2009]</td>
  <td valign="top">Shan Y, Seeliger MA, Eastwook MP, Xu H, Jensen M, Dror RO,
    Kuriyan J, Shaw DE;
    A conserved protonation-dependent switch controls drug binding in the
    Abl-kinase
    <span class="nobr"><i>PNAS 106:1</i></span>
    (<b>2009</b>) 139&ndash;144
  </td>
</tr>
<!--<tr>
  <td valign="top">[Steinbeck&nbsp;2009]</td>
  <td valign="top">Steinbeck C, Al-Lazikani B, Hermjakob J, Overington J,
    Thornton J;
    New open drug activity data at EBI
    <span class="nobr"><i>Chemistry Central Journal 3(Suppl I):O3</i></span>
    (<b>2009</b>)
    <br>Kinase SARfari Website:
    <a href="https://www.ebi.ac.uk/chembl/sarfari/kinasesarfari/">
    https://www.ebi.ac.uk/chembl/sarfari/kinasesarfari/</a>
  </td>
</tr>-->
<tr>
  <td valign="top">[Vieth&nbsp;2003]</td>
  <td valign="top">Vieth M, Higgs RE, Robertson DH, Shapiro M, Gregg EA,
    Hemmerle H;
    Kinomics - structural biology and chemogenomics of kinase inhibitors and
    targets
    <span class="nobr"><i>Biochem et Biopys Acta 1697</i></span>
    (<b>2003</b>) 243&ndash;257
  </td>
</tr>
</table>

<a name="SeeAlso"></a><h2>See Also</h2>

<p>
<a href="projsrch.htm">Project Search</a><br />
<a href="prosearch.htm">Homology Searching</a><br />
<a href="promodel.htm">Building Homology Models</a><br />
<a href="../apps/progeom.htm">Protein Geometry</a><br />
<a href="align.htm">Sequence and Structure Alignment</a><br />
<a href="pdb_mdb.htm">Protein Sequence and Structure Data</a>

<noscript>
	<hr noshade="noshade" />
	<font size="2"> Copyright &copy; 1997&ndash;2015
	<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a> </font>
</noscript>
<div id="MoeFooter"></div>
</body>
</html>
