<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!	/proteins/moeprojconfig.htm		MOE Project Configuration Help
!!-->

<!--
!!    MOE Online Manuals
!!    COPYRIGHT (C) 1997-2015
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

<meta keywords project, database organizer, alignment, superposition, configure />
<meta panel MOE Project Configuration>

<script type="text/javascript" 
src="../include/jsincludes_moe.js"></script>

<link rel="stylesheet" type="text/css"
href="../include/manstyle.css" />

<style>
h2 {margin:3.6rem 0 1.0rem 0;}
h6 {margin:1.2rem 0 0.4rem 0;}
dt {font-weight: bold;}
</style>

<title>MOE Project Configuration Help</title>

</head>

<body>
<div id="MoeHeader"></div>
<noscript>
	<span class="warning">Warning: JavaScript is disabled. This page will not display correctly.</span>
	<h1 class="title">MOE Documentation</h1>
	<hr noshade="noshade" />
	<style>.LaTeX {color: #C08080;}</style>
</noscript>

<h6 class="h-two">Overview</h6>
This page describes all of the fields that can be created via the
<a href="../moe/moeweb_intro.htm#Configure_Project_Database">Configure Project Database</a> tool 
on the MOE/web Homepage, which is an interactive editor
to create the configuration file needed for <a href="../proteins/moeproject.htm">MOE Project</a>. 
The content here is identical to the bubble help that is available within the interactive editor.

<h6 class="h-two">Contents</h6>
<div id="toc-list"><script>var gman_OrderedListTOC=false; </script></div>


<!-- All content from here until the See Also is auto-generated from the MoeProject xsd file bubble help content. -->
<h2>Common Options</h2>

<p>The MDB database fields determine various field properties in the database and their
related details.
<!-- TSedit: add the next line for context. -->
The Common Options here are properties that are used in all of the database fields.</p>
</p>

<dl>
    <dt>Field Name</dt>

    <dd>The field name to be used in the MDB file.</dd>

    <dt>Field Type</dt>

    <dd>The field data type in the MDB file.</dd>

    <dt>Field Precision</dt>

    <dd>The field precision to be used in the MDB file when displaying float and double
    fields. This specifies the number of digits to display after the decimal point, in
    either scientific notation or standard numeric notation, depending on whether Precision
    Sig. Figs. is checked. This affects only the display of the data; the full input data
    is always stored in the MDB file.</dd>

    <dt>Precision Sig. Figs.</dt>

    <dd>If checked, scientific notation is specified under Field Precision, otherwise the
    value is taken as number of decimal places to display. For example, if checked, a Field
    Precision of 3 will display 4 significant digits in scientific notation (3 after the
    decimal); if unchecked, a Field Precision of 3 will display 3 digits after the decimal
    place in standard numeric notation.</dd>

    <dt>Sort Priority</dt>

    <dd>Specify the desired sort order of the output MDB file, using numbers to indicate
    the columns to sort. Records will be sorted first by the column with a 1 for Sort
    Priority, and any ties will be broken by sorting the column with a value of 2 for Sort
    Priority, and so on. If tie-breaking is not desired, specify a 1 in the column to be
    sorted, and leave the other columns blank.</dd>

    <dt>Sort Direction</dt>

    <dd>Choose whether the field's data should be sorted in ascending or descending order.
    This has an effect only if Sort Priority indicates the field is being sorted.</dd>

    <dt>Display</dt>

    <dd>If Hide is selected, the field will be hidden by default in the output MDB
    file.</dd>

    <dt>Foreground Color</dt>

    <dd>The foreground color for this field in the MDB file, expressed as a MOE coloring
    expression. See Coloring Fields in the Molecular Database Viewer section of Chapter 1
    in the MOE manuals for details on how to create a coloring expression.</dd>

    <dt>Background Color</dt>

    <dd>The background color for this field in the MDB file, expressed as a MOE coloring
    expression. See Coloring Fields in the Molecular Database Viewer section of Chapter 1
    in the MOE manuals for details on how to create a coloring expression.</dd>

    <dt>Creation Order</dt>

    <dd>Specify the desired order in which the fields should appear in the output MDB file,
    using numbers here to indicate the order.</dd>

    <dt>Bubble Help</dt>

    <dd>The text of the Bubble Help that will be displayed for the field in the Database
    Viewer for the output MDB file.</dd>

    <dt>Temporary</dt>

    <dd>If checked, the field will be deleted at the end of the procedure and will not be
    present in the final MDB file. This can be useful if the field is a data source for
    some of the other fields but is not needed in its own right for the final MDB.</dd>

    <dt>Comment</dt>

    <dd>A comment can be entered here, explaining why certain options were chosen within
    this field, for example. The comment is ignored in the resulting output MDB file and is
    only used for display purposes when editing the configuration file.</dd>
</dl>

<h2>MOE</h2>

<p>The MOE field is used to generate a moe datatype field in the database that contains the
main entry for visualization. There is only one such field. Various processing elements can
be added to prepare the data. Options related to alignment, superposition, crystal
contacts, and electron density are also provided here.</p>

<h3>Reference Alignment</h3>

<p>This section specifies properties related to the Reference Alignment for the project.
This defines the frame of reference in which the structures are superposed, among other
things.</p>

<dl>
    <dt>Site</dt>

    <dd>A MOE selection language string that indicates where the ligands, or other area of
    interest, in the structure is located. These can be real or dummy atoms, and they
    affect where grids are truncated, and what area to zoom in on by default when a
    structure is loaded.</dd>

    <dt>Site Extend</dt>

    <dd>The distance, in Angstroms, to extend around the selection specified in Site
    (above) when displaying grids, electron density, and so on.</dd>

    <dt>Renumber Solvent Set</dt>

    <dd>A MOE selection expression for the solvent atoms in the reference structure that
    should be used for renumbering solvent atoms in each family member. If this is left
    blank, no solvent renumbering will be performed. Otherwise, as each structure is
    superposed on the reference, any solvent atom within a certain distance of a solvent
    atom in this set will have its UID and insertion code copied from the reference atom.
    If it is close to more than one atom in the reference set, the closest will be used.
    The distance cutoff is specified in Renumber Solvent Extend (below). If renumbering the
    solvent atoms results in non-unique identifiers, then other residues will be renumbered
    to avoid the naming clash if possible. Waters in the structures should be put in their
    own chain, so that there is no chance of the renumbering colliding with existing
    residues.</dd>

    <dt>Renumber Solvent Extend</dt>

    <dd>The distance, in Angstroms, that a solvent atom must be to a solvent atom defined
    in the Renumber Reference Set (above), in order for the UID and insertion code to be
    copied from it.</dd>
</dl>

<h3>Group By (subdirectory)</h3>

<p>This section specifies details of Group By values based on subdirectory names. This
allows for dynamic grouping in the MOE system manager. Structures can then be regrouped
on-the-fly by the various properties. The level of the subdirectory within the hierarchy
must also be specified.</p>

<dl>
    <dt>Property</dt>

    <dd>The name for the property encoded in the subdirectory names.</dd>

    <dt>Directory Depth</dt>

    <dd>The depth of the subdirectory from the root of the project data directory, from
    which the named Property (above) is to be extracted. Subdirectories within the root of
    the project data folder have a depth of 1, subdirectories within these have a depth of
    2, and so on. Any structure files located at a depth smaller than the given value will
    not receive any value for the given property.</dd>

    <dt>Split Character</dt>

    <dd>Specifies the characters that are used to split subdirectory names for use with
    Group By. If the full directory name should always be used, leave this blank or specify
    a character that will never appear in the directory names and use $1 for the Split
    Pattern. This field is used in conjunction with Split Pattern; see there for more
    details.</dd>

    <dt>Split Pattern</dt>

    <dd>Used in conjunction with Split Character to specify the value of the property to
    group by. The first element has index 1. Any $n used within this string, where n is a
    digit, will be replaced by the nth element extracted via the Split Character. Other
    characters are taken literally. For example, if a subdirectory was named
    P01242_protonated_raw, Split Character was '_' and Split Pattern was '$2-$3', then the
    Group By value would be 'protonated-raw'.</dd>
</dl>

<h3>Sequence File</h3>

<p>This section specifies details of sequence files that indicate the amino acid sequence
of chains within the structure files. This is only needed if SEQRES is missing from the
structure files, and it is then used to compute where empty residues (with no electron
density and thus no coordinates) belong.</p>

<dl>
    <dt>Filename Pattern</dt>

    <dd>The source file pattern from which to get a FASTA file for the protein. Use * to
    refer to the filename (without extension) of the structure file &mdash; the file name
    of the FASTA file must be defined relative to this. If provided, the sequence will be
    used in place of any existing SEQRES records in the structure, in order to construct
    empty residues (those with no atomic coordinates).</dd>
</dl>

<h3>Sequence Alignment</h3>

<p>This section specifies details on how the sequence alignment between each structure and
the reference is performed, and whether any renumbering of residues should be
performed.</p>

<dl>
    <dt>Renumber Receptor</dt>

    <dd>If checked, the receptor residues will be renumbered to match the reference
    alignment (UID and insertion codes). The first sequence in the reference is assumed to
    be some sort of consensus and thus should be numbered accordingly, as it will be used
    as the basis of the numbering.</dd>

    <dt>Realign Chains</dt>

    <dd>The chains of interest from each structure are aligned to the reference, but this
    does not guarantee that the sequences will then all be aligned to each other, as each
    may result in different gaps with respect to the reference. If Realign Chains is
    checked, all of the sequences will be loaded at the end of the whole procedure, and
    they will be realigned. The realignment occurs with the constraint that all residues
    that are aligned to a particular column in the reference alignment will be aligned to
    each other in the result.</dd>
</dl>

<h3>Inherit Annotations</h3>

<p>This section lists any MOE collections that should be copied from the reference
alignment to each structure as it is aligned and processed.</p>

<dl>
    <dt>Name</dt>

    <dd>The name of a MOE Collection (Set) specified in the reference alignment file. After
    alignment, the specified sets will be copied from the reference to the target
    structure. This can be used to copy sequence annotations of key regions in the protein.
    Name patterns such as 'set*' can be used as well, in which case all matching sets will
    be inherited.</dd>
</dl>

<h3>Ligand Details</h3>

<p>This section provides details on how ligand corrections, if any, should be performed, as
well as how the ligand should be named in the MOE system manager.</p>

<dl>
    <dt>Source SD File Pattern</dt>

    <dd>The source file pattern from which to get a dictionary SD file for ligand
    correction. Use * to refer to the filename (without extension) of the structure file
    &mdash; the file name must be defined relative to this. A field within these files can
    also specify all or part of the ligand name. If provided for a given structure, the
    first ligand in the SD file will also be compared to all candidate ligands found within
    the Site specification under Reference Alignment. Only those matching the heavy-atom
    skeleton of the SD file will be considered as a ligand. This provides an easy way to
    filter out unwanted co-factors, for example, and focus on the ligand of interest in the
    binding pocket. If an empty (zero bytes) SD file is given, ALL molecules in the binding
    pocket are assumed to be uninteresting, and no ligand will be specified.</dd>

    <dt>Source Property File Pattern</dt>

    <dd>The source file pattern from which to get properties to be used for all or part of
    the Ligand Name (below); see there for more details.</dd>

    <dt>Ligand Name</dt>

    <dd>An expression describing how the name for the ligand should be constructed for
    display in the MOE System Manager. If blank, no special naming will be applied to the
    ligand chain. Several special tokens can be used as follows: 
	&nbsp;&nbsp;&nbsp;$PDB$
    will be replaced with the HETNAM or HETSYN from the PDB file.
	&nbsp;&nbsp;&nbsp;$SDF:xyz$ will be replaced with the contents of field 'xyz' from the
    SD file matching the Source Property File Pattern (above). 
	&nbsp;&nbsp;&nbsp;$PROP:xyz$
    will take property 'xyz' from the property file that matches the Source Property File
    Pattern. 
	&nbsp;&nbsp;&nbsp;$FILE:cc:NN$ will take the filename of the structure file
    (minus extension), split it by the characters 'cc', and take the NNth element from the
    result. For example if a file was named P01242_LIGxyz_raw.pdb, then $FILE:_:2$ would
    become 'LIGxyz'. 
	&nbsp;&nbsp;&nbsp;$FILE$ can be used to incorporate the full filename
    (without extension). 
	&nbsp;&nbsp;&nbsp;Any characters other than these special tokens
    will be interpreted literally.</dd>

    <dt>Correct ligand from SDF</dt>

    <dd>If checked, a unique mapping between the heavy atoms of the dictionary molecule
    (the SD file above) and the target ligand will be searched for, and then the bond
    orders and charges will be copied over based on this mapping.</dd>
</dl>

<h3>Structure Preparation</h3>

<p>This section defines any preparation steps that should be run on each structure before
further processing is done. For example, missing sidechains could be constructed, or
protonation states adjusted.</p>

<dl>
    <dt>Delete Atoms Far from Site</dt>

    <dd>In checked, atoms far from the Site (specified in the Reference Alignment section)
    will be deleted from the system before writing to the database. This would normally be
    used to delete chains or domains far away from the active site that are not of
    interest, to reduce the final database size.</dd>

    <dt>Wash Ligand</dt>

    <dd>If checked, the protonation and/or tautomerization state of the ligand will be
    modified before performing further analyses.</dd>

    <dt>Use Structure Preparation</dt>

    <dd>If checked, Structure Preparation will be used to correct issues diagnosed in the
    protein, using default fixes. Otherwise, the structure will be used as is.</dd>

    <dt>Use Protonate 3D for Protonation</dt>

    <dd>If checked, Protonate3D will be used on each structure as it is processed, to add
    hydrogens and assign ionization states. Otherwise, hydrogens will be added in default
    extended conformations.</dd>

    <dt>Allow ASN/GLN/HIS &quot;Flips&quot; in Protonate3D</dt>

    <dd>If checked, side chain &quot;flips&quot; will be permitted for terminal amide,
    sulfonamide, and imidazole groups in the Protonate3D calculation (which must also be
    checked).</dd>

    <dt>Delete Water Molecules Farther than 4.5A from Ligand or Receptor</dt>

    <dd>If checked, waters farther than 4.5 Angstroms from either the ligand or the
    receptor will be deleted.</dd>

    <dt>Tether Receptor</dt>

    <dd>If checked, tethers will be created for the receptor atoms.</dd>

    <dt>Tether Receptor Strength</dt>

    <dd>The force constant used for the restraints of receptor atoms.</dd>

    <dt>Tether Receptor Buffer</dt>

    <dd>The size (in Angstroms) of the flat-bottom restraint given to receptor atoms. No
    restraint forces will be applied for distances less than the specified value.</dd>

    <dt>Tether Receptor Hydrogens</dt>

    <dd>If checked, hydrogens in the receptor will be tethered.</dd>

    <dt>Tether Ligand</dt>

    <dd>If checked, tethers will be created for the ligand atoms.</dd>

    <dt>Tether Ligand Strength</dt>

    <dd>The force constant used for the restraints of ligand atoms.</dd>

    <dt>Tether Ligand Buffer</dt>

    <dd>The size (in Angstroms) of the flat-bottom restraint given to ligand atoms. No
    restraint forces will be applied for distances less than the specified value.</dd>

    <dt>Tether Ligand Hydrogens</dt>

    <dd>If checked, hydrogens in the ligand will be tethered.</dd>

    <dt>Tether Solvent</dt>

    <dd>If checked, tethers will be created for the solvent atoms.</dd>

    <dt>Tether Solvent Strength</dt>

    <dd>The force constant used for the restraints of solvent atoms.</dd>

    <dt>Tether Solvent Buffer</dt>

    <dd>The size (in Angstroms) of the flat-bottom restraint given to solvent atoms. No
    restraint forces will be applied for distances less than the specified value.</dd>

    <dt>Tether Solvent Hydrogens</dt>

    <dd>If checked, hydrogens in the solvent will be tethered.</dd>

    <dt>Fix Atoms Far from Ligands</dt>

    <dd>If checked, atoms farther than the Fix Atoms Distance Cutoff distance (below) from
    a ligand will be fixed (in addition to tethers).</dd>

    <dt>Fix Atoms Distance Cutoff</dt>

    <dd>The cutoff (in Angstroms) to which atoms should be fixed, if the Fix Atoms Far from
    Ligands option (above) is selected. A value of 0 will fix all non-ligand atoms.</dd>

    <dt>Hydrogens Close to Ligands will not be Fixed</dt>

    <dd>If checked, hydrogens close to the ligand will be free; otherwise, hydrogens near
    the ligand will be fixed.</dd>

    <dt>Refine the Complex</dt>

    <dd>If checked, a short refinement will be carried out on the structure.</dd>

    <dt>Refinement RMS Gradient Endpoint</dt>

    <dd>Defines the termination condition for the RMS gradient, in kcal/mol/A, if the
    complex is refined.</dd>

    <dt>Merge Waters to Single Chain</dt>

    <dd>If checked, all water residues will be combined into a single chain that contains
    only the waters. This may result in the total number of chains in the system changing.
    Note also that this may result in UID and insertion code no longer being unique within
    the water chain.</dd>

    <dt>SVL Function to Call</dt>

    <dd>The name of an optional SVL function to call to perform further preparation steps,
    such as building missing sidechains. The function must be global and take no arguments,
    and operate on the currently loaded system.</dd>
</dl>

<h3>Crystal Contacts</h3>

<p>This section controls how crystal contacts will be generated when loading each
structure. These are residues that come from symmetry image molecules within the crystal
but are near to the real atoms.</p>

<dl>
    <dt>Near</dt>

    <dd>A MOE selection language string that specifies the atom set near which crystal
    contacts are to be retained. After generating all crystal-symmetry related chains, only
    those residues with at least one atom near to the given selection will be
    retained.</dd>

    <dt>Near Distance</dt>

    <dd>The distance, in Angstroms, from the Near atoms (above) that crystal contacts
    should be retained.</dd>

    <dt>Set</dt>

    <dd>The name of the MOE Collection to create for the crystal contacts. This allows for
    easy identification and selection when later loaded into MOE.</dd>

    <dt>Make Atoms Inert</dt>

    <dd>If checked, crystal contact atoms will be made Inert.</dd>

    <dt>Make Atoms Fixed</dt>

    <dd>If checked, crystal contact atoms will be made Fixed.</dd>
</dl>

<h3>Superpose</h3>

<p>This section defines the details of how the superposition of each structure onto the
reference alignment is performed.</p>

<dl>
    <dt>Reference Selection</dt>

    <dd>The MOE selection expression for the atoms to be used as the basis for
    superposition. This expression is evaluated on the reference structures, and mapped
    through the sequence alignment onto the target structure. If empty, this indicates that
    the structures have already been superposed and should not be moved. In the latter
    case, the SCALE matrix of the PDB file MUST be filled in, and it must provide the
    transformation back to the crystal coordinate frame, fractional space. This is
    especially important if electron densities are to be rotated into the superposed frame.
    Additionally in this latter case, the CRYST1 from the original structure must be
    provided.</dd>

    <dt>Superposition Method</dt>

    <dd>The method used for the superposition. 
	&nbsp;&nbsp;&nbsp;If set to 'alignment', a
    standard structural superposition with correspondences determined by the sequence
    alignment will be performed. 
	&nbsp;&nbsp;&nbsp;For 'pocket', the initial superposition
    from sequence will be used only to identify the pocket (area of interest), followed by
    pocket superposition on the pocket region. 
	&nbsp;&nbsp;&nbsp;For 'structure', a
    sequence-independent structural superposition will be performed instead, trying to find
    an optimal superposition of secondary structure elements between the query and the
    FIRST structure in the reference alignment (any additional reference structure will be
    ignored). 
	&nbsp;&nbsp;&nbsp;Select 'none' if the structures have already been
    superposed or they should not be superposed. In this case, the SCALE matrix in the PDB
    file can be used to provide the transformation back to the crystal frame of reference,
    fractional space. Be sure to also include the CRYST1 from the original structure as
    well in this case.</dd>

    <dt>Enumerate Protein Copies</dt>

    <dd>This option only has an effect when more than one copy of a protein appears (as
    multiple chains) in an input structure file. If checked, a separate row will be created
    in the output database (MDB) for each copy, each being superposed in turn on the
    reference. Otherwise, just a single row will be created in the output MDB, superposed
    on the first chain matching the alignment. In all cases, this column will contain all
    chains from the input structure file, so when multiple rows are produced, they will
    differ only in superposition and not content.</dd>

    <dt>Exclude</dt>

    <dd>If a subset of input structures should be excluded from the superposition process
    (for example, if they have already been superposed manually), provide a comma-separated
    list of their filenames (including full absolute path) here. Wildcards such as * can be
    used here as well. Note that to exclude ALL structures from being superposed, simply
    select 'none' as the Superposition Method above.</dd>
</dl>

<h3>Grids</h3>

<p>This section specifies an arbitrary number of grids to be calculated for each structure,
after superposition. These could be electron densities or any other sorts of grid, such as
water placement probability density. The grids will be linked to the MOE Surface and Maps
panel.</p>

<dl>
    <dt>Filename Pattern</dt>

    <dd>Source file pattern from which to get the structure factors file or CCP4 map file.
    Use * to refer to the filename (without extension) of the structure file &mdash; the
    filename must be defined relative to this.</dd>

    <dt>Map Type</dt>

    <dd>The map type, which applies to MTZ files. For other file types, 'grid' should be
    selected. 2Fo-Fc maps are used to visualize how well the atoms fit into the density,
    with a reduction in model bias. Fo-Fc maps are used to highlight areas of density not
    covered by atoms, as well as atoms in the model that lack density.</dd>

    <!-- TSedit: the <sub> tags need to be fixed from auto-generate. -->
    <!-- <dt>F&lt;sub&gt;obs&lt;/sub&gt; Column</dt> -->
    <dt>F<sub>obs</sub> Column</dt>

    <dd>This is used only for MTZ files, and specifies the column name to use as the Fobs
    (experimental structure factors). If left blank, MOE will try to choose the best column
    available, and certain special column names will be recognized and used if present.
    These are 'FWT' or '2FOFCWT' for 2Fo-Fc maps, and 'DELFWT' or 'FOFCWT' for Fo-Fc maps.
    The corresponding Phi column (below) will be chosen in each case as well. To make use
    of these special columns, simply leave this entry blank, or specify the Fo column name
    here and leave the Fcalc Column name (below) empty. MOE will understand what is meant
    and perform the correct calculation as long as Map Type (above) is set correctly.</dd>

    <!-- TSedit: the <sub> tags need to be fixed from auto-generate. -->
    <!-- <dt>F&lt;sub&gt;calc&lt;/sub&gt; Column</dt> -->
    <dt>F<sub>calc</sub> Column</dt>

    <dd>This is used only for MTZ files, and specifies the column name to use as the Fcalc
    (simulated structure factors). If left blank, MOE will try to choose the best column
    available or simulate the Fcalc from the atomic coordinates.</dd>

    <dt>&Phi; Column</dt>

    <dd>This is used only for MTZ files, and specifies the column name to use as the Phi
    (simulated phases). If left blank, MOE will try to choose the best column available, or
    simulate Phi from the atomic coordinates.</dd>

    <dt>Title</dt>

    <dd>The title to assign to the isosurface that will be created. Use $PDB to substitute
    for the PDB code + chain letter of the structure as part of the title.</dd>

    <dt>Near Distance</dt>

    <dd>Distance, in Angstroms, from the Site atoms (under Reference Alignment) at which
    the grid will be truncated.</dd>

    <dt>&sigma; Level</dt>

    <dd>The sigma level at which to compute an isosurface of the grid. This is the number
    of standard deviations above the mean value of the grid at which the contour is drawn.
    For Electron Density maps, usually this is viewed at a level of 1 for 2Fo-Fc maps, and
    3 for Fo-Fc maps. The allowed ranged is [0.1, 10].</dd>

    <dt>Normalize</dt>

    <dd>If checked, the final grid will be normalized to a mean of zero and a standard
    deviation of one.</dd>

    <dt>Rotate</dt>

    <dd>This applies only to CCP4 grids. If checked, the grid will be rotated using the
    same transformation as that computed during the superposition of the structure on the
    reference. Otherwise, the grid will not be rotated, and it will be assumed to already
    be in the desired frame of reference.</dd>

    <dt>Quality</dt>

    <dd>The level of detail to store with the isosurface. 
	&nbsp;&nbsp;&nbsp;'mesh': store
    only triangles of the isosurface; the sigma level cannot be adjusted in this case.
	&nbsp;&nbsp;&nbsp;'grid': store the entire grid, so that the sigma level can be
    adjusted dynamically within MOE, at the cost of a much larger size for the output MDB
    file.</dd>

    <dt>Include Structure Factors</dt>

    <dd>This applies only to MTZ files. If checked, the structure factors (for the chosen
    columns) will be retained in the output MDB file, allowing for recalculation of the
    electron density, if the need arises.</dd>

    <dt>Color (+)</dt>

    <dd>The color to use for the positive isosurface (for example, the sigma = 1 surface).
    Choose 'Other' to enter a MOE color expression in the textbox.</dd>

    <dt>Color (-)</dt>

    <dd>The color to use for the negative isosurface (for example, the sigma = -1 surface).
    Normally this applies only to Fo-Fc maps. With other map types or grids, negative sigma
    levels are usually not shown or are not of interest. Choose 'Other' to enter a MOE
    color expression in the textbox.</dd>

    <dt>Material</dt>

    <dd>The material for the generated isosurface. Choose from the list, or specify a
    material explicitly in TSGB format.</dd>
</dl>

<h3>Additional Rendering Options</h3>

<p>This section specifies rendering options for indicated sets of atoms. These are applied
after any options that are given in the Rendering Style under MOE Properties above.</p>

<dl>
    <dt>Selection</dt>

    <dd>A MOE selection expression to which the following rendering options are to be
    applied. The expression must be in the MOE Selection language, such as C(=O)C, aCharge
    &lt; -0.5, ligand &gt;&gt; 4.5, Ala23. The selection will be applied to a system
    containing the reference alignment and the structure currently being processed, though
    only atoms of the latter will be retained for the final result.</dd>

    <dt>Atom Style</dt>

    <dd>The atom style for this rendering.</dd>

    <dt>Atom Color</dt>

    <dd>The color scheme to use for atoms. Choose 'Other' to enter a MOE color expression
    in the textbox.</dd>

    <dt>Color Carbons Only</dt>

    <dd>If checked, the color specified in Atom Color (above) will be applied only to
    carbon atoms. Otherwise all atom types will be colored as specified.</dd>

    <dt>Ribbon Style</dt>

    <dd>The ribbon style for this rendering.</dd>

    <dt>Ribbon Color</dt>

    <dd>The color scheme to use for ribbons. Choose 'Other' to enter a MOE color expression
    in the textbox.</dd>

    <dt>Hide Backbone</dt>

    <dd>If checked, backbone atoms in this rendering set will be hidden.</dd>

    <dt>Hide Sidechains</dt>

    <dd>If checked, sidechain atoms in this rendering set will be hidden.</dd>

    <dt>Hide Polar Hydrogens</dt>

    <dd>If checked, polar hydrogens in this rendering set will be hidden.</dd>

    <dt>Hide Nonpolar Hydrogens</dt>

    <dd>If checked, nonpolar hydrogens in this rendering set will be hidden.</dd>

    <dt>Label Style</dt>

    <dd>The label style for this rendering.</dd>
</dl>

<h3>Post-Processing</h3>

<p>This section specifies any post-processing steps to be performed after all the
structures in the project have been processed and added to the MDB file. This could be used
for clustering, for example, or any other activity which must be postponed until all of the
structures of the project have been processed.</p>

<dl>
    <dt>SVL Function to Call</dt>

    <dd>The name of an SVL function to call after the file processing has completed and the
    final MDB has been created. The function must be global and take one argument &mdash;
    the MDB key. Note that the XML configuration file is stored within the database
    environment, making it accessible if needed. This could be used to perform calculations
    on all the structures in the project together, such as clustering.</dd>

	<!--The following was added manually; MoeDocs issue #40-->
    <dt>Function String Argument</dt>

    <dd>The value of an optional string argument to be passed to the
      post-processing function.  If more complex arguments are required,
      use this to provide a filename in which the arguments are given,
      and have the post-processing function read the arguments from this
      file.</dd> 
</dl>

<h3>Other MOE Options</h3>

<dl>
    <dt>Include All Models</dt>

    <dd>If checked, all models will be loaded for each structure; otherwise only the first
    model will be loaded.</dd>

    <dt>Include Alternate Locations</dt>

    <dd>If checked, atoms with alternate locations (those atoms with an occupancy less than
    one) will be preserved for each structure; otherwise only the highest occupancy atom is
    kept and the others are discarded. Alternate locations for atoms are set to be Inert
    and are placed in separate, adjacent residues.</dd>

    <dt>Rendering Style</dt>

    <dd>Controls how the rendering in the output database should be handled. This includes
    ribbon display and coloring, atom visibility and coloring, residue coloring in the
    sequence editor, and so on. 'SiteView' uses the standard MOE SiteView to set the
    rendering in each input structure, and the camera position will be oriented to match
    the initial view in the reference alignment. If set to 'Inherit', the closest structure
    in the reference to each input structure will be used as the source for copying
    rendering styles. Only rendering of the protein chain is affected. If set to
    'Preserve', the existing rendering styles, coloring, and view represented in the input
    files will not be modified. This can be used to provide a custom look and style, for
    example by providing a .moe file corresponding to some or all of the input structure
    files, with the structure displayed in the desired fashion. Also see the Additional
    Rendering Options below to specify with more fine-grained control how the rendering
    should appear. These latter options are applied after the setting chosen here.</dd>

    <dt>Split Character for PDB Code</dt>

    <dd>Specifies the characters that are used to split filenames for extracting the PDB
    code. This field is used in conjunction with Split Pattern; see there for more details.
    Leave this blank if you wish to use the PDB code within the PDB files and not extract
    it from the filenames.</dd>

    <dt>Split Pattern for PDB Code</dt>

    <dd>Used in conjunction with Split Character to specify the value of the PDB code from
    the filename. The first element has index 1. Any $n used within this string, where n is
    a digit, will be replaced by the nth element extracted via the Split Character. Other
    characters are taken literally. For example, if a file was named kinase_P01242_raw.pdb,
    Split Character was '_' and Split Pattern was '$2-$3', then the PDB code would be set
    to 'P01242-raw'. Note the file extension is always ignored. If this field is empty, the
    PDB code will be taken from its standard location inside the PDB file.</dd>
</dl>

<h2>ID</h2>

<p>The ID field is used to create a unique identifier on each row of the resulting output
database.</p>

<dl>
    <dt>ID Type</dt>

    <dd>The type of ID to generate: 
	&nbsp;&nbsp;&nbsp;'sequence' generates successive
    integers in each row of the MDB file. 
	&nbsp;&nbsp;&nbsp;'uuid' generates a UUID
    (Universally Unique Identifier) for each row of the database, which is guaranteed to be
    unique every time the database is regenerated. It can therefore serve as a unique key
    when combining multiple databases. 
	&nbsp;&nbsp;&nbsp;'psilo' will record the PSILO
    structure_id of the record that the structure information was taken from, when the
    database is generated using PSILO Families.</dd>

    <dt>Sequence Base</dt>

    <dd>The base (starting) sequence number for the 'sequence' ID Type (above).</dd>
</dl>

<h2>Subsystem</h2>

<p>The Subsystem field is used to extract atoms of interest from another field that
contains some or all of the structure. For example, it can be used to extract only the
ligand, or only the binding pocket, into a separate field.</p>

<h3>Atoms</h3>

<p>This section specifies the atoms that will comprise the subsystem. This could be
receptors atoms, ligand atoms, solvent atoms, or any other atom group of interest.</p>

<dl>
    <dt>Selection</dt>

    <dd>A selection expression, in the MOE Selection Language, that specifies the atoms of
    interest. For example: C(=O)C, aCharge &lt; -0.5, ligand &gt;&gt; 4.5, Ala23. The
    selection will be applied to a system containing the reference alignment and the
    contents of the source field for the structure currently being processed, though only
    atoms of the latter will be retained for the final result.</dd>

    <dt>Include Graphics</dt>

    <dd>If checked, graphics objects such as surfaces will be included in the extracted
    subsystem. These can only be stored in the MDB file if the Field Type (specified above)
    is set to 'moe'.</dd>
</dl>

<h3>Other Subsystem Options</h3>

<dl>
    <dt>Source Field</dt>

    <dd>The source field for the Subsystem field. The Selection expression (specified
    below) is applied to the atoms in this field to define the Subsystem field. The source
    field must be created before the Subsystem field which depends on it.</dd>
</dl>

<h2>Loop Library</h2>

<p>The Loop Library field is a structure field that can be used with the Loop Modeler,
during Homology Modeling for example. This involves keeping only the backbone atoms of the
protein chains, and ordering the atoms in a specific way.</p>

<dl>
    <dt>Selection</dt>

    <dd>A selection expression, in the MOE Selection Language, that specifies the atoms of
    interest. For example: C(=O)C, aCharge &lt; -0.5, ligand &gt;&gt; 4.5, Ala23. The
    selection will be applied to a system containing the reference alignment and the
    contents of the source field for the structure currently being processed, though only
    atoms of the latter will be retained for the final result.</dd>

    <dt>Source Field</dt>

    <dd>The source field for the Loop Library field. The Selection expression (specified
    above) is applied to the atoms in this field to define the Loop Library field. The
    source field must be created before the Loop Library field which depends on it.</dd>
</dl>

<h2>Ligand</h2>

<p>The Ligand field is intended to provide a sketch of the ligand. It is not guaranteed to
be in any particular frame of reference and therefore should not be combined with any other
molecule fields to carry out a calculation. To extract the ligand in the same frame of
reference as another field, use the Subsystem field.</p>

<dl>
    <dt>Depict</dt>

    <dd>If checked, the ligand will be depicted. Its coordinates will be modified to create
    a displayable 2D sketch.</dd>

    <dt>Source</dt>

    <dd>The source for the ligand structure. If 'sdf' is selected, the ligand specified in
    the Ligand Details section of the MOE field will be used to generate the ligand sketch.
    If 'pdb' is selected, the actual ATOM records in the structure file will be used
    instead. In the latter case, if any atoms are missing from the structure file, they
    will be missing from the ligand sketch as well.</dd>

    <dt>Copy HETNAM from MOE field</dt>

    <dd>If checked, the ligand molecule name will be copied from the Ligand Name specified
    in the Ligand Details section of the MOE field.</dd>
</dl>

<h2>PDB Record</h2>

<p>PDB Record fields extract information from the supplied PDB (or mmCIF) file. These are
parsed versions of the information of the relevant entities in the PDB file.</p>

<dl>
    <dt>PDB Card</dt>

    <dd>The PDB card to extract data from. For 'Remarks', specify the remark number in the
    text box.</dd>

    <dt>Change Case to</dt>

    <dd>Since PDB files are traditionally all uppercase, use this option to specify that
    the case should be modified, or left as-is, when extracting the contents of the PDB
    card.</dd>

    <dt>Group By</dt>

    <dd>If checked, the property can be used for dynamic grouping in the MOE system
    manager. Structures can then be regrouped on-the-fly by the various properties assigned
    with this option.</dd>
</dl>

<h2>Calculated Property</h2>

<p>Calculated Property fields are descriptors of various sorts that are calculated from a
protein or small molecule. The source of the atoms must be specified. Color coding can be
specified based on the values in this field.</p>

<h3>Arguments</h3>

<p>Specify the arguments that will be passed to the Service being called (specified above).
The exact nature of this section will vary depending on the type of service and the
arguments it expects.</p>

<dl>
    <dt>Name</dt>

    <dd>For MOE/web, SOAP function calls, or REST/HTTP, provide the name of the
    argument.</dd>

    <dt>Value</dt>

    <dd>The MDB field that is used to retrieve the argument value (to obtain the atoms
    from, or to perform the calculation on, for example). MDB fields can be of type: moe,
    molecule, or string, and must have been defined before the Calculated Property field.
    This Value can also be a literal string value, indicated by enclosing it in double
    quotes (&quot;).</dd>

    <dt>Format</dt>

    <dd>The file format in which the argument is sent to the specified service. Allowed
    formats are: PDB file, MOE file, SDF, SMILES, FASTA, or string. The argument Value
    (indicated above) will be converted into the specified format if possible.</dd>
</dl>

<h3>Other Calculated Property Options</h3>

<dl>
    <dt>Service</dt>

    <dd>Describes the type of service that will be used to calculate the descriptor, and
    also affects how the Source Property field (below) is interpreted.
    
	&nbsp;&nbsp;&nbsp;'MOE Descriptor': the value is a standard MOE descriptor, the name of
    which is specified in Source Property. 
	&nbsp;&nbsp;&nbsp;'SVL Function': the value is
    returned by an SVL function, named in Source Property and which takes arguments as
    defined in the Arguments section (below). The actual arguments will be atom keys,
    except for FASTA arguments which will be passed as a string of amino acid letters.
	&nbsp;&nbsp;&nbsp;'MOE Fingerprint': the value is the name of a standard MOE
    Fingerprint definition, which is specified in Source Property.
	&nbsp;&nbsp;&nbsp;'MOE/web Function': the value is a function provided by MOE/web,
    which is specified in Source Property. 
	&nbsp;&nbsp;&nbsp;'WSDL URL': the value is
    returned by a SOAP function. Specify the URL of the WSDL for the SOAP server in the
    textbox here, and the function name in Source Property. 
	&nbsp;&nbsp;&nbsp;'REST/HTTP URL': the value is returned by an 
    HTTP call (which may or may not use REST). In the
    textbox here, specify the base URL to call, including a '?' if appropriate. Specify the
    arguments and their values in the Arguments section (below). Source Property is not
    used in this case.</dd>

    <dt>Source Property</dt>

    <dd>The name of the MOE descriptor or MOE fingerprint to be computed, or the name of
    the SVL function, MOE/web function, or SOAP function to call, depending on the setting
    of Service (above). Does not apply to HTTP calls.</dd>

    <dt>Result Format</dt>

    <dd>The format in which the service data is returned. The service MUST return one of
    these formats, or it cannot be used. When Service (above) is set to 'MOE Descriptor',
    'SVL Function', or 'MOE Fingerprint', this should be set to 'SVL vector' so that the
    return value in SVL will be used directly. When Service is set as a 'WSDL URL' or
    'MOE/web Function', this must be set to 'SOAP/XML'. When Service is set as an 'HTTP
    URL', this can be set to 'SD file' or 'Delimited text file', with Result Field (below)
    specifying where to find the desired value within these files.</dd>

    <dt>Result Field</dt>

    <dd>This applies only when Result Format (above) is 'SD file', 'SOAP/XML', or
    'Delimited text file'. In the former, it is used to specify the field in the SD file
    containing the desired result. For SOAP/XML calls, it is the Xpath in the XML where the
    value will be found. For text files, it is either the column number of a tab-delimited
    text file to be used for the result (if the file contains multiple rows, it will be
    treated as a vector of return values), or the column heading (as a string) of the first
    row of the returned file. By default, text files are treated using any whitespace as a
    delimiter. To explicitly specify the delimiter, append :[wcmt][q] to the end of the
    Result Field, where the first letter indicates the delimiter used, with w=whitespace,
    c=comma, m=semicolon, or t=tab; and q indicates that items are enclosed in quotes
    (&quot;). For example, 3:tq indicates to fetch the result from column 3, splitting
    columns by tabs and assuming items are quoted.</dd>

    <dt>Group By</dt>

    <dd>If checked, the property can be used for dynamic grouping in the MOE system
    manager. Structures can then be regrouped on-the-fly by the various properties assigned
    with this option.</dd>
</dl>

<h2>Experimental Property</h2>

<p>Experimental Property fields are values associated with the structure or ligand, which
are obtained from the input data, rather than computed. Examples might include melting
temperature, isoelectric point, or IC50.</p>

<dl>
    <dt>Source Filename Pattern</dt>

    <dd>The source file pattern from which to get the property. Use * to refer to the
    filename (without extension) of the structure file &mdash; the file name of the
    property file must be defined relative to this. For each structure, the corresponding
    property file will be parsed and the desired property extracted. The Field in Source
    File (below) defines exactly which property should be extracted. If the given file is
    an SDF file, the property must be given in a specified field, such as &lt;IC50&gt;. If
    a PROP file is given, this should be a plain text file with one property per line, of
    the form 'field=value', and again the desired field should be specified by Field in
    Source File.</dd>

    <dt>Field in Source File</dt>

    <dd>The field in the source SDF or PROP file that contains the experimental property to
    be extracted.</dd>

    <dt>Group By</dt>

    <dd>If checked, the property can be used for dynamic grouping in the MOE system
    manager. Structures can then be regrouped on-the-fly by the various properties assigned
    with this option.</dd>
</dl>

<h2>Alignment Quality</h2>

<p>The Alignment Quality field stores one of various possible measures that indicate the
quality of the superposition onto the reference structure(s) &mdash; for example, the
RMSD.</p>

<dl>
    <dt>Quality Measure</dt>

    <dd>The quality measure to report, for the superposition of the subject on to the
    reference structure(s). For 'RMSD', large values may indicate that the subject did not
    align successfully to the reference. For 'Pocket/Motif Score', an energy-like score is
    returned depending on which Superposition Method is chosen in the Superposition section
    of the MOE field. If a Pocket superposition was carried out, the score indicates how
    well the corresponding pocket residues matched up, with a score of 26 being the
    absolute minimum cutoff, and a score of 40 or higher indicating a strong hit. If a
    Structure superposition was chosen, then the score gives an indication of how well the
    secondary structure elements of the subject superpose on the reference, with higher
    scores being better. Other superposition methods do not have scores associated with
    them and will result in a score of zero.</dd>
</dl>

<h2>PLIF</h2>

<p>The PLIF field generates a protein-ligand interaction fingerprint. 
<!-- TSedit: add the following lines as a supplement. -->
For more information about how to use PLIF correctly, see 
<a href="../proteins/moeproject.htm#UsingPlif">Using PLIF</a> in the MOE Project Configuration File.
</p>

<dl>
    <dt>Include Metal and Arene bits</dt>

    <dd>If checked, PLIF will include bits for metal ligation and arene interactions.
    Earlier versions of PLIF did not include these interaction types.</dd>
</dl>

<a name="SeeAlso" />
<h2>See Also</h2>
<a href="../proteins/moeproject.htm">MOE Project</a><br />
<a href="../moe/moeweb_intro.htm#Configure_Project_Database">MOE/web: Configure Project Database</a>

<noscript>
	<hr noshade="noshade" />
	<font size="2"> Copyright &copy; 1997&ndash;2015
	<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a> </font>
</noscript>
<div id="MoeFooter"></div>
</body>
</html>
