<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!	probuild.htm		Protein Builder and Rotamer Explorer
!!-->

<!--
!!    MOE Online Manuals
!!    COPYRIGHT (C) 1997-2015
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

<script type="text/javascript" 
src="../include/jsincludes_moe.js"></script>

<link rel="stylesheet" type="text/css"
href="../include/manstyle.css" />


<meta panel Protein Builder>
<meta panel Rotamer Explorer>

<meta keywords amino acid />
<meta keywords non-natural amino acid, non-standard amino-acid />
<meta keywords modified amino-acid />
<meta keywords rotamer, rotamer explorer, protein builder, mutate />
<meta keywords rename, renumber, mutate, append, prepend />
<meta keywords create sequence />
<meta keywords split, join, delete residue, disulfide />
<meta keywords rotamer library />
<meta keywords sidechain, side-chain, peptide, peptide bond />
<meta keywords repack, repack sidechain />

<title>Protein Builder and Rotamer Explorer</title>

</head>

<body>
<div id="useMathJax"></div>
<div id="MoeHeader"></div>
<noscript>
	<span class="warning">Warning: JavaScript is disabled. This page will not display correctly.</span>
	<h1 class="title">MOE Documentation</h1>
	<hr noshade="noshade" />
	<style>.LaTeX {color: #C08080;}</style>
</noscript>

<p>
This chapter describes how to create and edit proteins using the Protein
Builder and Rotamer Explorer.
</p>

<ul>
<li><a href="#CreatingProteins">Protein Builder</a>
    <ul>
    <li><a href="#Selection"	>Residue Selection</a></li>
    <li><a href="#Build"	>Build a New Molecule or Sequence</a></li>
    <li><a href="#Mutate"	>Mutate Residue</a></li>
    <li><a href="#AppendPrepend">Append/Prepend Residue</a></li>
    <li><a href="#AcetylateAmidate">Acetylate/Amidate Chain Terminus</a></li>
    <li><a href="#AppendPrepend">Append/Prepend Residue</a></li>
    <li><a href="#Protonation"	>Set Residue Protonation State</a></li>
    <li><a href="#Repack"	>Repack Residue Sidechain</a></li>
    <li><a href="#AppendPrepend">Append/Prepend Residue</a></li>
    <li><a href="#BreakSidechainBond">Break Sidechain-Sidechain Bond</a></li>
    <li><a href="#Join"		>Join Residues</a></li>
    <li><a href="#Split"	>Split Residues</a></li>
    <li><a href="#Rename"	>Rename Residue</a></li>
    <li><a href="#Renumber"	>Renumber Residue</a></li>
    <li><a href="#Minimize"	>Minimize Residue</a></li>
    <li><a href="#Delete"	>Delete Residue</a></li>
    <li><a href="#ProteinBuilderSettings">Protein Builder Settings</a></li>
    </ul>
</li>
<li><a href="#RotamerExplorer">Rotamer Explorer</a></li>
    <ul>
    <li><a href="#ConformationGeneration">Conformation Generation</a></li>
    <li><a href="#Scoring"	>Scoring</a></li>
    <li><a href="#StartExplorer">Explore a Residue</a></li>
    <li><a href="#Results"	>Reported Information</a></li>
    <li><a href="#Visualize"	>Visualizing the Results</a></li>
    <li><a href="#MultiTarget"	>Explore Multiple Targets</a></li>
    <li><a href="#WriteDatabase">Save Rotamer Results to Database</a></li>
    <li><a href="#RotamerExplorerSettings">Rotamer Explorer Settings</a></li>
    </ul>
</li>
<li><a href="#PTM">Post-Translational Modifications (PTM)</a></li>
<li><a href="#Codes">Residue Code Reference</a></li>
<li><a href="#SVL">SVL Commands</a></li>
<li><a href="#SeeAlso">See Also</a></li>
</ul>

<a name="CreatingProteins" />
<h2>Protein Builder</h2>

<table><tr><td>

<div class="floatright">
<img src="probuild/rotexpl.png" alt="Protein Builder Panel" />
</div>

The MOE Protein Builder is used to build and modify polypeptides, primarily
in 3D, but it can also work through the Sequence Editor with sequence-only
chains. It provides tools to add, remove, mutate, and optimize peptide
residues; or split and join chains of residues. It also has a built-in
<a href="#RotamerExplorer">Rotamer Explorer</a> to explore and optimize
sidechain conformations.

<p>
To open the Protein Builder, choose
<b>MOE | Edit | Build | Protein</b> or
<b>MOE or SE | Protein | Protein Builder</b>.
</p>

<p>
The Protein Builder is a standalone panel, but can also be attached
to the MOE window (like the <a href="../moe/molsystems/build.htm">
Atom Builder</a>) and to the Sequence Editor as well.
</p>

<p>
The Protein Builder operates on
<a href="#Selection">atom/residue selections</a>.
The actions and functionality provided by the Builder depend on the
current selection. A residue is considered selected by the Protein Builder
when any of its atoms are selected (similar to the <b>MOE | Select |
Synchronize</b> option).
</p>

<p>
The following table lists actions and functionality provided by the Protein
Builder panel, as well as how to use them:
</p>

</td></tr></table>

<p>
<table class="titlebar">
<tr>
    <th>Action</th>
    <th>How to do it with the Protein Builder</th>
</tr>
<tr>
    <td valign="top">
	<a href="#Build">Build</a> a new molecule or sequence.
    </td>
    <td valign="top">
	<ul>
	    <li>Ensure no atoms or residues are selected.</li>
	    <li>Press one of the residue buttons or enter a sequence
	    string at the top of the panel and press <b>Build</b>.</li>
	    <li>To build non-natural amino acids, select the desired amino
	    acid variant under the <b>More</b> button.
	</ul>
    </td>
</tr>

<tr>
    <td valign="top">
	<a href="#Mutate">Mutate</a> residues.
    </td>
    <td valign="top">
	<ul>
	    <li>Select one or more residues in the Sequence Editor or atoms of
	    the residues in the MOE main window.</li>
	    <li>Press the residue button of the standard amino acid type to mutate
	    the selected residues to.</li>
	    <li>To mutate to a non-natural amino acid, select the desired amino
	    acid under the <b>More</b> button.
	</ul>
    </td>
</tr>

<tr>
    <td valign="top">
	<a href="#AppendPrepend">Append or prepend</a> residues.
    </td>
    <td valign="top">
	<ul>
	<li>Select the tail or head residue of the chain, which should be
	appended or prepended in the Sequence Editor, or atoms of
	that residue in the MOE main window.</li>
	<li>Press the residue button of the amino acid type to append
	or prepend to the selected terminal residue. Multiple residues
	can be added by entering their single-letter sequence into the
	sequence popup of the <b>AA</b> button.
	</li>
	</ul>
	Or
	<ul>
	<li>Select any residues in the connected chain of residues,
	which should be appended or prepended.</li>
	<li>Then, select the <b>Append</b> or <b>Prepend</b> mode
	at the top of the Protein Builder panel.</li>
	<li>Press the residue button of the amino acid type to append
	or prepend.</li>
	</ul>
    </td>
</tr>

<tr>
    <td valign="top">
	<a href="#AcetylateAmidate">Acetylate or amidate</a>
	chain termini.
    </td>
    <td valign="top">
	<ul>
	<li>Select one or more residues in the connected chain of residues
	whose termini should be acetylated or amidated.</li>
	<li>Press - to select - the <b>Ace</b> (acetylate) or <b>Nme</b>
	(amidate) toggle buttons.</li>
	<li>To remove an acetyl or amide terminus group,
	press and release the <b>Ace</b> or <b>Nme</b> button.</li>
	</ul>
    </td>
</tr>

<tr>
    <td valign="top">
	Set residue <a href="#Protonation">protonation state</a>.
    </td>
    <td valign="top">
	<ul>
	<li>Select one residue or its atoms that can have different
	protonation states (e.g. His, Arg, etc.).</li>
	<li>A button bar of the protonation states available for selection
	appears under the block of amino acids.</li>
	<li>Press the desired protonation state button.</li>
	</ul>
    </td>
</tr>

<tr>
    <td valign="top">
	<a href="#Repack">Repack</a> residue sidechain.
    </td>
    <td valign="top">
	<ul>
	<li>Select the residue whose sidechain should be repacked.</li>
	<li>Press the <b>Repack</b> button.</li>
	</ul>
    </td>
</tr>

<tr>
    <td valign="top">
	Explore sidechain <a href="#RotamerExplorer">rotamers</a>.
    </td>
    <td valign="top">
	<ul>
	<li>Select one residue or its atoms whose sidechain rotamers
	should be explored and/or mutated.</li>
	<li>Press the <b>Rotamers</b> button.</li>
	</ul>
    </td>
</tr>

<tr>
    <td valign="top">
	Create <a href="#Disulfide">disulfide</a> bond.
    </td>
    <td valign="top">
	<ul>
	<li>Select the cysteine residues which should form a disulfide
	bond.</li>
	<li>Press the <b>S-S</b> button. If the geometric criteria for a
	disulfide bond between the selected residues are satisfied, the disulfide bond is
	created.</li>
	</ul>
    </td>
</tr>

<tr>
    <td valign="top">
	<a href="#BreakSidechainBond">Break sidechain</a>
	bond (e.g. disulfide).
    </td>
    <td valign="top">
	<ul>
	<li>Select the residues that are connected by a sidechain-sidechain
	bond.</li>
	<li>Press the <b>R-<font color="#ff0000">X</font>-R</b> button.</li>
	</ul>
    </td>
</tr>

<tr>
    <td valign="top">
	<a href="#Join">Join</a> or <a href="#Split">split</a> residues/chains.
    </td>
    <td valign="top">
	<ul>
	<li>Select the two residues that should be joined by a
	peptide bond, or whose peptide bond should be split.</li>
	<li>Press the <b>Join</b> or <b>Split</b> button.</li>
	</ul>
    </td>
</tr>

<tr>
    <td valign="top">
	<a href="#Rename">Rename</a> single residue.
    </td>
    <td valign="top">
	<ul>
	<li>Select the residue whose name, UID number or insertion code
	should be changed.</li>
	<li>Edit name, number or insertion code in text fields below the
	<b>Delete</b> and <b>Minimize</b> buttons. Apply each change with the
	<tt>Return</tt> key.</li>
	</ul>
    </td>
</tr>

<tr>
    <td valign="top">
	<a href="#Renumber">Renumber</a> residues.
    </td>
    <td valign="top">
	<ul>
	<li>Select residues or range of residues to be renumbered.</li>
	<li>Press the 
	<img src="probuild/probuild_renum.png" style="vertical-align:middle" />
	button.</li>
	</ul>
    </td>
</tr>

<tr>
    <td valign="top">
	<a href="#Minimize">Minimize</a> residues.
    </td>
    <td valign="top">
	<ul>
	<li>Select the residues that should be minimized after
	modification.</li>
	<li>Press the <b>Minimize</b> button to select an
	appropriate minimization protocol.</li>
	</ul>
    </td>
</tr>

<tr>
    <td valign="top">
	<a href="#Delete">Delete</a> residues.
    </td>
    <td valign="top">
	<ul>
	<li>Select the residues to be deleted.</li>
	<li>Press the <b>Delete</b> button.</li>
	</ul>
    </td>
</tr>

</table>
</p>

<!-- ======================== SELECTION =========================== -->

<a name="Selection" />
<h4>Residue Selection</h4>

<p>
The Protein Builder operates on atom/residue selections.
The actions and functionality provided by the Builder depend on the
current selection. Some actions are only available if a single residue is
selected, while others require a selection of two or more residues.
A residue is considered selected by the Protein Builder
when any of its atoms are selected (similar to the <b>MOE | Select |
Synchronize</b> option).
</p>
<p>
The current selection is also represented in the Protein Builder
panel by coloring the buttons of amino acids that are in the current
selection. Different colors convey additional information about the
selected residues:

<center>
<table class="titlebar">
<tr>
    <th>Color</th>
    <th>Description</th>
</tr>
<tr>
    <td><font color="0000FF"><b>Blue</b></font></td>
    <td>
	The structure of the selected residue matches its residue name.
    </td>
<tr>
<tr>
    <td><font color="FF0000"><b>Red</b></font></td>
    <td>
	The structure of the selected residue does not match its name.
    </td>
<tr>
<tr>
    <td><font color="00CCCC"><b>Cyan</b></font></td>
    <td>
	Selected residue is a sequence-only residue without atoms.
    </td>
<tr>
<tr>
    <td><font color="CC00CC"><b>Magenta</b></font></td>
    <td>
	The current selection includes multiple of the above states.
    </td>
<tr>
</table>
</center>

</p>

<!-- ========================== BUILD =========================== -->

<a name="Build" />
<h4>Build a New Molecule or Sequence</h4>

</p>
<div class="floatright">
<img src="probuild/probuild_build.png" alt="Build From Sequence"/>
</div>
Building a peptide molecule with the Protein Builder is possible by
specifying the sequence with the residue buttons one residue at a time,
or by entering a sequence string and building it in one operation. To enable
the Build mode of the Protein Builder, ensure that no atom or
residue is selected. The top of the Protein Builder will then show
a text field where a sequence string can be entered and a <b>Build</b>
button next to it.
</p>

<p>
The sequence string text field accepts the single letter
<a href="#Codes">codes</a> of
the 20 standard amino acids and, in parentheses, the three letter codes for non-natural amino acids.
Hit <tt>Return</tt> or press <b>Build</b>
to create a new molecule with the specified sequence. A panel will
pop up, in which the chain and tag names of the new
molecule chain can be specified, as well as, in the case of a sequence-only
chain:
</p>

<p>
<table width="80%" cellpadding="5">
<tr>
    <td valign="top" rowspan="4">
	<img src="probuild/probuild_create.png" alt="Create New Molecule"/>
    </td>
    <td rowspan="4"> &nbsp; </td>
    <td valign="top"><b>Name</b></td>
    <td valign="top">
	Enter the chain name of the new molecule to be built.
    </td>
</tr>
<tr>
    <td valign="top"><b>Tag</b></td>
    <td valign="top">
	Enter the tag name of the new molecule to be built.
    </td>
</tr>
<tr>
    <td valign="top"><b>Build Sequence Only Chain</b></td>
    <td valign="top">
	If on, a sequence only chain without atoms will be built.
    </td>
</tr>
<tr>
    <td valign="top"><b>OK</b></td>
    <td valign="top">
	Build molecule with specified options.
    </td>
</tr>
</table>
</p>

<p>
Alternatively to specifying a sequence string, it is also possible to build a
peptide one residue at a time, by pressing the standard amino acid buttons
in the Protein Builder main window, or non-natural amino acids in
the <b>More</b> popup panel. When creating the first residue, the
<b>Create New Molecule</b> panel will open to set the build options.
After the creation of the first residue in the new molecule chain, the
Builder will switch to the <a href="#AppendPrepend">append mode</a> and
each residue button click will add the specified amino acid.
</p>

<p>
<a name="Geometry" />

<table width="100%" cellpadding="5">
<tr>
    <td valign="top">
	<img src="probuild/probuild_geom.png" alt="Current Build Geometry"/>
	<br />
	<br />
	<img src="probuild/probuild_geometry.png"
	 alt="Backbone Geometry Panel"/>
    </td>

    <td valign="top">
	When building amino acid residues with atoms,
	the backbone geometry (i.e. secondary structure of the residues) can be specified
	to be added with the <b>Geometry</b> button, which opens a panel
	with the backbone conformation options. The currently selected
	conformation for building is shown right next to the button.

	<p>
	The panel provides presets for several common backbone conformations
	(see table). Custom <b>phi</b> and <b>psi</b> angles can be specified
	when the <tt>Custom</tt> radio option is selected. 
	If the <b>Cis Conformation</b> option is turned on,
	the omega angle is set to 0 degrees.
	The <b>Left-Handed Helices</b> option forces left-handed geometry,
	negating the phi and psi angles of the standard helix conformations.
	Non-helix geometries are not affected.
	</p>

	<p>
	<span class="tip"><b>Tip!</b></span>
	To set the backbone geometry of an already-built sequence of residues,
	select the residues in question, switch to the <b>Append</b> or
	<b>Prepend</b> mode of the Protein Builder, so that the <b>Geometry</b>
	panel is available, and then select the desired geometry and press the
	<b>Apply to Selection</b> button.
	</p>
    </td>

    <td valign="top">
	<table class="titlebar">
	    <tr>
		<th>Label</th>
		<th>Phi angle</th>
		<th>Psi angle</th>
	    </tr>
	    <tr><td>Helix</td><td>-65</td><td>-39</td></tr>
	    <tr><td>3-10-Helix</td><td>-50</td><td>-25</td></tr>
	    <tr><td>pi-Helix</td><td>-45</td><td>-80</td></tr>
	    <tr><td>Strand</td><td>-120</td><td>113</td></tr>
	    <tr><td>Anti-Strand</td><td>-140</td><td>135</td></tr>
	    <tr><td>Poly Proline</td><td>-65</td><td>140</td></tr>
	    <tr><td>Extended</td><td>180</td><td>180</td></tr>
	</table>
	</center>
    </td>
</tr>
</table>

</p>

<p>
For the sidechains (chi angles), the lowest strain conformation
in the rotamer library is used by default when building new residues.
Alternatively, the <a href="#Repack"><b>Auto Repack</b></a> option can
be enabled to repack the sidechains while building.

<b>Note!</b> Repacking can be slow if multiple sidechains are repacked
at once or when operating on sidechains with many rotamer conformations
in the rotamer library.
</p>

<!-- ========================== MUTATE =========================== -->

<a name="Mutate" />
<h4>Mutate Residue</h4>

<p>
<div class="floatleft">
<img src="probuild/probuild_mutate.png" alt="Mutate Residue"/>
</div>

Selected residues are mutated easily by pressing the residue button
of the amino acid to which to mutate the sidechain. A depiction
of the amino acid associated to each button is shown when hovering with
the mouse over the button in question. If an amino acid sidechain has
multiple possible protonation states, the most common state is picked
during mutation. To change it, use the <a href="#Protonation" />protonation
state</a> button bar.
</p>

<p>
By default, the Protein Builder tries to preserve the chi angles of
the previous sidechain while mutating. To optimize the new sidechain 
conformation, use the <a href="#Minimize"><b>Minimize</b></a>
or <a href="#Repack"><b>Repack</b></a> functionality.
</p>


<p>
<b>Note:</b> When selecting the first or last residue of a residue
block, the Protein Builder assumes that the block will be appended or prepended
and switches automatically to the <b>Append/Prepend</b>
mode. Press <b>Mutate</b> if a mutation of the selected residue is desired
instead.
</p>


<h5>Non-Natural Amino Acids</h5>

<p>
To mutate a residue to a non-natural amino acid, open the Non-Natural
Amino Acid panel by clicking the <b>More</b> button:

<table>
<tr>
    <td valign="top">
	<img src="probuild/probuild_more.png" alt="Non-natural Amino Acids"/>
    </td>
    <td> &nbsp; </td>

    <td valign="top">
	The panel provides a selection of common non-natural amino acids
	(see on the left), which can be extended with custom rotamer
	libraries (see <a href="buildrotlib.htm">Build Rotamer Library</a>).
	To assist the user in finding the desired amino acid among the three-letter
	code buttons, the bubblehelp of the buttons shows a 2D depiction
	of the amino acid and the given name. One can also filter the
	list of non-natural amino acid buttons by a <b>Filter</b> string
	and a <b>Class</b> pull-down menu. If the rotamer library was
	extended with custom non-natural amino acids, only one page at a time
	is shown in the panel (see on the right). To navigate the pages,
	use the selection list and browse buttons below the Filter line.
    </td>

    <td valign="top">
	<img src="probuild/probuild_morex.png" alt="Non-natural Amino Acids"/>
    </td>
</tr>
</table>
</p>

<!-- ======================= APPEND/PREPEND ======================== -->

<a name="AppendPrepend" />
<h4>Append/Prepend Residues</h4>

<p>
<div class="floatright">
<img src="probuild/probuild_append.png" alt="Append Residue"/>
<br />
<br />
<img src="probuild/probuild_prepend.png" alt="Prepend Residue"/>
<br />
<br />
<img src="probuild/probuild_aa.png" alt="Append Multiple Residue"/>
</div>

To append or prepend amino acids with the residue buttons
in the Protein Builder, first specify the residue
onto which to append or prepend additional residues.
This can either be done by selecting the terminal residue of a
chain or residue block directly or by selecting any residue in that
block. Then, the Protein Builder is switched to the <b>Append</b> or
<b>Prepend</b> mode. If a terminal residue is selected, the
<b>Append</b> or <b>Prepend</b> mode is enabled automatically,
otherwise it must be set manually at the top of the
Protein Builder.
</p>

<p>
Appending or prepending residues is done one at a time with
the residue buttons in the Protein Builder (adding standard amino acids),
or the <b>More</b> popup panel (adding non-natural amino acids).
When the new residue is added, it is automatically selected, and the
Builder stays in Append/Prepend mode.
To add multiple residues together one can enter a single letter sequence
string in the sequence popup of the <b>AA</b> button. Non-natural amino
acids can be specified by their three letter code in parentheses.
Similar to the building of new molecules,
the backbone geometry of the appended/prepended residues can also be selected in the
<a href="#Geometry"><b>Geometry</b></a> popup panel.
</p>

<p>
If the selected residue to be appended/prepended is an empty residue
(sequence-only, no atoms), the added residue will be empty as well.
</p>

<p>
<span class="tip"><b>Tip!</b></span>
Alternatively, it is also possible to <a href="#Build">build</a> a sequence
of residues as a new molecule and then attach it with the
<a href="#Join"><b>Join</b></a> button.
</p>

<!-- ========================== ACE/NME =========================== -->

<a name="AcetylateAmidate" />
<h4>Acetylate/Amidate Chain Terminus</h4>

To prepend or append an acetyl or amide group to the termini of a chain
or residue block, select either the terminal residue directly or any
residues within the residue block and then press the <b>Ace</b> or
<b>Nme</b> button to add the groups to the N- or C-terminus.
The buttons act as toggle buttons: To remove the acetyl/amide groups,
press again and release the toggle buttons.

<!-- ========================== PROTONATE =========================== -->

<a name="Protonation" />
<h4>Set Residue Protonation State</h4>

<p>
<div class="floatright">
<img src="probuild/probuild_variant.png" alt="Select Protonation State"/>
</div>

If a residue can exist in multiple protonation states (e.g. histidine),
the desired state can be selected with the Protein Builder. To
set the protonation, first select the residue to change. If multiple
protonation states are present for that residue type in the rotamer
library, they will be shown in a row of buttons in the Protein Builder
below the residue build and mutate buttons. The current protonation state
of the selected residue will be marked in blue on the corresponding
button. Hovering with the mouse over the buttons will show a 2D molecule
diagram of the protonation state to help with the selection. To change/set
the protonation, press the button representing the desired protonation.
</p>

<p>
<b>Note:</b> When building or mutating residues, the default protonation
state is set. To build a residue in a specific state, first
build it in the default state, then change its state in a second step.
</p>

<p>
<span class="tip"><b>Tip!</b></span>
Residue protonation states can also be chosen during
<a href="#RotamerExplorer"><b>Rotamer Exploration</b></a> by setting or adding
a specific sidechain protonation as the exploration <b>Target</b>.
</p>

<!-- ========================== REPACK =========================== -->

<a name="Repack" />
<h4>Repack Residue Sidechain</h4>

When mutating residue sidechains, by default no clash check or
energy optimization is done, so an unfavorable sidechain conformation
can occur. To optimize the sidechain conformation of the selected
residue, either the <a href="#Minimize"><b>Minimize</b></a>
or <b>Repack</b> options of the Protein Builder can be used. Repacking will search
the rotamer library for a sidechain conformation that fits into the
local environment. To perform the repacking automatically
after every <a href="#Minimize"><b>Mutate</b></a> or
<a href="#Build"><b>Build</b></a> operation, enable the
<b>Auto Repack</b> option.

<p>
<b>Note!</b> Repacking can be slow if multiple sidechains are repacked
at once, or when operating on sidechains with many rotamer conformations
in the rotamer library, i.e. sidechains with many rotatable bonds (e.g.
arginine).
</p>

<!-- ========================== DISULFIDE =========================== -->

<a name="Disulfide" />
<h4>Create Disulfide Bond</h4>

To create a disulfide bond/bridge, select the cysteine residues which
should form the bond and press <b>S-S</b>. If the geometric criteria
are met, i.e. the sidechains can be reoriented to connect, the
disulfide bond will be created.

<p>
<b>Note:</b> The residues of an already existing
disulfide bond can be selected. Pressing <b>S-S</b> will check the geometry of that
bond and disconnect it if the criteria are not satisfied.
</p>


<!-- =================== BREAK SIDECHAIN-SIDECHAIN ================ -->

<a name="BreakSidechainBond" />
<h4>Break Sidechain-Sidechain Bond</h4>

Disulfide bonds and other sidechain-sidechain bonds can be broken
by selecting the residues that are connected through sidechains with
other residues and pressing <b>R-<font color="#ff0000">X</font>-R</b>.
All sidechain-sidechain bonds will be broken, but bonds to backbone atoms
will stay intact.

<!-- ========================== JOIN =========================== -->

<a name="Join" />
<h4>Join Residues/Chains</h4>

Two chains or residue blocks can be joined by selecting
the two residues to be joined by a peptide bond. One of the
residues must be a C-terminal residue, and the other must be an N-terminal residue.
Pressing <b>Join</b> will create the peptide bond between the
selected residues. To perfom a rigid body minimization of the joined
fragment, <tt>Shift</tt>-click the <b>Join</b> button.

<p>
If the selected residues are in different chains, a panel will pop up
in which to specify whether the joined residues should be merged
into a single chain as well as which residues will be merged. The panel has
different options depending on the selection state of the <b>Merge
Residues Into One Chain</b> checkbox:
</p>

<p>
<table width="100%" cellpadding="5">
<tr>
    <td rowspan="5" valign="top">
	<img src="probuild/probuild_join.png" alt="Join Chains Without Merge"/>
    </td>
    <td rowspan="5" valign="top"> &nbsp; </td>
    <td valign="top"><b>Do Not Minimize After Join</b></td>
    <td valign="top">
	Join the selected residues with a peptide bond,
	but do not minimize residues after joining.
    </td>
</tr>
<tr>
    <td valign="top"><b>Minimize Tail Residues</b></td>
    <td valign="top">
	Join the selected residues with a peptide bond and minimize
	the <b>appended</b> residues (new <b>Tail Residues</b>) as a
	rigid body, while holding the other residues fixed.
    </td>
</tr>
<tr>
    <td valign="top"><b>Minimize Head Residues</b></td>
    <td valign="top">
	Join the selected residues with a peptide bond and minimize
	the <b>prepended</b> residues (new <b>Head Residues</b>) as a
	rigid body, while holding the other residues fixed.
    </td>
</tr>
<tr>
    <td valign="top"><b>?</b></td>
    <td valign="top">
	Briefly select the atoms and residues which will be
	minimized as a rigid body.
    </td>
</tr>
<tr>
    <td valign="top"><b>OK</b></td>
    <td valign="top">
	Join chain residues with the specified options.
    </td>
</tr>
</table>
</p>

<p>
<table width="100%" cellpadding="5">
<tr>
    <td valign="top"><b>Append Residue Block</b></td>
    <td valign="top">
	Append a block of residues after the selected target residue.
	The residues will be merged with the target chain.
	The <b>Residue Block</b> consists of the residues from
	the selected residue up to the <b>next chain break</b> in that chain.
    </td>
    <td rowspan="7" valign="top">
	<img src="probuild/probuild_merge.png" alt="Join and Merge Chains"/>
    </td>
</tr>
<tr>
    <td valign="top"><b>Append All Tail Residues</b></td>
    <td valign="top">
	Append all tail residues of the source chain after the selected
	target residue. The residues will be merged with the target chain.
	The <b>Tail Residues</b> are the residues from the selected residue
	up to the <b>end of the chain</b>.
    </td>
</tr>
<tr>
    <td valign="top"><b>Prepend Residue Block</b></td>
    <td valign="top">
	Prepend a block of residues before the selected target residue.
	The residues will be merged with the target chain.
	The <b>Residue Block</b> consists of the residues from the
	selected residue back to the <b>previous chain break</b> in that chain.
    </td>
</tr>
<tr>
    <td valign="top"><b>Prepend All Head Residues</b></td>
    <td valign="top">
	Prepend all head residues of the source chain before the selected
	target residue. The residues will be merged with the target chain.
	The <b>Head Residues</b> are the residues from the selected residue
	back to the <b>start of the chain</b>.
    </td>
</tr>
<tr>
    <td valign="top"><b>Minimize After Join</b></td>
    <td valign="top">
	If on, minimize the appended/prepended residues as a rigid body
	while the other residues are kept fixed.
    </td>
</tr>
<tr>
    <td valign="top"><b>?</b></td>
    <td valign="top">
	Briefly select the atoms and residues which will be appended/prepended
	and merged into one chain.
    </td>
</tr>
<tr>
    <td valign="top"><b>OK</b></td>
    <td valign="top">
	Join and merge chain residues with the specified options.
    </td>
</tr>
</table>
</p>

<p>
The following table shows the effects of the different <b>Append/Prepend</b>
options. Residues of two separate chains are joined and merged.
The selected residues that are to be joined with a peptide bond are
marked blue. The green residues are appended/prepended at the residue
selection in the target chain.

<center>
<table class="titlebar">
<tr>
    <th>Merge Option</th>
    <th>Before Merge</th>
    <th>After Merge</th>
</tr>
<tr>
    <td valign="top"><b>Append Residue Block</b></td>
    <td valign="top">
	<img src="probuild/probuild_merge1.png"
	 style="vertical-align:middle" />
    </td>
    <td valign="top">
	<img src="probuild/probuild_merge1a.png"
	 style="vertical-align:middle" />
    </td>
</tr>
<tr>
    <td valign="top"><b>Append All Tail Residues</b></td>
    <td valign="top">
	<img src="probuild/probuild_merge2.png"
	 style="vertical-align:middle" />
    </td>
    <td valign="top">
	<img src="probuild/probuild_merge2a.png"
	 style="vertical-align:middle" />
    </td>
</tr>
<tr>
    <td valign="top"><b>Prepend Residue Block</b></td>
    <td valign="top">
	<img src="probuild/probuild_merge3.png"
	 style="vertical-align:middle" />
    </td>
    <td valign="top">
	<img src="probuild/probuild_merge3a.png"
	 style="vertical-align:middle" />
    </td>
</tr>
<tr>
    <td valign="top"><b>Prepend All Head Residues</b></td>
    <td valign="top">
	<img src="probuild/probuild_merge4.png"
	 style="vertical-align:middle" />
    </td>
    <td valign="top">
	<img src="probuild/probuild_merge4a.png"
	 style="vertical-align:middle" />
    </td>
</tr>
</table>
</center>
</p>


<!-- ========================== SPLIT =========================== -->

<a name="Split" />
<h4>Split Residues/Chains</h4>

To split a peptide bond between two amino acid residues, select the
bond or the two residues in the MOE main window or the Sequence Editor
and press the <b>Split</b> button.

<p>
After breaking the peptide bond, the residue structures will be adjusted
to form proper N- and C-terminal residues. I.e. on the now N-terminal
residue, the backbone nitrogen will be positively charged and
hydrogens will be added. On the C-terminal residue, a negatively charged
oxygen is added to form the terminal carboxyl group.
</p>

<p>
By default, only the peptide bond is split and all chain residues are
kept in the same chain. The chain can also be split
into two, creating a new chain with the split-off residues. To achieve
the split, <tt>Shift</tt>-click the <b>Split</b> button to open the Split
Chain panel, in which to specify which residues will be moved into
a new chain. <b>Note</b>: Splitting sequence only residues (residues
without atoms) will always perform a chain split.
</p>

<p>
<table width="100%" cellpadding="5">
<tr>
    <td rowspan="5" valign="top">
	<img src="probuild/probuild_split.png" alt="Split Chains Options"/>
    </td>
    <td rowspan="5" valign="top"> &nbsp; </td>
    <td valign="top"><b>Split Peptide Bond Only</b></td>
    <td valign="top">
	Split the peptide bond between the selected residues,
	but do not move the split residues into a new chain.
    </td>
</tr>
<tr>
    <td valign="top"><b>Split Residue Block</b></td>
    <td valign="top">
	Split the peptide bond between the selected residues and
	move the <b>Residue Block</b> (residues from the selection up
	to the <b>next chain break</b>) into a new chain. The new chain will have
	the same name and tag as the original.
    </td>
</tr>
<tr>
    <td valign="top"><b>Split Chain Residues</b></td>
    <td valign="top">
	Split the peptide bond between the selected residues and
	move the <b>Chain Residues</b> (residues from the selection up
	to the <b>end of the chain</b>) into a new chain. The new chain
	will have the same name and tag as the original.
    </td>
</tr>
<tr>
    <td valign="top"><b>?</b></td>
    <td valign="top">
	Briefly select the atoms and residues
	which will be split off into a new chain.
    </td>
</tr>
<tr>
    <td valign="top"><b>OK</b></td>
    <td valign="top">
	Split chain residues with the specified options.
    </td>
</tr>
</table>
</p>

<!-- ========================== RENAME =========================== -->

<a name="Rename" />
<h4>Rename Residue</h4>

<p>
<div class="floatleft">
<img src="probuild/probuild_rename.png" alt="Rename Residue"/>
</div>
If exactly 1 residue is selected, its name, UID number and
insertion code (if set) are shown in the Protein Builder in three
text fields. To make changes, edit the text field contents
and press the <tt>Return</tt> key to accept the change.
</p>

<p>
<div class="floatright">
<img src="probuild/probuild_rename2.png" alt="Rename Residue"/>
</div>
<span class="tip"><b>Tip!</b></span>
The Protein Builder monitors residue names and UID number + insertion codes
and flags them if they might be not correct. E.g.: The residue
name does not match the residue structure or a UID number + insertion
code does not fit into the chain sequence. If such a problem is
detected, the offending name or number is colored red, and the
suggested correction can be selected from a pulldown menu.
</p>

<!-- ========================== RENUMBER =========================== -->

<a name="Renumber" />
<h4>Renumber Residue</h4>

<p>
If more than 1 residue or a range of residues is selected, the
number of selected residues or the selection range is shown in the
Protein Builder panel.
</p>

<p>
<center>
<table width="90%" cellpadding="5">
<tr>
    <td>
	Selection of a residue block:
	<img src="probuild/probuild_selrange.png" style="vertical-align:middle" 
	 alt="Selected Residue Range"/>
    </td>
    <td>
	Non-contiguous residue selection:
	<img src="probuild/probuild_selres.png" style="vertical-align:middle" 
	 alt="Selected Residues"/>
    </td>
</tr>
</table>
</center>
</p>

<p>
Press the button 
<img src="probuild/probuild_renum.png" style="vertical-align:middle" />
to pop up the renumber panel with options to change or assign UID numbers of
the selected residues:
</p>

<p>
<table width="100%" cellpadding="5">
<tr>
    <td rowspan="4" valign="top">
	<img src="probuild/probuild_renumber.png" alt="Renumber Residues"/>
    </td>
    <td rowspan="4" valign="top"> &nbsp; </td>
    <td valign="top"><b>Start From Number</b></td>
    <td valign="top">
	Renumber selected residues sequentially starting
	with the specified number. If the residue selection
	is not contiguous, the residues in the selection gaps
	are renumbered with insertion codes.
    </td>
</tr>
<tr>
    <td valign="top"><b>Copy From Chain</b></td>
    <td valign="top">
	Renumber selected residues by copying the
	UID numbers from the aligned residues of
	the specified chain.  Alignment gaps are
	filled with insertion codes.
    </td>
</tr>
<tr>
    <td valign="top"><b>Set to Alignment Positions</b></td>
    <td valign="top">
	Renumber selected residues by assigning the
	sequence alignment positions as residue UID
	numbers.
    </td>
</tr>
<tr>
    <td valign="top"><b>Apply</b></td>
    <td valign="top">
	Apply selected renumber scheme.
    </td>
</tr>
</table>

</p>

<!-- ========================== MINIMIZE =========================== -->

<a name="Minimize" />
<h4>Minimize Residue</h4>

After modifying residues with the Protein Builder, it might be necessary
to perform an energy minimization on that residue and its neighbors.
The <b>Minimize</b> button provides several minimization protocols for
various situations. The available protocols are dependent on the
the current selection of residues.

<p>
<center>
<table class="titlebar">
<tr>
    <th>Residues Selected?</th>
    <th>Minimize Option</th>
    <th>Description</th>
</tr>
<tr>
    <td align="center" rowspan="4">Yes</th>
    <td valign="top"><b>Selected Sidechains Only</b></td>
    <td valign="top">
	Minimize the sidechains of the selected residues.
	The sidechain atoms of residues near the selected residues
	(less than 4.5 angstroms away) are minimized as well, but
	tethered to their current positions. All other atoms and
	all backbone atoms are kept fixed.
    </td>
</tr>
<tr>
    <td valign="top">
	<b><span class="nobr">Selected Sidechains</span> + Tether BB</b>
    </td>
    <td valign="top">
	Minimize the sidechains of the selected residues. The
	backbone atoms of the selected residues are tethered to their
	current positions. The atoms of the residues near the selected
	residues are also tethered. All other atoms are kept fixed.
    </td>
</tr>
<tr>
    <td valign="top"><b>Selected Residues</b></td>
    <td valign="top">
	Minimize the selected residues (sidechains and backbone).
	The atoms of the residues near the selected residues are
	tethered to their current positions. All other atoms are
	kept fixed.
    </td>
</tr>
<tr>
    <td valign="top"><b>Joined Rigid Bodies</b></td>
    <td valign="top">
	Minimize the atoms of the selected residues. All other atoms
	are treated as separate rigid bodies.<br />
	E.g. if two chains were joined with three new residues and
	these three residues are selected, they will be minimized,
	while the two original chains will be treated each as rigid bodies.
    </td>
</tr>
<tr>
    <td align="center" rowspan="3">No</th>
    <td valign="top"><b>Sidechains Only</b></td>
    <td valign="top">
	Minimize the sidechain atoms of all residues while the backbone
	atoms are kept fixed.
    </td>
</tr>
<tr>
    <td valign="top"><b>Sidechains + Tether BB</b></td>
    <td valign="top">
	Minimize the sidechain atoms of all residues. The backbone
	atoms are tethered to their current positions.
    </td>
</tr>
<tr>
    <td valign="top"><b>All Residues</b></td>
    <td valign="top">
	Minimize all atoms of all residues.
    </td>
</tr>
</table>
</center>
</p>

<!-- ========================== DELETE =========================== -->

<a name="Delete" />
<h4>Delete Residue</h4>

The <b>Delete</b> button deletes the selected residues and their atoms,
as well as all bonds connecting them to other residues. If the residues were
connected by peptide bonds to other residues, these peptide bonds are
split correctly before the deletion, i.e. terminal oxygen or nitrogen
atoms are added and charged, as needed, to form proper N- or C-terminal
residues (see also <a href="#Split">Split Residues</a>).

<!-- ========================== SETTINGS =========================== -->

<a name="ProteinBuilderSettings" />
<h4>Protein Builder Settings</h4>

The Protein Builder has a couple configurable options.
Press the <img src="probuild/config.png" style="vertical-align:middle" />
button to open the Protein Builder Settings panel:

</p>

<p>
<div class="floatleft">
<img src="probuild/probuild_config.png" alt="Protein Builder Settings"/>
</div>

By default, the Protein Builder conveys additional information about
the <a href="#Selection">selected residues</a> via the colors of
the residue buttons. To disable this feature, switch off the
<b>Additional Residue Button Colors</b> option. The standard
amino acid button labels can also be switched between <b>Three-Letter</b> and
<b>One-Letter</b> codes.
</p>
<p>
A horizontal and a vertical <b>Layout</b> of the Protein Builder is available.
By default, the vertical layout is only used when the Builder is
attached to the MOE window. The Protein Builder panel can be attached to
either the <tt>left</tt> or <tt>right</tt> side of the MOE or
Sequence Editor window. The Protein Builder must be restarted for changes
to both options to take effect.
</p>

<p>
The <a href="#Repack"><b>Repack</b></a> cutoff specifies the
distance (in Angstroms) used to cluster the conformers during repacking.
Higher values yield smaller effective sizes of the rotamer library, but
speed up the repacking process. By default the protein builder uses
PDB standard residue names when adding/modifying residues, but
one can switch to <b>Amber Residue Names</b>. Amber uses e.g. separate names
for the different histidine protonation states (<tt>Hie, Hid, Hip</tt>).
When the <b>Scale View While Building</b>
option is on, the molecule view will be zoomed, so that newly built
(e.g. appended) residues are visible. This is convenient when
building long extended poly-peptides with the Builder.
By default, the Protein Builder synchronizes the atom and residue
selection, even if the option <span class="nobr">
<b>MOE | Select | Synchronize</b></span> is off.
The Protein Builder uses both the atom and residue selection; it is
therefore recommended to enable this option.
</p>

<p>
All of these Protein Builder settings are stored similar to
<a href="../appendix/settings.htm"><b>MOE Settings</b></a>
and will be retained for future MOE sessions.
</p>


<!-- ========================== ROTAMER EXPLORER ====================== -->


<a name="RotamerExplorer" />
<h2>Rotamer Explorer</h2>

The purpose of the Rotamer Explorer is to explore the conformational
space of amino acid sidechains within proteins in the presence of
surrounding amino acids, ligands, and solvent. The 
potential sidechain conformations can be visually inspected and ranked by multiple scores.
Additionally, the Rotamer Explorer can be used to mutate the selected
sidechain (and explore rotamers of these mutations).

<!-- =================== ROTAMER EXPLORER - CONF GENERATION ============= -->

<a name="ConformationGeneration"></a>
<h4>Conformation Generation</h4>

<p>
A rich discrete all-atom backbone-independent rotamer (or conformation)
library is used by the Rotamer Explorer for both sidechain conformation
exploration and residue mutation. The sidechain coordinates are optimally
rotated and translated onto a particular backbone of a particular protein
structure. (Previous versions of the Rotamer Explorer used a
systematic conformational search that varied dihedral angles
but used ideal bond lengths and angles.)
</p>

<p>
The rotamer library was created using LowModeMD.
The <a href="buildrotlib.htm"><b>Build Rotamer Library</b></a> chapter
explains the methodology and compares it against historical rotamer
libraries derived from PDB data.
</p>


<!-- =================== ROTAMER EXPLORER - SCORING ============= -->

<a name="Scoring"></a>
<h4>Scoring</h4>

The sidechain conformation score is a sum of three terms:
<ol>
<li>The sidechain strain energy from the rotamer library.
<li>The AMBER van der Waals interaction energy with the environment.
<li>A hydrogen bond score made up of a radial Lennard-Jones 12-10 potential
and a dipole-dipole electrostatic term.
</ol>

The hydrogen bond radial 12-10 distribution is consistent with
contact statistics derived from the PDB.

<p align="center"><img src="probuild/penalty.png" alt="Penalty Term" /></p>
<!-- LATEX rotamer/penalty \[
    P = \exp\left[-\frac{1}{kT}\left(
    {5}\left[\frac{2.85}{r_{ij}}\right]^{12}
    -{6}\left[\frac{2.85}{r_{ij}}\right]^{10}
    \right)\right]

\] ENDLATEX -->

where <i>r<sub>ij</sub></i> is the distance between the polar heavy atoms in
the sidechain and the environment.

<p>
Each polar hydrogen bearing heavy atom is given a point dipole in a
direction consistent with its hybridization and with a magnitude consistent
with the TIP5P water model. Dipole-dipole electrostatic interactions are
evaluated for each such pair:
</p>

<p align="center"><img src="probuild/dipole.png" alt="Dipole Energy Term" /></p>
<!-- LATEX rotamer/dipole \[

    E_{dip-dip} = \frac{1}{4\pi \varepsilon_{0}}
    \left(\mathbf{d}_{i}\cdot \mathbf{d}_{j}
    -3 \left( \mathbf{d}_{i}\cdot \mathbf{r}_{ij} \right)
    \left( \mathbf{r}_{ij}\cdot \mathbf{d}_{j} \right)
    \right)

\] ENDLATEX -->

where <b>d</b><sub><i>i</i></sub> and <b>d</b><sub><i>j</i></sub>
are the dipole vectors of the
heavy atoms and <b>r</b><sub><i>ij</i></sub> the vector between the atom
positions.

<p>
The total hydrogen bond interaction energy E<sub>hb</sub>, which takes
directional effects into account, is the product of the radial term P
and the dipole-dipole energy term E<sub>dip-dip</sub>.
</p>

<a name="StartExplorer"></a>
<h4>Exploring a Residue</h4>

<p>
To start exploring rotamers, select the residue and press the
<b>Rotamers</b> button. Only one residue can be explored
at a time. The rotamers of the selected residue are automatically
searched by the Rotamer Explorer, displaying the results in the list.
</p>


<!-- =================== ROTAMER EXPLORER - LIST INFO ============= -->

<h4><a name="Results">Reported Information</a></h4>

<p>Once the search is completed, the Rotamer Explorer reports the information
in the panel list. For each rotamer, the Rotamer Explorer reports:

<p>
<table width="100%" cellpadding="5" cellspacing="5">
<tr>
    <td valign="top"><b>Res</b></td>
    <td valign="top">
	The residue type of the rotamers being explored.
	When exploring amino acids with multiple protonation states (e.g.
	histidine), the residue type also indicates the tautomer (e.g.
	<tt>Hie</tt>, <tt>Hid</tt>, or <tt>Hip+1</tt>).
    </td>
    <td rowspan="7">
	<img src="probuild/rotexpl_main.png"
	 alt="Rotamer Explorer in Protein Builder"/>
    </td>
</tr>
<tr>
    <td valign="top"><b>E</b></td>
    <td valign="top">
	The sidechain conformation score is calculated as the sum
	of the sidechain strain energy, the van der Waals interaction
	energy with the environment, and the hydrogen bond interaction
	energy. It is normalized to the lowest value of all rotamers.
    </td>
</tr>
<tr>
    <td valign="top"><b>Erot</b></td>
    <td valign="top">
	Sidechain strain energy from the rotamer library.
    </td>
</tr>
<tr>
    <td valign="top"><b>Ehb</b></td>
    <td valign="top">
	Hydrogen bond interaction energy.
    </td>
</tr>
<tr>
    <td valign="top"><b>Evdw</b></td>
    <td valign="top">
	Van der Waals interaction energy with the environment.
    </td>
</tr>
<tr>
    <td valign="top"><b>RMSD</b></td>
    <td valign="top">
	Root Mean Square Deviation, in &Aring;, of the rotamer to the
	original sidechain conformation.
    </td>
</tr>
<tr>
    <td valign="top"><b>chi n</b></td>
    <td valign="top">
	The dihedral chi angles of the rotamer (optional).
    </td>
</tr>
</table>
</p>

<!-- =================== ROTAMER EXPLORER - VISUALIZE ============= -->

<a name="Visualize"></a>
<h4>Visualizing the Results</h4>
<p>
Clicking on rotamers in the list causes them to appear in the MOE window.
<tt>CTRL</tt>-click or <tt>SHIFT</tt>-click on the list to display
multiple rotamers.  During the exploration of the rotamers, the actual
residue in the system is unchanged.
</p>

<p>
<div class="floatright">
<img src="probuild/rotexpl_render.png" alt="Rotamer Color and Render Style"/>
</div>
To show the old/current sidechain conformation overlayed with the
rotamer search results, enable the <b>Show Current</b> checkbox on
the bottom of the Rotamer Explorer panel. The rendering of the
rotamer results can be changed to emphasize them and more easily
study their interactions with the local environment. Click
on the <b>Render</b> button to open the rotamer customization panel.
The <b>Style</b> and <b>Color</b> of the rotamer sidechains can be
controlled separately. By default, only the carbon color (see <b>C</b>
checkbox) is changed to green, but the current render style is retained.
To isolate the rotamer exploration site visually, press the 
<b>Isolate</b> button. It will center the view on the rotamer sidechain and
show only the residues that are in proximity of the selected
residue and its rotamer conformations.
</p>

<p>
In addition to selecting the rotamers to be shown in the list, they can also
be browsed with the
<img src="probuild/rotexpl_updown.png" style="vertical-align:middle" />
buttons below the list,
or by using the <tt>Up</tt> and <tt>Down</tt> keys.
Press
<img src="probuild/cancel.png" style="vertical-align:middle" />
to stop exploring, discard the rotamer result, and clear the list.
The result list is active as long as the explored residue or some of its atoms
are selected in the Sequence Editor or MOE main window. If the residue 
selection is lost, the search result list is desensitized, but not
cleared. To reactivate the list and continue exploring,
select the residue again or press the reselect
<img src="probuild/rotexpl_reselect.png" style="vertical-align:middle" />
button.

The list is cleared automatically if the search results become
invalid, e.g. if the molecule is changed, minimized, etc.
</p>

<p>
If a suitable sidechain conformation is found, it can be applied permanently
by pressing the <b>Keep</b> button. <br />
<b>Note</b>: After applying the requested rotamer conformation,
the search result list will be cleared. Alternatively, it is possible to
<a href="#WriteDatabase"><b>Write</b></a> all, or selected, rotamer
conformations in a database.
</p>

<!-- =================== ROTAMER EXPLORER - MULTI TARGET ============= -->

<a name="MultiTarget"></a>
<h4>Explore Multiple Targets</h4>

<p>
<div class="floatright">
<img src="probuild/rotexpl_target.png" alt="Target Selection"/>
</div>

By default, the Rotamer Explorer will perform a rotamer conformation search
of just the selected residue's sidechain type. However, it is also possible to
perform a search on multiple different amino acid sidechains or tautomers.
Press <b>Target</b> to specify the rotamer amino acid mutation targets;
a panel with the list of amino acids available in the
<a href="buildrotlib.htm">rotamer library</a> will pop up.
The standard amino acids and their tautomers are at the top of the list, above
the non-natural amino acids. For each amino acid target, the three-letter
code, sidechain charge, and name, as specified in the rotamer library, are
listed. To help find and select the desired target(s), the list can be
filtered by <b>Class</b> (e.g. <tt>Aromatic</tt>, <tt>Polar</tt>,
<tt>D-Amino</tt>, etc.) or by a <b>Filter</b> string. One or multiple
amino acid mutation targets can be selected.
</p>

<div class="floatleft">
<img src="probuild/rotexpl_multitarget.png" alt="Exploring multiple Targets"/>
</div>

<p>
At the bottom of the panel are several quick-selection buttons.
<b>Current Only</b> will clear the selection and select only the current
amino acid type of the residue to explore (i.e. resetting to default selection).
<b>Protomers</b> will select all (if any) tautomers of the currently selected
amino acids in the list (e.g. if exploring the histidine tautomers is desired).
<b>Polar</b>, <b>Hydrophobic</b>, <b>Positive</b>, <b>Negative</b> will
select the amino acids of that type (as defined in the rotamer library
class annotations) in the list, while pressing the
<b><font color="FF0000">X</font></b> will deselect them.
</p>

<p>
The number of selected mutation targets will be shown on the
<b>Target</b> button of the Rotamer Explorer/Protein Builder panel.
If multiple targets are selected and the <b>Rotamers</b> button is
pressed, all of these targets will be used to find suitable rotamer
conformations. The conformations are presented in the results list and
scored against each other. Press <b>Res</b> in the list header to sort
it so that the conformations of the amino acid targets are grouped together.
</p>

<!-- =================== ROTAMER EXPLORER - WRITE DATABASE ============= -->

<a name="WriteDatabase"></a>
<h4>Save Rotamer Results to Database</h4>

<p>
<div class="floatright">
<img src="probuild/rotexpl_write.png" alt="Write Rotamers to Database"/>
</div>
Rotamers reported in the list may be saved in a database for future
reference. Each column of information displayed will be stored in
a separate database field. To save the rotamers, press <b>Write</b>.
The Write to Database panel appears:
</p>

<p>
This panel can be used to save
some or all of the rotamers to a database.
</p>

<p>Press <b>OK</b> to perform the write and update/create the database
of rotamers.

<p>
<table width="100%" cellpadding="5" cellspacing="5">

<tr>
    <th valign="top" align="left"><b>Database</b></th>
    <td>
	The file to which the database is written.
    </td>
</tr>

<tr>
    <th valign="top" align="left"><b>Open in Database Viewer</b></th>
    <td valign="top">
	Opens the database in a Database Viewer after the write
	operation is completed.
    </td>
</tr>

<tr>
    <th valign="top" align="left"><b>Append New Entries</b></th>
    <td valign="top">
	When this option is on, if the specified database already exists,
	the new entries will be appended to it. Otherwise, the
	database will be overwritten. If the database does not exist,
	a new database will be created regardless of the state of this option.
    </td>
</tr>

<tr>
    <th valign="top" align="left"><b>Selected Rotamers Only</b></th>
    <td valign="top">
	If on, only the conformations selected in the Rotamer
	Explorer list will be written. Otherwise, all conformations will be
	written.
    </td>
</tr>
</table>


<!-- =================== ROTAMER EXPLORER - SETTINGS ============= -->

<a name="RotamerExplorerSettings"></a>
<h4>Rotamer Explorer Settings</h4>

<div class="floatright">
<img src="probuild/rotexpl_config.png" alt="Rotamer Explorer Settings"/>
</div>

<p>
The Rotamer Explorer has several per-session configurable options. The
settings are only retained while the Rotamer Explorer remains open.
Press the <img src="probuild/config.png" style="vertical-align:middle" />
button to open the Rotamer Explorer Settings panel.
</p>

<p>
A default rotamer search will include all active atoms, even if hidden,
into the search environment, but will exclude salts and solvent atoms.
The environment specifies which atoms interact with the rotamer and they
are used in calculating the sidechain conformation score, hydrogen bond
interaction energy, and to determine whether the conformation is free of steric
conflict. <b>Salts and Solvent</b> can be included in the environment,
e.g. if they are an important part of the active site, by enabling the
checkbox. <b>Hidden Atoms</b> can be removed from the environment as
well, if desired.
</p>

<p>
The <b>Cluster</b> value specifies, in angstroms, the distance to be used
to cluster the conformers for each rotamer type. Higher values yield
smaller effective sizes for the rotamer library, i.e. fewer
individual conformations will be returned. The <b>Energy Cutoff</b>
limits the results as well. All rotamer conformations
with an energy above the cutoff value (in relation to the lowest
energy of the specific rotamer type) are discarded and not shown in the
list. By default, the <b>Rotamer Chi Angles</b> are not included in
the result list. They are added, if that option is enabled.
</p>

<!-- ============================== PTM ========================= -->

<a name="PTM" />
<h3>Post-Translational Modifications (PTM)</h3>

<p>
To apply post-translational modifications to residues, first select the atoms
or residues where the PTMs should be attached.  Choose
<span class="nobr"><b>MOE | Edit | Build | PTM</b></span> (to use
selected atoms) or <span class="nobr"><b>SE | Edit | PTM</b></span>
(to use selected residues), followed by the desired modification.
The chosen modification will then be made to the selected atoms or
residues, provided that it can chemically be attached at those points and
there is physically room. A simple van der Waals potential is used
to check for clashes and ensure the additional atoms will fit, and any solvent
atoms which clash with the new modification are deleted by this operation.
Note that for smaller modifications, the residue name (3-letter code) may
change. A larger modification will result in a new residue being created in
a new chain, but with the same chain letter and tag as the residue they are
being attached to.


<!-- ============================ RESIDUE CODES ========================= -->

<a name="Codes"></a>
<h2>Residue Code Reference</h2>

<p>
<center>
<table class="titlebar">
    <tr>
	<th>1-letter code</th>
	<th>3-letter code</th>
	<th>Amino acid</th>
  </tr>
  <tr> <td><tt>A</tt></td> <td><tt>ALA</tt></td> <td>Alanine</td> </tr>
  <tr> <td><tt>R</tt></td> <td><tt>ARG</tt></td> <td>Arginine</td> </tr>
  <tr> <td><tt>N</tt></td> <td><tt>ASN</tt></td> <td>Asparagine</td> </tr>
  <tr> <td><tt>D</tt></td> <td><tt>ASP</tt></td> <td>Aspartate</td> </tr>
  <tr> <td><tt>C</tt></td> <td><tt>CYS</tt></td> <td>Cysteine</td> </tr>
  <tr> <td><tt>Q</tt></td> <td><tt>GLN</tt></td> <td>Glutamine</td> </tr>
  <tr> <td><tt>E</tt></td> <td><tt>GLU</tt></td> <td>Glutamate</td> </tr>
  <tr> <td><tt>G</tt></td> <td><tt>GLY</tt></td> <td>Glycine</td> </tr>
  <tr> <td><tt>H</tt></td> <td><tt>HIS</tt></td> <td>Histidine</td> </tr>
  <tr> <td><tt>I</tt></td> <td><tt>ILE</tt></td> <td>Isoleucine</td> </tr>
  <tr> <td><tt>L</tt></td> <td><tt>LEU</tt></td> <td>Leucine</td> </tr>
  <tr> <td><tt>K</tt></td> <td><tt>LYS</tt></td> <td>Lysine</td> </tr>
  <tr> <td><tt>M</tt></td> <td><tt>MET</tt></td> <td>Methionine</td> </tr>
  <tr> <td><tt>F</tt></td> <td><tt>PHE</tt></td> <td>Phenylalanine</td> </tr>
  <tr> <td><tt>P</tt></td> <td><tt>PRO</tt></td> <td>Proline</td> </tr>
  <tr> <td><tt>S</tt></td> <td><tt>SER</tt></td> <td>Serine</td> </tr>
  <tr> <td><tt>T</tt></td> <td><tt>THR</tt></td> <td>Threonine</td> </tr>
  <tr> <td><tt>W</tt></td> <td><tt>TRP</tt></td> <td>Tryptophan</td> </tr>
  <tr> <td><tt>Y</tt></td> <td><tt>TYR</tt></td> <td>Tyrosine</td> </tr>
  <tr> <td><tt>X</tt></td> <td><tt>UNK</tt></td> <td>Unknown</td> </tr>
  <tr> <td><tt>V</tt></td> <td><tt>VAL</tt></td> <td>Valine</td> </tr>
</table>
</center>
</p>

<!-- DNA codes, if the protein builder can also build DNA
<p>
<center>
<table class="titlebar">
    <tr> <th>1-letter code</th> <th>Nucleic Acid</th> </tr>
    <tr> <td><tt>a</tt></td> <td>Adenosine</td> </tr>
    <tr> <td><tt>c</tt></td> <td>Cytidine</td>  </tr>
    <tr> <td><tt>g</tt></td> <td>Guanosine</td> </tr>
    <tr> <td><tt>t</tt></td> <td>Thymidine</td> </tr>
</table>
</center>
</p>
-->

<!-- ===================== OBSOLETE: CREATE SEQUENCE ===================== -->
<!-- mk/2014: The Create Sequence panel is no longer accessible through the
menus, but it can come up through copy and paste.


<h2><a name="Create_Sequence"></a>Creating Sequences</h2>

<p>To open the Create Sequence panel, choose 
<b>SE&nbsp;|&nbsp;Edit&nbsp;|&nbsp;Create&nbsp;Sequence</b>
in the Sequence Editor.  This panel is used to create
polypeptide and DNA fragments from a string of single-letter identifiers.
</p>

<p align="center">
<img src="probuild/bioseq.png" alt="Create Sequence Panel" />
</p>

<p>
The Create Sequence input line recognizes single-letter codes of residues (see
<a href="#Codes">below</a>).  Sequence Type options allow you to create a
sequence of empty residues (i.e. without atoms) or a fully-populated
peptide chain in either extended or alpha-helix conformation.  The DNA options
create fully-populated strands in A, B or C conformation [Arnott 1975].</p>

<p>
The name, tag and header attributes of the chain can be set in this
panel. Press <b>OK</b> to create the sequence. Press <b>Apply</b> to create
the sequence without closing the panel.
</p>

<p><font color="#ff0000"><b>Tip</b></font>&nbsp;&nbsp;
To cut and paste a sequence into the text field use the keyboard cut and paste
facilities.  For more information on the text field see <a
href="../moe/hellomoe.htm#GUI_Basics">GUI Basics</a>.
</p>
-->

<!-- ============================== SVL COMMANDS ========================= -->

<a name="SVL"></a>
<h2>SVL Commands</h2>
<a href="fcnref/intropro.htm">Protein Functions</a>


<!-- ============================== REFERENCES ========================= -->
<!-- mk/2014: The reference below is only used by the no longer documented
Create Sequence panel (see above).

<a name="References"></a><h2>References</h2>

<table width="100%" cellpadding="5" border="0">
<tr>
    <td valign="top">[Arnott&nbsp;1975]</td>
    <td valign="top">
	Arnott, S., Selsing, E.; The conformation of C-DNA;
	<i>Journal of Molecular Biology</i>
	<i>98</i> (<b>1975</b>) 265&ndash;269.
    </td>
</tr>
</table>
-->

<!-- ============================== SEE ALSO ========================= -->

<a name="SeeAlso"></a>
<h2>See Also</h2>

<a href="buildrotlib.htm">Build Rotamer Library</a><br />
<a href="propred.htm">Predicting Secondary Structure and Solvent Accessibility</a><br />
<a href="pdb_mdb.htm">Protein Sequence and Structure Data</a><br />
<a href="../moe/gui/se.htm">Sequence Editor Window</a><br />

<noscript>
	<hr noshade="noshade" />
	<font size="2"> Copyright &copy; 1997&ndash;2015
	<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a> </font>
</noscript>
<div id="MoeFooter"></div>
</body>
</html>
