<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!      /proteins/projsrch.htm        MOE Project Search
!!
!!-->

<!--
!!    MOE Online Manuals
!!    COPYRIGHT (C) 1997-2015
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

<script type="text/javascript" 
src="../include/jsincludes_moe.js"></script>

<link rel="stylesheet" type="text/css"
href="../include/manstyle.css" />


<title>Project Search</title>

<meta panel Project Search>

</head>

<body>
<div id="MoeHeader"></div>
<noscript>
	<span class="warning">Warning: JavaScript is disabled. This page will not display correctly.</span>
	<h1 class="title">MOE Documentation</h1>
	<hr noshade="noshade" />
	<style>.LaTeX {color: #C08080;}</style>
</noscript>

<div id="toc-list"><script>var gman_OrderedListTOC=false; </script></div>

<!-- ======================= Project Search Panel ====================== -->

<a name="ProjectSearch"></a>
<h2>The Project Search Panel</h2>

<p>The Project Search Panel offers a multitude of features that allows the user to
extract potentially complex data sets quickly and efficiently. The flexible
setup of the application does not imply any predefined search modes for
searching a database. </p>

<p>The hallmark feature of the Project Search application is that the
results of each individual search are stored and may be combined with other
search results that have been generated along the search process. This way,
data sets of interest that are based on complex search parameters can
be generated, stored, and used for downstream applications.</p>

<p>Additionally, a personalized <a href="moeproject.htm">MOE Project</a> 
database can be created, which can be searched and filtered using the Project 
Search panel.</p>

<p>To open the Project Search Panel choose any of the databases listed below the
<b>Domain Motif</b> menu item, in either of the following menus: 
<blockquote>
<span class="menu">MOE | Protein | Search</span> or <br>
<span class="menu">SE | Protein | Search</span>.
</blockquote>
</p>

<p align="center">
<img src="projsrch/ProjSearch_menu.png"  width="507px" 
alt="Protein-Search menu" style="border:1px solid grey" />
</p>

<p>Selecting any of the databases will open the relevant Project Search panel.
Most Project Search panels will be similar to the Nuclear Receptor panel shown
below. Details about the fields used in Project Databases can be found
in the chapter about <a href="pdb_mdb.htm#projDB">Protein Sequences
and Structure Data</a>.</p>

<p align="center">
<img src="projsrch/ProjectSearch_NuclearReceptor.png"  width="659px"
alt="Nuclear Receptor Project Search Panel" style="border:1px solid grey" />
</p>

<p>As can be seen on the following screenshot, the Project Search panel shown
when a protein kinase database is opened contains additional options for the
ligand binding mode and for searching unique subfamilies (highlighted in red).
It also displays additional data columns by default.
Details about the fields used in Kinase Databases can be found
in the chapter about <a href="pdb_mdb.htm#kinDB">Protein Sequences
and Structure Data</a>.</p>

<p align="center">
<img src="projsrch/ProjectSearch_ProteinKinase.png"  width="814px"
alt="Protein Kinase Project Search Panel" style="border:1px solid grey" />
</p>

<p>Upon opening the Project Search application the entire content of the 
database is shown in the result list. The results can be sorted on any of the
data columns by clicking on its column heading. Click on the heading again
to alternate the column sorting between ascending and descending sorts.
</p>

<p>By default, the referred database
is an <tt>mdb</tt> file located in the <tt>$MOE/lib/project</tt> directory. 
The default directory can be changed by selecting the 
<a href="../appendix/settings.htm#ProteinDefaults">Protein Defaults</a>
in the <b>Configuration Options</b> panel.
</p>


<a name="listMarkEntries"></a>
<h3>Listing and Marking Entries</h3>
<p>The top portion of the Project Search panel is used to list and mark 
search entries.</p>

<p align="center">
 <img src="projsrch/topRow.png"  width="814px"
 alt="Listing and Marking" style="border:1px solid grey" />
</p>

<table width="100%" cellpadding="5">
 <tr>
   <td valign="top"><b>List</b></td>
   <td valign="baseline">
        Select the filter to apply to the results. Filters include
		<i>Full Database</i>, only <i>Marked Entries</i>, and the results
		of previous searches.
		<p align="center">
		 <img src="projsrch/projSearch_list_cut.png"  width="811px"
		 alt="List options" style="border:1px solid grey" />
		</p>
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Edit</b> 
		<img src="projsrch/editButton.png"  width="26px" alt="Edit icon" />
	</td>
   <td valign="baseline">
        After a search has been performed (for example, searching on the 
		<b>Species</b> of <i>Human</i>), press the <b>Edit button</b> 
		to name the query results. The name given, such as <b>human</b> for example,
		will be displayed in the <b>List</b> results, replacing the generic
		name of <b>Search 1</b>. Using the <b>Edit button</b>, a
		query result can also be renamed or deleted, or the entire history of
		query results can be cleared.
		<p align="center">
		 <img src="projsrch/projSearch_edit.png"  width="813px"
		 alt="Edit options" style="border:1px solid grey" />
		</p>
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Mark / Unmark</b></td>
   <td valign="baseline">
    Create a persistent subset of interest with the current entry selection by
    pressing the <b>Mark</b> button. These entries are then indicated in the
    <i>Mark</i> column with the "+" character, and are even available if the
    result set or the selection status has changed. It can be used as
    a permanent selection across multiple search result sets. The set of
    currently <i>Marked Entries</i> can be selected in the <b>List</b>
    drop-down.
    Pressing <b>Unmark</b> will remove the specifier from the selection.
		<p align="center">
		 <img src="projsrch/projSearch_mark_cut.png"  width="814px"
		 alt="Edit options" style="border:1px solid grey" />
		</p>
  </td>
 </tr>
</table>
 
 
<a name="paramGeneral"></a>
<h3>General Search/Filter Parameters</h3>

<p>In the section below the displayed results,
the Project Search panel allows filtering the database by
specifying one or more search terms. In addition to these search terms,
parameters that contain information about the kinase regulatory conformation,
family association, and structural parameters can be used as additional filters
to reduce the search space.
</p>

<p align="center">
 <img src="projsrch/searchArea.png"  width="814px"
 alt="Listing and Marking" style="border:1px solid grey" />
</p>

<table width="100%" cellpadding="5">
 <tr>
   <td valign="top"><b>Keywords</b></td>
   <td valign="baseline">
        Enter the search terms in the text field, with different terms
		separated by the space character. The search is case insensitive.
		<p>By default,
        wildcard ("*") characters are used on both sides of each search term. 
		Use double quotes around a word to change this behavior. 
		For an exact match to a phrase, enclose 
		the phrase in double quotes.</p>
		<p>A column header can be used for specializing the search on that column
		data. The format for such a specialization is
		<i>ColumnHeader:SearchPattern</i>.</p>
		<p>The column character &quot;:&quot; matches exact values.
		Use <i>greater than</i> and <i>less than</i>
		operators (&quot;&gt;&quot; and &quot;&lt;&quot;, respectively) to match 
		a range of values.</p>
		<p>For matching dates, the expected input format is <i>YYYY-MM-DD</i>.</p>
		<p>The Boolean operator to use between search terms is specified
		by the dropdown that follows <b>Keywords</b>.</p>
		<p>The example in the following figure shows a search for 
		<tt>Species:*yeast Date&gt;2000-01-01 Res&lt;3.0</tt>, using a Boolean
		operator of <i>And</i>. The returned results display both <i>Fission yeast</i>
		and <i>Baker's yeast</i> as species, which have a date after January 1, 2000,
		and which have a resolution less than 3.0 &Aring;.</p>
		
 		<p align="center">
		 <img src="projsrch/projSearch_keyword.png"  width="837px"
		 alt="Keyword options" style="border:1px solid grey" />
		</p>
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Species</b></td>
   <td valign="baseline">Specify the species the resulting compounds should
   originate from, as given in the PDB record. In addition to common
   names, scientific species names are also presented.
    <p align="left">
    <img src="projsrch/species.png"  width="400px"
	alt="Species Panel" style="border:1px solid grey" />
    </p>
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Family</b></td>
   <td valign="baseline">
     Specify a collection of families to be included in the search. An entire group
	 can be selected by double-clicking any family in the group.
     <p align="left">
    <img src="projsrch/selectfam.png" width="162px"
	alt="Family Selection Panel" style="border:1px solid grey" />
     </p>
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Ligand</b></td>
   <td valign="baseline">
		<div class="floatright">
		<img src="projsrch/fieldLigand.png" height="22px"
		alt="Ligand field" style="border:1px solid grey" />
		</div>
        Specify a MOE descriptor expression to match ligands in the database.
        If SMARTS strings are used as search criteria, they must be enclosed in
        double-quotes. A common set of descriptor expressions are given in the
        drop-down list.
	<p>
	<span class="tip"><b>Tip!</b></span>
	To quickly find all entries with apo structures (complexes
	without ligands) use the descriptor expression <i>!Weight</i>, or
	<i>Weight>0</i> if interested in structures with ligand.
	</p>
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Similar to Selected</b></td>
   <td valign="baseline">
		<div class="floatright">
		<img src="projsrch/fieldSimilar.png" height="22px"
		alt="Similarity field" style="border:1px solid grey" />
		</div>
		If enabled, the currently selected atoms in the MOE main window
		will be used to perform a fingerprint similarity search of
		ligands in the database.
		<p>For the similarity calculation, the
        <i>MACCS</i> keys are applied at a Tanimoto-distance cutoff as specified
        in the drop-down field to the right. Only structures with a
		Tanimoto-distance above the specified value are retained as results.
		The Tanimoto-distances will be listed after the search in the result
		set column <i>Sim</i>.</p>
   </td>
  </tr>
  <tr>
   <td valign="top"><b>Ligand Binding Mode</b></td>
   <td valign="baseline">
		<div class="floatright">
		<img src="projsrch/fieldBinding.png" height="22px"
		alt="Ligand Binding field" style="border:1px solid grey" /><br/>
        <div class="floatright"><img src="projsrch/binding.png" width="52px"
         alt="Ligand Binding Panel" style="border:1px solid grey" />
		</div></div>
        Ligands can be searched based
        on their binding profile to the kinase ATP-binding cleft. The regions
        within the pocket include the G-Loop, Hyd1, alphaC, Hinge, HRD, and DFG,
        and are abbreviated by the letters <i>G, Y, A, H, R, D</i> in the
	<i>Bind</i> column of the results display. This allows a search
	for inhibitors with mandatory interactions in the specified region. 

		<p>This filter is available only for kinase databases.</p>
   </td>
  </tr>
 <tr>
   <td valign="top"><b>Structure</b></td>
   <td valign="baseline">
		<div class="floatright">
		<img src="projsrch/fieldStructure.png" height="22px"
		alt="Structure field" style="border:1px solid grey" />
		</div>
        Specify key interaction regions to use in the structure search.
	For kinases, valid <a href="kinase.htm#kinClustering">residue sets</a>
	include <i>Core, Pocket, Canonical, Extended,</i> or <i>Complete</i>.
	For non-kinases, structure searches are always done on
	the <i>Complete</i> structure.
        The structure search is based on superposition of the C-alpha
	backbone atoms in the specified regions. The input structure to
	be searched for in the database is selected from the drop-down list.
	RMSD values above a specified cutoff value are discarded from the
        results. The RMSD cutoff (in &Aring;) is given in the <b>RMSD</b> field.
	The RMSD values will be listed after the search in the result
	set column <i>RMSD</i>.
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Sequence</b></td>
   <td valign="baseline">
		<div class="floatright">
		<img src="projsrch/fieldSequence.png" height="24px"
		alt="Sequence field" style="border:1px solid grey" />
		</div>
		Select the sequence search method. <i>Identity</i> will calculate the
		%-identity to the specified sequence search string over the 
		complete sequence. 
		For kinases, <p><i>Core, Pocket, Canonical,</i> or <i>Extended</i> will calculate the
		%-identity to the specified sequence search string over the kinase core, pocket,
		canonical, or extended <a href="kinase.htm#kinClustering">portion of the 
		sequence</a>, as selected.</p>
        <p>Specify the query sequence by either typing, pasting from the clipboard
        using <b>Ctrl+V</b> keys, or using one of the kinases already loaded in
        MOE. In the latter case, press 
		<img src="projsrch/loadButton.png" width="22px" alt="Load button" /> 
		to obtain a list of kinases currently present in MOE.</p>
		<p>Specify the <b>%-Identity Cutoff</b> value in the drop-down to the 
		right of the entered sequence. The %-identity values will be listed
		after the search in the result set column <i>SeqID</i>.</p>
		
		<p> If the special search method <i>Expression</i> is selected, one
		can specify a sequence residue selection expression which is evaluated
		against the sequences in the database. If the selection expression 
		matches a database entry's sequence, it is included in the the search
		result set. </p>
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Combine Search</b></td>
   <td valign="baseline">
		<div class="floatright">
		<img src="projsrch/fieldCombine.png" height="24px"
		alt="Combine Search field" style="border:1px solid grey" />
		</div>
		Combine the current search parameters with a previous search.
		Any of the previous searches or the marked entries, shown in
		<b>List</b> can be used. Use the Boolean operators
		(<i>And, Or, Not</i>) to fine-tune the search.
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Unique</b></td>
   <td valign="baseline">
		<div class="floatright">
		<img src="projsrch/fieldUnique.png" height="24px"
		alt="Unique field" style="border:1px solid grey" />
		</div>
		Enable any of these options to display unique (first occurence)
		search results in respect to the selected options.
   </td>
 </tr>
</table>

<a name="bottomButtons"></a>
<h3>Main Actions</h3>

<p>The bottom row of buttons in the Project Search panel contains the primary 
actions for displaying the filtered results, as well as performing actions
upon those results.
</p>
<p align="center">
 <img src="projsrch/bottomButtons.png"  width="615px" alt="Main actions" />
</p>

<table width="100%" cellpadding="5">
  <tr>
    <td valign="top"><nobr><b>Search</b></nobr></td>
    <td valign="baseline">Filter the database based on the current search 
	parameters. This button remains inactive until search criteria are entered
	in the Project Search panel. After a search is performed, the button again
	becomes inactive until the search criteria change.
    </td>
  </tr>
  <tr>
    <td valign="top"><nobr><b>Clear</b></nobr></td>
    <td valign="baseline">Clear and reset all search parameters. When this button
	is pressed, a dialog box asks for confirmation before proceeding.
    </td>
  </tr>
  <tr>
    <td valign="top"><nobr><b>Load</b></nobr></td>
    <td valign="baseline">
       Load the selected entries into the MOE main window.
       <p align="center">
         <img src="projsrch/load2MOE.png"  width="299px" alt="Load into MOE" />
       </p>
       <p>
       <table width="100%" cellpadding="5">
        <tr>
          <td valign="top"><b>Copy</b></td>
          <td valign="baseline">Specify the result set to be loaded in MOE:
          either everything in the result list, or only the
          marked collection, or only those currently selected in the list.
		  </td>
        </tr>
        <tr>
          <td valign="top"><b>Options</b></td>
          <td valign="baseline">
		  <p><b>Center Molecule</b>: the loaded structures will be centered in the 
		  MOE main window.</p>
		  <p><b>Apply Sequence Alignment</b>: the loaded structures will have their
		  sequences aligned using the alignment data from the database
		  before being displayed in the MOE main window.</p>
		  <p><b>Load Ligand Only</b>: only load ligand structures from the database.
		  <p><b>Select New Objects</b>: the loaded structures will be selected when
		  displayed in the MOE main window.</p>
		  <p><b>Close Current System</b>: all existing entities in MOE will be 
		  erased before displaying the loaded structures in the MOE main window.</p>
          </td>
        </tr>
       </table>
    </td>
  </tr>
  <tr>
    <td valign="top"><nobr><b>Save</b></nobr></td>
    <td valign="baseline">
       Save the selected entries in a separate database for future use.
       <p align="center">
         <img src="projsrch/save.png"  width="469px" alt="Save Panel" />
       </p>
       <p>
       <table width="100%" cellpadding="5">
        <tr>
          <td valign="top"><b>Copy</b></td>
          <td valign="baseline">Specify the result set to be loaded in MOE:
          either everything in the result list, or only the
          marked collection, or only those currently selected in the list.
		  </td>
        </tr>
        <tr>
          <td valign="top"><b>Copy Scores Values</b></td>
          <td valign="baseline">If the current result set includes
	  score columns from similarity, structure, or sequences searches,
	  they can be included in the output database.
		  </td>
        </tr>
        <tr>
          <td valign="top"><b>File</b></td>
          <td valign="baseline">Specify the path and file name of the database
		  to be saved.</td>
        </tr>
       </table>
    </td>
  </tr>
  <tr>
    <td valign="top"><nobr><b>PLIF</b></nobr></td>
    <td valign="baseline">Open the <a href="../apps/plif.htm">Protein Ligand 
	Interaction Fingerprints (PLIF) tool</a> for the current database.
	The subset of entries currently shown in the search panel list will
	be preselected in the PLIF tool.
       <p align="center">
         <img src="projsrch/plif.png"  width="869px" alt="PLIF panel" />
       </p>
    </td>
  </tr>
  <tr>
    <td valign="top"><nobr><b>Stats</b></nobr></td>
    <td valign="baseline">Open the <b>Project Database Statistics</b> panel
	for the currently displayed results.
	Statistics for each of the result columns are calculated and displayed
	as a pie chart or as a table of results, depending on whether <i>Chart</i>
	or <i>Table</i> is selected from the panel drop-down.
	<p>
	To only show the statistics of the currently selected entries in the
	Project Search panel list, enable the <b>Selected Only</b> checkbox.
	The statistics panel is synchronized with the Project Search panel: any
	changes in the search result set list or its selection is immediately
	reflected in the Statictics panel.
	</p>
	<p>
	Press <b>Export</b> to store the chart in commonly used
	graphics formats. To copy/export the numerical data from the <i>Table</i>
	view, select the table items of interest and copy them onto the clipboard
	by opening the list popup menu with a right mouse click and choose the
	option <b>Copy To Clipboard</b>.
	</p>
       <p><span class="floatleft">
         <img src="projsrch/dbStats.png"  width="389px" alt="Statistics panel" />
       </span>
       <span class="floatright">
         <img src="projsrch/dbStatsUID.png"  width="389px" 
		 alt="Statistics panel UID" />
       </span></p>
	   <p style="clear:both">The final set of data that can be displayed 
	   in the Project Database Statistics panel
	   is the <i>UID</i>. This is not a column of data from the displayed results.
	   Rather, it represents a specific position in the alignment of the 
	   reference structure. <i>UID</i> displays the residue statistics 
	   of all the displayed results for that
	   selected position. The position can be moved using the arrows in the
	   upper right of the panel, or a position number can be typed directly 
	   in the text box between the arrows.
	   </p>
    </td>
  </tr>
  <tr>
    <td valign="top"><nobr><b>Browse</b></nobr></td>
    <td valign="baseline">Open the <a href="../mdb/dbbrowse.htm">Database 
	Browser</a> for the currently displayed results. The Project Search
	browser shows in the MOE main window the annotated pocket region of the
	currrent entry and the 2D diagram of the ligand in the Browser panel.
	Additionally, the entry's <i>Compound name</i>, <i>Code</i>, and
	<i>Species</i> are listed. If available, group and family information
	is automatically selected to be shown in the configurable data field
	section, where one can select to show additional data fields of the
	database.
	<p>
	The Database Browser panel is also synchronized with the Project
	Search panel and the set of browsed entries will be adjusted to the
	result set list in the search panel. If entries are selected in
	the list, one can enable to <b>Browse Only Selected</b> in the 
	Browser panel.
	</p>
       <p align="center">
         <img src="projsrch/dbBrowse.png"  width="470px" alt="Database browser" />
       </p>
    </td>
  </tr>
  <tr>
    <td valign="top"><nobr><b>Close</b></nobr></td>
    <td valign="baseline">Close the Project Search panel without 
	saving any information. <br />
	<span class="warning">Note:</span> When this
	button is pressed, the panel closes immediately, without any confirmation.
    </td>
  </tr>
</table>


<!-- =============================== FOOTER =============================== -->

<a name="SeeAlso"></a>
<h2>See Also</h2>
<p>
<a href="kinase.htm">Kinase Search</a><br />
<a href="prosearch.htm">Homology Searching</a><br />
<a href="promodel.htm">Building Homology Models</a><br />
<a href="../apps/progeom.htm">Protein Geometry</a><br />
<a href="align.htm">Sequence and Structure Alignment</a><br />
<a href="pdb_mdb.htm">Protein Sequence and Structure Data</a>
</p>

<noscript>
	<hr noshade="noshade" />
	<font size="2"> Copyright &copy; 1997&ndash;2015
	<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a> </font>
</noscript>
<div id="MoeFooter"></div>
</body>
</html>
