<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!	propred.htm	Protein Sequence-based Property Prediction
!!-->

<!--
!!    MOE Online Manuals
!!    COPYRIGHT (C) 1997-2015
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

<script type="text/javascript" 
src="../include/jsincludes_moe.js"></script>

<link rel="stylesheet" type="text/css"
href="../include/manstyle.css" />


<title>Secondary Structure and Solvent Accessibility Prediction</title>
</head>

<body>
<div id="MoeHeader"></div>
<noscript>
	<span class="warning">Warning: JavaScript is disabled. This page will not display correctly.</span>
	<h1 class="title">MOE Documentation</h1>
	<hr noshade="noshade" />
	<style>.LaTeX {color: #C08080;}</style>
</noscript>

<ul>
<LI><A HREF="#Introduction">Introduction</A>
<LI><A HREF="#Methodology">Methodology</A>
<LI><A HREF="#PredictStructure">Predicting Secondary Structure</A>
<LI><A HREF="#PredictAccessibility">Predicting Solvent Accessibility</A>
<li><A HREF="#SVLCommands">SVL Commands</A>
<LI><A HREF="#References">References</A>
</ul>


<a name="Introduction"><h2>Introduction</h2>

<p>The MOE Sequence Editor provides commands for predicting secondary
structure and solvent accessibility from  primary structure information,
i.e. amino acid sequences.  Predictions are based on an
implementation of the Bayesian prediction formalism of Thompson &
Goldstein [Thompson&nbsp;1997], in which probability distributions are
directly estimated from a non-redundant training set of solved protein
structures.

<p>The following two commands display probabilities in  
residue scalar plots in the Sequence Editor:
<ul>
<li><b>SE | Annotate| Plot | Structure Propensity</b>
<li><b>SE | Annotate| Plot | Surface Accessibility</b>
</ul>

<p>Whereas the <b>SE | 2o Structure | Predict</b>
command assigns secondary structure labels to residues of selected chains.

<p>The following commands color residues in the Sequence Editor based on
their structure propensities. A darker color indicates a higher propensity
for a particular secondary structure.
<ul>
<li><b>SE | Annotate | Propensity | Helix</b>
<li><b>SE | Annotate | Propensity | Strand</b>
<li><b>SE | Annotate | Propensity | None</b>
</ul>


<p>All commands are located in the 
<a href="../moe/gui/se.htm">Sequence Editor</a>.


<a name="Methodology"><h2>Methodology</h2>
<p>In order to calculate the conditional probability of observing a particular
structural class <em>k</em> at residue site <em>i</em>, given the amino
acid sequence in a window around i of size <em>j</em>, Thompson <em>et al</em>
perform a Bayes reversal :

<p><center><img src="propred/propred1.gif"></center>

<p>In principle, one need only estimate two probabilities to predict
a class at each location in a sequence:

<ol>
<li>Conditional probabilities of each amino acid sequence of the 
chosen window size for each class type of interest 
<li>Independent probabilities of each class type
</ol>

<p>Both of these could be estimated by
counting up the relevant observations in a training set.

<p>In fact, the data sets are far too small to reliably estimate the
required distributions.
Therefore, the amino acid identity at each
site in a segment is assumed to be dependent only on the structure at that
site. Thus we have:

<p><center><img src="propred/propred2.gif"></center>

<p> In order to estimate these distributions, the number of instances of
each structural class and the number of instances of each amino acid,
at each window position, for each structural class were counted.

<p>An important element of this approach was the choice of the structural
classes which were counted. Each residue in the training set was classified
according to secondary structure type and solvent accessibility class.

<p>
Four secondary structure classes were defined, based on Kabsch and Sander
[Kabsch&nbsp;1983] assignments: <i>none</i>, <i>turn</i> (3-turn, 4-turn, bend,
3-10-helix), <i>helix</i>, (alpha helix, pi-helix) and <i>strand</i> (strand,
anti-strand).

<p>
Solvent accessibility classes were defined relative to values calculated for
all residue types in a GLY-X-GLY tripeptide (both backbone and sidechain
conformations being fully extended).  Those reference values were:

<pre>
    ALA:137.280 ARG:270.064 ASN:172.658 ASP:176.658 CYS:156.051
    GLN:199.846 GLU:207.425 GLY:100.309 HIS:219.886 ILE:222.251
    LEU:226.904 LYS:235.474 MET:244.611 PHE:262.057 PRO:171.714
    SER:145.820 THR:174.424 TRP:311.039 TYR:264.295 VAL:195.398
</pre>

<p>
Three solvent accessibility classes were defined : <i>buried</i> (less than
nine percent of reference value, <i>partially-exposed</i> (less than 36% of the
reference value) and <i>exposed</i>.

<a name="PredictStructure"><h2>Predicting Secondary Structure</h2>

<p>
The Predicted Secondary Structure command in the Measure menu of the Sequence
Editor calculates, for each amino acid residue in the system, the probability
of its being in one of three structural states:

<ul>
<li>Helix
<li>Extended (strand or antistrand) 
<li>None 
</ul>

<p>The following picture shows predicted secondary structure
probabilities. In this
example, the display of the <font face="Arial, Helvetica, sans-serif">predicted:none</font> plot has been turned off.
Note that the probabilities are scaled to values between 
one and negative one.  

<p align=center><img src="propred/pred2.png" alt="Predicted Secondary Structure"></p>

<p>The statistics upon which the predictive model
was based employed DSSP secondary structure assignments
[Kabsch&nbsp;1983], which is the automatic secondary structure
assignment algorithm built into MOE.

<p>For its part, the Predict Secondary Structure command in the Homology menu
assigns, to each amino acid residue of any selected chain, the most probable 
secondary structure label.

<a name="PredictAccessibility"><h2>Predicting Solvent Accessibility</h2>

<p>The Predicted Solvent Accessibility command from the Measure menu 
calculates the probability,
for each amino acid in the system,
that it will fall into one of two solvent accessibility 
classes:
<ul>
<li>Buried
<li>Exposed
</ul>

<p>Choose one of the scalars (1-3) to plot Predicted Solvent Accessibility
values for residues. Only the probability of the residue being in the buried
class will be plotted.  The plot legend at the bottom of the Sequence Editor
will indicate the color used to indicate the predicted probability. The plot
colors are assigned to scalars in the Measure menu.

<p>In the training regime for this command, a residue was classified
as buried if its solvent accessible surface area was less than 9%
of the value calculated for that residue in a GLY-X-GLY tripeptide
(see <a href="#Methodology">Methodology</a>).

<p>The following screen image displays the probability of a residue being
buried, along with a calculation of the actual solvent accessible surface of
each residue.

<p align=center>
<img src="propred/propred3.png" alt="Predicted Solvent Accessibility">
</p>


<h2> <a name="SVLCommands">SVL Commands</a> </h2>

<tt><a href="fcnref/pro_pred.htm">pro_Predict2Struct</a><br>
<a href="fcnref/pro_pred.htm">pro_PredictAccessibility</tt></a>

<p><a href="fcnref/intropro.htm">Protein Functions</a>


<h2> <a name="References">References</a> </h2>

<table width="100%" cellpadding="5" border="0">

<tr><td valign="top">[Kabsch&nbsp;1983]
    <td valign="top">
    Kabsch, W. and Sander, C.;  Dictionary of Protein Secondary Structures:
    Pattern Recognition of Hydrogen-Bonded and Geometrical
    Features; <i>Biopolymers 22</i> (<b>1983</b>) 2577&ndash;2637. </tr>

<tr><td valign="top">[Thompson&nbsp;1996]
    <td valign="top">
    Thompson, M.J. and Goldstein, R.A.;  Predicting Solvent Accessibility:
    Higher Accuracy Using Bayesian Statistics and Optimized Residue
    Substitution Classes; <i>Proteins 25</i> (<b>1996</b>) 38&ndash;47. </tr>

<tr><td valign="top">[Thompson&nbsp;1997]
    <td valign="top">
    Thompson, M.J. and Goldstein, R.A.; Predicting Protein Secondary Structure
    with Probabilistic Schemata of Evolutionarily Derived
    Information; <i>Protein Science 6</i> (<b>1997</b>) 1963&ndash;1975.  </tr>
</table>


<noscript>
	<hr noshade="noshade" />
	<font size="2"> Copyright &copy; 1997&ndash;2015
	<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a> </font>
</noscript>
<div id="MoeFooter"></div>
</body>
</html>
