define({'/Maestro11_Handbook.html':{i:[0],t:['Maestro Quick Reference Guide'],b:['']},'/QM_for_beginners.pdf':{i:[1508],t:['QM for Beginners'],b:['']},'/academy_tutorials/SA-1.html':{i:[3],t:['Introduction to Structure Preparation and Visualization'],b:['']},'/adme_tox_help/amphiphilic_moment.html':{i:[4809],t:['Amphiphilic Moment Panel'],b:['']},'/adme_tox_help/membrane_permeability.html':{i:[4810],t:['Membrane Permeability Panel'],b:['']},'/adme_tox_help/p450_examine_results.html':{i:[4811],t:['P450 Site of Metabolism - Examine Results Panel'],b:['']},'/adme_tox_help/p450_perform_calculation.html':{i:[4812],t:['P450 Site of Metabolism - Perform Calculation Panel'],b:['']},'/adme_tox_user_manual/adme_tox_intro.htm':{i:[1293],t:['Physics-Based ADME/Tox Introduction'],b:['#adme_tox_intro_2185511463_741933']},'/adme_tox_user_manual/amphiphilic_moment.htm':{i:[1299],t:['Amphiphilic Moment'],b:['#amphiphilic_moment_1814732899_741933']},'/adme_tox_user_manual/cyp450som.htm':{i:[1294],t:['P450 Site of Metabolism'],b:['#cyp450som_2053449233_743850']},'/adme_tox_user_manual/cyp450som_methodology.htm':{i:[1295],t:['Methodology for P450 Site of Metabolism'],b:['#cyp450som_2053449233_743854']},'/adme_tox_user_manual/cyp450som_results.htm':{i:[1297],t:['Examining the P450 Site of Metabolism Results'],b:['#cyp450som_2053449233_743889']},'/adme_tox_user_manual/cyp450som_running.htm':{i:[1296],t:['Running the P450 Site of Metabolism Calculation'],b:['#cyp450som_2053449233_743860']},'/adme_tox_user_manual/membrane_permeability.htm':{i:[1298],t:['Membrane Permeability'],b:['#membrane_permeability_497308932_741933']},'/autoqsar_help/autoqsar.html':{i:[4089],t:['AutoQSAR Panel'],b:['']},'/autoqsar_help/autoqsar_advanced_options.html':{i:[4090],t:['AutoQSAR - Advanced Options Dialog Box'],b:['']},'/autoqsar_help/autoqsar_properties.html':{i:[4092],t:['AutoQSAR Output Properties'],b:['']},'/autoqsar_help/autoqsar_validation_set.html':{i:[4091],t:['Select Validation Set Dialog Box'],b:['']},'/autoqsar_tutorial/autoqsar_tutorial.htm':{i:[5],t:['Creating and Applying Predictive QSAR Models with AutoQSAR'],b:['']},'/autoqsar_tutorial/autoqsar_tutorial_categorical.htm':{i:[13],t:['Creating a Categorical QSAR Model for Mutagenicity'],b:['']},'/autoqsar_tutorial/autoqsar_tutorial_categorical_analyzing.htm':{i:[15],t:['Analyzing a Categorical QSAR Model for Mutagenicity '],b:['']},'/autoqsar_tutorial/autoqsar_tutorial_categorical_creating.htm':{i:[14],t:['Creating a QSAR Model for Categorical Mutagenicity Data'],b:['']},'/autoqsar_tutorial/autoqsar_tutorial_categorical_predicting.htm':{i:[16],t:['Making Predictions from a Categorical QSAR Model for Mutagenicity'],b:['']},'/autoqsar_tutorial/autoqsar_tutorial_categorical_results.htm':{i:[17],t:['Analyzing Predictions from a Categorical QSAR Model for Mutagenicity'],b:['']},'/autoqsar_tutorial/autoqsar_tutorial_copying.htm':{i:[6],t:['Copying the AutoQSAR Tutorial Files'],b:['']},'/autoqsar_tutorial/autoqsar_tutorial_numeric.htm':{i:[7],t:['Creating a Numeric QSAR Model for Binding Affinity'],b:['']},'/autoqsar_tutorial/autoqsar_tutorial_numeric_analyzing.htm':{i:[10],t:['Analyzing a QSAR Model for Binding Affinity'],b:['']},'/autoqsar_tutorial/autoqsar_tutorial_numeric_creating.htm':{i:[9],t:['Creating a QSAR Model for Binding Affinity'],b:['']},'/autoqsar_tutorial/autoqsar_tutorial_numeric_predicting.htm':{i:[11],t:['Making Predictions from a QSAR Model for Binding Affinity'],b:['']},'/autoqsar_tutorial/autoqsar_tutorial_numeric_preparing.htm':{i:[8],t:['Preparing Data for QSAR Modeling'],b:['']},'/autoqsar_tutorial/autoqsar_tutorial_numeric_results.htm':{i:[12],t:['Analyzing Predictions from a QSAR Model for Binding Affinity and Identifying Compounds Outside the Domain of Model Applicability'],b:['']},'/bioluminate_help/aggregation_profiles.html':{i:[4094],t:['Aggregation Profiles Dialog Box'],b:['']},'/bioluminate_help/aggregation_surface.html':{i:[4096],t:['Aggregation Surface Panel'],b:['']},'/bioluminate_help/aggregation_surface_options.html':{i:[4095],t:['Aggregation Surface - Options Dialog Box'],b:['']},'/bioluminate_help/antibody_cdr_grafting.html':{i:[4099],t:['Antibody Humanization: CDR Grafting Panel'],b:['']},'/bioluminate_help/antibody_cdr_grafting_import_seq.html':{i:[4109],t:['Import Framework Sequences Dialog Box'],b:['']},'/bioluminate_help/antibody_cdr_grafting_options.html':{i:[4103],t:['CDR Grafting - Advanced Options Dialog Box'],b:['']},'/bioluminate_help/antibody_database.html':{i:[4098],t:['Antibody Database Management Panel'],b:['']},'/bioluminate_help/antibody_database_options.html':{i:[4097],t:['Antibody Database - Advanced Options Dialog Box'],b:['']},'/bioluminate_help/antibody_mutation_homology_suggestions.html':{i:[4100],t:['Antibody Humanization: Homology Suggestions Dialog Box'],b:['']},'/bioluminate_help/antibody_prediction.html':{i:[4102],t:['Antibody Prediction Panel'],b:['']},'/bioluminate_help/antibody_residue_mutation.html':{i:[4101],t:['Antibody Humanization: Residue Mutation Panel'],b:['']},'/bioluminate_help/choose_database.html':{i:[4104],t:['Choose Database Dialog Box'],b:['']},'/bioluminate_help/consensus_visualization.html':{i:[4105],t:['Consensus Visualization Panel'],b:['']},'/bioluminate_help/crosslink_proteins.html':{i:[4106],t:['Crosslink Proteins Panel'],b:['']},'/bioluminate_help/cysteine_mutation.html':{i:[4107],t:['Cysteine Mutation Panel'],b:['']},'/bioluminate_help/homology_model.html':{i:[4108],t:['Homology Model Panel'],b:['']},'/bioluminate_help/insertion_deletion_loop_swap.html':{i:[4110],t:['Insertions, Deletions, and Loop Swaps Panel'],b:['']},'/bioluminate_help/loop_clusters.html':{i:[4111],t:['Loop Clusters Dialog Box'],b:['']},'/bioluminate_help/low_mode_sampling.html':{i:[4112],t:['Low Mode Vibrational Sampling Panel'],b:['']},'/bioluminate_help/mutation_results_viewer.html':{i:[4136],t:['Results Viewer Panel'],b:['']},'/bioluminate_help/nonstandard_residues.html':{i:[4113],t:['Non-Standard Residues Panel'],b:['']},'/bioluminate_help/peptide_docking.html':{i:[4114],t:['Peptide Docking Panel'],b:['']},'/bioluminate_help/peptide_helicity.html':{i:[4115],t:['Peptide Helicity Panel'],b:['']},'/bioluminate_help/peptide_qsar.html':{i:[4117],t:['Peptide QSAR Panel'],b:['']},'/bioluminate_help/peptide_qsar_kpls_options.html':{i:[4118],t:['Peptide QSAR — Advanced Options - KPLS Dialog Box'],b:['']},'/bioluminate_help/peptide_qsar_load.html':{i:[4116],t:['Peptide QSAR - Load Sequences and Observables Dialog Box'],b:['']},'/bioluminate_help/peptide_qsar_pls_options.html':{i:[4119],t:['Peptide QSAR — Advanced Options - PLS Dialog Box'],b:['']},'/bioluminate_help/peptide_qsar_results.html':{i:[4120],t:['Peptide QSAR — Results Tab'],b:['']},'/bioluminate_help/peptide_qsar_setup.html':{i:[4121],t:['Peptide QSAR — Setup Tab'],b:['']},'/bioluminate_help/protein_interaction_analysis.html':{i:[4122],t:['Protein Interaction Analysis Panel'],b:['']},'/bioluminate_help/protein_interaction_analysis_options.html':{i:[4123],t:['Protein Interaction Analysis — Advanced Options Dialog Box'],b:['']},'/bioluminate_help/protein_protein_docking.html':{i:[4129],t:['Protein-Protein Docking Panel'],b:['']},'/bioluminate_help/protein_structure_quality.html':{i:[4124],t:['Protein Structure Quality Viewer Panel'],b:['']},'/bioluminate_help/protein_surface_analyzer.html':{i:[4126],t:['Protein Surface Analyzer Panel'],b:['']},'/bioluminate_help/protein_surface_analyzer_settings.html':{i:[4125],t:['Protein Surface Analyzer - Settings Dialog Box'],b:['']},'/bioluminate_help/protein_titration_curve.html':{i:[4127],t:['Protein Titration Curve Panel'],b:['']},'/bioluminate_help/protein_titration_curve_settings.html':{i:[4128],t:['Protein Titration Curve Settings Dialog Box'],b:['']},'/bioluminate_help/reactive_residues.html':{i:[4130],t:['Reactive Protein Residues Panel'],b:['']},'/bioluminate_help/residue_analysis.html':{i:[4131],t:['Residue Analysis Panel'],b:['']},'/bioluminate_help/residue_loop_mutation.html':{i:[4135],t:['Residue and Loop Mutation Panel'],b:['']},'/bioluminate_help/residue_scanning.html':{i:[4134],t:['Residue Scanning Panel'],b:['']},'/bioluminate_help/residue_scanning_homology_suggestions.html':{i:[4133],t:['Residue Scanning - Homology Suggestions Dialog Box'],b:['']},'/bioluminate_help/residue_scanning_options.html':{i:[4132],t:['Residue Scanning - Advanced Options Dialog Box'],b:['']},'/bioluminate_help/toggle_table.html':{i:[4137],t:['Toggle Table Panel'],b:['']},'/bioluminate_quick_start/antibody_modeling_tut.htm':{i:[58],t:['Homology Modeling of an Antibody'],b:['#antibody_modeling_tut_2547520322_744641']},'/bioluminate_quick_start/antibody_modeling_tut_basic_models.htm':{i:[62],t:['Generating the Basic Loop Models for 1fsk'],b:['#antibody_modeling_tut_2547520322_750136']},'/bioluminate_quick_start/antibody_modeling_tut_cleanup.htm':{i:[66],t:['Cleaning Up after the Antibody Modeling Exercise'],b:['#antibody_modeling_tut_2547520322_752527']},'/bioluminate_quick_start/antibody_modeling_tut_filter_templates.htm':{i:[60],t:['Removing 1fsk from the Templates To Be Used'],b:['#antibody_modeling_tut_2547520322_749520']},'/bioluminate_quick_start/antibody_modeling_tut_humanize.htm':{i:[65],t:['Humanizing the 1fsk Antibody Model'],b:['#antibody_modeling_tut_2547520322_750613']},'/bioluminate_quick_start/antibody_modeling_tut_import.htm':{i:[59],t:['Importing the 1fsk Antibody Structure'],b:['#antibody_modeling_tut_2547520322_750016']},'/bioluminate_quick_start/antibody_modeling_tut_template_search.htm':{i:[61],t:['Searching for Framework Templates for 1fsk'],b:['#antibody_modeling_tut_2547520322_752422']},'/bioluminate_quick_start/antibody_modeling_tut_xray_structure_compare.htm':{i:[64],t:['Comparing the 1fsk Model with the X-Ray Structure'],b:['#antibody_modeling_tut_2547520322_750448']},'/bioluminate_quick_start/antibody_modeling_tut_xray_structure_prep.htm':{i:[63],t:['Preparing the 1fsk X-Ray Structure for Comparison'],b:['#antibody_modeling_tut_2547520322_750352']},'/bioluminate_quick_start/antigen_docking_tut.htm':{i:[76],t:['Antibody-Antigen Docking with 1fsk'],b:['#antigen_docking_tut_169328780_754402']},'/bioluminate_quick_start/antigen_docking_tut_antibody_prep.htm':{i:[78],t:['Preparing the 1fsk Antibody'],b:['#antigen_docking_tut_169328780_749326']},'/bioluminate_quick_start/antigen_docking_tut_cleanup.htm':{i:[81],t:['Cleaning Up after the Antibody-Antigen Docking Exercise'],b:['#antigen_docking_tut_169328780_754171']},'/bioluminate_quick_start/antigen_docking_tut_prep.htm':{i:[77],t:['Preparing for the Antibody-Antigen Docking Exercise'],b:['#antigen_docking_tut_169328780_753817']},'/bioluminate_quick_start/antigen_docking_tut_reference_comparison.htm':{i:[80],t:['Comparing the Best Pose with the Reference'],b:['#antigen_docking_tut_169328780_749341']},'/bioluminate_quick_start/antigen_docking_tut_setup_and_run.htm':{i:[79],t:['Setting Up and Running the Job'],b:['#antigen_docking_tut_169328780_749173']},'/bioluminate_quick_start/bioluminate_tutorial.htm':{i:[19],t:['Protein Modeling with BioLuminate'],b:['#bioluminate_tutorial_3196872184_741933']},'/bioluminate_quick_start/consensus_waters_cleanup.htm':{i:[39],t:['Cleaning Up after the Consensus Water Exercise'],b:['#consensus_waters_tut_3047652980_750660']},'/bioluminate_quick_start/consensus_waters_examining.htm':{i:[38],t:['Examining the Consensus Waters'],b:['#consensus_waters_tut_3047652980_750485']},'/bioluminate_quick_start/consensus_waters_finding.htm':{i:[37],t:['Finding Consensus Waters from Homologs'],b:['#consensus_waters_tut_3047652980_750532']},'/bioluminate_quick_start/consensus_waters_proteinprep.htm':{i:[36],t:['Preparing the Protein for Consensus Water Display'],b:['#consensus_waters_tut_3047652980_751837']},'/bioluminate_quick_start/consensus_waters_tut.htm':{i:[35],t:['Identifying Consensus Waters'],b:['#consensus_waters_tut_3047652980_744641']},'/bioluminate_quick_start/cysteine_mutation_tut.htm':{i:[51],t:['Locating Possible Mutations for Disulfide Bridges in 1vhu'],b:['#cysteine_mutation_tut_1371472847_746989']},'/bioluminate_quick_start/cysteine_mutation_tut_cleanup.htm':{i:[57],t:['Cleaning Up after the Cysteine Mutation Exercise'],b:['#cysteine_mutation_tut_1371472847_753808']},'/bioluminate_quick_start/cysteine_mutation_tut_minimize.htm':{i:[54],t:['Minimizing the Mutated 1vhu Structure'],b:['#cysteine_mutation_tut_1371472847_753327']},'/bioluminate_quick_start/cysteine_mutation_tut_mutate_ala.htm':{i:[53],t:['Mutating Cysteines to Alanines in 1vhu'],b:['#cysteine_mutation_tut_1371472847_753171']},'/bioluminate_quick_start/cysteine_mutation_tut_proteinprep.htm':{i:[52],t:['Preparing the 1vhu Protein for Mutation'],b:['#cysteine_mutation_tut_1371472847_753078']},'/bioluminate_quick_start/cysteine_mutation_tut_results.htm':{i:[56],t:['Examining the 1vhu Cysteine Mutation Results'],b:['#cysteine_mutation_tut_1371472847_754668']},'/bioluminate_quick_start/cysteine_mutation_tut_searching.htm':{i:[55],t:['Searching for Mutations to Cysteine in the Mutated 1vhu'],b:['#cysteine_mutation_tut_1371472847_753375']},'/bioluminate_quick_start/homology_modeling_tut.htm':{i:[40],t:['Simple Homology Modeling of Proteins'],b:['#homology_modeling_tut_4015710100_744641']},'/bioluminate_quick_start/homology_modeling_tut_cleanup.htm':{i:[44],t:['Cleaning Up after the Homology Modeling Exercise'],b:['#homology_modeling_tut_4015710100_749174']},'/bioluminate_quick_start/homology_modeling_tut_compare_xray.htm':{i:[43],t:['Comparing the 2pcy Homology Model to the X-ray Structure'],b:['#homology_modeling_tut_4015710100_745813']},'/bioluminate_quick_start/homology_modeling_tut_examining_results.htm':{i:[42],t:['Examining 2pcy Homology Modeling Results'],b:['#homology_modeling_tut_4015710100_748928']},'/bioluminate_quick_start/homology_modeling_tut_setup.htm':{i:[41],t:['Setting Up and Running the Homology Modeling Job for 2pcy'],b:['#homology_modeling_tut_4015710100_748848']},'/bioluminate_quick_start/protein_docking_tut.htm':{i:[67],t:['Docking 1ba7 to 1qqu'],b:['#protein_docking_tut_2950702707_744641']},'/bioluminate_quick_start/protein_docking_tut_as_is.htm':{i:[69],t:['Docking the 1ba7 Ligand Protein As Is'],b:['#protein_docking_tut_2950702707_748845']},'/bioluminate_quick_start/protein_docking_tut_binding_sites.htm':{i:[74],t:['Examining the Poses and the Binding Sites from Protein-Protein Docking'],b:['#protein_docking_tut_2950702707_755939']},'/bioluminate_quick_start/protein_docking_tut_cleanup.htm':{i:[75],t:['Cleaning Up after the Protein-Protein Docking Exercise'],b:['#protein_docking_tut_2950702707_753781']},'/bioluminate_quick_start/protein_docking_tut_comparing_poses.htm':{i:[72],t:['Comparing the Top Poses From Protein-Protein Docking'],b:['#protein_docking_tut_2950702707_748884']},'/bioluminate_quick_start/protein_docking_tut_native_ligand.htm':{i:[71],t:['Docking the Native 1avx Ligand to the Native 1avx Receptor'],b:['#protein_docking_tut_2950702707_747626']},'/bioluminate_quick_start/protein_docking_tut_preparation.htm':{i:[68],t:['Preparing for the Protein-Protein Docking Exercise'],b:['#protein_docking_tut_2950702707_753832']},'/bioluminate_quick_start/protein_docking_tut_prepared_protein.htm':{i:[70],t:['Docking the Prepared 1ba7 Ligand Protein'],b:['#protein_docking_tut_2950702707_748854']},'/bioluminate_quick_start/protein_docking_tut_rmsd.htm':{i:[73],t:['Calculating the RMSD Between the Top Poses from Protein-Protein Docking'],b:['#protein_docking_tut_2950702707_748389']},'/bioluminate_quick_start/reactive_residues_tut.htm':{i:[31],t:['Identifying Reactive Residues in 2pcy'],b:['#reactive_residues_tut_3387413851_744641']},'/bioluminate_quick_start/reactive_residues_tut_cleanup.htm':{i:[34],t:['Cleaning Up after the Reactive Residues Exercise'],b:['#reactive_residues_tut_3387413851_748546']},'/bioluminate_quick_start/reactive_residues_tut_identifying.htm':{i:[33],t:['Identifying the Reactive Residues'],b:['#reactive_residues_tut_3387413851_748506']},'/bioluminate_quick_start/reactive_residues_tut_proteinprep.htm':{i:[32],t:['Preparing the Protein'],b:['#reactive_residues_tut_3387413851_749016']},'/bioluminate_quick_start/residue_scanning_tut.htm':{i:[45],t:['Scanning for Residue Mutations'],b:['#residue_scanning_tut_2687493990_744641']},'/bioluminate_quick_start/residue_scanning_tut_cleanup.htm':{i:[50],t:['Cleaning Up after the Residue Scanning Exercise'],b:['#residue_scanning_tut_2687493990_750525']},'/bioluminate_quick_start/residue_scanning_tut_examine_results.htm':{i:[49],t:['Examining the Results'],b:['#residue_scanning_tut_2687493990_750517']},'/bioluminate_quick_start/residue_scanning_tut_options_job.htm':{i:[48],t:['Setting Options and Running the Job'],b:['#residue_scanning_tut_2687493990_750198']},'/bioluminate_quick_start/residue_scanning_tut_proteinprep.htm':{i:[46],t:['Preparing the 1brs Protein'],b:['#residue_scanning_tut_2687493990_750068']},'/bioluminate_quick_start/residue_scanning_tut_set_up_mutations.htm':{i:[47],t:['Setting Up the Mutations'],b:['#residue_scanning_tut_2687493990_750146']},'/bioluminate_quick_start/toggle_table_cleanup.htm':{i:[25],t:['Cleaning Up after the Exercises'],b:['#toggle_table_tut_738302857_749710']},'/bioluminate_quick_start/toggle_table_import.htm':{i:[21],t:['Importing a Structure'],b:['#toggle_table_tut_738302857_749517']},'/bioluminate_quick_start/toggle_table_selection.htm':{i:[24],t:['Performing Actions on Selected Atoms'],b:['#toggle_table_tut_738302857_747533']},'/bioluminate_quick_start/toggle_table_structure.htm':{i:[23],t:['Performing Actions on the Entire Structure'],b:['#toggle_table_tut_738302857_747221']},'/bioluminate_quick_start/toggle_table_tut.htm':{i:[20],t:['Using the Toggle Table'],b:['#toggle_table_tut_738302857_744641']},'/bioluminate_quick_start/toggle_table_view.htm':{i:[22],t:['Changing the View'],b:['#toggle_table_tut_738302857_747792']},'/bioluminate_quick_start/viewing_sequences_aligning.htm':{i:[28],t:['Aligning Sequences to 3max'],b:['#viewing_sequences_tut_773466387_749529']},'/bioluminate_quick_start/viewing_sequences_annotating.htm':{i:[29],t:['Annotating Sequences'],b:['#viewing_sequences_tut_773466387_749811']},'/bioluminate_quick_start/viewing_sequences_cleanup.htm':{i:[30],t:['Cleaning Up after Examining Sequences'],b:['#viewing_sequences_tut_773466387_749843']},'/bioluminate_quick_start/viewing_sequences_import.htm':{i:[27],t:['Importing Chains for Three Proteins'],b:['#viewing_sequences_tut_773466387_749406']},'/bioluminate_quick_start/viewing_sequences_tut.htm':{i:[26],t:['Examining Sequences'],b:['#viewing_sequences_tut_773466387_744641']},'/bioluminate_user_manual/antibody_modeling.htm':{i:[513],t:['Antibody Modeling'],b:['#antibody_modeling_3917355573_741933']},'/bioluminate_user_manual/antibody_modeling_databases.htm':{i:[536],t:['Antibody Databases'],b:['#antibody_modeling_3917355573_746195']},'/bioluminate_user_manual/antibody_modeling_humanize_grafting.htm':{i:[535],t:['Humanizing Antibodies by CDR Grafting'],b:['#antibody_modeling_3917355573_750947']},'/bioluminate_user_manual/antibody_modeling_humanize_mutation.htm':{i:[524],t:['Humanizing Antibodies by Residue Mutation'],b:['#antibody_modeling_3917355573_742127']},'/bioluminate_user_manual/antibody_modeling_humanize_mutation_criteria.htm':{i:[529],t:['Setting Up Residue Selection Criteria for Antibody Humanization'],b:['#antibody_modeling_3917355573_747244']},'/bioluminate_user_manual/antibody_modeling_humanize_mutation_homologs.htm':{i:[527],t:['Finding Homologs for Antibody Humanzation'],b:['#antibody_modeling_3917355573_746694']},'/bioluminate_user_manual/antibody_modeling_humanize_mutation_importing.htm':{i:[526],t:['Importing the Antibody for Humanization'],b:['#antibody_modeling_3917355573_746690']},'/bioluminate_user_manual/antibody_modeling_humanize_mutation_properties.htm':{i:[531],t:['Selecting Properties to Calculate for Antibody Humanization'],b:['#antibody_modeling_3917355573_758292']},'/bioluminate_user_manual/antibody_modeling_humanize_mutation_refinement.htm':{i:[532],t:['Setting Refinement Options for Mutated Residues in Antibody Humanization'],b:['#antibody_modeling_3917355573_771042']},'/bioluminate_user_manual/antibody_modeling_humanize_mutation_regions.htm':{i:[528],t:['Selecting Regions to Mutate for Antibody Humanization'],b:['#antibody_modeling_3917355573_747156']},'/bioluminate_user_manual/antibody_modeling_humanize_mutation_results.htm':{i:[534],t:['Analyzing the Antibody Humanization Results'],b:['#antibody_modeling_3917355573_747726']},'/bioluminate_user_manual/antibody_modeling_humanize_mutation_running.htm':{i:[533],t:['Running the Mutation Job for Antibody Humanization'],b:['#antibody_modeling_3917355573_747568']},'/bioluminate_user_manual/antibody_modeling_humanize_mutation_selecting.htm':{i:[530],t:['Selecting the Residues and Their Mutations for Antibody Humanization'],b:['#antibody_modeling_3917355573_747294']},'/bioluminate_user_manual/antibody_modeling_humanize_mutation_summary.htm':{i:[525],t:['Summary of Antibody Humanization'],b:['#antibody_modeling_3917355573_775520']},'/bioluminate_user_manual/antibody_modeling_numbering.htm':{i:[537],t:['Antibody Numbering Schemes'],b:['#antibody_modeling_3917355573_759545']},'/bioluminate_user_manual/antibody_modeling_structure.htm':{i:[514],t:['Modeling an Antibody Structure'],b:['#antibody_modeling_3917355573_748307']},'/bioluminate_user_manual/antibody_modeling_structure_choosing_database.htm':{i:[517],t:['Choosing an Antibody Database'],b:['#antibody_modeling_3917355573_744807']},'/bioluminate_user_manual/antibody_modeling_structure_clusters.htm':{i:[522],t:['Selecting Clusters for an Antibody Loop'],b:['#antibody_modeling_3917355573_761408']},'/bioluminate_user_manual/antibody_modeling_structure_filtering_database.htm':{i:[519],t:['Filtering the Antibody Database by Property'],b:['#antibody_modeling_3917355573_745114']},'/bioluminate_user_manual/antibody_modeling_structure_fr_coords.htm':{i:[518],t:['Selecting the Coordinates for the Framework Region of the Antibody'],b:['#antibody_modeling_3917355573_743780']},'/bioluminate_user_manual/antibody_modeling_structure_generating_model.htm':{i:[520],t:['Generating the Antibody Loop Model'],b:['#antibody_modeling_3917355573_743805']},'/bioluminate_user_manual/antibody_modeling_structure_import_sequence.htm':{i:[516],t:['Importing the Antibody Sequence'],b:['#antibody_modeling_3917355573_749077']},'/bioluminate_user_manual/antibody_modeling_structure_model_options.htm':{i:[521],t:['Advanced Antibody Model Building Options'],b:['#antibody_modeling_3917355573_761406']},'/bioluminate_user_manual/antibody_modeling_structure_refining_loops.htm':{i:[523],t:['Refining Antibody Loops'],b:['#antibody_modeling_3917355573_744448']},'/bioluminate_user_manual/antibody_modeling_structure_summary.htm':{i:[515],t:['Antibody Homology Modeling Summary'],b:['#antibody_modeling_3917355573_775264']},'/bioluminate_user_manual/bioluminate_interface.htm':{i:[402],t:['The BioLuminate Interface'],b:['#bioluminate_interface_3614894846_741933']},'/bioluminate_user_manual/bioluminate_interface_main_window.htm':{i:[403],t:['The BioLuminate Main Window'],b:['#bioluminate_interface_3614894846_744092']},'/bioluminate_user_manual/bioluminate_interface_structures_projects.htm':{i:[404],t:['Structures and Projects in BioLuminate'],b:['#bioluminate_interface_3614894846_744085']},'/bioluminate_user_manual/bioluminate_interface_toggle_table.htm':{i:[405],t:['The BioLuminate Toggle Table'],b:['#bioluminate_interface_3614894846_744325']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu.htm':{i:[408],t:['The Toggle Table Action Menu'],b:['#bioluminate_interface_3614894846_744818']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_addh.htm':{i:[422],t:['Adding and Removing Hydrogens with the Toggle Table'],b:['#bioluminate_interface_3614894846_747187']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_appearance.htm':{i:[411],t:['Changing the Appearance of Structures with the Toggle Table'],b:['#bioluminate_interface_3614894846_745181']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_contacts.htm':{i:[412],t:['Displaying Polar Contacts with the Toggle Table'],b:['#bioluminate_interface_3614894846_744845']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_creating.htm':{i:[421],t:['Creating Project Entries with the Toggle Table'],b:['#bioluminate_interface_3614894846_747120']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_crystal_mates.htm':{i:[414],t:['Displaying Atoms Related By Crystallographic Symmetry with the Toggle Table'],b:['#bioluminate_interface_3614894846_744854']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_deleting.htm':{i:[420],t:['Deleting Entries from the Project with the Toggle Table'],b:['#bioluminate_interface_3614894846_747091']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_duplicating.htm':{i:[418],t:['Duplicating Rows with the Toggle Table'],b:['#bioluminate_interface_3614894846_744895']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_exclude.htm':{i:[419],t:['Removing Entries from the Workspace with the Toggle Table'],b:['#bioluminate_interface_3614894846_747053']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_minimize.htm':{i:[410],t:['Minimizing the Structure Energy from the Toggle Table'],b:['#bioluminate_interface_3614894846_745177']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_modifying_selecti.htm':{i:[415],t:['Modifying an Atom Selection with the Toggle Table'],b:['#bioluminate_interface_3614894846_746044']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_properties.htm':{i:[424],t:['Computing Properties with the Toggle Table'],b:['#bioluminate_interface_3614894846_747124']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_removing_selectio.htm':{i:[416],t:['Removing Selections with the Toggle Table'],b:['#bioluminate_interface_3614894846_744892']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_renaming.htm':{i:[417],t:['Renaming Rows with the Toggle Table'],b:['#bioluminate_interface_3614894846_747095']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_selection.htm':{i:[413],t:['Generating an Atom Selection with the Toggle Table'],b:['#bioluminate_interface_3614894846_745414']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_view.htm':{i:[409],t:['Changing the View with the Toggle Table'],b:['#bioluminate_interface_3614894846_745102']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_action_menu_waters.htm':{i:[423],t:['Removing Waters with the Toggle Table'],b:['#bioluminate_interface_3614894846_747233']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_color_menu.htm':{i:[428],t:['The Toggle Table Color Menu'],b:['#bioluminate_interface_3614894846_747355']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_hide_menu.htm':{i:[426],t:['The Toggle Table Hide Menu'],b:['#bioluminate_interface_3614894846_747357']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_label_menu.htm':{i:[427],t:['The Toggle Table Label Menu'],b:['#bioluminate_interface_3614894846_747356']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_quick_access.htm':{i:[406],t:['Quick Access Buttons in the Toggle Table'],b:['#bioluminate_interface_3614894846_744627']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_rows.htm':{i:[407],t:['Toggle Table Rows'],b:['#bioluminate_interface_3614894846_744675']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_shortcut_menus.htm':{i:[429],t:['Shortcut Menus in BioLuminate'],b:['#bioluminate_interface_3614894846_747370']},'/bioluminate_user_manual/bioluminate_interface_toggle_table_show_menu.htm':{i:[425],t:['The Toggle Table Show Menu'],b:['#bioluminate_interface_3614894846_744823']},'/bioluminate_user_manual/bioluminate_overview.htm':{i:[395],t:['Overview of BioLuminate'],b:['#bioluminate_overview_1304004408_741933']},'/bioluminate_user_manual/bioluminate_overview_antibody.htm':{i:[401],t:['Antibody Tools'],b:['#bioluminate_overview_1304004408_744376']},'/bioluminate_user_manual/bioluminate_overview_peptide.htm':{i:[398],t:['Peptide Tools'],b:['#bioluminate_overview_1304004408_747979']},'/bioluminate_user_manual/bioluminate_overview_protein_alignment.htm':{i:[399],t:['Protein Alignment and Docking Tools'],b:['#bioluminate_overview_1304004408_744407']},'/bioluminate_user_manual/bioluminate_overview_protein_mutation.htm':{i:[400],t:['Protein Mutation and Modification Tools'],b:['#bioluminate_overview_1304004408_744417']},'/bioluminate_user_manual/bioluminate_overview_protein_structure.htm':{i:[397],t:['Protein Structure Tools'],b:['#bioluminate_overview_1304004408_744402']},'/bioluminate_user_manual/bioluminate_references.htm':{i:[538],t:['BioLuminate References'],b:['#bioluminate_references_2456494482_741933']},'/bioluminate_user_manual/consensus_viewer.htm':{i:[434],t:['Identifying Consensus Molecules'],b:['#consensus_viewer_3366156288_744641']},'/bioluminate_user_manual/consensus_viewer_homologs.htm':{i:[436],t:['Finding and Aligning Homologs for Consensus Viewing'],b:['#consensus_viewer_3366156288_747923']},'/bioluminate_user_manual/consensus_viewer_results.htm':{i:[437],t:['Viewing Consensus Molecules'],b:['#consensus_viewer_3366156288_748019']},'/bioluminate_user_manual/consensus_viewer_target.htm':{i:[435],t:['Choosing the Target Protein for Consensus Viewing'],b:['#consensus_viewer_3366156288_747860']},'/bioluminate_user_manual/crosslink_proteins.htm':{i:[478],t:['Cross-Linking Proteins'],b:['#crosslink_proteins_3834381992_743724']},'/bioluminate_user_manual/crosslink_proteins_connection.htm':{i:[480],t:['Picking the Connection Residues for Cross-Linking Proteins'],b:['#crosslink_proteins_3834381992_752245']},'/bioluminate_user_manual/crosslink_proteins_job.htm':{i:[482],t:['Choosing Methods and Running the Protein Cross-Linking Job'],b:['#crosslink_proteins_3834381992_744955']},'/bioluminate_user_manual/crosslink_proteins_linkers.htm':{i:[481],t:['Defining the Linkers for Cross-Linking Proteins'],b:['#crosslink_proteins_3834381992_746035']},'/bioluminate_user_manual/crosslink_proteins_preparing.htm':{i:[479],t:['Preparing the Proteins for Cross-Linking'],b:['#crosslink_proteins_3834381992_752227']},'/bioluminate_user_manual/crosslink_proteins_results.htm':{i:[483],t:['Examining the Results of Protein Cross-Linking'],b:['#crosslink_proteins_3834381992_749451']},'/bioluminate_user_manual/cysteine_mutation.htm':{i:[506],t:['Locating Possible Mutations for Disulfide Bridges'],b:['#cysteine_mutation_1346260660_746989']},'/bioluminate_user_manual/cysteine_mutation_job.htm':{i:[511],t:['Running the Cysteine Mutation Job'],b:['#cysteine_mutation_1346260660_747820']},'/bioluminate_user_manual/cysteine_mutation_protein.htm':{i:[508],t:['Selecting and Analyzing the Protein for Cysteine Mutation'],b:['#cysteine_mutation_1346260660_748054']},'/bioluminate_user_manual/cysteine_mutation_relaxation.htm':{i:[510],t:['Relaxing the Structure Around the Cysteine Mutation Site'],b:['#cysteine_mutation_1346260660_748530']},'/bioluminate_user_manual/cysteine_mutation_residue_pairs.htm':{i:[509],t:['Choosing Residue Pairs to Mutate to or from Cysteine'],b:['#cysteine_mutation_1346260660_747833']},'/bioluminate_user_manual/cysteine_mutation_results.htm':{i:[512],t:['Examining the Cysteine Mutation Results'],b:['#cysteine_mutation_1346260660_748605']},'/bioluminate_user_manual/cysteine_mutation_summary.htm':{i:[507],t:['Cysteine Mutation Workflow Summary'],b:['#cysteine_mutation_1346260660_755626']},'/bioluminate_user_manual/homology_modeling.htm':{i:[468],t:['Homology Modeling of Proteins'],b:['#homology_modeling_1606914295_744641']},'/bioluminate_user_manual/overview_protein_analysis.htm':{i:[396],t:['Protein Analysis Tools'],b:['#bioluminate_overview_1304004408_744275']},'/bioluminate_user_manual/peptide_properties.htm':{i:[446],t:['Predicting Peptide Properties'],b:['#peptide_properties_3968443385_744641']},'/bioluminate_user_manual/peptide_properties_docking.htm':{i:[456],t:['Peptide Binding to a Receptor'],b:['#peptide_properties_3968443385_749188']},'/bioluminate_user_manual/peptide_properties_docking_job.htm':{i:[460],t:['Running the Peptide Docking Job'],b:['#peptide_properties_3968443385_758176']},'/bioluminate_user_manual/peptide_properties_docking_ligands.htm':{i:[458],t:['Specifying the Peptides to Dock'],b:['#peptide_properties_3968443385_758922']},'/bioluminate_user_manual/peptide_properties_docking_options.htm':{i:[459],t:['Setting Peptide Docking Options'],b:['#peptide_properties_3968443385_758229']},'/bioluminate_user_manual/peptide_properties_docking_receptor.htm':{i:[457],t:['Defining the Protein Receptor Region for Peptide Docking'],b:['#peptide_properties_3968443385_758180']},'/bioluminate_user_manual/peptide_properties_qsar.htm':{i:[448],t:['Peptide QSAR from Sequences'],b:['#peptide_properties_3968443385_749090']},'/bioluminate_user_manual/peptide_properties_qsar_applying.htm':{i:[452],t:['Applying the Peptide QSAR Model to Other Sequences'],b:['#peptide_properties_3968443385_749267']},'/bioluminate_user_manual/peptide_properties_qsar_building.htm':{i:[450],t:['Building a Peptide QSAR Model'],b:['#peptide_properties_3968443385_750189']},'/bioluminate_user_manual/peptide_properties_qsar_examining.htm':{i:[451],t:['Examining the Model'],b:['#peptide_properties_3968443385_749264']},'/bioluminate_user_manual/peptide_properties_qsar_input.htm':{i:[449],t:['Loading the Sequences and Properties'],b:['#peptide_properties_3968443385_749246']},'/bioluminate_user_manual/peptide_properties_qsar_options.htm':{i:[453],t:['Setting Peptide QSAR Model Options for PLS Methods'],b:['#peptide_properties_3968443385_749377']},'/bioluminate_user_manual/peptide_properties_qsar_options_KPLS.htm':{i:[455],t:['KPLS Options for Peptide QSAR'],b:['#peptide_properties_3968443385_749453']},'/bioluminate_user_manual/peptide_properties_qsar_options_PLS.htm':{i:[454],t:['PLS Options for Peptide QSAR'],b:['#peptide_properties_3968443385_754837']},'/bioluminate_user_manual/peptide_properties_stability.htm':{i:[447],t:['Alpha Helix Stability'],b:['#peptide_properties_3968443385_745959']},'/bioluminate_user_manual/protein_interface_viewing.htm':{i:[444],t:['Viewing Protein Interaction Properties in the Workspace'],b:['#protein_interfaces_591355133_746282']},'/bioluminate_user_manual/protein_interfaces.htm':{i:[440],t:['Analyzing Protein Interface Interactions'],b:['#protein_interfaces_591355133_741933']},'/bioluminate_user_manual/protein_interfaces_results.htm':{i:[443],t:['Protein Interface Analysis Results'],b:['#protein_interfaces_591355133_743884']},'/bioluminate_user_manual/protein_interfaces_running.htm':{i:[442],t:['Running the Analysis of the Protein Interface'],b:['#protein_interfaces_591355133_743839']},'/bioluminate_user_manual/protein_interfaces_selecting.htm':{i:[441],t:['Selecting the Interacting Proteins for Analysis'],b:['#protein_interfaces_591355133_743819']},'/bioluminate_user_manual/protein_low_modes.htm':{i:[445],t:['Locating Large-Scale Motions'],b:['#protein_low_modes_3368881234_749264']},'/bioluminate_user_manual/protein_protein_docking.htm':{i:[461],t:['Protein-Protein Docking'],b:['#protein_protein_docking_4223258233_744641']},'/bioluminate_user_manual/protein_protein_docking_antibody.htm':{i:[467],t:['Docking an Antigen to an Antibody'],b:['#protein_protein_docking_4223258233_749268']},'/bioluminate_user_manual/protein_protein_docking_constraints.htm':{i:[464],t:['Protein-Protein Docking Constraints'],b:['#protein_protein_docking_4223258233_748640']},'/bioluminate_user_manual/protein_protein_docking_multimers.htm':{i:[466],t:['Creating Dimers and Trimers in Protein-Protein Docking'],b:['#protein_protein_docking_4223258233_748653']},'/bioluminate_user_manual/protein_protein_docking_preparation.htm':{i:[462],t:['Preparation of Proteins for Protein-Protein Docking'],b:['#protein_protein_docking_4223258233_748440']},'/bioluminate_user_manual/protein_protein_docking_restraints.htm':{i:[465],t:['Protein-Protein Docking Distance Restraints'],b:['']},'/bioluminate_user_manual/protein_protein_docking_standard.htm':{i:[463],t:['Docking a Protein to Another Protein'],b:['#protein_protein_docking_4223258233_751370']},'/bioluminate_user_manual/protein_structure_quality.htm':{i:[430],t:['Analyzing Protein Quality'],b:['#protein_structure_quality_1022208676_744641']},'/bioluminate_user_manual/protein_structure_quality_ramachandran.htm':{i:[431],t:['Ramachandran Plot Tab'],b:['#protein_structure_quality_1022208676_751892']},'/bioluminate_user_manual/protein_structure_quality_report.htm':{i:[432],t:['Protein Report Tab'],b:['#protein_structure_quality_1022208676_750899']},'/bioluminate_user_manual/reactive_residues.htm':{i:[438],t:['Identifying Reactive Residues'],b:['#reactive_residues_2722908081_744641']},'/bioluminate_user_manual/reactive_residues_custom.htm':{i:[439],t:['Defining Custom Reactive Groups'],b:['#reactive_residues_2722908081_749231']},'/bioluminate_user_manual/residue_analysis.htm':{i:[433],t:['Analyzing Residue Properties'],b:['#residue_analysis_3638916663_744641']},'/bioluminate_user_manual/residue_loop_mutation.htm':{i:[469],t:['Residue and Loop Mutation'],b:['#residue_loop_mutation_513857013_744641']},'/bioluminate_user_manual/residue_loop_mutation_loops.htm':{i:[474],t:['Insertions, Deletions, and Loop Swaps with the Residue and Loop Mutation Panel'],b:['#residue_loop_mutation_513857013_748945']},'/bioluminate_user_manual/residue_loop_mutation_loops_choosing.htm':{i:[475],t:['Choosing the Original and Replacement Loop'],b:['#residue_loop_mutation_513857013_748571']},'/bioluminate_user_manual/residue_loop_mutation_loops_editing.htm':{i:[476],t:['Editing the Replacement Loop'],b:['#residue_loop_mutation_513857013_748640']},'/bioluminate_user_manual/residue_loop_mutation_loops_output.htm':{i:[477],t:['Choosing the Output and Method for Loop Replacement'],b:['#residue_loop_mutation_513857013_748643']},'/bioluminate_user_manual/residue_loop_mutation_residues.htm':{i:[470],t:['Residue Mutations with the Residue and Loop Mutation Panel'],b:['#residue_loop_mutation_513857013_748707']},'/bioluminate_user_manual/residue_loop_mutation_residues_mutation.htm':{i:[472],t:['Defining the Residue Mutation'],b:['#residue_loop_mutation_513857013_748953']},'/bioluminate_user_manual/residue_loop_mutation_residues_refining.htm':{i:[473],t:['Refining the Mutated Structure from Residue Mutation'],b:['#residue_loop_mutation_513857013_749063']},'/bioluminate_user_manual/residue_loop_mutation_residues_selecting.htm':{i:[471],t:['Selecting the Residue to Mutate'],b:['#residue_loop_mutation_513857013_747891']},'/bioluminate_user_manual/residue_scanning.htm':{i:[484],t:['Scanning for Residue Mutations'],b:['#residue_scanning_1293582990_744641']},'/bioluminate_user_manual/residue_scanning_command_line.htm':{i:[504],t:['Running Residue Scanning from the Command Line'],b:['#residue_scanning_1293582990_783242']},'/bioluminate_user_manual/residue_scanning_homologs.htm':{i:[488],t:['Using Homologs for Setting Up Mutations in Residue Scanning'],b:['#residue_scanning_1293582990_777307']},'/bioluminate_user_manual/residue_scanning_homologs_aligning.htm':{i:[491],t:['Aligning Homologs for Residue Scanning'],b:['#residue_scanning_1293582990_777331']},'/bioluminate_user_manual/residue_scanning_homologs_attributes.htm':{i:[493],t:['Selecting Residues by Structural Attributes for Residue Scanning'],b:['#residue_scanning_1293582990_777354']},'/bioluminate_user_manual/residue_scanning_homologs_blast.htm':{i:[489],t:['Obtaining Homologs from a BLAST Search for Residue Scanning'],b:['#residue_scanning_1293582990_777310']},'/bioluminate_user_manual/residue_scanning_homologs_importing.htm':{i:[490],t:['Importing Homologs for Residue Scanning'],b:['#residue_scanning_1293582990_777320']},'/bioluminate_user_manual/residue_scanning_homologs_selecting.htm':{i:[492],t:['Selecting Residues by Homology for Residue Scanning'],b:['#residue_scanning_1293582990_777345']},'/bioluminate_user_manual/residue_scanning_job.htm':{i:[498],t:['Running the Residue Scanning Job'],b:['#residue_scanning_1293582990_747899']},'/bioluminate_user_manual/residue_scanning_manual.htm':{i:[487],t:['Setting Up the Mutations Manually for Residue Scanning'],b:['#residue_scanning_1293582990_773377']},'/bioluminate_user_manual/residue_scanning_nonstandard_aa.htm':{i:[505],t:['Residue Scanning with Nonstandard Amino Acids'],b:['#residue_scanning_1293582990_772239']},'/bioluminate_user_manual/residue_scanning_options.htm':{i:[494],t:['Setting Options for Residue Scanning'],b:['#residue_scanning_1293582990_778339']},'/bioluminate_user_manual/residue_scanning_options_affinity.htm':{i:[496],t:['Setting Optimization Options for Affinity Maturation'],b:['#residue_scanning_1293582990_753998']},'/bioluminate_user_manual/residue_scanning_options_properties.htm':{i:[497],t:['Selecting Properties to Calculate for Residue Scanning'],b:['#residue_scanning_1293582990_761914']},'/bioluminate_user_manual/residue_scanning_options_refinement.htm':{i:[495],t:['Choosing Refinement Options for Residue Scanning'],b:['#residue_scanning_1293582990_778320']},'/bioluminate_user_manual/residue_scanning_preparation.htm':{i:[485],t:['Selecting and Preparing the Protein for Residue Scanning'],b:['#residue_scanning_1293582990_747418']},'/bioluminate_user_manual/residue_scanning_results.htm':{i:[499],t:['Examining the Mutation Results for Residue Scanning'],b:['#residue_scanning_1293582990_747349']},'/bioluminate_user_manual/residue_scanning_results_affinity.htm':{i:[500],t:['Binding Affinity Prediction from Residue Scanning'],b:['#residue_scanning_1293582990_749262']},'/bioluminate_user_manual/residue_scanning_results_complementarity.htm':{i:[503],t:['Surface Complementarity from Residue Scanning'],b:['#residue_scanning_1293582990_782031']},'/bioluminate_user_manual/residue_scanning_results_sasa.htm':{i:[502],t:['Solvent-Accessible Surface Areas from Residue Scanning'],b:['#residue_scanning_1293582990_763888']},'/bioluminate_user_manual/residue_scanning_results_stability.htm':{i:[501],t:['Stability Prediction from Residue Scanning'],b:['#residue_scanning_1293582990_749531']},'/bioluminate_user_manual/residue_scanning_type.htm':{i:[486],t:['Choosing the Residue Scanning Calculation Type'],b:['#residue_scanning_1293582990_747426']},'/canvas_command_reference/canvas_command.htm':{i:[1625],t:['Running Canvas Applications from the Command Line'],b:['#canvas_command_2298859567_743685']},'/canvas_command_reference/canvas_command_3dpharm.htm':{i:[1629],t:['3D Pharmacophore Fingerprint Programs'],b:['#canvas_command_2298859567_747029']},'/canvas_command_reference/canvas_command_canvas.htm':{i:[1646],t:['canvasSOM and canvasSOMBits'],b:['#canvas_command_2298859567_787959']},'/canvas_command_reference/canvas_command_canvasBayes.htm':{i:[1644],t:['canvasBayes'],b:['#canvas_command_2298859567_751336']},'/canvas_command_reference/canvas_command_canvasConvert.htm':{i:[1649],t:['canvasConvert'],b:['#canvas_command_2298859567_785931']},'/canvas_command_reference/canvas_command_canvasDBCS.htm':{i:[1631],t:['canvasDBCS'],b:['#canvas_command_2298859567_canvasdbcs']},'/canvas_command_reference/canvas_command_canvasFPGen.htm':{i:[1628],t:['canvasFPGen'],b:['#canvas_command_2298859567_796765']},'/canvas_command_reference/canvas_command_canvasFPHist.htm':{i:[1633],t:['canvasFPHist'],b:['#canvas_command_2298859567_779415']},'/canvas_command_reference/canvas_command_canvasFPMatrix.htm':{i:[1634],t:['canvasFPMatrix'],b:['#canvas_command_2298859567_777665']},'/canvas_command_reference/canvas_command_canvasJob.htm':{i:[1650],t:['canvasJob'],b:['#canvas_command_2298859567_751697']},'/canvas_command_reference/canvas_command_canvasKMeans.htm':{i:[1635],t:['canvasKMeans'],b:['#canvas_command_2298859567_747443']},'/canvas_command_reference/canvas_command_canvasKPLS.htm':{i:[1643],t:['canvasKPLS'],b:['#canvas_command_2298859567_784529']},'/canvas_command_reference/canvas_command_canvasLC.htm':{i:[1636],t:['canvasLC'],b:['#canvas_command_2298859567_768370']},'/canvas_command_reference/canvas_command_canvasLibOpt.htm':{i:[1632],t:['canvasLibOpt'],b:['#canvas_command_2298859567_785677']},'/canvas_command_reference/canvas_command_canvasMCS.htm':{i:[1654],t:['canvasMCS'],b:['#canvas_command_2298859567_766805']},'/canvas_command_reference/canvas_command_canvasMDS.htm':{i:[1638],t:['canvasMDS'],b:['#canvas_command_2298859567_750629']},'/canvas_command_reference/canvas_command_canvasMLR.htm':{i:[1639],t:['canvasMLR'],b:['#canvas_command_2298859567_750724']},'/canvas_command_reference/canvas_command_canvasMolDescriptors.htm':{i:[1640],t:['canvasMolDescriptors'],b:['#canvas_command_2298859567_750823']},'/canvas_command_reference/canvas_command_canvasNnet.htm':{i:[1645],t:['canvasNnet'],b:['#canvas_command_2298859567_786308']},'/canvas_command_reference/canvas_command_canvasPCAReg.htm':{i:[1641],t:['canvasPCAReg'],b:['#canvas_command_2298859567_canvaspcareg']},'/canvas_command_reference/canvas_command_canvasPLS.htm':{i:[1642],t:['canvasPLS'],b:['#canvas_command_2298859567_751052']},'/canvas_command_reference/canvas_command_canvasProjectDB.htm':{i:[1651],t:['canvasProjectDB'],b:['#canvas_command_2298859567_751797']},'/canvas_command_reference/canvas_command_canvasSDMerge.htm':{i:[1652],t:['canvasSDMerge'],b:['#canvas_command_2298859567_751700']},'/canvas_command_reference/canvas_command_canvasScaffold.htm':{i:[1655],t:['canvasScaffold'],b:['#canvas_command_2298859567_781560']},'/canvas_command_reference/canvas_command_canvasSearch.htm':{i:[1653],t:['canvasSearch'],b:['#canvas_command_2298859567_751701']},'/canvas_command_reference/canvas_command_canvas_app.htm':{i:[1648],t:['canvas_app'],b:['#canvas_command_2298859567_canvas_app']},'/canvas_command_reference/canvas_command_common.htm':{i:[1626],t:['Common Syntax Descriptions for Canvas Programs'],b:['#canvas_command_2298859567_765386']},'/canvas_command_reference/canvas_command_fingerprint.htm':{i:[1627],t:['2D Fingerprint Programs'],b:['#canvas_command_2298859567_743714']},'/canvas_command_reference/canvas_command_model.htm':{i:[1637],t:['Model Building and Related Application Programs'],b:['#canvas_command_2298859567_747126']},'/canvas_command_reference/canvas_command_scripting.htm':{i:[1656],t:['Scripting with Canvas'],b:['#canvas_command_2298859567_777478']},'/canvas_command_reference/canvas_command_similarity.htm':{i:[1630],t:['Similarity, Dissimilarity, and Clustering Programs'],b:['#canvas_command_2298859567_773639']},'/canvas_command_reference/canvas_command_utilities.htm':{i:[1647],t:['Canvas Utilities'],b:['#canvas_command_2298859567_786150']},'/canvas_help/3d_minimization.html':{i:[4139],t:['3D Minimization Dialog Box'],b:['']},'/canvas_help/3d_pharmacophore_fingerprints.html':{i:[4140],t:['3D Pharmacophore Fingerprints Dialog Box'],b:['']},'/canvas_help/3d_pharmacophore_fingerprints_advanced.html':{i:[4141],t:['3D Pharmacophore Fingerprints - Advanced Options Dialog Box'],b:['']},'/canvas_help/applications_menu.html':{i:[4142],t:['Applications Menu'],b:['']},'/canvas_help/apply_logic_to_view.html':{i:[4143],t:['Apply Logic to View Dialog Box'],b:['']},'/canvas_help/apply_to_master.html':{i:[4144],t:['Apply to Master Dialog Box'],b:['']},'/canvas_help/bayes_classification.html':{i:[4145],t:['Bayes Classification Dialog Box'],b:['']},'/canvas_help/bayes_classification_bins.html':{i:[4146],t:['Bayes Classification - Bins Dialog Box'],b:['']},'/canvas_help/bayes_classification_options.html':{i:[4147],t:['Bayes Classification - Advanced Options Dialog Box'],b:['']},'/canvas_help/binary_fingerprints_structures.html':{i:[4148],t:['Binary Fingerprints from Structures Dialog Box'],b:['']},'/canvas_help/calculator.html':{i:[4149],t:['Calculator Dialog Box'],b:['']},'/canvas_help/chart_menu.html':{i:[4150],t:['Chart Menu'],b:['']},'/canvas_help/choose_random_subset.html':{i:[4151],t:['Choose Random Subset Dialog Box'],b:['']},'/canvas_help/choosing_model.html':{i:[4152],t:['Choosing a Model'],b:['']},'/canvas_help/clustering_results.html':{i:[4153],t:['Clustering Results Panel'],b:['']},'/canvas_help/data_menu.html':{i:[4154],t:['Data Menu'],b:['']},'/canvas_help/details.html':{i:[4155],t:['Details Dialog Box'],b:['']},'/canvas_help/diversity_based_selection.html':{i:[4156],t:['Diversity-Based Selection Dialog Box'],b:['']},'/canvas_help/edit_menu.html':{i:[4157],t:['Edit Menu'],b:['']},'/canvas_help/edit_partition_db.html':{i:[4158],t:['Edit Partition Dialog Box'],b:['']},'/canvas_help/edit_structure.html':{i:[4159],t:['Edit Structure Dialog Box'],b:['']},'/canvas_help/edit_structure_name.html':{i:[4160],t:['Renaming Structures'],b:['']},'/canvas_help/export_options_db.html':{i:[4161],t:['Export Options Dialog Box'],b:['']},'/canvas_help/external_application.html':{i:[4162],t:['External Application Dialog Box'],b:['']},'/canvas_help/feature_selection.html':{i:[4163],t:['Feature Selection Dialog Box'],b:['']},'/canvas_help/feature_selection_correlation_matrix.html':{i:[4164],t:['Correlation Matrix Dialog Box'],b:['']},'/canvas_help/feature_selection_viewer.html':{i:[4165],t:['Feature Selection Viewer Panel'],b:['']},'/canvas_help/file_menu.html':{i:[4166],t:['File Menu'],b:['']},'/canvas_help/find_panel.html':{i:[4167],t:['Find Panel'],b:['']},'/canvas_help/getting_help.html':{i:[4168],t:['Getting Help in Canvas'],b:['']},'/canvas_help/hashed_fingerprints_options.html':{i:[4169],t:['Hashed Fingerprints - Advanced Options Dialog Box'],b:['']},'/canvas_help/heat_map.html':{i:[4170],t:['Heat Map Dialog Box'],b:['']}});