<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta http-equiv="Content-Style-Type" content="text/css">
<meta name="categories" content="tasks_menu ">
<meta name="product" content="{product}">
<meta name="profile" content="!BioLuminate">
<link rel="stylesheet" href="../support/help.css" charset="ISO-8859-1" type="text/css">
<title>Tasks Menu</title>
</head>
<script type="text/javascript">
function setTitle()
{
top.document.title = document.title + " - " + parent.parent.WINDOW_TITLE;
}
</script>
<body onload="setTitle();">
<table bgcolor="#dcdcdc" border="0" cellspacing="0" width="100%">
<tr><td>
<p><img src="../support/schrodinger_logo.gif" align="left" alt=""
 border="0" hspace="5" vspace="5" /></p>
</td></tr>
<tr><td>
<h1 class="title"><span class="schrored">Tasks Menu</span></h1>
</td></tr>
</table>


<p>The <span class="GUI">Tasks</span> menu provides acces to the tasks you can
run with Maestro Elements. 
</p>

<ul>
<li><p><span class="GUI">Minimize selected atoms</span>&mdash;Minimize the
energy of the selected atoms with respect to their positions, using the
OPLS_2005 force field. 
</p></li>
<li><p><span class="GUI">Measurements</span>&mdash;Measure distances, angles, or
dihedrals, and display them in the Workspace. The submenu has three items: <span
class="GUI">Pick to Measure</span>, which allows you to pick atoms in the
Workspace; <span class="GUI">Advanced Measurements</span>, which opens the  
<a href="../tools_menu/measurements_panel.html"><span
class="GUI">Measurements</span> panel</a> for more measurement features; and
<span class="GUI">Delete Measurements</span>, for deleting measurement-related
markers from the Workspace. The <span class="GUI">Pick to Measure</span> item
does the same as the toolbar button, and measures whatever type of measurement
you selected from the toolbar button menu.
</p></li>
<li><p><span class="GUI">H-Bonds and Contacts</span>&mdash;Show hydrogen bonds
or contacts in the Workspace, and change settings for hydrogen bonds and
contacts. Opens the <span class="GUI">H-Bonds and Contacts</span> panel<a
href="../tools_menu/hbonds_contacts.html">. You can also show H-bonds from the
toolbar.</a>
</p></li>
<li><p><span class="GUI">Superposition</span>&mdash;Superimpose structures, in
the <a href="../tools_menu/super_panel.html"><span class="GUI">Superposition</span> panel</a>.
</p></li>
<li><p><span class="GUI">Flexible Ligand Alignment</span>&mdash;Align ligands
with torsional rotation to find the best alignment, in the <a 
href="../tools_menu/flexible_ligand_alignment.html"><span class="GUI">Flexible
Ligand Alignment</span> panel</a>.
</p></li>
<li><p><span class="GUI">Ligand Interaction Diagram</span>&mdash;View a 2D 
diagram of the interactions between the ligand and the receptor that are in the
Workspace, in the <a href="../tools_menu/ligand_interaction_diagram.html"><span
class="GUI">Ligand Interaction Diagram</span> panel</a>.
</p></li>
<li><p><span class="GUI">Interactive Molecular Properties</span>&mdash;
</p></li>
<li><p><span class="GUI">Empirical pKa Prediction (Epik)</span>&mdash;Predict
the p<i>K</i><sub>a</sub> of molecules by a rule-based method, in the
<a href="../epik/epik_panel.html"><span class="GUI">Epik</span> panel</a>.
</p></li>
<li><p><span class="GUI">Ab Initio pKa Prediction (Jaguar)</span>&mdash;Predict
the p<i>K</i><sub>a</sub> of molecules by ab initio calculation with an
empirical correction, using Jaguar.
</p></li>
<li><p><span class="GUI">Torsion Scan</span>&mdash;Perform a rapid scan of a
torsion to view its energy profile with the OPLS_2005 force field, in the <a
href="../tools_menu/torsion_scan.html"><span class="GUI">Rapid Torsion
Scan</span> panel</a>.
</p></li>
<li><p><span class="GUI">Minimization</span>&mdash;Minimize the entire
workspace structure either in the gas phase or in a selected solvent, using the 
OPLS_2005 force field. 
</p></li>
<li><p><span class="GUI">Conformer Generation</span>&mdash;Generate conformers
of the Workspace structure, and add them to the project.
</p></li>
<li><p><span class="GUI">Ligand Tools</span>&mdash;Perform a range of tasks on
ligand-sized molecules. 
<ul>
<li><p><span class="GUI">Prepare Ligands (LigPrep)</span>&mdash;;Prepare
ligand-sized molecules for use in modeling, with <a
href="../ligprep/ligprep.html">LigPrep</a>. Includes 2D&rarr;3D conversion,
adjustment or expansion of ionization states, tautomers, ring conformations,
stereochemistry, and minimization.
</p></li>
<li><p><span class="GUI">Shape-Based Similarity Screening</span>&mdash;
</p></li>
<li><p><span class="GUI">Core Hopping</span>&mdash;Substitute the core of a
ligand for other cores, using alignment or shape to find similar cores, while
retaining the side chains. Opens the <span class="GUI">Ligand-Based Core
Hopping</span> panel.</p></li>
<li><p><span class="GUI">Isosteric Matching</span>&mdash;
</p></li>
<li><p><span class="GUI">R-Group Analysis</span>&mdash;Analyze the side chains
of a set of ligands and display properties related to these side chains, using 
the <a href="../tools_menu/r_group_analysis.html"><span class="GUI">R-Group
Analysis</span> panel.</a>
</p></li>
<li><p><span class="GUI">ADME Prediction (QikProp)</span>&mdash;Predict ADME
properties of molecules, in the <a href="../qikprop/qikprop.html"><span
class="GUI">QikProp</span> panel</a>.
</p></li>
<li><p><span class="GUI">Ligand Filtering</span>&mdash;Filter ligands based on
their properties, in the <a href="../tools_menu/ligfilter.html"><span
class="GUI">Ligand Filtering</span> panel</a>.
</p></li>
<li><p><span class="GUI">Merge Duplicate Ligands</span>&mdash;Merge properties
of duplicate ligands and write the merged records to a 2D SD file, a SMILES
file, or a CSV file with a SMILES string for the structure.
</p></li>
<li><p><span class="GUI">Electrostatic Potential</span>&mdash;Calculate the
electrostatic potential for the Workspace atoms, and return values as partial
charges, or a surface, or both.
</p></li>
</ul>
</p></li>
<li><p><span class="GUI">Protein Tools</span>&mdash;Perform a range of tasks on
protein molecules. The submenu has the following tools:
<ul>
<li><p><span class="GUI">Protein Preparation Wizard</span>&mdash;Prepare
proteins for use in modeling, with the 
<a href="../workflows/protein_preparation.html">Protein Preparation Wizard</a>.
</p></li>
</p></li>
<li><p><span class="GUI">Protein Structure Alignment</span>&mdash;Align two or
more protein structures, based on sequence and secondary structure in the 
<a href="../tools_menu/protein_structure_alignment.html"><span
class="GUI">Protein Structure Alignment</span> panel</a>.
</p></li> 
<li><p><span class="GUI">Align Binding Sites</span>&mdash;Align the binding
sites of two or more proteins, in the <a 
href="../tools_menu/align_binding_sites.html"><span class="GUI">Align Binding
Sites</span> panel</a>.
</p></li>
<li><p><span class="GUI">Protein Reports</span>&mdash;Display reports on protein
properties, such as geometric deviations from ideal values, G-factors, B-factors,
stereochemistry, planarity, missing atoms, in the 
<a href="../tools_menu/protein_reports_panel.html"><span class="GUI">Protein
Reports</span> panel</a>.
</p></li>
<li><p><span class="GUI">Ramachandran Plot</span>&mdash;Display <a
href="../tools_menu/ramachandran_panel.html">Ramachandran plot</a> of protein
dihedrals.
</p></li>
<li><p><span class="GUI">Binding Site Characterization
(SiteMap)</span>&mdash;Find potential binding sites on a protein, with <a
href="../glide/sitemap_panel.html">SiteMap</a>. 
</p></li>
<li><p><span class="GUI">Hydrophobic/philic Surface
Mapping</span>&mdash;Generate hydrophobic and hydrophilic surfaces for the
Workspace structure.
</p></li>
<li><p><span class="GUI">Multiple Sequence Viewer</span>&mdash;View and perform
tasks on multiple protein sequences, such as alignment, analysis, and homology
modeling, in the <a href="../tools_menu/multiple_sequence_viewer.html"><span
class="GUI">Multiple Sequence Viewer</span> panel</a>.
</p></li>
</ul>
</p></li>
<li><p><span class="GUI">Design and Scoring</span>&mdash;Design and score
ligands for a receptor binding site. The submenu has the following tasks:
<ul>
<li><p><span class="GUI">Docking and Lead Optimization</span>&mdash;Dock a
ligand to a receptor, then modify its structure (in 2D) and redock it, to
optimize its docking score.
</p></li>
<li><p><span class="GUI">Ligand Designer: Rigid Binding Site (uses Glide)</span>&mdash;Dock a
ligand to a receptor, then modify its structure in the Workspace and redock it,
to optimize its docking score.
</p></li>
<li><p><span class="GUI">Glide Grid Generation and Docking</span>&mdash;Dock one
or more ligands from the Project Table to a receptor. 
</p></li>
<li><p><span class="GUI">Filter Docked Poses</span>&mdash;Filter out docked
poses that do not match criteria for interaction with the receptor that you
specify.
</p></li>
</ul>
</p></li>
<li><p><span class="GUI">Pharmacophore Search (Phase)</span>&mdash;Create a
pharmacophore hypothesis for a ligand, and screen structures to match this
hypothesis.
</p></li>

<li><p><span class="GUI">Monitor Jobs</span>&mdash;Monitor jobs that you have
started, in the <a href="../misc/monitor_panel.html"><span
class="GUI">Monitor</span> panel</a>.
</p></li>

</ul>


<hr />
<table width="100%">
<td><p class="small"><a href="../support/legal_notice.html" target="LegalNoticeWindow">Legal Notice</a></p></td>
<tr><td>
<table align="right">
<tr><td><p class="small">
File: tasks_menu/tasks_menu.html<br />
Last updated: 07 Aug 2012
</p></td></tr>
</table>
</td></tr>
</table>
</body>
</html>
