<!DOCTYPE html>
<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd" lang="en-US" xml:lang="en-US" data-mc-search-type="Stem" data-mc-help-system-file-name="Documentation.xml" data-mc-path-to-help-system="../" data-mc-target-type="WebHelp2" data-mc-runtime-file-type="Topic" data-mc-preload-images="false" data-mc-in-preview-mode="false" data-mc-toc-path="Command References|Prime Command Reference|Prime Command Syntax">
    <head>
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <meta charset="utf-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <link href="../Skins/Default/Stylesheets/Slideshow.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/TextEffects.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Topic.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Components/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" /><title>sta—Single Template Alignment</title>
        <link href="../Resources/TableStyles/StandardRules.css" rel="stylesheet" data-mc-stylesheet-type="table" />
        <link href="../Resources/Stylesheets/docs.css" rel="stylesheet" />
        <script src="../Resources/Scripts/custom.modernizr.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/jquery.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.config.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/foundation.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/plugins.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/MadCapAll.js" type="text/javascript">
        </script>
    </head>
    <body><a name="aanchor6476"></a>
        <MadCap:concept term="Command Reference" /><a name="aanchor6477"></a>
        <MadCap:concept term="Prime" /><a name="prime_commands_1525853049_745390"></a><a name="prime_commands_1525853049_76040"></a>
        <table class="NavLink" data-mc-conditions="Default.ScreenOnly">
            <tr class="NavLink">
                <td class="NavLeft"><span class="NavText">◀ <a class="NavLink MCXref xref xrefNavLink" href="prime_commands_ssp.htm">ssp—Secondary Structure Prediction</a></span>
                </td>
                <td class="NavCenter"><span class="NavText"><a class="NavLink MCXref xref xrefNavLink" href="prime_command_referenceTOC.htm">Prime Command Reference Manual</a></span>
                </td>
                <td class="NavRight">
                &#160;
        </td>
            </tr>
        </table>
        <h1 class="Title">sta—Single Template Alignment</h1>
        <p>The Single Template Alignment (STA) program in PRIME is designed for protein sequences with medium to high sequence identity (&gt;20%). Features of this program include the following:</p>
        <ul>
            <li value="1">
                <p class="ListPara">STA differs from standard sequence alignment programs such as BLAST by taking into account secondary structure matching as well as profile-sequence matching. As a result, STA often generates better alignment than BLAST in the regions where sequence conservation is relatively weak. </p>
            </li>
            <li value="2">
                <p class="ListPara">STA uses a position-specific substitutional matrix (PSSM) for the query sequence, derived from PSI-BLAST, to match the template sequence.</p>
            </li>
            <li value="3">
                <p class="ListPara">In order to minimize the inaccuracy in a single secondary structure prediction, a novel and robust algorithm has been designed to derive a composite secondary structure for the query sequence from multiple predictions. The composite SSP is aligned to the SSA of the template.</p>
            </li>
            <li value="4">
                <p class="ListPara">STA can apply constraints, such as user-selected residue pairs to be aligned together or residues in either query or template that are not to be aligned (gaps).</p>
            </li>
            <li value="5">
                <p class="ListPara">STA can use templates directly from the PDB, and can also use templates from the Prime fold library.</p>
                <p class="Heading3Nonum">Syntax </p>
            </li>
        </ul><pre>
$SCHRODINGER/sta [<i class="UserReplace">options</i>] <i class="UserReplace">jobname</i></pre>
        <p>The input is read from the file <i class="UserReplace">jobname</i><code>.inp</code> and the output is written to <i class="UserReplace">jobname</i><code>.out</code>. </p>
        <p class="Heading3Nonum">Command Input File </p>
        <p>The input file consists of lines containing keyword-value pairs, one per line. The keywords are described in <a class="Table_Number MCXref xref xrefTable_Number selected" href="#prime_commands_1525853049_777631">Table 1</a>. </p>
        <table class="TableStyle-StandardRules" style="caption-side: top;mc-table-style: url('../Resources/TableStyles/StandardRules.css');" cellspacing="0">
            <caption valign="top" style="padding-bottom: 7pt;">
                <p class="Table_Caption" data-mc-autonum="Table 1. "><span class="autonumber"><span>Table 1. </span></span><a name="prime_commands_1525853049_777631"></a>Keywords for the sta program </p>
            </caption>
            <thead>
                <tr class="TableStyle-StandardRules-Head-Header1">
                    <td class="TableStyle-StandardRules-HeadE-Column1-Header1">
                        <p class="Cell_Heading">Keyword syntax</p>
                    </td>
                    <td class="TableStyle-StandardRules-HeadD-Column1-Header1">
                        <p class="Cell_Heading">Description</p>
                    </td>
                </tr>
            </thead>
            <tbody>
                <tr class="TableStyle-StandardRules-Body-Body1">
                    <td class="TableStyle-StandardRules-BodyH-Column1-Body1">
                        <p class="Cell_Body"><code>QUERY_NAME </code><i class="UserReplace">name</i>
                        </p>
                    </td>
                    <td class="TableStyle-StandardRules-BodyG-Column1-Body1">
                        <p class="Cell_Body">Name of the query sequence.</p>
                    </td>
                </tr>
                <tr class="TableStyle-StandardRules-Body-Body1">
                    <td class="TableStyle-StandardRules-BodyE-Column1-Body1">
                        <p class="Cell_Body"><code>QUERY_FILE </code><i class="UserReplace">filename</i>
                        </p>
                    </td>
                    <td class="TableStyle-StandardRules-BodyD-Column1-Body1">
                        <p class="Cell_Body">File name of the query sequence in FASTA format.</p>
                    </td>
                </tr>
                <tr class="TableStyle-StandardRules-Body-Body2">
                    <td class="TableStyle-StandardRules-BodyH-Column1-Body2">
                        <p class="Cell_Body"><code>FORMAT </code><i class="UserReplace">string</i>
                        </p>
                    </td>
                    <td class="TableStyle-StandardRules-BodyG-Column1-Body2">
                        <p class="Cell_Body">Alignment file format. Default is Maestro. Set to <code>dev</code> for plain text format—see <a class="page__ MCXref xref xrefpage__ selected" href="#prime_commands_1525853049_768375">Below is an example of the plain text format for output alignment files. This format is generated when you include ALIFORMAT dev in the command input file. </a> for an example. </p>
                    </td>
                </tr>
                <tr class="TableStyle-StandardRules-Body-Body2">
                    <td class="TableStyle-StandardRules-BodyE-Column1-Body2">
                        <p class="Cell_Body"><code>BGNRES </code><i class="UserReplace">n</i>
                        </p>
                    </td>
                    <td class="TableStyle-StandardRules-BodyD-Column1-Body2">
                        <p class="Cell_Body">The first residue of the query that is used in alignment. Default: 1.</p>
                    </td>
                </tr>
                <tr class="TableStyle-StandardRules-Body-Body1">
                    <td class="TableStyle-StandardRules-BodyH-Column1-Body1">
                        <p class="Cell_Body"><code>ENDRES </code><i class="UserReplace">n</i>
                        </p>
                    </td>
                    <td class="TableStyle-StandardRules-BodyG-Column1-Body1">
                        <p class="Cell_Body">The last residue of the query sequence that is used. Default: the last residue of the original query sequence.</p>
                    </td>
                </tr>
                <tr class="TableStyle-StandardRules-Body-Body1">
                    <td class="TableStyle-StandardRules-BodyE-Column1-Body1">
                        <p class="Cell_Body"><code>PREDSS</code>[<code>_</code><i class="UserReplace">i</i>]<code></code><i class="UserReplace">filename</i></p>
                    </td>
                    <td class="TableStyle-StandardRules-BodyD-Column1-Body1">
                        <p class="Cell_Body">Secondary structure prediction file of the query sequence, in CASP format. One occurrence of this keyword is required for each file, and there must be at least one SSP file specified. If only one prediction is used, the keyword is <code>PREDSS</code>; if more predictions are used, the keyword is suffixed with an index <i class="UserReplace">i</i>, starting from zero: <code>PREDSS_0,</code> <code>PREDSS_1</code>, and so on. </p>
                    </td>
                </tr>
                <tr class="TableStyle-StandardRules-Body-Body2">
                    <td class="TableStyle-StandardRules-BodyH-Column1-Body2">
                        <p class="Cell_Body"><code>TEMPLATE_PDB </code><i class="UserReplace">filename</i>
                        </p>
                    </td>
                    <td class="TableStyle-StandardRules-BodyG-Column1-Body2">
                        <p class="Cell_Body">Filename of the template structure, in PDB format.</p>
                    </td>
                </tr>
                <tr class="TableStyle-StandardRules-Body-Body2">
                    <td class="TableStyle-StandardRules-BodyE-Column1-Body2">
                        <p class="Cell_Body"><code>MODE </code><i class="UserReplace">string</i>
                        </p>
                    </td>
                    <td class="TableStyle-StandardRules-BodyD-Column1-Body2">
                        <p class="Cell_Body">Optional. Needed in the input file only when there are user-specified constraints. In that case, the value is set to <code>user</code>.</p>
                    </td>
                </tr>
                <tr class="TableStyle-StandardRules-Body-Body1">
                    <td class="TableStyle-StandardRules-BodyH-Column1-Body1">
                        <p class="Cell_Body"><code>PAIR </code><i class="UserReplace">pair-list</i>
                        </p>
                    </td>
                    <td class="TableStyle-StandardRules-BodyG-Column1-Body1">
                        <p class="Cell_Body">Optional. User-specified residue pairing, used only when <code>MODE</code> is set to <code>user</code>. All the residue pairs can be added after the keyword in one or multiple lines. For example, the following lines pair residue 12, 20, 60 from the query sequence with residue 15, 24, 65 from the template.</p><pre>
MODE user
PAIR 12 15 20 24
PAIR 60 65
</pre>
                    </td>
                </tr>
                <tr class="TableStyle-StandardRules-Body-Body1">
                    <td class="TableStyle-StandardRules-BodyB-Column1-Body1">
                        <p class="Cell_Body"><code>GAP </code><i class="UserReplace">gap-list</i>
                        </p>
                    </td>
                    <td class="TableStyle-StandardRules-BodyA-Column1-Body1">
                        <p class="Cell_Body">Optional. User-specified gaps, used only when <code>MODE</code> is set to <code>user</code>. Can be used with or without <code>PAIR</code>. Gaps are specified by the residues in either query (P) or template (T) that are aligned with them. For example, the following opens gaps against query residue 10, 11, template residue 34 and 35.</p><pre>
MODE user
GAP  P10 P11 T34 T35
</pre>
                    </td>
                </tr>
            </tbody>
        </table>
        <p class="Heading3Nonum">Files</p>
        <ul>
            <li value="1">
                <p class="ListPara">Input file—named <i class="UserReplace">jobname</i><code>.inp</code>. Contains keywords for running the program.</p>
            </li>
            <li value="2">
                <p class="ListPara">Query sequence file—File containing the complete sequence of the query, in FASTA format. Specified in the input file. </p>
            </li>
            <li value="3">
                <p class="ListPara">Secondary structure prediction files in CASP format for the query sequence.</p>
            </li>
            <li value="4">
                <p class="ListPara">Template PDB file—Template structure file, in PDB format. The sequence used for the template is taken from <code>SEQRES</code> lines in the PDB file. If <code>SEQRES</code> is not found in the PDB file, the template sequence is then taken from <code>ATOM</code> records.</p>
            </li>
            <li value="5">
                <p class="ListPara">Output alignment file—File named <i class="UserReplace">jobname</i><code>.out</code> containing pairwise alignment between the query and the template. By default, the alignment is in Maestro format. To generate plain text format, include <code>FORMAT dev</code> in the command input file. In plain text format, aligned residues from the query and the template are placed on top of each other—see below for an example.</p>
            </li>
            <li value="6">
                <p class="ListPara">Log file—File named <i class="UserReplace">jobname</i><code>.log</code>containing progress of the <code>sta</code> program, including warnings and error messages. </p>
                <p class="Heading3Nonum">Return Value and Errors</p>
            </li>
        </ul>
        <p><code>sta</code> returns 0 for success and non-zero for failure. In case of failure, check the log file for details, and search for <code>ERROR</code>. To further analyze the failure, use the <code>-DEBUG</code> option when you run <code>sta</code>. </p>
        <p class="Heading3Nonum">Examples</p>
        <p>Below is an example input file, where <code>query_seq.fasta</code> stores the query sequence in FASTA format.</p><pre>
QUERY_NAME      query
QUERY_FILE      query_seq.fasta
PREDSS_0        query-ssp1.casp
PREDSS_1        query-ssp2.casp
PREDSS_3        query-ssp3.casp
BGNRES          1
ENDRES          149
TEMPLATE_PDB    pdb1vpe.ent
</pre>
        <p><a name="prime_commands_1525853049_768375"></a>Below is an example of the plain text format for output alignment files. This format is generated when you include <code>ALIFORMAT dev</code> in the command input file. </p><pre>
probe length=70======tmplt_T0281_d1dq3a2 length=79 TotalScore= 42 SEQ= -4 SSTR= 102
 SEQID= 0.061538 nali= 65
probe bgnRes=1 endRes=70    template bgnRes=3 endRes=75
ProbeAA: ..MWMPPRPEEVARKLRRLGFVERMAKGGHRLYTHPDGRIVVVPFH...SGELP....KG
ProbeSS:   LLLLLLHHHHHHHHHHLLLEEEEELLLEEEEELLLLLEEEeLLL   LLLLL    HH
Fold AA: GNFGLPLNFNAFKEWASEYGVEFKTNGSQTIAIIND...ERISLGQWHTRNRVSKAVLVK
Fold SS: LLLEELLLHHHHHHHHHLLLLEEEEELLEEEEEELL   EEEELLLHHHHLLEEHHHHHH
ProbeAA: TFKRILRDAGLTEEEFHNL....
ProbeSS: HHHHHHHHhLLLHHHHHLL
Fold AA: MLRKLYEATK.DEEVKRMLHLIE
Fold SS: HHHHHHHHHL LHHHHHHHHHHL
</pre>
        <p>The file contains header information at the top, then in the alignment section, the query sequence (<code>ProbeAA</code>), query composite secondary structure (<code>ProbeSS</code>), template sequence (<code>Fold AA</code>) and template secondary structure (<code>Fold SS</code>) are given. Gaps are denoted by periods in the sequences and by spaces in the secondary structures.</p>
        <p>&#160;</p>
        <table class="NavLink" data-mc-conditions="Default.ScreenOnly">
            <tr class="NavLink">
                <td class="NavLeft"><span class="NavText">◀ <a class="NavLink MCXref xref xrefNavLink" href="prime_commands_ssp.htm">ssp—Secondary Structure Prediction</a></span>
                </td>
                <td class="NavCenter"><span class="NavText"><a class="NavLink MCXref xref xrefNavLink" href="prime_command_referenceTOC.htm">Prime Command Reference Manual</a></span>
                </td>
                <td class="NavRight">
                &#160;
        </td>
            </tr>
        </table>
    </body>
</html>