<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<!--	moetour.html
!!
!!	14-oct-2009 (al) resnapped images
!!	16-sep-2009 (gk) added tauto button in builder
!!	13-apr-2009 (al) update for new Render panel
!!	13-jan-2009 (al) popup name: MOE | Popup
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../manstyle.css" />


<style type="text/css">
body { background-color: white; }
.cntr { text-align: center; }
.resnap { border: solid thick maroon; }
#dbx { display: block; color: red; background-color: yellow; }
.clear {clear:both;}
</style>

<title>MOE Tour</title>
</head>

<body bgcolor="#ffffff">
<a href="../index.htm"><img src="../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>MOE Tour</h2>
<hr noshade="noshade" />

<p>Before you begin this tutorial, you might want to 
familiarize yourself with some basic concepts relative to the MOE 
<a href="../moe/hellomoe.htm#GUI_Basics">GUI</a>
(<i>Graphical User Interface</i>). 
To find information on
a specific panel, use the <a href="../panelindex.html">Index of MOE
panels</a>.
</p>

<p><b>Note:</b> If MOE does not correspond to what is presented in this
tutorial, you might be running a customized version. Try using
&quot;standard&quot; MOE. Restart MOE, this time with the <tt>-std</tt>
command line switch (i.e. <tt>moe -std</tt>).
</p>

<p>This tutorial includes the following exercises:
</p>
<ul>
  <li><a href="#BuildingSmallMolecule">Building a Small Molecule</a></li>
  <li><a href="#SaveToFile">Saving and Loading a Molecule File</a></li>
  <li><a href="#SavingInMDB">Saving a Molecule in a Database</a></li>

  <li><a href="#RenderingModes">Rendering the Molecule</a></li>
  <li><a href="#SelectingAtoms">Selecting Atoms</a>
  <ul>
    <li><a href="#MoveSelectedAtoms">Repositioning Selected Atoms</a></li>
    <li><a href="#RotateAroundBond">Rotating About a Bond</a></li>
  </ul>
  </li>
  <li><a href="#AtomManager">Getting Information About Molecules</a></li>
  <li><a href="#Measurements">Measuring Angles and Distances</a></li>
  <li><a href="#MeasureEnergy">Measuring Energy</a></li>
  <li><a href="#ConfSearch">Running a Conformation Search</a>
  <ul>
    <li><a href="#DatabaseCreation">Database Creation</a></li>
    <li><a href="#DBPlot">Plotting Using the Database Viewer</a></li>
    <li><a href="#LoadToMOE">Loading Molecules from the Database to
	MOE and Back</a></li>
  </ul>
  </li>
  <li><a href="#MolecularDynamics">Running and Animating a
	Molecular Dynamics Simulation</a></li>

  <li><a href="#SequenceEditor">Introducing the Sequence Editor</a>
  <ul>
    <li><a href="#SEPlots">Sequence Editor Plots</a></li>
    <li><a href="#SESelectingAtoms">Selecting Atoms and Residues</a></li>
  </ul>
  </li>
  <li><a href="#Alignment">Aligning and Superposing Sequences</a></li>
  <li><a href="#Superpose">Superposing Protein-Ligand Complexes</a></li>
</ul>


<h3>Conventions Used in this Tutorial</h3>

<table cellpadding="3" cellspacing="3">
<tr>
<td valign="top" bgcolor="#cccccc">MOE</td>
<td valign="top">MOE Window</td>
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">SE</td>
<td valign="top">Sequence Editor</td>
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">DBV</td>
<td valign="top">Database Viewer</td>
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">CLI</td>
<td valign="top">Command Line Interface</td>
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">MOE | RHS</td>
<td valign="top">Right Hand Side Button Bar (MOE Window)</td>
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">MOE | Footer</td>
<td valign="top">Footer Area (MOE Window)</td>
</tr>
</table>

<p>This tutorial assumes that you are using a three-button mouse.  For
information on using a two-button mouse see
<a href="../moe/gui/mouse.html">Using the Mouse</a>.
</p>
<h3><a name="BuildingSmallMolecule">Building a Small Molecule</a></h3>

<p>When first started, MOE always displays its main window, the
<a href="../moe/gui/mainwin.htm">MOE Window</a>, in which
molecular systems are rendered and manipulated.
</p>
<p>Small molecules can be built using the Molecule Builder.
In this exercise, we will build a molecule of aspirin, as pictured below: 
</p>
<p class="cntr">
<img src="moetour/asa.png" alt="Aspirin Molecule Drawn in MOE" />
</p>

<ol>

<li><p>Open the Molecule Builder:

<span class="menu">MOE | Edit | Build | Molecule</span>
</p></li>
<li><p>Create an aromatic ring by pressing the benzene button, located on the 
right of the Molecule Builder.
</p>
<p class="cntr">
<img src="moetour/benzene.png" alt="Building a Benzene Ring" />
</p>
</li>
<li><p>Center the molecular system in the MOE Window by clicking 
<span class="menu">MOE | RHS | Center</span>
</p></li>
<li><p>Render the molecule as cylinders by clicking on the
appropriate button in:
<span class="menu">MOE | Render | Atoms</span></p>
<p class="cntr"><img src="moetour/atomstyle.png" alt="Render atoms as cylinders" /></p>
</li>
<li><p>Choose <span class="menu">MOE | Render | Atoms | Label : Element</span> 
to label all atoms by element.
</p></li>
<li><p>To add a carboxyl group, select one of the benzene hydrogens by
clicking the left mouse button on it.  Substitute it with a carboxyl
group by clicking on <b>-CO2</b> in the Builder.</p>

<p class="cntr">
<img src="moetour/cooh.png" alt="Adding a Carboxyl Group" />
</p>
</li>
<li><p>To add a hydroxyl group, select the hydrogen atom
<i>ortho</i> to the carboxyl group. In the Builder, click on
<b>O</b> in the Element buttons.</p></li>

<li><p>To acetylate the hydroxyl group, select the hydroxyl hydrogen atom in the MOE
Window and click on the <b>-C=O</b> substituent button in the Builder, followed
by the <b>C</b> element button. 
You have now completed the aspirin molecule.
</p>
<p>We could also have built the aspirin molecule by typing: </p>
<pre>
    O(C(=O)C)c1c(C(=O)O)cccc1
</pre>
in the SMILES String text field of the Molecule Builder
and then pressing the SMILES button (the smiley face).

<p class="cntr">
<img src="moetour/smiles.png" alt="Building a Molecule using SMILES" />
</p>
</li>
<li><p>Press the Center button to center the molecular system in the 
MOE Window:

<span class="menu">MOE | RHS | Center</span>
</p></li>
<li><p>Rotate the molecule by dragging the middle mouse button
in the MOE Window.  You can also rotate the molecule using the trackball in the
RHS.</p>
<p class="cntr"><img src="moetour/trackball1.png" alt="Trackball" /></p>

<p>When rotating the molecule in the window, the center of rotation
is the center of mass of the entire molecular system.
When zoomed in close to an atom, however, you may prefer to localize
movement by forcing rotation about the atom. To do so, place the mouse cursor 
over the atom and click the
middle button. To regain the
default setting, simply click the middle mouse button in the MOE
Window away from any atoms.</p></li>


<li><p>To pan the molecule, press <tt>Shift</tt> and drag the middle mouse
button.  To pan the molecule with the trackball
(<span class="menu">MOE | RHS | Trackball</span>), activate panning mode
by clicking in the gray square at the bottom left of the trackball
(the square is labeled &quot;R&quot; when in rotation mode and
&quot;T&quot; when in translation mode).
</p>
<p class="cntr"><img  src="moetour/trackball2.png" alt="Trackball" /></p>
</li>
<li><p>To zoom in or out, press <tt>Ctrl</tt> and drag the middle
mouse button. Alternatively, if your mouse is equipped with a scroll wheel,
it can also be used to control zooming. Another alternative is using the wheel 
below the trackball.</p>
<p class="cntr"><img src="moetour/trackball3.png" alt="Trackball" /></p>
</li>

<li><p>The current forcefield setting is shown on the left of the MOE Footer: </p>
<p class="cntr"><img src="moetour/ffsetting.png" alt="Forcefield setting" /></p>

<p>To change the forcefield, click the button showing the forcefield name, or 
use <span class="menu">MOE | Window | Potential Setup</span>.
</p>
<p>In the Potential Setup panel, ensure the <b>MMFF94x</b> forcefield is active 
by selecting it in the <b>Load...</b> pulldown menu. Close the panel.
</p>
</li>


<li><p>Compute partial charges by choosing:
</p>
<p><span class="menu">MOE | Compute | Partial Charges</span>
</p>
<p class="cntr">
<img src="moetour/pcharge.png" alt="pcharge" />
</p>

<p>Leave the <b>Method</b> as <b>Current Forcefield</b> and press
<b>OK</b>.  This uses the default charge
calculation associated with the current forcefield. 
</p></li>
<li><p>Minimize the energy of the molecular system by choosing 
the <b>Minimize</b> button in the Right Button Bar of the MOE Window.</p></li>
</ol>


<p>See the tutorial on <a href="builder_tut.htm">MOE builders</a> for more
information on the Molecule Builder.
</p>
<h3><a name="SaveToFile">Saving and Loading a Molecule File</a></h3>

<ol>

<li><p>Save the acetylsalicylic acid molecule you have just built:
</p>
<p><span class="menu">MOE | File | Save...</span> 
</p>
<p>Double-click on the directory in which to save the molecule file.
Next, enter <tt>aspirin</tt> in the text field and save with a
<tt>.moe</tt> extension (the MOE file extension) by clicking Save.
</p>
<p><span class="tip">Tip</span> For quick access to
MOE directories, click on the arrow in the text field. This opens the
shortcut list of recent directory paths.
</p>
<p class="cntr">
<img src="moetour/savepanel.png" alt="Click the down arrow for recent directories" />
</p>
</li>
<li><p>Now, you will load the newly saved molecule back into MOE. However,
before doing so, you must first clear the data currently in MOE:
</p>
<p><span class="menu">MOE | File | Close</span> or <span class="menu">MOE | RHS | Close</span>
</p></li>
<li><p>Load the molecule back into MOE:
</p>
<p><span class="menu">MOE | File | Open</span>
</p>

<p>In the Open panel, select <tt>aspirin.moe</tt> from the list
of files and press OK.
</p></li>
</ol>

<p>If after loading the molecule file no atoms are shown in the MOE
window, it may be that the molecule is not visible with the current viewing 
parameters. To change the view so that the molecule is fully visible, 
choose <span class="menu">MOE | RHS | Center</span>.
</p>
<h3><a name="SavingInMDB">Saving a Molecule in a Database</a></h3>

<p>When building molecules in MOE, it is convenient to save and 
retrieve molecular fragments to and from a database.  Below is a short
example on how to create a database and copy the aspirin molecule from 
MOE to the database.
</p>
<ol>

<li><p>To create a molecular database, choose:
</p>
<p><b>MOE | File | New | Database</b>
</p>
<ol type="a">   
<li><p>In the New Database panel, 
select the directory in which the new database will be saved.
</p></li>
<li><p>In the Path text field, enter <tt>aspirin.mdb</tt>. 
</p></li>
<li><p>Press OK. The new database appears in a MOE Database Viewer. The
database contains a single field called <b>mol</b>, which is of type
<b>molecule</b>.
</p></li>
</ol>
</li>
<li><p>Create a second field in the database, which will contain the molecule
names, by selecting:
</p>
<p><span class="menu">DBV | Edit | New | Field...</span>
</p>
<p>The command line in the Database Viewer changes to a prompt.  
Choose <b>char</b> in the Type option menu to create a textual
field. Next, enter <b>Molecule Name</b> in the Name text box and press
<tt>Return</tt>. A new field called <b>Molecule Name</b> is created in the
database.
</p></li>
<li><p>To copy the aspirin molecule from the current system to the
database, choose:
</p>
<p><span class="menu">DBV | Edit | New | Entry...</span>
</p>
<p>In the Add Entry panel, select <b>Mode: System</b>
and type <tt>aspirin</tt> in the
Molecule Name text field. Press OK. Both fields now contain information in
the database.
</p></li>
<li><p>In the Database Viewer, to get a better view of the molecule in
the database, position the cursor over the cell in the first field
(called <tt>mol</tt>), press the left mouse button and drag down (and slightly
to the right) until you clearly see the molecular drawing. You can
rotate the drawing by dragging the middle mouse button over it.
</p></li>
<li><p>Only the heavy atoms are displayed in the Database Viewer. To
display hydrogen atoms, choose:
</p>
<p><span class="menu">DBV | Display | Molecule | Hydrogens</span>
</p></li>
</ol>


<h3><a name="RenderingModes">Rendering the Molecule</a></h3>

<p>The Render menu allows you to change the visual display mode of the
current molecule.
The primary rendering controls are found in
<a href="../moe/gui/mainwin.htm#Rendering">
<span class="menu">MOE | Render | Atoms</span></a>, which
can also be opened using
<span class="menu">MOE | Footer | Atoms</span>.
</p>

<p class="cntr">
<img src="moetour/atom_render.png" alt="Atom Rendering Panel" />
</p>

<p>
The default rendering mode is Line mode, which is also
the fastest drawing mode.  In building the aspirin molecule, you
previously used the Render menu to change to Stick mode. Three other
modes are: Ball and Line, Ball and Stick, and Space Filling.  There
is even a None (no bonds) mode. Take a moment now to try each rendering mode.
</p>

<p>Note that if you have any selected atoms, the chosen rendering will
affect only those atoms. The following steps will show you how to render
different parts of a system in distinct modes:
</p>
<ol>
<li><p>To set the entire molecule in Stick mode, make sure no atoms are selected and use 
<span class="menu">MOE | Render | Atoms</span>.
</p></li>

<li><p>To select all the benzene carbons, open the Atom Selector
with <span class="menu">MOE | Selection | Atom Selector</span>.
In the SMILES text field, 
type <tt>a</tt> to indicate aromatic atoms and press
<tt>All</tt>.
   
This will perform a substructure search and select all atoms
within the matching strings.
</p></li>
<li><p>Choose the appropriate button in
<span class="menu">MOE | Render | Atoms</span> to display
the atom nuclei of only the ring carbons as small spheres.</p></li>

</ol>

<p>The rendering options allow atoms to be labeled and colored according
to charge and element type.  You may wish to take a moment now to experiment
with the various <b>Label</b> and <b>Color</b> buttons in the
<span class="menu">MOE | Render | Atoms</span> panel.
</p>


<p>Individually selected atoms, hydrogens,
residues and sidechains may be shown or hidden using the
<span class="menu">MOE | Render | Hide</span> and 
<span class="menu">MOE | Render | Show</span> submenus.
The Backbone, Sidechain
and All options are useful when studying proteins and polymers. You will
have an opportunity to experiment with these commands later when we create
a protein.  For now, hide the hydrogens on the aspirin molecule with
<span class="menu">MOE | Render | Hide | Hydrogens</span>.
To make the atoms visible
again, use <span class="menu">MOE | Render | Show | All</span>.  Hydrogen atom 
display can also be toggled using the <span class="menu">MOE | RHS | Hydrogens</span> 
button; clicking this cycles through displaying all, polar only, or no hydrogen atoms.
</p>

<p>The Render menu is also used for setting the
<a href="../moe/gui/mainwin.htm#Viewing">viewpoint</a> from which the
molecule is observed.
<span class="menu">MOE | Render | Center</span> is
equivalent to <span class="menu">MOE | RHS | Center</span>.
These operate on selected atoms or on all atoms if none are selected,
and adjust the view so that all atoms in question can be seen and
are centered in the Main Window.
The View can also be set using the
<span class="menu">MOE | Footer | Trackball Popup | View</span>
menu, which allows the user to save and load particular viewing angles.
</p>

<p>To demonstrate the use of
<span class="menu">MOE | Render | Center</span> on selected atoms,
we begin by selecting a few atoms in the aspirin molecule:
</p>

<ol>
<li><p>Select all oxygen atoms with
<span class="menu">MOE | Selection | Atom Selector</span>.  Open the 
Periodic <b>Table</b> and choose <b>O</b>.
</p>
</li>

<li><p>Select <span class="menu">MOE | Render | Center</span>. In the MOE
window, the view is now zoomed in on the part of the molecule
containing the selected atoms.  </p></li>

</ol>

<h3><a name="SelectingAtoms">Selecting Atoms</a></h3>

<img src="moetour/selatom.png" align="right" hspace="10" vspace="10"
alt="Selected Atom in MOE" />
<img src="moetour/selatom2.png" align="right" hspace="10" vspace="10"
alt="Selected Atom in MOE" />

<p>To change the properties of a molecule or manipulate a subset of its
atoms, you must first indicate which atoms are to be manipulated by
creating an atom <i>selection set</i>.  Selected atoms are identified by
a change in color and a (small) sphere depending on the current
rendering mode of the atom selected.
</p>
<p>Use the mouse to manually select atoms:
</p>
<ol>
<li><p>Make a selection set consisting of a single atom by
clicking the left mouse button on any atom.  
This will cause all other selected atoms to become unselected.</p></li>

<li><p>Select the atoms that form a bond by clicking 
on the bond that links them.</p></li>

<li><p>Select all atoms of a residue by pressing <tt>Ctrl</tt>
while clicking the left mouse button 
on an atom.
In the case of the aspirin molecule, all
atoms belong to the same residue, hence, all atoms are selected.
Hidden atoms are also selected.</p></li>

<li><p>Add a single atom to the selection set by pressing
<tt>Shift</tt> while clicking the left mouse button on an atom.  
This toggles the atom selection state.  You can add all atoms of a residue to
the selection set by holding down both <tt>Shift</tt> and
<tt>Ctrl</tt> when clicking the left mouse button.  You cannot,
however, click on hidden atoms.</p></li>

<li><p>Draw a selection box around a group of atoms by dragging the left
mouse button over the atoms you want to select.  A selection box is drawn
as you drag the mouse. When you release the left mouse button, all atoms
within the box become selected.  To add more atoms to the selection set,
drag a new box while pressing <tt>Shift</tt>.
</p></li>
</ol>

<p>The <b>Selection</b> menu allows you to add atoms to the atom selection set
according to element type and geometry (hybridization).  In addition,
atoms can be selected based on whether they belong to the molecule
backbone or to a side chain and whether they have been hidden 
(e.g. using <span class="menu">MOE | Render | Hide</span>) or fixed
(e.g. using <span class="menu">MOE | Edit | Potential | Fix</span>).  The
<span class="menu">MOE | Selection | Extend</span> submenu allows you to extend
the atom selection set from the currently selected atoms.</p>

<p>To remove atoms from the atom selection set, press
<tt>Shift</tt> and click on the atoms you wish to
deselect. 
</p>
<h4><a name="MoveSelectedAtoms">Repositioning Selected Atoms</a></h4>

<p>Selected atoms can be moved independently of the rest of the molecule
by dragging the middle mouse button while holding down the
<a href="../moe/gui/mouse.html"><tt>Alt</tt></a> key, with or without the <tt>Shift</tt> key. 
Pressing both the
<tt>Alt</tt> and <tt>Shift</tt> keys specifies that selected atoms are to be moved; if only the 
<tt>Alt</tt> key is depressed, the selected atoms are rotated about their center 
of mass.  The <span class="menu">RHS | Trackball</span> can also be used 
to manipulate the selected atoms by pressing <tt>Alt</tt> while
using the trackball.
</p>
<h4><a name="RotateAroundBond">Rotating About a Bond</a></h4>

<ol>
<li><p>Clear the selection set by clicking in the MOE Window away
from any atoms.</p></li>

<li><p>Select the two atoms that form a bond by clicking on the bond that 
links them.</p></li>

<li><p>The MOE Footer changes to show the current bond length, plus a dial, 
compass button and Profile.  Left click and drag on the dial to rotate the bond. As 
you do this you will see that one of the bond atoms is highlighted with a compass; this 
shows which end of the bond will rotate.  To reverse this, click on the compass button.
(If this button is not clickable, check to see if any atoms in the molecule are 
fixed.) </p>
<p class="cntr">
<img src="moetour/rotatebond.png" alt="MOE Footer showing rotate dial" />
</p>
<p>To see a plot of potential energy of the molecule vs dihedral angle of the selected 
bond, click on the <b>Profile</b> button.  This plot is interactive; left click 
in it to set the bond to that dihedral angle.  To see the value of the dihedral 
angle in the Footer rather than the bond length, select the four atoms that 
fully define the dihedral angle.</p>
</li>
</ol>
<p>An alternative method for rotating about a bond used in previous versions of 
MOE will still work:</p>
<ol>
<li><p>Clear the selection set by clicking in the MOE Window away
from any atoms.</p></li>

<li><p>Select a single atom with the left mouse button.</p></li>

<li><p>Select a second atom that is bonded to the first by holding down
<tt>Shift</tt> and clicking the left mouse button on the atom.  To rotate
about a bond, exactly two atoms joined by a bond must be selected.</p></li>

<li><p>Press the <tt>Alt</tt> key and drag the left mouse button.</p></li>
</ol>

<p>When rotation is possible, the smaller group will rotate, while the
larger group remains still.  If no rotation occurs, it may be because the
bond is in a ring, or because there are fixed atoms preventing the
rotation.  Also, if one of the selected atoms has no other atoms bonded to
it, no apparent rotation will result.</p>


<h3><a name="AtomManager">Getting Information About Molecules</a></h3>

<p>If the mouse cursor is held over the <span class="menu">MOE Footer</span>, 
a yellow bubble appears reporting the number of atoms 
currently in the system.  If a single atom is selected, the MOE footer changes 
to show the name or number of the selected atom's chain and residue.  The atom 
name is shown in the text box (and can be edited), while the element is shown 
to the right in square brackets.  By holding the mouse cursor 
over the displayed text, more information about the atom is shown, while 
holding the mouse cursor to the right of the text shows information about 
the number and properties of the selected atoms.
</p>
<p class="cntr">
<img src="moetour/footerinfo.png" alt="MOE Footer showing rotate dial" />
</p>
<p>The <a href="../moe/gui/atommgr.html">Atom Manager</a> provides more 
detailed information about the individual atoms in a molecular
system, such as element name, hybridization and charge, as well as
information about other molecular objects.
</p>
<p>Molecules have a hierarchical composition. Atoms are the children
of <i>residues</i>, and residues are the children of <i>chains</i>.
The chain-residue relationship is explicitly mapped out in the
<a href="../moe/gui/se.html">Sequence Editor</a>.
The Atom Manager can be used to modify atom, residue and chain properties.
</p>

<ol>
<li><p>Double-click on an atom.
This opens the Atom Manager which lists all atoms, residues and chains
found in the MOE Window, along with their properties. (Another
way of opening the Atom Manager is by choosing
<span class="menu">MOE | Window | Atom Manager</span>.)
</p></li>
<li><p>Select a residue line in the list by clicking the left mouse button 
on a <tt>Residue</tt> line.  In the edit area at the bottom of
the panel, residue properties such as name and type can be edited. 
You must click <b>Apply</b> to accept any changes.</p></li>

<li><p>Select a <tt>Chain</tt> line in the list to see the chain properties
available for editing.</p></li>

<li><p>Press the <b>Compress All</b> button.  Only the chain listings are
displayed.</p></li>

<li><p>To uncompress a single chain, double-click the left mouse button
on it.  Similarly, double-click on a residue line to uncompress it.</p></li>

<li><p>Press <b>Expand All</b> to restore the full listing.</p></li>

<li><p>Click the left mouse button on an atom line.  The properties of
this atom are now editable.</p></li>

<li><p>Press <span class="menu">MOE | RHS | Select | Atom Selector</span> 
to open
the Atom Selector panel. Select all carbons using the <b>Table</b> pulldown
menu in the <b>Element</b> section.
In the Atom Manager, turn on the <b>Selection Only</b>
checkbox.  The list now displays information only for the selected
atoms.</p></li>

</ol>


<h3><a name="Measurements">Measuring Angles and Distances</a></h3>

<p>Steric measurements can be shown in the main MOE window using  
<span class="menu">MOE | RHS | Measure</span>.  
Distances are measured in angstroms, and bond angles and torsions in degrees.
</p>
<ol>

<li><p>Choose <span class="menu">MOE | RHS | Measure | Distance</span>.
The CLI in the MOE window becomes a prompt area for measuring distances, 
and the cursor becomes a crosshair. You are
prompted to select the first and second atoms.</p></li>

<li><p>Select any two atoms.
A colored meter appears, with its distance measured in
angstroms.  (If meters are not being displayed, check the settings in
<b>MOE | Render | Setup</b> checkbox.)</p></li>

<li><p>Cancel the meters prompt by pressing the <tt>Esc</tt> key.
</p></li>

<li><p>Meters are dynamically updated whenever changes occur in a
molecule's conformation. You can verify this by selecting one
of the atoms implicated in a meter and moving it 
(hold down <tt>Shift</tt> and <tt>Alt</tt> and drag the middle mouse button).
</p></li>

<li><p>Meters can be deleted using the
<span class="menu">MOE | RHS | Measure | Remove</span> options.
</p></li>
</ol>

<p><b>Note:</b> If, during prompting, you wish to enter an SVL command, use
the SVL Commands window or another CLI in the system (e.g. in the Sequence Editor).
</p>

<p> As an alternative to creating a meter, distances, bond angles and dihedrals 
are shown in the MOE Footer whenever two, three or four appropriately-bonded 
atoms are selected.</p>

<h3><a name="MeasureEnergy">Measuring Energy</a></h3>

<p>Display the potential energy of the entire system of atoms using
<span class="menu">MOE | Compute | Potential Energy</span>.</p>

<p>The
<span class="menu">MOE | Compute | Conformations | Dihedral Energy Plot</span> 
command also calculates the potential energy of the entire
molecular system. It makes a succession of calculations as the atoms
attached to one side of the user-specified rotatable bond are rotated
through 360 degrees. The differences in potential energies are then
plotted against the dihedral angle, with the current angle and energy
indicated by a vertical line.</p>

<ol>
<li><p>Choose
<span class="menu">MOE | Compute | Conformations | Dihedral Energy Plot</span>.
A prompt appears in the CLI.</p></li>

<li><p>Select four atoms specifying the bond about which the dihedral
energy at different angles of rotation is to be calculated.  Rotation may
be impossible if the selected atoms belong to a ring system, or if there
are fixed atoms. </p></li>

<li><p>A dihedral energy vs. angle plot appears. </p> </li>

<li><p>To cancel the atom-prompting operation, press the
<tt>Esc</tt> key.</p></li>
</ol>

<p>As mentioned above, the <span class="menu">MOE | Footer | Profile</span> button 
can also be used to show a dihedral energy plot whenever two atoms forming a 
suitable bond (or four atoms defining a dihedral) are selected.  This plot is 
interactive, so clicking in it will set the dihedral angle for the selected 
bond.</p>

<p>The
<span class="menu">MOE | Compute | Conformations | Dihedral Contour Plot</span>
command performs a similar calculation except that the
potential energy is calculated as a function of <b>two</b> dihedral
angles, thus yielding an energy surface. The surface is plotted in 2D as a
contour plot.  To stop the energy calculation once rotations have begun,
use the Cancel menu in the upper right corner of the MOE window.</p>

<p>The <span class="menu">MOE | GizMOE | Energy</span> command runs a potential energy
calculation job in the background. This continuously calculates the
energy, which it displays in the MOE
Window.  Select and move an atom (<tt>Shift-Alt</tt> - drag the middle mouse
button) in the MOE Window.  The displayed energy values update
dynamically.</p>

<h3><a name="ConfSearch">Running a Conformation Search </a></h3>

<p>MOE has several conformational search modules.  We will use the
Systematic Conformational Search to search for the minimum
energy conformations of the aspirin molecule.
The search is conducted by calculating the energy of the molecule for the
different angles of <i>all</i> its rotatable bonds. At each step, the
dihedral bond angles are varied systematically, and the molecule is
minimized. The energies of the different conformations are monitored,
and the product of the search is a series of conformations with
energies below a user-specifiable cutoff.</p>


<ol>

<li><p>Start the conformation search by choosing:  </p>

<p><span class="menu">MOE | Compute | Conformations | Conformational Search</span>
</p>
<p>This opens the Conformational Search panel. For the purposes
of this tutorial, you will use most of the default values.  Select the
<span class="menu">Method: Systematic</span>
option.  Make
sure that the Open Database Viewer option is selected as this will
automatically open the database (by default, <tt>csearch.mdb</tt>) in
which the resulting molecule conformations will be stored.
</p></li>
<li><p>Press OK to start the search. During the search, you can observe
the database file being written in real-time.</p></li>
</ol>

<h4><a name="DatabaseCreation">Database Creation</a></h4>

<p>In the first step of the conformation search, the database is
filled with the conformations of the aspirin molecule -- all
generated by bond rotations -- whose energies fall within 7 kcal/mol
(by default) of the minimum energy conformation found.  
Finally, duplicate conformations are eliminated.
</p>
<p>To tell when the search has terminated, click on the
<b>Cancel</b> menu in the upper right-hand corner of the MOE Window. If
<tt>Conformational Search</tt> no longer appears, this means that the search is
over. In any event, you can terminate the searching operation at any time
by choosing <tt>Conformational Search</tt> in the Cancel menu.
</p>

<p>In the database created by the conformation search, molecules
are saved in the field labeled <tt>mol</tt>.
The <tt>mol</tt> field contains line drawings of the conformations of the
aspirin molecule. When it first appears, it is compressed to its minimum size.  
</p>
<ul>
<li><p>To enlarge the <tt>mol</tt> field, click the left mouse button
anywhere in the <tt>mol</tt> column (except on the topmost box containing
the field name) and drag the mouse sideways and down. All the cells will
resize.  Enlarging the cells sufficiently allows element labels and bond
orders to be displayed.</p></li>

<li><p>Rotate a molecule line drawing by
dragging the middle mouse button within a molecule cell.
</p></li>
<li><p>Zoom into a molecule by dragging the middle mouse button
while pressing <tt>Ctrl</tt>.</p></li>

<li><p>The <span class="menu">Display | Molecule</span> menu in the Database Viewer menu bar contains various
commands that control molecular drawing. For example, you can draw
hydrogens and element symbols for each molecular entry.
</p></li>
</ul>


<h4><a name="DBPlot">Plotting Using the Database Viewer</a></h4>

<p>The Database Viewer includes an area for plotting numerical data.
</p>
<ol>

<li><p>Open the plot area by turning on the
<span class="menu">DBV | Display | Plot</span> toggle button.</p></li>

<li><p>Plot the energies of all the molecules by selecting the energy
(<b>E</b>) toggle from the <b>Plot Fields</b> menu. 
Once selected, the color of the toggle box will be the same as the
corresponding line in the plot.
</p></li>
<li><p>Resize the plot to suit the range of data values by choosing
<span class="menu">Zoom | Fit</span> in the Plot popup.  The popup menu is opened by clicking the
<a href="../moe/gui/mouse.html">mouse popup button</a>
in the plot area.</p></li>

<li><p> The plot area has a ticked vertical axis. Position your mouse
cursor on or near a data point in the plot, and hold it motionless for one
second. A yellow bubble will display the database entry number
corresponding to the plot point.</p></li>

<li><p>The selection of entries in the database corresponds to 
selected points in the plot area.  Select some entries in the database by
clicking the left mouse button on the entry numbers.  
Notice that the corresponding data points are circled in
the plot area.  Try the following selection techniques:
</p>

  <ol type="a">
    <li>Select a range of entries: click the left mouse button on the
    first entry, then hold <tt>Shift</tt> and click the left mouse
    button on the last entry of the range.</li>

    <li>Add another entry to the selected entries:  press
    <tt>Ctrl</tt> while clicking the left mouse button.</li>

    <li>Add a second range of entries to the first:  press <tt>Shift</tt>
    and click the left mouse button on the end of the range.</li>
  </ol>
</li>
<li><p>Entries can also be selected by selecting data points in the plot
area.  Try the following:
</p>
  <ol type="a">
    <li>Select individual plot points by  clicking the left mouse button
    in the plot area near data points.  </li>

    <li>Select a group of points by dragging the left mouse button.
    All points in the rectangular area that is swept out will
    be selected.</li>

    <li>Select a group of points whose values fall within a given range by
    clicking the left mouse button in the vertical ruler and dragging the
    mouse up or down.  All points in the delineated range are specified.</li>

  </ol>
</li>
<li><p>To clear the selection, choose <span class="menu">Select | Clear</span> from the
Plot popup menu.</p></li>

</ol>

<p>If you have two numerical data fields, you can use
<span class="menu">DBV | Compute | Analysis | Correlation Plot</span>
to make a correlation plot in which
the data are plotted against each other.  If two fields are already
selected, then the plot will be performed immediately. Otherwise, you will
be prompted to select two numeric fields.  A new window with the
calculated plot will be displayed.  If you wish to compare more than two 
numerical fields, <span class="menu">DBV | Compute | Analysis | Correlation Matrix</span> 
will show a matrix of inter-field correlation coefficients; clicking on one of the buttons that 
form the matrix will launch the corresponding Correlation Plot.
</p>
<h4><a name="LoadToMOE"></a>
Loading Molecules from the Database to MOE and Back</h4>

<p>The molecules in a Database Viewer molecule field can be loaded
into MOE by double-clicking the left mouse button within one of the
cells. The Copy Database Molecule to MOE panel appears.  You will
be queried whether you wish to delete the data in the current system or to
add to it.
</p>

<p>You can load several database molecules into MOE
at the same time by first selecting the desired entries and then using the
Molecule popup menu. Position the cursor over a cell in the molecule field 
and click the <a href="../moe/gui/mouse.html">mouse popup button</a>.  Choosing
<span class="menu">Selected to MOE</span>
will copy all the molecules of the
selected entries.
</p>
<p>Conversely, molecules can be copied from MOE and stored into an
existing molecule cell.
Click the right mouse button in a database molecule cell to
open the Molecule popup menu and choose
<span class="menu">Get from MOE</span>.
(Remember that to create a new entry you use 
<span class="menu">DBV | Edit | New | Entry</span>.)
</p>

<h3><a name="MolecularDynamics"></a>
Running and Animating a Molecular Dynamics Simulation</h3>

<p>A molecular dynamics simulation models a molecule's motion under
particular conditions of heat and pressure that may change at various
stages of the simulation. Molecular dynamics simulations are used for
such purposes as conformation search, and for calculating
thermodynamic properties.
</p>
<ol>
  <li><p>Reload the aspirin molecule into the current system:
</p>
<p><b>MOE | File | Open</b>
</p>
<p>In the Open panel, select
<tt>$MOE/sample/mol/asa.mol</tt>. 
If the MOE Window contains any
molecular data, be sure to close the system first.
</p></li>
<li>Click <span class="menu">MOE | RHS | Minimize</span> to automatically apply 
forcefield partial charges and relieve any strain in the molecule's 
conformation.</li>
<li><p>Start the molecular dynamics operation by choosing:  
</p>
<p><span class="menu">MOE | Compute | Simulations | Dynamics</span>
</p>
<p>This opens the Dynamics panel.  For the purposes of this
tutorial, you will modify some of the default values in order to shorten
the time to run the simulation:
</p>
<ol type="a">
    <li><p>Change the picosecond <b>Time Step</b> from 0.002 to 0.003</p></li>
    <li><p>Change the picosecond <b>Sample Time:</b>
	from 0.5 to 1.0</p></li>
</ol>
<p>
    Press <b>OK</b> to start the simulation.  By default the simulation consists of 
    two stages, a 100ps equilibration followed by a 500ps production run.  The 
    output of the simulation will take a few minutes to generate and be written 
    to two databases, one for each stage of the process.
</p></li>
<li><p>To collate the output databases into a single output database, enter the 
following SVL command:</p>
<p><tt>MD_Import [ 'dynamics.mdb', 'dynamics' ]</tt></p>
<p>You can then use the <span class="menu">MOE | File | Open</span> panel to open the collated output 
database <tt>dynamics.mdb</tt> in a database viewer.</p>
</li>
</ol>

<p>Once the simulation is complete, you can review the dynamics simulation
as a movie animation:
</p>
  <ol>
  <li><p>Close the current molecular data:
</p>
  <p><span class="menu">MOE | File | Close</span>
</p></li>
  <li><p>Open the Database Browser panel in the Database Viewer:
</p>
  <p><span class="menu">DBV | File | Browse...</span>
</p></li>
  </ol>

<p>The Database Browser loads the molecular
conformations into the MOE Window at a certain animation speed.
In other words, it animates the various conformations generated by the
molecular dynamics run.
</p>
<p>There are several ways of controlling the animation:
</p>
<ol type="a">
<li><p>Move the <b>Entry</b> slider by dragging the left mouse button or by 
holding the mouse over it and using the scroll wheel. The slider is used to 
manually animate the conformations stored in the database. 
</p>
<p>The entry number of the molecule currently loaded in MOE
appears in the text field beside the slider.
</p></li>
<li><p>You can step forwards or backwards through the animation using the
<img src="moetour/forward.gif" alt="Forward Button" /> and
<img src="moetour/back.gif" alt="Back Button" /> buttons. The
<img src="moetour/end.gif" alt="Beginning Button" /> and
<img src="moetour/begin.gif" alt="End Button" /> buttons will respectively
jump to the last and first entry of the database.
</p></li>
<li><p>
The <b>Animation</b> wheel controls the speed of the animation. Moving the
wheel to the right accelerates the animation rate. Moving the wheel all
the way to the left stops the animation.
</p>
<p>You can run a continuous animation loop by dragging the animation wheel
right and selecting the <b>Loop</b> option.
To stop the looping, click on the grey square to the right of the wheel.
</p></li>
</ol>


<h3><a name="SequenceEditor">Introducing the Sequence Editor</a></h3>

<p>The Sequence Editor is a tool for visualizing molecules in terms
of their residues, and acts as a starting point for performing
full sequence-to-structure reconstruction. This includes homology
analysis as well as secondary structure analysis and prediction on
proteins.
</p>
<ol>
<li><p>Clear the current molecular data with
<span class="menu">MOE | File | Close</span>.
</p></li>
<li><p>We will load a protein from the Protein Database panel (this panel
might take a few seconds to load):
</p>
<p><span class="menu">MOE | File | Protein Database</span>
</p></li>
<li><p>By default, the list orders proteins alphabetically
according to the last three characters of their four-letter PDB
codes.  Find <tt>2POR PORIN</tt>.
A quick way of finding proteins is to use the Search field: 
</p>
<p>Enter <tt>2POR</tt> in the Search text field.
The list displays <tt>2POR</tt>. Select this entry and press 
<span class="menu">Copy to MOE</span>, check the "Center Molecule" box 
and click <b>OK</b> in the prompt to load it into MOE.  Close the Protein 
Database panel.
</p></li>
<li><p>To see the beta-barrel,
turn on backbone ribbon drawing using the appropriate button in
<span class="menu">MOE | Render | Ribbon</span>.
</p></li>
<li><p>Open the Sequence Editor with
<span class="menu">MOE | Window | Sequence Editor</span>.
</p>
<p><span class="tip">Tip</span> 
A shortcut to open the Sequence Editor is to use the keystrokes
<b><tt>&lt;Ctrl&gt;-q</tt></b>.
</p>
<p>The residues of the currently loaded molecular system are laid out linearly
in a schematic diagram in the Sequence Editor. Protein structures may comprise
one or more chains which are each composed of a linear succession of residues.
</p></li>
<li><p>The Display menu in the Sequence Editor controls certain visual
features:
</p>
<ul>
<li><p>Change from three-letter to single-letter residue display mode
with 
<span class="menu">SE | Display | Single Letter Residues</span>.</p></li>

<li><p>Display secondary structure indicators with
<span class="menu">SE | Display | Actual Secondary Structure</span>.  The
colored bars correspond to the backbone ribbon displayed in the MOE
Window.  Both are calculated dynamically.</p></li>

<li><p>Display backbone hydrogen bonds with
<span class="menu">SE | Display | Hydrogen Bonds</span>.
Bonds appear as arcs.</p></li>

</ul>
</li>
<li><p>Some of the colors used in the Sequence Editor can be modified
using the Configuration Options panel.  Open the panel with
<span class="menu">SE | Window | Options</span> and switch to the
<b>Sequence Editor</b> page
to modify the color of the displayed H-Bonds.  Choose lightShadow and
press Apply to put the change into effect. Close the options panel.
</p>
<p>Note that this change will be saved in your private resource file when
you quit MOE (for more information, see 
<a href="../appendix/moeconfig.htm">resource configuration file</a>).  Hence,
if you do not like this color, reset the color to one you prefer in the
Configuration Options.
</p></li>
<li><p>Change the Sequence Editor residue colors:
</p>
<p><b>SE | Display | Color Residues</b>
</p>
<p>Select Function to color residues according to function. Press
Apply to see the changes.
</p></li>
<li><p>Now color the atoms in the MOE Window by residue:
</p>
<p><span class="menu">MOE | Render | Atoms | Uncheck "C" box</span>
</p>
<p><span class="menu">MOE | Render | Atoms | Color | Residue</span>
</p></li>
</ol>

<h4><a name="SEPlots">Sequence Editor Plots</a></h4>

<p>Various sequence-only properties can be measured and plotted in the
Sequence Editor: 
</p>

<ol>
<li><p>Measure and plot the solvent accessibility:
</p>
<p><b>SE | Measure | Solvent Accessibility | Scalar1</b>
</p>
<p>Solvent accessibility is plotted for each residue. The Plot footer in
the Sequence Editor contains a toggle box for Accessible Surface.
</p></li>
<li><p>Hide the plot by turning off the Accessible Surface toggle
located in the SE footer.</p></li>

<li><p>Measure and display predicted secondary structure:
</p>
<p><b>SE | Measure | Predicted Secondary Structure</b>
</p>
<p>Three plots are generated, indicating the
probability that each residue will or will not have a structure.
The solvent accessibility plot is no longer available.
</p></li>
</ol>

<p>
In addition, MOE provides several other plots that are displayed
in the Protein Geometry application.
</p>
<ol>
<li>To display a Ramachandran plot (phi-psi):

<p><span class="menu">SE | Measure | Protein Geometry</span>
</p>
<p><span class="tip">Tip</span> 
You can locate which residues correspond to the data points in the plot
panel by dragging a selection box around the points.
This will select corresponding atoms in the MOE window.
The corresponding residues in the Sequence Editor can be selected
with
<span class="menu">SE | Selection | Residues | With Selected Atoms</span>.
</p>
</li>

<li><p>To examine sidechain rotamer quality, select <b>Rotamers</b> in the
<b>Check</b> menu of the Protein Geometry application.</p></li>
</ol>

<h4><a name="SESelectingAtoms">Selecting Atoms and Residues</a></h4>

<p>The selection facilities of the Sequence Editor can be used to
locate the atoms belonging to a given residue or the residues
containing selected atoms.  They can also be used to select residues
of a specific structure or property, and residues conserved across an
alignment. 
</p>

<ul>
<li><p>Select a group of residues by dragging the left mouse button
over residues in the Sequence Editor. This draws a selection box around 
the residues.
All residues
within this box will be selected.  Now
select a residue in the Sequence Editor by clicking on
it with the left mouse button. This deselects all other residues.
</p>
<p>Choose
<span class="menu">SE | Selection | Atoms | Of Selected Residues</span>. 
In the MOE Window, the atoms of that residue will be
added to the current atom selection set.</p></li>

<li><p>Clear the residue selection in the Sequence Editor by clicking the
left mouse button within the residue area <i>away</i> from any residue.
Select an atom in the MOE Window.  Now use the
<span class="menu">SE | Selection | Residues | With Selected Atoms</span>
command to find the residue containing that atom.</p></li>

<li><p>Select a residue by clicking in the horizontal ruler above the
residues.  The ruler indicates residue positions for alignment purposes. If
several sequences are loaded in the Sequence Editor, clicking in the ruler
will select all the residues (across chains) at that position or, in other
words, within the <i>column of residues</i>.</p></li>

<li><p>Select the atoms of that residue by choosing 
<span class="menu">Atoms | Select</span>
in the <b>Selected Residues</b> popup.  To activate the popup, click
the <i>mouse popup button</i>
in the residue drawing area of the Sequence Editor
<i>away</i> from any residue. 
</p></li>

<li><p>There are also Chain, Selected Chain, Residue, and Residue Column popup
menus.</p></li>

<li>You can link the selection states of the Sequence Editor and the main MOE 
window by enabling the <span class="menu">MOE (or SE) | Selection | Synchronize</span> 
option.  When this is enabled, selecting residues in the Sequence Editor 
automatically selects that residue's atoms in the main MOE window, and vice 
versa.</li>

</ul>


<h3><a name="Alignment">Aligning and Superposing Proteins</a></h3>

<p>This section covers two important homology
operations: sequence alignment and superposition.
</p>
<ol>
 <li><p>Choose <span class="menu">File | Close</span>
  to clear molecular data.</p>
 </li>

 <li><p>
  <span class="menu">File | Open | <tt>$MOE/sample/mol/serine_prot.moe</tt></span>
  </p>
 </li>
 <li><p>Open the Protein Alignment panel:
  </p>
  <p><b>SE | Homology | Align</b>
  </p>
  <p>Press OK to perform the alignment using the default values.
  </p>
  <p>Once the alignment is complete, MOE-Align will superpose the chains.
  </p>
  <p><b>Note:</b> For a more comprehensive view of the residues in the Sequence
   Editor, turn on
   <span class="menu">SE | Display | Single Letter Residues</span>
  (if previously turned off).
  </p>
 </li>
 <li><p>Next we will examine an alignment region by isolating the atoms of a
  selected range of residues.  
  </p>
  <ol type="a">
    <li><p>To select all residues within columns 1 and 60: click the left mouse
     button in the ruler at position 1, then press <tt>Shift</tt> and click the
     left mouse button at position 60. This selects the range of residues.
     </p>
    </li>
    <li><p>Position the mouse cursor away from any selected residue and open
     the Selected Residues popup menu.  Choose
     <span class="menu">View | Isolate &amp; View</span>.  Only the atoms of
     the selected
     residues now appear in the MOE Window.
     </p>
    </li>
    <li><p>In the MOE Window, hide the side chains by choosing
     <span class="menu">MOE | Render | Hide | Sidechain</span> and the hide the 
     ribbon display by choosing <span class="menu">MOE | Render | Ribbon | X</span>.
     This gives you a  better view of the backbone atoms.</p>
    </li>
  </ol>
 </li>
 <li><p>To fine tune the alignment, you can move residues directly in the
  Sequence Editor.  The movement is performed by clicking and dragging the middle
  mouse button on the residue to move.  Try dragging a residue now.
  </p>
  <p>To move a group of residues, select the residues and press
  <a class="svl" href="../moe/gui/mouse.html#MovingResidues">Alt</a> while
  dragging the middle mouse button.  In this case, all selected residues
  are moved.
  </p>
  <p>Note that a residue cannot be moved left of position 1. 
  </p>
 </li>
 <li><p>Finally, you can clear the alignment (all chains will be reset to
  start at position 1):
  </p>
  <ol type="a">
   <li>Select all the chains by clicking the left mouse button on 
    the box titled <b>Chain</b> next to the residue ruler.
   </li>
   <li>Reset the alignment using
    <b>SE | Homology | Reset Alignment</b>.
   </li> 
  </ol>
 </li>
</ol>

<h3><a name="Superpose">Superposing Protein-Ligand Complexes</a></h3>
<ol>
<li>
<p>Here we will load five protein-ligand complexes into MOE and superpose 
them using the <b>Superpose</b> application.  Remove any molecules you have 
loaded in MOE by choosing <span class="menu">MOE | RHS | Close</span>.
The first step will be to load and prepare five co-crystal CDK2 complexes in MOE.
</p>
<p><span class="menu">MOE | File | Open</span> Use ctrl-click to select PDB 
files 1g5s.pdb, 1gih.pdb, 1gij.pdb, 1ke8.pdb, 1ke9.pdb from <tt>$MOE/sample/mol/</tt>.
</p>
<p>Click <b>Load PDB File</b> and click <b>Yes</b> when 
prompted, then press <b>OK </b>to 
load the five PDB files into MOE.
</p>
<p class="cntr">
<img src="moetour/kinases.png" alt="Molecules in MOE Window" />
</p>
</li>
<li>
<p>Begin customizing the view by isolating the ligands and rendering the
protein backbones.</p>

<p><span class="menu">MOE | Popup | Hide | All</span></p>
<p><span class="menu">MOE | Popup | Show | Ligand</span></p>
<p><span class="menu">MOE | Popup | Ribbon | Line</span></p>

<p>Set the ribbon color to <b>Chain color.</b> Each protein will be
colored the same color as its chain number in the <b>SE</b>.</p>
<p>
<span class="menu">MOE | Popup | Ribbon | Color: Chain</span>
</p>
<p class="cntr"><img src="moetour/kinribbons.png" alt="Backbone Ribbons" /></p>
</li>
<li>
<p>Click the <b>SE</b> button to open the <b>Sequence Editor</b> and
view the protein sequence information. Next, we will align the
protein sequences and superpose the complexes. For this, open the
<b>Superpose</b><b> </b>menu.</p>

<p><span class="menu">SE | Homology | Superpose</span></p>

<p>In the <b>Superpose</b> set <b>Atom Sets</b> to Main and <b>Options</b> to 
Pocket Residues, then click <b>Superpose</b>.</p>

<p class="cntr"><img src="moetour/prosup.png" alt="Protein Superpose Panel" /></p>

<p>Click <b>Yes</b> when prompted to align the protein sequences. The sequence
alignment and superposition will be performed. During superposition, chains 
having the same tag will not move in relation to each other. This way, 
the protein ligand complexes will be aligned. Click <b>Report RMSD</b> in the 
<b>Superpose</b> panel and have a look at 
the resulting plot.</p>

<p class="cntr"><img src="moetour/rmsd_plot.png" alt="RMSD Report" /></p>

<p>Highlighted in yellow you can see the RMSD between chain 1 and chain 4. 
The RMSD values for the other superposed chains are shown in the color plot.
Click <b>Close</b> in the Protein Superpose panel to close both that panel 
and the RMSD matrix.</p>
</li>
<li><p>We will now visualise the active site. Click 
<span class="menu">MOE | RHS | SiteView</span> to view the aligned active sites. 
Now choose <span class="menu">MOE | RHS | Ligand | Ligand Interactions</span> to 
launch a 2D display of the interactions between each ligand and its receptor. 
Use the up and down arrows in the Ligand Interactions panel to browse through 
the five complexes, and click Isolate in the to view a particular complex in 
the main MOE window.</p>
<p class="cntr"><img src="moetour/ligint.png" alt="Ligand Interactions" /></p>

</li>
<li>
<p>The System panel can be used to control visibility, selection state and 
rendering of molecules in MOE. Click <span class="menu">MOE | RHS | System</span> 
to raise the panel; you will see a list of the chain tags in the current 
system. Click on the + button next to <b>1G5S</b> to expand that chain tag 
group; this allows control over the receptor, solvent and ligand molecules 
within that tag group.  Hold the mouse over the various buttons to read about 
the other functions available from this panel.</p>
<p class="cntr"><img src="moetour/kinsystem.png" alt="System panel" /></p>
</li>
</ol>


<h2>See Also</h2>
<a href="../moe/gui/mainwin.htm">MOE Window</a><br />
<a href="../moe/gui/atommgr.html">Atom Manager</a><br />
<a href="../mdb/db.htm">Database Viewer</a><br />
<a href="../moe/gui/se.html">Sequence Editor</a><br />
<a href="../moe/molsystems/build.html">Molecule Builder</a>

<p><a href="../index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
