<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<!--
!!	plif_tut.html	PLIF Tutorial
!!	$MANSRC/tutorials
!!
!!	19-oct-2009 (al) edited
!!	24-aug-2009 (kg) updated
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../manstyle.css" />


<title>PLIF Tutorial</title>
<style type="text/css">
body { background-color: white; }
.cntr { text-align: center; }
.resnap { border: solid thick maroon; }
#dbx { display: block; color: red; background-color: yellow; }
</style>
</head>

<body>
<a href="../index.htm"><img src="../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>PLIF Tutorial</h2>
<hr noshade="noshade" />

<p>This tutorial has been broken down into the following steps:</p>
<ul>
<li><a href="#I">Load and prepare five co-crystal CDK2 complexes in MOE</a></li>
<li><a href="#II">Generate and Analyze PLIFs</a></li>
<li><a href="#III">Generate a Pharmacophore Query from the PLIFs</a></li>
</ul>

<p>The Protein Ligand Interaction Fingerprints (PLIF) application is a
method for summarizing the interactions between ligands and proteins
using a fingerprint scheme. Interactions such as hydrogen bonds,
ionic interactions and surface contacts are classified according to
the residue of origin, and built into a fingerprint scheme which is
representative of a given database of protein-ligand complexes.</p>

<p>The input data for PLIF can be from a variety of sources, which most
commonly include X-ray crystal structures and docking results.
Fingerprints generated using PLIF are compatible with other
fingerprint tools found in MOE. Standard fingerprint tools such as
clustering and diverse subsets can be applied to fingerprints
generated by PLIF. There is a PLIF visualization interface which
illustrates the specific structural meaning of the fingerprints and
reveals which binding site residues interact with which parts of the
ligands throughout the database.</p>

<p>This tutorial provides an example of calculating, interpreting, and
utilizing PLIF on a set of five structures where CDK2 is
co-crystallized with different inhibitors.</p>

<h3>Conventions Used in this Tutorial</h3>

<table border="0" cellpadding="3">
<tr><td bgcolor="#cccccc">MOE</td><td>MOE Window</td></tr>
<tr><td bgcolor="#cccccc">MOE | Popup</td><td>MOE Window Popup</td></tr>
<tr><td bgcolor="#cccccc">SE</td><td>Sequence Editor</td></tr>
<tr><td bgcolor="#cccccc">DBV</td><td>Database Viewer</td></tr>
</table>

<p>
For example, <b>MOE&nbsp;|&nbsp;File&nbsp;|&nbsp;Open</b> refers to
the "Open" command of the "File" pull-down menu of the
MOE Window.
</p>

<h3><a name="I">Load and prepare five co-crystal CDK2 complexes in MOE</a></h3>

<ol>

<li><p><span class="menu">MOE | File | Open </span>
</p>

<p>
Change to the <tt>$MOE/sample/mol/</tt> directory and
hold down the <tt>&lt;Ctrl&gt;</tt> key to select the following PDB files:</p>

<blockquote>
<tt>1g5s.pdb</tt><br />
<tt>1gih.pdb</tt><br />
<tt>1gij.pdb</tt><br />
<tt>1ke8.pdb</tt><br />
<tt>1ke9.pdb</tt>
</blockquote>

<p>
Click <b>Load&nbsp;PDB&nbsp;File</b> and press <b>Yes</b>.
In the <b>Load&nbsp;PDB&nbsp;File</b> panel leave the default
options and press <b>OK</b> to load the five structures into MOE.</p> 

<p align="center">
<img src="plif_tut/plif2.png" alt="Molecules in MOE Window" />
</p>

</li>

<li><p>Open the Sequence Editor by pressing the <b>SEQ</b> button in the 
top right corner of the MOE Window or with 
<span class="menu">MOE | Window | Sequence Editor</span> </p>
<p>Select:</p>
<blockquote>
<span class="menu">SE | Display | Compound Name</span> <br />
<span class="menu">SE | Display | Color Chains | By Tag</span>.</p>
<blockquote>

<p align="center">
<img src="plif_tut/seqed_chnum.png" alt="Sequence Editor chain numbers" />
</p>
</li>

<li><p>Isolate the ligands and render the
protein backbones.  In the MOE Window, right mouse click to open the popup
menu and choose:</p>

<blockquote>
<span class="menu">MOE | Popup | Hide | All</span>
<span class="menu">MOE | Popup | Show | Ligand</span>
</blockquote>

<table>
<tr>
<td>
Open the <span class="menu">MOE | Popup | Ribbon</span>
panel and <br /> choose the <b>Line</b> mode and color by <b>Chain</b>.
</td>
<td>
<img src="plif_tut/render.png" alt="Ribbon Render panel" />
</td>
</tr>
</table>

</li>

<li><p>Click <span class="menu">MOE | RHS | System</span>.</p>
<p>The <a href="../apps/ligx.htm#System">System Popup</a> 
gives an overview of the molecular objects loaded in MOE.</p>
<p>To expand the list of objects for <tt>1G5S</tt>
click the top <img src="../apps/ligx/sys_plus.png" alt="+" /> button.</p>

<p>To color the carbon atoms by the chain color for all the ligands, hold the
<tt>&lt;Ctrl&gt;</tt> key, click the gray box next to 
<tt>Ligand 1G5S.A</tt> and in the popup, click the <b>Chain</b> button.</p>

<p align="center"><img src="plif_tut/system.png" alt="The System Popup" />
</p>

<p><span class="tip"><b>Tip:</b></span> &nbsp;&nbsp;
Bubble help text is shown listing keyboard modifiers
if the mouse cursor is held over widgets in the panel.</p>

<p align="center"><img src="plif_tut/plif3.png" 
alt="Ribbons and ligands colored by chain" /></p>

</li>

<li><p>Superpose the complexes. Open the <b>Protein Superpose</b> panel using
one of:</p>

<blockquote>
<span class="menu">MOE | Compute | Biopolymer | Superpose</span><br />
<span class="menu">SE | Homology | Superpose</span>
</blockquote>

<p>Use the defaults in the <b>Superpose</b> panel and press the
<b>Superpose</b> button.
Click <b>Yes</b> when prompted to align the protein sequences prior
to the superposition.</p>

<p align="center"><img src="plif_tut/super.png" alt="Protein Superpose panel" /></p>

<p>Chains with the same tag will move together as
a unit.  In this way protein-ligand complexes may be aligned. Close the
<b>Superpose</b> window and take a moment to view and rotate the aligned
structures in MOE.</p>

<p align="center"><img src="plif_tut/plif4.png" alt="Superposed proteins" /></p>
</li>

<li><p>Compute 2D Ligand Interaction diagrams for the five complexes with:</p>

<blockquote>
<span class="menu">MOE | Compute | Ligand Interactions</span>
</blockquote>

<p align="center"><img src="plif_tut/ints.png" alt="Ligand Interaction diagram" /></p>

<p>Since the ligand interactions were computed for an aligned set of complexes, 
the application will first compute a consensus configuration of the binding site 
residues. In the
<a href="../apps/prolig2d.htm">Ligand Interactions</a> panel, use
the &quot;up/down&quot; arrows to scroll between complexes &ndash; you
will see that the residues are displayed in the same orientation for
each diagram. Please note that the residues are displayed in a consistent 
orientation for each diagram. Note that the backbone oxygen of Leu83 
(Val83 in complex #3) consistently forms a hydrogen bond with each ligand.
Close the <b>Ligand Interactions</b> panel.</p>

</li>

<li>
<p>Create database of complexes which we will use for the PLIF calculations.</p>

<blockquote>
<span class="menu">MOE | File | New | Database</span>
</blockquote>

<p>Enter <b>cdk_pdbs.mdb</b> for the database name. Press <b>OK</b> to
open the new database into a Database Viewer. The new database will be empty
except for a <b>mol</b> field. Next, add the five complexes as
entries in the database.</p>

<blockquote>
<span class="menu">DBV | Edit | New | Entry</span>
</blockquote>

<p>In the <b>Add Entry</b> panel set the <b>Mode</b> as <b>Chain Tag</b> 
and press <b>OK</b>.  This will group together chains with the same 
tag into the same database entry.</p>

<table align="center">
<tr>
<td>
<img src="plif_tut/addentry.png" alt="Add Entry" />
</td>
<td>
<img src="plif_tut/plif5.png" alt="Database" />
</td>
</tr>
</table>

<p><span class="tip"><b>Tip:</b></span> &nbsp;&nbsp;
PLIF calculations can either use ligands from entries
in the database, with a protein structure in the main MOE window,
or protein-ligand complexes in each entry of the database.
In this case, the protein structures should have consistent residue 
UID numbers.</p>

</li>
</ol>

<h3><a name="II">Generate and Analyze PLIFs</a></h3>

<p> The <b>PLIF Setup </b>panel is used to compute the interactions and 
generate the fingerprints for a database of complexes. Thresholds
can be set for the 8 types of protein-ligand interaction which 
comprise the fingerprint (see the 
<a href="../apps/plif.htm">PLIF Documentation</a> 
for details on these interactions). </p>

<p>If no interactions of a particular category are found, or none pass 
the thresholds, no bits are set for that category. If an
interaction between the ligand and a protein residue passes the lower 
interaction threshold (<b>Min Score 1</b>), 
the low order fingerprint bit is set. If the interaction passes the
higher interaction threshold (<b>Min Score 2</b>), then the low order
and high order bits are both set. If there are several interactions
between atoms in the ligand and the protein residue, only the
strongest interactions are annotated in the fingerprint.</p>

<p>In the <tt>cdk2_pdbs.mdb</tt> database, open the <b>PLIF Setup</b> 
panel.</p>

<blockquote>
<span class="menu">DBV | Compute | PLIF | Generate...</span>
</blockquote>

<p align="center"><img src="plif_tut/plif6.png" alt="PLIF Setup panel" /></p>

<ol>

<li><p>Specify the receptor and the ligand.</p>

<p>Set the <b>Receptor </b>to <b>Database</b>. Enter <tt>ligand </tt> for
the <b>Ligand</b> field name (ligands will be written to this new
database field).</p>
</li>

<li><p>Perform raw contact analysis.  This scores the interactions
for each protein-ligand complex.</p>

<p>Using the default settings, press <b>Prepare </b>to perform the raw contact
analysis. A new field <b>PLIF_raw </b>will be written to the
database. Once the <b>Prepare </b>step is finished, a message will
appear reporting that the preparation is complete. Press <b>OK</b> to close the
message panel.</p>
</li>

<li><p>Compute the PLIFs.  This will select the most important interactions 
for the series of protein-ligand complexes.  By default up to 250 fingerprint 
bits will be stored.</p>

<p>Click <b>Generate </b> and then <b>Yes</b> when prompted
to build the fingerprints. The PLIFs will be written to a new
database field called <b>FP:PLIF</b>. The <b>PLIF</b> panel will
automatically open with a visual display of the results.
</p>
</li>

</ol>

<p>The <b>PLIF</b>
panel is used to visually analyze the fingerprints. From
this panel, statistical analyses can be applied on the fingerprint
bits and several operations such as generating a pharmacophore query
can be performed.  These fingerprints can also be used in the normal
MOE Fingerprint tools.</p>

<p align="center"><img src="plif_tut/plif7.png" alt="PLIF Analysis" /></p>

<p>By default, the <b>Display Mode</b> is set to <b>Barcodes</b>. In the
graphical display at the top of the panel, you can view the barcodes
for the five complexes along the y-axis. On the x-axis, you will see
the single letter codes of the residues for which fingerprint bits
were generated. For an example, the Ile10, shown in green, interacts
with the ligands in 3 out of the 5 complexes. The residues of the x-axis are 
color-coded consistently throughout the different
<b>Display Modes</b>.</p>

<p>The bottom portion of the <b>PLIF</b> panel contains the entry list,
which shows all of the fingerprints and other fields in a listbox,
allowing them to be selected. Various display options affect the
fields shown in the entry list, but at a minimum, the composition of
the visible fingerprint bits is shown. Each fingerprint bit is
denoted by a character to indicate its meaning: 
</p>

<blockquote><table border="0" cellpadding="5">
<tr><td><tt>-</tt></td><td>bit not set</td></tr>
<tr><td><tt>D</tt></td><td>sidechain hydrogen bond donor</td></tr>
<tr><td><tt>A</tt></td><td>sidechain hydrogen bond acceptor</td></tr>
<tr><td><tt>d</tt></td><td>backbone hydrogen bond donor</td></tr>
<tr><td><tt>a</tt></td><td>backbone hydrogen bond acceptor</td></tr>
<tr><td><tt>O</tt></td><td>solvent hydrogen bond</td></tr>
<tr><td><tt>I</tt></td><td>ionic attraction</td></tr>
<tr><td><tt>C</tt></td><td>surface contact</td></tr>
</table></blockquote>

<p>The <b>Display Mode</b> option buttons
control the presentation and interaction with the underlying data.</p>

<p><b>Display Mode: Population</b>.  This displays a
histogram showing the number of ligands with which each residue
interacts.</p>

<p align="center"><img src="plif_tut/plif8.png" alt="PLIF Population Plot" /></p>

<p><b>Display Mode: Ligands</b>.  This displays 2D depictions of
the ligands.
The color highlights of the ligands correspond to the interactions
shown in the <b>Barcodes</b> and <b>Population </b>modes. Notice that
all interactions with a single protein residue will have the same
color in all the display modes.  For instance, the interactions with
the Leucine/Valine at position 83 are common to all five 
complexes shown in the <b>PLIF</b> panel &ndash; as seen in the 2D 
ligand interaction diagrams.</p>

<p align="center"><img src="plif_tut/plif9.png" alt="PLIF Aligned Ligands" /></p>

<p><b>Display Mode: Significance</b>. When an activity threshold
has been specified, this shows the interactions which are found
more frequently in active ligands.
</p>

<h3><a name="III">Generate a Pharmacophore Query from the PLIFs</a></h3>

<p>In the <b>PLIF</b> panel, open the <b>Query Generator</b> panel:</p>

<blockquote>
<span class="menu">Tools: Query Generator</span>
</blockquote>

<p>This panel enables the user to generate pharmacophore queries based
on the frequencies of protein-ligand contacts.</p>

<ol>

<li><p>Generate the pharmacophore features.  
To use all ligands in the query generation, choose:</p>
<blockquote>
<span class="menu">Select Entries: All </span>
</blockquote>

<p>Press <b>Build </b>to construct the pharmacophore.</p>

<p align="center"><img src="plif_tut/plif10.png" 
alt="PLIF Pharmacophore Query Generator" /></p>

<p>The two pharmacophore features created by the <b>Query Generator</b> are 
also shown in the MOE Window. </p>
</li>

<li><p>Add receptor-generated excluded volumes to the query.</p>
<p>Turn on the <b>Excluded Volumes </b>option.</p>
</li>

<li><p>Create the pharmacophore query.</p>

<p>Press <b>Create </b>in the <b>Query Generator </b>to convert the 
pharmacophore features into a searchable pharmacophore query. Press <b>Yes </b>
when prompted.  The query will be converted into a searchable pharmacophore 
and the <b>Pharmacophore Query Editor</b> containing this query will open.</p>

<p align="center"><img src="plif_tut/plif11.png" 
alt="Pharmacophore Query Editor" /></p>

</li>
</ol>

<p>Now that you have finished this example, generating and analyzing PLIFs, you 
may want to take some time to explore the other options in the <b>PLIF</b> panel. 
For example, the similarity calculator selects ligands that make similar
interactions; the pose selector can be used to analyze docking results.
</p>

<h2>See Also</h2>

<a href="../apps/plif.htm">Protein Ligand Interaction Fingerprints</a><br /> 
<a href="../apps/ph4_query.htm">Pharmacophore Query Editor</a><br /> 
<a href="../apps/ph4_search.htm">Pharmacophore Search</a><br />
<a href="../apps/prolig2d.htm">Ligand Interaction Diagrams</a><br />
<br />
<a href="../moe/gui/mainwin.htm">MOE Window</a><br />
<a href="../mdb/db.htm">Database Viewer Window</a><br />
<a href="../moe/gui/se.html">Sequence Editor Window</a>

<p><a href="../index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
