<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<!--
!!	protalign_tut.htm Protein Alignment Tutorial
!!
!!	02-oct-2010 (hs) formatted
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../manstyle.css" />


<title>Protein Alignment Tutorial</title>
</head>

<body bgcolor="#ffffff">
<a href="../index.htm"><img src="../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>Protein Alignment Tutorial</h2>
<hr noshade="noshade" />

<h2>Introduction</h2>

<p>
3D protein alignment can be challenging due to small molecules associated
with the protein chain, movements of loops and domains unrelated to
binding, low sequence similarity and existence of multimers.
Alignment/superposition tools in MOE can take advantage of additional
information to address these issues, as listed here:
</p>

<p>
<b>Molecules associated with main protein chains</b>.
It is often desirable to move these molecules along with the master protein
chain during superposition.
(In MOE, this can be done by designating associated chains
as a unique <i>Protein Unit</i>
<i>Chain Tags</i>.)
</p>

<p>
<b>Pocket Superposition</b>.
If ligand structures are present, the superposition can be
limited to pocket residues to improve the results, and to allow a close
comparison of ligand-active site interactions among peer complexes.</p>


<p>
<b>Reducing the effect of moving domains on superposition</b>.
These areas can be located and excluded from alignment and superposition.
</p>

<p>
<b>Enhancing the similarity signal</b>.
When the similarity between protein structures is low, the sequence
and structural data of an internal database of pre-aligned protein families
may be used to enhance the similarity signal and improve the alignment
results.</p>


<p>The following is a brief list of MOE applications that are helpful for such
alignment studies:</p>



<p>
<blockquote>
<table cellpadding="3" width="75%">
<tr>
<td valign="top"><u>Application</u>
<td valign="top"><u>Functions</u>
</tr>
<tr>
<td valign="top">Protein-Align
<td valign="top">Sequence and 3D alignment.
</tr>
<tr>
<td valign="top">Protein-Superpose
<td valign="top">3D Superposition panel.
</tr>
<tr>
<td valign="top">Protein-Consensus
<td valign="top">Find/Analyze conserved regions of 3D superposed chains.
</tr>
<tr>
<td valign="top">MOE-PDB Search
<td valign="top">Search pre-aligned protein family database that
are homologues to a query.
</tr>

</table>
</blockquote>
</p>

<p>
This tutorial will demonstrate how to handle these issues using four exercises:</p>
</p>

<ol>
<li>
In the <a href="#Exercise1">first exercise</a>, we define <i>protein units</i>
to allow simple alignment of a molecular system containing miscellaneous chains
associated with the main protein chain. This will be done by editing the
<i>chain tags</i> in the <b>Protein-Superpose</b> application.

</li></p><p><li>
In the <a href="#Exercise2">second exercise</a>, to exclude the effect
of protein movements that are unrelated to binding site on the alignment
results, we limit the superposition to pocket residues using the
<b>Protein-Superpose</b> application.

</li></p><p><li>

In the <a href="#Exercise3">third exercise</a>, we use other related
proteins to enhance the similarity signal and improve the alignment. This
is done using the <b>PDB-Search</b> application and the internal protein
family database (<tt>pdb.fam</tt>).

</li></p><p><li>

In the <a href="#Exercise4">fourth</a> exercise we define a consensus
from structurally conserved regions of the proteins. Then we limit the
alignment/superposition to this consensus. This will improve the results
and will highlight the differences.  This will be done using the <b>Protein
Consensus</b> panel.

</li>
</ol>





<h3>Terminology</h3>

<p>This tutorial makes use of the following terms:
</p>

<p>
<center>
<table class="titlebar"  width="80%">
 <tr>
  <th>Term </th>
  <th>Domain </th>
  <th> Definition </th>
 </tr>


 <tr>
  <td><i>Protein&nbsp;Unit</i></td>
  <td>This tutorial</td>

  <td>A distinct protein complex including a single copy of a master protein
  chain and the associated molecules (if present). These may include ligand,
  solvent, and even the interacting peptides and secondary proteins.</td>

 </tr>


 <tr>
  <td><i>PDB ID</i></td>
  <td>General</td>
  <td>The four letter code identifying a Protein Data Bank entry.</td>
 </tr>

 <tr>
  <td><i>Chain&nbsp;Name</i></td>
  <td>General</td>

  <td>A single letter code used to refer to individual chains within a
  PDB file.  Often when multiple <i>protein units</i> are present in the
  PDB file, they are designated with a unique <i>chain name</i>.</p></td>
 </tr>

 <tr>
  <td><i>Chain Tag</i></td>
  <td>MOE</td>

  <td>An alphanumeric string used in MOE to tag individual chains. When
  opening a PDB file, by default the PDB-ID is copied to the <i>chain
  tags</i> of <b>all</b> the chains.Some MOE applications including the
  <b>Protein-Superpose</b> use unique <i>chain tags</i> to define <i>protein
  units</i>, thus there might be the need to manually edit the <i>chain
  tags</i> as will be demonstrated in this tutorial.</td>

 </tr>
</table>
</center>
</p>

<h3>Conventions Used in this Tutorial</h3>

<p><table cellpadding="3">
<tr>
<td valign="top" bgcolor="#cccccc">MOE
<td valign="top">MOE Window
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">SE
<td valign="top">Sequence Editor
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">RHS
<td valign="top">The right hand side button bar
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">Footer
<td valign="top">The footer button bar
</tr>
</table>



<a name="Exercise1"></a>
<h2>Exercise 1: Designating <i>Protein Units</i>
Using <i>Chain Tags</i>
</h2>

When trying to superpose protein chains, if
there are other corresponding structures present (ligands, water molecules etc.)
it is desirable to treat and move them along with the master protein
chain. Along with superposing the master proteins, MOE can move the
associated structures simultaneously, if they are designated as a <i>protein
unit</i>. In MOE, <i>chain tags</i> are used designate <i>protein units</i>.
The first part of this example shows how we can designate <i>protein
units</i> within a crystal structure containing multiple instances of the
protein.</p>

<p>In this example we use the
structure of polypeptide deformylase from two different bacterial species:
<b>2AI8</b>
is from <i>E.Coli</i>, and <b>2AIE</b> is from <i>Streptococcus pneumonia</i>.
</p>


<p>Clear the molecular data in MOE with <b>RHS | Close</b> and ensure that there
are no running tasks (use the Cancel menu in the MOE window).</p>


<ol><p><li>

<i>Load and examine the structure of the E.Coli protein.</i>

<blockquote><span class="menu">
MOE | RHS | Open | <tt>$MOE/sample/mol/2ai8.pdb</tt>
</span></blockquote>


<p>This will bring up the <b>Load PDB File</b> panel. Keep the default options
and press <b>OK</b>.</p>

<p>
Examine the contents using the <b>Sequence Editor</b> (<b>MOE | SEQ</b>);
this file contains three copies of the protein (chains 1&ndash;3)
followed by three
chains that each contain a ligand <tt>SB7</tt> and a nickel ion. The last three
chains are water molecules.
</p>

<p>

The <i>protein units</i> in this file are properly marked using the
<i>chain name</i>.  The <i>chain names</i> can be displayed in the
<b>Sequence Editor</b> by

</p>

<blockquote><span class="menu">
SE | Display | Compound Name
</span></blockquote>

<p>
The <i>chain names</i> will be displayed as shown below. The first unit
(<tt>unit A</tt>) includes chains <tt>1,4,7,</tt> thus their <i>chain
names</i> are labeled as <tt>2AI8.A</tt>.  Similarly, the second unit
includes of chains <tt>2,5,8</tt> which are named <b><tt>2AI8.B</tt></b>
and finally the third unit includes chains <tt>3,6,9</tt> which are
named <tt>2AI8.C</tt>.
</p>

<p align="center">
<img src="protalign_tut/apat_image001.png" alt="MOE-Protalign">
</p>

<p><b>Note:</b> Sometimes the <i>chain names</i> are not designated
properly in the original PDB file. In such cases, the corresponding chains
could be identified through visual inspection. For this example
however the chain names properly correspond to protein units, and will be
used to create <i>chain tags</i>.</p>

</li></p><p><li>

<i>Load the structure of the second protein.</i></p>

<blockquote> <span class="menu">
RHS | Open | <tt>$MOE/sample/mol/2aie.pdb</tt>
</span></blockquote>

<p>
This will bring up the <b>Load PDB File</b> panel. Keep the default
options and press <b>OK</b>.
</p>

<p>This file contains only one copy of the receptor and a different
non-peptide inhibitor <tt> SB7</tt>.  At this point, you should have a
total of 12 chains loaded in the system. Use the <b>Sequence Editor</b>
to verify this.
</p>


</li></p><p><li>

<i>Run the superposition.</i><br />

Open the <b>Protein Superpose</b> panel with

<blockquote> <span class="menu">
MOE | Compute | Biopolymer | Superpose
</span></blockquote>

<p>
or from the <b>Sequence Editor</b> using
</p>

<blockquote> <span class="menu">
SE | Homology | Superpose
</span></blockquote>

<p>
The panel will list all the chains in the system. The <b>Group
By Chain</b> option is selected by default, and since the default <i>chain
tags</i> are set to the PDB-ID and are similar, thus all chains from
<b><tt>2AI8</tt></b> are grouped together as one group (as shown below) and the
three chains loaded from 2AIE are grouped together as <b><tt>2AIE</tt></b>
as well.
</p>

<p align="center"><img src="protalign_tut/apat_image002.png" alt="MOE-Protalign"></p>

</li></p><p><li>
<i>Designate the first protein unit.</i>

<p> 
Select all the chains
labeled as <tt>2AI8.A</tt> from the list by pressing on the corresponding
lines. These would be chains <tt>1,4,7</tt>.  Enter a new and unique
<i>chain tag</i> for this unit in the <b>New Tag</b> textbox (for example,
<b><tt>unit1</tt></b>) and press <b>Apply</b>.<br>


<p align="center"><img src="protalign_tut/apat_select_unit_A.png"
alt="MOE-Protalign"></p>

<p>
As a result, the three related chains will be grouped based on their new
<i>chain tag</i> as shown here:
</p>

<p align="center"><img src="protalign_tut/apat_image004.png" alt="MOE-Protalign"></p>


</li></p><p><li>
<i>Designate the second protein unit.</i></p>

<p>There are still two <i>protein units</i> remaining from 2AI8 which
have the same <b>chain tag</b> (2AI8) and thus will be distinguished
as one <i>protein unit</i> if alignment is performed at this point. To
separate the two, one of these needs to be designated with a unique tag,
similar to what we did for the first protein unit.  To do this, select
all the chains labeled as <b><tt>2AI8.B</tt></b> from the list by
pressing on the corresponding lines. These would be chains 2,5,8.
Enter a new and unique <i>chain tag</i> for this unit in the <b>New Tag</b>
textbox (for example, <b><tt>unit2</tt></b> and press <b>Apply</b>.
</p>


<p>
Notice that we do not need to repeat this for the two remaining <i>protein
units</i>, as their chain tags are unique now (2AI8 and 2AIE). Now
that we have four unique <i>protein units</i>, we are ready to proceed
with the alignment and superposition.
</p>

</li></p><p><li>
<i>Verify all four protein units.</i>

<p>Press selection hint. This will select the master chain from each unit
as shown below.  The system is ready to perform superposition on
all four units now.</p>

<p align="center"><img src="protalign_tut/apat_image005.png" alt="MOE-Protalign"></p>


</li></p><p><li>
<i>Perform the superposition.</i>

<p>Press <b>Superpose</b> to run the superposition. You will
be asked to realign the chains before superposition. Select <b>Yes</b>
to proceed.</p>

<p> By default, the superposition algorithm starts with alignment of
the sequences using a modified version of Needleman and Wunsch algorithm
[Needleman&nbsp;1970]. Briefly, the alignment begins with building up
the initial pairwise similarity matrix using either a progressive
or a tree based method. Then a round-robin realignment is performed,
followed by a randomized iterative refinement. Finally, a structure based
realignment is performed using the existing or predicted protein secondary
element information. After this, 3D superposition will be done on
the best aligned model. For more details about the method, please see the
chapter on <a href="../proteins/align.htm">Multiple
Sequence and Structure Alignment</a>.


</li></p><p><li>

<i>Read the superposition RMSD.</i>

<p>Notice the superposition RMSD is
displayed on the top-left corner of the panel (4.7&Aring;).
</p>

<p align="center"><img src="protalign_tut/apat_image006.png" alt="MOE-Protalign"></p>


</li></p><p><li>
<i>Inspect the superposition results.</i>

<p>To visualize the resulting superposition perform the following
operations:
</p>

<p>
<ul>

<li>
Center the aligned structures in the MOE Window by
<blockquote> <span class="menu">
MOE | RHS | Center
</span></blockquote>
</li>

<li>
Draw the backbones and color them based on chain using the <b>Footer</b>:</li>
<blockquote>
<span class="menu">
MOE | Footer | Ribbon |
</span>
<img src="protalign_tut/apat_image007.png" style="vertical-align: bottom" />
</blockquote>

<p>
Select <b>Color: </b>
<img src="protalign_tut/apat_ribboncolorby_chains.png"
style="vertical-align:bottom" />
</p>
</li>

<li>
Hide the atomic structures using the following commands:
<blockquote> <span class="menu">
MOE | RHS | Hide | All Atoms
</span></blockquote>
</li>


<p>The result as shown here reveals that one <i>protein unit</i> shown in
dark green is considerably different from the others. This is the protein
unit from 2AIE.
</p>

<p align="center"><img src="protalign_tut/apat_image009.png" alt="MOE-Protalign"></p>


</ul>
</p>

</li></p><p><li>

<i>Compare the superposed ligands.</i>

<p>
To further investigate the superposition results, display the ligands by:
</p>

<blockquote> <span class="menu">
MOE | RHS | Show | Ligand
</span></blockquote>


<p>
To color the ligands by chain:
</p>

<blockquote> <span class="menu">
MOE | Footer | Atoms
</span></blockquote>

<p>
Choose <b> Color: Chain</b>
</p>

<p>Render the ligand using the ball&amp;stick model by:
</p>

<blockquote> <span class="menu">
MOE | Footer | Atom |
<img src="protalign_tut/apat_image010.png"
style="vertical-align: bottom;"></span>
</blockquote>


<p>
Then hide the ribbons by: 
</p>

<blockquote> <span class="menu">
MOE | Footer | Ribbon | Style |
<img src="protalign_tut/apat_image011.png"
style="vertical-align: bottom;"></span>
</blockquote>

<p>
Center the ligands in the view with
</p>

<blockquote> <span class="menu">
MOE | RHS | Center
</span>
</blockquote>


<p>The result, shown below demonstrates that the three ligands from the
three units of 2AI8 are aligned properly (cyan, orange, and yellow),
but the forth ligand from the 2AIE crystal structure (gray) shows poor
alignment. This is due to the movement of secondary elements in 2AIE
which are unrelated to ligand binding. In the next exercise, we will limit
the superposition to the pocket residues to improve the results.</p>

<p align="center"><img src="protalign_tut/apat_image012.png" alt="MOE-Protalign"></p>

<p>In order to compare corresponding non-bonded interactions
among related <i>protein units</i>, the RMSD should preferably be below
1&Aring;. Thus, for the current system, a more sophisticated superposition is
required which will be performed in the next exercise.</p>

</ol>



<a name="Exercise2"></a>
<h2>Exercise 2: Pocket Superposition</h2>

<p>
When proteins or enzymes bind to similar ligands and substrates, the similarity
between their pocket residues (in sequence and 3D conformation) is often higher
in comparison with the areas that are unrelated to binding. MOE can take
advantage of this and improve the superposition results by limiting the
superposition function to the active site residues.</p>


<ol>
<p>
<li>
<i>Limit the superpose to pocket residues.</i></p>

Continuing with the results from previous exercise, we will now improve the
superposition by limiting the superposition to the pocket residues.
This can be done
using the <b>Protein Superpose</b> panel.

<blockquote> <span class="menu">
MOE | Compute | Biopolymer | Superpose
</span></blockquote>

or<br>
<blockquote> <span class="menu">
SE | Homology | Superpose
</span></blockquote>

Under <b>Options</b>, change
the default (<b>All Residues</b>) to <b>Pocket Residues</b>.

<p>
Perform the superposition by pressing <b>Superpose</b> and choosing
<b>Yes</b> when prompted to realign the chains. Examine the improved
superposition via the improved RMSD printed on the top-left box (0.796&Aring;).</p>

</li></p><p><li>
<i>Examine the improved ligand superposition.</i>
<blockquote>
<span class="menu">
MOE | RHS | Hide | All Atoms
</span>
<br />
<span class="menu">
MOE | RHS | Show | Ligand
</span></blockquote>

<i>Center the ligands in the view.</i>
<blockquote> <span class="menu">
MOE | RHS | Center
</span></blockquote>


<p>As shown below, all four ligands from different
protein units show a good overlap now that the high similarity of
the corresponding binding pockets is used to improve the superposition.</p>


<p align="center"><img src="protalign_tut/apat_image013.png" alt="MOE-Protalign"></p>


</li></p><p><li>
<i>Compare the superposed pockets.</i> </p>
<blockquote>
<span class="menu">
MOE | Footer | Ribbon |
<img border=0 src="protalign_tut/apat_image007.png"
style="vertical-align: bottom;" />
</span>
<br />
<span class="menu">
MOE | RHS | SiteView
</span></blockquote>


<p>This will render the ligands, active site residues (including the backbone
cartoons) and water molecules from all four protein units as shown below.</p>

<p align="center"><img src="protalign_tut/apat_image014.png" alt="MOE-Protalign"></p>
</ol>


<a name="Exercise3"></a>
<h2>Exercise 3: Enhancing the Similarity Signal Using Related
Sequences</h2>


<p>Clear the molecular data in MOE with
<span class="menu">MOE | RHS | Close</span> and ensure that
there are no running tasks (use the Cancel menu in the MOE Window).</p>


<ol>
<p>
<li>
<i>Load the first protein structure.</i></p>
Load the structure of E.Coli polypeptide deformylase with
<blockquote> <span class="menu">
MOE | RHS | Open
| <tt>$MOE/sample/mol/2ai8.pdb</tt>
</span></blockquote>

<p>This will bring up the
<b>Load PDB File</b> panel. Keep the default options and press <b>OK</b>.</p>

</li></p><p><li>
<i>Load the second protein structure.</i>

<p> The file <tt>2ai8.pdb</tt> contains three <i>protein units</i>,
i.e. three main chains and their associated ligands and solvent
molecules. Only one unit is required for this exercise so we need to delete
the two units made from chains <tt>B</tt>, <tt>C</tt>.
</p>


<ul>
<li>
Bring up the <b>Sequence Editor</b> using
<blockquote> <span class="menu">
MOE | SEQ
</span></blockquote>

</li></p><p><li> Select chains the unwanted chains: hold <tt>Ctrl</tt> on
the keyboard and press on chain numbers <tt>2,3,5,6,8</tt> and <tt>9</tt>
to select them.  The result is as shown below.</p>

<p align="center"><img src="protalign_tut/apat_image015.png" alt="MOE-Protalign"></p>

</li></p><p><li>
Delete these chains using the <b>Sequence Editor</b> menu
<blockquote> <span class="menu">
SE | Edit | Delete Selected Chains
</span></blockquote>


<p>Press <b>OK</b> to confirm deletion.</p>


</ul>

</li></p><p><li>
<i>Perform a raw alignment.</i>
<blockquote> <span class="menu">
MOE | Compute | Biopolymer | Superpose
</span></blockquote>

<p> Keep the default options and press <b>Superpose</b>. When prompted
to allow realigning the chains select <b>Yes</b>. You will see in the
MOE Window that the two proteins are superposed. Also the <b>Sequence
Editor</b> shows that they are aligned.  However, due to low sequence
similarity (about 17%) , the RMSD shown on the <b>Protein-Superpose</b>
panel is high (6.733&Aring;).</p>


</li></p><p><li>
<i>Inspect the results.</i></p>
Hide all atoms by using
<blockquote> <span class="menu">
MOE | RHS | Hide | All Atoms
</span></blockquote>
Then show the ligands by using
<blockquote> <span class="menu">
MOE | RHS | Show | Ligand
</span></blockquote>

<p>Press <b>Center</b> to see the superposed ligands. The result (shown
below) demonstrates poor superposition of the two ligands.</p>

<p align="center"><img src="protalign_tut/apat_image016.png" alt="MOE-Protalign"></p>

To see the protein backbones use
<blockquote> <span class="menu">
MOE | Footer | Ribbon |
<img border=0 src="protalign_tut/apat_image007.png"
style="vertical-align: bottom;" />
</span></blockquote>

<p>Press <b>Center</b> to see the entire protein chains. It is clear
that the poor sequence similarity has resulted in poor alignment and
superposition.</p>


</li></p><p><li>
<i>Search the PDB for structurally related proteins.</i>

<p> In this step, we attempt to search the internal protein-family
database for sequences homologous to the first chain in the system
(i.e. <tt>2AI8</tt>). First, bring up the <b>Search-PDB</b> panel: </p>

<blockquote> <span class="menu">
MOE | Compute | Biopolymer | PDB Search
</span></blockquote>


<p>To use the sequence of chain 1 as a query, press the button
labeled <b>1</b>, as shown below. The
sequence of chain 1 will appear in the <b>Query</b> box.</p>

<p align="center"><img src="protalign_tut/apat_image018.png" alt="MOE-Protalign"></p>

<p> To run the search press <b>Search</b>. This will search the internal
<tt>pdb.fam</tt> database which is a pre-aligned database of protein
families. The result of this search will contain protein families that have
members with high sequence similarity to that of the query.  These families
will be listed under <b>Results</b>, sorted based on <i>E score</i> as
shown below.  </p>

<p align="center"><img src="protalign_tut/apat_image019.png" alt="MOE-Protalign"></p>


</li></p><p><li>
<i>Inspect the search results.</i>

<p>To view the members of a family, double click on the corresponding item
listed under <b>Results</b>. For this exercise, click on the first line that
contains a family.  This will bring up the <b>Load Alignment</b> panel as
shown below. Here you can see a list of PDB entries of the selected protein
family which are pre-aligned.  You can select some or all entries and load
them in MOE.  To load them all, press <b>Load All</b> . This will
load 20 proteins into MOE.</p>


<p align="center"><img src="protalign_tut/apat_image020.png" alt="MOE-Protalign"></p>


<p>Close the <b>MOE-SearchPDB</b> panel by pressing <b>Close</b>.</p>

</li></p><p><li>
<i>Align the query chains to new chains.</i>

<p>
Use <b>MOE-Align</b> to align the query protein chains (<tt>2AI8, 2AIE</tt>)
to the pre-aligned sequences. To do this, bring up the <b>MOE-Align</b>
panel with
</p>

<blockquote>
<span class="menu"> MOE | Compute | Biopolymer | Align</span>
</blockquote>

<p>To avoid re-alignment of the pre-aligned chains select these chains
from the list by clicking on the first entry, scrolling down to the bottom
of the list, and, with <tt>Shift</tt> held down, selecting the last entry
by clicking on it. Now press <b>Freeze</b>.  You will notice that the
first column changes from <b>*</b> (which means use any alignment group)
to A. This assigns all the extra chains to a single group (group A)
and freezes their alignment (which means designated alignment group A).
Turn off <b>Superpose Chains</b> and press <b>Apply</b>. Close the <b>Protein
Align</b> panel by pressing <b>Close</b>.</p>

</li></p><p><li>
<i>Repeat the superposition using new alignment.</i>

<p>
First, bring up the <b>Protein Superpose</b> panel with

<blockquote> <span class="menu">
MOE | Compute | Biopolymer | Superpose
</span></blockquote>

<p> To avoid re-alignment of the chains, select
<span class="menu">Sequence Alignment | Use Current Alignment</span>.</p>

<p>
Check <b>Print RMSD Table to CLI</b> to get the textual output of the
results.<br> Check <b>Accent Secondary Structural Matches</b> which helps
improve the superposition by increasing the weight of residues with matching
secondary structures.
</p>

<p>From <b>Options:</b> choose <b>Pocket Residues</b>. Press
<b>Selection Hint</b> to add the original chains to the selection.
Press <b>Superpose</b>.
</p>


<p> The superposition will be performed and the overall RMSD that is
printed on the <b>Protein Superpose</b> panel (0.725&Aring;) denotes an
improvement in comparison with previous result (6.7&Aring;).  </p>


</li></p><p><li>
<i>Read the detailed RMSD results</i>

<p> Open the SVL window (<b>MOE | SVL</b>) and find the corresponding RMSD
for the first two chains i.e. 2AI8.A and 2AIE.P (<tt>1.23</tt>&Aring;).  </p>

</li></p><p><li>
<i> Inspect the detailed RMSD results.</i></p>

<p>To see more details press <b>Report RMSD</b>. This will bring up the
<b>Protein Superpose RMSD</b> panel as shown below. There are two graphs on
this panel, the upper one is a heat-map style display of the RMSD matrix
and the one below is a histogram of the RMSD-by-residue comparison of two
chains that can be selected using the lists.</p>

<p align="center"><img src="protalign_tut/apat_image021.png" alt="MOE-Protalign"></p>


<p>A more detailed report of the superposition results can be obtained by
pressing <b>Report</b>.</p>



</li></p><p><li>
<i>Inspect the improved results.</i>

<p>First delete the added chains. These chains were only added to assist us
during the alignment operation and are not needed anymore. To delete these
chains, select the added chains in the <b>Sequence Editor</b> by clicking on
the chain number of the first added chain (should be number <tt>7</tt>),
then hold the <tt>Shift</tt> key on the keyboard, scroll down and click on
the last chain (number <tt>27</tt>).</p>


<p>
Delete the selected chains by:
</p>

<blockquote> <span class="menu">
SE | Edit | Delete Selected Chains 
</span></blockquote>

<p>
Hide all atoms by 
</p>

<blockquote> <span class="menu">
MOE | RHS | Hide | All Atoms
</span></blockquote>

<p>
Show the ligands by using
</p>

<blockquote> <span class="menu">
RHS | Show | Ligand
</span></blockquote>

<p>Press <b>Center</b> to see the superposed ligands. </p>

<p>
To see the protein backbones use
</p>

<blockquote> <span class="menu">
MOE | Footer | Ribbon |
<img border=0 src="protalign_tut/apat_image007.png"
style="vertical-align: bottom;" />
</span></blockquote>

<p>Press <b>Center</b> to see the protein chains. The results (shown below)
show good overlay of the ligands and the protein backbone.</p>

<p align="center"><img src="protalign_tut/apat_image022.png" alt="MOE-Protalign"></p>


<a name="Exercise4"></a>
<h2>Exercise 4: Alignment of Related Proteins on Conserved Regions</h2>

<h3>Introduction</h3>

<p> Proteins with high sequence similarity may be straightforward to align
by sequence, but the resulting 3D superposition may be less than desirable
if the structures are sufficiently dissimilar. In many cases the structural
dissimilarity between two related proteins is confined to localized mobile
regions such as activation loops, with the structural core remaining
conserved. Initial alignments of systems such as these are typically poor
because all of the residues in the aligned sequences are give equal weight in
the superposition, and the dissimilar regions guide the superposition away
from the conserved core. These alignments can be improved by superposing
and realigning the system based only on the conserved core.</p>

<p>In this example two holo forms of the EGFR tyrosine kinase domain
(PDB codes <tt>1M17</tt> and <tt>1XKK</tt> ) are aligned. Although
there is very high sequence similarity between the structures, there are
significant structural differences that arise from activation loop motion
and a hinge-like displacement of the N-terminal lobe. As a result, the
initial alignment of these structures needs to be improved by focusing
the alignment on the structurally conserved regions.</p>

<p>Clear the molecular data in MOE with
<span class="menu">MOE | RHS | Close</span> and ensure that there are no
running tasks (use the <b>Cancel</b> menu in the MOE Window).</p>


<ol>
<p>
<li>
<i> Load the two protein structures.</i></p>
<blockquote> <span class="menu">
MOE | RHS | Open
</span></blockquote>

<p> Then browse to the samples folder <tt>$MOE/sample/mol</tt> and select
the two pdb files by holding <tt>Ctrl</tt> on the keyboard and clicking
on the two files: <tt>1m17.pdb</tt> and <tt>1xkk.pdb</tt>.
</p>

<p>Press <b>OK</b> and choose <b>Yes</b> when asked to confirm. This
will bring the Load PDB File panel. Turn on <b>Ignore Waters</b> and
press <b>OK</b>.</p>


</li></p><p><li>
<i>Perform a simple alignment and superposition.</i>

<p>
First bring up the <b>Protein-Superpose</b> panel by
</p>

<blockquote> <span class="menu">
MOE | Compute | Biopolymer | Superpose
</span></blockquote>

<p>Then press <b>Superpose</b> and choose <b>Yes</b> when asked to allow
the alignment of the two chains.</p>

<p>When the alignment/ superposition was performed, the RMSD will be
displayed on the <b>Protein-Superpose</b> Panel (11.105&Aring;).</p>


</li></p><p><li>
<i>Visualize the results.</i>

<p>
To see the alignment of the ligands, hide all atoms by using
</p>

<blockquote> <span class="menu">
MOE | RHS | Hide | All Atoms
</span></blockquote>

Then show the ligands by using
<blockquote> <span class="menu">
RHS | Show | Ligand
</span></blockquote>

<p>Press <b>Center</b> to see the superposed ligands. The result (shown below)
demonstrates poor superposition of the two ligands.</p>

<p align="center"><img src="protalign_tut/apat_image023.png" alt="MOE-Protalign"></p>

<p> To see the alignment of the protein backbones using the MOE footer
controls, display the ribbons using

<blockquote> <span class="menu">
MOE | Footer | Ribbon |
<img border=0 src="protalign_tut/apat_image007.png"
style="vertical-align: bottom;" />
</span></blockquote>

<p>This will show the ribbon model of the two proteins as below. From the
pictures it is clear that even though the ligands corresponding secondary
structural elements are in proximity, lack of close superposition has
resulted in high RMSD.</p>

<p align="center"><img src="protalign_tut/apat_image025.png" alt="MOE-Protalign"></p>


</li></p><p><li>
<i>Limit the alignment/superposition to structurally conserved regions.</i>

<p>
First bring up the <b>Protein Consensus</b> panel using
</p>

<blockquote> <span class="menu">
MOE | Compute | Biopolymer | Consensus
</span></blockquote>

<p>Then press <b>Calculate</b>. This enables many controls in the <b>Protein
Consensus Definition</b> section.</p>

<p> To define the consensus as areas of the proteins with high structural
similarity, check <b>Mainchain RMSD</b>, and move the corresponding
<b>Max</b> slider to change the values of maximum from 10 to 3&Aring;. Press
<b>Calculate</b> to recalculate the consensus, then press <b>Superpose</b>
to repeat the superposition. The protein backbone and ligand structures
in the main MOE window show a much improved superposition, as displayed
below.</p>

<p align="center"><img src="protalign_tut/apat_image026.png" alt="MOE-Protalign"></p>

</li></p><p><li>
<i> Read the text report of the alignment.</i></p>
<p>Press <b>Report</b>. In the text report find the line for <b>RMSD
of Gapless Consensus Set </b>(1.46&Aring;). This low RMSD along with the
visual results confirms that most of the protein has not changed between the
two conformations. Inspect the 3D structure to see the areas that have
moved significantly between the two conformations. These are areas
that were excluded in defining the consensus sets.</p>


</li></p>
</ol>


<h2>References</h2>


<p><table width="100%" cellpadding="5">

<tr>
<td valign="top">[Needleman&nbsp;1970]</td>
<td valign="top">
Needleman, S. B.; Wunsch, C. D.; A General Method Applicable to
the Search for Similarities in the Amino Acid Sequences of Two Proteins;
<i> Journal of Molecular Biology 48 </i> (<b>1970</b>) 443&ndash;453.
</td>
</tr>
</table>
</p>



<h2> See Also</h2>
<a href="../proteins/prosup.htm">Protein Structure Superposition panel</a><br />
<a href="../proteins/procore.htm">Analysis of Structurally Conserved Core</a><br />
<a href="../proteins/align.htm">Multiple Sequence and Structure Alignment</a><br />


<h2>SVL Commands</h2>
<a class="svl" href="../proteins/fcnref/pro_sup.html">pro_Superpose</a><br />
<a class="svl" href="../proteins/fcnref/pro_alig.html">pro_Align</a><br />
<a class="svl" href="../moe/fcnref/superpos.htm">Superpose</a>

<p><a href="../index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
