<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
<meta http-equiv="Content-Style-Type" content="text/css" />
<link rel="stylesheet" href="../support/help.css" charset="ISO-8859-1" type="text/css" />
<title>Prime Structure Prediction &mdash; Input Sequence Step</title>
</head>
<body>
<table bgcolor="#dcdcdc" border="0" cellspacing="0" width="100%">
<tr><td>
<p><img src="../images/common/schrodinger_logo.png" align="left" alt=""
 border="0" hspace="5" vspace="5" /></p>
</td></tr>
<tr><td>
<h1 class="title">Prime Structure Prediction &mdash; Input Sequence Step</h1>
</td></tr>
</table>

<ul>
<li><a href="#summary">Summary</a></li>
<li><a href="#working">Working in the <span class="GUI">Input Sequence</span> Step</a></li>
<li><a href="#features"><span class="GUI">Input Sequence</span> Step Features</a></li>
<li><a href="#links">Related Topics</a></li>
</ul>

<a name="summary"></a>
<h2>Summary</h2>

<p>This step is used to read in protein (amino acid) sequences and designate the
query sequence.
</p>


<a name="working"></a> 
<h2>Working in the Input Sequence Step</h2> 

<p>The purpose of this step is to define a query sequence. You can read the
sequence from a sequence file, add it as text, obtain it from the PDB, or obtain
it from the Workspace. Several common sequence file formats are supported,
including FASTA, EMBL, GENBANK, and PIR. Sequences can be read from PBB files,
from the SEQRES records, if present, or from the ATOM records if SEQRES records
are missing. You can add more than one sequence, but only one can be used as a
query.
</p>

<p>Once you have obtained one or more sequences, you can select the sequence you
want to use for the query from the <span class="GUI">Sequences</span> list.
</p>

<p>If you want to modify the sequence you have selected as a query, click 
<span class="GUI">Edit Sequence</span>
and make any changes you want in the dialog box that opens. You might, for
example, want to remove extra terminal residues or insertions, or mutate the
sequence.
</p>

<p>When you are satisfied with the query sequence, click <span
class="GUI">Next</span> to go to the Find Homologs step.
</p>

<h3>Error Messages</h3>

<p>If one of these messages appears when you click <span
class="GUI">Next</span>, you will not be able to proceed until the problem is
resolved.</p>

<dl>
<dt><span class="GUI">Invalid File Message</span></dt>
<dd><p>This message is shown when a valid sequence cannot be extracted from the
file.</p>
<ul>
<li>Check that the sequence is in one of the supported sequence file
formats.</li>
<li>If it is in FASTA format, check that there are no extra spaces between the
"&gt;" and the sequence name and that there are no extra spaces at the end of
the line.</li>
</ul>
</dd>

<dt><span class="GUI">No Sequences Found</span></dt>
<dd><p>This message is shown when no valid sequences can be extracted
from the Maestro Workspace.</p>
<ul>
<li>Make sure that the correct project is open.</li>
<li>Check the Project Table to make sure the correct entries are included in the
Workspace.</li>
<li>Display the Workspace sequence viewer to check that the structure has a
sequence.</li>
</ul>
</dd>

<dt><span class="GUI">Invalid Sequence Characters</span></dt>
<dd><p>The sequence contains nonstandard one-letter codes.</p>
<ul>
<li>Check that the sequence is in one-letter code and is all in upper case.</li>
<li>Change or edit out any nonstandard characters.</li>
</ul>
</dd>

</dl>

<a name="features"></a> 
<h2>Input Sequence Step Features</h2> 


<ul>
<li><a href="#feature1"><span class="GUI">Get sequence from</span> buttons</a></li>
<li><a href="#feature2"><span class="GUI">Sequences</span> table</a></li>
<li><a href="#feature3"><span class="GUI">Edit Sequence</span> button</a></li>
</ul>

<dl>

<a name="feature1"></a>
<dt><span class="GUI">Get sequence from</span> buttons</dt>
<dd><p>Use these buttons to get sequences from various sources.
</p></dd>

<ul>
<li><p><span class="GUI">File</span>&mdash;Read the sequence from a
file. Several file formats are supported, including PDB, FASTA, EMBL, GENBANK,
and PIR. File formats are auto-detected by the program. Sequences from PDB files
are read from the SEQRES records, if they exist; otherwise they are read from
the ATOM records. If SEQRES records do not exist and residues are missing from
the ATOM records, the query sequence will contain gaps. For other file formats,
sequences must be in standard uppercase one-letter code.
</p></li>

<li><p><span class="GUI">Text</span> button&mdash;Enter the sequence as
text. Opens the <span class="GUI">Get Sequence from Text</span> dialog box, in
which you can type in the sequence, or paste it from the clipboard with <span
class="Keys">Ctrl+V</span> (<span class="Keys">&#x2318;V</span>).
</p></li>

<li><p><span class="GUI">PDB ID</span>&mdash;Obtain sequences from the Protein
Data Bank (PDB). Opens the <a href="get_sequence_from_pdb.html"><span
class="GUI">Get Sequence from PDB ID</span> dialog box</a>, in which you can
enter the PDB ID and optional chain name. The PDB file is downloaded and the
sequence read from the file, as described above.
</p></li>

<li><p><span class="GUI">Workspace</span>&mdash;Read sequences from proteins in
the Workspace. If there are duplicates, the sequence is read in only once.
</p></li>
</ul>

<a name="feature2"></a>
<dt><span class="GUI">Sequences</span> table</dt>
<dd><p>This table lists the sequences that have been added to the run. You can
select a single sequence as the query. This sequence is displayed in the
sequence viewer.
</p></dd>

<a name="feature3"></a>
<dt><span class="GUI">Edit Sequence</span> button</dt>
<dd><p>Edit the sequence that is selected in the <span
class="GUI">Sequences</span> table. Opens a panel in which you can edit the
sequence as a text string.
</p></dd>


</dl>


<a name="links"></a>
<h2>Related Topics</h2>
<ul>

<li><a href="find_homologs_step.html">Prime Structure Prediction - Find Homologs Step</a></li>
<li><a href="toolbar.html">The Prime Toolbar</a></li>
<li><a href="info_layout.html">Prime Panel Layout and Use</a></li>
<li><a href="info_help.html">Prime General Help</a></li>
<li><a href="info_nav.html">Prime Navigation and the Guide</a></li>

</ul>
<hr />
<table width="100%">
<tr><td><p class="small"><a href="../support/legal_notice.html" target="LegalNoticeWindow">Legal Notice</a></p></td>
<td>
<table align="right">
<tr><td><p class="small">
File: prime/input_sequence_step.html<br />
Last updated: 11 Jun 2014
</p></td></tr>
</table>
</td></tr>
</table>
</body>
</html>
