<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
<meta http-equiv="Content-Style-Type" content="text/css" />
<link rel="stylesheet" href="../support/help.css" charset="ISO-8859-1" type="text/css" />
<title>Prime Structure Prediction &mdash; Edit Alignment Step</title>
</head>
<body>
<table bgcolor="#dcdcdc" border="0" cellspacing="0" width="100%">
<tr><td>
<p><img src="../images/common/schrodinger_logo.png" align="left" alt=""
 border="0" hspace="5" vspace="5" /></p>
</td></tr>
<tr><td>
<h1 class="title">Prime Structure Prediction &mdash; Edit Alignment Step</h1>
</td></tr>
</table>

<ul>
<li><a href="#summary">Summary</a></li>
<li><a href="#working">Working in the <span class="GUI">Edit Alignment</span> Step</a></li>
<li><a href="#features"><span class="GUI">Edit Alignment</span> Step Features</a></li>
<li><a href="#links">Related Topics</a></li>
</ul>

<a name="summary"></a>
<h2>Summary</h2>

<p>The purpose of this step is to find a sastisfactory sequence alignment
between the templates and the query, which can be used to build a structure.
</p>


<a name="working"></a> 
<h2>Working in the Edit Alignment Step</h2> 

<p>There are several ways to produce an alignment for use in the <span
class="GUI">Build Structure</span> step:
</p>

<ul>
<li>Accept the BLAST/PSI-BLAST alignment generated in the <span class="GUI">Find
Homologs</span> step and currently displayed in the Sequence Viewer.</li>
<li>Import an alignment, by clicking <span class="GUI">Import</span>, to the
right of the <span class="GUI">Alignments</span> table.</li>
<li>Generate an alignment, either with ClustalW or with Prime SSP, by selecting
the appropriate <span class="GUI">Alignment method</span> option, then clicking
<span class="GUI">Align</span>. Prime SSP requires secondary structure
predictions (SSPs) (see below). </li>
<li>Manually edit any of these alignments.</li>
<li>If you are modeling a GPCR, select <span class="GUI">GPCR-specific
mode</span> and click <span class="GUI">Align</span> to create the
alignment.</li>
</ul>

<p>For technical details about the alignment program, see the
<a href="../../../prime/prime_user_manual.pdf"
target="pdfmanuals">Edit Alignment chapter</a> in the
<i>Prime User Manual</i>.</p>

<h3>Secondary Structure Predictions</h3>

<p>To generate an SSP for the query, click <span class="GUI">Run</span>. 
If you want to save an SSP for later use, right-click on an SSP for a menu of
options, including deleting the SSP, hiding all SSPs, or opening the
<span class="GUI">Export SSP</span> panel. If there is a difficulty with
exporting a file, an error message is displayed. To import an SSP, click the 
<span class="GUI">Import</span> button for <span class="GUI">Secondary structure
prediction</span>, and use the file selector to locate and import the SSP.</p>

<p>If you believe an SSP is incorrect, you may delete it entirely or edit the
incorrect portions. To delete the SSP, right-click on the SSP and choose
<span class="GUI">Delete</span>. To edit the SSP, click the <span
class="GUI">Edit SSP</span> toolbar button,</p>
<img src="../images/MaestroToolbarIcons/prime_edit_ssp.png">
<p>  
then highlight the part of the SSP you wish to change, and type the character
<tt>e</tt>, <tt>h</tt>, or <tt>-</tt>.  The highlighted region of the prediction
is changed to your choice: E (strand), H (helix), or - (loop).</p>

<p>To retrieve an unmodified SSP after editing, right-click on it and
choose <span class="GUI">Revert</span>.</p>

<h3>Editing the Alignment</h3>

<p>You can edit any alignment using combinations of the Prime toolbar buttons
<span class="GUI">Slide Freely</span>,
<span class="GUI">Slide as Block</span>,
<span class="GUI">Set Anchors</span>,
<span class="GUI">Lock Gaps</span>, and
<span class="GUI">Unlock Gaps</span>.

A preliminary model of the query is displayed in the Workspace,
colored by <span class="GUI">Residue Property</span>,
revealing the location of charged, polar,
and hydrophobic residues.</p>

<h3> Error Messages</h3>

<dl>
<dt><span class="GUI">Alignment check</span></dt>
<dd><p>This error dialog box lists possible alignment problems that you may want
to fix before continuing to the next step.  For example, gaps in secondary
structure elements of the template, or gaps in the query that are aligned to
secondary structure elements of the template.  Click <span
class="GUI">Continue</span> to go on to the next step anyway; click <span
class="GUI">Cancel</span> to remain in <span class="GUI">Edit
Alignment</span>.</p></dd>
</dl>

<a name="features"></a> 
<h2>Edit Alignment Step Features</h2> 


<ul>
<li><a href="#feature4"><span class="GUI">Alignment method</span> options</a></li>
<li><a href="#feature5"><span class="GUI">GPCR-specific alignment</span>
option</a></li>
<li><a href="#feature1"><span class="GUI">Run</span> button</a></li>
<li><a href="#feature2"><span class="GUI">Import</span> button</a></li>
<li><a href="#feature3"><span class="GUI">Align</span> button</a></li>
<li><a href="#feature7"><span class="GUI">Identify Globally Conserved
Residues</span> button</a></li>
<li><a href="#feature6"><span class="GUI">Family name</span> text box</a></li>
<li><a href="#feature8"><span class="GUI">Alignments</span> table</a></li>
<li><a href="#feature9"><span class="GUI">Import</span> and <span
class="GUI">Export</span> buttons</a></li> 
</ul>

<p></p>

<dl>


<a name="feature4"></a>
<dt><span class="GUI">Alignment method</span> options</dt>
<dd><p>Choose the alignment program to be used for alignment.</p>
<ul>
<li><p><span class="GUI">ClustalW</span>&mdash;Align the query and template
using ClustalW, based only on the sequence information. This is usually very
fast, and is best used when there is high sequence identity between the query
and the template.
</p></li>
<li><p><span class="GUI">Prime STA</span>&mdash;Align the query and template
using the Prime STA program, which uses secondary structure assignments as well
as sequence information. This choice requires secondary structure predictions
(SSPs).
</p></li>
</ul>
</dd>
<a name="feature5"></a>
<dt><span class="GUI">GPCR-specific alignment</span> option</dt>
<dd><p>Run an alignment that includes special features for aligning GPCRs, which
include fingerprint matching, a customized GPCR sequence database, and
identification of transmembrane helices.
</p></dd>

<a name="feature1"></a>
<dt><span class="GUI">Run</span> button</dt>
<dd><p>Run all available secondary structure prediction programs. SSpro is
bundled with Prime, but PSIPRED, which is an optional third-party program, is
highly recommended for optimal results, especially for GPCRs. PSIPRED is not
available on Windows. See <a href="info_third.html">Prime Third-Party
Programs</a> for more information on this program.
</p></dd>

<a name="feature2"></a>
<dt><span class="GUI">Import</span> button</dt>
<dd><p>Import a secondary structure prediction for the query. Opens a file
selector in which you can select the SSP file. The SSP must be in FASTA,
Maestro, or PSPIRED format. If the file contains bad data or the sequence
doesn't match the query sequence, an error message is displayed.
</p></dd>

<a name="feature3"></a>
<dt><span class="GUI">Align</span> button</dt>
<dd><p>Run a full alignment of the query to the template, using the chosen
alignment method. 
</p></dd>


<a name="feature6"></a>
<dt><span class="GUI">Query family name</span> text box</dt>
<dd><p>The family name for the query is displayed in this text box after you
search for families.
</p></dd>

<a name="feature7"></a>
<dt><span class="GUI">Identify Globally Conserved Residues</span> button</dt>
<dd><p>Run HMMER on the Pfam database to search for globally conserved residues
from a multiple sequence alignment of homologous proteins, and identify the
protein family. The conserved residues are shown in the sequence viewer, colored
according to the template color if both the template and the query have the
conserved residue. This allows you to easily identify conserved residues that
are not in the template or in the query.
</p></dd>

<a name="feature8"></a>
<dt><span class="GUI">Alignments</span> table</dt>
<dd><p>Lists the templates with their alignment score, number of identities,
positives, and gaps. 
</p></dd>

<a name="feature9"></a>
<dt><span class="GUI">Import</span> and <span class="GUI">Export</span>
buttons</dt>
<dd><p>Import an alignment from a file, or export an alignment to a file. These
buttons open a file selector, in which you can navigate to the desired location
and select or name the file. </p> 
<p><b>Note:</b> The template PDB ID must be uppercase in the alignment file.
If it is not, you must edit the alignment file before importing it.
</p></dd>

</dl>


<a name="links"></a>
<h2>Related Topics</h2>
<ul>

<li><a href="find_homologs_step.html">Prime Find Homologs Step</a></li>
<li><a href="build_structure_step.html">Prime Build Structure Step</a></li>
<li><a href="info_help.html">Prime General Help</a></li>
<li><a href="info_layout.html">Prime Panel Layout and Use</a></li>
<li><a href="info_nav.html">Prime Navigation and the Guide</a></li>
<li><a href="info_jobs.html">Prime Job Options and Results</a> </li>
<li><a href="toolbar.html">Prime Toolbar</a></li>
<li><a href="info_third.html">Prime Third-Party Programs</a></li>

</ul>
<hr />
<table width="100%">
<tr><td><p class="small"><a href="../support/legal_notice.html" target="LegalNoticeWindow">Legal Notice</a></p></td>
<td>
<table align="right">
<tr><td><p class="small">
File: prime/edit_alignment_step.html<br />
Last updated: 11 Jun 2014
</p></td></tr>
</table>
</td></tr>
</table>
</body>
</html>
