<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta http-equiv="Content-Style-Type" content="text/css">
<link rel="stylesheet" href="../support/help.css" charset="ISO-8859-1" type="text/css">
<meta name="categories" content="new_features prime">
<title>New Features in Prime 3.1</title>
</head>
<script type="text/javascript">
function setTitle()
{
top.document.title = document.title + " - " + parent.parent.WINDOW_TITLE;
}
</script>
<body onload="setTitle();">
<table border=0 cellspacing=0 bgcolor=#dcdcdc width=100%>
<tr><td>
<p><img src="../support/schrodinger_logo.gif" border=0 alt="" align="left" vspace=5 hspace=5 /></p>
</td></tr>
<tr><td>
<h1 class="title"><span class="prime">New Features in Prime 3.1</span></h1>
</td></tr>
</table>

<p>New and changed features in Prime are listed below.
</p>

<ul>

<h3>Structure Refinement</h3>

<li><p>When refining a homology model, you can automatically select the loops,
side chains, and atoms that did not come from the templates, by clicking
buttons.
</p></li> 


<li><p>You can refine multiple structures in a single run by specifying a
Maestro file with multiple structures as the structure input. By default all
structures in the file are processed, but you can specify which structures to
process with the <span class="GUI">ENTRY</span> keyword. 
</p></li>

<li><p>Positional constraints can be applied to non-C-alpha atoms during loop
refinement, by adding the appropriate <code>CONSTRAINT_</code><i>i</i> keywords
to the input file. The constraints are applied in the loop buildup stage. 
</p></li> 


<h3>Homology Modeling</h3>

<li><p>When choosing to do homology modeling, a dialog box opens offering
you the choice of using the <span class="GUI">Structure Prediction</span> 
panel or the <span class="GUI">Multiple Sequence Viewer</span>.
</p></li> 


<li><p>The toolbar has been reorganized, so that only the relevant buttons are
shown. 
</p></li> 

<li><p>Two more ways of importing a sequence have been added: from the PDB, by
specifying the PDB ID and optional chain ID, and from text, which can be typed
or pasted in.
</p></li> 

<li><p>Editing a sequence can only be done in the <span class="GUI">Input
Sequence</span> step. The toolbar button and menu item have been replaced with
an <span class="GUI">Edit Sequence</span> button below the <span
class="GUI">Sequences</span> table.
</p></li> 

<li><p>Multiple templates can be imported in the <span class="GUI">Find
Homologs</span> step from the Project Table, by selecting them and clicking 
<span class="GUI">From Project Table</span>. 
</p></li> 

<li><p>The <span class="GUI">Find Family</span> button is now labeled 
<span class="GUI">Identify Globally Conserved Residues</span> to indicate more
accurately what is being done. Along with this change, only the conserved
residues (upper case) are shown in the sequence viewer, colored if they match
both the query and the template.
</p></li> 

<li><p>Two alignment programs can be used for alignment: ClustalW and Prime
STA. The ClustalW alignment was available previously as <span class="GUI">Quick
Align</span>, but it is now the preferred method if the sequence identity is
high. The ClustalW alignment does not use secondary structure information. The
<span class="GUI">Prime STA</span> option is the method that was formely used
when you clicked <span class="GUI">Align</span>, and is still used for GPCR
alignment. 
</p></li> 

<li><p>There are two methods available for building structures: knowledge-based
and energy-based, which correspond to the former &quot;preview model&quot; and
&quot;build model&quot;. The options in the <span class="GUI">Build Structure -
Options</span> dialog box apply to both methods. 
</p></li> 

<li><p>The template residue numbering is preserved by default in the built
model, as far as possible, and there is an option in the <span class="GUI">Build
Structure - Options</span> dialog box to control this action, supported by an
input file keyword.
</p></li> 

<li><p>You can select multiple ligands for inclusion in a model with the usual
list-selection actions, shift-click and control-click.
</p></li> 

<li><p>The <span class="GUI">Ligands and cofactors</span> table shows only the
ligands and cofactors that are within 5&nbsp;&Aring; of the selected chain by
default. You can list all of them by selecting <span class="GUI">Show
all</span>. (Update 2)
</p></li> 

<li><p>Secondary structure predictions can be imported in native (Maestro) format
as well as FASTA format. (Update 2)
</p></li> 

<li><p>The full Pfam annotation can be displayed with a choice from the <span
class="GUI">Display</span> menu. (Update 2)
</p></li> 

<li><p>Fold recognition is no longer available.
</p></li> 



<h3>MM-GBSA</h3>

<li><p>You can adjust the flexible region of the protein by adding selected
residues, excluding selected residues, or limiting the region to selected
residues. An option menu and a set of picking tools has been added to the panel
for this purpose. 
</p></li> 

<li><p>The MM-GBSA program has been substantially rewritten, with new output. If
you are relying on output in the earlier format, you can run the old version
with the command</p>
<p><code>$SCHRODINGER/run -FROM psp prime_mmgbsa2011.py</code>
</p></li> 

<h3>Covalent Docking</h3>

<li><p>You can import and export ligand reactive group definitions. They are
stored in a CSV file, with the SMARTS pattern as the first column and the atom
index as the second.
</p></li> 

<h3>Other New Features</h3>

<li><p>Prime has been ported to MacOSX and is now available on Mac computers. 
</p></li>

<li><p>A unified script for structure building and refinement has been
introduced, named <code>prime</code>. This script replaces
<code>bldstruct</code>, <code>refinestruct</code>, and
<code>multirefine</code>. It can be used directly in place of these scripts,
with one addition to the input file: the keyword <code>jobname</code>, which
specifies whether building or refinement is being done, with the values
<code>model</code> and <code>refine</code>. 
</p></li> 

<li><p>The RMSD and the score from structural alignment are now available as
Maestro properties (<code>r_psp_StructAlign_RMSD</code>, 
<code>r_psp_StructAlign_Score</code>) with the aligned structures. 
</p></li>

<li><p>The automatic scanning alignment in the <span class="GUI">Protein
Structure Alignment</span> and the <span class="GUI">Align Binding Sites</span>
tools is always performed, so the option to run it has been removed from these
panels.
</p></li> 

<li><p>The <span class="GUI">Protein Structure Alignment</span> panel has been
updated. It now provides a choice of the reference residues and the residues in
the proteins that are to be aligned to the reference. It runs the <span
class="GUI">structalign</span> utility directly, which uses the optimized
alignment. The option for this feature and the <span class="GUI">Options</span>
button have therefore been removed. You can also force an alignment for low
similarity with a new option.
</p></li> 

<li><p>The results of a protein structure alignment have been moved out of the
<span class="GUI">Protein Structure Alignment</span> panel into a dialog box. 
(Update 2)
</p></li> 

</ul>


<hr />
<table width="100%">
<td><p class="small"><a href="../support/legal_notice.html" target="LegalNoticeWindow">Legal Notice</a></p></td>
<td>
<table align="right">
<td><p class="small">
File: new_features/prime_new_feat.html<br />
Last updated: 27 Aug 2012
</p></td>
</table>
</td>
</table>
</body>
</html>
