<!DOCTYPE html>
<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd" lang="en-US" xml:lang="en-US" data-mc-search-type="Stem" data-mc-help-system-file-name="Documentation.xml" data-mc-path-to-help-system="../" data-mc-target-type="WebHelp2" data-mc-runtime-file-type="Topic" data-mc-preload-images="false" data-mc-in-preview-mode="false" data-mc-toc-path="Command References|MacroModel Command Reference|MacroModel Operation Codes|Conformational Searching Opcodes">
    <head>
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <meta charset="utf-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <link href="../Skins/Default/Stylesheets/Slideshow.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/TextEffects.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Topic.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Components/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" /><title>Conformational Searching Opcodes</title>
        <link href="../Resources/Stylesheets/docs.css" rel="stylesheet" />
        <script src="../Resources/Scripts/custom.modernizr.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/jquery.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.config.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/foundation.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/plugins.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/MadCapAll.js" type="text/javascript">
        </script>
    </head>
    <body><a name="aanchor1255"></a>
        <MadCap:concept term="Command Reference" /><a name="aanchor1256"></a>
        <MadCap:concept term="MacroModel" /><a name="opcodes_509685220_801537"></a>
        <table class="NavLink" data-mc-conditions="Default.ScreenOnly">
            <tr class="NavLink">
                <td class="NavLeft"><span class="NavText">◀ <a class="NavLink MCXref xref xrefNavLink" href="opcodes_coordinate.htm" xrefformat="{paratext}">Coordinate Manipulation Opcodes</a></span>
                </td>
                <td class="NavCenter"><span class="NavText"><a class="NavLink MCXref xref xrefNavLink" href="macromodel_command_referenceTOC.htm" xrefformat="{paratext}">MacroModel Command Reference Manual</a></span>
                </td>
                <td class="NavRight"><span class="NavText"><a class="NavLink MCXref xref xrefNavLink" href="opcodes_mdmc.htm" xrefformat="{paratext}">Molecular Dynamics and Monte Carlo Simulation Opcodes</a> ▶</span>
                </td>
            </tr>
        </table>
        <h1 class="Title">Conformational Searching Opcodes</h1>
        <p>The entire contents of the input file are used as the “seed” for the search for most search methods. At the start of a run, the full contents of this file is read, and the search proceeds as if all the structures in the input file had been found in the current search. This allows a new search to use the results of a previously run, partial search as its input. Of course, for a new search, the input file will contain a single structure.</p>
        <p>The output file from the search contains in the header lines of the individual structures the number of times each structure was found. Therefore, the full benefits of usage-directedness are obtained in a subsequently run search. This degeneracy information is ignored, for technical reasons, during a network-distributed search. In a distributed search, or in any search seeded from an output file produced by an earlier version of MacroModel, the structures are treated as if they each occurred exactly once in a previous search. Debug (<span class="OpcodeLink"><a href="opcodes_misc_debug.htm#opcodes_509685220_debg">DEBG</a></span>) flag 360, if set, instructs the program to ignore the degeneracy information even if it would otherwise utilize it.</p>
        <p>Searches may be updated during program execution. When a search is updated, the information in the temporary file (<i>filename</i><code>.tmp</code>) is written to the output file, after discarding structures that are not within the specified energetic window of the current global minimum. A summary of the job progress is also printed, which includes information on the number of times various structures were found and the convergence of these structures during minimization. Automatic, periodic updates can be controlled with the <span class="OpcodeLink"><a href="#opcodes_509685220_mcop">MCOP</a></span> command.</p>
        <p>For a discussion of conformational search protocols, see <a class="TopicLink MCXref xref xrefTopicLink" href="../macromodel_user_manual/mm_conf_search.htm#mm_conf_search_3329033292_59461" xrefformat="{paratext}">MacroModel Conformational Searches</a>. As discussed there, it is advisable to perform the search without exhaustive minimization of all found structures and to subsequently reminimize the surviving structures. </p>
        <p>This can be done without initiating a new MacroModel job, by inserting into the <code>.com</code> file a <span class="OpcodeLink"><a href="opcodes_flowcontrol.htm#opcodes_509685220_rwnd">RWND</a></span> command following the <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> command that terminates the search setup, then inserting a <span class="OpcodeLink"><a href="opcodes_flowcontrol.htm#opcodes_509685220_bgin">BGIN</a></span><code>/</code><span class="OpcodeLink"><a href="opcodes_flowcontrol.htm#opcodes_509685220_end">END</a></span> loop within which a <span class="OpcodeLink"><a href="opcodes_file.htm#opcodes_509685220_read">READ</a></span><code>/</code><span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> sequence is specified. The <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_comp">COMP</a></span> atoms used in the search will continue to be active during the reminimization.</p>
        <p>The maximum number of conformations that can be stored when using <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_spmc">SPMC</a></span>, or <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_mult">MULT</a></span> is no longer limited to 10000. See the respective command descriptions and <a class="TopicLink MCXref xref xrefTopicLink" href="macromodel_capabilities_capacity.htm#macromodel_capabilities_1857967633_745216" xrefformat="{paratext}">MacroModel Program Capacity</a> for further details.</p>
        <p>The opcodes in this section are linked below:</p>
        <p>
        </p>
        <h2 class="Opcode"><a name="opcodes_509685220_739858"></a><code class="Courier"><a name="opcodes_509685220_auto">AUTO</a></code> — AUTOmatic setup</h2>
        <p>Perform automatic setup for <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>, <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span>, <span class="CodeLink"><a href="#opcodes_509685220_spmc">SPMC</a></span>, <span class="CodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span>, <span class="CodeLink"><a href="#opcodes_509685220_lmc2">LMC2</a></span>, and combinations of <span class="CodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> with <span class="CodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> or <span class="CodeLink"><a href="#opcodes_509685220_lmc2">LMC2</a></span>, on the current structure. This procedure takes into account the current substructure information. The setup for <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> is compatible with <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>/<span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> and <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>/<span class="OpcodeLink"><a href="#opcodes_509685220_lmc2">LMC2</a></span>. In addition, it can be applied in a serial manner (across different molecules in the input structure file) in <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>/<span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span>, and <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>/<span class="OpcodeLink"><a href="#opcodes_509685220_lmc2">LMC2</a></span> serial searches, but not in pure <span class="OpcodeLink"><a href="#opcodes_509685220_lmc2">LMC2</a></span> serial searches. </p>
        <p><code>AUTO</code> is incompatible with <span class="OpcodeLink"><a href="#opcodes_509685220_loop">LOOP</a></span> and must be used carefully with <span class="OpcodeLink"><a href="opcodes_embrace.htm#opcodes_509685220_mbae">MBAE</a></span>. In addition, <code>AUTO</code> is incompatible with frozen atoms unless a substructure is being used.</p>
        <p class="Note" data-mc-autonum="&lt;b&gt;Note:&lt;/b&gt; "><span class="autonumber"><span><b>Note:</b> <![CDATA[ ]]></span></span>Serial searches can generate large numbers of output structures, and the resulting output structure file can be large enough to exceed file size limits.</p>
        <p><code>AUTO</code> should be preceded by the <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_msym">MSYM</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_mcnv">MCNV</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_mcop">MCOP</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_lmc2">LMC2</a></span>, <span class="CodeLink"><a href="#opcodes_509685220_spmc">SPMC</a></span>, <span class="OpcodeLink"><a href="opcodes_constrained_mini.htm#opcodes_509685220_subs">SUBS</a></span>, and <span class="OpcodeLink"><a href="opcodes_file.htm#opcodes_509685220_read">READ</a></span> opcodes, if present, and is best placed just before the <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> line in the <code>.com</code> file.</p>
        <p>The type of calculation in use is detected automatically. For <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> calculations <code>AUTO</code> sets up comparison atoms, chiral atoms, and torsional constraints unless instructed not to. For <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> calculations <code>AUTO</code> sets up comparison atoms, chiral atoms, torsional constraints, molecule moves, variable torsions, and ring closures. For <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> and <span class="OpcodeLink"><a href="#opcodes_509685220_lmc2">LMC2</a></span> calculations, molecule moves, variable torsions, and ring closures are set up but not used, unless the setup is turned off. Series of bonded atoms that connect fixed or frozen atoms in substructures have ring closure bonds created within them to prevent rotation of fixed or frozen regions. Failure to find such a ring closure turns off all variable torsions in the <span class="OpcodeLink"><a href="opcodes_constrained_mini.htm#opcodes_509685220_subs">SUBS</a></span> atoms connecting such regions.</p>
        <p><code>AUTO</code> arg8 controls the extent to which variable torsions are applied to the input structures. This option is relevant mostly to amide and ester linkages, and supplies a greater degree of control over sampling of standard and nonstandard amides and esters. arg8=1 applies torsion moves to the amide/ester linkage of nonstandard groups like anhydrides, carbamates, hydrazones, and so on. Normal esters and amides are not sampled with this setting, but a torsional constraint is applied to them to ensure that the relative conformation of these groups is maintained. arg8=2 adds torsion moves to sample normal esters and amides in addition to amide and ester linkages of nonstandard groups. arg8=3 adds torsion moves to C=N and N=N bonds, in addition to all amide and ester derivatives. arg8=4 in addition removes torsional constraints involving hydrogen atoms.</p>
        <p>In automatic setup, the maximum number of torsions varied at the same time is set to the minimum of arg2 of <span class="OpcodeLink"><a href="#opcodes_509685220_mcnv">MCNV</a></span> and the number of torsions present in the current system. If <span class="OpcodeLink"><a href="#opcodes_509685220_mcnv">MCNV</a></span> arg2 has not been set then the number of torsions present in the current system is used. In serial calculations, the number of torsions varied at the same time is determined for each system.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">AUTO On/Off</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">−1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Turn off automatic setup.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Turn on automatic setup and generate a new setup using the current system.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Turn on automatic setup. If automatic setup is already on, use the existing setup.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Comparison atom setup</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">−1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use existing list of comparison atoms.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use previous setting. If none exists, arg2 is set to 2.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Create a new list of comparison atoms containing all non-hydrogen atoms.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Create a new list of comparison atoms containing all non-hydrogen atoms and hydrogen atoms bound to oxygen atoms.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Create a new list of comparison atoms containing all of the atoms.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Create a new list of comparison atoms containing all non-hydrogen atoms. Also use dihedral angle differences involving polar H atoms (bonded to a O, S or N atom) in redundant conformer elimination. If the dihedral angles in two conformers differ by more than a threshold (controlled by <code>CRMS</code> arg7) then the conformers are judged to be distinct.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">5 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Create a new list of comparison atoms containing all ring atoms. If no ring atoms are found, use heavy atoms and hydrogen atoms bonded to oxygen atoms.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Chiral atom setup</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">−1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use existing list of chiral atoms.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use previous setting. If none exists, arg3 is set to 1.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Create a new list of chiral atoms (overrides <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span> for the current structure).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Torsion checks</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">−1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use existing list of torsion checks.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use previous setting. If none exists, arg4 is set to 1.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Create a new list of torsion checks for carbon-carbon double bonds, amides, and esters.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Create a new list of torsion checks for carbon-carbon double bonds only.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Torsional selection</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">−1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use existing list of torsions.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use previous setting. If none exists arg5 is set to 1.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Create a new list of torsions to sample.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Serial calculation</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">−1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Turn off serial mode for <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> calculations.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use previous setting.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Turn on serial mode for <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> calculations. Serial <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> calculations are off by default. If <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> serial calculations have already been requested, then serial <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> calculations are also turned on automatically when the <code>AUTO</code> opcode is used regardless of the value of arg6.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg7</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Minimum ring size for ring closures and torsional sampling</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use the previous setting. If there is no previous setting, use the default setting.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Attempt to sample rings with at least this many atoms as members. The default value is 5 (five-membered rings or larger).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg8</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Sampling of torsions around bonds involving planar groups</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Specify sampling of torsions around amide and ester linkages, azo (N=N) and imino (C=N) groups.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Do not sample torsions around planar groups.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Sample torsions around nonstandard amide and ester linkages; do not sample other planar groups.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">2 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Sample all amide and ester linkages; do not sample azo and imino linkages.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara"> Sample all amide and ester linkages, and sample azo and imino linkages.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Sample all of the above and include hydrogen atoms in the sampling.</p>
                </td>
            </tr>
        </table>
        <p>Related <span class="OpcodeLink"><a href="opcodes_misc_debug.htm#opcodes_509685220_debg">DEBG</a></span> flags: 520, 521.</p>
        <h2 class="Opcode"><a name="opcodes_509685220_790835"></a><code class="Courier"><a name="opcodes_509685220_auop">AUOP</a></code>  — AUto OPtions </h2>
        <p>When <span class="OpcodeLink"><a href="#opcodes_509685220_auto">AUTO</a></span> is used in conjunction with an MCMM, mixed MCMM/LMCS, mixed MCMM/LMC2, SPMC, or CGEN conformational search, <code>AUOP</code> may be used to modify the number of steps depending on some measure of the number of degrees of freedom present. <code>AUOP</code> should be used immediately before the <span class="OpcodeLink"><a href="#opcodes_509685220_auto">AUTO</a></span> command.</p>
        <p>For <span class="OpcodeLink"><a href="opcodes_confgen.htm#opcodes_509685220_cgen">CGEN</a></span> by default the number of moves can also depend on ConfGen’s internal estimate for the number of degrees of freedom. See <span class="OpcodeLink"><a href="opcodes_confgen.htm#opcodes_509685220_cgo2">CGO2</a></span> arg3 for more information.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Number of times global minimum is found</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">The search will terminate when the global minimum is found this many times. Not recommended for ConfGen searches. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Exclude backbone torsions from torsion list</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Include protein backbone torsions in the conformational search.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Exclude protein backbone torsions from the conformational search.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg3 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Exclude variable chirality atoms from chiral atom list</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0,1 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use full list of chiral atoms.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">2 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Exclude atoms whose chirality can vary in solution. This currently only excludes nitrogen atoms bonded to 4 atoms one of which much be a hydrogen atom.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Number of steps</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&lt; 1 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">The search will use the number of steps specified by the search command itself, e.g. arg1 of <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> . </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue"> &gt; 1 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">The search will use (number of degrees of freedom) * arg5 steps if this is less than the number of steps specified by the search command itself. For MCMM, mixed MCMM/LMCS, mixed MCMM/LMC2 and SPMC searches the number of degrees of freedom is given by the number of torsions (<span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span>) and molecules moved (<span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span>) identified by AUTO. For CGEN jobs, the number of degrees of freedom can be estimated this way or from ConfGen’s internal estimate for the number of degrees of freedom, depending on the value of <span class="OpcodeLink"><a href="opcodes_confgen.htm#opcodes_509685220_cgo2">CGO2</a></span> arg3.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_763304"></a><code class="Courier"><a name="opcodes_509685220_lmcs">LMCS</a></code> — Low-Mode Conformational Search</h2>
        <p>This method is described by Kolossváry and Guida [<a class="Ref_Number_Only MCXref xref xrefRef_Number_Only" href="mmod_references.htm#mmod_references_3670527510_741428" xrefformat="{paranumonly}">30</a>, <a class="Ref_Number_Only MCXref xref xrefRef_Number_Only" href="mmod_references.htm#mmod_references_3670527510_741436" xrefformat="{paranumonly}">31</a>]. In tests so far, it has proved highly efficient, and it has the advantage that ring structures and variable torsion angles do not have to be specified. <code>LMCS</code> works by exploring the low-frequency eigenvectors of the system, which are expected to follow “soft” degrees of freedom, such as torsions. It is, however, helpful to specify independently movable molecules, by means of <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> commands. Like the <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> procedure, <code>LMCS</code> may be seeded or restarted with multiple input structures, and <code>LMCS</code> may also be used with distributed MacroModel. Most features that work with <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> also work with <code>LMCS</code>. <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_comp">COMP</a></span>, <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span>, <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_demx">DEMX</a></span>, and <span class="OpcodeLink"><a href="#opcodes_509685220_mcss">MCSS</a></span> commands should ordinarily be specified, and several <span class="OpcodeLink"><a href="#opcodes_509685220_mcop">MCOP</a></span> arguments take on special meaning when used with <code>LMCS</code>. </p>
        <p>The <span class="OpcodeLink"><a href="#opcodes_509685220_auto">AUTO</a></span> opcode can be used for automatic setup of <code>LMCS and </code><span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>/<code>LMCS</code> calculations.</p>
        <p>LMCS comes in two varieties: LMCS-SHAKE and mode-following. LMCS-SHAKE follows the low-mode vector, but every 2&#160;Å it applies a few steps of steepest descent minimization to relieve any strain due to distorted bond lengths and bond angles introduced by the move. </p>
        <p>The mode-following (eigenvector-following) option allows the user to apply the original low-mode search concept: “Since the potential energy hypersurface is a network of interconnected minima and saddle points, we reasoned that one could utilize a procedure that relies on eigenvector following for conformational searching. Thus, one could initiate the search by starting with any local minimum. By using one of the eigenvector-following techniques, one could locate a saddle point associated with this minimum and then the other minimum associated with this saddle point. By application of the eigenvector-following technique to the second minimum or to a different eigenvector of the first minimum, additional minima could be located which could then be used to find additional saddles, etc.” (quoted from the <i>J. Comp. Chem.</i> article cited above). </p>
        <p>Mode-following uses the same eigenvector-following saddle point search method as in the <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_sdlp">SDLP</a></span> command (see details there). If LMCS mode-following locates a saddle point, it proceeds as follows. The eigenvector for the negative eigenvalue is computed, followed by a short move along that eigenvector away from the starting structure. The resulting structure, which is slightly lower in energy than the saddle point structure, is then energy-minimized to locate the minimum energy structure on the other side of the saddle point.</p>
        <p>If LMCS mode-following cannot locate a saddle point, the starting (minimum) structure is restored and is automatically “rejected by starting geometry.” Thus, only minima found via saddle points are stored. This makes LMCS mode-following useful for the mapping of conformational interconversions in a local region of the potential energy surface. The recommended procedure for this is to set <code>LMCS</code> arg5=−1 (every step mode-following), and <span class="OpcodeLink"><a href="#opcodes_509685220_mcss">MCSS</a></span> arg1=0 (random walk structure selection) or <code>LMCS</code> arg4=1 (local search mode). </p>
        <p class="Note" data-mc-autonum="&lt;b&gt;Note:&lt;/b&gt; "><span class="autonumber"><span><b>Note:</b> <![CDATA[ ]]></span></span>LMCS mode-following is not intended for general conformational searches. Simple LMCS or LMCS-SHAKE (<code>LMCS</code> arg5&gt;=0) is much more efficient for this purpose. LMCS mode-following is only recommended for local searches exploring the conformational interconversions of a molecule. <span class="OpcodeLink"><a href="opcodes_misc_debug.htm#opcodes_509685220_debg">DEBG</a></span><code> 920</code> saves all the intermediate structures during LMCS-SHAKE or mode-following in the output structure file and colors them so that you can conveniently visualize the different multiple LMCS moves. </p>
        <p><code>LMCS</code> can be combined with <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> search. Tests have confirmed that the most efficient use of <code>LMCS</code> is to allow explicit torsional rotation of key torsion bonds, especially in acyclic structures. Therefore, you can combine <code>LMCS</code> with <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> commands (with <span class="OpcodeLink"><a href="#opcodes_509685220_rca4">RCA4</a></span> if necessary) without restriction. <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> commands can also be applied to translate/rotate independently movable molecules. See also the <span class="OpcodeLink"><a href="#opcodes_509685220_mcop">MCOP</a></span> command.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Number of Monte Carlo steps to be carried out before stopping</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Carry out the search until arg2 structures have been found. We do not recommend the use of this default.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg2 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum number of structures to retain while running </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">The maximum number of conformations to retain while running is the value specified in arg1 or 10,000, whichever is greater.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">The maximum number of conformations to retain while running.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Number of low-frequency modes </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara"><code>LMCS</code> will explore the first |arg3| number of modes. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 10 (pure modes).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&lt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara"><code>LMCS</code> explores a random linear combination of the first |arg3| number of modes, rather than exploring single pure modes at a time. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Search mode</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Global search (each Monte Carlo step begins with the preceding Monte Carlo structure, providing the structure is within 100 kJ of the global minimum). This is equivalent to <span class="OpcodeLink"><a href="#opcodes_509685220_mcss">MCSS</a></span> (arg1=0, arg2=0, arg4=100.0).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Local search (each Monte Carlo step begins with the original structure). Generally used with small coordinate variations in <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> or <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> to find other minima which are closely related to the starting structure. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Control of multiple LMCS steps </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: Single LMCS leap (MacroModel 6.0 behavior). </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Frequency of using LMCS-SHAKE (must be in range 0 to 1). </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&lt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">|arg5| is frequency of using mode-following (arg5 must be in range 0 to −1).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Allowable interatomic approach distance</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Fraction of sum of van der Waals radii used as a closest atomic approach limit. If a van der Waals pair comes closer together than this as the result of an LMCS move, the move is rejected before minimization.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 0.25</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other fraction.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg7</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Minimum distance (Å)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">In an LMCS move, a random total travelling distance is selected between the specified minimum and maximum values. The distances specified here and in arg6 correspond to the motion of the fastest-moving atom.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">It is often useful to perform a short conformational search with the default values of arg7 and arg8, and, based on the results, adjust these arguments accordingly. If many conformations minimize back to the starting conformation, increase arg7 (and perhaps arg8). If many conformations are ruled out because of distorted sp<sup>3</sup> carbons, decrease arg8 (and perhaps arg7). The information needed to make these choices will become visible by specifying <span class="OpcodeLink"><a href="#opcodes_509685220_mcop">MCOP</a></span> arg1=1.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 3 Å</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other value to be used.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg8</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum distance (Å)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 6 Å</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other value to be used.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740031"></a><code class="Courier"><a name="opcodes_509685220_lmc2">LMC2</a></code> — Low-Mode Conformational search for large molecules</h2>
        <p>This method, termed <code>LLMOD</code>, is described by Kolossváry and Keseru <a class="Reference_Number MCXref xref xrefReference_Number" href="mmod_references.htm#mmod_references_3670527510_741444" xrefformat="[{color #0E6895}{paranumonly}{/color}]">[<span style="color: #0e6895;" class="mcFormatColor">32</span>]</a>. <code>LLMOD</code> has been developed for large-scale conformational searching, such as protein loop optimization, homology model refinement, and fully flexible docking for induced fit modeling, only to mention a few applications <a class="Reference_Number MCXref xref xrefReference_Number" href="mmod_references.htm#mmod_references_3670527510_741464" xrefformat="[{color #0E6895}{paranumonly}{/color}]">[<span style="color: #0e6895;" class="mcFormatColor">33</span>]</a>. <code>LLMOD</code> is very similar to <code>LMOD</code>, which is implemented in MacroModel as the <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> command, but <code>LLMOD</code> utilizes the ARPACK package to compute low-mode eigenvectors of a Hessian matrix that is only referenced implicitly, through its product with a series of vectors. The Hessian <span class="Symbol">´</span> vector product can be calculated by a number of methods. <code>LLMOD</code> is the first conformational search method that can be applied to fully flexible, unconstrained protein structures.</p>
        <p><code>LMC2</code> is implemented to extend the basic use of <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> to molecules of virtually any size. Note that two main features of <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span>, LMCS-SHAKE and LMCS mode-following, are not implemented in <code>LMC2</code>. The command arguments of <code>LMC2</code> are fairly similar to those of <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span>. Like <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span>, <code>LMC2</code> can also be used in conjunction with the <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> command to explicitly alter key torsion bonds via <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> commands (with <span class="OpcodeLink"><a href="#opcodes_509685220_rca4">RCA4</a></span> to open macrocyclic structures if necessary) and apply explicit translation/rotation of a ligand with the <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> command for docking. <code>LMC2</code> is primarily designed for fully flexible molecules, but the use of frozen atoms is allowed, and recommended in cases where parts of a macromolecule can be treated rigidly. Also note that you can set certain parameters controlling the ARPACK package via the <span class="OpcodeLink"><a href="#opcodes_509685220_arpk">ARPK</a></span> command.</p>
        <p>The <span class="OpcodeLink"><a href="#opcodes_509685220_auto">AUTO</a></span> opcode can be used for automatic setup of <code>LMC2</code> and <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>/<code>LMC2</code> calculations. It can be used for <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>/<code>LMC2</code> serial searches but not for pure <code>LMC2</code> serial searches.</p>
        <p><span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> is more efficient than <code>LMC2</code> when the number of atoms is small enough for the entire Hessian to be stored in computer memory without swapping.</p>
        <p class="Note" data-mc-autonum="&lt;b&gt;Note:&lt;/b&gt; "><span class="autonumber"><span><b>Note:</b> <![CDATA[ ]]></span></span>It has proven to be good practice and time-saving to minimize molecular structures during a conformational search to a gradient that is not too low and only re-minimize entirely the final set of low-energy conformations. In that respect, <code>LMC2</code> needs special attention. It is still recommended to follow this scheme, e.g., for unconstrained proteins, a gradient RMS of 1&#160;kJ/mol/Å is sufficient during the search, but the very first structure, for which the low-modes are computed, must be pre-minimized to a low, &lt;&#160;0.1&#160;kJ/mol/Å gradient RMS in order to derive reasonable modes. Pre-minimization can either be done in a separate MacroModel job, or the <span class="OpcodeLink"><a href="opcodes_flowcontrol.htm#opcodes_509685220_rwnd">RWND</a></span> command can be used to fully minimize the first structure, rewind the output file and start the <code>LMC2</code> conformational search from there.</p>
        <p class="Note" data-mc-autonum="&lt;b&gt;Note:&lt;/b&gt; "><span class="autonumber"><span><b>Note:</b> <![CDATA[ ]]></span></span>Large-scale low-mode calculations can use a large amount of memory. If memory issues are encountered, consider eliminating the use of solvation. In addition, reducing non-bonded cutoffs also reduces the memory required for the computation.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Number of Monte Carlo steps to be carried out before stopping</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Carry out the search until arg2 structures have been found. We do not recommend the use of this default.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg2 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum number of structures to retain while running </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">The maximum number of conformations to retain while running is the value specified in arg1 or 10,000, whichever is greater.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">This is the maximum number of conformations to retain while running.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Number of low-frequency modes</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara"><code>LMC2</code> explores the first |arg3| modes. A maximum of 100 modes is allowed.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 30 (pure modes).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&lt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara"><code>LMC2</code> explores a linear combination of all |arg3| low-frequency modes, rather than exploring single pure modes at a time. Note that <code>LMC2</code> employs a better mixing strategy than <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span>, which simply uses a random mixture of low modes. <code>LMC2</code> perturbs the starting structure by moving along the first |arg3| modes successively. Each move is carried out exactly the same way as for a selected pure mode, but each move employs a different random distance (see arg7/arg8).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Search mode</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Global search: each Monte Carlo step begins with the preceding Monte Carlo structure, providing the structure is within 100 kJ of the global minimum. This is equivalent to <span class="OpcodeLink"><a href="#opcodes_509685220_mcss">MCSS</a></span> (arg1=0, arg2=0, arg4=100.0).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Local search: each Monte Carlo step begins with the original structure. Generally used with small coordinate variations in <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> or <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> to find other minima which are closely related to the starting structure. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Determines when low-mode eigenvectors are calculated</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">ARPACK calculations on proteins are time consuming. The time span of a single low-mode calculation can vary from a few minutes to an overnight job. It is highly recommended that eigenvectors be re-calculated only when it is absolutely necessary, i.e., when there is evidence that low modes calculated for a particular conformation cannot be transferred to other conformations. In other words, as long as a single set of low modes can produce efficient search directions starting from a number of different local minima, there is no need to calculate a different set of modes for every new conformation.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: Calculate the low modes for the very first minimized structure only and use those modes throughout the entire search (recommended).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Low-mode eigenvectors are recalculated every time the search finds a new global minimum. The modes of the current global minimum are always used.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&lt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Low modes are recalculated for every new structure (<span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> behavior).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Allowable interatomic approach distance</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Fraction of sum of van der Waals radii used as a closest atomic approach limit. If a van der Waals pair comes closer together than this as the result of an LMCS move, the move is rejected before minimization.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 0.25</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other fraction.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg7</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Minimum distance (Å)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Same as <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span>, except that the default values are different, depending on arg3.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 3 (Å) for single mode, arg3 &gt;= 0</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">(Å) for mixed mode, arg3 &lt; 0</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other value to be used.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg8</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum distance (Å)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Same as <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span>, except that the default values are different, depending on arg3.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 6 (Å) for single mode, arg3 &gt;= 0</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">(Å) for mixed mode, arg3 &lt; 0</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other value to be used.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740091"></a><code class="Courier"><a name="opcodes_509685220_loop">LOOP</a></code> — protein LOOP conformation generation</h2>
        <p><code>LOOP</code> is a conformational search method for protein loops that works in conjunction with the <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> opcode in a manner similar to <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>.</p>
        <p>Given a single protein structure <code>LOOP</code> attempts to generate candidate loop conformations using either the specified loop from the protein structure provided or an alternate user specified sequence for the loop. <code>LOOP</code> rapidly generates a candidate structure with the appropriate bond lengths and angles. In addition, heavy atoms in this structure (i.e., atoms heavier than hydrogen) will not approach each other more closely than a specified distance. These candidate structures are then energy minimized and can be checked for uniqueness in a manner analogous to that for <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>. A more general discussion of the use of <code>LOOP</code> is available in <a class="TopicLink MCXref xref xrefTopicLink" href="mm_protein_loop.htm#mm_protein_loop_2245627268_738161" xrefformat="{paratext}">Protein Loop Construction with MacroModel</a>.</p>
        <p><code>LOOP</code> generates candidate structures in two stages: </p>
        <ul>
            <li value="1">
                <p class="ListPara">Initial loop generation via random selection of rotatable dihedral angles without concern for loop closure</p>
            </li>
            <li value="2">
                <p class="ListPara">“Tweak,” an iterative process involving the systematic adjustment of the rotatable bonds to attain loop closure [<a class="Ref_Number_Only MCXref xref xrefRef_Number_Only" href="mmod_references.htm#mmod_references_3670527510_741474" xrefformat="{paranumonly}">34</a>, <a class="Ref_Number_Only MCXref xref xrefRef_Number_Only" href="mmod_references.htm#mmod_references_3670527510_741483" xrefformat="{paranumonly}">35</a>] while eliminating very close encounters amongst heavy atoms.</p>
            </li>
        </ul>
        <p>In the loop generation stage, the N-terminus and C-terminus halves of the loop are constructed separately, starting from their attachment points, by growing the chain out to the next rotatable bond. Then a random dihedral angle is chosen for this bond and the heavy atoms positioned unambiguously as a result are checked for close approaches with heavy atoms in the rest of the protein (not the loop itself). If there are such close approaches, the process of selecting a new random angle and checking for close approaches is repeated until either <i>N</i><sub>try</sub> such attempts are made or no close approaches result. If <i>N</i><sub>try</sub> such attempts fail then the loop construction process retreats back to the previous rotatable bond for which a new random angle is chosen. When a dihedral angle has been chosen successfully, the process focuses on the next rotatable bond out from the attachment point. Rotatable bonds in side chains are also sampled in this manner. The loop generation stage produces a random structure for the protein loop that is not connected in the middle and does not have close approaches between the loop atoms and the rest of the protein, but there may such close approaches within the loop itself.</p>
        <p>In the second stage the Tweak algorithm is used to bring the middle of the loop together in the right geometry. Briefly, this algorithm proceeds by iteratively selecting a set of minimal changes to the rotatable bonds that lead toward loop closure based upon a linear approximation. Loop closure is specified by a set of target distances that must be met to within a tolerance of 0.001 Angstroms. Along the way close approaches involving heavy atoms of the loop atoms and the rest of the protein are removed by temporarily introducing additional target distances between the overlapping atoms. When the set of distances for closing the loop are met to within 1 Angstrom the Tweak algorithm also attempts to remove close approaches amongst the heavy atoms within the loop. By using a minimal set of changes to the rotatable bonds the Tweak algorithm retains as much of the original, random loop conformation in torsional space as possible and thus provides a mechanism for generating a diverse set of candidate loop structures.</p>
        <p><b class="Note">Limitations:</b>
        </p>
        <ul>
            <li value="1">
                <p class="ListPara">Only one loop can be treated, although successive calculations may target different loops.</p>
            </li>
            <li value="2">
                <p class="ListPara">An all-atom representation of the protein must be used.</p>
            </li>
            <li value="3">
                <p class="ListPara">The protein loop and the residues that it is immediately attached to must consist of alpha amino acids.</p>
            </li>
            <li value="4">
                <p class="ListPara">Rings (e.g., the one in proline) are treated as rigid.</p>
            </li>
            <li value="5">
                <p class="ListPara">No disulfide bonds are permitted within the loop or between the loop and the rest of the protein.</p>
            </li>
            <li value="6">
                <p class="ListPara">No atoms in the loop can be frozen.</p>
            </li>
        </ul>
        <p>Using <code>LOOP</code> usually results in a renumbering of the atoms within the structure such that the loop atoms become the highest numbered atoms in the structure. <code>LOOP</code> tries to make this work by automatically shifting the atom numbers provided in <span class="OpcodeLink"><a href="opcodes_constrained_mini.htm#opcodes_509685220_subs">SUBS</a></span>, <span class="OpcodeLink"><a href="opcodes_constrained_mini.htm#opcodes_509685220_fxat">FXAT</a></span>, <span class="OpcodeLink"><a href="opcodes_constrained_mini.htm#opcodes_509685220_fxdi">FXDI</a></span>, <span class="OpcodeLink"><a href="opcodes_constrained_mini.htm#opcodes_509685220_fxba">FXBA</a></span>, <span class="OpcodeLink"><a href="opcodes_constrained_mini.htm#opcodes_509685220_fxta">FXTA</a></span>, <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_comp">COMP</a></span>, and <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span> commands. There are also routines for automatically generating <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_comp">COMP</a></span> and <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span> atoms for the loop atoms as described for arg3, below. To facilitate subsequent studies using the structures generated, a new substructure file, <i>out_filename</i><code>.sbc</code>, containing the shifted atom numbers is generated automatically. In addition, shifted <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_comp">COMP</a></span> and <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span> commands suitable for use in <code>.com</code> files are written in the <code>.log</code> file.</p>
        <p>If comparison atoms (<span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_comp">COMP</a></span>) are specified explicitly or implicitly (see arg 3) then <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_msym">MSYM</a></span> must be used.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">N-terminus atom number for the loop</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">This is the atom number for the peptidic Nitrogen atom that joins the loop to the protein at the N-terminus of the loop. This must be nonzero.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg2 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">C-terminus atom number of the loop</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">This is the atom number for the peptidic Carbon atom that joins the loop to the protein at the C-terminus of the loop. This must be nonzero.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg3 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Type of LOOP Generation</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">−1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Turn off <code>LOOP</code> and clear out information specified in an earlier loop command. All other arguments are ignored.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Generates alternate conformations for the loop specified by args 1 and 2. In this mode the first structure minimized is that originally provided.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Remove the loop specified by args 1 and 2 and replace it with the amino acid sequence specified in an <i>in_filename</i><code>.lsq</code> file. Candidate loop structures will be generated for this new loop. This file consists of a list containing codes for the amino acids; one to a line starting from the N terminus of the loop. For the 20 common amino acids the standard 3-letter codes are used. However, the code for any amino acid present in Maestro’s modifiable fragment tables may be given provided that it is an alpha amino acid.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">The number of loop conformations to generate</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 100</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">The maximum number of loop conformations to keep</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 10000</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Allowable interaction approach distance</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">If a given pair of atoms is closer than arg6 times the sum of their van der Waals radii after loop generation just prior to minimization then the structure is rejected and another loop structure is generated.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 0.25 </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg7</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Automatic generation of COMP and CHIG atom lists for the loop</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">−1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Don’t generate <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_comp">COMP</a></span> or <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span> commands for the atoms in the loop.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Automatically generate <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_comp">COMP</a></span> commands for the heavy atoms in the loop and <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span> commands for atoms in the loop.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Automatically generate <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_comp">COMP</a></span> commands and <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span> commands for all the atoms in the loop.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Automatically generate <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_comp">COMP</a></span> commands for the heavy atoms in the loop and do not generate any <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span> commands.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Automatically generate <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_comp">COMP</a></span> commands for all the atoms and do not generate any <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span> commands.</p>
                </td>
            </tr>
        </table>
        <p>Related <span class="OpcodeLink"><a href="opcodes_misc_debug.htm#opcodes_509685220_debg">DEBG</a></span> flags: 555</p>
        <h2 class="Opcode"><a name="opcodes_509685220_740138"></a><code class="Courier"><a name="opcodes_509685220_lpop">LPOP</a></code> — LooP OPtions</h2>
        <p><code>LPOP</code> provides a means to modify some of the parameters related to protein loop generation with the <span class="OpcodeLink"><a href="#opcodes_509685220_loop">LOOP</a></span> opcode. If no <code>LPOP</code> commands are provided the default values for the parameters are used. Multiple <code>LPOP</code> commands (e.g., with different arg1 values) may be used. It is not expected that these parameters would need to be modified from their default values under normal conditions so typically one would not need to include a <code>LPOP</code> command in the <code>.com</code> file.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Parameter set modified by LPOP</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">This argument controls which set of parameters is modified by the remaining arguments to <code>LPOP</code>.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Tweak parameters</p>
                    <table class="ArgTable">
                        <tr class="ArgTable">
                            <td class="ArgTable">
                                <p class="ArgDef">arg 2</p>
                            </td>
                            <td class="ArgTable">
                                <p class="ArgDef">Maximum number of Tweak iterations that can be used while trying to generate an acceptable loop conformation. </p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">Default: 600</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">
                                <p class="ArgDef">arg5</p>
                            </td>
                            <td class="ArgTable">
                                <p class="ArgDef">Early check iteration fraction.</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">Loop generation attempts will be abandoned at arg5 * arg2 tweak iterations if the distance criteria specified by arg6 is not met.</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">Default: 0.167</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">
                                <p class="ArgDef">arg6</p>
                            </td>
                            <td class="ArgTable">
                                <p class="ArgDef">Early check maximum deviation.</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">If the maximum deviation of the distances from their desired values is greater than the value of arg6 the current tweak process is abandoned and the process for generating a new loop is restarted.</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">Default: 7.0&#160;Å</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">
                                <p class="ArgDef">arg7</p>
                            </td>
                            <td class="ArgTable">
                                <p class="ArgDef">Maximum ratio of constraints to rotatable dihedral angles.</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">If the ratio of the number of target distances to the number of rotatable bonds exceeds this value then the current loop generation attempt is abandoned and the process for generating a new loop is restarted.</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">Default: 1.1</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">
                                <p class="ArgDef">arg8</p>
                            </td>
                            <td class="ArgTable">
                                <p class="ArgDef">Maximum dihedral angle change.</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">Tweaking a conformation involves iteratively adjusting the dihedral angles. The size of the changes in the dihedral angles for any one adjustment are scaled so that the largest change is less than arg7. </p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">Default: 5 degrees</p>
                            </td>
                        </tr>
                    </table>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Initial construction parameters</p>
                    <table class="ArgTable">
                        <tr class="ArgTable">
                            <td class="ArgTable">
                                <p class="ArgDef">arg2</p>
                            </td>
                            <td class="ArgTable">
                                <p class="ArgDef">The number of attempts at generating a given dihedral angle before retreating to the previous rotatable bond.</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">Default: 10</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">
                                <p class="ArgDef">arg3</p>
                            </td>
                            <td class="ArgTable">
                                <p class="ArgDef">The maximum total number of failed attempts at generating a new loop conformation.</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">If this number is exceeded then the loop construction process fails and MacroModel terminates with an error message.</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">Default: 100,000</p>
                            </td>
                        </tr>
                    </table>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Geometric parameters</p>
                    <table class="ArgTable">
                        <tr class="ArgTable">
                            <td class="ArgTable">
                                <p class="ArgDef">arg5</p>
                            </td>
                            <td class="ArgTable">
                                <p class="ArgDef">The closest permissible approach of a loop heavy atom to a heavy atom within the rest of the protein.</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">Default: 2&#160;Å</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">
                                <p class="ArgDef">arg6</p>
                            </td>
                            <td class="ArgTable">
                                <p class="ArgDef">The closest permissible approach of a loop heavy atom with another loop heavy atom.</p>
                            </td>
                        </tr>
                        <tr class="ArgTable">
                            <td class="ArgTable">&#160;</td>
                            <td class="ArgTable">
                                <p class="ArgParaIndent">Default: 2&#160;Å</p>
                            </td>
                        </tr>
                    </table>
                </td>
            </tr>/n</table>
        <p>Related <span class="OpcodeLink"><a href="opcodes_misc_debug.htm#opcodes_509685220_debg">DEBG</a></span> flags: 555</p>
        <h2 class="Opcode"><a name="opcodes_509685220_740171"></a><code class="Courier"><a name="opcodes_509685220_arpk">ARPK</a></code> — ARPacK parameters </h2>
        <p>This opcode applies to <span class="OpcodeLink"><a href="#opcodes_509685220_lmc2">LMC2</a></span>, <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_vbr2">VBR2</a></span>, and LTNCG (<span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> arg1=11)</p>
        <p>The ARPACK package was developed by Danny Sorensen, Richard Lehoucq, Chao Yang, and Kristi Maschhoff at Rice University [<a class="Ref_Number_Only MCXref xref xrefRef_Number_Only" href="mmod_references.htm#mmod_references_3670527510_741500" xrefformat="{paranumonly}">36</a>, <a class="Ref_Number_Only MCXref xref xrefRef_Number_Only" href="mmod_references.htm#mmod_references_3670527510_741506" xrefformat="{paranumonly}">37</a>]. Instead of attempting to solve a huge eigenproblem directly, ARPACK solves it indirectly by solving a series of small problems of much lower dimensionality and references the huge matrix only implicitly through its product with some vectors. ARPACK is ideally suited for LLMOD calculations, because direct calculation of the low-mode eigenvectors of a huge Hessian matrix of a protein molecule is prohibitive. Instead, LLMOD utilizes ARPACK to compute the low-mode eigenvectors indirectly. The <code>ARPK</code> command allows a user to fine tune ARPACK parameters. Note that the <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_vbr2">VBR2</a></span> command also uses ARPACK to calculate vibrational modes.</p>
        <p><span class="OpcodeLink"><a href="opcodes_misc_debug.htm#opcodes_509685220_debg">DEBG</a></span><code> 999</code> can be used to provide greater verbosity for <code>ARPK</code>.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Select a method to calculate Hessian <span class="Symbol">´</span> vector</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: The Hessian is calculated analytically, but it is stored in a sparse vector representation, not in matrix form, and allows for fast matrix-vector multiplication, which scales only linearly with the number of dimensions. <i>This is the recommended method.</i></p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara"><i>H</i><span class="Symbol">´</span><i>v</i> is calculated by a finite difference formula based on gradients (see Kolossváry and Keseru <a class="Reference_Number MCXref xref xrefReference_Number" href="mmod_references.htm#mmod_references_3670527510_741444" xrefformat="[{color #0E6895}{paranumonly}{/color}]">[<span style="color: #0e6895;" class="mcFormatColor">32</span>]</a>). This method does not need second derivatives. It is fast, but is often unstable and converges only very slowly.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara"><i>H</i><span class="Symbol">´</span><i>v</i> is calculated using a LBFGS (see <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> arg1=10) representation of an approximation to the inverse Hessian matrix. This method is the fastest, but it can only be used to calculate the three lowest modes. Higher modes are usually only computational artifacts: they are totally useless for LLMOD. <i>Check arg4 if you choose this method</i>.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">This method is a variant of arg1=1 using spectral transformation to boost convergence. Extremely slow, <i>not</i> recommended. If you choose to try it, check arg6 and arg7.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Same as arg1=0, but sparse Hessian storage is built on top of an explicit Hessian matrix. It is faster than arg1=0, but explicit Hessian storage makes it prohibitive for more than approx. 1000 atoms. (The Hessian storage scheme is the same used for TNCG minimization (<span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> arg1=9).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara"><i>H</i><span class="Symbol">´</span><i>v</i> is calculated with <i>exact</i> arithmetic. The Hessian is not stored in any form, its elements are always recalculated as they are needed for computing v<sub>k</sub>(i)=H(i,j)*v<sub>l</sub>(j) and v<sub>k</sub>(j)=H(i,j)*v<sub>l</sub>(i). This is the most accurate method, but it is prohibitively slow on large molecules because of the burden of recalculating the Hessian numerous times.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Dimension of “small” problems to solve (length of Arnoldi factorization)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 300</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other value to be used. arg2 &gt;= requested number of modes + 1 (<span class="OpcodeLink"><a href="#opcodes_509685220_lmc2">LMC2</a></span> arg3, <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_vbr2">VBR2</a></span> arg2), recommended value: arg2 = 10 * requested number of modes</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Number of “small” problems to solve (number of Arnoldi iterations)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 450</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other value to be used (maximum 25,000).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">This argument is multipurpose.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">If arg1=2, then arg4 specifies the number of <i>extra</i> LBFGS steps applied after minimization has reached convergence, to build a more accurate LBFGS representation of the Hessian matrix.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 500</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other value to be used.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">If arg1=0, then arg4 is ignored. The amount of memory used for sparse Hessian is increased automatically as needed.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">If arg1 is not 0 or 2, then arg4 is used to specify the amount of memory used for sparse Hessian storage.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 20 (MB)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other value to be used.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Note that the user-defined memory is evenly distributed between storage of Hessian values and storage of corresponding (<i>i</i>, <i>j</i>) indices.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Accuracy of ARPACK calculation in terms of ABS(<span class="Symbol">l</span><sub>c</sub> − <span class="Symbol">l</span><sub>t</sub>) &lt; arg5* ABS(<span class="Symbol">l</span><sub>c</sub>) where <span class="Symbol">l</span><sub>c</sub> is a calculated eigenvalue and <span class="Symbol">l</span><sub>t</sub> is the true eigenvalue.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 0.0001</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&lt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Machine precision (double precision).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other value to be used.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Eigenvalue shift of spectral transformation. Used with arg1=3.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 5.0</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other value to be used.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Spectral transformation transforms the eigenvalue spectrum such that the closest eigenvalue to arg5 becomes the largest eigenvalue and therefore, most likely the first eigenvalue to be found by ARPACK.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg7</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Conjugate gradient tolerance criterion for spectral transformation. Used with arg1=3.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 0.01</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other value to be used.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Spectral transformation involves minimum residual conjugate gradient iteration for which arg7 sets the convergence limit.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg8</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Sparse Hessian cutoff value. Used with arg1=0 and arg1=4. (Similar to <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span>/arg6).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 0.001</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Other value to be used.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Used with arg1=0, any <i>H</i><sub class="SubItal">ij</sub> term that is less than arg8 in magnitude is excluded from the sparse Hessian representation. A single <i>H</i><sub class="SubItal">ij</sub> element is the sum of a number of <i>H</i><sub class="SubItal">ij</sub> terms coming from different interactions that map to the same coordinate indices (<i>i</i>, <i>j</i>). On the other hand, used with arg1=4, arg8 is interpreted the same way as arg6 in the <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> command, i.e., only those elements of the Hessian matrix that are greater than arg8 are used in the sparse Hessian representation.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740226"></a><code class="Courier"><a name="opcodes_509685220_mcmm">MCMM</a></code> — Monte Carlo Multiple Minimum</h2>
        <p>MCMM [<a class="Ref_Number_Only MCXref xref xrefRef_Number_Only" href="mmod_references.htm#mmod_references_3670527510_741510" xrefformat="{paranumonly}">38</a>, <a class="Ref_Number_Only MCXref xref xrefRef_Number_Only" href="mmod_references.htm#mmod_references_3670527510_741515" xrefformat="{paranumonly}">39</a>] is our recommended conformational search method. The input structure will be modified by random changes in torsion angles and/or molecular position as specified by the <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> or <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> commands. Ordinarily, whether a single structure or multiple structures appear in the input file, they will first all be read in, minimized and treated as if already found by the <code>MCMM</code> procedure. This allows a new search to be initialized from the output of a previous search, by using the output file of the old search as input for the new one. However, if the necessary <span class="OpcodeLink"><a href="opcodes_file.htm#opcodes_509685220_read">READ</a></span> and <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> commands are placed within a <span class="OpcodeLink"><a href="opcodes_flowcontrol.htm#opcodes_509685220_bgin">BGIN</a></span>/<span class="OpcodeLink"><a href="opcodes_flowcontrol.htm#opcodes_509685220_end">END</a></span> loop, then a separate search is carried out for each input structure. This is called a <i>serial search</i>. For such serial calculations, the <span class="OpcodeLink"><a href="#opcodes_509685220_auto">AUTO</a></span> opcode is also needed.</p>
        <p>The <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> command is used to specify dihedral angles to be varied; the <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> command specifies relative positions of multiple molecules, as in an enzyme-substrate docking procedure. In addition, <span class="OpcodeLink"><a href="#opcodes_509685220_rca4">RCA4</a></span> commands and <span class="OpcodeLink"><a href="#opcodes_509685220_ligb">LIGB</a></span> commands can be used to open rings and break ligand bonds, respectively, before performing torsional or relative molecular motion. <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span> commands should be specified for ring-closure atoms as well as to retain chirality about other centers. <span class="OpcodeLink"><a href="#opcodes_509685220_torc">TORC</a></span> commands may be used to hold double-bond configurations constant. </p>
        <p>The <span class="OpcodeLink"><a href="#opcodes_509685220_auto">AUTO</a></span> opcode can be used for automatic setup of any kind of <code>MCMM</code> calculation.</p>
        <p>Only unique structures will be retained, as in the <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_mult">MULT</a></span> conformational searches. Use <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_demx">DEMX</a></span> to set an energetic window to select low energy conformations. It is usually found that not all structures converge in minimization during a conformational search. A <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_mult">MULT</a></span> minimization of the output file is recommended to achieve convergence for the final result.</p>
        <p>While the default search method is random walk, we find that the usage-directed search (<span class="OpcodeLink"><a href="#opcodes_509685220_mcss">MCSS</a></span> arg1=2) gives improved search performance.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Number of Monte Carlo steps to be carried out before stopping</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Carry out the search until arg2 structures have been found. We do not recommend the use of this default.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg2 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum number of structures to retain while running </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">The maximum number of conformations to retain while running is the value specified in arg1 or 10,000, whichever is greater.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">This is the maximum number of conformations to retain while running.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Number of variables altered in each step</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">The number of torsional angles varied and/or molecules moved in each step (default: 3). The value specified is randomly varied by +/−1 to prevent concentration of the search in local areas of conformational space. The default (3) thus alters 2–4 variables at each step.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Greater control over this parameter may be obtained using the <span class="OpcodeLink"><a href="#opcodes_509685220_mcnv">MCNV</a></span> command; this is the most common procedure. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Search mode</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Global search (each Monte Carlo step begins with the preceding Monte Carlo structure, providing the structure is within 100 kJ of the global minimum). This is equivalent to <span class="OpcodeLink"><a href="#opcodes_509685220_mcss">MCSS</a></span> (arg1=0, arg2=0, arg4=100.0).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Local search (each Monte Carlo step begins with the original structure). Generally used with small coordinate variations in <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> or <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> to find other minima which are closely related to the starting structure.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Allowable interatomic approach distance</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Fraction of sum of van der Waals radii which is used as a closest atomic approach limit (default: 0.25).</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740274"></a><code class="Courier"><a name="opcodes_509685220_spmc">SPMC</a></code> — Systematic Pseudo-Monte Carlo search</h2>
        <p>Similar to <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>, but invokes systematic search in place of random search. The search begins at low torsional resolution (120°), searches all angles without duplicating coverage, then doubles the resolution, etc. This method has the advantage of not retracing its path and consequently converges the final stages of the conformational search more efficiently than <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>. Like <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>, the method is effectively open-ended: it will search conformational space until stopped by the user or with arg1.</p>
        <p><code>SPMC</code> searches are conducted starting from a single input structure. However, if the necessary <span class="OpcodeLink"><a href="opcodes_file.htm#opcodes_509685220_read">READ</a></span> and <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> opcodes are placed within a <span class="OpcodeLink"><a href="opcodes_flowcontrol.htm#opcodes_509685220_bgin">BGIN</a></span>/<span class="OpcodeLink"><a href="opcodes_flowcontrol.htm#opcodes_509685220_end">END</a></span> loop, then a separate search is carried out ion each input structure. This is called a <i>serial search</i>. <span class="OpcodeLink"><a href="#opcodes_509685220_auto">AUTO</a></span> may be used to set up single <code>SPMC</code> searches and is required for serial <code>SPMC</code> searches.</p>
        <p>It is suggested that torsional memory (<span class="OpcodeLink"><a href="#opcodes_509685220_mcss">MCSS</a></span> arg3) be activated when using <code>SPMC</code> to prevent retracing of points in conformational space when starting from different starting geometries. If rings are being varied (i.e., <span class="OpcodeLink"><a href="#opcodes_509685220_rca4">RCA4</a></span> commands are being used), geometrical preoptimization (<span class="OpcodeLink"><a href="#opcodes_509685220_mcop">MCOP</a></span> arg2) should also be activated.</p>
        <p>Use with <span class="OpcodeLink"><a href="#opcodes_509685220_mcnv">MCNV</a></span> arg1=1 and arg2=<i>N</i>−1, where <i>N</i> is the number of variable torsions plus the number of molecules being independently translated/rotated with <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> commands.</p>
        <p>Details of the method are provided by Goodman and Still <a class="Reference_Number MCXref xref xrefReference_Number" href="mmod_references.htm#mmod_references_3670527510_741528" xrefformat="[{color #0E6895}{paranumonly}{/color}]">[<span style="color: #0e6895;" class="mcFormatColor">40</span>]</a><i>.</i></p>
        <p>The arguments are the same as for <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>, except arg3:</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum resolution for torsional alterations</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 24, implying angular resolution of 360° / 24 = 15°.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740292"></a><code class="Courier"><a name="opcodes_509685220_mcop">MCOP</a></code> — Monte Carlo OPtions</h2>
        <p>This command alters the data written to the <code>.log</code> file, and also the geometrical optimization routine. If <code>MCOP</code> is omitted, this is equivalent to setting arg1=250 and arg2=0.</p>
        <p class="Note" data-mc-autonum="&lt;b&gt;Note:&lt;/b&gt; "><span class="autonumber"><span><b>Note:</b> <![CDATA[ ]]></span></span>Despite its name, <code>MCOP</code> specifies parameters for use in low-mode (<span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> and <span class="OpcodeLink"><a href="#opcodes_509685220_lmc2">LMC2</a></span>) and <span class="OpcodeLink"><a href="#opcodes_509685220_loop">LOOP</a></span> calculations, as well as Monte Carlo searches. Arguments 4 and 5 only apply to <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> and <span class="OpcodeLink"><a href="#opcodes_509685220_lmc2">LMC2</a></span> jobs. Starting in MacroModel 6.5, these arguments have new meanings and arguments 6 and 7, which pertained to the earlier LMCS methodology, have been eliminated.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Number of steps between printout to log-file</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Print to log file every 250 Monte Carlo steps.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Print to log file every step.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue"><i>n</i>
                    </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Print to log file every <i>n</i> steps. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Geometrical preoptimization</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Off.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">On. Preoptimizes variable internal coordinates to improve ring closure distances. Recommended for <span class="OpcodeLink"><a href="#opcodes_509685220_spmc">SPMC</a></span> of ring systems.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Frequency of updating a conformational search</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">When an update is performed, the following actions take place:</p>
                    <ul class="ArgList">
                        <li class="ArgList" value="1">
                            <p class="ArgList">The current <code>.tmp</code> file is removed, and structures that are to be saved (based on the value of the current global energetic minimum) are written to the output file.</p>
                        </li>
                        <li class="ArgList" value="2">
                            <p class="ArgList">A summary of the progress of the search so far is written to the log file.</p>
                        </li>
                        <li class="ArgList" value="3">
                            <p class="ArgList">If a <i>jobname</i><code>.upt</code> file is created in the directory from which the job was initiated, an update is performed immediately, in addition to those specified by arg3. </p>
                        </li>
                    </ul>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: Perform an update every tenth of a run, but not more often than every ten steps nor less often than every 500 steps. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue"><i>n</i>
                    </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Perform an update every <i>n</i> steps. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef"> arg4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">LMCS serial job</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Not an <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> serial job. A single low-mode search is performed, with all input structures treated as seed structures, as if found in previous iterations. </p>
                </td>
            </tr>
        </table>
        <p class="Note" data-mc-autonum="&lt;b&gt;Note:&lt;/b&gt; "><span class="autonumber"><span><b>Note:</b> <![CDATA[ ]]></span></span>arg4=0 is unsuitable for non-conformers.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue"><span class="Symbol">¹</span>0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara"><span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> serial job; this implies that a separate conformational search will be performed for each structure in the input file. This takes advantage of the ability of <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> to define fruitful search directions without specification of variable torsions. An <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> serial job can be run only when there are no commands specifying atom numbers—such as <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> or <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span>—which might translate to incorrect specifications in the different input structures. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef"> arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Probability of taking a TORS/MOLS step</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">If this is an <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> job, all steps will be LMCS steps.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue"><i>n</i>
                    </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">If this is an <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> job and there are <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> or <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> commands present, this fraction of moves will be <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> or <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> (i.e., not LMCS) moves.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6 </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum number of conformers to write to the output structure file for each search performed </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">For example, in a serial search, the search for each input structure would produce at most the number of structures specified here. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Using a value other than 0 or 1 for this argument is not permitted in a <code>ligprep</code> licensed calculation where only one output structure is permitted per structure processed.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use setting from the previous <code>MCOP</code> command.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">If no previous <code>MCOP</code> command has been issued, use the number of structures to save internally, which is controlled by arg2 of the search command (<span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_lmc2">LMC2</a></span>).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Save up to this many structures per search.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg7</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">The number of conformers read in from the input structure file to seed each search in a serial search.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">This argument is primarily intended for use in <code>MBAE</code> conformational searches when <span class="OpcodeLink"><a href="opcodes_coordinate.htm#opcodes_509685220_algn">ALGN</a></span> has been used to produce multiple orientations for ligand structures.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Do not use this argument in <span class="OpcodeLink"><a href="#opcodes_509685220_loop">LOOP</a></span> calculations.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Do not use this argument in <code>ligprep</code> licensed calculations.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0, 1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Seed each search with one structure.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Seed each search with this many input structures.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740343"></a><code class="Courier"><a name="opcodes_509685220_mcss">MCSS</a></code> — Monte Carlo Structure Selection</h2>
        <p>This command allows the program to select starting geometries for Monte Carlo search steps in several different ways. Arg1 selects between random walk and two usage-related criteria. Arg2 affects weighting among selected structures for low energy geometries. Arg5 gives an optional energy window which prevents structures which are high in energy from being chosen.</p>
        <p>This command is active only when doing global searching (i.e., when <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> or <span class="OpcodeLink"><a href="#opcodes_509685220_spmc">SPMC</a></span> arg4 = 0). In any case, structures must be within 100 kJ of the current global minimum to be candidates for starting geometry selection.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Starting structure selection criterion</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Random walk. Most recent structure will be chosen whose energies are allowed by args2 and 5.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use-directed. The least used structures will be used as starting geometries if their energies are allowed by args2 and 5. “Use” is defined as (times used as starting structure) − (times resulting structure is kept).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use-directed. The least used structures will be used as starting geometries if their energies are allowed by args2 and 5. Use defined simply as (times used as starting structure).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Energetic window modifier</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Arg5 will be used directly; recommended.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Arg5 will be multiplied by a random number between 0 and 1. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Torsional memory selection</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Torsional memory is not used. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Torsional memory is used. Structures are considered identical if all torsions match within RES/arg3 where RES is the operative search resolution (smallest value = 2).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Energetic window, kJ/mol</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Current structure will be used as a starting geometry for a subsequent step only if its energy is within arg5 kJ/mol of the lowest energy structure yet found. A good choice for arg5 is simply the value of the overall energetic window being used in <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_demx">DEMX</a></span>. Default: 100 kJ/mol.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_777761"></a><code class="Courier"><a name="opcodes_509685220_flap">FLAP</a></code> — use FLAP algorithm for ring conformations</h2>
        <p>Use the FLAP ring vertex reflection algorithm <a class="Reference_Number MCXref xref xrefReference_Number" href="mmod_references.htm#mmod_references_3670527510_743523" xrefformat="[{color #0E6895}{paranumonly}{/color}]">[<span style="color: #0e6895;" class="mcFormatColor">54</span>]</a> to generate ring conformations in MCMM, low-mode, or mixed MCMM/low-mode conformational searches. This opcode must be added before the relevant <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> or <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> opcode.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Probability of using a FLAP step</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Specify the fraction of conformational search steps that will use FLAP. If used in conjunction with <span class="OpcodeLink"><a href="#opcodes_509685220_mcrc">MCRC</a></span>, the sum of the FLAP and template probabilities must not be greater than 1. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use the default value of 0.5</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use this value. Should be between 0 and 1.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_777713"></a><code class="Courier"><a name="opcodes_509685220_mcrc">MCRC</a></code> — Monte Carlo Ring Conformations</h2>
        <p>Use templates of ring conformations in MCMM, low-mode, or mixed MCMM/low-mode conformational searches. This opcode must be added before the relevant <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> or <span class="OpcodeLink"><a href="#opcodes_509685220_lmcs">LMCS</a></span> opcode.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Use of templates</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use templates in conformational search (default)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">−1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Do not use templates in conformational search</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Probability of using templates</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Specify the fraction of conformational search steps that will use templates. If used in conjunction with <span class="OpcodeLink"><a href="#opcodes_509685220_flap">FLAP</a></span>, the sum of the FLAP and template probabilities must not be greater than 1. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use the default value of 0.5</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use this value. Should be between 0 and 1.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740362"></a><code class="Courier"><a name="opcodes_509685220_mcts">MCTS</a></code> — Monte Carlo Torsion Selection</h2>
        <p>Experience has shown that this command is not terribly useful.</p>
        <p>Use in conjunction with <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> to favor torsion angle selection near local torsional minima for the angles being varied. One <code>MCTS</code> command is required for each torsion which is to be effected. Arg1-4 are the atoms defining a particular torsion angle (i.e., arg2-3 should appear as a rotatable bond in a <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> command). Arg5-7 are 1-fold, 2-fold, and 3-fold torsional barriers which are used to compute local torsional energies as part of the test for an allowable value of the randomly selected Monte Carlo angular change. After a Monte Carlo variation of a torsion angle described by arg1-4 of this command is performed, a local torsional energy (E<sub>T</sub>) is computed based on the value of the dihedral angle, <span class="Symbol">t</span>, using the usual MM2 formula:</p>
        <p class="EquationNoNum">E<sub>T</sub> = (V<sub>1</sub>/2)(1+cos<span class="Symbol"> t</span>) + (V<sub>2</sub>/2)(1−cos 2<span class="Symbol">t</span>) + (V<sub>3</sub>/2)(1+cos 3<span class="Symbol">t</span>) − E<sub>T,min</sub></p>
        <p>where E<sub>T,min</sub> is the minimum possible value of E<sub>T</sub>. If E<sub>T</sub> is greater than 1, the torsion is rejected and a new random torsion angle is chosen. Otherwise, E<sub>T</sub> is compared with a random number between 0 and 1 and, if E<sub>T</sub> is larger than that number, a new random torsion angle is chosen. This scheme selects for local torsions which are low in energy. For sp3-sp3 linkages, one can favor the gauche and anti conformers (the minima) by using a V<sub>1</sub> (arg5) and V<sub>3</sub> (arg7)of 0.25. This will cause totally eclipsed (0-degree torsion) torsions to be strongly disfavored, 120-degree torsions to be moderately disfavored and the gauche and anti conformations to be favored. By choosing V<sub>1</sub>-V<sub>3</sub> with care and using a random number for comparison, even high energy geometries are occasionally explored.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1-4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Atoms defining the torsion</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">V<sub>1</sub> (positive number gives anti minima)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">V<sub>2 </sub>(positive number gives eclipsed minima)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg7</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">V<sub>3 </sub>(positive number gives staggered minima)</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_757771"></a><code class="Courier"><a name="opcodes_509685220_mcsm">MCSM</a></code> — Monte Carlo Single Minimum</h2>
        <p>This procedure is similar to that described by Li and Scheraga <a class="Reference_Number MCXref xref xrefReference_Number" href="mmod_references.htm#mmod_references_3670527510_741534" xrefformat="[{color #0E6895}{paranumonly}{/color}]">[<span style="color: #0e6895;" class="mcFormatColor">41</span>]</a> and is a global search for the single lowest energy structure. The search can be conducted with or without minimization (depending on arg3). While this method is good at finding a single low energy conformer, there is no guarantee that it will locate the true global minimum energy conformer.</p>
        <p>Monte Carlo internal coordinate conformational search is performed with minimizations on a single structure. Variable internal coordinates are specified by <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> and/or <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span>. </p>
        <p><code>MCSM</code> can be used with cyclic structures providing that either ring bonds are not varied or that ring closure commands (<span class="OpcodeLink"><a href="#opcodes_509685220_rca4">RCA4</a></span>) are used for each ring in which variable torsions are used. During the run, random variations will be applied to 2-4 randomly selected dihedral angles from the <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> lists or molecular translations/rotations from the <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> lists for each Monte Carlo step. If some other range of variable coordinates is desired, the range is set with the <span class="OpcodeLink"><a href="#opcodes_509685220_mcnv">MCNV</a></span> command.</p>
        <p>If used within a <span class="OpcodeLink"><a href="opcodes_flowcontrol.htm#opcodes_509685220_bgin">BGIN</a></span>/<span class="OpcodeLink"><a href="opcodes_flowcontrol.htm#opcodes_509685220_end">END</a></span> loop, all structures in the input file will be read and the global minimum found will be listed to the output file at the end of every MC iteration set for each input structure. The ultimate global minimum would be found by examining the energies of each of the output structures.</p>
        <p>Unlike <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>, <code>MCSM</code> requires no <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> command. It must appear after a <span class="OpcodeLink"><a href="opcodes_file.htm#opcodes_509685220_read">READ</a></span> command.</p>
        <p>In general, Monte Carlo searches should use <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span> commands to maintain all chiral centers and <span class="OpcodeLink"><a href="#opcodes_509685220_torc">TORC</a></span> commands to hold double bond geometries constant. If chirality or double-bond geometry is lost in any step, then following steps will be wrong if the resulting geometry is used for subsequent steps.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Minimization mode</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Steepest descent.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">PR conjugate gradient (best general method).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Variable Metric (not recommended with <code>MCSM</code>).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Full matrix NR (not recommended with <code>MCSM</code>).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">9</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">TNCG (good for flexible structures). </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Line-search control for second-derivative methods (arg1=4 or 9)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">No line searching (best choice).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Line searching on.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum number of minimization iterations</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Number of Monte Carlo cycles (default: 100)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Initial temperature (K)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0.0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Metropolis sampling will not be done; every structure will be used as the next starting point </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Final Temperature (K)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0.0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Continuous sampling at the arg5 temperature will be done throughout the run. If a nonzero temperature is supplied, cooling from the arg5 to the arg6 temperature will be carried out during the run. This slow cooling is equivalent to simulated annealing.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg7</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Step size buffer</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">
                    </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">As in <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> arg5.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg8</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">TNCG Hessian cutoff</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">
                    </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">As in <span class="OpcodeLink"><a href="opcodes_minimization.htm#opcodes_509685220_mini">MINI</a></span> arg6.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740420"></a><code class="Courier"><a name="opcodes_509685220_mcnv">MCNV</a></code> — Monte Carlo Number of Variables</h2>
        <p>This command resets the degrees of freedom (number of torsion angles varied plus number of molecules moved in space) altered in a single search step. If this command is not used, the value is taken from arg3 of the <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> command. This command overrides the <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> arg3. </p>
        <p>For <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> on single unsymmetrical molecules, we find it best to specify the range as 1 to <i>N</i>, where <i>N</i> is the number of variable dihedral angles, when <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> commands are being used. It is best to use a range of values, rather than a single value. When <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> is being used, <i>N</i> should be incremented by one. When <span class="OpcodeLink"><a href="opcodes_mdmc.htm#opcodes_509685220_zmat">ZMAT</a></span> is being used, <i>N</i> should take account of all degrees of freedom.</p>
        <p>Used with <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>, <span class="OpcodeLink"><a href="opcodes_mdmc.htm#opcodes_509685220_mcsd">MCSD</a></span>, <span class="OpcodeLink"><a href="opcodes_mc.htm#opcodes_509685220_mclo">MCLO</a></span>, <span class="OpcodeLink"><a href="opcodes_mdmc.htm#opcodes_509685220_imps">IMPS</a></span>, and <span class="OpcodeLink"><a href="#opcodes_509685220_spmc">SPMC</a></span> commands.</p>
        <p>In the context of MC or MC(JBW) simulations this command sets the number of degrees of freedom to be changed at each MC step or randomized at each MC(JBW) step. A degree of freedom is either a bond length, bond angle or torsion or a molecular translation or rotation along or about a single axis. This number should be set in such a way to provide a compromise between acceptance rate and conformational interconversions. When args 1 and 2 of this command differ, <code>MCNV</code> defines a range for the number of degrees of freedom to be changed at each step. When the two arguments are identical, <code>MCNV</code> defines an exact number for the degrees of freedom to be changed at each step. In both cases, the initial values will be modified during the run by the adaptive mechanism unless debug 103 is defined. </p>
        <p>For MC, MC(JBW) or <span class="OpcodeLink"><a href="opcodes_mdmc.htm#opcodes_509685220_mcsd">MCSD</a></span> simulations, a number of degrees of freedom must be defined and consequently, the <code>MCNV</code> command should be present; however, its two arguments can be 0, in which case the program will provide a default range, from 1 to maximum number of the degrees of freedom of the molecules. Such a range is probably not efficient. For MC(JBW)/SD, no randomization in the JBW part is needed and all the randomization can effectively be done by the SD part of the simulation. Doing so will increase the number of conformational interconversions. In order to achieve that, the <code>MCNV</code> command should be omitted and arg 2 of the <span class="OpcodeLink"><a href="opcodes_mdmc.htm#opcodes_509685220_mcsd">MCSD</a></span> command should be set to a negative number.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Minimum number of degrees of freedom altered</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: 1</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum number of degrees of freedom changed in a MC step</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Default: number of variable degrees of freedom in the system.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Cluster torsions varied at one time</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">This is no longer seen as a useful option.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Do not cluster torsions; recommended.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">All torsions rotated will be in a contiguous group as defined by the ordering of torsions in the <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> commands.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Allow a single intervening unused torsion.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue"><i>n</i>
                    </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">If, at a given stage, <i>m</i> torsions are rotated, these are selected from a contiguous group of (<i>m</i>+<i>n</i>−1) torsions selected from the list taken from the <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> commands.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740451"></a><code class="Courier"><a name="opcodes_509685220_seed">SEED</a></code> — random number generator SEED</h2>
        <p>This command sets a starting value for the random number generator. Used to start the Monte Carlo random number generator at a different point so that repeated Monte Carlo or molecular dynamics runs give different results. The starting value should not exceed 700 000; if it does it will be divided by 2 until it is less than 700 000. </p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Seed value</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">−1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use a value generated from the time and date.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use the default value.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Use the value specified (must be less than 700 000), or use positive values less than 78593 if the MacroModel random-number generator is used (<span class="OpcodeLink"><a href="opcodes_misc_debug.htm#opcodes_509685220_debg">DEBG</a></span><code> 178</code>). </p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740457"></a><code class="Courier"><a name="opcodes_509685220_tors">TORS</a></code> — variable TORSion selection </h2>
        <p>Each <code>TORS</code> command specifies up to two torsions, using the numbers of the two central atoms. These will be used as variable dihedral angles by the <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_spmc">SPMC</a></span> or <span class="OpcodeLink"><a href="#opcodes_509685220_mcsm">MCSM</a></span> commands. The actual number of torsions which will be varied during a single Monte Carlo step depends on the search method, but the number varied will be taken from the list specified in <code>TORS</code> commands. A given random torsional variation will be plus or minus a random number selected from the range extending from the arg5 to the arg6 specification.</p>
        <p>Variable torsions within rings require the ring-closure commands <span class="OpcodeLink"><a href="#opcodes_509685220_rca4">RCA4</a></span> in addition to <code>TORS</code> commands. It is advisable to specify at least two variable torsions within each ring containing <span class="OpcodeLink"><a href="#opcodes_509685220_rca4">RCA4</a></span> ring closures. The atoms of a ring closures (args 2 and 3 in <span class="OpcodeLink"><a href="#opcodes_509685220_rca4">RCA4</a></span>) must not be listed as variable torsions.</p>
        <p>The minimum and maximum angular increment (arg5 and arg6) refer to the torsions given in arg1-4. It is possible to use a different angular increment for each torsion by using only arg1, arg2, arg5, and arg6 and a different <code>TORS</code> command for each torsion. For global searching, arg5 and arg6 of 0.0° and 180.0° are appropriate values. If you wish to focus the search on conformations having only small angular variations from the starting conformation, a value of 30.0° for arg6 could be used.</p>
        <p>If you are searching multicyclic ring systems, you should include <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_chig">CHIG</a></span> commands for any substituted atoms at the ring closure atoms (arg2 and arg3 in <span class="OpcodeLink"><a href="#opcodes_509685220_rca4">RCA4</a></span>) to assure maintenance of stereochemistry.</p>
        <p>When doing substructure MC searching, always order the pairs of atoms defining torsions such that the second atom of each pair is not connected to any fixed atoms (<span class="OpcodeLink"><a href="opcodes_constrained_mini.htm#opcodes_509685220_fxat">FXAT</a></span>) except via the first atom (in the torsional movements, the chain connected to the second atom is the one that will actually be moved). If both ends are anchored by <span class="OpcodeLink"><a href="opcodes_constrained_mini.htm#opcodes_509685220_fxat">FXAT</a></span> commands, then a ring closure (<span class="OpcodeLink"><a href="#opcodes_509685220_rca4">RCA4</a></span>) command will be necessary. <code>TORS</code> and <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> commands can be used together.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1-2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Atom numbers specifying first torsion</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg3-4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Atom numbers specifying second torsion</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Minimum dihedral angle variation</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">A positive number in degrees (default: 0.0°). </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum dihedral angle variation</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">A positive number in degrees (default: 180.0°). </p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740485"></a><code class="Courier"><a name="opcodes_509685220_tres">TRES</a></code> — Torsional RESolution</h2>
        <p>This command may be used in the <span class="OpcodeLink"><a href="#opcodes_509685220_spmc">SPMC</a></span> systematic pseudo Monte Carlo searches to alter the initial resolution of the search around a particular torsion angle. This command must come after the <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> command.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1-2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Atoms defining the torsional angle</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">These must have been listed already with a <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> command. </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg3</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Resolution for this angle</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">The value in degrees of the initial moves will be 360°/arg3; thus arg3=3 (the default) gives 120° resolution.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740495"></a><code class="Courier"><a name="opcodes_509685220_mols">MOLS</a></code> — variable MOLeculeS selection</h2>
        <p>This command selects molecules to be independently rotated and/or translated during a conformational search. It is used for configurational/conformational searches of complexes. In particular, given a docked bimolecular complex, <code>MOLS</code> can be used to translate and rotate the smaller molecule within the binding site of the larger one in order to explore possible binding geometries. <code>MOLS</code> and <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> commands can be used together; this allows the internal geometry of the separate molecules to be explored together with the relative orientation. If there are <i>N</i> molecules in the system, it suffices to specify <i>N</i>−1 of them in <code>MOLS</code> commands to ensure that all are adjusted.</p>
        <p><code>MOLS</code> commands must come after <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> commands.</p>
        <p>During a search, random molecules are selected for motion. For each molecule, rotation about all three axes and translation along all three axes are performed by amounts selected randomly from within the ranges specified in arg5-6 and arg7-8. <code>MOLS</code> has the same relationship to <span class="OpcodeLink"><a href="#opcodes_509685220_ligb">LIGB</a></span> as <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> has to <span class="OpcodeLink"><a href="#opcodes_509685220_rca4">RCA4</a></span>: the bonds specified in a <span class="OpcodeLink"><a href="#opcodes_509685220_ligb">LIGB</a></span> command are broken before the <code>MOLS</code>-specified molecular motion is carried out, then the <span class="OpcodeLink"><a href="#opcodes_509685220_ligb">LIGB</a></span> bonds are remade prior to minimization of the resulting structure.</p>
        <p class="Note" data-mc-autonum="&lt;b&gt;Note:&lt;/b&gt; "><span class="autonumber"><span><b>Note:</b> <![CDATA[ ]]></span></span>During a long enough search, if pair-list cutoffs are in effect, as is normal (see <span class="OpcodeLink"><a href="opcodes_ffld.htm#opcodes_509685220_exnb">EXNB</a></span>), one molecule may eventually wander far enough away from another that no nonbonded energies between them exist anymore. In this situation, further searching just explores random spatial dispositions of this pair with no energetic contribution from their mutual interaction. Unless already-found binding conformations are lower in energy by at least the <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_demx">DEMX</a></span>-specified energy, this will lead to essentially an infinite and fruitless search of conformational space. To avoid this, use <span class="OpcodeLink"><a href="opcodes_constrained_mini.htm#opcodes_509685220_fxdi">FXDI</a></span> to constrain the distance between atom pairs spanning pairs of molecules to some maximum distance. This maximum distance should be specified as the half-width of an <span class="OpcodeLink"><a href="opcodes_constrained_mini.htm#opcodes_509685220_fxdi">FXDI</a></span> potential. In practice, we do this only if we encounter difficulties without doing so.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1-4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Atom in a molecule to be moved</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Each nonzero atom given specifies independent motion of the entire molecule containing the atom. Thus, to specify independent motion of two molecules, put an atom number from the first in arg1 and an atom number from the second in arg2.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&lt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Perform rotations about the atom number given.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Perform rotations about the center of mass of the molecule containing the atom.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Minimum rotational variation</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Angle in degrees (default: 0.0).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum rotational variation</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Angle in degrees (default: 0.0, i.e., no rotation). 180° is a reasonable value.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg7</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Minimum translational variation</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Movement in Angstroms (default: 0.0).</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg8</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum translational variation</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Movement in Angstroms (default: 0.0). A value in the range 3-5 Å is reasonable value.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740539"></a><code class="Courier"><a name="opcodes_509685220_ligb">LIGB</a></code>  — LIGand Bonds</h2>
        <p>Used chiefly for configurational searches of inorganic complexes in conjunction with the <span class="OpcodeLink"><a href="opcodes_energy.htm#opcodes_509685220_vdwb">VDWB</a></span> command.</p>
        <p>This command defines bonds to be broken, creating molecular fragments which will be moved independently during the search. A <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> command must be present for each fragment to be moved. For a bidentate ligand, there will be two <code>LIGB</code> bonds specified for the single <span class="OpcodeLink"><a href="#opcodes_509685220_mols">MOLS</a></span> command that moves the ligand; for a tridentate ligand, there will be three, and so on.</p>
        <p>This procedure allows an <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> search to find, for example, both <i>mer</i> and <i>fac</i> isomers of an octahedral complex with stoichiometry MA<sub>3</sub>B<sub>3</sub>. <code>LIGB</code> can also be used to extend a conformational search to a configurational search; for example, by specifying bonds to chiral carbons as <code>LIGB</code> bonds, the R and S configurations about these carbons will be explored.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">First atom in bond to be broken (typically a metal atom)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Bond(s) to be broken</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Add all bonds to arg1 to the <code>LIGB</code> list.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&gt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Add the arg1-arg2 bond to the <code>LIGB</code> list for arg1.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">&lt; 0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Remove the bond between arg1 and |arg2| from the <code>LIGB</code> list for arg1, if such a list already exists, or, if no such lists exists, create one with all bonds to arg1 on it except this one.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740557"></a><code class="Courier"><a name="opcodes_509685220_rca4">RCA4</a></code>  — Ring Closure Atoms </h2>
        <p>This command directs the program to temporarily break a bond to sever a ring for the purpose of Monte Carlo torsion angle searching. The first 4 arguments are four atoms within the ring which comprise a ring-closure torsion angle. One <code>RCA4</code> command is necessary for each ring with dihedral angles specified in a <span class="OpcodeLink"><a href="#opcodes_509685220_tors">TORS</a></span> command. Used with <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_spmc">SPMC</a></span>, <span class="OpcodeLink"><a href="opcodes_mdmc.htm#opcodes_509685220_imps">IMPS</a></span>, and <span class="OpcodeLink"><a href="#opcodes_509685220_mcsm">MCSM</a></span>. </p>
        <p>With <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span> and <span class="OpcodeLink"><a href="#opcodes_509685220_mcsm">MCSM</a></span>, and for small-medium rings, arg5 and arg6 should be approximately 0.5 and 2.5. For large rings, arg5 and arg6 should be ca 0.1 and 5.0.</p>
        <p>With <span class="OpcodeLink"><a href="#opcodes_509685220_spmc">SPMC</a></span> and for small-medium rings, arg5 and arg6 should be approximately 1.0 and 2.0. For larger rings, arg5 and arg6 should be ca 0.5 and 3.0-4.0. The exact choice is not very important but has a minor effect on search efficiency.</p>
        <p>It is forbidden that an arg2 or arg3 atom be used in more than one <code>RCA4</code> command (common ring closure atoms are not allowed). The closure angle arguments (arg7 and arg8) are optional and refer to both closure angles 1-2-3 and 2-3-4. Our tests so far indicate that there is little reason to use arg7 and arg8 closure angle constraints.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1-4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Atom numbers within ring</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Four contiguous atoms within a ring. The closure bond is the one between arg2 and arg3.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Minimum allowable ring closure distance (Å) (default: 0)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum allowable ring closure distance (Å)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">The default is essentially infinite, so that ring-closure distance never precludes minimization on a structure produced by Monte Carlo variations.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg7</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Minimum allowable closure angle (°) </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">Default = 0.0</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg8</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum allowable closure angle (°) </p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">&#160;</td>
                <td class="ArgTable">
                    <p class="ArgPara">The default is 180°, so that ring-closure angle never precludes minimization on a structure produced by Monte Carlo variations.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740580"></a><code class="Courier"><a name="opcodes_509685220_disc">DISC</a></code> — DIStance Check</h2>
        <p>This command causes the distance between atoms given in arg1 and arg2 to be monitored and structures which have such distances outside the allowable range to be eliminated from the output file. Used only with <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_mult">MULT</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_mcsm">MCSM</a></span>, and <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1-2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Atom numbers defining distance</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Control</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Check distance before minimization.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Check distance after minimization.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Minimum allowable distance (Å)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum allowable distance (Å)</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740592"></a><code class="Courier"><a name="opcodes_509685220_torc">TORC</a></code> — TORsional Check</h2>
        <p>This command causes the absolute value of the torsion angle defined by arg1-4 to be monitored and structures which have such torsions outside the allowable range to be eliminated from the output file. Used only with <span class="OpcodeLink"><a href="opcodes_conf_comparison.htm#opcodes_509685220_mult">MULT</a></span>, <span class="OpcodeLink"><a href="#opcodes_509685220_mcsm">MCSM</a></span>, and <span class="OpcodeLink"><a href="#opcodes_509685220_mcmm">MCMM</a></span>. The test is applied after minimization in each case, and acts as a filter, particularly during conformational searches.</p>
        <p>This command is intended to reject unwanted cis or trans isomers created during a conformation search. Appropriate values for rejecting all trans isomers of a cis torsion (original torsion angle ~0) are arg5=0 and arg6=90. To reject all cis isomers of a trans torsion (original torsion angle ~180), use arg5=90, and arg6=180.</p>
        <p class="Note" data-mc-autonum="&lt;b&gt;Note:&lt;/b&gt; "><span class="autonumber"><span><b>Note:</b> <![CDATA[ ]]></span></span>Torsion checks do not limit the range of conformers searched or constrain the torsion during minimization. See <span class="OpcodeLink"><a href="opcodes_constrained_mini.htm#opcodes_509685220_fxta">FXTA</a></span> for details on applying a torsional constraint during minimization.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1-4</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Atom numbers specifying a dihedral angle</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg5</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Minimum allowable angle (|degrees|)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg6</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum allowable angle (|degrees|)</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740606"></a><code class="Courier"><a name="opcodes_509685220_mcmf">MCMF</a></code> — Monte Carlo Maximum constraint Failures</h2>
        <p>This command allows you to adjust the maximum number of constraint failures, e.g. ring closures (<span class="OpcodeLink"><a href="#opcodes_509685220_rca4">RCA4</a></span>) or constrained torsion (<span class="OpcodeLink"><a href="#opcodes_509685220_torc">TORC</a></span>), that are allowed before accepting a faulty structure. A limit of some kind is appropriate since the user could supply constraints that make valid structure generation impossible. Thus the program tries a given starting geometry and a given number of varying torsions (or other coordinates) repeatedly until arg1 failures (or the default of 10000) have occurred. Then, the program allows a different number of varying torsions and makes arg1 new attempts to create a valid structure. This process is repeated arg2 times before a faulty structure is accepted. Thus when constraint tests have failed arg1 times, the program allows arg2 failures (of arg1 tries each) before accepting the structure anyway. Such faulty structures often fail to minimize properly.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum number of constraint failures with constant torsion set (default: 10000)</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg2</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Maximum number of attempts with variable torsion sets (default: 10)</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_740613"></a><code class="Courier"><a name="opcodes_509685220_smpl">SMPL</a></code> — monte carlo SaMPLing</h2>
        <p>This command allows one to write sample structures to disk during a Monte Carlo Single Minimum (<span class="OpcodeLink"><a href="#opcodes_509685220_mcsm">MCSM</a></span>) run.</p>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Sampling interval</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue">0</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Write the last structure sampled and the global minimum; the latter appears last in the output file.</p>
                </td>
            </tr>
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgValue"><i>n</i>
                    </p>
                </td>
                <td class="ArgTable">
                    <p class="ArgPara">Write every <i>n</i>th structure sampled.</p>
                </td>
            </tr>
        </table>
        <h2 class="Opcode"><a name="opcodes_509685220_781054"></a><code class="Courier"><a name="opcodes_509685220_srno">SRNO</a></code>— SeRial Number Ordering</h2>
        <table class="ArgTable">
            <tr class="ArgTable">
                <td class="ArgTable">
                    <p class="ArgDef">arg1</p>
                </td>
                <td class="ArgTable">
                    <p class="ArgDef">Number to be added to serial number property in output structure file</p>
                </td>
            </tr>
        </table>
        <table class="NavLink" data-mc-conditions="Default.ScreenOnly">
            <tr class="NavLink">
                <td class="NavLeft"><span class="NavText">◀ <a class="NavLink MCXref xref xrefNavLink" href="opcodes_coordinate.htm" xrefformat="{paratext}">Coordinate Manipulation Opcodes</a></span>
                </td>
                <td class="NavCenter"><span class="NavText"><a class="NavLink MCXref xref xrefNavLink" href="macromodel_command_referenceTOC.htm" xrefformat="{paratext}">MacroModel Command Reference Manual</a></span>
                </td>
                <td class="NavRight"><span class="NavText"><a class="NavLink MCXref xref xrefNavLink" href="opcodes_mdmc.htm" xrefformat="{paratext}">Molecular Dynamics and Monte Carlo Simulation Opcodes</a> ▶</span>
                </td>
            </tr>
        </table>
    </body>
</html>