<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->


<!--
!!	se.html		Sequence Editor GUI chapter
!!
!!	08-oct-2010 (mk) fixes for moe 2010
!!	26-may-2010 (al) added chain tags, updated
!!	23-sep-2004 (jd) Removed Save As... link to new save panel
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../../manstyle.css" />


<meta panel Sequence Editor>
<meta panel Color Residues>

<title>Sequence Editor</title>
</head>

<body bgcolor="#ffffff">
<a href="../../index.htm"><img src="../../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>Sequence Editor</h2>
<hr noshade="noshade" />

<p>This document introduces the Sequence Editor and provides an
overview of its functionality.  If you are looking for information on a
specific menu or command, use the 
<a href="#MenuBar">Sequence Editor Menu Bar</a>.</p>

<ul>
<li><a href="#Introduction">Introduction</a></li>
<li><a href="#Components">Components of the Sequence Editor</a>
    <ul>
    <li><a href="#MenuBar">Sequence Editor Menu Bar</a>
    </li><li><a href="#CommandLine">Command Line</a>
    </li><li><a href="#ChainNumbers">Chain Numbers</a>
    </li><li><a href="#ResidueDisplayArea">Residue Display Area</a>
    </li><li><a href="#Footer">Footer</a>
    </li><li><a href="#PopupMenus">Popup Menus</a>
    </li>
    </ul>
</li><li><a href="#SequenceEditorPopupMenus">Sequence Editor Popup Menus</a>
    <ul>
    <li><a href="#Atom_Commands">Atom Commands</a>
    </li><li><a href="#Residue_Commands">Residue Commands</a>
    </li><li><a href="#Chain_Commands">Chain Commands</a>
    </li>
    </ul>
</li><li><a href="#SelectingMolecularObjects">Selecting Molecular Objects</a>
    <ul>
    <li><a href="#UsingTheLeftMouseButton">Using the Left Mouse Button</a></li>
    <li><a href="#UsingPopupMenus">Using Popup Menus</a></li>
    <li><a href="#UsingSelectionMenu">Using the Selection Menu</a>
	<ul>
	<li><a href="#SelectingAtoms">
	    Selecting and Isolating Atoms in the MOE Window</a></li>
	<li><a href="#SelectingResChains">Selecting Residues and Chains</a></li>
	<li><a href="#InvertingSelection">Inverting Chain or Residue
Selection</a></li>
	<li><a href="#ClearingSelection">Clearing Chain and Residue
Selection</a></li>
	</ul>
	</li>
    <li><a href="#ResidueSelectorPanel">Using the Residue Selector Panel</a></li>
    </ul>
</li><li><a href="#EditingMolecularObjects">Editing Molecular Objects</a>
    <ul>
    <li><a href="#MovingResidues">Moving Residues</a></li>
    <li><a href="#MovingChains">Moving Chains</a></li>
    <li><a href="#JoiningAndSplitting">Joining and Splitting Chains</a></li>
    <li><a href="#DeletingChainsAndResidues">Deleting Chains and Residues</a></li>
    </ul>
</li><li><a href="#ColoringMolecularObjects">Coloring Molecular Objects</a>
    <ul>
    <li><a href="#ColoringResidues">Coloring Residues</a></li>
    <li><a href="#ColorByChainTag">Coloring Chains by Chain Number
	and Tag</a></li>
    <li><a href="#ColoringSelectedChains">
		Coloring Atoms of Selected Chains in the MOE Window</a></li>
    <li><a href="#ColoringTheBackbone">Coloring the Backbone</a></li>
    </ul>
</li><li><a href="#Homology_Modeling">Homology Modeling</a></li>
<li><a href="#Measuring">Measuring Residue Properties
	and Stereochemistry</a></li>
<li><a href="#ModifyingTheDisplay">Modifying the Display (Compound Name,
Secondary Structure, Hydrogen Bonds, <br />
Residue Name, Residue Indices, Residue UID)</a></li>

<li><a href="#SVL">SVL Commands</a></li>
<li><a href="#References">References</a></li>
<li><a href="#See_Also">See Also</a></li>
</ul>

<a name="Introduction"></a><h2>Introduction</h2>

<p>In MOE, a molecular system consists of one or more chains, and each chain
consists of zero or more residues. A residue consists of zero or more
atoms.  The Sequence Editor allows you to organize and
modify molecules at the chain and residue levels. It also serves
as the primary launch point for MOE's protein modeling and protein structure
analysis commands.</p>

<p>The Sequence Editor displays the current molecular system as a residue
schematic, where each
chain is displayed as a list of residues.
For instance, the figure below shows a residue schematic of
<tt>$MOE/sample/mol/serine_prot.moe</tt>, a collection of serine
proteases.</p>

<p style="text-align:center"><img src="se/se.png" alt="Sequence Editor"></p>

<p>To open the Sequence Editor from the 
<a href="mainwin.htm">MOE Window</a>, select 
<b>Sequence Editor</b> from the <b>Window</b> menu.
Alternatively, press the <b>SEQ</b> button in the upper
right corner of the MOE Window.
</p>

<p>Chain numbering is reflected in the chain display order in the Sequence
Editor.  Chain numbers are shown in the column labeled <b>Chain</b>
on the left.
You can enable (or disable) display elements such as the compound
name or hydrogen bonds by turning on (or off) toggles in the 
<a href="#ModifyingTheDisplay"><b>Display</b></a> menu.
Operations on chains are performed using the mouse, the menu bar commands,
the Chain popup or the Selected Chains popup.</p>

<p>Residues composing the chains are drawn with either
<a href="#ModifyingTheDisplay">1- or 3-letter residue codes</a>, 
depending on the selected display mode. 
The horizontal alignment ruler 
in the Sequence Editor indicates the alignment position of residues. 
A given alignment position in the residue schematic
is referred to as a <i>residue column</i>.
Operations on residues are performed using the mouse, the menu bar
commands, or the popup menus.</p>

<p>To close the Sequence Editor, choose 
<b><span class="nobr">SE | File | Quit</span></b>.
The Cancel menu, in the top right corner of the window,
allows you to cancel tasks running in MOE.  Note that the
MOE Window and Database
Viewer also provide Cancel menus.  Next to the Cancel menu, the 
<b>SVL</b> and <b>MOE</b> buttons raise the
<a href="svlcommands.html">SVL Commands Window</a> and MOE Window,
respectively.</p>


<a name="Components_of_the_Sequence_Editor"></a><a name="Components"></a>
<h2>Components of the Sequence Editor</h2>

<p>
<center>
<img src="se/se_components.png" alt="Sequence Editor panel" />
</center>
</p>

<a name="Sequence_Editor_Menu_Bar"></a><a name="MenuBar"></a>
<h3>Sequence Editor Menu Bar</h3>

<p>The Sequence Editor menu bar
provides direct access to commands, applications and tools,
some of which can also be found in the
<a href="mainwin.htm#MOEWindow">MOE Window</a> (e.g. Protein Builder).</p>


<table width="100%" cellpadding="5">
<td valign="top" nowrap="nowrap" width="33%">

<fieldset>
    <legend> <b>File</b> </legend>

New<br />
&nbsp;&nbsp;<a href="../../mdb/db.htm">Database</a><br />
&nbsp;&nbsp;<a href="../../apps/ph4_query.htm">Pharmacophore Query</a><br />
&nbsp;&nbsp;<a href="ted.html">Text Editor</a><br />
&nbsp;&nbsp;<a href="../../apps/moevie.htm">MOEvie</a><br />
<a href="fileopen.htm">Open</a><br />
<a href="../../apps/filesave.htm">Save</a><br />
<a href="mainwin.htm">Close</a><br />
<a href="../../apps/print.htm">Print</a><br />
<a href="../../proteins/pdb_mdb.htm">Protein Database</a><br />
<a href="../../proteins/pdb_mdb.htm">PSILO/RCSB</a><br />
Quit

</fieldset>

<br />

<fieldset>
    <legend> <b>Edit</b> </legend>
    Copy SEQ<br />
    Paste<br />
    Set Chain Tag<br />
Protein<br />
&nbsp;&nbsp;<a href="../../apps/rotamer.htm">Mutate</a><br />
&nbsp;&nbsp;<a href="../../proteins/protool.htm">Amidate C Terminus</a><br />
&nbsp;&nbsp;<a href="../../proteins/protool.htm">Acetylate N Terminus</a><br />
&nbsp;&nbsp;<a href="../../proteins/protool.htm">Structure</a><br />


<a href="../../proteins/protool.htm">Protein Builder</a><br />
<a href="../../proteins/protool.htm">Create Sequence</a><br />
<a href="../../apps/rotamer.htm">Rotamer Explorer</a><br />
<a href="../../proteins/align.htm">Alignment Constraints</a><br />
<a href="../../proteins/align.htm">Constrain Residues</a><br />
<a href="../../proteins/align.htm">Unconstrain Residues</a><br />
<a href="#DeletingChainsAndResidues">Delete Selected Residues</a><br />
<a href="#MovingChains">Move Chains</a><br />
<a href="#JoiningAndSplitting">Split Chains</a><br />
<a href="#JoiningAndSplitting">Join Chains</a><br />
<a href="#SortChains">Sort Chains</a><br />
&nbsp;&nbsp;By Name<br />
&nbsp;&nbsp;By Tag<br />
<a href="#DeletingChainsAndResidues">Delete Selected Chains</a>

</fieldset>


<td valign="top" nowrap="nowrap" width="33%">
<fieldset>
    <legend> <b>Selection</b> </legend>

<a href="#ClearingSelection">Clear</a><br />
<a href="#InvertingSelection">Invert Residues</a><br />
<a href="#InvertingSelection">Invert Chains</a><br />
<a href="select.htm">Residue Selector</a><br />
<a href="#Conserved">Conserved Residues</a><br />
&nbsp;&nbsp;<a href="#Identity">Residue Identity</a><br />
&nbsp;&nbsp;<a href="#Structure">Structure</a><br />
&nbsp;&nbsp;<a href="#Hydrophobic">Hydrophobic</a><br />
&nbsp;&nbsp;<a href="#Hydrophilic">Hydrophilic</a><br />
&nbsp;&nbsp;<a href="#Acidic">Acidic</a><br />
&nbsp;&nbsp;<a href="#Basic">Basic</a><br />
&nbsp;&nbsp;<a href="#Aromatic">Aromatic</a><br />
&nbsp;&nbsp;<a href="#Aliphatic">Aliphatic</a><br />
&nbsp;&nbsp;<a href="#Polar">Polar</a><br />

<a href="#SelectionResidues">Residues</a><br />
&nbsp;&nbsp;<a href="#WithSelectedAtoms">With Selected Atoms</a><br />
&nbsp;&nbsp;<a href="#OfSelectedChains">Of Selected Chains</a><br />
&nbsp;&nbsp;<a href="#All">All Residues</a><br />
&nbsp;&nbsp;<a href="#Empty">Empty Residues</a><br />

<a href="#SelectionChains">Chains</a><br />
&nbsp;&nbsp;<a href="#ChainsWithSelectedAtoms">With Selected Atoms</a><br />
&nbsp;&nbsp;<a href="#ChainsWithSelectedResidues">With Selected Residues</a><br />
&nbsp;&nbsp;<a href="#WaterChains">Water Chains</a><br />
&nbsp;&nbsp;<a href="#AllChains">All Chains</a><br />
&nbsp;&nbsp;<a href="#EmptyChains">Empty Chains</a><br />
&nbsp;&nbsp;<a href="#ColoringSelectedChains">Color Separate Selected Chains</a><br />

<a href="#SelectingAtoms">Atoms</a><br />
&nbsp;&nbsp;<a href="#AtomsOfSelectedResidues">Of Selected Residues</a><br />
&nbsp;&nbsp;<a href="#AtomsOfSelectedChains">Of Selected Chains</a><br />
&nbsp;&nbsp;<a href="#IsolateAtoms">Isolate Atoms of Selected Residues</a>
<br /><a href="#SynchTip">Synchronize</a>

</fieldset>

<td valign="top" nowrap="nowrap" width="33%">
<fieldset>
    <legend> <b>Homology</b> </legend>
<a href="../../proteins/prosearch.htm">PDB Search</a><br />
<a href="../../proteins/kinase.htm">Kinase Search</a><br />
<a href="#ResetAlignment">Reset Alignment</a><br />
<a href="#PackAlignment">Pack Alignment</a><br />
<a href="../../proteins/align.htm">Align</a><br />
<a href="../../proteins/prosup.htm">Superpose</a><br />
<a href="../../proteins/procore.htm">Consensus</a><br />
<a href="../../proteins/promodel.htm">Homology Model</a><br />
<a href="../../proteins/antibody.htm">Antibody Modeler</a><br />
<a href="#PredictSecondaryStructure">Predict Secondary Structure</a>
</fieldset>

<br />

<fieldset>
    <legend> <b>Measure</b> </legend>
<a href="#Hydrophobicity">Hydrophobicity</a><br />
<a href="#ResidueCharge">Residue Charge</a><br />
<a href="#SolventAcc">Solvent Accessibility</a><br />
<a href="../../proteins/propred.htm">Predicted Solvent Accessibility</a><br />
<a href="../../proteins/propred.htm">Predicted Secondary Structure</a><br />
<a href="../../proteins/procontacts.htm">Protein Contacts</a><br />
<a href="../../apps/progeom.htm">Protein Geometry</a>
</fieldset>

<br />

<fieldset>
    <legend> <b>Display</b> </legend>
<a href="#ModifyingTheDisplay">Compound Name</a><br />
<a href="#ModifyingTheDisplay">Chain Tag</a><br />
<a href="#ModifyingTheDisplay">Actual Secondary Structure</a><br />
<a href="#ModifyingTheDisplay">Predicted Secondary Structure</a><br />
<a href="#ModifyingTheDisplay">Hydrogen Bonds</a><br />
<a href="#ModifyingTheDisplay">Single Letter Residues</a><br />
<a href="#ModifyingTheDisplay">Residue Indices</a><br />
<a href="#ModifyingTheDisplay">Residue UID</a><br />
<a href="#ColoringResidues">Color Residues</a><br />
<a href="#ColorByChainTag">Color Chains</a><br />
&nbsp;&nbsp;By Number<br />
&nbsp;&nbsp;By Tag

</fieldset>

</tr>
</table>

<a name="Command_Line"></a><a name="CommandLine"></a>
<h3>Command Line</h3>

<p>The <a href="svlcommands.html">Command Line Interface (CLI)</a>
is where SVL commands are typed and issued.  While the Sequence Editor
has the focus, all keystrokes are automatically directed to the CLI
regardless of where the mouse is positioned within the window.</p>

<p>The command line area is also used as a prompt area for several
MOE functions.  This prompt area may be for entering data
or simply for displaying messages.  In either case, to cancel
the function which is displaying the prompt, press <tt>&lt;Esc&gt;</tt>.  While
the Sequence Editor's CLI is occupied by a prompt, 
other CLIs in MOE function as usual.</p>

<p>For more information on the Command Line Interface see
the <a href="svlcommands.html">SVL Commands Window</a>.</p>

<a name="Chain_Numbers"></a><a name="ChainNumbers"></a>
<h3>Chain Numbers</h3>
<div class="floatright">
<img src="se/se_chainnumbers.png" alt="CLI in the Sequence Editor" />
</div>
<p>The chain numbers are the boxes to the left of the residue display area.
Chains are always numbered sequentially, and each chain number is
colored by its <i>chain color</i>.
If you color the atoms by chain color 
(in the MOE Window), this is the color which will be used.
Chain color can be set using the <b>Chain Color</b> command
in the Chain popup.  Chains can also be colored by tag using
<span class="menu">SE | Display | Color Chains | By Tag</span>.
A selected chain has its number displayed inset.</p>

<p>The area labeled <b>Chain</b> above the chain numbers is called the
<i>chain label</i> and when pressed, selects or deselects all chains.
A popup menu which acts
upon the selected chains can be accessed here using the mouse popup button.</p>

<br style="clear:right;" />

<a name="Residue_Display_Area"></a><a name="ResidueDisplayArea"></a>
<h3>Residue Display Area</h3>
<p>The residue display area comprises the column ruler,
the residues themselves and the empty space between the drawn residues.
The popup menus <b>Residue Column</b>, <b>Residue</b>, and 
<b>Selected Residues</b> are located in these areas.</p>


<br style="clear:right;">
<a name="Footer"></a>
<h3>Footer</h3>
<img src="se/se_footer.png" alt="Sequence Editor Footer" class="floatright">

<p>The bottom part of the window is referred to as the footer and is used
to turn on or off the display of scalar plots.  The Sequence Editor can
plot scalar values assigned to residues (see 
<a href="../fcnref/residue.htm">rSetScalar</a>) in a graph which
is aligned with the residues. The scalar plot displays values in the range
-1 to 1; values outside this range are clamped (i.e. the graphs are
clipped).


<p>The Measure menu provides several functions which use these plots.
See <a href="#Measuring">Measuring Residue Properties and
Stereochemistry</a> for
more information.

<a name="Popup_Menus"></a>
<h3>Popup Menus</h3>

<p>The Sequence Editor features several popup menus with a variety of
commands.  Familiarity with these popups will give you a much greater
control over the display of atoms in the MOE Window and of residues in the
Sequence Editor.  Press the <a href="mouse.html">mouse popup button</a>
over each of the following objects to display their respective popup menus:

<p>
<center>
<img src="../molsystems/molrep/seq_editor_popup.png"
alt="Sequence Editor popup menus" />
</center>
</p>

<p>
<center>
<table class="titlebar" width="95%">
	<tr>
	<th width="20%">Popup Menu</th>
	<th>Location </th>
	<th>Action</th>
	</tr>
	<tr>
	<td valign="top">
	<b>Chain</b></td>
	<td valign="top">Over a chain number.</td>
	<td valign="top">Commands apply to the
	selected residues in a chain. If no residues are selected, 
	all residues in the chain will be used.</td>
	</tr>
	
	<tr>
	<td valign="top">
	<b>Selected Chains</b></td>
	<td valign="top">Over the <b>Chain</b> label.</td>
	<td valign="top">Commands apply to the selected chains.</td>
	</tr>
	
	<tr>
	<td valign="top">
	<b>Residue</b></td>
	<td valign="top">Over a residue.</td>
	<td valign="top">Commands apply to the single residue only.</td>
	</tr>
	
	<tr>
	<td valign="top">
	<b>Selected Residues</b></td>
	<td valign="top">In the display area away from residues, chains or ruler.</td>
	<td valign="top">Commands apply to all selected residues.</td>
	</tr>
	
	<tr>
	<td valign="top">
	<b>Residue Column</b></td>
	<td valign="top">In the alignment ruler.</td>
	<td valign="top">Commands apply to the residues in the column. </td>
	</tr>
</table>
</center>

<a name="Sequence_Editor_Popup_Menus"></a>
<a name="SequenceEditorPopupMenus"></a>
<h2>Sequence Editor Popup Menus</h2>

<a name="Atom_Commands"></a>
<h3>Atom Commands</h3>

<p>
Atom commands are located in all popup menus, possibly more than once
as they can also appear in the Residues and Selected Residues submenus.
These commands apply to the atoms belonging to the
residue or chain as specified by the location in which the popup was invoked.
</p>

<p>
<center>
<table class="titlebar" width="95%">
	<tr><th colspan="3">Atom Visibility and Rendering</th></tr>
	<tr>
	  <th>Command</th>
	  <th>Action</th>
	</tr>
	  <tr>
	    <td valign="top"><span class="menu">View | Atoms</span></td>
	    <td valign="top">Show the child atoms of the residues or chains.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">View | Isolate &amp; View</span></td>
	    <td valign="top">Center and show the child atoms in the MOE Window;
	    hide all other atoms.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">View | Isolate</span></td>
	    <td valign="top">Show the child atoms in the MOE Window; hide all
	    other atoms.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Hide | Atoms</span>,
		<span class="menu">Show | Atoms</span></td>
	    <td valign="top">Hide or show the child atoms in the MOE Window.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Hide | Backbone</span>, 
		<span class="menu">Show | Backbone</span></td>
	    <td valign="top">Hide or show the child backbone atoms in the MOE Window.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Hide | Sidechain</span>, 
		<span class="menu">Show | Sidechain</span></td>
	    <td valign="top">Hide or show the child sidechain atoms in the MOE Window.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Hide | Near Residues (4.5A)</span>, 
		<span class="menu">Show | Near Residues (4.5A)</span></td>
	    <td valign="top">Hide or show the atoms of residues that contain an atom 
	    within 4.5 angstroms of the child atoms.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Hide | Far Residues</span>, 
		<span class="menu">Show | Far Residues</span></td>
	    <td valign="top">Hide or show the atoms of residues that do not 
	    contain an atom within 4.5 angstroms of the child atoms.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Atoms</span></td>
	    <td valign="top">Render the atoms of the involved residues in the 
			MOE Window in the specified rendering style.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Ribbon</span></td>
	    <td valign="top">Render the backbone ribbon of the involved residues in the 
			MOE Window in the specified rendering style.</td>
	  </tr>
</table>
</center>
</p>

<p>
<center>
<table class="titlebar" width="95%">
	<tr><th colspan="3">Atom Selection</th></tr>
	<tr>
	  <th>Command</th>
	  <th>Action</th>
	</tr>
	  <tr>
	    <td valign="top"><span class="menu">Select | Atoms</span></td>
	    <td valign="top">
		Select the child atoms of the residues or chains.
	    </td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Select | Invert Atoms</span></td>
	    <td valign="top">
		Invert the atom selection state of each child atom
		of the residues or chains.
	    </td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Select | Sidechain</span></td>
	    <td valign="top">
		If the residues or chains belong to a protein,
		select the child sidechain atoms.
	    </td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Select | Backbone</span></td>
	    <td valign="top">
		If the residues or chains belong to a protein,
		select the child backbone atoms.
	    </td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Select | Near Residues (4.5A)</span></td>
	    <td valign="top">Select the atoms of
		residues that contain an atom within 4.5
		angstroms of any child atom of the residues/chains.
	    </td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Select | Far Residues</span></td>
	    <td valign="top">Select that atoms of residues
		that do not contain an atom within 4.5
		angstroms of any child atom of the residues/chains.
	    </td>
	  </tr>
</table>
</center>
</p>

<a name="Residue_Commands"></a>
<h3>Residue Commands</h3>

<p>Residue commands control residue selection, editing, deletion, and
display color in the Sequence Editor.
The drawing color of the residue name is called
the <i>residue color</i>, and can be selected from
a palette of pre-set colors.
</p>

<p>
<center>
<table class="titlebar" width="95%">
	<tr><th colspan="3">Residue Selection</th></tr>
	<tr>
	  <th width=20%>Command</th>
	  <th>Popup Location</th>
	  <th>Action</th>
	</tr>
	
	  <tr>
	    <td valign="top"><span class="menu">Select | Near Residues (4.5A)</span></td>
	    <td valign="top">
	    Residue<br />
	    Selected Residues<br />
	    Residue Column<br />
	    Chain<br />
	    Selected Chains<br />
	    <span class="nobr">... | Selected Residues</span>
	    </td>
	    <td valign="top">Select residues that contain an atom within 4.5
		angstroms of any child atom of the residues/chains.
		The atoms of the selected residues will also be selected.
	    </td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Select | Far Residues</span></td>
	    <td valign="top">
	    Residue<br />
	    Selected Residues<br />
	    Residue Column<br />
	    Chain<br />
	    Selected Chains<br />
	    ... | Selected Residues
	    </td>
	    <td valign="top">Select residues that do not contain an
		atom within 4.5
		angstroms of any child atom of the residues/chains.
		The atoms of the selected residues will also be selected.
	    </td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Residues | Select All</span></td>
	    <td valign="top">
	    Chain<br />
	    Selected Chains
	    </td>
	    <td valign="top">Select all residues in the chain
		or selected chains.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Residues | Invert Selection</span></td>
	    <td valign="top">
	    Chain<br />
	    Selected Chains
	    </td>
	    <td valign="top">Invert the residue selection state of each residue
		in the chain or selected chains.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Residues | Clear Selection</span></td>
	    <td valign="top">
	    Chain<br />
	    Selected Chains
	    </td>
	    <td valign="top">Deselect all residues in the chain or
		selected chains.</td>
	  </tr>
<!--
	  <tr>
	    <td valign="top">To Clipboard</td>
	    <td valign="top">
	    Chain<br />
	    Selected Chains
	    </td>
	    <td valign="top">Copy the single letter codes for all residues in the
		chain to the clipboard.</td>
	  </tr>
-->
</table>
</center>
</p>


<p>
<center>
<table class="titlebar" width="95%">
	<tr><th colspan="3">Residue Editing and Deletion</th></tr>
	<tr>
	  <th width=20%>Command</th>
	  <th>Popup Location</th>
	  <th>Action</th>
	</tr>
	<tr>
	  <td valign="top"><span class="menu">Color</span></td>
	  <td valign="top">
		Residue<br />
		Selected Residues<br />
		Residue Column<br />
		<span class="nobr">... | Selected Residues</span><br />
		... | Residues
	  </td>
	  <td valign="top">Draw the residue name in the Sequence Editor display
	    in the specified color.
	    Atoms will be drawn in this color when the coloring scheme
	    in the MOE Window is set to
	    <span class="menu"><b>Render | Atoms | Residue</b></span>.
<!--
	    In this mode, the atom colors are updated automatically
	    when the residue color is changed.
-->
	  </td>
	</tr>
	  <tr>
	    <td valign="top"><span class="menu">Rotamer Explorer</span></td>
	    <td valign="top">
	    Residue
	    </td>
	    <td valign="top">Open the 
		<a href="../../apps/rotamer.htm">Rotamer Explorer</a>
	    to explore conformations of the current residue.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Delete this Residue</span></td>
	    <td valign="top">
	    Residue
	    </td>
	    <td valign="top">Delete the current residue.
	    </td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Delete Selected Residues</span></td>
	    <td valign="top">
	    Residue<br />
	    Selected Residues
	    </td>
	    <td valign="top">Delete all currently selected residues.
	    </td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Delete this Column</span></td>
	    <td valign="top">
	    Residue Column
	    </td>
	    <td valign="top">Delete all residues in the current residue column.
	    </td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Delete</span></td>
	    <td valign="top">
	    ... | Selected Residues
	    </td>
	    <td valign="top">Delete all selected residues belonging
		to the current chain, selected chains, or residue column.
	    </td>
	  </tr>
</table>
</center>
</p>

<a name="Chain_Commands"></a>
<h3>Chain Commands</h3>

<p>Chain commands are used to reorder, split, join, delete,
pack alignment, reset alignment, set tag name,
and set chain number display color of chains.
The commands to move chains also appear in the
<span class="menu">SE | Edit</span> menu.
The display color of the
chain number in the Sequence Editor is called the <i>chain color</i>,
and can be selected from a palette of pre-set colors.
</p>

</p>

<p>
<center>
<table class="titlebar" width="95%">
	<tr><th colspan="3">Chain Reordering Commands</th></tr>
	<tr>
	  <th>Command</th>
	  <th>Popup Location</th>
	  <th>Action</th>
	</tr>
	
	  <tr>
	    <td valign="top"><span class="menu">Move | Top</span></td>
	    <td valign="top">Chain</td>
	    <td valign="top">Move the current chain to the top of the list.</td>
	
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Move | Bottom</span></td>
	    <td valign="top">Chain</td>
	    <td valign="top">Move the current chain chains to the bottom 
			of the list.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Move | Before</span></td>
	    <td valign="top">Chain</td>
	    <td valign="top">Prompt for a chain number and move the current
		chain just above it.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Move Chains | Top</span></td>
	    <td valign="top"><span class="nobr">Selected Chains</span></td>
	    <td valign="top">Move the selected chains to the top of the
		list, maintaining their relative order.</td>
	
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Move Chains | Bottom</span></td>
	    <td valign="top">Selected Chains</td>
	    <td valign="top">Move the selected chains to the bottom 
			of the list, maintaining their relative order.</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Move Chains | Before</span></td>
	    <td valign="top">Selected Chains</td>
	    <td valign="top">Prompt for a chain number and move the 
		selected chains just above it, maintaining their
		relative order.</td>
	  </tr>
</table>
</center>
</p>

<p>
<center>
<table class="titlebar" width="95%">
	<tr><th colspan="3">Chain Selection</th></tr>
	<tr>
	  <th>Command</th>
	  <th>Popup Location</th>
	  <th>Action</th>
	</tr>
	  <tr>
	    <td valign="top"><span class="menu">Select | Chains with Same Tag</span></td>
	    <td valign="top">Chain</td>
	    <td valign="top">Select all chains with the same tag as
		the selected chain.</td>
	  </tr>
</table>
</center>
</p>
	
<!--
<p><b>Delete</b> is found in each popup as well as the Selected
Residues submenu in several of the popups.  In each case, the items
to delete will apply to the object over which the popup menu was 
invoked.  For instance, over a chain, the deletion applies to the
chain; over a residue, the deletion applies to the single
residue.  <b>Split</b> is located in the Residue popup, the Selected Residue
popup, and in the Edit menu of the menu bar.  <b>Join</b> is located in the
Selected Chains popup as well as in the Edit menu of the menu bar.</p>
-->

<p>
<center>
<table class="titlebar" width="95%">
	<tr><th colspan="3">Chain Editing and Deletion</th></tr>
	<tr>
	<th>Command</th>
	<th>Popup Location</th>
	<th>Action</th>
	</tr>
	
	<tr>
	<td valign="top"><span class="menu">Set Chain Tag</span></td>
	<td valign="top">
	    Chain
	</td>
	<td valign="top">Edit the chain tag of the current chain.
	</td>
	</tr>
	<tr>
	<td valign="top"><span class="menu">Set Chain Tag</span></td>
	<td valign="top">
	    <span class="nobr">Selected Chains</span>
	</td>
	<td valign="top">Edit the chain tags of the currently selected chains.
	    At the end of the operation, all selected chains will have the
	    same tag.
	</td>
	</tr>
	<tr>
	  <td valign="top"><span class="menu">Chain Color</span></td>
	  <td valign="top">
		Chain<br />
		Selected Chains
	  <td valign="top">Draw the chain number in the Sequence Editor display
	    in the specified color.
	    Atoms will be drawn in this color when the coloring scheme
	    in the MOE Window is set to
	    <span class="menu"><b>Render | Atoms | Chain</b></span>.
	  </td>
	</tr>
	  <tr>
	    <td valign="top"><span class="menu">Reset Alignment</span></td>
	    <td valign="top">
	    Chain<br />
	    Selected Chains
	    </td>
	    <td valign="top">Reset the alignment for the chain(s)
		(remove all gaps).</td>
	  </tr>
	  <tr>
	    <td valign="top"><span class="menu">Pack Alignment</span></td>
	    <td valign="top">
	    Selected Chains
	    </td>
	    <td valign="top">Pack the alignment for the
		selected chains (remove all empty columns).</td>
	  </tr>
	<tr>
	<td valign="top"><span class="menu">Split Chain</span></td>
	<td valign="top">Residue</td>
	<td valign="top">Split the chain at the current residue.
	    The current residue will become the first residue in
	    the split-off chain which will be positioned immediately
	    below the original chain.
	</td>
	</tr>
	<tr>
	<td valign="top"><span class="menu">Split Chains</span></td>
	<td valign="top">Selected Residues</td>
	<td valign="top">Split the chains at the selected residues.
	    Each selected residue will become the first residue in
	    a split-off chain.
	</td>
	</tr>
	<tr>
	<td valign="top"><span class="menu">Join Chains</span></td>
	<td valign="top">Selected Chains</td>
	<td valign="top">Join the selected chains in sequence,
	    according to chain number.  The chain with the smallest
	    chain number will become the parent of all residues.
	    Note that no bonds are formed between atoms.
	</td>
	</tr>
	<tr>
	<td valign="top"><span class="menu">Delete this Chain</span></td>
	<td valign="top">
	    Chain
	</td>
	<td valign="top">Delete the current chain.
		The remaining chains will be renumbered.
	</td>
	</tr>
	<tr>
	<td valign="top"><span class="menu">Delete Selected Chains</span></td>
	<td valign="top">
	    Chain<br />
	    Selected Chains<br />
	</td>
	<td valign="top">Delete all selected chains.
		The remaining chains will be renumbered.
	</td>
	</tr>
</table>
</center>
</p>

<a name="Selecting_Molecular_Objects"></a><a name="SelectingMolecularObjects"></a>
<h2>Selecting Molecular Objects</h2>

<p>You can select chains and residues as well as atoms in the Sequence
Editor using the mouse, the Sequence Editor Selection menu or the
Sequence Editor popup menus.</p>

<a name="SynchTip"></a>
<p><span class="tip">Tip</span>&nbsp;&nbsp;
<span class="menu">SE | Selection | Synchronize</span> synchronizes the
selection of atoms in the MOE Window with the selection of residues in
the Sequence Editor.</p>

<a name="Using_the_Left_Mouse_Button"></a><a name="UsingTheLeftMouseButton">
<h3>Using the Left Mouse Button in the Sequence Editor</h3></a>

<p>
<center>
<table class="titlebar" width="95%">
	<tr><th colspan="2">Chains</th></tr>
	<tr>
	<td valign="top">Select a single chain, deselecting all other
	chains.
	</td>
	<td valign="top">Click on a chain number.
	</td>
	</tr>
	<tr>
	<td valign="top">Select all chains
	</td>
	<td valign="top">Click on <b>Chain</b> label 
	(if all chains are already selected, clicking on the <b>Chain</b> label 
	will deselect all chains).
	</td>
	</tr>
	<tr>
	<td valign="top">Select contiguous chains
	</td>
	<td valign="top">Click on first chain then <tt>Shift</tt> - 
	click on last chain.
	</td>
	</tr>
	<tr>
	<td valign="top">Toggle selection state of a single chain
	</td>
	<td valign="top"><tt>Ctrl</tt> - click on chain number.
	</td>
	</tr>
	<tr>
	<td valign="top">Clear chain selection
	</td>
	<td valign="top">Click below chain numbers.
	</td>
	</tr>

<tr><th colspan="2">Residues</th></tr>

  <tr>
    <td valign="top">Select a single residue deselecting all other residues.
    </td>
    <td valign="top">Click on residue.
    </td>
  </tr>
  <tr>
    <td valign="top">Toggle selection state of a single residue
    </td>
    <td valign="top"><tt>Ctrl</tt> - click on residue.
    </td>
  </tr>
  <tr>
    <td valign="top">Select a column of residues, 
    i.e. residues across chains that occupy the same alignment position 
    (as indicated by the ruler)</td>
    </td>
    <td valign="top">Click in alignment ruler.
    </td>
  </tr>
  <tr>
    <td valign="top">Select contiguous residues (within the same chain) or
    residue columns
    </td>
    <td valign="top">Click on first residue (or residue column)
	then <tt>Shift</tt> - click  on last residue (or residue column).
    </td>
  </tr>
  <tr>
    <td valign="top">Toggle selection state of a column of residues
    </td>
    <td valign="top"><tt>Ctrl</tt> - click on alignment ruler
    (if all residues in the column are already selected, <tt>Ctrl</tt> - click
    will deselect all residues).
    </td>
  </tr>
  <tr>
    <td valign="top">Draw selection box around residues
    </td>
    <td valign="top">Drag over desired residues using the left mouse button.
    </td>
  </tr>
  <tr>
    <td valign="top">Add to selection using a selection box
    </td>
    <td valin="top"><tt>Ctrl</tt> - drag over residues using
	the left mouse button.
    </td>
  <tr>
    <td valign="top">Clear residue selection</td>
    <td valign="top">Click away from residues, chains or ruler.
    </td>
  </tr>
</table>
</center>
</p>

<a name="Using_Popup_Menus"></a><a name="UsingPopupMenus"><h3>Using Popup Menus</h3></a>

<p>The popup menus in the Sequence Editor allow you to select, view,
hide or show atoms of a chain, a residue, a group of selected residues,
or a residue column (based on alignment). Furthermore, the 
<b><span class="nobr">Residues | Select All</span></b>
option in the Chain and Selected Chains popups selects
all residues in a chain or in all selected chains.  For a full description
of the popup menus see <a href="#SequenceEditorPopupMenus">Sequence Editor
Popup Menus</a>.</p>

<a name="Using_the_Selection_Menu"></a><a name="UsingSelectionMenu">
<h3>Using the Selection Menu</h3></a>

<a name="Selecting_Atoms_in_the_MOE_Window"></a><a name="SelectingAtoms"></a>
<h4>Selecting and Isolating Atoms in the MOE Window</h4>

<p>Commands in the <span class="menu">SE | Selection</span>
menu enable you to select
and isolate atoms of the molecular system.  Every selection operation adds
to the current selection set.  In other words, previously selected atoms
remain selected.</p>

<!--
<p>The <span class="menu">Selection | Atoms</span> submenu contains the following:</p>

<ul>
	<li><a name="AtomsOfSelectedResidues"></a><b>Of Selected Residues</b>.
	Selects the child atoms of residues selected in the Sequence Editor.</li>
	
	<li><a name="AtomsOfSelectedChains"></a><b>Of Selected Chains</b>.
	Selects the child atoms of chains selected in the Sequence Editor.</li>
	
	<li><a name="IsolateAtoms"></a><b>Isolate Atoms of Selected Residues</b>.
	Shows
	and centers atoms of selected residues. All atoms that do not belong
	to the selected residues are hidden.</li>
</ul>
-->

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Selection | Atoms</th></tr>
	<tr>
	<th>Command</th>
	<th>Action</th>
	</tr>
    <tr>
    <td valign="top">
    <a name="AtomsOfSelectedResidues"></a>
	<span class="menu">Of Selected Residues</span>
    </td>
    <td valign="top">
    Select the child atoms of all selected residues.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="AtomsOfSelectedChains"></a>
	<span class="menu">Of Selected Chains</span>
    </td>
    <td valign="top">
    Select the child atoms of all selected chains.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="IsolateAtoms"></a>
	<span class="menu">Isolate Atoms of Selected Residues</span>
    </td>
    <td valign="top">
    Shows and centers the atoms of all selected residues.
    Atoms not belonging to the selected residues are hidden.
    </td>
    </tr>
</table>
</center>
</p>

<p>
Atoms of selected chains can be isolated by color using the
<span class="menu">SE | Selection | Chains | Color Separate Selected
Chains</span> command.
</p>

<a name="Selecting_Residues_and_Chains"></a><a name="SelectingResChains"></a>
<h4>Selecting Residues and Chains</h4>

<p>The <span class="menu">SE | Selection</span> menu enables you to highlight
and isolate chains or residues of the molecular system.  Selection operations
located in the Selection menu 
add to the current selection set.  In other
words, previously selected objects remain selected.</p>

<a name="SelectionChains"></a>
<!--
<p>Using the <span class="menu">Selection | Chains</span> submenu, 
you can select chains according to the following criteria:</p>

<ul>
<li><a name="ChainsWithSelectedAtoms"></a><b>With Selected Atoms</b>.
Select the chains that have at least one atom selected in the MOE
Window.</li>

<li><a name="ChainsWithSelectedResidues"></a><b>With Selected Residues</b>.
Select the chains that have at least one residue selected in the Sequence
Editor.</li>

<li><a name="WaterChains"></a><b>Water Chains</b>.
MOE stores all water residues from a given file in the same chain
when reading Protein Data Bank<sup>TM</sup> files. If such a chain is
found then it will be selected.</li>

<li><a name="AllChains"></a><b>All Chains</b>.</li>

<li><a name="EmptyChains"></a><b>Empty Chains</b>.
Select chains with no residues.</li>
</ul>
-->

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Selection | Chains</th></tr>
	<tr>
	<th>Command</th>
	<th>Action</th>
	</tr>
    <tr>
    <td valign="top">
    <a name="ChainsWithSelectedAtoms"></a>
	<span class="menu">With Selected Atoms</span>
    </td>
    <td valign="top">
    Select the chains that have at least one atom selected.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="ChainsWithSelectedResidues"></a>
	<span class="menu">With Selected Residues</span>
    </td>
    <td valign="top">
    Select the chains that have at least one residue selected.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="WaterChains"></a>
	<span class="menu">Water Chains</span>
    </td>
    <td valign="top">
    MOE stores all water residues from a given file in the same chain
    when reading Protein Data Bank<sup>TM</sup> files. If such chains are
    found, they will be selected.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="AllChains"></a>
	<span class="menu">All Chains</span>
    </td>
    <td valign="top">
    Select all chains in the system.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="EmptyChains"></a>
	<span class="menu">Empty Chains</span>
    </td>
    <td valign="top">
    Select chains with no residues.
    </td>
    </tr>
</table>
</center>
</p>

<a name="SelectionResidues"></a>
<!--
<p>The <span class="nobr"><b>Selection | Residues</b></span> command is used to select
residues according to the following criteria:</p>

<ul>

<li><a name="WithSelectedAtoms"></a><b>With Selected Atoms.</b>
Select the residues with at least one
atom selected in the MOE Window.  Enable 
<span class="menu">SE | Selection | Synchronize</span>
to perform this operation
on a continual basis.</li>

<li><a name="OfSelectedChains"></a><b>Of Selected Chains.</b>
Select
the residues belonging to chains that are selected in the Sequence
Editor.</li>

<li><a name="All"></a><b>All Residues.</b></li>

<li><a name="Empty"></a><b>Empty Residues.</b> 
Select residues containing no atoms.</li>

</ul>
-->

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Selection | Residues</th></tr>
	<tr>
	<th>Command</th>
	<th>Action</th>
	</tr>
    <tr>
    <td valign="top">
    <a name="WithSelectedAtoms"></a>
	<span class="menu">With Selected Atoms</span>
    </td>
    <td valign="top">
    Select the residues with at least one atom selected.  The
    <span class="menu">SE | Selection | Synchronize</span>
    will keep the atom and residue selections synchronized automatically.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="OfSelectedChains"></a>
	<span class="menu">Of Selected Chains</span>
    </td>
    <td valign="top">
    Select the residues belonging to selected chains.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="All"></a>
	<span class="menu">All Residues</span>
    </td>
    <td valign="top">
    Select all residues in the system.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Empty"></a>
	<span class="menu">Empty Residues</span>
    </td>
    <td valign="top">
    Select residues with no atoms.
    </td>
    </tr>
</table>
</center>
</p>


<a name="Conserved"></a>
<p>The <span class="menu">Selection | Conserved Residues</span>
menu is used to
select those residues where the specified property (identity, structure,
etc.) is conserved for the entire column (alignment position).  Only
alignment positions with no gaps are
considered conserved, i.e. the residue columns must be complete.
This command applies to protein chains only.</p>

<!--
<ul>
<li><a name="Identity"></a><b>Residue Identity.</b>
Select residues
according to residue name.</li>

<li><a name="Structure"></a><b>Structure.</b>
Select amino acid
residues according
to their secondary structure type as well as columns where the
secondary structure type is entirely conserved.</li>
<li><a name="Hydrophobic"></a><b>Hydrophobic.</b>
ALA, VAL, ILE, LEU, MET, TRP, PHE, TYR</li>
<li><a name="Hydrophilic"></a><b>Hydrophilic.</b>
	SER, THR, ASN, GLN, GLU, ASP, ARG, LYS, HIS </li>
<li><a name="Acidic"></a><b>Acidic.</b> GLU, ASP </li>
<li><a name="Basic"></a><b>Basic.</b> ARG, LYS, HIS</li>
<li><a name="Aromatic"></a><b>Aromatic.</b> PHE, TYR, TRP, HIS</li>
<li><a name="Aliphatic"></a><b>Aliphatic.</b> VAL, LEU, ILE, GLY, ALA, MET, PRO </li>
<li><a name="Polar"></a><b>Polar.</b> ASN, SER, THR, GLN</li>
</ul>
-->

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Selection | Conserved Residues</th></tr>
	<tr>
	<th>Command</th>
	<th>Action</th>
	</tr>
    <tr>
    <td valign="top">
    <a name="Identity"></a><span class="menu">Residue Identity</span>
    </td>
    <td valign="top">
    Select residues according to residue name.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Structure"></a><span class="menu">Structure</span>
    </td>
    <td valign="top">
    Select amino acid residues according
    to their secondary structure type as well as columns where the
    secondary structure type is entirely conserved.
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Hydrophobic"></a><span class="menu">Hydrophobic</span>
    </td>
    <td valign="top">
	Ala, Val, Ile, Leu, Met, Trp, Phe, Tyr
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Hydrophilic"></a><span class="menu">Hydrophilic</span>
    </td>
    <td valign="top">
	Ser, Thr, Asn, Gln, Glu, Asp, Arg, Lys, His
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Acidic"></a><span class="menu">Acidic</span>
    </td>
    <td valign="top">
	Glu, Asp
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Basic"></a><span class="menu">Basic</span>
    </td>
    <td valign="top">
	Arg, Lys, His
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Aromatic"></a><span class="menu">Aromatic</span>
    </td>
    <td valign="top">
	Phe, Tyr, Trp, His
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Aliphatic"></a><span class="menu">Aliphatic</span>
    </td>
    <td valign="top">
	Val, Leu, Ile, Gly, Ala, Met, Pro
    </td>
    </tr>
    <tr>
    <td valign="top">
    <a name="Polar"></a><span class="menu">Polar</span>
    </td>
    <td valign="top">
	Asn, Ser, Thr, Gln
    </td>
    </tr>

</table>
</center>
</p>

<a name="Inverting_Chain_or_Residue_Selection"></a><a name="InvertingSelection"></a>
<a name="Clearing_Chain_and_Residue_Selection"></a>
<h4>Inverting and Clearing the Chain and Residue Selection</h4>

<p>The <span class="menu">SE | Selection</span> menu
contains commands for clearing and inverting chain and residue selection:

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Selection</th></tr>
	<tr>
	<th>Command</th>
	<th>Action</th>
	</tr>
    <tr>
    <td valign="top">
	<span class="menu">Invert Residues</span>
    </td>
    <td valign="top">
	Invert the current residue selection (selected residues become
	unselected, and vice-versa).
    </td>
    </tr>
    <tr>
    <td valign="top">
	<span class="menu">Invert Chains</span>
    </td>
    <td valign="top">
	Invert the current chain selection (selected chains become
	unselected, and vice-versa).
    </td>
    </tr>
    <tr>
    <td valign="top">
	<span class="menu">Clear</span>
    </td>
    <td valign="top">
	Deselect all chains and all residues.
    </td>
    </tr>
</table>
</center>
</p>

<!--
<p>To invert the chain selection in the Sequence Editor, choose <b>Invert
Chains</b> in the Selection menu.  To invert the residue selection, choose
<b>Invert Residues</b> in the Selection menu.</p>

<h4><a name="ClearingSelection"></a>Clearing Chain and Residue Selection</h4>

<p>The <span class="menu">Selection | Clear</span> command deselects all chains
and all residues in the Sequence Editor.</p>
-->

<p>To clear the chain selection only, click the left mouse button in the
space beneath the Chain column (away from chain numbers) or on the
<b>Chain</b> label (you will have to press the <b>Chain</b> label
twice if not all chains are currently selected).</p>

<p>To clear the residue selection only, click the left mouse button 
away from any residue, within the residue display area.</p>


<a name="Using_the_Residue_Selector_Panel"></a><a name="ResidueSelectorPanel"><h3>Using the Residue Selector Panel</h3>

<p>The Residue Selector is one of several panels in MOE providing
detailed selection of molecular data. These applications are
fully described in the <a href="select.htm">Selection Panels</a> chapter.</p>

<a name="Editing_Molecular_Objects"></a><a name="EditingMolecularObjects"></a><h2>Editing Molecular Objects</h2>

<a name="Moving_Residues"></a><a name="MovingResidues"></a>
<h3>Moving Residues</h3>

<p>Residues are moved with mouse drag movements within the residue display
area.  They can be moved to the right or left, but not beyond the start
of the alignment ruler.</p>

<p>
<center>
<table class="titlebar" width="95%">
<tr>
<th>Action</th>
<th>Description</th>
</tr>
<tr>
<td valign="top">To move one residue</td>
<td valign="top">Drag the residue in the Sequence Editor using the middle
mouse button.  A red border highlights the residue being moved.

<p>The residue is moved right or left (changing its alignment
position). Any residues encountered will be pushed 
along, like beads on a string. 
Gaps between residues are not preserved.</p></td>
</tr>

<tr>
<td valign="top">To move several residues </td>
<td valign="top">
Select the residues to move (residues can belong to more than one chain
and do not have to be consecutive), and press <tt>Alt</tt> while dragging
the middle mouse button.

<p>Selected residues are moved right or left. Any residues
encountered will be pushed along, like beads on a string.
Selected residues maintain their relative alignment positions while moving.</p></td>
</tr>

<tr>
<td valign="top">To push residues in a chain</td>
<td valign="top">Press <tt>Shift</tt> 
while dragging the middle mouse button over a residue in the chain.
<p>
All residues to the right or left of the dragged residue are pushed to
the right or left along with it, while maintaining gaps.
</td>
</tr>

<tr>
<td valign="top">To slide all residues in an entire chain</td>
<td valign="top">Press <tt>Ctrl</tt> and drag the middle mouse button
over any residue in the chain.  Gaps are maintained.</td>
</tr>
</table>
</center>
</p>

<a name="Moving_Chains"></a><a name="MovingChains"></a>
<h3>Reordering Chains</h3>

<p>You can reorder the chains in the system by moving one or more 
chains using commands in the 
<span class="nobr"><b>Edit | Move Chains</span></b> submenu or in the
Chain and Selected Chains popup menus. </p>

<p>When using the Edit menu, you can move several chains at once, but you
must first select the chains you wish to move.  This applies also to the
Selected Chains popup menu.</p>

<p>The equivalent Chain popup commands move a single chain at a time.</p>

<!--
<ul>
<li><b>Top.</b> Move the selected chains to the top of the list.</li>

<li><b>Bottom.</b> Move the selected chains to the bottom of the list.</li>

<li><b>Before.</b> Move the selected chains before a given chain.  You will
be prompted to select the chain before which the selected chains will
be moved.</li>

</ul>
-->

<p>
<center>
<a name="SortChains"></a>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Edit</th></tr>
	<tr>
	<th>Command</th>
	<th>Action</th>
	</tr>
    <tr>
    <td valign="top">
	<span class="menu">Move Chains | Top</span>
    </td>
    <td valign="top">
	Move the selected chains to the top of the chain list.
    </td>
    </tr>
    <tr>
    <td valign="top">
	<span class="menu">Move Chains | Bottom</span>
    </td>
    <td valign="top">
	Move the selected chains to the bottom of the chain list.
    </td>
    </tr>
    <tr>
    <td valign="top">
	<span class="menu">Move Chains | Before</span>
    </td>
    <td valign="top">
	Move the selected chains before a given chain.  You will
	be prompted to select the chain before which the selected chains will
	be moved.
    </td>
    </tr>
    <tr>
    <td valign="top">
	<span class="menu">Sort Chains | by Name</span>
    </td>
    <td valign="top">
	Order the chains alphabetically by chain name.
	The name can be displayed in the Sequence Editor using
	<span class="menu">SE | Display | Compound Name</span>.
    </td>
    </tr>
    <tr>
    <td valign="top">
	<span class="menu">Sort Chains | by Tag</span>
    </td>
    <td valign="top">
	Order the chains alphabetically by chain tag.
	Chain tags can be edited using
	<span class="menu">SE | Edit | Chain Tag</span>
	or with the Chain and Selected Chains popup menus.
    </td>
    </tr>
</table>
</center>
</p>

<!--
<p><b>Note:</b> When you reorder chains in the Sequence Editor display,
you cause the actual chain numbering to be reordered as well.</p>
-->

<a name="Joining_and_Splitting_Chains"></a><a name="JoiningAndSplitting"></a>
<h3>Joining and Splitting Chains</h3>

<p>To join two or more chains:</p>
<ol>
<li>Select the chains to join.</li>
<li>Choose <b><span class="nobr">SE | Edit | Join Chains</span></b> or choose
<b>Join Chains</b> from the Selected Chains popup.</li>
</ol>

<p>The residues of the selected chains are re-parented to the lowest numbered
chain. <!--If the last residue of one chain and the first residue of
the next belong to the MOE standard residue database, the two residues are
automatically polymerized.--></p>

<p>To split a chain at a single residue, open the
residue popup over the residue and select <b>Split Chain</b>.</p>

<p>To split the system at several residues:
<ol>
<li>Select the residues that will mark the beginning of the new chains.</li>
<li>Choose <span class="menu">SE | Edit | Split Chains</span>.
The selected residues each become the first residue of a new chain.  Each
new chain appears directly below the chain from which it was split.</li>

</ol>


<a name="Deleting_Chains_and_Residues"></a><a name="DeletingChainsAndResidues"></a>
<h3>Deleting Chains and Residues</h3>

To delete a single chain:
<ol>
<li>Position the cursor over the number of the chain to delete.</li>
<li>Display the Chain popup and choose <b>Delete this Chain</b>.</li>

</ol>

<p>To delete two or more chains at a time:
<ol>
<li>Select the chains to delete.</li>
<li>Display the Chain or Selected Chains popup and choose <b>Delete Selected Chains</b>.
The same functionality is found in
<span class="menu">SE | Edit | Delete Selected Chains</span>.</li>

</ol>

<p>Deleting residues is accomplished similarly.  To delete a single
residue, you can use the <b>Delete this Residue</b> option from the Residue
popup.  To delete one or more residues, select the residues to
delete, then use either the <b>Delete Selected Residues</b> option in the
either the Selected Residues or Residue popup, or the 
<span class="menu">SE | Edit | Delete Selected Residues</span> menu command.
To delete all residues in a residue column, use the <b>Delete this Column</b>
option in the Residue Column popup.
After the residue column is deleted, the alignment is &quot;packed&quot;
(all residues
to the right of the deleted column are moved left by one column to
close the gap).
</p>

<p>When deleting chains or residues, MOE displays the total number of
chains, residues and atoms that will be deleted, and asks you to confirm
or cancel the operation.  You also have the option to &quot;pack&quot; the
alignment; i.e. eliminate empty residue columns that may
result from the deletion operation.</p>

<a name="Coloring_Molecular_Objects"></a><a name="ColoringMolecularObjects"></a><h2>Coloring Molecular Objects</h2>

<a name="Coloring_Residues"></a><a name="ColoringResidues"></a>
<h3>Coloring Residues</h3>

<p>Residues can be colored in the Sequence Editor using one of the
following:

<ul>

<li>The <b>Residues | Color</b> menu item from the Chain and Selected Chains
popup menus in the Sequence Editor

<li>The <b>Color</b> menu item from the Residue, Selected Residues, and
Residue Column popup menus in the Sequence Editor

<li>The Color Residues panel
<span class="nobr">(<b>SE | Display | Color Residues</b>)</span>

</ul>

<p>The Sequence Editor popup menus allow you to color individual or
multiple residues.  The colors are selected from a pre-defined palette which
is accessible as a submenu of colors.

<p>The Color Residues panel defines a group of color-coded amino acid
equivalence classes. Choosing a color scheme and pressing Apply will color
all the residues currently in MOE.  When residues are created or mutated
they are <i>not</i> automatically colored.  In this case, you must press
Apply in the panel to update the residue coloring.

<p align="center"><img src="se/colormap.png" alt="Color Residues"></p>

<p>The <a href="mainwin.htm#Color">MOE Window</a> includes the rendering option 
<span class="menu">MOE | Render | Atoms | Color: Residue</span> to
automatically color atoms according to the color assigned to their parent
residue.

<a name="ColorByChainTag"></a>
<h3>Coloring Chains by Chain Number and Tag</h3>

<p>
Chain colors can be assigned according to chain number and chain tag using the
<span class="menu">SE | Display | Color Chains</span>
menu commands.
The chain color is the color used to display the chain name in
the Sequence Editor.  Atoms can be colored by chain color using
the <span class="menu">MOE | Render | Atoms | Color: Chain</span> command.

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Display | Color Chains</th></tr>
	<tr>
	<th>Command</th>
	<th>Action</th>
	</tr>
    <tr>
    <td valign="top"><span class="menu">By Number</span></td>
    <td valign="top">
	Color each chain a unique color.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">By Tag</span></td>
    <td valign="top">
	Color chains of a common tag by the same color.
	Chain tags can be edited using
	<span class="menu">SE | Edit | Set Chain Tag</span> or
	similar commands in the Chain and Selected Chains popup menus.
    </td>
    </tr>
</table>
</center>
</p>

<a name="ColoringSelectedChains"></a>
<a name="Coloring_Atoms_of_Selected_Chains_in_the_MOE_Window"></a>
<h3>Coloring Atoms of Selected Chains in the MOE Window</h3>

From the Selection menu, choose the 
<span class="menu">SE | Selection | Chains | Color Separate Selected Chains</span>
command. This command is greyed out if no chains are selected. Atoms of
selected chains appear in one color in the MOE Window, whereas atoms
of all other chains are a different color.  
(To otherwise modify the coloring scheme of atoms in the MOE
Window, use the <span class="menu">MOE | Render | Atoms</span> panel.)

<a name="Coloring_the_Backbone"></a><a name="ColoringTheBackbone"></a>
<h3>Coloring the Backbone</h3>

<p>The display of the backbone ribbon in the MOE Window can be controlled
from within the Sequence Editor.  Each of the Sequence Editor popup menus
contains a submenu called <b>Ribbon</b> that allows you to render the
backbone ribbon of specific residues in various styles or to change the color of
that portion of the backbone to one of the pre-defined colors found in the
<span class="menu">Ribbon</span> submenu.  Note that changing the color
will not automatically turn on the backbone display, so to see the color,
first draw the backbone.  To see the different backbone drawing styles,
see <a href="mainwin.htm#RenderingBackbone">Rendering the Backbone</a>.

<a name="Homology_Searching"></a>
<a name="HomologySearching"></a>
<a name="Homology_Modeling"></a>
<h2>Homology Modeling</h2>

<p>The Homology menu provides access to MOE's suite of homology modeling
applications.  Most of the menu commands apply specifically to
proteins, with the exception of Reset Alignment and Pack Alignment,
both of which can operate on any chain.

<!--
<ul>

<li><b><a href="../../proteins/prosearch.htm">PDB Search...</a></b><br />
Open the MOE-SearchPDB panel, for performing a homology search.</li>

<li><a name="ResetAlignment"></a>
<b>Reset Alignment</b><br />
Reset the alignment positions of residues of selected chains. Residues
become positioned according to cardinal numbering. This command is
greyed out if no chain is selected.</li>

<li><a name="PackAlignment"></a>
<b>Pack Alignment</b><br />
Eliminate empty residue columns from the current alignment.  Such
empty columns may arise, for example, when chains or residues are deleted.</li>

<li><b><a href="../../proteins/align.htm">Align...</a></b><br />
Open the MOE-Align panel, for performing multiple sequence
and structure alignment.</li>

<li><b><a href="../../proteins/prosup.htm">Superpose...</a></b><br />
Open the MOE-ProSuperpose panel, for performing protein
structure superposition.</li>

<li><b><a href="../../proteins/procore.htm">Consensus...</a></b><br />
Open the Protein Consensus panel, for performing an
analysis of the structurally conserved core of a group of aligned proteins.</li>

<li><b><a href="../../proteins/promodel.htm">Homology Model...</a></b><br />
Open the MOE-Homology panel, for building a 3D protein model.</li>

<li><b><a href="../../proteins/antibody.htm">Antibody Modeler...</a></b><br />
Open the Antibody Modeler panel, for building a 3D protein model based on 
antibody templates.</li>


<li><a name="PredictSecondaryStructure"></a><b>Predict Secondary Structure</b><br />

Assign to each amino acid residue of the selected chains its most probable
secondary structure label. A sequence-based structure prediction is performed
on the selected chains, and color-coded bars are drawn below amino acid
residues where applicable. Choosing this command automatically turns on the
Predicted Secondary Structure option in the Display menu.  Note that the
command is greyed out if no chains are selected.  The display of results is
controlled using <span class="menu">Display | Predicted Secondary Structure</span>.
</li>
</ul>
-->

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Homology</th></tr>
	<tr>
	<th>Command</th>
	<th>Description</th>
	</tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/prosearch.htm">PDB Search</a>
    </span></td>
    <td valign="top">
	Open the MOE-SearchPDB panel, for performing a homology search.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/kinase.htm">Kinase Search</a>
    </span></td>
    <td valign="top">
	Open the Kinase Search panel, for browsing and searching
	in the Kinome Database.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a name="ResetAlignment"></a>Reset Alignment
    </span></td>
    <td valign="top">
	Reset the alignment positions of residues of selected chains. Residues
	become positioned according to cardinal numbering. This command is
	greyed out if no chain is selected.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a name="PackAlignment"></a>Pack Alignment
    </span></td>
    <td valign="top">
	Eliminate empty residue columns from the current alignment.  Such
	empty columns may arise, for example, when chains or residues
	are deleted.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/align.htm">Align</a>
    </span></td>
    <td valign="top">
	Open the Protein Align panel, for performing multiple sequence
	and structure alignment.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/prosup.htm">Superpose</a>
    </span></td>
    <td valign="top">
	Open the Protein Superpose panel, for performing protein
	structure superposition.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/procore.htm">Consensus</a>
    </span></td>
    <td valign="top">
	Open the Protein Consensus panel, for performing an
	analysis of the structurally conserved core of a group
	of aligned proteins.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/promodel.htm">Homology Model</a>
    </span></td>
    <td valign="top">
	Open the Homology Model panel, for building a 3D protein model.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/antibody.htm">Antibody Modeler</a>
    </span></td>
    <td valign="top">
	Open the Antibody Modeler panel, for building a 3D protein
	model based on antibody templates.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a name="PredictSecondaryStructure"></a>
	<a href="../../proteins/propred.htm">Predict Secondary Structure</a>
    </span></td>
    <td valign="top">
	Assign to each amino acid residue of the selected chains its most
	probable secondary structure label. A sequence-based structure
	prediction is performed on the selected chains, and color-coded
	bars are drawn below amino acid residues where applicable.
	Choosing this command automatically turns on the
	Predicted Secondary Structure option in the Display menu.
	Note that the command is greyed out if no chains are selected.
	The display of results is controlled using 
	<span class="menu">Display | Predicted Secondary Structure</span>.
    </td>
    </tr>
</table>
</center>
</p>

<a name="Measuring_Stereochemistry"></a>
<h2>
<a name="Measuring">Measuring Residue Properties and Stereochemistry</a>
</h2>

<p>The Measure menu offers various residue property and
stereochemistry evaluation tools:</p>
<ul>
<li>Residue scalar plots displayed in the Sequence Editor</li>
<li>Protein stereochemistry evaluation tools and plots</li>
</ul>

<p>Residue scalar plots measure and display residue
attributes such as:</p>
<ul>
<li>Hydrophobicity</li>
<li>Residue charge</li>
<li>Solvent accessibility</li>
<li>Predicted solvent accessibility</li>
<li>Predicted secondary structure</li>
</ul>
<p>MOE displays
each numerical attribute as a <i>scalar plot</i>, which is represented by
a bar graph aligned with residue sequences. Up to three scalar values --
or attributes -- may be plotted at the same time.
Plotted values range from -1 to 1, with a horizontal black line marking
0. Plotted values exceeding this range are "clamped"; i.e. MOE
clips the graph.  Colors assigned to the plots can be configured in the
<a href="../../appendix/moeconfig.htm#SequenceEditor">Sequence Editor Colors</a>
page of the Configuration Options panel.</p>

<p>
<center>
<img src="se/se_scalar.png" alt="SE Scalar Plots" />
</center>
</p>

<p>The Plot footer contains checkboxes corresponding to each of the
scalar plots.  Toggling one of these hides/displays the corresponding plot.
When a particular residue attribute is calculated and assigned to a given
plot, the name of the corresponding checkbox in the Plot footer changes to
reflect the attribute that is plotted.  For example, in the figure above,
Hydrophobicity is assigned to Scalar 1, Charge to Scalar 2,
and Accessible Surface to Scalar 3.
</p>

<p>The following are the predefined attributes that can be plotted.  To
plot any one of them, select the appropriate command from the Measure menu
and specify which scalar plot to use: Scalar1, Scalar2, or Scalar3.</p>

<!--
<ul>

<li><a name="Hydrophobicity"></a>
<b>Hydrophobicity</b><br />
Plot the amino acid hydrophobicity values of
Eisenberg <i>et al.</i> [Eisenberg 1984]. </li>

<li><a name="ResidueCharge"></a>
<b>Residue Charge</b><br />
Plot the charge of a residue, which is the sum of the formal charges
of its constituent atoms.</li>

<li><a name="SolventAcc"></a>
<b>Solvent Accessibility</b><br />
Calculate the solvent accessible
surface areas of all amino acid residues in the system and plot the
ratios of these values to pre-calculated values for each amino acid
type in a <tt>GLY-X-GLY</tt> tri-peptide in extended conformation.
The following reference values are used:</li>

<pre>
ALA:119.77&nbsp;ARG:242.85&nbsp;ASN:161.57&nbsp;ASP:155.38&nbsp;CYS:145.01&nbsp;
GLN:172.92&nbsp;GLU:174.09&nbsp;GLY: 86.63&nbsp;HIS:193.65&nbsp;ILE:187.39&nbsp;
LEU:189.51&nbsp;LYS:208.26&nbsp;MET:195.41&nbsp;PHE:213.44&nbsp;PRO:145.71&nbsp;
SER:130.84&nbsp;THR:154.64&nbsp;TRP:254.40&nbsp;TYR:229.11&nbsp;VAL:167.65&nbsp;
</pre>
</li>
<li><b><a href="../../proteins/propred.htm">Predicted Solvent Accessibility</a></b><br />
Plot the probabilities that residues are buried based
on the residue sequence of each chain.</li>

<li><b><a href="../../proteins/propred.htm">
Predicted Secondary Structure</a></b><br />
Plot the probabilities that residues have a
helix or strand secondary structure based
on the residue sequence of each chain. The Predicted Secondary Structure
command populates all three scalar plots.</li>

</ul>
-->

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Measure</th></tr>
	<tr>
	<th>Command</th>
	<th>Description</th>
	</tr>
    <tr>
    <td valign="top"><span class="menu">
	<a name="Hydrophobicity"></a>Hydrophobicity
    </span></td>
    <td valign="top">
	Plot the amino acid hydrophobicity values of
	Eisenberg <i>et al.</i> [Eisenberg 1984].
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a name="ResidueCharge"></a>Residue Charge
    </span></td>
    <td valign="top">
	Plot the charge of a residue, which is the sum of the formal charges
	of its constituent atoms.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a name="SolventAcc"></a>Solvent Accessibility
    </span></td>
    <td valign="top">
	Calculate the solvent accessible
	surface areas of all amino acid residues in the system and plot the
	ratios of these values to pre-calculated values for each amino acid
	type in a <tt>GLY-X-GLY</tt> tri-peptide in extended conformation.
	The following reference values are used:

<pre>
ALA:119.77&nbsp;ARG:242.85&nbsp;ASN:161.57&nbsp;ASP:155.38&nbsp;CYS:145.01&nbsp;
GLN:172.92&nbsp;GLU:174.09&nbsp;GLY: 86.63&nbsp;HIS:193.65&nbsp;ILE:187.39&nbsp;
LEU:189.51&nbsp;LYS:208.26&nbsp;MET:195.41&nbsp;PHE:213.44&nbsp;PRO:145.71&nbsp;
SER:130.84&nbsp;THR:154.64&nbsp;TRP:254.40&nbsp;TYR:229.11&nbsp;VAL:167.65&nbsp;
</pre>

	The midline represents exactly 0.25 of the standard exposed
	surface area for each residue type.  Bars above the midline
	(pointing up) indicate buried residues, below the midline,
	exposed residues.

    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/propred.htm">Predicted Solvent Accessibility</a>
    </span></td>
    <td valign="top">
	Plot the probabilities that residues are buried based
	on the residue sequence of each chain.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/propred.htm">Predicted Secondary Structure</a>
    </span></td>
    <td valign="top">
	Plot the probabilities that residues have a
	helix or strand secondary structure based
	on the residue sequence of each chain. The Predicted Secondary Structure
	command populates all three scalar plots.
    </td>
    </tr>
</table>
</center>
</p>

<p>The remaining commands in the Measure menu provide information on
protein conformation:</p>

<!--
<ul>

<li><a href="../../proteins/procontacts.htm"><b>Protein Contacts</b></a><br />
Opens the Protein Contacts panel, for analyzing stabilizing contacts.</li>


<li><a href="../../apps/progeom.htm"><b>Protein Geometry</b></a><br />
Opens the Protein Geometry panel, for evaluating protein
stereochemical quality.</li>

</ul>
-->

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Measure</th></tr>
	<tr>
	<th>Command</th>
	<th>Description</th>
	</tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../proteins/procontacts.htm">Protein Contacts</a>
    </span></td>
    <td valign="top">
	Opens the Protein Contacts panel, for analyzing stabilizing contacts.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="../../apps/progeom.htm">Protein Geometry</a>
    </span></td>
    <td valign="top">
	Opens the Protein Geometry panel, for evaluating protein
	stereochemical quality.
    </td>
    </tr>
</table>
</center>
</p>

<a name="Modifying_the_Display"></a><a name="ModifyingTheDisplay"></a><h2>Modifying the Display</h2>

<p>The Display menu controls various
aspects of the display within the residue area of the Sequence Editor. </p>

<p align="center"><img src="se/sedisplay.png" alt="Display Menu in Sequence Editor"></p>

<!--
<ul>
<li><b>Compound Name</b><br />
Displays the name and header of the chain. For Protein Data Bank<sup>TM</sup>
files, MOE assigns an accession code and chain letter to the chain name,
as well as a molecule ID to the chain header.</li>

<li><b>Actual Secondary Structure</b><br />
Displays the DSSP secondary structure assignments [Kabsch 1983]
as color-coded bars above the amino acid residues to which they apply.
The standard default coloring corresponds to the coloring of the backbone
ribbon in the MOE Window; by default helix is red, strand is yellow,
turns are blue.</li>

<li><b>Predicted Secondary Structure</b><br />
Displays the predicted secondary structure as calculated from the
<a href="#PredictSecondaryStructure">
<span class="menu">Homology | Predict Secondary Structure</span></a> command.  The
secondary structure is shown as color-coded bars below the residues.</li>


<li><b>Hydrogen Bonds</b><br />
Displays the protein backbone hydrogen bonds used by 
the DSSP secondary structure assignment algorithm to identify 3-turns,
4-turns and 5-turns. Bonds are drawn as arcs
connecting the donor and acceptor residues.
</li>

<li><b>Single Letter Residues</b><br />
Displays single-letter abbreviations of amino and nucleic acids (nucleic acids 
are shown in lower case). 
Non-standard amino acids and nucleic acids are displayed using an
<tt>X</tt>. Other residues, such as waters and moieties, are displayed as
empty boxes. In single-letter mode, dots are used to represent alignment
gaps.  For the complete list of residue abbreviations, see
<a href="../../proteins/protool.htm">Building Proteins and DNA</a>.</li>

<li><b>Residue Indices</b><br />
Displays the ordinal numbers of residues within chains. Choosing this
option precludes the display of residue UID (described below). </li>


<li><b>Residue UID</b><br />
Displays residue UID numbers.
Choosing this option precludes the display of residue indices. </li>

<li><a href="#ColoringResidues"><b>Color Residues</b></a><br />
Opens the Color Residues panel, for coloring amino acid residues by class.</li>

</ul>
-->

<p>
<center>
<table class="titlebar" width="95%">
    <tr><th colspan="2">SE | Display</th></tr>
    <tr>
	<th>Command</th>
	<th>Description</th>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	Compound Name
    </span></td>
    <td valign="top">
	Display the name and header of the chain. For
	Protein Data Bank<sup>TM</sup>
	files, MOE assigns an accession code and chain letter to the chain name,
	as well as a molecule ID to the chain header.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	Chain Tag
    </span></td>
    <td valign="top">
	Display the chain tag, enclosed in square brackets.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	Actual Secondary Structure
    </span></td>
    <td valign="top">
	Display the DSSP secondary structure assignments [Kabsch 1983]
	as color-coded bars above the amino acid residues to which they apply.
	The standard default coloring corresponds to the coloring of the
	backbone
	ribbon in the MOE Window; by default, helix is red, strand is yellow,
	turns are blue.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	Predicted Secondary Structure
    </span></td>
    <td valign="top">
	Display the predicted secondary structure as calculated from the
	<a href="#PredictSecondaryStructure">
	<span class="menu">Homology | Predict Secondary Structure</span></a>
	command.  The secondary
	structure is shown as color-coded bars below the residues.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	Hydrogen Bonds
    </span></td>
    <td valign="top">
	Display the protein backbone hydrogen bonds used by 
	the DSSP secondary structure assignment algorithm to
	identify 3-turns,
	4-turns and 5-turns. Bonds are drawn as arcs
	connecting the donor and acceptor residues.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	Single Letter Residues
    </span></td>
    <td valign="top">
	Display single-letter abbreviations of amino and nucleic acids
	(nucleic acids are shown in lower case).  Non-standard amino
	acids and nucleic acids are displayed using an <tt>X</tt>. Other
	residues, such as waters and moieties, are displayed as empty
	boxes. In single-letter mode, dots are used to represent alignment
	gaps.  For the complete list of residue abbreviations, see
	<a href="../../proteins/protool.htm">Building Proteins and DNA</a>.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	Residue Indices
    </span></td>
    <td valign="top">
	Display the ordinal numbers of residues within chains. Choosing this
	option precludes the display of residue UID (described below).
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	Residue UID
    </span></td>
    <td valign="top">
	Display residue UID numbers.
	Choosing this option precludes the display of residue indices.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="#ColoringResidues">Color Residues</a>
    </span></td>
    <td valign="top">
	Open the Color Residues panel, for
	coloring amino acid residues by class.
    </td>
    </tr>
    <tr>
    <td valign="top"><span class="menu">
	<a href="#ColorByChainTag">Color Chains</a>
    </span></td>
    <td valign="top">
	Assign chain colors.  If <span class="menu">By Number</span>
	is chosen, then each chain is assigned a unique chain color.
	If <span class="menu">By Tag</span> is chosen, then chains
	having the same tag are assigned the same color.
	Chain colors can also be set using the
	<span class="menu">Color Chain</span> command in the Chains
	and Selected Chains popup menus.
    </td>
    </tr>
</table>
</center>
</p>

<p>The default values of all the above options can be configured in the 
<a href="../../appendix/moeconfig.htm">Configuration Options panel</a>. </p>


<a name="SVL_Commands"></a><a name="SVL"></a><h2>SVL Commands</h2>

<a href="../fcnref/chain.htm">Chain Functions</a><br />
<a href="../../proteins/fcnref/intropro.html">Protein Functions</a><br />
<a href="../fcnref/residue.htm">Residue Functions</a>

<a name="References"></a><h2>References</h2>

<table>
<tr>
<td valign="top">
[Eisenberg&nbsp;1984] 

<td valign="top">
Eisenberg, D., Schwarz, E., Komaromy, M., and Wall, R.;
<i>J.Mol.Biol. 179(1)</i> (<b>1984</b>) 125&ndash;142.

<tr>
<td valign="top">
[Kabsch&nbsp;1983] 

<td valign="top">
Kabsch, W. and Sander, C.;
Dictionary of Protein Secondary Structures: Pattern Recognition of
Hydrogen-Bonded and Geometrical Features; <i>Biopolymers 22</i>
(<b>1983</b>) 88&ndash;47.

</table>


<a name="See_Also"></a><h2>See Also</h2>

<a href="select.htm#Residue_Selector">Residue Selector</a><br />
<a href="../../proteins/protool.htm#Building_Proteins_and_DNA">Building Proteins and DNA</a><br />
<a href="../../proteins/align.htm#Constraining_and_Unconstraining_Residues">Constraining and Unconstraining Residues</a><br />
<a href="mainwin.htm#MOE_Window">MOE Window</a><br />


<p><a href="../../index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
