<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<!--
!!   	antibody.htm		Antibody modeler
!!
!!   	26-oct-2009 (ac) edits
!!      07-oct-2009 (jm) update with regards to changes in the application
!!   	16-oct-2008 (al) edits
!!   	10-oct-2008 (jm) created
!!-->

<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../manstyle.css" />



<title>Antibody Modeler</title>

<meta panel Antibody Modeler>
<meta panel Antibody Builder>
<meta panel Antibody Modeler Settings>

</head>

<body bgcolor="#ffffff">
<a href="../index.htm"><img src="../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>Antibody Modeler</h2>
<hr noshade="noshade" />
<ul>

<li><a href="#Overview">Overview</a>
<li><a href="#Methodology">Methodology</a>
    <ul>
    <li><a href="#MethodWorkFlow">Antibody Homology Modeling Workflow</a>
    <li><a href="#MethodFabDB">Antibody Database</a>
    <li><a href="#MethodFabRef">Reference Sets</a>
    <li><a href="#MethodNumbering">Numbering Scheme</a>
    <li><a href="#MethodSearch">Template Search</a>
    <li><a href="#MethodEval">Template Evaluation</a>
    <li><a href="#MethodGrafting">CDR Loop Grafting</a>
    <li><a href="#MethodCDRLoops">CDR Loop Assignment</a>
      <ul>
        <li>
          <a href="#MethodCDRBounds">
            Analysis of Canonical CDR Loop Conformations
          </a>
        </li>
        <li>
          <a href="#MethodCDRAssign">Representative CDR Loop Collection</a>
        </li>
      </ul>
    </ul>
<li><a href="#Run">Running the Application</a>
  <ul>
    <li><a href="#RunSearch">Antibody Template Search</a>
        <ul>
        <li><a href="#RunResults">Search Results</a>
           <ul>
            <li><a href="#RunVLVH">VL + VH</a>
            <li><a href="#RunVL">VL</a>
            <li><a href="#RunVH">VH</a>
	    <li><a href="#RunReport">Search Report</a>
           </ul>
        <li><a href="#RunAlignment">Alignments</a>
	    <ul>
	    <li><a href="#RunAlignmentViews">Alignment Views</a>
	    <li><a href="#RunPrint">Print</a>
	    </ul>
        </ul>
    <li><a href="#RunBuild">Building Antibody Models</a>
	<ul>
	    <li><a href="#RunAssignLoops">Assigning CDR Loops</a>
	    <li><a href="#RunBuildSettings">Antibody Builder</a>
	</ul>
    <li><a href="#Settings">Antibody Modeler Settings</a>
    <li><a href="#RunSchemes">Numbering</a>
    <li><a href="#FabExample" class="example">Example: Creating an
	Antibody Model</a>
  </ul>
<li><a href="#FabDBUpdate">Compiling and Updating the Antibody Database</a>
  <ul>
    <li><a href="#MethodFabDB">Antibody Database</a>
    <li><a href="#MethodFabFam">Antibody Family File</a>
    <li><a href="#MethodUpdate">Updating the Antibody Database</a>
  </ul>
<li><a href="#SeeAlso">See Also</a>
<li><a href="#References">References</a>

</ul>

<!-- ============================== OVERVIEW ============================== -->

<p><a name="Overview"></a><h2>Overview</h2>

<p>
Antibodies are heavy (~150 kDa) globular plasma proteins also known as
immunoglobulins. They are part of a large interconnected network of cells and
molecules whose primary function is to eliminate pathogens from the body.
The hallmark feature of antibodies is their ability to bind to an almost
unlimited collection of targets with remarkably high affinity. The binding mode
of antibodies to antigens is similar to that of enzyme-substrate interactions.
Five different classes of
immunoglobulins are found in serum (A, M, D, E and G) that differ in certain
physical aspects and specific roles in immune response [Morea 2000].

<p>
Despite differences in biological function and vast diversity in antigen
binding, antibodies have a common structural composition. They are "Y" shaped
molecules composed of two heterodimers with
each set containing a heavy chain (V<sub>H</sub>) and light chain (V<sub>L</sub>) linked
by conserved disulfide bridges. The V<sub>L</sub> and V<sub>H</sub>  chains themselves are composed
of beta-barrel shaped domains and, based on protein sequence variability,
termed variable
or constant domains. The light chains harbor one variable (V<sub>L</sub>)
and one constant (C<sub>L</sub>) domain while the heavy chains contain
one variable
(V<sub>H</sub>) and three or four constant domains (C<sub>H1</sub>,
C<sub>H2</sub>, C<sub>H3</sub>, C<sub>H4</sub>) depending on the heavy-chain
isotype. In most antibodies,
binding to an antigen is facilitated primarily by six
loops, three from the V<sub>L</sub>, termed L1, L2 and L3,
and
three from the V<sub>H</sub> domain, termed H1, H2 and H3. The antigen
binding loops together form the Complementarity Determining Regions (CDR)
originally discovered as a genetic hot-spot for extreme variability within the
domain.
The regions outside the antigen binding loops are termed Framework (FR)
regions and
- in comparison to CDR regions - are structurally fairly well conserved
[Morea 2000].

<p>
<div align=center>
<img src="antibody/stru-2.jpg"  alt="Structural Composition of Antibodies">
<br>&nbsp;<br>
<b>Figure 1:</b> <i>Structural Composition of Antibodies</i>
</div>
<p>
Their modular composition combined with high target specificity make
antibodies particularly attractive for transformation into real
drugs. Recent efforts in applying antibodies in immunotherapy have proven
successful in the treatment of rheumatoid arthritis, multiple sclerosis and
various cancers [Taylor&nbsp;2001, Hudson&nbsp;2003]. However, the
immunogenicity of the applied xenogeneic
antibodies has also been shown to compromise drug efficacy.
Therefore, protein engineering has become increasingly important in generating
therapeutically effective antibodies. By altering or composing specific
domains, it is possible to change the affinity, the effector
functions, half-life, and the immunogenicity of
antibodies. In addition, domains from different antibodies can be swapped
and/or linked to other molecules to create molecules with new properties.
antibody homology modeling techniques have often been applied in order to
achieve the above mentioned goals [Morea&nbsp;2000].

<p>
The MOE Antibody Homology Modeling application offers a collection of procedures
as well as an interface to meet the demands of effective antibody homology
modeling. The
application has flexible components allowing the integration of
various work-flows associated with this particular type of modeling.
The routines account for the particular structural composition
of antibodies when searching for template candidates and composing templates.
As a result, models may be generated based on templates containing framework
and CDR loops from different sources (grafting) composed as dimers or as single
chains. A knowledge-based approach is applied with an underlying
database of antibody structures currently in the Protein
Data Bank (PDB), clustered by class, species, subclass and framework sequence
identity.
This database may be enriched with
in-house antibody structures and can be continually updated and
reclustered.

<!-- =========================== Methodology ========================== -->

<p><a name="Methodology"></a><h2>Methodology</h2></p>
<p>
The following sections outline the methods that are implemented in order to run
the Antibody Homology Modeling application.
</p>

<!-- ===================== Homology Modeling Workflow ================== -->

<a name="MethodWorkFlow"></a><h3>Antibody Homology Modeling Workflow</h3>

<p> Homology modeling works on the premise that regions of common function (e.g.
the active site of enzymes) assume a common structural fold. This is in
contrast to antibody homology modeling where only
the region outside the antigen binding loops, the framework region, is expected
to exhibit commonality in the structural folding pattern, and
the regions of interest (the
antigen binding loops) are expected, by their very nature, to be diverse.
In particular, the antigen binding region has to be
modeled as a dimer composed of light and heavy variable domains.  This always
presents a challenge in homology modeling.  Since binding to an antigen is
facilitated by six CDR loops, an awareness of the subdomain composition of the
complex is required throughout the modeling process in order to obtain the
accuracy that is expected for antigen binding site models. Fortunately, five of
the six CDR loops assume a rather small set of main chain conformations and are
therefore termed <b>canonical CDR loops</b>. The exception is the CDR H3 loop,
with a broad range of main chain conformations that do not fit any categories.
When taken together, it is clear that the
commonly applied homology modeling workflow needs
to be restructured for modeling the antigen binding sites of antibodies. In
Figure 2, the essential steps of the modeling procedure are illustrated: </p>

<p>
<div align=center style="padding:15px">
<img src="antibody/workflow.png"  alt="Structural Composition of Antibodies">
<br>
<b>Figure 2:</b> <i>Workflow of Antibody Homology Modeling</i>
</div>
<p>
<ol>

<li><b>Search for Framework templates.</b></li> Every homology modeling
procedure starts with the submission of a query sequence.  In general homology
modeling, the search for template candidates would include a scoring scheme
using <a href="prosearch.htm#Methodology">E-Values</a> and
<a href="prosearch.htm#Methodology">Z-scores</a>
which gives a confidence assessment for the query sequence being associated with
a known structural fold. The model is then built by applying common
homology techniques (Figure 2, workflow diagram in blue).  In the
case of an antigen
binding site, two queries, one for the light and one for the heavy chain,
have to
be submitted. The first task in antibody homology modeling is to find a suitable
framework template which forms the structural core of the designated antigen
binding site model.  This means that suitable template candidates for the VL and
VH chains must be found.  Then, a search must be made
for candidates that are most suitable
for forming the structural core of the antibody binding site complex.
Furthermore, in the event that there are several promising
candidates, an additional scoring scheme is required for selecting
the best template; this should be
based not only on sequence but also on structural fitness, in particular
backbone integrity, which is crucial in homology modeling.


</p><p>

<li><b>Assignment of suitable CDR loops.</b></li> Once a candidate for
F<sub>V</sub>,
the variable region VL-VH framework, is determined,
the CDR loop regions have to be assigned. In principle,
two methods are available for prediction of the coordinates for the respective
CDR loops. Predictions based on the application of a canonical structure method
are based on  identification of loop signatures for the canonical CDR loops L1,
L2, L3, H1 and H2 respectively, and for the non-canonical H3 loop the
predictions are based on a defined set of rules [Morea&nbsp;2000].
Alternatively, application of a knowledge-based method can be used, in which the
CDR loops are assigned based on loop length and similarity from a specific
antibody loop database prior to the grafting of CDRs onto the
F<sub>V</sub> dimer
framework. Local geometries, species and particular conformations can be
accounted for in this process.

</p><p>

<li><b>Building the Antibody Homology Model.</b></li> A hallmark feature of the
MOE Antibody Homology Modeler is that the antigen binding site can be composed
from multiple sources in the designated  Framework and CDR Loop regions (<a
href="#MethodGrafting">loop grafting</a>) in  order to use or preserve distinct
CDR loop conformations for the modeling  procedure. Since the antibody template
preparation steps account for the complexity of the antigen binding site, the
antibody homology models  are then built using the standard homology techniques
with no awareness of the structural composition of an antigen binding site. A
fully grafted
F<sub>V</sub> template as generated requires only sidechain mutations and
sidechain placements in order to finish the antigen binding site model. If no
structure template for a CDR loop is specified then  the implemented loop
building procedure of the homology modeling application will be used to model
the conformation of the CDR loop in question. </ol>

<p> As indicated by the red line in Figure 2, the MOE Antibody Modeler is based
on a knowledge-based method, since a rich collection of framework and CDR loop
structures is available  in the current PDB that can be harvested in order to
successfully conduct this type of modeling. </p><p>

<a name="MethodFabDB"></a><h3>Antibody Database</h3>

The central building block for knowledge-based antibody homology modeling is
the  antibody structure database, which constitutes the  information pool of the
antibody homology modeling procedures. This database is compiled by applying a
fully automated protocol (<a href="#FabDBUpdate">Compiling and Updating
the Antibody Database</a>).
The system is designed to continuously scan the Protein Data Bank
(PDB) and potentially available in-house databases for new antibody structures
thus continuously enhancing the knowledge base. However, not all stored antibody
structures are suitable for homology modeling purposes because of issues with
respect to structural quality. In addition to the 3D coordinates of the
structures, the database also stores data regarding the structural fitness of an
entry.

<br> The antibody chains are grouped according to class and species (for
V<sub>L</sub> chains also subclass) and further clustered at a resolution of
0.85 &Aring; with respect to CA atoms superposition of framework residues. The
alignments of the resulting clusters are stored in a antibody family file
(<tt>fab.fam</tt>). The antibody structures are kept in <tt>fab.mdb</tt>  (<a
href="#FabDBUpdate">Compiling and Updating the Antibody Database</a>). The
current antibody collection (MOE 2009) is composed of 1,531 V<sub>L</sub>  and
1,488 V<sub>H</sub> chains.

<!--  ==================== Reference Sets ================================  -->

<a name="MethodFabRef"></a><h3>Reference Sets</h3>

<p> A crucial component for compiling an antibody database and for working with
antibody structures in general is a method that allows detection and
specification of immunoglobulin regions involved in antigen binding. In MOE,
this task is facilitated by use of a defined set of either light or heavy chain
structures. The reference sets are expected to fulfill the following
requirements: </p>

<ol>
<li>Identification of a given protein chain as a F<sub>V</sub> variable domain.
<li>Classification of the identified antibody chain as either V<sub>L</sub> or V<sub>H</sub>.
<li>Assignment of CDR loop regions within the domain.
</ol>

<p> In addition, the reference sets should ideally be kept small in order to
speed up calculations while still covering the  full spectrum of framework
diversity within a given class.

<p> The heavy and light chain reference sets in MOE are based on published
reference collections [Martin&nbsp;2002] that have not been updated since. These
sets were enriched with antibody structures submitted to the PDB since 2001. No
restrictions with respect to the species or source were applied. The structures
were aligned using <a href="align.htm">MOE-Align</a> and then clustered to
obtain the maximum spread with respect to framework sequence identity.

<p> The current antibody VL reference set comprises 101 antibody light
chain structures at a maximum of 85% sequence identity in the framework
region. The antibody VH reference set comprises 85 antibody heavy
chain structures at a
maximum framework sequence identity of 75%. Only the variable domains of the
dimer F<sub>V</sub> complex &ndash;
usually eight amino acids in the post-CDR3 loop
region &ndash; are considered in the reference sets.

<p> The high selectivity and sensitivity of the reference collections for
antibody chains is shown in Figure 3.  The results
were produced by scanning the entire Protein Data Bank
for immunoglobulins and are shown as a correlation plot between both
V<sub>L</sub> and V<sub>H</sub> reference sets (only chains identified as
antibodies are shown).  Despite the high agreement (up to 50%) with respect to
sequence identity in the framework region to either class, it is possible to
classify each antibody chain indisputably as either  V<sub>L</sub> or
V<sub>H</sub>. The antibodies shown in the top right corner are the exception as
their high identity score to both reference sets makes a clear distinction
impossible. Analysis of these proteins revealed that they are engineered
antibodies with linked V<sub>L</sub> and V<sub>H</sub> domains; such
linked domains are called <i>single-chain variable fragments</i>,
abbreviated as <i>scFv</i>s.

<p>
<div align=center style="padding:15px">
<img src="antibody/cplot.png"  alt="Antibody Classification">
<br>
<b>Figure 3:</b> <i>Antibody Classification using MOE Antibody Reference Sets</i>
</div>


<!-- ======================= Numbering Scheme ====================== -->

<a name="MethodNumbering"></a><h3>Numbering Scheme</h3>

<p>
When working with antibodies, it is convenient to have a numbering scheme
that provides a standard way of identifying specific locations within an
antibody sequence.  The Antibody Modeler implements a variety of
numbering schemes: Kabat, Chothia, Aho, IMGT, and MOE (CCG).
</p>

<p>
The most commonly used schemes, now universally accepted by immunologists,
were introduced by Kabat [Kabat&nbsp;1970] and Chothia [Chothia&nbsp;1987].
The Kabat numbering scheme is solely based on sequence
alignments, derived from the analysis of a limited set of
sequences; the specifications are very
rigid and can result in insertions being introduced
on residues that are in fact structurally conserved.
Chothia <i>et al.</i>
performed CDR loop conformation analysis and found a mismatch of
the insertion sites in the L1 and H1 loops when using the Kabat numbering.
To correct for this,
the Chothia numbering system assigns the same number to topologically equivalent
residues in these loops [Chothia&nbsp;1987].

<p>
Alternative numbering schemes have since been introduced by Lefranc
[Lefranc&nbsp;1997a]
and by Honegger [Honegger&nbsp;2001].
The IMGT numbering scheme is based on
germline sequences for use with the Immunogenetics (IMGT) database
[Lefranc&nbsp;1997b].
It has the advantage that it unifies numbering across
antibody <i>lambda</i> and <i>kappa</i> light chains, heavy chains and the
various T-cell receptor subclasses. Since the alignments are based on germline
sequences, however, the numbering scheme stops in the CDR3 region. This
shortcoming
is addressed by the numbering scheme introduced by Honegger (Aho) which
is otherwise similar in approach.  The Aho
scheme additionally has two locations, as opposed to only one
in the IMGT scheme, at which gaps may be introduced: in CDR1 and
CDR2 [Honegger&nbsp;2001].
</p>

<p>With the exception of Honegger's system, none of the common antibody
numbering schemes facilitates an indisputable numbering of antibody
structures currently found in the PDB.

The MOE numbering scheme (CCG) goes some way towards addressing
this with a flexible system
based on the CDR loop definitions as outlined in
Andrew Martin's Antibody Website [Martin&nbsp;2002].  It combines the loop
definitions of Kabat with the residue numbering of
the Chothia numbering schemes.  Insertions are in accordance with
structural positioning, and loop definitions are particularly suitable for
loop
grafting purposes.
</p>

<p>A detailed illustration of the various numbering schemes is given
in Figure 4.
</p>

<p>
<div align=center style="padding:15px;">
<img src="antibody/schemes.png"  alt="Antibody Numbering Schemes">
<br>
<b>Figure 4:</b> <i>Antibody Numbering Schemes</i>
</div>

<!-- ======================= Template Search ====================== -->

<a name="MethodSearch"></a><h3>Template Search</h3>

<p> Since the structures of antibodies are well known and well
preserved throughout most species, the conventional method for
searching for templates applying E-Values and Z-Scores (see <a
href="prosearch.htm#Methodology">Homology&nbsp;Searching</a> for details)
has to be modified. When used for antibodies, the E-Value/Z-Score will
show a high level of confidence that a target structure is a member of an
immunoglobulin fold family, which is an insufficient result when searching
for templates of an antigen binding site. Hence, a more detailed scoring scheme, one that
can account for the subdomain composition of antigen binding sites, is required.

<p>
The antibody search algorithm calculates scores for each of the
antibody subdomains using two distinct scoring methods. The scores for the
framework regions are calculated as a percent identity within the aligned
sequence. Gaps within the alignment are factored into the calculations.
In contrast, the scores for the CDR loops are calculated as similarity of the
gapless loop sequence with the loop region
of the target. The residue similarity is calculated according to Sonnhammer and
Hollich [Sonnhammer&nbsp;2005]. In Figure 5, the methods underlying the antibody template search are illustrated.
<div align=center style="padding:15px;">
<img src="antibody/templ_search.png"  alt="Antibody Numbering Schemes">
<br>&nbsp;<br>
<b>Figure 5:</b> <i>Antibody Template Search</i>
</div>

<!-- ======================= Template Evaluation ====================== -->

<a name="MethodEval"></a><h3>Template Evaluation</h3>

<p>
In many cases a search for antibody templates will reveal more than one potential candidate.
This is particularly true when looking for a suitable
F<sub>V</sub> dimer framework. A high
sequence similarity score alone does not necessarily result in a suitable
template for homology modeling. In the worst
case, the template candidate might be a CA-trace structure that is not
usable at all for modeling. It is therefore
important to factor in structural aspects when choosing a particular chain as
template, especially when loop grafting is
involved.

<p> The structural assessment of antibody
F<sub>V</sub> chains in MOE has been designed to
emphasize backbone integrity and is based on a scoring scheme that combines
global structure factors (Resolution, R-Value, R<sub>free</sub> ) with local
backbone assessment by inspecting the main chains of the individual Framework
and respective CDR regions separately. The backbone is assessed locally for
correct topology and geometry as well as for appropriate atomic occupancy and
temperature factors of the backbone atoms N, CA, C and O. The resulting scores
are calculated from distribution frequencies within the respective categories
and permit relative structural comparisons within
F<sub>V</sub> dimer regions.

The following components are calculated:

<blockquote>
<table width="90%" cellpadding="3">
  <tr>
    <td valign="baseline" width="20%"><b>Backbone Topology</b></td>
    <td valign="baseline">
        The antibody residues are inspected for
        missing backbone atoms or erroneous backbone bond orders. For each
        of the subdomains the occurrences are counted.
        The score represents the frequency of the occurrence within a
        respective antibody subdomain.
    </td>
  </tr>
  <tr>
    <td valign="baseline"><b>Backbone Geometry</b></td>
    <td valign="baseline">
        The backbone dihedral angles are evaluated
        <i>omega</i> and <i>phi</i>/<i>psi</i> for each residue within the
        subdomains. The outliers are counted and scored based on the
        frequency distribution of geometric
        outliers within a subdomain.
    </td>
  </tr>
  <tr>
    <td valign="baseline"><b>Backbone B-Factors</b></td>
    <td valign="baseline">
        Atomic temperature factors (B-Factors) provide empirical
	estimates of the positional spread of each atom.  The values
        are calculated as the average of backbone atoms N, CA,
        C and O, and are averaged over the respective subdomain.
        In general, B-Factor values lower than 20.0 and higher than 40.0
        indicate some reason for concern [Vriend&nbsp;1990].
        The B-Factor score is calculated from the frequency distribution in the
        antibody collection database.
    </td>
  </tr>
  <tr>
    <td valign="baseline"><b>Backbone Occupancy</b></td>
    <td valign="baseline">
        The occupancy monitors the presence of an atom at its mean position and
        ranges from 0.0 to 1.0, if assigned in the PDB record. The score
        represents the
        average occupancy of backbone atoms N, CA, C and O averaged over the
        entire subdomain.
    </td>
  </tr>
  <tr>
    <td valign="baseline"><b>Resolution</b></td>
    <td valign="baseline">
        Resolution scores are calculated from the frequency distribution of
        the antibody database. The maximum score is assigned for structures
        with a resolution less than 2.0 &Aring;.
    </td>
  </tr>
  <tr>
    <td valign="baseline"><b>R-value/ R<sub>free</sub></b></td>
    <td valign="baseline">
        The agreement of the fit of a model to the diffraction data is given by
        the R-Value as a measurement of the discrepancy between the observed
        (F<sub>o</sub>) and calculated (F<sub>c</sub>) structure factor
        amplitudes [Kleywegt&nbsp;1996]. The R<sub>free</sub> value is
        associated with the accuracy of the atomic model.

	<p>
        The difference between R-value and R<sub>free</sub> is also assessed.
        Large differences often indicate incomplete models or
        overfitting [Kleywegt&nbsp;1996]. Ideally, the difference should
        be small.

	<p>
        The score is calculated as the average of the R-value,
        R<sub>free</sub>, and (R-value - R<sub>free</sub>) scores.
        The individual scores are calculated from the respective frequency
        distributions. In the case that either R-value or R<sub>free</sub>
	is not assigned in the PDB record, the final score is the score
	of the component that is available.
    </td>
  </tr>
</table>
</blockquote>

<p>
The overall structure score of an antibody subdomain is calculated as
a combination of the individual components described above.
Since errors in the backbone topology and/or geometry are much more severe with
respect to homology modeling, they are weighted more heavily.
The overall structure score of an antibody subdomain is calculated as follows:
</p>

<div align=center style="padding:15px">
<img src="antibody/score_function.png" alt="Scoring Function" />
</div>

<p> The scores range from 0 (not suitable for homology modeling purposes) to 100
(ideal protein structure). Values below 50 often indicate structural issues of
the backbone integrity, that should be addressed before assigning a particular
structure as a template for antibody homology modeling. </p>

<!-- ======================= Loop grafting ====================== -->

<a name="MethodGrafting"></a><h3>CDR Loop Grafting</h3>

<p> In loop grafting, multiple templates are combined together, with different
loops originating in different templates. The antibody chains of the designated
framework and  CDR loops are aligned in the framework region. The backbone atoms
N, CA,  C and O of three framework residues on either side of the designated CDR
loop are then optimally superposed onto the corresponding residues of the
framework template. The CDR loop residues of the framework template and the
framework residues of the template CDR   are then deleted and the CDR residues
are bonded to the framework template.

<P> Four rounds of tethered energy minimization are then applied to relieve
strained  geometry and atom clashes in the newly composed antibody subdomain.
The CDR loop residues as well as flanking framework residues of the template are
given flat-bottom tether restraints applying different radii and weights in
each round. All other residues are kept fixed during the refinement procedure.
</p>

<!-- ======================= Loop conformations ====================== -->
<a name="MethodCDRLoops"></a><h3>CDR Loop Assignment</h3>
<a name="MethodCDRBounds"></a><h4>Analysis of Canonical CDR Loop Conformations </h4>

<p> The antibody database offers a rich collection of loop conformations for
each of the six hypervariable CDR loops.  Since the dominant criteria for
selecting a particular CDR loop candidate is the similarity calculation, it is
worthwhile to determine the range of loop conformations observed at a maximum of
similarity thus establishing a measure of precision that can be expected for
antibody models in the CDR loop regions. Since the non-canonical CDR H3 loop is
known to have a high range of conformational freedom the analysis is focused on
the canonical CDR loops, L1, L2, L3 and H1, H2 respectively.

<p>For evaluations of the lower bounds of canonical CDRs, the antibody database
was searched for loop multiplets - loops identical in sequence and length - and
the Root Mean Square Distance (RMSD) of the multiplet cluster was measured by
superimposing the backbone atoms N, CA, C and O of the loop residues using the
<a href="#MethodNumbering">CCG Numbering Scheme</a> as the underlying CDR loop
definition scheme. The results of the analysis is shown in Figure 6.
For each of the canonical CDR loops a range of up to 2 &Aring; is observed. The
median for each category is calculated to be 0.2 to 0.3 &Aring;. These values
represent lower bounds obtained from experimental data.

<p><center>
<table border="0" cellpadding="3" width="400px">
<tr align="center">
  <td bgcolor="#cccccc"><b>CDR</b></td>
  <td bgcolor="#cccccc"><b>Median</b></td>
  <td bgcolor="#cccccc"><b>Min</b></td>
  <td bgcolor="#cccccc"><b>Max</b></td>
  <td bgcolor="#cccccc"><b>Count</b></td>
</tr>
<tr>
  <td valign=top align="center"><b>L1</b></td>
  <td valign=top align="center">0.26</td>
  <td valign=top align="center">0.01</td>
  <td valign=top align="center">2.91</td>
  <td valign=top align="center">201</td>
</tr>
<tr>
  <td valign=top align="center"><b>L2</b></td>
  <td valign=top align="center">0.23</td>
  <td valign=top align="center">0.02</td>
  <td valign=top align="center">1.98</td>
  <td valign=top align="center">146</td>
</tr>
<tr>
  <td valign=top align="center"><b>L3</b></td>
  <td valign=top align="center">0.21</td>
  <td valign=top align="center">0.01</td>
  <td valign=top align="center">1.69</td>
  <td valign=top align="center">213</td>
</tr>
<tr>
  <td valign=top align="center"><b>H1</b></td>
  <td valign=top align="center">0.26</td>
  <td valign=top align="center">0.02</td>
  <td valign=top align="center">2.29</td>
  <td valign=top align="center">198</td>
</tr>
<tr>
  <td valign=top align="center"><b>H2</b></td>
  <td valign=top align="center">0.32</td>
  <td valign=top align="center">0.00</td>
  <td valign=top align="center">1.95</td>
  <td valign=top align="center">227</td>
</tr>
</table>
<br>
<b>Figure 6:</b> <i>Analysis of CDR Loop Conformations</i>
</center>


<a name="MethodCDRAssign"></a><h4>Assignment of Representative CDR Loop Collections</h4>

<p> Based on the analysis described above, it is evident that
similarity/identity is not the only criteria when selecting a CDR template
candidate. In addition to similarity data, a collection of representative
conformations for a respective CDR loop should also be available.

<p>All CDR loops of identical length to the corresponding query sequence are
collected from the antibody database. Templates with a CDR domain structure
score of 65 and lower as well as those with sequence similarity of 55% (CDR H3
loop = 45%) and
lower are discarded from the set. The remainders are clustered applying a
single-linkage algorithm using the RMSD of CA atoms of the CDR loop
residues as the distance matrix. The clusters are merged at a resolution of 0.4
&Aring; (CDR L2 loop at 0.3 &Aring;, H2 and H3 at 0.5 &Aring;) and the candidate
with the highest sequence similarity to the query CDR
is kept as the representative of the bin. The resulting sets of CDR
conformations is then ranked by cluster popularity. In Figure 7, the essential
steps of this procedure are illustrated.


<div align=center style="padding:15px;">
<img src="antibody/cdr_assignloops.png" alt="CDR Loop Assignment" />
<br>&nbsp;<br>
<b>Figure 7:</b> <i>CDR Loop Conformation Clustering</i>
</div>
<!-- ======================= Running the application ====================== -->

<p><a name="Run"></a><h2>Running the Application</h2>

<p><font color="#FF0000"><b>Tip!</b></font>&nbsp;&nbsp; Since many actions in
the homology modeling procedures are dependent on accessing the antibody
database, it is advantageous to keep a copy of the default antibody database
<span class="nobr">(<tt>$MOE/lib/fab.mdb</tt>)</span>
on your local  machine in order to reduce network
traffic and speed up the search for templates and model building procedures.
</p>

<p>
The application can be accessed through the main MOE panel
using <b><nobr>MOE | Compute | Biopolymer | Antibody Modeler</b></nobr>
or via the Sequence Editor using
<b><nobr>SE | Homology | Antibody Modeler</b></nobr>.

<p><div align=center>
<img src="antibody/mainpanel.png"  alt="Main Panel Antibody Modeler">
</div>
<p>
The Antibody Modeler panel can be divided into two sections:
<ol>
    <li><b>Query and Template Specification.</b>
    The upper part of the panel changes depending on whether the panel is in
    template searching or model building mode.
    In <b>Template Search</b> mode,
    this area provides fields and options for
    submitting and editing queries for template searching.  In <b>Model
    Build</b> mode, this area is used for
    composing an antibody
F<sub>V</sub> dimer template.
    </li>
    <br />

    <li><b>Results Display.</b>
    The lower part of the panel is used to display the results of
    template searching and to display alignments.
    Multiple views on the template search result sets are available.
    This area does not change when the mode is switched from
    template searching to model building.
    </li>

</ol>

<!-- ==========================  Template Search ========================== -->

<p><a name="RunSearch"></a><h3>Antibody Template Search</h3><p>

In contrast to general homology modeling, an antibody model is in most cases a
dimer composed of a V<sub>L</sub> and V<sub>H</sub> chain. Therefore, the
antibody template search is in fact a combination of two separate  searches -
one for the light and one for the heavy chain template.

<p>
The query sequences may be typed in directly or pasted from the clipboard with
<tt>Ctrl-V</tt>. If the chain is already available in the Sequence Editor,
the user may load the sequence directly into the application by choosing
the corresponding chain from the option menu.
The <b>Load Chain</b> option will
result in the reverse behavior, with a chain being created in MOE from
the respective sequence in the panel.

<p>
In general, the sequence specified as <b>Query 1</b> represents, by
default, an antibody V<sub>L</sub> chain.
In this case, the template search procedure would only
consider V<sub>L</sub> family clusters when searching for templates.  If,
however, a dimer
of heavy chains is the desired model (VH:VH), <b>VHH</b>
must be chosen from the <b>Query 1</b> option menu.
The labeling of the result sets will reflect this change in
order to distinguish the set from the results of the other heavy chain
in the system (which will be labeled VH).

<p>
Analogous to the V<sub>L</sub> chain, <b>Query 2</b>
specifies the heavy chain query which is set to <b>VH</b> by default.
Similarly to
V<sub>L</sub> chains, it is also possible to define the input sequence
as a light chain, if the resulting antibody dimer should be composed
of light chains only (VL:VL). The menu
setting must then be changed to <b>VLL</b>.  Although this
scenario is not relevant to natural biology, it may be of interest to
synthetic bioengineering.

<p>
When a query sequence has been edited in one of the text fields,
the <b>Search</b> button at the bottom of the main panel is activated.
Press <b>Search</b> to initiate the antibody template search.

<!-- =========================== Search Results =========================== -->

<a name="RunResults"></a><h3>Search Results</h3>

<p>
The search results are displayed in a table if the radio button is
set to <b>Results</b>. In general, only results with an overall
(the entire
F<sub>V</sub> domain) score higher than the cutoff value as
specified in the <a href="#Settings">Settings</a> panel
will be displayed. A double-click in the result table will load
the selected antibody chain(s) into MOE.
</p>

<p> The search results for
F<sub>V</sub> dimer templates are usually stored in three sets,
one for each of the specified query classes (VL or VH) and one that shows the
results forming the best suitable
F<sub>V</sub> dimer <span class="nobr">(VL + VH)</span>.
These three result sets can
be viewed in distinct viewing modes specified by <b>Display</b>: </p>

<table width="100%" cellpadding="5">
  <tr>
    <td valign="top"><a name="RunVLVH"></a><nobr><b>VL + VH</b></nobr></td>
    <td valign="baseline">

    The results in this viewing mode are particularly useful when searching for
    a suitable
F<sub>V</sub> dimer template which is in most cases the first step of
    building an antibody homology model (see Figure 2). Therefore, these results
    are presented by default immediately after the search is completed. All of
    the candidates shown in the table resemble
F<sub>V</sub> dimers with respect to
    structure.  The individual chain name as well the sequence identity scores
    for the entire
F<sub>V</sub> domain and the framework region are listed in the columns
    <b>Overall</b> and  <b>Framework</b> for each of the candidate dimers. The
    framework scores of the
F<sub>V</sub> dimer are  given as the average of the FR1, FR2
    and FR3 scores of each of the chains in the column <b>Score</b>. The results
    may be filtered by <b>Antigen</b>, <b>Species</b> and/or <b>Subclass</b>.

    <p>Note: When Query 1 is set to VH, then the mode is labeled
    <b>VHH + VH</b>, and similarly, when Query 2 is set to VL, <b>VL + VLL</b>.
    </p>
    </td>
  </tr>
  <tr>
    <td valign="top"><a name="RunVL"></a><b>VL</b></td>
    <td valign="baseline">

    This viewing mode displays the search results for the sequence  specified in
    the <b>Query 1</b> text field.  (Note that when Query 1 is set to VH, the
    Display mode is also called VHH.) There are a number of views of the results
    available that can be specified by <b>Region</b>. The idea is here to offer
    various levels of detail for each of the relevant subdomains of an antibody
    chain.  An overall view listing the sequence similarity/identity values for
    each subdomain is offered in <b>All</b> viewing mode. When viewing the
    results for framework or one of the CDR loops, then the sequence
    identity/similarity results are enriched with structure scores of the
    selected region.

    <table width="100%" cellpadding="3">
	<tr>
	<td valign="baseline"><b>All</b></td>
	<td valign="baseline">

	In this mode the sequence identity and similarity for each subdomain of
	an antibody chain are listed. Scores for the entire
F<sub>V</sub> region of the
	light chain and the framework regions FR1, FR2, FR3 and FR4 represent
	sequence identities (unless changed in the <a
	href="#Settings">Settings</a> panel) and are listed in columns
	<b>Overall</b>, <b>FR1</b>, <b>FR2</b>, <b>FR3</b>  and <b>FR4</b> (at a
	maximum length of eight residues). The similarities of the gapless CDR
	loops with the respective query loop are listed in columns <b>L1</b>,
	<b>L2</b> and <b>L3</b>. The scores are set to 0.0 in the event that the
	length of the CDR loop of the query does not match the length of the
	corresponding CDR loop of the target. The  column <b>Ag</b> specifies
	the type of antigen the antibody structure of the target was associated
	with. In general, three different types of antigens are specified: P =
	Protein, H = Hapten and W = crystallographic waters.   In the column
	<b>Species</b> the source of the antibody is listed.

	<p>The data set can be sorted by each column as specified by
	<b>Sort</b>; by default, sorting is according to the overall score. In
	addition, the results can also be filtered by the type of antigen
	<b>Ag</b>, <b>Species</b> or <b>Subclass</b>. </p>

	<p> <div align=center>
	<img src="antibody/VL_all.png"  alt="VL All Display">
	</div>
	</p>
	</td>
      </tr>
      <tr>
	<td valign="baseline"><b>FR</b></td>
	<td valign="baseline">

	This viewing mode offers a detailed report on the framework results as a
	combination of the framework regions FR1, FR2 and FR3 respectively. The
	sequence identity of the query to this region is shown in the column
	<b>Framework</b> and the residue length  is shown in the <b>Length</b>
	column.  In addition to the identity scores, the structural scores of
	the framework templates are also listed here.  An overall structural
	assessment of the template is given by the <a
	href="#MethodEval">scores</a> displayed in the <b>Structure</b> column.
	When labeled as <i>CA-trace</i>, the structure is comprised exclusively
	of CA-coordinates and is not suitable for modeling.

	<p> Structure values below 50 indicate issues that should be addressed
	or accounted for when choosing the chain(s) as framework templates. The
	defining scores for backbone assessment of the subdomain are given by
	the backbone topology and geometry scores, displayed in columns
	<b>Topology</b> and <b>Geometry</b>. Errors in either category usually
	result in low scores, hence the corresponding structures should be
	carefully assessed before being used for modeling.  In addition to the
	structure scores, the average B-Factors of the backbone atoms N, CA, C
	and O over the length of the subdomain are given in the column
	<b>B-Factor</b>.	These values may be of particular interest when
	comparing the CDR loop candidates. The X-ray crystallographic resolution
	of the template structure is shown in column <b>Resolution</b>. </p>

	<p> By default, the results are sorted by the
	framework sequence identity but can be sorted by each column as
	specified by the <b>Sort</b> menu. In addition, the results may be
	filtered by Antigen <b>Ag</b>, <b>Species</b> or <b>Subclass</b>.
	</p>

	<p>
	In <a href="#RunBuild"><b>Model Build</b></a> mode,
	double-clicking in the list will
	result in the chain code of the selected template being
	entered in the <b>FR</b> text field in the template
	specification area of the panel.
	</p>
	<p>
	<div align=center style="padding:15px;">
	  <img src="antibody/VL_fr.png"  alt="VL FR Display">
	</div>
	</p>

	</td>
      </tr>
      <tr>
	<td valign="baseline"><a name="L1"></a><b>L1</b></td>
	<td valign="baseline">

	A detailed view of the CDR L1 loop results is presented here. The
	emphasis in this viewing mode is not the sequence similarity to the
	query CDR as shown in column <b>L1</b>, but moreover an assessment of
	the structural quality of the potential CDR loop templates. This allows
	choosing designated templates based on structural fitness in addition to
	sequence similarity. In addition to structural and similarity data, the
	conformational spread of a suitable collection of L1 loops may be
	displayed as  well (see <a href="#MethodCDRAssign">
	CDR Loop Assignment</a> for details).

	The L1 sequence similarity of the templates to the query is shown in
	column <b>L1</b>. An overall structure quality assessment of the L1 loop
	is shown in column <b>Structure</b>. The defining scores for backbone
	quality assessment of the CDR L1 loop are given by backbone topology and
	geometry, listed in columns <b>Topology</b> and <b>Geometry</b>. Errors
	in either category usually result in low overall scores, hence the
	corresponding chains should be carefully assessed before being used for
	modeling. In addition to the structure scores, the average B-Factor
	values of the backbone atoms N, CA, C and O over the length of the
	subdomain are given in the column <b>B-Factor</b>. These values may be
	of particular interest when looking at the CDR loops of the candidates.
	The X-ray crystallographic resolution of the template structure is shown
	in column <b>Resolution</b>. The antigen column <b>Ag</b> specifies the
	type of antigen the antibody  structure was associated with. Three
	different types of antigens are specified: P = Protein, H = Hapten and W
	= crystallographic waters.

	<p> By default, the results are sorted by similarity to L1 but can also
	be sorted by each column as specified by the <b>Sort</b> menu. As
	already mentioned in the previous views, the search results may be
	filtered by antigen type <b>Ag, Species or Subclass</b>. </p>

	<p> Double-clicking in the result list will load the selected chains in
	MOE when in <a href="#RunSearch">Template Search</a> mode.  In <a
	href="#RunBuild">Model Build</a> mode, double-clicking in the list will
	create an entry of the top ranking candidate within the current
	selection in the <b>L1</b> text field.   </p>

	<p>
	<div align=center style="padding:15px;">
	<img src="antibody/VL_L1.png"  alt="VL L1 Display">
	</div>
	</p>
	<br>

	The  broad range CDR Loop conformations (see
	<a href="#MethodCDRBounds">Analysis of Canonical CDR Loop
	Conformations</a> for details) observed in
	multiplet clusters warrants an evaluation of potential CDR loop
	candidates on the basis of conformational diversity. The conformational
	range of pre-selected loop templates is offered as a dendrogram when
	<b>Show CDR Loop Dendrogram</b> is pressed. All template
	candidates shown in the panel have at least 50% similarity to CDR Loop
	query and domain structure scores of 65 and higher (structurally
	suitable for homology modeling purposes). The underlying distance matrix
	for applying the single-linkage clustering algorithm is based on the
	root mean square distance (RMSD) of CDR loop CA atoms. The
	dendrogram allows to select the templates based on structural
	disposition independent from similarity data. This viewing mode is
	particularly of interest when distinct L1 loop conformations are liked
	to be preserved e.g. in a antibody humanization project. For those
	tasks, similarity data alone will not necessarily deliver accurate
	models.

	<div align=center style="padding:15px;">
	  <img src="antibody/L1_conf.png"  alt="VL FR Display">
	</div>
	The functionality of the CDR Loop clustering panel is described below in more detail.
        </td>
      </tr>
      <tr>
	<td valign="baseline"><b>L2</b></td>
	<td valign="baseline">

	A detailed view of the L2 loop results is presented in this table.
	The functionality and the various views on the data is analogous to CDR
	<a href="#L1">L1</a> described above.
	</td>
      </tr>
      <tr>
	<td valign="baseline"><b>L3</b></td>
	<td valign="baseline">

	A detailed view of the CDR L3 loop results is presented in this table.
	The functionality and the various views on the data is analogous to CDR
	<a href="#L1">L1</a> described above.
	</td>
	</tr>

    </table>
    </tr>
    <tr>
      <td valign="top"><a name="RunVH"></a><b>VH</b></td>
      <td valign="baseline">

    This display mode shows the results of the query specified by the <b>Query
    2</b>. The descriptions of the viewing modes are the same as for VL
    described in detail above. The only difference is in the labels: VH instead
    of VL and H1, H2, H3 instead of L1, L2, L3.  If Query 2 has been set to VL,
    the labels become modified accordingly.

    </td>
    </tr>
</table>

<!-- ===========================     Report    ============================ -->

<a name="RunReport"></a><h3>Search Report</h3>

<p>

A report of the search results will be generated by pressing <b>Report</b>,
which is only accessible when the display mode is set to <b>Results</b>.
The search report consists of the combined results (as described in
the <a href="#RunVLVH"><span class="nobr">VL + VH</span></a>
viewing mode above) and the
individual results for heavy and light chains (viewing mode
<a href="#RunVL">VL</a> and <a href="#RunVH">VH</a>). The
individual search results list the scores of sequence identity (Framework)
and similarity (CDRs) for each subdomain along with the overall score
of the respective V<sub>L/H</sub> domain.  In addition to the scores,
the specifications that have been applied for conducting the search as
well as the input query sequences are included in the report. The report
is shown in a text editor window and can be printed or saved.

</p>

<div align=center>
<img src="antibody/report.png"  alt="Search Report">
</div>

<!-- ============================= Alignment =============================== -->

<a name="RunAlignment"></a><h3>Alignments</h3>

<p>The sequence identity/similarity scores for individual antibody subdomains
are important factors when selecting appropriate antibody templates. This is
particularly true when searching for framework templates.  However, a number of
projects associated with antibody homology modeling often are based on only
subtle changes of single amino acids, e.g. to improve the solubility of the
antigen binding site, key residues in the framework have to be changed, or when
loop shaping of CDR loops is sought to be altered. As a result, despite
promising similarity/identity scores on a subdomain level, the presence of
particular residues at certain locations might be unfavorable.  In these cases,
a more detailed view of the results may be an advantage, allowing the inspection
of alignments on a residue level.

<p>When instantiating this view, promising template candidate(s) will have
usually been identified and selected in the <b>Results</b> table. Press
<b>Alignment</b> to switch the viewing mode. The cluster of the top ranking
candidate within the selection will be shown
and also selected in the alignment. If
no template has been selected, then the alignment with the top-ranking candidate
from the search results is shown by default. </p>

<p>In <b>Alignment</b> viewing mode the query sequence is always shown on top,
followed by the template candidate(s) as selected in the <b>Results</b> table
(of course, only the ones that are also in this cluster). The CDR loop regions
are outlined in the header by arrow and line characters
<span class="nobr"><tt>(&lt;&ndash;&ndash;&ndash;&ndash;&ndash;&ndash;&gt;)</tt></span>
which makes it easy to
identify the topologically relevant antibody regions within an alignment. Key
information on the target cluster as described in the <a
href="#MethodFabFam">family file</a> is displayed and includes class, species
and subclass and - if available - the framework RMSD (&Aring;) of the template
cluster. </p>

<!-- ===========================   Alter Alignment View ================ -->

<a name="RunAlignmentViews"></a><h4>Alignment Views</h4>

A variety of views reflecting the topology of the antibody chain are offered that can be modified by pressing the <b>Region</b> button.
<div align="center" style="padding:15px;">
  <img src="antibody/alignment.png"  alt="Alignment Display">
</div>

When in <b>All</b> viewing mode, the alignment of the query with the entire
F<sub>V</sub>
region of the target cluster is shown. This is the default mode when inspecting
alignments. The F<sub>V</sub> region of the targets is restricted to eight
post-CDR3
residues. This viewing mode is suggested when inspecting the alignment of
framework residues. A more detailed view of the alignment in respective CDR loop
regions is offered when pressing one of the CDR loop buttons in <b>Region</b>.
Depending on the class of the target cluster, the CDR regions are labeled <b>L1
L2 L3</b> when a light chain alignment is shown, or <b>H1 H2 H3</b> in the case
of an alignment with heavy chains. The alignments in CDR viewing mode include
the CDR loop residues (with three residues on either side of the CDR) and allows
for inspection of the CDR&ndash;Framework transition region.  In addition
to the alignment, the
sequence identity of CDR loop residues of the targets with the
query CDR is listed. The <b>global</b> identity is calculated as the percentage
of matching residues with the query, as shown in the panel. Gaps are factored
into the calculations. Since the alignment is performed on the entire
F<sub>V</sub> region
during the template search, the positions within the CDR loop regions might not
be optimized. By pressing the button <b>Realign CDR</b>, the CDR region
(including three framework residues on either side) is globally aligned using
the constraints at the first framework residue on either side of the CDR. The
alignment then is then updated and the global identity recalculated based on the
new residue positions. The <b>local</b> identity is based on the calculation of
an alignment of the CDR loop residues with the corresponding residues of each
member of the target cluster.

<div align="center" style="padding:15px;">
  <img src="antibody/align_L3.png"  alt="Alignment Display CDR Loops">
</div>

<p> When the residues in the alignment have a high degree of agreement with the
query sequence, it is often difficult to detect mismatching residues within an
alignment. Checking the <b>Show only Mismatches</b> checkbox on will change the
viewing mode by only showing mismatching residues to the query sequence.
Identical residues to the query are then shown with the "." character, while
the mismatched residues keep the single letter residue code. This way, it
easy to detect the regions that are not identical to the query.

<p> If the <b>Alignment</b> was entered when the display mode is set to
<span class="nobr"><b>VL + VH</b></span>,
an option list will appear in the panel. This widget contains
the chain names of the top ranking F<sub>V</sub> dimer complex as selected in
<b>Results</b>. By default, the alignment of the light chain is shown first, but
the user can toggle easily between the alignments of the light and heavy chain
by changing the selection in the option list.

<p>
<div align=center style="padding:15px;">
<img src="antibody/align_all_mismatch.png"  alt="Alignment Diff Viewing Mode">
</div>

A suitable template candidate within an alignment may be selected and the
corresponding similarity/identity scores and/or structural scores be viewed by
switching back to <b>Results</b>.  This quick, easy toggling between the two
viewing modes allows to gather information on both the subdomain and the residue
level. In <b>Alignment</b> viewing mode, it is possible to select one or more
entries and then load them into MOE by pressing <b>Create in MOE</b>. The entire
cluster can be loaded into MOE by first selecting all or none of the entries
(use the popup on the right mouse button). The antibody chains are aligned and
annotated as framework (FR) and CDR regions corresponding to the antibody
class. The annotations appear as a residue coloring scheme in the Sequence
Editor, with the CDR regions colored in blue. Residues of a respective  antibody
subdomain may be selected in MOE using the <b>Sets</b> menu in the <a
href="../moe/gui/select.htm#ResidueSelector">Residue&nbsp;Selector</a>. </p>

<!-- ===========================   Print Alignment  ==================== -->

<a name="RunPrint"></a><h4>Print</h4>

<p>
The alignment can be printed using
the <b>Print</b> button, which
launches the <a
href="../apps/print.htm#PrintSequence">Sequence&nbsp;Editor&nbsp;Print</a>
panel. The CDR loop regions are given colored backgrounds, and the respective
antibody subdomain regions are annotated at the top. All the above described
alignment viewing modes can be printed as seen in the alignment panel.
</p>

<p>Note: Printing is only possible
in <b>Alignment</b> display mode; the
<b>Print</b> button is otherwise unavailable.
</p>

<div align=center>
<img src="antibody/alignprint.png"  alt="Print Alignment">
</div>

<!-- ========================== Template composition ====================== -->

<a name="RunBuild"></a><h3>Building Antibody Models</h3>

<p> A hallmark feature of MOE Antibody Homology Modeler is the ability to
compose each component of the F<sub>V</sub> template dimer from potentially
four different
sources (<a href="#MethodGrafting">loop grafting</a>) for framework and each
respective CDR loop region. As this resembles a general requirement when
humanizing an antibody, it has the advantage that distinct conformations can be
conserved with full control on type and conformation of the input source.  Press
<b>Model Build</b> to switch the panel into antibody model building mode. </p>

<div align=center align="center" style="padding:15px;">
  <img src="antibody/model_build.png"  alt="Model Build">
</div>

<p>
<a name="RunAssignLoops"><h3>Assigning CDR Loops</h3></a>

<p>The composition of antigen binding site templates from various sources is
represented by four text fields denoted as <b>FR</b>, <b>L1</b>, <b>L2</b>,
<b>L3</b> in the event that a light chain is being composed, and as <b>FR</b>,
<b>H1</b>, <b>H2</b>, <b>H3</b> in the case of building a heavy chain template.
In each of the text fields, the chain codes of the designated template regions
must be entered (the values are automatically entered upon double-click of the
result table).  A representative collection of CDR loop candidates are presented
in the shortcut list (see <a href="#MethodCDRAssign">above</a> for details).
The templates in this collection are at
least 0.4 &Aring; distant with respect to CDR loop CA RMSDs and ranked according
to cluster popularity. As a result, the top ranking candidate in the shortcut
list represents the candidate from the most popular CDR loop conformation and
the conformational outliers are listed on the bottom. This allows for
fast generation
of templates with common CDR loop conformations.  The distance
relationship (based on CA RMSDs) among the candidates can be viewed as a
dendrogram by pressing the <b>?</b> button.

<!-- ======================= CDR Loop cluster Panel ===================== -->

<div align=center style="padding:15px;">
  <img src="antibody/CDR_assign.png"
    alt="Dendrogram CDR Loop Conformations">
</div>

Each entry in the shortcut list represents the template with the highest
sequence similarity within a cluster (see <a href="#MethodCDRAssign">above</a>
for details). The chain
names along with the similarity values are shown in the <b>Code</b> and
<b>Similarity</b> columns.
The <b>Rank</b> column denotes the order the candidates are
listed the respective shortcut list, whereas the order within the cluster
dendrogram is shown in the <b>#</b> column. Selection of templates can be done
either by clicking in the dendrogram or by selection in the list. A report of
the clustering tree can be obtained by pressing the <b>Report</b> button. The
dendrogram may be saved in common graphics formats by pressing the
<b>Export</b> button. The colors of the dendrogram, the background, and the
axis may be altered by pressing the <b>Settings</b> button. The dendrogram
can also be opened by pressing <b>Show CDR Loop Conformations</b> when the
display mode is set to either <b>VL</b> or <b>VH</b> and the region set to one
of the CDR regions. The collection of CDR templates can get quite large,
reducing the usability of the clustering dendrogram. By moving the <b>Zoom
X-Axis</b> ruler, the number of visible clusters in the dendrogram can be
reduced to as few as 10, creating a zooming effect. Navigation along the
X-Axis is possible by either typing in the X-Axis starting point in the text
field <b>Start at</b> or by using the slider next to it. Similarly, the Y-Axis
range can zoomed by moving the <b>Zoom Y-Axis</b> slider.

<p>Once the search results and the query-specific CDR loop conformations are
available, it is easy to generate antigen binding site models of high
precision using the MOE Antibody Homology Modeler. A suitable F<sub>V</sub>
dimer template
can be composed by selecting the top ranking framework candidate (highest
ranking from the <span class="nobr"><b>VL + VH</b></span>
result set) and then pressing the <b>Assign
Loops</b> button for the light and heavy template chains. The text fields
denoting the respective CDR loops of the F<sub>V</sub> dimer are automatically
filled with
the top-ranking (most prominent CDR loop conformation) candidate from the
respective CDR shortcut list. Since only structurally valid templates have been
considered during the clustering process, browsing through the result sets for
light and heavy chains becomes unnecessary. By using only a few mouse
clicks, all relevant components of the F<sub>V</sub>
dimer template are assigned with
reasonable input  structures and the antigen binding site model is ready to be
built. However, depending on the underlying task, much more detail must be
considered when choosing the best template(s). If a new template should be
composed or various CDR loops should be reassigned, then the CDR loop text
fields can be emptied by pressing the <b>Clear</b> button.

<p> It is noteworthy that only the framework templates have to be specified for
building an antibody homology model. The text fields denoting the input
structures for CDR loops can be left empty, if the templates do not satisfy the
expected criteria. Then the loop building routine as implemented in the
<a href="promodel.htm">Homology Modeling</a> application
is used to model the CDR loop.

<p> In <b>Results</b> viewing mode a variety of views on the search results are
offered. Double-clicking in the result list triggers an entry of the chain name
in the corresponding CDR loop text field. If more than one entries are selected
in the list, then only the top ranking template is used.  In case the display
mode is set to <span class="nobr"><b>VL + VH</b></span>,
all Framework and CDR text fields of the
designated F<sub>V</sub> template are filled with the chain names of the top
ranking F<sub>V</sub> dimer. </p>

<p>
When the framework regions of the F<sub>V</sub> template are specified then
the <b>Build</b> button becomes available.  Pressing <b>Build</b>
will open the Antibody Builder panel, described next.
</p>

<!-- =========================== Builder Settings ========================= -->

<a name="RunBuildSettings"></a><h3>Antibody Builder</h3>

<p>The Antibody Builder is opened by pressing the <b>Build</b> button in the
Antibody Modeler panel, which appears only when the Antibody Modeler panel is in
<b>Model Build</b> mode. The panel is divided in a section specifying the
parameters relevant for composing the antigen binding site template, the
visualization and the refinement of the resulting F<sub>V</sub> dimer model and
a section specifying the parameters (<b>Homology Modeling Parameters</b>)
used by the <a href="promodel.htm">Homology Modeling</a> application.
For a fully grafted F<sub>V</sub> dimer template (a template with continuous
backbone trace) the homology modeler is used only for conducting the necessary
sidechain mutations
and sidechain placements in order to finish the model. Consequently, most of the
homology modeling parameters are relevant only if the homology
modeler is assigned to model one or more CDR loops
(see <a href="promodel.htm">Homology Modeling</a> for details).
</p>

<p>
<div align=center style="padding:15px;">
  <img src="antibody/builder.png" alt="Antibody Builder">
</div>
</p>

<p>
<table width="100%" cellpadding="3">
    <tr>
      <td valign="baseline"><b>Current System</b></td>
      <td valign="baseline">

      Before the modeling procedure begins, the current system will be saved
      into the specified file.<p> </td>

    </tr>
    <tr>
      <td valign="baseline"><b>Chain Name</b></td>
      <td valign="baseline">

      The name of the antibody model. If no name is  given, antibody models will
      be named using the prefix  <tt>fab_model</tt>. <p> </td>

    </tr>
    <tr>
      <td valign="baseline"><b>Side Chain Packing</b></td>
      <td valign="baseline">

      If on, then all sidechain atoms of the CDR loops will be destroyed. As a
      result, the repacking of the sidechain rotamers  will be enforced in the
      modeling phase. <p> </td>

    </tr>
    <tr>
      <td valign="baseline"><b>Clashing Energy Cutoff</b></td>
      <td valign="baseline">

      Atom clashes in the interface of the VL:VH template  as well as intrachain
      clashes are calculated and summed over each residue. If the repulsion
      energy (kcal/mol) exceeds the specified cutoff, then the  sidechain atoms
      of these residues will be destroyed enforcing repacking of the sidechain
      rotamers of these residues in the modeling phase.  <p> </td>

    </tr>
    <tr>
      <td valign="baseline"><b>Template</b></td>
      <td valign="baseline">

	<i>Keep Template after Building Model.</i>
      If on, the F<sub>V</sub> template will be preserved after the antigen
      binding site model has been built.<p>

	<p>
	<i>Transpose Template to
F<sub>V</sub>-Dimer Coordinates.</i>
      If on, the F<sub>V</sub> template will be transposed to
	suitable dimer coordinates after the antigen
      binding site model has been built.
	<p>
	</p>

	<p> </td>

    </tr>

    <tr>
      <td valign="baseline"><b>Model</b></td>
      <td valign="baseline">

      If on, the full antibody model is built after building the antibody
      template. Otherwise, only the F<sub>V</sub> dimer template is composed.
      <p>
      </td>

    </tr>
    <tr>
      <td valign="baseline">&nbsp;</td>
      <td valign="baseline">

      The second checkbox in this section is only relevant in the context of an
      antibody humanization project. If on, then the sequence of the target
      model is composed of the template framework sequence and the CDR loop
      sequences of the input queries. <p> </td>

    </tr>
    <tr>
      <td valign="baseline"><b>Model Refinement</b></td>
      <td valign="baseline">

      If on, then the antigen binding site model will be subjected to a
      structure refinement procedure in order to relieve strained geometry.
      Specify, if prior to the structure refinement step the <a
      href="../apps/protonate3d.htm">Protonate 3D</a> application should be used
      to optimize ionization states and proton placement in the F<sub>V</sub>
      model. <p>
      </td>

    </tr>
    <tr>
      <td valign="baseline"><b>Color VL</b></td>
      <td valign="baseline">

	Once the models and/or templates are built, the CDR loops and  framework
	residues of the light chain are colored according to the  <b>FR</b>,
	<b>L1</b>, <b>L2</b>, <b>L3</b>  color specifications. In addition,
	residue sets of the antibody subdomains are assigned; these
	can be used in the
	<a href="../moe/gui/select.htm#ResidueSelector">Residue Selector</a>.
	<p> </td>

    </tr>
    <tr>
      <td valign="baseline"><b>Color VH</b></td>
      <td valign="baseline">

	Once the models and/or templates are built, the CDR loops and  framework
	residues of the heavy chain are colored according to the  <b>FR</b>,
	<b>H1</b>, <b>H2</b>, <b>H3</b>  color specifications. In addition,
	residue sets of the antibody subdomains are assigned; these
	can be used in the
	<a href="../moe/gui/select.htm#ResidueSelector">Residue Selector</a>.
	<p> </td>

    </tr>
    <tr>
      <td colspan="2">

	The following parameters are for the
    	<a href="promodel.htm">Homology Modeling</a> application:

        <p>
      </td>
    <tr>
      <td valign="baseline"><b>Output Database</b></td>
      <td valign="baseline">
	The name of the molecular database file that will contain the
      generated models and various measurements. This can be either a new
      database or an existing one, in which case the contents will be
      overwritten.<p>
      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Loop Dictionary</b></td>
      <td valign="baseline">

	The loop dictionary is a requirement for loop modeling using the
	homology modeling application, but is not relevant for a fully grafted F<sub>V</sub>
	antibody template. However, if the loop building routine of the homology
	modeler is assigned for modeling a CDR loop, the loop dictionary must
	be specified.  <p><b>Loop Database</b> specifies the MDB file containing
	the protein structures used for loop  modeling. The antibody database
	<tt>$MOE/lib/fab.mdb</tt> is used by default.

	<p> <b>Loop Codes</b> specifies a text file containing the PDB codes  of
	the protein chains to be included in building the loop database for
	knowledge-based loop modeling. If the field is blank, all structures  in
	the loop database will be used. It is recommended to specify the loop
	codes, in case the <tt>$MOE/lib/pdb.mdb</tt> database is chosen (default
	<span class="nobr"><tt>$MOE/lib/segment.lis</tt>).</span> </p>

      <p>
      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Models</b></td>
      <td valign="baseline">

	Specify the number of main chain models. This entry is only relevant if
	the homology modeling application is assigned to modeling one or more
	CDR loops. In this case, a stochastic element is involved in building
	the loops and the number of independent main chain models have to be
	specified. If the CDR loops already been grafted, then only one main
	chain model is built. The first model is always built using the UQO
	method, the remaining side chain models are built using a sampling
	method.

      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Intermediates</b></td>
      <td valign="baseline">

      Specifies the degree of refinement by molecular mechanics to be  applied
      to each of the independently constructed models in the ensemble  of
      intermediate models. If <b>Medium</b> is chosen, a brief series of  highly
      tethered minimizations meant only to relieve strained geometries is run.
      If <b>Fine</b> is chosen, each model is also submitted to an
      electrostatics-enabled minimization run which terminates when the
      gradient falls below the specified <b>RMS Gradient</b>. <p> </td>

    </tr>
    <tr>
      <td valign="baseline"><b>Model Scoring</b></td>
      <td valign="baseline">

      Specifies the scoring method used to choose the final model from the
      ensemble of intermediate models.<p> </td>

    </tr>
    <tr>
      <td valign="baseline"><b>Final Model</b></td>
      <td valign="baseline">

      Specifies the energy minimization scheme to be applied to the highest
      scoring model from the ensemble of intermediate models. If <b>Medium</b>
      is chosen, the final model is minimized using a built-in protocol.  If
      <b>Fine</b> is chosen, the model is minimized within the full
      specification of the currently loaded forcefield, including cutoffs and
      the treatment of electrostatics.<p> </td>

    </tr>
    <tr>
      <td valign="baseline"><b>Forcefield</b></td>
      <td valign="baseline">

      Displays the currently loaded forcefield, which will be used for all
      energy minimizations, and for the GB/VI scoring. Press the <b>Potential
      Setup</b>  button to change the forcefield specifications. </td>

    </tr>

</table>
</p>

<!-- ===========================   Numbering   ============================ -->

<a name="RunSchemes"></a><h3>Numbering</h3>

<p>

The various
<a href="antibody.htm#MethodNumbering">antibody numbering systems</a>
can be viewed by pressing the <b>Numbering</b>
button.  This will open a panel illustrating the different numbering schemes
in relation to each other.  Heavy and light chains are illustrated in two
separate depictions. Highly conserved antibody residues are shown in
color as summarized in the legend on the bottom. The image in the panel
can be exported in commonly known image formats by pressing <b>Export</b>
or can be printed directly by pressing <b>Print</b>.

</p>

<div align=center>
<img src="antibody/numberingpanel.png"  alt="Numbering Schemes">
</div>

<!-- =========================== Settings ============================ -->

<a name="Settings"></a><h2>Antibody Modeler Settings</h2>

<p>

In the Antibody Modeler Settings panel, the parameters relevant for template
search, sequence alignment and template grafting are specified. The panel is
opened by pressing the <b>Settings</b> button on the Antibody Modeler panel:
</p>

<p>
<div align=center style="padding:15px;">
<img src="antibody/settings.png"  alt="Antibody Modeler Settings">
</div>
</p>

<table width="100%" cellpadding="3">
    <tr>
      <td valign="baseline"><b>Antibody&nbsp;Structures</b></td>
      <td valign="baseline">
	Specifies the name of the <a href="#MethodFabDB">antibody database</a>.
	This database is the core of the knowledge-based antibody homology
	modeling.
        <p>
      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Family File</b></td>
      <td valign="baseline">Specifies the name of the clustered
	<a href="#MethodFabFam">family file</a>. This
        file contains the antibody sequences clustered by class, subclass,
        species and framework sequence identity. This file is the
        core file for the antibody template search.
        <p>
      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Display&nbsp;Score&nbsp;Cutoff</b></td>
      <td valign="baseline">When showing template search results, the entries
      with an overall
	<a href="#MethodEval">score</a> below the designated value will not be
	shown.
        <p>
      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Framework&nbsp;Alignment</b></td>
      <td valign="baseline">Amino acid substitution matrix used when
      calculating the sequence
      alignment scores of framework regions. The default matrix is
      <tt>blosum62</tt>.
      <p>
      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>CDR Alignment</b></td>
      <td valign="baseline">Amino acid substitution matrix used when calculating the sequence
      alignment scores of CDR regions. The default matrix is <tt>blosum62</tt>.
      <p>
      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Score</b></td>
      <td valign="baseline">Specifies the
	<a href="#MethodSearch">scoring mode</a>
	when scoring antibody loop or framework
      regions. By default, the scoring mode is set to <b>Identity</b>. When
      calculating CDR loop scores the default mode is <b>Similarity</b>.
      <p>
      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Gap Start</b></td>
      <td valign="baseline">The penalty for creating a gap in a sequence alignment. Large values
      penalize gap creation.
      <p>
      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Gap Extend</b></td>
      <td valign="baseline">The penalty for extending a gap in a sequence alignment. Large values
      penalize gap extension.
      <p>
      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>CDR Length Mismatch</b></td>
      <td valign="baseline">When calculating the CDR loop scores, the gapless loop sequence is
      used. If on, then the score will be set to zero if the length of the
      query
      CDR loop does not match the CDR loop length of the target.
      </td>
    </tr>
</table>

<p>
As outlined in the <a href="antibody.htm#MethodGrafting">Methodology</a>
section, tethered minimization
in four stages is conducted when creating chimeric antibody templates. The
following parameters are applied during the procedure.
</p>

<table width="100%" cellpadding="3">
    <tr>
      <td valign="baseline"><b>Weight</b></td>
      <td valign="baseline"> The tether weights of heavy atoms used in Stages 1-4.<p> </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Radius</b></td>
      <td valign="baseline"> The flat-bottom tether radius used in Stages 1-4.<p> </td>
    </tr>
     <tr>
      <td valign="baseline"><b>Gradient</b></td>
      <td valign="baseline"> The RMS gradient applied in Stages 1-4. The forcefield energy
      minimization performed will stop when the root mean square (RMS) gradient
      of the potential energy falls below the specified values.
      <p> </td>
    </tr>
</table>

<!-- ============================= Example =============================-->

<a name="FabExample"></a>
<div class="example">
<h2>Example: Building an Antibody Homology Model</h2>
<ol>
    <p>
    <li><i>Clear the molecular data in MOE.</i>
    <blockquote>
    <span class="menu">MOE | RHS | Close</span>
    </blockquote>

    </li></p><p><li>

    <i>Load the protein sequence into MOE.</i>

    <blockquote>
    <span class="menu">MOE | File | Open | <tt>$MOE/sample/mol/3D9A.fst</tt>
    </span>
    </blockquote>

    </li></p><p><li>

    <i>Open the Antibody Modeler application.</i>

    <blockquote>
    <span class="menu">
    MOE | Compute | Biopolymer | Antibody Modeler
    </span>
    </blockquote>

    </li></p><p><li>
    <i>Specify the query sequences.</i>
    <br />
    From the <b>Load Chain</b> pulldown, select
    <tt>3D9A.L</tt> in for <b>Query 1</b> and
    <tt>3D9A.H</tt> for <b>Query 2</b>.
    This will copy the sequence for the respective chains
    into the query fields.

    </li></p><p><li>
    <i>Perform the template search.</i>
    <br />
    
    Press <b>Search</b>.

    <p>Once the search results are displayed, you can
    to analyze the data based on the framework and overall similarity scores.
    To analyze the results for individual loops, select either <b>VL</b> or
    <b>VH</b> from the <b>Display</b> pulldown from the <b>Antibody
    Modeler</b> panel.
    </p>

    <p>
    <center>
    <img src="antibody/ab_search_results.png" alt="Search results" />
    </center>
    </p>

    </li></p><p><li>
    <i>Set the model building parameters.</i>

    <p>First, press <b>Model Build</b> to switch to the model building
    page of the Antibody Modeler.
    </p>

    <p>
    Set the <b>Display</b> to <b>VL + VH</b> and select 1GP0 as
    the template by double clicking on
    <tt>1GP0M/I</tt> in 
    the Results list. (This will simulate blind prediction since we are using
    a template with a score of &lt;90%). This will automatically fill
    <tt>1GP0.M</tt> and <tt>1GP0.I</tt> into the template boxes.
    </p>

    <p>
    <center>
    <img src="antibody/ab_1gp0.png" alt="Template selection" />
    </center>
    </p>

    <p>
    Press <b>Assign Loops</b>.
    for both the <b>VL</b> and <b>VH Template</b>.
    This will automatically select the best loops as determined
    by the scoring function.
    </p>

    </li></p><p><li>
    <i>Build the model.</i>
    <br />
    
    Press <b>Build</b>.  This will open the Antibody Builder panel.

    <p>In the Antibody Builder panel,
    select an appropriate forcefield for proteins
    by pressing <b>Potential Setup</b>.
    In the Potential Setup panel, select <tt>AMBER99</tt> from
    the <b>Load</b> pulldown.  Set the solvation model
    by selecting <b>R-Field</b> from the <b>Solvation</b>
    pulldown. Press <b>OK</b>.
    </p>

    <p>
    Press <b>Build</b> to start creating the model.
    </p>

    </li></p><p><li>
    <i>Compare the model to the 3D9A crystal.</i>
    <br />
    Once the model has been created,
    load the crystal structure into the system.

    <blockquote>
    <span class="menu">
    MOE | File | Open | <tt>$MOE/sample/mol/3d9a.pdb</tt>
    </span>
    </blockquote>

    <p>Render the image for better visualization:</p>

    <p>
    <ul>
    	<li>
	<span class="menu">MOE | Hide | All Atoms</span>

	</li><li>

    	Choose a ribbon rendering using
	<span class="menu">MOE | Footer | Ribbons | Ribbon</span>.

	</li><li>

    	Color the ribbon by chain with
	<span class="menu">MOE | Footer | Ribbons | Color | Chain</span>.

	</li>

    </ul>
    </p>

    <p>
    Superpose the structures using the Protein Superpose panel:

    <blockquote>
    <span class="menu">
    MOE | Compute | Biopolymer | Superpose
    </span>
    </blockquote>

    <p>Press <b>Selection Hint</b>,
    followed by <b>Superpose</b>. In the image, the magenta and orange
    ribbons represent the model, while the other ribbons represent the crystal
    structure.
    </p>

    </li></p>

    <p>
    <center>
    <img src="antibody/fabExample.png"  alt="Fab Example">
    </center>
    </p>

    <p>
    Note the antigen (turquoise, on the right) in the X-ray structure.
    </p>
</ol>
</div>


<!-- ========================== Database Update ======================== -->

<h2><a name="FabDBUpdate">Compiling and Updating the Antibody Database</a></h2>
<p>
As mentioned above, the antibody structure database is a key component in MOE's knowledge-based Antibody Homology Modeler.

<a name="MethodCompileDB"></a><h3>Compiling the Antibody Database</h3>

<p> Once the V<sub>L</sub> and V<sub>H</sub> reference sets are established, the
MOE antibody database is then compiled by screening the entire protein
collection of the Protein Data Bank (PDB) for immunoglobulins. The sequence of
each protein chain is aligned  against MOE's V<sub>L</sub> and V<sub>H</sub>
reference sets and the  sequence identity is calculated as percentage of
identical residues within  the alignment to the F<sub>V</sub> domain (overall)
as well as
the  framework region (FR1, FR2 and FR3).  Chains are classified as antibodies
if the overall sequence identity with one  of the reference collections exceeds
40%. The current antibody collection (MOE 2009) is composed of 1,531
V<sub>L</sub>  and 1,488 V<sub>H</sub> chains.

<p>Classifying a protein chain as either V<sub>L</sub> or V<sub>H</sub> requires a
framework sequence identity higher
than 60% with the one class and less than 50% with the other class.
If a protein chain is detected as a V<sub>L</sub> chain, further sub-classification into
either <i>kappa</i> or <i>lambda</i> subclasses is achieved by alignment with
reference germline sequences originating
from the Immunogenetics database [Lefranc&nbsp;1997]. The cutoff value for
subclass assignment
is 65% sequence identity (composed of FR1, CDR1, FR2, CDR2, FR3 and parts
of CDR3).
Finally, the CDR loop residues are assigned based on the alignment positions
with the reference.

<p>
The result of an antibody structure scan on the Protein Data Bank (PDB) is
shown in Figure 8 as a correlation plot of framework identity to both
V<sub>L</sub> and V<sub>H</sub> reference sets.

<p> In addition to the structure coordinates, a variety of complementary
information regarding the antibody chain is stored in an database entry in the
data fields as following:

<p><table border="1" cellpadding="3" width="100%">
<tr>
  <td bgcolor="#cccccc"><b>Field</b></td>
  <td bgcolor="#cccccc"><b>Description</b></td>
</tr>
<tr>
  <td valign=top><b><tt>mol</tt></b></td>
  <td>

  A molecule field storing the structure data of the antibody chain.</td>

</tr>
<tr>
  <td valign=top><b><tt>Code</tt></b></td>
  <td>

  A character field specifying the PDB code of the structure.</td>

<tr>
<tr>
  <td valign=top><b><tt>Chain</tt></b></td>
  <td>

  A character field specifying the chain name of the antibody chain.</td>

</tr>
<tr>
  <td valign=top><b><tt>Resolution</tt></b></td>
  <td>

  The X-ray structure resolution as specified in the PDB header.</td>

</tr>
<tr>
  <td valign=top><b><tt>R_Value</tt></b></td>
  <td>

  The X-ray structure R-Value as specified in the PDB header.</td>

</tr>
<tr>
  <td valign=top><b><tt>R_free</tt></b></td>
  <td>

  The X-ray structure R<sub>free</sub> value as specified in the PDB header.</td>

</tr>
<tr>
  <td valign=top><b><tt>Date</tt></b></td>
  <td>

  The submission date of the structure into the PDB.</td>

</tr>
<tr>
  <td valign=top><b><tt>Class</tt></b></td>
  <td>A character field specifying the antibody class (VL or VH)</td>
</tr>
<tr>
  <td valign=top><b><tt>Subclass</tt></b></td>
  <td>

  A character field specifying the subclass of antibody chains. This field is
  only relevant for VL chains.</td>

</tr>
<tr>
  <td valign=top><b><tt>Source</tt></b></td>
  <td>

  A character field specifying the species of the antibody chain.</td>

</tr>
<tr>
  <td valign=top><b><tt>SciSource</tt></b></td>
  <td>

  A character field specifying the scientific notation of the species.</td>

</tr>
<tr>
  <td valign=top><b><tt>Segments</tt></b></td>
  <td>

  A vector of 7 integers specifying the number of residues in antibody
  subdomains FR1, CDR1, FR2, CDR2, FR3, CDR3 and FR4 respectively.</td>

</tr>
<tr>
  <td valign=top><b><tt>bb_topology</tt></b></td>
  <td>

  This field is stores the raw data for the backbone topology for each of the
  seven antibody subdomains (FR1-4, CDR1-3). The vector length is seven and
  serves as the basis for calculating the backbone topology score.</td>

</tr>
<tr>
  <td valign=top><b><tt>bb_geom</tt></b></td>
  <td>

  This field is stores the raw data for the backbone geometry data for each of
  the seven antibody subdomains (FR1-4, CDR1-3). The vector length is seven, the
  values are the counts of outliers (Z-scores > 4.0) of the dihedral angles
  phi/psi and omega per subdomain. </td>

</tr>
<tr>
  <td valign=top><b><tt>bb_BFactor</tt></b></td>
  <td>

  This field is stores the B-Factor values of the backbone atoms N, CA, C and O
  for each of the seven antibody subdomains (FR1-4, CDR1-3). The vector length
  is seven, the values represent the average of the B-Factor values over the
  respective subdomain.</td>

</tr>
<tr>
  <td valign=top><b><tt>bb_occ</tt></b></td>
  <td>

  This field is stores the occupancy values of the backbone atoms N, CA, C and O
  for each of the seven antibody subdomains (FR1-4, CDR1-3). The vector length
  is seven, the values represent the average occupancies over the respective
  subdomain.</td>

</tr>
<tr>
  <td valign=top><b><tt>bb_CAonly</tt></b></td>
  <td>Flag to indicate if the structure is a CA-trace structure. </td>
</tr>
<tr>
  <td valign=top><b><tt>Scores</tt></b></td>
  <td>

  The structure scores as calculated from the structure raw data stored in
  columns <i>bb_geom</i>, <i>bb_topology</i>, <i>Resolution</i>, <i>R_Value</i>
  and <i>R_free</i>. The total length per entry is 32.<br>

  <pre>
        1:  Overall structure score
      2-7:  Topology scores of subdomains FR1, CDR1, FR2, CDR2, FR3, CDR3
     8-13:  Geometry scores of subdomains FR1, CDR1, FR2, CDR2, FR3, CDR3
    14-19:  B-Factor scores of subdomains FR1, CDR1, FR2, CDR2, FR3, CDR3
    20-25:  Occupancy scores of subdomains FR1, CDR1, FR2, CDR2, FR3, CDR3
       26:  Score for Resolution
       27:  R-Value/R<sub>free</sub> scores
    28-31:  Residue length of domains FR, CDR1, CDR2, CDR3
       32:   flag to specify, if antibody structure is a CA-trace structure
  </pre>
  </td>
</tr>
<tr>
  <td valign=top><b><tt>Dimer</tt></b></td>
  <td>

  A character field to specify the chain code of the chain
  with which the current entry chain forms a F<sub>V</sub>
  dimer complex.
  </td>

</tr>
<tr>
  <td valign=top><b><tt>DimerType</tt></b></td>
  <td>

  A character field to specify the type of antibody F<sub>V</sub> dimer.
  In most cases, the F<sub>V</sub> complex is composed of an antibody light and
  heavy chain (VL:VH), for
  camel antibodies the entry is VH:VH and for rare and engineered
  F<sub>V</sub>s the entry
  is VL:VL. This entries are used to filter the
  <span class="nobr">VL + VH</span>
  result sets according to
  the classes specified by the search queries.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>Complex</tt></b></td>
  <td>

  A character field to specify the type of antigen the antibody chain is
  complexed with. The values are single characters specifying "P" (Protein), "H"
  (Hapten) or "W" (crystallographic waters).</td>

</tr>
<tr>
  <td valign=top><b><tt>Comment</tt></b></td>
  <td>

  A character field to store additional information. Currently, a note is stored
  in this field, if an antibody chain was detected as a sc
F<sub>V</sub> chain.</td>

</tr>
<tr>
</table>

<!-- ======================= Family File ====================== -->

<a name="MethodFabFam"></a><h3>Antibody Family File</h3>

<p> For efficient template searching, the chains in the database are grouped by
class (V<sub>L</sub> or V<sub>H</sub>) and species.  Additionally, the light
chains are further grouped into <i>kappa</i> or <i>lambda</i> subclusters. The
immunoglobulin clusters obtained from the structure database SCOP
[Chothia&nbsp;1997] are used as seeds for the clustering procedure: each chain
of the antibody database is aligned with clusters of matching class, subclass
(V<sub>L</sub> only) and species.  The framework identity (composed of the
subdomains FR1, FR2 and FR3) with each member of these clusters is calculated
and new clusters are formed if the identity is lower
than 60%. Otherwise, the antibody chain is added to the cluster with the
highest identity.

<p> After updating the antibody database, the newly formed or enriched clusters
are  further subclustered based on structural superposition of framework
residues. The Root Mean Square Distances (RMSD in &Aring;)  are calculated upon
superposition of the framework CA atoms  and a single-linkage clustering
calculation is applied to determine clusters at a resolution of 0.75 &Aring;.
The database clustering scheme is illustrated in Figure 8. The alignments of the
clustering scheme are stored in a text file termed <tt>fab.fam</tt> located in
the MOE working directory. This file is the core of the antibody template
search.

<p>
<div align=center style="padding:15px;">
<img src="antibody/abcoll.png"  alt="Antibody Family Clustering">
<br>
<b>Figure 8:</b> <i>Antibody Database Clustering Scheme</i>
</div>


<!-- ======================= Datbase Update ====================== -->

<a name="MethodUpdate"></a><h3>Updating the Antibody Database</h3>

<p> Since knowledge-based antibody homology modeling is dependent on the wealth
and variety of structures in the antibody database, it is important to implement
a continuous and fully automated update system in order to keep the knowledge
base current. An antibody database update can be performed via an user interface
or via a MOE/batch script. The system is based on an incremental updating scheme
that keeps a record of structures already integrated in the system. The base
source for updating is the Protein Data Bank (PDB), but in-house antibody
structures may be integrated as well.

<p>The graphical version of the antibody database update can be launched at
the <a href="../moe/hellomoe.htm#GUI_Basics">CLI</a> by typing

<pre>
    svl> run 'fab_update.svl'
</pre>

or
through the main MOE panel  by  <b><nobr>MOE | File | Open</b></nobr>. Select
<tt>fab_update.svl</tt> located in the
<span class="nobr"><tt>$MOE/lib/svl/run</tt></span> directory and press
<b>Run SVL File</b> in the dialog panel.

<div align=center style="padding:15px;">
  <img src="antibody/update.png"  alt="Antibody Database Update Panel">
</div>

The antibody database update procedure is based on a differential system, using
the current antibody database as the starting point along with a structure
family file and a look-up file. The look-up file is a text file that lists the
protein structures files inspected and/or used in a previous update. Once the
antibody update is successfully completed, three new files are created:
the new antibody database, a new family file and an updated
structure look-up file. These files are then used as input for the next antibody
database update.

<p>
<table width="100%" cellpadding="3">
    <tr>
      <td valign="baseline"><b>Source Structures</b></td>
      <td valign="baseline">

      Specifies the name of the input antibody database, usually the most recent
      database. By default, the antibody database in the $MOE/lib directory is
      specified but should be changed to the most recent version. <p>

      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Output Database</b></td>
      <td valign="baseline">

      Specifies the name of the new antibody database after the antibody update
      procedure is completed. <p>

      </td>
    </tr>
    <tr>
      <td valign="baseline"><b><nobr>Overwrite Existing Entries</nobr></b></td>
      <td valign="baseline">

      By default, existing chain entries in the current database are not
      overwritten. If on, then this behavior is changed to allow rewriting of
      existing entries. This setting is useful, if changes in a structure file
      have been made and the database needs to be updated.  <p>

      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Source Family File</b></td>
      <td valign="baseline">

      Specifies the name of the antibody family file used to
      seed the framework clusters. By default, a clustering seed file is
      specified in <tt>$MOE/lib/fabseeds.fam</tt>. This file is an excerpt of
      the immunoglobulin clustering scheme of the SCOP database
      [Chothia&nbsp;1997]. This default seed file will enforce structure
      clustering of most of the current antibody structures. If the update
      interval is short and only a few new antibody structures are submitted
      into the system, then the previous fam-file might be selected as input in
      order to speed up the clustering procedure. <p>

      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Target Family File</b></td>
      <td valign="baseline">

      Specifies the name of the antibody family file once the antibody update
      procedure is completed. <p>

      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Diff File</b></td>
      <td valign="baseline">

      Specifies the name of the look-up file. This file is a text file with a
      list of protein structures that have been considered in previous antibody
      database updates. Using this file, most entries in the PDB are bypassed
      and only the new submissions are scanned for immunoglobulins. <p>

      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Target Diff File</b></td>
      <td valign="baseline">

      Specifies the name of the new protein structure look-up file once the
      antibody update procedure is successfully completed. <p>

      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Structure Directory</b></td>
      <td valign="baseline">

      Specifies the top level directory where the protein structures are
      located. All subdirectories will searched for structure files.  <p>

      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>File Pattern</b></td>
      <td valign="baseline">

      Specify a file pattern with which the structure files can be identified. For
      structure files from the PDB, the files are searched by the extension
      <tt>.ent.gz</tt> (compressed format) or <tt>.ent</tt>.  <p>

      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Resolution</b></td>
      <td valign="baseline">

      As outlined above, the final clustering step is based on superposition of
      framework backbone atoms. Specify the resolution (in Angstroms) in
      framework residues of the final antibody clusters. <p>


      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>&nbsp;</b></td>
      <td valign="baseline">

      The antibody database  update does not always have to be conducted by
      screening structure files for immunoglobulins followed by reclustering of
      the entries. Alternatively, it is possible to perform only the clustering
      steps of the process. Then only a new fam-file will be generated. This
      type of update is performed by checking <b>Generate Family File
      Exclusively</b>, in which case the fields in the interface that are
      relevant for screening new input structures are deactivated.

      <p> In order  to increase the performance while clustering antibody
      chains, only the clusters with modified entries are subjected to the final
      clustering scheme. Activating the checkbox <b>Recluster All</b> will
      override this default behavior and all the clusters will then be
      clustered according to the specified resolution.

      </td>
    </tr>
    <tr>
</table>

<p> In order to update the antibody structure database, it is not necessary to
employ a computing cluster. A full compilation of the antibody database as seen
in <tt>$MOE/lib/fab.mdb</tt> scanning the entire PDB (59,842 structure files of
September 2009) was finished in approximately 10 hours on a Macintosh Mac Pro, 2
Dual-Core Intel Xeon Processors at 2.66 GHz. An update performed every 14 days
(on average approximately 250 structures)
including a full clustering of the database
can be done in approximately 25 minutes on the same machine.
<b>A full scan of the PDB
is required the first time the system is used.</b>

<p> A convenient method to update the antibody database is to launch the
application using MOE/batch. The example below shows an example using
<b>bash</b> in a Unix-type operating system:

<pre>
#!/bin/sh -x

MOE=/path_to_moe                        #MOE directory

pdbdir=/your_top_level_structure_dir    #directory of the PDB protein files
pattern='*.ent'                         #file pattern of protein files
targetdir=/specify/target/directory     #target directory of the new files


        #specify the relevant input files from the previous update
        #database, fam-file and structure look-up file

lastmdb=/specify/the/name/here.mdb      #current antibody database
lastfam=$MOE/lib/fabseeds.fam           #or previous fam-file
laststruct=/specify/the/name/here.txt   #current look-up structure file

newmdb=$targetdir'/fab.mdb'             #new fab database
newfam=$targetdir'/fab.fam'             #new fam file
newstruct=$targetdir'/struct.txt'       #new look-up file

mkdir $targetdir                        #create the new directory
chmod 766 $targetdir


$MOE/bin/moebatch -load "fabutils.svl" \
    -exec "run ['fab_update.svl', ['$lastmdb', '$newmdb', '$lastfam', '$newfam',
    '$laststruct', '$newstruct', '$pdbdir', [overwrite:0,
    filepattern:'$pattern']]]" \
    -licwait -exit
</pre>

<!-- =============================== FOOTER =============================== -->

<a name="SeeAlso"></a><h2>See Also</h2>

<p>
<a href="prosearch.htm">Homology Searching</a><br />
<a href="promodel.htm">Building Homology Models</a><br />
<a href="../apps/progeom.htm">Protein Geometry</a><br />
<a href="align.htm">Sequence and Structure Alignment</a>


<a name="References"></a><h2>References</h2>

<table cellpadding="5" width="100%">
<tr>
<td valign="top">[Chothia&nbsp;1987]</td>
<td valign="top">Chothia, C., Lesk, A.M.;
Canonical Structures for the Hypervariable Regions of Immunoglobulins;
<i>J. Mol. Biol. 196</i> (<b>1987</b>) 901&ndash;917.
</td></tr>

<tr>
<td valign="top">[Chothia&nbsp;1997]</td>
<td valign="top">Murzin, A.G., Brenner, S.E., Hubbard, T., Chothia, C.;
SCOP: A Structural Classification of Proteins Database for the
Investigation of Sequences and Structures;
<i>J. Mol. Biol. 247</i> (<b>1995</b>) 536&ndash;540.
<a href="http://scop.berkeley.edu/">http://scop.berkeley.edu/</a>.
</td></tr>

<tr>
<td valign="top">[Honegger&nbsp;2001]</td>
<td valign="top">Honegger, A., Pl&uuml;ckthun A.;
Yet Another Numbering Scheme for Immunoglobulin Variable Domains: An Automatic
Modeling and Analysis Tool;
<i>J Mol Biol. 309</i> (<b>2001</b>) 657&ndash;670.
</td></tr>

<tr>
<td valign="top">[Hudson&nbsp;2003]</td>
<td valign="top">Hudson, P.J., Souriau, C.;
Engineered Antibodies;
<i>Nature Medicine 9</i> (<b>2003</b>) 129&ndash;134.
</td></tr>

<tr>
<td valign="top">[Kabat&nbsp;1970]</td>
<td valign="top">Wu, T.T., Kabat, E.A.;
An Analysis of the Sequences of the Variable Regions of Bence-Jones Proteins
and Myeloma Light Chains and Their Implications for Antibody Complementarity;
<i>J. Exp. Med. 132</i> (<b>1970</b>) 211&ndash;250.
</td></tr>

<tr>
<td valign="top">[Kleywegt&nbsp;1996]</td>
<td valign="top">Kleywegt, G.J., Br&uuml;nger, A.T.; Checking your imagination:
application of the free R value;
<i>Structure 4</i> (<b>1996</b>) 897&ndash;904.
</td></tr>

<tr>
<td valign="top">[Lefranc&nbsp;1997a]</td>
<td valign="top">Lefranc, M.-P.;
Unique Database Numbering for Immunogenetic Analysis;
<i>Immunology Today 18</i> (<b>1997</b>) 509.
</td></tr>

<tr>
<td valign="top">[Lefranc&nbsp;1997b]</td>
<td valign="top">Lefranc, M.-P., Giudicelli, V., Ginestoux, C.,Bodmer, J.,
M&uuml;ller, W., Bontrop, R., Lemaitre, M., Malik, A., Barbi&eacute, V.,
Chaume, D.;
IMGT, The International ImMunoGeneTics Database;
<i>Nucleic Acids Research 27</i> (<b>1997</b>) 209&ndash;212.
</td></tr>

<tr>
<td valign="top">[Martin&nbsp;2002]</td>
<td valign="top">Allcorn, L.C., Martin, A.C.R.;
SACS - A Self-Maintaining Database of Antibody Crystal Structures;
<i>Bioinformatics 18</i> (<b>2002</b>) 175&ndash;181
<a href="http://acrmwww.biochem.ucl.ac.uk/index.html">
http://acrmwww.biochem.ucl.ac.uk/index.html</a>.
</td></tr>

<tr>
<td valign="top">[Morea&nbsp;2000]</td>
<td valign="top">Morea, V., Lesk, A.M., Tramontano, A.;
Antibody Modeling: Implications for Engineering and Design;
<span class="nobr"><i>Methods 20</i></span> (<b>1970</b>) 267&ndash;279.
</td></tr>

<tr>
<td valign="top">[Sonnhammer&nbsp;2005]</td>
<td valign="top">Sonnhammer E., Hollich V.; Scoredist: A Simple and Robust
Protein Sequence Distance Estimator;
<i>BMC Bioinformatics 6</i> (<b>2005</b>) 108&ndash;116.
</td></tr>

<td valign="top">[Taylor&nbsp;2001]</td>
<td valign="top">Taylor, P.C., Williams, R.O., Maini, R.N.;
Immunotherapy for Rheumatoid Arthritis;
<span class="nobr"><i>Current Opinion in Immunology 13</i></span>
(<b>2001</b>) 611&ndash;616.
</td></tr>

<tr>
<td valign="top">[Vriend&nbsp;1990]</td>
<td valign="top">Vriend, G.; WHAT IF: A Molecular Modeling and Drug Design
Program;
<i>J. Mol. Graph. 8</i> (<b>1990</b>) 52&ndash;56.
<a href="http://swift.cmbi.ru.nl/gv/pdbreport/checkhelp/explain.html#bfac">
http://swift.cmbi.ru.nl/gv/pdbreport/checkhelp/explain.html#bfac</a>.
</td></tr>

</table>
<p><a href="../index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
