<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<head>
    
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../manstyle.css" />


<title>Homology Model Tutorial</title>
</head>

<body bgcolor="#ffffff">
<a href="../index.htm"><img src="../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>Homology Model Tutorial</h2>
<hr noshade="noshade" />

<p>In this tutorial we will introduce MOE's Protein Modeling applications:</p>
<a href="../proteins/prosearch.htm">MOE-SearchPDB</a>,
<a href="../proteins/align.htm">MOE-Align</a>,
<a href="../proteins/promodel.htm">Homology Model</a>, and
<a href="../apps/progeom.htm">Protein Geometry</a>.
The tutorial has been broken down into the following steps:

<ul>
<li><a href="#SearchingCluster">Performing a Homology Search</a></li>
<li><a href="#BuildingModel">Building a Homology Model</a></li>
<li><a href="#EvaluatingModel">Evaluating the Homology Model</a></li>
</ul>

<h3>Conventions Used in this Tutorial</h3>

<table cellpadding="3">
<tr>
<td valign="top" bgcolor="#cccccc">MOE</td>
<td valign="top">MOE Window</td>
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">SE</td>
<td valign="top">Sequence Editor</td>
</tr>
<tr>
<td valign="top" bgcolor="#cccccc">DBV</td>
<td valign="top">Database Viewer</td>
</tr>
</table>

<p>In this tutorial, we will assume that you are using a three-button mouse.
For information on using a two-button mouse see
<a href="../moe/gui/mouse.html">Using the Mouse</a>.</p>



<h3><a name="SearchingCluster">Performing a Homology Search</a></h3>

<ol>
<li><p> Close the current system using </p>
<blockquote>
<span class="menu">MOE | File | Close</span>
</blockquote>
</li>

<li><p>Open the file containing the protein sequence for which we will build 
a homology model. We will be using the sequence for one of the comparative
modeling targets from the Fourth Critical Assessment of Structure
Prediction (<a href="http://predictioncenter.org/casp3/">CASP3</a>)
"blind" test of protein structure prediction methods. The sequence for target
69 is in the MOE sample directory:</p>

<blockquote>
<span class="menu">MOE | File | Open  | <tt>$MOE&#47;sample&#47;mol&#47;casp_69.pir</tt></span>
</blockquote>

<p>After you have loaded this file into MOE, you will see a single chain
object in Sequence Editor window. To display
chain names and to display sequences in single-letter mode, choose:</p>

<blockquote>
<span class="menu">SE | Display | Compound Name</span> <br />
<span class="menu">SE | Display | Single Letter Residues</span>
</blockquote>

<p align="center">
<img src="hom_tut/t69seq.png" alt="MOE-SearchPDB" /></p>
</li>

<li><p>We will now use the <b>MOE-SearchPDB</b> application to search for a suitable template for homology modeling. The search is performed on a database of
protein structures and sequences that have been clustered into families.
Open the <b>MOE-SearchPDB</b> panel with</p>
<blockquote>
<span class="menu">SE | Homology | PDB Search</span>
</blockquote>

<p align="center">
<img src="hom_tut/spanel.png" alt="MOE-SearchPDB" />
</p>

<p>Press the <b>1</b> from the buttons beside the <b>Chain:</b> label
at the top right corner of the panel to load the t69 sequence into 
the search area and press <b>Search</b>.</p>

<p>As the search progresses, pre-aligned families appear in the Results section
of the panel with their calculated E-values. Once the screening of E-values is
complete, the Z-Scores are calculated and the Z-Score threshold applied. Note
that if the E-value is sufficiently low, the Z-Score is not calculated
as it presumably will be large.
(See <a href="../proteins/prosearch.htm">Homology Searching</a>
for more details.)</p>

<p>In this example, the search identifies members of the C-type lectin-like
superfamily as homologues of our target sequence.
Each item in the Results section of the panel represents one
or more PDB structures, possibly augmented by sequence-only entries from
the Uniref50 database, clustered into a family and aligned by a sequence
and structure based alignment protocol. The displayed code represents the 
PDB structure from within the family that scored the highest against the
target sequence.</p>
</li>

<li><p>Double-click on <tt>PDB_3G81.A</tt> in the <b>Results</b> area,
or select the result and press the <b>Load&nbsp;Alignment</b> button. The
<span class="nobr">MOE-SearchPDB: Load Alignment</span> panel appears, displaying the
pre-aligned family. Note that the PDB structure <tt>3G81.A</tt>, which was displayed
in the Search results area, is merely one of a large number of PDB structures
in this family. </p>

<p align="center">
<img src="hom_tut/ldalign.png" alt="MOE-SearchPDB: Load Alignment" />
</p>

</li>

<li><p>Press <b>Load&nbsp;All</b> to load the entire protein
family into MOE. You may now close the MOE-SearchPDB panel.  To view the protein sequences that have been loaded,
open the Sequence Editor using</p>
<blockquote>
<span class="menu">MOE | Window | Sequence Editor</span>
</blockquote>
<p>In the Sequence Editor, you now see the target sequence (Chain 1) as well
as the family of homologous proteins that you just loaded.
In order to see the secondary structure information for the
protein structures you have just loaded, select</p>
<blockquote>
<span class="menu">SE | Display | Actual Secondary Structure</span>
</blockquote>

<p>The colored bars over the sequences reflect the secondary structure of
those sequences having associated atomic coordinates.  The target sequence,
Chain 1, is a <i>sequence-only</i> chain, and therefore has no such bars.</p>

<p align="center">
<img src="hom_tut/alignsec.png" alt="Secondary Structures" />
</p>

</li>

</ol>


<h3><a name="BuildingModel">Building a Homology Model</a></h3>

<p>In order to build a homology model, we will align the target sequence to the
protein family, decide which chain is to be the template for our model,
and then build the model.</p>
<ol>
<li><p>Select the t69 chain in the Sequence Editor and then invert the chain
selection with </p>
<blockquote>
<span class="menu">SE | Selection | Invert Chains</span>
</blockquote>

<p align="center">
<img src="hom_tut/invsel.png" alt="Selected Chains" /></p>
</li>

<li><p>Now we will use the <b>MOE-Align</b> application to align the target 
sequence to the C-type lectin-like protein superfamily.
Start <b>MOE-Align</b> with</p>
<blockquote>
<span class="menu">SE | Homology | Align</span>
</blockquote>

<p align="center">
<img src="hom_tut/align.png" alt="MOE-Align" /></p>

<p>In the <b>MOE-Align</b> panel, click the Selected: <b>Freeze</b> button.
This <b>Freeze</b> option allows us to perform a
<i>sequence-to-group</i> alignment between the target sequence and the
entire protein family - that is, the alignments of each of the selected chains
to each other will not be disturbed. Press <b>OK</b> to perform the alignment. 
</p>

<p>Target 69 has now been aligned to the protein family.</p>
<p align="center">
<img src="hom_tut/t69aligned.png" alt="Target Aligned" /></p>
</li>

<li><p>When <b>MOE-Align</b> finishes calculating the sequence alignment,
it writes a  pairwise sequence identity matrix
to the SVL Commands Window.  Open the SVL Commands Window with</p>
<blockquote>
<span class="menu">SE | Window | Commands</span>
</blockquote>

<p align="center">
<img src="hom_tut/matrix.png" alt="Residue Identity Matrix" />
</p>

<p>The sequence identity matrix is non-symmetric.  The percentage
sequence identity is calculated from the number of identical residues
between both chains, divided by the number of residues in the column
chain for the top right half of the matrix, and the number of residues
in the row chain for the bottom left half of the matrix.</p>
</li>

<li><p>The chain with the highest sequence identity to the query
sequence is from <tt>3G81.A</tt>.  Hold the shift key down and 
click on the first two chains in the sequence editor. Then
select the identical residues in these chains using</p>
<blockquote>
<span class="menu">SE | Selection | Conserved Residues | Residue Identity</span>
</blockquote>

<p align="center">
<img src="hom_tut/resident.png" alt="Select Identical Residues" />
</p>

<p>Then click the <i>mouse popup button</i> on the gray background 
of the Sequence Editor window and set :</p>
<blockquote>
<span class="menu">Selected Residues | Color | Red</span>
</blockquote>

<p>Deselect the residues to show the identical residues are colored red.</p>

<p>Scroll along the sequence alignment and compare the sequences.
Near position 140, gaps are placed to align against sequences in the
rest of the cluster, but the alignment between the t69 and
<tt>3G81.A</tt> chains could be improved. The sequence <tt>NWADGEPN</tt> in
t69 is not aligned with <tt>NWAPGEPN</tt> in <tt>3G81.A</tt></p>

<p align="center">
<img src="hom_tut/mismatch.png" alt="Mismatched Sequences" />
</p>

<p>The sequence identity matrix in the SVL commands window shows 
that the first five of the protein template chains have a sequence identity
of greater than 25% to t69. These are from <tt>3G81.A</tt>, <tt>1HUP.A</tt>, 
<tt>1R13.A</tt>, <tt>1BV4.A</tt> and <tt>1AFA.1</tt>.  
</p>

<p>To align the t69 sequence with the five closest sequences, select 
chains 1-6 in the Sequence Editor.  Then open <b>MOE-Align</b> with:</p>
<blockquote>
<span class="menu">SE | Homology | Align</span>
</blockquote>
<p>Click the Selected Chains: <b>Align</b> button.
The selected chains will be aligned against each other, ignoring
the other chains.  Press <b>OK</b> to start the alignment.</p>

<p align="center">
<img src="hom_tut/realign.png" alt="MOE-Align" /></p>

<p>
Note in the SVL Commands Window that the sequence identity has increased from
53.8% to 59%. </p>

<p>Select the t69 and <tt>3G81.A</tt> chains again and select the identical 
residues between then using:</p>
<blockquote>
<span class="menu">SE | Selection | Conserved Residues | Residue Identity</span> 
</blockquote>
<p>Note that the alignment is improved near position 140, and that 
all of the residues which were colored red earlier, are still 
selected, so the alignment has only been improved.  </p>
</li>

<li><p>You can now build the homology model. 
Open <b>Homology&nbsp;Model</b> with</p>
<blockquote>
<span class="menu">SE | Homology | Homology Model</span>
</blockquote>

<p align="center">
<img src="hom_tut/hompanel.png" alt="MOE Homology Model" /></p>
</li>

<li><p>There is only one sequence loaded in MOE with no associated atomic
data (Chain 1); it is therefore detected by default as the sequence to
be modeled.  it therefore is by default the only sequence to be modeled.
Its <b>Template</b> will default to the first chain in the system which
has associated atomic coordinates: Chain 2 (<tt>3G81.A</tt>).</p>

<p>Select <b>Open&nbsp;Database&nbsp;Viewer</b> near the top of the
<b>Homology&nbsp;Model</b> panel.</p>
</li>

<li><p>The currently loaded forcefield is displayed at the bottom of the
panel. By default, MOE uses a version of the Merck Molecular Forcefield
[Halgren], however, this forcefield is not specifically designed for
protein simulations and modeling. Press <b>Potential&nbsp;Setup</b> to select
a more appropriate forcefield. We suggest Amber99 [Wang 2001] with a
Reaction Field (<b>R&ndash;Field</b>) treatment of solvation
electrostatics [Tironi 1995] for this exercise. Homology Model will
creates or destroys hydrogens and lone pairs as necessary to complete 
outer valence electron shells and assigns partial charges to all atoms
so there is no need to press <b>Fix&nbsp;Hydrogens</b> and 
<b>Fix&nbsp;Charges</b>.</p>

<p>Finally, press <b>OK</b> to put the changes into effect close the 
<b>Potential&nbsp;Setup</b> window.</p>

<p align="center">
<img src="hom_tut/amber99.png" alt="Potential Setup" /></p>
</li>

<li><p>In the Homology Modeling panel Press <b>OK</b>. In the output database 
<tt>promodel.mdb</tt> you can view the results as they are being calculated.</p>
</li>
</ol>

<p><b>Notes:</b></p>
<ul>

<li><p>The entire system is saved prior to modeling as 
a MOE file specified in the <b>Current&nbsp;System</b> field.
The default filename is <tt>promodel.moe</tt>.</p>
</li>

<li><p>The various models will be written to the database specified in the
<b>Output&nbsp;Database</b> field.  The default filename is 
<tt>promodel.mdb</tt>.</p>
</li>

<li><p>The 
<span class="menu">Disable C-terminal &amp; N-terminal Outgap Modeling</span> 
option is, by default, turned on.  When outgaps are ignored, the amino acids of the target 
sequence that extend beyond (either before or after) the template sequence
in the alignment are clipped off before the model is built.</p>
</li>

<li><p>By default, ten independent intermediate models will be built.
These different homology models will be the result of the permutational
selection of different loop candidates and sidechain rotamers.</p>
</li>

<li><p>The intermediate model which scores best according to
the chosen scoring function is chosen as the final model,  
subject to optional further energy minimization.</p>
</li>

<li><p>Once the calculations have finished, the output database will include
the ten intermediate models and a refined final model (entry eleven).</p>
</li>
</ul>

<p>To get a better view of the molecules in the database, you can
enlarge the molecular cells in the Database Viewer.  Position the cursor
over any one of the cells in the first column (the <b>mol</b> field),
press the left mouse button and drag down (and slightly to the right)
until you clearly see all the molecular drawings. You can rotate each
molecule individually by dragging the middle mouse
button over it.</p>

<p align="center">
<img src="hom_tut/db_hom.png" alt="Homology Models in Database Viewer" />
</p>

<ol>
<li><p>When the calculation finishes, the final,
refined model will be loaded in the MOE window.  
Atom sets are created with the atoms which have
been copied from the template structure, and for
the newly-built atoms.</p>

<p>To color the atoms which differ from the template
structure, open the <b>Atom Selector</b>:</p>
<blockquote>
<span class="menu">MOE | RHS | Select | Atom Selector...</span> 
</blockquote>

<p>From the <b>Sets:</b> menu select <tt>A_new</tt> to select the
201 heavy atoms which differ from the <tt>3G81.A</tt> template structure.</p>

<p>Color the carbon atoms that are now selected using</p>
<blockquote>
<span class="menu">MOE | Footer | Atoms | Green</span> 
</blockquote>
<p>Then click on the background of the main MOE window to deselect the atoms.</p>

<table cellpadding="3">
    <tr>
	<td><p align="center">
	    <img src="hom_tut/aselect.png" alt="Atom Selector" />
	    </p>
	</td>
	<td><p align="center">
	    <img src="hom_tut/newatoms.png" alt="New Atoms" />
	    </p>
	</td>
    </tr>
</table>
</li>


<li><p>Then choose</p>
<blockquote>
<span class="menu">MOE | Render | Hide | All </span> 
</blockquote>
<p>and turn on backbone ribbon drawing using the yellow arrow button in </p>
<blockquote>
<span class="menu">MOE | Render | Ribbon </span>
</blockquote>
<p>to render the final model as shown here.</p>

<p align="center">
<img src="hom_tut/final.png" alt="Final Model" />
</p>
</li>

</ol>

<h3><a name="EvaluatingModel">Evaluating the Homology Model</a></h3>

<p>It is good practice to examine the stereochemistry of homology
models for unusual or geometrically unreasonable features. Serious
problems - particularly problems that cluster in one or a few
areas of the model can suggest a either a problem with the choice
or template, or, more likely, a problem with the alignment of
the target sequence to the template. </p>

<p>Investigate the stereochemical fitness of our model using
<b>Protein&nbsp;Geometry</b> from </p>
<blockquote>
<span class="menu">SE | Measure | Protein Geometry</span> 
</blockquote>

<p align="center">
<img src="hom_tut/progeom.png" alt="Protein Geometry" /></p>

<p>The user interface of the <b>Protein&nbsp;Geometry</b> application presents
data in the form of plots, such as the Ramachandran plot shown above,
and as tabular lists. Press <b>Export...</b> to generate an image from
which the severity of any psi-phi outliers in our model can be judged.</p>

<p align="center">
<img src="hom_tut/export.png" alt="Ramachandran" /></p>

<p>
<font color="#ff0000"><b>Important!</b></font>
<em>
MOE uses stochastic algorithms in its homology modeling, and results may differ slightly each time the model is constructed.  Therefore, your results and the results shown here may not be identical.
</em>
</p>

<p>Take a moment to examine the exported Ramachandran plots and then press
<b>Report...</b>. Press <b>OK</b> in the <b>Protein&nbsp;Report:&nbsp;Options</b>
prompt panel.</p>

<p align="center">
<img src="hom_tut/report.png" alt="Protein Geometry" /></p>

<p>Shown is a report of the outliers from the protein geometry calculations. You can scroll down to find the residues with dihedral angles that are outside the usually observed ranges. To examine the area of the model in which these outliers are found, switch the Protein Geometry display to <b>Data</b>.</p>

<p align="center">
<img src="hom_tut/report2.png" alt="Protein Geometry" /></p>

<p>Select the outlier(s) in the list, and press <b>Select&nbsp;Atoms</b> at the
bottom of the panel. Now, with the atoms of the outlier residue selected,
you may perform operations such as a fine grain energy minimization in an
attempt to obtain more typical phi and psi angles.</p>

<h2>Summary</h2>

<p>
In this tutorial, we have show how to use the protein modeling tools in MOE,
searching with a protein amino acid sequence, for a structure 
from a similar protein.  <b>MOE-Align</b> was used to refine the alignment 
between the sequence and the template.  A homology model was built using
MOE-Homology, and the geometry of this was checked using the
Protein Geometry application.</p>

<h2>References</h2>

<table cellpadding="5">
<tr> <!-- KEEP -->
<td valign="top">[Halgren&nbsp;1996]
</td>
<td>
Halgren, T.A.; The Merck Force Field; <i>J. Comput. Chem. 17</i> (<b>1996</b>) 490-641.
</td>
</tr>
<tr> <!-- KEEP -->
<td valign="top">[Tironi 1995]
</td>
<td>
Tironi, I.G., Sperb, R., Smith, P.E., Van Gunsteren, W.F.;
A Generalized Reaction Field Method for Molecular Dynamics Simulations;
<i>Journal of Chemical Physics 102</i> (<b>1995</b>) 5451-5459.
</td>
</tr>
<tr> <!-- KEEP -->
<td valign="top">[Wang&nbsp;2001]
</td>
<td>
Wang, J., Cieplak, P., Kollman, P.A.;
How Well Does a Restrained Electrostatic Potential (RESP) Model Perform in
Calculating Conformational Energies of Organic and Biological Molecules;
<i>J. Comput. Chem. 21</i> (<b>2001</b>) 1049-1074.
</td>
</tr>
</table>

<h2>See Also</h2>

<a href="../proteins/prosearch.htm">Homology Searching</a><br />
<a href="../proteins/align.htm">Sequence and Structure Alignment</a><br />
<a href="../proteins/promodel.htm">Building Homology Models</a><br />
<a href="../apps/progeom.htm">Protein Geometry</a><br />
<p></p>
<a href="../moe/pot.htm">Potential Energy Selection and Configuration</a><br />
<p></p>
<a href="../moe/gui/mainwin.htm">MOE Window</a><br />
<a href="../mdb/db.htm">Database Viewer Window</a><br />
<a href="../moe/gui/se.html">Sequence Editor Window</a><br />

<p><a href="../index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
