<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
<meta http-equiv="Content-Style-Type" content="text/css" />
<link rel="stylesheet" href="../support/help.css" charset="ISO-8859-1" type="text/css" />
<title>Multiple Sequence Viewer Panel</title>
</head>
<body>
<table bgcolor="#dcdcdc" border="0" cellspacing="0" width="100%">
<tr><td>
<p><img src="../images/common/schrodinger_logo.png" align="left" alt=""
 border="0" hspace="5" vspace="5" /></p>
</td></tr>
<tr><td>
<h1 class="title">Multiple Sequence Viewer Panel</h1>
</td></tr>
</table>

<ul>
<li><a href="#summary">Summary</a></li>
<li><a href="#opening">Opening the <span class="GUI">Multiple Sequence Viewer</span> Panel</a></li>
<li><a href="#using">Using the <span class="GUI">Multiple Sequence Viewer</span> Panel</a></li>
<li><a href="#features"><span class="GUI">Multiple Sequence Viewer</span> Panel Features</a></li>
<li><a href="#acknowledgement">Acknowledgement</a></li>
<li><a href="#links">Related Topics</a></li>
</ul>

<a name="summary"></a>
<h2>Summary</h2>

<p>The Multiple Sequence Viewer is an alignment, visualization, and
manipulation toolkit for multiple sequences.
</p>

<a name="opening"></a>
<h2>Opening the Multiple Sequence Viewer Panel</h2>

<p>To open the <span class="GUI">Multiple Sequence Viewer</span> panel, you can:</p>
<ul>
<li><p>Choose <span class="GUI">Tools &rarr; Multiple Sequence Viewer</span> 
in the main window.
</p></li>
</ul>


<a name="using"></a> 
<h2>Using the Multiple Sequence Viewer Panel</h2> 


<p>The <span class="GUI">Multiple Sequence Viewer</span> panel provides a range
of tools for manipulating and aligning multiple sequences.
</p>

<p>The Multiple Sequence Viewer (MSV) has its own projects, which contain all the
sequences in the project along with associated data. The project is stored in a
single file with a <code>.msv</code> extension, and by default is stored inside
the Maestro project. You can save it externally if you wish. </p>
<p>The project can include sequences imported directly into the project, and
sequences that are displayed in the Workspace. Directly imported sequences
remain in the project unless explicitly deleted. Sequences in the Workspace are
transient: when structures are inlcuded in or excluded from the Workspace, the
sequence is added to or removed from the project.</p>

<p>Although the data is stored separately from the Maestro project data, there
are interactions between the Workspace and the MSV, which depend on settings
that you make: 
</p>
<ul>
<li><p>Changes that are made in the Workspace are propagated to the MSV. These
changes cover inclusion and exclusion of entries; deletion, mutation, or
insertion of residues; and selection of residues. 
</p></li>
<li><p>Changes to the residue selection in the MSV can be propagated back to the
Workspace.
</p></li>
<li><p>Changes to the sequences in the MSV are <i>not</i> propagated back to the
Maestro structure.
</p></li>
<li><p>Color schemes can be transferred between the MSV and the Workspace.
</p></li>

</ul>


<h3>Speed Optimization Tips</h3>

<p>The speed of the sequence viewer depends on the number and length of sequences
loaded into the viewer. Typically, alignments of 20 sequences of 300 residues or
fewer can be interactively viewed and edited. Below are a few speed optimization
tips.</p>  

<ul>
<li>Hide the ruler (<span class="GUI">Settings &rarr; Display Ruler</span>
off)</li> 
<li>Use unwrapped mode (<span class="GUI">Settings &rarr; Wrap
Sequences</span> off)</li>
<li>Disable annotation grouping (<span class="GUI">Settings &rarr; Group
Annotations</span> off)</li> 
<li>Collapse all sequences (<span class="GUI">Sequences &rarr; Collapse All
Sequences</span>)</li>
<li>Hide annotations (<span class="GUI">Annotations &rarr; Clear All
Annotations</span>)</li>
<li>Reduce the size of the sequence area: move sequence area splitter to the
right, shrink entire window</li>
</ul> 
<p>Remember that operations involving Maestro sequences may depend on the
complexity of the Maestro workspace.</p>

<a name="features"></a> 
<h2>Multiple Sequence Viewer Panel Features</h2> 


<ul>
<li><a href="#feature1">Menu bar</a></li>
<li><a href="#feature2">Toolbars</a></li>
<li><a href="#feature3">Sequence display area</a></li>
<li><a href="#feature4">Status area</a></li>
</ul>


<a name="feature1"></a>
<h3>Menu bar</h3>
<p>
</p>

<ul>
<li><a href="#file"><span class="GUI">File</span> menu</a></li>
<li><a href="#edit"><span class="GUI">Edit</span> menu</a></li>
<li><a href="#sequences"><span class="GUI">Sequences</span> menu</a></li>
<li><a href="#alignment"><span class="GUI">Alignment</span> menu</a></li>
<li><a href="#color"><span class="GUI">Color</span> menu</a></li>
<li><a href="#annotations"><span class="GUI">Annotations</span> menu</a></li>
<li><a href="#tools"><span class="GUI">Tools</span> menu</a></li>
<li><a href="#maestro"><span class="GUI">Maestro</span> menu</a></li>
<li><a href="#settings"><span class="GUI">Settings</span> menu</a></li>
</ul>

<dl>


<a name="file"></a>
<dt><span class="GUI">File</span> menu</dt>
<dd><p>This menu provides tools for creating, opening, and saving Multiple
Sequence Viewer projects; importing and exporting sequences; saving images, and
closing the panel.
</p>
<dl>

<dt><span class="GUI">New Query</span></dt>
<dd><p>Create a new, empty query in a new tab.
</p></dd>

<dt><span class="GUI">Rename Query</span></dt>
<dd><p>Rename the current query. The name appears on the tab for the query.
</p></dd>

<dt><span class="GUI">Duplicate Query</span></dt>
<dd><p>Create a copy of the selected sequences in the current query in a new tab. 
</p></dd>

<dt><span class="GUI">Delete Query</span></dt>
<dd><p>Delete the current query. All sequences, structures, and data associated
with the query are removed (unless they are also in another query tab). 
</p></dd>

<dt><span class="GUI">Open</span></dt>
<dd><p>Open an existing Multiple Sequence Viewer (MSV) project. Opens a file
selector, in which you can navigate to and select the project.
</p></dd>

<dt><span class="GUI">Save</span></dt>
<dd><p>Save the current MSV project. If the project has not yet been saved,  a
file selector opens, in which you can navigate to a location and name the
project. If the project has been saved previously, the project is simply saved.
</p> </dd>

<dt><span class="GUI">Save As</span></dt>
<dd><p>Save the current MSV project with a new name. Opens a file selector, in
which you can navigate to a location and name the project. After saving, the
current project is the one with the new name.
</p> </dd>

<dt><span class="GUI">Import Sequences</span></dt>
<dd><p>Import sequences into the project. Sequences can be imported from a range
of file formats: FASTA, SWISSPROT, GCG, PIR, EMBL, as well as PDB and Maestro.
Opens a file selector, in which you can choose the file type, navigate to and
select the sequence file.
</p>
<p>The file selector has four options:</p>
<ul>
<li><span class="GUI">Align to query sequence</span>&mdash;When the sequences
are imported, align them to the query sequence.</li>
<li><span class="GUI">Replace matching sequences</span>&mdash;If an imported
sequence exactly matches a sequence that is already in the MSV, replace the
existing sequence with the imported sequence, while preserving the
alignment. If the imported sequence has structural data, import the data as
well. If there are multiple matching sequences in the MSV, replace only the
first instance.</li>
<li><span class="GUI">Translate DNA / RNA sequences</span>&mdash;Translate DNA
and RNA sequences into the sequence for the proteins they code for, using
standard genetic code. The protein sequence is imported instead of the nucleic
acid sequence.</li>
<li><span class="GUI">Incorporate PDB files into Maestro</span>&mdash;When PDB
sequences are imported, add the corresponding structures as entries to the
Maestro project. </li>
</ul>
<p>Structural data (ATOM records), B-factors, and secondary structure
assignments are also imported if the data are in the PDB file. A nonstandard
version of FASTA format is accepted, in which a residue can be preceded by its
residue number; numbering is otherwise sequential, starting from 1.
</p>
</dd>

<dt><span class="GUI">Export Sequences</span></dt>
<dd><p>Export sequences from the project to a file in FASTA format or as a plain
text file. Opens a file selector, in which you can navigate to a location and
name the file. The file selector has three options:</p>
<ul>
<li><span class="GUI">Save annotations</span>&mdash;Save the SSP and SSA
annotations in the file.</li> 
<li><span class="GUI">Save similarity values</span>&mdash;Save the calculated
percent similarity between sequences. The values are added to the sequence name
in the exported FASTA file as ID (identity), SIM (similarity), HOM (homology),
e.g.
<pre>
&gt;1abc|ID:7.14|SIM:19.64|HOM:17.86
</pre>
</li>
<li><span class="GUI">Export only selected part of the
alignment</span>&mdash;Export only the residues that are selected in all
sequences. </li>
</ul>
<p>These options only apply to export in FASTA format.</p>
</dd>

<dt><span class="GUI">Save Image</span></dt>
<dd><p>Save an image of the sequence viewer. Available formats are PNG, EPS, and
PDF. Opens a file selector, in which you can navigate to a location and name the
file. The selector has one option: <span class="GUI">Export image of the entire
alignment</span>, which is selected by default. If it is not selected, the image
is of only the visible part of the alignment (what you can see in the sequence
display area of the panel).
</p></dd>

<dt><span class="GUI">Close</span></dt>
<dd><p>Close the current Multiple Sequence Viewer (MSV) project and close the panel.
</p></dd>
</dl>

</dd>


<a name="edit"></a>
<dt><span class="GUI">Edit</span> menu</dt>
<dd><p>From this menu you can choose to undo or redo actions, edit sequences,
and delete sequences. The sequence editor panel accepts standard editor key
strokes, such as <span class="Keys">Ctrl+C</span> (<span class="Keys">&#x2318;C</span>), <span
class="Keys">Ctrl+X</span> (<span class="Keys">&#x2318;X</span>), and <span class="Keys">Ctrl+V</span> (<span class="Keys">&#x2318;V</span>) for copying,
cutting, and pasting text. 
</p>
<dl>

<dt><span class="GUI">Undo</span></dt>
<dd><p>Undo the last editing operation. The operation is appended to the menu item
text, for example <span class="GUI">Undo Load File</span>.
</p></dd>

<dt><span class="GUI">Redo</span></dt>
<dd><p>Redo the last undone editing operation. The operation is appended to the
menu item text, for example <span class="GUI">Redo Load File</span>
</p></dd>

<dt><span class="GUI">New Sequence</span></dt>
<dd><p>Create a new sequence by entering the letter codes for the residues. Opens
the <span class="GUI">Sequence Editor</span> dialog box, in which you can name
the sequence and type in the letter codes for the sequence.
</p></dd>

<dt><span class="GUI">Edit Sequence</span></dt>
<dd><p>Edit an existing sequence as a string of letter codes or create a new
sequence.  Opens the <span class="GUI">Sequence Editor</span> dialog box, in
which you can change the name of the sequence, and add or delete residues or
gaps by character code. Only the 20 standard amino acid codes, <code>X</code>
(unknown residue), and <code>-</code> and <code>~</code> (gap symbols) are
recognized. When you have finished editing, you can either replace the existing
sequence (if one was selected), or add it as a new sequence to the end of the
list.
</p></dd> 

<dt><span class="GUI">Paste in FASTA Format</span></dt>
<dd><p>Insert entire sequences into the sequence viewer in FASTA format. Opens
the <span class="GUI">Sequence Editor</span> dialog box, in which you can paste
the sequences and edit them. The same editing rules apply, except that lines
beginning with a <code>&gt;</code> character are treated as a sequence header.
These lines also separate one sequence from the next. Multiple sequences thus
delineated are saved as separate sequences.</p>
<p>
This feature is useful for copying an alignment from a web site and adding it
directly to the MSV without having to save a file. You can also add sequences by
importing a file.
</p></dd>

<dt><span class="GUI">Duplicate Selected Sequences</span></dt>
<dd><p>Duplicate the selected sequences, and place each duplicate immediately
below its parent.
</p></dd>

<dt><span class="GUI">Find Pattern</span></dt>
<dd><p>Find a pattern in the sequences. Displays the <a
href="#feature2.4">Find toolbar</a> if it is not displayed, and puts focus in
the text box so you can start typing the pattern.
</p></dd>

<dt><span class="GUI">Renumber Residues</span></dt>
<dd><p>Renumber the residues in one or more sequences. Opens the <span
class="GUI">Renumber Residues</span> panel, where you can either renumber the
selected sequences so that residues at the same position in the sequence viewer
have the same residue number; or import and align a template for a single
sequence, and apply the numbering of the template to the selected sequence.
</p></dd>

<dt><span class="GUI">Delete Selection</span></dt>
<dd><p>Delete the sequences that are selected in the viewer.
</p></dd>

<dt><span class="GUI">Remove Redundant Sequences</span></dt>
<dd><p>Remove sequences whose sequence identity is greater than a given
threshold. Opens a dialog box in which you can set the threshold, then click 
<span class="GUI">Remove</span> to perform the action. The default threshold is
100%. For each pair of sequences that are considered identical, the shorter of
the two is deleted; if they are of the same length, the second (lower down in
the sequence viewer) is deleted. In the latter case, you can change which
sequence is discarded by reordering the sequences. This task operates on all
sequences. If you click <span class="GUI">Cancel</span> in the dialog box, the
redundant sequences become the selection, so you can perform operations on
them. 
</p></dd>
</dl>

</dd>


<a name="sequences"></a>
<dt><span class="GUI">Sequences</span> menu</dt>
<dd><p>From this menu you can control which sequences are selected, which
sequences are shown, and the order in which they are listed.
</p>
<dl>

<dt><span class="GUI">Hide Selected</span></dt>
<dd><p>Hide the selected sequences.
</p></dd>

<dt><span class="GUI">Show All</span></dt>
<dd><p>Display all sequences.
</p></dd>

<dt><span class="GUI">Clear All</span></dt>
<dd><p>Delete all sequences from the project.
</p></dd>

<dt><span class="GUI">Select All</span></dt>
<dd><p>Select all sequences.
</p></dd>

<dt><span class="GUI">Deselect All</span></dt>
<dd><p>Deselect all sequences.
</p></dd>

<dt><span class="GUI">Invert Selection</span></dt>
<dd><p>Invert the selection of the sequences: select the unselected sequences,
and deselect the selected sequences.
</p></dd>

<dt><span class="GUI">Expand All</span></dt>
<dd><p>Expand all sequences so that the associated data, such as secondary
structure assignment, is displayed. This command is mapped to <span
class="Keys">Ctrl+DOWN ARROW</span> (<span class="Keys">&#x2318;DOWN ARROW</span>).
</p></dd>

<dt><span class="GUI">Collapse All</span></dt>
<dd><p>Hide the associated data for all sequences. This command is mapped to
<span class="Keys">Ctrl+UP ARROW</span> (<span class="Keys">&#x2318;UP ARROW</span>).
</p></dd>

<dt><span class="GUI">Set Color of Sequence Name</span></dt>
<dd><p>Set the color used for the name of the selected sequences. Opens a color
selector, in which you can select a color. This feature allows you to color-code
the names of the sequences. 
</p></dd>

<dt><span class="GUI">Sort by Tree Order</span></dt>
<dd><p>Order sequences by the phylogeny tree generated by ClustalW. The tree is
displayed in the leftmost part of the display area, which is normally
hidden. When this option is selected, the other sorting items are not available.
The nodes of the tree have a shortcut menu, with items <span class="GUI">Swap
Branches</span> to swap the order of the branches originating at the node, <span 
class="GUI">Select Sequences</span> for selecting sequences from the branches
originating at the node, and <span class="GUI">Hide Branch</span>, for hiding
the sequences for the branch. 
</p></dd>

<dt><span class="GUI">Sort Ascending</span></dt>
<dd><p>Sort the sequences by the value of a given property, in ascending order.
The properties that can be used are <span class="GUI">Name</span>, 
<span class="GUI">Chain ID</span>, <span class="GUI">Length</span>, 
<span class="GUI">Number of Gaps</span>, <span class="GUI">Sequence 
Identity</span>, <span class="GUI">Sequence Similarity</span>, 
<span class="GUI">Sequence Homology</span>, and 
<span class="GUI">Sequence Score</span>. Homology is calculated as the
percentage of residues with identical side-chain chemical properties (as defined
for the Side-Chain Chemistry color scheme).
</p></dd>

<dt><span class="GUI">Sort Descending</span></dt>
<dd><p>Sort the sequences by the value of a given property, in descending order.
The properties that can be used are the same as for <span
class="GUI">Sort Ascending</span>.
</p></dd>

<dt><span class="GUI">Move Up</span></dt>
<dd><p>Move the selected sequences up one position in the list.
</p></dd>

<dt><span class="GUI">Move Down</span></dt>
<dd><p>Move the selected sequences down one position in the list.
</p></dd>

<dt><span class="GUI">Move to Top</span></dt>
<dd><p>Move the selected sequences to the top of the list.
</p></dd>

<dt><span class="GUI">Move to Bottom</span></dt>
<dd><p>Move the selected sequences to the bottom of the list.
</p></dd>

<dt><span class="GUI">Get PDB Structures</span></dt>
<dd><p>Download structural information (secondary structures, B-factors,
coordinates) for sequences that came from the PDB, such as in a Blast
search. The information is obtained from a local copy of the PDB or from the
RCSB web site. If sequences are selected, information is obtained for these
sequences, otherwise it is obtained for all sequences. The PDB sequence replaces
the corresponding sequence in the MSV.
</p></dd>
</dl>

</dd>


<a name="alignment"></a>
<dt><span class="GUI">Alignment</span> menu</dt>
<dd><p>This menu provides tools for automatic sequence alignment with ClustalW,
residue selection and manual sequence alignment tasks. Some of the features are
also available on the toolbar. Residues are not renumbered when the gaps are
added or removed.
</p>
<dl>

<dt><span class="GUI">Multiple Alignment</span></dt>
<dd><p>Align the selected sequences simultaneously using ClustalW.  If there are 
columns (residues) selected, the alignment is performed only on the selected
residues. You can run an alignment on several discontinuous selected regions at
the same time.
</p></dd>

<dt><span class="GUI">Pairwise Alignment</span></dt>
<dd><p>Align the selected sequences pairwise using a Smith-Waterman algorithm,
and the settings from the <span class="GUI">Alignment Settings</span> dialog
box.  
</p></dd>

<dt><span class="GUI">Align and Merge</span></dt>
<dd><p>Align new sequences with a query sequence without changing the existing
alignment. Gaps are inserted into the existing alignment to preserve the residue
matching.
</p></dd>

<dt><span class="GUI">Align by Residue Numbers</span></dt>
<dd><p>Align sequences so that residues with identical residue numbers (and
insertion codes) are aligned. This is useful for families of proteins that share
common numbering schemes, such as antibodies.
</p></dd>

<dt><span class="GUI">Use Constraints</span></dt>
<dd><p>Apply constraints on pairwise alignments, so that the constrained
residues are in the same position (same column) after the alignment.
</p>
<p>When you select this option, a constraint row is displayed between the
query sequence and the other sequences. To add a constraint, click on a
residue in the query sequence and then on a position in one of 
the other sequences. The constraints are displayed as red lines connecting the
constrained residue pair. To remove a constraint, click on the constrained
residue pair again. To remove all constraints, choose <span class="GUI">Clear
Constraints</span> from the <span class="GUI">Alignment</span> menu or from the
sequence shortcut menu.
</p>
<p>You can also allow or disallow gaps in secondary structure elements, in the 
<span class="GUI">Pairwise Alignment Settings</span> dialog box.</p>

</dd>

<dt><span class="GUI">Clear Constraints</span></dt>
<dd><p>Remove all constraints on the alignment.
</p></dd>

<dt><span class="GUI">Alignment Settings</span></dt>
<dd><p>Opens the <span class="GUI">Pairwise Alignment Settings</span> dialog
box, in which you can choose the similarity matrix type, set the gap opening
penalty and the gap extension penalty, choose whether to allow gaps in
secondary structure elements, and create a substitution matrix from an existing
alignment. This substitution matrix can be selected as <span
class="GUI">Custom</span> from the matrix option menu for subsequent alignments. 
</p></dd>

<dt><span class="GUI">Lock Gaps</span></dt>
<dd><p>Lock gaps in the alignment so that they are not filled when performing
manual alignment. If you insert a gap after locking the gaps, the new gap is not
automatically locked. If you have a residue selection, the gaps are only locked
in the selected region.
</p></dd>

<dt><span class="GUI">Unlock Gaps</span></dt>
<dd><p>Unlock previously locked gaps so that they can be filled.
</p></dd>

<dt><span class="GUI">Select Identities</span></dt>
<dd><p>Select residues that are identical in all sequences. Gaps are ignored.
</p></dd>

<dt><span class="GUI">Select Aligned Blocks</span></dt>
<dd><p>Select blocks of residues for which there are no gaps in any of the
sequences.
</p></dd>

<dt><span class="GUI">Select Columns with Structure</span></dt>
<dd><p>Select columns in which at least one of the residues has 3D structure
(atom coordinates) associated with it. This is useful for multiple aligned
structures with SEQRES regions that don't have crystal coordinates in parts of
their sequences.
</p></dd>

<dt><span class="GUI">Expand Selection</span></dt>
<dd><p>Expand the residue selection in the selected sequences to include the
corresponding residues in all of the selected sequences. If no sequences are
selected, the residue selection is expanded to cover all sequences.
</p></dd>

<dt><span class="GUI">Expand Selection from Query</span></dt>
<dd><p>Expand the residue selection in the query sequence to include the
corresponding residues in the selected sequences, or in all sequences if no
sequences are selected.
</p></dd>

<dt><span class="GUI">Hide Selected Columns</span></dt>
<dd><p>Hide the columns in which residues in all sequences are selected.
</p></dd>

<dt><span class="GUI">Hide Unselected Columns</span></dt>
<dd><p>Hide the columns in which not all residues are selected (columns in which
one or more residues are not selected). 
</p></dd>

<dt><span class="GUI">Show All Columns</span></dt>
<dd><p>Show all columns for all sequences.
</p></dd>

<dt><span class="GUI">Select All</span></dt>
<dd><p>Select all residues. 
</p></dd>

<dt><span class="GUI">Invert Selection</span></dt>
<dd><p>Invert the selection of the residues: select the unselected residues, and
deselect the selected residues.
</p></dd>

<dt><span class="GUI">Deselect All</span></dt>
<dd><p>Deselect all residues. 
</p></dd>

<dt><span class="GUI">Delete</span></dt>
<dd><p>Delete the selected residues. 
Also <span class="Keys">Shift+Backspace</span> (<span class="Keys">&#x21E7;Delete</span>).
</p></dd>

<dt><span class="GUI">Crop</span></dt>
<dd><p>Delete the unselected residues. 
</p></dd>

<dt><span class="GUI">Remove Empty Columns</span></dt>
<dd><p>Remove columns that consist entirely of gaps.
</p></dd>

<dt><span class="GUI">Fill with Gaps</span></dt>
<dd><p>Replace the selected residues with gaps, in the selected sequences. 
</p></dd>

<dt><span class="GUI">Remove Gaps</span></dt>
<dd><p>Remove gaps in the residue selection of the selected sequences by
shifting residues to the left. If there is no selection, all gaps are removed,
including gaps at the beginning of the sequences.
</p></dd>

<dt><span class="GUI">Track Changes</span></dt>
<dd><p>Track the regions of the sequence that have been edited, by showing an
annotation that marks the edited regions. The latest edit is shown in black, and
earlier edits in progressively lighter shades, up to the fifth last edit. 
</p></dd>

<dt><span class="GUI">Reset History</span></dt>
<dd><p>Clear the history of changes stored when changes are tracked.
</p></dd>

</dl>

</dd>


<a name="color"></a>
<dt><span class="GUI">Color</span> menu</dt>
<dd><p>Apply a color scheme to the sequences.
</p>
<dl>

<a name="color.1"></a>
<dt><span class="GUI">Residue Type</span></dt>
<dd><p>Color the residues by residue type. The colors are:
</p>
<table>
<tr>
<td style="padding-right: 10px;">ACFILMPVW</td>
<td>blue</td>
<td>(hydrophobic)</td>
</tr>
<tr>
<td style="padding-right: 10px;">DE</td>
<td>red</td>
<td>(acidic)</td>
</tr>
<tr>
<td style="padding-right: 10px;">HKR</td>
<td>green-yellow</td>
<td>(basic)</td>
</tr>
<tr>
<td style="padding-right: 10px;">GNQSTY</td>
<td>orange</td>
<td>(other)</td>
</tr>
</table>
<br/>
</dd>


<a name="color.2"></a>
<dt><span class="GUI">Residue Similarity</span></dt>
<dd><p>Color residues by similarity. Identical residues are red, similar
residues (positive BLOSUM62 pairwise score) are orange, other residues are white.
</p></dd>

<a name="color.3"></a>
<dt><span class="GUI">Hydrophobicity (Kyte-Doolittle)</span></dt>
<dd><p>Color residues by Kyte-Doolittle hydrophobicity. Hydrophilic residues are
blue, hydrophobic residues are red, residues with zero hydrophobicity are white.
</p></dd>

<a name="color.4"></a>
<dt><span class="GUI">Hydrophilicity (Hopp-Woods)</span></dt>
<dd><p>Color residues by Hopp-Woods hydrophilicity. Hydrophilic residues are
red, hydrophobic residues are blue, residues with zero hydrophobicity are white.
</p></dd>

<a name="color.5"></a>
<dt><span class="GUI">Taylor Scheme</span></dt>
<dd><p>Color residues with the Aminochromography color scheme developed by
William Taylor (Protein Engineering <b>1997</b>, <i>10</i>, 743&ndash;746). In
this scheme, well conserved parts of the alignment exhibit bright, clear colors,
while parts that are not well conserved have brownish, dull colors.
</p></dd>

<a name="color.6"></a>
<dt><span class="GUI">Constant color</span></dt>
<dd><p>Color all residues with the color chosen from the submenu. Twelve colors
are offered on the submenu, with a <span class="GUI">Custom</span> item so that
you can choose your own color in a color selector.
</p></dd>

<a name="color.6s"></a>
<dt><span class="GUI">Secondary Structure</span></dt>
<dd><p>Color the sequence by the secondary structure assignment. If no SSA is
available but one or more secondary structure predictions are available, the
predictions are used to color the sequence. The colors for multiple predictions
are averaged, so positions where all predictions agree have bright colors, and
the positions of disagreement are more gray. If no SSP nor SSA is available, the
color of the sequence is not changed.
</p></dd>

<dt><span class="GUI">B Factor</span></dt>
<dd><p>Color the residues by their temperature factor (PDB B factor), on a
green-white-red scale, with green for the lowest values and red for the highest.
</p></dd>

<a name="color.7"></a>
<dt><span class="GUI">Residue Propensities</span></dt>
<dd><p>Color residues by the residue propensities. 
The schemes that are available on the submenu are described in the table below.
</p>
<table rules="groups" frame="hsides">
<thead>
<th>Scheme
</th>
<th>Residues
</th>
<th>Color
</th>
<th>Description
</th>
</thead>
<tbody>
<tr>
<td><span class="GUI">Helix Propensity</span></td>
<td>AMLEQK</td>
<td>red</td>
<td>helix-forming</td>
</tr>
<tr><td>&nbsp;</td>
<td>VIFW</td>
<td>magenta</td>
<td>weak helix-forming</td>
</tr>
<tr><td>&nbsp;</td>
<td>CSTNDHR</td>
<td>gray</td>
<td>ambivalent</td>
</tr>
<tr><td>&nbsp;</td>
<td>PGY</td>
<td>blue</td>
<td>helix-breaking</td>
</tr>
<tr>
<td><span class="GUI">Strand Propensity</span></td>
<td>VILMTFWY</td>
<td>blue</td>
<td>strand-forming</td>
</tr>
<tr><td>&nbsp;</td>
<td>ACSNQHR</td>
<td>gray</td>
<td>ambivalent</td>
</tr>
<tr><td>&nbsp;</td>
<td>DEKGP</td>
<td>red</td>
<td>strand-breaking</td>
</tr>
<tr>
<td><span class="GUI">Turn Propensity</span></td>
<td>GSDNP</td>
<td>cyan</td>
<td>turn-forming</td>
</tr>
<tr><td>&nbsp;</td>
<td>AVLIMHFWC</td>
<td>magenta</td>
<td>turn-breaking</td>
</tr>
<tr><td>&nbsp;</td>
<td>EQTKRY</td>
<td>gray</td>
<td>ambivalent</td>
</tr>
<tr style="vertical-align: text-bottom;">
<td style="vertical-align: text-bottom;"><span class="GUI">Helix Terminators</span><sup>*</sup></td>
<td style="vertical-align: text-bottom;">GTMRKHF</td>
<td style="vertical-align: text-bottom;">green</td>
<td style="vertical-align: text-bottom;">helix-starting</td>
</tr>
<tr><td>&nbsp;</td>
<td>SNDELWP</td>
<td>red</td>
<td>helix-ending</td>
</tr>
<tr><td>&nbsp;</td>
<td>CQAVIY</td>
<td>gray</td>
<td>ambivalent</td>
</tr>
<tr>
<td><span class="GUI">Exposure Tendency</span></td>
<td>RNDQEHK</td>
<td>blue</td>
<td>surface</td>
</tr>
<tr><td>&nbsp;</td>
<td>ACGPSTWY</td>
<td>gray</td>
<td>ambiguous</td>
</tr>
<tr><td>&nbsp;</td>
<td>ILMFV</td>
<td>orange</td>
<td>buried</td>
</tr>
<tr>
<td><span class="GUI">Steric Group</span></td>
<td>GACS</td>
<td>red</td>
<td>small, noninterfering</td>
</tr>
<tr><td>&nbsp;</td>
<td>TVNDILPM</td>
<td>magenta</td>
<td>ambiguous</td>
</tr>
<tr><td>&nbsp;</td>
<td>QEKR</td>
<td>cyan</td>
<td>sticky polar</td>
</tr>
<tr><td>&nbsp;</td>
<td>HFYW</td>
<td>blue</td>
<td>aromatic</td>
</tr>
<tr>
<td><span class="GUI">Side-Chain Chemistry</span></td>
<td>DE</td>
<td>red</td>
<td>acidic, hydrophilic</td>
</tr>
<tr><td>(the default)</td>
<td>RKH</td>
<td>blue</td>
<td>basic, hydrophilic</td>
</tr>
<tr><td>&nbsp;</td>
<td>GAVILM</td>
<td>green</td>
<td>neutral, hydrophobic, aliphatic</td>
</tr>
<tr><td>&nbsp;</td>
<td>FYW</td>
<td>orange</td>
<td>neutral, hydrophobic, aromatic</td>
</tr>
<tr><td>&nbsp;</td>
<td>STNQ</td>
<td>cyan</td>
<td>neutral, hydrophilic</td>
</tr>
<tr><td>&nbsp;</td>
<td>C</td>
<td>yellow</td>
<td>primary thiol</td>
</tr>
<tr><td>&nbsp;</td>
<td>P</td>
<td>dark gray</td>
<td>imino acid</td>
</tr>
</tbody>
</table>
<sup>*</sup>This item is only present on the <span class="GUI">Color
Blocks</span> submenu of the <span class="GUI">Annotations</span> menu.
<br/><br/>
</dd>

<a name="color.11"></a>
<dt><span class="GUI">Mark Residues</span></dt>
<dd><p>Mark the selected residues with the color chosen from this submenu. This
color overrides any other color applied. To remove this color, select the
residues and choose <span class="GUI">Unmark Residues</span> from the submenu.
</p></dd>

<a name="color.10"></a>
<dt><span class="GUI">Adjust Color Range</span></dt>
<dd><p>Set the limits of sequence identity used when weighting the color
density by alignment quality. 
Opens a dialog box, in which you can set the lower and upper threshold
for the sequence identity. Residues with identity below the lower threshold are 
colored white; residues with identity above the upper threshold have full color,
and the color density for residues with identity between the two thresholds
is set using a linear scale. 
</p></dd>

<dt><span class="GUI">Color Sequences</span></dt>
<dd><p>Show or hide the sequence coloring. 
</p></dd>

<dt><span class="GUI">Adjust Text Contrast</span></dt>
<dd><p>Use white for the text on residues colored with dark colors, and black
for the text on residues colored with light colors. If this option is not
selected, the text is black for all residues.
</p></dd>

</dl>

</dd>


<a name="annotations"></a>
<dt><span class="GUI">Annotations</span> menu</dt>
<dd><p>Annotations are a means of representing additional information
associated with the sequences. There are two classes of annotations: global
annotations (consensus sequence, mean hydrophobicity) and local annotations. The
global annotations are calculated for the entire set of sequences, the local
annotations are computed for each sequence individually. Depending on the
annotation type, they are presented as histogram 
plots (hydrophobicity, B-factor), color bars (&quot;Color Blocks&quot;),
alphanumeric strings (consensus sequence, SSP, Pfam), graphical
representations (secondary structure assignments).
</p>
<dl>

<dt><span class="GUI">Consensus Sequence</span></dt>
<dd><p>Display the consensus sequence at the top of the panel. The consensus
sequence is the sequence that is composed of the most frequently occurring
residue at each position in the sequence; if there are two residues that have
the same frequency of occurrence, a + symbol is used, and the residues for this
position are shown in its tooltip.  The sequence is annotated with a histogram
of the number of sequences that are represented by each residue in the
consensus, with information on the percentage in the tooltip.
</p></dd>

<dt><span class="GUI">Consensus Symbols</span></dt>
<dd><p>Display a row at the top of the panel that contains symbols for the degree
of consensus. The symbols follow the ClustalW conventions:</p>
<table cellpadding=5>
<tr>
<td style="padding-right: 10px;"><code>*</code>
</td>
<td>Single, fully conserved residue.
</td>
</tr>
<tr>
<td style="padding-right: 10px;"><code>:</code>
</td>
<td>One of the following &quot;strong&quot; groups is fully conserved: STA,
NEQK, NHQK, NDEQ, QHRK, MILV, MILF, HY, FYW. 
</td>
</tr>
<tr>
<td style="padding-right: 10px;"><code>.</code>
</td>
<td>One of the following &quot;weaker&quot; groups is fully conserved: CSA, ATV,
SAG, STNK, STPA, SGND, SNDEQK, NDEQHK, NEQHRK, FVLIM, HFY.
</td>
</tr>
</table>
<br/></dd>

<dt><span class="GUI">Sequence Logo</span></dt>
<dd><p>Display logo annotation. In this annotation, residue symbols at each
position whose frequency of occurence at that position is greater than a
threshold are drawn in a vertical stack in order of frequency, with the height 
of the residue symbols proportional to the frequency of occurrence. (See
Schneider T.D.; Stephens R.M. Sequence Logos: A New Way to Display Consensus
Sequences. <i>Nucleic Acids Res.</i> <b>1990</b>, <i>18</i>, 6097.)
</p></dd>

<dt><span class="GUI">Mean Hydrophobicity</span></dt>
<dd><p>Display histogram of Kyte-Doolittle hydrophobicity for each residue in
the alignment, averaged over all sequences. Hydrophobic residues have positive
values; hydrophilic residues have negative values. 
</p></dd>

<dt><span class="GUI">Mean Isoelectric Point</span></dt>
<dd><p>
</p></dd>

<dt><span class="GUI">Add Global Annotations</span></dt>
<dd><p>Add all global annotations (listed above) to the display.
</p></dd>

<dt><span class="GUI">Remove Global Annotations</span></dt>
<dd><p>Remove all global annotations from the display. Does not affect
sequence-dependent annotations.
</p></dd>

<dt><span class="GUI">Residue Numbers</span></dt>
<dd><p>Display residue numbers above the sequence. The numbers are given every 5
residues, and are left-aligned to the left edge of the residue in the
sequence. This is useful for tracking sequence changes after residue deletion,
for example. The ruler only gives absolute alignments.
</p></dd>

<dt><span class="GUI">Secondary Structure Assignment</span></dt>
<dd><p>Display secondary structure assignment for the sequence.
</p></dd>

<dt><span class="GUI">B-Factor</span></dt>
<dd><p>Display histogram of temperature factors for each residue in the
sequence.
</p></dd>

<dt><span class="GUI">Disulfide Bonds</span></dt>
<dd><p>Display disulfide bonds as lines connecting cysteine residues, colored
from black (strongest prediction) to light gray (weakest prediction).
</p></dd>

<dt><span class="GUI">Hydrophobicity</span></dt>
<dd><p>Display histogram of Kyte-Doolittle hydrophobicity for each residue in
the sequence. Hydrophobic residues have positive values; hydrophilic residues
have negative values. 
</p></dd>

<dt><span class="GUI">Isoelectric Point</span></dt>
<dd><p>Display histogram of isoelectric points in a 5-residue window. The
isoelectric point of the isolated amino acid and the 5-residue window
value are displayed in the tooltip for each histogram bar.
</p></dd>

<dt><span class="GUI">Ligand Contacts</span></dt>
<dd><p>Display a row in which residue positions are colored by the shortest
distance between any ligand heavy atom and any heavy atom in the residue at that
position. Red is used if the distance is less than 4&nbsp;&Aring;, orange is
used if the distance is less than 6&nbsp;&Aring;, and gray is used
otherwise. This annotation takes a little time to generate, and requires a
structure for the sequence.
</p></dd>

<dt><span class="GUI">Antibody CDRs</span></dt>
<dd><p>Display assignment of the three VL and VH regions. The residues are
colored red, and a red line is displayed in the annotation for each region,
labeled L<i>n</i> or H<i>n</i> for <i>n</i> = 1, 2, or 3.
</p></dd>

<dt><span class="GUI">Select Antibody CDRs</span></dt>
<dd><p>If the sequence is an antibody, select the CDRs in the sequence. 
</p></dd>

<dt><span class="GUI">Antibody Numbering Scheme</span></dt>
<dd><p>Choose the numbering scheme to use for residues in an antibody, from
<span class="GUI">Chothia</span>, <span class="GUI">Enhanced Chothia</span>,
<span class="GUI">Kabat</span>, <span class="GUI">IMGT</span>,
and <span class="GUI">AHo</span>. This numbering scheme is only in effect when
the <span class="GUI">Antibody CDRs</span> annotation is turned on. When you
export the antibody to a file, it is exported with the selected numbering scheme.
</p></dd>


<dt><span class="GUI">Color Blocks</span></dt>
<dd><p>Display a single row of color blocks that are colored according to one of
the residue properties described under <a href="#color.7"><span
class="GUI">Residue Propensities</span></a> for the <span
class="GUI">Color</span> menu. The row is presented like a sequence, but without
letter codes, only the colors for the property. This submenu includes all the
items that are on the <span class="GUI">Residue Propensities</span> submenu of
the <span class="GUI">Color</span> menu, with the addition of the <span
class="GUI">Helix Terminator</span> item, and also has commands to display
annotations for all properties on the submenu or to remove all color block
annotations.
</p></dd>

<dt><span class="GUI">User Annotations</span></dt>
<dd><p>Add or remove customizable annotations. There are three types of user
annotation offered:</p>
<ul>
<li><span class="GUI">Mark Alignment Region</span>&mdash;Draw a rectangle over
the selected part of the entire alignment (all columns). The rectangle position
is fixed at the initial ruler position: it doesn't change if you change the
alignment.</li>
<li><span class="GUI">Mark Rectangular Region</span>&mdash;Draw a rectangle over
the currently selected block of residues. The rectangle is anchored to the top
left residue and its position moves with this residue if you edit the
alignment.</li>
<li><span class="GUI">Add Custom Annotation</span>&mdash;Add an annotation that
you can edit to mark residues with whatever text symbols you choose. You can
edit the annotation in Edit mode, and change its name from the default
&quot;Custom annotation&quot; by right-clicking on the annotation and choosing
<span class="GUI">Rename Sequence</span>. </li>
</ul>
</dd>

<dt><span class="GUI">Clear Annotations</span></dt>
<dd><p>Remove all annotations from the selected sequences, leaving just the
sequences. This action removes any secondary structure assignments and
predictions, as well as the annotations added from this menu. If no sequences
are selected, annotations are removed from all sequences.
</p></dd>
</dl>

</dd>


<a name="tools"></a>
<dt><span class="GUI">Tools</span> menu</dt>
<dd><p>This menu provides access to programs for finding homologs, finding
families, and predicting secondary structure.
</p>
<dl>

<dt><span class="GUI">Find Homologs (BLAST)</span></dt>
<dd><p>Run a BLAST search to find homologs for the first sequence. Opens the
<span class="GUI">Blast Search Settings</span> panel, in which you can make
choices for the search and start the job. A progress dialog box is displayed,
and the results are displayed in the <span class="GUI">BLAST Search
Results</span> dialog box. 
</p></dd>

<dt><span class="GUI">Show Results of BLAST Search</span></dt>
<dd><p>Show the output of the latest BLAST search, in the <span class="GUI">BLAST
Search Results</span> dialog box. The dialog box allows you to select homologs,
sort results by one of a range of properties, download PDB structures for the
homologs, and incorporate selected homologs into the project.
</p></dd>

<dt><span class="GUI">Find Family (Pfam)</span></dt>
<dd><p>Run a Pfam search to find families for the selected sequences, or for all
sequences if no sequence is selected.
</p></dd>

<dt><span class="GUI">Predict</span></dt>
<dd><p>Run prediction programs. This submenu has the following items:
</p>
<dl>
<dt><span class="GUI">Run All Predictions</span></dt>
<dd><p>Run all the predictions listed below.
</p></dd>

<dt><span class="GUI">Secondary Structure</span></dt>
<dd><p>Run the secondary structure programs to obtain a prediction of the
secondary structure of the selected sequences, or for all
sequences if no sequence is selected. A dialog box opens that shows job
progress.
</p></dd>

<dt><span class="GUI">Solvent Accessibility</span></dt>
<dd><p>Predict accessibility of each residue to solvent. If more than 25% of
total residue surface area is predicted to be exposed to the solvent, the
residue is marked &quot;e&quot; (exposed, colored blue), otherwise it is marked
&quot;b&quot; (buried, colored yellow).
</p></dd>

<dt><span class="GUI">Domain Arrangement</span></dt>
<dd><p>Predict the arrangement of domains. Residues marked gray are likely to
form a domain. Residues marked red are likely to be in linker (inter-domain)
regions.
</p></dd>

<dt><span class="GUI">Disordered Regions</span></dt>
<dd><p>Calculate a disorder score and classify residues by this score The score
is normalized to a 0 to 1 range. If a residue has a disorder score less than
0.5, it is marked light gray. If the score is between 0.5 and 0.9, the 
residue is marked orange. If the score is greater than 0.9, the residue is
marked red.
</p></dd>

<dt><span class="GUI">Disulfide Bridges</span></dt>
<dd><p>Predict disulfide bridges between cysteines. Predicted bonds are drawn as
lines connecting cysteine residues, colored from black (strongest prediction) to
light gray (weakest prediction).
</p></dd>

<dt><span class="GUI">Beta Strand Contacts</span></dt>
<dd><p>Predict the contacts between beta sheets. 
</p></dd>

</dl>
</dd>

<dt><span class="GUI">Clear Predictions</span></dt>
<dd><p>Remove secondary structure predictions. 
</p></dd>

<dt><span class="GUI">Build Homology Model</span></dt>
<dd><p>Build a structure for the query sequence, using the Prime Build Structure
tools. The structure is built for the query sequence, and templates must be
selected from among the sequences that have PDB structures. The <a
href="msv_build_model.html"><span class="GUI">Build Homology Model</span>
panel</a> opens, in which you can select options for use of templates, for
multimer models, and for building the structure; and then run the job. The
structure is incorporated into the Maestro project when the job finishes, and
the sequence is added to the MSV and aligned with the query (or queries, for a
heteromultimer).
</p></dd>

<dt><span class="GUI">Analyze Binding Site</span></dt>
<dd><p>Calculate sequence identity in multiple alignment columns that are within
a certain spatial distance from the ligand in the query. You must have a query
sequence with a structure and a set of sequences that are aligned to the query. 
Choosing this menu item opens the <a href="msv_analyze_binding_site.html"><span 
class="GUI">Analyze Binding Site</span> dialog box</a>, in which you can select
sequences by percentage identity to the query in the binding site, or analyze
the percentage identity, similarity, and homolgy of the aligned sequences to the
query within a range of distances from the binding site.
</p></dd>

<dt><span class="GUI">Compare Sequences</span></dt>
<dd><p>Compare all sequences or the selected sequences, by identity, similarity,
or homology. The percentage is displayed in a table, like a heat map, with cells
color-coded by the percentage value. Opens the <span class="GUI">Compare
Sequences</span> panel, in which you can choose the comparison measure to
display, switch between all or selected sequences, and refresh the display after
changing the alignment. 
</p></dd>

<dt><span class="GUI">Show Job Settings</span></dt>
<dd><p>Opens the <span class="GUI">Multiple Sequence Viewer Job Settings</span>
dialog box, so that you can make settings for any job that is run from the MSV.
</p></dd>

<dt><span class="GUI">Show Job Log</span></dt>
<dd><p>Show the log file for the most recent job in a dialog box.
</p></dd>
</dl>
<br />
</dd>


<a name="maestro"></a>
<dt><span class="GUI">Maestro</span> menu</dt>
<dd><p>This menu provides options and actions for the interaction with the
sequences and their structures as stored in the Maestro project and displayed in
the Workspace.
</p>

<dl>

<dt><span class="GUI">Incorporate Entries from Workspace</span></dt>
<dd><p>Incorporate the sequences and the corresponding structures from the
entries that are in the Workspace into the MSV.
</p></dd>

<dt><span class="GUI">Incorporate Selected Entries from Project Table</span></dt>
<dd><p>Incorporate the sequences and the corresponding structures from the
entries that are selected in the Project Table into the MSV.
</p></dd>

<dt><span class="GUI">Include Incorporated Entries in Workspace</span></dt>
<dd><p>When structures are imported into the MSV and incorporated into the
Maestro project, include the structures in the Workspace. 
</p></dd>

<dt><span class="GUI">Associate Maestro Entries</span></dt>
<dd><p>Associate Maestro entries with sequences in the MSV without adding new
sequences, modifying the alignment,or importing structures into the MSV. Opens
the <span class="GUI">Associate Maestro Entries with MSV Sequences</span> dialog
box, in which you can choose an entry chain from the Workspace (list on the
left) and an MSV sequence (list on the right), and click <span
class="GUI">Associate Selected Pair</span> to associate the sequence with the
entry. The sequence identity for the selected pair is shown below the lists, and
colored green if the identity exceeds 95% or red if it does not. The residues
that do not match the entry sequence are marked as structureless in the MSV by
using a less intense color. This facility is useful if the sequences are
imported into the MSV and the structures are independently imported into
Maestro.
</p></dd>

<dt><span class="GUI">Superimpose Structures According to Sequence
Alignment</span></dt>
<dd><p>Superimpose structures according to their sequence alignment. Uses the
<a href="super_panel.html"><span class="GUI">Superposition</span>
panel</a>, with sequence identities selected as the atoms for superposition.
</p></dd>

<dt><span class="GUI">Protein Structure Alignment</span></dt>
<dd><p>Run the Prime Protein Structure Alignment program on the selected
(or all) protein structures and return the alignments. The sequences you select
must have structures associated with them.
</p></dd>

<dt><span class="GUI">Get Colors from Maestro Workspace</span></dt>
<dd><p>Color sequences with the colors that they have in the Maestro
Workspace. The color of the alpha carbon is used to color the residues.
</p></dd>

<dt><span class="GUI">Apply Colors to Workspace</span></dt>
<dd><p>Apply the colors from the MSV to the sequences and structures in Maestro.
</p></dd>

<dt><span class="GUI">Color Entry Surface</span></dt>
<dd><p>Color the molecular surface in the Workspace using the colors from the
corresponding sequence in the MSV. If sequences are selected, only the colors of
the selected sequences are applied.
</p></dd>

<dt><span class="GUI">Update from Maestro</span></dt>
<dd><p>Update the sequences in the MSV that originated from Maestro with any
changes made in Maestro.
</p></dd>

<dt><span class="GUI">Update Maestro Workspace Selection from MSV</span></dt>
<dd><p>Update the atom selection in the Maestro Workspace from the residue
selection in the MSV.
</p></dd>

<dt><span class="GUI">Update Automatically from Maestro</span></dt>
<dd><p>When this option is selected, changes made to sequences in the Maestro
project are automatically propagated to the MSV. If it is not selected, you can
choose <span class="GUI">Synchronize with Maestro</span> to apply changes made
in Maestro to the MSV.
</p></dd>

<dt><span class="GUI">Allow Structural Changes</span></dt>
<dd><p>When in Edit mode, allow mutation operations on the sequence to change
the structure in Maestro. If this option is not selected, you will not 
be allowed to mutate sequences that came from Maestro structures. This option
has no effect on deletions in the sequence, which are not propagated to the
structure.
</p></dd>

</dl>

</dd>


<a name="settings"></a>
<dt><span class="GUI">Settings</span> menu</dt>
<dd><p>This menu provides settings for control of what is displayed.
</p>
<dl>

<dt><span class="GUI">Wrap Sequences</span></dt>
<dd><p>When selected, wrap the sequences so that the display consists of multiple
rows of sequences, and can be scrolled vertically. When unselected, display the
sequences in a single row that scrolls horizontally. Operations on unwrapped
sequences are generally faster, especially when there are many sequences.
</p></dd>

<dt><span class="GUI">Group Annotations by Type</span></dt>
<dd><p>When selected, group the annotations of the same type and display the
groups as separate rows, below the sequences. When unselected, display the
annotations for each sequence directly below the sequence. 
</p></dd>

<dt><span class="GUI">Font Size</span></dt>
<dd><p>Change the font size for the text in the sequence viewer. This submenu
offers a selection of point sizes for the font.
</p></dd>

<dt><span class="GUI">Display Ruler</span></dt>
<dd><p>Display a &quot;ruler&quot; that marks the residue positions in the
sequence viewer. These positions are not the same as the residue numbers in each
sequence, which can be offset from the origin and have gaps.  
</p></dd>

<dt><span class="GUI">Display Tooltips</span></dt>
<dd><p>Show information about the panel and its contents in tooltips (text
displayed when the pointer pauses over the relevant part of the panel).
</p></dd>

<dt><span class="GUI">Display Header Row</span></dt>
<dd><p>Display a heading row above the sequences with labels for each section. 
</p></dd>

<dt><span class="GUI">Replace Identities with Dots</span></dt>
<dd><p>Replace residue symbols with dots for all residues that are identical to
those in the query sequence. This feature makes it easy to find the
mutations from the query sequence in the other sequences. 
</p></dd>

<dt><span class="GUI">Pad Alignment with Gaps</span></dt>
<dd><p>Add gaps to the end of each sequence so that the sequences are the same
length.
</p></dd>

<dt><span class="GUI">Display Sequence Boundaries</span></dt>
<dd><p>Display the residue number of the first and last visible residue in each row
for each sequence. The numbers are displayed to the left and right of the
sequence. 
</p></dd>

<dt><span class="GUI">Display Percentage Identity</span></dt>
<dd><p>Display the percentage identity with the query sequence to the right
of the sequence. By default, the query sequence is the consensus sequence.
You can set the query sequence by right-clicking on a sequence and choosing 
<span class="GUI">Set as Query</span> from the shortcut menu. The name of
the query sequence is displayed in the status area.
</p></dd>

<dt><span class="GUI">Display Percentage Similarity</span></dt>
<dd><p>Display the percentage similarity to the query sequence to the right
of the sequence. Two residues are similar if they have a positive BLOSUM62
pairwise score.  The percentage similarity is calculated from the number of
similar residues divided by the number of aligned residues.  By default, the
query sequence is the consensus sequence. You can set the query sequence by
right-clicking on a sequence and choosing <span class="GUI">Set as Query</span>
from the shortcut menu. The name of the query sequence is displayed in the
status area.
</p></dd>

<dt><span class="GUI">Display Percentage Homology</span></dt>
<dd><p>Display the percentage homology to the query sequence to the right
of the sequence. Homology is calculated as the percentage of residues with
identical side-chain chemical properties (as defined for the Side-Chain
Chemistry color scheme). By default, the query sequence is the consensus 
sequence. You can set the query sequence by right-clicking on a sequence and
choosing <span class="GUI">Set as Query</span> from the shortcut menu. The
name of the query sequence is displayed in the status area.
</p></dd>

<dt><span class="GUI">Display Score</span></dt>
<dd><p>Display the BLOSUM62 similarity score to the right of the sequence. The
score is calculated relative to the query sequence.
</p></dd>

<dt><span class="GUI">Include Gaps in Sequence Identity Calculations</span></dt>
<dd><p>When calculating sequence identity, count gaps as though they were
residues, rather than ignoring them. For example, a column consisting
of 2 different residues and 8 gaps would have a sequence identity of 20% if gaps
are included, but 50% if gaps are ignored.
</p></dd>

<dt><span class="GUI">Calculate Sequence Identity Only in Selected
Columns</span></dt>
<dd><p>When calculating sequence identity, restrict the calculation to the
columns (residues) that are selected. This allows you to calculate the identity
of regions of a sequence rather than the whole sequence.
</p></dd>

<dt><span class="GUI">Update Sequence Profile</span></dt>
<dd><p>Update the internal profile that is used for sequence identity
and consensus calculations, and sequence coloring. If <span
class="GUI">Automatically Update Sequence Profile</span> is off, use this
command to manually update the profile before doing calculations or applying
coloring.
</p></dd>

<dt><span class="GUI">Automatically Update Sequence Profile</span></dt>
<dd><p>Update the internal profile that is used for sequence identity
and consensus calculations, and sequence coloring automatically when changes are
made. Deselect this option to improve performance when you have a large number
of sequences. 
</p></dd>

<dt><span class="GUI">Ask Before Accessing a Remote Server</span></dt>
<dd><p>Display a dialog box requesting confirmation of the action before
retrieving information from a web server.
</p></dd>

<dt><span class="GUI">Return to Default Settings</span></dt>
<dd><p>Reset all settings on the <span class="GUI">Settings</span> menu to the
default values.
</p></dd>

</dl>

</dd>

</dl>

<a name="feature2"></a>
<h3>Toolbars</h3>
<p>There are two rows of tools on the tool bars. The first row contains toolbars
that only have buttons. These toolbars are described together. The second row
contains toolbars that have other tools besides buttons. These toolbars are
described separately.  You can show or hide any of the toolbars from the
shortcut menu.</p>

<ul>
<li><a href="#feature2.1">Toolbar Buttons</a></li>
<li><a href="#feature2.2">Sequence Editing Modes</a></li>
<li><a href="#feature2.3">Fetch Toolbar</a></li>
<li><a href="#feature2.4">Find toolbar</a></li>
</ul>


<a name="feature2.1"></a>
<h4>Toolbar Buttons</h4>

<p>The buttons on the toolbars that only have buttons are described below. 
</p>

<table cellpadding=5>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_open.png" /></td>
<td><span class="GUI">Import Sequences</span><br />
Import sequences into the Multiple Sequence Viewer project from a file.
The file must be in FASTA or PDB format. Opens a file selector, in which you can
navigate to and select the file.  Same as <span class="GUI">File &rarr; Import
Sequences</span>.</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_save.png" /></td>
<td><span class="GUI">Export Sequences</span><br />
Export sequences from the Multiple Sequence Viewer project to a FASTA file.
Opens a file selector, in which you can navigate to a location and name the
file. Same as <span class="GUI">File &rarr; Export Sequences</span>.</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_undo.png" /></td>
<td><span class="GUI">Undo</span><br />
Undo the last action.  Same as <span class="GUI">Edit &rarr; Undo</span>.</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_redo.png" /></td>
<td><span class="GUI">Redo</span><br />
Redo the last action. Same as <span class="GUI">Edit &rarr; Redo</span>.</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_lock.png" /></td>
<td><span class="GUI">Lock gaps</span><br />
Lock gaps in the sequences so that they are not filled when performing manual
alignment. The locking is applied once. Gaps created after locking is done are
not automatically locked. To lock them, click this button again. Locked gaps are
indicated by a dash (<code>-</code>); unlocked gaps are indicated by a tilde
(<code>~</code>). If you have a residue selection, gaps are only locked in the
selected region.
</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_unlock.png" /></td>
<td><span class="GUI">Unlock gaps</span><br />
Unlock gaps in the sequences after they have been locked. This allows gaps to be
filled when performing manual alignment.</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_pairwise_align.png" /></td>
<td><span class="GUI">Pairwise Alignment</span><br />
Align multiple sequences pairwise using ClustalW. Same as <span
class="GUI">Alignment &rarr; Pairwise Alignment</span>.
</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_multiple_align.png" /></td>
<td><span class="GUI">Multiple Alignment</span><br />
Align multiple sequences simultaneously using ClustalW. Same as <span
class="GUI">Alignment &rarr; Multiple Alignment</span>.
</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_identity.png" /></td>
<td><span class="GUI">Color Matching Residues Only</span><br />
Apply the current color scheme only to residues that are identical in all
sequences (gaps are ignored).
</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_weight.png" /></td>
<td><span class="GUI">Weight Colors by Alignment Quality</span><br />
Set the color density according to the sequence identity.
</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_average.png" /></td>
<td><span class="GUI">Average Colors in Columns</span><br />
Average the colors in each column and color all residues in the column by the
average color.</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_zoom_in.png" /></td>
<td><span class="GUI">Zoom in</span><br />
Increase the width of each residue so that the horizontal scale is expanded.
</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_zoom_out.png" /></td>
<td><span class="GUI">Zoom out</span><br />
Decrease the width of each residue so that the horizontal scale is contracted.
When the residues are narrower than the text, the text is no longer displayed.
The residues can be identified by their tooltips. 
</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_wrap.png" /></td>
<td><span class="GUI">Wrap Sequences</span><br />
Wrap the sequences so that the display consists of multiple rows of sequences,
and can be scrolled vertically. When unselected, display the sequences in a
single row that scrolls horizontally. Same as <span class="GUI">Settings &rarr;
Wrap Sequences</span>.</td>
</tr>
<tr>
<td><img src="../images/MaestroToolbarIcons/msv_3d_model.png" /></td>
<td><span class="GUI">Build Homology Model</span><br />
Build a 3D model of the sequence using Prime. Same as Tools &rarr; Build 3D
Model.</td> 
</tr>
</table>

<a name="feature2.2"></a>
<h4>Sequence Editing Modes</h4>
<p>The <span class="GUI">Mode</span> option menu allows you to select one of
four sequence editing modes so that you can edit the alignment, and in some
cases, edit the sequence itself. </p>
<p>You can lock the sequence downstream (to the right) of the residue or block
that you are moving, so that the downstream part of the sequence moves as a
block, without creating or removing gaps. To lock or unlock the sequence
downstream, click the <span class="GUI">Lock Sequence Downstream</span> button
to the right of the <span class="GUI">Mode</span> option menu. Locking is on by
default. </p> 
<img src="../images/MaestroToolbarIcons/msv_lock_down.png" />
<p>You can also use the <span class="GUI">Lock Gaps</span> and <span
class="GUI">Unlock Gaps</span> toolbar buttons to prevent gaps from collapsing
while editing the alignment. </p>
<p>
The four sequence editing modes are described below.
</p>
<dl>
<dt><span class="GUI">Select and Slide</span></dt>
<dd><p>Use this mode to select multiple residues and slide them.
</p>
<ul>
<li><p>To select residues, drag over the residues, or shift-click the first and
last residues in the range. You can drag across multiple sequences to select
residues, and you can drag in the ruler to select residues in all sequences. 
</p></li>
<li><p>To deselect selected residues, control-click the residues.
</p></li>
<li><p>To slide the selected residues, drag them to their new location. 
</p></li>
</ul>

</dd>

<dt><span class="GUI">Grab and drag</span></dt>
<dd><p>Use this mode to drag single residues to a new location. If there are
residues adjacent to the residue you drag, they are also moved as you drag. Gaps
are filled as you drag, unless they are locked. 
</p></dd>

<dt><span class="GUI">Edit</span></dt>
<dd><p>Edit the sequence or the SSPs by typing. You can mutate residues by typing
in the replacement residue code, and delete residues. If the sequence is a
Maestro sequence, a mutation operation also mutates the residues in the
structure, but you must have <span class="GUI">Allow Structural Changes</span>
selected on the <span class="GUI">Maestro</span> menu to perform the mutation.
Deletions do not change the structure, regardless of the <span class="GUI">Allow
Structural Changes</span> setting.
You can also edit sequences as text by using the tools on the <span
class="GUI">Edit</span> menu. You can change an SSP by typing in the
replacement code (<span class="Keys">E</span>, <span class="Keys">H</span>, or 
<span class="Keys">-</span>). </p>
<p>The allowed key strokes are listed below.
</p>
<ul>
<li><span class="Keys">SPACE</span>&mdash;insert a single gap under the cursor
and move the downstream sequence one position to the right.
</li>
<li><span class="Keys">BACKSPACE</span>
(<span class="Keys">DELETE</span>)&mdash;delete a single gap to the left 
of the cursor and move the downstream sequence one position to the left. The
cursor stays on the current residue, which moves to the left.
</li>
<li><span class="Keys">DELETE
(<span class="Keys">Fn+DELETE</span>)</span>&mdash;delete the single gap under
the cursor and move the downstream sequence one position to the left. The cursor
stays at the same ruler position.
</li>
<li><span class="Keys">Shift+BACKSPACE</span> (<span class="Keys">&#x21E7;DELETE</span>)&mdash;delete the selected
residues. 
</li>
<li>Arrow keys&mdash;move the cursor around the alignment. Using <span
class="Keys">CTRL</span> with the left and right arrow keys moves the cursor by
10 residues.
</li>
<li><span class="Keys">HOME</span>&mdash;move the cursor to the leftmost
position that is displayed in the current sequence (the left side of the
sequence display). Pressing <span class="Keys">HOME</span> again moves the
cursor to the first residue in unwrapped mode, scrolling the display.
</li>
<li><span class="Keys">END</span>&mdash;move the cursor to the rightmost position
that is displayed in the current sequence (the right side of the sequence
display).  Pressing <span class="Keys">END</span> again moves the
cursor to the last position in unwrapped mode, scrolling the
display. The cursor is not necessarily over a residue.
</li>
</ul>

</dd>

<dt><span class="GUI">Insert and remove gaps</span></dt>
<dd><p>Insert single gaps by clicking with the left mouse button. Delete single
gaps by clicking with the right mouse button. If multiple sequences are selected
and you click in one of the selected sequences, the gaps are inserted or deleted
in all selected sequences. If you click in a single selected sequence or in a
sequence that is not selected, the gaps are inserted or deleted only in the
sequence you clicked on.
</p></dd>

</dl>

<a name="feature2.3"></a>
<h4>Fetch Toolbar</h4>
<p>The <span class="GUI">Fetch</span> tool allows you to fetch sequences from
the Protein Data Bank or the Entrez Protein Database, from the appropriate web
sites.
</p>
<ul>
<li><p>To fetch sequences from the PDB, type the four-character code into the
<span class="GUI">Fetch</span> text box and press <span
class="Keys">ENTER</span>. The sequence is retrieved from the RCSB web site and
added to the project.</p></li> 
<li><p>To fetch sequences from Entrez, type the access code into the <span
class="GUI">Fetch</span> text box and press <span class="Keys">ENTER</span>. 
The access code format is <i>database</i><code>|</code><i>code</i>, where
<i>database</i> is the code for the database (<code>gi</code> for GeneBank,
<code>pdb</code> for the Protein Data Bank, <code>emb</code> for the EMBL
Sequence Database, and so on), and <i>code</i> is the sequence code for the
database. Examples: <code>gi|12345</code>, <code>pdb|2aba</code>,
<code>emb|CAA44029.1</code>. The sequence is retrieved from the NCBI web site
and added to the project.</p></li>
</ul>

<a name="feature2.4"></a>
<h4>Find Pattern Toolbar</h4>
<p>This toolbar provides a tool for finding a pattern in the sequences. The
pattern used in the search is an extended PROSITE pattern, which makes use of
secondary structure and property information. The pattern has the
following syntax.
</p>
<table cellpadding=5>
<tr>
<td style="padding-right: 10px;"><i>residue</i></td>
<td>Find occurrences of the specified residue. The residue must be given as
an upper case letter. For example, <code>A</code> finds alanine.</td>
</tr>
<tr>
<td style="padding-right: 10px;"><code>[</code><i>list</i><code>]</code></td>
<td>Find occurrences of any of the residues listed. For example
<code>[AIL]</code> finds occurrences of A, I, or L</td>
</tr>
<tr>
<td style="padding-right: 10px;"><code>{</code><i>list</i><code>}</code></td>
<td>Exclude all occurrences of any of the residues listed. For example
<code>{ED}</code> ensures that occurrences of E and D are <i>not</i> found.</td>
</tr>
<tr>
<td style="padding-right: 10px;"><code>a</code>&nbsp;</td>
<td>Find acidic residues (D and E)</td>
</tr>
<tr>
<td style="padding-right: 10px;"><code>b</code>&nbsp;</td>
<td>Find basic residues (K and R)</td>
</tr>
<tr>
<td style="padding-right: 10px"><code>e</code>&nbsp;</td>
<td>Find residues in an extended (beta-strand) region</td>
</tr>
<tr>
<td style="padding-right: 10px"><code>f</code>&nbsp;</td>
<td>Find residues in a flexible region (B-factor greater than the chain average)
</td>
</tr>
<tr>
<td style="padding-right: 10px"><code>h</code>&nbsp;</td>
<td>Find residues in a helical region</td>
</tr>
<tr>
<td style="padding-right: 10px;"><code>o</code>&nbsp;</td>
<td>Find hydrophobic residues (A, C, F, I, L, P, V, W, and Y)</td>
</tr>
<tr>
<td style="padding-right: 10px"><code>p</code>&nbsp;</td>
<td>Find aromatic residues (F, W, and Y)</td>
</tr>
<tr>
<td style="padding-right: 10px"><code>s</code>&nbsp;</td>
<td>Find solvent-exposed residues</td>
</tr>
<tr>
<td style="padding-right: 10px"><code>x . ?</code>&nbsp;</td>
<td>Find any residue. Any of these three characters can be used.</td>
</tr>
<tr>
<td style="padding-right: 10px"><code>@</code><i>number</i>&nbsp;</td>
<td>Find the residues with the specified PDB residue number (not ruler
position). Insertion codes are not recognized, so all residues with a given
insertion code are found.</td>
</tr>
<tr>
<td style="padding-right: 10px"><code>(</code><i>m</i><code>)</code><br/>
<code>(</code><i>m</i><code>,</code><i>n</i><code>)</code>&nbsp;</td>
<td>Find the specified number of contiguous occurrences of a residue (or residue
type). The second form specifies a variable number of occurrences,
e.g. <code>o(2-4)</code> means find two to four consecutive hydrophobic
residues.
</td>
</tr>
</table>
<p>To run a simple search for a sequence of residues, just type in the
sequence. 
</p>

<p>For a more complex search, in which you apply conditions at each residue
position, you can combine these elements to create a search pattern, such as
<code>G-[IL]-o{AC}</code>. There is an implied AND between contiguous elements:
so in the example given, <code>o{AC}</code> means a hydrophobic residue that is
not Ala or Cys. Each such sequence of elements that applies to a single residue
must be separated from the next sequence by a <code>-</code> character. The
search takes place when you press <span class="Keys">Enter</span>. The patterns
that are found are highlighted, and all other residues are colored white.
</p>

<p>As well as typing in patterns, you can store and retrieve your own patterns
from the <span class="GUI">Select Pattern</span> option menu. The patterns are
listed at the top of this option menu, with four default items, <span
class="GUI">Deamidation Site</span>, <span class="GUI">Glycosylation
Site</span>, <span class="GUI">Proteolysis Site</span>, and <span
class="GUI">Oxidation Site</span>. The last item is <span class="GUI">Edit
Patterns</span>, which opens the <span class="GUI">Edit Patterns</span> dialog
box, in which you can change patterns, including these default patterns, and add
and delete patterns. You can edit the table cells to change the pattern name,
the  definition, and the &quot;hotspot&quot;. The <span
class="GUI">Hotspot</span> column contains the residue index in the pattern that
should be selected when the pattern is found. This feature is ignored by the MSV
but is used in other applications that rely on the MSV (notably BioLuminate).
</p>


<a name="feature3"></a>
<h3>Sequence display area</h3>

<p>The MSV can handle multiple query sequences. Each is displayed in a separate
tab in the sequence display area. You can create a new tab by clicking the
&quot;+&quot; button to the top right of the display area, or by choosing <span
class="GUI">Sequences &rarr; New Query</span>. Using the &quot;+&quot; button
opens a dialog box to import a sequence; the menu choice creates an empty
query. The tabs are labeled <span class="GUI">Query 1</span>, <span
class="GUI">Query 2</span>, and so on, by default. You can change the name by
choosing <span class="GUI">Rename Query</span> from the tab shortcut
menu. Closing a tab with the <span class="GUI">X</span> button removes the
sequences and related data from the MSV.
</p>

<p>The sequence display area in each tab is divided into three sections. The
first of these is empty until you do an alignment with ClustalW. When the
alignment is done, this area contains a phylogeny tree diagram. The second
section displays the sequence name and the names of the various annotations. The
annotations can be expanded or collapsed individually, by clicking the
&quot;tree node&quot; (box) immediately to the left of the sequence name. They
can also be expanded or collapsed globally with <span class="Keys">Ctrl+DOWN
ARROW</span> and <span class="Keys">Ctrl+UP ARROW</span> (<span class="Keys">&#x2318;UP ARROW</span>).  Selected sequences are colored
slate blue. The third section displays the sequences and their annotations,
global annotations, and a ruler.
</p>
<p>Sequences are represented by the standard residue letter symbols. Residues
for which atom coordinates are missing are colored in a paler shade than
residues for which atom coordinates are available.
</p>
<p>Secondary structure predictions are represented by the characters H (helix),
E (extended), and - (everything else). Secondary structure assignments are
indicated by tubes for helices and arrows for extended structures. 
</p>

<p>There are three shortcut menus. The sequence shortcut menu opens when you
right-click in the sequence name section. The alignment shortcut menu opens when
you right-click on a sequence. The query shortcut menu opens when you right-click
on the tab name.
</p>

<h4>Sequence shortcut menu</h4> 
<p>This shortcut menu contains items from several of the main menus, and one
additional item. If you right-click on a sequence when there is no selection,
the sequence you click on is selected. Otherwise, right-clicking does not change
the selection. The menu items are listed below, with descriptions or links to
their descriptions in the main menus.
</p>
<dl>
<dt><span class="GUI">Set As Query Sequence</span></dt>
<dd><p>Make the selected sequence the query sequence. This item is only
present on the menu if there is a single sequence selected. To make the
consensus sequence the query, first display it with <span
class="GUI">Annotations &rarr; Consensus Sequence</span>, then use this command
to make it the query.
</p></dd>
<dt><span class="GUI">Select Ligand Contacts</span></dt>
<dd><p>Select the residues that are in contact with the ligand. This is a useful
way of selecting residues in a binding site. This item is present when you
right-click on a <span class="GUI">Ligand Contacts</span> annotation.</p></dd>
<dt><span class="GUI">Rename Sequence</span></dt>
<dd><p>Rename the sequence. Opens a dialog box in which you can enter a new
name. 
</p></dd>

<dt><span class="GUI">Translate DNA / RNA sequence</span></dt>
<dd><p>Translate a DNA or RNA sequence into the sequence for the protein it
codes for, using standard genetic code.</p></dd>

<dt><span class="GUI">Find Homologs (BLAST Search)</span></dt>
<dt><span class="GUI">Find Family (Pfam Search)</span></dt>
<dt><span class="GUI">Predict</span></dt>
<dd><p>These items are the same as on the <a href="#tools"><span
class="GUI">Tools</span> menu</a>.
</p></dd>
<dt><span class="GUI">Get PDB Structures</span></dt>
<dt><span class="GUI">Select All</span></dt>
<dt><span class="GUI">Deselect All</span></dt>
<dt><span class="GUI">Invert Selection</span></dt>
<dt><span class="GUI">Hide Selected</span></dt>
<dt><span class="GUI">Show All</span></dt>
<dt><span class="GUI">Set Color of Sequence Name</span></dt>
<dt><span class="GUI">Move Up</span></dt>
<dt><span class="GUI">Move Down</span></dt>
<dt><span class="GUI">Move to Top</span></dt>
<dt><span class="GUI">Move to Bottom</span></dt>
<dd><p>These items are the same as on the <a href="#sequences"><span
class="GUI">Sequences</span> menu</a>.
</p></dd>
<dt><span class="GUI">Delete Selection</span></dt>
<dd><p>This item is the same as on the <a href="#edit"><span
class="GUI">Edit</span> menu</a>.
</p></dd>
<dt><span class="GUI">Annotations</span></dt>
<dd><p>This submenu is a copy of the 
<a href="#annotations"><span class="GUI">Annotations</span> menu</a>.
</p></dd>
<dt><span class="GUI">Clear Annotations</span></dt>
<dd><p>Clear (remove) the annotations for the selected sequences.
</p></dd>
</dl>


<h4>Alignment shortcut menu</h4> 
<p>This shortcut menu contains items from the <span class="GUI">Alignment</span>
menu, and some additional items. 
</p>

<dl>
<dt><span class="GUI">Select Identities</span></dt>
<dt><span class="GUI">Select All</span></dt>
<dt><span class="GUI">Invert Selection</span></dt>
<dt><span class="GUI">Deselect All</span></dt>
<dt><span class="GUI">Delete</span></dt>
<dt><span class="GUI">Multiple Sequence Alignment</span></dt>
<dd><p>These items are the same as on the <a href="#sequences"><span
class="GUI">Alignment</span> menu</a>.
</p></dd>
<dt><span class="GUI">Anchor Residues Outside Selection</span></dt>
<dd><p>Set anchors on the residues outside the selection so that they do not
move at all during alignment.  The residues outside the selection are grayed
out. This action prevents new gaps from being created in a sequence, but you can
slide residues into existing gaps. To remove the anchors, click in an area
outside the sequences.
</p></dd>
<dt><span class="GUI">Clear Restricted Regions</span></dt>
<dd><p>Remove the anchors that were set by <span class="GUI">Anchor Residues
Outside Selection</span>.
</p></dd>

</dl>

<h4>Query shortcut menu</h4>

<p>This shortcut menu is displayed when you right-click the tab that contains
the query name. It has two items, which are also on the <span
class="GUI">Sequences</span> menu. 
</p>

<dl>
<dt><span class="GUI">Rename Query</span></dt>
<dd><p>Rename the current query. The name appears on the tab for the query.
</p></dd>

<dt><span class="GUI">Duplicate Query</span></dt>
<dd><p>Create a copy of the selected sequences in the current query in a new tab.
</p></dd>
</dl>

<a name="feature4"></a>
<h3>Status area</h3>
<p>The status area at the foot of the panel displays information on the current
task, on the sequences in the MSV project, and on the query sequence. 
</p>

<a name="acknowledgement"></a>
<h2>Acknowledgement</h2>
<p>The Multiple Sequence Viewer was developed in collaboration with
Dr. Jano Jusuf and Dr. Stanley Krystek from Bristol-Myers Squibb.</p>

<a name="links"></a>
<h2>Related Topics</h2>
<ul>
<li><a href="msv_build_model.html">Build Homology Model Panel</a></li>
<li><a href="msv_analyze_binding_site.html">Analyze Binding Sites Dialog Box</a></li>
<li><a href="../prime/find_homologs_step.html">Prime Find Homologs Step</a></li>
<li><a href="../prime/edit_alignment_step.html">Prime Edit Alignment Step</a></li>
<li><a href="../workspace/sequence_viewer.html">Sequence Viewer</a></li>
<li><a href="../project_menu/get_pdb_file_db.html">Get PDB File Dialog
Box</a></li>
</ul>
<hr />
<table width="100%">
<tr><td><p class="small"><a href="../support/legal_notice.html" target="LegalNoticeWindow">Legal Notice</a></p></td>
<td>
<table align="right">
<tr><td><p class="small">
File: tools_menu/multiple_sequence_viewer.html<br />
Last updated: 04 Nov 2014
</p></td></tr>
</table>
</td></tr>
</table>
</body>
</html>
