<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../../manstyle.css" />



<meta keywords>
<meta functions pro_SearchPDB>

<title>Protein Functions: pro_SearchPDB</title>

</head>

<body bgcolor="#ffffff">
<a href="../../index.htm"><img src="../../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>Protein Functions</h2>
<hr noshade="noshade" />

<h2>Syntax</h2>

<pre>
    family_lists = <a class="fcnlink" href="#pro_SearchPDB">pro_SearchPDB</a> [ query_set, search_options]

    search_options = [
	ktup		: int
	E_cutoff	: real,
	E_accept	: real,
	Z_cutoff	: real,
	Z_iter		: int,
	matrix		: 'token',
	gapstart	: real,
	gapextend	: real
    ]
</pre>

<h2>Description</h2>

<p>
Performs a multi-stage homology search for each sequence in the query set
against a database of protein structures and sequences 
that has been pre-clustered into families. An initial fast scan is performed,
using a generalized version of the Fasta methodology. Those families containing
at least one high-scoring sequence are then submitted to a Z-score
calculation.

<p>
The return vector <tt>family_lists</tt> is a vector of tagged vectors, one for
each query in the query set. The tagged vectors will include information
on each family found for a query.

<h4>Arguments</h4>

<dl>

<dt><tt>query_set</tt>


<dd>A vector of amino acid sequences specified by strings of single letter
codes.

<p><dt><tt>search_options</tt>

<dd>Vector of search parameters.
Optional arguments are attribute-value colon-separated pairs:

<dl>

<p><dt><tt>ktup:int</tt>
<dd> Strictly positive integer specifying the tuple size used during the
initial fast scan of the database. The maximum value is 3.
The default value is 1.

<p><dt><tt>E_cutoff:real</tt>
<dd>Maximum allowable expectation value for a family to be retained in the
family list. The default value is 10.

<p><dt><tt>E_accept:real</tt>
<dd>Families which report expectation values less than <tt>E_accept</tt>
 will always be retained in the family list, and the Z-score will not be
estimated. The default is 1e-12.

<p><dt><tt>Z_cutoff:real</tt>

<dd>Minimum Z-score that must be achieved in
order to retain a family in the family list. The Z-score is
estimated by generating a baseline score and standard deviation from the
alignment of a number of random permutations of the query sequence against the
the family. It is equal to the difference between the score of the
optimal alignment, between the query sequence and the family,
and the estimated baseline score, expressed in terms of the 
standard deviation of the random scores. The default is 6.

<p><dt><tt>Z_iter:int</tt>

<dd>Minimum number of randomized alignments to perform when estimating the
Z-scores. The default is 100.

<p><dt><tt>matrix:'token'</tt>

<dd>Name of the amino acid scoring matrix that is used in the sequence
alignment.  The name will be concatenated onto the value of the
configuration parameter in the 
<a href="../../appendix/moeconfig.htm">MOE configuration resource file</a>
(or using the Matrix Path field in the Protein Defaults page of the 
<a href="../../appendix/moeconfig.htm">Configuration Options</a> panel) to
produce a full path specification.  The default name is
<tt>'blosum62'</tt>.

<p><dt><tt>gapstart:real</tt>

<dd>Penalty for opening a gap in a sequence alignment. The default is -12.

<p><dt><tt>gapextend:real</tt>

<dd>Penalty for each residue aligned to a gap position in a sequence
alignment. The default is -2.

</dl>
</dl>


<h4>Return Values</h4>

For each query in the query set, a tagged vector is returned
giving details for all families identified in the search.
The tags, and their meanings, are:

<dl>
<p><dt><tt>E</tt></dt>
<dd> For each family, a vector of real numbers is returned representing
the expectation values calculated by the initial fast scan for each sequence
in the family.

<p><dt><tt>Z</tt></dt>
<dd> For each family, an estimated Z-score is returned unless the option
<tt>Z_iter</tt> is equal to zero. In that case, this value will be null.

<p><dt><tt>header</tt>
<dd>For each family, a vector of tokens is returned containing text information
for each of the sequences in the family.

<p><dt><tt>seq</tt>
<dd>For each family, a vector of character strings is returned, representing the
single-letter codes of the sequences.
The sequences are aligned to the query, and gap characters ("-") are inserted
into the sequences accordingly.

<p><dt><tt>query</tt>
<dd>For each family, a vector of characters representing the single-letter
codes of the query sequence. The query sequence has been aligned to the
family, and gap characters ("-") are inserted accordingly.

</dl>
<h2>References</h2>
<table width="100%" cellpadding="5" border="0">

<tr>
<td valign="top">[Pearson&nbsp;1996]</td>
<td valign="top">
Pearson, W.R.; Effective Protein Sequence Comparison; <i>Meth.
Enz.</i>; Vol. 266, Doolittle, R.F. (Ed.); Academic Press,
San Diego, (1996) 227-258.
</td>
</tr>
</table>


<h2>See Also</h2>

<a href="../prosearch.htm">Homology Searching</a><br />
<a href="intropro.html">Protein Functions</a>

<p>
    <a class="svl" href="pro_alig.html">pro_Align</a>
</p>

<p><a href="../../index.htm">MOE Table of Contents</a>&nbsp;&nbsp;<a href="../../fcnindex.html">SVL Function Index</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
