<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../../manstyle.css" />



<meta keywords>
<title>Protein Functions: pro_Search</title>


</head>
<body bgcolor="#ffffff">
<a href="../../index.htm"><img src="../../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>Protein Functions</h2>
<hr noshade="noshade" />

<h2>Syntax</h2>

<h3 class="obsolete">Obsolete</h3>

<pre>
    hitlist = pro_Search [ chain_key, search_options ]
    hitlist = pro_Search [ sequence_string, search_options ]
    pro_Search []

    search_options = [
	ID_cutoff   : real,
	Z_cutoff    : real,
	mda         : 'token',
	gapstart    : real,
	gapextend   : real,
	sec_struct  : flag,
	hfc         : flag
    ]
</pre>

Please note that this function has been
superseded by <a class="svl" href="pro_srch.htm">pro_SearchPDB</a>.

<h2>Description</h2>

<p>Searches the MOE homology databank for protein
structures that are homologous to a specified amino acid sequence.
The databank is a library of alignments
that was generated by clustering the chains in the MOE protein database along
with selected sequences from public domain sequence databases
into families of related structures.

<p>In batch mode, or when called with explicit arguments, a
<tt>hitlist</tt> is returned. The hitlist is a vector which contains a
sub-vector corresponding to each chain or family in the homology databank
which met the search criteria when compared to the query.  For each hit, a
vector with the following information is returned:

<pre>
    [ Zscore, hfc, code, header, family_info, query_positions ]
</pre>

<p>If passed null, <tt>pro_Search</tt> opens the MOE-SearchPDB panel and
allows many searches to be performed.



<h4>Arguments</h4>

<dl>

<dt><tt>chain_key</tt> or <tt>sequence_string</tt>


<dd>Key of chain in MOE or 
string of characters specifying the single-letter codes of the query sequence.

<p><dt><tt>search_options</tt>
<dd>Vector of search parameters.
Optional arguments are attribute-value colon-separated pairs:

<dl>

<p><dt><tt>ID_cutoff:real</tt>

<dd>Minimum percentage residue
identity that must be achieved in order to be submitted to the
Z-score (significance) test.  The default is 20.

<p><dt><tt>Z_cutoff:real</tt>

<dd>Minimum Z-score that must be achieved in
order to retain a search candidate in the hitlist. The Z-score is
estimated by generating a baseline score and standard deviation from the
alignment of a number of random permutations of the query sequence against the
candidate sequence. It is equal to the difference between the score of the
optimal alignment, between the query sequence and the candidate sequence,
and the estimated baseline score, expressed in terms of the 
standard deviation of the random scores. The default is 5.

<p><dt><tt>mda:'token'</tt>

<dd>Name of the amino acid scoring matrix that is used in the sequence
alignment.  The name will be concatenated onto the value of the
configuration parameter in the 
<a href="../../appendix/moeconfig.htm">MOE configuration resource file</a>
(or using the Matrix Path field in the Protein Defaults page of the 
<a href="../../appendix/moeconfig.htm">Configuration Options</a> panel) to
produce a full path specification.  The default name is
<tt>'blosum62'</tt>.

<p><dt><tt>gapstart:real</tt>

<dd>Penalty (a positive real
number) to be levied for each gap start. The default is 3.

<p><dt><tt>gapextend:real</tt>

<dd>Penalty (a real number) to be 
levied for each residue which matches a gap. The default is 1.

<p><dt><tt>sec_struct:flag</tt>

<dd>Set to 1 to use the predicted secondary structure
probabilities of the candidate and target sequence in the alignment
calculation. The default is 0 (off).

<p><dt><tt>hfc:flag</tt>

<dd>Set to 1 to calculate
the <a href="pro_thr.html">hydrophobic fitness score</a> of the
alignments of the query sequence to the top-ranked candidate structures.
The default is 0 (off).

</dl>

</dl>


<pre>
    Close[];
    pdb_Open '1HVC';		
    hits = pro_Search first Chains[]
</pre>


<h4>Return Values</h4>

For each hit,
a vector with the following information is returned:

<dl>
<dt><tt>Z-score</tt>
<dd>See explanation of Z-scores.

<p><dt><tt>hfc</tt>
<dd>Hydrophobic fitness score (if requested).

<p><dt><tt>code</tt>
<dd>Code of the PDB chain in the family most similar to the query.

<p><dt><tt>header</tt>
<dd>Header of the PDB chain in the family most similar to query.

<p><dt><tt>family_info</tt>
<dd>Sub-vector of the hitlist containing complete 
information on all sequences in the family:

<p>
<table cellpadding="3">
<tr>
<td valign="baseline"><tt>db_ref</tt></td>
<td valign="baseline">Tokens specifying database sources (<tt>'PDB'</tt>, <tt>'SWS'</tt>).</td>
</tr>
<tr>
<td valign="baseline"><tt>chain_codes</tt></td>
<td valign="baseline">Accession codes.</td>
</tr>
<tr>
<td valign="baseline"><tt>chain_headers</tt></td>
<td valign="baseline">Header information (PDB entries only).</td>
</tr>
<tr>
<td valign="baseline"><tt>sequences</tt></td>
<td valign="baseline">Vectors of 3-letter amino acid codes for each sequence.</td>
</tr>
<tr>
<td valign="baseline"><tt>res_positions</tt></td>
<td valign="baseline">Alignment positions of query sequence when aligned to family.</td>
</tr>
</table>

<p><dt><tt>query_positions</tt></td>
<dd>Alignment positions of query when aligned to family.

</dl>

<h2>See Also</h2>

<a href="../prosearch.htm">MOE-SearchPDB Panel</a><br />
<a href="intropro.html">Protein Functions</a>

<p>
    <a class="svl" href="pro_thr.html">Hydrophobic_Fitness_Z</a><br />
    <a class="svl" href="pro_alig.html">pro_Align</a>
</p>

<p><a href="../../index.htm">MOE Table of Contents</a>&nbsp;&nbsp;<a href="../../fcnindex.html">SVL Function Index</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
