<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<!--
!!   	plif.htm		Protein Ligand Interaction Fingerprints
!!   
!!	06-jun-2010 (kk) minor edits, screenshots
!!  	28-oct-2009 (sg) correction
!!	06-oct-2008 (kk) minor grammar corrections; table format
!!	03-oct-2008 (hf) minor grammar corrections
!!	24-sep-2008 (kk) copy edit; references updated
!!   	08-aug-2008 (ac) minor clarification
!!   	21-sep-2007 (ac) minor revisions
!!   	18-sep-2007 (ac) updated setup panel
!!   	16-sep-2007 (ac) finished
!!   	17-jul-2007 (ac) created
!!-->

<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../manstyle.css" />


<title>Protein Ligand Interaction Fingerprints</title>

<meta panel PLIF (Protein Ligand Interaction Fingerprints)>

</head>
<body bgcolor="#ffffff">
<a href="../index.htm"><img src="../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>Protein Ligand Interaction Fingerprints</h2>
<hr noshade="noshade" />

<ul>
<li><a href="#Overview">Overview</a>

<li><a href="#Generate">Generating Fingerprints</a>
    
<li><a href="#Using">Using Fingerprints</a>

    <ul>
    <li><a href="#UsingBarcodes">Barcode Display</a>
    <li><a href="#UsingPopulation">Population Display</a>
    <li><a href="#UsingSignificance">Significance Display</a>
    <li><a href="#UsingLigand">Ligand Display</a>
    <li><a href="#UsingFieldSel">Field Selection</a>
    <li><a href="#UsingFingSel">Fingerprint Selection</a>
    <li><a href="#UsingSelByFing">Entry Selection by Fingerprint</a>
    </ul>

<li><a href="#Tools">Fingerprint Tools</a>

    <ul>
    <li><a href="#ToolsBitSel">Bit Selector</a>
    <li><a href="#ToolsPoseSel">Pose Selector</a>
    <li><a href="#ToolsQuery">Query Generator</a>
    <li><a href="#ToolsSimil">Similarity</a>
    </ul>

</ul>

<!-- ============================== OVERVIEW ============================= -->

<p><a name="Overview"></a><h2>Overview</h2>

<p>The Protein Ligand Interaction Fingerprints (PLIF) tool is a method for
summarizing the interactions between ligands and proteins using a fingerprint
scheme. Interactions such as hydrogen bonds, ionic interactions and surface
contacts are classified according to the residue of origin, and built into a
fingerprint scheme which is representative of a given database of protein-ligand
complexes.

<p>The input data for PLIF can be from a variety of sources, which most
commonly include X-ray crystal structures and docking results. Using
fingerprints to collectively represent protein-ligand interactions for a large
database is an effective way of dealing with databases which are noisy and
error-prone due to the many difficulties involved in modeling ligands bound to
proteins.

<p>Fingerprints generated using PLIF are compatible with other fingerprint tools
found in MOE. Standard fingerprint tools such as clustering and diverse subsets
can be applied to fingerprints generated by PLIF. There is a specialized
visualization interface which is designed to take into account the specific
structural meaning of each fingerprint bit.

<p>In order to classify interaction fingerprints, each amino acid residue of a
protein is broken down into categories, as shown below:

<p><div align="center"><img src="plif/scheme.png"></div>

<p>There are 6 types of interactions in which a residue may participate:
sidechain hydrogen bonds (donor or acceptor), backbone hydrogen bonds (donor or
acceptor), ionic interactions, and surface interactions. The most potent of each
of these interactions in each category, if any, are considered.

<p>If no interactions of a particular category are found, or none pass the
thresholds, no bits are set for that category. If the strongest interaction 
passes the lower interaction threshold, the low order fingerprint bit is set. If
the strongest interaction passes the higher interaction threshold, then the low
order and high order bits are both set. Therefore the bit patterns for each
category can take on values of 00, 10 or 11, respectively.

<p>Hydrogen bonds between polar atoms are calculated using a method based on
protein contact statistics, whereby a pair of atoms are scored by distance and
orientation. The score is expressed as a percentage probability of being a good
hydrogen bond [Labute&nbsp;2001].

<p>Ionic interactions are scored by calculating the inverse square of the
distance between atoms with opposite formal charge (e.g. a carboxylate oxygen
atom and a protonated amine), and expressing as a percentage (100% corresponds
to 1 &Aring; distance).

<p>Surface contact interactions are determined by calculating the solvent
exposed surface area of the residue, first in the absence of the ligand, then in
the presence of the ligand. The difference between the two values is the extent
to which the ligand has shielded the residue from exposure to solvent, which is
potentially indicative of a hydrophobic interaction. The solvent exposed surface
area is determined by adding 1.4 &Aring; to the van der Waals radii of each
heavy atom, and computing the fraction of this total surface which does not lie
within the radius of any other.


<!-- ============================== GENERATE ============================= -->

<p><a name="Generate"></a><h2>Generating Fingerprints</h2>

<p>Before PLIF can be used, it is necessary to assemble one or more proteins to
serve as the receptor species, and some number of ligands with bound
conformations. There are two main ways to go about this:

<ol>

<p><li>Load a receptor into the MOE main window, and provide a database which
contains a molecule-type field for which each entry contains a single ligand
pose, the coordinates of which correspond to its orientation within the active
site;

<p><li>Provide a database which has a molecule-type field which contains both the
ligand and the receptor together in each entry.

</ol>

<p>The first method is most commonly applicable
when the source data comes from rigid-receptor docking results, in which a
single protein structure applies to many poses of many ligands for which bound
conformations have been calculated.
The second method is most commonly applicable when the source data comes from
a collection of crystal structures, for each of which the receptor is similar,
but not necessarily identical. 

<p>Open the MOE database, and select
<b>DBV&nbsp;|&nbsp;Compute&nbsp;|&nbsp;PLIF&nbsp;|&nbsp;Generate</b>.
The following panel will appear:

<p><div align="center"><img src="plif/generate.png"></div>

<p>The default options are configured to be reasonable for a database of bound
poses which are assumed to be complementary to a receptor which is currently
loaded in MOE.

<p><table width="100%" cellpadding="5">

<tr>
<td valign="top"><b>Receptor<br>Ligand</b>
<td valign="top">

For each entry in the underlying database, the preparation step which
calculates the raw interactions needs both a ligand and a receptor definition.
For databases which have many poses which are relative to a single receptor,
e.g. docking runs, it is conventional to store only the ligand in the database.
In such cases the receptor must be loaded into MOE, and it must apply to all
the entries which are being processed during preparation. For databases of
complexes where the receptor varies for each entry, it is necessary to store the
receptor for each of the bound ligands within the same molecule field.
For the <b>Receptor</b>, select <b>MOE</b> to use a currently loaded
protein as the receptor for all database entries for which interactions are to
be calculated. Select <b>Database</b> if the protein and ligand atoms are stored
together in the same database field.</p>

<ol>

<li>


<p>When <b>MOE</b> is selected, a choice of atom sets to use for the receptor
is offered:


<ul>

<li>Receptor Atoms
<li>Receptor+Solvent
<li>Selected Atoms
<li>Selected Residues
<li>Selected Chains

</ul>

<p><img src="plif/prosource1.png"/>

<p>The default selection is <tt>Receptor+Solvent</tt>. For this setting, it is
important to ensure that just one protein is loaded, and that all of the ligands
in the underlying database have 3D coordinates relative to the active site of
this protein.

<p>For <b>Ligand</b>, one of the molecule fields in the database must be
selected. This field is not modified by the generation of interaction data or
fingerprints.</p>

<li>

<p>If the receptor type is changed to <b>Database</b>, then a molecule field (to
immediate right) must be selected, and for each entry must contain a protein
<i>and</i> a ligand. Fields which do not contain distinguishable protein ligand
complexes will not give rise to meaningful fingerprint data.


<p><img src="plif/prosource2.png"/>



<p>The <b>Ligand</b> field must also be specified, and is indicated as a text
box. It is valid to select a field which does not exist, as it will be
overwritten with the extracted ligand.
The field must be different from the molecule field that contains the
protein-ligand complex.

</ol>

<tr>
<td valign="top"><b>Selected&nbsp;Entries Only</b>
<td valign="top">

If this option is checked, only entries which are selected in the Database
Viewer will be used to generate interactions or fingerprints.

<tr>
<td valign="top"><b>Rebuild&nbsp;Interactions</b>
<td valign="top">

If this option is not checked, the calculation of interactions which is
triggered by the <b>Prepare</b> button skips entries which already have
interactions assigned to them. If this option is checked, then all applicable
entries will have their interactions recalculated, and preexisting calculated
interactions will be overwritten. The default behavior is suitable for
sequential building of PLIF databases.

<tr>
<td valign="top"><b>Min Score 1<br>Min Score 2</b>
<td valign="top">

By default, each interaction category can occupy as many as two interaction
bits: low and high. It is possible to disable the generation of these bit types
by manually setting the minimum threshold to an impossibly high value, but for
convenience it is also possible to disable the two-bit scheme by un-checking
<b>Min&nbsp;Score&nbsp;1</b> or <b>Min&nbsp;Score&nbsp;2</b>

<tr>
<td valign="top"><b>Sidechain&nbsp;H-donor<br>H-acceptor</b>
<td valign="top">

Up to two fingerprint bits per residue can be assigned for interactions
between protein sidechain atoms acting as H-bond donors, and two more
fingerprint bits for protein sidechain atoms acting as H-bond acceptors. For
convenience, the generation of either of these types of interactions can be
disabled by un-checking the checkbox. The minimum thresholds are set by default
to 1% for the low bit, and 10% for the high bit, and may be modified as
necessary.

<tr>
<td valign="top"><b>Backbone&nbsp;H-donor<br>H-acceptor</b>
<td valign="top">

Up to two fingerprint bits per residue can be assigned for interactions
between protein backbone atoms acting as H-bond donors, and two more
fingerprint bits for protein backbone atoms acting as H-bond acceptors.

<tr>
<td valign="top"><b>Solvent&nbsp;H-donor<br>H-acceptor</b>
<td valign="top">

Up to two fingerprint bits per solvent molecule can be assigned for
interactions between solvent atoms acting as H-bond donors, and two more
fingerprint bits for solvent atoms acting as H-bond acceptors.

<tr>
<td valign="top"><b>Ionic&nbsp;Attraction</b>
<td valign="top">

Up to two fingerprint bits per residue can be assigned for ionic interactions
between some charged atom within the residue, and some oppositely charged atom
on the ligand.

<tr>
<td valign="top"><b>Surface&nbsp;Contact</b>
<td valign="top">

Up to two fingerprint bits per residue can be assigned for reduction of
solvent exposure on the residue due to the presence of the ligand. The property
is expressed as displaced surface area, scaled to a fraction.

<tr>
<td valign="top"><b>Maximum # Bits</b>
<td valign="top">

This value sets an upper limit on the number of bits in the fingerprint
scheme. Only the most common bits will be stored if this limit is lower than the
number of fingerprint bits that could be used to describe all of the
different types of qualifying interactions.

<tr>
<td valign="top"><b>Sequential Bit Definition</b>
<td valign="top">

When this option is checked, interactions will be described using the 
preexisting PLIF scheme for this database. Interactions that were not scored in
that scheme will be ignored. If there is no preexisting PLIF scheme in the
database, then the setting for this option is ignored.

<tr>
<td valign="top"><b>Prepare</b>
<td valign="top">

This button initiates the calculation of protein-ligand interactions, which
is the most resource intensive part of generating fingerprints. The raw
interactions are written to the underlying database, where they can be retrieved
in order to construct fingerprints. If the definition of the <b>Protein</b>
field indicates a molecule object in the MOE main window, it must be opened
before this button is pressed. By default, all of the entries in the database
are considered, and only those which have no interaction data are recalculated.

<tr>
<td valign="top"><b>Generate</b>
<td valign="top">

This button causes the generation of fingerprint bits, using the previously
calculated raw protein-ligand interaction data. By default, the fingerprint
scheme is rebuilt, and the contributions of all database entries are considered
when constructing it. Once the fingerprints have been calculated and written to
the database, the setup panel will automatically close, to be replaced by the
results panel.

<tr>
<td valign="top"><b>Show Results</b>
<td valign="top">

If some or all protein ligand interaction fingerprints have already been
calculated, this button can be used to open the results panel. 

</table>

<!-- ================================ USING ============================== -->

<p><a name="Using"></a><h2>Using Fingerprints</h2>

<p>After the protein ligand interaction fingerprints have been generated and
written to the database, the field <tt>FP:PLIF</tt> will be populated with data,
and the fingerprints available to generic MOE fingerprint functionality, such as
clustering and diverse subset calculation. The remainder of this page describes
visualization and derivative tools which are designed especially for use with
PLIF.

<p>To bring up the PLIF results panel, open the MOE database, and select
<b>DBV&nbsp;|&nbsp;Compute&nbsp;|&nbsp;PLIF&nbsp;|&nbsp;Analyze</b>.
The panel which appears should resemble the following:

<p><div align="center"><img src="plif/results.png"></div>

<p>The results panel is divided up into two main parts:

<ul>

<p><li>Graphic display (top), which reflects a particular view of the currently
selected entries.

<p><li>Entry list (center), which shows all of the fingerprints and other fields
in a listbox, allowing them to be selected. Various display options affect the
fields shown in the entry list, but at a minimum, the composition of the
visible fingerprint bits is shown. Each fingerprint bit is denoted by a
character to indicate its meaning:

<blockquote><table border="0" cellpadding="5">

<tr><td><tt>-</tt><td>bit not set

<tr><td><tt>D</tt><td>sidechain hydrogen bond donor

<tr><td><tt>A</tt><td>sidechain hydrogen bond acceptor

<tr><td><tt>d</tt><td>backbone hydrogen bond donor

<tr><td><tt>a</tt><td>backbone hydrogen bond acceptor

<tr><td><tt>O</tt><td>solvent hydrogen bond

<tr><td><tt>I</tt><td>ionic attraction

<tr><td><tt>C</tt><td>surface contact

</table></blockquote>

</ul>

<p>It is also adorned with a number of options and buttons, which control the
presentation and interaction with the underlying data.

<p><table width="100%" cellpadding="5">

<tr>
<td valign="top"><b>Display Mode</b>
<td valign="top">

The graphical display shown at the top of the panel contains a representation
which is based on the entries currently selected in the entry list, and the
selected display mode. The available display modes are  <a
href="#UsingBarcodes"> <b>Barcodes</b></a>, <a href="#UsingPopulation">
<b>Population</b></a>,  <a href="#UsingSignificance"> <b>Significance</b></a>
and  <a href="#UsingLigand"> <b>Ligands</b></a>, each of which is described
subsequently.

<tr>
<td valign="top"><b>Export...</b>
<td valign="top">

Allows the current state of the graphical display to be exported as a
picture, using a number of different formats.

<tr>
<td valign="top"><b>Tools</b>
<td valign="top">

When clicked, offers a selection of <a href="#Tools">external tools</a>
designed for use with PLIF.

<tr>
<td valign="top"><b>Select</b>
<td valign="top">

The selection buttons <b>Prev</b>, <b>Next</b> and <b>All</b> are designed
for convenient selection of groups of entries within the main entry list. While
the <b>All</b> button consistently selects all entries, the behavior of
<b>Prev</b> and <b>Next</b> vary depending on the sort order of the entries. By
default, entries are sorted by order of occurrence in the database, so both of
these buttons will select a single entry which is either before or after the
current selection. If sorting criteria have been provided, these buttons will
select the previous or next distinct group which has the same primary index
values.

<p>For example, if the database is sorted by a field called <tt>id</tt> which
has values of 1, 2 and 3, then pressing the <b>Next</b> button will cycle
through entries which have this value.

<tr>
<td valign="top"><b>Activity</b>
<td valign="top">

For some visualization and calculation purposes, it is necessary to define
entries as having a binary activity classification, which evaluates to 1
(active) or 0 (inactive). If such information is not available, activity will
not be used. The Activity field consists of three parts: field, comparison and
value. The field list contains numeric fields in the underlying database. The
comparison can be greater to or less than, and the value can be any number. 

<p>For the sake of convenience, the results panel checks to see if there is a
numeric field called <tt>is_active</tt>, and if so, selects this field and
defines entries to be active if the value is greater than 0.5, which is
appropriate if all entries contain a value of 1 for actives and 0 for inactives.
If activity is stored in some other form, such as pKi or IC50, then a threshold
may be customized as necessary. If none of the fields contain activity,
select the field labeled <tt>&lt;none&gt;</tt>.

<p>Note that in the entry list, entries which are classified as active are
labeled with a <tt>*</tt> character under the column heading <tt>A</tt>.

<tr>
<td valign="top"><b>Verbose List</b>
<td valign="top">

This option controls the level of detail displayed in the entry list. If off
(default), then the fingerprints for each entry are displayed in a concise
char-packed form, with no labels to indicate which residues are associated with
each bit:

<p><img src="plif/verbose0.png">

<p>If on, the bit patterns will be split up into groups of residues, each of
which is labeled by residue number in the column header:

<p><img src="plif/verbose1.png">

<tr>
<td valign="top"><b>Include&nbsp;Selected&nbsp;Only</b>
<td valign="top">

If on, the next time the <b>Update from DBV</b> button is pressed, the
results panel will be populated only with database entries that were selected in
the database viewer. If the database viewer is not open, or there are no
selected entries, then all entries will be loaded, as if this option were not
checked.

<tr>
<td valign="top"><b>Show&nbsp;Ligand&nbsp;in&nbsp;MOE</b>
<td valign="top">

If on, or whenever changed, the ligands corresponding to the selected entries
in the entry list are created in the main MOE window, and colored by chain
color. When the entry selection changes, or this option is turned off, or the
panel closes, these molecules will be removed.

<tr>
<td valign="top"><b>Fields...</b>
<td valign="top">

Brings up the <a href="#UsingFieldSel">Field Selection</a> dialog.

<tr>
<td valign="top"><b>Fingerprints...</b>
<td valign="top">

Brings up the <a href="#UsingFingSel">Fingerprint Selection</a> dialog.

<tr>
<td valign="top"><b>Select...</b>
<td valign="top">

Brings up the <a href="#UsingSelByFing">Entry Selection by Fingerprint</a>
dialog.

<tr>
<td valign="top"><b>Recalculate</b>
<td valign="top">

Brings up the <a href="#Generate">Fingerprint Setup</a> dialog, which can be
used to modify the fingerprints in the database.

<tr>
<td valign="top"><b>Update from DBV</b>
<td valign="top">

Re-reads the underlying data from the source database, and replaces the
current content. This is necessary if the content of the database has been
changed by external means, and it is desired that the results panel reflect
these changes.

<tr>
<td valign="top"><b>Select in DBV</b>
<td valign="top">

Synchronizes the selection in the entry list with the corresponding database
viewer. The database viewer will be opened if it is not already.

<tr>
<td valign="top"><b>View</b>
<td valign="top">

For the selected entries in the entry list, the ligands and the corresponding
proteins will be created in the MOE main window, deleting any preexisting
content. The ensemble will then be viewed using the Ligand Interactions panel.
If there are multiple selected entries, then the concurrent alignment method
will be used by default.

<tr>
<td valign="top"><b>Print...</b>
<td valign="top">

Brings up the standard MOE print preview dialog, with the following options:

<p><img src="plif/print.png">

<p>If <b>Bit Descriptions</b> is on, the page layout begins with a summary of
the fingerprint scheme, which provides the fingerprint indices, residue IDs and
interaction types for each. <b>Font Size</b> may be adjusted as necessary.

<p>If any or all of <b>Barcode Form</b>, <b>Population Histogram</b>,
<b>Activity Significance</b> or <b>Ligands</b> are checked, then each of these
graphical display types will be included in the page layout. The manner in which
they are drawn is the same as for the panel.

</table>

<p><a name="UsingBarcodes"></a><h2>Barcode Display</h2>

<p>The "barcode" display represents the selected entries and selected
fingerprints as a matrix (one row per entry, one column per fingerprint bit)
in which a set bit is drawn as a black rectangle:

<p><div align="center"><img src="plif/gfx_bc.png"></div>

<p>The X-axis shows the residue numbers that correspond to each group of
fingerprint bits, and is coded with an arbitrary sequence of colors. Information
about the amino acid codes which compose the residues is presented when
available. If all of the proteins involved have the same residue composition
with regards to the interactions, then the 3-letter amino acid codes are drawn
above the residue ID numbers (e.g. His, Asp, Met, etc.). If there is a mixture
of residue compositions, then single letter codes will be used (e.g. H, D, M,
etc). If a particular interaction fingerprint involves two different types of
residues, the two letters will be concatenated (e.g. HD). If there are three or
more types of residues involved, the description will be replaced by <tt>*</tt>.
If the residue code does not correspond to a naturally occurring amino acid, the
description will be <tt>?</tt>.

<p><a name="UsingPopulation"></a><h2>Population Display</h2>

<p>The population histogram displays the frequency of occurrence amongst the
selected entries for each of the selected fingerprint bits. The X-axis is
constructed in the same way as for the barcode display, while the Y-axis shows a
black bar which shows the relative counts for the bits:

<p><div align="center"><img src="plif/gfx_pop.png"></div>

<p>At the top of the largest bar is shown the bit count, the number of entries
involved, and the percentage of the entries which feature this bit.

<p>When the population display is selected, there is an additional option called
<b>Normalize</b>. If this is checked, the base peak is scaled to take up the
maximum amount of size in the graphic. If not checked, then the base peak will
be scaled in proportion to the fraction of the entries in which it occurs (e.g.
if 100 entries are selected, and the most populous bit occurs in 50 entries,
then the largest bar will be approximately half the size of the picture).

<p>If activity information is available, then each of the frequency bars are
further annotated with a qualitative indication of the overabundance of actives
vs. inactives. The fraction of actives containing the bit (<b>a</b>) is compared
with the fraction of actives overall (<b>b</b>). If <b>a&nbsp;&gt;&nbsp;b</b>,
then the extent to which it is greater is indicated by drawing a
black line above the bar. If <b>a&nbsp;&lt;&nbsp;b</b>, then the extent to
which it is less is indicated by drawing an inverse (white) line below the top
of the bar:

<p><div align="center"><img src="plif/overabund.png"></div>

<p>The combination of height and overabundance is often an indication that the
corresponding interaction is important for activity. If an interaction occurs
frequently, but no more often in active compounds than inactive, then it is
unlikely to be of interest as a distinguishing feature of actively binding
ligands; similarly if an interaction occurs much more often in active compounds
than in inactive, but its overall occurrence is rare, it is unlikely to be a
crucial determining factor.

<p><a name="UsingSignificance"></a><h2>Significance Display</h2>

<p>The significance chart is related to the population histogram, except that
the heights of the bars are constructed to indicate the extent to which each
fingerprint bit is an important prerequisite for activity. The display differs
in that the bars can have positive or negative values, and make use of grayscale
hues:

<p><div align="center"><img src="plif/gfx_sig.png"></div>

<p>The height of the bars for each fingerprint bit are a product of the
frequency of the bit and the normalized probability of an entry being active if
it contains the bit. The probability is calculated as:

<p><div align="center"><img src="plif/sigfmla1.png"></div>

<p>where A<sub>0</sub> and I<sub>0</sub> are the number of actives and inactives
within the set overall, and A<sub>b</sub> and I<sub>b</sub> are the number of
actives and inactives within the subset of entries for which the particular bit
is set.

<p>The height of the bar for the particular bit, Q<sub>b</sub>, is calculated
as:

<p><div align="center"><img src="plif/sigfmla2.png"></div>

<p>where C<sub>b</sub> is the number of entries within the set which have the
bit set, and C<sub>0</sub> is the size of the set. The value of Q<sub>b</sub>
lies between -1 and 1. A positive value indicates that the bit occurs
frequently in general, and even more frequently within active compounds,
relative to the proportion of actives within the set. A negative value indicates
that the bit occurs more often in the inactives.

<p>The significance of each bit is calculated by proposing the hypothesis that
if the bit is set, then the compound is active. This hypothesis is the compared
to a random model based on the overall fraction of entries which are marked as
active:

<p><div align="center"><img src="plif/sigfmla3.png"></div>

<p><div align="center"><img src="plif/sigfmla4.png"></div>

<p><div align="center"><img src="plif/sigfmla5.png"></div>

<p>The properties of the hypothesis, N<sub>b</sub>, are summarized by the number
of true positives, true negatives, false positives and false negatives (TP, TN,
FP, FN). The properties of the random model, E<sub>b</sub>, are obtained from
the fraction of the set which contains the bit, and the fraction of the set
which is classified as active.

<p>The bars are color-coded so that highly significant fingerprint bits (low
<img src="plif/sigfmla5a.png">) are drawn in black/dark gray, while less
significant bits are drawn in lighter shades of gray.

<p><a name="UsingLigand"></a><h2>Ligand Display</h2>

<p>The ligand display graphic shows the ligands of the selected entries, in 2D
diagrammatic form, with a concurrent alignment that is representative of
bound conformation, and further annotated with an indication of the interactions
in which it participates:

<p><div align="center"><img src="plif/gfx_lig.png"></div>

<p>The 3D coordinates of the ligands are first converted into 2D coordinates
using the 2D depiction method [Clark&nbsp;2006], with a flattening constraint,
which forces the layout algorithm to compromise pure aesthetics in order to
find a planar representation which resembles the 3D form [Clark&nbsp;2007].
Once all of the poses have been flattened, they are superposed onto their
original 3D positions, then collectively rotated onto the plane of the page.
In order to compose this graphic, they are shown alongside each other.

<p>The relative layout of the ligands will be meaningful only if the ligand
coordinates for the selected entries are relative to the same protein, or are
relative to multiple proteins which have been superimposed onto a common space.

<p>For each ligand, the fingerprints are examined, and those which correspond to
a hydrogen bond are correlated to the individual ligand atoms which were found
to be involved in the interaction. These ligand atoms are annotated with a
circle, which is colored in the same way as the X-axis for the barcode,
population and significance graphics. Surface contact interactions are annotated
differently, as the number of ligand atoms affected is typically many; a dotted
outline is drawn around the ligand atoms which are close to the residue.

<p>If more than 50 ligands have been selected, only the first 50 will be
displayed.

<p><a name="UsingFieldSel"></a><h2>Field Selection</h2>

<p>By default, the main entry list shows index, activity and fingerprint bits
for each entry, and sorts these by database order. It is possible to show
additional fields in this display, and also to sort by numerical fields:

<p><div align="center"><img src="plif/dlg_fields.png"></div>

<p>The fields listed underneath <b>Fields to Show</b> can be toggled on or off,
and will be updated accordingly in the main entry list.

<p>The sort order can be modified by selecting up to six fields underneath
<b>Sorting Indices</b>, and each individual sort may be performed either up or
down (e.g. if the field <tt>is_active</tt> is defined, selecting this field in
<b>Sort #1</b> and changing the direction to Down would cause all of the active
compounds to be listed at the beginning; also, the <b>Select/Next</b> button
would toggle between display of actives vs. inactives).

<p><a name="UsingFingSel"></a><h2>Fingerprint Selection</h2>

<p>Groups of fingerprint bits can be selected or deselected as necessary, which
can clear up room in the main entry list and graphical display:

<p><div align="center"><img src="plif/dlg_fprints.png"></div>

<p>Each check box corresponds to a particular residue ID. If unchecked, none
of the corresponding fingerprint bits will be shown, and more space will be
available to visualize the remaining bits. 

<p><b>Select All</b> and <b>Clear All</b> can be used for convenience.
<b>Include Surface Contacts</b> can optionally be unchecked in order to exclude
bits which are described as surface contacts, leaving only the polar
interactions.

<p><a name="UsingSelByFing"></a><h2>Entry Selection by Fingerprint</h2>

<p>Entries in the main list can be selected based on the presence of selected
fingerprint bits, using the selection panel:

<p><div align="center"><img src="plif/dlg_selby.png"></div>

<p>Some combination of fingerprint specifiers must be selected in the main list
in this dialog panel. For each residue in the fingerprint scheme, there is one
entry labeled "any interaction", which translates to a positive match if any
fingerprint bit related to that residue is set. For each of the distinct bit
types, an additional entry is provided.

<p>Each entry either matches or does not, based on the fingerprints selected,
and the value of <b>Logical Operation</b>. In the case of <tt>OR</tt>, an entry
matches if any of the selected fingerprints applies. In the case of
<tt>AND</tt>, an entry only matches if all of the selected fingerprints apply.

<p>The changes to the selection in the main results panel are then modified by
the <b>Current Selection</b> state. If set to <b>Clear</b>, then the selection
is completely replaced whether or not the entry matched. If <b>Add To</b>,
then any entry that is matched is selected, and anything previously selected
remains so. If <b>Subtract</b>, then any entry which matches is removed from the
selection, if it was selected previously. If <b>Restrict</b>, then an entry must
match and already be selected in order to remain so.

<!-- ================================ TOOLS ============================== -->

<p><a name="Tools"></a><h2>Fingerprint Tools</h2>

<p>The tools described in this section are not an integral part of PLIF, but can
be accessed conveniently from the <b>Tools</b> menu in the PLIF results panel.
Each of the tools makes use of existing fingerprint data, and may derive a
particular property, or modify the content of the database.

<p>Each tool brings up an independently operated panel. Any changes to the data
which are made by an external tool will not be automatically reflected in the
corresponding PLIF results panel, so it may be necessary to use the
<b>Update&nbsp;from&nbsp;DBV</b> button in order to refresh the data view.
Similarly, tools which require a set of entries to be provided do not obtain
this selection from the PLIF result panel, but may instead use the selection
state in the database viewer, which may be synchronized using the
<b>Select&nbsp;in&nbsp;DBV</b> button.

<p><a name="ToolsBitSel"></a><h3>Bit Selector</h3>

<p>Detailed information about the nature and statistics of each fingerprint is
presented by this tool: 

<p><div align="center"><img src="plif/tool_bitsel.png"></div>

<p>The fingerprint bits may be sorted and selected using
particular criteria, which may then be used to remove unwanted bits from the
fingerprint scheme. Also, the raw data used to generate the fingerprint bits can
be extracted.

<p><table width="100%" cellpadding="5">

<tr>
<td valign="top"><b>Activity</b>
<td valign="top">

The definition of active vs. inactive may be specified with this option, by
selecting a field and a cutoff threshold. If a boolean field called
<tt>is_active</tt> is defined, this will be set by default. The definition of
activity affects some of the statistics which are presented.

<tr>
<td valign="top"><b>List</b>
<td valign="top">

In the main list, each of the fingerprint bits is shown on one line, in the
currently sorted order. The bit position, residue ID and interaction type are
presented in the first three columns. The final six columns display statistical
information about the participation of each of the bits within the database as a
whole:

<blockquote><dl>

<p><dt><tt>%abund</tt>

<dd>The percentage abundance of the fingerprint bit throughout the database. If
the fingerprint bit occurs in every entry, the value will be 100.

<p><dt><tt>%active</tt>

<dd>The percentage abundance of the fingerprint bit within the entries which are
classified as active.

<p><dt><tt>%relabn</tt>

<dd>The percentage abundance of the fingerprint bit, scaled so that the most
abundant bit has a value of 100%, e.g. if the most populous fingerprint bit
occurred in 30 entries, a fingerprint bit which occurred in 15 entries would
have a <tt>%relabn</tt> value of 50%, regardless of the total number of entries.

<p><dt><tt>%relact</tt>

<dd>The percentage abundance of the fingerprint bit within the entries
classified as active, scaled so that the most populous active bit is set to
100%. The calculation is identical as for <tt>%relabn</tt>, but using only
active entries.

<p><dt><tt>%ovract</tt>

<dd>The over-activity, which is given as the difference between <tt>%active</tt>
and <tt>%abund</tt>. A positive value means that the bit occurs more often in
active compounds than it does in the database overall.

<p><dt><tt>%wovact</tt>

<dd>The value of <tt>%ovact</tt> multiplied by <tt>%abund</tt>, expressed as a
percentage. Scaling the over-activity by the overall abundance gives a number
which gives a rough indication of when the bit frequently selects in favor of
activity.  Bits with higher values are likely to be necessary for activity,
whereas bits with small or negative values are likely to be unimportant or
actually anti-selective.

</dl></blockquote>

<tr>
<td valign="top"><b>Sort by</b>
<td valign="top">

The default sorting value is <tt>none</tt>, in which case the bits are listed
in their natural order. The value can be changed to any of the calculated
statistical values presented in the list above, in which case the list will be
sorted highest first according to the selected property.

<tr>
<td valign="top"><b>Where</b>
<td valign="top">

Minimum or maximum ranges of a selected property can be used to select or
deselect entries in the list. Select first the desired property, then enter a
minimum and/or maximum bound, and press the <b>Select</b> or <b>Deselect</b>
buttons to apply the effect to the list.

<tr>
<td valign="top"><b>Select&nbsp;All<br>Clear&nbsp;Selection
<td valign="top">

For convenience, the entire set of bits can be selected or deselected with a
single button click.

<tr>
<td valign="top"><b>Prune</b>
<td valign="top">

Given that some number of bits have been selected in the list, the
fingerprint scheme will be modified so that <i>only</i> the selected bits remain
in the scheme. All other bits will be removed, and each entry in the database
will be updated accordingly. Next time the PLIF results panel is updated, the
bit subset will be reflected in the display.

<tr>
<td valign="top"><b>Extract</b>
<td valign="top">

Since the interaction data upon which the PLIF fingerprints are based is
stored internally within the database, it is possible to extract this data, and
for each residue/interaction type, create a new column in the database and
output the strength of the corresponding interaction with the ligand (or 0 if
none, maximum if more than one). The resulting columns can then be used as
continuous descriptors, e.g. for QSAR studies. 

</table>

<p><a name="ToolsPoseSel"></a><h3>Pose Selector</h3>

<p>One of the possible uses for interaction fingerprints is to disambiguate the
many poses generated by computational docking algorithms, by studying the
binding modes of a number of ligands, and deciding which of their putative poses
is most likely to be the correct one, in the context of all the ligands. The
<b>Pose Selector</b> tool uses activity data for the ligands to discover a
common binding motif which is indicative of activity. The interaction patterns
which most commonly emerge from active ligands should contain a structural motif
which, if discovered, will lead to a way to eliminate many of the superfluous
poses.

<p>This collective method is quite orthogonal to conventional docking scoring
functions, which typically consider each ligand in isolation, often resulting in
many poses which score similarly well but have quite different binding
orientations. The method used by this tool is however based on the assumptions
that (1) there is just one binding mode, and that (2) for most of the ligands,
at least one pose exists which demonstrates the important interactions.

<p>The algorithm operates by postulating a series of hypotheses of the form:

<blockquote>
<p>ACTIVE = {contains bits <b>A</b>} AND {does not contain bits <b>B</b>}
</blockquote>

<p>where <b>A</b> and <b>B</b> are arbitrary sets of bits. A ligand is
considered as being predicted to be active if any of its constituent poses is
classified as active by the hypothesis. The efficacy of each
hypothesis is determined by:

<blockquote>
<p>SCORE = {% true positives} + {% true negatives} 
- {% false positives} - {% false negatives}
</blockquote>

<p>Hypotheses are initially generated at random, where the set <b>A</b> is
weighted in favor of selecting bits which are frequently observed in active
compounds. After a predetermined time, the best randomly determined hypotheses
are subjected to iterative refinement in order to improve the efficacy scores.

<p>If the final hypothesis has a score close to zero, it means that no pattern
was found which distinguishes actives from inactives. If the final score is
close to 100%, then the selected motif has a near perfect ability to distinguish
actives from inactives.

<p>For each group of poses belonging to an active ligand, if there is only one
pose which satisfies the hypothesis, then that pose is selected as being the
most probable active pose. If none or multiple poses satisfy the hypothesis,
then that with the highest Tanimoto similarity to the set <b>A</b> is retained,
or in the event of a draw, the first entry.

<p><div align="center"><img src="plif/tool_posesel.png"></div>

<p>When the <b>Fit</b> button is selected, the algorithm for selecting active
poses begins. When it is complete, the <b>Write</b> button becomes available,
and the results may be written to the underlying database. A database field will
be created if necessary: <tt>PLIF_apose</tt>, the value of which will be 0 for
any entry which is classified as inactive. For each group of poses from active
ligands, one entry will have a value of 1, and the others will have a value of
0. The database will thence be split into three divisions: active poses,
inactive poses, and inactive ligands.

<p><a name="ToolsQuery"></a><h3>Query Generator</h3>

<p>Because ligand interaction fingerprints are directly derived from actual
structural information, it is possible to at any time backtrack the fingerprint
data to actual 3D interactions. The <b>Query Generator</b> tool operates on the
principle that a modest selection of poses with a homogeneous set of interaction
fingerprints will most likely also possess a homogeneous set of pharmacophoric
feature points. The pharmacophoric feature points can be grouped and filtered
according to which residues they interact with, and those with a sufficiently
tight grouping can be converted into a pharmacophore query feature.

<p>The tightness of the resulting collection of query features is highly
dependent on the uniformity of the selected poses. While not all queries
generated in such a way are suitable for screening databases of conformations
without binding modes, when applied in absolute mode to poses already docked
into the protein, or as a constraint on further docking calculations, the
resultant queries are a very effective way to make use of interaction
information gleaned from PLIF.

<p><div align="center">
    <img src="plif/tool_querygen.png">
    <img src="plif/ph4query.png">
</div>

<p>The main dialog box allows control of the various parameters that go into the
construction of the 3D query (above left). The query features are shown in the
MOE main window (above right).

<p><table width="100%" cellpadding="5">

<tr>
<td valign="top"><b>Entries</b>
<td valign="top">

Before any queries can be generated, the subset of poses to operate on must
be established. This subset should ideally be between 10 and 100 poses, and
should have a high degree of commonality in binding modes, i.e. the interaction
fingerprint bits which are important to activity should be conserved throughout
the set to the extent possible.

<p>If set to <b>All</b>, then the entire database will be considered as the
input. If set to <b>Selected</b>, then all entries which are selected in the
Database Viewer will be considered (note that this is <i>not</i> the selection
within the main PLIF results panel; use the <b>Select&nbsp;in&nbsp;DBV</b>
button to synchronize these). If set to <b>Active&nbsp;Poses</b>, then all
entries for which the value of the field <tt>PLIF_apose</tt> is true will be
considered.

<tr>
<td valign="top"><b>Scheme</b>
<td valign="top">

The pharmacophore query scheme to use for generating the query features. All
of the schemes available in MOE are presented in this list.

<tr>
<td valign="top"><b>List of fingerprints</b>
<td valign="top">

The fingerprint bits which correspond to at least one pharmacophore feature
are listed in this list box, along with the type and number of poses in which
they occur. These fingerprint bits are all selected by default, which means any
of them may be combined into the final query features. They may be deselected,
in which case they will be removed from consideration.

<tr>
<td valign="top"><b>Max.&nbsp;Radius</b>
<td valign="top">

Defines the maximum radius of a query feature. If a group of interaction
feature points cannot be grouped together with a radius of less than this value,
the query feature will be split into smaller groups, or excluded.

<tr>
<td valign="top"><b>Feature Coverage</b>
<td valign="top">

Each proposed query feature must hit at least this fraction of the provided
input poses, e.g. if 10 poses are provided, and the threshold is set to 60%, a
proposed query feature which only matches the pharmacophore points of 5 or less
poses will be rejected, but one which matches 6 or more will be included in the
final results.

<tr>
<td valign="top"><b>List of query features</b>
<td valign="top">

All of the query features which qualify according to the thresholds will be
shown in this list box, and display in the MOE main window. By default all are
selected, but they can be deselected, in which case they will not be displayed,
nor will they be included when the pharmacophore query is saved.

<tr>
<td valign="top"><b>Show Original Features</b>
<td valign="top">

If on, the individual pharmacophoric features of all the input poses will be
displayed in the MOE main window, in addition to the query features derived from
them.

<tr>
<td valign="top"><b>Excluded Volumes</b>
<td valign="top">

If on, then the region of space occupied by the protein will be included in
the query in the form of excluded volumes. This is of interest when the query is
intended to be used in screening mode, for matching conformations without
explicitly binding to the protein.

<tr>
<td valign="top"><b>Build</b>
<td valign="top">

Constructs the pharmacophore interaction points using the indicated input
pose and the selected scheme. From this point on, the pharmacophore query result
is updated dynamically.

<tr>
<td valign="top"><b>Evaluate</b>
<td valign="top">

Runs through the source database and evaluates the extent to which the
pharmacophore query matches the contents, in both absolute and relative modes,
and provides information as to the ratio of actives and inactives which match.
An entry is considered to match in absolute mode if the bound pose matches  the
pharmacophore query without any rotation or translation. It is considered a
match in relative mode if the conformation is capable of fitting the query by
applying any rigid transformation. This is a convenient way of approximately
evaluating the efficacy of the query, without proper statistical validation. The
output is written to the SVL Commands window.

<tr>
<td valign="top"><b>Create</b>
<td valign="top">

Creates the pharmacophore query, closes the dialog, then opens the
pharmacophore query editor in its stead.

</table>

<p><a name="ToolsSimil"></a><h3>Similarity</h3>

<p>Using the interaction fingerprint bits to compare similarity between a
selected subset of the database and all the rest can be a useful way to
estimate properties or segregate the data in various ways. For instance, if one
or more entries have been obtained from crystal structures of known active
ligands, the activity of the remaining data might be estimated by means of
comparing the Tanimoto coefficients between their fingerprints and those of the
remaining entries.

<p><div align="center"><img src="plif/tool_simil.png"></div>

<p><table width="100%" cellpadding="5">

<tr>
<td valign="top"><b>Compare&nbsp;To</b>
<td valign="top">

The subset of entries to which the remaining entries are to be compared to is
defined by this field. The default value, <tt>selected</tt>, implies that the
selected entries in the Database Viewer define that set. The other entries
consist of the numerical fields in the database, and these may be used in the
form of a cutoff. Generally the set should be small, e.g. a handful of
known-active correct binding poses.

<tr>
<td valign="top"><b>Similarity</b>
<td valign="top">

The similarity metric may be either the Tanimoto coefficient or the
Euclidean distance. Note that for Tanimoto coefficients, larger numbers indicate
more similarity, while for the Euclidean the opposite is true.

<tr>
<td valign="top"><b>Collective</b>
<td valign="top">

The collective behavior of comparison must be selected. If <b>max</b> is
selected, then each entry is compared with the selection set, and the highest
value is chosen (for Tanimoto, this means most similar, for Euclidean this means
least similar). Conversely, <b>min</b> causes the lowest value to be chosen. If
<b>mean</b> is selected, then the similarities between a particular entry and
all of the selected set are computed, then the mean average taken.

<tr>
<td valign="top"><b>Output To</b>
<td valign="top">

The field to which the resulting similarity value is written, which defaults
to <tt>PLIF_sim</tt>.

</table>

<p>To conduct the calculation and write the results to the underlying database,
press the <b>Calculate</b> button.


<!-- ============================ REFERENCES ============================= -->

<p>
<h2>See Also</h2>
<p>
<a href="surfmap.htm">Surfaces and Maps</a><br>
<a href="sitefind.htm">Site Finder</a><br>
<a href="docking.htm">Molecular Docking</a><br>


<p><a name="Ref"></a><h2>References</h2>

<table cellpadding=5 width=100%>


<tr>
<td valign="top">[Labute&nbsp;2001]
<td valign="top">
Labute, P.; Probabilistic Receptor Potentials;
<i>Journal of the Chemical Computing Group</i> (<b>2001</b>)
<a href="http://www.chemcomp.com/journal/cstat.htm">
http://www.chemcomp.com/journal/cstat.htm</a>.

<tr>
<td valign="top">[Clark&nbsp;2006]
<td valign="top">
(a) Clark, A.M., Labute, P., Santavy, M.;
2D Structure Depiction;
<i>J. Chem. Inf.  Model. 46</i> (<b>2006</b>) 1107&ndash;1123. 
<br>(b) <a href="http://www.chemcomp.com/journal/depictor.htm">
http://www.chemcomp.com/journal/depictor.htm</a>
</tr>

<tr> 
<td valign="top">[Clark&nbsp;2007] 
<td valign="top"> 
(a) Clark, A.M., Labute, P.; 2D Depiction of Protein-Ligand Complexes; <i>J.
Chem. Inf. Model. 47</i> (<b>2007</b>) 1933&ndash;1944.
<br/>
(b) <a href="http://www.chemcomp.com/journal/ligintdia.htm">
http://www.chemcomp.com/journal/ligintdia.htm</a>
</tr>

</table>

<p><a href="../index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
