<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<!--
!!   	kinase.htm	Kinase Search and Kinase Update
!!
!!   	26-oct-2010 (jm) minor changes
!!   	05-oct-2010 (jm) created
!!-->

<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../manstyle.css" />


<title>Kinase Search</title>

<meta panel Kinase Database Search>

</head>

<body bgcolor="#ffffff">
<a href="../index.htm"><img src="../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>Kinase Search</h2>
<hr noshade="noshade" />

<ul>
<li><a href="#Background">Background and Methods</a>
  <ul>
    <li><a href="#PKClassification">Classification of Protein Kinases</a>
    <li><a href="#PKIdentification">Identification of Protein Kinases in MOE</a>
    <li><a href="#PKStructure">ATP Pocket Architecture and Catalytic
        Reaction Mechanism
        </a>
    <li><a href="#kinRegulate">Kinase Regulation</a>
    <li><a href="#kinInhibitors">Kinase Inhibitors</a>
        <ul>
            <li><a href="#kinInhClasses">Kinase Inhibitor Classes</a>
        </ul>
    <li><a href="#kinClustering">Kinase Structure Clustering in MOE</a>
  </ul>
<li><a href="#kinSearch">Searching the Kinase Database</a>
  <ul>
    <li><a href="#paramGeneral">General Search/Filter Parameters</a>
    <li><a href="#paramKinase">Kinase-specific Search Parameters</a>
    <li><a href="#paramLigand">Ligand-specific Search/Filter Parameters</a>
     <li><a href="#SearchMode">Kinase Database Search Mode</a>
     <li><a href="#SearchResults">Kinase Search Results</a>
  </ul>

<li><a href="#Update">Compiling and Updating the Kinase Database</a>
  <ul>
    <li><a href="#kinCompile">Compiling the Kinase Database</a>
    <li><a href="#kinUpdate">Updating the Kinase Database</a>
  </ul>
<li><a href="#SeeAlso">See Also</a>
<li><a href="#References">References</a>

</ul>


<!-- ====================== Background and Methods ======================== -->

<p><a name="Background"></a><h2>Background and Methods</h2>
The Protein Kinases (PKs) represent one of the largest superfamilies in
eukaryotes (~ 2% of vertebrate genes encode kinases) that are functionally
related by their catalytic domain, a region of approx. 250-300 amino acids.
PKs are ATP-dependent enzymes that facilitate the transfer of a phosphoryl
group from the &gamma;-position of ATP to
hydroxyls of serine, threonine or tyrosine of their respective substrates. Since
the phosphoryl transfer reaction represents one of the most important
posttranslational modifications in the cell it is not surprising to find PKs in
pivotal roles in regulation and control of a wide variety of critical cellular
processes including cell cycle, cell growth, cell death, metabolism and
differentiation.
<br>
Initial interest in protein kinases as pharmacological targets was triggered
by the finding that many viral oncogenes encode structurally modified cellular
PKs with constitutive enzyme activity. Consequently, deregulated kinase
activity has been implicated in the pathology of a number of important human
diseases, including cancer and disorders in the immune system
[Kolibaba&nbsp;1997].

<p>
<a name="PKClassification"></a><h3>Classification of Protein Kinases</h3>
<p>
Despite the fact that almost all kinases use the same donor (usually ATP) and
catalyze the same transfer reaction, PKs display a remarkable
diversity in structure and substrate recognition mechanism. One of the most
elaborate efforts to group PKs based on structures/sequence criteria was
presented by Grishin <i>et al.</i> [Grishin&nbsp;2005] classifying the majority
of PKs (> 98%) into 10 distinct fold groups. The most prominent kinase fold
architecture is represented by Ser/Thr-Tyr which is pharmcologically the most
important group and the sole focus of the MOE Kinase Search application.
<br>
Based on sequence homology, substrate recognition and specificity as well as
cellular locus of enzyme activity, the Ser/Thr Tyr kinases may be further
subdivided in distinct kinase groups, families and subfamilies. Commonly
accepted classification schemes distinguish "conventional" protein kinases
(ePKs) and "atypical" protein kinases [Manning&nbsp;2002].
The ePKs - the larger group - have been subclassified into 8 groups by
examining sequence similarity between catalytic domains, the presence of
accessory domains, and regulation mode  [Manning&nbsp;2002][Hanks&nbsp;1995].
Figure 1 shows a classification tree of the Ser/Thr-Tyr kinases
applied in MOE to classify kinases.
<p>
<div align=center>
<img src="kinase/grouptree.png" alt="Kinase classification tree">
<br>&nbsp;<br>
<b>Figure 1:</b> <i>Kinase Classification Tree</i>
</div>
<p>
The catalytic core of PKs can be further subdivided into 12 smaller subdomains
termed <i>Hank's Regions</i> defined as regions uninterrupted by large amino
acid insertions as well as containing a characteristic set of conserved residues
[Hanks&nbsp;1995]. Surprisingly, sequence similiarity within these domains
is only significantly high within close relatives of the kinase families and
subfamilies. Only twelve residues within the catalytic core of approx.
300 amino acids are recognized as being invariant or nearly invariant
(conserved in over 95%) and hence strongly implicated in playing an essential
role in ATP-binding and enzyme catalysis.


<!-- ======================= Kinase Identification ======================= -->
<p>
<a name="PKIdentification"></a><h3>Identification of Protein Kinases in MOE</h3>
<p>
In order to achieve reliable detection of kinase structures using sequence
alignments, a set of reference sequences for each of the above mentioned kinase
groups is employed. This reference set is based on a modified version of
alignments originating from the EMBL site supplemented with sequences from the
SUGEN sequence collections [Steinbeck&nbsp;2009] [Manning&nbsp;2002].
<br>
The kinase reference sets in MOE have been aligned globally allowing for
assignment of an unified kinase alignment index. Consequently, all structurally
relevant residues map to the same kinase residue index allowing fast
identification of relevant subdomains and comparison of kinase structures.
In addition to <i>conventional</i> kinases, various reference collections for
the <i>atypical</i> PKs have also been integrated into this indexing scheme.
The group harboring aPKs is termed <i>OPK</i> (other protein kinases). The
kinase reference alignments are stored in the text file
<tt>$MOE/lib/kin_ref.fam</tt>.
A separate reference collection exists for a special group of kinases termed
<i>Pseudo</i> kinases kept in <tt>$MOE/lib/kin_refpseudo.fam</tt>. The hallmark
feature of these enzymes is their catalytic inactivity due to
sequence changes/deletions in critical kinase motifs. Pseudo kinases have been
implicated as regulators of signaling processes. Approx. 10% of the kinase
sequences in the human kinome are believed to be pseudo kinases
[Scheeff&nbsp;2009].
Proteins classified in MOE as Pseudo kinases (<i>PSEUDO</i>) lack the global
kinase alignment index.
<br>
The kinase detection algorithm in MOE is based on the alignments with the
reference collections within the Hank's regions. Sequence identity is calculated
as percentage of identical residues within the specified sequence domains. The procedure involves the following steps:
 <ol>
  <li>Detection of distinct kinase signature motifs G-Loop, HRD and DFG (see
  below for more detail on the function of these motifs). At least two of the
  three motifs are expected to be present for positive kinase identification.
  In addition to the actual motif sequences, signature pattern in the vicinity
  are included and an estimate, if the detected motifs are in acceptable
  distances to each other. If found negative, then a check for pseudo kinases
  is executed by pairwise sequence alignments with the members of the pseudo
  kinase reference collection. If the maximum sequence identity with this set
  falls below 35% then the sequence is discarded.

  <li>Identification of the closest kinase group based on one-to-group
  alignments with the reference collection. Based on statistical analysis
  sequence identity cutoff of 45% is applied for positive kinase classification.

  <li>Identification of the reference kinase with the highest sequence identity
  (pairwise alignment with the group members) that determines kinase family and
  subfamily association.

  <li>Assignment of the global kinase index based on the alignment with the
  reference.
 </ol>

<!-- ======================= Kinase Structure ======================= -->
<p>
<a name="PKStructure"></a><h3>ATP Pocket Architecture and Catalytic Reaction Mechanism</h3>
<p>
 The solution of the PKA crystal structure in 1991 offered insights in the
 understanding of the catalysis and regulation of kinases [Knighton&nbsp;1991].
 The kinase residue references issued in this manual are in accordance with the
 residue numbering of PKA.<br>
 Protein kinases are bilobal enzymes with the active site located in the cleft
 of the two lobes. The smaller N-lobe is dominated by a five-stranded
 antiparallel &beta;-sheet (&beta;1-&beta;5) and an &alpha;-helix (C). The first
 two &beta;-strands contain a glycine-rich sequence also termed the
 <i>G-Loop</i>. The third &beta;-sheet contains the invariant Lys72 that is
 stabilized by interaction with the invariant Glu91 residue located in the
 center of helix C.<br>
 The larger C-lobe has seven &alpha;-helices and a few short &beta;-strands.
 Several residues interact with the triposphate group of ATP, the catalytic
 subdomain (HRD motif) between &beta;6 and &beta;7 and the DFG motif (
 Mg-binding loop) between &beta;8 and &beta;9 are of primary importance for the
 enzyme reaction.
<p>
<div align=center>
  <img src="kinase/kinase300.png" alt="Kinase Core Structure">
  <img src="kinase/catalysis300.png" alt="Kinase Phosphotransfer">
  <br><b>Figure 2:</b> <i>Kinase structure (A) and enzyme reaction mechanism</i><br>
</div>
<p>
In table 1, the highly conserved kinase residues and their role in ATP-binding
and catalysis are summarized:
<p>
<table border="0" cellpadding="3" width="100%">
<tr align="center">
  <td bgcolor="#cccccc" width="20%"><b>Region</b></td>
  <td bgcolor="#cccccc" width="20%"><b>Residues</b></td>
  <td bgcolor="#cccccc" width="60%"><b>Description</b></td>
</tr>
<tr>
  <td valign=top align="center"><b>G-Loop</b></td>
  <td valign=top align="center"><i>GxGxxGxxV</i></td>
  <td valign=top align="left">
    Important component for ligand binding. The invariant conserved Glycines
    allow a close approach of the nucleotide to the peptide backbone.
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>Hyd1</b></td>
  <td valign=top align="center"><i>Lys72</i></td>
  <td valign=top align="left">
    Residues within this region are in hydrophobic contact with the adenine ring
    of ATP forming a hydrophobic pocket. The invariant Lys72 residue interacts
    with &beta;-phosphoryl groups of the bound ATP and is thought to be crucial
    in mediating correct orientation of the ligand.
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>alphaC</b></td>
  <td valign=top align="center"><i>Glu91</i></td>
  <td valign=top align="left">
    The spacial position of Lys72 is secured by a salt-bridge with Glu91 in the
    center of the only helix in the C-lobe of the enzyme. This helix is thought
    to be important for enzyme regulation since the activation of kinases is
    often accompanied by changes in it's orientation.
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>Hinge</b></td>
  <td valign=top align="center"><i>&nbsp;</i></td>
  <td valign=top align="left">
    Linker region between the N and C-lobe. The adenine ring of ATP forms
    hydrogen bonds with backbone atoms of the hinge residues. The residue
    N-terminal to these H-Bond contacts is termed the <i>Gatekeeper</i> and
    controls access to the back cleft. The C-terminal residues of the hinge
    region form in concert with the G-Loop the ribose binding pocket.
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>HRD</b></td>
  <td valign=top align="center"><i>Asp166<br>Lys168<br>Asn171</i></td>
  <td valign=top align="left">
    Catalytic center with the characteristic sequence motif <i>HRDxxxxN</i> that
    facilitates phosphoryl-transfer to substrate. Lys168 forms direct contact to
    the &gamma;-phosphoryl group of ATP, neutralizes the negative charge and
    stabilizes the intermediate state. Asp166 represents the catalytic base that
    abstracts the proton from the hydroxyl-group of the substrate, Asn171
    coordinates with the second Mg<sup>2+</sup>-ion (Mg2) that chelates the
    &alpha;- and &gamma;-phosphate oxygens.
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>DFG</b></td>
  <td valign=top align="center"><i>Asp184<br>Phe185</i></td>
  <td valign=top align="left">
    This motif is located at the base of the kinase activation segment. The
    carboxyl-group of Asp184 is in complex with the chelating
    Mg<sup>2+</sup>-ion (Mg1) which coordinates &beta;- and &gamma;-phosphoryl
    oxygens forming almost an optimal octahedral coordinate scheme. The
    orientation of Phe185 residue has in many kinases been shown to contribute
    to stabilizing the active site pocket by forming hydrophobic contacts with
    residues from the &alpha;C-helix and the nearby HRD motif (hydrophobic
    spine).
  </td>
</tr>
</table>
<br>
<div align=center>
<b>Table 1:&nbsp;</b><i>Conserved kinase residues and their role in ATP catalysis</i>
</div>

<p>
<a name="kinRegulate"></a><h3>Kinase Regulation</h3>
<p>
PKs are highly flexible enzymes and the binding of substrates, cofactors,
autoinhibitory domains or interacting proteins are often accompanied with large
rotational movements within the activation segment. The activation segment
includes a stretch of 25 to 30 residues starting with the DFG motif to the
<b>APE</b> site, a crucial motif for substrate recognition and binding and is
comprised of the secondary elements <i>DFG, &beta;9, Activation Loop and P+1
loop</i> [Nolen&nbsp;2004]. Many PKs are activated upon phosphorylation of one
up to three phosphorylation events in the <i>Activation Loop</i> that facilitate
the movement of the activation segments (RMSD > 10 &Aring;) observed for
many kinases. Movement of the activation segment is in many kinases a
requirement for ATP to gain full access to the binding pocket. The
phosphorylated residues
in the Activation Loop are stabilized by Arg165, the residue preceding Asp166 in
the catalytic subdomain (RD kinases).
In addition, active site regulation of some kinases is regulated by a
characteristic conformational change in the DFG motif ("DFG flip"). In the fully
activated kinase, Asp184 interacts with the chelating Mg2 ion and Phe185 with
residues of &alpha;C and HRD-base forming the <i>DFG-in</i> conformation. In
the <i>DFG-out</i> conformation the DFG aspartate is pointed away from the
active site and the phenylalanine reaching into the ATP binding pocket making
it incompatible for ATP binding. Many inhibitors target selectively the
<i>DFG-out</i> conformation. Recent publication also show that the DFG-flip is
accompanied by an outward  displacement of the &alpha;-helix forming a
<i>&alpha;C-out</i> conformation that disrupts the characteristic ionic
interaction of Lys72 with Glu91 seen in the <i>&alpha;C-in</i> conformation
[Shan&nbsp;2009][Pargellis&nbsp;2002].

<!-- ======================= Inhibitors ====================== -->
<p>
<a name="kinInhibitors"></a><h3>Kinase Inhibitors</h3>
<p>
Since the discovery that deregulation of PK activity is associated with a
variety of diseases, particularly cancer, the development of specific kinase
inhibitors is one of the key challenges for research in pharmaceutical companies
as well as academic science. A large number of kinase inhibitors that target
distinct signaling cascades are currently used as modifiers of the disease
pathway. The strategies employed range from ATP analogues to silencing RNA
(siRNA) each bearing their own issues with respect to specificity and/or target
delivery.<br>
The majority of kinase inhibitors that have been developed to date  are ATP
analogues also known as <i>type I</i> inhibitors. These inhibitors require the
kinase to be in an active conformation (<i>DFG-in</i>) identical to that used to
bind ATP. Type-I inhibitors typically occupy the adenine region and form 1-3
hydrogen bonds with the kinase <i>hinge</i> region [Ghose&nbsp;2008]. Although
empirical screening of compound libraries revealed a number of ATP-competitive
inhibitors with a high degree of selectivity within the kinase family, many
type-I kinase inhibitors have failed in clinical trials due to intolerable
side-effects. A prominent type-I inhibitor with high potency and low specificity
is the microbial alkaloid <i>Staurosporine</i>.

<p>
Recently, a second group of kinase inhibitors has emerged that is termed <i>type
II</i> revealing a new binding mode. These inhibitors also bind to the ATP
binding site and often form similar hydrogen bond and hydrophobic interactions
in the adenine pocket of the kinase similar to type-I inhibitors. However, the
hallmark feature of type-II inhibitors is the interactions with an allosteric
site of the kinase that is only formed when the enzyme assumes <i>DFG-out</i>
conformation [Pargellis&nbsp;2002], locking (or even inducing) the kinase in an
inactive state. Since the regulation of kinases offers a much larger degree of
conformational variation, it has been proposed that type-II inhibitors may
target selectivity much more efficiently than type-I inhibitors. Prominent
representatives of the type-II inhibitor class are <i>Gleevec (Imatinib)</i> and
<i>Sorafenib</i>.
<p>
<!--<a href="#kinInhClassification">Search AA</a>-->
<a name="kinInhClasses"></a><h4>Kinase Inhibitor Classes</h4>

In order to gain insights in the details of kinase and inhibitor interactions
and thus selectivity it is necessary to obtain a broad collection of chemical
scaffolds or building blocks of available kinase inhibitors. The database
<tt>$MOE/lib/kin_liglcasses.mdb</tt> offers a ligand scaffold classification
scheme as published by Vieth <i>et al.</i> and De Moliner <i>et al.</i>
[Vieth&nbsp;2004][De Moliner&nbsp;2003]. The database harbors references
structures for each class member as well as SMART-strings that are used for
substructure search and may be extended with additional inhibitor scaffolds or
variants of existing class members. An excerpt of the kinase ligand class
database is shown in the table 2:
<p>
<table border="0" cellpadding="3" width="100%">
<tr align="center">
  <td bgcolor="#cccccc" width="25%"><b>Inhibitor Class</b></td>
  <td bgcolor="#cccccc" width="65%"><b>Structures</b></td>
</tr>
<tr>
  <td valign=top align="center"><b>adenosines</b></td>
  <td valign=top align="left">
    <img src="kinase/adenosines_1.png">
    <img src="kinase/adenosines_3.png">
    <img src="kinase/adenosines_5.png">
    <img src="kinase/adenosines_7.png">
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>staurosporines</b></td>
  <td valign=top align="left">
    <img src="kinase/staurosporines_1.png">
    <img src="kinase/staurosporines_2.png">
    <img src="kinase/staurosporines_3.png">
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>oxoindoles</b></td>
  <td valign=top align="left">
    <img src="kinase/oxoindoles_3.png">
    <img src="kinase/oxoindoles_4.png">
    <img src="kinase/oxoindoles_8.png">
    <img src="kinase/oxoindoles_9.png">
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>purines</b></td>
  <td valign=top align="left">
    <img src="kinase/purines_2.png">
    <img src="kinase/purines_4.png">
    <img src="kinase/purines_6.png">
    <img src="kinase/purines_7.png">
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>isoquinolines</b></td>
  <td valign=top align="left">
    <img src="kinase/isoquinolines_1.png">
    <img src="kinase/isoquinolines_2.png">
    <img src="kinase/isoquinolines_3.png">
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>quinazolines</b></td>
  <td valign=top align="left">
    <img src="kinase/quinazolines_1.png">
    <img src="kinase/quinazolines_2.png">
    <img src="kinase/quinazolines_3.png">
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>ureas</b></td>
  <td valign=top align="left">
    <img src="kinase/ureas_1.png">
    <img src="kinase/ureas_2.png">
    <img src="kinase/ureas_3.png">
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>imidazoles</b></td>
  <td valign=top align="left">
    <img src="kinase/imidazoles_1.png">
    <img src="kinase/imidazoles_3.png">
    <img src="kinase/imidazoles_5.png">
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>aminopyrimidines</b></td>
  <td valign=top align="left">
    <img src="kinase/aminopyrimidines_1.png">
    <img src="kinase/aminopyrimidines_2.png">
    <img src="kinase/aminopyrimidines_3.png">
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>anthraquinones</b></td>
  <td valign=top align="left">
    <img src="kinase/anthraquinone-like_1.png">
    <img src="kinase/anthraquinone-like_2.png">
    <img src="kinase/anthraquinone-like_3.png">
  </td>
</tr>
</table>
<br>
<div align=center>
<b>Table 2:&nbsp;</b><i>Kinase inhibitor classes used in MOE</i>
</div>

<!-- ======================= Inhibitors ====================== -->
<p>
<a name="kinClustering"></a><h3>Kinase Structure Clustering in MOE</h3>
<p>
The comparison of protein kinase structures is an efficient way to elucidate
subtle structural differences in relevant regions. This might help to understand
particular issues with respect to binding selectivity. MOE uses distinct residue
sets that offer various degrees of structural abstractions centered around the
ATP-binding pocket. The residue sets are described in detail as the following:

<table border="0" cellpadding="3" width="100%">
<tr align="center">
  <td bgcolor="#cccccc" width="15%"><b>Set</b></td>
  <td bgcolor="#cccccc" width="45%"><b>Description</b></td>
  <td bgcolor="#cccccc" width="40%"><b>Topology</b></td>
</tr>
<tr>
  <td valign=top align="center"><b>core</b></td>
  <td valign=top align="left">
    The residue set is based on the 12 invariant residues in the kinase core
    domain (see above) that are crucial for binding and catalysis of ATP. The
    residues include Gly56,Gly58, Lys72, Glu91,Asp166, Asn171, Asp184,Phe185.
    This residue set represents the ultimate abstraction of the kinase ATP
    binding pocket.
  </td>
  <td valign=middle align="center">
      <img src="kinase/core.png" alt="Kinase Core Residue Set">
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>pocket</b></td>
  <td valign=top align="left">
    This set of kinase residues is based on interaction statistics of an unique
    set of 645 kinase inhibitors in the ATP binding pocket. In this study, the
    hydrophobic contacts, hydrogen bonds and ionic interactions have been
    measured and analyzed with respect to location in the ATP binding pocket.
    The outlined residue set resembles the regions of most common contacts and
    include G-Loop, Hyd1, Hinge, HRD and DFG as described above. It is
    noteworthy, that the &alpha;C helix is not included in this residue set.
  </td>
  <td valign=middle align="center">
      <img src="kinase/pocket.png" alt="Kinase Pocket Residue Set">
  </td>
</tr>
<tr>
  <td valign=top align="center"><b>canonical</b></td>
  <td valign=top align="left">
    This set of kinase residues expands the <i>pocket</i> collection with the
    core residues of the <i>&alpha;C</i> helix. Sequence analysis of kinase
    structures in the database reveal that all residues with comparatively high
    frequency of occurrence are included in this set.
  </td>
  <td valign=middle align="center">
      <img src="kinase/canonical.png" alt="Kinase Canonical Set">
  </td>
</tr>
</table>
<br>
<div align=center>
<b>Table 3:&nbsp;</b>
    <i>Residue sets used in MOE for clustering kinase structures</i>
</div>
<p>
The collection of PK structures along with their respective ligands, solvent
and cofactors are stored in the kinase database located in
<tt>$MOE/lib/kinome.mdb</tt>. The details on compiling and updating the kinome
database are specified in the chapter below. All kinase structures in the
database have been pre-clustered based on the above listed residue collection at
a resolution of 1.0 &Aring; RMSD (root mean square distance).
<p>
The pre-calculated clusters allow for fast comparison of the entire database
within a relevant residue set independent of any family associations or other
constraints. Furthermore, the changes of RMSDs within these residue sets of a
kinase collection of interest might be very telling with respect to range of
movement, ligand binding and/or enzyme regulation.


<!-- ======================= Kinase Search Panel ====================== -->

<a name="kinSearch"></a><h2>Searching the Kinase Database</h2>
The Kinase Search Panel offers a multitude of features that allows the user to
extract potentially complex data sets quickly and efficiently. The flexible
setup of the application does not imply any predefined search modes for
searching the kinase database. As a result, the user may issue queries that
target only the inhibitor set of the database or tap into aspects that are
strictly relevant in a kinase structure context or any combination of the two
approaches. The hallmark feature of the Kinase Search application is that the
results of each individual search are stored and may be combined with other
search results that have been generated along the search process. This way,
data sets of interest that are based on rather complex search parameters can
be generated, stored and used for downstream applications.
<p>
The application can be accessed through the main MOE panel
using <b><nobr>MOE | Compute | Biopolymer | Kinase Search</b></nobr>

<p><div align=center>
<img src="kinase/mainpanel.png"  alt="Kinase Search Main Panel">
</div>
<p>

Upon opening the Kinase Search application the entire content of the kinase
database is shown in the result list. By default, the referred kinase database
is located in the <tt>$MOE/lib</tt> directory named <tt>kinome.mdb</tt>. This
may be changed if the user chooses to search an updated kinase database in the
<b>Configuration Options</b> panel using
<b><nobr>MOE | Window | Options</b></nobr> and selecting for <b>Protein
Defaults</b>.

<table width="100%" cellpadding="5">
  <tr>
    <td valign="top"><nobr><b>Search</b></nobr></td>
    <td valign="baseline">Conduct a kinase database search based on the
    specified parameters.
    </td>
  </tr>
    <td valign="top"><nobr><b>Load into MOE</b></nobr></td>
    <td valign="baseline">
       Load kinases from the result set in the MOE main window.
       <p><div align=center>
         <img src="kinase/load2MOE.png"  alt="Load Panel">
       </div></center>
       <p>
       <table width="100%" cellpadding="5">
        <tr>
          <td valign="top"><b>Copy</b></td>
          <td valign="baseline">Specify the result set to be loaded in MOE,
          either all kinases in the result list or only the selected or the
          marked collection.
        </tr>
        <tr>
          <td valign="top"><b>Include</b></td>
          <td valign="baseline">
          Specify, which parts of the kinase complex should be loaded in MOE.
          The kinase complex is composed of the kinase, ligand, solvent and
          add-on molecules. The latter represents a molecules that are in
          contact with the kinase but not ligand or solvent (e.g. cofactors,
          substrate, etc.).
          </td>
        </tr>
        <tr>
          <td valign="top"><b>Options</b></td>
          <td valign="baseline">
          Specify visual modes of the newly loaded structures that should be
          centered in the MOE main window (<b>Center Molecules</b>) or selected
          (<b>Select New Objects</b>). All existing chains in MOE will be
          erased, if <b>Close Current System</b> is checked.
          </td>
        </tr>
       </table>
    </td>
  </tr>
    <td valign="top"><nobr><b>Save as Database</b></nobr></td>
    <td valign="baseline">
       The entries displayed in the result table may be saved in a separate
       database for future use.
       <p><div align=center>
         <img src="kinase/save.png"  alt="Save Panel">
       </div></center>
       <p>
       <table width="100%" cellpadding="5">
        <tr>
          <td valign="top"><b>Copy</b></td>
          <td valign="baseline">Specify the result set to be saved, either all
          kinases in the result list or only the selected or the marked
          collection.
        </tr>
        <tr>
          <td valign="top"><b>Path</b></td>
          <td valign="baseline">Specify the path and file name of the kinase
          database.
        </tr>
       </table>
    </td>
  </tr>
</table>
<p>
The Kinase Search panel can be divided in three sections:
<ol>
  <li>Upper section to specify the search parameters that have a general,
  structure specific or ligand specific scope.
  <li>Middle section to specify the search mode.
  <li>Bottom section to list the search results that may be sorted, marked or
  browsed.
</ol>

<p>
<a name="paramGeneral"></a><h3>General Search/Filter Parameters</h3>
<p>

The upper section of the Kinase Search panel allows searching the database
specifying one or more search terms. In addition to these search terms,
parameters that contain information about the kinase regulatory conformation,
family association and structural parameters could be used as additional filters
to reduce the search space.
<p><div align=center>
 <img src="kinase/searchtxt.png"  alt="Text Search">
</div></center>
<table width="100%" cellpadding="5">
 <tr>
   <td valign="top"><b>Text Field</b></td>
   <td valign="baseline">
        Enter the search term(s) in the text field, separated by the space
        character, in case multiple search terms were submitted. By default,
        wildcard ("*") characters on both sides of the search phrase. Use double
        quotes to change this behavior. For an exact match, enclose the phrase
        in double quotes with a preceding and trailing space.
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Date</b></td>
   <td valign="baseline">Submission date of the structure in the RCSB
   repository. The expected input format is <i>YYYY-MM-DD</i>. Only structures
   with submission date equal or newer to the specified value will be included
   in the search results.</td>
 </tr>
 <tr>
   <td valign="top"><b>Resolution</b></td>
   <td valign="baseline">Resolution (in &Aring;) of the kinase crystal
   structure. Only structures with a resolution equal to or lower than the
   specified value will be considered in the search.</td>
 </tr>
  <tr>
   <td valign="top"><b>Species</b></td>
   <td valign="baseline">Specify the species the resulting kinases should
   originate from as given in the PDB record. In addition to common
   specifications, scientific notations are also presented.
    <p>
    <div align=center> <img src="kinase/species.png"  alt="Species Panel"></div>
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Family</b></td>
   <td valign="baseline">
     Specify a collection of kinase families that should be included in the
     search.
     <p>
     <div align=center> <img src="kinase/selectfam.png"
        alt="Family Selection Panel">
     </div>
   </td>
 </tr>
 </tr>
 <tr>
   <td valign="top"><b>DFG</b></td>
   <td valign="baseline">
       Specify the DFG conformation state (<i>DFG-out</i> or <i>DFG-in</i>)
       of the kinases to be searched. By default, the conformation state in the
       DFG motif is not considered (<i>Any</i>).
   </td>
 </tr>
 </tr>
  <tr>
   <td valign="top"><b>alphaC</b></td>
   <td valign="baseline">
       Specify the interaction state (<i>alphaC-out</i> or <i>alphaC-in</i>) of
       the highly conserved Lys72 with Glu91 in the center of &alpha;C-helix. In
       the <i>in</i> conformation, Glu91 forms an ionic interaction with Lys72.
       By default, the conformation state of &alpha;C helix is not considered
       (<i>Any</i>).
   </td>
 </tr>
  </tr>
  <tr>
   <td valign="top"><b>RD</b></td>
   <td valign="baseline">Specify, if the search space should be filtered for RD
   kinases only.
   </td>
 </tr>
</table>
<p>
<a name="paramKinase"></a><h3>Kinase-specific Search Parameters</h3>
<p>
The search for kinase structures can either be specified based on sequence
similarity/identity to query sequence as well as structural superposition of an
input kinase structure with the kinase structures of the database. If both are
issued, then only the results that satisfy the specified sequence AND structural
parameters are kept in the result set.
<p><div align=center>
 <img src="kinase/kinsearch.png"  alt="Kinase Search Parameters">
</div></center>
<p>
<table width="100%" cellpadding="5">
 <tr>
   <td valign="top"><b>Sequence</b></td>
   <td valign="baseline">
        Specify the query sequence by either typing, pasting from the clipboard
        using <b>Ctrl+V</b> keys or using one of the kinases already loaded in
        MOE. In the latter case, press <b>Load Sequence</b> to obtain a list of
        kinases currently present in MOE.<p>
        The sequence search is conducted by pairwise alignments with the kinase
        sequences of the database. It is worth noting that the sequence search
        is restricted to the core domain of the kinase. The N-terminus of the
        target sequences is truncated eight residues upstream the G-Loop motif.
        The search results may be calculated as percentage of sequence
        <b>Identity</b> or <b>Similarity</b> as specified in the drop-down list.
        The algorithm of Sonnhammer <i>et al.</i> [Sonnhammer&nbsp;2005] is
        employed for calculating sequence similarity.
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Structure</b></td>
   <td valign="baseline">
        Specify the kinase input structure from the drop-down list.
        The structure search is based on superposition of backbone atoms
        (N, CA, C and O) within a defined <b>Residue Set</b>. Valid sets include
        <i>Core, Pocket</i> or <i>Canonical</i> as described above. Kinases with
        RMSD values above a specified cutoff value are discarded from the
        results. The RMSD cutoff (in &Aring;) is given in the <b>RMSD</b> field.
        It is worth noting that the input structure is stored with the search
        result set. This way, the search parameters may be issued again even if
        the input structure in the MOE main window has been deleted or modified.
   </td>
 </tr>
</table>
<p>
<a name="paramLigand"></a><h3>Ligand-specific Search/Filter Parameters</h3>
<p>
A variety of search parameters are possible when searching for particular kinase
inhibitors. The range of parameters include one or many descriptor as used in
MOE as well as SMARTS and fingerprint similarity. In addition, filter
criteria include a specified set of kinase ligand classes as listed above and
the binding mode of the ligand to the ATP-binding pocket of the kinase. As for
the kinase structure search, all of the specified parameters have to be
fulfilled in order to be displayed in the result list.
<p><div align=center>
 <img src="kinase/ligspecific.png"  alt="Ligand-specific Search Parameters">
</div></center>
<p>
<table width="100%" cellpadding="5">
 <tr>
   <td valign="top"><b>Ligand Filter</b></td>
   <td valign="baseline">
        Issue search one or more parameters based on descriptors as used in MOE.
        Double-quoted input is expected, if SMART strings are used as search
        criteria. A common set of descriptor parameters are given in the
        drop-down list.
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Similarity</b></td>
   <td valign="baseline">
        Search the kinase database for inhibitors based on fingerprint
        similarity with a given input ligand structure that are listed the
        drop-down list. The input may also be defined as a collection of
        selected atoms or residues. In this case, the selection should include
        at least eight heavy atoms which has been calculated in a kinase ligand
        study as a valueable lower cutoff. For the similarity calculation, the
        <i>MACCS</i> keys are applied at a tanimoto-distance cutoff as specified
        in the <b>Tanimoto</b> text field. Only structures with a tanimoto
        distance above the specified value are retained as results.
        The input structure is conserved in the result set
        and may be used/combined in subsequent searches even if the original
        structure has been removed in MOE.
   </td>
  </tr>
  <tr>
   <td valign="top"><b>Ligand Class</b></td>
   <td valign="baseline">
        Search the kinase database for inhibitors based on membership to one or
        multiple kinase ligand classes as outlined above. The class(es) are
        selected from the selection panel. By default, kinase ligand classes are
        not considered.
        <p><div align=center>
         <img src="kinase/ligclass.png"  alt="Ligand Class Panel">
        </div></center>
   </td>
  </tr>
  <tr>
   <td valign="top"><b>Binding</b></td>
   <td valign="baseline">
        In addition to the search criteria above, ligands may be searched based
        on their binding profile to the kinase ATP-binding cleft. The regions
        within the pocket include the G-Loop, Hyd1, alphaC, Hinge, HRD and DFG
        as outlined above and abbreviated by the letters <i>G, Y, A, H, R,
        D</i>. This allows a search for inhibitors with mandatory interactions
        in the specified region. The precise combination of these pocket regions
        may be assigned in the panel as shown below. By default, the pocket
        interaction mode is not included in a database search.
        <p><div align=center>
         <img src="kinase/binding.png"  alt="Ligand Binding Panel">
        </div></center>
   </td>
  </tr>
</table>

<p>
  <a name="SearchMode"></a><h3>Kinase Database Search Mode</h3>
<p>
A particularly important feature of the Kinase Search application is the storage
of each individual search result set along with all the associated search
parameters. Two search result sets (A,B) may be combined as unions (A<b> And</b>
B), complements (A <b>Or</b> B) or exclusions (A <b>Not</b> B) as specified in
the drop-down list (And, Or, Not). The default set in A is the entire content of
the kinase database but could also be one of the search result sets. These may
then serve as basis for newly specified search parameters.
This way, not only already existing sets can be combined but also existing sets
with the search results of current parameters.
<p>Press the <b>Reset Search</b> button to erase all search fields in the Kinase
Search panel and reset them to their respective default values.

<p><div align=center>
 <img src="kinase/mode.png"  alt="Kinase Search Mode">
</div></center>

<p>
  <a name="SearchResults"></a><h3>Kinase Search Results</h3>
<p>
The results of the kinase database searches are displayed in the result list.
<p><div align=center>
 <img src="kinase/resultlist.png"  alt="Kinase Search Results">
</div></center>
<table width="100%" cellpadding="5">
 <tr>
   <td valign="top"><b>List</b></td>
   <td valign="baseline">
    Each individual database search is assigned a consecutive ID with the number
    of matches shown in brackets. Since the application starts with the
    entire content of the kinase database, the first entry of the search list is
    the input database.
   </td>
 </tr>
 <tr>
   <td valign="top"><b>Sort By</b></td>
    <td valign="baseline">
    Sort the displayed search results by each of the columns as shown in the
    drop-down list. The arrow indicates the sort order of the data set. Press
    the <b>Fields</b> button to control the columns that are visible in the
    result table.
    </td>
 </tr>
 <tr>
   <td valign="top"><b>Selected Entries</b></td>
   <td valign="baseline">
    Create a persistent subset of interest with the current entry selection by
    pressing the <b>Mark</b> button. These entries are then indicated in the
    <i>Mark</i>-column with the "+" character and available if the result set or
    the selection status has changed.  Press <b>Unmark</b> to reverse the
    specifier.
   </td>
 </tr>
  <tr>
   <td valign="top"><b>View</b></td>
   <td valign="baseline">
    Specify the results used as basis for calculating the statics and database
    browsing. Choose between all results shown in the table (<b>Listed</b>),
    only the selected entries (<b>Selected</b>) or only the marked entries
    (<b>Marked</b>).
   </td>
 </tr>
  <tr>
   <td valign="top"><b>Statistics</b></td>
   <td valign="baseline">
    Based on the above specified result statistics for each of the result
    columns are calculated and displayed as pie charts. Press <b>Export</b> to
    store the plot in commonly used graphics formats.
    <p><div align=center>
       <img src="kinase/stats.png"  alt="Pie Chart Panel">
    </div></center>
   </td>
 </tr>
  <tr>
   <td valign="top"><b>Browser</b></td>
   <td valign="baseline">
    The search results may also be browsed in 3D applying the kinase-specific
    MOE database browser. In the kinase browsing mode, the structurally relevant
    regions of the ATP-binding pocket are highlighted by distinct ribbon colors.
    Kinase ligand and solvent molecules interacting with the complex are shown
    as well. In addition to the 3D viewing mode, a 2D depiction of the ligand is
    shown in the browser panel. Core data specific to the currently displayed
    kinase is displayed including the chain name (<b>Code</b>) of the kinase,
    the resolution of the crystal structure (<b>Resolution</b>), the submission
    date (<b>Date</b>) and the association of the kinase in <b>Group, Family</b>
    and <b>Subfamily</b>. All kinase structures in the kinase database are shown
    in a standardized viewing mode based on the superposition in the core region
    to a kinase reference structure (<tt>$MOE/lib/kin_ref.moe</tt>).
    <p>
    Several widgets in the kinase browser panel control the 3D viewing mode of
    the kinase complex that can be modified or adjusted according to the
    specification of the user.
    <p><div align=center>
       <img src="kinase/dbrowser.png"  alt="Kinase Database Browser Panel">
       <img src="kinase/browserMOE.png"  alt="3D Depiction Kinase Database
       Browser">
    </div></center>
    </p>
    <table width="100%" cellpadding="5">
     <tr>
      <td valign="top"><b>Complex</b></td>
      <td valign="baseline">
        If on, then the ligand (if available) is shown in addition to the kinase
        structure.
      </td>
     </tr>
     <tr>
      <td valign="top"><b>Ribbons</b></td>
      <td valign="baseline">
        By default, the ribbon mode of the kinase structure is restricted to the
        ATP-binding pocket regions G-Loop, Hyd1, alphaC, Hinge, HRD and DFG. If
        on, then the ribbon mode for the entire kinase structure is activated.
      </td>
     </tr>
     <tr>
      <td valign="top"><b>Addon</b></td>
      <td valign="baseline">
        Molecules that are in contact with the kinase and not classified as
        ligand (cofactors, substrate, etc.) are stored in the database in the
        <i>addon</i> molecule field. If on. then these molecules are shown as
        well.
      </td>
     </tr>
     <tr>
      <td valign="top"><b>Core</b></td>
      <td valign="baseline">
        Show the highly conserved residues that are relevant for ATP-binding and
        catalysis. These residues might serve as reference points within the
        ATP-binding pocket.
      </td>
     </tr>
     <tr>
      <td valign="top"><b>Labels</b></td>
      <td valign="baseline">
        If on, then the names of residues that are in contact with the ligand or
        in the <b>Core</b> set are shown.
      </td>
     </tr>
     <tr>
      <td valign="top"><b>Settings</b></td>
      <td valign="baseline">
        Specify the ribbon colors for each of the binding region within the ATP
        pocket. Press <b>Assign Default Colors</b> to restore the default
        colors.
        In addition to the ribbon colors, the color settings of residues that
        are in contact with the ligand (<b>Pocket</b>), residues of the
        <b>Core</b> set, the ligand,  molecules that interact with the kinase
        (and are not ligand or solvent molecules) &ndash; <b>Addon</b> &ndash;
        can be modified as well.
        <p><div align=center>
           <img src="kinase/browsersettings.png"
           alt="Kinase Database Browser Panel">
        </div></center>
      </td>
     </tr>
    </table>
   </td>
 </tr>
</table>


<!-- ======================= Datbase Update ====================== -->

<a name="Update"></a><h2>Compiling and Updating the Kinase Database</h2>
The kinase database is the driving engine in the MOE Kinase Search application,
all relevant information used in this application is stored here. The
following chapter outlines the details underlying the compilation of the kinase
database and instructions on how to update the kinase knowledge-base
continuously.

<a name="kinCompile"></a><h3>Compiling the Kinase Database</h3>
<p>
Once the kinase reference sets are established, the
MOE kinase database is then compiled by screening the entire protein
collection of the Protein Data Bank (PDB) for kinase structures as described
above.
<p>Classifying a protein chain as a kinase requires sequence identity within the
kinase core of at least 45% with one of the classes in the reference file (see
above for more detail), otherwise a check is conducted to verify, if the
sequence is sufficiently close to one of the known pseudo kinases in the pseudo
reference file. If the sequence identity is higher than 35%, the database is
labeled as <i>PSEUDO</i>. Sequences that are positive with respect to kinase
pattern detection (G-Loop, DFG and HRD) but lack sufficient homology with the
reference sequences are classified as <i>UNK</i> (unknown with respect to kinase
group, family or subfamily).
Since the reference collection is biased towards human kinase detection, this
might be case for distant species like bacteria, parasites, nematodes etc.


<p> In addition to the structure coordinates, a variety of complementary
information with respect to kinase/inhibitor complex is stored in an database
entry in the following data fields:
<p><center>
<table border="0" cellpadding="3" width="100%">
<tr>
  <td bgcolor="#cccccc"><b>Field</b></td>
  <td bgcolor="#cccccc"><b>Description</b></td>
</tr>
<tr>
  <td valign=top><b><tt>kinase</tt></b></td>
  <td>A molecule field storing the structure data of the kinase chain.</td>
</tr>
<tr>
  <td valign=top><b><tt>ligand</tt></b></td>
  <td>A molecule field storing the structure data of the ligand residue(s).
        The ligand is expected to be in contact with at least one of the five
        pocket regions (G-Loop, Hyd1, Hinge, HRD or DFG).
  </td>
</tr>
<tr>
  <td valign=top><b><tt>solvent</tt></b></td>
  <td>A molecule field storing solvent molecules that are in contact with
        the kinase.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>addon</tt></b></td>
  <td>A molecule field storing molecules that are in contact with the kinase
        (other than solvent and ligand). This might be cofactors, substrate or
        alternate ligand conformations.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>code</tt></b></td>
  <td>A character field specifying the PDB code of the structure.</td>
<tr>
<tr>
  <td valign=top><b><tt>resolution</tt></b></td>
  <td>The X-ray structure resolution as specified in the PDB header.</td>
</tr>
<tr>
  <td valign=top><b><tt>compnd</tt></b></td>
  <td>A character field with the description as specified in the PDB compnd
        specifier.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>title</tt></b></td>
  <td>A character field with the description as specified in the PDB title
        specifier.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>date</tt></b></td>
  <td>A character field with the submission date of the structure into the
        PDB.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>organism_common</tt></b></td>
  <td>A character field specifying the species of the kinase.</td>
</tr>
<tr>
  <td valign=top><b><tt>organism_scientific</tt></b></td>
  <td>A character field specifying the scientific notation of the species.</td>
</tr>
<tr>
  <td valign=top><b><tt>group</tt></b></td>
  <td>A character field specifying the kinase group.</td>
</tr>
<tr>
  <td valign=top><b><tt>family</tt></b></td>
  <td>A character field specifying the kinase family.</td>
</tr>
<tr>
  <td valign=top><b><tt>subfamily</tt></b></td>
  <td>A character field specifying the kinase subfamily.</td>
</tr>
<tr>
  <td valign=top><b><tt>famidx</tt></b></td>
  <td>An integer field specifying a reference identifier. The number represents
  a byte-packed index to the kinase with highest sequence identity in the
  reference file.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>kinUID</tt></b></td>
  <td>An integer field denoting the global kinase residue index for each kinase
        residue. The numbers represent the alignment positions of the reference
        collection.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>segments</tt></b></td>
  <td>An integer field that represents a segmentation vector. Splitting the
  kinase residues with this vector denote the Hank's regions of the kinases.
  Every alternating segment represents one of the Hank's region.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>sequence</tt></b></td>
  <td>A character field with the sequence (1-letter representation) of the
  kinase in the catalytic core. The N-terminal end is set to start eight
  residues downstream the G-Loop.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>dfg</tt></b></td>
  <td>An integer field to specify the DFG conformation state. Kinases
  in DFG-in conformation (active form) are set to 1 and to -1 if DFG-out
  conformation is assumed. The value 0 is set,  if the criteria for either
  DFG-in or DFG-out are not met.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>alphaC</tt></b></td>
  <td>
    An integer field defining the interaction of Lys72 and Glu91 residues. It
    is commonly accepted that the interaction of these highly conserved residues
    is critical for assuming appropriate pocket architecture. If
    an ionic interaction between these  two residues is formed within 5.5
    &Aring; the value is
    set to 1, otherwise the value is set to -1. In case the kinase does not
    harbor any of these highly conserved residues the value is 0.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>RD</tt></b></td>
  <td>  An integer field that specifies, if the residue preceding the
  catalytically critical Asp166 is an arginine. If so, the value is set to 1,
  otherwise -1. For kinases lacking the DFG motif the value is set to 0.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>ligclass</tt></b></td>
  <td>
   A character field that presents a comma-separated list of classes as
   specified in the ligand class database the ligand has been positively
   identified.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>bind_pocket</tt></b></td>
  <td>
   An integer field that masks binding of the ligand to the ATP-binding pocket
   regions G-Loop, Hyd1, alphaC, Hinge, HRD and DFG.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>FP:BIT_MACCS</tt></b></td>
  <td>
     An integer field storing the MACCS key fingerprint data of the ligand.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>cid_core</tt></b></td>
  <td>An integer field denoting the membership to predefined kinase clusters.
  The RMSD of the backbone atoms N,Ca,C and O of the <i>core</i> residues is
  measured with each representative of the kinase cluster collection.
  The cluster ID of the representative with the lowest RMSD is assigned if the
  value is below 1.0 &Aring;, otherwise a new cluster (and id) is formed.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>cid_pocket</tt></b></td>
  <td>An integer field denoting the membership to predefined kinase clusters.
  The RMSD of the backbone atoms N,Ca,C and O of the <i>pocket</i> residue
  collection is measured with each representative of the kinase cluster
  collection. The cluster ID of the representative with the lowest RMSD is
  assigned if the value is below 1.0 &Aring; otherwise a new cluster is formed.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>cid_canonical</tt></b></td>
  <td>An integer field denoting the membership to predefined kinase clusters.
  The RMSD of the backbone atoms N,Ca,C and O of the <i>canonical</i> residue
  collection is measured with each representative of the kinase cluster
  collection. The cluster ID of the representative with the lowest RMSD is
  assigned if the value is below 1.0 &Aring; otherwise a new cluster is formed.
  </td>
</tr>
<tr>
  <td valign=top><b><tt>comment</tt></b></td>
  <td>
  A character field to store additional information. Currently, a note is stored
  in this field, if more than one candidate for ligand has been detected.
  </td>
</tr>
<tr>
</table></center>

<a name="kinUpdate"></a><h3>Updating the Kinase Database</h3>
<p>In order to extract a maximum of detail it is important to implement
a continuous and fully automated update system in order to keep the knowledge
base current. A kinase database update can be performed via an user interface
or via a MOE/batch script. The system is based on an incremental updating scheme
that keeps a record of update date in the newly compiled or updated kinase
database. The base source for updating is the Protein Data Bank (PDB), but
in-house kinase structures may be integrated as well.

<p>The GUI version of the kinase database update can be launched at
the <a href="../moe/hellomoe.htm#GUI_Basics">CLI</a> by typing
<pre>
    svl> run 'kinupdate.svl'
</pre>
or through the main MOE panel  by  <b><nobr>MOE | File | Open</b></nobr>.
Select <span class="nobr"><tt>$MOE/lib/svl/run/kinupdate.svl</tt></span>
directory and press
<b>Run SVL File</b> in the dialog panel.

<div align=center style="padding:15px;">
  <img src="kinase/kinupdate.png"  alt="Kinase Database Update Panel">
</div>

The kinase database update procedure is based on a differential system, using
the current kinase database as the starting point. Once the
kinase update is successfully completed, the date of the update is stored in
the database.

<p><center>
<table width="100%" cellpadding="3">
    <tr>
      <td valign="baseline"><b>Input Database</b></td>
      <td valign="baseline">
      Specifies the name of the input kinase database, usually the most recent
      database. By default, the kinase database in the $MOE/lib directory is
      specified but should be modified to the most recent version. <p>
      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>Output Database</b></td>
      <td valign="baseline">
      Specifies the name of the new kinase database after the update
      procedure is completed. It is possible to update the input database and
      hence specify the same name as the input database. The exception is the
      kinase database <tt>$MOE/lib/kinome.mdb'</tt>.
      <p>
      </td>
    </tr>
    <tr>
      <td valign="baseline"><b><nobr>Overwrite Existing Entries</nobr></b></td>
      <td valign="baseline">
      By default, existing kinase entries in the input database are skipped
      during an update. If on, then this behavior is changed to
      allow rewrite of already existing entries. This setting might be useful,
      if corrections to a structure file have been made that require updating of
      an existing record.
      <p>
      </td>
    </tr>
    </tr>
    <tr>
      <td valign="baseline"><b>Structure Directory</b></td>
      <td valign="baseline">
      Specifies the top level directory where the protein structures are
      located. All subdirectories will be searched recursively.
      <p>
      </td>
    </tr>
    <tr>
      <td valign="baseline"><b>File Pattern</b></td>
      <td valign="baseline">

      Specify a file pattern with which the structure files can be identified.
      For structure files from the PDB, the files are searched by the extension
      <tt>*.ent.gz</tt> (compressed format) or <tt>*.ent</tt>. <p>
      </td>
    </tr>

</table>
</center>

<p> For conducting the kinase database update it is not necessary to
employ a computing cluster. A full compilation of the kinase database as seen
in <tt>$MOE/lib/kinome.mdb</tt> scanning the entire PDB (approx. 60,000
structure files up until September 2010) was finished in approximately 12 hours
on a Macintosh Mac Pro, 2 Dual-Core Intel Xeon Processors at 2.66 GHz. An
update performed every 14 days can be performed in less than 30 minutes on the
same computer.

<p> A convenient method to update the kinase database is to launch the
application using MOE/batch. The example below shows a script using
<b>bash</b> in a Unix-type operating system:

<pre>
#!/bin/sh -x

MOE=/path_to_moe                        #MOE directory
inputmdb=name_of_input_mdb              #name of input kinase database
outputmdb=$inputmdb                     #name of output kinase database
classmdb=$MOE/lib/kin_ligclasses.mdb    #name of ligand class database
pdbdir=/your_top_level_structure_dir    #directory of the PDB protein files
pattern='*.pdb'                         #file pattern of protein structure files

$MOE/bin/moebatch -load "kinutils.svl" \
    -exec "run ['kinupdate.svl', ['$inputmdb', '$outputmdb', '$pdbdir', \
    [overwrite:0, classmdb:'$classmdb', filepattern:'$pattern']]]" \
    -licwait -exit
</pre>

<!-- =============================== FOOTER =============================== -->

<a name="SeeAlso"></a><h2>See Also</h2>

<p>
<a href="prosearch.htm">Homology Searching</a><br />
<a href="promodel.htm">Building Homology Models</a><br />
<a href="../apps/progeom.htm">Protein Geometry</a><br />
<a href="align.htm">Sequence and Structure Alignment</a>

<!-- ----------------------  References ----------------------------------- -->
<a name="References"></a><h2>References</h2>

<table cellpadding="5" width="100%">
<tr>
  <td valign="top">[De Moliner&nbsp;2003]</td>
  <td valign="top">De Moliner E, Moro S, Sarno S, Zagotto G, Zanotti G, Pinna
    LA, Battistutta R;
    Inhibition of Protein Kinase CK2 by Anthraquinone-related Compounds
    <span class="nobr"><i>J Bio Chem 278:3</i></span>
    (<b>2003</b>) 1831&ndash;1836
  </td>
</tr>
 <tr>
  <td valign="top">[Ghose&nbsp;2008]</td>
  <td valign="top">Ghose AK, Herbertz T, Pippin DA, Salvino JM, Mallamo JP;
    Knowledge Bases Prediction of Ligand Binding Modes and Rational Inhibitor
    Design for Kinase Drug Discovery
    <span class="nobr"><i>J Med Chem 51:17</i></span>
    (<b>2008</b>) 5149&ndash;5171
  </td>
 </tr>
 <tr>
  <td valign="top">[Grishin&nbsp;2005]</td>
  <td valign="top">Cheek S, Ginalski K, Zhang H, Grishin NV;
    A comprehensive update of the sequence and structure classification of
    kinases
    <span class="nobr"><i>BMC Structural Biology 5:6</i></span>
    (<b>2005</b>)
  </td>
</tr>
<tr>
  <td valign="top">[Hanks&nbsp;1995]</td>
  <td valign="top">Hanks SK, Hunter T;
    The eukaryotic protein kinase superfamily: kinase (catalytic) domain
    structure and classification
    <span class="nobr"><i>FASEB J 9:8</i></span>
    (<b>1995</b>) 576&ndash;596
  </td>
</tr>
<tr>
  <td valign="top">[Hanks&nbsp;2003]</td>
  <td valign="top">Steven K. Hanks;
     Genomic analysis of the eukaryotic protein kinase superfamily:
     a perspective
      <span class="nobr"><i>Genome Biology</i></span> Vol 4, Issue 5
      (<b>2003</b>) 111.1&ndash;111.7.
  </td>
</tr>
<tr>
  <td valign="top">[Knighton&nbsp;1991]</td>
  <td valign="top">Knighton DR, Zheng JH, Ten Eyck LF, Ashford VA, Xuong NH,
    Taylor SS, Sowadski JM;
    Crystal structure of the catalytic subunit of cyclic adenosine
    monophosphate-dependent protein kinase
    <span class="nobr"><i>Science 253</i></span>
    (<b>1991</b>) 407&ndash;414
  </td>
</tr>
<tr>
  <td valign="top">[Kolibaba&nbsp;1997]</td>
  <td valign="top">Kolibaba KS and Druker BJ;
    Protein tyrosine kinases and cancer
    <span class="nobr"><i>Biochim Biophys Acta 13333:3</i></span>
    (<b>1997</b>) F217&ndash;248
  </td>
</tr>
<tr>
  <td valign="top">[Manning&nbsp;2002]</td>
  <td valign="top">Manning G, Whyte DB, Martinez R, Hunter T, Sudarsanam S;
    The Protein Kinase Complement of the Human Genome
    <span class="nobr"><i>Science 298</i></span>
    (<b>2002</b>) 1912&ndash;1934
  </td>
</tr>
<tr>
  <td valign="top">[Nolen&nbsp;2004]</td>
  <td valign="top">Nolen B, Taylor S, Ghosh G;
    Regulation of Protein Kinases: Controlling Activation through Activation
    Segment Conformation
    <span class="nobr"><i>Molecular Cell 15</i></span>
    (<b>2004</b>) 661&ndash;675
  </td>
</tr>
<tr>
  <td valign="top">[Pargellis&nbsp;2002]</td>
  <td valign="top">Pargellis C, Tong L, Churchill L, Cirillo PF et al.;
    Inhibition of p38 MAP kinase by utilizing a novel allosteric binding site
    <span class="nobr"><i>Nature Structural Biology 9:4</i></span>
    (<b>2002</b>) 268&ndash;272
  </td>
</tr>
<tr>
  <td valign="top">[Scheeff&nbsp;2009]</td>
  <td valign="top">Scheeff ED, Eswaran J, Bunkoczi G, Knapp S, Manning, G;
    Structure of the Pseudokinase VRK3 Reveals a Degraded Catalytic Site, a
    Highly Conserved Kinase Fold, and a Putative Regulatory Binding Site
    <span class="nobr"><i>Structure 17</i></span>
    (<b>2009</b>) 128&ndash;138
  </td>
</tr>

<tr>
  <td valign="top">[Shan&nbsp;2009]</td>
  <td valign="top">Shan Y, Seeliger MA, Eastwook MP, Xu H, Jensen M, Dror RO,
    Kuriyan J, Shaw DE;
    A conserved protonation-dependent switch controls drug binding in the
    Abl-kinase
    <span class="nobr"><i>PNAS 106:1</i></span>
    (<b>2009</b>) 139&ndash;144
  </td>
</tr>
<tr>
  <td valign="top">[Sonnhammer&nbsp;2005]</td>
  <td valign="top">Sonnhammer E., Hollich V.; Scoredist: A Simple and Robust
    Protein Sequence Distance Estimator;
    <i>BMC Bioinformatics 6</i> (<b>2005</b>) 108&ndash;116.
  </td>
</tr>
<tr>
  <td valign="top">[Steinbeck&nbsp;2009]</td>
  <td valign="top">Steinbeck C, Al-Lazikani B, Hermjakob J, Overington J,
    Thornton J;
    New open drug activity data at EBI
    <span class="nobr"><i>Chemistry Central Journal 3(Suppl I):O3</i></span>
    (<b>2009</b>)
    <br>Kinase SARfari Website:
    <a href="http://www.sarfari.org/chembl/sarfari/kinasesarfari/">
    http://www.sarfari.org/chembl/sarfari/kinasesarfari/</a>
  </td>
</tr>
<tr>
  <td valign="top">[Vieth&nbsp;2003]</td>
  <td valign="top">Vieth M, Higgs RE, Robertson DH, Shapiro M, Gregg EA,
    Hemmerle H;
    Kinomics - structural biology and chemogenomics of kinase inhibitors and
    targets
    <span class="nobr"><i>Biochem et Biopys Acta 1697</i></span>
    (<b>2003</b>) 243&ndash;257
  </td>
</tr>
</table>

<p><a href="../index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
