<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<!--
!!	pdb_mdb.htm	Protein Database chapter
!!
!!	27-sep-2010 (hf)    PSILO Download panel
!!	27-sep-2010 (kk)    TER option for PDB; updated snaps
!!	23-oct-2009 (hf)    Updated for release 2009.10
!!	23-oct-2009 (hf)    mmCif panel strict chains option
!!	12-jun-2009 (hf)    mmCif panel
!!	16-oct-2008 (kk)    minor edit
!!	06-oct-2008 (hf)    minor corrections
!!	02-oct-2008 (kk)    updated ReadPDB; added WritePDB
!!	01-oct-2008 (hf)    update to new method, 2008 stats
!!	26-sep-2007 (lc)    update to sync with the inclusion of NMR chains
!!	08-sep-2007 (lc)    Updated for release 2007
!!	08-apr-2003 (kk)    Updated for release 2003.02
!!	18-mar-2002 (m2)    Updated for release 2003.02
!!	09-may-2002 (kk)    Updated for release 2002.03
!!	12-jan-2001 (al)    Updated for release 2001.01
-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../manstyle.css" />


<meta panel Protein Database>
<meta panel Protein Database: Copy to MOE>
<meta panel Load PDB File>
<meta panel Load mmCIF File>
<meta panel PSILO/RCSB Browse>
<meta panel PSILO Download>
<meta panel RCSB Download>


<title>Protein Sequence and Structure Data</title>
</head>
<body bgcolor="#ffffff">
<a href="../index.htm"><img src="../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>Protein Sequence and Structure Data</h2>
<hr noshade="noshade" />

<ul>
<li><a href="#MOEPDB">MOE Protein Database</a></li>
<li><a href="#ProteinDatabasePanel">The Protein Database Panel</a>
  <ul>
    <li><a href="#SearchingtheDatabase">Searching the Database</a></li>
    <li><a href="#CopyingfromtheDatabase">Copying Proteins from the Database</a></li>
  </ul></li>
<li><a href="#OpeningPDBFiles">Reading and Writing PDB Files</a></li>
<li><a href="#OpeningmmCIFFiles">Reading mmCIF Files</a></li>
<li><a href="#PSILODownload">Downloading PDB Files from PSILO or
    the RCSB</a>
</li>
<li><a href="#SVLCommands">SVL Commands</a></li>
<li><a href="#References">References</a></li>
</ul>

</head>

<a name="MOEPDB"><h2>MOE Protein Database</h2>

<p>Three databases of proteins are provided as part of the MOE distribution:

<ul>

<li><i>Non-redundant Chain Database</i>.

MOE provides a database of non-redundant protein chains from the Protein Data
Bank [Berman 2000] in the file <tt>$MOE/lib/pdb.mdb</tt>.

Entries consist of single protein chains (not full protein molecules).
Solvent, ions and ligands are not included.
In MOE&nbsp;2010, the
database contains 26,000 non-redundant chains obtained from the over
68,000 PDB entries containing protein molecule models in September&nbsp;2010.

<p>This database is used by MOE's homology modeling.
The database contains over 20,000 chains of X-ray models from crystallographic
data with a resolution of 2.8&nbsp;&Aring; or better.
Additionally, it also includes over 4,100 chains of
NMR models which are not significantly homologous to the
above group of high-resolution chains.
Chains that have fewer than 40 residues
with complete backbone coordinates were excluded, as were X-ray models
with greater than 3.5&Aring; resolution.
No two chains in the database have mutual sequence identities 
larger than 90% unless they have substantially different structures 
when superposed. When selecting representatives
from groups of similar chains, the following qualities were considered:

<p><ul>
<li>Number of residues with missing coordinates</li>
<li>Longest sequence of residues with missing coordinates</li>
<li>Number of non-standard amino acids</li>
<li>R Free</li>
<li>Resolution </li>
</ul>


<p>
<li><i>Full Protein Database</i>.

An mdb file containing the full release of the Protein Data Bank&trade; 
is part of the MOE distribution.  Entries have all been processed to have 
correct ligand bonding and information, standard atom names and 
correct protein sequence data,
including residues with no atomic data.
The database is located on the distribution DVD under the name
<tt>pdb_<i>yyyy_mm_dd</i>.mdb</tt>, where <tt><i>yyyy</i></tt> indicates
the year and <tt><i>mm_dd</i></tt> indicates the month and day of the
last update.

<p>
<li><i>Structural Family Database</i>.

The database of structural families is located the file
<tt>$MOE/lib/pdb.fam</tt>.   It has been obtained by clustering
the full release of the Protein Data Bank&trade; using
CCG's custom exhaustive clustering algorithms.
The file is used by protein sequence searches.

</ul>

<p>The default protein database file used by MOE is
<tt>$MOE/lib/pdb.mdb</tt>.  This is the database that is displayed by
default in the Protein Database panel described in the next section.
The subset of chains
used by <a href="promodel.htm">Homology Model</a> for segment matching is
specified by <tt>$MOE/lib/segment.lis</tt>.  Defaults can be changed in
the <b>Protein Defaults</b> page of the 
<a href="../appendix/moeconfig.htm#ProteinDefaults">Configuration Options</a>
panel
(<span class="nobr"><b>MOE | Window | Options</b></span>).  

<p>For more information on how to prepare a Protein Data Bank file
for use in MOE, see the <a href="proprep.htm">Preparing a Protein
or Protein-Ligand Complex</a>.
</p>

<a name="ProteinDatabasePanel"></a>
<h2>The Protein Database Panel</h2>

<p>
The Protein Database panel can be opened from the
<a href="../moe/gui/mainwin.htm">MOE Window</a> or the
<a href="../moe/gui/se.html">Sequence Editor</a> using
<b>File&nbsp;|&nbsp;Protein&nbsp;Database</b>.

<p align=center><img src="pdb_mdb/pdb.png" alt="Protein Database Panel"></p>

<p>This panel allows you to search for proteins having a given name, code,
date, resolution, or containing a particular <tt>het</tt> group, and then to
save them to a database or load them into MOE.

When the panel opens, it must scan the database and populate the display list.
This may take a while, especially when the database is accessed
for the first time or over a network.

<p>By default, the Protein Database panel lists all the entries found
in the MOE protein database file <tt>$MOE/lib/pdb.mdb</tt>.  To look at
a different database, use the <b>Browse</b> button.  The database whose
entries are currently listed is given in the <b>Database</b> field.

<p>The following information is displayed for each entry:

<blockquote>
<table class="titlebar" border="1" cellpadding="5">
<tr>
<th>Title</th>
<th>Description</th>
</tr>
<tr><td><b>Code</b></td>
<td>PDB Accession Code (with chain letter, if applicable)</td></tr>
<tr><td><b>Date</b></td><td>PDB Accession date (most recent, if revised)</td>
</tr>
<tr><td><b>Res.</b></td><td>X-ray resolution ("-" if not applicable)</td></tr>
<tr><td><b>Name</b></td><td>Molecule name  or Entry title</td></tr>
</table>
</blockquote>

<img src="pdb_mdb/pdb_sort.png" alt="Sort Options"  class = "floatright" />
<p>

The can be sorted by code, date, or resolution.
When the list is sorted by accession code, only the last three letters
of the 4-letter code are used.  The leading digit is used only
to determine the order of codes with the last three letters identical.

<!--
Chain number, header and cross-reference are given
for every chain.  The chains are listed in lexical order, determined by
the last three letters of their PDB accession codes.  If the last three
letters are the same, then the leading digit of the code determines the
sort order. The cross-reference makes it easier to locate structures that
were excluded from the database due to a very high sequence identity with
structures that are already present.
-->

<a name="SearchingtheDatabase"></a>
<h3>Searching the Database</h3>

<p>The lower part of the panel provides options for searching the protein
database for several kinds of information.  When you change the search
options, the entry list becomes restricted to those entries satisfying
the search criteria.  The search criteria can thus be thought of as
<i>filters</i> for the entry list.

The following search options can be used:

<ul>

<li>
<img src="pdb_mdb/pdb_search.png" alt="Search Text Field"  
class = "floatright" />

<b>Search.</b> 
Enter a pattern (words, word fragments, or phrases) that will be searched for
in text derived from all <tt>COMPND, HEADER, TITLE</tt> and <tt>KEYWDS</tt> 
PDB fields in the
database.  You can also search for date, code, and resolution information.

See below for more information about how to use this field.

<p>
<b>Note:</b> Some of the matched entries displayed in the list may not
show the pattern they match.  This is because the pattern matches
a field that is not shown in the list.

<p>
<li>

<img src="pdb_mdb/pdb_date.png" alt="Date Text Field"  class = "floatright" />

<b>Date.</b> 
Enter a date in the text field to restrict the displayed entries
to those that were deposited into the PDB on or after that date.
The date must be of the format <i>yyyy-mm-dd</i>.  An incomplete date,
such as <i>yyyy</i> or <i>yyyy-mm</i> may be also used.

<p>
<li><img src="pdb_mdb/pdb_data.png" alt="Data Toggles" class = "floatright" />

<b>Data.</b>
Choose one or more experimental techniques (Diffraction, NMR, Other)
to restrict the displayed entries to those solved by the selected techniques.

<p>
<li>
<img src="pdb_mdb/pdb_resolution.png" alt="Resolution Field"
class = "floatright" />

<b>Resolution.</b>
Enter the X-ray resolution (in angstroms).  The displayed entries will be
restricted to those solved at this resolution or finer.  Enter 0
to indicate that the X-ray resolution should not be used to restrict
the displayed entries.

<p>
<li>
<img src="pdb_mdb/pdb_het.png" alt="Hets Text Field" class="floatright" />

<b>Hets.</b>
Enter a pattern (words, word fragments, or phrases)
that must match at least of one of the PDB
<tt>HET, HETNAM</tt> or <tt>HETSYN</tt> fields of the database.

See below for more information about how to use this field.
<p>
<b>Note:</b> The default protein database, <tt>pdb.mdb</tt> contains
<i>only</i> protein chains, and thus this field cannot be used.  This field
can only be used when looking at a database that contains ligands and
small molecules such as the full protein database found on the
distribution DVD.

</ul>

<p>The <b>Search</b> and <b>Hets</b> text fields both operate in the
same manner:

<ul>
<li>The following wildcards are accepted:

<dir>
  <table cellspacing="0" border="0" cellpadding="2">
    <tr>
      <td width="10%" valign="top"><tt>*</tt></td>
      <td width="90%" valign="top">matches 0 or more of any character </td>
    </tr>
    <tr>
      <td valign="top"><tt>?</tt></td>
      <td valign="top">matches any single character </td>
    </tr>
    <tr>
      <td valign="top"><tt>@</tt></td>
      <td valign="top">matches any single character in [a..z, A..Z, _ ] </td>
    </tr>
    <tr>
      <td valign="top"><tt>#</tt></td>
      <td valign="top">matches any single digit [0..9] </td>
    </tr>
    <tr>
      <td valign="top"><tt>[...]</tt></td>
      <td valign="top">matches any character in the set;
      ranges (using dash (<tt>-</tt>) symbol) are allowed</td>
    </tr>
    <tr>
      <td valign="top"><tt>[^</tt>...<tt>]</tt></td>
      <td valign="top">matches any character <i>not</i> in the set</td>
    </tr>
  </table>
</dir>

For more on wildcards, see
<a href="../svl/fcnref/findetc.html" class="svl">findmatch</a>.

<li>An implicit <b>*</b> is placed at the beginning and end of each word</li>
<li>Words are space or comma-separated</li>
<li>Exact phrases, such as those containing spaces or commas, must be
enclosed in double quotes (<tt>&quot;</tt>)</li>
<li>The search words and phrases are used in one of the following modes:
   <ul>
      <li><b>And.</b> Only entries containing <i>all</i> of the terms are displayed</li>
      <li><b>Or.</b> Entries containing <i>any</i> of the terms are displayed</li>
      <li><b>Not.</b> Only entries containing <i>none</i> of the terms are displayed</li>
    </ul></li>
</ul>

<p>To reset all of the search fields to the values they had when the
panel was first opened, press <b>Reset&nbsp;Search</b>.  Note that this will
clear all the search text fields.

<p>For more information on a given entry, select the entry and press
<b>Entry&nbsp;Details</b> to open the Protein Database: Entry Details panel.
This panel gives summaries of the PDB entry information for each selected
entry in the list.

<p align=center>
<img src="pdb_mdb/pdb_details.png" alt="Protein Database: Entry Details Panel">

<!--
<p>To search for a protein, enter a character string in the Search
text field and press <tt>Enter</tt>. Proteins matching the string
are displayed in the Protein Database. To retrieve the complete list
of proteins after a search, press the Reset button at the far right of the text
field. You can
also clear the text field and press <tt>Enter</tt>.
Note that any selected entries will remain selected. 
-->

<a name="CopyingProteinsfromtheDatabase"></a>
<h3>Copying Proteins from the Database</h3>

<p>Once you have completed your search of the protein database, you
can preserve the data you have found by one of the following two options:


<img src="pdb_mdb/pdb_copymoe.png" class="floatright"
alt="Protein Database: Copy to MOE">

<ul>
<li><i>Copy the proteins into MOE.</i>

To load one or more proteins into MOE, select them in the list and press
<b>Copy&nbsp;to&nbsp;MOE</b>.  This will open the
Protein&nbsp;Database:&nbsp;Copy&nbsp;to&nbsp;MOE
panel which gives you load options.

<p>When proteins are loaded from the MOE protein database, the attributes
of the newly created chains are set as follows: the chain tag is set to the
4-character PDB accession code, and the chain name is set to the accession
code and chain letter (if any), separated by a period. The chain header
field will contain the entry annotation.

<p>
The UID and INS attributes of the newly created residues are set according to
the PDB sequence numbers ("resSeq") and PDB insertion codes ("iCode") of the
original entry.

<p>
<li><i>Create a new database containing the proteins.</i>

<p>To create a new database containing only the selected entries in the
list, press <b>Copy to Database</b>.  This will open a Copy to Database file
selection box which will prompt you for the name of a destination database.
A number of fields will be written to this new database, including a
molecule field containing the actual molecular data, and several other
fields containing PDB field information.  The information contained in
this new database is such that the database can be examined using the
Protein Database panel (press <b>Browse</b> to change the current database).

</ul>

<a name="OpeningPDBFiles"></a>
<h2>Reading and Writing PDB Files</h2>

To read molecules from PDB files into MOE, choose
<b>File&nbsp;|&nbsp;Open</b> in either the MOE Window or
the Sequence Editor.
When PDB files are selected in the file list and <b>Load&nbsp;PDB</b>
is pressed, the following panel will appear:

<p>
<center><img src="pdb_mdb/ReadPDB.png" alt="Load PDB File Panel"></center>

<p>
The options for reading PDB files are as follows:

<p>
<table class="titlebar" width="100%" cellpadding="5" border="1">

<tr>
<th>Option</th>
<th>Description</th>
</tr>
<tr>
<td valign="top"><b>Non-Crystallographic</b></td>
<td valign="top">
If <b>Apply&nbsp;NCS&nbsp;Transforms</b> is on, then any
non-crystallographic symmetry
operators in the PDB file (represented by <tt>MTRIXn</tt> records) will be 
applied to create those atoms in the asymmetric unit that are not explicitly
represented in the file. If <b>Apply&nbsp;BIOMT&nbsp;Transforms</b> is on, then all
atoms representing the presumed "biologically significant
oligomerization state" of the file's primary entity will be created by
applying the transforms in the REMARK 350 record to the chains specified there.
</td></tr>

<tr>
<td valign="top"><b>Crystallographic</b></td>
<td valign="top">
If <b>Install&nbsp;Crystal&nbsp;Parameters</b> is on, then the current
<a href="../apps/crystal.html">
<span class="nobr">Crystal Parameters</span>
</a>
(spacegroup, unit cell size and shape) will be updated according to information
in the PDB file.  If <b>Enable Periodic System</b> is on, then the display and
enforcement of the periodic system will be enabled.
</td></tr>

<tr>
<td valign="top"><b>Multiple Models</b></td>
<td valign="top">
If <b>Load&nbsp;All</b> is on, all of the models in a multiple-model PDB file
(such as an NMR file) will be loaded into the system. Otherwise, only the
first model will be loaded.
</td></tr>

<tr>
<td valign="top"><b>Sequence</b></td>
<td valign="top">
If <b>Integrate&nbsp;SEQRES&nbsp;Records</b> is on, then all residues in the 
<tt>SEQRES</tt>
records will be created, including those without corresponding entries
in the <tt>ATOM</tt> records.
</td></tr>

<tr>
<td valign="top"><b>Atom Connect</b></td>
<td valign="top">
If <b>Enable Auto-connect</b> is on, automatic bonding of atoms [Labute 2005]
is performed after <tt>CONECT</tt> records are applied.  Automatic bonding
is always done on atoms of amino acid residues.

If <b>Ignore CONECT&nbsp;Records</b> is on, then <tt>CONECT</tt> records will
not be used when creating bonds.
</td></tr>

<tr>
<td valign="top"><b>Alternate Locations</b></td>
<td valign="top">
Specifies the treatment of alternate atom locations. If <b>Ignore</b>
is chosen, only the highest occupancy positions will be read
into MOE. Otherwise all alternative positions will also be read,
and assigned to extra, inert atoms, and atom collections will be
created named PDB_ALTLOC_?, where <b>?</b> will be replaced by
the alternative location character.  In the latter case, the
<b>Show</b> and <b>Hide</b> options control whether these extra
atoms will be initially visible or not.
</td></tr>

<tr>
<td valign="top"><b>Ignore</b></td>
<td valign="top">
If <b>Waters</b> is on, then all waters in the PDB file will be ignored.
If <b>Hetero</b> is on, then all hetero residues
in the PDB file will be ignored.
</td></tr>

<tr>
<td valign="top"><b>New Chains</b></td>
<td valign="top">
If <b>Center&nbsp;Atoms</b> is on, the view in the main MOE window will be adjusted
so as to place any new atoms at the center.
If <b>Select&nbsp;Chains</b> is on, all new chains from the PDB file
will be selected in the Sequence Editor.
If <b>Close&nbsp;Current&nbsp;System</b> is on, the current system will be closed
before the new molecules are loaded.
</td></tr>

</table>

<p>To save molecular data to a PDB file,
choose <b>File&nbsp;|&nbsp;Save</b> in either the MOE Window or the
Sequence Editor, then choose <i>Molecule</i> from the <b>Save</b> pulldown menu
and <i>Protein Data Bank (.pdb)</i> from the <b>Format</b> pulldown menu.
The <b>Save</b> Panel will look like this:

<p>
<center><img src="pdb_mdb/WritePDB.png" alt="Save PDB File Panel"></center>


<p>The options for writing PDB files are as follows:

<p>
<table class="titlebar" width="100%" cellpadding="5" border="1">

<tr>
<th>Option</th>
<th>Description</th>
</tr>
<tr>
<td valign="top"><b>Preserve Chain ID</b></td>
<td valign="top">
If on, an attempt will be made to preserve any chain identifier found
in the chain name. If no chain identifier can be found, then the first
available identifier in the sequence [A..Z] is used.
If off, then chains are sequentially assigned a chain identifier
in the range [A..Z].
</td></tr>

<tr>
<td valign="top"><b>Conform Naming to</b></td>
<td valign="top">
If on, hydrogen names will be translated before being written according to
the chosen nomenclature convention. <b>PDB</b> refers to the conventions
in place prior to the RCSB data remediation project; this choice is appropriate
when writing files intended to be read by older, unmaintained software. 
<b>PDB&nbsp;v3.0</b> refers to the atom naming conventions adopted during the
remediation project by members of the Worldwide Protein Data Bank consortium
[Berman&nbsp;2003]. <b>IUPAC</b> [IUPAC&nbsp;1998] is identical to
<b>PDB&nbsp;v3.0</b>
except for the naming of the peptide hydrogen atom, which is "HN" as opposed
to "H".
If <b>AMBER</b> is chosen, neutral versions of the typically
charged residues Asp, Glu, Arg and Lys will be renamed.
Histidine residues will be named according to their
protonation state; half-cystines will be named CYX,
as will de-protonated cysteines.
<p><b>Note:</b> the IUPAC treatment of the carboxyl or carboxylate
terminus (O', O'' and H'') is <i>not</i> implemented here. 
</td></tr>

<tr>
<td valign="top"><b>Replace B-Factor with</b></td>
<td valign="top">
If on, then the temperature factor field will be used to store the selected
atomic property.  If off, then temperature factor values will be saved.

<p>For more information on the available atomic properties, see
<a href="../moe/fcnref/asmref.htm" class="svl">aSlogP</a>,
<a href="../moe/fcnref/atom_p.htm" class="svl">aCharge</a>,
<a href="../moe/fcnref/atom.htm" class="svl">aIdealVSA</a>,
<a href="../moe/fcnref/potfcn.htm" class="svl">Potential</a>,
<a href="../moe/fcnref/atom_d.htm" class="svl">aScalar</a>,

and
<a href="../moe/fcnref/atom.htm" class="svl">aTempFactor</a>.
</td></tr>

<tr>
<td valign="top"><b>Scale B-Factor Values</b></td>
<td valign="top">
If on, then values saved in the temperature factor field are scaled to the
range [0,100]; otherwise the values are written verbatim.
</td></tr>

<tr>
<td valign="top"><b>Write TER Records for All Chains</b></td>
<td valign="top">
If on, chains will
be written in the same order that they are seen in the Sequence Editor,
and TER records will be written after each chain. Otherwise, protein
chains will be written before non-protein chains, and only protein
chains will be followed by TER records.

<p>
Enabling this option may be necessary for some non-standard PDB readers
to properly interpret PDB files written by MOE.
</p>
</td></tr>

</table>

<a name="OpeningmmCIFFiles"></a>
<h2>Reading mmCIF Files</h2>

To read molecules from mmCIF files into MOE, choose
<b>File&nbsp;|&nbsp;Open</b> in either the MOE Window or
the Sequence Editor.
When mmCIF files are selected in the file list and <b>Open&nbsp;CIF</b>
is pressed, the following panel will appear:

<p>
<center><img src="pdb_mdb/ReadCIF.png" alt="Load CIF File Panel"></center>

<p>
The options for reading mmCIF files are as follows:

<p>
<table class="titlebar" width="100%" cellpadding="5" border="1">

<tr>
<th>Option</th>
<th>Description</th>
</tr>

<tr>
<td valign="top" width="12%"><b>Max. # Models</b></td>
<td valign="top">
If <b>All</b> is selected, all of the models in a multiple-model mmCIF file
(such as an NMR file) will be loaded into the system. Otherwise, at most the
number of models specified will be loaded.
</td></tr>

<tr>
<td valign="top"><b>Symmetries</b></td>
<td valign="top">
If <b>Non-crystallographic</b> is selected, then any non-crystallographic
symmetry (NCS) operators in the mmCIF file (represented by the
<tt>_struct_ncs_oper</tt> tables) will be 
applied to create those atoms in the asymmetric unit that are not explicitly
represented in the file.   When set to <b>Biological&nbsp;Assembly</b>,
all atoms representing the presumed &quot;biologically significant
oligomerization state&quot; of the file's primary entity will be created by
applying the transforms in the <tt>_struct_assembly</tt> tables to the chains
specified there.  NCS images will never be generated when this option is used
since they are mutually exclusive.  When set to <b>None</b>, neither NCS
nor biological assembly information will be applied.

When <b>Install&nbsp;Crystal&nbsp;Parameters</b> is on, the current
<a href="../apps/crystal.html">
<span class="nobr">Crystal Parameters</span>
</a>
(spacegroup, unit cell size and shape) will be updated according to information
in the CIF file.</td></tr>

<tr>
<td valign="top"><b>Chain Grouping</b></td>
<td valign="top">
If <b>Strict CIF Chains</b> is selected, then each distinct molecule
(represented by a unique <tt>struct_asym.id</tt> in mmCIF) will get its own chain in MOE.
Otherwise, molecules will be grouped together by PDB chain code with the exception
that waters and polymers will always be in their own chains.
</td></tr>

<tr>
<td valign="top"><b>New Objects</b></td>
<td valign="top">
If <b>Center</b> is on, the view in the MOE Window will be adjusted
so as to place any new atoms at the center.
If <b>Select</b> is on, all new atoms, residues and chains will be selected
once loaded.
If <b>Close&nbsp;Current&nbsp;System</b> is on, the current system will be
closed before the new molecules are loaded.
</td></tr>

</table>

<p>
<center><img src="pdb_mdb/SelectAssembly.png" alt="Select Assembly Panel"></center>
</p>

<p>If <b>Symmetries</b> is set to <b>Biological Assembly</b> and multiple
assemblies are provided in the mmCIF file, a second panel will appear listing
all the assemblies in the file (provided by the file author).  Choose the
assembly you wish to load, or <b>All</b> to load all the assemblies, and press
OK.</p>

<h2>
<a name="RCSBDownload"></a>
<a name="PSILODownload">Downloading PDB Files from PSILO or the RCSB</a>
</h2>

<p>The <b>PSILO/RCSB Browse</b> panel is a direct link between MOE and a 
<a href="http://www.chemcomp.com/psilo-info.htm" target="_blank">PSILO
server</a>, the <a href="http://www.rcsb.org">RCSB Protein Data Bank</a>
internet site, or a user specified RCSB mirror site (see below).
PDB files can be downloaded with an option to immediately load the protein
into MOE.

<blockquote>
<b>Note:</b>&nbsp;&nbsp; Downloading files from the internet or PSILO requires that
Java has been configured to run with MOE. Please read instructions on <a
href="../install/install.htm#JDBC">installing&nbsp;Java</a> if there is any trouble
using the download tool.
</blockquote>

<p>To open the PSILO/RCSB Browse panel, choose 
<b>File&nbsp;|&nbsp;PSILO/RCSB&nbsp;Download</b> from either the 
MOE Window or the Sequence Editor.  The following panel appears:

<p align="center">
<img src="pdb_mdb/PSILO_browse.png">
</p>

<p>Upon opening, the PSILO/RCSB Browse panel
immediately searches for an existing
<i>Index&nbsp;File</i> (<tt>rcsb_entries.idx</tt> or <tt>psilo_entries.idx</tt>
depending on the source).  If one exists then the
<b>Database&nbsp;Index</b> list will be populated with the
information in this file.
Each time the index list is updated, a new <tt>*_entries.idx</tt> file will be
created.  By default, the index file is created in the current MOE directory,
however, this directory can be modified by setting the <tt>rcsb.index</tt>
resource, either in the MOE resource configuration file or using
the SVL command <a href="../moe/fcnref/rc.htm" class="svl">SetRC</a>.

<h4>Browsing the List</h4>

<p>The <b>Site</b> and <b>URL</b> fields specify the site used
for any operation performed from this panel.  The currently displayed index
list, however, is not necessarily derived from this site since an already
existing local <tt>*_entries.idx</tt> file may have been used upon opening the
panel.

<p>A user specified RCSB mirror site can be added to the <b>Site</b>
option menu by setting the resource <tt>rcsb.mirror</tt>, 
either in the MOE resource configuration file or using
the SVL command <a href="../moe/fcnref/rc.htm" class="svl">SetRC</a>.
This site will show up in the menu as <b>Local</b>.  The
URL is expected to refer to a site with the same file structure as the RCSB. If
the download site is invalid then the panel will fail to function properly.

<p>If <a href="../appendix/moeconfig.htm">PSILO connectivity</a> has been
configured then PSILO will automatically be added to the <b>Site</b> option
menu as well.

<p>The index list displays the code, date, resolution (Res), and description
(Name) found in the <tt>*_entries.idx</tt> file.  The
<b>Search</b> field can be used to search for specific proteins.  This field
searches the entries for patterns (words, word fragments or phrases) and shows
any of the matching entries.

<p>When the desired protein is located, the entry must be selected to indicate
that it should be downloaded.  Multiple entries may be selected from the list.
All selected entries, together with any specified accession codes listed in the
<b>Extra&nbsp;Codes</b> text field, will be downloaded.


<h4>Downloading Files</h4>

<p>Once the accession codes have been specified, the actual downloading of the
protein files can be done by pressing the <b>Download</b> button.  The controls
for the download operation are as follows:

<p>
<table class="titlebar" width="100%" cellpadding="5" border="1">

<tr>
<th>Name</th>
<th>Description</th>
</tr>
<tr>
<td valign="top" nowrap><b>Site</b></td>
<td valign="top">The site used for the download.
</td></tr>

<tr>
<td valign="top" nowrap><b>Selected Entries</b></td>
<td valign="top">
Accession codes are taken from each selected entry in the list.
</td></tr>

<tr>
<td valign="top" nowrap><b>Extra Codes</b></td>
<td valign="top">
List of additional codes to download.  Multiple codes are listed comma
separated.
</td></tr>

<tr>
<td valign="top" nowrap><b>Directory</b></td>
<td valign="top">
The directory in which to save the downloaded PDB files.  The default is the
current MOE directory.
</td></tr>

<tr>
<td valign="top" nowrap><b>Copy Proteins to MOE</b></td>
<td valign="top">If <b>on</b> then each successfully downloaded file will be
loaded into MOE.
</td></tr>

<tr>
<td valign="top" nowrap><b>Uncompress Files<br>After Download</b></td>
<td valign="top">
By default, files are stored in a compressed state when obtained from the RCSB.
These filenames have the extension <tt>.gz</tt>.  If <b>Uncompress</b> is on
then the files will be uncompressed and the <tt>.gz</tt> extension removed.
Files obtained from PSILO are always uncompressed.
</td></tr>

<tr>
<td valign="top" nowrap><b>Overwrite Existing<br>Files</b></td>
<td valign="top">
If a file with the same accession code exists in the specified directory then
they will be preserved unless <b>Overwrite</b> is on.
</td></tr>

</table>

<p>The download panel appears:

<p>
<center>
<img src="pdb_mdb/PSILO_download.png">
</center>

<p>Each code can have one of the following status states upon opening:

<dir>
<table>
<tr>
<td><tt>pending</tt></td>
<td>File will be downloaded.
</td></tr>
<tr>
<td><tt>overwrite</tt></td>
<td>File will be downloaded and the old file will be overwritten.
</td></tr>
<tr>
<td><tt>exists</tt></td>
<td>File will not be downloaded as it already exists in the designated
directory.
</td></tr>
</table>
</dir>

<p>When the download is complete, each file will have one of the
following status states:

<dir>
<table>
<tr>
<td valign="top"><tt>complete</tt></td>
<td valign="top">File downloaded and the operation completed as specified.
</td></tr>
<tr>
<td valign="top"><tt>overwritten</tt></td>
<td valign="top">File downloaded, the old file was overwritten and the operation
completed as specified.
</td></tr>
<tr>
<td valign="top"><tt>preserved</tt></td>
<td valign="top">File was not downloaded due to an existing file.  The operation
was completed successfully.
</td></tr>
<tr>
<td valign="top"><tt>failed</tt></td>
<td valign="top">An error occurred either during the connection, the download, the
writing of the file or the loading into MOE.
</td></tr>
</table>
</dir>

<a name="SVLCommands"><h2>SVL Commands</h2>

<a href="fcnref/pdb.html" class="svl">pdb_List</a><br />
<a href="fcnref/pdb.html" class="svl">pdb_Open</a><br />
<a href="../moe/fcnref/rcsb.htm" class="svl">rcsb_download</a><br />

<p><a href="fcnref/intropro.html">Protein Functions</a>


<a name="References"><h2>References</h2>


<table width="100%" cellpadding="5" border="0">

<tr>
<td valign="top">[Berman&nbsp;2000]</td>
<td valign="top">
Berman, H.M., Westbrook, J., Feng, Z., Gilliland, G., Bhat, T.N., Weissig,
H., Shindyalov, I.N., Bourne, P.E.;  The Protein Data Bank;
<i>Nucleic Acids Research&nbsp;28</i> (<b>2000</b>) 235&ndash;242.
</td></tr>

<tr>
<td valign="top">[Berman&nbsp;2003]</td>
<td valign="top">
Berman, H.M., Henrick, K., Nakamura, H.;
Announcing the worldwide Protein Data Bank;
<i>Nat. Struct. Biol.&nbsp;10</i> (<b>2003</b>) 980.
</td></tr>

<tr>
<td valign="top">[IUPAC&nbsp;1998]</td>
<td valign="top">
Markley, J.L., Bax, A., Arata, Y., Hilbers, C.W., Kaptein, R., Sykes, B.D.,
Wright, P.E., Wüthrich, K.;
Recommendations for the Presentation of NMR Structures of Proteins and
Nucleic Acids;
<i>Pure &amp; Appl. Chem.&nbsp;70</i> (<b>1998</b>) 117&ndash;142.
</td></tr>

<tr>
<td valign="top">[Labute&nbsp;2005]</td>
<td valign="top">
Labute, P.; On the Perception of Molecules from 3D Atomic Coordinates;
<i>J. Chem. Inf. Model.&nbsp;45</i> (<b>2005</b>) 215&ndash;221.
</td></tr>

</table>


<h2>See Also</h2>

<a href="../moe/gui/mainwin.htm">MOE Window</a><br />
<a href="../moe/gui/se.html">Sequence Editor</a><br />
<a href="../moe/fcnref/rcsb.htm">RCSB Download</a><br />
<a href="prosearch.htm">MOE-SearchPDB</a><br />
<a href="../apps/crystal.html">Crystal Parameters</a><br />

<br />

<a href="http://www.rcsb.org">RCSB Protein Data Bank</a>

<p><a href="../index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
