<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<!--
!!   	protool.htm		Protein and Sequence Builders
!!
!!   	02-sep-2009 (sm) new screenshots
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../manstyle.css" />



<meta panel Protein Builder>
<meta panel Create Sequence>

<title>Building Proteins and DNA </title>

</head>

<body bgcolor="#ffffff">
<a href="../index.htm"><img src="../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>Building Proteins and DNA </h2>
<hr noshade="noshade" />

<p>
This chapter describes how to create and edit proteins using the Protein
Builder, the Create Sequence panel, and the Edit Protein commands in the
Sequence Editor.
</p>

<ul>
<li><a href="#CreatingProteins">Creating Proteins</a>
    <ul>
    <li><a href="#ProteinBuilderPanelRef">Protein Builder Panel Reference</a></li>
    </ul></li>
<li><a href="#Create_Sequence">Creating Sequences</a></li>
<li><a href="#Editing_Proteins">Editing Proteins</a>
    <ul>
    <li><a href="#Acetylate">Acetylate N-terminus</a></li>
    <li><a href="#Amidate">Amidate C-terminus</a></li>
    <li><a href="#SecondaryStructure">Secondary Structure</a></li>
    </ul></li>
<li><a href="#Codes">Residue Codes</a></li>
<li><a href="#SVL">SVL Commands</a></li>
<li><a href="#References">References</a></li>
</ul>

<h2><a name="CreatingProteins">Creating Proteins</a></h2>

<p>
The MOE Protein Builder is used to construct polypeptides from the standard 20
amino acids, plus acetyl and amide end-caps, as well as three variants of
histidine and one of proline.  It is intended to be used when proteins with
atomic data are desired.  The Create Sequence command (see next section) is
used when sequence only data is required.</p>

<p>To open the Protein Builder, choose
<b>SE&nbsp;|&nbsp;Edit&nbsp;|&nbsp;Protein&nbsp;Builder</b>
in the Sequence Editor (or
<b>MOE&nbsp;|&nbsp;Edit&nbsp;|&nbsp;Build&nbsp;|&nbsp;Protein</b>).
The following panel appears: </p>

<p align="center">
<img src="protool/probuilder.png" alt="Protein Builder Panel" />
</p>

<p>
Press one of the buttons in the Residue bank to append the selected amino acid
to the carboxyl terminal of the chain being built.  The Geometry button bank
can be used to select the backbone dihedral angles of any subsequent residue.
</p>

<p>Press <b>New Chain</b> to create the new chain.

Note that when New Chain is pressed, any previously created chain can no longer
be edited using the Protein Builder.
</p>

<a name="ProteinBuilderPanelRef" />
<h3>Protein Builder Panel Reference</h3>

<table width="100%" cellpadding="5">
<tr>
<td valign="top"><b>Residue</b></td>
<td valign="top">
Appends the selected amino acid to
the carboxyl terminal of the chain being built. 

<p><b>Acetylate N-Term</b> creates an ACE N-terminus, and <b>Amidate
C-Term</b>, an NME C-terminus.  Once an NME has been attached to a protein, no
further residues can be appended: pressing buttons from the Residue bank will
have no effect.  The ACE end-cap can be prepended at any time; it is the only
residue that can be joined into the sequence out of order.
</p>
<p>Residue codes appear <a href="#Codes">below</a>.
</p>
</td>
</tr>
<tr>
<td valign="top"><b>Geometry</b></td>
<td valign="top">Geometry buttons are used to select the backbone dihedral
angles of any subsequent residue. 

<p><b>poly pro</b> refers to poly-proline geometry.
</p>
<p>
If <b>custom</b> is selected, the backbone conformation angles will be taken
from the phi and psi text fields (interpreted as angles in degrees, measured
clockwise).
</p></td>
</tr>
<tr>
<td valign="top"><b>phi/psi</b></td>
<td valign="top">
Backbone dihedral angles associated with the various geometry buttons:

<br /><br />
<center><table class="titlebar">
  <tr>
    <th>label</th>
    <th>phi angle</th>
    <th>psi angle</th>
  </tr>
    <tr><td>helix</td><td>-65</td><td>-39</td></tr>
    <tr><td>3-10-helix</td><td>-50</td><td>-25</td></tr>
    <tr><td>pi-helix</td><td>-45</td><td>-80</td></tr>
    <tr><td>strand</td><td>-120</td><td>113</td></tr>
    <tr><td>anti-strand</td><td>-140</td><td>135</td></tr>
    <tr><td>poly pro</td><td>-65</td><td>140</td></tr>
    <tr><td>extended</td><td>180</td><td>180</td></tr>
</table>
</center>

<p>The chi angles are set to 180 degrees (extended conformation).
</p>

<p>If the <b>cis conformation</b> option is turned on,
this sets the omega angle to 0 degrees. 
</p>

<p>If the <b>left-handed helices</b> option is turned on, this forces
left-handed geometry, negating the phi and psi angles of the standard helix
conformations.  Non-helix geometries are not affected.
</p>
</td>
</tr>
</table>

<h2><a name="Create_Sequence"></a>Creating Sequences</h2>

<p>To open the Create Sequence panel, choose 
<b>SE&nbsp;|&nbsp;Edit&nbsp;|&nbsp;Create&nbsp;Sequence</b>
in the Sequence Editor.  This panel is used to create
polypeptide and DNA fragments from a string of single-letter identifiers.
</p>

<p align="center">
<img src="protool/bioseq.png" alt="Create Sequence Panel" />
</p>

<p>
The Create Sequence input line recognizes single-letter codes of residues (see
<a href="#Codes">below</a>).  Sequence Type options allow you to create a
sequence of empty residues (i.e. without atoms) or a fully-populated
peptide chain in either extended or alpha-helix conformation.  The DNA options
create fully-populated strands in A, B or C conformation [Arnott 1975].</p>

<p>
The name, tag and header attributes of the chain can be set in this
panel. Press OK to create the sequence. Press Apply to create the sequence
without closing the panel.
</p>

<p><font color="#ff0000"><b>Tip</b></font>&nbsp;&nbsp;
To cut and paste a sequence into the text field use the keyboard cut and paste
facilities.  For more information on the text field see <a
href="../moe/hellomoe.htm#GUI_Basics">GUI Basics</a>.
</p>

<h2><a name="Editing_Proteins"></a>Editing Proteins</h2>

<p>
The <b>SE&nbsp;|&nbsp;Edit&nbsp;|&nbsp;Protein</b> submenu in the Sequence Editor
contains commands to amidate or acetylate protein chains, as well as to
manipulate protein secondary structure.
</p>

<a name="Amidate" /><h3>Amidate C Terminus</h3>

<p>Choose <b>SE&nbsp;|&nbsp;Edit&nbsp;|&nbsp;Protein&nbsp;|&nbsp;Amidate_C_Terminus</b>.
This command allows you to append the NME C-terminal residue to any protein
chain.  You will be prompted in the Sequence Editor for the chain to amidate.
</p>
<p>The command will remain active until it is cancelled with <tt>Esc</tt>.
</p>
<h3><a name="Acetylate">Acetylate N Terminus</a></h3>

<p>
Choose <b>SE&nbsp;|&nbsp;Edit&nbsp;|&nbsp;Protein&nbsp;|&nbsp;Acetylate_N_Terminus</b>.
This command allows you to prepend any protein chain with the ACE N-terminal
residue provided that the first residue of the chain has a backbone nitrogen
and alpha-carbon. You will be prompted in the Sequence Editor for the chain to
acetylate.
</p>
<p>The command will remain active until it is cancelled with <tt>Esc</tt>.
</p>

<h3><a name="SecondaryStructure">Secondary Structure</a></h3>

<p>
The <b>SE&nbsp;|&nbsp;Edit&nbsp;|&nbsp;Protein&nbsp;|&nbsp;Structure</b>
command allows you to
edit the user-assigned secondary structure labels of selected residues. You can
set them to one of the legal segment types (see <a
class="svl" href="../moe/fcnref/moe_const.html"> R_SEGMENT_TYPES</a>), or you can copy
the actual secondary structure labels - as determined by the protein
conformation - to the user-assigned labels.
</p>
<p>You can also adjust the backbone dihedral angles of the protein to values
which correspond to the assigned segment labels with the
<b>Edit&nbsp;|&nbsp;Protein&nbsp;|&nbsp;Structure&nbsp;|&nbsp;To_Actual</b>
command.
</p>

<p>See <a class="svl" href="fcnref/pro_seg.html">pro_SegmentToDihedrals</a>
for a table of the backbone dihedral angles which will be used for each
segment type.
</p>

<p><b>Note:</b> This function will have no effect where bonds are not
rotatable (e.g. if there is a bond cycle due to the existence of
disulfide bonds).
</p>

<a name="Codes"></a>
<h2>Residue Codes</h2>

<p>
<center>
<table class="titlebar">
  <tr>
    <th>1-letter code</th>
    <th>3-letter code</th>
    <th>Amino acid</th>
  </tr>
  <tr> <td><tt>A</tt></td> <td><tt>ALA</tt></td> <td>Alanine</td> </tr>
  <tr> <td><tt>R</tt></td> <td><tt>ARG</tt></td> <td>Arginine</td> </tr>
  <tr> <td><tt>N</tt></td> <td><tt>ASN</tt></td> <td>Asparagine</td> </tr>
  <tr> <td><tt>D</tt></td> <td><tt>ASP</tt></td> <td>Aspartate</td> </tr>
  <tr> <td><tt>C</tt></td> <td><tt>CYS</tt></td> <td>Cysteine</td> </tr>
  <tr> <td><tt>Q</tt></td> <td><tt>GLN</tt></td> <td>Glutamine</td> </tr>
  <tr> <td><tt>E</tt></td> <td><tt>GLU</tt></td> <td>Glutamate</td> </tr>
  <tr> <td><tt>G</tt></td> <td><tt>GLY</tt></td> <td>Glycine</td> </tr>
  <tr> <td><tt>H</tt></td> <td><tt>HIS</tt></td> <td>Histidine</td> </tr>
  <tr> <td><tt>I</tt></td> <td><tt>ILE</tt></td> <td>Isoleucine</td> </tr>
  <tr> <td><tt>L</tt></td> <td><tt>LEU</tt></td> <td>Leucine</td> </tr>
  <tr> <td><tt>K</tt></td> <td><tt>LYS</tt></td> <td>Lysine</td> </tr>
  <tr> <td><tt>M</tt></td> <td><tt>MET</tt></td> <td>Methionine</td> </tr>
  <tr> <td><tt>F</tt></td> <td><tt>PHE</tt></td> <td>Phenylalanine</td> </tr>
  <tr> <td><tt>P</tt></td> <td><tt>PRO</tt></td> <td>Proline</td> </tr>
  <tr> <td><tt>S</tt></td> <td><tt>SER</tt></td> <td>Serine</td> </tr>
  <tr> <td><tt>T</tt></td> <td><tt>THR</tt></td> <td>Threonine</td> </tr>
  <tr> <td><tt>W</tt></td> <td><tt>TRP</tt></td> <td>Tryptophan</td> </tr>
  <tr> <td><tt>Y</tt></td> <td><tt>TYR</tt></td> <td>Tyrosine</td> </tr>
  <tr> <td><tt>X</tt></td> <td><tt>UNK</tt></td> <td>Unknown</td> </tr>
  <tr> <td><tt>V</tt></td> <td><tt>VAL</tt></td> <td>Valine</td> </tr>
</table>
</center>
</p>

<br />

<p>
<center>
<table class="titlebar">
  <tr>
    <th>1-letter code</th>
    <th>Nucleic Acid</th>
  </tr>
  <tr>
    <td><tt>a</tt></td>
    <td>Adenosine</td>
  </tr>
  <tr>
    <td><tt>c</tt></td>
    <td>Cytidine</td>
  </tr>
  <tr>
    <td><tt>g</tt></td>
    <td>Guanosine</td>
  </tr>
  <tr>
    <td><tt>t</tt></td>
    <td>Thymidine</td>
  </tr>
</table>
</center>
</p>

<br />
<a name="SVL"></a><h2>SVL Commands</h2>
<a href="fcnref/intropro.html">Protein Functions</a>

<a name="References"></a><h2>References</h2>

<table width="100%" cellpadding="5" border="0">
<tr>
<td valign="top">[Arnott&nbsp;1975]</td>
<td valign="top">
Arnott, S., Selsing, E.; The conformation of C-DNA;
<i>Journal of Molecular Biology</i> <i>98</i> (<b>1975</b>) 265&ndash;269.
</td>
</tr>
</table>

<h2>See Also</h2>

<a href="../apps/rotamer.htm">Residue Mutation and Rotamer Exploration</a><br />
<a href="propred.htm">Predicting Secondary Structure and Solvent Accessibility</a><br />
<a href="pdb_mdb.htm">Protein Sequence and Structure Data</a><br />
<a href="../moe/gui/se.html">Sequence Editor Window</a><br />

<p><a href="../index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
