<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<!--
!!	molrw.htm	Read/Write Molecular Data Functions
!!	moe/fcnref
!!
!!      30-sep-2009 (al) clarification
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../../manstyle.css" />



<meta keywords>
<meta functions ReadMOE, WriteMOE>
<meta functions ReadMDLMOL, WriteMDLMOL>
<meta functions ReadPDB, WritePDB>
<meta functions ReadTriposMOL2, WriteTriposMOL2>
<meta functions Read_Formats, Write_Formats>
<meta functions ReadAuto, WriteAuto>
<meta functions ReadClustal, WriteClustal>
<meta functions ReadEMBL, WriteEMBL>
<meta functions ReadFasta, WriteFasta>
<meta functions ReadGB, WriteGB>
<meta functions ReadGCG, WriteGCG>
<meta functions ReadIG, WriteIG>
<meta functions ReadMSF, WriteMSF>
<meta functions ReadPIR, WritePIR>
<meta functions ReadPhylip, WritePhylip>
<meta functions ReadMacroModel, WriteMacroModel>
<meta functions ReadCIF>

<title>Read/Write Molecular Data Functions</title>

</head>

<body bgcolor="#ffffff">
<a href="../../index.htm"><img src="../../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>Read/Write Molecular Data Functions</h2>
<hr noshade="noshade" />

<h2>Syntax</h2>
<pre>
    new_chains = <a class="fcnlink" href="#ReadAuto">ReadAuto</a> [ 'filename', options, 'format' ]
    <a class="fcnlink" href="#WriteAuto">WriteAuto</a> [ 'filename', options, 'format' ]

    format_tokens = <a class="fcnlink" href="#Read_Formats">Read_Formats</a> []
    format_tokens = <a class="fcnlink" href="#Write_Formats">Write_Formats</a> []

    <a class="fcnlink" href="#ReadXXX">ReadXXX</a> filename
    <a class="fcnlink" href="#ReadXXX">ReadXXX</a> [ filename, options ]
    chain_keys = <a class="fcnlink" href="#ReadXXX">ReadXXX</a> filename
    chain_keys = <a class="fcnlink" href="#ReadXXX">ReadXXX</a> [ filename, options ]

    <a class="fcnlink" href="#WriteXXX">WriteXXX</a> filename
    <a class="fcnlink" href="#WriteXXX">WriteXXX</a> [ filename, options ]
</pre>

<p><a name="Read_Formats"></a></p>
<h2>Description</h2>

<p>MOE can read and write several molecular formats. The <tt>ReadXXX</tt>
functions read molecular files with explicitly specified formats and
create the molecules in MOE.  Similarly,
the <tt>WriteXXX</tt> functions can be used to write the
current molecular system into files of given formats.
In the case of <tt>ReadAuto</tt> and <tt>WriteAuto</tt>,
MOE automatically detects the file format based on
file extension alone.</p>

<p>MOE is able to read and write the following formats:
<ul>
<li>Molecular files
<li>Sequence-only files
<li>Custom formats
</ul>

<p><a name="Read_Formats"></a></p>
<p><a name="Write_Formats"></a></p>

<p>The current list of recognized formats can be obtained using
<tt>Read_Formats</tt> and <tt>Write_Formats</tt>.


<table border="0" width="100%" cellpadding="5" cellspacing="3">
<tr>
<td valign="baseline"><tt><span class="fcndef">Read_Formats</span></tt></td>
<td valign="baseline">
Returns a vector of tokens, each specifying a currently supported read
format. The presence of a token, for example, <tt>'XXX'</tt>, in the
<tt>format_tokens</tt> vector indicates that there exists an SVL function
called <tt>ReadXXX</tt>.

<pre>
svl&gt; Read_Formats[]
['Auto','moe','pdb','tripos','mdl_mol','macmod','IG/Stan','genbank',
'embl','pir','gcg','fasta','clustal','gcg_msf','phylip','xtl','cif']
</pre>

<tr>
<td valign="baseline"><tt><span class="fcndef">Write_Formats</span></tt></td>
<td valign="baseline">Returns a vector of tokens, each specifying a currently
supported write format. The presence of token <tt>'XXX'</tt> in the
returned vector indicates that there exists an SVL function called
<tt>WriteXXX</tt>. </p>

<pre>
svl&gt; Write_Formats[]
['Auto','moe','pdb','tripos','mdl_mol','macmod','IG/Stan','genbank',
'embl','pir','gcg','fasta','clustal','gcg_msf','phylip','xtl','cif']
</pre>
</table>

<p><a name="ReadAuto"></a></p>
<p><a name="WriteAuto"></a></p>

<p><tt><span class="fcndef">ReadAuto</span></tt>
reads molecular data from a file in the format specified
by the token <tt>format</tt>. For example, if <tt>'PDB'</tt> is specified, then
Protein Data Bank data is assumed. If the format is <tt>[]</tt>, then an
attempt is made to detect the format from the file extension.  If the filename
is <tt>[]</tt> or <tt>''</tt>,
a prompt is displayed for file and format selection. </p>

<p> 
<tt>options</tt> is a tagged vector which can contain the options: 

<ul>
  <tt><li>center:flag</tt> a non-zero <tt>flag</tt> value cause the newly
    read-in atoms to have their center of mass at the origin, otherwise
    not.</li>
  <tt>
  <li>select:flag</tt> if <tt>flag</tt> is non-zero, all newly created objects
    (atoms, residues, chains) will be added to their respective selection
    sets. </li>
</ul>

<p><tt><span class="fcndef">WriteAuto</span></tt>
writes the current molecular data from MOE to the file
<tt>filename</tt> in the format. If the file name is <tt>[]</tt> or
<tt>''</tt>, a prompt is displayed for file and format selection.</p>

<p><a name="ReadXXX"></a></p>
<p><a name="WriteXXX"></a></p>

<p>The naming convention for standard reading and writing functions is
<tt>ReadXXX</tt> and <tt>WriteXXX</tt>, where <tt>XXX</tt> is a format
identifier such as <tt>MOE</tt> or <tt>MDLMOL</tt> (e.g.
<tt>ReadMOE</tt>, <tt>WriteMDLMOL</tt>).


<p>The standard calling syntax is:

<pre>
    <span class="fcndef">ReadXXX</span> 'filename'
    chains = <span class="fcndef">ReadXXX</span> 'filename'
    <span class="fcndef">ReadXXX</span> [ 'filename', options ]
    chains = <span class="fcndef">ReadXXX</span> [ 'filename', options ]

    <span class="fcndef">WriteXXX</span> 'filename'
</pre>

<p>where the <tt>options</tt> argument is a tagged vector of
attribute-value pairs: 

<ul>
  <tt>
  <li>center:flag</tt> if <tt>flag</tt> is non-zero, then the newly read-in
    atoms will have their center of mass at the origin;</li>
  <tt>
  <li>select:flag</tt> if <tt>flag</tt> is non-zero, then the newly read-in
    chains, residues, and atoms will be part of their respective selection
    sets.</li>
</ul>

<p>If the return value of <tt>ReadXXX</tt> is needed, <tt>ReadXXX</tt> returns
the keys of the newly read-in chains. Otherwise, <tt>ReadXXX</tt> returns
null. </p>

<h4>Reading and Writing Molecular Files</h4>

<p>The following read and write file formats store complete 3D representations
of molecular data.

<p>

<table border="0" width="100%" cellpadding="4">

<tr>
<td valign="baseline"><tt>ReadMOE</tt></td>
<td valign="baseline"><tt>WriteMOE</tt></td>
<td valign="baseline"><tt>moe</tt></td>
<td valign="baseline">Native <a href="../../moe/fcnref/moefmt.htm">MOE file
format</a></td>
</tr>

<tr>
<td valign="baseline"><tt>ReadTriposMOL2</tt></td>
<td valign="baseline"><tt>WriteTriposMOL2</tt></td>
<td valign="baseline"><tt>mol2</tt></td>
<td valign="baseline">Tripos Associates MOL2 format</td>
</tr>

<tr>
<td valign="baseline"><tt>ReadPDB</tt></td>
<td valign="baseline"><tt>WritePDB</tt></td>
<td valign="baseline"><tt>pdb</tt></td>
<td valign="baseline">Protein Data Bank&trade; format</td>
</tr>

<tr>
<td valign="baseline"><tt>ReadMDLMOL</tt></td>
<td valign="baseline"><tt>WriteMDLMOL</tt></td>
<td valign="baseline"><tt>mdl_mol</tt></td>
<td valign="baseline">MDL Standard Database format</td>
</tr>

<tr>
<td valign="baseline"><tt>ReadMacroModel</tt></td>
<td valign="baseline"><tt>WriteMacroModel</tt></td>
<td valign="baseline"><tt>macmod</tt></td>
<td valign="baseline">Schrodinger molecular model format</td>
</tr>

<tr>
<td valign="baseline"><tt>ReadCIF</tt></td>
<td valign="baseline">&nbsp;</td>
<td valign="baseline"><tt>cif</tt></td>
<td valign="baseline">[macromolecular] Crystallographic Information File</td>
</tr>
</table>

<h4>Reading and Writing Sequence-Only Files</h4>

<p>The following read and write file formats store sequence-only information 
for proteins, DNA and RNA sequences. <tt>filename</tt> must be a token. There
is no return vector.

<p>
<table border="0" width="100%" cellpadding="4">

<tr>
<td valign="baseline"><tt>ReadClustal</tt></td>
<td valign="baseline"><tt>WriteClustal</tt></td>
<td valign="baseline"><tt>clustal</tt></td>
<td valign="baseline">Format used by the CLUSTAL multiple sequence alignment
software</td>
</tr>

<tr>
<td valign="baseline"><tt>ReadEMBL</tt></td>
<td valign="baseline"><tt>WriteEMBL</tt></td>
<td valign="baseline"><tt>embl</tt></td>
<td valign="baseline">European Molecular Biology Laboratories data bank format</td>
</tr>

<tr>
<td valign="baseline"><tt>ReadFasta</tt></td>
<td valign="baseline"><tt>WriteFasta</tt></td>
<td valign="baseline"><tt>fasta</tt></td>
<td valign="baseline">Fasta format</td>
</tr>

<tr>
<td valign="baseline"><tt>ReadPIR</tt></td>
<td valign="baseline"><tt>WritePIR</tt></td>
<td valign="baseline"><tt>pir</tt></td>
<td valign="baseline">Protein Identification Resource format </td>
</tr>

<tr>
<td valign="baseline"><tt>ReadGB</tt></td>
<td valign="baseline"><tt>WriteGB</tt></td>
<td valign="baseline"><tt>genbank</tt></td>
<td valign="baseline">GenBank format </td>
</tr>

<tr>
<td valign="baseline"><tt>ReadGCG</tt></td>
<td valign="baseline"><tt>WriteGCG</tt></td>
<td valign="baseline"><tt>gcg</tt></td>
<td valign="baseline">Genetics Computer Group single sequence format</td>
</tr>

<tr>
<td valign="baseline"><tt>ReadIG</tt></td>
<td valign="baseline"><tt>WriteIG</tt></td>
<td valign="baseline"><tt>IG/Stan</tt></td>
<td valign="baseline">Stanford/Intelligenetics format</td>
</tr>

<tr>
<td valign="baseline"><tt>ReadMSF</tt></td>
<td valign="baseline"><tt>WriteMSF</tt></td>
<td valign="baseline"><tt>gcg_msf</tt> </td>
<td valign="baseline">Genetics Computer Group multiple sequence format</td>
</tr>

<tr>
<td valign="baseline"><tt>ReadPhylip</tt></td>
<td valign="baseline"><tt>WritePhylip</tt></td>
<td valign="baseline"><tt>phylip</tt> </td>
<td valign="baseline">PHYLogeny Inference Package format</td>
</tr>
</table>

<p>
<h4>Format Specific Notes</h4>

<p><tt>WriteMDLMOL</tt> uses the CTAB format which can specify up to 999
atoms in a molecular system. Attempting to write a larger system to this format
is an error.

<p>
<h4><a href="rwcustom.html">Creating Customized Reading and Writing
Formats</a></h4>

<table border="0" width="50%" cellpadding="4">

<tr>
<td valign="baseline"><tt>Read_ProcessStandardOptions</tt></td>
<td valign="baseline"></td>
</tr>
<tr>
<td valign="baseline"><tt>Read_Formats</tt></td>
<td valign="baseline"><tt>Write_Formats</tt></td>
</tr>
<tr>
<td valign="baseline"><tt>Read_Prompt</tt></td>
<td valign="baseline"><tt>Write_Prompt</tt></td>
</tr>
<tr>
<td valign="baseline"><tt>Read_Return</tt></td>
<td valign="baseline"></td>
</tr>
</table>

<p>
<h2>See Also</h2>

<table>
<tr>
<td><a class="svl" href="openclo.html">Close</a>
<td><a class="svl" href="openclo.html">Open</a>
<tr>
<td><a class="svl" href="save.html">DataFilename</a>
<td><a class="svl" href="save.html">DataModified</a>
<tr>
<td><a class="svl" href="save.html">Save</a>
<td><a class="svl" href="save.html">SaveAs</a>
</table>


<p><a href="../../index.htm">MOE Table of Contents</a>&nbsp;&nbsp;<a href="../../fcnindex.html">SVL Function Index</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
