<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!	/apps/filesave.htm	Save panel
!!-->

<!--
!!    MOE Online Manuals
!!    COPYRIGHT (C) 1997-2015
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

<script type="text/javascript" 
src="../include/jsincludes_moe.js"></script>

<link rel="stylesheet" type="text/css"
href="../include/manstyle.css" />


<title>Saving the System</title>

<meta panel Save />
<meta panel Ray Tracing Settings />

<meta main_keywords POV-Ray, MDL, SDF, SD />
<meta main keywords PNG, JPEG, GIF, BMP, TIF, EPS, PDB, MOE />
<meta main_keywords Fasta, PIR, ALN, EMBL, GB, GCG, IG, MSF, PHY />
<meta main_keywords Schrodinger, MacroModel, mmd, mol, mol2, sdf />
<meta main_keywords mae, xtl, protein data bank, maestro, MSI />
<meta main_keywords tripos, Phylip, NBRF />
<meta main_keywords POV-Ray, encapsulated postscript />
<meta main_keywords theme />

<meta keywords export, graphics, ray-tracing, periodic box, Wall restraint />
<meta keywords viewing, crystal parameters, temperature factor, format />
<meta keywords temperature factor />
<meta keywords raw, sequence, graphic, quality />
<meta keywords anti-aliasing, file system, color, resolution, dimension />
<meta keywords depict, 2D, B-factor, transparent, compression />
<meta keywords CMYK, YCbCr, sRGB, RGB, clipboard, scene file, bitmap />

</head>

<body>
<div id="MoeHeader"></div>
<noscript>
	<span class="warning">Warning: JavaScript is disabled. This page will not display correctly.</span>
	<h1 class="title">MOE Documentation</h1>
	<hr noshade="noshade" />
	<style>.LaTeX {color: #C08080;}</style>
</noscript>

<ul class="index">
    <li><a href="#Intro">Introduction</a></li>
    <li><a href="#Select">Selecting a Format</a>
	<ul>
	<li><a href="#Molecule">Molecular Format</a></li>
	<li><a href="#Sequence">Sequence Format</a></li>
	<li><a href="#Picture">Graphic Format: Saving an Image File</a></li>
	<li><a href="#RTrace">Using POV-Ray</a></li>
	</ul>
	</li>
    <li><a href="#SVL">SVL Commands</a></li>
</ul>

<a name="Intro"><h2>Introduction</h2></a>

<p>The MOE Save menu command allows you to write the current molecular system
to a file. Different file formats are available to accomodate a variety of uses,
including recording project data, sharing information with colleagues or
illustrating presentations and reports.</p>

<p>To save the current molecular system to a file,
choose <nobr><b>File | Save</b></nobr> in either the MOE Window
or the Sequence Editor.</p>

<p style="text-align:center">
<img src="filesave/save_panel.png" alt="Save Panel" />
</p>

<p>Specifying the save filename is done in the
file selection box area of the Save panel (upper half),
and can be accomplished either by selecting
a file from the files list or by entering the name in the Files
text field beneath the Files list.  
For more information, please see
<a href="../moe/hellomoe.htm#File_Selection_Box">File Selection Box</a>.
</p>

<h2><a name="Select">Selecting a Format</a></h2>

<div class="floatright">
<img src="filesave/save_select.png" alt="Format Selection">
</div>

<p>Three categories of file formats are available:
Molecule, Sequence and Picture. Molecule formats preserve the
3D coordinate data of the original system. Sequence formats record
sequence-only information. Picture formats store
a graphical representation of the current molecular system.</p>

<p>The <b>Save</b> pulldown menu is used to pick a
category of file formats while the <b>Format</b> pulldown menu selects
a specific format within that category.</p>

<p>The <b>Automatic Extension</b> checkbox specifies whether the typical
file extension for the selected format should automatically be appended
to the filename, when no extension is specified.</p>

<p>When only parts of a molecular system are of interest, use the
<b>Only Selected</b> checkbox to save only the selected atoms, residues or
chains. This feature is available only for Molecule and Sequence formats.</p>

<p>When a given file format has been selected, the bottom of the Save panel
displays additional save options available for that format.</p>

<h3><a name="Molecule">Molecule Format</a></h3>

<p>The following molecule formats are currently available:</p>

<ul>
<li><a href="#MOE">MOE Molecule File</a> (.moe)</li>
<li><a href="#MDL">MDL SDfile or Molfile</a> (.sdf, .mol)</li>
<li>Tripos MOL2 Molecule File (.mol2)</li>
<li><a href="#PDB">Protein Data Bank File (.pdb)</a></li>
<li>Schrodinger MacroModel File (.mmd)</li>
<li>Schrodinger Maestro File (.mae)</li>
<li>MSI Crystal File (.xtl)</li>
</ul>

<p>Options specific to the MOE and MDL file formats are described below;
options specific to the PDB file format are described in the chapter on
<a href="../proteins/pdb_mdb.htm">Protein Sequence and Structure Data</a>.</p>

<h4><a name="MOE">MOE Molecule File</a></h4>

<div class="floatright">
<img src="filesave/save_moe.png" alt="MOE Parameters" class="snap">
</div>

<p>The MOE molecule file format can save graphic objects, periodic box
parameters, and view parameters in addition to molecular data.
Optionally, sequence-only information of the
molecular objects can be saved.
Note that rendering parameters, such as atom and bond rendering modes, are
automatically saved, as are parameters for backbone ribbon rendering
and for coloring atoms and residues.</p>

<ul>
<p>
<li><b>Graphics.</b> Specifies whether to save user-created graphics
objects, such as molecular surfaces.

    <p>
    <ul>
    <li><i>None.</i> No graphics objects are saved.</li>

    <li><i>Visible Only.</i> Only the currently-visible graphics objects are
    saved.  Hidden graphics objects (e.g. using the <a
    href="../moe/gui/gobjman.htm">Graphic Object Manager</a>) are not saved.</li>

    <li><i>All.</i> All graphics objects (hidden and visible) are saved.</li>
    </ul>
    </p>
</li></p>

<p><li><b>Save.</b>  Specifies whether to save viewing and periodic box
parameters, and whether or not to save atom data.

    <p>
    <ul>
    <li><i>View Parameters.</i> If on, the current viewing parameters are
    saved.  The parameters to be saved are the view "look-at" point, the view
    y- and z- direction vectors, the view scale and the Z-clip front and depth
    values.  When the molecular data is loaded from the file into MOE, the
    viewing parameters will be set to these saved values.  If off, the viewing
    parameters are not saved, and no change to the viewing parameters will be
    made when the molecule is opened into MOE from the file.

    <p>For more information on these parameters see
    <a href="../moe/fcnref/view.htm" class="svl">ViewLookAt</a>,
    <a href="../moe/fcnref/view.htm" class="svl">ViewOrientation</a>,
    <a href="../moe/fcnref/view.htm" class="svl">ViewScale</a>
    and
    <a href="../moe/fcnref/view.htm" class="svl">ViewZFront</a>,
    <a href="../moe/fcnref/view.htm" class="svl">ViewZWidth</a>.
    </p>
    </li>


    <li><i>Sequence Only.</i> If on, only residue and chain
    data are saved; i.e. atom data are not written.  If off, atoms
    are saved along with their residues and chains.</li>

    <li><i>Periodic Box.</i>  If on, the periodic box state and dimensions
    are saved.  Otherwise, the periodic box parameters are not saved.
    The dimensions of the periodic box are set using the
    <a href="./crystal.htm">Crystal Parameters</a> panel.</li>

    <li><i>Wall Restraints.</i> If on, wall restraints are saved. Wall
    restraints are defined using the Wall page of the
    <a href="../moe/pot.htm#Wall">Potential Setup</a> window.</li>

    </ul>
    </p>
    </li></p>
</ul>

<h4><a name="MDL">MDL SDfile or Molfile</a></h4>

<div class="floatright">
<img src="filesave/save_mdl.png" alt="MDL Mol Parameters"></p>
</div>

<ul>
    <p>
    <li><i>One Molecule per File (.mol)</i>.
	If on, molecules will be saved into individual MDLMOL format files,
	with filenames generated from the <b>File Prefix</b> field and
	molecule chain names.
	The files will be written to the directory
	currently displayed in the file selection box.
	<b>Note:</b> Any filename
	specified in the file selection box is ignored.

	<p>If off, molecules are saved, one per record,
	into a single SD file.  In this case, the filename specified
	in the file selection box is used, and the <b>File Prefix</b> field
	is ignored.
	</p>
    </li></p>
    <p><li><i>Use Molecule Name in Title Field</i>.
	If on, molecule chain names will be written into
	the record title field.
    </li></p>
    <p><li><i>Omit H and LP Atoms</i>.
	If on, only heavy atoms are saved. Otherwise, all atoms are saved.
    </li></p>
    <p><li><i>Depict as 2D</i>. If on, atom coordinates are rebuilt in 2D using a
    flat layout algorithm. Otherwise, current atom coordinates are saved.
    </li></p>
</ul>

<h4><a name="PDB">PDB File</a></h4>

<div class="floatright">
<img src="filesave/save_pdb.png" alt="PDB Parameters"></p>
</div>

<ul>
    <p>
    <li><i>Preserve Chain ID</i>. If on, an attempt will be made to
    preserve any chain identifier found in the chain name. If no chain 
    identifier can be found, then the first available identifier in the 
    sequence [A..Z] is used. If off, then chains are sequentially assigned
    a chain identifier in the range [A..Z]. </li>

    </p><p>

    <li><i>Write Crystal Parameters</i>.  The current crystal space 
    group and cell dimensions will be written to a CRYST1 record in 
    the output file.  This option is only available if the periodic 
    system is enabled in MOE 
    (see <a href="./crystal.htm">Crystal Parameters</a>).</li>

    </p><p>

    <li><i>Conform Naming</i>. If on, hydrogen names will be translated 
    before being written according to the chosen nomenclature convention.
    <table cellspacing="0" border="0" cellpadding="3" style="margin:0.5em;">
    
    <tr>
    <td>PDB</td> 
    <td>Refers to the conventions in place prior to the RCSB data 
    remediation project; this choice is appropriate when writing files 
    intended to be read by older, unmaintained software. 
    </td>
    </tr>
    
    <tr>
    <td><nobr>PDB v3.0</nobr></td>
    <td>Refers to the atom naming conventions adopted during the remediation 
    project by members of the Worldwide Protein Data Bank consortium 
    [Berman 2003]. 
    </td>
    </tr>
    
    <tr>
    <td>IUPAC</td>
    <td>[IUPAC 1998] is identical to PDB v3.0 
    except for the naming of the peptide hydrogen atom, which is "HN" as 
    opposed to "H".
    </td>

    <tr>
    <td>AMBER</td> 
    <td>Neutral versions of the 
    typically charged residues Asp, Glu, and Arg will be renamed. Histidine
    residues will be named according to their protonation state; 
    half-cystines will be named CYX, as will de-protonated cysteines.
    </td>
    </tr>
    </table>

    <li><i>Replace B-Factor</i>. If on, atom temperature factors will be 
    replaced with one of the following:

    <table cellspacing="0" border="0" cellpadding="3" style="margin:0.5em;">
	    <tr>
	    <td>Partial Charges</td>
	    <td>The atom's partial charge.</td>
	    </tr>
	    <tr>
	    <td>Hydrophobicity</td>
	    <td>The value of the SlogP descriptor for the atom.</td>
	    </tr>
	    <tr>
	    <td>Ideal VSA</td>
	    <td>The idealized VDW surface area for the atom,
		calculated using molecular 
		topology rather than 3D methods.</td>
	    </tr>
	    <tr>
	    <td>Force</td>
	    <td>The gradient of the potential energy of the atom, 
	    as calculated using the current force field.</td>
	    </tr>
	    <tr>
	    <td>Atom Scalar</td>
	    <td>The value stored in the atom scalar 
	    (see <a href="../moe/fcnref/atom_d.htm" class="svl">aScalar</a> 
	    and <a href="../moe/fcnref/atom_d.htm" class="svl">aSetScalar</a>).</td>
	    </tr>
	    <tr>
	    <td>Accessible Surface Area</td>
	    <td>The solvent accessible surface area for the atom, calculated using
	    a probe radius of 1.4 angstroms.</td>
	    </tr>
    </table>
    </li>

    </p><p>

    <li><i>Scale B-Factor Values</i>. If on, atom
    temperature factors will be scaled to fit in the range [0, 100].</li>

    </p><p>

    <li><i>Write TER Records for All Chains</i>. If on, chains will
    be written in the same order that they are seen in the Sequence Editor,
    and TER records will be written after each chain. Otherwise, protein
    chains will be written before non-protein chains, and only protein
    chains will be followed by TER records.
    </li>

    </p><p>
    <li><i>Separate into Files By</i>. If on, molecules will be written
    individually (<b>Molecule</b>) or separated by tag (<b>Tag</b>)
    into files.  The filenames are generated from the <b>Prefix</b> field
    and chain tag.
    The files will be written to the directory
    currently displayed in the file selection box, but any filename
    specified in the file selection box is ignored.
    </li></p>
</ul>



<h3><a name="Sequence">Sequence Format</a></h3>

<p>The Sequence formats currently available are:</p>

<ul>
	<li>Fasta sequence file (.fst)</li>
	<li>NBRF/Protein Identification Resource sequence file (.pir)</li>
	<li>Clustal sequence file (.aln)</li>
	<li>European Molecular Biology Labs sequence file (.embl)</li>
	<li>GenBank sequence file (.gb)</li>
	<li>GCG single sequence file (.gcg)</li>
	<li>Stanford/Intelligenetics (.ig)</li>
	<li>GCG Multiple Sequence File (.msf)</li>
	<li>Phylip sequence file (.phy)</li>
	<li>Formatted sequence file (.fsq)</li>
	<li>Raw sequence file (.seq)</li>
</ul>

<div class="floatright">
<img src="filesave/save_seq.png" alt="Sequence Parameters">
</div>

<p>All sequence formats define a common parameter used to determine if
the sequence alignment should be preserved.</p>

<ul>
<li><i>Reset Alignment</i>. If on, the residues become positioned according to
    cardinal numbering. Otherwise, any gaps in the alignment are recorded.
</ul>

<h3><a name="Picture">Graphic Format</a></h3>

<p>The Picture category of the Save As panel comprises all graphic file formats.
MOE can currently write (output) image files in the following formats:</p>

<ul>
	<li><a href="#PNG">Portable Network Graphics</a> (.png)</li>
	<li><a href="#JPG">JPEG</a> (.jpg)</li>
<!--
	<li><a href="#GIF">CompuServe Bitmap</a> (.gif)</li>
-->
	<li><a href="#BMP">Windows Bitmap</a> (.bmp)</li>
	<li><a href="#TIF">Tagged Image File</a> (.tif)</li>
	<li><a href="#EPS">Encapsulated PostScript</a> (.eps)</li>
	<li><a href="#RTrace">POV-Ray Scene File</a> (.pov)</li>
</ul>

<!--
<p>Additionally, <a href="#RTrace">POV-Ray</a>
(the Persistence of Vision Raytracer) can be used
to render ray-traced images of your molecular system.
POV-Ray is not provided with MOE but is freely available from 
the <a href="http://www.povray.org">POV-Ray</a> website.</p>
-->

<p>Selecting one of the image formats changes the Save panel to show options 
relevant to that format.  Four of the options are common to all formats:
<ul>

<p>
<li><i>Dimension</i>.
Specifies the image size in pixels, inches, centimeters or percent. Images of 
any size can be created; if a size larger than the current screen resolution 
is specified, it will be drawn as separate tiles and assembled into a single 
output file off-screen. Units of percent are calculated relative to the current 
MOE window size.
</li>

</p><p>

<li><i>Resolution</i>.
Specifies the image printing resolution in Dots per Inch (dpi)
or Dots per Centimeter (dpc). The image printing resolution determines the
size of the image when printed. For example, a 300 dpi image of 300x300 pixels
will occupy a 1 inch square when printed.
</li>

</p><p>

<li><i>Anti-Alias Picture</i>.
This will apply high quality anti-aliasing to the output image.
If disabled, the level of anti-aliasing specified in the 
<a href="../appendix/viewsetup.htm">Visualization Setup</a> panel will be used.
</li>

</p><p>

<li><i>High Quality</i>.
If enabled, the maximum ribbon and focal blur quality will be applied
during the image creation. Otherwise the current quality settings from the 
<a href="../appendix/viewsetup.htm">Visualization Setup</a> panel
will be used.
</li>
</p>

</ul>

<p>Other image options are specific to particular formats:</p>

<h4><a name="PNG">Portable Network Graphics</a></h4>

<div class="floatright">
<img src="filesave/save_png.png" alt="PNG Parameters">
</div>

<p>
Additional options available for the PNG format allow control of the 
compression level, color space,
the ability to have a transparent background
and use of POV-Ray to render the image.</p>

<ul>

<p>
<li><i>Color Space</i>. The selected color space affects how colors are
stored in the image file. All image formats support the <b>RGB</b> color space
which is used by most computer monitors. PNG's also support
<a href="http://www.w3.org/Graphics/Color/sRGB">sRGB</a>, a standard
internet colorspace.</li>

</p><p>

<li><i>Compression</i>. The compression level of the image can be set
between 0 (no compression) and 10 (maximum compression). Since the PNG format
uses a <i>lossless</i> algorithm, the compression level does not influence
the image quality. However, writing a PNG file using a high compression level
typically require more time compared to writing the same image with a lower
compression level.</li>

</p><p>

<li><i>Transparent Background</i>. With transparency turned-on, the image
pixels matching the color of the MOE Window background are marked as
transparent. If off, the background color is retained in the image file.

Modifying the background color of the MOE Window is accomplished using the
<a href="../appendix/viewsetup.htm">Visualization Setup</a> panel.</li>

</p><p>

<li><i>Render with POV-Ray</i>. If on, use the free ray-tracer
<a href="http://www.povray.org">POV-Ray</a> to render the current
molecular system. See the <a href="#RTrace">Using POV-Ray</a> section
below for more information (only available after installing POV-Ray).

<p><b>Note:</b> POV-Ray is not provided with MOE and must be
obtained from the POV-Ray website. 
Version 3.7 or above is required for full compatibility with MOE. 
Click
<a href="http://www.povray.org/beta/">here</a>
to be taken to the current beta download area; download and
install the appropriate version of POV-Ray for your workstation.</p>
</li>
</p>
</ul>

<h4><a name="JPG">JPEG</a></h4>

<div class="floatright">
<img src="filesave/save_jpg.png" alt="JPEG Parameters"></p>
</div>

<p>Additional options for the JPEG format allow control of the quality,
and the option to render the image with POV-Ray.</p>

<ul>

<p>

<li><i>Quality</i>. The Quality level of the image can be set from 0 (lowest)
to 100 (best). The size of the file is proportional to the quality of the
image. Setting the quality level to a low value generates smaller files and,
as expected, setting the quality level to a high value produces larger files.</li>

</p><p>

<li><i>Render with POV-Ray</i>. If on, use the free ray-tracer
<a href="http://www.povray.org">POV-Ray</a> to render the current
molecular system. See the <a href="#RTrace">Using POV-Ray</a> section
below for more information (only available after installing POV-Ray).

<p><b>Note:</b> POV-Ray is not provided with MOE and must be
obtained from the POV-Ray website. 
Version 3.7 or above is required for full compatibility with MOE. 
Click here to be taken to the
<a href="http://www.povray.org/beta/">current beta download area</a>; download and
install the appropriate version of POV-Ray for your workstation. </p>
</li>
</p>
</ul>

<!--
<h4><a name="GIF">CompuServe Bitmap</a></h4>

<div class="floatright">
<img src="filesave/save_gif.png" alt="GIF Parameters">
</div>

<p>The Save GIF Parameters provides the option to set the background transparent
or to render the the image with POV-Ray.</p>

<ul>
<p>
<li><i>Transparent Background</i>. With transparency turned-on, the image
pixels matching the color of the MOE Window background are marked as
transparent. If off, the background color is retained in the image file.

Modifying the background color of the MOE Window is accomplished using the
<a href="../appendix/viewsetup.htm">Visualization Setup</a> panel.</li>

</p><p>

<li><i>Render with POV-Ray</i>. If on, use the free ray-tracer
<a href="http://www.povray.org">POV-Ray</a> to render the current
molecular system. See the <a href="#RTrace">Using POV-Ray</a> section
below for more information (only available after installing POV-Ray).

<p><b>Note:</b> POV-Ray is not provided with MOE and must be
obtained from the POV-Ray website. 
Version 3.7 or above is required for full compatibility with MOE. 
Click here to be taken to the
<a href="http://www.povray.org/beta/">current beta download area</a>; download and
install the appropriate version of POV-Ray for your workstation. </p>
</li>
</p>
</ul>
-->

<h4><a name="BMP">Windows Bitmap</a></h4>

<div class="floatright">
<img src="filesave/save_bmp.png" alt="BMP Parameters">
</div>

<p>The Save BMP Parameters section covers all parameters related to saving
a BMP file.</p>

<ul>

<p>
<li><i>Transparent Background</i>. With transparency turned-on, the image
pixels matching the color of the MOE Window background are marked as
transparent. If off, the background color is retained in the image file. This
option is only available if the color channels are set to RGBA.

Modifying the background color of the MOE Window is accomplished using the
<a href="../appendix/viewsetup.htm">Visualization Setup</a> panel.</li>

</p><p>

<li><i>BMP Version</i>. Choose the BMP file format version to be written.
The default and most commonly supported format is version 3. Versions 4
and 5 support RGBA channels, which are needed to save images with a
transparent background, but very few programs can read BMP files with
an alpha channel (RGBA).
</li>
</p><p>

<li><i>Render with POV-Ray</i>. If on, use the free ray-tracer
<a href="http://www.povray.org">POV-Ray</a> to render the current
molecular system. See the <a href="#RTrace">Using POV-Ray</a> section
below for more information (only available after installing POV-Ray).

<p><b>Note:</b> POV-Ray is not provided with MOE and must be
obtained from the POV-Ray website. 
Version 3.7 or above is required for full compatibility with MOE. 
Click here to be taken to the
<a href="http://www.povray.org/beta/">current beta download area</a>; download and
install the appropriate version of POV-Ray for your workstation. </p>
</li>

</p><p>

<li><i>Save to Clipboard</i>. This function is only available under
Microsoft Windows. If selected, the image is saved to the clipboard,
otherwise the image is saved to the specified file.</li>
</p>
</ul>

<h4><a name="TIF">Tagged Image File</a></h4>

<div class="floatright">
<img src="filesave/save_tif.png" alt="EPS Parameters">
</div>

<p>The additional options available when saving a TIF file allow control over 
the compression method, whether the background should be transparent or not, 
and whether to use POV-Ray to render the scene.</p>

<ul>

<p>
<li><i>Compress</i>.  This controls the method of file compression used 
(this is not image compression such as in PNG or JPEG formats).  <b>None</b> 
results in no file compression, <b>RLE</b> compresses the file using a run 
length encoding while <b>ZIP</b> makes use of the same compression algorithm
as used by zip files.
</li>

</p><p>

<li><i>Transparent Background</i>. With transparency turned-on, the image
pixels matching the color of the MOE Window background are marked as
transparent. If off, the background color is retained in the image file. This
option is only available if the color space is set to RGBA.

Modifying the background color of the MOE Window is accomplished using the
<a href="../appendix/viewsetup.htm">Visualization Setup</a> panel.</li>

</p><p>

<li><i>Render with POV-Ray</i>. If on, use the free ray-tracer
<a href="http://www.povray.org">POV-Ray</a> to render the current
molecular system. See the <a href="#RTrace">Using POV-Ray</a> section
below for more information (only available after installing POV-Ray).

<p><b>Note:</b> POV-Ray is not provided with MOE and must be
obtained from the POV-Ray website. 
Version 3.7 or above is required for full compatibility with MOE. 
Click here to be taken to the
<a href="http://www.povray.org/beta/">current beta download area</a>; download and
install the appropriate version of POV-Ray for your workstation. </p>
</li>
</p>
</ul>

<h4><a name="EPS">Encapsulated PostScript</a></h4>

<div class="floatright">
<img src="filesave/save_eps.png" alt="EPS Parameters">
</div>

<p>When saving an encapsulated postscript file, the colormap can be specified.</p>

<ul>
<li><i>Colormap</i>. Selecting Color for the colormap will preserve the
colors of the image, while the Grayscale colormap will generate a black and
white (grayscale) picture.</li>
</ul>

<h4><a name="POVRayScene">POV-Ray Scene File</a></h4>

<p>Saves the POV-Ray settings required to render the current image using
POV-Ray.</p>

<p style="text-align:center"><img src="filesave/save_pov.png" alt="Ray Tracing Settings"></p>

<ul>
<p>
<li><i>Atom Theme</i>. Selects a pre-defined material style
for rendering atom nuclei and bonds. The default material is plastic.
</li>

</p><p>

<li><i>Ribbon Theme</i>. Selects a pre-defined material style
for rendering backbone ribbons. The default material is plastic.
</li>
</p>

</ul>


<h3><a name="RTrace">Using POV-Ray</a></h3>

<p>Fully ray-traced images of MOE scenes can be created using
<a href="http://www.povray.org">POV-Ray</a> (Persistence of Vision Raytracer).
POV-Ray is not included with MOE and must be obtained from the POV-Ray
website <a href="http://www.povray.org/beta/">current beta download area</a> and
installed on your workstation. Version 3.7 or above is required for 
full compatibility with MOE.</p>

<p>Images can be rendered using POV-Ray directly from MOE by checking the 
<b>Render with POV-Ray</b> box in the image format's options.  Additionally, 
a <i>POV-Ray scene file</i> may be saved by choosing that format.</li> 

<p>To open the Ray Tracing Settings panel, click the <b>POV-Ray Settings</b>
button in any of the image formats listed above.</p>

<p style="text-align:center"><img src="filesave/rtset.png" alt="Ray Tracing Settings"></p>

<p>Within the Ray Tracing Settings panel you can specify the path to your
POV-Ray executable. Additionally, this panel can be used to customize many
ray tracing related options</p>

<ul>
<p>
<li><i>POV-Ray Path</i>. Specifies the location of the POV-Ray executable
on your workstation.</li>

</p><p>

<li><i>Hide Preview Window</i>. If enabled, POV-Ray will show the image's progress 
during the render process.  If disabled, no preview will be shown.</li>

</p><p>

<li><i>Anti-Aliasing</i>.  Anti-aliasing is used to eliminate ray-tracing
sampling errors or at least reduce the negative impact they have on the image
(rough edges).
The following anti-aliasing method are available:

<ul>
    <li><tt>None</tt>. No anti-aliasing is performed.
    <p>
    <li><tt>Adaptive</tt>. 
    Pixels differing from their neighbors by more than a given 
    threshold are super-sampled to construct a pixel representing the
    appropriate color based on the color of the surrounding pixels.
    <p>
    <li><tt>Recursive</tt>. Each pixel is divided into quarters which are
    subsequently divided into quarters again in a recursive manner until each
    new pixel is a uniformed color that is defined by the assigned threshold
    value.  Once the colors for all the sub-pixels are determined the average
    color for the entire pixel is determined. 
</ul>
<p>
The anti-aliasing <b>Threshold</b> is the maximum difference between two
neighboring pixels allowing the pixels to be considered the same color and
thus not triggering the anti-aliasing procedure. The color of each pixel
is defined as the components of rgb, for example r1,g1,b1 and r2,g2,b2
for pixel1 and pixel2, respectively; allowing their difference to be
calculated as follows: </p>
<p style="text-align:center"><tt>diff = abs(r1-r2) + abs(g1-g2) + abs(b1-b2)</tt></p>
<p>
The value of each rgb component ranges between 0.0 and 1.0; therefore the
maximum deviation of two pixels is 3.0. In case the anti-aliasing threshold is
set to 0.0 every pixel is super-sampled, and conversely, when threshold is
set to 3.0 no anti-aliasing is performed.</p>
<p>
Depending on the anti-aliasing mode selected, the anti-aliasing <b>Depth</b>
specifies how the pixels are sampled. In <tt>Adaptive</tt> mode, the
antialiasing depth assigns the matrix size of a super-sampled pixel. For
example, in <tt>Adaptive</tt> mode a <b>Depth</b> value of 4
would give 4 * 4 = 16 samples per pixel. In <tt>Recursive</tt> mode,
the anti-aliasing <b>Depth</b> specifies the maximum number of subdivisions
per pixel.</p>

</p><p>

<li><i>Atom Theme</i>. Selects a pre-defined material style
for rendering atom nuclei and bonds. The default material is plastic.
</li>

</p><p>

<li><i>Ribbon Theme</i>. Selects a pre-defined material style
for rendering backbone ribbons. The default material is plastic.
</li>
</p>
</ul>

<h2><a name="SVL">SVL Commands</a></h2>

<a href="../moe/fcnref/molrw.htm">Write Molecular Data Functions</a><br />
<a href="../moe/fcnref/draw.htm">Display Attribute Functions</a><br />
<a href="../moe/fcnref/quality.htm">Rendering Quality Functions</a><br />
<a href="../moe/fcnref/atom_d.htm">Atom Display Properties Functions</a><br />

<h2>See Also</h2>

<a href="../moe/gui/fileopen.htm">MOE Open System</a><br />
<a href="../appendix/viewsetup.htm">Visualization Setup</a><br />
<a href="../moe/gui/mainwin.htm#Copy">Copy and Paste</a>

<noscript>
	<hr noshade="noshade" />
	<font size="2"> Copyright &copy; 1997&ndash;2015
	<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a> </font>
</noscript>
<div id="MoeFooter"></div>
</body>
</html>
