<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!	prodesign.htm	Protein Design 
!!-->

<!--
!!    MOE Online Manuals
!!    COPYRIGHT (C) 1997-2015
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->


<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

<script type="text/javascript" 
src="../include/jsincludes_moe.js"></script>

<link rel="stylesheet" type="text/css"
href="../include/manstyle.css" />


<title>Protein Design</title>

<meta panel Protein Design>
<meta main_keywords protein design, scan, alanine, residue sequence />
<meta main_keywords disulfide, resistance sequence />


<style type="text/css">
td { vertical-align: top; }
</style>

<!-- *** These are for the old equations, that are now done with MathJax. ***
<style type="text/css">
.subsup_f { line-height: 1.8; }
.subsup_f sub { margin-right: 1ex; }
.subsup_f sup { margin-left: -1ex; }
</style>
-->
</head>

<body>
<div id="useMathJax"></div>
<div id="MoeHeader"></div>
<noscript>
	<span class="warning">Warning: JavaScript is disabled. This page will not display correctly.</span>
	<h1 class="title">MOE Documentation</h1>
	<hr noshade="noshade" />
	<style>.LaTeX {color: #C08080;}</style>
</noscript>

<div id="toc-list"><script>var gman_OrderedListTOC=false; </script></div>

<!--
<ol>
<li><a href="#Introduction">Introduction</a></li>
<li><a href="#Methodology">Methodology Overview</a>
    <ol>
    <li><a href="#Searching">Searching Conformational and Sequence Space</a></li>
    <li><a href="#Refinement">Refining Mutants Generated by UQO</a></li>
    <li><a href="#Scoring">Scoring Changes in Protein Stability</a></li>
    <li><a href="#Ensemble">Generation of Conformational Ensembles</a></li>
    <li><a href="#Variants">Protein Design Variants</a>
		<ol>
		<li><a href="#AlanineScanMethod">Alanine Scan Methodology</a></li>
		<li><a href="#DisulfideScanMethod">Disulfide Scan Methodology</a></li>
		<li><a href="#ResistanceScanMethod">Resistance Scan Methodology</a></li>
		<li><a href="#ResidueScanMethod">Residue Scan Methodology</a></li>
		<li><a href="#SampleSequenceMethod">Sample Sequence Methodology</a></li>
		<li><a href="#SequenceDesignMethod">Sequence Design Methodology</a></li>
		</ol></li>
    </ol></li>
<li><a href="#Run">Running the Protein Design Application</a>
    <ol>
    <li><a href="#Preparing_the_Data">Preparing the Data</a></li>
    <li><a href="#Using_the_Panel">Using the Protein Design Panel</a>
		<ol>
		<li><a href="#MutationEditor">Using the Mutation List Editor</a>
			<ol>
			<li><a href="#ConstantMutation">Defining a Constant Mutation</a></li>
			<li><a href="#MultipleMutations">Defining Multiple Mutations</a></li>
			<li><a href="#NonNaturalAA">Mutating to/from Non-Natural Amino Acids</a></li>
			</ol></li>
		<li><a href="#panel_variants">Using Protein Design Variants</a>
			<ol>
			<li><a href="#AlanineScan">Alanine Scan</a></li>
			<li><a href="#DisulfideScan">Disulfide Scan</a></li>
			<li><a href="#ResistanceScan">Resistance Scan</a></li>
			<li><a href="#ResidueScan">Residue Scan</a></li>
			<li><a href="#SampleSequence">Sample Sequence</a></li>
			<li><a href="#SequenceDesign">Sequence Design</a></li>
			</ol></li>
		<li><a href="#panel_input">Input/Output Options</a></li>
		<li><a href="#display_opt">Display Options</a></li>
		<li><a href="#ensemble_opt">Ensemble Options</a></li>
		<li><a href="#affinity_opt">Affinity Options</a></li>
		<li><a href="#properties_opt">Properties Options</a></li>
		</ol></li>
	<li><a href="#Output">Working with Output</a>
		<ol>
		<li><a href="#Output_Database">The Output Database</a></li>
		<li><a href="#Browsing_the_Output_Database">Browsing the Output Database</a></li>
		</ol></li>
	<li><a href="#Using_MOEsmp">Running Protein Design with MOE/cluster (MOE/smp)</a></li>
	<li><a href="#Continuing">Continuing an Incomplete Protein Design Job</a></li>
	<li><a href="#Restarting">Restarting from a Protein Design Output File</a></li>
	</ol></li>
<li><a class="example" href="#Examples">Examples</a>
	<ol>
    <li><a class="example" href="#Example1_Data_Preparation">
	    Example 1: Data Preparation for Protein Design</a></li>
    <li><a class="example" href="#Example2_DisulfideScan">
	    Example 2: Disulfide Scan on 2ST1</a></li>
    <li><a class="example" href="#Example3_ResidueScan">
	    Example 3: Residue Scan on 1VFB</a></li>
	</ol></li>
<li><a href="#References">References</a></li>
<li><a href="#See_Also">See Also</a></li>
</ol>
-->

<a name="Introduction"></a>
<h2>Introduction</h2>

<p>Biologics are human or animal protein derived compounds.  Such
substances play an ever-increasing role in everyday life, from laundry
to therapeutics,
and their great promise has given rise to an
urgent need for efficient methods for engineering
new biomolecules.
Computational techniques can assist in suggesting modifications
to existing structures for obtaining desirable properties and functions.
In MOE, the Protein Design application permits unlimited
simultaneous residue mutations of target structures, and
then calculates the properties of the resultant mutant, with
the purpose of examining how mutations of a protein
modulate its protein properties.
</p>

<a name="Methodology"></a>
<a name="Methodology_Overview"></a>
<h2>Methodology Overview</h2>

<p>
Computational approaches to protein design face a difficult problem,
as illustrated in the figure below.  Given a protein structure, the
generation of an arbitrary mutant in principle requires taking into
consideration all possible residue mutations and all rotamers of each
mutation state.  A selection process must then choose a single state
for each residue, which ideally should coincide with
the minimum global energy structure as a whole.
</p>

<p align="center">
<a href="prodesign/size_of_problem.png">
<img src="prodesign/size_of_problem_thumb.png" alt="Protein Design Problem" />
</a>
</p>

<br clear="all">

<p>
<div class="floatleft">
<a href="prodesign/method.png">
<img src="prodesign/method_thumb.png" alt="Protein Design Method"
style="margin: 0px 15px 0px 0px" />
</a>
</div>

<p>
The Protein Design methodology reduces the size of the problem
by partitioning the system and then pruning unpropitious states.
To make the search for optimal solutions computationally tractable,
mutation/rotamer states are encoded
into an energy matrix that lends itself to solution
by Unary Quadratic Optimization (UQO) [Labute&nbsp;2008].
The Protein Design algorithm is as follows:
</p>

<ol>
<p>
<li><b>Partition the System.</b>
The atoms of the system are divided into two groups: <i>sidechains</i> and
<i>environment</i>.  The sidechain atoms are collected after the specified
residue mutations have been made.
All other atoms, including backbone atoms, are considered part
of the environment.
</li></p><p>

<li><b>Read mutation and rotamer states.</b> The desired mutations and
associated rotamers are obtained from the rotamer library (see
<span class="nobr"><span class="menu">MOE | Settings |
Protein Defaults</span>).</span>
For example, for histidine, four tautomer and protomer forms
are read out.
The rotamer library also contains the self energy and strain energy for each
rotamer of the desired mutation(s).
The backbone atoms of the rotamers are
superposed onto the backbone atoms of the residues to be mutated.
</li></p><p>

<li><b>Matrix Setup.</b>  The interaction energies between
each pair of mutation and rotamer states are calculated and
stored in a matrix, U. High energy states are removed during
construction of the
matrix, garnering savings in both computational and memory requirements.
</li></p><p>

<li><b>Unary Quadratic Optimization (UQO)</b>.
First, Dead-End Elimination [Goldstein&nbsp;1994] is used
to reduce the complexity of the optimization
procedure in subsequent steps.  This step is followed by a search for the
particular configuration of states (mutation and/or rotamer) that minimizes
a quadratic energy function, which is described in more detail 
<a href="#binvec">below</a>.
</li></p><p>

<li><b>Refinement of Mutations</b>.  A final energy minimization of the UQO
solution is performed to relax the structure.  The refinement protocol
includes a severe clash resolution stage, if necessary.
</li></p><p>

<li><b>Stability/Affinity Scoring</b>.  
A final scoring for stability and affinity is performed on the 
refined structure. Additionally some common protein
property descriptors (see <a href="#Output">Working with Output</a>)
are automatically calculated. The units of affinity and stability are kcal/mol.
</li></p><p>

<li><b>Ensemble Generation</b>.
the Protein Design application can optionally
generate an ensemble of protein conformations using any of the
<a href="../proteins/probuild.htm#RotamerExplorer">Rotamer Explorer</a>,
<a href="confsrch.htm#LowModeMD">LowMode&nbsp;MD</a> or
<a href="md.htm">Molecular Dynamics</a>
methods.
</li></p><p>

<li><b>Ensemble Refinement</b>.  If an ensemble
is being generated,
the conformations generated by the ensemble
method are refined using the same method as in Step 5,
&quot;Refinement of Mutations&quot;.
The ensemble is then pruned to remove similar and high energy 
conformations.
</li></p><p>

<li><b>Ensemble Scoring</b>.  If an ensemble is being
generated, each conformation can be scored for stability,
affinity, and protein properties. 
The values are averaged using a Boltzmann distribution.
</li></p>

</ol>

<br clear="all">

<a name="Searching"></a>
<h3>Searching Conformational and Sequence Space</h3>
<div class="floatright">
<a href="prodesign/prodes_partition.png">
<img src="prodesign/prodes_partition_thumb.png"
 alt="Protein Design Partition" />
</a>
</div>

<p>
In the first step of the Protein Design algorithm, the system is
partitioned by implicitly cutting the bond joining
the <span class='LaTeX'>\( \alpha \)</span> and 
<span class='LaTeX'>\( \beta \)</span> carbons in the residues to be mutated.
The result of the partitioning is a collection of non-intersecting atom sets
(sidechains)
each associated with a particular residue. Each such atom set will have
an associated collection of one or more mutation and rotamer states
which have been read in from the rotamer library (see
<span class="nobr"><span class="menu">MOE | Settings
| Protein Defaults</span>).</span>
Rotamer and mutation backbone atoms are superposed onto the
backbone atoms of the residue to be mutated. Since the rotamers in the library
do not include any backbone atoms,
the strain energy due to a backbone carbonyl oxygen is
added to the strain energy of the sidechain.
A <b>Repack Environment</b> option allows nearby residues
to be included in the conformational search.
</p>

<a name="binvec"></a>
<p>
A binary vector <i>x</i> is used to encode a particular selection
of mutation/conformational states of the residues to be
mutated <i>i</i>, <i>j</i>,
<i>k</i>,&nbsp;&hellip;.
The bits of <i>x</i> are arranged in blocks, where each block represents a
residue, and each element of the block represents one of the possible
mutation/rotamer configurations of that residue.
Consider the image at right in which residue <i>i</i>, an alanine,
is to be mutated into one of three possible residues,
with one conformation (green)
for the first mutation, and two for each of the second (blue)
and third (purple), for a total of five possible
mutation/rotamer configurations.
These states would correspond to a block of 5 bits in the
binary vector.
Similarly, residue <i>j</i> is depicted as having two possible mutations
with one conformation each, for a total of two bits in the binary
vector, and residue <i>k</i> has 3 mutations and is represented
by 4 bits in the binary vector.
The vector <i>x</i> with the following sequences of bits:
</p>

<p align="center">
<span class="nobr"><i>x</i> = [ 0 1 0 0 0 | 1 0 | 0 0 1 0 | ... ]</span>
</p>

<p>
would then specify the configuration of state 2 for <i>i</i>,
state 1 for <i>j</i> and state 3 for 
<i>k</i>. In this representation, a value of 1 in <i>x</i>
marks the presence of a particular state, and 0, its absence.
Admissible values of <i>x</i> are those in which there is exactly one
bit of value 1 in each block (all the rest are 0's).
This constraint on the values of <i>x</i> is 
called a <i>unary constraint</i> since the individual states of the
mutation are encoded in base 1 or unary notation. 
</p>

<p>
The energy of a configuration is determined from a quadratic function:
</p>
<!-- Old HTM
<p align="center">
<i>E</i>(<i>x</i>) = 1 &frasl; 2&middot;<i>x</i>&middot;<i>U</i><i>x</i> +
<i>u</i>&middot;<i>x</i> + <i>u</i><sub>0</sub> 
</p>
-->

<p align="center"><span class='LaTeX'>
\[
E(x) = \frac{1}{2} \cdot x \cdot Ux + u \cdot x +u_0
\]
</span></p>


<p>
which is taken to be a multidimensional quadratic.
The function is determined by an interaction matrix <i>U</i>
and a &quot;self&quot; energy vector <i>u</i>.
</p>

<p align="center">
<a href="prodesign/prodes_matrix.png">
<img src="prodesign/prodes_matrix_thumb.png" alt="Protein Design Partition" />
</a>
</p>

<p>
The matrix contains the interaction energy between each pair of
residue states. Within a block, the matrix contains 0's and 1's,
since exactly one state can be chosen within each block. 
The vector <i>u</i> contains interactions
with the rest of the system, for instance, backbone atoms. For a given vector, 
subject to the unary constraint, the function E returns the free energy
of the system.  The Protein Design application uses van der Waals,
hydrogen bond, pKa and a disulfide bond terms to populate the matrix.
The functional form for an individual conformer would therefore be:
</p>
<!-- Old htm
<p align="center">
S = <i>E<sub>self</sub></i> + <i>E<sub>vdw</sub></i> + <i>E<sub>hb</sub></i> +
&Delta;<i>E<sub>pH-pK<sub>a</sub></sub></i> + <i>E<sub>SS</sub></i>
</p>
-->

<p align="center"><span class='LaTeX'>
\[
S = E_{self} + E_{vdW} + E_{hb} + \Delta E_{pH-pK_a} + E_{SS}
\]
</span></p>

<p align="left">
where
</p>

<p>
<blockquote>
<table border="0" cellpadding="5" width="85%">
<tr>
<!-- <td valign="top"><i>E<sub>self</sub></i></td>-->
<td valign="top"><span class='LaTeX'>\(E_{self}\)</span></td>
<td valign="top">The self energy of the specific
conformation of the mutation. <span class='LaTeX'>\(E_{self}\)</span> is comprised of the
AMBER angle, torsion and van der Waals terms.
</td>
</tr>

<tr>
<!--<td valign="top"><i>E<sub>vdw</sub></i></td>-->
<td valign="top"><span class='LaTeX'>\(E_{vdW}\)</span></td>
<td valign="top">
The AMBER van der Waals interaction energy from the
matrix <i>u</i>.
</td>
</tr>

<tr>
<!--<td valign="top"><i>E<sub>hb</sub></i></td>-->
<td valign="top"><span class='LaTeX'>\(E_{hb}\)</span></td>

<td valign="top"> A dipole-based hydrogen bond term
from the matrix <i>u</i>. A component of this term is a hydrogen bond radial
12-10 distribution that is consistent with contact statistics derived from the PDB
and dipole interaction based energy. The 12-10 distribution has the following
formalism:

<p align="center"><span class='LaTeX'>
\[
P = \exp \left[ - \frac{1}{kT} \left(
5 \left[ \frac {2.85}{r_{ij}} \right] ^{12} -
6 \left[ \frac {2.85}{r_{ij}} \right] ^{10}
\right) \right]
\]
</span></p>

<p>
where <span class='LaTeX'>\( r_{ij} \)</span> is the distance between the polar heavy atoms in the
sidechain and the environment.

<p>
Each polar hydrogen-bearing heavy atom is given a point dipole,
with a direction consistent with its hybridization, and with
a magnitude consistent with the TIP5P water model.
Dipole-dipole electrostatic interactions are evaluated for each such pair according to:

<p align="center"><span class='LaTeX'>
\[
E_{dip-dip} = \frac {1}{4 \pi \epsilon _0 } \left(
d_i \cdot d_j - 3 (d_i \cdot r_{ij})(r_{ij} \cdot d_i)
\right)
\]
</span></p>

<p>
where <span class='LaTeX'>\( d_i \)</span> and <span class='LaTeX'>\( d_j \)</span> are the dipole vectors of the heavy
atoms and <span class='LaTeX'>\( r _{ij} \)</span> is the vector between the atom positions.
<p>
The total hydrogen bond interaction energy <span class='LaTeX'>\( E_{hb} \)</span>,
which takes directional effects into account, is the product of the
radial term <span class='LaTeX'>\( P \)</span> and the dipole-dipole energy term <span class='LaTeX'>\( E_{dip-dip} \)</span>.


</td>
</tr>

<tr>
<!--<td valign="top">&Delta;<i>E<sub>pH-pK<sub>a</sub></sub></i></td>-->
<td valign="top"><span class='LaTeX'>\( \Delta E_{pH-pK_a}\)</span></td>
<td valign="top">A term
used to account for <span class='LaTeX'>\( pK_{a} \)</span> shifts due to conformational and environmental
effects. This allows for prediction of the correct protonation state 
for a given conformation. <span class='LaTeX'>\( \Delta E_{pH-pK_a}\)</span> is
given by:

<p align="center"><span class='LaTeX'>
<!--
&Delta;<i>E<sub>pH-pK<sub>a</sub></sub></i> = (kT log 10)(pH - pK<sub>a</sub>)
-->
\[
\Delta E_{pH-pK_a} = (kT \log 10)(pH - pK_a)
\]
</span></p>

<p>
where <span class='LaTeX'>\( pK_{a} \)</span> for a particular mutation and its conformation is calculated using
<a href="residuepka.htm">PROPKA</a>.
</td>
</tr>

<tr>
<!--<td valign="top"><i>E<sub>SS</sub></i></td>-->
<td valign="top"><span class='LaTeX'>\(E_{SS}\)</span></td>
<td valign="top">A geometry-based energy term for disulfide bonds.
In the creation of the energy matrix <i>U</i>, whenever two cysteine 
sulfurs are within a specified distance cutoff,
they are treated as if they were forming
a disulfide bond. <span class='LaTeX'>\(E_{SS}\)</span> is equal to 0 when the geometry is
distant from that of the ideal disulfide bond.
The definition of an ideal disulfide bond in the Protein Design application
is:
<ul>
<li>The distance of between two cysteine sulfurs atoms is in the range 
1.99&Aring;&ndash;2.44&Aring;.
<li>The angles between the &beta; carbons and both sulfurs are within
45&deg; of the ideal angle of 103.7&deg;. 
</ul>
<p>
When matching these geometry constraints, <span class='LaTeX'>\(E_{SS}\)</span>
is equal to:
  
<p align="center"><span class='LaTeX'>
<!--
<i>E<sub>SS</sub></i> = -300 + abs (2.04-<i>d<sub>SS</sub></i>)
+ max abs (103.7 - &theta;<sub>C<sub>&beta;i</sub>
S<sub>i</sub>S<sub>j</sub></sub>)
-->
\[
E_{SS} = -300 + \text{abs } (2.04-d_{SS})
+ \max \text {abs } (103.7 - \theta _{C_{\beta_i}S_iS_j} )
\]
</span></p>

</td>
</tr>

</table>
</center>
</blockquote>
</p>

<p>
Following the creation of the energy matrix <i>U</i>,
the protein design problem can
now be reduced to finding the binary vector <i>x</i>,
subject to the unary constraint, that minimizes
<span class="nobr"><i>E</i>(<i>x</i>).</span>
This is done with a Unary Quadratic Optimization algorithm. For more
information on the Unary Quadratic Optimization itself see the
<a href="../apps/protonate3d.htm#Method">Protonate3D Methodology</a> section
which describes the optimization technique in detail.
</p>

<a name="Refinement"></a>
<h3>Refining Mutations Generated by UQO</h3>
<p>
After producing a mutant using the above methodology, it is beneficial to
perform an energy minimization on the mutated structure to relax it
prior to scoring.
If the structure has severe clashes, then a special refinement
protocol is used prior to a more conservative protocol.
</p>

<p>
The conservative refinement protocol allows atoms in the mutated residue
to be free to move,
tethers neighboring atoms within 8 &Aring; with a weight of
10 kcal/&Aring; and
a deviation of 0.25 &Aring;, and fixes all other atoms.
The severe clash refinement protocol involves applying a tether
of 300 kcal/&Aring;
and a deviation of 0.25 &Aring; to all atoms, while disabling
electrostatics. Multiple energy minimizations are then performed
whilst increasing the van der Waals scaling factor from 0.1 to 1.
</p>

<a name="Scoring"></a>
<h3>Scoring Protein Stability</h3>

<div class="floatleft">
<a href="prodesign/stability_prediction.png">
<img src="prodesign/stability_prediction_thumb.png"
alt="Protein Design Prediction of Stability" />
</a>
</div>

<p>
Accurate modeling of the impact of a mutation on a protein must
capture both the conformation change upon mutation as well as
the change in stability from wild type to mutant protein.
The change in stability due to mutation is measured experimentally as the
difference in free energy of folding of the mutant (Mut), and wild type (WT).
Here, we predict this change from the difference in stability (<i>s</i>)
between the mutant and wild type in the folded (<i>f</i>)
and unfolded (<i>u</i>) states of the protein:
</p>

<!--
<p align="center">
&Delta;&Delta;<i>G<sub>s</sub></i>
 = &Delta;<i>G<span class="subsup_f"><sub>f</sub><sup>Mut</sup></i></span>
 - &Delta;<i>G<span class="subsup_f"><sub>f</sub><sup>WT</sup></i></span>
 = &Delta;<i>G<span class="subsup_f"><sub>s</sub><sup>WT&rarr;Mut</sup></i></span>
 - &Delta;<i>G<span class="subsup_f"><sub>s</sub><sup>WT&rarr;Mut</sup></i></span>
<p align="center">
-->

<p align="center"><span class='LaTeX'>
\[
\Delta \Delta G_s
= \Delta G _f ^{Mut} - \Delta G _f ^{WT}
= \Delta G _{s_f} ^{WT \to Mut} - \Delta G _{s_u} ^{WT \to Mut}
\]
</span><p>
Using this formalism, we can create a model based on Linear Interaction
Energy (LIE). According to LIE, <span class='LaTeX'>\(\Delta G\)</span> can be expressed as the
change in the residue environment interaction energy going from
wild-type to mutant, where the interaction energy is the change 
between the energy of the protein and the energy of the protein without
the residue of interest. LIE is usually expressed as:
</p>

<!--
<p align="center">
&Delta;<i>G</i> = &Delta;<i>E<sub>vdw</sub></i> +
&Delta;<i>E<sub>ele</sub></i> + <i>T</i>&Delta;<i>S</i>
</p>
-->

<p align="center"><span class='LaTeX'>
\[
\Delta G = \Delta E_{vdW} + \Delta E_{ele} + T \Delta S
\]

</span><p>
In the case of the folded state, this is
calculated using the following equation, applying the ideal theoretical LIE
values of 1 for <span class='LaTeX'>\( \Delta E_{vdW} \)</span> and
0.5 for <span class='LaTeX'>\( \Delta E _{ele}\)</span>.
</p>

<!--
<p align="center">
&Delta;<i>G<span class="subsup_f"><sub>s</sub><sup>WT&rarr;Mut</sup></span></i>
 = <i>&alpha;</i>[&Delta;<i>E<sub>vdw</sub></i>
 + 0.5(&Delta;<i>E<sub>coul</sub></i> + &Delta;<i>E<sub>sol</sub></i>)
 ] + &beta;&Delta;<i>E<sub>SS</sub></i>
 + &gamma;&middot;&Delta;<i>SA<sub>sc</sub></i>
</p>
-->
<span class='LaTeX'>
\[
\Delta G _{s_f} ^{WT \to Mut} = \alpha 
[ \Delta E _{vdW} + 0.5 (\Delta E_{coul} + \Delta E_{sol})] + \beta E_{SS}
\]
</span>
<p>
where  <span class='LaTeX'>\( \alpha \)</span> is a scaling factor accounting for configurational
entropy effects, <span class='LaTeX'>\( \Delta E_{vdW} \)</span>
is the AMBER van der Waals interaction
energy, <span class='LaTeX'>\( \Delta E_{coul} \)</span> is the AMBER Coulomb interaction
energy,  <span class='LaTeX'>\( \Delta E_{sol} \)</span> is the change in solvation energy
calculated using
<a href="../moe/pot.htm#Equation">GBVI</a>,
and  <span class='LaTeX'>\( \Delta E_{SS} \)</span> is the change in energy due to the presence
of a disulfide bond. This value is calculated using the same formalism
as described in the 
<a href="#Searching">Sequence and Conformational Search</a> section above.
</p>

<p>
Predicting the change in stability of the unfolded states proves to be more
difficult since there are typically no structures available of
this state. It may be possible to use a computational method to predict
the unfolded state but this would require a significant investment in time,
both real and computational. In addition, the unfolded state may, in fact,
comprise multiple conformations, making this prediction even more
difficult.
</p>

<p>
Using the structure of the unfolded step can be side-stepped if it is
assumed that the stability change of the unfolded state is in fact a constant.
This assumption can be made on the basis that this unfolded state is
unrelated to the sequence of the protein and its environment. In other words,
the stability is only dependent on the change of wild type residue
to mutant. Therefore the change in stability in the unfolded state was broken down
into 3 key terms:
</p>
<span class='LaTeX'>
\[
\Delta G _{s_u} ^{WT \to Mut} = 
\gamma \Delta \Delta G_{sol} + \epsilon \Delta \delta _{gly} +
\zeta \Delta \delta _{His}
\]
</span>
<p>
In this equation, <span class='LaTeX'>\( \Delta \Delta G_{sol} \)</span> is the change in the unfolded
state free energy of solvation. This term is a topological solvation term
derived from extend Hückel theory. <span class='LaTeX'>\( \Delta \delta _{gly} \)</span> is the change
in the number of Glycines, and <span class='LaTeX'>\( \Delta \delta _{cyc} \)</span> is the change in the
number of cyclic amino acids (proline). These terms capture the change in
backbone entropy upon mutation. Lastly, <span class='LaTeX'>\( \Delta \delta _{His} \)</span> 
is the change in the number of histidines. Since histidine is a special
amino acid with multiple protonation states existing at biological pH,
this term captures the cost associated with change the protomer state
of histidine.
 
</p>

<a name="stability_scoring_function"></a>
<p>
The stability scoring function therefore has the following functional
form:
</p>

<!--
<p align="center">
&Delta;&Delta;<i>G<sub>s</sub></i>
 = &alpha;[&Delta;<i>E<sub>vdw</sub></i>
 + 0.5(&Delta;<i>E<sub>coul</sub></i> + &Delta;<i>E<sub>sol</sub></i>)
 ] + &beta;&Delta;<i>E<sub>SS</sub></i>
 + &gamma;&middot;&Delta;<i>SA<sub>sc</sub></i>
 + &Delta;<i>G<span class="subsup_f">
 <sub>uf</sub><sup>WT&rarr;Mut</sup></span></i>
</p>
-->
<span class='LaTeX'>
\[
\Delta G _{s} =
\alpha [ \Delta E _f ^{vdW} + 0.5 (\Delta E _f ^{coul} + \Delta E _f ^{sol})]
+ \beta E_{SS}
+ \gamma \Delta \Delta G _u ^{sol}
+ \epsilon \Delta \delta _{gly}
+ \zeta \Delta \delta _{His}

\]
</span>


<p>
In the Protein Design application,
the stability scoring function has been trained on over 3000 single point
mutations taken from the FoldX [Guerois&nbsp;2002]
and PoPMuSiC-2.0 [Dehouck&nbsp;2009] datasets.
</p>


<a name="Ensemble"></a>
<h3>Generation of Conformational Ensembles</h3>
<p>
The ensemble options is used to generate
an ensemble of protein conformations and to
calculate average stability, affinity
and protein properties using a Boltzmann distribution. There are three
different methods for generating an ensemble:
Rotamer Explorer, LowMode and Dynamics.
</p>

<p>
The <a href="../proteins/probuild.htm#RotamerExplorer">Rotamer Explorer</a>
uses a similar search method as that used for Protein Design: the 
mutation along with residues within a cutoff are
partitioned into sidechains atoms and an environment atom set. The
conformations for residues in the sidechain set are read in from the
rotamer library. A matrix is built and searched using UQO.
This is followed by Monte Carlo search performed on the matrix around
an optimal
solution. The search finishes when the number of conformations found
reaches twice the desired ensemble size.
</p>

<p>
The LowMode ensemble generation method uses
<a href="confsrch.htm#LowModeMD">LowMode MD</a> to search the
conformational space of the mutant. It should be used when a mutation
may cause larger changes in the backbone conformation. 

All atoms of the mutated residue are free to move, while residues within
the "Fix Cutoff" are tethered. To speed up the simulation, atoms that are
farther than the <a href="../moe/pot.htm#Forcefield">non-bonded cutoff</a>
+ 5&Aring; are marked as inert. The number of iterations is a parameter
that can be modified.
Each LowMode interation minimizes the current conformation
to a root mean square gradient of 1 to enable the search to escape
small local energy wells.
</p>

<p>
The Molecular Dynamics ensemble generation method uses
<a href="md.htm">Molecular Dynamics</a> to create a conformational
ensemble. It is recommended to use the explicit solvent option under this
ensemble method. The Molecular
Dynamics protocol uses the same tethering
and fixed scheme as LowMode MD. A 1ps
heating phase is performed followed by another 0.5ps heating
phase in which the heavy atom restraints
are gradually reduced. This is followed by a
production phase whose length is a parameter that can be specified
(the default 50ps). All phases use a time step of 2ps. A conformation
is saved every 5ps.
</p>

<p>
After an ensemble is generated, conformations within
an RMSD threshold of each other (i.e. similar conformations) are removed.
The remaining conformations are then scored and
sorted using the
<a href="#stability_scoring_function">stability scoring function</a>.
Conformations outside of the
energy window of the lowest energy conformation are removed.
Additionally, the best <i>N</i> conformations, where <i>N</i> is
a settable parameter, are kept.
The affinity and protein properties of these conformation can
optionally be calculated.
All properties and scores are then averaged using a Boltzmann distribution.
</p>

<a name="Variants"></a>
<h3>Protein Design Variants</h3>

<p>
The Protein Design application has 6 variants, each tailored
to solving different
yet similar problems. In all cases, the goal of the experiment is to modulate
some physical protein properties, such as stability or affinity, with each
variant
restricting the search in a different way.
In each variant, the desired mutations are specified in a mutation list
which can be applied either to chains currently loaded in the system
or to a database containing a conformational ensemble.
The following sections explain the
distinct aspects of each of the Protein Design variants.

<a name="AlanineScanMethod"></a>
<h4>Alanine Scan Methodology</h4>
<p>
Alanine Scanning is a technique used to determine
the importance of a residue to the stability, affinity or property of
a given protein. Alanine is used because its
sidechain is non-bulky and chemically
inert while retaining secondary structure preferences
similar to most amino acids.
</p>

<p>
The Alanine Scan variant of Protein Design sequentially
mutates all residues, specified in the mutation list, to alanine.
</p>

<div class="floatright">
<a href="prodesign/cys_distance.png">
<img src="prodesign/cys_distance_thumb.png"
alt="Protein Design Prediction of Stability - Unfolded State" />
</a>
</div>

<a name="DisulfideScanMethod"></a>
<h4>Disulfide Scan Methodology</h4>
<p>
Disulfide bonds play an important role in the folding and stability of 
proteins. An easy way to increase the stability of a protein
is to add a new disulfide bond.

<p>
The Disulfide Scan variant of the Protein Design application analyzes the
residues specified in the mutation list and determines whether there is
another residue close enough to create a disulfide bond. Two residues are
said to be close enough to form a disulfide bond if their &beta;
carbons are within 5&Aring; of each other. With such a pair identified, the 
Disulfide Scan mutates both residues to Cysteine using the
<a href="#Searching">Sequence and Conformational Search</a> methodology
described above.
</p>

<br clear="all">

<div class="floatleft">
<a href="prodesign/codon_table.png">
<img src="prodesign/codon_table_thumb.png"
alt="Codon Table" />
</a>
</div>

<a name="ResistanceScanMethod"></a>
<h4>Resistance Scan Methodology</h4>
<p>
Resistance Scanning is a technique used to identify single point mutations,
typically in viral and bacterial proteins, which
cause a loss of affinity toward a ligand.  This decreased affinity is also 
known as <i>resistance</i>.
These single point mutations represent
a low genetic barrier to resistance and cause
the target to become resistant quicker.
<!-- DOESN'T MAKE SENSE.
The goal of this technique is to identify ligands which exhibit this profile.
-->
</p>

<p>
In Resistance Scanning, mutations are limited to
single nucleotide polymorphisms (SNP) of the
wild type sequence. A SNP mutation is where there is a single nucleotide
change in the codon encoding an amino acid, inducing a change in the protein
sequence.
For example, if the wild-type sequence is Ala, the possible mutations for this
residue are Asp, Glu, Gly, Pro, Ser, Thr, and Val. The mutations are limited to
SNPs to emulate the mutations that are more likely to happen naturally over the
evolution of the protein.
</p>

<p>
When using Resistance Scan, all mutations
specified in the mutation list are created sequentially,
and their affinities are calculated
automatically.  Large positive increases
in affinity indicate that the target may become resistant
to the ligand easily.
</p>

<br clear="all">

<a name="ResidueScanMethod"></a>
<h4>Residue Scan Methodology</h4>
<p>
Residue Scanning, also known as site-directed mutagenesis, is used
to generate mutations that may produce rationally designed proteins with
modulated properties. 

<p>
Residue Scanning mutates in a sequential fashion
all residues specified in the mutation list.
The mutants of a particular residue can be edited with
a Mutation Editor.
It is also possible to perform multiple point mutations.
</p>

<a name="SampleSequenceMethod"></a>
<h4>Sample Sequence Methodology</h4>
<p>
Sample Sequence randomly searches the sequence space, as opposed to enumerating
all possibilities as in Residue Scanning. This provides the ability to search
and optimize larger sequence spaces in conjunctions with the
<a href="mutanal.htm">Mutation Analysis application</a> through accumulation
of statistics on the search space.
</p>

<p>
Sample Sequence mutates in a random fashion
all residues specified in the mutation list.
The mutants of a particular residue can be edited with
a Mutation Editor. This option should be used when a mutation space becomes large
enough to prohibit enumeration (for example, larger than 10&nbsp;000).
</p>

<a name="SequenceDesignMethod"></a>
<h4>Sequence Design Methodology</h4>

<p>
When dealing with multiple point mutations, the number of possible mutations
grows exponentially.
For example, there are 8000 possible mutations for 3 residues if
they are allowed to be any of the
20 standard amino acids. In these cases, it is
beneficial to optimize the sequence space.  This can be
done similarly to optimizing the conformation of a protein.
</p>

<p>
Sequence Design is similar
in principle to Residue Scan, but is meant to handle situations
where the combinatorial explosion due to the number of residues
and mutations reaches a point where producing all mutants is not feasible.
Sequence Design uses the algorithm as described in
<a href="#Searching">Sequence and Conformational Search</a>, but unlike
the other variants which restrict the UQO search to one mutant per 
residue, multiple mutants for each residue are allowed to compete with each
other during the UQO search. This enables an efficient search of sequence
space without the need to explicitly create all mutations. In essence,
Sequence Design optimizes the sequence for a given backbone
structure. Please note that when using Sequence Design,
the wild type mutation is always allowed to compete.
In some cases, if no better mutant is
possible, the best mutant found will be simply the wild type.
</p>

<a name="Run"></a>
<h2>Running the Protein Design Application</h2>

<a name="Preparing_the_Data"></a>
<h3>Preparing the Data</h3>

<p>
The Protein Design application can operate on either
a protein structure loaded in MOE or on a database of containing
a conformational ensemble, such as a Molecular Dynamics
trajectory.  
When working on a database, the first entry of the first molecule field
in the database file must be loaded into MOE, and the residues
to mutate specified.

The structure data under consideration
are typically crystallographic data read
from a <a href="../proteins/pdb_mdb.htm#OpeningPDBFiles">PDB file</a>,
or obtained from the MOE
<a href="../proteins/pdb_mdb.htm#MOEPDB">protein database</a>
or directly from the
<a href="../proteins/pdb_mdb.htm#RCSBDownload">PDB or PSILO</a>.
The data might also be loaded from a
<a href="../mdb/db.htm">MOE database</a> or pasted from the
<a href="../moe/gui/mainwin.htm#Copy">clipboard</a>.
</p>

<p>Structural data often contain errors.  Some additional preparatory
steps are commonly required,
such as capping, completing residues with missing
atoms and selecting appropriate alternate locations.
In MOE, structural issues can be automatically corrected
using the Structure Preparation
application <span class="nobr">(<span class="menu">MOE | Compute
| Prepare | Structure Preparation</span></span> or
<span class="nobr"><span class="menu">MOE | Protein
| Structure Preparation</span>).</span>
Once the panel is raised, press <b>Correct</b>.
This will result in an attempt to correct all issues detected by
the application.
For more information on data preparation, please see
<a href="structprep.htm">Structure Preparation</a>.

</p>

<p>
Although not required by the Protein Design application,
it may be desirable to optimize the hydrogen bond network
<span class="nobr">(<span class="menu">MOE | Compute | Prepare |
<a href="protonate3d.htm">Protonate&nbsp;3D</a></span>)</span>
and calculate charges
<span class="nobr">(<span class="menu">MOE | Compute | Prepare |
<a href="../moe/pot.htm#Charges">Partial&nbsp;Charges</a></span>)</span>
as optimal hydrogens positions and charges are necessary 
if any molecular mechanics
refinement is to be done on the mutants produced.
Note: If molecular mechanics refinements are to be performed, an appropriate
molecular mechanics <a href="../moe/pot.htm">forcefield</a>
should be selected.
</p>

<p>
It is also recommended to perform tethered energy minimization
prior to using the Protein Design application to relieve bad crystallographic
contacts or other bad geometries.
Note that in this case,
the <a href="../apps/emin.htm">Energy Minimize</a> application
itself can be used to adjust hydrogens and lone pairs and
to calculate partial charges.
Tethering, which prevents receptor atoms from deviating too
far from their initial coordinates, can be done through the
Energy Minimize panel.  Alternatively, the
<span class="menu">MOE | RHS |
<a href="../apps/ligx.htm#Prepare">LigX</a></span> panel
can be used to
add tether restraints to receptor, ligand, and/or solvent atoms
(<b>Receptor</b>, <b>Ligand</b>, <b>Solvent</b>) and then
perform energy minimization on the system (enable <b>Refine</b>).
</p>

<p>An additional step that may be taken is
to delete extraneous co-factors or unbound water molecules.  Water
molecules which play structural roles within the protein, such as
bridging waters, and important cofactors should be retained.
</p>

<p>
Finally, the
<a href="progeom.htm">Protein Geometry</a> application can be used
to assess the stereochemical quality of the protein.
</p>

<p>
<ul class="help">
<li class="help" style="padding-left: 25px;">
For more information on protein structure preparation, please see
<a href="structprep.htm">Structure Preparation</a>.
</li>
</ul>
</p>

<a name="Using_the_Panel"></a>
<h3>Using the Protein Design Panel</h3>

<p>To use the Protein Design application, select
<span class="nobr"><b>MOE | Protein | Design | <i>Variant</i></b></span>.
The following panel will open:
</p>

<p align="center">
<img src="prodesign/prodes_panel.png" alt="Protein Design Panel" />
</p>

<p>
The following sections describe details of the Protein Design Panel:
		<ul>
		<li><a href="#MutationEditor">Using the Mutation List Editor</a></li>
			<ul>
			<li><a href="#ConstantMutation">Defining a Constant Mutation</a></li>
			<li><a href="#MultipleMutations">Defining Multiple Mutations</a></li>
			<li><a href="#NonNaturalAA">Mutating to/from Non-Natural Amino Acids</a></li>
			</ul>
		<li><a href="#panel_variants">Using Protein Design Variants</a></li>
			<ul>
			<li><a href="#AlanineScan">Alanine Scan</a></li>
			<li><a href="#DisulfideScan">Disulfide Scan</a></li>
			<li><a href="#ResistanceScan">Resistance Scan</a></li>
			<li><a href="#ResidueScan">Residue Scan</a></li>
			<li><a href="#SampleSequence">Sample Sequence</a></li>
			<li><a href="#SequenceDesign">Sequence Design</a></li>
			</ul>
		<li><a href="#panel_input">Input/Output Options</a></li>
		<li><a href="#display_opt">Display Options</a></li>
		<li><a href="#ensemble_opt">Ensemble Options</a></li>
		<li><a href="#affinity_opt">Affinity Options</a></li>
		<li><a href="#properties_opt">Properties Options</a></li>
		</ul>
</p>

<!-- Mutation Editor -->

<a name="MutationEditor"></a>
<h4>Using the Mutation List Editor</h4>
<p>
There are two variations of the <b>Mutation List Editor</b>: one for
defining a constant mutation, and the other for defining multiple mutations.
</p> 


<a name="ConstantMutation"></a>
<h5>Defining a Constant Mutation</h5>
<p>
The first variation of the <b>Mutation List Editor</b> is used for
<a href="#AlanineScan">Alanine Scanning</a>,
<a href="#DisulfideScan">Disulfide Scanning</a>, and
<a href="#ResistanceScan">Resistance Scanning</a>. 
For these three <a href="#Variants">Protein Design Variants</a>, the mutation
for each residue is constant and therefore only the following options are
available. Note: When no mutations are listed for a residue, that 
residue will not be used for the search.
</p>
<p><img src="prodesign/panel_edit1.png" alt="Protein Design Edit Options" /></p>

<table cellpadding="5">
<tr>
<td  valign="top"><b>Clear All</b></td>
<td valign="top">Remove mutations for all residues in the <b>Mutation List</b>.</td>
</tr>

<tr>
<td  valign="top"><b>Mutate All</b></td>
<td valign="top">Add default mutations to residues that have no mutations
listed in their <b>Mutation</b> column.</td>
</tr>

<tr>
<td  valign="top"><b>Clear/Mutated</b></td>
<td valign="top">
If a residue cannot be mutated, it will be
identified in the bubble help of
the <b>Clear/Mutated</b> button.</td>
</tr>
</table>


<a name="MultipleMutations"></a>
<h5>Defining Multiple Mutations</h5>
<p>
The second variation of the <b>Mutation List Editor</b> is used for
<a href="#ResidueScan">Residue Scanning</a>,
<a href="#SampleSequence">Sample Sequence</a>, and
<a href="#SequenceDesign">Sequence Design</a>. 
For these three <a href="#Variants">Protein Design Variants</a>, the mutation list can be 
edited using the mutation list editor.
Note: When no mutations are listed for a residue, that
residue will not be used for the search.
</p>
<p><img src="prodesign/panel_edit2.png" alt="Protein Design Edit Options" /></p>

<table cellpadding="5">
<tr>
<td  valign="top"><b>Clear/Mutate</b></td>
<td valign="top">Remove mutations from, or add mutations to,
residues selected in the <b>Mutation List</b>.</td>
</tr>

<tr>
<td  valign="top"><b>Edit Box</b></td>
<td valign="top">
Manually define a mutation using either a sequence of single letter codes (such as <tt>AKW</tt>) or
a list of three letter codes separated by a space or comma
(such as <tt>Arg Lys Trp</tt>). Three
letter codes must be entered with the first letter capitalized and the
other two letters lowercase; for example, &quot;Arg&quot;, but not &quot;ARG&quot; or &quot;arg&quot;.
Non-natural amino acids must be defined in the
database chosen as the Rotamer Library and specified
using its full name in parentheses
(such as <tt>A,(HYP),K</tt>).
In addition to the 20 standard amino acids (<tt>Xxx</tt>), other supported mutations include <tt>J</tt> (Non-polar),
<tt>O</tt> (Polar), <tt>B</tt> (<tt>N</tt>, <tt>D</tt>)
and <tt>Z</tt> (<tt>Q</tt>, <tt>E</tt>).
</td></tr>

<tr>
<td valign="top"><b>Apply</b></td>
<td valign="top">Apply the mutation in the edit box to the
residues selected in the <b>Mutation List</b>.</td>
</tr>

<tr>
<a name="MutationEditorShortcut">
<td valign="top"><b>Mutation Editor Shortcut</b></td>
<td valign="top">
Clicking on 
<img src="prodesign/panel_edit2_shortcut.png" alt="Mutation Editor Shortcut"/>
will open the 
<b>Mutation Editor Shortcut</b> panel.
This shortcut provides an easy way to browse, add, and remove mutations for
the residues selected in the <b>Mutation List</b>. 
By default, the <b>Mutation Editor Shortcut</b> will select all of the
common amino acids defined as mutations for each of the residues, and will start
with the <b>Class</b> filter set to <i>All</i>. Selecting/deselecting an amino
acid will add/remove it from the list of mutations for the selected residues
in the <b>Mutation List</b>.

<table>
<tr><td>
<img src="prodesign/panel_edit2_shortcut_2014.png" alt="Edit Short Cut"
style="margin: 0px 15px 0px 0px;" />
</td>
<td>

<center>
<table>

<tr><td><b>Class</b></td>
<td>
Restrict the display of amino acids in the amino acid list 
to a particular amino acid class. The <i>All</i> option displays all classes.
</td></tr>

<tr><td><b>Filter</b></td>
<td>
Filter the display
of amino acids in the list. If no filter is specified, then no restriction
is made other than the <b>Class</b> restriction. If a filter is specified, then in addition to
the restrictions imposed by <b>Class</b>, only those lines in the list with text
containing each of the words in the <b>Filter</b> text field will be displayed. 
<p>
For example, if the filter text is "Standard", the display is restricted to
lines that contain the consecutive letters (case insensitive) "standard".
If the filter text is "Standard Polar", the display is restricted to
lines that contain the substring "standard" as well as the substring
"polar" (not necessarily in that order).
</p>
<p>
Typing in the <b>Filter</b> text field will automatically update the list.
The red <b>X</b> button deletes all characters from the <b>Filter</b> text.
</p>
</td></tr>

<tr><td><b>Select</b></td>
<td>
Select amino acids in the listbox that correspond to the
chosen category. If the red <b>x</b> is pressed, it will deselect
the amino acids that correspond to that category.
<ul>
<li><b>Reset</b>: Sets the selection in the listbox to the initial
selection when the <b>Mutation Editor Shortcut</b> opened.</li>
<li><b>Aromatic</b>: Selects aromatic amino acids (Phe, Tyr, Trp).</li>
<li><b>Polar</b>: Selects polar amino acids (Arg, Asn, Asp, Cys,
Gln, Glu, His, Lys, Ser, Thr).</li>
<li><b>Hydrophobic</b>: Selects hydrophobic amino acids (Ala, Gly, Ile, Leu,
Met, Phe, Pro, Trp, Tyr, Val).</li>
<li><b>Positive</b>: Selects positively charged amino acids (Arg, Lys).</li>
<li><b>Negative</b>: Selects negatively charged amino acids (Asp, Glu).</li>
</ul>
</td></tr>

</table>
</td></tr>

</table>
</td></tr>
</td></tr>


</table>
</center>
</p>

<!-- Non-Natural Amino Acids -->

<a name="NonNaturalAA"></a>
<h5>Mutating to/from Non-Natural Amino Acids</h5>

<p>
If the parent residue or mutation is a non-natural amino acid
it must be added to the Rotamer Library. The
<a href="../proteins/buildrotlib.htm">Build Rotamer Library</a> application can be used
to create MDB files which can then be added to the Rotamer Library
( see
<span class="menu">MOE | Settings | Protein Defaults | Rotamer Library</span> 
). After a non-natural amino acid is added to the Rotamer Library, the Protein Design application will 
need to be restarted. 

<p>
Non-natural amino acids will appear in the Editor Shortcut when
the class is changed to <i>All</i>.
Mutations to a non-natural amino acid can be performed via
the Mutation List, by first selecting the residue to be mutated and then
editing the mutation string in the mutation list editor.
Non-natural amino acids must be specified using the residue
name (rName) in parentheses; for example, <tt>A,(HYP),K</tt>.

</p>


<!-- Protein Design Variants -->


<a name="panel_variants"></a>
<h4>Using Protein Design Variants</h4>

<p>
The Protein Design application has 5 different variants each of which is
optimized for a specific application. 
They can be selected by clicking on the desired variant.
For more information, see <a href="#Variants">Protein Design Variants</a>.
A site is defined as the residues to be mutated. For example, in
the following mutation &quot;Asp24[Ala;Ser], Ala25[Asp,Ser]&quot;,
the site would be
defined as Asp24 and Ala25. A mutation combination refers to the
specific mutations for a site. For the example
&quot;Asp24[Ala,Ser], Ala25[Asp,Ser]&quot;, there would
be 4 mutation combinations:
"Asp24Ala, Ala25Asp&quot;;
&quot;Asp24Ser, Ala25Asp&quot;;
&quot;Asp24Ala, Ala25Ser&quot;;
&quot;Asp24Ser, Ala25Ser&quot;. 
</p>

<div class="floatleft"> 
<img src="prodesign/prodes_variants.png" alt="Protein Design Variants" />
</div>

<ul>
<li>
<a href="#AlanineScan"><i>Alanine Scan</i></a>
<span class="nobr">(<span class="menu">MOE | Protein |
Design | Alanine Scan</span>)</span>
<p>
All residues displayed in the mutation list will be mutated to Alanine, as
shown in the Mutation column in the list. All mutations are done
sequentially, also known as single point mutations.
</p>
</li>

<li>
<a href="#DisulfideScan"><i>Disulfide Scan</i></a>
<span class="nobr">(<span class="menu">MOE | Protein |
Design | Disulfide Scan</span>)</span>
<p>
All residues displayed in the mutation list will be assessed for the
possibility of creating a disulfide bond with a nearby residue. If
disulfide bond creation is possible,
the implicated residues and their partner(s)
will be mutated to cysteine systematically,
followed by an attempt to create the disulfide bonds.
</p>
</li>

<li>
<a href="#ResistanceScan"><i>Resistance Scan</i></a>
<span class="nobr">(<span class="menu">MOE | Protein |
Design | Resistance Scan</span>)</span>
<p>
Single nucleotide polymorphism (SNP) mutations will be performed on 
residues displayed in the mutation list. All mutations will be single point 
mutations and will be built sequentially.
</p>
</li>

<li>
<a href="#ResidueScan"><i>Residue Scan</i></a>
<span class="nobr">(<span class="menu">MOE | Protein |
Design | Residue Scan</span>)</span>
<p>
Mutations listed in the mutation column of the mutation list
will be performed on
residues displayed in the mutation list. Mutations for a given residue can be 
edited using the Mutation List Editor. The permissible number of 
concurrent N point mutations can be controlled using
the Site Limit option. All mutation combinations, as defined by the Mutation
list and Site Limit will be created and output.
</p>
</li>

<li>
<a href="#Sample Sequence"><i>Sample Sequence</i></a>
<span class="nobr">(<span class="menu">MOE | Protein |
Design | Sample Sequence</span>)</span>
<p>
Mutations listed in the mutation column of the mutation list
will be used to define the mutation space for
residues displayed in the mutation list. When using the Sample Sequence
variant, the mutation space will be randomly searched, creating the number of
mutants specified by Sequences. Mutations for a given residue can be 
edited using the Mutation List Editor. The permissible number of 
concurrent N point mutations can be controlled using
the Site Limit option.
</p>
</li>

<li>
<a href="#SequenceDesign"><i>Sequence Design</i></a>
<span class="nobr">(<span class="menu">MOE | Protein |
Design | Sequence Design</span>)</span>
<p>
Mutations listed in the mutation column of the mutation list
will be performed on
residues displayed in the mutation list. Mutations for a given residue can 
be edited using the Mutation List Editor. The permissible number
of concurrent N point mutations can be controlled using
the Site Limit option. Unlike in Residue Scan, here the sequence is optimized,
with mutations being allowed to compete with each other for a specific site
instead of all possible mutations being produced. Only the best scoring
mutations per mutation site will be output.
</p>
</li>
</ul>

<br clear="all">

<!-- Alanine Scan -->

<a name="AlanineScan"></a>
<h5>Alanine Scan</h5>

<p>
For more information on input options for Alanine Scan, please see
<a href="#panel_input">Input/Output Options</a>. For details concerning the
Alanine Scan method, please see its <a href="#AlanineScanMethod">Methodology</a>.

<h6>Mutation List and Display Options</h6>

<table cellpadding="5">
<tr>
<td  valign="top"><b>Mutation List</b></td>
<td valign="top">
All residues as defined by the <a href="#display_opt">Display Options</a> appear in
this list. For each residue,  the following details are given:
<ul>
<li><b>Chain</b>: Chain number and name.
<li><b>Res</b>: Residue Type.
<li><b>UID</b>: Residue UID and Insertion code.
<li><b>Mutation</b>: If this field is blank, the residue will not be
mutated.  An &quot;A&quot; marks the residue for mutation to alanine.
If no mutations are listed for a residue, no mutation will be applied. 
</ul>
When the <a href="#display_opt">display option</a> <b>Contour in MOE window</b> is enabled, 
a single click in the <b>Mutation List</b> will display the
contours for the selected residues in the list. A double click will
toggle the selection state of the associated atoms of the selected residues.

</td><tr>
</table>


<h6>Mutation Editor</h6>
<p><img src="prodesign/panel_edit1.png" alt="Protein Design Edit Options" /></p>

The <a href="#ConstantMutation">first variation</a> of the <b>Mutation List Editor</b> is used for
Alanine Scanning.


<h6>Generation Options</h6>

<table cellpadding="5">
<tr>
<td  valign="top"><b>Site Limit</b></td>
<td valign="top">
A site refers to the residues being mutated. 
<b>Site Limit</b> gives the maximum number of residues which may
constitute a mutation site.
In Alanine Scan, <b>Site Limit</b> is fixed at a value of 1.
To allow for multi-point alanine
mutations, use <a href="#ResidueScan">Residue Scan</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Mutation Space</b></td>
<td valign="top">This is the number of mutations that will be searched.
The value will change color depending on the size of the space:
<ul>
<li>Yellow = 0
<li>Blue = 1
<li>Green &gt;1 and &lt; 1,000,000
<li>Red &ge; 1,000,000
</ul>
When the mutation space is large, it is recommended to use
<a href="#SequenceDesign">Sequence Design</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Sample</b></td>
<td valign="top">
This option is enabled only when using <a href="#SampleSequence">
Sample Sequence</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Sequences</b></td>
<td valign="top">
This option is available only with <a href="#SampleSequence">
Sample Sequence</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Settings</b></td>
<td valign="top">

<table cellpadding="5">

<tr>
<td valign="top">
<p><img src="prodesign/uqo_opt1.png" alt="options" />
</td>
<td valign="top">
Pressing the Settings icon
<img src="prodesign/settings.png" alt="settings" align="bottom" />
will raise the popup menu shown at left. 

<ul>
<p><li><b>Repack environment</b>. When enabled, residues within "Cutoff"
of the mutation will be included in conformational search.
</li></p>

<p><li><b>Cutoff</b>. The distance (&Aring;) used to determine which residues
are added to the conformational search in addition to the residue being
mutated.
</li></p>

<p><li><b>Alternate protomers</b>. Alternate protonation states of residues
to be mutated
are considered when
<b>Repack environment</b> is enabled. For example,
when enabled and mutating to Asp,
both the charged and neutral forms are allowed to compete against each other.
Disabling would allow only the charged form. 
This option is by default enabled in Alanine Scan, Disulfide Scan, Resistance
Scan and Residue Scan. In Sequence Design, which searches over all sequence
possibilities, this option is by default disabled
to reduce the search space.
</li></p>

<p><li><b>pH</b>. Specifies the pH of the system when performing
the mutation. This,
along with
<a href="../apps/residuepka.htm">PROPKA</a>, is
used for scoring alternate protonation states,
to account for local <span class='LaTeX'>\(pK_{a}\)</span> shifts.
</li></p>

<p><li><b>Refine Mutations</b>. When enabled,
the mutations will be refined following the conformational
search using UQO.
For more information on the
refinement protocol, see <a href="#Searching">Searching Conformation and
Sequence Space</a>. By default, this option is enabled.
</li></p>

<p><li><b>RMS Gradient</b>.
If "Refine Mutations" is enabled, the final refinement
will be performed until the RMS of the gradient of the potential energy is less
than a specified value.
</li></p>

<p><li><b>Enumerate Site Limit</b>.
This option is only used by Residue Scanning, Sample Sequence, and
Sequence Design. 
</li></p>
</ul>


</td>
</tr>

</table>
</td>
</tr>

</table>

<!-- Disulfide Scan -->

<a name="DisulfideScan"></a>
<h5>Disulfide Scan</h5>

<p>
For more information on input options for Disulfide Scan, please see
<a href="#panel_input">Input/Output Options</a>. For details concerning the
Disulfide Scan method, please see its
<a href="#DisulfideScanMethod">Methodology</a>.


<h6>Mutation List and Display Options</h6>

<table cellpadding="5">
<tr>
<td  valign="top"><b>Mutation List</b></td>
<td valign="top">
All residues as defined by the <a href="#display_opt">Display Options</a> appear in
this list. For each residue,  the following details are given:
<ul>
<li><b>Chain</b>: Chain number and name.
<li><b>Res</b>: Residue Type.
<li><b>UID</b>: Residue UID and Insertion code.
<li><b>Mutation</b>: Shows the possible disulfide bond partners for
a given residue. The chain is separated from the
partners by a colon (:). For
example, 1:S24C specifies that Serine 24 of 
Chain 1 will be mutated to Cysteine. 
If no mutations are listed for a residue, no mutation will be done. 
</ul>

When the <a href="#display_opt">display option</a> <b>Contour in MOE window</b> is enabled, 
a single click in the <b>Mutation List</b> will display the
contours for the selected residues in the list. A double click will
toggle the selection state of the associated atoms of the selected residues.
</td><tr>
</table>


<h6>Mutation Editor</h6>
<p><img src="prodesign/panel_edit1.png" alt="Protein Design Edit Options" /></p>

The <a href="#ConstantMutation">first variation</a> of the <b>Mutation List Editor</b> is used for
Disulfide Scanning.


<h6>Generation Options</h6>

<table cellpadding="5">
<tr>
<td  valign="top"><b>Site Limit</b></td>
<td valign="top">
A site refers to the residues being mutated. 
<b>Site Limit</b> gives the maximum number of residues which may
constitute a mutation site.
In Disulfide Scan,
<b>Site Limit</b> is fixed at a value of 2. To allow for varying
multi-point Disulfide
mutations, use <a href="#ResidueScan">Residue Scan</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Mutation Space</b></td>
<td valign="top">This is the number of mutations that will be searched.
The value will change color depending on the size of the space:
<ul>
<li>Yellow = 0
<li>Blue = 1
<li>Green &gt;1 and &lt; 1,000,000
<li>Red &ge; 1,000,000
</ul>
When the mutation space is large, it is recommended to use
<a href="#SequenceDesign">Sequence Design</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Sample</b></td>
<td valign="top">
This option is enabled only when using <a href="#SampleSequence">
Sample Sequence</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Sequences</b></td>
<td valign="top">
This option is available only with <a href="#SampleSequence">
Sample Sequence</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Settings</b></td>
<td valign="top">

<table cellpadding="5">

<tr>
<td valign="top">
<p><img src="prodesign/uqo_opt1.png" alt="options" /></p>
</td>
<td valign="top">
Pressing the Settings icon 
<img src="prodesign/settings.png" alt="settings" align="bottom" />
will raise the popup menu shown at left. 

<ul>
<p><li><b>Repack environment</b>. When enabled, residues within <b>Cutoff</b>
of the mutation will be included in the conformational search.
</li></p>

<p><li><b>Cutoff</b>. The distance (in &Aring;) used to determine which residues
are added to the conformational search, in addition to the residue being
mutated.
</li></p>

<p><li><b>Alternate protomers</b>. When enabled, alternate protonation states of residues
to be mutated
are considered. For example,
when enabled and mutating to Asp,
both the charged and neutral forms are allowed to compete against each other.
Disabling would allow only the charged form. By default, this option is
enabled. This option is available only when
<b>Repack environment</b> is enabled. 
</li></p>

<p><li><b>pH</b>. Specifies the <span class='LaTeX'>\(pH\)</span> of the system when performing
the mutation. This,
along with
<a href="../apps/residuepka.htm">PROPKA</a>, is
used for scoring alternate protonation states,
to account for local <span class='LaTeX'>\(pK_{a}\)</span> shifts.
</li></p>

<p><li><b>Refine Mutations</b>. When enabled,
the mutations will be refined, using UQO, following the conformational
search.
For more information on the
refinement protocol, see <a href="#Searching">Searching Conformation and
Sequence Space</a>. By default, this option is enabled.
</li></p>

<p><li><b>RMS Gradient</b>.
Specifies the cutoff RMS of the gradient of the potential energy.
If <b>Refine Mutations</b> is enabled, the final refinement
will be performed until the RMS of the gradient of the potential energy is less
than this specified value.
</li></p>

<p><li><b>Enumerate Site Limit</b>.
This option is only used by Residue Scanning, Sample Sequence, and
Sequence Design. 
</li></p>
</ul>
</td>
</tr>
</table>
</td>
</tr>

</table>

<!-- Resistance Scan -->

<a name="ResistanceScan"></a>
<h5>Resistance Scan</h5>

<p>
For more information on input options for Resistance Scan, please see
<a href="#panel_input">Input/Output Options</a>. For details concerning the
Resistance Scan method, please see its
<a href="#ResistanceScanMethod">Methodology</a>.
</p>

<h6>Mutation List and Display Options</h6>

<table cellpadding="5">
<tr>
<td  valign="top"><b>Mutation List</b></td>
<td valign="top">
All residues as defined by the <a href="#display_opt">Display Options</a> appear in
this list. For each residue,  the following details are given:
<ul>
<li><b>Chain</b>: Chain number and name.
<li><b>Res</b>: Residue Type.
<li><b>UID</b>: Residue UID and Insertion code.
<li><b>Mutation</b>:  List of possible mutations, given
as single letter residue codes. Only 
<a href="#ResistanceScan">single nucleotide polymorphism</a> (SNP)
mutations are allowed.
If no mutations are listed for a residue, no mutation will be done. 
</ul>

When the <a href="#display_opt">display option</a> <b>Contour in MOE window</b> is enabled, 
a single click in the <b>Mutation List</b> will display the
contours for the selected residues in the list. A double click will
toggle the selection state of the associated atoms of the selected residues.
</td><tr>
</table>


<h6>Mutation Editor</h6>
<p><img src="prodesign/panel_edit1.png" alt="Protein Design Edit Options" /></p>

The <a href="#ConstantMutation">first variation</a> of the <b>Mutation List Editor</b> is used for
Resistance Scanning. 


<h6>Generation Options</h6>


<table cellpadding="5">
<tr>
<td  valign="top"><b>Site Limit</b></td>
<td valign="top">
A site refers to the residues being mutated. 
<b>Site Limit</b> gives the maximum number of residues which may
constitute a mutation site.
In Resistance Scan,
<b>Site Limit</b> is fixed at a value of 1. To allow for multi-point Resistance
Scan mutations, use <a href="#ResidueScan">Residue Scan</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Mutation Space</b></td>
<td valign="top">This is the number of mutations that will be searched.
The value will change color depending on the size of the space:
<ul>
<li>Yellow = 0
<li>Blue = 1
<li>Green &gt;1 and &lt; 1,000,000
<li>Red &ge; 1,000,000
</ul>
When the mutation space is large, it is recommended to use
<a href="#SequenceDesign">Sequence Design</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Sample</b></td>
<td valign="top">
This option is enabled only when using <a href="#SampleSequence">
Sample Sequence</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Sequences</b></td>
<td valign="top">
This option is available only with <a href="#SampleSequence">
Sample Sequence</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Settings</b></td>
<td valign="top">

<table cellpadding="5">

<tr>
<td valign="top">
<p><img src="prodesign/uqo_opt1.png" alt="options" />
</td>
<td valign="top">
Pressing the Settings icon 
<img src="prodesign/settings.png" alt="settings" align="bottom" />
will raise the popup menu shown at left. 

<ul>
<p><li><b>Repack environment</b>. When enabled, residues within <b>Cutoff</b>
of the mutation will be included in the conformational search.
</li></p>

<p><li><b>Cutoff</b>. The distance (in &Aring;) used to determine which residues
are added to the conformational search, in addition to the residue being
mutated.
</li></p>

<p><li><b>Alternate protomers</b>. When enabled, alternate protonation states of residues
to be mutated
are considered. For example,
when enabled and mutating to Asp,
both the charged and neutral forms are allowed to compete against each other.
Disabling would allow only the charged form. By default, this option is
enabled. This option is available only when
<b>Repack environment</b> is enabled. 
</li></p>

<p><li><b>pH</b>. Specifies the <span class='LaTeX'>\(pH\)</span> of the system when performing
the mutation. This,
along with
<a href="../apps/residuepka.htm">PROPKA</a>, is
used for scoring alternate protonation states,
to account for local <span class='LaTeX'>\(pK_{a}\)</span> shifts.
</li></p>

<p><li><b>Refine Mutations</b>. When enabled,
the mutations will be refined, using UQO, following the conformational
search.
For more information on the
refinement protocol, see <a href="#Searching">Searching Conformation and
Sequence Space</a>. By default, this option is enabled.
</li></p>

<p><li><b>RMS Gradient</b>.
Specifies the cutoff RMS of the gradient of the potential energy.
If <b>Refine Mutations</b> is enabled, the final refinement
will be performed until the RMS of the gradient of the potential energy is less
than this specified value.
</li></p>

<p><li><b>Enumerate Site Limit</b>.
This option is only used by Residue Scanning, Sample Sequence, and
Sequence Design. 
</li></p>
</ul>
</td>
</tr>

</table>
</td>
</tr>

</table>

<!--Residue Scan -->

<a name="ResidueScan"></a>
<h5>Residue Scan</h5>

<p>
For more information on input options for Residue Scan, please see
<a href="#panel_input">Input/Output Options</a>. For details concerning the
Residue Scan method, please see its
<a href="#ResidueScanMethod">Methodology</a>.
</p>

<h6>Mutation List and Display Options</h6>

<table cellpadding="5">
<tr>
<td  valign="top"><b>Mutation List</b></td>
<td valign="top">
All residues as defined by the <a href="#display_opt">Display Options</a> appear in
this list. For each residue,  the following details are given:
<ul>
<li><b>Chain</b>: Chain number and name.
<li><b>Res</b>: Residue Type.
<li><b>UID</b>: Residue UID and Insertion code.
<li><b>Mutation</b>:  List of possible mutations, given as
single letter residue codes for the 20 standard amino acids. 
For non-natural amino acids, the
full residue name within parentheses is required;
For example, <tt>(SEP)</tt> specifies phosphorylated serine. 
If no mutations are listed for a residue, no mutation will be carried out. 
</ul>

When the <a href="#display_opt">display option</a> <b>Contour in MOE window</b> is enabled, 
a single click in the <b>Mutation List</b> will display the
contours for the selected residues in the list. A double click will
toggle the selection state of the associated atoms of the selected residues.
</td><tr>
</table>
<p>
To create a single multi-point mutation using the Residue Scan
variant, first ensure that all residues listed in <b>Mutation List</b> have only
one mutation. If this condition is met, <b>Site Limit</b> can be set to 0,
which enables the creation of the exact mutation described in <b>Mutation List</b>.
For example, if 2 residues are present in <b>Mutation List</b> and these residues 
have only one mutation each, then, when <b>Site Limit</b> is set to 0, only a single
2-point mutant (as described by the mutation) will be created in addition to
the wild type protein.
</p>


<h6>Mutation Editor</h6>
<p><img src="prodesign/panel_edit2.png" alt="Protein Design Edit Options" /></p>

The <a href="#MultipleMutations">second variation</a> of the <b>Mutation List Editor</b> is used for
Residue Scanning.


<h6>Generation Options</h6>

<table cellpadding="5">
<tr>
<td  valign="top"><b>Site Limit</b></td>
<td valign="top">
A site refers to the residues being mutated. 
<b>Site Limit</b> gives the maximum number of residues which may
constitute a mutation site.
For Residue Scan, all sites of sizes 1 to <b>Site Limit</b> will be generated.
</td>
</tr>

<tr>
<td  valign="top"><b>Mutation Space</b></td>
<td valign="top">This is the number of mutations that will be searched.
The value will change color depending on the size of the space:
<ul>
<li>Yellow = 0
<li>Blue = 1
<li>Green &gt;1 and &lt; 1,000,000
<li>Red &ge; 1,000,000
</ul>
When the mutation space is large, it is recommended to use the
<a href="#SampleSequence">Sample Sequence variant</a> or
<a href="#SequenceDesign">Sequence Design variant</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Sample</b></td>
<td valign="top">
This option is enabled only when using <a href="#SampleSequence">
Sample Sequence</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Sequences</b></td>
<td valign="top">
This option is available only with <a href="#SampleSequence">
Sample Sequence</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Settings</b></td>
<td valign="top">

<table cellpadding="5">

<tr>
<td valign="top">
<p><img src="prodesign/uqo_opt1a.png" alt="options" />
</td>
<td valign="top">
Pressing the Settings icon 
<img src="prodesign/settings.png" alt="settings" align="bottom" />
will raise the popup menu shown at left. 

<ul>
<p><li><b>Repack environment</b>. When enabled, residues within <b>Cutoff</b>
of the mutation will be included in the conformational search.
</li></p>

<p><li><b>Cutoff</b>. The distance (in &Aring;) used to determine which residues
are added to the conformational search, in addition to the residue being
mutated.
</li></p>

<p><li><b>Alternate protomers</b>. When enabled, alternate protonation states of residues
to be mutated
are considered. For example,
when enabled and mutating to Asp,
both the charged and neutral forms are allowed to compete against each other.
Disabling would allow only the charged form. By default, this option is
enabled. This option is available only when
<b>Repack environment</b> is enabled. 
</li></p>

<p><li><b>pH</b>. Specifies the <span class='LaTeX'>\(pH\)</span> of the system when performing
the mutation. This,
along with
<a href="../apps/residuepka.htm">PROPKA</a>, is
used for scoring alternate protonation states,
to account for local <span class='LaTeX'>\(pK_{a}\)</span> shifts.
</li></p>

<p><li><b>Refine Mutations</b>. When enabled,
the mutations will be refined, using UQO, following the conformational
search.
For more information on the
refinement protocol, see <a href="#Searching">Searching Conformation and
Sequence Space</a>. By default, this option is enabled.
</li></p>

<p><li><b>RMS Gradient</b>.
Specifies the cutoff RMS of the gradient of the potential energy.
If <b>Refine Mutations</b> is enabled, the final refinement
will be performed until the RMS of the gradient of the potential energy is less
than this specified value.
</li></p>


<p><li><b>Enumerate Site Limit</b>.
When enabled, the number of mutated sites will be enumerated from 1 to
N (where N = <b>Site Limit</b>). When disabled, only mutations of
size N will be created.
</li></p>
</ul>
</td>
</tr>

</table>

</td>
</tr>
</table>


<!--Sample Sequence -->

<a name="SampleSequence"></a>
<h5>Sample Sequence</h5>

<p>
For more information on input options for Sample Sequence, please see
<a href="#panel_input">Input/Output Options</a>. For details concerning the
Sample Sequence method, please see its
<a href="#SampleSequenceMethod">Methodology</a>.
</p>

<h6>Mutation List and Display Options</h6>

<table cellpadding="5">
<tr>
<td  valign="top"><b>Mutation List</b></td>
<td valign="top">
All residues as defined by the <a href="#display_opt">Display Options</a> appear in
this list. For each residue,  the following details are given:
<ul>
<li><b>Chain</b>: Chain number and name.
<li><b>Res</b>: Residue Type.
<li><b>UID</b>: Residue UID and Insertion code.
<li><b>Mutation</b>:  List of possible mutations, given as
single letter residue codes for the 20 standard amino acids. 
For non-natural amino acids, the
full residue name within parentheses is required.
For example, <tt>(SEP)</tt> specifies phosphorylated serine. 
If no mutations are listed for a residue, no mutation will be carried out. 
</ul>

When the <a href="#display_opt">display option</a> <b>Contour in MOE window</b> is enabled, 
a single click in the <b>Mutation List</b> will display the
contours for the selected residues in the list. A double click will
toggle the selection state of the associated atoms of the selected residues.
</td><tr>
</table>


<h6>Mutation Editor</h6>
<p><img src="prodesign/panel_edit2.png" alt="Protein Design Edit Options" /></p>

The <a href="#MultipleMutations">second variation</a> of the <b>Mutation List Editor</b> is used for
Sample Sequence.


<h6>Generation Options</h6>

<table cellpadding="5">
<tr>
<td  valign="top"><b>Site Limit</b></td>
<td valign="top">
A site refers to the residues being mutated. 
<b>Site Limit</b> gives the maximum number of residues which may
constitute a mutation site.
For Sample Sequence, the mutation space can be randomly searched, creating mutants
with 1 to N mutations, where N is defined by <b>Site Limit</b>.
</td>
</tr>

<tr>
<td  valign="top"><b>Mutation Space</b></td>
<td valign="top">This is the number of mutations that will be searched.
The value will change color depending on the size of the space:
<ul>
<li>Yellow = 0
<li>Blue = 1
<li>Green &gt;1 and &lt; 1,000,000
<li>Red &ge; 1,000,000
</ul>
</td>
</tr>

<tr>
<td  valign="top"><b>Sample</b></td>
<td valign="top">
When enabled,
this option indicates that the mutation space will be searched randomly.
</td>
</tr>

<tr>
<td  valign="top"><b>Sequences</b></td>
<td valign="top">
When <b>Sample</b> is enabled, <b>Sequences</b> determines the number
of random mutants that will be generated.
</td>
</tr>


<tr>
<tr>
<td  valign="top"><b>Settings</b></td>
<td valign="top">

<table cellpadding="5">
<tr>
<td valign="top">
<p><img src="prodesign/uqo_opt1a.png" alt="options" />
</td>
<td valign="top">
Pressing the Settings icon 
<img src="prodesign/settings.png" alt="settings" align="bottom" />
will raise the popup menu shown at left. 

<ul>
<p><li><b>Repack environment</b>. When enabled, residues within <b>Cutoff</b>
of the mutation will be included in the conformational search.
</li></p>

<p><li><b>Cutoff</b>. The distance (in &Aring;) used to determine which residues
are added to the conformational search, in addition to the residue being
mutated.
</li></p>

<p><li><b>Alternate protomers</b>. When enabled, alternate protonation states of residues
to be mutated
are considered. For example,
when enabled and mutating to Asp,
both the charged and neutral forms are allowed to compete against each other.
Disabling would allow only the charged form. By default, this option is
enabled. This option is available only when
<b>Repack environment</b> is enabled. 
</li></p>

<p><li><b>pH</b>. Specifies the <span class='LaTeX'>\(pH\)</span> of the system when performing
the mutation. This,
along with
<a href="../apps/residuepka.htm">PROPKA</a>, is
used for scoring alternate protonation states,
to account for local <span class='LaTeX'>\(pK_{a}\)</span> shifts.
</li></p>

<p><li><b>Refine Mutations</b>. When enabled,
the mutations will be refined, using UQO, following the conformational
search.
For more information on the
refinement protocol, see <a href="#Searching">Searching Conformation and
Sequence Space</a>. By default, this option is enabled.
</li></p>

<p><li><b>RMS Gradient</b>.
Specifies the cutoff RMS of the gradient of the potential energy.
If <b>Refine Mutations</b> is enabled, the final refinement
will be performed until the RMS of the gradient of the potential energy is less
than this specified value.
</li></p>

<p><li><b>Enumerate Site Limit</b>.
When enabled, the number of mutated sites will be enumerated from 1 to
N (where N = <b>Site Limit</b>). When disabled, only mutations of
size N will be created.
</li></p>
</ul>
</td>
</tr>

</table>

</td>
</tr>
</table>


<!-- Sequence Design -->

<a name="SequenceDesign"></a>
<h5>Sequence Design</h5>

<p>
For more information on input options for Sequence Design, please see
<a href="#panel_input">Input/Output Options</a>. For details concerning the
Sequence Design method, please see its
<a href="#SequenceDesignMethod">Methodology</a>.
</p>

<h6>Mutation List and Display Options</h6>

<table cellpadding="5">
<tr>
<td  valign="top"><b>Mutation List</b></td>
<td valign="top">
All residues as defined by the <a href="#display_opt">Display Options</a> appear in
this list. For each residue,  the following details are given:
<ul>
<li><b>Chain</b>: Chain number and name.
<li><b>Res</b>: Residue Type.
<li><b>UID</b>: Residue UID and Insertion code.

<li><b>Mutation</b>:  List of possible mutations, given as
single letter residue codes for the 20 standard amino acids. 
For non-natural amino acids, the
full residue name within parentheses is required.
For example, <tt>(SEP)</tt> specifies phosphorylated serine. 
If no mutations are listed for a residue, no mutation will be carried out. 
</ul>

When the <a href="#display_opt">display option</a> <b>Contour in MOE window</b> is enabled, 
a single click in the <b>Mutation List</b> will display the
contours for the selected residues in the list. A double click will
toggle the selection state of the associated atoms of the selected residues.

</td><tr>
</table>


<h6>Mutation Editor</h6>
<p><img src="prodesign/panel_edit2.png" alt="Protein Design Edit Options" /></p>

The <a href="#MultipleMutations">second variation</a> of the <b>Mutation List Editor</b> is used for
Sequence Design.


<h6>Generation Options</h6>

<table cellpadding="5">
<tr>
<td  valign="top"><b>Site Limit</b></td>
<td valign="top">
A site refers to the residues being mutated. 
<b>Site Limit</b> gives the maximum number of residues which may
constitute a mutation site.
For Sequence Design, all sites of sizes 1 to <b>Site Limit</b> will be generated.
</td>
</tr>

<tr>
<td  valign="top"><b>Mutation Space</b></td>
<td valign="top">This is the number of mutations that will be searched.
The value will change color depending on the size of the space:
<ul>
<li>Yellow = 0
<li>Blue = 1
<li>Green &gt;1 and &lt; 1,000,000
<li>Red &ge; 1,000,000
</ul>
</td>
</tr>

<tr>
<td  valign="top"><b>Sample</b></td>
<td valign="top">
This option is enabled only when using <a href="#SampleSequence">
Sample Sequence</a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Sequences</b></td>
<td valign="top">
This option is available only with <a href="#SampleSequence">
Sample Sequence</a>.
</td>
</tr>


<tr>
<td  valign="top"><b>Settings</b></td>
<td valign="top">

<table cellpadding="5">

<tr>
<td valign="top">
<a name="settings_seqdesign">
<p><img src="prodesign/uqo_opt2.png" alt="options" />
</td>
<td valign="top">
Pressing the Settings icon 
<img src="prodesign/settings.png" alt="settings" align="bottom" />
will raise the popup menu shown at left. 

<ul>
<p><li><b>Repack environment</b>. When enabled, residues within <b>Cutoff</b>
of the mutation will be included in the conformational search.
</li></p>

<p><li><b>Cutoff</b>. The distance (in &Aring;) used to determine which residues
are added to the conformational search, in addition to the residue being
mutated.
</li></p>

<p><li><b>Alternate protomers</b>. When enabled, alternate protonation states of residues
to be mutated
are considered. For example,
when enabled and mutating to Asp,
both the charged and neutral forms are allowed to compete against each other.
Disabling would allow only the charged form. By default, this option is
enabled. This option is available only when
<b>Repack environment</b> is enabled. 
</li></p>

<p><li><b>pH</b>. Specifies the <span class='LaTeX'>\(pH\)</span> of the system when performing
the mutation. This,
along with
<a href="../apps/residuepka.htm">PROPKA</a>, is
used for scoring alternate protonation states,
to account for local <span class='LaTeX'>\(pK_{a}\)</span> shifts.
</li></p>

<p><li><b>Refine Mutations</b>. When enabled,
the mutations will be refined, using UQO, following the conformational
search.
For more information on the
refinement protocol, see <a href="#Searching">Searching Conformation and
Sequence Space</a>. By default, this option is enabled.
</li></p>

<p><li><b>RMS Gradient</b>.
Specifies the cutoff RMS of the gradient of the potential energy.
If <b>Refine Mutations</b> is enabled, the final refinement
will be performed until the RMS of the gradient of the potential energy is less
than this specified value.
</li></p>

<p><li><b>Energy Window</b>.
Specifies the cutoff energy.
Unique sequences whose energy for a
mutation site is greater than this value (above the global minimum) will be discarded.
</li></p>

<p><li><b>Maximum Number</b>.
The maximum number of unique sequences that will be output per mutation site.
</li></p>

<p><li><b>Enumerate Site Limit</b>.
When enabled, the number of mutated sites will be enumerated from 1 to
N (where N = <b>Site Limit</b>). When disabled, only mutations of
size N will be created.
</li></p>
</ul>
</td>
</tr>

</table>
</td>
</tr>

</table>

<!-- End of Protein Variants -->


<a name="panel_input"></a>
<h4>Input/Output Options</h4>
The Protein Design application uses the system currently in the
MOE Window for running the simulation. Before running, ensure the system has
been <a href="#Preparing_the_Data">prepared properly</a>.

<p align="center">
<img src="prodesign/panel_input.png" alt="Protein Design Input Options" />


<table cellpadding="5">
<tr>
<td  valign="top"><b>Chains</b></td>
<td valign="top">
The collection of chains which the Protein Design application will use.
All chains containing only dummy atoms will be ignored.
The default is Active Chains.
</td>
</tr>

<tr>
<td  valign="top"><b>MDB File</b></td>
<td valign="top">
It is also possible to use an MDB file containing a
conformational ensemble generated from such applications as
<a href="../apps/confsrch.htm#LowModeMD">LowMode MD</a> or 
<a href="../apps/md.htm">Molecular Dynamics</a>. When an MDB file is defined,
The Protein Design application will loop over the MDB performing
the desired mutations defined in the mutation list.

<p>
To specify a MOE database (MDB) file, enter a filename in the
<span class="menu">Protein Design | MDB File</span> text field.
Alternatively, use the <b>Browse</b> button to select a file
through a file browser.
For Protein Design to run correctly, the
system currently loaded in MOE must
corresponds to the molecule defined in the first molecule field
in the database.
Note that the first molecule field of an MDB file will be
used irrespective of the name, and ignoring all other molecule fields.
</td>
</tr>

<tr>
<td  valign="top"><b>Output</b></td>
<td valign="top">
MDB file output name.
</td>
</tr>

</table>

<a name="display_opt"></a>
<h4>Display Options</h4>
<p>
The Protein Design application <b>Display Options</b> work with the <b>Mutation List</b>
that is displayed for the different <a href="#Variants">Protein Design Variants</a>.
</p>

<p align="center">
<img src="prodesign/panel_display_options.png" alt="Protein Design Display Options" />
</p>


<table cellpadding="5">
<tr>
<td  valign="top"><b>Display</b></td>
<td valign="top">
This option filters which residues are displayed in the <b>Mutation List</b>.
Residues not shown in the list will not be mutated. 
<b>Display</b> has the following options: 
<ul>
<li>All Residues</li>
<li>Selected Atoms</li>
<li>Selected Residues</li>
<li>Selected Chains</li>
<li>Pocket</li>
<li>Interface</li>
<li><i>Sets</i></li>
</ul>
The initial value for <b>Display</b>
is dependent on what is currently loaded. For example, if
atoms are selected in the system, <b>Display</b> will default
to <i>Selected Atoms</i>.
</td>
</tr>

<tr>
<td  valign="top"><b>Contour in MOE window</b></td>
<td valign="top">
When enabled,
a transparent surface is created for the residues selected in the <b>Mutation List</b>.
A green contour indicates that the associated residues will
be mutated, while a magenta color indicates otherwise.
By default, <b>Contour in MOE window</b> is disabled. 
</td>
</tr>

</table>

<a name="ensemble_opt"></a>
<h4>Ensemble Options</h4>
<p>
The Ensemble options listed below are the same for all Protein Design variants.

<div class="floatleft">
<img src="prodesign/ensemble_opt.png">
</div>
If the site being mutated is flexible, then generating an
ensemble of conformations may yield more accurate results. To enable generation
of a conformational ensemble, enable the <b>Ensemble</b> checkbox
in the Protein Design panel. Once enabled, select one of
the <a href="#Ensemble">ensemble methods</a>. Press the Settings
icon to open a pop-up window for ensemble options.


<table cellpadding="5">

<tr>
<td>
<b>Rotamer Explorer</b>
<p><img src="prodesign/rotexp_opt.png">
</td>
<td>
<ul>
<li><b>RMSD Limit</b>. Check for duplicates in the generated ensemble.
Two conformations are considered duplicates if the RMSD between them,
taking molecular symmetry into account, is less than the RMSD Limit
parameter.
<li><b>Energy Window</b>. Energy minimized conformations will be discarded
if their potential energy is greater than
<span class='LaTeX'>\(E_{min} + s\)</span>,
<!--<span class="nobr"><i>E<sub>min</sub></i> + <i>s</i></span>, -->
where <span class='LaTeX'>\(E_{min}\)</span> is the
lowest energy among the minimized conformations and <i>s</i> is the value
specified for Energy Window, in kcal/mol.
<li><b>Conformation Limit</b>. Specifies the limit for the resulting number
of conformations. If a value <i>k</i> is specified, then only the
<i>k</i> lowest
energy conformations will be retained and the rest will be discarded. 
<li><b>Fix residues farther than</b>. Residues farther than this distance 
(&Aring;) will be fixed. Residues within this distance will be searched using
the Rotamer Explorer.
</ul>
</td>
</tr>

<tr>
<td>
<b>LowMode</b>
<p><img src="prodesign/lowmode_opt.png">
</td>
<td>
<ul>
<li><b>Iteration Limit</b>. Maximum number of LowMode iterations.
<li><b>RMSD Limit</b>. Check for duplicates in the generated ensemble.
Two conformations are considered duplicates if the RMSD between them,
taking molecular symmetry into account, is less than the RMSD Limit
parameter.
<li><b>Energy Window</b>. Energy minimized conformations will be discarded
if their potential energy is greater than
<span class='LaTeX'>\(E_{min} + s\)</span>,
<!--<span class="nobr"><i>E<sub>min</sub></i> + <i>s</i></span>, -->
where <span class='LaTeX'>\(E_{min}\)</span> is the
lowest energy among the minimized conformations and <i>s</i> is the value
specified for Energy Window, in kcal/mol.
<li><b>Conformation Limit</b>. Specifies the limit on the resulting number
of conformations. If a value <i>k</i> is specified then only the
<i>k</i> lowest
energy conformations will be retained and the rest discarded. 
<li><b>Fix residues farther than</b>. Residues farther than this distance 
(&Aring;) will be fixed. Residues within this distance will be searched using
LowMode MD.
<li><b>Tether Sidechains</b>. A tether weight of this value will be applied to
side chain atoms of residues within the above cutoff.
A flat bottom tether is used with a maximum deviation of 0.25.
<li><b>Tether Backbone</b>. A tether weight of this value will be applied to
backbone atoms of residues within the above cutoff. 
A flat bottom tether is used with a maximum deviation of 0.25.
<li><b>Add Explicit Solvent</b>. Add explicit water molecules when performing
the ensemble generation. The system will be neutralized by adding the minimum
number of Na+ and Cl- ions. The solvent will not be saved in the output MDB
file.
</ul>
</td>
</tr>

<tr>
<td>
<b>Dynamics</b>
<p><img src="prodesign/dynamics_opt.png">
</td>
<td>
<ul>
<li><b>Simulation Length (ps)</b>: The length of time in picoseconds of the 
Molecular Dynamic production stage. 
<li><b>Energy Window</b>. Energy minimized conformations will be discarded
if their potential energy is greater than
<span class='LaTeX'>\(E_{min} + s\)</span>,
<!--<span class="nobr"><i>E<sub>min</sub></i> + <i>s</i></span>, -->
where <span class='LaTeX'>\(E_{min}\)</span> is the
lowest energy among the minimized conformations and <i>s</i> is the value
specified for Energy Window, in kcal/mol.
<li><b>Conformation Limit</b>. Specifies the limit for the resulting number
of conformations. If a value <i>k</i> is specified, then only the
<i>k</i> lowest
energy conformations will be retained and the rest discarded. 
<li><b>Fix residues farther than</b>. Residues farther than this distance 
(&Aring;) will be fixed. Residues within this distance will be
allowed to move during the Molecular Dynamics simulation.
<li><b>Tether Sidechains</b>. A tether weight of this value will be applied to
side chain atoms of residues within the above cutoff.
A flat bottom tether is used with a maximum deviation of 0.25.
<li><b>Tether Backbone</b>. A tether weight of this value will be applied to
backbone atoms of residues within the above cutoff. 
A flat bottom tether is used with a maximum deviation of 0.25.
<li><b>Add Explicit Solvent</b>. Add explicit water molecules when performing
the ensemble generation. The system will be neutralized by adding the minimum
number of Na+ and Cl- ions. The solvent will not be saved in the output MDB
file.
<li><b>Save Trajectories</b>. All trajectory files will be saved in a folder
of the same name root name as the output MDB file. Each
mutation entry will have its own trajectory file
(<i>rootname</i>/<i>rootname</i>_<i>mutation</i>.mdb).
</ul>
</td>
</tr>
</table>

<a name="affinity_opt"></a>
<p><h4>Affinity Options</h4>
<p>
The Affinity options listed below are the same for all Protein Design variants.
<p>
When affinity is enabled, the <a href="../apps/docking.htm#Scoring"
>GBVI/WSA dG scoring function</a> or MM/GBVI is used to calculate 
the affinity of the molecule defined by <i>Atoms</i> with the rest of the system.
MM/GBVI is used when the molecule defined by <i>Atoms</i> contains more than 10
residues. This is because GBVI/WSA dG
is appropriate only for Protein:Ligand
complexes, and not Protein:Protein ones. In both cases the units for affinity
are kcal/mol.

<table cellpadding="5">
<tr>
<td>
<b>Atoms</b>
</td>
<td>
Used to defined the atoms of binding partner (e.g. a ligand in a 
Protein:Ligand complex or an antigen in an Antibody:Antigen complex).
</td>
</tr>
<tr>
<td>
<b>?</b>
</td>
<td>
Flash the selection state of the atoms defined by <i>Atoms</i>.
</td>
</tr>
</table>

<a name="properties_opt"></a>
<p><h4>Properties Options</h4>
<p>
The Properties options listed below are the same for all Protein Design variants.
<p>
When Properties is enabled, then <a href="../proteins/proprop.htm">
protein properties</a> will be calculated for
the set defined by <i>Chains</i>.
<table cellpadding="5">
<tr>
<td>
<b>Chains</b>
</td>
<td>
Used to define the protein chain(s) for which the protein properties
will be calculated.
</td>
</tr>
<tr>
<td>
<b>?</b>
</td>
<td>
Briefly flash the atoms defined by <i>Chains</i>.
</td>
</tr>
</table>



<a name="Output"></a>
<h3>Working with Output</h3>

<p>
The Protein Design application will write the mutations to a database
containing some or all of the fields described below. 
Please note that for all
mutations the Protein Design application automatically performs a mutation
to self for the wild type protein to calculate relative changes to the
stability and other predicted properties.



<a name="Output_Database"></a>
<h4>The Output Database</h4>
<p><center><table class="titlebar" border="1" width="90%">

<tr>
<th>Field</th>
<th>Description</th>
</tr>

<tr>
<td valign="top"><b><tt>mol</tt></b></td>
<td>The output mutation. When an ensemble is generated, the lowest energy
conformation of the ensemble is saved in this field.</td>
</tr>

<tr>
<td valign="top"><b><tt>mol:conf</tt></b></td>
<td>Packed conformational ensemble of the mutant. This field is only
present if an ensemble was generated.</td>
</tr>

<tr>
<td valign="top"><b><tt>mseq</tt></b></td>
<td>
The molecule sequence number. If an MDB is being used as input, the mseq will
correspond to the entry number in the input database. If the input was
from the system loaded MOE, then the mseq will be 1.

When examining the output database,
entries with different mseq numbers arise
from different input structures.
</tr>

<tr>
<td valign="top"><b><tt>sseq</tt></b></td>
<td>
The site sequence number corresponds to the unique mutation expression used
to generated the entry's mutation.
</td>
</tr>

<tr>
<td valign="top"><b><tt>mutseq</tt></b></td>
<td>
The mutation sequence number. This is used to identify the id of the
mutation within an individual mutation expression. A mutseq of 0 corresponds to
the wild type.
</td>
</tr>

<tr>
<td valign="top"><b><tt>mutation</tt></b></td>
<td>
The mutation from the input required to produce the structure in mol. The
mutation is defined as follows for the example of 1:Y61N.
<ul>
<li><i>1</i> = Chain number
<li><i>Y</i> = Wild Type Single letter residue name
<li><i>61</i> = Residue UID and insertion code
<li><i>Asn</i> = Single letter mutation name
</ul>
Multiple point mutations will have mutations defined by a comma separated list.
</td>
</tr>

<tr>
<td valign="top"><b><tt>Stability</tt></b></td>
<td>
The absolute thermostability of the mutation.
When generating an ensemble, Stability is equal to the Boltzmann average
of the stabilities of the ensemble. 
A more negative value indicates a more stable mutation.
The units of stability are kcal/mol.
</td>
</tr>

<tr>
<td valign="top"><b><tt>dStability</tt></b></td>
<td>
The relative thermostability of the mutation with respect
to the wild type protein.
When generating an ensemble, dStability is equal to the Boltzmann average
of the relative stabilities of the ensemble.
A more negative value indicates a more stable mutation. 
The units of stability are kcal/mol.
</td>
</tr>

<tr>
<td valign="top"><b><tt>Affinity</tt></b></td>
<td>
The absolute binding affinity of the mutation.
This field is only present if the affinity calculation is enabled.
When generating an ensemble, Affinity is equal to the Boltzmann average
of the affinities of the ensemble.
A more negative value indicates a mutation with better affinity.
The units of affinity are kcal/mol.
</td>
</tr>

<tr>
<td valign="top"><b><tt>dAffinity</tt></b></td>
<td>
The relative binding affinity of the mutation to the wild type protein.
This field is only present if the affinity calculation was enabled,
which is the case by default for Resistance Scanning.
When generating an ensemble, dAffinity is equal to the Boltzmann average
of the relative affinities of the ensemble.
A more negative value indicates a mutation with better affinity. 
The units of affinity are kcal/mol.
</td>
</tr>

<tr>
<td valign="top"><b><tt>dAffinity_all</tt></b></td>
<td>
A vector of numbers, where element <i>i</i> corresponds 
to the dAffinity of conformation <i>i</i> in the ensemble.
This field is only present if an ensemble was generated.
</td>
</tr>

<tr>
<td valign="top"><b><tt>S-S</tt></b></td>
<td>The absolute number of disulfide bonds.</td>
</tr>

<tr>
<td valign="top"><b><tt>dS-S</tt></b></td>
<td>The relative number of disulfide bonds to the wild type.
By default, this field is always written when performing
a Disulfide Scan.
This field makes it possible to
to quickly identify entries where a new disulfide bond has been created.
</td>
</tr>

<tr>
<td valign="top"><b><tt>mutexp</tt></b></td>
<td>The mutation expression that the mutation is derived from.</td>
</tr>

<tr>
<td valign="top"><b><tt>dpro_<i>XXX</i></tt></b></td>
<td>
The relative difference between the mutation and wild type for various
<a href="../proteins/proprop.htm">protein properties</a>,
where XXX is the following:
<ul>
<li><i>mass</i>: Protein Mass in kDa.
<li><i>debye</i>: Debye Screening Length.
<li><i>pI_seq</i>: Sequence-based pI Prediction.
<li><i>pI_3D</i>: Structure-based pI Prediction.
<li><i>coeff_280</i>: Extinction coefficient at 280 nm.
<li><i>coeff_fric</i>: Frictional Coefficient.
<li><i>coeff_diff</i>: Diffusion Coefficient.
<li><i>r_gyr</i>: Radius of Gyration.
<li><i>r_solv</i>: Hydrodynamic Radius.
<li><i>sed_const</i>: Sedimentation Constant.
<li><i>eccen</i>: Protein Eccentricity.
<li><i>asa_vdw</i>: Accessible Surface Area (Water Probe).
<li><i>asa_hyd</i>: Hydrophobic Surface Area.
<li><i>asa_hph</i>: Hydrophilic Surface Area.
<li><i>volume</i>: Protein Volume.
<li><i>mobility</i>: Protein Mobility.
<li><i>helicity</i>: Protein Helix Ratio.
<li><i>henry</i>: Henry's Function f(ka).
<li><i>net_charge</i>: Protein Net Charge.
<li><i>app_charge</i>: Protein Charge at Debye Length.
<li><i>dipole_moment</i>: Protein Dipole Moment.
<li><i>hyd_moment</i>: Hydrophobicity Moment.
<li><i>zeta</i>: Zeta Potential at Debye Length.
<li><i>zdipole</i>: Zeta Dipole Moment.
<li><i>zquadrupole</i>: Zeta Quadruple Moment.
<ul>
</td>
</tr>

</table></center>

<a name="Browsing_the_Output_Database"></a>
<h4>Browsing the Output Database</h4>

To use the custom browser for the Protein Design application, choose
<span class="menu">DBV | File | Browse</span>. The following browser
will then appear.

<p>
<center>
<img src="prodesign/browser.png">
</center>
</p>

<blockquote>
<table cellpadding="5">

<tr>
<td  valign="top"><b>Selected</b></td>
<td valign="top">
Toggle selection state of current entry being viewed.
</td>
</tr>

<tr>
<td  valign="top"><b>Browse Selected</b></td>
<td valign="top">
Browse only selected entries.
</td>
</tr>

<tr>
<td  valign="top"><b>Center Mutation</b></td>
<td valign="top">
Similar to
<span class="menu">MOE | RHS | SiteView</span>.
<b>Center Mutation</b> will perform the following rendering changes:
<ol>
<li>Isolate the mutation by hiding distant atoms. Distant backbone ribbons
will also be hidden.
<li>Render the mutation in stick mode; all other atoms will be rendered
in line mode.
<li>Hide non-polar hydrogens and show polar hydrogens.
</ol>
</td>
</tr>

<tr>
<td  valign="top"><b>Keep</b></td>
<td valign="top">
Normally, browsing through the database replaces the currently
browsed entry by another entry.  Pressing <b>Keep</b> causes the
the current entry to be retained in MOE. 
</td>
</tr>

<tr>
<td  valign="top"><b>Stability</b></td>
<td valign="top">
The relative stability compared to the wild type protein.
</td>
</tr>

<tr>
<td  valign="top"><b>Overlay Wild Type</b></td>
<td valign="top">
Overlay wild type protein with current entry.
</td>
</tr>

<tr>
<td  valign="top"><b>Color</b></td>
<td valign="top">
When enabled, the carbon color of the mutant and wild-type structures
are controlled using the <b>Mutant</b> and <b>Wild Type</b> color widgets.
When disabled, the atom coloring will be
controlled using, for example,
the <span class="menu">MOE | Footer | Atom</span> or 
the <a href="../moe/gui/system.htm"><b>System Manager</b></a>.
</td>
</tr>

<tr>
<td  valign="top"><b>Mutant/Wild Type Color</b></td>
<td valign="top">
Allows you to control the color of the wild type and mutant carbon atoms.
</td>
</tr>
</table>
</blockquote>


<a name="Using_MOEsmp"></a>
<h3>Running Protein Design with MOE/cluster (MOE/smp)</h3>

<p>Installing and running MOE in MPU mode is described in the <a
href="../install/smp_inst.htm">MOE/smp manual</a>. If MOE is being run on a
multiprocessor workstation, then this can be setup by
invoking MOE with the appropriate command line
<a href="../install/install.htm#RunSwitch">switches</a>, e.g.:

<pre>
    cmd% moe -mpu 2
</pre>

<p>The above example will run MOE with two MPU nodes, both on the same
workstation. When MOE/cluster (MOE/smp) is being used, multiple mutation sites will be
searched simultaneously.

<a name="Continuing"></a>
<h3>Continuing an Incomplete Protein Design Job</h3>

<p>
Sometimes it is necessary to cancel a computational experiment
due to some unforeseen reason, such as server maintenance,
priority given to another job, or perhaps the computer unexpectedly
shut down. When this occurs, and a Protein Design job was stopped, it is
possible to continue the Protein Design experiment. To do this, start the
SVL Commands window.
</p>

<blockquote>
<span class="menu">MOE | SVL </span>
</blockquote>
<p>
In the SVL window type the following function:
</p>

<blockquote>
<pre>
svl&gt; ProteinDesign '<i>prodesign_outputfile</i>'
</pre>
</blockquote>

<p>
where <i>prodesign_outputfile</i> is the previously running job.
This will automatically continue the Protein Design job specified by 
<i>prodesign_outputfile</i>. Additionally, it will load the structure 
into the MOE window. 
</p>
<p>
To start the job from the beginning and delete all of 
the current data in the MDB file, the following option can be used:
</p>

<blockquote>
<pre>
svl&gt; ProteinDesign ['<i>prodesign_outputfile</i>', [restart : 0]]
</pre>
</blockquote>

<p>
This will start the Protein Design job specified in 
<i>prodesign_outputfile</i> from the beginning, overwriting
all data currently in the output. When using the <tt>restart : 0</tt>
option, all other Protein Design options can be used to modify the
parameters of the Protein Design experiment.
</p>

<p>
When using the Sample Sequence variant of Protein Design, it is
possible to add more mutants to the database even if the
run was completed. This provides the ability to augment the search
space if it was not searched adequately. This can be done using the
the following option:
</p>

<blockquote>
<pre>
svl&gt; ProteinDesign ['<i>prodesign_outputfile</i>', [
    seq_sample_iter : <i>number_of_total_mutants</i>
]]
</pre>
</blockquote>

<p>
This option will force Protein Design to add additional mutants so that
the total (previous + new mutants) is equal to <i>number_of_total_mutants</i>.
This option is similar to the
<b><a href="#SampleSequence">Sequences</a></b> option of the Sample Sequence
variant.

<a name="Restarting"></a>
<h3>Restarting from a Protein Design Output File</h3>

<p>
To enable the ability to see which parameters where used when conducting the
Protein Design experiment, it is possible to restart the Protein Design
application with the options originally used.  To do this, start the
SVL Commands window.
</p>

<blockquote>
<span class="menu">MOE | SVL </span>
</blockquote>
<p>
In the SVL window type the following command:
</p>

<blockquote>
<pre>
svl&gt; run ['prodesign_ui.svl', '<i>prodesign_outputfile</i>']
</pre>
</blockquote>


<p>
This will load the input structure used for
the Protein Design experiment into the MOE window.
It will automatically create atom sets that initialize
the following:

<ul>
<li><i>MOEProDes_Env</i> is used to define the <b>Chains</b> that were 
specified in the <a href="#panel_input">Input/Output Options</a>.
<li><i>MOEProDes_Mut</i> is used for residues defined by the <b>Display</b> 
that was specified in the Display Options of the <a href="#panel_variants">Protein Design Variant</a>.
<li><i>MOEProDes_Lig</i> is used to define the Ligand <b>Atoms</b> that were
used when the <a href="#affinity_opt">Affinity Options</a> were enabled.</li>
<li><i>MOEProDes_Prop</i> is used to define the Property <b>Chains</b>
that were used when the <a href="#properties_opt">Properties Options</a> were enabled.
</ul>


<!-- ------------------------ Examples ---------------------------- -->

<a name="Examples"></a>
<h2>Examples</h2>

<!-- Example 1 --> 

<a name="Example1_Data_Preparation"></a>
<div class="example">
<h3>Example 1: Data Preparation for Protein Design</h3>

<p>
This example demonstrates how to prepare a protein for any of
the Protein Design Variants.
Subtilisins are protein-digesting enzymes which are commonly found in
laundry detergents to allow for efficient cleaning of clothes at
cold temperatures.
</p>

<p>
<ol>
<p><li>
<i>Load the structure into MOE.</i>

<blockquote>
<span class="menu">MOE | File | Open | <tt>$MOE/sample/mol/2st1.pdb</tt></span>
</blockquote>

In the Load PDB File panel, press <b>OK</b>. 
The molecular data comprises the crystallized protein.

</li></p><p><li>
<i>Delete the solvent chain.</i>

<blockquote>
<span class="menu">MOE | RHS | Select | Solvent</span>
<br />
<span class="menu">MOE | RHS | Delete</span>
</blockquote>

<div class="floatright">
<img src="prodesign/example1_2st1_prep.png" alt="Preparation of 2ST1" />
</div>

</li></p><p><li>

<i>Setup the forcefield and assign partial charges.</i>

<blockquote>
<span class="menu">MOE | Footer | Potential Setup</span>
</blockquote>

If the currently loaded forcefield is not Amber10EHT, select it
from the <b>Load</b> menu.  The change is effective immediately.
To assign charges to both the protein and the ligand, click on
<span class="menu">Potential Setup | Fix Charges</span>
Close the Potential Setup panel.

</li></p><p><li>
<i>Correct Structural Issues.</i>
<br />
<blockquote>
<span class="menu">MOE | Compute | Prepare | Structure Preparation</span>
or 
<span class="menu">MOE | Protein | Structure Preparation</span>
</blockquote>

Press <b>Correct</b> in the panel to add hydrogens and correct structural issues
in the protein-ligand complex.

</li></p><p><li>
<i>Optimize hydrogen bond network</i>
<br />

<blockquote>
<span class="menu">Structure Preparation | Protonate3D</span>
</blockquote>

Press <b>OK</b> in the Protonate3D panel to optimize the hydrogen
positions of the protein.
The calculation may take a moment. Once done, you may close the
Structure Preparation panel.

</li></p><p><li>
<i>Save current system as MOE file.</i>

<blockquote>
<span class="menu">MOE | File | Save...| <tt>2st1_prep.moe</tt></span>
</blockquote>

Save the current system as 2st1_prep.moe
for future use in other examples.

</li></p>

</li></p>
</ol>
</p>
<br clear="all">
</div>

<!-- Example 2 -->

<a name="Example2_DisulfideScan"></a>
<div class="example">
<h3>Example 2: Disulfide Scan on 2ST1</h3>
<p>
<p>
This example demonstrates the use of Disulfide Scan.
Subtilisins are protein-digesting enzymes which are commonly found in
laundry detergents to allow for efficient cleaning of clothes at
cold temperatures.
</p>

<p>
<ol>

<p><li>
<i>Load the prepared structure from Example 1, Data Preparation.</i>

<blockquote>
<span class="menu">MOE | File | Open | <tt>2st1_prep.moe</tt></span>
</blockquote>

</li></p><p><li>
<i>Open the Disulfide Scan Panel</i>.

<blockquote>
<span class="menu">MOE | Protein | Design | Disulfide Scan ...</span>
</blockquote>

<div class="floatright">
<img src="prodesign/example2_2st1_site.png" alt="2ST1 Mutation Site" />
</div>

</li></p><p><li>
<i>Identify residues to be mutated.</i>
<br />
<p>
By default since there are no atoms or residues selected, no ligands,
and no multiple protein chains, all residues are displayed in the
Mutation List. To reduce the list to only the desired residues, the
<b>Display</b> controls
should be used. Scroll down in the Mutation List to <tt>SER87</tt> and double
click; this will select the atoms in the MOE Window.  Select
<span class="menu">Display | Selected Atoms</span> in the panel.

</li></p><p><li>
<i>Isolate mutation site.</i>
<blockquote>
<span class="menu">MOE | RHS | SiteView</span>
</blockquote>

SiteView is used to show only selected
mutation sites and nearby residue atoms. Enable <b>Contour in MOE Window</b>
to view the residues to be mutated. In green is SER87, the selected residue,
while in yellow are the residues whose <span class='LaTeX'>\(C_{\beta}\)</span> is within 5.0&Aring;.
Each possible pair of residues in yellow with SER87 will be sequentially
mutated to Cys and the application will attempted to create a disulfide bond.

</li></p><p><li>
<i>Start the Disulfide Scan.</i>
<br />
<p>
Press <b>OK</b>. The output database opens automatically into a
Database Viewer. This database will contain the mutations along with 
the associated wild type protein.
The Disulfide Scan progress is reported to the
<a href="../moe/gui/svlcommands.htm">SVL Commands window</a>
as well as to the top left corner of the MOE Window.

</li></p>
</ol>
<br clear="all">

<h6>Analyzing Results</h6>


<p>
For more information on all the columns of the output database, please
see <a href="#Output_Database">Output Database</a>. The main columns
of interest in this example are: 
<ul>
<li><i>mutseq</i>, which is the id of the
mutation within an individual mutation expression, where 0 indicates the
wild type protein.
<li><i>mutation</i>, which
identifies the exact mutation made for the corresponding mdb entry.
<li><i>dStability</i> which represents the change in stability over the wild-type
protein.
<li><i>dS-S</i> which is the number of disulfide bonds compared to the wild
type protein.
</ul>

<p>
Of the three mutations made, two have created disulfide bonds. To easily
analyze the resulting structures, you can browse the mutants by
<span class="menu">DBV | File | Browse ...</span>, this will start the
customized <a href="#Browsing_the_Output_Database">
DB Browser for Protein Design</a>.

<p>
The disulfide bonds created in this example
have been created in Subtilisin and reported in
[Katz&nbsp;1986].

<div class="floatright">
<img src="prodesign/example2_2st1_mutation.png" alt="2ST1 Mutation" />
</div>

<ol>
<p><li>
<i>Start the Protein Design Browser</i>.

<blockquote>
<span class="menu">DBV | File | Browse ...</span>
</blockquote>

</p></li><p><li>
<i>Hide the original 2st1 structure</i>.

<blockquote>
<span class="menu">MOE | System | 2ST1 </span>
</blockquote>

</p></li><p><li>
<i>Isolate the mutation</i>.
<br>

<p>
Go to entry 4 in the MDB by clicking the next arrow in the browser. Follow
this by enabling <b>Center Mutation</b> and <b>Overlay Wild Type</b>. This
will isolate the mutation in the center of the screen, similar to
<a href="../apps/ligx.htm#SiteView"
>SiteView</a> and allow you to easily compare the wild type
conformation shown in gray and the mutant shown in green.

</p></li>
</ol>
<br clear="all">

</div>

<!-------------------------- Example 3 ---------------------------------->

<a name="Example3_ResidueScan"></a>
<div class="example">
<h3>Example 3: Residue Scan on 1VFB</h3>
<p>
<p>
1VFB is a crystal structure of the Fv fragment of an anti-hen egg white
lysozyme antibody D1.3 antigen bound form. This example will be use to
highlight how residue scanning can be used to modulate antibody
affinity. This example assumes that 1VFB (found in
<tt>$MOE/sample/mol/1vfb_prep.moe</tt>)
has been prepared
using similar steps as in <a href="#Example1_Data_Preparation">Example 1</a>.
</p>

<p>

<div class="floatright">
<img src="prodesign/example3_1vfb_h1loop.png" alt="1VFB H1 Loop" />
</div>

<ol>

</li></p><p><li>
<i>Load the prepared structure into MOE.</i>

<blockquote>
<span class="menu">MOE | File | Open | <tt>1vfb_prep.moe</tt></span>
</blockquote>

</li></p><p><li>
<i>Render Structure to help identify key mutations.</i>
<br>
<p>
To help reduce the number of residues to mutate, it is beneficial to render
the protein in such a way so that the antigen and antibody are clearly
distinguished. 

    <ol type="i">
    <li><i>Annotate secondary structure using antibody detection.</i>

    <blockquote>
    <span class="menu">SE | Annotate | Antibody | CCG</span>
    </blockquote>

    <li><i>Color atoms by chain.</i>

    <blockquote>
    <span class="menu">MOE | Footer | Atoms | Color by Chain</span>
    </blockquote>

    <li><i>Select H1 loop</i>
    
    <blockquote>
    <span class="menu">MOE | Popup | Select | Set | <i>CDR_H1</i></span>
    </blockquote>

    <li><i>Color H1 Loop atoms Green.</i>

    <blockquote>
    <span class="menu">MOE | Footer | Atoms | Green</span>
    </blockquote>

    <li><i>Isolate H1 loop</i>
    
    <blockquote>
    <span class="menu">MOE | RHS | SiteView</span>
    </blockquote>

    </ol>

<br clear="all">

<div class="floatright">
<img src="prodesign/example3_1vfb_panel.png" alt="Residue Scan Panel" />
</div>

</li></p><p><li>
<i>Open the Residue Scan Panel</i>

<blockquote>
<span class="menu">MOE | Protein | Design | Residue Scan ...</span>
</blockquote>

</li></p><p><li>
<i>Identify residues to be mutated.</i>
<br />
<p>
By default, since there are no atoms or residues selected and
no ligands, and since 1VFB
contains multiple protein chains, only the interface residues are shown in the
Mutation List. Since this example focuses on the H1
CDR loop region, change the <b>Display</b> option:

<blockquote>
<span class="menu">Residue Scan | Display | <i>CDR_H1</i></span>
</blockquote>

<p>
There should be 10 residues shown in the Mutation list. Select all residues
and click on <b>Clear</b> in the Mutation Editor. By removing
the mutations for these residues, they will not be searched. 
Most of the residues in the H1 loop do not interact with the antigen and
therefore will be excluded in this example.  We will focus only on Gly31,
since there is space for a larger mutation to fit.
Examination of the selected mutation site reveals that there are many
charged residues present in the antigen (e.g. Arg112 and Lys116). 
To limit the mutation space, we will focus on
creating either novel salt bridges by mutating to Glu or Asp, or Pi-Cation
interactions by mutating to an aromatic residue.
Select Gly31 in the list, and in the mutation editor box type <b>DEFWY</b>
and click <b>Apply</b> to limit mutations to Asp, Glu, Phe, Trp and Tyr.

</li></p><p><li>
<i>Enable binding affinity calculation.</i>
<p>
Since the antigen contains more than 10 residues, the affinity calculation
will use MM/GBVI scoring. This scoring function should only be used as
a pass/fail type score, where the user will defined the pass/fail mark. 

<blockquote>
<span class="menu">Residue Scan | Affinity</span>
</blockquote>

Enable the checkbox to allow the affinity calculation. Select the 
antigen (the 1VFB.C chain) in the Sequence Editor.

<blockquote>
<span class="menu">Residue Scan | Affinity | Atoms | Selected Chains</span>
</blockquote>
To calculate protein properties for only the antibody chain, ensure first
that Properties is enabled. With Properties enabled,
the Properties Chains pulldown becomes available; choose Unselected Chains
(the antigen chain was selected in the previous step).
<blockquote>
<span class="menu">Residue Scan | Properties | Chains | Unselected Chains</span>
</blockquote>

</li></p><p><li>
<i>Start the Residue Scan.</i>
<br />
<p>
Press <b>OK</b>. The output database opens automatically into a
Database Viewer. This database will contain the mutations along with 
the associated wild type protein.
The Residue Scan progress is reported to the
<a href="../moe/gui/svlcommands.htm">SVL Commands window</a>
as well as
to the top left corner of the MOE Window. This calculation will take a few
minutes.

</li></p>
</ol>

<h6>Analyzing Results</h6>

<p>
For more information on all the columns of the output database please
see <a href="#Output_Database">The Output Database</a>. The main columns
of interest in this example are: 
<ul>
<li><i>mutseq</i>, which is the id of the
mutation within an individual mutation expression where 0 indicates the
wild type protein.
<li><i>mutation</i>, which
identifies the exact mutation made for the corresponding mdb entry.
<li><i>dStability</i> which represents the change in stability over the
wild-type protein.
<li><i>dAffinity</i> which is the relative affinity of the mutation compared
to the wild-type protein.
</ul>

<p>
To better understand the gains or losses in
affinity, the <a href="#Browsing_the_Output_Database">Protein
Design Browser</a> can be used. 

<div class="floatright">
<img src="prodesign/example3_1vfb_mutation.png" alt="1vfb Mutation" />
</div>

<ol>
<p><li>
<i>Start the Protein Design Browser</i>.

<blockquote>
<span class="menu">DBV | File | Browse ...</span>
</blockquote>

</p></li><p><li>
<i>Hide the original 1VFB structure</i>.

<blockquote>
<span class="menu">MOE | System | 1VFB </span>
</blockquote>

</p></li><p><li>
<i>Isolate the mutation</i>.
<br>

<p>
Enable <b>Center Mutation</b>
and <b>Overlay Wild Type</b>. This
will isolate the mutation in the center of the screen, similarly to
<span class="menu">MOE | RHS | <a href="ligx.htm#SiteView">SiteView</a></span>,
and allow you to easily compare the wild type
conformation shown in gray to the mutant shown in green.

<p>
Using these option will allow you to easily view and asses the mutations
made and whether any new interactions were formed.

</p></li>
</ol>
<br clear="all">

</div>

<a name="References"></a>
<h2>References</h2>

<p>
<table width="100%" cellpadding="5">
<tr>
<td valign="top">[Dehouck&nbsp;2009]</td>
<td valign="top">
Dehouck, Y., Grosfils, A., Folch, B., Gilis, D., Bogaerts, P., Rooman, M.;
Fast and accurate predictions of protein stability changes upon
mutations using statistical potentials and neural networks:
PoPMuSiC-2.0;
<i>Bioinformatics 25</i> (<b>2009</b>) 2537&ndash;2543.
</td></tr>

<tr>
<td valign="top">[Goldstein&nbsp;1994]</td>
<td valign="top">
Golstein, R.F.;
Efficient rotamer elimination applied to protein side
chains and related spin glasses;
<i>Biophys J. 66</i> (<b>1994</b>) 1335&ndash;1340.
</td></tr>

<tr>
<td valign="top">[Guerois&nbsp;2002]</td>
<td valign="top">
Guerois, R., Nielsen, J.E., Luis Serrano, L.;
<i>J. Mol. Biol. 320</i> (<b>2002</b>) 369&ndash;387.
</td></tr>

<tr>
<td valign="top">[Katz&nbsp;1986]</td>
<td valign="top">
Katz, B.A. and Kossiakoff, A.;
<i>J. Biol. Chem. 261</i> (<b>1986</b>) 15480&ndash;15485.
</td></tr>

<tr>
<td valign="top">[Labute&nbsp;2008]</td>
<td valign="top">
Labute, P.;
Protonate3D: Assignment of Ionization States and Hydrogen Coordinates
to Macromolecular Structures; <i>Proteins 75</i> (<b>2008</b>) 187&ndash;205.
</td></tr>

</table>

<a name="See_Also"></a>
<h2>See Also</h2>

<a href="structprep.htm">Structure Preparation</a><br />
<a href="../proteins/proprop.htm">Protein Properties</a><br />
<a href="../apps/mutanal.htm">Mutation Analysis</a>
<a href="../proteins/probuild.htm">Protein Builder and Rotamer Explorer</a>


<noscript>
	<hr noshade="noshade" />
	<font size="2"> Copyright &copy; 1997&ndash;2015
	<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a> </font>
</noscript>
<div id="MoeFooter"></div>
</body>
</html>
