<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<!--
!!    MOE 2010.10 On-Line Manuals
!!    COPYRIGHT (C) 2010
!!        CHEMICAL COMPUTING GROUP INC.  ALL RIGHTS RESERVED.
!!-->

<!--
!!	view2d.htm
!!
!!	01-oct-1010 (cc) minor edits
!!	06-jun-2010 (kk) new screenshot; minor edits
!!      02-sep-2009 (sm) new screenshots
!!	06-oct-2008 (kk) minor grammar edits
!!  	10-sep-2008 (ac) created
!!-->

<head>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css"
href="../manstyle.css" />



<meta panel 2D Molecules>
<meta panel Depict as 2D>

<title>2D Molecules</title>
</head>

<body bgcolor="#ffffff">
<a href="../index.htm"><img src="../images/logo.gif"
alt="CCG Logo" align="right" width="174" height="55" border="0" /></a>
<h4>MOE 2010.10</h4>
<h2>2D Molecules</h2>
<hr noshade="noshade" />

<ul>
  <li><a href="#Depiction">Structure Depiction</a></li>
  <li><a href="#Display">Display Options</a></li>
  <li><a href="#Acquire">Acquiring Molecules</a></li>
  <li><a href="#Manage">Managing Molecules</a></li>
  <li><a href="#Output">Output Graphics</a></li>
  <li><a href="#SeeAlso">See Also</a></li>
</ul>

<p>The <b>2D Molecules</b> panel is a repository for a small set of molecular
structures, which are rendered onscreen in 2D, one at a time. The manner in
which they are displayed can be adjusted with a number of options, which are
applied to the whole set.

<p>The molecule collection can be acquired from a number of sources, such as the
MOE main window, the clipboard, or various file formats. The molecules can be
transferred or exported to various destinations, such as the MOE main window,
database files, raster or vector graphics files, or various clipboard flavors.

<p>The molecules can be displayed using their original layout coordinates, or
they can be redepicted automatically, in various modes, including unconstrained
aesthetics, flattening to match original coordinates, flattening with group
alignment, or alignment according to designated common scaffolds. Various
display options can be changed, such as atom coloring, background, bond style,
hydrogen rendering, stereo labels, and supplementary annotations.

<p>The 2D molecule viewing panel can be invoked from the menu <nobr><b>MOE |
Compute | 2D Molecules</b></nobr>, <nobr><b> MOE | RHS | Ligand 
| 2D Molecules</b></nobr> or from the command line by <nobr><tt>run
'view2d.svl'</tt></nobr>:

<p align="center"><img src="view2d/panel.png"></p>

<h2><a name="Depiction">Structure Depiction</a></h2>

<p>The molecules loaded into the current set are drawn in 2D diagram style,
using either the original source coordinates, or calculated 2D layout
coordinates. The various options for the <b>Depict</b> mode are as follows:

<p><table border="0" cellspacing="5" width="100%">

<tr><td valign="top"><b>Yes</b>

</td><td>The coordinates for each molecule will be replaced by those calculated
by the automatic 2D layout algorithm, as implemented by
the <tt>DepictionCoordinates</tt> function.

</td></tr><tr><td valign="top"><b>No</b>

</td><td>The original coordinates will always be used, even if they are
unsuitable for 2D depiction.

</td></tr><tr><td valign="top"><b>Maybe</b>

</td><td>The original coordinates will be replaced with automatic layout
coordinates for molecules which appear to be 3D (i.e. the Z-coordinate is not
always zero).

</td></tr><tr><td valign="top"><b>Flatten</b>

</td><td>The 2D coordinates will be recalculated using the
<tt>Flatten3DStructure</tt> function, which produces an aesthetically viable
diagram layout, which matches the original inter-atomic distances as closely as
possible without violating any rules of aesthetics.

</td></tr><tr><td valign="top"><b>Align</b>

</td><td>The molecules are rendered collectively. This option applies only in
cases where the original molecules are all aligned in 3D space. First, each
molecule is "flattened", in order to produce a 2D layout which resembles the
original 3D embedding. Then, each molecule is superimposed onto its original
coordinates, and these are collectively rotated onto the page. The result is
that the 2D coordinates of the molecule set are directly comparable, in an
approximate way.

</td></tr><tr><td valign="top"><b>Scaffold</b>

</td><td>Each of the molecules is aligned on the page according to a notion of
common scaffold. Specification of the scaffold is done in a sub-section of the
panel:

<p><div align="center"><img src="view2d/scaffold.png"></div>

<p>If the <b>Auto</b> checkbox is on, then the scaffolds are calculated,
analyzed and depicted automatically, using the same algorithm as is used by the
<a href="sar.htm">Structure Activity Report</a>. Otherwise, when the 
<b>Auto</b> checkbox is off, the text box to its right is used to provide 
the means of ascertaining the common scaffold. If a <tt>SMARTS</tt> pattern is
entered, it will be used to identify the scaffolds.

<p>Pressing the
<a href="../moe/gui/mouse.html#MousePopupButton">mouse popup button</a>
on the text box brings down a list of recently-used
<tt>SMARTS</tt> patterns, and also two special options. <b>user-selection</b>
examines the selected atoms, and computes a <tt>SMILES</tt> string,
which is used to identify the scaffold. The <b>predefined</b> option applies
only when the input molecules already have their scaffold definitions encoded
within the <i>atom number</i> field, which can be imported from MDL MOL and SD
files.

</td></tr></table>

<p>To change the depiction mode, select the new option, and press the
<b>Apply</b> button. All molecules will be updated.

<h2><a name="Display"></a>Display Options</h2>

<p>A variety of options can be selected which influence the way in which
molecules are displayed on the screen, and also how they are exported in
graphics formats.

<p><table border="0" cellspacing="5" width="100%">

<tr><td valign="top"><b>Rotate All</b>

</td><td>To rotate all of the current molecules by a selected value in degrees
(anti-clockwise).

</td></tr><tr><td valign="top"><b>Invert</b>

</td><td>If checked, then the coordinates of the 2D molecules will be inverted.
Any wedge bonds will be adjusted accordingly.

</td></tr><tr><td valign="top"><b>Bonds</b>

</td><td>The rendition of bond orders for certain functional groups may be
modified. The options are:

<ul>

<li>Default

<li>Kekul&eacute;

<li>Resonance

<li>Skeleton

</ul>

<p>By default, all bonds are displayed in their incoming style. For formats
which require bond orders to be calculated, these consist of single, double and
triple bonds, with Kekul&eacute; forms for aromatic rings. When the
<b>Kekul&eacute;</b> option is selected, any aromatic rings which were
originally specified in resonance form are converted to alternating
single/double bonds. Conversely, the <b>Resonance</b> option will convert
Kekul&eacute; style aromatic rings to the "ring" resonance notation.

<p>The <b>Skeleton</b> option displays all bonds as single lines, regardless of
their bond order.

</td></tr><tr><td valign="top"><b>Font</b>

</td><td>
There are 3 settings for font size, with the default being the smallest.

</td></tr><tr><td valign="top"><b>Hydrogen</b>

</td><td>Hydrogen atoms for incoming molecules may be specified explicitly
as separate atoms, or implicitly as part of a heteroatom heavy atom label,
or not provided at all.  The options for hydrogen display are as follows:

<blockquote>
<table width="80%" border="0" cellpadding="3">

<tr>
<td valign="top"><b>Inline</b></td>

<td valign="top">
Hydrogen counts are incorporated into heteroatom heavy
atom labels.  They are not displayed when attached to
carbon atoms. In some cases, such as dihydrogen, or hydrogens attached
to chiral ring junctions, explicit hydrogens are included in the diagram
as distinct atoms.
This is the most common convention for complete-molecule sketches.
</td>

</tr>
<tr>

<td valign="top"><b>Explicit</b></td>

<td valign="top">
All explicit hydrogens are drawn as distinct atoms. Note that this
will not cause explicit hydrogens to be added to the structure, i.e. if
the incoming structure is hydrogen-suppressed, then this option will
have no effect.
</td>

</tr>
<tr>

<td valign="top"><b>None</b></td>

<td valign="top">
All hydrogen representation is
excluded from the diagram, whether explicit or implicit.
</td>

</tr>
<tr>

<td valign="top"><b>Chiral</b></td>

<td valign="top">
Any explicit hydrogen atom attached to a chiral center is drawn as a distinct
atom, while other hydrogen atoms will be treated in the default way.
<b>Note:</b> To show chirality in the window, the
<a href="../moe/molsystems/build.html#MolecularGeometry">chirality constraint</a>
must be set to the current geometry.
</td>

</table>
</blockquote>

</td></tr><tr><td valign="top"><b>Stereo</b>

</td><td>Chirality and double bond stereochemistry are traditionally annotated
by using wedges and planar layout, respectively, but are ideally also
represented by parity-based schemes. If the <b>R/S</b> button is selected, then
the diagrams will be annotated with Cahn-Ingold-Prelog (CIP) notation, which is
<b>R</b> or <b>S</b> for chiral centers, and <b>E</b> or <b>Z</b> for
stereoactive double bonds. 


<p>Alternatively, if the <b>Parity</b> button is selected, then the diagrams are
annotated with odd/even notation, where one dot encodes odd, and two dots
encodes even. This notation has cheminformatic rather than chemical meaning.

<p><b>Note:</b> To show chirality in the window, the
<a href="../moe/molsystems/build.html#MolecularGeometry">chirality constraint</a>
must be set to the current geometry.

</td></tr><tr><td valign="top"><b>Color&nbsp;Atoms</b>

</td><td>When checked, the heteroatoms are drawn according to a predefined color
scheme, which corresponds to that used for atoms in the MOE main window, e.g.
oxygen is red, nitrogen is blue, etc.

</td></tr><tr><td valign="top"><b>Foreground</b>

</td><td>The default foreground color may be changed, which is black by default.
The foreground color is used to draw all atoms and bonds, unless specified
otherwise, e.g. if the atoms are explicitly colored, or if scaffolds are
defined.

</td></tr><tr><td valign="top"><b>Background</b>

</td><td>The paper color for rendering the molecules may be changed. The default
paper color is transparent, i.e. white when displayed onscreen, but when
exported using any graphics format which supports transparency, the bounding box
of the molecule will not obscure any underlying content.

</td></tr><tr><td valign="top"><b>Annotations</b>

</td><td>The annotation options include a color, which defaults to blue, and a
choice of additional annotation features is presented. The default, <b>None</b>,
does not add any of these features to the diagram.

<p>For incoming molecule formats which support an <i>atom number</i> field, such
as MDL MOL, the <b>Acol</b> option can be selected, which causes any atom with
an atom number of other than zero to be drawn using the annotation color, which
overrides any other coloring scheme. This is particularly useful for showing
reaction centers, or precomputed scaffolds.

<p>Alternatively, the <i>atom number</i> field can be shown explicitly with
drawn-in labels, by selecting the <b>Anum</b> option. The labels are arranged
around the atoms themselves.

<p>To display the index of each atom, select the <b>Index</b> options. This is
useful for debugging cheminformatics algorithms, which refer to atoms by
indices. Note that if the input molecule contains explicit hydrogens which are
not being displayed, the original atom numbers are preserved, so the numbers
will be non-consecutive where hydrogen atoms are removed.

</td></tr><tr><td valign="top"><b>Scaffolds</b>

</td><td>When the <b>Depict</b> mode is set to <b>Scaffolds</b>, the drawing
color for scaffold atoms can be selected, and the method of scaffold selection
can be specified, as described previously.

</td></tr><tr><td valign="top"><b>Name</b>

</td><td>Molecule formats generally provide some way to record a name for an
explicit molecule species, and this is displayed in the <b>Name</b> text field
for the selected molecule. To change the name of a molecule, edit the text, and
press ENTER, or click <b>Apply</b>.

</td></tr><tr><td valign="top"><b>Rotate</b>

</td><td>The rotation of just the currently selected molecule can be modified
using this input control, and is applied immediately to the diagram. An explicit
number can be entered, or the wheel control may be used. 

</td></tr></table>

<p>All of the options, with the exception of <b>Rotate</b>, do not take effect
until the <b>Apply</b> button is pressed.

<h2><a name="Acquire">Acquiring Molecules</a></h2>

<p>When the panel opens, the default behavior is to examine the contents of the
MOE main window, and for each chain which is not too large, nor considered to be
solvent, convert the input into a sketch representation. This collection of
molecules comprises the initial starting set, and may be empty. If there are
atoms selected in the main window, then only components for which at least one
atom is selected are included in the set.

<p>Additional molecules can be added to the current set by clicking on the
<b>Acquire</b> radio button, which opens up a set of action buttons:

<p><table border="0" cellspacing="5" width="100%">

<tr><td valign="top"><b>FromMOE</b>

</td><td>The contents of the MOE main window will be examined, in the same way
as when the panel starts. Any new molecular species will be appended to the
current list of molecules. No duplicate checking is done, so pressing this
button multiple times will continually grow the set.

</td></tr><tr><td valign="top"><b>Clipboard</b>

</td><td>The system clipboard will be examined for formats which are readily
translated into chemical sketch representations. Such formats include MOE, MDL
MOL and SD files, and ISIS/ChemDraw sketch data. Non-molecular data, such as
pictures or text, will not be processed.

<p>For single-sketch formats, such as MOL or ISIS/ChemDraw, the whole chemical
species will be added to the set as a single entity. For MOE data, the appended
molecules will be split by chain, one entity each. For MDL SD files, one entity
is added for each record.

<p>Entities with more than 1000 heavy atoms are excluded, and a maximum of 100
molecules may be loaded at any time.

</td></tr><tr><td valign="top"><b>ClipComp</b>

</td><td>The same as the <b>Clipboard</b> button, except that each entity is
further separated into individual connected components. This is useful when a
number of molecules are drawn into the same sketch.

</td></tr><tr><td valign="top"><b>File...</b>

</td><td>Prompts for an input file to be selected, then reads that file and adds
each entity to the current set. Supported file formats include MOE, MDL MOL, MDL
SDF, ISIS/ChemDraw, and MDB.

</td></tr></table>

<h2><a name="Manage">Managing Molecules</a></h2>

<p>The 2D Molecules panel stores a set of between 0 and 100 molecules, one of
which is displayed at a given time. When there are currently 2 or molecules
loaded, part of the molecule display is taken up by a button-bank:

<p><div align="center"><img src="view2d/molbtn.png"></div>

<p>Each of the buttons shows a thumbnail depiction of the molecule that it
represents. Clicking on any of the buttons makes that molecule the current
molecule. The button for the currently selected molecule is shown with the
selected background color (or white if transparent), while the non-selected
molecules have standard gray buttons. The images shown in the buttons reflect
the display settings in the same way as for the larger display underneath. When
there are more than 10 molecules loaded, navigation buttons are added to their
right.

<p>As previously mentioned, the initial set of molecules is derived from the
content of the MOE main window. New molecules can be added to the set using the
<b>Acquire</b> buttons, as described above.

<p>The currently selected molecule from the loaded set is shown in the main
molecule display area. This will be blank if there are no molecules loaded.

<p>If there is more than one compound in the loaded set, it can be selected by
clicking on the miniature icon button, above the main molecule display. If there
are more icons than will fit on one page, the left, right, start and end buttons
can be used to navigate through the set.

<p>To remove the currently selected molecule from the loaded set, click the
<b>Remove</b> button. To clear out all of the loaded molecules, and optional
replace them with the contents of the MOE main window, click the <b>Reset</b>
button.

<h2><a name="Output">Output Graphics</a></h2>

<p>The molecules shown in the panel may be exported to a variety of destinations
in various formats, and to the extent that the format allows, will resemble what
is seen in the panel.

<p>Molecules can be exported as molecular object formats, raster graphics and
vector graphics. They can be exported to files or copied to the clipboard, where
platform capabilities allow. Molecular formats which allow multiple entries
incorporate all loaded molecules, while those which do not provide the option of
exporting one file per molecule. Molecules can also be transferred back to the
MOE main window, and printed in tiled form.

<h3>Clipboard</h3>

<p>Select the <b>Copy</b> radio button to bring up the clipboard submenu, which
consists of action buttons, which correspond to the respective format. All
clipboard copy operations function on the currently selected molecule only.

<p>The following formats can be used to copy data onto the clipboard:

<ul>

<li>MOE
<li>MOL
<li>EMF+
<li>SVG
<li>BMP

</ul>

<p>Note that EMF+ and BMP formats are only supported for the clipboard on
Microsoft Windows platforms. On POSIX systems (Linux, Mac OS X, etc.), these
formats can be generated, but only by writing to files, as is described below.

<p>When copying EMF+ graphics to the clipboard on Microsoft Windows platforms,
it is possible with many presentation applications to paste the graphic into a
document. With some applications, such as older versions of Microsoft Office, it
may be necessary to use a menu action such as "Paste Special", and select the
EMF+ portion of the clipboard data. The resulting graphic will be rendered with
anti-aliased lines and fonts, scale seamlessly, and print at maximum resolution.

<h3>Export</h3>

<p>The current set of molecules can be written to one file, or a group of files,
or in some cases, the currently selected molecule may be written to a single
file, depending on the format. The following export formats are supported:

<ul>

<li>MOE
<li>MOL
<li>SDF
<li>EMF+
<li>SVG
<li>PNG
<li>JPG
<li>GIF
<li>BMP

</ul>

<p>The MOE and SDF formats are naturally capable of holding multiple molecules
(separate chains and records, respectively), and so when the currently loaded
set has more than one entry, only a single file is required to export them.

<p>All of the other formats are not readily compatible with multiple entries,
and thus a series of files will be written, one for each molecule.
The output filenames will be a series based on the filename prefix
that is entered in the textfield of the export panel:

<p><div align="center"><img src="view2d/export.png"></div>

<p>Pressing <tt>&lt;Return&gt;</tt> or <b>OK</b> will
populate the lower part of the panel with the
names of the files that are to be written.
Pressing <b>OK</b> once more will initiate the export operation.
</p>

<p>If only the current molecule is to be exported, turn on the
<b>Current Molecule Only</b> checkbox.  Just a single file will be created.

<h3>Print</h3>

<p>The loaded molecules can be arranged in a printable format, which arranges
the molecules on pages in tiled mode, in a print preview dialog:

<p align="center"><img src="view2d/print.png"></p>

<p>The molecules are rendered with display options at the time the print panel
was launched. The display of molecule names, numbers and borders between tiles
can be toggled with the respective options.

<h3>Graphics Formats</h3>

<p>There are a number of formats which can be used to capture the visual
representation of a molecule. These formats are divided between raster and
vector formats. Raster formats, often described collectively as "bitmaps" or
"pixmaps", are matrices of pixels, each of which have a specific color value.
They tend to be well supported and easy to deploy using the largest possible
variety of image presentation software, but they use up an amount of disk space
proportional to the resolution, cannot be scaled to a larger size than that
at which they were generated, and not all software is capable of gracefully
shrinking raster graphics. Raster graphics also do not print well unless they
are generated at a particularly high resolution.

<p>Vector graphics, on the other hand, can be grown or shrunk to any resolution
without penalty, and so in principle can be displayed on any device without
prior knowledge of the destination. There are few universally supported vector
formats, however, and the output quality is heavily dependent on the properties
of the rendering software, if it supports the format at all. Many vector formats
are designed for printers, but do not support features such as alpha blending,
and may be rendered poorly on-screen due to lack of anti-aliasing. Problems are
also often encountered with portability of fonts, and the absence of a vector
graphics standard for web browsers can be an issue.

<p>The 2D Molecules panel can generate graphics in the following formats:

<p><table border="0" cellspacing="5" width="100%">

<tr><td valign="top"><b>PNG</b>

</td><td>Portable Network Graphics. Raster. Supports alpha transparency levels
and lossless compression. Compatible with almost all graphics manipulation and
presentation software, including all modern browsers.

</td></tr><tr><td valign="top"><b>JPG</b>

</td><td>Joint Photographic Experts Graphics format. Raster. Does not support
any kind of transparency, and does not support lossless compression, which makes
it a relatively poor choice for molecular diagrams, but is useful for some
compatibility purposes.

</td></tr><tr><td valign="top"><b>GIF</b>

</td><td>Graphical Interchange Format. Raster. Supports binary
transparency and lossless compression, but is restricted to 8-bit paletted
color. This format is adequate for molecular diagram graphics, but use of
transparent backgrounds it not recommended.

</td></tr><tr><td valign="top"><b>BMP</b>

</td><td>Microsoft Windows Bitmap format. Raster. Supports 24-bit color and is
lossless, but even though modern variants of the format include compression,
this is not used, in order to maximize compatibility. Lack of well supported
transparency, unnecessarily large output files, and lack of support from web
browsers make this format mainly useful when interacting with certain Windows
applications.

</td></tr><tr><td valign="top"><b>EMF+</b>

</td><td>Enhanced Metafile Plus format. Vector. This is a Windows-specific
variant of the Enhanced Metafile format, which has been extended to include
advanced features such as alpha-level transparency and anti-aliasing, which are
critical to rendering molecular graphics equally well on all devices, at any
level of magnification. While EMF+ graphics can be generated from MOE on any 
supported platform, support for rendering EMF+ is largely confined to Microsoft
Windows, from 2000 onwards. This format should not be confused with EMF, which
is closely related, but not as sophisticated.

</td></tr><tr><td valign="top"><b>SVG</b>

</td><td>Scalable Vector Graphics format. Vector. XML-based. This is an
international standard for sophisticated vector graphics, the features of which
are a superset of those needed to render molecules in 2D. Support for this
format should not be taken for granted, and is often incomplete. Rendering will
not necessarily take advantage of high quality anti-aliasing, and inconsistent
support for embedded fonts may cause atom labels to vary in appearance depending
on the device. However, a fully compliant SVG renderer can reproduce graphics of
a quality at least as good as is seen in the 2D Molecules panel. 

</td></tr></table>

<p>Before exporting graphics or copying to the clipboard, it is often useful to
control the properties of the output, in particular the bounding box or
resolution of the resulting images. For raster formats, this is particularly
important, because the resolution cannot be increased after the file is written.
For vector formats, it can be convenient to specify the default graphic size,
even though the images can subsequently be magnified without penalty.

<p>When the <b>Image</b> radio button is pressed, the following options are
provided:

<p align="center"><img src="view2d/image.png"></p>

<p>When the property of <b>Boundary</b> is set to <b>None</b>, the size of
each molecular graphic is determined by using industry standard settings
for the number of angstroms (in the units of the diagram) per millimeter
(units of the output graphic). The information which is required is
the number of dots the intended rendering device can display per unit
of length, which is typically given as dots-per-inch (DPI). A value
of 300 dpi is sufficient for a low budget printer, but far too high
for contemporary monitors. It is safer to overestimate the resolution
than to underestimate it, because most software is capable of scaling
raster images down, but it is not possible to scale it up with adequate
results. Unfortunately, target resolution specifications within graphics
formats are not always reliable: many rendering engines will display
a raster graphic at 1:1 scale by default, which can be inconvenient.
For many purposes, it is advisable to select the correct resolution
for the device. The same advice applies to vector formats, though it is
only for convenience.

<p>If the <b>Boundary</b> is set to <b>Box</b>, then the parameters which
are requested are width (<b>W</b>) and height (<b>H</b>). For raster
formats, this measurement is in pixels, and for vector formats it is an
arbitrary unit of dimension roughly equivalent to screen pixels. When
a bounding box is used, instead of a resolution, the dimensions of the
output graphics are guaranteed to be that which is specified, for all
molecules. This is particularly useful for a series, e.g. when rendering
in order to place inside a table which needs to have a uniform cell size.

<a name="SVLCommands"></a>
<h2>SVL Commands</h2>

<a class="svl" href="../moe/fcnref/depictor.htm">DepictionCoordinates</a><br />
<a class="svl" href="../moe/fcnref/depictor.htm">Draw2DStructure</a><br />
<a class="svl" href="../moe/fcnref/depictor.htm">Draw2DStructureCTAB</a><br />
<a class="svl" href="../moe/fcnref/depictor.htm">Flatten3DStructure</a>

<a name="SeeAlso"></a>
<h2>See Also</h2>

<a href="sar.htm">Structure Activity Report</a>

<p><a href="../index.htm">MOE Table of Contents</a></p>
<hr noshade="noshade" />
<a href="http://www.chemcomp.com"><img src="../images/flogo.gif"
alt="CCG Logo"
align="left" width="30" height="30" border="1" hspace="5" vspace="3" /></a>
<font size="2">
<a href="../legal.html">Copyright</a> &copy; 1997-2010
<a href="http://www.chemcomp.com">Chemical Computing Group Inc.</a><br />
<a href="mailto:info@chemcomp.com"><i>info@chemcomp.com</i></a>
</font>
</body>
</html>
