<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta http-equiv="Content-Style-Type" content="text/css">
<meta name="categories" content="tasks_menu applications_menu">
<meta name="product" content="{product}">
<meta name="profile" content="!BioLuminate">
<link rel="stylesheet" href="../support/help.css" charset="ISO-8859-1" type="text/css">
<title>Tasks Menu</title>
</head>
<script type="text/javascript">
function setTitle()
{
top.document.title = document.title + " - " + parent.parent.WINDOW_TITLE;
}
</script>
<body onload="setTitle();">
<table bgcolor="#dcdcdc" border="0" cellspacing="0" width="100%">
<tr><td>
<p><img src="../support/schrodinger_logo.gif" align="left" alt=""
 border="0" hspace="5" vspace="5" /></p>
</td></tr>
<tr><td>
<h1 class="title"><span class="schrored">Tasks Menu</span></h1>
</td></tr>
</table>

<p>The <span class="GUI">Tasks</span> menu provides access to applications,
workflows, and tools organized by task rather than by application. To display
this menu, choose <span class="GUI">Applications &rarr; Tasks View</span>. The
<span class="GUI">Applications</span> and <span class="GUI">Workflows</span>
menus are replaced with the <span class="GUI">Tasks</span> menu. To change back,
choose <span class="GUI">Tasks &rarr; Applications View</span>.
</p>

<p>The <span class="GUI">Tasks</span> menu contains items that are on the 
<span class="GUI">Tools</span> menu, the <span class="GUI">Workflows</span>
menu, and the <span class="GUI">Applications</span> menu, but it does not
contain the full range of items on these menus.
</p>

<ul>
<li><p><span class="GUI">Ligand Preparation</span>&mdash;Prepare ligand-sized
molecules for use in modeling, with <a
href="../ligprep/ligprep.html">LigPrep</a>. Includes 2D&rarr;3D conversion,
adjustment or expansion of ionization states, tautomers, ring conformations,
stereochemistry, and minimization.
</p></li>

<li><p><span class="GUI">Ligand Properties</span>&mdash;Generate properties for
ligand molecules that can be used as descriptors in a QSAR or QSPR model. (See
<a href="../workflows/lsbd_panel.html">Ligand and Stucture-Based
Descriptors</a>.)
</p></li>

<li><p><span class="GUI">Library Design</span>&mdash;Prepare a library of
molecules from a core structure by addition of fragments at designated
attachment points (see <a
href="../combiglide/library_enumeration.html">Combinatorial Library
Enumeration</a>).
</p></li>

<li><p><span class="GUI">pKa Prediction</span>&mdash;Predict the
p<i>K</i><sub>a</sub> of ligand-sized molecules. Two methods are available:
<span class="GUI">Empirical</span>, which uses Hammett and Taft methodology (<a
href="../epik/epik_panel.html">Epik</a>), and <span class="GUI">Quantum
Mechanical</span>, which uses ab initio methods and continuum solvation, with an
empirical correction term (<a href="../jaguar/pka.html">Jaguar</a>).
</p></li>

<li><p><span class="GUI">Protein Preparation</span>&mdash;Prepare proteins for
use in modeling, with the 
<a href="../workflows/protein_preparation.html">Protein Preparation Wizard</a>.
</p></li>

<li><p><span class="GUI">Protein Analysis</span>&mdash;Perform various kinds of
protein analyses. The submenu offers the following choices:
<ul>
<li><span class="GUI">Binding Site Detection</span>&mdash;Find potential binding
sites on a protein, with <a href="../glide/sitemap_panel.html">SiteMap</a>.</li>
<li><span class="GUI">Protein Reports</span>&mdash;Display reports on protein
properties, such as geometric deviations from ideal values, G-factors, B-factors,
stereochemistry, planarity, missing atoms, in the 
<a href="../tools_menu/protein_reports_panel.html"><span class="GUI">Protein
Reports</span> panel</a>.</li>
<li><span class="GUI">Ramachandran Plot</span>&mdash;Display <a
href="../tools_menu/ramachandran_panel.html">Ramachandran plot</a> of protein
dihedrals.</li>
</ul>
</p></li>

<li><p><span class="GUI">Homology Modeling</span>&mdash;Predict protein
structure from a protein sequence using information from homologous proteins,
with <a href="../prime/info_layout.html">Prime Structure Prediction</a>.
</p></li>

<li><p><span class="GUI">Minimization</span>&mdash;Minimize the energy of a
structure with respect to geometry changes. The submenu offers the 
following choices:
<ul>
<li><p><span class="GUI">Force-Field</span>&mdash;Minimize the force-field
energy with <a href="../macromodel/energy_minimization.html">MacroModel</a>.
</p></li>
<li><p><span class="GUI">Semiempirical QM</span>&mdash;Minimize the
<a href="../nddo/semiempirical_nddo.html">semiempirical NDDO energy</a>.
</p></li>
<li><p><span class="GUI">Ab Initio QM</span>&mdash;Minimize the ab initio
quantum mechanical energy with <a href="../jaguar/optimizations.html">Jaguar</a>.
</p></li>
<li><p><span class="GUI">QM/MM</span>&mdash;Minimize the mixed
quantum-mechanical/molecular mechanics energy with <a
href="../qsite/qsite_panel.html">QSite</a>.
</p></li>
</ul>
</p></li>

<li><p><span class="GUI">Quantum Mechanics</span>&mdash;Run electronic structure
calculations with Jaguar.
</p></li>

<li><p><span class="GUI">Conformational Search</span>&mdash;Perform a
search for the conformations of one or more molecules. The submenu offers the
following choices:
<ul>
<li><span class="GUI">Bioactive Search</span>&mdash;Search for ligand
conformations that are likely to be bioactive, by varying only the torsions,
with <a href="../macromodel/confgen_standard.html">ConfGen</a>.</li>
<li><span class="GUI">Advanced Conformational Search</span>&mdash;Perform a
complete search for ligand conformations with extensive control over the search
parameters, using <a
href="../macromodel/conformational_search.html">MacroModel</a>.</li> 
<li><span class="GUI">Monte Carlo</span>&mdash;Search for low-energy
conformations of proteins, ligands, or protein-ligand complexes using a Monte
Carlo algorithm, with MCPRO<sup>+</sup>.</li>
<li><p><span class="GUI">Ligand-Receptor Complex</span>&mdash;Perform a
conformational search on a ligand-receptor complex and its components, with 
<a href="../macromodel/embracecsearch.html">MacroModel Embrace</a>.
</p></li>
<li><p><span class="GUI">Macrocycle Sampling</span>&mdash;Run a hybrid simulated
annealing/large-scale low-mode sampling job to sample the conformational space
of macrocycles.
</p></li>
</ul>
</p></li>

<li><p><span class="GUI">Protein Refinement</span>&mdash;Refine the structure of
a protein, by predicting loops and side chains, and minimizing the energy, using
<a href="../prime/refine_panel.html">Prime Refinement</a>.
</p></li>

<li><p><span class="GUI">Protein X-Ray Refinement</span>&mdash;Refine the X-ray
structure of a protein using PrimeX.
</p></li>

<li><p><span class="GUI">Docking</span>&mdash;Dock potential ligands to a
receptor . The submenu offers the following tasks:
<ul>
<li><span class="GUI">Grid Generation</span></li>
<li><span class="GUI">Glide Docking</span></li>
<li><span class="GUI">Interactive</span> (enumeration and docking, combiglide)</li>
<li><span class="GUI">Combinatorial Screening</span></li>
<li><span class="GUI">Covalent Docking</span></li>
<li><span class="GUI">Visualize XP Interactions</span></li>
<li><span class="GUI">Virtual Screening Workflow</span></li>
<li><span class="GUI">Induced-Fit Docking</span></li>
<li><span class="GUI">QM-Polarized Ligand Docking</span></li>
</ul>
</p></li>

<li><p><span class="GUI">Molecular Dynamics</span>&mdash;Run molecular dynamics
simulations on a periodic system that can include a protein, solvent, salt, and
a membrane, using Desmond. The submenu offers the following choices:
<ul>
<li><a href="../desmond/system_builder.html"><span class="GUI">System
Setup</span></a></li>
<li><a href="../desmond/molecular_dynamics_panel.html"><span
class="GUI">Molecular Dynamics</span></a></li>
<li><a href="../desmond/simulated_annealing_panel.html"><span
class="GUI">Simulated Annealing</span></a></li> 
<li><a href="../desmond/replica_exchange_panel.html"><span class="GUI">Replica
Exchange</span></a></li>
<li><span class="GUI">Replica Exchange Review</span></li>
<li><a href="../desmond/simulation_quality_analysis.html"><span
class="GUI">Simulation Quality Analysis</span></a></li>
<li><a href="../desmond/metadynamics.html"><span
class="GUI">Metadynamics</span></a></li>
<li><a href="../desmond/metadynamics_analysis.html"><span
class="GUI">Metadynamics Analysis</span></a></li>
</ul>
</p></li>

<li><p><span class="GUI">Binding Energy Estimation</span>&mdash;Estimate the
binding energy of a ligand to a protein. The submenu offers the following
choices: 
<ul>
<li><span class="GUI">MM-GBSA</span>&mdash;Calculate ligand binding energies
using <a href="../prime/prime_mmgbsa_panel.html">Prime MM-GBSA</a>.</li>
<li><span class="GUI">Linear Interaction Model</span>&mdash;Generate a model
for predicting the binding free energy of new ligands using a model based on
known binding energies and simulations, using 
<a href="../liaison/liaison_panel.html">Liaison</a>.</li>
<li><span class="GUI">Ligand Ring Atom Mutation</span>&mdash;Calculate relative
binding energies by FEP for mutation of ligand ring atoms, with Desmond or
MCPRO<sup>+</sup>.</li> 
<li><span class="GUI">Ligand Functional Group Mutation</span>&mdash;Calculate
relative binding energies by FEP for mutation of ligand functional groups, with
Desmond or MCPRO<sup>+</sup>.</li> 
<li><span class="GUI">Protein Residue Mutation</span>&mdash;Calculate
relative binding energies by FEP for mutation of protein residues, with
Desmond.</li> 
</ul>
</p></li>

<li><p><span class="GUI">Coordinate Scan</span>&mdash;Calculate the energy of a
structure as a function of one or more coordinates at defined values of the
coordinates. The submenu offers the following items:
<ul>
<li><span class="GUI">Force-Field</span>&mdash;Scan one or two coordinates with
<a href="../macromodel/coordinate_scan.html">MacroModel</a>.</li>
<li><span class="GUI">QM Relaxed</span>&mdash;Scan up to 5 coordinates with
<a href="../jaguar/coordscan.html">Jaguar</a>, optimizing the
remaining geometric degrees of freedom.</li>
<li><span class="GUI">QM Rigid</span>&mdash;Scan up to 5 coordinates with
<a href="../jaguar/coordscan.html">Jaguar</a>, keeping the
remaining geometric degrees of freedom fixed.</li>
</ul>
</p></li>

<li><p><span class="GUI">Core Hopping</span>&mdash;Substitute the core of a
ligand for other cores, retaining the side chains. The submenu offers the
following choices: 
<ul>
<li><span class="GUI">Receptor-Based</span>&mdash;Substitute cores in the presence
of a receptor, using Glide for docking of the new ligands.</li>
<li><span class="GUI">Ligand-Based</span>&mdash;Substitute cores using alignment
to find similar cores, based only on the ligand. </li>
<li><span class="GUI">Isosteric Matching</span>&mdash;Substitute cores using
shape overlap to find similar cores, based only on the ligand. </li>
</ul>
</p></li>

<li><p><span class="GUI">Shape Screening</span>&mdash;Screen a set of molecules
to find those that have the same shape as a query molecule (or shape), using
<a href="../phase/shape_screening.html">Phase Shape Screening</a>.
</p></li>

<li><p><span class="GUI">Pharmacophore Modeling</span>&mdash;Develop
pharmacophore models of ligands (hypotheses) and use them to screen a database
for matches; create and manage a database for screening. Uses Phase.
</p></li>

<li><p><span class="GUI">ADME</span>&mdash;Calculate ADME properties with 
<a href="../qikprop/qikprop.html">QikProp</a>.
</p></li>

<li><p><span class="GUI">P450 Site of Metabolism</span>&mdash;Predict the likely
sites of metabolism of ligands in various P450 sites.
</p></li>

<li><p><span class="GUI">QSAR</span>&mdash;Build and apply QSAR models. The
submenu offers the following choices:
<ul>
<li><span class="GUI">Atom-Based</span>&mdash;Build and apply a QSAR model based
on atom types, using <a href="../phase/atom_based_qsar.html">Phase</a>.</li>
<li><span class="GUI">Field-Based</span>&mdash;Build and apply a QSAR model
based on <a href="../phase/field_based_qsar.html">molecular fields</a> (like
CoMFA or CoMSIA).</li>
<li><span class="GUI">Property-Based</span>&mdash;Build and apply a QSAR model
based on molecular properties, with <a
href="../strike/strike_build.html">Strike</a>.</li>
</ul>
</p></li>
<li><p><span class="GUI">Align Structures</span>&mdash;Align structures to a
reference structure. This submenu duplicates items on the <span
class="GUI">Tools</span> menu:
<ul>
<li><a href="../tools_menu/super_panel.html"><span class="GUI">Superposition</span></a></li>
<li><a href="../tools_menu/flexible_ligand_alignment.html"><span class="GUI">Flexible Ligand Alignment</span></a></li> 
<li><a href="../tools_menu/protein_structure_alignment.html"><span class="GUI">Protein Structure Alignment</span></a></li>
<li><a href="../tools_menu/align_binding_sites.html"><span class="GUI">Align Binding Sites</span></a></li>
</ul>
</p></li>

<li><p><span class="GUI">KNIME</span>&mdash;Run KNIME workflows from Maestro. 
</p></li>

<li><p><span class="GUI">Monitor Jobs</span>&mdash;Open the 
<a href="../misc/monitor_panel.html"><span class="GUI">Monitor</span> panel</a>,
to monitor the progress of jobs.
</p></li>

</ul>



<hr />
<table width="100%">
<td><p class="small"><a href="../support/legal_notice.html" target="LegalNoticeWindow">Legal Notice</a></p></td>
<tr><td>
<table align="right">
<tr><td><p class="small">
File: tasks_menu/tasks_menu.html<br />
Last updated: 07 Aug 2012
</p></td></tr>
</table>
</td></tr>
</table>
</body>
</html>
