<!DOCTYPE html>
<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd" lang="en-US" xml:lang="en-US" data-mc-search-type="Stem" data-mc-help-system-file-name="Documentation.xml" data-mc-path-to-help-system="../" data-mc-target-type="WebHelp2" data-mc-runtime-file-type="Topic" data-mc-preload-images="false" data-mc-in-preview-mode="false" data-mc-toc-path="Help|Prime Help">
    <head>
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <meta charset="utf-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <link href="../Skins/Default/Stylesheets/Slideshow.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/TextEffects.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Topic.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Components/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Resources/Stylesheets/docs.css" rel="stylesheet" type="text/css" /><title>Prime Structure Prediction Menu Bar</title>
        <script src="../Resources/Scripts/custom.modernizr.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/jquery.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.config.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/foundation.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/plugins.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/MadCapAll.js" type="text/javascript">
        </script>
    </head>
    <body><a name="aanchor1491"></a>
        <MadCap:concept term="Help Topics" /><a name="aanchor1492"></a>
        <MadCap:concept term="Prime" />
        <h1>Prime Structure Prediction Menu Bar</h1>
        <p>The Prime Structure Prediction menu bar contains four menus, <span class="GUI">File</span>, <span class="GUI">Edit</span>, <span class="GUI">Display</span>, and <span class="GUI">Step</span>. Some of the options in these menus are also available as toolbar buttons.</p>
        <ul>
            <li value="1"><a href="#file">The File Menu</a>
            </li>
            <li value="2"><a href="#edit">The Edit Menu</a>
            </li>
            <li value="3"><a href="#display">The Display Menu</a>
            </li>
            <li value="4"><a href="#step">The Step Menu</a>
            </li>
            <li value="5"><a href="#links">Related Topics</a>
            </li>
        </ul>
        <h2><a name="file"></a>The File Menu</h2>
        <p>The <span class="GUI">File</span> menu contains commands for creating, naming, opening, and deleting Prime runs (see <a href="info_runs.html">Using Prime Runs</a>), exporting sequence and setting job options.  </p>
        <p>The run commands are:</p>
        <dl>
            <dt><span class="GUI">New</span>
            </dt>
            <dd>
                <p>Create a new run. You are asked for a name (all names must be unique.) The new run starts at the first step, Input Sequence.</p>
            </dd>
            <dt><span class="GUI">Save As</span>
            </dt>
            <dd>
                <p>Copies the current run under a new run name and switches to the new run. The original run is saved under the original name. If you open a new project and begin a workflow without explicitly giving the run a name, it will be given the default name of <span class="GUI">run1</span>.</p>
            </dd>
            <dt><span class="GUI">Rename</span>
            </dt>
            <dd>
                <p>Specify a new name for the current run. If the run name is not unique, a warning message appears.</p>
            </dd>
            <dt><span class="GUI">Delete</span>
            </dt>
            <dd>
                <p>Delete the current run. You will be asked for confirmation.</p>
            </dd>
            <dt><span class="GUI">Open</span>
            </dt>
            <dd>
                <p>Shows a list of options for switching to another run. The list includes the four most recently used runs. If there are more than four runs in the current project, click the <span class="GUI">More</span> option and select the run you want to open from the dialog box.</p>
            </dd>
        </dl>
        <p>The other options are:</p>
        <dl>
            <dt><span class="GUI">Export Sequences</span>
            </dt>
            <dd>
                <p>Click to open the <span class="GUI">Export Sequences</span> panel. All sequences (including secondary structure assignments and predictions) currently visible in the Sequence Viewer will be exported to a single file in native Maestro format. If HMMER/Pfam, ssa, or ssp data is currently hidden under the parent sequence it describes, only the parent sequence is exported.</p>
            </dd>
            <dt><span class="GUI">Job Options</span>
            </dt>
            <dd>
                <p>Open the <a href="job_options.html"><span class="GUI">Structure Prediction - Job Options</span> dialog box</a>, so that you can set options for running jobs for each stage of the workflow.</p>
            </dd>
        </dl>
        <h2><a name="edit"></a>The Edit Menu</h2>
        <p>The Edit menu contains commands for editing sequences and secondary structure predictions. These editing tools are also available as buttons on the Prime toolbar, and are described in the <a href="toolbar.html">Prime Toolbar</a> topic:</p>
        <p><span class="GUI">Select</span>
            <br /> <span class="GUI">Crop</span><br /> <span class="GUI">Edit SSP</span><br /> <span class="GUI">Set Template Regions</span><br /> <span class="GUI">Slide Freely</span><br /> <span class="GUI">Slide As Block</span><br /> <span class="GUI">Add Anchors</span><br /> <span class="GUI">Lock Gaps</span><br /> <span class="GUI">Unlock Gaps</span></p>
        <h2><a name="display"></a>The Display Menu</h2>
        <p>The <span class="GUI">Display</span> menu contains the following options governing the visualization of structures and sequences. Color schemes are described briefly here; use the <span class="GUI">Legend</span> command in this menu to view the assignment of each color and symbol used in each color scheme and sequence representation.</p>
        <ul>
            <li value="1"><a href="#display.1">Color Scheme</a>
            </li>
            <li value="2"><a href="#display.2">Font Size</a>
            </li>
            <li value="3"><a href="#display.3">View Structure</a>
            </li>
            <li value="4"><a href="#display.4">View SSA</a>
            </li>
            <li value="5"><a href="#display.4a">View Full PFAM Annotation</a>
            </li>
            <li value="6"><a href="#display.5">Legend</a>
            </li>
            <li value="7"><a href="#display.6">Preferences</a>
            </li>
        </ul>
        <p />
        <h3><a name="display.1"></a>Color Scheme</h3>
        <p>This command opens a menu of coloring options. Only one color scheme can be used at any given time. Each step in Prime has an appropriate default color scheme. You can choose a different scheme using this menu. When sequences are added to the Sequence Viewer, they are colored according to the scheme that is currently selected.</p>
        <p>When the structure associated with a sequence is displayed in the Workspace, it is generally colored using the same scheme as the sequence. However, protein structures in the Workspace are sometimes colored using other color schemes:</p>
        <ul>
            <li value="1">
                <p>When a PDB structure is imported, it is displayed in the Workspace using a Maestro error reporting color scheme in which complete recognized residues are gray, incomplete residues are red, unknown/nonstandard residues are orange, residues with multiple location indicators are green, and recognized residues with some unrecognized atoms are blue. See the <a href="../maestro_help/pdb.html#errors">PDB Conversion</a> topic section on Color-Based Error Reporting for more information.</p>
            </li>
            <li value="2">
                <p>When the Build Structure step is complete, a color scheme is applied to the structure in which blue atoms are those derived directly from the template, cyan atoms have backbone atoms from the template, and predicted side chains, and red atoms have been predicted or modeled.</p>
            </li>
        </ul>
        <p>The Prime color schemes are:</p>
        <ul>
            <li value="1"><a href="#display.1.1">None</a>
            </li>
            <li value="2"><a href="#display.1.2">Color by Sequence</a>
            </li>
            <li value="3"><a href="#display.1.3">Residue Type</a>
            </li>
            <li value="4"><a href="#display.1.4">Residue Property</a>
            </li>
            <li value="5"><a href="#display.1.5">Residue Matching</a>
            </li>
            <li value="6"><a href="#display.1.6">Residue Homology</a>
            </li>
            <li value="7"><a href="#display.1.7">Secondary Structure</a>
            </li>
            <li value="8"><a href="#display.1.8">Template ID</a>
            </li>
            <li value="9"><a href="#display.1.9">Proximity</a>
            </li>
        </ul>
        <dl>
            <dt><a name="display.1.1"></a>None</dt>
            <dd>
                <p>All sequences are colored standard gray.</p>
            </dd>
            <dt><a name="display.1.2"></a>Color by Sequence</dt>
            <dd>
                <p>The query sequence is colored standard gray. Each of the other sequences uses a different color.</p>
            </dd>
            <dt><a name="display.1.3"></a>Residue Type</dt>
            <dd>
                <p>Each amino acid sequence is colored according to the array of residue type colors.</p>
            </dd>
            <dt><a name="display.1.4"></a>Residue Property</dt>
            <dd>
                <p>Each amino acid sequence is colored according to the array of residue property colors (fewer colors than residue type).</p>
            </dd>
            <dt><a name="display.1.5"></a>Residue Matching</dt>
            <dd>
                <p>The query and templates are colored in a pairwise manner. Matching residues are colored according to their residue type, and non-matching residues are colored standard gray. If more than one template is chosen, the query will have more colored residues because the effect is cumulative. When only the query sequence is shown, it is colored standard gray for all residues.</p>
            </dd>
            <dt><a name="display.1.6"></a>Residue Homology</dt>
            <dd>
                <p>Aligned query and template residues that are identical (highly conserved) are colored red. Aligned query and template residues that are similar according to the BLOSUM62 scoring matrix (conserved) are colored yellow. Gray is used for residues with no homology.</p>
            </dd>
            <dt><a name="display.1.7"></a>Secondary Structure</dt>
            <dd>
                <p>The query sequence, which has no secondary structure assignment (SSA), is colored standard gray. All other amino acid sequences should have SSAs. Secondary structure assignments are displayed as a linear series of red cylinders (alpha helices), cyan arrows (beta strands), and gray line segments (loops/random coils.) This option applies the same color scheme to the amino acid sequences that have SSAs, and to the model in the Workspace.</p>
            </dd>
            <dt><a name="display.1.8"></a>Template ID</dt>
            <dd>
                <p>Only available when a model is being built. Before a structure is built, the query sequence is colored standard gray. Afterward, both the query sequence and the predicted structure are colored according to template ID for each region (using a cycle of assigned colors). Each template sequence is colored with its assigned color, but only in the region used to build the model. The unused portion of the template is colored dark gray.</p>
            </dd>
            <dt><a name="display.1.9"></a>Proximity</dt>
            <dd>
                <p>At any step in which the Sequence Viewer is displaying sequences that have associated structure (all steps except Input Sequence), color by proximity mode is available. When you select this color scheme, the current sequence colors do not change immediately. Instead, when the mouse is over a sequence that has structure, the cursor changes to indicate that proximity can be calculated. You can then select one or more contiguous residues in the sequence, and the system will color that sequence based on the proximity to the selected residues. No other sequences will have modified colors (even if they have structure too and are in proximity of the selected sequence). Proximity is calculated based on the shortest distance between any two atoms in two residues, not including the bonded C and N atoms in the residues next to the selected residues.</p>
                <p>To change the proximity cutoff, choose <span class="GUI">Display → Preferences</span> and change the number in the <span class="GUI">Proximity cutoff</span> text box.</p>
            </dd>
        </dl>
        <h3><a name="display.2"></a>Font Size</h3>
        <p>The font size of letters and numbers in the Sequence Viewer can be changed by choosing a point size from this submenu. The default size is 10 points.</p>
        <h3><a name="display.3"></a>View Template Structure</h3>
        <p>View the template structures in the Workspace. The options for showing structures in the Workspace include:</p>
        <ul>
            <li value="1"><span class="GUI">All</span>
            </li>
            <li value="2"><span class="GUI">None</span>
            </li>
            <li value="3"><span class="GUI">Predicted Only</span>
            </li>
        </ul>
        <h3><a name="display.4"></a>View SSA</h3>
        <p>Show secondary structure assignments in the Sequence Viewer. The default is not to show secondary structure assignments.</p>
        <h3><a name="display.4a"></a>View Full PFAM Annotation</h3>
        <p>Show the full PFAM annotation. By default only the highly conserved residues are shown in the annotation. In the full annotation, capital letters indicate highly conserved residues, lower case letters indicate a match to the HMM (Hidden Markov Model), + means the match is conservative, and a blank indicates that the residue does not match the HMM.</p>
        <h3><a name="display.5"></a>Legend</h3>
        <p>Opens the <span class="GUI">Sequence Viewer Legend</span> panel, which allows you to view the assignment of each color and symbol used in each color scheme and sequence representation. Click the <span class="GUI">Colors</span> tab and select a color scheme from the list, or click the <span class="GUI">Symbols</span> tab and select a type of sequence data, such as Pfam/HMMER or SSP. It is also available when you right-click on a sequence in the Sequence Viewer.</p>
        <h3><a name="display.6"></a>Preferences</h3>
        <p>Open the <span class="GUI">Preference</span> dialog box, which has a single text box, <span class="GUI">Proximity cutoff</span>. The value entered in this text box defines the proximity cutoff for the <span class="GUI">Proximity</span> color scheme. The default is 4.00&#160;Å.</p>
        <h2><a name="step"></a>The Step Menu</h2>
        <p>The <span class="GUI">Step</span> menu provides another way to navigate between steps in Prime. All the steps are listed on the menu. The current step is selected. To go to a different step, select that step on the menu. Steps that are not available are dimmed.</p>
        <p>You can also show or hide the Guide, by selecting <span class="GUI">Guide</span> on this menu. When the check box is selected, the Guide is displayed. Deselect the check box to hide the Guide. The Guide is hidden by default.</p>
        <h2><a name="links"></a>Related Topics</h2>
        <ul>
            <li value="1"><a href="structure_prediction.html" class="MCXref xref" xrefformat="{paratext}">Structure Prediction Panel</a>
            </li>
            <li value="2"><a href="info_runs.html" class="MCXref xref" xrefformat="{paratext}">Using Prime Runs</a>
            </li>
            <li value="3"><a href="job_options.html" class="MCXref xref" xrefformat="{paratext}">Structure Prediction - Job Options Dialog Box</a>
            </li>
            <li value="4"><a href="toolbar.html" class="MCXref xref" xrefformat="{paratext}">Prime Structure Prediction Toolbar</a>
            </li>
        </ul>
    </body>
</html>