<!DOCTYPE html>
<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd" lang="en-US" xml:lang="en-US" data-mc-search-type="Stem" data-mc-help-system-file-name="Documentation.xml" data-mc-path-to-help-system="../" data-mc-target-type="WebHelp2" data-mc-runtime-file-type="Topic" data-mc-preload-images="false" data-mc-in-preview-mode="false" data-mc-toc-path="Help|Prime Help">
    <head>
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <meta charset="utf-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <link href="../Skins/Default/Stylesheets/Slideshow.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/TextEffects.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Topic.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Skins/Default/Stylesheets/Components/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../Resources/Stylesheets/docs.css" rel="stylesheet" type="text/css" /><title>Prime Sequence Viewer</title>
        <script src="../Resources/Scripts/custom.modernizr.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/jquery.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/require.config.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/foundation.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/plugins.min.js" type="text/javascript">
        </script>
        <script src="../Resources/Scripts/MadCapAll.js" type="text/javascript">
        </script>
    </head>
    <body><a name="aanchor2959"></a>
        <MadCap:concept term="Help Topics" /><a name="aanchor2960"></a>
        <MadCap:concept term="Prime" />
        <h1>Prime Sequence Viewer</h1>
        <p>The Prime sequence viewer displays the sequences for the query and templates, and in some steps it shows secondary structure assignments (SSAs) and secondary structure predictions (SSPs). It is divided into two sections. The name of the sequence, SSA, or SSP is given in the left section, and the sequence, SSP, or SSA itself is displayed in the right section. The sequence viewer includes a shortcut menu (right click) that provides access to some functions.</p>
        <p>To resize the sequence viewer, drag the dotted line below the sequence viewer. If you cannot enlarge it any further, drag the bottom border of the panel to make the panel larger, and then you will be able to enlarge the sequence viewer some more. You can also drag the dotted line between the right and left sections of the sequence viewer to resize the sections.</p>
        <p>The sequence viewer can be configured to display a single line for each sequence that can be scrolled horizontally, or the sequences can be wrapped so that they scroll vertically. The configuration is controlled by the Wrap Sequences option on the sequence viewer shortcut menu.</p>
        <p>The sequence viewer also includes a "ruler", which marks the residue position relative to the leftmost residue displayed in the sequence viewer. This residue position is displayed in parentheses in the tooltip for each residue. The positions are arbitrary, and can change when you display residues in the viewer. They are intended to help you keep track of the sequence as you scroll the sequence viewer.</p>
        <p>You can select a single residue in a sequence by clicking on it, and you can use control-click to select or deselect individual residues. To select a range of residues, you can click the first and shift-click the last, or drag over the residues. The sequence in which the residues are selected is determined by where you start to drag. Likewise, you can invert the selection of multiple residues by holding down <span class="Keys">CTRL</span> when you drag.</p>
        <p>The shortcut menu items are described below.</p>
        <table border="0" cellpadding="5">
            <tr>
                <td style="padding-right: 10px;"><span class="GUI">Show All</span>
                    <br /><span class="GUI">Hide All</span>
                </td>
                <td>This command is present when you right-click on a sequence or an SSP. It can
be used to show or hide all the family and SSP information that is associated with a
sequence. It is equivalent to clicking the &amp;plus; or âˆ’ sign to the left of the
sequence name. 
</td>
            </tr>
            <tr>
                <td style="padding-right: 10px;"><span class="GUI">Run SSP</span>
                </td>
                <td>Run a secondary structure prediction for a sequence. This command is only
available in the <span class="GUI">Find Homologs</span>, <span class="GUI">Fold
Recognition</span> and <span class="GUI">Edit Alignment</span> steps. 
</td>
            </tr>
            <tr>
                <td style="padding-right: 10px;"><span class="GUI">View SSA</span>
                </td>
                <td>Show or hide the secondary structure assignment. This command is only
present when you right-click on an SSA or a sequence that has an SSA.
</td>
            </tr>
            <tr>
                <td style="padding-right: 10px;"><span class="GUI">View Full PFAM
Annotation</span>
                </td>
                <td>This option controls whether the PFAM annotation shows only the highly
conserved residues, or shows other family annotations as well. In the full
annotation, capital letters indicate highly conserved residues, lowercase
letters indicate a match to the HMM, + means the match is conservative, and a
blank indicates that the residue does not match the HMM. This command is only
present when you right-click on a Pfam annotation.</td>
            </tr>
            <tr>
                <td style="padding-right: 10px;"><span class="GUI">Revert</span>
                </td>
                <td>Revert the SSP to that generated by the SSP program. This is useful to
restore the original SSP after making changes. This command is only present for
SSPs, and is only available in the <span class="GUI">Edit Alignment</span> step.
</td>
            </tr>
            <tr>
                <td style="padding-right: 10px;"><span class="GUI">Delete</span>
                </td>
                <td>Delete the SSP that you right-clicked on. This command is only present for
SSPs, and is only available in the <span class="GUI">Edit Alignment</span> step 
</td>
            </tr>
            <tr>
                <td style="padding-right: 10px;"><span class="GUI">Export Single Sequence</span>
                </td>
                <td>Export an SSP to a file. A <a href="../maestro_help/fileselector.html">file
selector</a> opens, in which you can navigate to a location and name the file. 
</td>
            </tr>
            <tr>
                <td style="padding-right: 10px;"><span class="GUI">Save Image</span>
                </td>
                <td>Save an image of the sequence viewer. A <a href="../maestro_help/fileselector.html">file
selector</a> opens, in which you can select a format, from PNG, TIFF, and JPEG,
and save the image.
</td>
            </tr>
            <tr>
                <td style="padding-right: 10px;"><span class="GUI">Export Sequences</span>
                </td>
                <td>Export the sequences to a single FASTA file. A <a href="../maestro_help/fileselector.html">file
selector</a> opens, in which you can navigate to and name the file. You can also
export sequences by choosing <span class="GUI">File &gt; Export Sequences</span>. 
</td>
            </tr>
            <tr>
                <td style="padding-right: 10px;"><span class="GUI">View Structure</span>
                </td>
                <td>Display the structures for the sequence in the Workspace. You can choose from
<span class="GUI">All</span>, <span class="GUI">None</span>, or <span class="GUI">Predicted Only</span>. This command is only present when there is a 
structure for the sequence. The query sequence does not have a structure
associated with it until a structure has been built. The template sequences do
have structures, which can be displayed. 
</td>
            </tr>
            <tr>
                <td style="padding-right: 10px;"><span class="GUI">Wrap Sequences</span>
                </td>
                <td>When this option is selected, the sequences in the sequence viewer are
wrapped, so that you can scroll vertically through the sequences. When it is
deselected, the sequences are displayed on a single line, which you can scroll
horizontally. This command is always present on the shortcut menu.
</td>
            </tr>
            <tr>
                <td style="padding-right: 10px;"><span class="GUI">Legend</span>
                </td>
                <td>Open the <span class="GUI">Sequence Viewer Legend</span> panel, which
explains the color schemes and the symbols used in the sequence viewer. 
This command is always present on the shortcut menu.</td>
            </tr>
        </table>
        <h2><a name="links"></a>Related Topics</h2>
        <ul>
            <li value="1"><a href="structure_prediction.html" class="MCXref xref" xrefformat="{paratext}">Structure Prediction Panel</a>
            </li>
        </ul>
    </body>
</html>